{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "import glob"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "69140"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "raw_yaml_path_list=glob.glob(\"database/raw_manuscripts/*.yaml\")\n",
    "len(raw_yaml_path_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "#clean text\n",
    "import re\n",
    "def remove_symbols(text):\n",
    "    repl_list=[\n",
    "            \"\\\\\\\\\",\n",
    "            \"\\\\n\",\n",
    "            \"\\\\u2019\",\n",
    "            \"\\\\\",\n",
    "            \"\\n\",\n",
    "    ]\n",
    "    for r in repl_list:\n",
    "        text=text.replace(r,\"\")\n",
    "    text=re.sub(' +', ' ', text)\n",
    "    return text "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 69140/69140 [31:15<00:00, 36.87it/s]\n"
     ]
    }
   ],
   "source": [
    "import yaml\n",
    "from tqdm import tqdm\n",
    "raw_data_list=[]\n",
    "target_path=raw_yaml_path_list[0]\n",
    "for target_path in tqdm(raw_yaml_path_list):\n",
    "\n",
    "    with open(target_path,\"r\") as f:\n",
    "        txt=f.read()\n",
    "\n",
    "    commons_pos=txt.rfind(\"Creative Commons\")\n",
    "    cite_this_pos=txt.rfind(\"Cite this article\")\n",
    "    if commons_pos>0:\n",
    "        with open(target_path,\"r\") as f:\n",
    "            #yamlを辞書として読み込み\n",
    "            d=yaml.safe_load(f)\n",
    "            license_text=txt[commons_pos:cite_this_pos]\n",
    "            license_text=remove_symbols(license_text)\n",
    "            license_text=license_text[:license_text.find(\"To view a copy of this \")]\n",
    "            d[\"Lisence\"]=license_text\n",
    "            raw_data_list.append(d)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['database/processed/0926.bin']"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import joblib\n",
    "joblib.dump(raw_data_list,\"database/processed/0926.bin\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "65329"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(raw_data_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'Creative Commons (CC) BY or CC0 license, totalling nu2009=u2009224,427,218 sentences. All words were converted to lower case and punctuation was removed. Tokenization was performed using a custom WordPiece tokenizer with a vocabulary size of 52,000 words and word occurrence frequency of greater or equal to two. Finally, the pre-trained DistilBERT model was further finetuned using 1500 manually annotated free-text radiological reports (sourced from the non-COVID-19 cohort), with a batch size of four, for five epochs using Adam optimizer with a learning rate of 1u2009xD7u200910 and Binary Cross-Entropy loss with logits. The finetuned multi-label DistilBERT model was trained to output probabilities of the following labelsu2013Atelectasis, Pleural Calcification, Cardiomegaly, Consolidation, Effusion, Emphysema, External Medical Device, Fracture, Internal Medical Device, Interstitial Opacity, Metalwork, Nodule, Pleural Thickening, Other Abnormality, and No Findings. The labels were selected due to occurrence in at least 20 radiological reports from the training set. Model performance was validated on an independent dataset of nu2009=u2009 500 manually-labelled reports.\" - Deep neural networks - Quality control classifiers - \"Two quality control (QC) classifiers were trained to differentiate (1) chest versus non-chest body part (u201Cbody part classifieru201D) and (2) AP versus PA projection (u201Cprojection classifieru201D). The Non-COVID-19 cohort (nu2009 =u2009284,904 images, non-COVID-19 cohort) was selected for both classifiers. Images were randomised into training (80%), validation (10%), and testing (10%) sets using stratified splits. To avoid data leakage, we ensured that patient identifiers do not overlap between splits.\" - \"QC classifiers were built using the InceptionV3 architecture and initialised with ImageNet weights. Global Average Pooling and two dense layers comprised the classification head. Softmax activation was applied to the final dense layer. Models were trained on 16-bit DICOM files with 32 images per batch using Adam optimizer with a learning rate of 1u2009xD7u200910, whilst minimising the Categorical Cross-Entropy loss. Input images were resized to 299u2009xD7u2009 299 using bilinear interpolation without preserving the aspect ratio. During training, images were subject to random augmentations, which included brightness adjustments, angular rotation, and leftu2013right flipping. Training was terminated early if validation loss did not improve after ten consecutive epochs.\" - Ensemble of deep neural networks for COVID-19 prediction - \"All networks used in the CovIx ensemble utilise an InceptionV3 backbone and a classification head comprising of a Global Average Pooling layer, Dense layer (nu2009=u20091024 neurons), Dropout (dropout rate of 0.2) layer, and a final Classification layer (a Dense layer with number of neurons reflecting the number of desired classes). The InceptionV3 backbone produced the best performing-classifiers compared to VGG16, DenseNet, and ResNet both in our experiments as well as external studies. Network weights for all InceptionV3 backbones were obtained by training a multi-label classifier to identify one or more of the NLP labels extracted from free-text radiological reports in nu2009=u2009284,904 images from the non-COVID-19 cohort. (see Ground Truth Generation and Natural Language Processing).\" - \"CovIx is an ensemble of three models (Fig._) designed to capture micro- and macro-level features of the datasetu2014the high-resolution patch-wise classifier, low resolution image-wise classifier, and a high-resolution image-wise classifier. The final probability value produced by the ensemble is the weighted mean of the output probabilities produced by the Softmax output of each constituent model.\" - \"Figure 3Constituents of the CovIx ensemble. The low- and high-resolution image-wise classifiers were trained on frontal CXRs scaled to 299u2009xD7u2009299 and 764u2009xD7u2009764 pixels respectively. The classification head (H) contained two outputs u2014an NLP multi-label classifier output (L1-LN) and a COVID-19 classifier (Softmax). The NLP output consisted of a Dense layer with a neuron per NLP target class (classesu2009=u200910) followed by a Sigmoid activation function, while the COVID-19 classifier output likewise consisted of a Dense layer with four output neurons representing Normal, Abnormal, Pneumonia and COVIDu2009+u2009respectively followed by a Softmax output. The patch-wise classifier was built by scaling each image to 1500u2009xD7u20091500 resolution, extracting lung and heart masks, and taking 50 random patches cropped to image masks with a size of 299u2009xD7u2009299 as the network inputs. At inference stage, 50 random patches were acquired for each image and fed to the classifier to generate class probability values for Normal, Abnormal, Pneumonia, and COVIDu2009 +u2009classes.\" - \"The low- and high-resolution image-wise classifiers were trained on frontal CXRs scaled to 299u2009xD7u2009299 and 764u2009xD7u2009764 pixels respectively. When constructing the InceptionV3 networks with varying input shapes, the number of channels in each layer of the network remained constant, with only the dimensions of the intermediate feature maps being affected. The final feature map output prior to Global Average Pooling had a dimension of 8u2009xD7u20098 in the 299u2009xD7u2009299 model and a dimension of 22u2009xD7u200922 in the 764u2009xD7u2009746 model. The classification head contained two outputsu2014 an NLP multi-label classifier and a COVID-19 classifier. The NLP multi-label classifier was trained to identify one or more of the NLP labels extracted from free-text radiological reports, whilst the COVID-19 classifier assigned a probability value to Normal, Abnormal Non-Pneumonia, Non-COVID-19 Pneumonia, and COVIDu2009 +u2009classes. The network was trained end-to-end, such that the NLP label outputs were used as auxiliary targets for the COVID-19 classifier. This auxiliary training objective served to regularize the network training by encouraging the neural networks to extract a variety of useful features from all input images, whether the COVID class was present or not, making the networks more generalizable and more resilient.\" - \"The patch-wise classifier was built by scaling each image to 1500u2009xD7u2009 1500 resolution (the lowest DICOM resolution in our training set) and taking 50 random patches with a size of 299u2009xD7u2009299 as the network inputs (default InceptionV3 input size). To ensure that random patches represent meaningful information, the centres of each patch were randomly selected from segmented lung areas. Segmentation masks were obtained by training a UNet model with a ResNet-50 backbone and ImageNet weights on a collection of 2,000 manually labelled lung and cardiac fields. At inference stage, 50 random patches were acquired for each image and fed to the classifier to generate class probability values for Normal, Abnormal, Pneumonia, and COVIDu2009+u2009classes. The final prediction was taken as the average class probability across 50 patches.\" - \"All models were trained on 16-bit DICOM files with 64 images per batch using Adam optimizer. For the models with multiple outputs (low- and high- resolution image-wise classifiers), the final loss function was the sum of the categorical cross entropy loss applied to the Softmax output and the binary cross entropy loss of the output of the NLP layer. The 16-bit DICOM images were linearly rescaled to the range [u2212u20091, 1] before being fed into the models. A learning rate of 1u2009xD7u200910 was applied to the neural network backbone, whilst layers within the classification head were trained with a learning rate of 1u2009 xD7u200910 to minimise effects of the double descent phenomenon. Images were subject to random train-time augmentations, which included brightness adjustments, angular rotation, and leftu2013right flipping. Training was terminated early if validation loss did not improve after ten consecutive epochs.\" - Comparison with COVID-Net, DeepCOVID-XR, and consensus radiologist interpretations - \"COVID-Net and DeepCOVID-XR models were used to establish testing set performance reference standard. Briefly, COVID-Net, trained and validated on nu2009=u2009 13,975 CXRs (nu2009=u2009358 COVIDu2009+u2009images), utilises a bespoke convolutional network architecture to differentiate Normal, COVID-19, and non-COVID-19 Pneumonia CXRs, whilst DeepCOVID-XR, trained and validated on nu2009=u200914,788 CXRs (nu2009=u20094253 COVIDu2009+u2009images), is an ensemble of 24 neural networks that assigns each CXR a probability of displaying signs of COVID-19. Prior to inference, all images in the prospectively-collected testing set were converted to 8-bit PNG files, preserving original resolutions. Pre-trained model weights were obtained from respective GitHub repositories and class probabilities calculated using the author-supplied inference scripts.\" - \"One hundred images were selected from patients presenting to ED in NHS GG&C in June 2020. Images were acquired over a continuous time period, representing u201Creal-worldu201D incidence of COVID-19 presentation. Expert interpretations were independently provided by four radiologists with 6_months to 4 years (average 2.5_years) post Fellowship of the Royal College of Radiologists examination. Radiologists were blinded to any identifying patient information or clinical characteristics.\" - Statistical analysis - \"The predictive performance of the NLP and AI systems was assessed by using the area under the receiver operating characteristic (AU ROC) and precision-recall (AU PR) curves and 95% Confidence Intervals (CIs) were produced using 2000 bootstrap samples. Sensitivity, positive predictive value (PPV), and F1-score (a measure of accuracy, reflecting the harmonic mean of PPV and sensitivity, where 1 represents perfect PPV and sensitivity) were determined. Interobserver agreement was measured using Cohens Kappa. Model sensitivity and specificity were compared using McNemars test and AU ROCs were compared using DeLong test. A two-tailed p value of 0.05 was considered statistically significant.\"- - Cohort characteristics - \"All CXRs in our dataset (nu2009=u2009314,042) were obtained between February 2008 and September 2020 across 14 acute sites in NHS GG&C. Of the 314,042 images, nu2009=u20092,313 (0.74%) and nu2009=u2009253,141 (80%) had missing Body Part Examined (0018, 0015) and View Position (0018, 5101) DICOM attributes respectively. To extrapolate the missing attribute values, we trained two classifiers that determine whether an X-ray is a chest radiograph (body part classifier) and whether its projection is AP or PA (projection classifier). Both classifiers achieved AUROCu2009>u20090.99 on a held-out testing set and were used to inform our quality control procedure (see u201Cu201D).\" - \"Of the 29,138 images in the COVID-19 cohort, nu2009=u200911,123 images (38%) from 8,511 patients passed our inclusion and QC criteria (4407 females, average age of 66, range 16u2013105_years, see u201Cu201D). The training set consisted of nu2009=u20098239 images obtained from patients presenting across 14 acute sites in NHS GG&C with symptoms of COVID-19 between March and May, 2020. Of these 63% (nu2009=u20095190) were obtained in ED, whilst remaining were obtained in in-patient facilities. The testing set images were collected continuously in ED from symptomatic NHS GG&C patients presenting between June and September 2020 (Table ). The rate of positivity for COVID-19 among chest radiographs in the test set (249/3,289; 7.6%) was lower than in the training (1,650/8,239; 20%). The proportion of anteroposterior radiograms was congruent between training and testing sets (28%).\" - Table 1 Patient characteristics in the training and testing sets. - Construction of CovIx ensemble for COVID-19 diagnosis - \"CovIx is a neural network ensemble that aims to capture macro- and micro-level features of the disease. All ensemble constituents utilise an InceptionV3 backbone, pretrained on CXRs from the non-COVID-19 cohort (nu2009=u2009284,904 images). The pre-training task was a multi-label classification problem that aimed to assign a CXR with one or more of the 15 labelsu2014Atelectasis, Pleural Calcification, Cardiomegaly, Consolidation, Effusion, Emphysema, External Medical Device, Fracture, Internal Medical Device, Interstitial Opacity, Metalwork, Nodule, Pleural Thickening, Other Abnormality, and No Findings.\" - \"To automate label extraction from free-text radiological reports, we trained a bespoke DistilBERT model using nu2009=u20092,067,531 full text PubMed articles (see u201Cu201D). NLP model performance on an independent set of 500 reports across the 15 labels achieved micro-average AUROC of 0.94 (AUROCExternal Medical Deviceu2009=u20090.71 to AUROCAbnormal Otheru2009=u20091.0). NLP labels were subsequently assigned to all CXRs. Following multilabel pre-training, weights of the InceptionV3 model were transferred for further finetuning on the COVID-19 cohort.\" - \"CovIx ensemble is comprised of four components (Fig._) u2013 (1) lung segmentation network, (2) high resolution patch-wise classification network, (3) low resolution image-wise classifier, and (4) high resolution image-wise classifier. The lung segmentation model was trained and validated on nu2009=u20092000 manually labelled lung fields. The resulting masks were used to select centres of the 50 random patches for every CXR, ensuring that only relevant information is captured. We have systematically assessed patch-wise model AUROC on a validation set using 10, 25, 50, and 100 patches per image. AUROC metric increased proportionally to the number of patches, with 50 and 100 patches producing identical validation set performance. The final patch-wise model consisted of 50 random patches, representing a balance between required computational resources and model performance.\" - \"Low- and high-resolution networks utilised 299u2009xD7u2009299 and 764u2009 xD7u2009764 sized images respectively. The networks were trained to label each image with one or more of the 15 NLP labels extracted from free-text reports and subsequently use label probabilities to classify an image as Normal, Abnormal, Pneumonia, or COVID+. Final CovIx class probabilities were obtained by averaging outputs produced by constituent classifiers.\" - Model performance - \"CovIx performance was evaluated on a prospective continuously-collected testing set of nu2009=u20093289 images (nu2009=u2009249 COVID-19 positive, collected June u2013 September 2020) obtained from patients referred to the COVID-19 pathway following ED presentation in NHS GG&C. Performance of individual CovIx models is shown in Supplementary Fig. . The CovIx ensemble identified COVID-19 CXRs with AUROC and AUPR of 0.86 and 0.51 respectively (sensitivityu2009=u2009 0.55, PPVu2009=u20090.40, and F1-scoreu2009=u20090.47 [Fig._, Table ]). Concurrent model identification of Normal, Abnormal, and Pneumonia CXRs resulted in AUROCs of 0.89, 0.70, and 0.96 respectively (Fig._).\" - Figure 4Discriminative performance of Chest-Xray classification algorithms on prospectively collected testing set of 3289 images. Discriminative capacity of CovIx (blue), COVID-Net (red), and DeepCOVID-XR (black) is represented as Receiver Operating Characteristics (ROC) (A-D) or Precision-Recall (PR) (E-H) curves. 95% Confidence Intervals (CI), generated using 2000 bootstrap samples, are visualised as pale curves. - Table 2 Model performance comparison in identifying COVID-19 CXRs. - \"Impact of age on model performance was assessed by evaluating sensitivity, PPV, and F1-scores for every age quintile. The model achieved peak COVID-19 sensitivity (0.83), PPV (0.61), and F1-score (0.71) in the 49u201360 age group (2nd age quintile) (Fig._). Furthermore, CovIx demonstrated increased COVID-19 detection in AP views and Male patients, exemplified by increased sensitivities (0.63, 0.68), PPVs (0.47, 0.45), and F1-scores (0.54) (Fig._).\" - Figure 5Effect of patient age, sex, and view position on CovIx performance. Model performance, expressed as sensitivity, positive predictive value (PPV), and F1 Score is represented as lines, whilst class frequencies, expressed as proportion of total images in the testing set, are shown as bars. - \"To determine whether CovIx identifies COVID-19-specific features from CXRs, we applied the algorithm to nu2009=u20095000 randomly selected radiographs (nu2009 =u20092819 normal radiological reports) obtained from patients presenting to NHS GG&C ED between September 2009 and August 2019. CovIx labelled 156 images (3%) as having radiological signs indicative of COVID-19. Of the 156 images 80 (51%) had normal radiological reports, 15 (10%) exhibited basal consolidations, and two (1%) had laboratory-confirmed Pneumonia. Remainder exhibited a diverse range of radiological signs, including cardiomegaly, emphysema, and atelectasis.\" - \"Finally, we compared CovIx algorithm to state-of-the-art, by evaluating COVID-Net and DeepCOVID-XR algorithms on our continuously-collected testing set. CovIx achieved better performance, expressed through significantly greater (DeLong pu2009<u20090.05) AUROC and AUPR values compared to other algorithms (Fig._ ) as well as higher PPV and F1-scores (Table ).\" - Comparison with expert radiologists - \"CovIx predictions were compared to board-certified radiologist interpretations on the first 100 continuously-collected CXRs of patients presenting to ED in June 2020 (nu2009=u200917 COVID-19 positive). Average inter-reader agreement, expressed as Cohens Kappa, for Normal, Abnormal, Pneumonia, and COVID-19 CXRs was 0.68, 0.49, 0.43, and 0.60 respectively (Fig._A).\" - \"Figure 6Comparison with board-certified radiologist interpretations. (A) Heatmaps visualising inter-reader agreement for Normal, Abnormal, Pneumonia, and COVID-19 images. (B-E) Receiver Operating Characteristic (ROC) curves and (F-I) Precision-Recall (PR) curves demonstrating CovIx model performance on the first 100 images collected in June 2020 across EDs in NHS GG&C (nu2009=u200917 COVID-19 positive). Individual radiologist performance is visualised as u201Cu2009+u2009u201D. Grey lines indicate 95% Confidence Intervals (CI) calculated on 2000 bootstrap samples.\" - \"The overall multi-class accuracy of CovIx on this test set was 60% compared with the reference standard, while the accuracy of individual radiologists ranged from 55 to 69% and the accuracy of the consensus interpretation of all four radiologists was 66%. Differences in overall performance were not statistically significant between CovIx and consensus radiologists labels (McNamara s p valueu2009=u20090.48, Supplementary Fig. ).\" - \"At single-label level, CovIx performance was comparable to radiologists in Normal and Abnormal CXRs (McNamaras p valueu2009=u20090.82 and 0.53 respectively, Fig._B-D, F-H). However, CovIx exhibited statistically significant performance improvements in Pneumonia and COVID-19 classes (McNamaras p valueu2009 =u20090.02 and 0.04_respectively,_Fig._E,I), further exemplified through greater sensitivity, F1-score, and PPV (Table , Fig. ).\" - Table 3 Comparison of model performance with Radiologist labels.- - \"In this paper we present development and prospective evaluation of an AI algorithmu2014 CovIxu2014for screening of putative COVID-19 CXRs in symptomatic patients presenting to emergency department. The study population, aggregated across NHS GG&C, is representative of u201Creal-worldu201D patients presenting to ED between the peaks of the COVID-19 pandemic. On a continuously-collected testing set of nu2009 =u20093,289 images (nu2009=u2009249 COVID-19 positive), CovIx achieved AU ROC and AU PR of 0.86 and 0.51 respectively, outperforming state-of-the-art COVID-Net and DeepCOVID-XR models. Additionally, on a continuously-collected sample of 100 test images, CovIx performed favourably when compared to four board-certified radiologists, achieving statistically significant performance improvements for Pneumonia and COVID-19 identification.\" - \"Our work introduces several advantages. First, we use an ensemble approach that evaluates macro- and micro-level features of COVID-19 CXRs. The image-wise classifiers (macro-level) were pre-trained on nu2009=u2009284,904 images using ground truths derived from a state-of-the-art NLP model trained on 224,427,218 sentences from medical literature. To the best of our knowledge this represents the largest medical corpus in a language modelling task, providing high-quality annotations. Second, the patch-wise classifier (micro-level) enabled training on a relatively small training set, whilst still outperforming state-of-the-art models, such as DeepCOVID-XR. A similar approach, utilising 100 random patches during inference step, has been previously proposed. We demonstrate that training a model using 50 random patches obtained from CXR lung fields, combined with a simple image augmentation schedule, yields superior performance. Third, AI models for COVID-19 detection have focused either on a binary COVID versus non-COVID classification task or on differentiation of COVID-19 pneumonia from viral or bacterial pneumonias. Our algorithm introduces simultaneous detection of normality, COVID-19 pneumonia, viral or bacterial pneumonias, as well as non-pneumonia abnormalities. This approach makes it more versatile in diverse clinical environments such as the ED, where earlier diagnosis of bacterial pneumonia reduces mortality and length of stay. Finally, most AI studies have been carried out at the time of considerable load on the healthcare system, with over-represented prevalence of COVID-19. As such, it is unclear how well these algorithms perform when COVID-19 is not the dominant viral pneumonia.\" - In this work, we rely heavily on the InceptionV3 architecture, which produced better performance compared to VGG16, DenseNet, and ResNet both in our experiments as well as external studies. However, deep neural network models may suffer from over-fitting when there is a small number of training exemplars, whilst shallow architectures may achieve comparable results with shorter training times. Shallow architectures have already been explored in the context of COVID-19 screening and may provide a plausible alternative in cases where limited training data is available. - \"We demonstrate first evidence of AI performance in u201Creal-worldu201D settings on continuously collected CXRs in patients presenting to ED between the peaks of the pandemic. As such, our experiments reflect the changing prevalence of COVID-19 in the symptomatic ED population (20% Marchu2013May, 2020 vs. 8% Juneu2013 September 2020). The training and testing sets represent an imbalanced machine learning problem, whereby the prevalence of a positive class (COVID-19) is considerably lower than that of the negative class (Normal, Abnormal-Other, Non-COVID Pneumonia). When class imbalance exists, learners will typically over-classify the majority group due to its increased prior probability. To address this phenomenon, both undersampling the majority class and over-sampling the minority class have been proposed. Generating synthetic samples through linear interpolation between data samples belonging in the same minority class or weighing the training loss function have also been suggested. These techniques assume that the prevalence of the minority class is a known and stable quantity, however prevalence of SARS-CoV-2 is changing rapidly. To mitigate the impact of class imbalance in our models, we pre-trained every constituent of the CovIx Ensemble using a large collection of frontal CXRs (nu2009=u2009284,904) obtained from patients prior to emergence of COVID-19 (Non-COVID Cohort, Figs._, , see u201Cu201D). This approach has been demonstrated to improve model robustness against imbalance and shown to outperform techniques such as over-/under-sampling and Synthetic Minority Oversampling Technique (SMOTE).\" - Furthermore, evaluation of CovIx on 5000 CXRs collected between September 2009 and August 2019, where COVID-19 prevalence is expected to be 0%, the algorithm identified only 156 images with high likelihood of COVID-19, suggesting that the algorithm is highly specific (97%). We believe this sets realistic expectations of AI performance. - Errors made by our algorithm were explainable. Of the 226 images with negative RT-PCR findings classified as COVID-19 positive by CovIx (false positives), 196 (87%) demonstrated signs including co-occurrence of bilateral small pleural effusions and unilateral lower lobe consolidation. Although individually these findings are present in a minority of COVID-19 patients, presence of multiple abnormalities on a single CXR resulted in greater COVID-19 probability values. Similarly, of the 105 images with positive RT-PCR findings classified as non-COVID-19 (false negatives), only 23 (22%) had typical COVID-19 findings, such as multifocal ground glass opacity, linear opacities, and consolidation. - \"Due to variabilities in COVID-19 severity across our testing cohort, it s likely that false negative predictions reflect limitations of CXR imaging rather than the algorithm itself. For example, 56% of symptomatic COVID-19 patients can demonstrate normal chest imaging, especially early in their disease course. Additionally, many of the findings seen in COVID-19 imaging are non-specific and overlap with other viral pneumonias. Consequently, CXR imaging alone is not recommended for COVID-19 diagnosis, but should be used concomitantly with clinical assessment, blood tests, and RT-PCR. As such, our model, either on its own or in consort with other biomarkers/clinical findings, could play an important triage role in earlier identification of patients likely to have COVID-19, enabling improved flow and infection control.\" - \"A number of research and industrial groups have published deep learning-based studies and non-peer reviewed preprints. Although the studies report extremely high sensitivity and specificity of AI algorithms to detect COVID-19 on CXRs, most have been limited by small sample sizes or have relied on images from publicly available datasets of variable quality and label accuracy. Although larger open access COVID-19 datasets are becoming more prevalent, for example the COVIDx dataset comprising of 13,975 CXR images across 13,870 patient cases, the utility of these resources is uncertain. Indeed, aggregation of disease-specific CXR datasets to produce a meta-training set can often lead to overinflated performance metrics. Given that neural networks have propensity to learn features that are specific of the dataset more than the ones that are specific of the disease, resulting models generalise poorly to independent testing sets. We demonstrate this characteristic by assessing performance of the COVID-Net model our testing set. The model classified 98% of all images as COVID+u2009, resulting in poor PPV, AU PR, and AU ROC values (Fig._).\" - \"Murphy et al. present an evaluation of a commercial patch-based convolutional neural network, CAD4COVID-Xray, on a cohort of continuously acquired CXRs (nu2009 =u2009454) obtained in patients suspected of having COVID-19 pneumonia presenting to a single centre between March 4 and April 6, 2020. The network was first trained on a large collection of CXRs for tuberculosis detection and subsequently finetuned using publicly-available pneumonia dataset (nu2009=u200922,184 images) and internally-curated COVID-19 images (nu2009=u2009416). The AI system correctly classified chest radiographs as COVID-19 pneumonia with an area under the receiver operating characteristic curve of 0.81. By contrast, our system was trained on four times as many COVID-19 cases obtained across 14 different institutions. Furthermore, our testing set represents u201Creal-worldu201D incidence of COVID-19 positivity (249/2,889 images, 9%) among patients presenting with symptoms of COVID-19 to ED.\" - \"More recently, an ensemble of 24 neural networks, DeepCOVID-XR, has demonstrated high accuracy of COVID-19 detection (AUROCu2009=u20090.90 compared to RT-PCR reference standard) and compared favourably to consensus of five thoracic radiologists (AUROCu2009=u20090.95) on an independent testing set. The network was pre-trained on a large CXR dataset of over 100,000 images and finetuned on 14,788 frontal CXRs (4,253 COVID-19 positive) from 20 sites, producing a binary prediction of COVID-19 likelihood. Evaluation of DeepCOVID-XR on our testing set demonstrated considerable performance boost compared to the COVID-Net model (AUROCu2009 =u20090.65, AUPRu2009=u20090.13, Table ). Nevertheless, DeepCOVID-XR did not perform on par with the CovIx ensemble (AUPRDeepCOVID-XRu2009=u20090.13 vs. AUPRCovIxu2009=u20090.51). Given similar inclusion criteria (RT-PCR positivity during a clinical encounter), and study population characteristics (comparable age and gender profiles), its likely that technical differences account for discrepancies in DeepCOVID-XR performance. For example, DeepCOVID-XR training and testing sets contained more AP images (89% and 97% respectively), compared to only 28% in our study population. Patients undergoing AP examination are more likely to exhibit severe symptoms with increasingly discernible signs of COVID-19 infection. This is further supported by improved CovIx performance on AP projections (Fig._G-I). Previous studies also report that AP CXRs have shown an overall better inter-rater agreement for COVID-19 diagnosis compared to PA.\" - \"CovIx ensemble performed best in patients within the 49u201360 age group (2 age quintile) (Fig._A-C). Young age has previously been associated with increased likelihood of false negative findings on CXR in retrospective multi-institutional study, of 254 RT-PCR verified COVID-19 positive patients. Additionally, older patients are more likely to present with more severe symptoms and multiple lobe involvement than young and middle-age groups.\" - \"Notably, whilst model performance for Normal, Abnormal, and Pneumonia classes was independent of patient sex, CovIx demonstrated decreased performance in female patients, as exemplified by reduction in sensitivity, F1-Score, and PPV (Fig._D-F). Sex differences in COVID-19 severity and outcomes are well documented, with men exhibiting more severe symptomatology, increased likelihood of intubation, and greater chances of mortality. CT imaging has also demonstrated significantly greater severity scores in men with a trend toward more bilateral lung involvement. Additionally, breast tissue may project onto lung fields, thus increasing the density of the lung periphery and simulating ground-glass opacities. To the best of our knowledge this is the first report of sex-related accuracy differences in AI-guided COVID-19 diagnosis using CXR imaging.\" - \"Our study has several limitations. First, the inclusion criteria was broadened to ensure sufficient numbers of COVID-19 positive images in our training set. As high-quality COVID-19 CXRs become more readily available, its likely that model performance can be refined further by building bespoke classifiers for AP and PA projections as well as opportunities to address age- and sex-driven discrepancies in model performance. Second, the performance of our algorithm was compared to RT-PCR as a reference standard, which itself has limited sensitivity due to sampling error or viral mutation. Third, although we used a continuously collected testing set for model validation, we did not assess model performance in an independent institution. Therefore, the generalisability potential of our algorithm is unclear. Finally, CovIx is limited to only a single data type u2013 frontal CXRs. It is anticipated that inclusion of multimodal dataset in clinical decision support will further improve model accuracy, reliability, and interpretation. To support this area of research, we made the pre-trained CovIx models and inference scripts available to the research community ().\" - \"Overall, we present and evaluate a deep learning algorithm for detection of COVID-19 infection in symptomatic patients presenting to emergency department. The algorithm was trained on a large representative population and tested on continuously collected data in a u201Creal-worldu201D setting. CovIx has the potential to mitigate unnecessary exposure to COVID-19 in busy ED settings by serving as an automated tool to rapidly triage patients for further testing and/or isolation. Planned future studies include (1) incorporation of imaging data with readily-available point-of-care clinical data such as demographics and vital signs to further boost the performance, (2) evaluation of model generalisability in external institutions outside NHS GG&C, and (3) adoption of the algorithm for risk prediction of clinically meaningful outcomes in patients with confirmed COVID-19. By providing the CovIx code base as an open-source project, we hope investigators will further improve, fine-tune, and test the algorithm using clinical images from their own institutions.\"- - \"WHO. Rolling updates on coronavirus disease (COVID-19). (2020).Dofitas, B. L., Kalim, S. P., Toledo, C. B. & Richardus, J. H. Yaws in the Philippines: First reported cases since the 1970s. Infect. Dis. Poverty 9, 1. (2020)._ _ _ _ Guan, W. J. et al. Clinical characteristics of coronavirus disease 2019 in China. N. Engl. J. Med. 382, 1708u20131720. (2020)._ _ _ _ Wang, D. et al. Clinical characteristics of 138 hospitalized patients with 2019 novel coronavirus-infected pneumonia in Wuhan, China. JAMA 323, 1061u20131069. (2020)._ _ _ _ _ Huang, C. et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet 395, 497u2013 506. (2020)._ _ _ _ _ Wu, Z. & McGoogan, J. M. Characteristics of and Important Lessons From the Coronavirus Disease 2019 (COVID-19) Outbreak in China: Summary of a Report of 72314 Cases From the Chinese Center for Disease Control and Prevention. JAMA 323, 1239u20131242. (2020)._ _ _ _ Wang, W. et al. Detection of SARS-CoV-2 in different types of clinical specimens. JAMA 323, 1843u2013 1844. (2020)._ _ _ _ _ Eggleton, E. J. Simple, fast and affordable triaging pathway for COVID-19. Postgrad. Med. J. 97, 192u2013195. (2021)._ _ _ Long, C. et al. Diagnosis of the Coronavirus disease (COVID-19): rRT-PCR or CT?. Eur. J. Radiol. 126, 108961. (2020)._ _ _ _ Long, D. R. et al. Occurrence and timing of subsequent severe acute respiratory syndrome coronavirus 2 reverse-transcription polymerase chain reaction positivity among initially negative patients. Clin. Infect. Dis. 72, 323u2013326. (2021)._ _ _ _ Tang, Y. W., Schmitz, J. E., Persing, D. H. & Stratton, C. W. Laboratory diagnosis of COVID-19: Current issues and challenges. J. Clin. Microbiol. (2020)._ _ _ _ Dinnes, J. et al. Rapid, point-of-care antigen and molecular-based tests for diagnosis of SARS-CoV-2 infection. Cochrane Database Syst. Rev. 3, CD013705. (2021)._ _ _ Yang, W. et al. The role of imaging in 2019 novel coronavirus pneumonia (COVID-19). Eur. Radiol. 30, 4874u2013 4882. (2020)._ _ _ _ _ Cleverley, J., Piper, J. & Jones, M. M. The role of chest radiography in confirming covid-19 pneumonia. BMJ 370, m2426. (2020)._ _ _ Rubin, G. D. et al. The role of chest imaging in patient management during the COVID-19 pandemic: A multinational consensus statement from the Fleischner society. Radiology 296, 172u2013180. (2020)._ _ _ Tavare, A. N. et al. Managing high clinical suspicion COVID-19 inpatients with negative RT-PCR: A pragmatic and limited role for thoracic CT. Thorax 75, 537u2013538. (2020)._ _ _ Nair, A. et al. A British Society of Thoracic Imaging statement: Considerations in designing local imaging diagnostic algorithms for the COVID-19 pandemic. Clin Radiol 75, 329u2013334. (2020)._ _ _ _ _ Wong, H. Y. F. et al. Frequency and distribution of chest radiographic findings in patients positive for COVID-19. Radiology 296, E72u2013E78. (2020)._ _ _ van Ginneken, B. The potential of artificial intelligence to analyze chest radiographs for signs of COVID-19 pneumonia. Radiology (2020)._ _ _ _ Wang, L., Lin, Z. Q. & Wong, A. COVID-Net: a tailored deep convolutional neural network design for detection of COVID-19 cases from chest X-ray images. Sci. Rep. 10, 19549. (2020)._ _ _ _ _ _ Khan, A. I., Shah, J. L. & Bhat, M. M. CoroNet: A deep neural network for detection and diagnosis of COVID-19 from chest X-ray images. Comput. Methods Programs Biomed. 196, 105581. (2020)._ _ _ _ Zhang, R. et al. Diagnosis of Coronavirus Disease 2019 pneumonia by using chest radiography: Value of artificial intelligence. Radiology 298, E88u2013 E97. (2021)._ _ _ Jin, C. et al. Development and evaluation of an artificial intelligence system for COVID-19 diagnosis. Nat. Commun. 11, 5088. (2020)._ _ _ _ _ _ Wehbe, R. M. et al. DeepCOVID-XR: An artificial intelligence algorithm to detect COVID-19 on chest radiographs trained and tested on a large US clinical dataset. Radiology. (2020)._ _ _ Roberts, M. et al. Common pitfalls and recommendations for using machine learning to detect and prognosticate for COVID-19 using chest radiographs and CT scans. Nat. Mach. Intell. 3, 199u2013217. (2021)._ _ Yi, P. H., Kim, T. K. & Lin, C. T. Generalizability of deep learning tuberculosis classifier to COVID-19 chest radiographs: New tricks for an old algorithm?. J. Thorac. Imaging 35, W102u2013W104. (2020)._ _ _ Cohen, J. P., Hashir, M., Brooks, R. & Bertrand, H. On the limits of Cross-domain generalization in automated X-ray prediction. arXiv:2002.02497 (2020). .Maguolo, G. & Nanni, L. A Critic Evaluation of Methods for COVID-19 Automatic detection from X-ray images. arXiv:2004.12823 (2020). .Tartaglione, E., Barbano, C. A., Berzovini, C., Calandri, M. & Grangetto, M. Unveiling COVID-19 from chest X-ray with deep learning: A hurdles race with small data. arXiv:2004.05405 (2020). .Johnson, J. M. & Khoshgoftaar, T. M. Survey on deep learning with class imbalance. J. Big Data 6, 27. (2019)._ _ Charter for Safe Havens in Scotland: Handling Unconsented Data from National Health Service Patient Records to Support Research and Statistics. (2015).Sanh, V., Debut, L., Chaumond, J. & Wolf, T. DistilBERT, a distilled version of BERT: Smaller, faster, cheaper and lighter. arXiv:1910.01108 (2019). .Drozdov, I. et al. Supervised and unsupervised language modelling in chest X-ray radiological reports. PLoS ONE 15, e0229963. (2020)._ _ _ _ _ The PMC Open Access Subset, (2021).Schuster, M. & Nakajima, K. Japanese and Korean Voice Search. In International Conference on Acoustics, Speech and Signal Processing. 5149u2013 5152 (IEEE, 2012).Kingma, D. P. & Ba, J. Adam: A method for stochastic optimization. arXiv:1412.6980 (2014). .Szegedy, C., Vanhoucke, V., Ioffe, S., Shlens, J. & Wojna, Z. Rethinking the inception architecture for computer vision. arXiv:1512.00567 (2015). .Russakovsky, O. et al. ImageNet large scale visual recognition challenge. arXiv:1409.0575 (2014). .Taylor, A. G., Mielke, C. & Mongan, J. Automated detection of moderate and large pneumothorax on frontal chest X-rays using deep convolutional neural networks: A retrospective study. PLoS Med. 15, e1002697. (2018)._ _ _ _ Oh, Y., Park, S. & Ye, J. C. Deep learning COVID-19 features on CXR using limited training data sets. IEEE Trans. Med. Imaging 39, 2688u20132700. (2020)._ _ _ Ronneberger, O., Fischer, P. & Brox, T. U-Net: Convolutional networks for biomedical image segmentation. arXiv:1505.04597 (2015). .He, K., Zhang, X., Ren, S. & Sun, J. Deep residual learning for image recognition. arXiv:1512.03385 (2015). .Heckel, R. & Furkan Yilmaz, F. Early stopping in deep networks: Double Descent and how to eliminate it. arXiv:2007.10099 (2020). .Kim, S. & Lee, W. Does McNemar s test compare the sensitivities and specificities of two diagnostic tests?. Stat Methods Med. Res. 26, 142u2013154. (2017)._ _ _ _ DeLong, E. R., DeLong, D. M. & Clarke-Pearson, D. L. Comparing the areas under two or more correlated receiver operating characteristic curves: A nonparametric approach. Biometrics 44, 837u2013845 (1988)._ _ _ _ Rasmy, L., Xiang, Y., Xie, Z., Tao, C. & Zhi, D. Med-BERT: Pre-Trained contextualized embeddings on large-scale structured electronic health records for disease prediction. arXiv:2005.12833 (2020). .Murphy, K. et al. COVID-19 on chest radiographs: A multireader evaluation of an artificial intelligence system. Radiology 296, E166u2013E172. (2020)._ _ _ Umer, M., Ashraf, I., Ullah, S., Mehmood, A. & Choi, G. S. COVINet: A convolutional neural network approach for predicting COVID-19 from chest X-ray images. J. Ambient Intell. Humaniz Comput. (2021)._ _ _ _ Rhodes, A. et al. Surviving sepsis campaign: International guidelines for management of sepsis and septic shock: 2016. Intensive Care Med 43, 304u2013377. (2017)._ _ _ Bianchini, M. & Scarselli, F. On the complexity of neural network classifiers: A comparison between shallow and deep architectures. IEEE Trans. Neural. Netw. Learn Syst. 25, 1553u20131565. (2014)._ _ _ Bressem, K. K. et al. Comparing different deep learning architectures for classification of chest radiographs. Sci. Rep. 10, 13590. (2020)._ _ _ _ _ _ Das, D., Santosh, K. C. & Pal, U. Truncated inception net: COVID-19 outbreak screening using chest X-rays. Phys. Eng. Sci. Med. 43, 915u2013925. (2020)._ _ _ _ Mukherjee, H. et al. Shallow convolutional neural network for COVID-19 outbreak screening using chest X-rays. Cognit. Comput. (2021)._ _ _ _ Akkasi, A. & Moens, M. F. Causal relationship extraction from biomedical text using deep neural models: A comprehensive survey. J. Biomed. Inform. 119, 103820. (2021)._ _ _ Huang, C., Li, Y., Loy, C. C. & Tang, X. In 2016 IEEE Conference on Computer Vision and Pattern Recognition (CVPR). 5375u20135384.He, H. & Garcia, E. A. Learning from Imbalanced Data. IEEE Trans. Knowl. Data Eng. 21, 1263u20131284. (2009)._ _ Chawla, N. V., Bowyer, K. W., Hall, L. O. & Kegelmeyer, W. P. SMOTE: Synthetic Minority Over-Sampling Technique. arXiv:1106.1813 (2011). .Japkowicz, N. The Class Imbalance Problem: Significance and Strategies. In Proceedings of the 2000 International Conference on Artificial Intelligence. 111u2013117 (2000).Pouwels, K. B. et al. Community prevalence of SARS-CoV-2 in England from April to November, 2020: Results from the ONS Coronavirus Infection Survey. Lancet Public Health 6, e30u2013e38. (2021)._ _ _ Hendrycks, D., Lee, K. & Mazeika, M. Using Pre-Training Can Improve Model Robustness and Uncertainty. arXiv:1901.09960 (2019). .Lomoro, P. et al. COVID-19 pneumonia manifestations at the admission on chest ultrasound, radiographs, and CT: Single-center study and comprehensive radiologic literature review. Eur. J. Radiol. Open 7, 100231. (2020)._ _ _ _ Bai, H. X. et al. Performance of radiologists in differentiating COVID-19 from Non-COVID-19 viral pneumonia at chest CT. Radiology 296, E46u2013E54. (2020)._ _ _ Castiglioni, I. et al. Machine learning applied on chest x-ray can aid in the diagnosis of COVID-19: A first experience from Lombardy, Italy. Eur. Radiol. Exp. 5, 7. (2021)._ _ _ _ Cohen, J. P., Morrison, P. & Dao, L. COVID-19 Image Data Collection. arXiv:2003.11597 (2020). .Pan, I., Cadrin-Chenevert, A. & Cheng, P. M. Tackling the radiological society of North America pneumonia detection challenge. AJR Am. J. Roentgenol. 213, 568u2013574. (2019)._ _ _ Wang, X. et al. ChestX-ray8: Hospital-Scale Chest X-ray Database and Benchmarks on Weakly-Supervised Classification and Localization of Common Thorax Diseases. arXiv:1705.02315 (2017). .Santosh, K. C. & Ghosh, S. Covid-19 imaging tools: How big data is big?. J. Med. Syst. 45, 71. (2021)._ _ _ _ _ Balbi, M. et al. Chest X-ray for predicting mortality and the need for ventilatory support in COVID-19 patients presenting to the emergency department. Eur Radiol. (2020)._ _ _ _ Stephanie, S. et al. Determinants of chest X-ray sensitivity for COVID-19: A multi-institutional study in the United States. Radiol Cardiothorac Imaging 2, e200337. (2020)._ _ _ _ Liu, K., Chen, Y., Lin, R. & Han, K. Clinical features of COVID-19 in elderly patients: A comparison with young and middle-aged patients. J Infect 80, e14u2013e18. (2020)._ _ _ _ _ Iaccarino, G. et al. Gender differences in predictors of intensive care units admission among COVID-19 patients: The results of the SARS-RAS study of the Italian Society of Hypertension. PLoS ONE 15, e0237297. (2020)._ _ _ _ _ Abate, B. B., Kassie, A. M., Kassaw, M. W., Aragie, T. G. & Masresha, S. A. Sex difference in coronavirus disease (COVID-19): A systematic review and meta-analysis. BMJ Open 10, e040129. (2020)._ _ _ _ Dehingia, N. & Raj, A. Sex differences in COVID-19 case fatality: SDo we know enough?. Lancet Glob Health 9, e14u2013 e15. (2021)._ _ _ Dangis, A. et al. Impact of gender on extent of lung injury in COVID-19. Clin. Radiol. 75, 554u2013556. (2020)._ _ _ _ _ Smith, D. L., Grenier, J. P., Batte, C. & Spieler, B. A characteristic chest radiographic pattern in the setting of the COVID-19 pandemic. Radiol. Cardiothorac. Imaging 2, e200280. (2020)._ _ _ _ Watson, J., Whiting, P. F. & Brush, J. E. Interpreting a covid-19 test result. BMJ 369, m1808. (2020)._ _ _ Santosh, K. C. AI-driven tools for coronavirus outbreak: Need of active learning and cross-population train/test models on multitudinal/multimodal data. J. Med. Syst. 44, 93. (2020)._ _ _ _ _ \"- - \"We thank NHS GG&C SafeHaven for data extraction and James Blackwood and Dr. Charlie Mayor for help with project coordination. We are grateful to Scan Computers and NVIDIA for providing access to a DGX-1 workstation. We would also like acknowledge assistance of Canon Medical Research Europe Limited in providing the SHAIP tool, assisting with the deidentification of data and the provision of a secure machine learning workspace. This work is supported by Bering Limited and the Industrial Centre for AI Research in Digital diagnostics (iCAIRD) which is funded by the Data to Early Diagnosis and Precision Medicine strand of the government s Industrial Strategy Challenge Fund, managed and delivered by Innovate UK on behalf of UK Research and Innovation (UKRI) [Project number 104690]. Views expressed are those of the authors and not necessarily those of Bering, the iCAIRD Consortium members, the NHS, Innovate UK or UKRI. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. ID and BS are employees of Bering Limited. The funder provided support in the form of salaries for authors ID and BS, but did not have any additional role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript. The specific roles of these authors are articulated in the u2018author contributions section.\"- - Authors and Affiliations - \"Bering Limited, London, UKIgnat Drozdov_&_Benjamin SzubertNHS Greater Glasgow and Clyde, Glasgow, UKElaina Reda,_Peter Makary,_Daniel Forbes,_Sau Lee Chang,_ Abinaya Ezhil,_Srikanth Puttagunta,_Mark Hall,_Chris Carlin_&_David J. Lowe\" - \"AuthorsIgnat DrozdovYou can also search for this author in _Benjamin SzubertYou can also search for this author in _Elaina RedaYou can also search for this author in _Peter MakaryYou can also search for this author in _Daniel ForbesYou can also search for this author in _Sau Lee ChangYou can also search for this author in _Abinaya EzhilYou can also search for this author in _Srikanth PuttaguntaYou can also search for this author in _Mark HallYou can also search for this author in _Chris CarlinYou can also search for this author in _David J. LoweYou can also search for this author in _\" - Contributions - I.D. and D.L. developed study design and wrote the main manuscript text. B.S and I.D. performed experiments and prepared manuscript figures. E.R. and P.M. generated lung segmentation ground truths and D.F. developed free text annotation pipelines. S.L.C., A.E., S.P., and M.H. and C.C. assisted with clinical model interpretation. S.L.C., A.E., S.P., and M.H. generated expert image annotations. All authors reviewed the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - ID and BS are employees of Bering Limited. The funder provided support in the form of salaries for authors ID and BS, but did not have any additional role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript. Remaining authors have no competing interests as defined by Nature Research, or other interests that might be perceived to influence the results and/or discussion reported in this paper. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC BY) licence (where permitted by UKRI, u2018Open Government Licence or u2018Creative Commons Attribution No-derivatives (CC BY-ND) licence may be stated instead) to any Author Accepted Manuscript version arising from this submission.\"- - Author notes - \\'These authors contributed equally: Xudong Pei, Liqi Zhou, Chen Huang.\\' - Authors and Affiliations - \"National Laboratory of Solid State Microstructures, Jiangsu Key Laboratory of Artificial Functional Materials, College of Engineering and Applied Sciences and Collaborative Innovation Center of Advanced Microstructures, Nanjing University, Nanjing, ChinaXudong Pei,_Liqi Zhou,_Yiming Hu_&_Peng WangDepartment of Physics, University of Warwick, Coventry, UKLiqi Zhou_&_Peng WangThe Rosalind Franklin Institute, Harwell Science and Innovation Campus, Didcot, UKChen Huang,_ Judy S. Kim,_Emanuela Liberti_&_Angus I. KirklandDivision of Structural Biology, Welcome Trust Centre for Human Genetics, University of Oxford, Oxford, UKMark Boyce,_Peijun Zhang_&_David I. StuartDepartment of Materials, University of Oxford, Oxford, UKJudy S. Kim,_Peter D. Nellist_&_Angus I. KirklandJEOL Ltd, Tokyo, JapanTakeo SasakiDiamond Light Source, Harwell Science and Innovation Campus, Didcot, UKPeijun Zhang,_David I. Stuart_&_Angus I. Kirkland\" - \"AuthorsXudong PeiYou can also search for this author in _Liqi ZhouYou can also search for this author in _Chen HuangYou can also search for this author in _Mark BoyceYou can also search for this author in _Judy S. KimYou can also search for this author in _Emanuela LibertiYou can also search for this author in _Yiming HuYou can also search for this author in _Takeo SasakiYou can also search for this author in _Peter D. NellistYou can also search for this author in _Peijun ZhangYou can also search for this author in _David I. StuartYou can also search for this author in _Angus I. KirklandYou can also search for this author in _Peng WangYou can also search for this author in _\" - Contributions - P.W. and A.I.K. conceived the overall project. X.D.P. and L.Q.Z. carried out the ptychographic reconstructions, multi-band 3D SPA analysis and relevant simulations supervised by P.W. P.W., L.Q.Z., C.H., J.S.K., and E.L. acquired the ptychographical data. T.S. provided technical assistance with the data acquisition. M.B., D.I.S., and P.Z. prepared the various cryo-samples and provided cryo-EM biological structural analysis. X.D.P., L.Q.Z., C.H., J.S.K., E.L., Y.M.H., P.D.N., A.I.K., and P.W. discussed ptychographical data analysis. P.W., X.D.P., L.Q.Z., J.S.K., and A.I.K. wrote the paper. All authors discussed the results and commented on the manuscript. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks Richard Leapman and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC BY) licence to any Author Accepted Manuscript version arising from this submission.\"- - Authors and Affiliations - \"Yusuf Hamied Department of Chemistry, University of Cambridge, Lensfield Road, Cambridge, UKDerya Emin,_Yu P. Zhang,_Evgeniia Lobanova,_Alyssa Miller,_ Zengjie Xia,_Helen Dakin,_Dimitrios I. Sideris,_Jeff Y. L. Lam,_Rohan T. Ranasinghe,_ Suman De,_Tuomas P. J. Knowles,_Michele Vendruscolo,_Francesco S. Ruggeri_ &_David KlenermanUK Dementia Research Institute, University of Cambridge, Cambridge, UKDerya Emin,_Yu P. Zhang,_Evgeniia Lobanova,_Zengjie Xia,_Helen Dakin,_ Jeff Y. L. Lam_&_David KlenermanLaboratory of Organic Chemistry, Wageningen University and Research, Wageningen, NetherlandsXuecong Li_&_Francesco S. RuggeriPhysical Chemistry and Soft Matter, Wageningen University and Research, Wageningen, NetherlandsXuecong Li_&_Francesco S. RuggeriJohn van Geest Centre for Brain Repair, Department of Clinical Neurosciences, University of Cambridge, Cambridge, UKAntonina Kouli_&_Caroline H. Williams-GrayMolecular Imaging Chemistry Laboratory, Wolfson Brain Imaging Centre, Department of Clinical Neurosciences, University of Cambridge, Cambridge, UKYanyan Zhao_&_Franklin I. AigbirhioSheffield Institute for Translational Neuroscience, University of Sheffield, Sheffield, UKSuman De\" - \"AuthorsDerya EminYou can also search for this author in _Yu P. ZhangYou can also search for this author in _Evgeniia LobanovaYou can also search for this author in _Alyssa MillerYou can also search for this author in _Xuecong LiYou can also search for this author in _Zengjie XiaYou can also search for this author in _Helen DakinYou can also search for this author in _Dimitrios I. SiderisYou can also search for this author in _Jeff Y. L. LamYou can also search for this author in _Rohan T. RanasingheYou can also search for this author in _Antonina KouliYou can also search for this author in _Yanyan ZhaoYou can also search for this author in _Suman DeYou can also search for this author in _Tuomas P. J. KnowlesYou can also search for this author in _Michele VendruscoloYou can also search for this author in _Francesco S. RuggeriYou can also search for this author in _Franklin I. AigbirhioYou can also search for this author in _Caroline H. Williams-GrayYou can also search for this author in _David KlenermanYou can also search for this author in _\" - Contributions - D.E., Y.P.Z., E.L., A.M., X.L., Z.X., H.D., D.I.S., J.Y.L.L., R.T.R, A.K., Y.Z., F.S.R. and S.D. performed and analysed the experiments. T.P.J.K., M.V., F.I.A., C.H.W.-G. and D.K. supervised the project. D.E. and D.K. conceived the idea, designed the study and wrote the manuscript. All authors discussed the results and proofread the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks the other anonymous reviewer(s) for their contribution to the peer review of this work. Peer review reports are available. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC BY) licence to any Author Accepted Manuscript version arising from this submission.- - Author notes - \\'These authors contributed equally: Brian R. Walker, Ruth Andrew.\\' - Authors and Affiliations - \"University/British Heart Foundation Centre for Cardiovascular Science, University of Edinburgh, Edinburgh, EH16 4TJ, Scotland, UKGregorio Naredo-Gonzalez,_Rita Upreti,_Maurits A. Jansen,_Scott Semple,_Brian R. Walker_&_Ruth AndrewEdinburgh Imaging, Queens Medical Research Institute, 47 Little France Crescent, University of Edinburgh, Edinburgh, EH16 4TJ, Scotland, UKMaurits A. Jansen,_ Scott Semple_&_Ian MarshallDepartment of Natural Sciences, Faculty of Science and Engineering, Manchester Metropolitan University, Chester Street, Manchester, M1 5GD, UKOliver B. SutcliffeCentre for Clinical Brain Sciences, Chancellor s Building, 49 Little France Crescent, University of Edinburgh, Edinburgh, EH16 4SB, Scotland, UKIan MarshallInstitute of Translational and Clinical Research, Newcastle University, Newcastle upon Tyne, NE1 3BZ, UKBrian R. Walker\" - \"AuthorsGregorio Naredo-GonzalezYou can also search for this author in _Rita UpretiYou can also search for this author in _Maurits A. JansenYou can also search for this author in _Scott SempleYou can also search for this author in _Oliver B. SutcliffeYou can also search for this author in _Ian MarshallYou can also search for this author in _Brian R. WalkerYou can also search for this author in _Ruth AndrewYou can also search for this author in _\" - Contributions - \"G.N.G.: Project administration, Methodology, Investigation, Data curation, Formal analysis, Writingu2014original draft, review and editing; R.U.: Methodology, Investigation, Data curation, Formal analysis, Writingu2014review and editing. M.J.: Methodology, Investigation, Data curation, Formal analysis, Writingu2014 review and editing; S.S.: Methodology, Investigation, Formal analysis, Data curation, Writingu2014review and editing; O.S.: Methodology, Investigation, Formal analysis, Writingu2014review and editing; I.M.: Conceptualization, Funding acquisition, Methodology, Formal analysis, Writingu2014review and editing; B.W.: Conceptualization, Funding acquisition, Project administration, Formal analysis, Writingu2014 review and editing. R.A.: Conceptualization, Funding acquisition, Project administration, Formal analysis Writingu2014review and editing.\" - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - \"BRW is a holder of a relevant patent for use of 11u03B2-hydroxysteroid dehydrogenase 1 inhibitors. All other authors do not have any competing interests to declare.\" - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC BY) licence to any Author Accepted Manuscript version arising from this submission.- - Author notes - \\'These authors contributed equally: Haidong Wang, Daniel Orejon.\\' - Authors and Affiliations - \"Department of Engineering Mechanics, Tsinghua University, Beijing, 100084, ChinaHaidong Wang,_Dongxing Song_&_Xing ZhangSchool of Engineering, Institute for Multiscale Thermofluids, University of Edinburgh, Edinburgh, EH9 3FB, Scotland, UKDaniel Orejon,_Glen McHale,_Yasuyuki Takata_&_Khellil SefianeInternational Institute for Carbon-Neutral Energy Research (WPI u2013 I2CNER), Kyushu University, Fukuoka, 819-0395, JapanDaniel Orejon,_Yasuyuki Takata_&_Khellil SefianeDepartment of Mechanical Engineering, Kyushu University, Fukuoka, 819-0395, JapanHiroshi Takamatsu_&_Yasuyuki Takata\" - \"AuthorsHaidong WangYou can also search for this author in _Daniel OrejonYou can also search for this author in _Dongxing SongYou can also search for this author in _Xing ZhangYou can also search for this author in _Glen McHaleYou can also search for this author in _Hiroshi TakamatsuYou can also search for this author in _Yasuyuki TakataYou can also search for this author in _Khellil SefianeYou can also search for this author in _\" - Contributions - K.S. conceived the idea, K.S., H.W. and D.O. developed the methodology; K.S., G.M., H.W. and D.O. carried out the investigation and analysis; H.W. and D.O. carried out the experimental observations and visualisation; K.S., Y.T., H.T., X.Z. and H.W. contributed with funding acquisition; K.S., was in charge of project administration; the project was supervised by K.S., Y.T., H.T. and X.Z; the original draft was completed by K.S., G.M., H.W. and D.O.; and all the authors H.W., D.O., D.S., X.Z., G.M., H.T., Y.T. and K.S. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \"Communications Materials thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editors: Aldo Isidori and John Plummer._ are available.\" - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC BY) licence to any Author Accepted Manuscript version arising.\"- - Author notes - \\'These authors contributed equally: Astrid H. Berge, Suzi M. Pugh.\\' - Authors and Affiliations - \"Department of Chemistry, University of Cambridge, Lensfield Road, Cambridge, CB2 1EW, UKAstrid H. Berge,_Suzi M. Pugh,_Marion I. M. Short_&_Alexander C. ForseCentre for Catalysis Research and Innovation (CCRI), Department of Chemistry and Biomolecular Sciences, University of Ottawa, Ottawa, Ontario, K1N 6N5, CanadaChanjot Kaur_&_Abdelhamid SayariDepartment of Materials Science and Metallurgy, University of Cambridge, 27 Charles Babbage Road, Cambridge, CB3 0FS, UKZiheng Lu_&_ Chris J. PickardComputational Science Research Center, Korea Institute of Science and Technology (KIST), Seoul, 02792, Republic of KoreaJung-Hoon LeeAdvanced Institute for Materials Research, Tohoku University, Aoba, Sendai, 980-8577, JapanChris J. Pickard\" - \"AuthorsAstrid H. BergeYou can also search for this author in _Suzi M. PughYou can also search for this author in _Marion I. M. ShortYou can also search for this author in _Chanjot KaurYou can also search for this author in _Ziheng LuYou can also search for this author in _Jung-Hoon LeeYou can also search for this author in _Chris J. PickardYou can also search for this author in _Abdelhamid SayariYou can also search for this author in _Alexander C. ForseYou can also search for this author in _\" - Contributions - A.H.B. and S.M.P. contributed equally to this work. A.H.B., S.M.P., Z.L., J.-H.L., C.J.P. and A.C.F. carried out DFT calculations and analysis. M.I.M.S., A.C.F. and S.M.P. carried out MOF synthesis and characterisation. C.K. and A.S. carried out amine-grafted silica synthesis and characterisation. S.M.P. carried out solid-state NMR measurements, and S.M.P., A.H.B., and A.C.F. analysed the NMR data. A.H.B. and S.M.P. wrote the manuscript with contributions from all the coauthors. A.C.F. designed the study. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \"Nature Communications thanks Niklas Hedin and the other, anonymous, reviewer(s) for their contribution to the peer review of this work._ are available.\" - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC BY) licence to any Author Accepted Manuscript version arising.\"- - Authors and Affiliations - \"Department of Physics and Astronomy, University of Exeter, Exeter, EX4 4QL, Devon, UKCameron P. Gallagher,_Ian R. Hooper,_J. Roy Sambles_&_Alastair P. HibbinsQinetiQ Ltd, Cody Technology Park, Ively Rd, Farnborough, GU14 0LX, UKJoshua K. Hamilton_ &_Christopher R. LawrencePepsiCo, Leicester, LE4 1ET, UKJohn Bows\" - \"AuthorsCameron P. GallagherYou can also search for this author in _Joshua K. HamiltonYou can also search for this author in _Ian R. HooperYou can also search for this author in _J. Roy SamblesYou can also search for this author in _Alastair P. HibbinsYou can also search for this author in _Christopher R. LawrenceYou can also search for this author in _John BowsYou can also search for this author in _\" - Contributions - C.P.G, J.B, and C.R.L. conceived the main idea of the project. J.K.H. and I.R.H produced the finite element models. J.R.S and A.P.H contributed in discussing the results and guiding the computational work. C.P.G fabricated the measured samples. J.K.H produced the experimental data and wrote the draft manuscript and all authors commented and provided help in building the final version of the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC BY) licence to any Author Accepted Manuscript version arising.- - Author notes - \\'These authors contributed equally: Gan Huang, Jingyuan Xu.\\' - Authors and Affiliations - \"Clean Energy Processes (CEP) Laboratory, Department of Chemical Engineering, Imperial College London, London, UKGan Huang,_Jingyuan Xu_&_Christos N. Markides\" - \"AuthorsGan HuangYou can also search for this author in _Jingyuan XuYou can also search for this author in _Christos N. MarkidesYou can also search for this author in _\" - Contributions - G.H. and C.N.M. developed the concept. G.H. and J.X. conducted the experiments. J.X. and G.H. prepared the models. G.H., C.N.M. and J.X. wrote the paper. C.N.M. directed the project. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks Chibuisi Okorieimoh and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC BY) licence to any Author Accepted Manuscript version arising.- - Authors and Affiliations - \"Electrical Engineering Division, Engineering Department, Cambridge University, Cambridge, CB3 0FA, UKN. C. A. van Fraassen,_K. M. Niang_&_A. J. FlewittDepartment of Chemistry, University of Bath, Bath, BA2 7AX, UKJ. D. Parish_&_A. L. Johnson\" - \"AuthorsN. C. A. van FraassenYou can also search for this author in _K. M. NiangYou can also search for this author in _J. D. ParishYou can also search for this author in _A. L. JohnsonYou can also search for this author in _A. J. FlewittYou can also search for this author in _\" - Contributions - N.C.A.F. and A.J.F. conceived of and designed the overall experiments. N.C.A.F. and K.M.N. fabricated the devices. J.D.P. and A.L.J. fabricated the SnO thin film. N.C.A.F. characterised and modelled the devices. N.C.A.F., K.M.N. and A.J.F. wrote the main manuscript text. All the authors discussed the results and commented on the manuscript. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC BY) licence to any author accepted manuscript version arising from this submission Disclaimer: The views expressed are those of the authors and not necessarily those of the United Kingdom (UK) Department of Health and Social Care, the National Health Service, the National Institute for Health and Care Research (NIHR), the UK Health Security Agency, the World Health Organization or of the commentators.\"- - Authors and Affiliations - \"MRC Centre for Global Infectious Disease Analysis & WHO Collaborating Centre for Infectious Disease Modelling, Jameel Institute, Imperial College London, London, UKDavid J. Haw,_Giovanni Forchini,_Patrick Doohan,_Paula Christen,_ Matteo Pianella,_Robert Johnson,_Alexandra B. Hogan,_Peter Winskill,_Peter J. White,_Azra C. Ghani,_Neil M. Ferguson_&_Katharina D. HauckUSBE, UmexE5 Universitet, UmexE5, SwedenGiovanni ForchiniDepartment of Zoology, University of Oxford, Oxford, UKSumali BajajSchool of Population Health, University of New South Wales, Sydney, AustraliaAlexandra B. HoganDepartment of Economics and Public Policy, Imperial College Business School, London, UKMarisa Miraldo_ &_Peter C. SmithModelling and Economics Unit, UK Health Security Agency, London, UKPeter J. WhiteCentre for Health Economics, University of York, York, UKPeter C. Smith\" - \"AuthorsDavid J. HawYou can also search for this author in _Giovanni ForchiniYou can also search for this author in _Patrick DoohanYou can also search for this author in _Paula ChristenYou can also search for this author in _Matteo PianellaYou can also search for this author in _Robert JohnsonYou can also search for this author in _Sumali BajajYou can also search for this author in _Alexandra B. HoganYou can also search for this author in _Peter WinskillYou can also search for this author in _Marisa MiraldoYou can also search for this author in _Peter J. WhiteYou can also search for this author in _Azra C. GhaniYou can also search for this author in _Neil M. FergusonYou can also search for this author in _Peter C. SmithYou can also search for this author in _Katharina D. HauckYou can also search for this author in _\" - Contributions - \"D.J.H., P.C.S., K.D.H., G.F., P.D. and P.C. conceived and designed the work. D.J.H. and P.D. undertook and K.D.H., P.C.S., G.F., P.D., P.C., R.J., M.P. and S.B. contributed to the analysis and interpretation of data. D.J.H. and P.D. created the new software used in the work. K.D.H. wrote the first draft of the manuscript and P.C.S., D.J.H., G.F., M.P., R.J. and P.D. substantially revised it. S.B., P.D., A.B.H., P.W., M.M., P.J.W., A.C.G. and N.M.F. contributed to interpretation of the data and substantially revised the manuscript. All authors have approved the submitted version and have agreed to be personally accountable for the authors own contributions and to ensure that questions related to the accuracy or integrity of any part of the work, even ones in which the author was not personally involved, are appropriately investigated, resolved and the resolution documented in the literature.\" - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - P.J.W. has received payments from Pfizer for teaching mathematical modeling of infectious-disease transmission and vaccination. P.W. has received personal fees for three days from WHO EURO for COVID-19 vaccine modeling. A.B.H. has received personal fees from WHO for COVID-19 vaccine modeling, and personal fees from Pfizer Inc. to advise on modeling RSV vaccination strategies. All other authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \\'Nature Computational Science thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editor: Ananya Rastogi, in collaboration with the Nature Computational Science team. are available.\\' - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \"Supplementary Notes: 1.1 Economic data, 1.2 Contact matrices, 1.3 Epidemiological parameters and calibration, 1.4 Hospital capacity, 1.5 Sensitivity analyses. Supplementary Figs. 1u201317, Tables 1u20138 and references.Economic activities classified under Sector Industry Classification P (56 Education). Source: .Closures of economic sectors under scenarios B (education open) and LDB (lockdown except education). The values denote percentage closures compared to pre-pandemic level of production for all economic sectors under scenarios B and LDB. 1: open, 0: fully closed. The two months denote periods of fixed economic configurations under scenario B. Education is open at least to 80% pf pre-pandemic production under scenario B. Under scenario LDA, the closure values for all sectors except for the education sector are the same as in LDB; Supplementary Table .Loss in gross value added under all scenarios. For all scenarios GVA loss is reported in xA3 millions against a fully open economy (FO). Scenario A: any economic sector may be closed down to 80% of observed closure during the lockdown period but not lower to sustain essential services; Scenario B: education closed no more than 80%; all other sectors may be closed down to 80% of observed closure during the lockdown period; Scenario LDA: all economic sectors closed to levels observed during the lockdown period March to May 2020; Scenario LDB: all economic sectors except for education are closed to levels of observed closure during the lockdown period; education is operational at 80%; three alternate assumptions on spare emergency hospital capacity for COVID-19 patients in scenarios A and B (H =12,000, 18,000, 24,000). all scenarios assume changes in economic configuration every 2 months except for B (6u2009xD7u20091, H =18,000) which allows changes every month; all scenarios including FO assume that stringency of NPIs and self-productive behavior reduce transmission substantially (u03B4 = 0.72 except for scenario B (u03B4 = 0.76, 18,000), which assumes weak stringency of non-pharmaceutical interventions and/or little self-protective behavior in the population. *Measuring the GVA contribution of education and human health activities is problematic (Schreyer, Paul. 2010. Towards Measuring the Volume Output of Education and Health Services. Paris: OECD. 10.1787/5kmd34g1zk9x-en).Percentage closures of economic sectors under weak stringency of non-pharmaceutical interventions (u03B4 = 0.76), Scenario B (education open), H =18,000. The values denote percentage closures of economic sectors compared to pre-pandemic level of production. 1 fully open and 0 fully closed.Data used for the UK application of DAEDALUS on workforce, total output, gross value added, and intermediate use and provision for 63 sectors, United Kingdom 2016. We maximize monthly gross value added in the optimization, i.e. annual GVA divided by 12. Intermediate use excludes use of imports and excludes taxes (less subsidies) on products. Intermediate provision excludes exports and gross capital formation. Sectors 44 and 45 are combined.\"- - Statistical source data.Statistical source data.Statistical source data.Statistical source data.Statistical source data.- - \\'\\'- - ',\n",
       " 'Creative Commons Attribution (CC BY) licence. - \\' \\'- - \\' \\' - \\'The code used in this publication can be freely downloaded from: under the terms of the Creative Commons Attribution (CC BY) licence.\\' - \\' \\'- - \"Scott, J., Gupta, N., Weber, C. & Newsome, S. Additive manufacturing: Status and opportunities. Sci. Technol. Policy Inst. 2012, 1u201319 (2012)._ Blakey-Milner, B. et al. Metal additive manufacturing in aerospace: A review. Mater. Des. 209, 110008 (2021)._ _ _ Mudge, R. P. & Wald, N. R. Laser engineered net shaping advances additive manufacturing and repair. Weld. J. (Miami, Fla) 86, 44u201348 (2007)._ Gisario, A., Kazarian, M., Martina, F. & Mehrpouya, M. Metal additive manufacturing in the commercial aviation industry: A review. J. Manuf. Syst. 53, 124u2013149 (2019)._ _ ATI. INSIGHT 08u2014additive manufacturingu2014applications in aerospace. Insight 2018, 1u201316 (2018)._ Marlin, T. Process Control, Designing Processes and Control Systems for Dynamic Performance (Thomas Marlin, 2014)._ Chaudhuri, U. R. & Chaudhuri, U. R. Fundamentals of automatic process control. Fund. Autom. Process Control (2012)._ _ _ William, D. & Callister, J. Fundamentals of Materials Science and Engineering (Wiley, 2001). ._ _ Hofmeister, W., Griffith, M., Ensz, M. & Smugeresky, J. Solidification in direct metal deposition by LENS processing. JOM 53, 30u2013 34 (2001)._ _ _ Chechik, L. et al. Variation of texture anisotropy and hardness with build parameters and wall height in directed-energy-deposited 316L steel. Addit. Manuf. 38, 101806 (2021)._ _ _ Song, L., Bagavath-Singh, V., Dutta, B. & Mazumder, J. Control of melt pool temperature and deposition height during direct metal deposition process. Int. J. Adv. Manuf. Technol. 58, 247u2013256 (2012)._ _ Nassar, A. R., Keist, J. S., Reutzel, E. W. & Spurgeon, T. J. Intra-layer closed-loop control of build plan during directed energy additive manufacturing of Ti-6Al-4V. Addit. Manuf. 6, 39u201352 (2015)._ _ _ Meacock, C. & Vilar, R. Laser powder microdeposition of CP2 Titanium. Mater. Des. 29, 353u2013361 (2008)._ _ _ Tang, L. & Landers, R. G. Melt pool temperature control for laser metal deposition processes-part II: Layer-to-layer temperature control. J. Manuf. Sci. Eng. Trans. ASME 132, 0110111u20130110119 (2010)._ _ Hofman, J. T., Pathiraj, B., Dijk, J. V., De Lange, D. F. & Meijer, J. A camera based feedback control strategy for the laser cladding process. J. Mater. Process. Tech. 212, 2455u2013 2462 (2012)._ _ Griffith, M. L. et al. Direct laser additive fabrication system with image feedback control (2002).Farshidianfar, M. H., Khodabakhshi, F., Khajepour, A. & Gerlich, A. P. Closed-loop control of microstructure and mechanical properties in additive manufacturing by directed energy deposition. Mater. Sci. Eng. A 803, 140483 (2021)._ _ _ Thomas, B. G. Modeling of hot tearing and other defects in casting processes. ASM Handb. 22, 362u2013374 (2009)._ Farshidianfar, M. H., Khajepour, A. & Gerlich, A. Real-time control of microstructure in laser additive manufacturing. Int. J. Adv. Manuf. Technol. 82, 1173u20131186 (2016)._ _ Gibson, B. T. et al. Melt pool size control through multiple closed-loop modalities in laser-wire directed energy deposition of Ti-6Al-4V. Addit. Manuf. 32, 100993 (2020)._ _ _ Hua, T., Jing, C., Xin, L., Fengying, Z. & Weidong, H. Research on molten pool temperature in the process of laser rapid forming. J. Mater. Process. Technol. 198, 454u2013462 (2008)._ _ Hu, D. & Kovacevic, R. Sensing, modeling and control for laser-based additive manufacturing. Int. J. Mach. Tools Manuf. 43, 51u2013 60 (2003)._ _ Baraldo, S., Vandone, A., Valente, A. & Carpanzano, E. Closed-loop control by laser power modulation in direct energy deposition additive manufacturing. In Proc. 5th Int. Conf. Ind. 4.0 Model Adv. Manuf. 15u201358 (2020). .Tang, L. & Landers, R. G. Melt pool temperature modeling and control for laser metal deposition processes. In Proc. Am. Control Conf. 4791u20134796 (2009). .Bi, G., Sun, C. N. & Gasser, A. Study on influential factors for process monitoring and control in laser aided additive manufacturing. J. Mater. Process. Technol. 213, 463u2013468 (2013)._ _ _ Farshidianfar, M. H., Khajepour, A. & Gerlich, A. P. Effect of real-time cooling rate on microstructure in laser additive manufacturing. J. Mater. Process. Tech. 231, 468u2013478 (2016)._ _ _ Bi, G., Schu, B., Gasser, A., Wissenbach, K. & Poprawe, R. Development and qualification of a novel laser-cladding head with integrated sensors. Int. J. Mach. Tools Manuf. 47, 555u2013561 (2007)._ _ Vasinonta, A., Beuth, J. L. & Ong, R. Melt pool size control in thin-walled and bulky parts via process maps. In Proc. 12th Solid Free. Fabr. Symp. 432u2013440 (2001). .Dehoff, R. R. et al. Site specific control of crystallographic grain orientation through electron beam additive manufacturing. Mater. Sci. Technol. 31, 931u2013 938 (2015)._ _ _ _ Dass, A. & Moridi, A. State of the art in directed energy deposition: From additive manufacturing to materials design. Coatings 9, 1u201326 (2019)._ _ Mukherjee, T., Manvatkar, V., De, A. & DebRoy, T. Dimensionless numbers in additive manufacturing. J. Appl. Phys. 121, 064904 (2017)._ _ _ _ Wei, H. L. et al. Mechanistic models for additive manufacturing of metallic components. Prog. Mater. Sci. 116, 100703 (2021)._ _ _ Kirka, M. M., Greeley, D. A., Hawkins, C. & Dehoff, R. R. Effect of anisotropy and texture on the low cycle fatigue behavior of Inconel 718 processed via electron beam melting. Int. J. Fatigue 105, 235u2013243 (2017)._ _ _ Ghorbanpour, S. et al. Effect of microstructure induced anisotropy on fatigue behaviour of functionally graded Inconel 718 fabricated by additive manufacturing. Mater. Charact. 179, 111350 (2021)._ _ _ Sanchez, S. et al. The creep behaviour of nickel alloy 718 manufactured by laser powder bed fusion. Mater. Des. 204, 109647 (2021)._ _ _ Bontha, S. & Klingbeil, N. W. Thermal process maps for controlling microstructure in laser-based solid freeform fabrication. Solid Free. Fabr. Symp. 2003, 219u2013226. (2003)._ _ Chechik, L. et al. Hardness variation in inconel 718 produced by laser directed energy deposition. Materialia 26, 101643 (2022)._ _ _ Freeman, F. S. H. B., Thomas, B., Chechik, L. & Todd, I. Multi-faceted monitoring of powder flow rate variability in directed energy deposition. Addit. Manuf. Lett. 2, 100024 (2022)._ _ Freeman, F., Chechik, L., Thomas, B. & Todd, I. Calibrated closed-loop control to reduce the effect of geometry on mechanical behaviour in directed energy deposition. J. Mater. Process. Technol. 311, 117823 (2023)._ _ Chechik, L. Scripts for process control of laser directed energy deposition manufacturing. Opt. Laser Technol. (2022).Bachmann, F., Hielscher, R. & Schaeben, H. Texture analysis with MTEX- Free and open source software toolbox. Solid State Phenom. 160, 63u201368 (2010)._ _ _ Lehto, P., Remes, H., Saukkonen, T., HxE4nninen, H. & Romanoff, J. Influence of grain size distribution on the Hall-Petch relationship of welded structural steel. Mater. Sci. Eng. A 592, 28u201339 (2014)._ _ _ Agazhanov, A. S., Samoshkin, D. A. & Kozlovskii, Y. M. Thermophysical properties of Inconel 718 alloy. J. Phys. Conf. Ser. 2019, 1382 (2019)._ _ _ Mukherjee, T., Zuback, J. S., De, A. & DebRoy, T. Printability of alloys for additive manufacturing. Sci. Rep. 6, 19717 (2016)._ _ _ _ _ _ Raghavan, A., Wei, H. L., Palmer, T. A. & DebRoy, T. Heat transfer and fluid flow in additive manufacturing. J. Laser Appl. 25, 052006 (2013)._ _ _ _ \"- - This work was supported by the EPSRC Rolls-Royce Strategic Partnership Grant, MAPP (grant EP/P006566/1) and EPSRC (grant EP/R512175/1). The provision of materials and supporting information from Rolls-Royce plc. is gratefully acknowledged. We also wish to acknowledge the Henry Royce Institute for Advanced Materials, funded through EPSRC grants EP/R00661X/1, EP/S019367/1, EP/P02470X/1 and EP/P025285/1, for BeAM Magic 2.0 access at The University of Sheffield and also for FEG JSM-7900F SEM access at Royce@Sheffield. Thanks to Dr. Le Ma for operation of the SEM/EBSD. For the purpose of open access, the author has applied a Creative Commons Attribution (CC BY) licence to any Author Accepted Manuscript version arising.- - Authors and Affiliations - \"Department of Materials Science and Engineering, The University of Sheffield, Mappin Street, Sheffield, S1 3JD, UKLova Chechik,_Alexander D. Goodall,_Katerina A. Christofidou_&_Iain Todd\" - \"AuthorsLova ChechikYou can also search for this author in _Alexander D. GoodallYou can also search for this author in _Katerina A. ChristofidouYou can also search for this author in _Iain ToddYou can also search for this author in _\" - Contributions - L.C. led the research program, ran the experiments and wrote the initial draft. The structure of the paper was discussed in depth, rearranged and streamlined with the help of all authors. A.G. contributed to design of figures. All authors edited the manuscript. K.C. and I.T. supervised the work. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC BY) license to any Author Accepted Manuscript version arising from this submission.\"- - Author notes - \\'These authors contributed equally: Sang S. Seo, Susana R. Louros.\\' - Authors and Affiliations - \"Centre for Discovery Brain Sciences, University of Edinburgh, Edinburgh, UKSang S. Seo,_Susana R. Louros,_Natasha Anstey,_Callista B. Harper,_Nicholas C. Verity,_Owen Dando,_Sophie R. Thomson,_Jennifer C. Darnell,_Peter C. Kind_ &_Emily K. OsterweilSimons Initiative for the Developing Brain, University of Edinburgh, Edinburgh, UKSang S. Seo,_Susana R. Louros,_Natasha Anstey,_Callista B. Harper,_Nicholas C. Verity,_Owen Dando,_Sophie R. Thomson,_Jennifer C. Darnell,_Peter C. Kind_&_Emily K. OsterweilDepartment of Molecular and Cellular Neurobiology, Center for Neurogenomics and Cognitive Research, Vrije Universiteit Amsterdam, Amsterdam, NetherlandsMiguel A. Gonzalez-Lozano_&_Ka Wan Li\" - \"AuthorsSang S. SeoYou can also search for this author in _Susana R. LourosYou can also search for this author in _Natasha AnsteyYou can also search for this author in _Miguel A. Gonzalez-LozanoYou can also search for this author in _Callista B. HarperYou can also search for this author in _Nicholas C. VerityYou can also search for this author in _Owen DandoYou can also search for this author in _Sophie R. ThomsonYou can also search for this author in _Jennifer C. DarnellYou can also search for this author in _Peter C. KindYou can also search for this author in _Ka Wan LiYou can also search for this author in _Emily K. OsterweilYou can also search for this author in _\" - Contributions - E.K.O. and S.S.S. conceptualized the study and prepared the manuscript. S.R.L. prepared tissue for proteomics and performed all biochemistry experiments. S.S.S., N.C.V., and S.R.T. performed TRAP experiments. S.S.S. performed all bioinformatics analyses with assistance from O.D. S.S.S. and N.A. performed mGluR-LTD recordings with the supervision of E.K.O. and P.C.K. M.A.G.L. performed Mass Spectroscopy experiments with supervision of K.W.L. and analyzed results with S.S.S. S.S.S. and C.B.H. performed confocal imaging analyses. J.C.D. provided essential insights and advice on the study and manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC BY) license to any Author Accepted Manuscript version arising.\"- - Authors and Affiliations - \"Department of Civil and Environmental Engineering, Imperial College London, London, UKIzhar Hussain Shah_&_Rupert J. MyersDepartment of Civil and Environmental Engineering, University of California, Davis, CA, USASabbie A. MillerDepartment of Civil, Construction, and Environmental Engineering, University of Alabama, Tuscaloosa, AL, USADaqian Jiang\" - \"AuthorsIzhar Hussain ShahYou can also search for this author in _Sabbie A. MillerYou can also search for this author in _Daqian JiangYou can also search for this author in _Rupert J. MyersYou can also search for this author in _\" - Contributions - R.J.M. originated the idea, led research design and write-up of the paper, and contributed to research execution; I.H.S. contributed to research design, write-up, and research execution; D.J. contributed to research design and write-up; and S.A.M. contributed to research design, research execution, and write-up. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \"Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work._ are available.\" - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC BY) license to any Author Accepted Manuscript version arising.- - Authors and Affiliations - \"Department of Oncology and Metabolism, Medical School, University of Sheffield, Beech Hill Road, Sheffield, S10 2RX, UKFeier Zeng,_Giovana Carrasco,_Boya Li_&_Aymen I. IdrisDepartment of Life Sciences, School of Sciences, European University Cyprus, 6 Diogenes Street, 1516, Nicosia, CyprusAntonia Sophocleous\" - \"AuthorsFeier ZengYou can also search for this author in _Giovana CarrascoYou can also search for this author in _Boya LiYou can also search for this author in _Antonia SophocleousYou can also search for this author in _Aymen I. IdrisYou can also search for this author in _\" - Contributions - \"CRediT author statementsF.Z., G.C., B.L., A.S., A.I.I. extracted, collected, collated and analyzed data; A.S. and A.I.I. designed study; A.I.I. conceived and supervised study, and wrote the manuscript. All co-authors reviewed and approved the final version of manuscript. The corresponding author, A.I.I., approved co-authors contribution, ensured each author met the authorship criteria, and that no other eligible authors have been omitted.\" - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - \"A.I. Idris is an inventor on a patent concerning the use of NFu03BAB inhibitors for the treatments of bone diseases. Other authors declare no conflict of interest.\" - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution (CC-BY) license. c Sketch of a quantum dot nano-heterojunction allowing strong photocapacitive current generation. Image adapted under the Creative Commons Attribution (CC-BY) license. d, e Representative examples of conductor/semiconductor/conductor devices for neuromodulation. d The image shows the POLYRETINA device for artificial vision. Image adapted under the Creative Commons Attribution (CC-BY) license. e Sketch of a photovoltaic patch exploited for wound healing. Image adapted under the Creative Commons Attribution (CC-BY) license. In each panel, the dashed box shows a layer-by-layer sketch of the device and the putative main neuromodulation mechanisms.\" - \"Several conductor/semiconductor interfaces have been explored for neuronal stimulation in-vitro. Rand et al. proposed a p-n bilayer structure composed of two organic pigments, phthalocyanine (H2Pc, p-type) and N,Nu2032-dimethyl perylene-3,4:9,10-tetracarboxylic diimide (PTCDI, n-type), layered on top of a chromium/gold anode. Results showed the direct activation of cultured primary neurons and retinal ganglion cells in an embryonic chick retina with short pulses in the deep red via a capacitive coupling. Concurrently, Nizamoglu et al. are investigating other device structures with nanomaterials to maximise the stimulation efficiency of the conductor/semiconductor interface. Han et al. introduced a hybrid organic/inorganic structure exploiting a thin film of aluminium antimonide (AlSb) nanocrystals coated on top of a pristine P3HT film as the hole transport layer and interface layer to cultured neurons (Fig._). The device structure had four layers: ITO as bottom conductor, zinc oxide (ZnO) as electron transport layer, P3HT and AlSb. Photocurrent measurements upon 20-ms blue light illumination showed strong capacitive currents at low irradiance leading to action potential generation in cultured neurons up to 20u2009Hz of repetition rate. Karatum et al. showed that interfaces based on InP/ZnS core/shell quantum dots (QDs) generated Faradaic currents leading to either membrane depolarisation and action potential generation or membrane hyperpolarisation depending on the device structure (ITO/titanium oxide/QD for depolarisation and ITO/QD/ZnO for hyperpolarisation). In a second report, authors further explore QDs for capacitive current generation at the material/electrolyte interface by exploiting an heterojunction composed of InP/ZnO/ZnS core/shell/shell QDs as the electron donor and PC60BM as the electron acceptor layered over a ITO/ZnO conductor (Fig._). Last, Srivastava et al. proposed a high open-circuit voltage bulk heterojunction composed of poly[4,8-bis(5-(2-ethylhexyl)thiophen-2-yl)benzo[1,2-b;4,5-bu2032 ]dithiophene-2,6-diyl-alt-(4-(2-ethylhexyl)-3-fluorothieno[3,4-b]thiophene-)-2-carboxylate-2-6-diyl)] (PTB7-Th) blended with [6,6]-Phenyl-C71-butyric acid methyl ester (PC71BM) coated over a ITO/ZnO conductor to elicit spikes in cultured primary neurons.\" - \"Ghezzi et al. developed a photovoltaic wide-field spherical array for retinal stimulation, embedding several thousands of photovoltaic pixels composed of a P3HT:PC60BM layered between a polymeric anode (PEDOT:PSS) and an inorganic cathode (titanium/titanium nitride, Ti/TiN) for efficient capacitive stimulation (conductor/semiconductor/conductor interface) (Fig._). Results demonstrated efficient network-mediated stimulation of retinal ganglion cells with very high spatial resolution equivalent to the pitch of the photovoltaic pixels embedded in the array. The same device was proposed exploiting a different electron donor material, the poly[2,6-(4,4-bis-(2-ethylhexyl)-4H-cyclopenta[2,1-b;3,4-bu2032 ]dithiophene)-alt-4,7(2,1,3-benzothiadiazole)] (PCPDTBT) blended with the PC60BM electron acceptor (PCPDTBT:PC60BM) to provide sensitivity in the near infrared spectrum, crucial for applications in retinal stimulation.\" - \"Besides capacitive and Faradaic neuronal stimulation, heat generation can be exploited for neuromodulation. Martino et al. showed that a thin P3HT film, or its bulk heterojunction with PC60BM, layered on an insulating glass substrate led to membrane depolarisation on Human Embryonic Kidney (HEK-293) cells, grown on top, mediated by heat transfer for sufficient long and intense exposure. Membrane depolarisation was associated with a temperature-dependent change in the membrane capacitance. Feyen et al. extended this approach reporting that prolonged illumination (e.g. ~150u2009ms or longer) of the P3HT film led to membrane hyperpolarization and silencing of neuronal firing in cultured primary neurons, epileptic hippocampal slices and explanted blind retinas. The ability to silence neuronal firing might find applications in several conditions, such as inhibition of nerve activity for pain relief or silencing epileptiform activity. While in these reports, the photothermal modulation of the cell membrane was independent of TRPV channels, Lodola et al. showed that the same interface could activate TRPV1 channels, when they were stably transfected in HEK-293 cells (Fig._). This possibility is also a key tool in neurotechnology due to the importance of TRPV1 channels as a receptor involved in body temperature regulation, heat and pain detection in sensory nociceptive fibres and functions in the central nervous system including response to neurogenic inflammation, cytoskeletal remodelling, synaptic remodelling and plasticity, neuronal excitability and cell survival.\" - \\'Fig. 3: Pure organic semiconductor bioelectronic interfaces.a The sketch shows a pristine P3HT layer interfaced with HEK-293 cells for the photothermal modulation of the membrane potential and other biological processes via the activation of TRPV1 channels. Image adapted under the Creative Commons Attribution (CC-BY) license. b The scanning electron microscopy image shows P3HT NPs being internalised in a HEK-293 cell. Upon absorption of light, the internalised P3HT NPs modulated intracellular calcium dynamics via a photocatalytic process. Image adapted under the Creative Commons Attribution (CC-BY) license. c The image shows an ultrastructural analysis of the interaction between a P3HT NP and a cultured primary neuron from a resin-embedded focused ion beam cross-section. NPs were not internalised but established a tight contact with the neuronal membrane. The injection of P3HT NPs in the subretinal space of blind rats endowed remaining retinal neurons with light sensitivity. The mechanism of interaction in-vivo is still unclear. Image reprinted by permission from Springer Nature. In each panel, the dashed box shows a layer-by-layer sketch of the device and the putative main neuromodulation mechanisms.\\' - Apart from neurons, photostimulation of astrocytes, critical for supporting neuronal function, was also attempted. Photoexcitation of the ITO/P3HT:PC60BM conductor/semiconductor interface caused membrane depolarisation mediated by the ClC-2 chloride channel, highlighting the possibility to use organic semiconducting materials to probe ion channels in non-excitable cells.- - A new exciting perspective for organic semiconductors is their exploitation as nanomaterials (e.g. nanoparticles, NPs). In bioelectronic and neural interfaces, nanomaterials have the advantage to decrease the cell-material distance by reaching the plasma membrane and closely interact with the target cells. - \"Semiconducting polymer nanobioconjugates sensitive to near infrared light (Poly{2,2u2032 -[(2,5-bis(2-hexyldecyl)-3,6-dioxo-2,3,5,6-tetrahydropyrrolo[3,4-c]pyrrole-1,4-diyl)dithiophene]-5,5u2032 -diyl-alt-thiophen-2,5-diyl}, PDPP3T) have been exploited as photothermal nanomodulators. The high photothermal conversion efficiency combined with antibodies selective to TRPV1 receptors allowed for precise spatio-temporal activation of the thermosensitive ion channel in an immortalised neuronal cell line. The binding of NPs with specific biological targets through established techniques, namely antigen-antibody interactions, presents an interesting alternative to optogenetics that is both selective and light-sensitive.\" - \"Pristine P3HT NPs (with hydrodynamic diameter between 150 and 300u2009nm) were first reported to be internalised in cultured HEK-293 cells and simple eyeless polyps. In HEK-293 cells, Bossio et al. demonstrated that P3HT NPs have photocatalytic activity (Fig._). Upon internalisation in cells and light absorption, P3HT NPs form ROS below toxic levels modulating intracellular calcium dynamics without affecting the cell viability or the NP optoelectronic properties. Likewise, the photosensitivity bestowed by P3HT NPs in the freshwater polyp Hydra vulgaris has shown increase in the contraction events of the animal body and enhancement of opsin3-like gene expression, part of a superfamily of G protein-coupled receptors that play a key role in vision. Authors ruled out a photothermal effect for P3HT NPs since the total heating due to the contribution of all the internalised NPs resulted to be negligible. In light of the previous report in HEK-293 cells, the increase in cytoplasmic ROS and the subsequent pathways seems to be the cause of the reported physiological changes. In addition, to avoid internalisation and direct the binding of P3HT NPs toward the cell membrane, polythiophenes functionalised with alkyl side chains terminating with N-hydroxysuccinimidyl ester groups have been synthesised (NHS NPs). These NHS NPs showed higher efficacy in the modulation of the membrane potential.\" - \"Recently, P3HT NPs were injected subretinally in rats and exploited as a retinal prosthesis. Contrary to the previous reports, P3HT NPs were engulfed but not internalised by neurons remaining extracellular (Fig._). When injected in close proximity to the retina, P3HT NPs endowed remaining retinal neurons with light sensitivity. In this case, ROS are not taken into consideration. Instead, authors proposed a capacitive mechanism relying on the close contact between NPs and the cell membrane as the driver of the evoked retinal responses; although, it remains unclear how a potential difference could affect the electrolyte solution surrounding a spherical P3HT NP without spatial asymmetry in the material or in the device.\"- - Further from neuromodulation, cardiac cells can also benefit from the photoexcitation of organic semiconductors. Heating cardiac cells can trigger them, allowing the manipulation of their rhythm contraction or reinstating synchronisation should there be a failure. Photothermal coupling obtained by light exposure of a P3HT film increased the contraction rate of human-induced pluripotent stem cell-derived cardiomyocytes in a spatially and temporally precise fashion. - \"Electrical stimulation can promote tissue regeneration and the ability of organic semiconductors to be wireless, lightweight and flexible makes them good candidates in a solar cell for wearable therapeutic purposes. This has been applied to ischaemic tissue, where it promoted angiogenesis in the mouse hindlimb, and to cutaneous wounds. In both cases, the small solar cell, composed of ITO/ZnO/P3HT:PC60BM/PEDOT:PSS with silver (Ag) electrodes, provided sustained low power electrical stimulation throughout the regenerative process (Fig._). By adding an active component to the wound dressing, the organic solar cell was aligned with the wound healing direction and showed marked improvement compared to the passive patch. Along the same vein, the interaction between endothelial colony-forming cells, critical in the reconstruction of vasculature, and a pristine P3HT film enhanced cell proliferation and tubulogenesis via the activation of TRPV1 channels upon light exposure.\" - The bulk of research into organic semiconductors for bioelectronic interfaces has focused on biomedical impact, being able to utilise the breadth of fundamental biology to elucidate the underlying mechanisms. Recent efforts have been made to introduce organic semiconductors in biotechnology to studies on plants, a critical component of our ecosystem, to modulate them and make them more resilient to environmental risks. Recently, an interface between the plant model Arabidopsis thaliana and P3HT NPs was investigated. Authors specifically studied the stoma, which is a pore made of two guard cells that controls fundamental aspects of plant physiology such as the uptake of carbon dioxide and the release of oxygen and water. They found that P3HT NPs were not internalised into these guard cells but were able to reduce the opening of the pore by >20%, which would thus reduce water loss. The exact mechanism of this control was unclear but was not linked to the steric interaction, tested through inert silicon dioxide NPs, and attributed to photocatalytic effects with the production of ROS and modulating the calcium signalling. While still in its infancy, the novel application of these organic semiconductors to plants presents a broad and potentially impactful field of research. - The majority of the approaches presented above have been inspired from research in organic photovoltaics. This community continues to excel with materials and methods aiming for thinner, cheaper and more efficient devices for wearable applications. Wearable devices capitalise on the flexible and light-weight nature of organic photovoltaic materials to be incorporated in clothing or to be worn on skin, harvesting solar energy for powering organic sensors. Further advancement has allowed organic photovoltaics to power wearable ion pumps. These encouraging results bodes well for organic photovoltaics to eventually power subcutaneous implantable devices.- - Current efforts in organic semiconductors have certainly shown promise as a tool to investigate and modulate biological processes or as part of a medical device for neuromodulation. A broad base has been established but several key challenges exist for applying organic semiconductors to further neurostimulation purposes. - For organic photovoltaic devices to be considered as an alternative to electrical neurostimulation, a key factor is the efficiency of the system. The predominant mechanism of organic semiconductors altering neurons is in the form of a solar cell through electrical currents. The power output of a solar cell is dependent on its area. A smaller device can offer better spatial resolution; however, the output power is reduced and further subject to resistive losses. An increase in efficiency would allow the existing applications to enhance in their capabilities, for instance through further miniaturisation of photovoltaic pixels in an array for retinal stimulation or cultured neurons. Further expansion of organic solar cells for neurostimulation where the threshold for activation is higher, such as nerve or epicortical stimulation, device size or efficiency must be increased. Larger devices are inherently more invasive and less spatially selective so improving efficiency is preferred either by careful consideration of the design of the device or proper choice and optimisation of materials, more likely a combination of both. The choice of the light absorbing material is doubly important when we consider that biological tissue absorbs a great deal of visible light but is more transparent in the near infrared spectrum. This aspect is crucial when light is delivered externally and a bulk of the irradiance is lost before hitting the organic semiconductor. As such, many materials lose their functionality due to the physical barrier and would benefit from a shift towards materials sensitive to near infrared light. - The combination of light and neuromodulation is also offered by optogenetics in fundamental research, which naturally have high resolution with cellular and subcellular targeting capability. The process to express optogenetic tools involves the insertion of genes into the cell for it to produce. Organic semiconductors offer a natural in-between from light to material to neuron in a non-genetic mechanism, thereby innately not immunogenic or subject to viral vector delivery. In NP form, organic semiconductors are a powerful tool to interface with cells with specific surface functionalisation, such as harnessing antigen-antibody interactions, supporting tremendous advancement in their biological application. Additionally, investigating the cell response to heat treatment and ROS exposure are additional options not yet achieved with optogenetics. - The photothermal and photocatalytic effects are most apparent when pristine organic semiconductors are the only element exposed to the electrolyte and offer interesting modulatory effects on cells and tissue. Focused heat and ROS generation offer highly novel means to modulate cells however their existence in excess may also hinder the translation to the clinic due to toxicity. Specific quantification and analysis of these semiconductor/cell interactions, particularly when generated from NPs, need to be closely probed as it ventures into the in vivo space. Through appropriate choices in the illumination protocol or the NP structure, independent control of each phototransduction mechanism could be achieved thus facilitating the use of NPs as a biological tool. - As the field progresses, using a combination of innovative nanomaterials and carefully designed devices, organic semiconductors have the potential to occupy a larger space in neuromodulation. Subsequently, it can then push for medical grade devices through defined regulations and convincing applications stemming from rigorous laboratory research. Future work should focus on efficiency, reproducibility, stability and safety to allow organic semiconductors to grow from the capabilities demonstrated thus far and enter the medical industry and clinics.- - \"Palanker, D., Le Mer, Y., Mohand-Said, S., Muqit, M. & Sahel, J. A. Photovoltaic restoration of central vision in atrophic age-related macular degeneration. Ophthalmology 127, 1097u20131104 (2020)._ _ Paulsen, B. D., Tybrandt, K., Stavrinidou, E. & Rivnay, J. Organic mixed ionicu2013 electronic conductors. Nat. Mater. 19, 13u201326 (2020)._ _ _ Ohayon, D. & Inal, S. Organic bioelectronics: from functional materials to nextu2010generation devices and power sources. Adv. Mater. (2020).Bystrenova, E. et al. Neural networks grown on organic semiconductors: neural networks grown on organic semiconductors. Adv. Funct. Mater. 18, 1751u20131756 (2008)._ _ _ Antognazza, M., Ghezzi, D., Musitelli, D., Garbugli, M. & Lanzani, G. A hybrid solid-liquid polymer photodiode for the bioenvironment. Appl. Phys. Lett. 94, 243501 (2009)._ _ _ Ferlauto, L. et al. Design and validation of a foldable and photovoltaic wide-field epiretinal prosthesis. Nat. Commun. 9, 992 (2018)._ _ _ Han, M. et al. Photovoltaic neurointerface based on aluminum antimonide nanocrystals. Commun. Mater. 2, 19 (2021)._ _ Feyen, P. et al. Light-evoked hyperpolarization and silencing of neurons by conjugated polymers. Sci. Rep. 6, 22718 (2016)._ _ _ SilverxE5-Ejneby, M. et al. A chronic photocapacitor implant for noninvasive neurostimulation with deep red light. bioRxiv (2020).Lodola, F. et al. Conjugated polymers optically regulate the fate of endothelial colony-forming cells. Sci. Adv. 5, eaav4620 (2019)._ _ _ Jang, H.-K. et al. A disposable photovoltaic patch controlling cellular microenvironment for wound healing. IJMS 19, 3025 (2018)._ _ _ Milos, F. et al. High aspect ratio and light-sensitive micropillars based on a semiconducting polymer optically regulate neuronal growth. ACS Appl. Mater. Interfaces 13, 23438u201323451 (2021)._ _ _ Benfenati, V. et al. Photostimulation of whole-cell conductance in primary rat neocortical astrocytes mediated by organic semiconducting thin films. Adv. Healthcare Mater. 3, 392u2013399 (2014)._ _ _ Bossio, C. et al. Photocatalytic activity of polymer nanoparticles modulates intracellular calcium dynamics and reactive oxygen species in HEK-293 cells. Front. Bioeng. Biotechnol. 6, 114 (2018)._ _ Tortiglione, C. et al. Semiconducting polymers are light nanotransducers in eyeless animals. Sci. Adv. 3, e1601699 (2017)._ _ _ Tullii, G., Gobbo, F., Costa, A. & Antognazza, M. R. A prototypical conjugated polymer regulating signaling in plants. Adv. Sustainable Syst. (2021).Paltrinieri, T. et al. Understanding photocapacitive and photofaradaic processes in organic semiconductor photoelectrodes for optobioelectronics. Adv. Funct. Mater. 31, 2010116 (2021)._ _ _ xD0erek, V., Rand, D., Migliaccio, L., Hanein, Y. & Gu0142owacki, E. D. Untangling photofaradaic and photocapacitive effects in organic optoelectronic stimulation devices. Front. Bioeng. Biotechnol. 8, 284 (2020)._ _ Bellani, S., Antognazza, M. R. & Bonaccorso, F. Carbon-based photocathode materials for solar hydrogen production. Adv. Mater. 31, 1801446 (2019)._ _ _ Albert, E. S. et al. TRPV4 channels mediate the infrared laser-evoked response in sensory neurons. J. Neurophysiol. 107, 3227u20133234 (2012)._ _ _ Shapiro, M. G., Homma, K., Villarreal, S., Richter, C.-P. & Bezanilla, F. Infrared light excites cells by changing their electrical capacitance. Nat. Commun. 3, 736 (2012)._ _ _ Jenkins, M. W. et al. Optical pacing of the embryonic heart. Nat. Photon. 4, 623u2013626 (2010)._ _ _ Carvalho-de-Souza, J. L. et al. Photosensitivity of neurons enabled by cell-targeted gold nanoparticles. Neuron 86, 207u2013 217 (2015)._ _ _ Gribi, S., Dunilac, S., du, B., de, Ghezzi, D. & Lacour, S. P. A microfabricated nerve-on-a-chip platform for rapid assessment of neural conduction in explanted peripheral nerve fibers. Nat. Commun. 9, 4403 (2018)._ _ _ Nelidova, D. et al. Restoring light sensitivity using tunable near-infrared sensors. Science 368, 1108 (2020)._ _ _ Ghezzi, D. et al. A hybrid bioorganic interface for neuronal photoactivation. Nat. Commun. 2, 166 (2011)._ _ _ Rand, D. et al. Direct electrical neurostimulation with organic pigment photocapacitors. Adv. Mater. 30, 1707292 (2018)._ _ _ Han, M. et al. Organic photovoltaic pseudocapacitors for neurostimulation. ACS Appl. Mater. Interfaces 12, 42997u201343008 (2020)._ _ _ SilverxE5 Ejneby, M. et al. Extracellular photovoltage clamp using conducting polymeru2010modified organic photocapacitors. Adv. Mater. Technol. 5, 1900860 (2020)._ _ _ Ghezzi, D. et al. A polymer optoelectronic interface restores light sensitivity in blind rat retinas. Nat. Photon. 7, 400u2013406 (2013)._ _ _ Gautam, V., Rand, D., Hanein, Y. & Narayan, K. A polymer optoelectronic interface provides visual cues to a blind retina. Adv. Mater. 26, 1751u2013 1756 (2014)._ _ _ Maya-Vetencourt, J. F. et al. A fully organic retinal prosthesis restores vision in a rat model of degenerative blindness. Nat. Mater. 16, 681u2013689 (2017)._ _ _ Karatum, O. et al. Nanoengineering InP quantum dot-based photoactive biointerfaces for optical control of neurons. Front. Neurosci. 15, 652608 (2021)._ _ Karatum, O. et al. Quantum dot and electron acceptor nano-heterojunction for photo-induced capacitive charge-transfer. Sci. Rep. 11, 2460 (2021)._ _ _ Srivastava, S. B. et al. Bulk-heterojunction photocapacitors with high open-circuit voltage for low light intensity photostimulation of neurons. J. Mater. Chem. C 9, 1755u20131763 (2021)._ _ _ Chenais, N. A. L., Leccardi, M. J. I. A. & Ghezzi, D. Capacitive-like photovoltaic epiretinal stimulation enhances and narrows the network-mediated activity of retinal ganglion cells by recruiting the lateral inhibitory network. J. Neural Eng. 16, 066009 (2019)._ _ Chenais, N. A. L., Airaghi Leccardi, M. J. I. & Ghezzi, D. Photovoltaic retinal prosthesis restores high-resolution responses to single-pixel stimulation in blind retinas. Commun. Mater. 2, 28 (2021)._ _ Airaghi Leccardi, M. J. I. et al. Photovoltaic organic interface for neuronal stimulation in the near-infrared. Commun. Mater. 1, 21 (2020)._ _ Martino, N. et al. Photothermal cellular stimulation in functional bio-polymer interfaces. Sci. Rep. 5, 8911 (2015)._ _ _ Lodola, F., Martino, N., Tullii, G., Lanzani, G. & Antognazza, M. Conjugated polymers mediate effective activation of the Mammalian Ion Channel Transient Receptor Potential Vanilloid 1. Sci. Rep. 7, 8477 (2017)._ _ _ Shuba, Y. M. Beyond neuronal heat sensing: diversity of TRPV1 heat-capsaicin receptor-channel functions. Front. Cell. Neurosci. 14, 612480 (2021)._ _ _ Lyu, Y., Xie, C., Chechetka, S. A., Miyako, E. & Pu, K. Semiconducting polymer nanobioconjugates for targeted photothermal activation of neurons. J. Am. Chem. Soc. 138, 9049u20139052 (2016)._ _ _ Zucchetti, E. et al. Poly(3-hexylthiophene) nanoparticles for biophotonics: study of the mutual interaction with living cells. J. Mater. Chem. B 5, 565u2013574 (2017)._ _ _ Zangoli, M. et al. Engineering thiophene-based nanoparticles to induce phototransduction in live cells under illumination. Nanoscale 9, 9202u20139209 (2017)._ _ _ Maya-Vetencourt, J. F. et al. Subretinally injected semiconducting polymer nanoparticles rescue vision in a rat model of retinal dystrophy. Nat. Nanotechnol. 15, 698u2013708 (2020)._ _ _ Lodola, F., Vurro, V., Crasto, S., Di Pasquale, E. & Lanzani, G. Optical pacing of humanu2010induced pluripotent stem cellu2010derived cardiomyocytes mediated by a conjugated polymer interface. Adv. Healthcare Mater. 8, 1900198 (2019)._ _ _ Jeong, G.-J. et al. Therapeutic angiogenesis via solar cell-facilitated electrical stimulation. ACS Appl. Mater. Interfaces 9, 38344u201338355 (2017)._ _ _ Hashemi, S. A., Ramakrishna, S. & Aberle, A. G. Recent progress in flexibleu2013wearable solar cells for self-powered electronic devices. Energy Environ. Sci. 13, 685u2013 743 (2020)._ _ _ Park, S. et al. Self-powered ultra-flexible electronics via nano-grating-patterned organic photovoltaics. Nature 561, 516u2013 521 (2018)._ _ _ Kaltenbrunner, M. et al. Ultrathin and lightweight organic solar cells with high flexibility. Nat. Commun. 3, 770 (2012)._ _ _ NxFAxF1ez, C. G., Navaraj, W. T., Polat, E. O. & Dahiya, R. Energy-autonomous, flexible, and transparent tactile skin. Adv. Funct. Mater. 27, 1606287 (2017)._ _ _ Jakeu0161ovxE1, M. et al. Wireless organic electronic ion pumps driven by photovoltaics. npj Flex. Electron. 3, 14 (2019)._ _ \"- - \"This work was supported by xC9cole polytechnique fxE9dxE9rale de Lausanne and Medtronic plc.\"- - Authors and Affiliations - \"Medtronic Chair in Neuroengineering, Center for Neuroprosthetics and Institute of Bioengineering, School of Engineering, xC9cole polytechnique fxE9dxE9 rale de Lausanne, Chemin des Mines 9, 1202, Geneva, SwitzerlandDanashi Imani Medagoda_ &_Diego Ghezzi\" - \"AuthorsDanashi Imani MedagodaYou can also search for this author in _Diego GhezziYou can also search for this author in _\" - Contributions - D.I.M. and D.G. wrote the article. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\'Peer review information Communications Materials thanks Eric Glowacki, Sedat Nizamoglu and the other, anonymous, reviewer for their contribution to the peer review of this work. Primary Handling Editor: John Plummer. Peer reviewer reports are available.\\' - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 2.0 License), , 2003). Second row: images processed through the model: (left) two bars and (right) a rosette pattern. Third row: heat maps showing summed response of eight narrowly tuned model neurons with equally spaced orientation preferences in an iso-orientation domain. Bottom row: heat maps showing summed response of eight broadly tuned model neurons with equally spaced orientation preferences in pinwheel centre.\" - Using two-photon imaging, Levy et al. found no relationship between the geometry of the dendritic arbour and the local homogeneity in the orientation map, suggesting that all layer 2/3 neurons integrate inputs similarly from all their neighbours. Therefore, we used the same parameters throughout the cortex and applied the same cortical interactions, irrespective of the local arrangement of orientation preference. The first component we used was distance-weighted excitation from surrounding oriented neurons. As cells in iso-orientation domains are surrounded by cells with similar orientation preference, excitation had the effect of differentially increasing the response to the preferred orientation. This, in turn, narrowed the orientation tuning and increased the cross-orientation suppression. In pinwheel domains, excitation from surrounding cells with different orientation preferences differentially increased cell response to non-preferred orientations, thus broadening orientation tuning and decreasing cross-orientation suppression. In addition, the effect of the surround excitation in our model was to reduce compression in the contrast response curves. - \"Our model assumed the same pre-cortical contrast response across the orientation map; thus, we needed to increase the compression more in iso-orientation domains than pinwheel centres to match the experimental data, while not adversely affecting the other cortical response properties. To achieve this, we used a standard hyperbolic form of self-normalization but replaced the conventional semi-saturation constant by the output of non-oriented inhibitory neurons with severely compressed response functions. As excitation increased responses more for iso-orientation domains than pinwheel centres, a hyperbolic normalization with the same parameters throughout the cortex had the effect of increasing compression more for iso-orientation domains than pinwheel centres, in accord with the physiological results. It is noteworthy that this form of normalization will reduce the amount of cross-orientation suppression when decreasing the cortical semi-saturation constant and thus will reduce the negative correlation between the two values that would be expected if there was no modification of thalamic inputs. We will refer to this model as the excitationu2013normalization (EN) model.\" - The combined effects of the two cortical components are illustrated and compared with the input from the thalamus in . The population receptive field of thalamic afferents is less balanced in ON and OFF responses at pinwheels, thus leading to a tendency for broader orientation selectivity than in other cortical domains. The excitatory cortical circuit further narrows the orientation tuning from the thalamic components in the iso-orientation domain as illustrated for a narrowly tuned cortical input in (left). It also broadens the tuning in the pinwheel neighbourhood as illustrated for a broadly tuned cortical input in (right). As a consequence, the tuning distribution is narrower for the iso-orientation domain than for the pinwheel centre, in agreement with the experimental results. Moreover, even though the normalization component is identical in the two cortical regions, it yields more saturation in cells within iso-orientation domains than pinwheel centres (). For completeness, shows the effects of intra-cortical EN circuits on narrowly and broadly tuned neurons within each cortical region (iso-orientation domain and pinwheel). - \"To predict the EN models performance in more complex stimulus patterns, we compared the thalamic and cortical outputs generated in response to a sinusoidal grating and plaids made of 2, 4 or 8 gratings with equally spaced orientations including the preferred one. A narrowly tuned cell exhibits cross-orientation suppression just from the thalamic input, as would be expected from the early response compression. The response decreases with the addition of more orientations to the stimulus, because contrast is reduced for the preferred grating to keep the total contrast equal to 1.0 ( left, black). In addition, suppression is further enhanced by the cortical network in iso-orientation domains ( left, red). The broadly tuned cell exhibits cross-orientation suppression that is similar to the narrowly tuned cell, due to the same early response compression. However, the broadly tuned cell responds to the additional oblique gratings; therefore, the responses to four and eight grating plaids increase ( right, black). In addition, the cross-orientation suppression is further reduced by the cortical network in pinwheels, leading to enhanced responses for the complex plaids ( right, blue).\" - The difference between iso-orientation domains and pinwheel centres is seen clearly by processing patterns and contours through two simulated extra-striate cells, one pooling outputs from eight cells in an iso-orientation domain and the other from eight cells in a pinwheel centre (). The former responds strongly to the isolated contours and hardly at all to the pattern. The latter gives a weak response to the isolated contours, but recreates the pattern. As a result, extra-striate neurons that pool outputs from iso-orientation domains will provide information about isolated edges and contours, whereas extra-striate neurons that pool outputs from pinwheels will provide information about patterns and textures. - \"After generating these predictions, we measured responses to pinwheel patterns in new experiments (three cats) and compared them with responses to gratings of preferred orientations. We found that responses to octotropic plaids (eight gratings) are significantly higher in pinwheels than in iso-orientation domains (, MUA: R=u22120.3730, P<0.0001) and relative responses increase with broadness of tuning width (, MUA: R=u22120.5339, P<0.0001), thus confirming the EN model s predictions.\" - \\'Figure 5: Association between multiple orientation stimulus response versus local orientation topography and orientation tuning width.Stimulus was an octotropic plaid composed by summing eight gratings equally spaced in orientation. Multi-unit activity from three cats (N=167). Octotropic plaid response is normalized to the maximum response of the unit. (a) Normalized octotropic plaid response as a function of local orientation homogeneity. (b) Normalized octotropic plaid response as a function of half-width at half-height of the orientation tuning curve.\\' - In summary, our simulations show that applying the same neural computations uniformly across cortex can generate different response properties in pinwheel centres and iso-orientation domains. The excitatory cortical circuit proposed in our model generates cortical compartments that differ in orientation homogeneity and tuning, but also in contrast saturation and response suppression to stimulus patterns. The narrow orientation tuning, high contrast sensitivity, and pronounced cross-orientation suppression of iso-orientation domains seems ideal to detect edges in natural images. Conversely, the broad orientation tuning and limited cross-orientation suppression in pinwheels should allow processing of stimulus patterns made of multiple orientations such as textures. - Given that divisive normalization seems (theoretically) so omnipresent and accounting for so many cortical response patterns, we present an explicit argument as to why such a normalization is excluded from our model of cortex. We demonstrate that local normalization fails to reproduce the differences in cross-orientation suppression and orientation tuning between iso-orientation domains and pinwheel centres that we empirically measured. To test the effects of local normalization, we used the same pre-cortical components as our model, but replaced the multiplicative surround excitation by divisive normalization from surrounding neurons (). In iso-orientation domains, divisive normalization from surrounding neurons broadens the tuning curve by reducing responses to preferred orientations (). In pinwheels, divisive normalization can leave the tuning curve essentially intact () or even narrow it by reducing responses to orientations on the flanks, that is, the opposite of the empirical results. Divisive normalization has been commonly used to explain the compression of contrast response curves and here it leads to greater nonlinearity for iso-orientation domains, consistent with the empirical results. However, divisive normalization makes the wrong prediction for processing multi-orientation patterns as shown in . For completeness, compares the effects of divisive normalization for narrowly and broadly tuned cells in iso-orientation domains and pinwheels. - \"Figure 6: Cortical normalization model predictions for differences between iso-orientation domains and pinwheel centres.(a) Model schematic: inputs were passed through a photoreceptor compressive nonlinearity. The retinal output was convolved with difference-of-Gaussian LGN receptive fields and combined in a pushu2013pull manner to create elongated cortical receptive fields that were run through an expansive spiking nonlinearity. Neighbouring V1 neurons had orientation preferences corresponding to iso-orientation and pinwheel domains. The cortical component is normalization from surrounding oriented neurons with a factor added to the denominator, to keep it from going to zero. (bu2013d) Left: predictions for iso-orientation domain cortical output (red triangles) versus thalamic input for narrowly tuned neurons (black circles). Right: predictions for pinwheel domain cortical output (blue triangles) versus thalamic input for broadly tuned neurons (black circles). (b) Orientation tuning. (c) Contrast response. (d) Responses to images of increasing orientation complexity: preferred grating and plaids of 2, 4 and 8 equally spaced orientations including the preferred.\" - Divisive normalization could be made consistent with the empirical results by normalizing the responses with distant cells that have orthogonal orientation preferences in iso-orientation domains and the same orientation preferences in pinwheels. This would not only be mathematically ad hoc by using different principles for different orientation domains, but would also go against a large body of physiological data. For example, it would predict that the strongest inhibition in most cortical neurons should be evoked by a stimulus orientation orthogonal to the optimal, whereas intracellular recordings have consistently demonstrated that the strongest inhibition is generated by the optimal orientation. It is noteworthy that there is an essential normalization component to the EN model. This component keeps the excitation within bounds and is responsible for the greater response compression in iso-orientation domains. - Busse et al. generalized measurements of cross-orientation suppression to different relative contrasts of tests and masks. They found summation in cross-orientation suppression when contrasts of the mask and test were similar, but a shift to a winner-take-all rule to mask or test whichever had significantly higher contrast. Busse et al. suggested that the winner-take-all result provided strong support to a surround normalization model expressed in terms of stimulus contrasts. We found that surround normalization was not needed to explain their results. We entered their stimulus contrast values into our model and discovered that even the classical thalamic cross-orientation model predicts summation when contrasts are similar and greater cross-orientation suppression when contrast is higher for one component (). Further, the additional cross-orientation suppression for iso-orientation domains predicted by our model completes the winner-take-all. Busse et al. presented population results for cat visual cortex and, as iso-orientation domain cells dominate in numbers, our model works well as an alternative to divisive normalization in reproducing both the summation and winner-take-all regimes. - \\'Figure 7: EN model prediction for plaids with varying component contrasts.Predicted responses for (a) the thalamic model with narrow input to cortex, (b) the EN model with narrow input to an iso-orientation domain, (c) the thalamic model with broad input to the cortex and (d) the EN model with broad input to a pinwheel centre. Dashed curves are the contrast response curves measured through variation of the contrast of the preferred oriented single grating. Responses are normalized to the 25% contrast model output. The solid lines depict the model predictions for plaids where the preferred orientation is fixed at 25% contrast and the contrast of the component (test) orientation is varied.\\'- - Our physiological results demonstrate that cortical neurons in iso-orientation domains not only have narrower orientation tuning, but also generate visual responses that are more sensitive to contrast and more suppressed by non-preferred orientations than cells in pinwheel centres. Neurons near pinwheel centres are more broadly tuned for orientation and generate visual responses that are more linearly related with contrast and less suppressed by patterns made of multiple orientations. The fact that cross-orientation suppression varies across different regions of the orientation map suggests a possible role for the cortical network in shaping neuronal response properties. Using a computational model, we show that applying the same cortical computation homogeneously across primary visual cortex, we can generate outputs that differ in contrast sensitivity, orientation tuning width and cross-orientation suppression, as the cortical orientation gradient varies between iso-orientation domains and pinwheel centres. As variation in these outputs can be used to process different features of natural scenes, our results have broader implications for cortical processing and provide a possible functional role of orientation maps in the visual cortex. - We propose the simplest model that can reproduce the empirical associations that we discovered between cortical topography and neuronal response properties. The most important component is the cortical excitation from surrounding neurons. In iso-orientation domains, excitation has the effect of narrowing the orientation tuning and increasing cross-orientation suppression. In pinwheel domains, excitation broadens orientation tuning and decreases cross-orientation suppression. The effects of orientation topography thus supplement evidence for the amplification of tuned thalamic excitation by cortical circuits. - The EN model needs a normalization component to replicate the physiological differences in contrast saturation between iso-orientation domains and pinwheel centres. Surround normalization is widely proposed as the mechanism for many kinds of cortical effects; however, we chose not to use it because it had undesired effects in iso-orientation domains and pinwheel centres. As neurons in iso-orientation domains are surrounded by cells with similar orientation preference, surround normalization strongly reduced the response to the preferred orientation, broadening the orientation tuning and weakening cross-orientation suppression. On the other hand, because cells in pinwheel centres have different orientation preferences, the surround normalization narrowed the orientation tuning and strengthened cross-orientation suppression. These effects are just the opposite of what physiological measurements demonstrate. It is worth noting that pinwheel centres constitute a small part of striate cortex. Therefore, our simulations indicate that surround normalization will generally broaden orientation tuning in cortical orientation maps, contrary to what is usually presumed in normalization models. - Making response functions more compressive in iso-orientation domains than pinwheel centres required using a normalization approach that was different from surround normalization. Therefore, we used a standard hyperbolic form of self-normalization, but replaced the conventional semi-saturation constant by the output of non-oriented inhibitory neurons with severely compressed response functions, which are common in the input layers of the visual cortex. It is noteworthy that in the absence of intra-cortical inhibition, response redundancy would be greater in iso-orientation domains than pinwheel centres. Therefore, the stronger normalization in iso-orientation domains has the additional benefit of equalizing redundancy across the cortical surface. - The importance of cells that extract contours and edges has been recognized since Hubel and Wiesel; however, extra-striate cells that are selective for specific classes of patterns and textures have only been reported recently. Our empirical results and model simulations suggest that cells in pinwheel centres provide suitable inputs for pattern selective cells, whereas cells in iso-orientation domains enhance sensitivity to edges. If excitatory connections between similar orientations dominate in the visual cortex of rodents, then these animals could have sharp orientation tuning with high contrast sensitivity and strong cross-orientation suppression even without orientation maps. Therefore, by lacking pinwheels, animals without orientation maps would have limited pattern recognition, but they could still rely on accurate detection of moving edges for survival.- - Recording - \"MUA was recorded from the primary visual cortex of six anaesthetized adult male cats via a 32 channel silicon probes with a 100u2009u03BCm inter-electrode distance (NeuroNexus Technologies). The impedance of each electrode was u223C 1u2009Mu03A9. Recorded signals were amplified and filtered by a computer running Plexon (Plexon). Offline software from Plexon was used to sort single units from MUA.\" - Animal surgery - \"Animals were sedated with ketamine (15u2009mgu2009kg) and acepromazine (0.2u2009 mgu2009kg), and anaesthetized with propofol (2u2009mgu2009kg). The animals were continuously infused with fluid (2u20133u2009mlu2009kgu2009h) for hydration, vecuronium bromide (0.2u2009mgu2009kgu2009h) for muscle paralysis, and sufentanil (10u201320u2009ngu2009kgu2009h) and propofol (u223C2u20135u2009mgu2009 kgu2009h) for anaesthesia through intravenous catheters placed in each hind leg. The eyes were treated with 1% atropine sulfate to dilate the pupil and 2% neosynephrine to relax the nictitating membrane. The refracted eyes were fit with contact lenses with a 3u2009mm pupil to protect the cornea and focus stimuli on the retina at 57u2009cm viewing distance. All vital signs were observed and maintained throughout the experiment, including temperature, electrocardiogram, blood pressure, expired CO2 and heart rate. Area 17 was exposed by removing the skull and dura overlaying the region. Electrode probes were inserted tangentially with a horizontal angle of less than 5xB0 at the centre of the gyrus. All procedures were performed in accordance to the guidelines of the U.S. Department of Agriculture and approved by the Institutional Animal Care and Use Committee at the State University of New York, College of Optometry.\" - Stimuli - \"We stimulated V1 neurons by monocular presentation of different visual stimuli on a 20-inch CRT monitor (Nokia 445Xpro, Salo). Two sequences of full field stimuli were generated and presented using Matlab routines in the Psychophysics Toolbox. The first set consisted of single sinusoidal gratings at eight equally spaced orientations: 0xB0, 22.5xB0, 45xB0, 67.5xB0, 90xB0, 112.5xB0, 135xB0 and 157.5xB0. Each stimulus was flashed at four different phases: 0, pi/2, pi and 3pi/2 radians, and at seven different contrasts: 1/64, 1/32, 1/16, 1/8, 1/4, 1/2 and 1. The second set consisted of plaids made by adding pairs of 50% contrast gratings in all possible phase and orientation combinations. Each image was flashed for 100u2009ms and followed by 100u2009ms of a blank mid-grey screen. In both sets, the image sequences were randomized and each image was presented 20 times. Multiple blocks of stimuli were run on individual cats.\" - Selection of sites - \"MUA sites used in subsequent analyses had to pass a number of criteria. First, they needed to pass a signal-to-noise ratio (>5), which was calculated as the ratio between mean response of the preferred orientation at full contrast and the s.d. of the least preferred orientation at the lowest contrast. This threshold was determined by careful inspection of peristimulus time histograms that were fit using the Bayesian adaptive regression splines method to get a smooth estimate of firing rate. They were also averaged across all phases and all trials. Response magnitude was quantified as the total number of spikes during stimulus presentation, that is, the area under the Bayesian adaptive regression splines curve. Recording sites were only included in the analyses if it was possible to calculate an LHI for that site. An accurate measure of LHI required at least three recording sites on either side of the reference site to pass the signal-to-noise ratio and contrast response fit requirements (Ru22650.95). Single units were isolated from sites where it was possible to calculate an LHI (we also required that the contrast response of the single units could be well fit by a Naka Rushton (Ru22650.95), and that the orientation tuning could be well fit by a Von Mises function (Ru22650.7)). Peristimulus time histograms for each plaid combination (averaged across all phase combinations and trials) were also fit with a smooth curve using the Bayesian adaptive regression splines method and the magnitude of the response was quantified in the same way as for gratings.\" - Contrast response - \"At each recording site, a contrast response function was fit to responses evoked by single gratings with the preferred orientation at varying contrasts. The classic Nakau2013Rushton formulation of the hyperbolic ratio was used:\" - \\'\\' - \"Here, f gives the response to a grating of contrast, c. Rmax is the maximal response, c50 is the semi-saturation constant and n is the exponent determining the shape of the function. Only recording sites at which the contrast response function could be well fit (Ru22650.95) were used for subsequent analysis. The parameters displayed on the figures were estimated from the best fitting curves. As f(1.0) can be different from Rmax, we calculated the maximum response as the fitted function output at 100% contrast and the semi-saturation constant as the contrast value at which the fitted function yields half of the function value at full contrast.\" - Orientation tuning width - Orientation tuning curves were fit to the full-contrast single-grating responses at each site with a Von Mises function. We used the half width at half height as a measurement for orientation tuning bandwidth and the stimulus orientation at the peak of the orientation tuning as the orientation preference. Based on the criterion on signal to noise and contrast response fit described above, all multi-units that remained in the analysis were generally well fit by the Von Mises function (mean R=0.9208). We also calculated the circular variance as an estimate of orientation tuning width. Results were fairly similar for the two measures. - Suppression index - \\'A SI was determined for each plaid containing the preferred orientation at each recording site:\\' - \\'\\' - \"R(u03B8p, u03B8i) represents the response to a plaid that includes the preferred grating, u03B8p, and R(u03B8p) represents the response to the preferred grating at 0.5 contrast. This SI highlights the fact that many plaids evoke responses smaller than their preferred component.\" - Local homogeneity index - \"The tangential insertion of the multi-electrode array and inter-electrode distance of 100u2009u03BCm allows us to estimate the location of each recording site within the orientation map. We used a one-dimensional (1D) version of the Nauhaus et al. LHI, to measure how quickly the orientation preference is changing around each site. The LHI for a given site was calculated by finding the magnitude of a vector addition. Each neighbouring neuron contributes a vector to the sum. The angle of the vector is given by the preferred orientation and the magnitude by a Gaussian weighting of cortical distance from the reference neuron. Explicitly the LHI for a given neuron is:\" - \\'\\' - \"The sum is over all neighbouring neurons. The distancej measures how far neuron j is from the given neuron in microns; u03A9j is the orientation preference of neuron j; u03C3 is set to 180u2009u03BCm (our measurements are spaced by 100u2009u03BCm); And k is a constant equal to the inverse of the theoretically maximum LHI, to ensure that LHI ranges from 0 to a maximum of 1. We use the same spatial Gaussian at all locations and analysed recording sites that were surrounded by other sites that passed our inclusion criteria. Therefore, all locations have the same number of elements contributing to the index.\" - We have checked our 1D estimates of homogeneity against two-dimensional (2D) estimates obtained from two published orientation maps and our own orientation map. First, we calculated the orientation maps (). Then, we compared the 2D LHI with the mean of eight 1D LHI measured at equally spaced penetration orientations in multiple locations (). Finally, we estimated the chance that recordings in pinwheel domains were misclassified as iso-orientation domains in our 1D recordings. We found that the 1D and 2D estimates are strongly correlated and are accurately described by a slightly curved function. Therefore, although the 1D LHI may slightly overestimate homogeneity, no area with 2D LHI of <0.2 will be classified as iso-orientation, as the 1D LHI will at most be 0.2 higher. These simulations show that linear arrays can be reliably used to identify and investigate orientation domains. - \"Figure 8: LHI computation simulations.(a) Three different orientation maps generated from optical imaging data from our lab (left) and data published by the Grinvald lab (middle and right). Scale bars below maps represent 1u2009mm in length. (b) Plots comparing 2D LHI calculation around 5,000 randomly chosen map locations with the average 1D LHI calculation of 8 radial angles of penetration. Solid grey lines depict the quadratic best fit polynomial. R values are given for this curve and the best-fit coefficients. Dashed grey lines depict the line of unity.\" - Nonlinearity index - \"To capture the saturation of the contrast responses curve with a single index, we joined the point of the curve corresponding to half the maximum response to the point of the curve corresponding to the maximum response. We then calculated the area between this line and the corresponding segment of the curve, positive for above the line and negative for below. We then normalized this value by the area between this line and the curve of maximum saturation. The curve of maximum saturation is formed by a vertical line from half the maximum response (half of the function output at 100% contrast) to the horizontal line at the maximum response (the function output at 100% contrast). Hence, the Nonlinearity Index ranged from u22121.0 for maximum expansion, to 0.0 for linearity, to 1.0 for maximum saturation.\" - Model specification - \"The luminance value of each pixel of a stimulus S(x,y) is passed through a point-wise Nakau2013Rushton function to get the photoreceptor output, P(x,y). Here, L50 is the semi-saturation luminance:\" - \\'\\' - \\'The photoreceptor output P(x,y) is convolved with the receptive fields of ON-Center and OFF-Center thalamic neurons, modelled as difference of Gaussian functions, D(x,y), to obtain the thalamic responses (Ton and Toff):\\' - \\'\\' - \\'\\' - \\'\\' - \"u03C31 is the s.d. of the receptive field centre and u03C32 is the s.d. of the surround (u03C31 is 1/3 of u03C32).\" - \"The total thalamic input (Ttot) to a cortical neuron with a preferred vertical orientation is formed by aligning multiple thalamic receptive fields in a vertical row and combining them with a pushu2013pull mechanism:\" - \\'\\' - \\'\\' - \\'\\' - where x1 and x2 define the vertical columns over which the i thalamic inputs are summed after they are rectified. The cortical orientation tuning was made narrower by increasing the number of thalamic neurons (i) and making longer vertical columns. Responses of neurons at other orientations were calculated by rotating the stimulus. - \\'The total thalamic input was then passed through a power-law spiking nonlinearity to get the cortical response, C:\\' - \\'\\' - A is a scaling constant adjusted to keep the response of the cortical neuron within a particular range and the exponent m is set to 2, which is within the range of measured values. - \\'We added distance-tuned excitation to the model by multiplying the cortical response by a sum of Gaussian weighted responses from orientation-tuned cells in the local cortical neighbourhood, CE:\\' - \\'\\' - \\'\\' - \"wj is the weight from each jth neuron from the cortical neighbourhood, given by a Gaussian with u03C3=180u2009u03BCm (as in the LHI calculation) and a constant k that is set so that all weights sum to 1.\" - \"The response of the cortical cell (CE) was then self-normalized using a Nakau2013 Rushton function. The normalization included also the output of a non-oriented inhibitory neuron CU, which acts as the semi-saturation constant in the Nakau2013 Rushton function. The normalized cortical response, CEN, is given by:\" - \\'\\' - \"Rmax is the maximum response of the neuron, CU is the response of an inhibitory un-oriented cell in the local cortical neighbourhood, u03B20 is a small number (set here to 0.01) that keeps the denominator above zero. u03B21 and u03B2 2 are weights given to each normalization term. These parameters can be manipulated in simulations, but were left at 1.0 for the predictions in .\" - It is important to note that the input to the model is exactly the same across the cortex, as are the parameters of local cortical excitation and normalization. The response differences in contrast saturation and orientation suppression arise solely because of the differences in the local distribution of orientation preferences between iso-orientation domains and pinwheel centres (). - \\'We used a basic form of response normalization for the predictions made in , using thalamic inputs rather than stimulus contrast. The thalamic contribution to the divisive normalization model is identical to that for the EN model. The cortical response from the thalamic afferents was divided by the weighted population response that was previously used for excitation in the EN Model. The response of a cortical neuron for the population normalization model,CN, is given by:\\' - \\'\\' - Here, all variables are as before. - Data availability - Source data available as .mat files and model matlab code is available as .m file upon request.- - \\'How to cite this article: Koch, E. et al. Functional implications of orientation maps in primary visual cortex. Nat. Commun. 7, 13529 doi: 10.1038/ncomms13529 (2016).\\' - \"Publishers note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \"Hubel, D. H. & Wiesel, T. N. Receptive fields, binocular interaction and functional architecture in the cats visual cortex. J. Physiol. 160, 106u2013154 (1962)._ _ _ Blasdel, G. G. & Salama, G. Voltage-sensitive dyes reveal a modular organization in monkey striate cortex. Nature 321, 579u2013 585 (1986)._ _ _ _ Bonhoeffer, T. & Grinvald, A. Iso-orientation domains in cat visual cortex are arranged in pinwheel-like patterns. Nature 353, 429u2013431 (1991)._ _ _ _ Ohki, K. et al. Highly ordered arrangement of single neurons in orientation pinwheels. Nature 442, 925u2013928 (2006)._ _ _ _ Miller, K. D. A model for the development of simple cell receptive fields and the ordered arrangement of orientation columns through activity-dependent competition between ON- and OFF-center inputs. J. Neurosci. 14, 409u2013441 (1994)._ _ _ Swindale, N. V. The development of topography in the visual cortex: a review of models. Network 7, 161u2013247 (1996)._ _ _ Paik, S. B. & Ringach, D. L. Retinal origin of orientation maps in visual cortex. Nat. Neurosci. 14, 919u2013925 (2011)._ _ _ Stevens, J. L., Law, J. S., Antolik, J. & Bednar, J. A. Mechanisms for stable, robust, and adaptive development of orientation maps in the primary visual cortex. J. Neurosci. 33, 15747u2013 15766 (2013)._ _ _ Van Hooser, S. D., Heimel, J. A., Chung, S., Nelson, S. B. & Toth, L. J. Orientation selectivity without orientation maps in visual cortex of a highly visual mammal. J. Neurosci. 25, 19u201328 (2005)._ _ _ Ohki, K., Chung, S., Ch\\'ng, Y. H., Kara, P. & Reid, R. C. Functional imaging with cellular resolution reveals precise micro-architecture in visual cortex. Nature 433, 597u2013603 (2005)._ _ _ _ Kaschube, M. et al. Universality in the evolution of orientation columns in the visual cortex. Science 330, 1113u2013 1116 (2010)._ _ _ _ Hashemi-Nezhad, M. & Lyon, D. C. Orientation tuning of the suppressive extraclassical surround depends on intrinsic organization of V1. Cereb. Cortex 22, 308u2013326 (2012)._ _ Carandini, M. & Sengpiel, F. Contrast invariance of functional maps in cat primary visual cortex. J. Vis. 4, 130u2013143 (2004)._ _ Kaschube, M. Neural maps versus salt-and-pepper organization in visual cortex. Curr. Opin. Neurobiol. 24, 95u2013102 (2014)._ _ _ Wilson, S. P. & Bednar, J. A. What, if anything, are topological maps for? Dev. Neurobiol. 75, 667u2013681 (2015)._ _ Kaas, J. H. Evolution of columns, modules, and domains in the neocortex of primates. Proc. Natl Acad. Sci. USA 109, (Suppl 1) 10655u2013 10660 (2012)._ _ _ Nauhaus, I., Benucci, A., Carandini, M. & Ringach, D. L. Neuronal selectivity and local map structure in visual cortex. Neuron 57, 673u2013679 (2008)._ _ _ Field, D. J., Hayes, A. & Hess, R. F. Contour integration by the human visual system: evidence for a local u2018association field. Vis. Res. 33, 173u2013 193 (1993)._ _ _ Goris, R. L., Simoncelli, E. P. & Movshon, J. A. Origin and function of tuning diversity in macaque visual cortex. Neuron 88, 819u2013831 (2015)._ _ _ Fleming, R. W., Holtmann-Rice, D. & Bulthoff, H. H. Estimation of 3D shape from image orientations. Proc. Natl Acad. Sci. USA 108, 20438u201320443 (2011)._ _ _ _ Freeman, J., Ziemba, C. M., Heeger, D. J., Simoncelli, E. P. & Movshon, J. A. A functional and perceptual signature of the second visual area in primates. Nat. Neurosci. 16, 974u2013981 (2013)._ _ _ Adelson, E. H. & Movshon, J. A. Phenomenal coherence of moving visual patterns. Nature 300, 523u2013525 (1982)._ _ _ _ Rust, N. C., Mante, V., Simoncelli, E. P. & Movshon, J. A. How MT cells analyze the motion of visual patterns. Nat. Neurosci. 9, 1421u20131431 (2006)._ _ _ Cohen, E. H. & Zaidi, Q. Symmetry in context: salience of mirror symmetry in natural patterns. J. Vis. 13, 22 (2013)._ _ Okazawa, G., Tajima, S. & Komatsu, H. Image statistics underlying natural texture selectivity of neurons in macaque V4. Proc. Natl Acad. Sci. USA 112, E351u2013E360 (2015)._ _ _ _ Li, A. & Zaidi, Q. Three-dimensional shape from non-homogeneous textures: carved and stretched surfaces. J. Vis. 4, 860u2013878 (2004)._ _ Cohen, E. H. & Zaidi, Q. Fundamental failures of shape constancy resulting from cortical anisotropy. J. Neurosci. 27, 12540u201312545 (2007)._ _ _ Li, A. & Zaidi, Q. Perception of three-dimensional shape from texture is based on patterns of oriented energy. Vis. Res. 40, 217u2013242 (2000)._ _ _ Morrone, M. C., Burr, D. C. & Maffei, L. Functional implications of cross-orientation inhibition of cortical visual cells. I. Neurophysiological evidence. Proc. R. Soc. Lond. B Biol. Sci. 216, 335u2013354 (1982)._ _ _ _ Blakemore, C. & Tobin, E. A. Lateral inhibition between orientation detectors in the cat\\'s visual cortex. Exp. Brain Res. 15, 439u2013440 (1972)._ _ _ Bonds, A. B. Role of inhibition in the specification of orientation selectivity of cells in the cat striate cortex. Vis. Neurosci. 2, 41u201355 (1989)._ _ _ Carandini, M., Heeger, D. J. & Senn, W. A synaptic explanation of suppression in visual cortex. J. Neurosci. 22, 10053u201310065 (2002)._ _ _ Li, B., Thompson, J. K., Duong, T., Peterson, M. R. & Freeman, R. D. Origins of cross-orientation suppression in the visual cortex. J. neurophysiol. 96, 1755u2013 1764 (2006)._ _ Priebe, N. J. & Ferster, D. Mechanisms underlying cross-orientation suppression in cat visual cortex. Nat. Neurosci. 9, 552u2013561 (2006)._ _ _ Albrecht, D. G. & Geisler, W. S. Motion selectivity and the contrast-response function of simple cells in the visual cortex. Vis. Neurosci. 7, 531u2013546 (1991)._ _ _ Kremkow, J. et al. Neuronal nonlinearity explains greater visual spatial resolution for darks than lights. Proc. Natl Acad. Sci. USA 111, 3170u20133175 (2014)._ _ _ _ Komban, S. J., Alonso, J. M. & Zaidi, Q. Darks are processed faster than lights. J. Neurosci. 31, 8654u20138658 (2011)._ _ _ Wool, L. E. et al. Salience of unique hues and implications for color theory. J. Vis. 15, 10 (2015)._ _ Baylor, D. A., Nunn, B. J. & Schnapf, J. L. Spectral sensitivity of cones of the monkey Macaca fascicularis. J. Physiol. 390, 145u2013160 (1987)._ _ _ Harris, K. D. & Mrsic-Flogel, T. D. Cortical connectivity and sensory coding. Nature 503, 51u2013 58 (2013)._ _ _ _ Ferster, D. Spatially opponent excitation and inhibition in simple cells of the cat visual cortex. J. Neurosci. 8, 1172u20131180 (1988)._ _ _ Hirsch, J. A., Alonso, J. M., Reid, R. C. & Martinez, L. M. Synaptic integration in striate cortical simple cells. J. Neurosci. 18, 9517u20139528 (1998)._ _ _ Priebe, N. J., Mechler, F., Carandini, M. & Ferster, D. The contribution of spike threshold to the dichotomy of cortical simple and complex cells. Nat. Neurosci. 7, 1113u20131122 (2004)._ _ _ Levy, M., Lu, Z., Dion, G. & Kara, P. The shape of dendritic arbors in different functional domains of the cortical orientation map. J. Neurosci. 34, 3231u20133236 (2014)._ _ _ Zhuang, J. et al. Layer 4 in primary visual cortex of the awake rabbit: contrasting properties of simple cells and putative feedforward inhibitory interneurons. J. Neurosci. 33, 11372u201311389 (2013)._ _ _ Kremkow, J., Jin, J., Wang, Y. & Alonso, J. M. Principles underlying sensory map topography in primary visual cortex. Nature 533, 52u201357 (2016)._ _ _ _ Heeger, D. J. Normalization of cell responses in cat striate cortex. Vis. Neurosci. 9, 181u2013197 (1992)._ _ _ Anderson, J. S., Carandini, M. & Ferster, D. Orientation tuning of input conductance, excitation, and inhibition in cat primary visual cortex. J. Neurophysiol. 84, 909u2013926 (2000)._ _ _ Priebe, N. J. & Ferster, D. Direction selectivity of excitation and inhibition in simple cells of the cat primary visual cortex. Neuron 45, 133u2013145 (2005)._ _ _ Monier, C., Chavane, F., Baudot, P., Graham, L. J. & Fregnac, Y. Orientation and direction selectivity of synaptic inputs in visual cortical neurons: a diversity of combinations produces spike tuning. Neuron 37, 663u2013680 (2003)._ _ _ Marino, J. et al. Invariant computations in local cortical networks with balanced excitation and inhibition. Nat. Neurosci. 8, 194u2013201 (2005)._ _ _ Liu, B. H. et al. Broad inhibition sharpens orientation selectivity by expanding input dynamic range in mouse simple cells. Neuron 71, 542u2013 554 (2011)._ _ _ Busse, L., Wade, A. R. & Carandini, M. Representation of concurrent stimuli by population activity in visual cortex. Neuron 64, 931u2013942 (2009)._ _ _ Ferster, D., Chung, S. & Wheat, H. Orientation selectivity of thalamic input to simple cells of cat visual cortex. Nature 380, 249u2013252 (1996)._ _ _ _ Li, Y. T., Ibrahim, L. A., Liu, B. H., Zhang, L. I. & Tao, H. W. Linear transformation of thalamocortical input by intracortical excitation. Nat. Neurosci. 16, 1324u20131330 (2013)._ _ _ Lien, A. D. & Scanziani, M. Tuned thalamic excitation is amplified by visual cortical circuits. Nat. Neurosci. 16, 1315u20131323 (2013)._ _ _ Sato, T. K., Haider, B., Hausser, M. & Carandini, M. An excitatory basis for divisive normalization in visual cortex. Nat. Neurosci. 19, 568u2013570 (2016)._ _ _ Carandini, M. & Heeger, D. J. Normalization as a canonical neural computation. Nat. Rev. Neurosci. 13, 51u201362 (2012)._ _ _ Xue, M., Atallah, B. V. & Scanziani, M. Equalizing excitation-inhibition ratios across visual cortical neurons. Nature 511, 596u2013600 (2014)._ _ _ _ Schwartz, O. & Simoncelli, E. P. Natural signal statistics and sensory gain control. Nat. Neurosci. 4, 819u2013825 (2001)._ _ _ Cossell, L. et al. Functional organization of excitatory synaptic strength in primary visual cortex. Nature 518, 399u2013403 (2015)._ _ _ _ Kass, R. E., Ventura, V. & Cai, C. Statistical smoothing of neuronal data. Network 14, 5u201315 (2003)._ _ Naka, K. I. & Rushton, W. A. An attempt to analyse colour reception by electrophysiology. J. Physiol. 185, 556u2013586 (1966)._ _ _ Shmuel, A. & Grinvald, A. Coexistence of linear zones and pinwheels within orientation maps in cat visual cortex. Proc. Natl Acad. Sci. USA 97, 5568u20135573 (2000)._ _ _ _ \"- - This study was supported by NIH grants EY05253, EY07556 and EY13312.- - Authors and Affiliations - \"Graduate Center for Vision Research, College of Optometry, State University of New York, 33 West 42nd Street, New York, 10036, New York, USAErin Koch,_Jianzhong Jin,_Jose M. Alonso_&_Qasim Zaidi\" - \"AuthorsErin KochYou can also search for this author in _Jianzhong JinYou can also search for this author in _Jose M. AlonsoYou can also search for this author in _Qasim ZaidiYou can also search for this author in _\" - Contributions - E.K., J.M.A. and Q.Z. designed the study. E.K., J.J. and J.M.A. performed and analysed the electrophysiological experiments. E.K. and Q.Z. made the computational model. E.K., J.M.A. and Q.Z. wrote the paper. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing financial interests. - \\' \\'- - Supplementary Figures 1 and 2 (PDF 98 kb)- - \\' \\' - \"This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution 3.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution 3.0 License (creativecommons.org/licenses/by/3.0/).\" - \"Consistent differences across multiple trophic guilds resulted in a positive effect of acoustic enrichment at the community level. Acoustically enriched reefs had twice the total abundance of juvenile fishes of dummy-loudspeaker and no-loudspeaker reefs, which were not significantly different from each other (LMM, loudspeaker treatment: u03C7u2009=u200921.107, dfu2009=u20092, pu2009 <u20090.001; Fig._; full model and post-hoc pairwise comparisons in Supplementary Table_). Acoustically enriched reefs also had 50% greater species richness than dummy-loudspeaker and no-loudspeaker reefs, with no significant difference between the two acoustically unmanipulated treatments (u03C7u2009=u200912.848, dfu2009=u20092, pu2009=u20090.002; Fig._; Supplementary Table_). Further, there was a significant effect of loudspeaker treatment on effective Shannon diversity, with acoustically enriched reefs having greater diversity than reefs with no loudspeaker; dummy-loudspeaker reefs were not significantly different to either of the other two treatments (u03C7u2009=u20095.990, dfu2009=u2009 2, pu2009=u20090.050; Fig._; Supplementary Table_). This development of a more abundant, more species-rich and more diverse fish community is potentially the result of increased detectability of acoustically enriched reefs (i.e. fishes could hear, and therefore detect and orient towards, reefs from a greater distance away), or altered settlement behaviour of juvenile fishes in response to additional sounds (i.e. fishes were more likely to settle onto the reef once they arrived at it), or a combination of both mechanisms.\" - \"Fig. 3Community-level effects of acoustic enrichment. a Total abundance, b species richness, and c effective Shannon diversity of juvenile fish communities on experimental patch reefs. Boxplots represent medians (central lines), interquartile ranges (boxes) and 95% ranges (whiskers). Linear mixed models revealed significant effects of loudspeaker treatment in all three cases; different letters above boxplots represent significant differences in post-hoc Tukeys HSD tests (Supplementary Table_). Absolute values for species richness and Shannon diversity are likely to be underestimates, as fish that were only identified to higher taxonomic levels (family or subfamily) may represent more than a single species.\"- - \"Our results demonstrate that acoustic enrichment has the potential to enhance community development of fishes on degraded coral reef habitat. Playback of healthy reef sound: (a) created temporally stable population increases in the most abundant taxonomic group of reef fishes (Pomacentridae; Fig._); (b) drove increases in fish recruitment across a broad range of trophic guilds (Fig._ ); and (c) led to increased abundance, species richness and diversity at the whole-community level (Fig._). The near-ubiquitous qualitative equivalence of communities associated with dummy-loudspeaker and no-loudspeaker control treatments demonstrates that observed increases on acoustically enriched reefs were due to the acoustic treatment, rather than the additional visual cues or structural complexity associated with the presence of loudspeakers. The current work did not attempt to investigate which sounds are most effective at attracting settlement-stage fishes. However, previous evidence from short-term trials shows_that healthy reef sound is more attractive than either degraded reef sound or white noise to settlement-stage fishes suggesting that the results of the current study are likely influenced by more than indiscriminate phonotaxis.\" - \"Acoustic enrichment may have effects on community dynamics at multiple spatial and temporal scales. First, many fishes migrate away from their initial settlement site during ontogeny. As such, even though loudspeakers inevitably create smaller acoustic halos than natural reefs, elevated settlement to acoustically enriched nucleus sites may promote fish community development at wider spatial scales. Additionally, initial increases in settlement driven by acoustic enrichment may facilitate a u2018snowball effect, whereby other fishes respond positively to communities established earlier, causing further increases in settlement. Finally, this community development might also cause natural soundscapes to increase in volume, because healthy reef ecosystems are louder and more acoustically complex than their degraded counterparts. These mechanisms all raise the possibility that the effects of acoustic enrichment could extend beyond those observed in the immediate vicinity of loudspeakers during their deployment, with benefits seen at larger spatial and temporal scales (Fig._).\" - \"Fig. 4Schematic illustrating the potential for acoustic enrichment to reverse harmful feedback loops on coral reefs. The left-hand cycle shows acoustically mediated feedback associated with degradation; the right-hand reverse cycle suggests how acoustic enrichment could facilitate ecosystem recovery through increasing recruitment and enhancing communities of fishes. Pictures of fishes are_adapted from ref. .\" - Acoustic enrichment shows potential as a sensory-based conservation tool for contributing to the restoration of coral reef ecosystems. Trials of sound playback in terrestrial contexts have previously revealed its potential to alter animal behaviour and increase settlement rates of acoustically specialised taxa. We show that a similar technique exhibits promise as a novel conservation tool for the management of degraded coral reefs. Existing management and restoration techniques can improve habitat quality in previously degraded areas; if combined with such techniques, acoustic enrichment might accelerate fish community development and enhance natural ecosystem recovery processes. Further work is now needed to investigate the translatability of this finding into different reef habitats and geographical contexts; the impacts of acoustic enrichment on adult fish behaviour; the long-term recovery of natural settlement cues; and the spatial scale of effects on fish communities and ecosystem processes. This will optimise acoustic enrichment as one of a suite of active management tools for restoring degraded coral reef ecosystems.- - Study site - \"This study was carried out during Octoberu2013December 2017 in the lagoon to the south-west of Lizard Island Research Station (14xB040.8u2032S, 145xB0 26.4u2032E; Fig._). Lizard Island is a continental mid-shelf island in the northern Great Barrier Reef with an extensive surrounding fringing and lagoonal reef system. In the two years preceding this study, severe mass bleaching events caused extensive coral mortality in this area (over 60% of live coral bleached), leading to widespread ecosystem change.\" - \"Fig. 5Study site map, showing experimental reefs and coral-rubble point. All experimental reefs were placed at a minimum distance of 100u2009m from their nearest neighbour, and at a fixed distance of 25u2009m from the nearest natural reef. Satellite image obtained from Google Maps, available at (Map data: Google, GBRMPA).\" - Experimental design - \"Prior to establishing experimental sites, the potential detection distance of reef-sound playback by juvenile fishes was determined using recordings at the study site. Recordings were taken from the deepest experimental reef (4.5u2009 m mid-tide depth), to provide a conservative estimate of acoustic isolation; lower frequency (longer-wavelength) sounds attenuate relatively quickly in shallow water, so the potential detection distance of reef-sound playback is likely to have been higher here than at all other sites. Full-night recordings of reef-sound playback were taken simultaneously in both the sound-pressure and particle-acceleration domains, at 1, 50 and 100u2009m distance from a loudspeaker. Loudspeaker playback was conducted using the same methods as in subsequent experimental trials, as described below. Recordings were made in sea states between 0 and 2 on the Beaufort scale, and never during rain. Recording equipment was suspended 1u2009m above the seabed, hanging from a submerged stand to avoid unwanted noise from waves slapping on the hull of a surface vessel. Sound pressure was recorded using an omnidirectional hydrophone with inbuilt digital recorder (SoundTrap 300 STD; Ocean Instruments NZ, Auckland, New Zealand), and particle-acceleration recordings were taken using a triaxial accelerometer (M20u2013040; sensitivity following a curve over the frequency range 0u20135u2009kHz; Geospectrum Technologies, Dartmouth, Canada) connected to a digital 8-track recorder (F8 field recorder; Zoom Corporation, Tokyo, Japan). Recordings were all taken at a sampling frequency of 48u2009kHz, and analysed using the paPAM (particle acceleration) and PAMGuide (sound pressure) packages on MATLAB, across a frequency range of 10u20134000u2009 Hz as the likely hearing range of many juvenile fishes, with a Hamming window, a 50% overlap and an FFT size of 2048.\" - \"Analysis of recordings of reef-sound playback could not distinguish a signal against the background noise floor at a distance of 50u2009m. Each recording was analysed both as a full-night track and as a series of 20 time-matched 2-min sub-samples, evenly spaced throughout the night (Fig._). Recordings showed that loudspeaker playback recorded at a distance of 1u2009m had significantly increased received sound-pressure level (SPL) and particle-acceleration level (PAL) relative to playback recorded from both 50 and 100u2009m, with visible differences in power spectral density plots (Fig._). By contrast, recordings taken at 50u2009m from the loudspeaker had power spectra and received SPLs and PALs that were equivalent to those taken at 100u2009m (Fig._). Further, intermittent recordings taken at 50u2009m from a loudspeaker showed no significant difference in received SPLs and PALs within the hearing range of young fishes (10u20134000u2009Hz) during 10-min periods of the night when the loudspeaker was turned on compared to equivalent 10-min periods when the loudspeaker was turned off (LMM, nu2009=u200920, SPL: u03C7u2009=u20092.785, dfu2009=u2009 1, pu2009=u20090.103; PAL: u03C7u2009=u20090.447, dfu2009=u20091, pu2009 =u20090.508). Thus, recordings of reef-sound playback taken at a distance of 50u2009m were unable to distinguish the playback signal over the natural ambient conditions at the study site. Based on this finding, all experimental patch reefs were placed a minimum of 100u2009m from each other (Fig._), to achieve acoustic isolation of experimental sounds between reefs.\" - \"Fig. 6Recordings of reef-sound playback taken at 1, 50 and 100u2009m from the loudspeaker. Shown are: a power spectral densities from a single full-night recording taken in sound-pressure; b boxplots of total received broadband (10u2013 4000u2009Hz) sound pressure level (SPL) from evenly spaced 2-min sub-samples throughout the night (nu2009=u200920); c power spectral densities similar to a but showing particle acceleration; and d boxplots of total received broadband (10u20134000u2009Hz) particle-acceleration levels (PAL) similar to b. Boxplots in b and d represent medians (central lines), interquartile ranges_(boxes), and 95% ranges_(whiskers).\" - \"Experimental coral-rubble patch reefs were used to assess the impact of acoustic enrichment on fish community development. At the start of the fish recruitment season, 33 patch reefs were built on open sand, placed at a fixed distance of 25u2009m (as determined by GPS) from the nearest natural reef in 2u20134.5u2009 m water depth (mid-tide depth; tidal range during experimentu2009xB1u20091.3u2009 m). There was no significant difference in depth between the three treatment groups (linear model: u03C7u2009=u20092.042, dfu2009=u20092, pu2009=u20090.15). Reefs consisted of 70u2009l of dead coral rubble, collected from a single degraded reef near the study site (Fig._), arranged around a double breeze block (40u2009 xD7u200940u2009xD7u200920u2009cm) to create a structurally complex habitat patch of 2u2009xD7u20092u2009xD7u20090.5u2009m. Each reef was composed of an approximately even volume of branching (60%), tabular (20%) and cuboidal/spherical (20%) rubble types. This created standardised patches of marginal reef habitat, to which many coral reef fish families have been seen to settle in earlier studies.\" - \"Reefs were assigned to one of three experimental treatments. Each reef was coupled with either no loudspeaker, a dummy loudspeaker system (to control for additional visual cues and structural complexity provided by loudspeaker systems), or a real loudspeaker system playing back healthy reef sound overnight (acoustic-enrichment treatment). Loudspeaker treatment was allocated pseudorandomly to reefs, avoiding allocation of the same treatment to reefs that were spatially adjacent. Acoustically enriched reefs were fitted with an underwater loudspeaker (University Sound UW-30; maximal output 156u2009dB and 1u2009u03BCPa at 1u2009m, frequency response 0.1u201310u2009kHz; Lubell Labs, Columbus, OH, USA) powered by an amplifier (M033N; 18u2009W, frequency response 0.04u201320u2009kHz; Kemo Electronic GmbH, Germany) and a 12u2009V 12u2009Ah sealed lead-acid battery, connected to an MP3 player (Clip Jam; SanDisk, Milpitas, CA, USA) that was powered by an external battery pack (MIX IT RockStar, 10,000u2009mAh; Belkin, Los Angeles, CA, USA). Loudspeakers were fixed to the centre of reefs and oriented upwards, to ensure even distribution of sound in all directions laterally. Amplification systems were housed in a barrel floating directly above reefs, attached by a rope to a breeze block. Dummy-loudspeaker reefs consisted of a concrete tile of the same size, shape and colour as the loudspeakers, fixed to the reef in the same manner as the loudspeakers, attached by a rope to a floating buoy of the same dimensions as the barrel used in loudspeaker systems.\" - \"The acoustic-enrichment treatment consisted of playback of recordings taken at a healthy reef in the middle of the study site in November 2015; this was before the study site experienced two severe mass bleaching events in 2016 and 2017 that caused extensive coral mortality and widespread ecosystem change throughout the region. Five different recordings were used to reduce pseudoreplication, with recordings being allocated to acoustically enriched reefs at random (each recording was allocated to 2u20133 reefs). Recordings used in playback were full-night recordings, taken using an omnidirectional hydrophone (HiTech HTI-96-MIN with inbuilt preamplifier, manufacturer-calibrated sensitivity u2212164.3u2009 dB re_1u2009Vu2009u03BCPa; frequency range 0.002u201330u2009kHz; calibrated by manufacturers; High Tech, Inc.) connected to a digital recorder (PCM-M10, 48u2009kHz sampling rate; Sony Corporation). The hydrophone was freely suspended 1u2009m above the seabed in water depth of 3.5u2009m, from a rope-anchored barrel that contained the recorder.\" - \"Reef-sound playback was conducted overnight. Fish settlement is predominantly a nocturnal behaviour; playback therefore started 0.5u20131.5u2009h before sunset and stopped 2u20135u2009h after sunrise the following morning. Playback was also matched by time of night to within 45u2009min of the original recording, such that playback tracks were playing approximately u2018real-time throughout the night. For an illustrative waveform and spectrogram of the reef-sound playback, see Supplementary Fig._. Equipment failure or rough weather conditions prevented loudspeaker deployment on some nights, but all acoustically enriched reefs had successful loudspeaker deployments on between 34 and 36 out of a total of 40 nights in the experiment.\" - Surveys of fish communities - \"The family Pomacentridae (damselfish) was surveyed regularly throughout reef deployment. The high abundance of damselfishes on coral reefs (up to 50% of reef fish communities) facilitates adequate statistical power to test for differences in community development, and their non-cryptic nature allows accurate surveying with minimal disturbance. Visual surveys by a SCUBA diver (T.A.C.G.) were used to monitor communities of juvenile damselfishes (those that had settled on reefs following a pelagic larval phase in the current season). The observer and dive buddy remained at least 1u2009m from the reef during surveys, in order to minimise disturbance to the community. Each reef was surveyed 10 times throughout a 40-day period that started immediately following construction, with 3u20139 days between consecutive surveys. The start of the survey period on each reef was staggered across a total duration of 10 days (i.e. construction and surveying of the final reef started 10 days after the first reef was built), to allow a single observer (T.A.C.G.) to complete surveys at the same experimental time point. Surveys took up to 1u2009h per reef; staggering was therefore necessary to allow for surveying of 33 patch reefs at the same experimental time points. Deployment order was counterbalanced such that the same number of reefs within each treatment were constructed and surveyed on each day.\" - \"After 40 days, the whole fish community on each reef was surveyed by dismantling the reef piece-by-piece. An observer using SCUBA (T.A.C.G.) checked each piece of rubble thoroughly, using dilute clove oil and a hand net to capture all juvenile fishes on the reef. The dive buddy kept a continuous watch for u2018stray fishes that attempted to escape across the sand flat or burrow into the sand during the survey process. All reefs were double-checked for missed fishes after surveys were completed; to our knowledge, no fishes were missed from surveys. All fishes were identified to species, except in cases where uncertainty meant that identification was only possible to family or sub-family level. In these cases, fishes that looked similar were assigned as the same species (e.g. u2018 Unknown goby 1\\'); this was the case for 4% of species, whose members together constituted 9% of the total abundance. Adult fishes were excluded from analyses, as their larger home ranges mean that they do not exhibit fixed associations with reef habitat to the same extent as juveniles. After the experiment, all fish were released alive onto neighbouring reefs.\" - Statistical analyses - Visual surveys throughout the 40-day period were used to create accumulation curves for juvenile damselfishes (family Pomacentridae). A generalised additive mixed model (GAMM) with a negative binomial distribution included time (days since patch reef creation) as a smooth term, loudspeaker treatment (acoustic enrichment, dummy loudspeaker, no loudspeaker) and patch-reef ID as parametric coefficients, and playback-track ID as a random term. A first-order autoregressive structure was used to account for temporal autocorrelation, and visual examination of diagnostic plots and comparison of model outputs to raw data were used to confirm goodness-of-fit. - \"Comprehensive surveys at the end of the 40-day period were analysed using LMMs and Poisson-distributed GLMMs. Separate models were run for abundance of juvenile damselfishes (family Pomacentridae), herbivores, omnivores, planktivores, invertivores and piscivores, as well as total community abundance, species richness and effective Shannon diversity (calculated as the exponential of the Shannonu2013Weiner index). Trophic classification was based on published literature and FishBase; for full details, see Supplementary Table_. For fish that were only identified to family or sub-family, trophic guilds were assigned to all species within the relevant taxonomic group that are known to occur in the Lizard Island area (using lists compiled by the Lizard Island Field Guide [lifg.australianmuseum.net.au]), and the most commonly occurring guild in the taxonomic group was chosen as the trophic guild for that individual. Corallivores made up <0.25% of all fish, and were found on only two of 33 reefs, so were excluded from this analysis due to a lack of statistical power. All other trophic guilds represented at least 4% of the total count and were found on at least 19 of 33 reefs. In each model, playback-track ID was included as a random term, and error distributions (Gaussian LMM or Poisson-distributed GLMM) were chosen such that there were no deviations from homoscedasticity or normality in visual examinations of residual plots. The effect of loudspeaker treatment on the dependent variable was tested through ANOVA comparisons with a null model and post-hoc Tukeys HSD testing following statistically significant initial results (pu2009u2264u20090.05).\" - All figure creation and statistical modelling was conducted in R v. 3.5.0. Figures were prepared using the packages cowplot, ggmap, and ggplot2. Statistical modelling was conducted using the packages lme4 and mgcv. - Ethical approval - Permission and ethical approval for this work was granted by Lizard Island Research Station, the Great Barrier Reef Marine Park Authority (G13/35909.1), James Cook University (A2408, A2361), and the University of Exeter (2013/247). - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - \"Raw data are available from the University of Exeters institutional repository at .\" - \\' \\'- - \"Hughes, T. P. et al. Spatial and temporal patterns of mass bleaching of corals in the Anthropocene. Science 359, 80u201383 (2018)._ _ _ _ _ Harborne, A., Rogers, A., Bozec, Y.-M. & Mumby, P. J. Multiple stressors and the functioning of coral reefs. Annu. Rev. Mar. Sci. 9, 445u2013468 (2017).Graham, N. A. J. et al. Managing resilience to reverse phase shifts in coral reefs. Front. Ecol. Environ. 11, 541u2013548 (2013)._ _ Hughes, T. P. et al. Coral reefs in the Anthropocene. Nature 546, 82u201390 (2017)._ _ _ _ _ Anthony, K. et al. New interventions are needed to save coral reefs. Nat. Ecol. Evol. 1, 1420u20131422 (2017)._ _ _ Mouillot, D., VillxE9ger, S., Scherer-Lorenzen, M. & Mason, N. W. H. Functional structure of biological communities predicts ecosystem multifunctionality. PLoS ONE 6, e17476 (2011)._ _ _ _ _ _ Hemingson, C. R. & Bellwood, D. R. Biogeographic patterns in major marine realms: function not taxonomy unites fish assemblages in reef, seagrass and mangrove systems. Ecography 41, 174u2013182 (2018)._ _ Hughes, T. P., Graham, N. A. J., Jackson, J. B. C., Mumby, P. J. & Steneck, R. S. Rising to the challenge of sustaining coral reef resilience. Trends Ecol. Evol. 25, 633u2013 642 (2010)._ _ _ Graham, N. A. J., Jennings, S., MacNeil, M. A., Mouillot, D. & Wilson, S. K. Predicting climate-driven regime shifts versus rebound potential in coral reefs. Nature 518, 94u201397 (2015)._ _ _ _ _ Leis, J. M., Siebeck, U. & Dixson, D. L. How nemo finds home: the neuroecology of dispersal and of population connectivity in larvae of marine fishes. Integr. Comp. Biol. 51, 826u2013843 (2011)._ _ _ Doherty, P. & Fowler, T. An empirical test of recruitment limitation in a coral reef fish. Science 263, 935u2013939 (1994)._ _ _ _ _ Dixson, D. L., Abrego, D. & Hay, M. E. Chemically mediated behavior of recruiting corals and fishes: a tipping point that may limit reef recovery. Science 345, 892u2013897 (2014)._ _ _ _ _ _ Gordon, T. A. C. et al. Habitat degradation negatively affects auditory settlement behavior of coral reef fishes. Proc. Natl Acad. Sci. USA 115, 5193u20135198 (2018)._ _ _ _ _ Feary, D. A., Almany, G. R., McCormick, M. I. & Jones, G. P. Habitat choice, recruitment and the response of coral reef fishes to coral degradation. Oecologia 153, 727u2013737 (2007)._ _ _ _ Pratchett, M. S. et al. Effects of climate-induced coral bleaching on coral-reef fishesu2014ecological and economic consequences. Oceanogr. Mar. Biol. Annu. Rev. 46, 251u2013296 (2008)._ Parmentier, E. et al. The influence of various reef sounds on coral-fish larvae behaviour. J. Fish. Biol. 86, 1507u20131518 (2015)._ _ _ _ Simpson, S. D., Meekan, M. G., Montgomery, J., McCauley, R. D. & Jeffs, A. Homeward sound. Science 308, 221 (2005)._ _ _ _ Ackerman, J. & Bellwood, D. Reef fish assemblages: a re-evaluation using enclosed rotenone stations. Mar. Ecol. Prog. Ser. 206, 227u2013 237 (2000)._ _ _ Goatley, C. H. R. & Bellwood, D. R. Body size and mortality rates in coral reef fishes: a three-phase relationship. Proc. R. Soc. B 283, (2016)._ _ _ _ Doherty, P. J. et al. High mortality during settlement is a population bottleneck for a tropical surgeonfish. Ecology 85, 2422u20132428 (2004)._ _ Bellwood, D. R., Hughes, T. P. & Hoey, A. S. Sleeping functional group drives coral-reef recovery. Curr. Biol. 16, 2434u20132439 (2006)._ _ _ _ Holles, S. H., Simpson, S. D., Radford, A. N., Berten, L. & Lecchini, D. Boat noise disrupts orientation behaviour in a coral reef fish. Mar. Ecol. Prog. Ser. 485, 295u2013300 (2013)._ _ _ Nagelkerken, I., Dorenbosch, M., Verberk, W. C. E. P., Cocheret de la Moriniere, E. & Van der Velde, G. Importance of shallow-water biotopes of a Caribbean bay for juvenile coral reef fishes: patterns in biotope association, community structure and spatial distribution. Mar. Ecol. Prog. Ser. 202, 175u2013 192 (2000)._ _ _ Jones, G. P. Some interactions between residents and recruits in two coral reef fishes. J. Exp. Mar. Biol. Ecol. 114, 169u2013182 (1987)._ _ Booth, D. J. Larval settlement patterns and preferences by domino damselfish Dascyllus albisella Gill. J. Exp. Mar. Biol. Ecol. 155, 85u2013104 (1992)._ _ Friesen, M. R., Beggs, J. R. & Gaskett, A. C. Sensory-based conservation of seabirds: a review of management strategies and animal behaviours that facilitate success. Biol. Rev. 92, 1769u20131784 (2017)._ _ _ Molles, L. E. et al. u2018Acoustic anchoring and the successful translocation of North Island kokako (callaeas cinerea wilsoni) to a New Zealand mainland management site within continuous forest. Notornis 55, 57u201368 (2008)._ Kiffner, C., Waltert, M., Meyer, B. & Muhlenberg, M. Response of lions (Panthera leo LINNAEUS 1758) and spotted hyaenas (Crocuta crocuta ERXLEBEN 1777) to sound playbacks. Afr. J. Ecol. 46, 223u2013226 (2007)._ _ Ladd, M. C., Burkepile, D. E. & Shantz, A. A. Near-term impacts of coral restoration on target species, coral reef community structure, and ecological processes. Rest. Ecol. (2019)._ _ Williams, S. L. et al. Large-scale coral reef rehabilitation after blast fishing in Indonesia. Rest. Ecol. 27, 447u2013456 (2019)._ _ Hughes, T. P. et al. Global warming and recurrent mass bleaching of corals. Nature 543, 373u2013 377 (2017)._ _ _ _ _ Stuart-Smith, R. D., Brown, C. J., Ceccarelli, D. M. & Edgar, G. J. Ecosystem restructuring along the Great Barrier Reef following mass coral bleaching. Nature 560, 92u2013 96 (2018)._ _ _ _ _ Richardson, L. E., Graham, N. A. J., Pratchett, M. S., Eurich, J. G. & Hoey, A. S. Mass coral bleaching causes biotic homogenization of reef fish assemblages. Glob. Chang. Biol. 24, 3117u20133129 (2018)._ _ _ _ Nedelec, S. L. et al. Particle motion: the missing link in underwater acoustic ecology. Methods Ecol. Evol. 7, 836u2013842 (2016)._ _ Merchant, N. D. et al. Measuring acoustic habitats. Methods Ecol. Evol. 6, 257u2013265 (2015)._ _ _ _ Wright, K. J., Higgs, D. M., Cato, D. H. & Leis, J. M. Auditory sensitivity in settlement-stage larvae of coral reef fishes. Coral Reefs 29, 235u2013243 (2010)._ _ _ Wright, K. J., Higgs, D. M. & Leis, J. M. Ontogenetic and interspecific variation in hearing ability in marine fish larvae. Mar. Ecol. Prog. Ser. 424, 1u201313 (2011)._ _ _ Tolimieri, N., Haine, O., Jeffs, A., McCauley, R. & Montgomery, J. Directional orientation of pomacentrid larvae to ambient reef sound. Coral Reefs 23, 184u2013191 (2004)._ _ Dufour, V. & Galzin, R. Colonization patterns of reef fish larvae to the lagoon at Moorea Island, French Polynesia. Mar. Ecol. Ser. 102, 143 (1993)._ _ Welsh, J. Q., Goatley, C. H. R. & Bellwood, D. R. The ontogeny of home ranges: Evidence from coral reef fishes. Proc. R. Soc. B 280, 1u20137 (2013)._ _ Jost, L. Entropy and diversity. Oikos 113, 363u2013375 (2006)._ _ Richardson, L. E., Graham, N. A. J., Pratchett, M. S. & Hoey, A. S. Structural complexity mediates functional structure of reef fish assemblages among coral habitats. Environ. Biol. Fishes 100, 193u2013207 (2017)._ _ Wilson, S. K. et al. Exploitation and habitat degradation as agents of change within coral reef fish communities. Glob. Chang. Biol. 14, 2796u20132809 (2008)._ _ _ Pratchett, M. S., Hoey, A. S., Wilson, S. K., Messmer, V. & Graham, N. A. J. Changes in biodiversity and functioning of reef fish assemblages following coral bleaching and coral loss. Diversity 3, 424u2013452 (2011)._ _ Green, A. L. & Bellwood, D. R. Monitoring functional groups of herbivorous reef fishes as indicators of coral reef resilienceu2014 a practical guide for coral reef managers in the Asia Pacific Region. In IUCN Working Group on Climate Change and Coral Reefs (IUCN, Gland, Switzerland, 2009). .Froese, R. & Pauly, D. FishBase (2017) (accessed 1 August 2018); .R Core Team. R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, Vienna, Austria, 2018).Wilke, C. O. Cowplot: streamlined plot theme and plot annotations for u2018ggplot2. R package version 0.9.2 (2017).Kahle, D. & Wickham, H. ggmap: spatial visualization with ggplot2. R. J. 5, 144u2013 161 (2013)._ _ Wickham, H. ggplot2: Elegant Graphics for Data Analysis (Springer-Verlag, New York, 2016).Bates, D., Maechler, M., Bolker, B. & Walker, S. Fitting linear mixed-effects models using lme4. J. Stat. Softw. 67, 1u201348 (2015)._ _ Wood, S. Fast stable restricted maximum likelihood and marginal likelihood estimation of semiparametric generalized linear models. J. R. Stat. Soc. 73, 3u201336 (2011)._ _ _ _ \"- - \"We thank the staff at Lizard Island Research Station for logistical support; Brendan Nedelec and Maggie Travis for fieldwork assistance; Howard Choat, Kendall Clements and William Robbins for assistance with fish identification; James Campbell for assistance with acoustic analyses; Danielle Dixson for drawings of fishes used in Fig. ; and Patick Kennedy,_Harry Harding and Katherine Maltby for statistical advice. This work was supported by funding from a Natural Environment Research Council Research Grant NE/P001572/1 (to S.D.S. and A.N.R.); a Natural Environment Research Council-Australian Institute of Marine Science CASE GW4+ Studentship NE/L002434/1 (to T.A.C.G.); an Australian Research Council Discovery Grant DP170103372 (to M.I.M.); the Australian Institute of Marine Science (to M.G.M.); and the University of Exeter (to S.D.S.).\"- - Authors and Affiliations - \"Biosciences, University of Exeter, Hatherly Laboratories, Prince of Wales Road, Exeter, EX4 4PS, UKTimothy A. C. Gordon,_Kieran McCloskey,_Sophie L. Nedelec_ &_Stephen D. SimpsonAustralian Institute of Marine Science, Perth, WA, 6009, AustraliaTimothy A. C. Gordon_&_Mark G. MeekanSchool of Biological Sciences, University of Bristol, 24 Tyndall Avenue, Bristol, BS8 1TQ, UKAndrew N. Radford_ &_Isla K. DavidsonDepartment of Marine Biology and Aquaculture, James Cook University, Townsville, QLD 4811, AustraliaKasey Barnes_&_Mark I. McCormickAustralian Research Council Centre of Excellence for Coral Reef Studies, James Cook University, Townsville, QLD 4811, AustraliaMark I. McCormick\" - \"AuthorsTimothy A. C. GordonYou can also search for this author in _Andrew N. RadfordYou can also search for this author in _Isla K. DavidsonYou can also search for this author in _Kasey BarnesYou can also search for this author in _Kieran McCloskeyYou can also search for this author in _Sophie L. NedelecYou can also search for this author in _Mark G. MeekanYou can also search for this author in _Mark I. McCormickYou can also search for this author in _Stephen D. SimpsonYou can also search for this author in _\" - Contributions - T.A.C.G., A.N.R., S.L.N., M.G.M., M.I.M. and S.D.S. designed the research; T.A.C.G., I.K.D., K.B., K.M., S.L.N. and S.D.S. performed the fieldwork; T.A.C.G., A.N.R. and S.D.S. analysed the data; T.A.C.G. wrote the initial manuscript and all authors contributed to its final revision. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 License. Photos of Cyphomyrmex (Figs._ and ) are used under a perpetual commercial license from Alexander Wild.\" - \\' \\'- - \"Sprenger, M. & Fukuda, K. New mechanisms, new worries. Science 351, 1263u2013 1264 (2016)._ _ _ _ Brown, E. D. & Wright, G. D. Antibacterial drug discovery in the resistance era. Nature 529, 336u2013 343 (2016)._ _ _ Newman, D. J. & Cragg, G. M. Natural products as sources of new drugs from 1981 to 2014. J. Nat. Prod. 79, 629u2013 661 (2016)._ _ _ Lewin, G. R. et al. Evolution and ecology of actinobacteria and their bioenergy applications. Annu. Rev. Microbiol. 70, 235u2013254 (2016)._ _ _ Baltz, R. H. Marcel Faber Roundtable: is our antibiotic pipeline unproductive because of starvation, constipation or lack of inspiration? J. Ind. Microbiol. Biotechnol. 33, 507u2013 513 (2006)._ _ _ Medema, M. H., Cimermancic, P., Sali, A., Takano, E. & Fischbach, M. A. A systematic computational analysis of biosynthetic gene cluster evolution: lessons for engineering biosynthesis. PLoS Comput. Biol. 10, e1004016 (2014)._ _ _ Doroghazi, J. R. et al. A roadmap for natural product discovery based on large-scale genomics and metabolomics. Nat. Chem. Biol. 10, 963u2013968 (2014)._ _ _ Smanski, M. J. et al. Synthetic biology to access and expand natures chemical diversity. Nat. Rev. Microbiol. 14, 135u2013149 (2016)._ _ _ Udwary, D. W. et al. Genome sequencing reveals complex secondary metabolome in the marine actinomycete Salinispora tropica. Proc. Natl Acad. Sci. USA 104, 10376u201310381 (2007)._ _ _ _ Ziemert, N. et al. Diversity and evolution of secondary metabolism in the marine actinomycete genus Salinispora. Proc. Natl Acad. Sci. USA 111, E1130u2013E1139 (2014)._ _ _ Jang, K. H. et al. Anthracimycin, a potent anthrax antibiotic from a marine-derived actinomycete. Angew. Chem. Int. Ed. 52, 7822u20137824 (2013)._ _ _ Schulze, C. J. et al. Genome-directed lead discovery: biosynthesis, structure elucidation, and biological evaluation of two families of polyene macrolactams against Trypanosoma brucei. ACS Chem. Biol. 150813113920008 (2015). Ling, L. L. et al. A new antibiotic kills pathogens without detectable resistance. Nature 517, 455u2013459 (2015)._ _ _ _ Pye, C. R., Bertin, M. J., Lokey, R. S., Gerwick, W. H. & Linington, R. G. Retrospective analysis of natural products provides insights for future discovery trends. Proc. Natl Acad. Sci. USA 114, 5601u20135606 (2017)._ _ _ Miller, I., Chevrette, M. & Kwan, J. Interpreting microbial biosynthesis in the genomic age: biological and practical considerations. Mar. Drugs 15, 165 (2017)._ _ Clardy, J., Fischbacha, M. & Currie, C. R. The natural history of antibiotics. Curr. Biol. 19, 1u20138 (2009)._ _ Adnani, N. et al. Coculture of marine invertebrate-associated bacteria and interdisciplinary technologies enable biosynthesis and discovery of a new antibiotic, keyicin. ACS Chem. Biol. 12, 3093u20133102 (2017)._ _ _ Zipperer, A. et al. Human commensals producing a novel antibiotic impair pathogen colonization. Nature 535, 511u2013516 (2016)._ _ _ _ Oh, D.-C., Poulsen, M., Currie, C. R. & Clardy, J. Dentigerumycin: a bacterial mediator of an ant-fungus symbiosis. Nat. Chem. Biol. 5, 391u2013393 (2009)._ _ _ Currie, C. R. et al. Ancient tripartite coevolution in the attine ant-microbe symbiosis. Science 299, 386u2013388 (2003)._ _ _ _ VanArnam, E. B. et al. Selvamicin, an atypical antifungal polyene from two alternative genomic contexts. Proc. Natl Acad. Sci. USA 113, 12940u2013 12945 (2016)._ _ _ Kroiss, J. et al. Symbiotic streptomycetes provide antibiotic combination prophylaxis for wasp offspring. Nat. Chem. Biol. 6, 261u2013263 (2010)._ _ _ Scott, J. J. et al. Bacterial protection of beetle-fungus mutualism. Science 322, 63u201363 (2008)._ _ _ _ Blodgett, J. A. V. et al. Common biosynthetic origins for polycyclic tetramate macrolactams from phylogenetically diverse bacteria. Proc. Natl Acad. Sci. USA 107, 11692u201311697 (2010)._ _ _ _ Poulsen, M., Oh, D. C., Clardy, J. & Currie, C. R. Chemical analyses of wasp-associated Streptomyces bacteria reveal a prolific potential for natural products discovery. PLoS One 6, e16763 (2011).Kim, K. H. et al. Natalamycin A, an Ansamycin from a termite-associated Streptomycessp. Chem. Sci. 5, 4333u20134338 (2014)._ _ _ Oh, D. C., Poulsen, M., Currie, C. R. & Clardy, J. Sceliphrolactam, a polyene macrocyclic lactam from a wasp-associated Streptomyces sp. Org. Lett. 13, 752u2013755 (2011)._ _ _ Carr, G. et al. Microtermolides A and B from termite-associated Streptomyces sp. and structural revision of vinylamycin. Org. Lett. 14, 2822u20132825 (2012)._ _ _ Stork, N. E., McBroom, J., Gely, C. & Hamilton, A. J. New approaches narrow global species estimates for beetles, insects, and terrestrial arthropods. Proc. Natl Acad. Sci. USA 112, 7519u20137523 (2015)._ _ _ _ Eisner, T. & Meinwald, J. Defensive secretions of arthropods. Science 153, 1341u20131350 (1966)._ _ _ _ Book, A. J. et al. Evolution of high cellulolytic activity in symbiotic Streptomyces through selection of expanded gene content and coordinated gene expression. PLoS Biol. 14, 1u201321 (2016)._ _ Chevrette, M. G. & Currie, C. R. Emerging evolutionary paradigms in antibiotic discovery. J. Ind. Microbiol. Biotechnol. (2018). Hug, J., Bader, C., Remu0161kar, M., Cirnski, K. & MxFCller, R. Concepts and methods to access novel antibiotics from actinomycetes. Antibiotics 7, 44 (2018)._ _ McDonald, B. R. & Currie, C. R. Lateral gene transfer dynamics in the ancient bacterial genus Streptomyces. mBio 8, e00644u201317 (2017)._ _ _ Blin, K. et al. antiSMASH 4.0u2014improvements in chemistry prediction and gene cluster boundary identification. Nucleic Acids Res. 1854, 1019u2013 1037 (2017)._ Navarro-MuxF1oz, J. et al. A computational framework for systematic exploration of biosynthetic diversity from large-scale genomic data._BioRxiv_(2018) Preprint at .Laatsch, H. Antibase: The Natural Compound Identifier (Wiley-VCH, Weinheim, 2014).PxE9rez, M. et al. PM100117 and PM100118, new antitumor macrolides produced by a marine Streptomyces caniferus GUA-06-05-006A. J. Antibiot. 69, 388u2013394 (2016)._ _ Takeuchi, T. et al. ATP depletion assay led to the isolation of new 36-membered polyol macrolides Deplelides A and B from Streptomyces sp. MM581-NF15. Org. Lett. 19, 4207u2013 4210 (2017)._ _ _ Zhao, M. et al. In vivo pharmacokinetics and pharmacodynamics of APX001 against Candida spp. in a neutropenic disseminated candidiasis mouse model. Antimicrob. Agents Chemother. 62, e02542u201317 (2018)._ _ _ _ Zhao, M., Lepak, A. J. & Andes, D. R. Animal models in the pharmacokinetic / pharmacodynamic evaluation of antimicrobial agents. Bioorg. Med. Chem. 24, 6390u20136400 (2016)._ _ _ Andes, D. R. & Lepak, A. J. In vivo infection models in the pre-clinical pharmacokinetic/pharmacodynamic evaluation of antimicrobial agents. Curr. Opin. Pharmacol. 36, 94u201399 (2017)._ _ _ Payne, D. J. Microbiology. Desperately seeking new antibiotics. Science 321, 1644u2013 1645 (2008)._ _ _ Fisher, M. C., Hawkins, N. J., Sanglard, D. & Gurr, S. J. Worldwide emergence of resistance to antifungal drugs challenges human health and food security. Science 360, 739u2013742 (2018)._ _ _ Fischbach, M. A. & Walsh, C. T. Antibiotics for emerging pathogens. Science 325, 1089u20131093 (2009)._ _ _ _ Hayakawa, M. & Nonomura, H. Humic acid-vitamin agar, a new medium for the selective isolation of soil actinomycetes. J. Ferment. Technol. 65, 501u2013509 (1987)._ _ _ Hanshew, A. S. et al. Characterization of actinobacteria associated with three antu2013 plant mutualisms. Microb. Ecol. 69, 192u2013203 (2015)._ _ Li, Q., Chen, X., Jiang, Y. & Jiang, C. in Actinobacteria - Basics and Biotechnological Applications (ed. Dhanasekaran, D.) (InTech, 2016). London, United Kingdom.Katoh, K. & Standley, D. M. MAFFT Multiple Sequence Alignment Software Version 7: improvements in performance and usability. Mol. Biol. Evol. 30, 772u2013780 (2013)._ _ _ Price, M. N., Dehal, P. S. & Arkin, A. P. FastTree 2 u2013 approximately maximum-likelihood trees for large alignments. PLoS One 5, e9490 (2010)._ _ _ Benjamini, Y. & Yekutieli, D. The control of the false discovery rate in multiple testing under dependency. Ann. Stat. 29, 1165u20131188 (2001)._ _ _ Kumar, L. & Futschik, M. E. Mfuzz: a software package for soft clustering of microarray data. Bioinformation 2, 5u20137 (2007)._ _ Liu, Y., SchrxF6der, J. & Schmidt, B. Musket: a multistage k-mer spectrum-based error corrector for Illumina sequence data. Bioinformatics 29, 308u2013315 (2013)._ _ _ Magou010D, T. & Salzberg, S. L. FLASH: fast length adjustment of short reads to improve genome assemblies. Bioinformatics 27, 2957u20132963 (2011)._ _ Bankevich, A. et al. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 19, 455u2013477 (2012)._ _ _ _ Hyatt, D. et al. Prodigal: prokaryotic gene recognition and translation initiation site identification. BMC Bioinforma. 11, 119 (2010)._ _ Eddy, S. R. Accelerated profile HMM searches. PLoS Comput. Biol. 7, e1002195 (2011)._ _ _ _ _ Stamatakis, A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics 30, 1312u2013 1313 (2014)._ _ _ Medema, M. H. et al. Minimum information about a biosynthetic gene cluster. Nat. Chem. Biol. 11, 625u2013631 (2015)._ _ _ Gritti, F. et al. Performance of columns packed with the new shell particles, Kinetex-C18. J. Chromatogr. A. 1217, 1589u2013 1603 (2010)._ _ _ Nielsen, K. F., MxE5nsson, M., Rank, C., Frisvad, J. C. & Larsen, T. O. Dereplication of microbial natural products by LC-DAD-TOFMS. J. Nat. Prod. 74, 2338u20132348 (2011)._ _ _ \"- - \"The authors would like to thank Daniel Udwary, Christina Hansen, Lindsay Kalan, Reed Stubbendieck, Charlotte Francoeur, Hongjie Li, and Jon Clardy for critical appraisal of the manuscript, and Steve Giles, Janet Newlands, Rachel Arango, Tim Aspenson, Miranda Dam, Josh Daniels, Daniel DeSautels, Zachary Dumar, Athena Golfinos, Megan Hunjadi, Samuel Marquardt, Cathryn Phaybandyt, Mitchell Prokop, Shelby Sandstrom, Mia Temkin, and Aijan Urmat for technical help; This project was supported through National Institutes of Health (NIH) U19 Al109673, NIH U19 TW009872, National Science Foundation MCB-0702025, Sistema de Estudios de Posgrado y VicerrectorxEDa de InvestigacixF3n de la Universidad de Costa Rica (research projects 801-B0u2013538 and 810-B3u2013185), and SxE3o Paulo Research Foundation FAPESP grants #2013/50954u20130 (FAPESP/FIC-NIH); #2014/14095u2013 6 (HEO); #2015/01001u20136 (WGPM). Additional support was provided to MGC through NIH National Research Service Award T32 GM008505.\"- - Authors and Affiliations - \"Laboratory of Genetics, University of Wisconsin-Madison, Madison, 53706, WI, USAMarc G. ChevretteDepartment of Bacteriology, University of Wisconsin-Madison, Madison, 53706, WI, USAMarc G. Chevrette,_Caitlin M. Carlson,_Adam J. Book,_ Julian Cagnazzo,_Camila Carlos,_Will Flanigan,_Kirk J. Grubbs,_Heidi A. Horn,_ Bradon R. McDonald,_Laura Muller,_Amber Schmitz,_Evelyn Wendt-Pienkowski_ &_Cameron R. CurrieSchool of Pharmaceutical Sciences of RibeirxE3o Preto, University of SxE3o Paulo, RibeirxE3o Preto, 14040-903, SP, BrazilHumberto E. Ortega,_ Weilan G. P. Melo_&_Monica T. PupoPharmaceutical Sciences Division, School of Pharmacy, University of Wisconsin-Madison, Madison, 53705, WI, USAChris Thomas,_ Kenneth J. Barns,_Scott Wildman,_Fan Zhang_&_Tim S. BugniMcArdle Laboratory for Cancer Research, Wisconsin Institute for Medical Research, University of Wisconsin-Madison, Madison, 53705, WI, USAGene E. Ananiev_&_F. Michael HoffmannDepartment of Molecular and Cell Biology, University of Connecticut, Storrs, 06269, CT, USAJonathan L. KlassenDepartment of Biology, Large Lakes Observatory, University of Minnesota-Duluth, Duluth, 55812, MN, USAJennifer J. KnackSchool of Biological Sciences, Georgia Institute of Technology, Atlanta, 30332, GA, USAGina R. LewinCenter for Research in Microscopic Structures and Department of Biochemistry, School of Medicine, University of Costa Rica, San JosxE9, 10102, Costa RicaAdrixE1 n A. Pinto-TomxE1sDepartment of Medicine, University of Wisconsin School of Medicine and Public Health, Madison, 53705, WI, USAMiao Zhao_&_David R. Andes\" - \"AuthorsMarc G. ChevretteYou can also search for this author in _Caitlin M. CarlsonYou can also search for this author in _Humberto E. OrtegaYou can also search for this author in _Chris ThomasYou can also search for this author in _Gene E. AnanievYou can also search for this author in _Kenneth J. BarnsYou can also search for this author in _Adam J. BookYou can also search for this author in _Julian CagnazzoYou can also search for this author in _Camila CarlosYou can also search for this author in _Will FlaniganYou can also search for this author in _Kirk J. GrubbsYou can also search for this author in _Heidi A. HornYou can also search for this author in _F. Michael HoffmannYou can also search for this author in _Jonathan L. KlassenYou can also search for this author in _Jennifer J. KnackYou can also search for this author in _Gina R. LewinYou can also search for this author in _Bradon R. McDonaldYou can also search for this author in _Laura MullerYou can also search for this author in _Weilan G. P. MeloYou can also search for this author in _AdrixE1n A. Pinto-TomxE1 sYou can also search for this author in _Amber SchmitzYou can also search for this author in _Evelyn Wendt-PienkowskiYou can also search for this author in _Scott WildmanYou can also search for this author in _Miao ZhaoYou can also search for this author in _Fan ZhangYou can also search for this author in _Tim S. BugniYou can also search for this author in _David R. AndesYou can also search for this author in _Monica T. PupoYou can also search for this author in _Cameron R. CurrieYou can also search for this author in _\" - Contributions - \"Conceptualization: M.G.C., C.M.C., H.E.O., A.A.P., T.S.B., D.R.A., M.T.P., C.R.C.; Methodology: M.G.C., C.M.C., H.E.O., C.T., G.E.A., K.J.B., T.S.B., D.R.A., M.T.P., C.R.C.; Software: M.G.C.; Validation: M.G.C., C.M.C., H.E.O., G.E.A., M.Z., F.Z.; Formal analysis: M.G.C., H.E.O.; Investigation: M.G.C., C.M.C., H.E.O., C.T., G.E.A., K.J.B., W.F., J.J.K., L.M., W.G.P.M., H.A.H., A.A.P., E.W., A.S., S.W., M.Z., F.Z.; Resources: C.M.C., C.T., G.E.A., K.J.B., A.J.B., K.J.G., F.M.H., J.L.K., G.R.L., B.R.M., W.G.P.M., E.W., S.W., M.Z., F.Z., T.S.B., D.R.A., M.T.P., C.R.C.; Data curation: M.G.C., C.M.C.; Writingu2014Original Draft: M.G.C., C.R.C.; Writingu2014Review and Editing: M.G.C., C.R.C., C.M.C., H.E.O., C.C., J.L.K., D.R.A., M.T.P., C.R.C.; Visualization: M.G.C.; Supervision: C.R.C., D.R.A., M.T.P., T.S.B., F.M.H.; Project administration: M.G.C., C.M.C., D.R.A., C.R.C.; Funding acquisition: T.S.B., D.R.A., M.T.P., C.R.C.\" - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\'Journal peer review information: Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work.\\' - \"Publishers note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported Licence). (b) Geometry of the shear test. - \"Figure presents an electron microscopic photograph of an MAE with embedded spherical particles. It serves as an illustration of the internal stricture of a composite material. Figure shows a solitary rigid particle that is rotated and translated as a whole under the engineering shear strain u03C8 caused by the shear stress u03C4. The vector of the magnetic field strength is directed perpendicular to the applied stress: ({bf{H}}parallel Oy).\" - \"The mechanism of single-particle magnetoelasticity makes it possible to explain the influence of the magnetic field on the magnitude of the shear modulus in MAEs. The contribution of this mechanism and its influence on the elastic characteristics of the MAE depends on the ratio of the anisotropy constant K of the magnetic particle to the shear modulus u03BC of the surrounding elastomer. The greater is the magnitude of the ratio K/u03BC, the stronger is the magnetorheological effect. The magnetic anisotropy constant K describes the uniaxial directional dependence of magnetic properties of filler particles. This means that the anisotropy energy EA of a single particle can be written in the form of ({E}_{A}=-(KV{ cos }^{2}theta )/2), where u03B8 is the angle between the easy axis and the particles magnetization, V is the volume of the particle. The elastomer matrix is non-magnetic and, therefore, its shear modulus u03BC is independent of magnetic field.\" - The principal difference between the calculation of effective elastic properties of MAEs and the effective elastic properties of nonmagnetic composites is the necessity to take into account the internal degrees of freedom of filler particles, namely directions of the vectors of the magnetic moment of the particles. As a consequence, the magnetic field will cause inhomogeneous elastic stresses in the polymer matrix. The influence of these stresses will inevitably lead to a contribution to the magnitude of the effective shear modulus of the composite material. - \"In, the equation connecting the shear angle u03C8 and the shear stress u03C4 has been obtained. The effective shear modulus G, relating the shear angle and the shear stress, has the following form\" - \"$$tau ={G}^{{rm{e}}}(p,H)psi ,$$ (3) \" - where - \"$${G}^{{rm{e}}}(p,H)={G}^{{rm{e}}}(p)+mu cdot {rm{Omega }}(H)p$$ (4) \" - and - \"$${rm{Omega }}(H)=frac{Hm/K}{(1+Hm/K)(1+4mu /K)-1}.$$ (5) \" - \"p is the concentration of filler particles, ({G}^{{rm{e}}}(p)) is the shear modulus of the composite material in the absence of external magnetic field and m is the particles magnetization. The case is selected when the directions of easy magnetization axes of all particles coincide with the direction of external field. Therefore, there is no second term in Eq. (). Figure illustrates the shear test geometry and the characteristic angles.\" - \"The effective modulus ({G}^{{rm{e}}}(p)) for non-magnetic composite materials was previously considered and calculated for a large number of different structures and various approximations, see, e.g.. As a rule, effective coefficients characterizing the properties of composites can be obtained in analytical form only in the approximation of a small concentration of inclusions pu2009u226Au20091. From full-scale and numerical experiments, it is known that, in the case of strong inhomogeneity, when the concentration of inclusions approaches the percolation threshold, pu2009u2192u2009pc, the effective coefficients grow strongly. As follows from percolation theory, the behavior of effective coefficients is analogous to the behavior of the order parameter in the theory of phase transitions of the second order and diverges, for example, as the shear modulus in (2) does. The exponent of the power law is called, as in the second-order phase transition, the critical exponent or the critical index.\" - Calculation of the critical exponent, for example, for the shear modulus of a composite consisting of spherical solid inclusions in an elastic matrix is a complex problem that cannot be accurately analyzed analytically (similarly to the critical exponent of a second-order phase transition). At the same time, critical exponents are crucial characteristics of the behavior of a composite near the percolation threshold and even their approximate calculation is an important statement of the problem. - \"Figure shows the dependence of the additional term u03BC u03A9(H) to G(p,H) on the normalized magnetic field mH/K and the normalized u201Cmagneto-mechanical rigidityu201D of the matrix u03BC/K. It is seen that the additional contribution u03BC u03A9(H) grows with the increasing magnetic field Hm/K and, in large magnetic fields (Hu2009>u2009K/m), comes to the saturation. The magnitude of this magnetoelastic effect is bigger for larger anisotropy constants of the inclusion and smaller shear moduli of the matrix.\" - \"Figure 2Dependence of u03A9(H) on the normalized magnetic field mH/K and magneto-mechanical rigidity u03BC/K. (a) Dependence of u03A9(H) on the normalized magnetic field for different values u03BC/Ku2009=u20090.05; 0.2; 0.5; 1; 2. (b) Dependence of u03A9(H) on the magnetomechanical rigidity for different values of the magnetic field mH/Ku2009=u20090.1; 0.5; 2; 10.\" - \"In the absence of magnetic field, the effective modulus increases with the growing concentration of inclusions and near the percolation threshold behaves like an order parameter in the theory of second order phase transitions, namely, it diverges accordingly to a power law. Qualitatively, this increase is explained by the fact that, as the concentration grows, finite clusters appear and their elastic interaction through the matrix increases. The imposition of an external magnetic field leads to an additional increase in the interaction (additional elastic stresses appear in the matrix). These additional stresses also increase with growing concentration. This effect should lead to the field dependence of the critical index. To estimate the influence of the magnetic field on the critical exponent of the shear modulus, its renormalization, we use the method of PadxE9 approximants.\"- - \"Calculations of the effective modulus ({G}^{{rm{e}}}(p,H)) have been made in the approximation of the small parameter. In our case, this small parameter is the concentration of inclusions p. However, by using (4) and (5) one can estimate the behavior of ({G}^{{rm{e}}}(p,H)) also at large concentration values in the critical region close to the percolation threshold. For this purpose, we resort to the method of PadxE9 approximants (PA).\" - \"The method of PadxE9 approximants is one of the approaches for approximate (and, in some cases, sufficiently accurate) methods of calculating critical exponents for the case when the behavior of effective coefficients at low concentrations is known. The PA method is one of extrapolation methods for estimating the asymptotic behavior of a power series from several first terms of the behavior (see, Chapter 6).\" - \"In the theory of second order phase transitions, method of PadxE9 approximants was used in the past for solving one of the most difficult problems of statistical physics, namely calculation of critical exponents. As it is written in, this method had a stunning success. In particular, the critical exponent of the magnetization was obtained on the basis of the low-temperature series of magnetization.\" - \"The formulation of the problem of determining the effective characteristic of the shearing experiment is similar both for an elastic matrix with rigid inclusions and for a viscous fluid, see, for example. Note that application of the method (u201Ctricku201D) of PadxE9-approximants works well for the investigation of suspensions. The expression for the effective viscosity of suspensions accurate to the first order of the concentration of hard inclusions was obtained by Einstein. In the work (see also, an expression for the effective viscosity, which is accurate up to the square of the concentration, was obtained. The experimental data show the divergence of the effective viscosity as the concentration approaches the threshold. Such a divergence, however, cannot be derived from any power-law expansion. In the simplest form of the PA method, a power series in the concentration of inclusions, for example, a series of shear modulus or effective viscosity of a suspension at pu2009u226Au20091 can be represented as the ratio of two polynomials. In this way, based on the Einsteins expression for the effective viscosity proportional to 1u2009+u20092.5p or a more accurate approximation 1u2009+u20092.5pu2009+u20095pu2009, the value of the percolation threshold according to Einstein was estimated as pcu2009=u20091/2, or, when the second order of concentration is taken into account, as a more accurate value pcu2009=u20092/5, which describes the experimental data well.\" - \"To determine the critical exponent, it is necessary to apply one of the versions of the PA method. For estimation of the critical exponent, one can use the so-called D-log method of PadxE9 approximants.\"- - \"In our case, the application of the method of PadxE9 approximants significantly differs from the applications considered in. We did not aim at finding the behavior of ({G}^{{rm{e}}}(p)), in particular its critical exponent S0 in (2). The numerical value of the critical index S0 in (2) will be assumed known from the literature. The problem consists in determination of the influence of an external magnetic field on this critical index. Therefore, we seek the renormalization of the critical index S(H) for ({G}^{{rm{e}}}(p,H)) by a magnetic field H, where S(Hu2009=u20090)u2009=u2009S0.\" - \"The value of the critical exponent S0 for the effective shear modulus for a u201C purelyu201D elastic problem in the absence of magnetic field (Hu2009=u2009 0) in the two-dimensional case was found in (see also) and is equal in the two-dimensional case to\" - \"$${S}_{0}=mathrm{1.3.}$$ (6) \" - \"Let us use the variation of method of PadxE9 approximants, proposed in, specifically for the determination of the critical exponent. We seek the effect of magnetic field H on the shear modulus ({G}^{{rm{e}}}(p,H)) in the form of\" - \"$${G}^{{rm{e}}}(p,H)=mu {(1-frac{p}{{p}_{c}})}^{-S(H)},$$ (7) \" - \"where pc is the given constant. The method of PadxE9 approximants allows one to reproduce the critical behavior (7) and to estimate the parameter S(H) (which is the critical exponent) from the expansion of (4).\" - Expansion of the logarithmic derivative of (7) into the Taylor series gives - \"$$frac{d}{dp},{rm{l}}{rm{n}},{G}^{{rm{e}}}(p,H)approx frac{S(H)}{{p}_{c}}{p}^{0}+...,$$ (8) \" - while the corresponding expansion of G(p, H) from (4), taking into account given expression for ({G}^{{rm{e}}}(p)), yields - \"$$frac{d}{dp},{rm{l}}{rm{n}}[{G}^{e}(p)+mu cdot pcdot {rm{ Omega }}(H)]approx (frac{{S}_{0}}{{p}_{c}}+{rm{Omega }}(H)){p}^{0}+...,$$ (9) \" - where ({G}^{{rm{e}}}(p)) is chosen accordingly to (2). - \\'Comparing (8) and (9), we obtain the critical exponent of an effective elastic modulus renormalized by an external magnetic field:\\' - \"$$S(H)={S}_{0}+{rm{Omega }}(H){p}_{c}.$$ (10) \" - Thus, the term dependent on external magnetic field is no longer linearly proportional to p as in (4), but it enters the power exponent in (7). - \"For a more accurate determination of the critical exponent, one can also use the PadxE9 approximant for the function\" - \"({F}_{2}(p)=({p}_{c}-p)frac{d}{dp},mathrm{ln},f(p)) in the point pu2009=u2009pc, where f(p) is a series expansion of ({mu }^{-1}({G}^{{ rm{e}}}(p)+mu cdot pcdot {rm{Omega }}(H))) with respect to concentration p.\" - \"The PadxE9 polynomial for the function F2(p) is chosen in the form:\" - \"$$P(p)=({S}_{0}+{rm{Omega }}(H){p}_{c})frac{1+Ap}{1+Bp},$$ (11) \" - \\'where A and B are the coefficients found by using the expansion for f(p) in the form:\\' - \"$$f(p)=1+(frac{{S}_{0}}{{p}_{c}}+{rm{Omega }}(H))p+frac{{S}_{0}({S}_{0}+1)}{2{{p}_{c}}^{2}}{p}^{2}.$$ (12) \" - \\'Now with the help of (11) we obtain the expression for the critical index:\\' - \"$$S(H)=P({p}_{c})=frac{(1+{S}_{0})({S}_{0}^{2}(2+{S}_{0})-2{S}_{0}^{2}{rm{ Omega }}(H){p}_{c}-{S}_{0}{{rm{Omega }}}^{2}(H){p}_{c}^{2})}{{S}_{0}(1+{S}_{0})(2+{S}_{0})-{ rm{Omega }}(H){p}_{c}(1+{S}_{0})(2+3{S}_{0})-2{{rm{Omega }}}^{2}(H){p}_{c}^{2}(2+3{S}_{0})-2{{ rm{Omega }}}^{3}(H){p}_{c}^{3}}.$$ (13) \" - \"For u03A9 (H)pcu2009<u2009S0, Equation () gives (10), and it satisfies the equality S(Hu2009=u20090)u2009=u2009S0.\" - The details of derivation of Eq. () are provided in the Appendix 1. - \\'MAEs are usually characterized by the magnitude of the relative magnetorheological effect (MRE), which is defined as the relative change of the shear modulus due to applied magnetic field:\\' - \"$$MRE=frac{{G}^{{rm{e}}}(Hne 0)-{G}^{{rm{e}}}(H=0)}{{G}^{{rm{e}}}(H=0)}.$$ (14) \" - \"Figure shows the concentration and field dependences of the effective shear modulus. As can be seen from Fig. , the effective modulus with a renormalized critical index increases with increasing concentration much faster than the module without normalizing the modulus. Figure shows the growth of the effective modulus with the external magnetic field. It is seen that the effective shear modulus nonlinearly depends on external magnetic field and goes to saturation in large magnetic fields. Fig. presents the dependence of the relative MR effect on the concentration of inclusions p. It is seen that the relative MR effect significantly grows with increasing concentration of inclusions p. More precisely, it theoretically goes to infinity when pu2009u2192u2009pc. This is the consequence of the dependence of the critical index on the external magnetic field and the approximations made (e.g. infinitely rigid inclusions). The obtained expression ((7) together with (10)) is asymptotic and is valid in the region pu2009<u2009pc. Similar to the theory of u201Csmearedu201D phase transitions, it has to be expected that the MRE will take large (up to 10%) but finite values in the vicinity of pc, as it was indeed observed in experiments.\" - \"Figure 3Field and concentration dependences of the normalized effective shear modulus for u03BC/Ku2009=u20090.1. (a) Dependences of the shear modulus on the concentration p in the field Hm/Ku2009=u20092 for different models: the black curve 1 is calculated from Eq. (), the green curve 2 is obtained from Eq. (), the red curve 3 takes into account the renormalization and it is calculated from Eq. () with exponent (10), the blue curve (4) takes into account the self-consistent approach and it is obtained from Eq. (); (b) Dependence of the normalized shear modulus on applied magnetic field; (c) Dependence of the relative magnetorheological effect on the filler concentration in the framework of the renormalized percolation model for different external fields mH/Ku2009=u20090.5, 2, 3.\"- - \"Let us go beyond the model (2), applying a self-consistent approach, which allows one to consider the case of higher concentrations. We examine one more variant of the linear approximation (3). Renormalization of the critical index by magnetic field (10) was performed on the basis of expression (4) for ({G}^{{rm{e}}}(p,H) ). This expression has been obtained in in the approximation of a single inclusion with infinite large elastic moduli in an elastic medium with the shear modulus u03BC. Several approaches are known allowing one to improve calculation of effective coefficients based on the approximation of single inclusions using the self-consistency method. The most successful approach is the Bruggeman-Landauer approximation. In this approximation, it is assumed that the solitary inclusion of the first phase is not surrounded by the first phase but by the medium with the effective elastic moduli to be found. The same assumption is also valid for the first phase.\" - Similar (but not exactly the same) approach can be used during derivation of Eq. (). First, one may assume that the solitary inclusion is immersed into a medium with effective elastic moduli of the composite materials, calculated in the absence of magnetic field. In this case, Eq. () should be replaced by - \"$${G}_{SC}^{{rm{e}}}(p,H)={G}^{e}(p)+{G}^{e}(p)cdot pcdot {rm{Omega }}(H).$$ (15) \" - The expressions for the effective shear modulus ({G}_{SC}^{{rm{e}}}(p,H)) will be different from (3). The detailed analysis of ({G}_{SC}^{{rm{e}}}(p,H)) and some other modifications of (3) is not the subject of this paper and will be given in the separate publication. - \"Here we are interested in the self-consistent approach, which instead of (4) leads to expression (15) for the shear modulus. The dependence of the shear modulus, obtained from (15) in Fig. is given by curve 4. This curve 4 goes much higher than curves 1 and 2 obtained with the help of expressions (2) and (4), and it is closer to curve 3 obtained by means of renormalization. However, as it is easily seen from (15), self-consistency does not lead to the effect of the anomalous growth of MRE when the concentration p tends to the threshold value pc (pu2009u2192u2009pc).\" - \"The most interesting question is how the self-consistency leading to ({G}_{SC}^{{ rm{e}}}(p,H)) influences the critical exponent S(H). For this calculation, it is necessary to substitute into (9) the expression for ({G}_{SC}^{{rm{e}}}(p,H) ) instead of ({G}^{e}(p)+mu cdot {rm{Omega }}(H)p). The first term in the expansion of the logarithmic derivative gives the expression for the renormalized critical exponent S(H). It turns out that the critical exponent for ({G}_{SC}^{{rm{e}}}(p,H)) remains the same as for ({G}^{e}(p,H) ) and does not depends on self-consistency. One can speak about u201Cuniversalityu201D of the renormalized critical index S(H)u2009=u2009S0u2009+u2009u03A9(H)p c.\"- - \"There are macroscopically inhomogeneous media (composite materials) where an analog of phase transition of the second order is observed. This is the percolation transition. Examples are elastic two-phase materials with a strong difference in the elastic properties of the phases. Percolation transitions (they mean sharp change of the effective properties of the entire system, i.e. the composite material as a whole) are observed near the percolation threshold, when the concentration of one of the phases approaches the percolation threshold. However, often such macroscopically inhomogeneous media exist and are considered at low concentrations pu2009u226Au20091. This case is also interesting for investigation and has broad field of applications. For example, in the pioneering work of Albert Einstein the viscosity of a suspension (solid particles in a viscous liquid), first considered in detail with accuracy of the first order p. Later, refined theories were developed with accuracy up to p and further. It is quite clear that a theory developed up to an arbitrarily large order of p (if it is possible) does not describe a percolation transition specifying divergence when approaching the percolation threshold ~(pcu2009u2212u2009p). The same thing happens in phase transitions. Expansion of the order parameter in powers of temperature does not give its critical behavior near the phase transition temperature. Nevertheless, having only such approximations and using the method of PadxE9 approximants it is possible to re-establish the critical behavior of the order parameter and with good approximation to calculate the basic u201Cworldu201D constants of the theory, i.e. critical indices. In this case, as originally suggested, the expansion of the logarithmic derivative of the order parameter is used. The divergence of the shear modulus at the percolation threshold is explained by the approximation of infinitely rigid particles. Such u201Chardenedu201D states with touching particles and therefore diverging elastic modulus are well known in the literature. In real-word applications, the effective modulus will be limited by the large but finite modulus of ferromagnetic inclusions.\"- - \"In the present work, the same approach as in the theory of second order phase transitions has been applied. However, we did not use it to find the critical exponent of effective shear modulus in the absence of magnetic field, which we considered to be known. We employed it to determine the renormalization of this exponent by a magnetic field. As it turned out, the critical index describing the elastic behavior of an MAE material in a magnetic field near the percolation threshold depends on the magnetic field. That allows one to describe the dependences of the absolute and relative magnetorheological effect in magnetoactive elastomers on external magnetic field and concentration of soft magnetic inclusions. In particular, the relative magnetorheological effect should strongly increase when the concentration of filler particles approaches the percolation threshold (pu2009<u2009pc, pu2009u2192u2009pc). Our renormalization approach can be applied to many other materials, also in the 3D case. For example, an extension to magnetorheological fluids and gels is obvious.\"- - \"Abramchuk, S. et al. Novel highly elastic magnetic materials for dampers and seals: Part I. Preparation and characterization of the elastic materials. Polym. Adv. Technol. 18(11), 883u2013890 (2007)._ _ _ Li, Y., Li, J., Li, W. & Du, H. A state-of-the-art review on magnetorheological elastomer devices. Smart Mater. Struct. 23, 123001 (2014)._ _ Filipcsei, G., Csetneki, I., SzilxE1gyi, A. & ZrxEDnyi, M. Magnetic field-responsive smart polymer composites. Adv. Polym. Sci. 206, 137u2013189 (2007)._ _ _ Ubaidillah, S. J., Purwanto, A. & Mazlan, S. A. Recent progress on magnetorheological solids: materials, fabrication, testing, and applications. Adv. Eng. Mater. 17, 563u2013597 (2015)._ _ _ Menzel, A. M. Tuned, driven, and active soft matter. Phys. Rep. 554, 1u201345 (2015)._ _ _ _ _ Odenbach, S. Microstructure and rheology of magnetic hybrid materials. Arch. Appl. Mech. 86, 269u2013279 (2016)._ _ _ Lopez-Lopez, M., DurxE1n, J. D., Iskakova, L. Y. & Zubarev, A. Y. Mechanics of Magnetopolymer Composites: A Review. J. Nanofluids 5, 479u2013495 (2016)._ _ Cantera, M. A., Behrooz, M., Gibson, R. F. & Gordaninejad, F. Modeling of magneto-mechanical response of magnetorheological elastomers (MRE) and MRE-based systems: a review. Smart Mater. Struct. 26(2), 023001 (2017)._ _ _ Stoll, A., Mayer, M., Monkman, G. J. & Shamonin, M. Evaluation of highly compliant magneto-active elastomers with colossal magnetorheological response. J. Appl. Polym. Sci. 131(2) (2014).Stauffer, D. & Aharoni, A. In Introduction To Percolation Theory, Revised 2 Ed. (Taylor & Francis, London, 1994).Milton, G. W. The Theory of Composites. (Cambridge University Press, Cambridge, UK, 2002).Torquato, S. Random Heterogeneous Materials. Microstructure and Macroscopic Properties. (Springer-Verlag, New York, 2002).Hunt, A. G. Percolation Theory for Flow in Porous Media 202 pp, (Springer Verlag, New York, 2005).Snarskii, A., Bezsudnov, I. V., Sevryukov, V. A., Morozovskiy, A. & Malinsky, J. Transport Processes in Macroscopically Disordered Media. From Mean Field Theory to Percolation (Springer Verlag, New York, 2016).Bergman, D. J. & Stroud, D. Physical properties of macroscopically inhomogeneous media. Solid State Phys. 46, 147u2013269 (1992)._ _ _ Kalita, V. M., Snarskii, A. A., Zorinets, D. & Shamonin, M. Single-particle mechanism of magnetostriction in magnetoactive elastomers. Phys. Rev. E 93(6), 062503 (2016)._ _ _ _ Linke, J. M., Borin, D., Yu. & Odenbach, S. First-order reversal curve analysis of magnetoactive elastomers. RSC Adv. 6, 100407 (2016)._ _ _ Coey, J. M. Magnetism and Magnetic Materials. (Cambridge University Press, Cambridge, UK, 2010).Kalita, V. M., Snarskii, A. A., Shamonin, M. & Zorinets, D. Effect of single-particle magnetostriction on the shear modulus of compliant magnetoactive elastomers. Phys. Rev. E 95(3), 032503 (2017)._ _ _ _ Christensen, R. M. Mechanics of Composite Materials (Wiley, 1979).Stanley, H. E. Introduction to Phase Transitions and Critical Phenomena (International Series of Monographs on Physics), Revised Ed. (Oxford University Press USA, New York, 1971).Landau, L. D. & Lifshitz, E. M. Statistical Physics, Third Edition, Part 1: Volume 5 (Elsevier, 2013).Ma, S.-K. Modern theory of critical phenomena, (Perseus Pub., Cambridge, Mass, 2000).Patashinskii, A. Z. & Pokrovskii, V. L. Fluctuation Theory of Phase Transitions (Pergamon Press, 1979).Baker, G. A. & Graves-Morris, P. PadxE9 Approximants 746 pp (Cambridge University Press, Cambridge, UK 1996).Andrianov, I. V., Manevitch, L. I. & Hazewinkel, M. Asymptotology: Ideas, Methods, and Applications 252 pp (Springer US, 2002).Einstein, A. A new determination of molecular dimensions. Ann. Phys. 19, 289u2013306 (1906), 34 591 (1911).Batchelor, G. K. & Green, J. T. The hydrodynamic interaction of two small freely-moving spheres in a linear flow field. J. Fluid Mech. 56, 375u2013400 (1972)._ _ _ _ Brouwers, H. J. H. Viscosity of a concentrated suspension of rigid monosized particles. Phys. Rev. E 81, 051402 (2010)._ _ _ _ Boroun, S. & Larachi, F. Tuning mass transport in magnetic nanoparticle-filled viscoelastic hydrogels using low-frequency rotating magnetic fields. Soft Matter 13(36), 6259u20136269 (2017)._ _ _ _ _ Bergman, D. J. Elastic moduli near percolation in a two-dimensional random network of rigid and nonrigid bonds. Phys. Rev. B 33, 2013u20132016 (1986)._ _ _ _ Annunziata, M. A., Menzel, A. M. & LxF6wen, H. Hardening transition in a one-dimensional model for ferrogels. J. Chem. Phys. 138(20), 204906 (2013)._ _ _ _ Lopez-Lopez, M. T., Iskakova, L. Y. & Zubarev, A. Y. To the theory of shear elastic properties of magnetic gels. Phys. A: Stat. Mech. Appl. 486, 908u2013914 (2017)._ _ \"- - This Open Access Publication was funded by the OTH Regensburg.- - Authors and Affiliations - \"Igor Sikorsky Kyiv Polytechnic Institute, Prospekt Peremohy 37, 03056, Kiev, UkraineAndrei A. Snarskii_&_Viktor M. KalitaInstitute for Information Recording NAS of Ukraine, Shpaka Street 2, 03113, Kiev, UkraineAndrei A. SnarskiiInstitute of Physics NAS of Ukraine, Prospekt Nauky 46, 03028, Kiev, UkraineViktor M. KalitaEast Bavarian Centre for Intelligent Materials (EBACIM), Ostbayerische Technische Hochschule Regensburg, PrxFCfeninger Strasse 58, 93049, Regensburg, GermanyMikhail Shamonin\" - \"AuthorsAndrei A. SnarskiiYou can also search for this author in _Viktor M. KalitaYou can also search for this author in _Mikhail ShamoninYou can also search for this author in _\" - Contributions - This is a collaborative theoretical research. All authors contributed equally to the manuscript. In particular, A.S. conceived the idea of the manuscript. A.S. and V.K. performed calculations and wrote the draft of the manuscript. M.S. put the considerations into the context of magnetoactive elastomers and wrote the final manuscript text. All authors reviewed the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing Interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\'Publisher\\'\\'s note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported Licence. ',\n",
       " 'Creative Commons Attribution 3.0 Unported Licence.\" - \"Figure 2P. falciparum life cycle synchronisation by DFMO. (a) Giemsa-stained morphological evaluation of thin blood smears of DFMO treated parasites for an entire 48u2009h IDC of P. falciparum 3D7 parasites (5% haematocrit and 3% parasitaemia,1000X enlarged, hpiu2009=u2009hours post-infection). (b) Synchronicity was evaluated through age-binning of parasites based on 2u2009h evaluation of Giemsa stained parasite morphology for parasites synchronised either with 3 consecutive but 6-hourly spaced sorbitol treatments (3xsorbitol) inspected directly after the last sorbitol treatment, or a single DFMO treatment (IC90) at the next invasion cycle, monitored for a total of 24u2009h after treatment (Cu2009=u2009control, 5% haematocrit and 5% parasitaemia). A minimum of 100 parasites was inspected for each condition. *Pu2009<u20090.05; ***Pu2009 <u20090.001 calculated using a student t-test. Box plots indicate 80% distribution of data in box and hinges, and 95% in whiskers (c) Quantification of DFMO synchronisation. Percentage of the parasites with increasing DNA content (Nu2009=u2009DNA copy number) measured by flow cytometry through SYBR Green I fluorescence collected in the FL-1 channel (FITC signal) on a Becton Dickenson FACSAria with 50 000 total events captured. Parasites were treated with DFMO (IC90) and sampled every 6u2009h for an entire IDC (48u2009h). Data are averagedu2009xB1u2009S.E. from at least 2 independent biological replicates, performed in technical triplicates.\" - \"Synchronisation is usually morphologically evaluated, and parasites are classified as ring, trophozoite or schizont stages with age range overlaps of 6u201312u2009 h (Supplementary Fig._). Morphological evaluation of the age range of the parasites after a single DFMO synchronisation is within a 3u20134u2009h window (18u2013 22 hpi, Fig._), compared to, at best, a 6u201310u2009h window achieved with sorbitol synchronisation. To quantify the DMFO-dependent synchrony, we used flow cytometry to trace the nuclear content of parasites through fluorescent SYBR Green I staining. After DFMO treatment, 96.6u2009xB1u20090.2% of parasites contained a single nucleus (1u2009N nuclear content), which was retained for the entire 48u2009h period evaluated, indicating that these parasites were synchronised as early trophozoites (Fig._) prior to DNA synthesis. Time-matched, control parasites progressed through schizogony, forming multinucleated schizonts (85% of parasites in 3u2009N and u22654u2009N populations, Fig._). A fraction of the parasites (12%) are still detected with 1u2009N nuclear content at 48u2009 h (3% with 2u2009N), which reflect either parasites that have not entered schizogony, or early developmental stage parasites from the next replication cycle.\" - The induced life cycle arrest corresponds to a cell cycle arrest at the G1/S phase transition - \"Cell cycle arrest should be fully reversible, with no overt negative perturbation effects and without compromising the cells viability or capacity to differentiate. DFMO-arrest is reversible by the addition of putrescine as a mitogen (product of AdoMetDC/ODC, 2 mM), either simultaneously with DFMO (Fig._) or after 24u2009 h of DFMO treatment (Fig._). In DFMO-treated cultures, the cell number remained constant for the 96u2009h monitored, supporting a cytostatic rather than cytotoxic response (Fig._). Putrescine-induced reversal significantly increased parasitaemia by 48u2009h (Pu2009u2264u20090.05, nu2009=u20094, Fig._), and these parasites subsequently developed normally into schizonts, releasing merozoites that can re-invade erythrocytes with multiplication rates similar to what was previously shown. Additionally, these parasites formed gametocytes, with comparable gametocytaemias at day 14 post-induction (Pu2009>u20090.05, Fig._). Previously, DFMO-treated parasites were shown to be metabolically viable, capable of ATP synthesis, and ultrastructurally unaffected and can be reversed even after 67 h. These data demonstrate that DFMO-treated parasites undergo cell cycle arrest, because they show no signs of overt perturbation effects, are tightly synchronised in their life cycle, and can re-initiate normal life cycle progression and differentiation into gametocytes.\" - \"Figure 3DFMO arrest and putrescine reversal associated to G1/S cell cycle control in intraerythrocytic P. falciparum parasites. In all instances, u25A0 refers to control parasites, u25BC refers to DFMO-treated parasites, and u25CF refers to putrescine-reversed parasites. (a) DFMO dose-response curves of asexual P. falciparum 3D7 proliferation over 96u2009h (initiated with ring-stage parasites, 1% haematocrit, 1% parasitaemia) at 37u2009xB0C in the absence or presence of 2u2009mM putrescine. Proliferation is expressed relative to untreated controls, with data averaged from nu2009=u20096 biological replicates and shownu2009 xB1u2009S.E. (b) Synchronised P. falciparum 3D7 cultures were treated with DFMO alone (IC90) or with putrescine (2u2009mM, after 24u2009h DFMO pressure) and parasitaemia monitored over 96u2009h with SYBR Green I fluorescence (10 000 infected erythrocytes counted). *Pu2009<u20090.05, student t-test. Data from nu2009=u20094 biological replicates in duplicate (c) Gametocytaemia of P. falciparum NF54 cultures over 14 days, determined microscopically. Control or DFMO-treated parasites (IC90, 24u2009h treatment of ring-stage parasites before DFMO removal with fresh media) before gametocytogenesis induction. Data averagedu2009 xB1u2009S.E. from nu2009u2265u20093 biological replicates with >1000 cells counted; where not shown, error bars fall within symbols. (d) Flow cytometric analysis of nuclear division in P. falciparum parasites following life cycle arrest and subsequent re-entry into the life cycle. Parasites (1% haematocrit, 10% parasitaemia) were sampled after 12 hpi, 36 hpi, 39 hpi, 42 hpi or treated with DFMO (IC90) for 24, 27 or 30u2009h before sampling. Additionally, following 24u2009h of DFMO treatment, putrescine (2u2009mM) was added to stimulate cell cycle re-entry, and samples taken 3, 6, 12u2009h after reversal. The nucleic acid content of parasitised erythrocytes was determined by consecutive staining with both SYBR Green I (DNA fluorescence) and PyroninY (RNA fluorescence); detected in the FITC or PE channel, respectively. Overlaid histograms for a representative sample of biological triplicates. DNA content (SYBR Green I fluorescence) confirmed using fluorescent microscopy on a Zeiss LSM 880 Confocal Laser Scanning Microscope.\" - \"To associate the life cycle-arrested parasites with a cell cycle phase, we used flow cytometric quantification of parasites stained with both SYBR Green I to trace DNA content as well as Pyronin Y to detect RNA (Fig._ and Supplementary Fig._). Both DNA and RNA levels increased in control parasites as they progressed from G1 to S, forming multinucleated schizonts (confirmed through fluorescence microscopy). Merozoites from these parasites subsequently reinvaded erythrocytes and formed new ring-stage parasites in the next cycle, returning to a single nucleated G1 state. The DFMO-arrested parasites were incapable of significant DNA or RNA synthesis and >91% of these parasites displayed parameters characteristic of single nucleated G1 phase or of the G1/S transition point. This arrest was observed for all 3-hourly samples time-matched to control parasites and taken during a single life cycle. The additional lack of increased RNA indicate entry into a quiescent G0-like state. The addition of putrescine to the DFMO-arrested parasites reversed the arrest and stimulated re-entry into the cell cycle, with 17u2009xB1u20092% parasites entering S-phase within 6u2009h, with nuclear content at 12u2009h corresponding to the control schizont-stage parasites (Fig._ ). The putrescine-reversed parasites subsequently complete their cell cycle and produce invasive daughter merozoites and ring-stage parasites. Based on this data, parasites that re-enter the cell cycle continue with asynchronous nuclear division characteristic of schizogony in the parasite, resulting in asynchronous populations.\" - \"To determine the timeframe of cell cycle arrest and re-initiation at higher resolution, we measured the global transcriptomes of cell cycle-arrested parasites (24u2009 h DFMO treatment) as well as of parasites reversed with putrescine and sampled after 3u2009h (RE1), 6u2009h (RE2) or 12u2009h (RE3) using DNA microarrays (Fig._, Supplementary File_). Global Pearson correlation demonstrated a consistent arrested response with strong correlations (ru2009=u20090.8u20130.94) between all the cell cycle-arrested time points (Fig._). This indicates that the arrested phenotype is immediate and complete after the earliest effective point of DFMO arrest. Progressive departure from the arrested phenotype was observed following re-entry (RE1 to RE2), resulting in anti-correlation between the transcriptome of parasites re-initiating cell cycle progression after 12u2009h and that of the arrested parasites (A1-A3 vs. re-entry time point 3, RE3, Fig._).\" - \"Figure 4The transcriptome profile of cell cycle arrest and re-entry. Ring-stage intraerythrocytic P. falciparum 3D7 parasites were arrested with DFMO (2u2009 mM) for 24u2009h prior to sampling (A1u20133, arrested, sampled at 0, 3 and 6u2009h after the 24u2009h DFMO treatment) or left untreated (C, control). Additionally, after 24u2009h DFMO pressure, the cell cycle arrest was reversed with 2u2009mM putrescine and sampled after 3u2009h (RE1), 6u2009h (RE2) or 12u2009h (RE3). (a) Correlation plot displays the global Pearson correlations within and between each of the treatment conditions. (b) Global Pearson correlation to the time-mapped published IDC transcriptomic profiles for the arrested samples (A1u20133, blue lines), and re-entered samples (RE1u20133, orange lines). Arrows are used to emphasise the peak areas of correlation for the arrested (A1u20133) and each RE sample (RE1u20133). Representative illustration of parasites shown below graph. Parasite drawings were modified from freely available images () under a Creative Commons Attribution 3.0 Unported Licence.\" - \"The P. falciparum transcriptome is characterised by the u2018just-in-time expression of genes. We therefore exploited the high-resolution hourly evaluation of the changing transcriptome of P. falciparum parasites during their IDC to finely map and stage our parasites (Supplementary File_). All cell cycle-arrested transcriptome time points correlated to the transcriptomes of late ring/early trophozoites at 15u201317 hpi (Pearson ru2009=u20090.50u20130.67, Fig._ ), demonstrating arrest at the G1/S transition. The transcriptome of the first point of re-entry (RE1) still correlates with the 17 hpi IDC time point (ru2009 =u20090.59), but the parasites subsequently rapidly enter S-phase, with re-entry time point 2 (RE2) corresponding to the IDC transcriptome from 22u201324 hpi (ru2009=u20090.36), when parasites initiate DNA synthesis. This progress towards normal cell cycle progression is further observed with RE3 showing positive correlation from 28 hpi onwards. Finer evaluation of the re-entry will require dynamic evaluation of RNA synthesis. Collectively, these data indicate that DFMO-treated parasites undergo cell cycle arrest at the G1/S transition and, upon external mitogen supply, re-initiate the cell cycle for proliferation.\" - Molecular characteristics of a cell cycle arrest at the G1/S phase transition - \"We subsequently used DFMO treatment to arrest parasites at the G1/S transition and define the molecular events of cell cycle arrest at this point in parasite development. We compared the DMFO-treated cell cycle-arrested transcriptome to the transcriptomes of P. falciparum parasites obtained after various chemical or physical perturbations through chemical signature clustering (Fig._). Compounds with similar chemical backgrounds or biological effects clustered together, but were distinct from the DFMO-treated transcriptome. Furthermore, the cell cycle-arrested transcriptome differs from previous descriptions of amino acid-induced hibernation in P. falciparum, with regulators of the translational control/eIF2u03B1 pathway (e.g. IK1, PK4 and eIF2u03B1) that cause a delay in completion of the IDC, not present in our dataset. We likewise did not observe a delay during ring stages as was seen during metabolic hibernation. Additionally, the transcriptome of DFMO-treated parasites is different from that of artemisinin-treated parasites, which also characteristically delays ring-stage development (Fig._). Lastly, it is unlikely that the cell cycle arrest induced here is due to general DNA damage signals, since the P. falciparum kinome does not include homologs of the master kinases involved in DNA damage associated G1/S checkpoints in mammals (it should be noted, however, that such a function may be fulfilled by unrelated kinases present in the parasites kinome). All markers associated with mismatch repair and double-strand break repair (e.g. repair proteins RAD54 and RAD51) show decreased abundance in the cell cycle-arrested transcriptome (Supplementary File_), in contrast to treatment with DNA-damaging agents such as methyl methanesulphonate, cisplatin and etoposide, which upregulate multiple DNA repair pathways.\" - \"Figure 5Compound transcription signature clustering. Transcriptomes of drug-treated asexual parasites and parasites undergoing amino acid starvation (turquoise circle, Starve) were separated with a support vector machine (SVM) classifier and subsequently clustered in order to establish relatedness of chemical perturbations to cell cycle-arrested parasites (2u2009mM DFMO-treated, 24u2009h, orange circle). Qu2009=u2009quinine, CQu2009=u2009chloroquine, Stauu2009=u2009 straurosporine, Roscu2009=u2009roscovitine, Apiu2009=u2009apicidine, Febru2009 =u2009febrifugine, Artu2009=u2009artemisinin, leuu2009=u2009leupeptine, Triu2009 =u2009trichostatin A, Colu2009=u2009colchicine, Cyclu2009=u2009cyclosporine.\" - \"The transcriptome of the DFMO-arrested parasites further correlated to molecular markers of cell cycle arrest associated with specific cell cycle compartments in quiescent and non-quiescent yeast transcriptomes (Table_, Supplementary Fig._ and File_). Of the 14 transcripts that characterise the transcriptome of quiescent yeast and for which orthologues exist in malaria parasites, 7 showed increased abundance in cell cycle-arrested P. falciparum, consistent with a quiescent phenotype (Table_). Orthologous transcripts (44/95) associated with the yeast G1 compartment were also increased in abundance during arrest, many of which encode proteins involved in protein synthesis (e.g. ribosomal proteins, eif4a and eif4e). A small subset (12/95) of transcripts related to DNA repair/replication, including that encoding the CDK protein kinase 5, pk5, show decreased abundance during arrest. Notably, 4/11 of the P. falciparum orthologues of G1/S yeast checkpoint-associated transcripts are also decreased in cell cycle-arrested parasites (e.g. pre-replicative complex components (pre-RC) including mcm3u2013 7 and orc2). Similarly, S-phase DNA synthesis processes are decreased in abundance in cell cycle-arrested parasites. However, whilst yeast employs START as a cell cycle checkpoint of growth, no clear correlation of transcriptional marks for this process in P. falciparum parasites was observed.\" - Table 1 Matched cell cycle related orthologues of P. falciparum and yeast transcriptomes perturbed in cell cycle arrest and re-initiation. P. falciparum parasite transcripts were matched to cell cycle associated transcripts from non-quiescent and quiescent yeast transcriptomes (). - \"The P. falciparum cell cycle-arrested transcriptome contained transcripts previously shown to define the G1 phase in the murine malaria parasite P. berghei (Supplementary Fig._ and File_). This included ribosomal protein families and a nucleolar complex protein 2. The majority of transcripts marking S/M transition in murine parasites showed decreased expression in the P. falciparum arrested transcriptome with concomitant increased expression during cell cycle re-entry.\" - \\'Molecular cues of processes involved in cell cycle arrest and re-entry in P. falciparum \\' - \"A proportion of the P. falciparum transcriptome revealed directly opposed expression profiles between cell cycle arrest and re-entry; 975 differentially expressed (DE) transcripts with decreased expression in arrested parasites matched with continual increased abundance profiles in parasites undergoing cell cycle re-entry, the opposite was true for 988 DE transcripts (Fig._, qPCR confirmation Supplementary Fig._). Progressive increases in differential expression was observed between the arrested and re-entered transcriptomes (4.6% and 16.4%, RE1 and RE2, respectively). The RE3 transcriptome drastically diverged from arrested parasites, confirming that re-entry is associated with profound changes. This data were mined for 1) genes that were either highly differentially expressed (u22652-fold DE, log2 FC of 0.75), or 2) genes potentially involved as cell cycle regulation based on functional classification as (i) kinases and (ii) phosphatases (228 individual transcripts); (iii) DNA replication (73 transcripts); (iv) DNA repair (66 transcripts); (v) transcription and (vi) chromatin dynamics (272 transcripts); and (vii) Ca signalling associated processes (95 transcripts, Supplementary Fig._ and File_).\" - \"Figure 6Molecular mechanisms controlling cell cycle re-entry. The transcriptomes of parasites that re-entered their cell cycles (RE1u20133) were analysed in context of genes matching key terms associated with cell cycle regulation using PlasmoDB (v33). (a) The expression profiles of genes of interest that were DE in RE1u20133 are shown in heatmap compared to the arrested transcriptome (Au2009 =u2009arrested). (b) The number of DE (either increased or decreased abundance) genes associated with specific cell cycle-related functional associations in histograms and genes of interest highlighted in grey boxes. Gene colours correspond to the GO functional classes as defined in the inserted key. (c) A gene association network was constructed between putative regulators of cell cycle re-entry by combining co-expression analysis (GRENITS, probability linkage scoreu2009>u2009 0.25) and functional association between genes (STRING v 10.0, combined probability score >0.8). Transcripts that were differentially expressed by RE1 (light grey), RE2 (grey) and RE3 (dark grey) are indicated according to their increased (u25CF ) or decreased (u25BC) transcript abundance. Genes of interest are indicated on the network, (RRu2009=u2009ribonucleoside-diphosphate reductase large (l) subunit pf3d7_1437200 and small subunit (s) pf3d7_1015800; orc3u2009=u2009 conserved unknown protein with orc3 domain, pf3d7_1029900) while the full gene association network is available in Supplementary File_. Associated gene sets are indicated in inserted pie charts coloured according to GO annotations for gene functions.\" - \"Several kinases involved in proliferation/cell division were differentially expressed, 43% of which (63/147) are essential to either P. falciparum or P. berghei development (Supplementary Fig._). This included ark2, nek1, the cdc-2 related kinases crk3, crk4 and crk5 and pk5, of which crk3 & 4 and pk5 are essential to parasite development. Other Ca sensitive kinases including pk2 (an essential Ca/calmodulin-dependent kinase), cdpk4 (a cyclin-dependent kinase), pkac and pkar (catalytic and regulatory subunits of cAMP-dependent PKAs) and cGMP-dependent pkg, were also differentially expressed.\" - \"Of the 27 member ApiAP2 transcription factor family, pf3d7_1115500 increased during DNA synthesis, while pf3d7_0802100, pf3d7_0613800, and pf3d7_0516800, decreased during endomitotic division (RE3, Supplementary Fig._). Additionally, the MYB1 transcription factor, which is essential for progression into schizogony, increased in transcript abundance at re-initiation (RE1). Furthermore, genes that encode proteins involved in epigenetic control are present in the DE set, including decreased levels of histone deacetylase complex subunit (sap18) and two histone acetyl transferases (gcn5 and pf3d7_1020700) and increased transcript abundance for the methyl transferase, set9.\" - \"The dynamics of the DE genes was assessed by probing for their occurrence either immediately upon re-entry in RE1 as early responders (14 genes) or subsequently during RE2 (91 genes, intermediate responders) or RE3 (229 genes, late responders) (Fig._). Re-entry into the cell cycle was characterised by the immediate expression of myb1 and cdpk4 upon polyamine supply, whilst sap18 and pk2 both display decreased expression upon cell cycle re-entry. After this initial expression of genes, RE2 includes kinases (pk5, pkac and pkar), set9, ApiAP2 pf3d7_1115500, DNA primase and an unannotated gene. The latter, pf3d7_0105800, contained an Interpro domain associated with cyclin dependent kinases (IPR000789). Genes involved in the late response (RE3) are enriched in transcription factors (3 ApiAP2 family members) and CDKs and Aurora kinases (crk3, 4, 5 and ark2), with increased expression of pre-RC genes orc1&2 and mcm3, 4, 5, 6, 8 also evident.\" - \"The dynamics in expression of cell cycle-associated genes led us to predict directional relationships between particularly important gene nodes during re-entry through gene association network analysis based on co-expression data obtained at various re-entry time points (Fig._ and Supplementary Fig._, Supplementary File_ ). Four early responders (cpdk4, myb1, sap18 and pf3d7_1112100) are defined as effectors. Cdpk4 is a particularly connected node effecting downstream signalling events (e.g. pk2, pkg, pkac and pkar, cdpk1) with 58 connections to cell cycle related genes involved in Ca signalling, transcription, kinases and phosphatases (Fig._). This suggests that CDPK4 expression could stimulate entry into S-phase and lead to the expression of genes essential for completion of schizogony in P. falciparum, similar to its requirement for entry into S-phase in P. berghei gametogenesis. Myb1 shares 12 connections with cdpk4, including the two essential ApiAP2 transcription factors (pf3d7_0802100 and pf3d7_0613800), providing a link between signalling events and associated transcriptional regulation. Additionally, myb1 associates with epigenetic modulators; its expression is associated with histone methyltransferase, set9 expression and decreased abundance of a putative histone acetyl transferase (pf3d7_1020700) in RE2. The involvement of epigenetic mechanisms is further evident in connections with sap18; its decreased abundance is associated with increased levels of genes involved in DNA replication and transcription, including pre-replicative licensing factors (mcm3u20136 & 8), DNA polymerases and replication factors, with a functional relationship confirmed for pk5 with members of the pre-replicative complex, orc1, orc2 and co-expression with mcm8. Pf3d7_1112100 was decreased in abundance by RE1, but it was co-expressed with genes that almost uniformly increased in abundance in RE2, supporting a repressive function for this regulator. Manual annotation revealed the presence of a Bub1/Mad3 motif (IPR015661), typically associated with regulation of the spindle assembly checkpoint in metazoans. The co-expressed genes of this regulator comprised 55 cell cycle related genes, mostly encoding kinases, transcription factors and Ca signalling mediators. The gene also co-expressed with pk6, a CDK-related kinase, which was increased in abundance during cell cycle arrest and re-initiation. The presence of this regulator within the dataset may indicate the existence of a G1 cell cycle checkpoint in P. falciparum.\" - Taken together, these results suggest that the proliferation decision in P. falciparum following exposure to mitogens is mediated by Ca signalling pathways, possibly influencing activity of MYB1 that subsequently initiates an epigenetically driven cascade of expression, ultimately enabling DNA replication through expression of the pre-RC.- - Here, we describe an effective tool for the study of cell cycle progression and regulation in malaria parasites, and provide an initial comparative transcriptome analysis of parasites undergoing cell cycle arrest and re-entry. DFMO-induced cell cycle arrest occurs at the G1/S transition, similar to other eukaryotes, is fully reversible, and provides a physiologically relevant system to study cell cycle regulatory mechanisms. This cell cycle arrest manifests within a tight window of development, associated to the precise temporal production of the biosynthetic PfAdoMetDC/ODC enzyme. DFMO-induced arrest is further advantageous in its ease of use because it can be induced at any point prior to the production of PfAdoMetDC/ODC, resulting in a relatively broad treatment window. Furthermore, the inhibition can easily be overcome with the addition of exogenous polyamines, without DFMO wash-out required. However, evaluation of re-entry is limited as we could not observe complete synchronous re-entry into the cell cycle, with parasites rather re-initiating schizogony and undergoing DNA replication in the asynchronous manner that is typically observed in untreated parasites. Moreover, the ability to use DFMO treatments across different parasite strains overcomes the problem of variation in timing of the IDC typically associated with laboratory adaptation of P. falciparum parasites. It therefore provides a useful resource for the further evaluation of cell cycle control in malaria parasites. - We used the system to show that P. falciparum parasites can enforce cell cycle control prior to G1/S transition and undergo quiescence-proliferation decision-making. Such an ability to perform quiescence-proliferation decision-making have been questioned in protists, leading to the hypothesis that that S/M phase completion is not controlled in these endomitotic environments, thereby enabling asynchronous nuclear division. However, the finding that core cell division cycling proteins are conserved in Apicomplexa may explain the ability of malaria parasites to tightly synchronise their asexual replication in vivo, implying endogenous cell cycle control. Moreover, it is well established that the parasite can respond to artemisinin-induced stress by delaying their cell cycle progression and inducing a state of dormancy during early ring-stage development. Dormancy is also observed in P. vivax hypnozoites during liver stage development. However, the cell cycle arrest and quiescence observed here is clearly distinct from these examples of dormancy, as (i) it does not delay ring-stage development, (ii) it is fully reversible, (iii) it does not result in irreversible damage to the parasite (in contrast to other perturbations including treatment with the CDK inhibitor Roscovitine and L-mimosine) and (iv) there is no evidence of DNA damage signals observed in our system. The halt in the cell cycle that we observe at the G1/S transition also does not show evidence of the typical markers for the yeast START checkpoint found in G1. Rather, the halt in cell cycle at the G1/S transition is due to growth factor restriction of polyamines as mitogens, as has been extensively described for various eukaryotes. This is more reminiscent of the metabolic hibernation induced by amino acid starvation, although direct comparison indicated that the molecular events associated with the G1/S halt and quiescence observed here are unique and are not observed in other transcriptomes resulting from environmental perturbations to the Plasmodium parasites. - We subsequently used DFMO-induced cell cycle arrest to explore the potential regulatory mechanisms involved in driving the quiescence-proliferation decision-making processes. Although our results cannot precisely define modulations in transcription during cell cycle arrest versus changes in mRNA stability, future studies can examine the effect of DFMO arrest using the recently described method to biosynthetically label nascent transcripts in P. falciparum. The dynamic nature of the transcriptome data describing cell cycle re-entry allowed inference of hierarchical causality. Thus, Ca signalling appear to be key for P. falciparum to progress into S-phase, with a central role proposed for CDPK4 and PK2 in crossing of the G1/S boundary. PbCDPK4 is required for DNA replication and progression to S-phase in P. berghei male gametocytes and changes in Ca levels are also associated with a G1/S transition in mammalian cells and yeast. Additional control of gene expression is implied through the dynamic pattern of expression of the transcription factor MYB1, which is essential for trophozoite to schizont transition and shows low levels in arrested parasites and increased expression upon re-entry. Transcriptional regulation by ApiAP2 transcription factors to progress through S-phase cannot be ruled out, since seven ApiAP2 transcription factors increased in transcript abundance during cell cycle re-entry, of which pf3d7_0802100 and pf3d7_0613800 are genetically validated as essential for asexual development in P. berghei. Interestingly, epigenetic control of gene expression during G1/S is proposed here to involve SAP18 as a transcriptional co-repressor and its functional association with HDAC1 and chromatin assembly factor 1. However, confirmation of changes in the typical euchromatic permissive chromatin state (reviewed in) associated with asexual development or changes in the dynamic chromatin proteome, needs to be further established. PK5 is a confirmed role player in cell cycle progression, particularly through regulating components of the pre-RC, including regulation of ORC1 through phosphorylation by PK5. This confirms that the pre-RC is central to cell cycle progression in Plasmodium. While several cyclins and CDKs are implicated in regulation of the mammalian cell cycle, the involvement of some of these proteins in cell cycle control in P. falciparum is unclear. Our data confirms the expression of NIMA (nek1) and Aurora (ark2) kinases and their involvement in cell cycle progression. - Taken together, our data demonstrate that P. falciparum parasites to undergo quiescence-proliferation decision-making as a means to control the initial phases of its cell cycle. Mitogen dependency should be considered a contributing feature to cell cycle control in Plasmodium spp., thereby inducing quiescence to ensure survival until more favourable conditions occurs. We show that the parasite can sense signals for cellular quiescence and induce a G1 arrest preventing the G1/S transition. Passing this boundary seems to be an absolute requirement for cell cycle progression before entering the endocyclic mitotic cycles. The ensuing endocyclic division however, might not be well-controlled, causing asynchronous nuclear division, resulting in a varied number of daughter merozoites produced from individual schizonts depending on the nutritional conditions in which the parasite is cultivated. This is reminiscent of early embryonic development in Drosophila, which is characterised by syncytial rounds of mitosis without cytokinesis, taking the parasynchronous division of this organism into account. - \"The regulatory mechanisms underlying the quiescence-proliferation decision-making process appear to be unique to malaria parasites and involve transcriptional control, Ca signalling and effector kinases. With this first set of evidence for cell cycle regulation in Plasmodium spp., DMFO-based cell cycle arrest will open avenues for future validation of the mechanisms underlying these processes in the parasite. Cell cycle regulators are being considered as drug targets in a variety of human disorders including cancer, neurodegenerative diseases, diabetes and inflammation. The unique nature of cell cycle control in P. falciparum therefore holds promise for exploiting essential molecules that regulate the parasites cell cycle as novel therapeutics.\"- - \\' In vitro cultivation of intraerythrocytic P. falciparum parasites\\' - \"In vitro cultivation of intraerythrocytic P. falciparum parasites holds ethics approval from the University of Pretoria (EC120821-077). P. falciparum (3D7, NF54, K1, Dd2 and HB3) parasites (MRA-102, MRA-1000, MRA-159, MRA-156 and MRA-155 from BEI resources/MR4, , mycoplasma free authenticated and genetically validated) were maintained as described(see Supplementary Methods). Parasites were synchronised by 3 repetitive, 6-hourly staggered rounds of iso-osmotic sorbitol exposure. Invasion was defined as time zero (synchronised cultures) and DFMO treatment initiated when 95% of the parasites were within 6u2009h of time zero. A time course in which parasite populations were synchronised with three successive rounds of sorbitol exposure of ring stages and subsequent magnetic purification of mature stages), captured 2 hourly from invasion to the end of the lytic cycle (46u201348u2009h later) was used as reference to ensure consistent hours post invasion (hpi) determination in subsequent experiments. Gametocytogenesis was induced with NF54 cultures as described. Polyamine levels were determined as described before.\" - \"Perturbation of intraerythrocytic P. falciparum parasites with DL-u03B1-difluoromethyl ornithine (DFMO)\" - \"DFMO dose-response analyses in the presence and absence of putrescine dihydrochloride (2u2009mM) were initiated with ring-stage intraerythrocytic P. falciparum parasite cultures (1% haematocrit, 1% parasitaemia) using SYBR Green I fluorescence measured with a Fluoroskan Ascent FL microplate fluorometer (Thermo Scientific, excitation at 485u2009nm and emission at 538u2009nm). Nonlinear regression curves were generated using GraphPad Prism 5.\" - Measurement of nucleic content - \"The effect of DFMO-induced arrest on parasite DNA content was determined flow cytometrically. Intraerythrocytic P. falciparum parasites (~5% parasitaemia, 5% haematocrit, 50000 events captured) were treated with DFMO (IC90, 2u2013 10u2009mM depending on parasite strain) from invasion (time zero) with samples taken every 6u2009h until 48u2009hours after treatment. Alternatively, DFMO treatment (IC90) of 2u201310 hpi ring-stage intraerythrocytic P. falciparum parasites (~1% parasitaemia, 1% haematocrit, 10 000 parasitised events captured) was reversed after 24u2009h with 2u2009mM putrescine and proliferation monitored 96u2009h post DFMO treatment. Cell suspension aliquots were fixed with 1:10 ratio of 0.025% glutaraldehyde (v/v) and cell pellets stained with 10u2009xB5 l 1:1000 SYBR Green I (Invitrogen) for 30u2009min in the dark at room temperature. Analyses were performed on a Becton Dickenson FACS ARIA with 515u2013545u2009 nm band pass filters (FL-1 channel, FITC signal) for SYBR Green I (DNA) and analysed using FlowJo vX.0.7, with gating as described in the Supplementary Methods. The effect of DFMO on nuclear division was confirmed by fluorescent microscopy by removing samples from treated and control parasites as above before methanol fixation on microscope slides and staining with 10u2009xB5l 1:1000 SYBR Green I for 30u2009min in the dark at room temperature. Stained samples were visualised using Zeiss LSM 880 Confocal Laser Scanning Microscope (LSM) - Airyscan detector for super-resolution imaging using a 488u2009nm laser and 495u2013550 BP filter.\" - \"Cell cycle compartment analysis was performed flow cytometrically by investigating both DNA and RNA content. DFMO treatment was initiated on 2u201310 hpi ring-stage parasitised erythrocytes (1% haematocrit, 10% parasitaemia) and uninfected erythrocytes (1% haematocrit) for 24u2009h prior to the addition of putrescine dihydrochloride. Samples were stained with 10u2009xB5l 1:1000 SYBR Green I (Invitrogen) for DNA quantification and subsequently with 10u2009u03BCl Pyronin Y (100u2009 u03BCg/ml) for RNA quantification for 30u2009min each in the dark at room temperature, as has been described. Analyses were performed for SYBR Green I (DNA) as described above with Pyronin Y (RNA) measured at 564u2013606u2009nm (FL-2 channel, PE signal, 10 000 events of parasitised cells captured). Compensation was calculated from single-stained Pyronin Y and single-stained SYBR Green I samples.\" - Oligonucleotide DNA microarray and analysis - \"Oligonucleotide DNA microarrays were performed to analyse the global transcriptomic changes of either DFMO-arrested parasites or DFMO-arrested parasites followed by putrescine-reversal using custom microarray slides that contained 12 468 oligos (60-mer, Agilent Technologies) based on the complete P. falciparum genome as described. Sorbitol synchronised ring-stage parasitised erythrocytes (5% haematocrit, 10u201315% parasitaemia) were incubated with DFMO (IC90) at 37u2009 xB0C for 24u2009h prior to sampling (Arrested) or left untreated (Control), for two independent biological replicates, performed in technical duplicates. Additionally, after 24u2009h DFMO pressure, the cell cycle arrest was reversed with 2u2009mM putrescine dihydrochloride and samples thereafter taken after 3u2009h (REu2009=u20091), 6u2009h (RE2) or 12u2009h (REu2009=u20093). From 50u2009ml parasite cultures sourced at specific time points, RNA isolation, cDNA synthesis and dye coupling with Cy3 (reference pool containing a mix of treated, control and re-entry samples) or Cy5 (individual samples) were performed as described. The slides were scanned using a GenePixu2122 4000B (Axon Instruments) scanner.\" - \"Initial data processing was performed using GenePix Pro 5.1 (Acuity). Normalization and the identification of differentially regulated genes was performed using R v3.2.3 () and Bioconductor with the limma and marray packages. Data normalization included robust-spline within-slide normalization, Gquantile between-slide normalization and the fit of a linear model to obtain log2 (Cy5/Cy3) expression values for each condition (complete dataset in Supplementary File_). DE genes were defined as those with log2 fold change (log2 FC) of 0.5 in either direction. DE genes were calculated between the control and arrested parasites at 24u2009h and between time-matched arrested (DFMO treated) and reversed (with putrescine following DFMO treatment for 24u2009h) samples for the re-entry time points. Microarray data was confirmed by qPCR, complete method in Supplementary Methods.\" - Data analyses - \"Using the hourly IDC transcriptome as a reference time line of global transcript abundance, Pearson correlation was used to align the transcriptional profiles of arrested, control and re-entered parasites to the HB3 IDC transcriptome (Supplementary File_). Where not otherwise stated, data visualization of microarray data was performed using TIGR MeV 4.9.0.\" - Compound signature clustering was performed on 185 expression patterns found within differential expression datasets for P. falciparum parasites treated with 21 compounds in 30 classes to determine genes whose expression patterns were significantly aligned due to the presence of a particular compound. The same process was followed for inclusion of cell cycle-arrested and amino acid starved parasites. A Support Vector Machine (SVM) was constructed using the SVM function from the e1071 package in R, additional details in Supplementary Methods. - \"The P. falciparum transcriptomes for the arrested parasites and those that re-entered their cell cycle (RE1, RE2 and RE3) were correlated to transcriptomes from model organisms with defined cell cycle compartments. Yeast transcriptomes for both non-quiescent, normally dividing yeast and quiescent yeast were obtained from the Saccharomyces Genome Database () using datasets for Friedlander_2006_PMID_16542486 and Aragon_2006_PMID_16507144. These transcriptomes were mapped to cell cycle compartments as defined within the database, with phenotype analysis for mitotic cell cycle compartment and associated events. P. falciparum orthologues for the yeast genes were identified with InParanoid8 (). In addition, P. falciparum orthologues to genes assigned to P. berghei cell cycle compartments were mined from Hall et al. (Supplementary File_).\" - \"Functional cluster analyses associated with cell cycle events were determined through text mining of PlasmoDB v33 (search terms: kinas*, u201CDNA replicationu201D , u201CDNA damageu201D, u201CDNA repairu201D, transcription, histon*, ApiAP2, calcium and Ca*) and matching resultant gene identifiers (from P. falciparum 3D7) with the differential expression dataset for the P. falciparum cell cycle-arrested and re-entry transcriptomes (Supplementary File_). Essentiality was evaluated from empirical data available on PlasmoGem () and PhenoPlasm (). For the Casubset, additional gene identifiers were obtained from literature.\" - Gene association network filtering and co-expression regulation network inference - \"A gene association network (GAN) was filtered for cell cycle associated DE transcripts in RE1, RE2, RE3 in STRING (v10.0), filtering categories for empirical interactions, co-expression inference, curated database entries and fusion-based entries, with a combined threshold of u22650.8. A de novo putative gene regulation network (GRN) was created from dynamic RNA transcription and decay data to evaluate co-expressing genes using the GRENITS package (Bioconductor in R). Genes that were differentially expressed in RE1 and RE2 in the cell cycle-related datasets (Supplementary File_) were probed for regulation by co-expression against the subsequent re-entered time points (RE2 and RE3). The number of links above a varied threshold range and the distribution of interaction strength over probability (Supplementary File_) were used to guide a threshold cut off (>0.25). The GAN and GRN were merged into a single GAN and visualised in Cytoscape v.3.5.0. Nodes with single interactions that were not identified as regulators but as regulated were collapsed into pie charts according to their functional cluster in order to visually simplify the network.\"- - \\' \\' - The microarray data has been submitted to the Gene Expression Omnibus with accession number GSE92289 (). - \\' \\'- - \"WHO, World Malaria Report. (2017).Bozdech, Z. et al. The transcriptome of the intraerythrocytic developmental cycle of Plasmodium falciparum. PLoS Biol 1, E5 (2003)._ _ Josling, G. & Llinas, M. Sexual development in Plasmodium parasites: knowing when its time to commit. Nature Rev Microbiol 13, 573u2013586 (2015)._ _ _ Arnot, D. E., Ronander, E. & Bengtsson, D. C. The progression of the intra-erythrocytic cell cycle of Plasmodium falciparum and the role of the centriolar plaques in asynchronous mitotic division during schizogony. Int J Parasitol 41, 71u201380 (2011)._ _ _ Doerig, C., Endicott, J. & Chakrabarti, D. Cyclin-dependent kinase homologues of Plasmodium falciparum. Int J Parasitol 32, 1575u20131585 (2002)._ _ _ Doerig, C., Chakrabarti, D., Kappes, B. & Matthews, K. The cell cycle in protozoan parasites. Prog Cell Cycle Res 4, 163u2013183 (2000)._ _ _ Doerig, C. & Chakrabarti, D. In Malaria parasites: genomes and molecular biology (eds Waters, A. P. & Janse, C. J.) 249u2013287 (Caister Academic Press, 2004).Francia, M. E. & Striepen, B. Cell division in apicomplexan parasites. Nat Rev Microbiol 12, 125u2013 136 (2014)._ _ _ Striepen, B., Jordan, C. N., Reiff, S. & van Dooren, G. G. Building the perfect parasite: cell division in apicomplexa. PLoS Pathog 3, e78 (2007)._ _ Jacobberger, J. W., Horan, P. K. & Hare, J. D. Cell cycle analysis of asexual stages of erythrocytic malaria parasites. Cell Prolif 25, 431u2013445 (1992)._ _ _ Gritzmacher, C. A. & Reese, R. T. Protein and nucleic acid synthesis during synchronized growth of Plasmodium falciparum. J Bacteriol 160, 1165u2013 1167 (1984)._ _ _ _ Merckx, A. et al. Identification and initial characterization of three novel cyclin-related proteins of the human malaria parasite Plasmodium falciparum. J Biol Chem 278, 39839u201339850 (2003)._ _ _ Chulay, J. D., Haynes, J. D. & Diggs, C. L. Plasmodium falciparum: assessment of in vitro growth by [H]hypoxanthine incorporation. Exp Parasitol 55, 138u2013146 (1983)._ _ _ Grimberg, B. T., Erickson, J. J., Sramkoski, R. M., Jacobberger, J. W. & Zimmerman, P. A. Monitoring Plasmodium falciparum growth and development by UV flow cytometry using an optimized Hoechst-thiazole orange staining strategy. Cytometry A 73, 546u2013554 (2008)._ _ Spencer, S. L. et al. The proliferation-quiescence decision is controlled by a bifurcation in CDK2 activity at mitotic exit. Cell 155, 369u2013383 (2013)._ _ _ Babbitt, S. E. et al. Plasmodium falciparum responds to amino acid starvation by entering into a hibernatory state. Proc Natl Acad Sci USA 109, E3278u20133287 (2012)._ _ _ Codd, A., Teuscher, F., Kyle, D. E., Cheng, Q. & Gatton, M. L. Artemisinin-induced parasite dormancy: a plausible mechanism for treatment failure. Malar J 10, 56 (2011)._ _ Naughton, J. A. & Bell, A. Studies on cell-cycle synchronization in the asexual erythrocytic stages of Plasmodium falciparum. Parasitology 134, 331u2013337 (2007)._ _ _ Oredsson, S. M. Polyamine dependence of normal cell-cycle progression. Biochem Soc Trans 31, 366u2013370 (2003)._ _ _ Landau, G. et al. Expression profiling and biochemical analysis suggest stress response as a potential mechanism inhibiting proliferation of polyamine-depleted cells. J Biol Chem 287, 35825u201335837 (2012)._ _ _ van Brummelen, A. C. et al. Co-inhibition of Plasmodium falciparum S-adenosylmethionine decarboxylase/ornithine decarboxylase reveals perturbation-specific compensatory mechanisms by transcriptome, proteome, and metabolome analyses. J Biol Chem 284, 4635u20134646 (2009)._ _ Llinas, M., Bozdech, Z., Wong, E. D., Adai, A. T. & DeRisi, J. L. Comparative whole genome transcriptome analysis of three Plasmodium falciparum strains. Nucleic Acids Res 34, 1166u20131173 (2006)._ _ _ Lambros, C. & Vanderberg, J. P. Syncrhonization of Plasmodium falciparum in erythrocytic stages in culture. J Parasitol 65, 418u2013420 (1979)._ _ _ Harper, J. V. Synchronization of cell populations in G1/S and G2/M phases of the cell cycle. Methods Mol Biol 296, 157u2013166 (2005)._ _ _ Toettcher, J. E. et al. Distinct mechanisms act in concert to mediate cell cycle arrest. Proc Natl Acad Sci USA 106, 785u2013790 (2009)._ _ _ _ Niemand, J., Louw, A. I., Birkholtz, L. & Kirk, K. Polyamine uptake by the intraerythrocytic malaria parasite. Plasmodium falciparum. Int J Parasitol 42, 921u2013929 (2012)._ _ _ Hoppe, H. C., Verschoor, J. A. & Louw, A. I. Plasmodium falciparum: a comparison of synchronisation methods for in vitro cultures. Exp Parasitol 72, 464u2013467 (1991)._ _ _ Khan, T., van Brummelen, A. C., Parkinson, C. J. & Hoppe, H. C. ATP and luciferase assays to determine the rate of drug action in in vitro cultures of Plasmodium falciparum. Malar J 11, 369 (2012)._ _ Assaraf, Y. G., Golenser, J., Spira, D. T., Messer, G. & Bachrach, U. Cytostatic effect of DL-alpha-difluoromethylornithine against Plasmodium falciparum and its reversal by diamines and spermidine. Parasitol Res 73, 313u2013318 (1987)._ _ _ Naniche, D., Reed, S. I. & Oldstone, M. B. Cell cycle arrest during measles virus infection: a G0-like block leads to suppression of retinoblastoma protein expression. J Virol 73, 1894u20131901 (1999)._ _ _ _ Darzynkiewicz, Z., Evenson, D., Staiano-Coico, L., Sharpless, T. & Melamed, M. R. Relationship between RNA content and progression of lymphocytes through S phase of cell cycle. Proc Natl Acad Sci USA 76, 358u2013 362 (1979)._ _ _ _ Painter, H. J., Carrasquilla, M. & Llinas, M. Capturing in vivo RNA transcriptional dynamics from the malaria parasite Plasmodium falciparum. Genome Res 27, 1074u20131086 (2017)._ _ _ Hu, G. et al. Transcriptional profiling of growth perturbations of the human malaria parasite Plasmodium falciparum. Nat Biotechnol 28, 91u201398 (2010)._ _ _ Leete, T. H. & Rubin, H. Malaria and the cell cycle. Parasitol Today 12, 442u2013444 (1996)._ _ _ de Hoon, M. J., Imoto, S., Nolan, J. & Miyano, S. Open source clustering software. Bioinformatics 20, 1453u20131454 (2004)._ _ Gupta, D. K., Patra, A. T., Zhu, L., Gupta, A. P. & Bozdech, Z. DNA damage regulation and its role in drugrelated phenotypes in the malaria parasites. Sci Rep 6, 23603 (2016)._ _ _ Aragon, A. D., Quinones, G. A., Thomas, E. V., Roy, S. & Werner-Washburne, M. Release of extractionresistant mRNA in stationary phase Saccharomyces cerevisiae produces a massive increase in transcript abundance in response to stress. Genome Biol 7, R9 (2006)._ _ Friedlander, G. et al. Modulation of the transcription regulatory program in yeast cells committed to sporulation. Genome Biol 7, R20 (2006)._ _ Patterson, S. et al. Pre-replication complex organization in the atypical DNA replication cycle of Plasmodium falciparum: characterization of the mini-chromosome maintenance (MCM) complex formation. Mol Biochem Parasitol 145, 50u201359 (2006)._ _ _ Johnson, A. & Skotheim, J. M. Start and the restriction point. Curr Opin Cell Biol 25, 717u2013723 (2013)._ _ _ Hall, N. et al. A comprehensive survey of the Plasmodium life cycle by genomic, transcriptomic, and proteomic analyses. Science 307, 82u201386 (2005)._ _ _ _ Ward, P., Equinet, L., Packer, J. & Doerig, C. Protein kinases of the human malaria parasite Plasmodium falciparum: the kinome of a divergent eukaryote. BMC Genomics 12, 79 (2004)._ _ Solyakov, L. et al. Global kinomic and phospho-proteomic analyses of the human malaria parasite Plasmodium falciparum. Nat Commun 2, 565 (2011)._ _ Painter, H. J., Campbell, T. L. & Llinas, M. The Apicomplexan AP2 family: integral factors regulating Plasmodium development. Mol Biochem Parasitol 176, 1u20137 (2011)._ _ _ Coetzee, N. et al. Quantitative chromatin proteomics reveals a dynamic histone post-translational modification landscape that defines asexual and sexual Plasmodium falciparum parasites. Nat Sci Rep 4, 607 (2017)._ _ _ Brochet, M. & Billker, O. Calcium signalling in malaria parasites. Mol Microbiol 100, 397u2013408 (2016)._ _ _ Sanderson, T. & Rayner, J. C. PhenoPlasm: a database of disruption phenotypes for malaria parasite genes. Wellcome Open Res 2 (2017)._ _ Gissot, M., Briquet, S., Refour, P., Boschet, C. & Vaquero, C. PfMyb1, a Plasmodium falciparum transcription factor, is required for intra-erythrocytic growth and controls key genes for cell cycle regulation. J Mol Biol 346, 29u201342 (2005)._ _ _ Billker, O. et al. Calcium and a calcium-dependent protein kinase regulate gamete formation and mosquito transmission in a malaria parasite. Cell 117, 503u2013514 (2004)._ _ _ Deshmukh, A. S. et al. Regulation of Plasmodium falciparum Origin Recognition Complex subunit 1 (PfORC1) function through phosphorylation mediated by CDK-like kinase PK5. Mol Microbiol 98, 17u201333 (2015)._ _ _ Butler, C. L. et al. Identifying Novel Cell Cycle Proteins in Apicomplexa Parasites through Co-Expression Decision Analysis. Plos One 9, e97625 (2014)._ _ _ Cubi, R. et al. Laser capture microdissection enables transcriptomic analysis of dividing and quiescent liver stages of Plasmodium relapsing species. Cell Microbiol 19, e12735 (2017)._ _ Birkholtz, L. In Encyclopedia of Malaria (eds Hommel, M. & Kremsner, P. G.) (Springer, 2013).Casero, R. A. & Marton, L. J. Targeting polyamine metabolism and function in cancer and other hyperproliferative diseases. Nature Rev Drug Discov 6, 373u2013390 (2007)._ _ _ Gerner, E. W. & Meyskens, F. L. Jr. Polyamines and cancer: old molecules, new understanding. Nature Rev Cancer 4, 781u2013792 (2004)._ _ _ Pegg, A. E. Mammalian polyamine metabolism and function. IUBMB Life 61, 880u2013 894 (2009)._ _ _ Wallace, H., Fraser, A. & Hughes, A. A perspective of polyamine metabolism. Biochem J 376, 1u201314 (2003)._ _ _ Wallace, H. M. The polyamines: past, present and future. Essays Biochem 46, 1u20139 (2009)._ _ _ Kahl, C. R. & Means, A. R. Regulation of cell cycle progression by calcium/calmodulin-dependent pathways. Endocr Rev 24, 719u2013736 (2003)._ _ _ Means, A. R. Calcium, calmodulin and cell cycle regulation. FEBS Lett 347, 1u20134 (1994)._ _ _ _ Modrzynska, K. et al. A Knockout Screen of ApiAP2 Genes Reveals Networks of Interacting Transcriptional Regulators Controlling the Plasmodium Life Cycle. Cell Host Microbe 21, 11u2013 22 (2017)._ _ _ Ay, F. et al. Multiple dimensions of epigenetic gene regulation in the malaria parasite Plasmodium falciparum: gene regulation via histone modifications, nucleosome positioning and nuclear architecture in P. falciparum. Bioessays 37, 182u2013194 (2015)._ _ _ Carvalho, T. G., Doerig, C. & Reininger, L. Nima- and Aurora-related kinases of malaria parasites. Biochim Biophys Acta 1834, 1336u2013 1345 (2013)._ _ _ OFarrell, P. H. Quiescence: early evolutionary origins and universality do not imply uniformity. Philos Trans R Soc Lond B Biol Sci 366, 3498u20133507 (2011)._ _ Mancio-Silva, L. et al. Nutrient sensing modulates malaria parasite virulence. Nature 547, 213u2013216 (2017)._ _ _ _ Crest, J., Oxnard, N., Ji, J. Y. & Schubiger, G. Onset of the DNA replication checkpoint in the early Drosophila embryo. Genetics 175, 567u2013584 (2007)._ _ _ Verlinden, B. K. et al. Interrogating alkyl and arylalkylpolyamino (bis)urea and (bis)thiourea isosteres as potent antimalarial chemotypes against multiple lifecycle forms of Plasmodium falciparum parasites. Bioorg Med Chem 23, 5131u20135143 (2015)._ _ _ Teng, R. et al. Metabolite profiling of the intraerythrocytic malaria parasite Plasmodium falciparum by H NMR spectroscopy. NMR Biomed 22, 292u2013302 (2009)._ _ _ Reader, J. et al. Nowhere to hide: interrogating different metabolic parameters of Plasmodium falciparum gametocytes in a transmission blocking drug discovery pipeline towards malaria elimination. Malar J 14, 213 (2015)._ _ Das Gupta, R. et al. 3-Aminooxy-1-Aminopropane and Derivatives Have an Antiproliferative Effect on Cultured Plasmodium falciparum by Decreasing Intracellular Polyamine Concentrations. Antimicrob Agents Chemother 49, 2857u201364 (2005)._ _ Schmid, I., Cole, S. W., Korin, Y. D., Zack, J. A. & Giorgi, J. V. Detection of cell cycle subcompartments by flow cytometric estimation of DNA-RNA content in combination with dual-color immunofluorescence. Cytometry 39, 108u2013116 (2000)._ _ _ Painter, H. J., Altenhofen, L., Kafsack, B. F. & Llinas, M. Whole-Genome Analysis of Plasmodium spp. Utilizing a New Agilent Technologies DNA Microarray Platform. Methods Mol Biol 923, 213u2013219 (2012)._ _ Szklarczyk, D. et al. STRINGv10: protein-protein interaction networks, integrated over the tree of life. Nucleic Acids Res 43, D447u2013452 (2015)._ _ _ GRENITS: Gene Regulatory Network Inference Using Time Series v. R package version 1.24.0. (2012).\"- - \"We thank Esmare Human for technical assistance and AI Louw and T Carvalho for helpful discussions and critical reading of the manuscript. This work was supported by grants from the South African National Research Foundation (UID 84627) and the European Commission u2018EviMalaru201D (ref 242095) to LMB. JN was supported by grants from the Claude Leon Foundation. LMB and JN acknowledge the South African Medical Research Council (MRC) for funding the University of Pretoria Institute for Sustainable Malaria Control as MRC Collaborating Centre for malaria research.\"- - Author notes - \"RixEBtte van Biljon and Jandeli Niemand contributed equally.\" - Authors and Affiliations - \"Department of Biochemistry, Genetics & Microbiology, Institute for Sustainable Malaria Control, University of Pretoria, Private Bag x20, Hatfield, 0028, South AfricaRixEBtte van Biljon,_Jandeli Niemand,_Roelof van Wyk,_Katherine Clark,_ Bianca Verlinden,_Clarissa Abrie,_Hilde von GrxFCning,_AnnxE9l Smit,_Janette Reader_&_Lyn-MarixE9 BirkholtzCentre for Bioinformatics and Computational Biology, Department of Biochemistry, Genetics & Microbiology, University of Pretoria, Private Bag x20, Hatfield, 0028, South AfricaWerner SmidtDepartment of Biochemistry & Molecular Biology, Centre for Malaria Research, Pennsylvania State University, University Park, Harrisburg, PA, 16802, USAHeather Painter_ &_Manuel LlinxE1sDepartment of Chemistry, Pennsylvania State University, University Park, Harrisburg, PA, 16802, USAManuel LlinxE1sInfection and Immunity Program, Monash Biomedicine Discovery Institute and Department of Microbiology, Monash University, Victoria, 3800, AustraliaChristian Doerig\" - \"AuthorsRixEBtte van BiljonYou can also search for this author in _Jandeli NiemandYou can also search for this author in _Roelof van WykYou can also search for this author in _Katherine ClarkYou can also search for this author in _Bianca VerlindenYou can also search for this author in _Clarissa AbrieYou can also search for this author in _Hilde von GrxFCningYou can also search for this author in _Werner SmidtYou can also search for this author in _AnnxE9l SmitYou can also search for this author in _Janette ReaderYou can also search for this author in _Heather PainterYou can also search for this author in _Manuel LlinxE1sYou can also search for this author in _Christian DoerigYou can also search for this author in _Lyn-MarixE9 BirkholtzYou can also search for this author in _\" - Contributions - L.M.B. and J.N. conceptualised the work. J.N., R.V.B., K.C., A.S., C.A. and J.R. performed the experiments and J.N., R.V.B., R.V.W., B.V., H.V.G., W.S. were involved in data analysis. H.P. contributed materials and analysed data. M.L., C.D. provided expertise on transcriptome and cell cycle analysis and with L.M.B., J.N., R.V.B. wrote the final versions of the paper. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing Interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License (). The source of the mouse icons used in Figs._h, a, c, , and Supplementary Figs._a, d, are Free vector graphics on Pixabay. Content on Pixabay is licensed under Pixabay License (). We also thank Jeremy Allen, Ph.D., from Edanz Group () for editing a draft of this manuscript. M.H. is supported by the Research Fellowship of the Uehara Memorial Foundation and grants from the Ministry of Education, Culture, Sports, Sciences and Technology of Japan (KAKENHI 19H03716).\"- - Author notes - \\'These authors contributed equally: Masaki Honda, Masashi Kadohisa.\\' - Authors and Affiliations - \"Department of Transplantation and Pediatric Surgery, Kumamoto University, 1-1-1 Honjo, Chuo-ku, Kumamoto, 860-8556, JapanMasaki Honda,_Masashi Kadohisa,_ Daiki Yoshii_&_Taizo HibiDepartment of Cell Pathology, Kumamoto University, 1-1-1 Honjo, Chuo-ku, Kumamoto, 860-8556, JapanDaiki Yoshii_&_Yoshihiro Komohara\" - \"AuthorsMasaki HondaYou can also search for this author in _Masashi KadohisaYou can also search for this author in _Daiki YoshiiYou can also search for this author in _Yoshihiro KomoharaYou can also search for this author in _Taizo HibiYou can also search for this author in _\" - Contributions - M.H. designed and performed experiments, analyzed data, and wrote the manuscript. M.K. performed experiments, analyzed data, and wrote the manuscript. D.Y. performed experiments and analyzed pathological specimens. Y.K. and T.H. provided material support and reviewed the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License (). b At day 7, lineage-specific gene expression induced by nanotopography was measured using population-based quantitative polymerase chain reaction (QPCR). c, d Image-based cell profiling. At day 2, various measures of chromatin (blue), actin (red), focal adhesion kinase (FAK, yellow), and phosphorylated FAK (pFAK, green) were obtained from images of single cells. Collectively, multivariate dataset containing single-cell measures of focal adhesions, the cytoskeleton, and chromatin is referred to as the u201Cmorphomeu201D. c Representative images of cells stained against different cellular aspects. Black lines show the cell and nucleus outlines, from which morphology measurements were extracted. Cell and nucleus outlines were obtained from the actin and chromatin images, respectively. d Morphome features extracted from 4 stains across single cells. e, f Machine learning for data-driven exploration and model building using the morphome. e Hierarchical clustering uncovered distinct patterns that delineate cell type and nanotopography within the morphome. f Bayesian linear regression created a predictive model that related the morphome to gene expression.\" - \"We employed cells of the musculoskeletal system due to the diverse responses of muscle, bone, cartilage, and fibrous cell types to nanotopographies. Mouse myoblasts, osteoblasts, chondrocytes, fibroblasts, and pre-osteoblast and pre-myoblast progenitors, were grown on nanotopographies. Responses from combinations of each cell type on all nanotopographies were measured, yielding 24 unique combinations of cell type and nanotopography. Effects of nanotopography on conventional morphological characteristics such as cell and nuclear area, actin intensity, focal adhesion area, and intensity (see Supplementary Figs._ and , and Supplementary Note_ ), and nuclear translocation of the mechanosensors YAP and TAZ (see Supplementary Figs._ and ) were evident. Quantitative polymerase chain reaction (QPCR) was then used to assess changes in lineage marker expression induced by nanotopography by day 7 (Fig._). At day 2, we performed image-based profiling (Fig._). From images of the chromatin and actin, the nucleus and the cell body, respectively, was robustly segmented. Within these cellular features, we measured morphology (shape and geometry of different compartments), texture (spatial patterns of fluorescence and therefore organization), intensity (total fluorescence value), and radial distribution of intensity (measuring radial arrangement of fluorescence) of chromatin, actin, focal adhesion kinase (FAK) and phosphorylated FAK (pFAK) (Fig._). The morphome consisted of 624 single-cell measurements (u201Cfeaturesu201D ), with 75 chromatin and nuclear features, 211 actin and whole-cell features, 168 FAK features, and 170 pFAK features. Machine learning was then applied on the morphome: hierarchical clustering was used to uncover distinct patterns of morphological features that distinguish cell type-specific responses to nanotopographies (Fig._); (ii) Bayesian linear regression was then used to predict myogenic, osteogenic, chondrogenic and fibrogenic gene expression induced by nanotopography using the morphome as predictors (Fig._).\" - The morphome captures changes induced by nanotopographies - \"Patterns of nanotopography-induced morphological changes were visible from the morphome (Fig._). Immediately apparent were large blocks of actin, FAK, and pFAK measurements with similar values within a cell type on a specific nanotopography. These features correspond to increasingly complex measures of texture, granularity and radial intensity distribution for chromatin, actin, FAK and pFAK (Fig._ , see Supplementary Table_). Frequency of pixel gray levels measure texture and homogeneity of pixels, with high values indicating coarseness. Granularity measures an objects coarseness, with higher values indicating heterogeneity of pixel intensities and coarser texture. The Zernike coefficient measures the spatial arrangement of intensity as it resembles the increasingly complex Zernike polynomials (Fig._). The Zernike coefficient was used to measure both cell shape and radial distribution of fluorescence intensity of chromatin, actin, pFAK, and FAK. Interestingly, higher order Zernike polynomials resemble the punctate shape and spatial distribution of focal adhesions. This provides an integrative analysis of focal adhesions at the single-cell level compared to traditional measures that define individual adhesion characteristics.\" - \"Fig. 2: Cell response to nanotopography at single-cell level.Osteoblastic, myoblastic, chondroblastic, and fibroblastic cell lines cultured on different nanotopographies were used to obtain the morphome. a Heat map of the morphome across cell types and nanotopographies. The morphome consisted of 624 features that quantitatively measure the cell and nucleus geometries, as well as chromatin, actin, FAK, pFAK characteristics within single cells (see Supplementary Data_). Each feature has an average of 0 and a standard deviation of 1 after normalization across all cell types. The color and intensity of each tile represents the average value of the feature for a particular cell type and nanotopography combination. bu2013f Schematic diagrams of representative morphome features.\" - Nanotopography changes gene expression - \"Gene expression was used to quantitatively determine the effect of nanotopographies on cell function (see Supplementary Fig._ and Supplementary Data_). For comparison, we differentiated the same cells cultured on conventional tissue culture plastic using established biochemical inducers of musculoskeletal differentiation (see Supplementary Methods). We discuss here the statistically significant changes induced by nanotopography on lineage-specific gene expression relevant to the cell type (see Supplementary Table_). Pre-myoblasts showed significantly higher expression of the early lineage marker MYOD1, and of the late markers MYOG and MYH7 when cultured on SQ surfaces relative to FLAT surfaces (Fig._). This myogenic gene expression profile was similar to pre-myoblasts stimulated with biochemical inducers of myogenic differentiation for 4 days (see Supplementary Fig._). Both pre-osteoblasts and osteoblasts showed increased expression of early (RUNX2, SP7) and late (BGLAP, SPP1) osteogenic markers when cultured on NSQ relative to FLAT (Fig._), in line with previous studies. The gene expression profile of both pre-osteoblasts and osteoblasts on NSQ resembled cells osteogenically differentiated after 4 days (see Supplementary Fig._). Chondrocytes cultured on HEX showed increased expression of COL2A1 (early marker) and ACAN (late marker) compared to those cultured on FLAT (Fig._). Chondrogenic gene expression profile induced by SQ and HEX showed the highest similarity with cells chondrogenically differentiated for 4 days (see Supplementary Fig._). Interestingly, this means that each nanotopography favors the gene expression of separate cell phenotypes. Meanwhile, fibroblasts showed increased expression of pathogenic fibrosis markers, TGFB1I1, COL3A1, and ELN on all nanotopographies compared with FLAT (Fig._ ). However, we did not observe any similarities in fibrotic gene expression profile induced by nanotopographies and fibrotic induction (see Supplementary Fig._ ).\" - \"Fig. 3: Gene expression is changed by nanotopography.Changes in musculoskeletal gene expression from musculoskeletal cell types in response to nanotopographies. Measurement of au2013c myogenic markers from pre-myoblasts and myoblasts, du2013 g osteogenic markers from pre-osteoblasts and osteoblasts, hu2013k chondrogenic markers from chondrocytes, lu2013n fibrotic markers from fibroblasts on nanotopographies. See Supplementary Data_ for the complete gene expression data. Precursor (pre-myoblast and pre-osteoblast) and lineage committed (myoblast and osteoblast) cells were from the same cell line, but with lineage committed cells cultured in the presence of inducers of osteogenic or myogenic differentiation. Gene expression is listed in order of increasing maturity for the given cell lineage. QPCR measurements shown here were normalized to the reference gene and cell type on FLAT. All QPCR measurements are given as meanu2009xB1u2009standard deviation from two independent experiments (nu2009=u20096). Open faced squares denote individual QPCR measurements. Significance levels obtained from one-way ANOVA with Tukey s post-hoc test for pairwise comparison. Significance levels were denoted by *(pu2009 <u20090.05), **(pu2009<u20090.01), ***(pu2009<u20090.001), and ****(pu2009 <u20090.0001).\" - Distinct nanotopographies are reflected in the morphome - \"A subset of the morphome, consisting of 185 features, varied significantly across cell types (Fig._, see Supplementary Data_). For ease of visualization of a multivariate dataset, hierarchical clustering was employed to group morphome features of high similarity together and thus reveal morphological profiles across different nanotopographies. Before clustering, each morphome feature was mean centered and normalized, transforming each morphome feature to a relative scale with negative values denoting decrease and positive values denoting increase from the meanu2009=u20090. When taken entirely, hierarchical clustering of the morphome revealed distinct morphological profiles of all combinations of cell type and nanotopography (see Supplementary Fig._). Our results were further distilled to hierarchical clustering of the morphome separated by cell type (Fig._ and Supplementary Data_). When compared to FLAT, pre-myoblasts on SQ showed high average values of: focal adhesion textures, pFAK radial distribution, nuclear morphometry, and chromatin textures (Fig._). The morphome of pre-myoblasts cultured on SQ reflects the need for FAK phosphorylation and preferential localization at stress fiber edges, which is necessary for myotube differentiation. In contrast, myoblasts on SQ showed a particularly high average value for chromatin granularity and nuclear morphometry, and near-zero values for radial distribution of actin and of focal adhesions (Fig._). High-chromatin granularity observed for both pre-myoblasts and myoblasts on SQ denotes chromatin heterogeneity and condensation and transcriptional activity, which is reportedly higher prior to myotube formation.\" - \"Fig. 4: The morphome reveals nanotopography-specific changes to cell morphology.Hierarchical clustering of the morphome within each cell type. The data morphome from each cell type were clustered separately (see Supplementary Data_). Within the cell type-specific morphome, each morphome feature was normalized to have a meanu2009 =u20090 and standard deviationu2009=u20091. The color and intensity of each tile represents the average value of the feature. The morphome features analyzed were comprises 21 chromatin, 42 actin, 23 FAK, and 60 pFAK features. Features included were changed significantly across topographies (pu2009<u20090.05 using one-way ANOVA). The total number of cells analyzed from two independent experiments were: a nu2009=u2009877 pre-myoblasts; b nu2009=u2009931 myoblasts; c nu2009=u2009644 pre-osteoblasts; d nu2009=u2009728 osteoblasts; e nu2009 =u2009619 chondrocytes; and f nu2009=u20091140 fibroblasts.\" - \"Pre-osteoblasts on SQ and NSQ had high average values for pFAK radial distribution, intensity, granularity and texture, and high average values for granularity of chromatin and actin (Fig._). However, pre-osteoblasts on SQ had higher order pFAK and FAK radial distribution than on NSQ, which induced the highest expression of osteogenic markers. The morphome of pre-osteoblasts grown on NSQ featured radially variable actin that resemble bone cells, which have high contractility and actin stress fibers.\" - \"For osteoblasts, the differences between the SQ and NSQ morphome were more prominent: NSQ induced lower average values of focal adhesion granularity, chromatin texture and nuclear morphometry, and higher average values for focal adhesion radial distribution (Fig._) compared to SQ. The osteoblast morphome on NSQ indicates that focal adhesions localize at regular intervals along the periphery, which is associated with osteogenesis. Furthermore, changes in nuclear morphometry attributed to spreading after growth on stiff surfaces is also associated with osteogenic differentiation.\" - \"Chondrocytes on HEX, which significantly increased chondrogenic marker gene expression relative to FLAT, showed high average values of radial distribution, texture and granularity of actin and FAK, high average nuclear morphometry, and low average values of pFAK and chromatin measurements (Fig._). These characteristics reflect the morphological changes (including reduced contractility and stress fiber formation, increased cell circularity, and decreased cell spreading, low-FAK phosphorylation and poor focal adhesion formation) of stem cells undergoing chondrogenesis.\" - \"The morphome of fibroblasts cultured on FLAT had high average values of both actin and focal adhesion measurements (Fig._). The highly uniform radial arrangement of focal adhesions and actin of cells on FLAT indicate reduced polarization and contractile morphology of fibroblasts activated to a fibrotic state. Inflammation pathways are reportedly increased in fibroblasts on HEX, inducing low adhesion that is reflected in low actin and focal adhesion radial distribution. Fibroblasts grown on NSQ and HEX showed low average values of focal adhesion and actin radial distribution but high values when grown on SQ.\" - \"Overall, the morphome reflected cell-type-specific responses to nanotopography. This was highlighted by the dissimilarity of the hierarchically clustered morphome from cell types with similar lineage or origin (e.g., pre-osteoblasts vs. osteoblasts, Table_). Furthermore, by training a Bayesian logistic regression classifier using the morphome as predictors we confirmed that the morphome contains sufficient information to distinguish six different cell types. We observed robust classification of cell type using either an out-of-sample morphome or a morphome obtained from the same dataset (see Supplementary Tables_ and , and Supplementary Note_ ), demonstrating the fact that the morphome contains sufficient information to describe cell types. The logistic regression classifier also indicates the radial arrangement of actin and focal adhesions were critically distinct between musculoskeletal cell types, while the arrangement of actin fibers into stress fibers or into cortical, circular bundles provided information on various cell states.\" - Table 1 Correlation coefficient of morphome features hierarchically clustered by cell type. - \"We also clustered the morphome based on nanotopography (see Supplementary Fig._ and Supplementary Note_). Patterns emerged in the morphome in direct response to nanotopography: NSQ induced high average values of pFAK radial distribution, texture and granularity; and HEX induced high average values of actin radial distribution. Correlation analysis between the dendrograms confirm that the morphome clusters of different nanotopographies were dissimilar to each other (see Supplementary Table_).\" - The morphome robustly predicts gene expression - \"The Spearman rank correlation revealed that varying degrees of correlation exist between morphome features and gene expression (see Supplementary Fig._). We hypothesized that the morphome would sufficiently encompass cell response induced by nanotopography. Thus, we utilized Bayesian linear regression to predict gene expression using the morphome features as predictors (for the explicit model definition, see Methods). A Bayesian linear regression model reflects uncertainty in the estimation of regression weights compared to point value estimates using maximum likelihood regression. Gene expression was modeled independently of each other, thereby creating 14 different equations with variable weighting of the morphome features. Importantly, the model was trained without any prior knowledge of cell type and topography type or parameters (e.g., nanodot diameter or center-to-center distance), instead relying on the morphome to encode both information.\" - \"The morphome clearly captured gene expression changes induced by nanotopography (Fig._). The heterogeneity inherent in single cells, usually uncaptured by population measurements of gene expression, are apparent in the variance of the predictions using the model. The mean absolute error (MAE) for prediction of all genes was between 10% (for prediction of MYOD1, MYOG, and MYH7) and 21% (for prediction of COL3A1, see Supplementary Table_).\" - \\'Fig. 5: The morphome reliably predicts nanotopography-induced gene expression.a The morphome was used to train a Bayesian linear regression model that predicted myogenic, osteogenic, chondrogenic, and fibrotic gene expression. The expression of each gene (response) was trained against linear combinations of morphome features (predictors) and without any prior knowledge on topography parameters. The linear regression model was trained using 60% of the dataset and tested using 40% of the data. Scatterplots show actual and predicted gene expression values by using the test set as input to the model, with each open faced circle showing predicted and actual gene expression from a single morphome. Black diamond shows the median predicted gene expression values. Colors represent the different musculoskeletal genes, with orange denoting myogenic, blue denoting osteoblastic, green denoting chondrogenic and brown denoting fibrotic genes. Mean absolute error (MAE) was obtained by first calculating the difference between actual and predicted gene expression values for each morphome then averaging differences across the entire morphome. b Testing the predictive power of the morphome by leave-one-out validation. To test the predictive power and bias of the morphome, the linear regression model was retrained after exclusion of one combination of cell type and topography. The excluded cell type and topography dataset was used for prediction, from which MAE was calculated. The tile position denotes the cell type and nanotopography combination that was excluded in the model and used for testing, while the color of each tile denotes the MAE.\\' - \"The magnitude of the regression weight reflects the contribution of each morphome feature in predicting gene expression (see Supplementary Data_). Across all 14 genes, pFAK activation, as indicated by pFAK/FAK integrated intensity ratio, consistently contributed to the prediction of all 14 genes. FAK texture and radial distribution, actin texture, and chromatin granularity features considerably contributed to prediction of gene expression (see Supplementary Fig._). pFAK was particularly important to the model due to its relevance in contractility induced by nanotopography, fibrosis and scar tissue formation, in vitro osteogenesis, and chondrogenic maintenance.\" - \"The sensitivity and predictive power of the model was verified by iteratively training new models using a morphome with a held-out combination of cell type and topography (Fig._). Essentially, we exploited the presence of multiple cell types and nanotopographies to be able to train predictive models of gene expression without overfitting to data from any experimental setup (e.g., background staining). Drastic increases in MAE were observed when predicting lineage-specific genes using models that excluded the particular cell type lineage being tested, regardless of nanotopography. The results are logically explained by the fact that a particular cell type contributes the most information to gene expression prediction by virtue of its lineage. Removal of the morphome from the particular cell type in question thus drastically reduces the amount of distinct information in the model. Model prediction after removal of nanotopographies showed consistency in MAE, indicating the generalizability of the model in predicting gene expression from nanotopographies outside of FLAT, SQ, NSQ, and HEX.\" - The morphome predicts cell behavior in a complex environment - \"We demonstrate the application of the linear regression model by predicting gene expression of pre-osteoblasts and fibroblasts co-cultured on nanotopographies. A new morphome was obtained from all cells on the entire nanotopography surface (see Supplementary Fig._). This co-culture morphome was then used as input in the model to predict gene expression (see Supplementary Fig._).\" - \"For visualization, the sum of predicted osteogenic (RUNX2, SP7, BGLAP, and SPP1) and fibrotic (TGFB1I1, COL3, and ELN) genes was plotted against the spatial coordinates of the pre-osteoblasts and fibroblasts. Osteogenic gene expression was highest on NSQ, wherein concentrated areas of enhanced expression levels were present (Fig._). These areas might represent hotspots or nuclei of osteogenic paracrine signaling induced by the NSQ nanotopography. In contrast, osteogenic gene expression was low and homogenous on the FLAT, SQ, and HEX topographies. The uniformity of cell distribution across each nanotopography (see Supplementary Fig._) eliminates the possibility of inadvertent cell clustering as the origin of gene expression changes.\" - \"Fig. 6: Single-cell analysis of morphology and gene expression using the morphome.a, b Predicted response of a pre-osteoblast and fibroblast co-culture to nanotopography. Contour plots show the sum of predicted a osteogenic (RUNX2, SP7, BGLAP, SPP1) and b fibrotic (TGFB1I1, COL3, ELN) gene expression for individual cells on FLAT, SQ, NSQ, and HEX topographies. Pre-osteoblast and fibroblast cells were co-cultured on FLAT, SQ, NSQ, and HEX nanotopographies for 2 days, and their morphome obtained from the entire nanotopography surface. The newly collected morphome was then used as input in the linear regression model (shown in Fig._ ) to predict gene expression. x and y axes of each contour plot shows are spatial coordinates on the nanotopogrpahy substrate, while the color of the contour represents the level of summed gene expression. Scale baru2009=u2009100u2009 xB5m. c, d Morphology and gene expression at the single-cell level is provided by the morphome. Each dot in the scatterplot denotes a single-cell. Nanotopographies are color coded, with FLAT denoted in pink, SQ denoted in purple, NSQ denoted in blue and HEX denoted in green. e, f Cellu2013cell interaction altered by nanotopography. The average changes in e cell morphology and f gene expression between two neighboring cells separated by a specified distance was measured and normalized to the maximum observed change. Data are presented as meanu2009 xB1u2009standard deviation and reported as a function of distance between two cells binned every 125u2009xB5m. Nu2009u2265u20095000 cells per topography from one independent experiment.\" - \"Fibrotic gene expression showed more spatial variability across nanotopographies but was also maximized on the NSQ nanotopography, and largely overlaps with the spatial pattern of osteogenic gene expression (Fig._). This is attributable to the synergistic interaction of osteoblasts and fibroblasts on osteogenic differentiation and mineralization. The predicted effect of high osteogenic gene expression induced by NSQ was verified in the increased osteogenic gene expression at 7 days and mineralization compared to FLAT at 28 days (see Supplementary Fig._). Given that this model is based on single-cell populations, our predictions using the co-culture morphome indicate the capability to encode not just cell-material interaction but a concerted response arising from the cellular milieu.\" - \"We additionally gleaned insights in single-cell responses induced by nanotopography using the morphome. The combination of spatial information and cellular response to nanotopography present in the morphome permits analysis of single cells similar to flow cytometry. As an example, we focused on correlating morphome features with high contribution to the model. Nanotopographies induced indistinct effects on either nuclear perimeter or pFAK activation, yet there was a clear gradient in predicted osteogenic gene expression (Fig._). As nuclei became smaller and pFAK activation increased, both osteogenic and fibrotic gene expression increased. In contrast, nanotopographies exhibited clearly separable effects on actin and pFAK radial distribution, with cells on SQ showing the lowest values (Fig._ ). These particular changes in cell morphology correlated more with predicted osteogenic gene expression than fibrotic gene expression, which lacked clear separation by nanotopography. From the morphome, we found strong evidence to implicate nucleus shape and pFAK activation changed by nanotopography as drivers of osteogenesis.\" - \"The morphome additionally uncovered effects of nanotopography on cellu2013cell interaction at the morphological level and on predicted gene expression (Fig._ ). On average, the effect of FLAT on cellu2013cell interaction and the resulting cell morphology extended up to 1u2009mm (Fig._), yet gene expression changed maximally at a separation distance of only 250u2013375u2009xB5m between neighboring cells (Fig._). In contrast, the average effect of SQ and HEX on cellu2013 cell interaction and morphology were minute and apparent only at short cellu2013 cell separation distances of 250 and 500u2009xB5m, respectively. A predominantly negative effect on pFAK activation was observed between neighboring cells grown on either SQ or HEX. The suppressive effect of nanotopography on cellu2013 cell interaction correlate strongly with the homogeneous gene expression predicted for SQ and HEX (Fig._). On the contrary, long-range effects between cells on NSQ were distinct (Fig._). In contrast to FLAT, neighboring cells on NSQ separated by 1u2009mm or less selectively exhibited drastic changes only in nucleus perimeter and pFAK activation. The long-range effects of cellu2013cell interaction observed in NSQ were clearly manifested in predicted gene expression (Fig._). In fact, NSQ showed a critical distance of 500u2013625u2009xB5m between neighboring cells where fibrotic and osteogenic gene expression were maximally changed. In summary, we observed a clear augmentation in cellu2013cell interaction induced by NSQ compared to FLAT, SQ and HEX. This long-range interaction between neighboring cells on NSQ separated by 1u2009mm or less drove changes in pFAK activation and osteogenesis.\"- - In this study, we present a system that robustly uses morphology to quantitatively predict cell type-specific responses to nanotopography. The morphome, which in this study is the collective morphological measurements of chromatin, actin and focal adhesions within single cells, were found to manifest nanotopography-induced changes in cell morphology and gene expression. The information in the morphome underpinned the robustness of a Bayesian linear regression model for predicting gene expression. The morphome also uncovered biological insights at both the morphological and gene expression levels resulting from nanotopographical perturbation of a complex cell microenvironment. - The morphome-based predictive model reported here offers two distinct advantages over the current state of the art. First, our predictive model utilizes hundreds of cell morphology features accurately predict expression levels of 14 different genes. Moreover, because our model exploits the relationship of the morphome to cell type-specific gene expression, it can be used to predict gene expression induced by new microenvironments, given a set of images of cells grown on them. Some groups have previously used cell morphology to glean insights on cell lineage commitment, cell response to topography, and design rules controlling cell behavior. However, all of these studies only go as far as describing correlations between the cell microenvironment and morphology from a single-cell type for classification. - Second, we use gene expression to determine magnitude of the cell response to nanotopography. This contrasts with the current state of the art, relying heavily on setting arbitrary boundaries in protein or gene expression levels to classify cells into functional cell classes. While this work does not indicate that our nanotopographies maximize functionality at 7 days, our predictive model allows us to easily rank nanotopographies in their effect on different musculoskeletal lineages. An important consequence of this work is in utilizing predicted gene expression levels to compare new topographies. Indeed, our morphome-based approach supports a function-focused exploration of new topographies that will make the current trial-and-error screening approach more efficient. Moreover, since gene expression is highly scalable, our system can be easily adapted to predict the expression level of any gene of interest. - The utility of our morphome-based models was validated by robust prediction of the outcome from a co-culture of osteoblasts and fibroblasts. The complexity of a co-cultured microenvironment prevents direct inference from our single-cell culture results. Yet the osteogenic function induced by NSQ from a complex co-culture system was predicted by the morphome models, and validated by high mineralization observed after 28 days of culture. Our results suggest that the morphome can also manifest cellular changes driven by chemical or paracrine cues. This property of the morphome can be exploited to predict cell behavior in more complex microenvironmental settings. - \"The co-culture experiment also showed that the morphome dataset encompasses, at high resolution, structural, functional and spatial information. Indeed, we took advantage of this informationally rich dataset to uncover enhancement of cellu2013cell interaction (from micron to millimeter range) resulting from a subtle change in nanotopography order. SQ and HEX, both of which present an ordered interface to the cell, suppressed cellu2013cell interaction while cellu2013 cell interaction was apparent at long distances on FLAT and NSQ. This result presents a new mechanism for nanotopography-induced-cell behavior.\" - Clearly, morphome capture is crucial to the ability of the linear regression model to predict nanotopography-induced gene expression. While population-level measures of gene expression strongly indicate cell function, they introduce a measure of uncertainty and biological variability into the linear regression model. Thus, a one-to-one relationship between the morphome and cell function is essential to develop. Non-destructive microscopic and molecular tools that combine spatial and structural information from the morphome with single-cell functional assays are vitally important for establishing quantitative topography structure- and cell-function relationships using the morphome. However, the use of routine methods, such as high-content imaging and QPCR, permits any lab to measure the morphome and to model it against the gene expression in question. - By generating a multivariate morphome dataset and combining it with machine learning, we have created a powerful platform for relating topography structure to gene expression. The predictive power of the Bayesian linear regression model presented here easily lends to sequential experimental design by exploiting uncertainty and variability within the model. Combined with bench-top lithographic techniques and in silico simulation of morphological response to nanotopography, we envision a completely closed-loop system that enables functionally oriented exploration of new topographies.- - Polycarbonate surfaces with nanotopography - \"Surfaces patterned with 120u2009nm diameter and 100u2009nm depth nanopits were fabricated on polycarbonate using injection molding. The following nanotopographies were used: surfaces without nanopits (FLAT); nanopits in a square array with 300u2009nm center-to-center spacing (SQ); nanopits in a square array with ~300u2009 nm center-to-center spacing distorted by 50u2009nmu2009in both x and y directions (NSQ); nanopits in a hexagonal array with 300u2009nm center-to-center spacing (HEX). Samples were cleaned in 70% ethanol and dried before treating with O2 plasma at 120u2009W for 1.5u2009min. Samples were sterilized using UV light in a biological safety cabinet for at least 20u2009min before cell seeding.\" - Cell culture - \"Mouse fibroblast cell line NIH3T3 (ATCC) was cultured in reduced sodium bicarbonate content (1.5u2009g per liter) Dulbeccos modified Eagles medium with (DMEM) supplemented with l-glutamate (2u2009mM), 10% bovine calf serum, and 1% penicillinu2013streptomycin. Mouse C2C12 myoblasts (ATCC) were cultured in DMEM with 20% FBS and 1% penicillinu2013streptomycin, and committed into mature myoblastic cells using DMEM supplemented with 2% horse serum and 1% penicillinu2013 streptomycin. Mouse chondrocytes were cultured in minimum essential medium alpha (MEMu03B1) with nucleosides, ascorbic acid, glutamate, sodium pyruvate supplemented with 10% FBS and 1% penicillinu2013streptomycin. Mouse MC3T3 cells (ATCC) were cultured in MEMu03B1 with nucleosides and l-glutamine without ascorbic acid and supplemented with 10% FBS and 1% penicillinu2013streptomycin. To commit MC3T3 into mature osteoblasts, MC3T3 media was supplemented with 10u2009nM dexamethasone, 50u2009xB5g per ml ascorbic acid and 10u2009mM u03B2-glycerophosphate. Lineage committed progenitor cells, referred here as pre-osteoblasts and pre-myoblasts, were also included in the study to mimic the osteogenic and myogenic regeneration profile in the adult tissue.\" - Cell seeding - \"Cells were harvested from flasks using trypsin in versene buffer and spun down at 400u2009xD7u2009g for 5u2009min. NIH3T3 and MC3T3 cells were resuspended in complete media and seeded at 4000 cells per cm. Chondrocytes and C2C12 were seeded at 2500 cells per cm. Cells were seeded at different densities to ensure single cells at ~30% confluency on each surface after 2 days culture. To ensure homogeneity of seeding, cells were seeded using a device that controls fluid flow. For co-culture studies, MC3T3 and NIH3T3 cells were simultaneously seeded at 2000 cells per cm per cell type in MC3T3 growth media. All cells were grown on nanotopographies for either 2 days (for image-based cell profiling) or 7 days (for gene expression measurement).\" - Gene expression measurement - \"After 7 days, total RNA was obtained from lysed cells according to manufacturer s instructions (Promega ReliaPrep Cell Miniprep kit). Gene expression was measured directly from 5u2009ng RNA using a one-step QPCR kit with SYBR dye (PrimerDesign). A list of the forward and reverse primers used to study different mouse genes is provided in Supplementary Table_. QPCR was run on the BioRad CFX96 platform. Relative gene expression was normalized to the 18S ribosomal RNA reference gene. Gene expression was measured at least twice from each independent experiment. One-way analysis of variance (ANOVA) with Tukeys post-hoc test for multiple comparisons was performed to determine the effect of nanotopography on gene expression compared with FLAT. Statistical significance was considered at pu2009 <u20090.05. Plotting and statistical testing of gene expression data were performed using GraphPad Prism (v7.0a).\" - Immunofluorescence staining - \"After 2 days, cells on surfaces were fixed with 4% paraformaldehyde solution in phosphate buffered saline at 4u2009xB0C for 15u2009min. Fixed cells were then permeabilized and blocked with 10% goat serum and 2% bovine serum albumin in phosphate buffered saline for 1u2009h at room temperature. Cells were stained with the following primary antibodies overnight at 4u2009xB0C: pFAK Y397 (Abcam 39967, 1:400) and FAK (ThermoScientific 396500, 1:400). Afterwards, Alexa Fluor-conjugated secondary antibodies (ThermoScientific, 1:500) against the host species of the primary antibody were used. Alexa Fluor 549-conjugated phalloidin (ThermoScientific, 1:200) were used to visualize the actin cytoskeleton. Cells were also stained with 4u2032,6-diamidino-2-phenylindole (DAPI; (ThermoScientific) to visualize chromatin inside the nuclei. DAPI was previously reported to contribute textural information as a means of alternatively representing chromatin. All surfaces were mounted on 0.17u2009xB5m thick glass coverslips with ProLong mounting medium (ThermoScientific) and dried overnight at 4u2009xB0C before imaging.\" - Image acquisition and morphome extraction - For single population studies, monochrome images of each fluorophore were obtained at x40 magnification (numeric aperture 1.3) using the EVOS FL1 System (ThermoScientific). For co-culture studies, the entire nanotopography surface was imaged and stitched through an automated microscope (EVOS FL2 Auto) with a x40 magnification (numeric aperture 1.3). All images from the same cell type were obtained using the same camera and light settings. Afterwards, image processing and morphome feature extraction were perfomed using CellProfiler (v2.4.0, The Broad Institute). Image processing, including illumination correction and channel alignment, was performed across each independent experiment from the same cell type. Nuclei and cells were segmented from each image, allowing single-cell analysis of shape or morphometric measurements, total and local intensities and textures from chromatin, actin, pFAK and FAK. Measurements were taken from distinct cells from each independent experiment. - Multivariate analysis - \"The morphome initially consisted of a total of 1050 measurements obtained from single cells. Features with zero variance within each batch (e.g., Zernike Phase measurements) were first removed from the dataset. Morphome measurements from single-cell populations and independent experiments were first combined then scaled by subtracting the mean and normalizing by the standard deviation of the dataset to result in a dataset with meanu2009=u20090 and standard deviationu2009 =u20091. Morphome data from co-culture studies were similarly scaled and normalized using the mean and standard deviation from the initial dataset consisting of single-cell populations. A Pearson correlation method at significance level 90% was used to remove features with correlation higher than 0.9 without significantly reducing total data variance (see Supplementary Fig._) using the KMDA (v1.0) package for R. After pre-processing, 624 morphome features were used in the study.\" - Hierarchical clustering - \"To determine the features that were significantly varied across nanotopography, a one-way ANOVA with Tukeys post-hoc test for multiple comparisons was performed. Prior to clustering, each morphome feature was mean centered and normalized to the standard deviation. An agglomerative hierarchical clustering algorithm was performed using a Euclidean distance metric and an average linkage method for cluster linkage using gplots (v3.0.1) package. Membership of each morphome in a cluster was obtained from silhouette analysis using the cluster (v2.0.7-1) package. Hierarchical clustering was visually represented with a dendrogram, and a heatmap with color intensity corresponding to the average values of the morphome features. Dendrogram correlation, which measures the similarities in the grouping of morphome features between two dendrograms, was performed using the corrplot package.\" - Bayesian linear log-Normal regression - \"Only morphome features with an absolute Spearman correlation coefficientu2009 u22650.7 against all examined gene expression markers were used in the linear regression model. The linear regression model used 243 morphome features, containing 22 nuclear morphometry and chromatin, 71 actin, 75 FAK and 75 pFAK measurements, as predictors of the model. QPCR data from each independent experiment was propagated across the corresponding single-cell morphome data from the same independent experiment. For each gene analyzed, data were rescaled from 0 to 1 by normalizing to the maximum gene expression.\" - \"Linear regression was performed as a simple approximation of the relationship between the morphome and myogenic, osteogenic, chondrogenic, and fibrotic gene expression. Established Bayesian inference methods were used to determine the probabilities of observing gene expression with a given morphome set. We consider a linear model where expression of one gene (response y) was predicted through a linear combination of the morphome features (predictors X) transformed by the inverse identity link function. We assume that y follows a log-Normal distribution parametrized by the mean u03BC and variance (sigma _i^2):\" - \"$$y_isim {mathrm{log}};{mathrm{Normal}}left( {mu _i,sigma _i^2} right)$$ (1) \" - \"And that u03BC is a linear function of X parametrized by u03B2:\" - \"$$mu _i = beta _0 + beta _1X_1 + beta _2X_2 + ldots + beta _nX_n$$ (2) \" - \"All model parameters u03B2 were assumed a priori to come from a normal distribution, parametrized by mean and standard deviation:\" - \"$$beta sim {mathrm{Normal}}(0,2)$$ (3) \" - \"Each gene was trained independently resulting in 14 different linear regression equations. A 60u201340% training and test split for Bayesian linear regression was performed randomly and with stratification using the caret (v6.0-81) package for R. The Bayesian linear model was created using the brms (v2.5.0) package for R, which utilizes the Hamiltonian Markov Chain Monte Carlo sampler for estimation of the posterior distribution of u03B2. Bayesian linear modeling was carried out using with 1000 warm-up iterations and 1000 sampling iterations within each chain for three independent chains. All models were confirmed to converge to the equilibrium distribution by confirming low-autocorrelation, potential scale reduction statistic split Rhatu2009u2265u20091, effective sample size was smaller than total sample size. We confirmed the suitability of the prior distribution by ensuring that the data regenerated using the prior predictive distribution (i.e., without seeing any data) closely aligned with the real dataset. Predicted gene expression was performed by using the test set or the morphome obtained from the co-culture study as input to the linear model. Predicted values were averaged across 50 draws from the posterior distribution. The magnitude of the average values of parameters u03B2 indicated feature importance as it effectively weighted the contribution of each morphome feature in predicting gene expression.\" - To determine the predictive power of the morphome, a specific combination of cell type, topography and independent experiment or replicate were iteratively omitted, and the remaining dataset was used to refit new models. Thus, 576 additional models were created to test 24 different cell type combinations across 12 genes and 2 independent experiments. The predictive quality of the models was assessed by predicting the expression of all 14 genes from the held-out cell type, topography and replicate dataset. We report the mean absolute error (MAE) of QPCR prediction for each cell type and topography combination averaged across two independent experiments. MAE was calculated as the average across all absolute differences between predicted and actual gene expression. - \"Analysis of cellu2013cell interaction\" - \"The co-culture morphome was used to predict gene expression at the single-cell level. The dataset was then used to determine changes in gene expression and morphome between neighboring cells of a given distance. Changes in cell morphology and gene expression was performed for each cell against all other cells. Distances between cells were binned to calculate average change in cell morphology and gene expression at intervals of 125u2009xB5m. The changes in cell behavior between two cells was normalized to the maximum value of change observed.\" - Statistics, visualization, and software - Statistical analysis and machine learning were performed using statistical software R (v3.4.3) and its graphical interface RStudio (v1.0). Scatterplots, boxplots, and histograms were generated using ggplot2 (v3.1.0) in R. Interpolation of x and y coordinates for contour plotting was performed using bivariate interpolation of a regularly gridded dataset using akima (v0.6-2) package. Contour plots were created using the fitted.contour function in R, with the nuclear centroid position used as spatial coordinates of the cell. Barcharts and one-way ANOVA analysis of QPCR values were obtained using GraphPad Prism (v7.0a). - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - Raw data (e.g., gene expression data, morphome data), R workspace data that contains all Bayesian linear regression models, and associated code that support the findings of this study are available in Zenodo with the identifier 10.5281/zenodo.3608197 - \\' \\'- - \\' \\' - The statistical models proposed and evaluated in this paper is realized using standard packages in R. The code used to create models and R workspace containing all fitted models are available in the dataset published on Zenodo with the identifier 10.5281/zenodo.3608197 - \\' \\'- - \\' \\' - Nanotopographies used in this study are made in house and can be obtained from the corresponding author upon reasonable request. - \\' \\'- - \"Reynolds, P. M., Pedersen, R. H., Riehle, M. O. & Gadegaard, N. A dual gradient assay for the parametric analysis of cell-surface interactions. Small 8, 2541u2013 2547 (2012)._ _ _ _ Di Rienzo, C. et al. Unveiling LOX-1 receptor interplay with nanotopography: mechanotransduction and atherosclerosis onset. Sci. Rep. 3, 1141, (2013)._ _ _ _ _ Chan, L. Y., Birch, W. R., Yim, E. K. F. & Choo, A. B. H. Temporal application of topography to increase the rate of neural differentiation from human pluripotent stem cells. Biomaterials 34, 382u2013392 (2013)._ _ _ _ Rasmussen, C. H. et al. Enhanced differentiation of human embryonic stem cells toward definitive endoderm on ultrahigh aspect ratio nanopillars. Adv. Funct. Mater. 26, 815u2013823 (2016)._ _ _ Yim, E. K. F., Pang, S. W. & Leong, K. W. Synthetic nanostructures inducing differentiation of human mesenchymal stem cells into neuronal lineage. Exp. Cell Res. 313, 1820u20131829 (2007)._ _ _ _ _ McMurray, R. J. et al. Nanoscale surfaces for the long-term maintenance of mesenchymal stem cell phenotype and multipotency. Nat. Mater. 10, 637u2013644 (2011)._ _ _ _ _ _ Dalby, M. J. et al. The control of human mesenchymal cell differentiation using nanoscale symmetry and disorder. Nat. Mater. 6, 997u20131003 (2007)._ _ _ _ _ Tsimbouri, P. M. et al. Using nanotopography and metabolomics to identify biochemical effectors of multipotency. ACS Nano 6, 10239u201310249 (2012)._ _ _ _ Yang, J. et al. Nanotopographical induction of osteogenesis through adhesion, bone morphogenic protein cosignaling, and regulation of microRNAs. ACS Nano 8, 9941u20139953 (2014)._ _ _ _ Teo, B. K. K. et al. Nanotopography modulates mechanotransduction of stem cells and induces differentiation through focal adhesion kinase. ACS Nano 7, 4785u2013 4798 (2013)._ _ _ _ Zhou, Q. et al. Directional nanotopographic gradients: a high-throughput screening platform for cell contact guidance. Sci. Rep. 5, 16240, (2015)._ _ _ _ _ _ Balaban, N. Q. et al. Force and focal adhesion assembly: a close relationship studied using elastic micropatterned substrates. Nat. Cell Biol. 3, 466u2013472 (2001)._ _ _ _ Kulangara, K., Yang, Y., Yang, J. & Leong, K. W. Nanotopography as modulator of human mesenchymal stem cell function. Biomaterials 33, 4998u20135003 (2012)._ _ _ _ _ Cavalcanti-Adam, E. A. et al. Cell spreading and focal adhesion dynamics are regulated by spacing of integrin ligands. Biophys. J. 92, 2964u20132974 (2007)._ _ _ _ _ Natale, C. F., Ventre, M. & Netti, P. A. Tuning the material-cytoskeleton crosstalk via nanoconfinement of focal adhesions. Biomaterials 35, 2743u2013 2751 (2014)._ _ _ _ Luan, F. & Cordeiro, M. N. D. S. in Recent Trends on QSAR in the Pharmaceutical Perceptions (Khan, M. T. H. ed.) 194u2013241 (Bentham Books, 2012).Jenson, J. M. et al. Peptide design by optimization on a data-parameterized protein interaction landscape. Proc. Natl Acad. Sci. USA 115, e10342u2013e10351 (2018)._ _ _ _ _ Kholodovych, V. et al. Accurate predictions of cellular response using QSPR: a feasibility test of rational design of polymeric biomaterials. Polymer 45, 7367u20137379 (2004)._ _ _ Hook, A. L. et al. Combinatorial discovery of polymers resistant to bacterial attachment. Nat. Biotechnol. 30, 868u2013875 (2012)._ _ _ _ _ Kukumberg, M., Yao, J. Y., Neo, D. J. H. & Yim, E. K. F. Microlens topography combined with vascular endothelial growth factor induces endothelial differentiation of human mesenchymal stem cells into vasculogenic progenitors. Biomaterials 131, 68u201385 (2017)._ _ _ _ Unadkat, H. V. et al. An algorithm-based topographical biomaterials library to instruct cell fate. Proc. Natl Acad. Sci. USA 108, 16565u201316570 (2011)._ _ _ _ _ _ Joergensen, N. L. et al. Topography-guided proliferation: distinct surface microtopography increases proliferation of chondrocytes in vitro. Tissue Eng. Part A 21, 2757u20132765 (2015)._ _ _ _ Dalby, M. J. et al. Nanomechanotransduction and interphase nuclear organization influence on genomic control. J. Cell. Biochem. 102, 1234u20131244 (2007)._ _ _ _ Dalby, M. J., Gadegaard, N. & Wilkinson, C. D. W. The response of fibroblasts to hexagonal nanotopography fabricated by electron beam lithography. J. Biomed. Mater. Res. A 84, 973u2013979 (2008)._ _ _ _ Dalby, M. J., Riehle, M. O., Yarwood, S. J., Wilkinson, C. D. W. & Curtis, A. S. G. Nucleus alignment and cell signaling in fibroblasts: response to a micro-grooved topography. Exp. Cell Res. 284, 272u2013280 (2003)._ _ _ Allan, C. et al. Osteoblast response to disordered nanotopography. J. Tissue Eng. 9, 2041731418784098 (2018)._ _ _ _ _ Quarles, L. D., Yohay, D. A., Lever, L. W., Caton, R. & Wenstrup, R. J. Distinct proliferative and differentiated stages of murine MC3T3-E1 cells in culture: an in vitro model of osteoblast development. J. Bone Miner. Res. 7, 683u2013692 (1992)._ _ _ _ Wagers, A. J. & Conboy, I. M. Cellular and molecular signatures of muscle regeneration: current concepts and controversies in adult myogenesis. Cell 122, 659u2013 667 (2005)._ _ _ _ Kumar, R. M. & Sreekumar, K. A survey on image feature descriptors. Int. J. Comput. Sci. Inf. Technol. 5, 7668u20137673 (2014)._ Van De Water, L., Varney, S. & Tomasek, J. J. Mechanoregulation of the myofibroblast in wound contraction, scarring, and fibrosis: opportunities for new therapeutic intervention. Adv. Wound Care 2, 122u2013141 (2013)._ _ Bollong, M. J. et al. Small molecule-mediated inhibition of myofibroblast transdifferentiation for the treatment of fibrosis. Proc. Natl Acad. Sci. USA 114, 4679u20134684 (2017)._ _ _ _ _ Quach, N. L., Biressi, S., Reichardt, L. F., Keller, C. & Rando, T. A. Focal adhesion kinase signaling regulates the expression of caveolin 3 and u03B21 integrin, genes essential for normal myoblast fusion. Mol. Biol. Cell 20, 3422u20133435, (2009)._ _ _ _ _ Clemente, C. F. M. Z., Corat, M. A. F., Saad, S. T. O. & Franchini, K. G. Differentiation of C2C12 myoblasts is critically regulated by FAK signaling. Am. J. Physiol. Regul. Integr. Comp. Physiol. 289, R862u2013R870 (2005)._ _ _ _ Bachmann, P. & Hinrichsen, K. Semiautomated evaluation of morphometric data of myoblasts and myotubes and the calculation of the relative DNA content with the television texture analysis system (TAS). Histochemistry 69, 233u2013242 (1980)._ _ _ _ Lassalle, B., Gautron, J., Martelly, I. & Le Moigne, A. Image analysis of rat satellite cell proliferation in vitro. Cytotechnology 2, 213u2013224 (1989)._ _ _ _ Treiser, M. D. et al. Cytoskeleton-based forecasting of stem cell lineage fates. Proc. Natl Acad. Sci. USA 107, 610u2013615 (2010)._ _ _ _ _ Kilian, K. A., Bugarija, B., Lahn, B. T. & Mrksich, M. Geometric cues for directing the differentiation of mesenchymal stem cells. Proc. Natl Acad. Sci. USA 107, 4872u20134877 (2010)._ _ _ _ _ _ Swift, J. et al. Nuclear lamin-A scales with tissue stiffness and enhances matrix-directed differentiation. Science 341, 1240104 (2013)._ _ _ _ _ Shin, H. et al. Focal adhesion assembly induces phenotypic changes and dedifferentiation in chondrocytes. J. Cell. Physiol. 231, 1822u20131831 (2016)._ _ _ _ Connelly, J. T., GarcxEDa, A. J. & Levenston, M. E. Interactions between integrin ligand density and cytoskeletal integrity regulate BMSC chondrogenesis. J. Cell. Physiol. 217, 145u2013154 (2008)._ _ _ _ Vasilopoulos, Y., Gkretsi, V., Armaka, M., Aidinis, V. & Kollias, G. Actin cytoskeleton dynamics linked to synovial fibroblast activation as a novel pathogenic principle in TNF-driven arthritis. Ann. Rheum. Dis. 66(Suppl 3), iii23u2013iii28 (2007)._ _ _ _ Dalby, M. J. Cellular response to low adhesion nanotopographies. Int. J. Nanomed. 2, 373u2013381 (2007)._ _ Wong, V. W. et al. Focal adhesion kinase links mechanical force to skin fibrosis via inflammatory signaling. Nat. Med. 18, 148u2013152 (2011)._ _ _ _ _ Castillo, A. B. et al. Focal adhesion kinase plays a role in osteoblast mechanotransduction in vitro but does not affect load-induced bone formation in vivo. PLoS ONE 7, e43291 (2012)._ _ _ _ _ _ Salasznyk, R. M., Klees, R. F., Williams, W. A., Boskey, A. & Plopper, G. E. Focal adhesion kinase signaling pathways regulate the osteogenic differentiation of human mesenchymal stem cells. Exp. Cell Res. 313, 22u201337 (2007)._ _ _ _ Shamir, L. Assessing the efficacy of low-level image content descriptors for computer-based fluorescence microscopy image analysis. J. Microsc. 243, 284u2013292 (2011)._ _ _ _ Wang, I. N. E. et al. Role of osteoblast-fibroblast interactions in the formation of the ligament-to-bone interface. J. Orthop. Res. 25, 1609u20131620 (2007)._ _ _ _ Vasilevich, A. S. et al. Designed surface topographies control ICAM-1 expression in tonsil-derived human stromal. Cells Front. Bioeng. Biotech. 6, 87 (2018)._ _ Guillaume-Gentil, O. et al. Tunable single-cell extraction for molecular analyses. Cell 166, 506u2013 516 (2016)._ _ _ _ Chaloner, K. & Verdinelli, I. Bayesian experimental design: a review. Stat. Sci. 10, 273u2013304 (1995)._ _ _ _ Huntington, M. D. & Odom, T. W. A portable, benchtop photolithography system based on a solid-state light source. Small 7, 3144u20133147 (2011)._ _ _ _ Heydari, T. et al. Development of a virtual cell model to predict cell response to substrate topography. ACS Nano 11, 9084u20139092 (2017)._ _ _ _ Stormonth-Darling, J. M., Saeed, A., Reynolds, P. M. & Gadegaard, N. Injection molding micro- and nanostructures in thermoplastic elastomers. Macromol. Mater. Eng. 301, 964u2013971 (2016)._ _ _ _ _ Yan, X. Z. et al. Effects of continuous passaging on mineralization of MC3T3-E1 cells with improved osteogenic culture protocol. Tissue Eng. Part C. Methods 20, 198u2013204 (2014)._ _ _ _ Reynolds, P. M. et al. Controlling fluid flow to improve cell seeding uniformity. PLoS ONE 13, e0207211 (2018)._ _ _ _ _ Gray, M. et al. NuMA promotes homologous recombination repair by regulating the accumulation of the ISWI ATPase SNF2h at DNA breaks. Nucleic Acids Res. 42, 6365u20136379 (2014)._ _ _ _ _ Kamentsky, L. et al. Improved structure, function and compatibility for CellProfiler: modular high-throughput image analysis software. Bioinformatics 27, 1179u20131180 (2011)._ _ _ _ _ Caicedo, J. C. et al. Data-analysis strategies for image-based cell profiling. Nat. Methods 14, 849u2013863 (2017)._ _ _ _ _ Zhan, X., Patterson, A. D. & Ghosh, D. Kernel approaches for differential expression analysis of mass spectrometry-based metabolomics data. BMC Bioinforma. 16, 77 (2015)._ _ _ Warnes, G. R. et al. gplots: Various R Programming Tools for Plotting Data. R package version 3.0.1. _(2016).Maechler, M., Rousseeuw, P., Struyf, A., Hubert, M. & Hornik, K. cluster: Cluster Analysis Basics and Extensions. R package version 2.0.7-1. (2018).Wei, T. and Simko, V. u201Ccorrplotu201D: Visualization of a Correlation Matrix. R package version 0.84. (2017).Kuhn, M. Building predictive Models in R Using the caret Package. J. Stat. Softw. 28, 1u201326 (2008)._ _ BxFCrkner, P. C. brms: an R package for Bayesian multilevel models using Stan. J. Stat. Softw. 80, 1u201328 (2017)._ _ Wickham, H. ggplot2: Elegant Graphics for Data Analysis. (Springer-Verlag, New York, 2009)._ _ _ Akima, H. & Gebhardt, A. akima: Interpolation of Irregularly and Regularly Spaced Data. R package version 0.6-2. (2016).\"- - \"We acknowledge ERC funding through FAKIR 648892 Consolidator Award. MFAC is financially supported by the University of Glasgow MG Dunlop Bequest, College of Science and Engineering Scholarship, and FAKIR consolidator award. NG acknowledges support by the Research Council of Norway through its Centres of Excellence funding scheme, project number 262613._We acknowledge the James Watt Nanofabrication Center for fabrication work, and Steen Lillelund for initiating the machine learning work. We thank Julie Russell for her contribution to the QPCR, Carmen Huesa for providing the primary cartilage cells and Rachel Love for the injection molding of nanotopographies.\"- - Authors and Affiliations - \"Divison of Biomedical Engineering, School of Engineering, University of Glasgow, Glasgow, UKMarie F. A. Cutiongco,_Paul M. Reynolds_&_Nikolaj GadegaardSchool of Computing Science, University of Glasgow, Glasgow, UKBjxF8rn Sand Jensen\" - \"AuthorsMarie F. A. CutiongcoYou can also search for this author in _BjxF8rn Sand JensenYou can also search for this author in _Paul M. ReynoldsYou can also search for this author in _Nikolaj GadegaardYou can also search for this author in _\" - Contributions - M.F.A.C., P.M.R., and N.G. designed the biological experiments. M.F.A.C. and B.S.J. designed the machine learning analysis. M.F.A.C. carried out imaging, image analysis, QPCR, machine learning. P.M.R. fabricated and characterized the nanotopographical surfaces. M.F.A.C., B.S.J., P.M.R., and N.G. wrote the manuscript. All authors have read and approved the manuscript before submission. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License (). j Representative flow cytometric dot plots demonstrating DilC-stained macrophages and PKH67-stained Gluc-CC or Gal-CC. DilCPKH67 double-positive cells represent phagocytosis. k Mean fold changeu2009xB1u2009sd of the DilCPKH67 double-positive population in the phagocytosis assay using monocyte-derived macrophages from 3 different healthy donors (nu2009=u20094 biological replicates for each condition; two-way ANOVA statistical analysis). ***pu2009<u20090.0001; *pu2009=u20090.047; nsu2009=u20090.651; **pu2009=u20090.0016; *pu2009 =u20090.019; *pu2009=u20090.018; nsu2009=u20090.652; **u2009=u20090.0095; nsu2009=u20090.988. lu2013n Mean fold changeu2009xB1u2009sd in cell death after 10 days co-cultivating PDAC cells with activated T cells isolated from healthy l Donor 1, m Donor 2, or n Donor 3 (nu2009=u20094 biological replicates per condition; Holmu2013Sidak t test statistical analysis), (red lineu2009 =u2009Control, basal toxicity set as 1.0). For l: ***pu2009=u20090.000016; ***pu2009=u20090.00075; **pu2009=u20090.0013. For m: ***pu2009=u20090.000002; ***pu2009=u20090.00025; ***pu2009<u20090.000001. For n: ***pu2009<u2009 0.000001; ***pu2009=u20090.00018; **pu2009=u20090.0023.\" - \"At the transcriptional level, we evaluated different epithelialu2013mesenchymal transition (EMT)-related genes. ZEB1, a key regulator of invasion and metastasis, as well as the EMT transcription factors (TF) SNAI2 and SNAI1 were significantly higher in Gal-CC compared to Gluc-CC, whereas CDH1 and VIM were unchanged (Fig._ ). We next evaluated transforming growth factor-u03B2 (TGFu03B2) secretion levels, one of the most important cytokines implicated in stemness, EMT, and extracellular matrix regulation, and measured significantly higher levels in Gal-CC compared to Gluc-CC (Fig._). We also found a significant overexpression of nuclear factor-u03BA B in Gal-CC, an important TF that mediates cytokine secretion (Fig._). Based on this result, we performed a cytokine array. Overall, we found differences in the levels of expression of different secreted cytokines or inflammatory-associated molecules in both culture conditions (Fig._ and Supplementary Fig._). Significant differences were found in C-C motif chemokine ligand 2 (CCL2), C-X-C motif chemokine ligand 12 (CXCL12), granulocyte macrophages colony-stimulating factor (GM-CSF), and soluble intercellular adhesion molecule (ICAM)-1 (Fig._), as well as CCL5, CXCL1, Serpin E1, and IL-8 (Fig._). Interestingly, the factors that significantly increased in Gal-CC have been associated with higher stemness (CCL2, IL-8); tumorigenesis (CCL2, CXCL12, IL-8); chemoresistance (CXCL12); metastasis, migration, and invasion (CXCL12, IL-8); and quiescence (CXCL12). On the other hand, the decrease of the other factors has been correlated with lower chemotaxis, communication and activation of immune cells like T cells, basophils, neutrophils (ICAM-1, CCL5, CXCL1), higher quiescence and infiltration of pro-inflammatory macrophages (GM-CSF), higher metastasis, and stemness (Serpin E1).\" - \"To test whether Gal-CC are more immunoevasive, we performed functional in vitro immune evasion assays. Co-culturing DilC-stained (red) macrophages from 3 healthy donors with PKH67-stained (green) tumor cells (Glu- and Gal-CC) (Fig._) resulted in less double-positive DilCPKH67 cells (indicative of macrophage-mediated phagocytosis) (Fig._) for Gal-CC compared to Gluc-CC (Fig._), indicating that macrophages are less able to detect and phagocytose Gal-CC than Gluc-CC, likely due to the overexpression of the u201Cdont eat meu201D CD47 receptor on the surface of Gal-CC (Fig._). Next, activated T cells were co-cultured with tumor cells (Glu- and Gal-CC). Regardless of the donor and PDAC cell line used, T cells were less efficient at eliminating Gal-CC compared to Gluc-CC when they were co-cultivated (Fig._). Importantly, the levels of cell death detected in T cell/Gal-CC co-cultures was very similar to control cultures (without T cells) for Donor 1 (Fig._) and Donor 3 (Fig._). T cells from Donor 2, in general, displayed higher cytolytic capacity (Fig._).\" - Upregulated in vivo immune evasion and metastasis in Gal-CC - \"To functionally test the invasive capacity of Gal-CC and Gluc-CC, zebrafish and NOD-SCID mouse models of cellular invasion were employed. Zebrafish embryos were microinjected into circulation (Duct of Cuvier), as previously described. While at 1 day post-injection (dpi) PANCA6L mCherry-H2B Gluc-CC migrated more efficiently to the tail compared to Gal-CC (Fig._), Gluc-CC were less viable at later time points, as evident by the loss in fluorescence observed at 4 and 6 dpi (Fig._). For PANC185scd, similar results were observed (Supplementary Fig._). To extend these observations, NOD-SCID mice were tail vein injected with Gluc-CC or Gal-CC and their capacity to disseminate and invade to distant organs was assessed at 10 dpi (short time) and 3 months pi (long time) (Fig._ ).\" - \"Fig. 7: In vivo immune evasion and metastasis are upregulated in Gal-CC.a Representative images of zebrafish embryo tails at 1, 4, and 6 days post-injection (dpi). Scale baru2009=u2009250u2009xB5M. b Meanu2009xB1u2009sem of the integrated density (fluorescence measured) 1 dpi (biological replicates: nu2009=u2009 29 for Glucose and nu2009=u200936 for Galactose; Students t test) **pu2009 =u20090.0025. c, d Meanu2009xB1u2009sem of proliferation ratio observed between Gluc-CC and Gal-CC at c 4 dpi and d 6 dpi. Proliferation ratios are represented in comparison to 1 dpi (red line) (biological replicates: 4 dpi: nu2009=u2009 26 for Glucose and nu2009=u200914 for Galactose; Students t test; **pu2009 =u20090.0021; 6 dpi: nu2009=u200918 for Glucose and nu2009=u200911 for Galactose; Students t test; **pu2009=u20090.0019). e Representative in vivo invasion assay scheme. Schemes were constructed using unmodified images from Servier Medical Art by Servier, licensed under a Creative Commons Attribution 3.0 Unported License (). f, g Mean fold changeu2009xB1u2009sem of mCherry cells present in different organs analyzed at f short time (10 days post-injection), (nu2009 =u20092 independent experiments with a total of nu2009=u200917 mice; Holmu2013 Sidak t test statistical analysis) or at g long time (3 months post-injection). Number (n) of samples analyzed are indicated for each organ. (nu2009=u2009 1 independent experiment with a total of nu2009=u20096 mice; Holmu2013Sidak t test statistical analysis). For f: nsu2009=u20090.25; nsu2009=u20090.26; **pu2009=u20090.0058; nsu2009=u20090.27; **pu2009=u20090.0016. For g: *pu2009=u20090.017; *pu2009=u20090.034; *pu2009=u20090.025; *pu2009=u2009 0.027. h Representative fluorescent IVIS images of different organs from mice at 3 months post-injection. i Meanu2009xB1u2009sem of total radiant efficiency [p/s]/[xB5W/cm] of different organs from h. Number (n) of samples analyzed are indicated for each organ. (nu2009=u20091 independent experiment with a total of nu2009=u20096 mice; Holmu2013Sidak t test statistical analysis). All data are not statistically significant. j, k Mean fold changeu2009xB1u2009 sem of the CD45CD11bF4-80 population present in the different organs analyzed at j short time (nu2009=u20092 independent experiments with a total of nu2009 =u200917 mice; Holmu2013Sidak t test statistical analysis) or at k long time (nu2009=u20091 independent experiment with a total of nu2009=u20096 mice; Holmu2013Sidak t test statistical analysis). Red lines indicate baseline levels of the CD45CD11bF4-80 population of control mice (nu2009=u20096), set as 1.0. Number (n) of samples analyzed for Gluc-CC- and Gal-CC-injected mice are indicated for each organ. For j: nsu2009=u20090.609; ***pu2009=u20090.000003; ***pu2009 =u20090.00057; nsu2009=u20090.738. For k: *pu2009=u20090.0361; *pu2009=u2009 0.033; *pu2009=u20090.0362; nsu2009=u20090.982.\" - \"At 10 dpi, while there was no significant increase in mCherry-positive cells in the spleen, liver, or lungs, there was a significant increase in the pancreas and blood in mice injected with Gal-CC (Fig._ and Supplementary Fig._). However, at 3 months pi, we found that mice injected with Gal-CC presented with significantly higher percentages of mCherry-positive cells in most organs compared to mice injected with Gluc-CC, including the liver, one of the main metastatic sites in PDAC patients (Fig._ and Supplementary Fig._). In some organs, such as kidneys, thymus, and lymphatic nodes, no significant differences were observed (Supplementary Fig._). Results were confirmed and validated ex vivo using IVIS imaging (Fig._) or by immunohistochemical (IHC) analysis_of human cytokeratin-19 (Supplementary Fig._). Interestingly, we found numerous and abnormally large lymphatic nodes in mice injected with Gal-CC, which were not detected in the mice injected with Gluc-CC (Fig._, yellow squares). Lastly, we analyzed in each organ the percentage of tumor-associated macrophages (TAMs) (CD45CD11bF4-80), which are known to play an important role both in the primary tumor and for pre-metastatic niche formation. In mice injected with Gal-CC, we observed at 10 dpi significantly lower TAMs in the liver and pancreas (Fig._); however, at 3 months pi the population of TAMs increased significantly in all the organs analyzed except the lungs and kidneys (Fig._ and Supplementary Fig._).\"- - We have developed a 2D in vitro system for long-term and sustained enrichment of PaCSCs based on forced OXPHOS. By culturing cells with galactose in lieu of glucose, cells that are exclusively dependent on glycolysis (non-PaCSCs) cannot survive due to differences in ATP yield and availability of glycolysis intermediates. Indeed, we observed cell death in the presence of galactose that coincided with an increase in the PaCSC population, confirming that non-PaCSCs cannot survive in galactose. To confirm the latter, we placed purified non-CSCs in galactose and observed that, while galactose favors the conversion of non-PaCSC in PaCSCs (i.e., plasticity), this contribution is secondary compared to the enrichment achieved due to loss of the non-PaCSC compartment. While we have not defined the mechanism of non-PaCSC plasticity, it is plausible that non-PaCSCs could increase their mitochondrial mass to survive in metabolically restrictive conditions. - \"In our OXPHOS-promoting system, we observed a profound increase in traditional CSC features, such as the expression of multiple CSC biomarkers, overexpression of pluripotency-associated genes and stem-related pathways (validated by RNAseq), increased NANOG promoter activity, enhanced self-renewal capacity, and, most notably, significantly higher in vivo tumorigenicity. Other studies using alternate carbon sources (i.e., fructose) in breast cancer cells observed similar results but their studies were not as exhaustive and did not evaluate all of the phenotypes presented herein. Not surprisingly, we did not obtain 100% CSC biomarker staining in Gal-CC, which we reconcile by acknowledging that heterogeneity exists within the CSC population(s), making it impossible to detect all CSC clones as not all CSCs (or OXPHOS-dependent hybrid CSCs) will express the same single biomarker or combinations of different biomarkers. Nonetheless, the increase observed in defined CSC markers (>15%) and others (mitochondrial markers of approximately 100% and immune evasion markers of >50%) clearly indicate an enrichment comparable to that obtained with traditional approaches. Apart from traditional CSC features, we also observed that Gal-CC were more slow cycling than Gluc-CC, linking OXPHOS to a quiescent-like state, a feature typically associated with a CSCs survival and metastatic capacity. Importantly, the latter was not permanent and/or irreversible, as Gal-CC are highly plastic and can exit this slow-cycling state in accordance to the carbon source available. Whether OXPHOS metabolism is favored by quiescent CSCs as a means of maintaining mitochondrial membrane potential, ATP production, and mtDNA content is still to be determined.\" - \"Many studies have shown that slow-cycling/dormant tumor cells are also more chemoresistant, have epithelialu2013mesenchymal plasticity (EMP), and are better able to evade the immune system, facilitating post-treatment relapse, tumor cell dissemination, and survival in hostile environments. Along these lines, Gal-CC were more chemoresistant to a panel of traditional chemotherapeutics, largely due to the overexpression of the ABCG2 transporter and the different levels of expression of genes related to GEM transport. In contrast, and as expected, Gal-CC were more sensitive to OXPHOS metabolic drugs and less sensitive to the glycolytic irreversible inhibitor 2-DG. Moreover, we also observed increased autophagy in Gal-CC, suggesting that autophagy is also likely an important mechanism used by OXPHOS-dependent cells in metabolically restrictive conditions, such as those faced by CSCs in the tumor microenvironment (TME) where oxygen and nutrients are reduced. It has also been observed that autophagy is strongly linked to quiescent/dormant processes in cancer cells as well as EMT and EMP. For example, two recent reports demonstrate that autophagy is intimately linked to the capacity of cancer cells to transition between epithelial and mesenchymal states. EMP is now considered a critical process for tumor cell dissemination, metastatic colonization, and chemoresistance. Not only were Gal-CC more chemoresistant but also different EMT TFs were significantly overexpressed, including ZEB1, which was recently linked to PDAC metabolic plasticity in vivo. In fact, biological associations between EMT and metabolic reprogramming in cancer have been described and shown to be a consequence of EMT TF signaling. Thus the overexpression of ZEB1 in cultures enriched in OXPHOS-dependent CSCs is not surprising but does validate the system and its biological relevance.\" - \"Finally, we show that OXPHOS-driven PaCSCs exhibit immune evasive phenotypic and functional properties. Indeed, studies have speculated that CSCs may maintain only limited antigen-presentation levels and hence are invisible to the immune system, but it is also possible that CSCs possess antigen editing-independent capacities that may endow them with immunoevasive properties. In this sense, Gal-CC secreted more TGFu03B2 and acquired a previously uncharacterized immune evasive molecular signature, enabling them to evade immune innate cells (i.e., macrophages and NK cells) and immune adaptive cells (T cells) thanks to the modulation of different immune-associated receptors, ligands, and secreted cytokines. As a consequence, Gal-CC were more invasive and metastatic in vivo and were able to modulate the immune infiltrating cell profile at early and late times post-injection, attracting fewer immune cells during the early stages of extravasation, while at later times promoting the accumulation of pro-tumor immune cells, specifically TAMs, which facilitate establishment of the niche and TME.\" - \"It is not surprising that mitochondria are important mediators of tumorigenesis. While we have previously shown that PaCSCs use OXPHOS to meet their energy requirements, herein we expand upon this observation by showing that Gal-CC have a denser and more extensive mitochondria network and have more functional mitochondria compared to glycolytic cells (non-PaCSCs). Indeed, Gal-CC overexpress mitochondrial-associated genes, such as PGC1u03B1, which promotes mitochondrial biogenesis and OXPHOS and these processes are linked to cancer cell migration, EMP, and immune evasion. Likewise, it has been shown in other systems that the aberrant expression of immune evasion receptors (e.g., PD-L1) is metabolically controlled, and in cancer, correlations between metabolic alterations, EMT, and PD-L1 upregulation have been demonstrated. Thus our system has allowed us to rigorously demonstrate that the metabolism of PaCSCs is intimately linked to their stem, slow-cycling, metastatic, and immunoevasive capacities.\"- - Primary human PDAC cells - \"PDAC PDXs were obtained from Dr. Manuel Hidalgo under a Material Transfer Agreement with the Spanish National Cancer Centre (CNIO), Madrid, Spain (Reference no. I409181220BSMH) and were originally described and genetically characterized in ref. . For primary cultures, PDX-derived tumor tissue fragments were minced, enzymatically digested with collagenase (Cat no. 07416, Stem Cell Technologies) for 60u2009min at 37u2009xB0C, and after centrifugation for 5u2009min at 500u2009xD7u2009g the pellets were resuspended and cultured in RPMI, 10% fetal bovine serum (FBS), and 50 units/ml penicillin/streptomycin as described previously. PDAC PDX-derived cultures are referred to by a random number designation [i.e., PANC185, PANC185scd (single-cell-derived), PANCA6L, and PANC286]. Primary cultures were tested for Mycoplasma at least every 4 weeks.\" - Establishment of OXPHOS-competent and -independent cultures - \"Low passage (<15) PDX-derived primary PDAC cultures were trypsinized and seeded at a concentration of 800,000 cells in p100 plates with RPMI medium supplemented with 10% FBS and 50u2009units/ml of penicillin and streptomycin. After 24 h, cells were cultured with either glucose-free Dulbeccos Modified Eagle Medium (DMEM; Thermo Fisher Scientific) supplemented with 5u2009mM glucose (0.9u2009g/l), 10% FBS, 50u2009units/ml of penicillin and streptomycin, and 1u2009mM of pyruvate [Glucose: OXPHOS-independent conditions] or glucose-free DMEM medium (Thermo Fisher Scientific) supplemented with 5u2009mM galactose (0.9u2009g/L), 10% FBS, 50u2009units/ml of penicillin and streptomycin, and 1u2009mM of pyruvate [Galactose: OXPHOS-competent enriched conditions]. Sugar concentrations of 5u2009mM were chosen to mimic physiological sugar levels (glucose, 5u2009mM) and to avoid potential biological artifacts mediated by supraphysiological sugar levels. Media for both conditions were changed every day, over a period of 14 days, after which cells were collected for further processing and analysis, as described below. Earlier time points (e.g., 7 days) were also tested, but maximum levels of markers and gene expression were obtained at 14 days, and thus this time point was chosen for all experiments described herein. Gal-CC cultures can be maintained in culture indefinitely (>70 days tested), trypsinized, and passaged if necessary, as well as cryopreserved for future thawing and usage.\" - Flow cytometry and sorting - \"Cells were resuspended in Flow buffer [1xD7 phosphate-buffered saline (PBS); 3% FBS (v/v); 3u2009mM EDTA (v/v)] before analysis with a 4-laser Attune NxT Acoustic Cytometer (Thermo Fisher Scientific). Cytometry data were acquired with the Invitrogenu2122 Attuneu2122 NxT software, version 3.1.1, unless specified otherwise. For cell surface marker expression, refer to antibodies listed in Supplementary Table_. For detection of PARKIN by cytometry, cells were fixed with 4% paraformaldehyde (PFA) in PBS for 10u2009min, washed, and permeabilized for 15u2009min with 1% Triton X-100. For Annexin-V staining, floating and attached cells were pooled and resuspended in 1xD7 Annexin-V staining buffer containing Annexin-V-fluorescein isothiocyanate (FITC) diluted 1:20 (Cat no. 29001, Biotium, Freemont, CA) and incubated for 20u2009min at room temperature (RT) prior to flow cytometric analysis. For autofluorescent detection, cells were excited with blue laser 488u2009nm and selected as intersection with the filters 530/40 (FITC) and 580/30 (phycoerythrin (PE)). For all assays, 2u2009mg/ml 4,6-diamidino-2-phenylindole (DAPI; Cat no. D9564, Sigma) was used to exclude dead cells with laser VL1. Data were analyzed with the FlowJo 9.3 software (Tree Star Inc., Ashland, OR.). For cell sorting, cells were resuspended in sorting buffer [1xD7 PBS; 0.1% FBS (v/v); 3u2009mM EDTA (v/v)] and a_FACS Vantage SE Flow Cytometer was used and events were_analyzed by the BD FACSDiva software v.9.0 (BD Biosciences, San Jose, CA). Examples of gating strategies for all of the aforementioned cytometry-based analyses are presented in Supplementary Fig._.\" - \"For mitochondria markers, Gluc-CC and Gal-CC were stained with Mitotracker Deep Red (Cat no. M22426), CM-H2XRos (Cat no. M7513), MitoTrackerxAE Red CMXRos (Cat no. M7512), Mitotracker Green FM (Cat no. M7514), and NAO (Cat no. A1372) (all from Invitrogen) in the absence of FBS for 30u2009min at 37u2009xB0 C, at concentrations of 2, 100, and 10u2009nM, respectively, and 0.1u2009xB5 M for Mitotracker Green FM and NAO. Fluorescence was detected using the filters RL1 (allophycocyanin), YL1 (PE), and BL1 (FITC). MitoSOX (Cat no. M36008, Invitrogen) was used at 5u2009xB5M for 30u2009min at 37u2009xB0C and detected with laser YL1 (PE). MitoBlue (kindly provided by Dr. JL MascarexF1as, Universidad de Santiago de Compostela) was used at 10u2009xB5M for 20u2009min at 37u2009 xB0C and detected with laser VL2. Data were analyzed with the FlowJo 9.3 software (Tree Star Inc., Ashland, OR).\" - Cell cycle analysis - \"One million cells were plated in 100u2009mm tissue-culture dishes, and after 24u2009h, their cell cycles were synchronized using serum-free medium. Twenty-four hours later, treatments with glucose and galactose media were initiated and cell cycle was determined 6 days later. Cells were trypsinized and fixed in cold 70% ethanol and stored at u221220u2009xB0C. After 24u2009h, cells were incubated with 100u2009xB5g/ml RNAase A (Sigma) in PBS during 30u2009min at 37u2009xB0C and then stained with 50u2009xB5g/ml of PI solution (Cat no. P4864, Sigma) and stored overnight at 4u2009xB0C. Approximately 20,000 cells/sample were analyzed using an Attune NxT Acoustic Cytometer (Thermo Fisher Scientific) with excitation at 561u2009nm and emission at 615u2009nm. The percentage of cells in each phase of the cell cycle was determined using the FlowJo 9.3 software (Tree Star Inc., Ashland, OR).\" - WB analysis - \"Cells were harvested in RIPA buffer (Cat no. R0278, Sigma) supplemented with protease inhibitor cocktail (Roche Applied Science, Indianapolis, IN). Fifty micrograms of protein were resolved by sodium dodecyl sulfate-polyacrylamide gel electrophoresis and transferred to polyvinylidene difluoride (PVDF) membranes (Amersham Pharmacia, Piscataway, NJ). Membranes were sequentially blocked with 1xD7 TBS containing 5% bovine serum albumin (BSA; w/v) and 0.1% Tween20 (v/v), incubated with a 1:1000 dilution of the indicated antibodies (Supplementary Table_) overnight at 4u2009xB0C, washed 3 times with 1xD7 PBS containing 0.05% Tween20 (v/v), incubated with horseradish peroxidase-conjugated donkey anti-rabbit or sheep anti-mouse antibody (Amersham), and washed again to remove unbound antibody. Bound antibody complexes were detected with SuperSignal chemiluminescent substrate (Amersham) with a_MyECL Imager (Thermo Fisher Scientific). Uncropped and unprocessed scans of all the main blots presented can be found in the Source data file.\" - Sphere-formation assay - \"PDAC spheres were generated and expanded in DMEM-F12 (Invitrogen) supplemented with B-27 (Cat no. A3582801, GIBCO) and basic fibroblast growth factor (PeproTech EC). One thousand cells/ml/well were seeded in ultra-low attachment 24-well plates (Corning) as described previously. For serial passaging, spheres were harvested at day 7 using a 40-xB5m cell strainer, dissociated to single cells with trypsin, and then re-grown in the same conditions for 7 days (14 days total). Numbers of spheres were determined by microscopy using an inverted EVOS FL microscope (Thermo Fisher Scientific) using a xD710 objective with phase contrast.\" - RNA extraction and reverse transcriptase quantitative PCR (RT-qPCR) - \"RNA was isolated by using the standard protocol of the guanidine thiocyanate (GTC) method. One microgram of purified RNA was used for the synthesis of cDNA using the Thermo Scientific Maxima First Strand cDNA Synthesis Kit (Cat no. K1672, Thermo Fisher Scientific) according to the manufacturers instructions, followed by RT-qPCR analysis using SYBR green and a StepOne Plus real-time thermo-cycler (Applied Biosystems). The thermal cycle consisted of an initial denaturation step of 10u2009min at 95u2009xB0C followed by 40 cycles of denaturation (15u2009 s at 95u2009xB0C) and annealing (1u2009min at 60u2009xB0C). The results obtained for each gene were normalized with the u03B2-actin levels. For primers used, see Supplementary Table_.\" - Lentivirus production and cell transduction - \"Lentiviral particles were produced by co-transfection of 293T cells (kindly provided by Dr. Amparo Cano, Universidad AutxF3noma de Madrid, Spain) with the indicated plasmids using the polyethylenimine-based transfection method. Briefly, 5u2009 xD7u200910 293T cells were co-transfected with 1u2009u03BCg packaging plasmid psPAX2, 1u2009u03BCg envelope plasmid VSVG, and 2u2009u03BCg of H2B-mCherry or the NANOG reporter lentiviral vector backbone (the sequence of the construct has been previously described in the publication by Hotta et al.). After 8u2009 h, the transfection medium was changed and recombinant lentiviruses were harvested 48 and 72u2009h later. The virus particle-containing supernatant was filtered through 0.45-xB5m PVDF membrane filters and aliquoted and stored at u2212 80u2009xB0C until needed. For lentivirus transduction, PDAC cells were seeded in 6-well plates at a concentration of 3u20135u2009xD7u200910 cells/well. One milliliter of virus was directly overlaid on cells and polybrene (Cat no. 638133, Sigma) was added at a final concentration of 8u2009u03BCg/ml. After 16u2009h, medium was changed. Stably transduced cells were obtained after mCherry-positive or YFP-positive cell sorting using a FACS Vantage SE Flow Cytometer and analyzed by the BD FACSDiva software (BD Biosciences).\" - Electron microscopy - \"After sorting cells for the indicated markers, cells were centrifuged and pellets were fixed with 0.1u2009M cacodylate buffer with a pH of 7.4 at RT and sections were processed by the UAM Electron Microscopy unit per standard protocols. Pictures were taken with a JEM-1010 transmission electron microscope (JEOL, USA) and analyzed by Adobe PhotoShop CS4 EXTENDED V11.0 (Adobe Systems, USA).\" - Optical microscopy - All the samples processed for optical microscopy were visualized and photographed with an Axiovert 135 TV microscope (ZEISS, Germany) equipped with an Olympus DP50 digital camera and a fluorescence lamp. The images were obtained with the program ViewFinder 7.1 (Better Light, USA) and processed with the program Adobe PhotoShop CS4 EXTENDED V11.0 (Adobe systems, USA). - Real-time proliferation assay - \"Fifty thousand PDAC cells were seeded in a 96-well plate. After 24u2009h, cells were cultured with glucose or galactose media for 14 days. Real-time analysis of proliferation and autofluorescence was performed by IncuCytexAE Zoom System (ESSEN BioScience, USA) taking images every 30u2009min for 40u2009h. The results were analyzed with the IncuCyte Zoom 2015A software (ESSEN BioScience, USA).\" - Fluorescence microscopy - \"Non-sorted and sorted cells cultured with glucose or galactose were seeded in 24-well culture dishes (Corning) and incubated at 37u2009xB0C. The mitochondrial probes MitoTracker DeepRed and MitoTracker CMXRos were added at 2 and 10u2009 nM, respectively, for 30u2009min at 37u2009xB0C (all from Invitrogen), followed by two washes with 1xD7 PBS, and a_final_5u2009min incubation with DAPI (2u2009mg/ml, Sigma). MitoTracker DeepRed was excited at 644u2009nm and the fluorescence emitted was detected at 665u2009nm (far red fluorescence) and assigned red fluorescence. MitoTracker CMXRos was excited at 579u2009nm and the fluorescence emitted was detected at 599u2009nm and assigned green fluorescence. For IF assays, cells were fixed with 4% PFA in PBS for 20u2009min at RT, washed with PBS, permeabilized with 1% Triton X-100 in PBS for 15u2009min, blocked with 1% BSA in PBS for 1u2009h at RT, and then incubated with specific antibodies (Supplementary Table_) in a solution of 1% BSA in PBS. ProLongxAE Gold Antifade Reagent with DAPI (Cat no. P36941) was then added to mark cell nuclei. The fluorescent images were collected with a laser scanning confocal microscope Zeiss 710 and analyzed using the software Zen2009 5.5 (Oberkochen, Germany).\" - In vivo assays - \"For in vivo tumorigenicity assays, serial dilutions of Gluc-CC and Gal-CC resuspended in Matrigel (Cat no. 356234, Corning) were subcutaneously injected into 8-week-old female nude mice (Hsd:Athymic Nude-Foxn1nu/Foxn1+; Envigo) and tracked for 10u2013 14 weeks for tumor formation. If a mouse in a specific dilution group warranted sacrifice, all of the mice (Gluc-CC and Gal-CC) in that dilution group were sacrificed in order to obtain the number of tumors for all mice at the exact same time. For the dilutions 50,000, 10,000 and 5000, mice were sacrificed at 10 weeks post inoculation. For the 1000-cell dilution group, mice were sacrificed at 14 weeks post inoculation. CSC frequencies were calculated using the ELDA software (). For in vivo tail vein injection invasion assays, 8-week-old NOD-SCID mice (Instituto de Investigaciones BiomxE9dicas u201CAlberto Solsu201D CSIC-UAM) were injected intravenously via the tail vein with either 5u2009xD7u2009 10 mCherry-H2B-labeled PANC185scd or PANCA6L Gluc-CC or Gal-CC (resuspended in 0.9% physiological saline solution) using a 27-G needle. The mice were sacrificed 10 dpi or 3 months pi. Indicated organs were collected, digested, and stained with antibodies to detect TAMs (CD45CD11bF4-80) (Supplementary Table_). In parallel, the percentage of mCherry+ cells present in each organ was determined by FACS using an Attune NxT Acoustic Cytometer (Thermo Fisher Scientific). The percentage of cells was determined using the FlowJo 9.3 software (Tree Star Inc., Ashland, OR). Organs were also analyzed by IVIS-Lumina II (Caliper Life Sciences, Hopkinton, MA, USA) and analyzed by the software Living image 3.2 (Perkin Elmer, Waltham, MA, USA).\" - \"For all in vivo experiments, mice were housed according to institutional guidelines and all experimental procedures were performed in compliance with the institutional guidelines for the welfare of experimental animals approved by the Universidad AutxF3noma de Madrid Ethics Committee (CEI 60-1057-A068 and CEI 103-1958-A337) and La Comunidad de Madrid (PROEX 335/14 and PROEX 294/19) and in accordance with the guidelines for Ethical Conduct in the Care and Use of Animals as stated in The International Guiding Principles for Biomedical Research involving Animals, developed by the Council for International Organizations of Medical Sciences (CIOMS). Briefly, mice were housed according to the following guidelines: a 12-h light/12-h dark cycle, with no access during the dark cycle; temperatures of 65u201375u2009xB0F (~18u201323u2009xB0C) with 40u201360% humidity; a standard diet with fat content ranging from 4u201311%; sterilized water was accessible at all times; for handling, mice were manipulated gently and as little as possible; noises, vibrations, and odors were minimized to prevent stress and decreased breeding performance; and enrichment was always used per the facility s guidelines to help alleviate stress.\" - Immunohistochemistry - \"For histopathological analysis, 3-u03BCm sections of formalin-fixed paraffin-embedded blocks were used for IHC analysis of human cytokeratin 19 using the primary and secondary antibodies and dilutions detailed in Supplementary Table_. Antigens were visualized using 3,3-diaminobenzidine tetrahydrochloride plus (DAB+). Counterstaining was performed with hematoxylin. Digital images were obtained using an Axiophot (Carl Zeiss) microscope with a xD710 objective and DP70 Olympus camera. Images were processed using the ImageJ v2.0.0 software (NIH, USA, ).\" - Autophagy flux assay - \"For autophagy flux analysis, Gluc-CC and Gal-CC were treated with the lysosomotropic reagent Bafilomycin A1 at a concentration of 150u2009nM (Calbiochem, Thermo Fisher Scientific, Waltham, MA, USA) for 5u2009h. After treatment, Gluc-CC and Gal-CC were harvested and analyzed by WB as described above. For densitometry, WB images were analyzed using PhotoShop CS4 EXTENDED V11.0 (Adobe Systems, USA). Autophagy flux compares the LC3B-II (Sigma Aldrich, St. Louis, MO, USA) levels with and without the autophagy inhibitor and is calculated as the ratio of LC3B-II in the presence and absence of the autophagy inhibitor compared to the control condition (Gal-CC versus Gluc-CC). The data were normalized to the indicated housekeeping protein.\" - Quiescence assay - \"Gluc-CC and Gal-CC were labeled with PKH26 red fluorescent cell membrane labeling dye (Cat no. MINI26-1KT, Sigma) according to the manufacturers instructions. At the indicated days post staining, for a total of 14 days, cells were trypsinized, washed, and the percentage of PKH26 cells was determined using an Attune NxT Acoustic Cytometer (Thermo Fisher Scientific). DAPI (Sigma) was used to exclude dead cells with laser VL1.\" - Senescence assay - \"u03B2-Galactosidase activity was measured in Gluc-CC y Gal-CC according to the manufacturers instructions (Cat no. 94433, Sigma). u03B2-Galactosidase cells were determined using an_Axiovert 135 TV optical microscope (ZEISS, Germany), and the ImageJ 2.0.0 software (NIH, USA) was used for further analysis.\" - Chemoresistance in vitro assays - \"PDAC cells were seeded in a 12-well plate at 10 cells/well. Following 14 days of culture with glucose or galactose, Gal-CC or Gluc-CC cells were treated with 1u2009u03BCg/ml of GEM, 3u2009xB5M of MTX, or 1u2009xB5M of DXR for 2 days. After 2 days, drugs were removed and PDAC cells were allowed to recover for 3 days. For metabolic drugs, Gluc-CC and Gal-CC were treated with 5u2009 xB5M of Menadione, 10u2009nM of Rotenone, 100u2009xB5M of Resveratrol, 3u2009 mM of Metformin, or 5u2009mM of 2-DG for 3 days. Following treatments, the supernatant was collected, and the ToxiLightu2122 Bioassay Kit (Cat no. LT07-217, Lonza, Switzerland) was used to evaluate the toxicity using a bioluminescence GLOMAX Luminometer (Promega).\" - Measurement of mitochondrial metabolism by Seahorse analyzer - \"Measurements of OCR were performed with a XF extracellular flux analyzer (Seahorse Bioscience, Agilent Technologies, Santa Clara, CA, USA), and data were collected with the XF96 1.4.2 Software (Agilent). Briefly, 14-day-old Gal-CC or Gluc-CC were trypsinized and re-seeded at a density of 10 cells per well in a XF24 cell culture microplate_with DMEM (4.5u2009g/L glucose, +L-Glutamine, -Pyruvate, pH 7.6 at RT) supplemented with 10% heat-inactivated FBS (Gibco) and 1% Pen/Strep. On the day of the assay (24u2009h after re-seeding), medium was changed to non-carbonated Seahorse measurement medium and the XF24 plate was transferred to a temperature-controlled (37u2009xB0C) XF24 Extracellular Flux analyzer and equilibrated for 10u2009min. To determine the basal respiration, four assay cycles (1-min mix, 2-min wait, and 3-min measuring period) were performed. Then oligomycin (4u2009u03BCM) was added by automatic pneumatic injection (three assay cycles) to inhibit ATP synthase and thus approximate the proportion of respiration used to drive ATP synthesis versus proton leak-linked respiration. Oligomycin was followed by an injection of FCCP (carbonyl cyanide p-trifluoromethoxyphenylhydrazone) (0.5u2009u03BCM) to completely dissipate proton motive force and maximally stimulate mitochondrial respiration (three assay cycles), thus determining SRC and substrate oxidation capacity. An injection of rotenone (4u2009u03BCM) and antimycin A (2u2009u03BCM) was used to correct for the non-mitochondrial respiration rate (three assay cycles), which was subtracted from all the other rates. Basal respiration was calculated as the oligomycin-sensitive fraction of mitochondrial respiratory activity, estimating the proportion of basal respiration used to drive ATP synthesis. To determine ECARs, experiments were terminated by injection of 2-DG (100u2009mM), correcting for non-glycolytic acidification. Injection of 2-DG enables the calculation of glycolytic acidification. Data obtained after oligomycin treatment induced glycolytic capacity. Raw data were normalized to total cell numbers, determined by optical absorbance of lysed crystal violet-stained cells.\" - RNA sequencing analysis - \"Total RNA was isolated by the GTC method using standard protocols. Sequencing libraries were prepared with the u201CNEBNext Ultra II Directional RNA Library Prep Kit for Illuminau201D (NEB no. E7760) as recommended by the kit manufacturer. Briefly, polyA+ fraction was purified and randomly fragmented, converted to double-stranded cDNA, and processed through subsequent enzymatic treatments of end-repair, dA-tailing, and ligation to adapters. Adapter-ligated library was completed by PCR with Illumina PE primers. The resulting purified cDNA libraries were applied to an Illumina flow cell for cluster generation and sequenced on an Illumina NextSeq 550 (with v2.5 reagent kits), according to the manufacturer s protocols. RNAseq data sets were analyzed using the tool Nextpresso. Nextpresso is comprised of four basic levels: (1) quality check, (2) read cleaning and/or down-sampling, (3) alignment, and (4) analysis (gene/isoform expression quantification, differential expression, gene set enrichment analysis, and fusion prediction). The gene signatures (Hallmark genesets, h.all.v7.1.symbold.gmt) from GSEAu2014 Molecular Signature Database for Gene set enrichment analysis was used for pathway enrichment analysis with GSEA v4.0.3 (Broad Institute).\" - Zebrafish maintenance and xenograft assays - \"Zebrafish embryos were obtained by mating adult zebrafish (Danio rerio, wild type), maintained in 30-l tanks with a ratio of 1 fish per liter of water, with 14u2009h/10u2009h light/dark cycle and a temperature of 28.5u2009xB0C according to published procedures. All the procedures used in the experiments as well as fish care were performed in agreement with the Animal Care and Use Committee of the University of Santiago de Compostela and the standard protocols of Spain (Directive 2012-63-UE). At the final point of the experiments, zebrafish embryos were euthanized by tricaine overdose.\" - \"For zebrafish xenograft assays and image analysis, zebrafish embryos were collected at 0u2009h post-fertilization (hpf) and incubated until 48 hpf at 28.5u2009 xB0C. At 48 hpf, hatched embryos were anesthetized with 0.003% of tricaine (Sigma). mCherry-H2B-labeled PANC185scd and PANCA6L Gluc-CC and Gal-CC were trypsinized, resuspended, and concentrated in an Eppendorf at 10 cells per tube for each condition. Cells were then resuspended in 10u2009xB5l of PBS with 2% polyvinylpyrrolidone to avoid cellular aggregation. Borosilicate needles (1u2009mm O.D.u2009xD7 u20090.75u2009mm I.D.; World Precision Instruments) were used to perform the xenograft assays in the zebrafish embryos. Between 100 and 150 cells were injected into the circulation of each fish (Duct of Cuvier) using a microinjector (IM-31 Electric Microinjector, Narishige) with an output pressure of 34u2009kPA and 30u2009ms of injection time per injection. Subsequently, the injected embryos were incubated at a temperature of 34u2009xB0C for 6 dpi in 30u2009ml Petri dishes for each condition with salt dechlorinate tap water. Imaging of the injected embryos was performed using a fluorescence stereomicroscope (AZ-100, Nikon) at 1, 4, and 6 dpi in order to measure the proliferation of the PANC185scd- and PANCA6L-injected human cancer cells inside the zebrafish circulation in each of the conditions assayed.\" - The image analysis of the injected embryos was carried out using the Quantifish software v2.1 (University College London, London, UK) in order to obtain the proliferation ratio of the cells in the region of the caudal hematopoietic tissue of the embryos, where the cells proliferate and metastasize. This program measures in each of the images provided the intensity of the fluorescence and the area of the positive pixel above a certain threshold of the cells. With these parameters, an integrated density value is obtained allowing the researcher to compare different times between images to reach a proliferation ratio. - Lactate production assay - \"Supernatant from Glu-CC and Gal-CC were collected to evaluate the changes in the levels of lactate production. The analysis was performed using the Lactate Assay Kit (Sigma Aldrich, St. Louis, MO, USA) according to the manufacturer s instructions. The optical density was determined using a Synergyu2122 HT Multi-Mode Microplate Reader (BioTek, Winooski, VT, USA) at a wavelength set to 570u2009 nm.\" - ATP determination assay - \"Lysate pellets of cells from Glu-CC and Gal-CC were collected to evaluate the changes in the levels of ATP. The analysis was performed using the Molecular Probes ATP Determination Kit (Thermo Fisher Scientific, Waltham, MA, USA) according to the manufacturers instructions. Bioluminescence was determined using a Synergyu2122 HT Multi-Mode Microplate Reader (BioTek, Winooski, VT, USA).\" - Isolation of macrophages and T cells from human blood - \"Blood samples from healthy donors were provided by the BioBank Hospital RamxF3 n y Cajal-IRYCIS (PT13/0010/0002), integrated in the Spanish National Biobanks Network (ISCIII Biobank Register No._ B.0000678). Samples were processed following standard operating procedures with the appropriate approval of the Hospital RamxF3n y Cajal Ethical and Scientific Committee (Control No.: DE-BIOB-73 AC65, RG.BIOB-57, and RG.BIOB-54), with informed consent and according to Declaration of Helsinki principles. Blood samples were diluted with PBS (Cat no. 10010023, Gibco) and Ficoll (Cat no. L6115, Merck) was used to isolate peripheral blood mononuclear cells (PBMCs). PBMCs were divided across three 6-well culture plates (per donor) with RPMI 1640 media containing 10% FBS and 50u2009units/ml penicillin/streptomycin. After 24u2009h, monocytes adhered to the plate surface were separated from the lymphocytes that remain in suspension and that were seeded in T-75 flasks for their subsequent activation and growth.\" - Cytokine array - \"Changes in protein levels related to immune evasion processes were determined by measuring changes in the levels of cytokines or inflammatory-related molecules released into the culture medium by Gluc-CC and Gal-CC cells, using the Proteome Profiler Human Cytokine Array Kit (R&D Systems, Minneapolis, MN, USA), according to the manufacturers instructions. Cytokine array images were obtained using the MyECL Imager (Thermo Fisher Scientific). For densitometry analysis, array images were analyzed using PhotoShop CS4 EXTENDED V11.0 (Adobe Systems, USA). Data were normalized to the indicated array positive control.\" - \"TFGu03B2 secretion assay\" - \"Culture medium from Gluc-CC and Gal-CC were collected to evaluate the changes in the levels of TGFu03B2 secretion. The analysis was performed using the Human TGFu03B21 Immunoassay Quantikine ELISA Kit (R&D Systems, Minneapolis, MN, USA) according to the manufacturers instructions. The optical density was determined using a Synergyu2122 HT Multi-Mode Microplate Reader (BioTek, Winooski, VT, USA) at a wavelength set to 450u2009nm.\" - Macrophage immunoevasion assay - \"Five days after isolation of macrophages from the blood of healthy donors (see above), macrophages were labeled with the red membrane dye DilC, according to the manufacturers instructions (Cat no. D3911, Thermo Fisher). Gluc-CC and Gal-CC were harvested and labeled with PKH67 green fluorescent cell membrane labeling dye (Sigma) according to the manufacturers instructions. Labeled cells were seeded together with labeled macrophages during 24u2009h. Afterwards, macrophage-mediated_phagocytosis (DilC+/PKH67+ cell percentages) was determined using the AttuneNxT cytometer (Thermo Fisher Scientific), and data were analyzed with the FlowJo 9.3 software (TreeStar, Ashland, OR).\" - T cell immunoevasion assay - \"Seven days after isolation and expansion of T cells from the blood of healthy donors, T cells were activated with phytohemagglutinin-P (Cat no. L8754, Merck) at a concentration of 5u2009u03BCg/ml and with 20u2009ng/ml of human rIL-2 (Cat no. PHC0026, Gibco) for 7 days. Gluc-CC and Gal-CC were harvested and co-seeded with activated T cells during 10 days. Afterwards, T cells were removed and the extent of cytotoxicity was determined with the ToxiLightu2122 Bioassay Kit (Cat no. LT07-217, Lonza, Switzerland) using a bioluminescence GLOMAX Luminometer (Promega). The data were analyzed with the Prism 8 software (San Diego, CA).\" - Statistical analysis and reproducibility - \"Results are presented as meansu2009xB1u2009standard deviation (stdev) or xB1 standard error of the mean (SEM), as indicated in the figure legends. Different two-sided statistical analyses were performed: one-way analysis of variance (ANOVA; two sided) with Bonferroni adjustment or with an unpaired two-tailed t test to determine significance, two-way ANOVA with Tukeys multiple comparisons test, or unpaired two-sided (confidence interval of 95%) t test for multiple comparisons using the Holmu2013Sidak method were used to determine differences between means of groups. p Values < 0.05 were considered statistically significant. GraphPad software (GraphPad Prism 8.0, GraphPad Software, La Jolla, CA, USA) was used to perform the statistical analysis of the data.\" - \"The number of biologically independent samples are indicated in the figure legends. Repeated independent experiments per each panel with similar results are shown below. nu2009=u20091 (Figs._, , , , , , and Supplementary Figs._, , , , , , ); nu2009=u20092 (Figs._, , , , , and Supplementary Figs._, , ); nu2009 =u20093 (Figs._, , and Supplementary Figs._, , ), nu2009=u20094 (Figs._ , and Supplementary Figs._, ).\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - RNAseq data from Gluc-CC and Gal-CC cells, generated in this study, have been deposited in the ArrayExpress database at EMBL-EBI () under accession number . are provided with this paper. - \\' \\'- - \"Rahib, L. et al. Projecting cancer incidence and deaths to 2030: the unexpected burden of thyroid, liver, and pancreas cancers in the United States. Cancer Res. 74, 2913u20132921 (2014)._ _ _ _ Ko, A. H. Progress in the treatment of metastatic pancreatic cancer and the search for next opportunities. J. Clin. Oncol. 33, 1779u20131786 (2015)._ _ _ _ Hermann, P. C. et al. Distinct populations of cancer stem cells determine tumor growth and metastatic activity in human pancreatic cancer. Cell Stem Cell 1, 313u2013323 (2007)._ _ _ _ Malladi, S. et al. Metastatic latency and immune evasion through autocrine inhibition of WNT. Cell 165, 45u201360 (2016)._ _ _ _ _ Chen, W., Dong, J., Haiech, J., Kilhoffer, M.-C. & Zeniou, M. Cancer stem cell quiescence and plasticity as major challenges in cancer therapy. Stem Cells Int. 2016, 1740936 (2016)._ _ _ Valle, S., Martin-Hijano, L., AlcalxE1, S., Alonso-Nocelo, M. & Sainz, B. Jr. The ever-evolving concept of the cancer stem cell in pancreatic cancer. Cancers 10, 33 (2018)._ _ _ _ Miranda-Lorenzo, I. et al. Intracellular autofluorescence: a biomarker for epithelial cancer stem cells. Nat. Methods 11, 1161u20131169 (2014)._ _ _ _ Sancho, P. et al. MYC/PGC-1u03B1 balance determines the metabolic phenotype and plasticity of pancreatic cancer stem cells. Cell Metab. 22, 590u2013 605 (2015)._ _ _ _ Jagust, P., de LuxxE1 n-Delgado, B., Parejo-Alonso, B. & Sancho, P. Metabolism-based therapeutic strategies targeting cancer stem cells. Front. Pharmacol. 10, 203 (2019)._ _ _ _ _ Frey, P. A. The Leloir pathway: a mechanistic imperative for three enzymes to change the stereochemical configuration of a single carbon in galactose. FASEB J. 10, 461u2013470 (1996)._ _ _ _ Warburg, O. The metabolism of carcinoma cells. J. Cancer Res. 9, 148u2013163 (1925)._ _ _ Shares, B. H., Busch, M., White, N., Shum, L. & Eliseev, R. A. Active mitochondria support osteogenic differentiation by stimulating u03B2-catenin acetylation. J. Biol. Chem. 293, 16019u201316027 (2018)._ _ _ _ _ AlcalxE1, S., Martinelli, P., Hermann, P. C., Heeschen, C. & Sainz, B. The anthrax toxin receptor 1 (ANTXR1) is enriched in pancreatic cancer stem cells derived from primary tumor cultures. Stem Cells Int. (2019).Guttman, M. et al. lincRNAs act in the circuitry controlling pluripotency and differentiation. Nature 477, 295u2013300 (2011)._ _ _ _ _ _ Yu, X., Lin, Y., Sui, W., Zou, Y. & Lv, Z. Analysis of distinct long noncoding RNA transcriptional fingerprints in pancreatic ductal adenocarcinoma. Cancer Med. (2017).Zagorac, S., Garcia-Bermejo, L. & Sainz, B. The epigenetic landscape of pancreatic cancer. Stem Cells Epigenomes 2, 10 (2018)._ _ _ Viale, A. et al. Oncogene ablation-resistant pancreatic cancer cells depend on mitochondrial function. Nature 514, 628u2013632 (2014)._ _ _ _ _ _ Yu, L., Chen, X., Sun, X., Wang, L. & Chen, S. The glycolytic switch in tumors: how many players are involved? J. Cancer 8, 3430u20133440 (2017)._ _ _ _ _ Wu, S., Lu, H. & Bai, Y. Nrf2 in cancers: a double-edged sword. Cancer Med. 8, 2252u20132267 (2019)._ _ _ _ Narendra, D., Tanaka, A., Suen, D.-F. & Youle, R. J. Parkin is recruited selectively to impaired mitochondria and promotes their autophagy. J. Cell Biol. 183, 795u2013803 (2008)._ _ _ _ _ De Luca, A. et al. Mitochondrial biogenesis is required for the anchorage-independent survival and propagation of stem-like cancer cells. Oncotarget 6, 14777u2013 14795 (2015)._ _ _ _ Zhao, J. et al. Mitochondrial dynamics regulates migration and invasion of breast cancer cells. Oncogene 32, 4814u20134824 (2013)._ _ _ _ Crncec, A. & Hochegger, H. Triggering mitosis. FEBS Lett. 593, 2868u20132888 (2019)._ _ _ _ Musacchio, A. The molecular biology of spindle assembly checkpoint signaling dynamics. Curr. Biol. 25, R1002u2013 R1018 (2015)._ _ _ _ Hwang, D. W. et al. Autophagy induced by CX-4945, a casein kinase 2 inhibitor, enhances apoptosis in pancreatic cancer cell lines. Pancreas (2017).Yang, M.-C. et al. Blockade of autophagy reduces pancreatic cancer stem cell activity and potentiates the tumoricidal effect of gemcitabine. Mol. Cancer 14, 179 (2015)._ _ _ _ _ Sui, X. et al. JNK confers 5-fluorouracil resistance in p53-deficient and mutant p53-expressing colon cancer cells by inducing survival autophagy. Sci. Rep. 4, 4694 (2014)._ _ _ _ _ Spratlin, J. L. & Mackey, J. R. Human equilibrative nucleoside transporter 1 (hENT1) in pancreatic adenocarcinoma: towards individualized treatment decisions. Cancers 2, 2044u20132054 (2010)._ _ _ _ _ Ding, X., Chen, W., Fan, H. & Zhu, B. Cytidine deaminase polymorphism predicts toxicity of gemcitabine-based chemotherapy. Gene 559, 31u201337 (2015)._ _ _ _ Kitamura, T., Qian, B.-Z. & Pollard, J. W. Immune cell promotion of metastasis. Nat. Rev. Immunol. 15, 73u201386 (2015)._ _ _ _ _ Chaffer, C. L. et al. Poised chromatin at the ZEB1 promoter enables breast cancer cell plasticity and enhances tumorigenicity. Cell 154, 61u201374 (2013)._ _ _ _ _ Rinkenbaugh, A. L. & Baldwin, A. S. The NF-u03BA B pathway and cancer stem cells. Cells 5, 16 (2016).Miao, Y. et al. Adaptive immune resistance emerges from tumor-initiating stem cells. Cell 177, 1172.e14u2013 1186.e14 (2019)._ _ _ Cioffi, M. et al. Inhibition of CD47 effectively targets pancreatic cancer stem cells via dual mechanisms. Clin. Cancer Res. 21, 2325u20132337 (2015)._ _ _ _ Cabezas-Sainz, P. et al. Improving zebrafish embryo xenotransplantation conditions by increasing incubation temperature and establishing a proliferation index with ZFtool. BMC Cancer 18, 3 (2018)._ _ _ _ _ Banerjee, A. et al. The effects of restricted glycolysis on stem-cell like characteristics of breast cancer cells. Oncotarget 9, 23274u2013 23288 (2018)._ _ _ _ Takeishi, S. & Nakayama, K. I. To wake up cancer stem cells, or to let them sleep, that is the question. Cancer Sci. 107, 875u2013881 (2016)._ _ _ _ _ Vera-Ramirez, L., Vodnala, S. K., Nini, R., Hunter, K. W. & Green, J. E. Autophagy promotes the survival of dormant breast cancer cells and metastatic tumour recurrence. Nat. Commun. 9, 1944 (2018)._ _ _ _ _ _ Sosa, M. S., Bragado, P. & Aguirre-Ghiso, J. A. Mechanisms of disseminated cancer cell dormancy: an awakening field. Nat. Rev. Cancer 14, 611u2013622 (2014)._ _ _ _ _ Wang, F. et al. Alisertib induces cell cycle arrest and autophagy and suppresses epithelial-to-mesenchymal transition involving PI3K/Akt/mTOR and sirtuin 1-mediated signaling pathways in human pancreatic cancer cells. Drug Des. Dev. Ther. 9, 575u2013601 (2015)._ _ Shinde, A. et al. Spleen tyrosine kinase-mediated autophagy is required for epithelial-mesenchymal plasticity and metastasis in breast cancer. Cancer Res. 79, 1831u20131843 (2019)._ _ _ _ _ Krebs, A. M. et al. The EMT-activator Zeb1 is a key factor for cell plasticity and promotes metastasis in pancreatic cancer. Nat. Cell Biol. 19, 518u2013529 (2017)._ _ _ _ Sciacovelli, M. & Frezza, C. Metabolic reprogramming and epithelial-to-mesenchymal transition in cancer. FEBS J. 284, 3132u20133144 (2017)._ _ _ _ _ Ilmer, M. & Horst, D. Pancreatic CSCs and microenvironment. Genes Cancer 6, 365u2013366 (2015)._ _ _ _ _ Watanabe, R. et al. Pyruvate controls the checkpoint inhibitor PD-L1 and suppresses T cell immunity. J. Clin. Investig. 127, 2725u20132738 (2017)._ _ _ _ Wangpaichitr, M. et al. Relationship of metabolic alterations and PD-L1 expression in cisplatin resistant lung cancer. Cell Dev. Biol. 6, 183 (2017).Martinez-Garcia, R. et al. Transcriptional dissection of pancreatic tumors engrafted in mice. Genome Med. 6, 27 (2014)._ _ _ _ _ Gallmeier, E. et al. Inhibition of ataxia telangiectasia- and Rad3-related function abrogates the in vitro and in vivo tumorigenicity of human colon cancer cells through depletion of the CD133(+) tumor-initiating cell fraction. Stem Cells 29, 418u2013429 (2011)._ _ _ _ Chomczynski, P. & Sacchi, N. Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction. Anal. Biochem. 162, 156u2013159 (1987)._ _ _ _ Hotta, A. et al. Isolation of human iPS cells using EOS lentiviral vectors to select for pluripotency. Nat. Methods 6, 370u2013376 (2009)._ _ _ _ GraxF1a, O., Rubio-Camarillo, M., Fdez-Riverola, F., Pisano, D. G. & Glez-PexF1a, D. Nextpresso: next generation sequencing expression analysis pipeline. Curr. Bioinformatics 13, 583u2013591 (2018).Athar, A. et al. ArrayExpress updateu2014from bulk to single-cell expression data._ Nucleic Acids Res. 47, D711u2013D715 (2019)._ _ _ _ \"- - \"We acknowledge and thank Dr. Nuria Malats and Jaime Villarreal from the Spanish National Cancer Research Center (CNIO) for RNA sequencing and analysis, funded by Fondo de Investigaciones Sanitarias (FIS) grant PI18/01347. We thank Patricia SxE1nchez-Tomero and Marina Ochando-Garmendia for technical assistance and support and Dr. RaxFAl SxE1nchez Lanzas for assistance with autophagy experiments. We want to particularly acknowledge the patients and the BioBank Hospital RamxF3 n y Cajal-IRYCIS (PT13/0010/0002) integrated in the Spanish National Biobanks Network for its collaboration and, in particular, AdrixE1n Povo Retana for macrophage isolation. We would also like to thank the Transmission Electron Microscopy Unit Laboratory, part of the UAM Interdepartmental Investigation Service (SIdI); Coral Pedrero for exceptional help with in vivo experiments; and the laboratories of Dr. Amparo Cano and Dr. JosxE9 GonzxE1lez CastaxF1 o for reagents and helpful discussions. S.V. was a recipient of an Ayuda de Movilidad del Personal Investigador del IRYCIS, a mobility grant from the Instituto RamxF3 n y Cajal de InvestigacixF3n Sanitaria (IRYCIS), Madrid, Spain, and a pre-doctoral fellowship from the Comunidad de Madrid, Ayudas Para La ContratacixF3n De Investigadores Predoctorales Y Posdoctorales (PEJD-2017-PRE/BMD-5062), Madrid, Spain. This study was supported by a RxE1mon y Cajal Merit Award (RYC-2012-12104) from the Ministerio de EconomxEDa y Competitividad, Spain (to B.S.); funding from la Beca Carmen Delgado/Miguel PxE9rez-Mateo from AESPANC-ACANPAN Spain (to B.S.); a Conquer Cancer Now Grant from the Concern Foundation (Los Angeles, CA, USA) (to B.S.); a Coordinated grant (GC16173694BARB) from the FundacixF3 n AsociacixF3n EspaxF1ola Contra el CxE1ncer (AECC) (to B.S.); FIS grants PI18/00757 (to B.S.), PI16/00789 (to M.A.F.-M.), PI18/00267 (to L.G.-B.; co-financed through Fondo Europeo de Desarrollo Regional (FEDER) u201CUna manera de hacer Europau201D ); a Miguel Servet award (CP16/00121) (to P.S.); a Max Eder Fellowship of the German Cancer Aid (111746) (to P.C.H.); and the German Research Foundation (DFG, CRC 1279 u201CExploiting the human peptidome for Novel Antimicrobial and Anticancer Agentsu201D; to P.C.H.).\"- - Author notes - \\'Christopher HeeschenPresent address: Center for Single-Cell Omics and Key Laboratory of Oncogenes and Related Genes, Shanghai Jiao Tong University School of Medicine, Shanghai, China\\' - Authors and Affiliations - \"Department of Biochemistry, Universidad AutxF3noma de Madrid (UAM) and Instituto de Investigaciones BiomxE9dicas u201CAlberto Solsu201D (IIBM), CSIC-UAM, Madrid, SpainSandra Valle,_Sonia AlcalxE1,_Laura Martin-Hijano,_Diego Navarro,_ Lourdes Yuste,_Laura Ruiz-CaxF1as,_Marta Alonso-Nocelo,_Miguel xC1ngel FernxE1 ndez-Moreno_&_Bruno Sainz JrCancer Stem Cells and Fibroinflammatory Microenvironment Group, Chronic Diseases and Cancer Area 3 - Instituto RamxF3n y Cajal de InvestigacixF3 n Sanitaria (IRYCIS), Madrid, SpainSandra Valle,_Sonia AlcalxE1,_Laura Martin-Hijano,_ Diego Navarro,_Lourdes Yuste,_Laura Ruiz-CaxF1as,_Marta Alonso-Nocelo_&_ Bruno Sainz JrDepartment of Zoology, Genetics and Physical Anthropology, Veterinary Faculty, Universidad de Santiago de Compostela, Lugo, SpainPablo Cabezas-SxE1 inz,_Juan A. Rubiolo_&_Laura SxE1nchezBiomarkers and Therapeutic Targets Group - IRYCIS, Madrid, SpainEdurne Ramos MuxF1oz_&_Laura Garcia-BermejoDepartment of Internal Medicine I, Ulm University, Ulm, GermanyKanishka Tiwary,_Karolin Walter_&_Patrick C. HermannDepartment of Neurology, Universidad AutxF3noma de Madrid (UAM), Madrid, SpainEmilio GonzxE1lez-ArnayStem Cells & Cancer Group, Molecular Pathology Programme, Spanish National Cancer Research Centre (CNIO), Madrid, SpainChristopher HeeschenTranslational Research Unit, IIS AragxF3n, Zaragoza, SpainPatricia SanchoCentro de InvestigacixF3n BiomxE9dica en Red en Enfermedades Raras (CIBERER), Madrid, SpainMiguel xC1ngel FernxE1ndez-MorenoInstituto de InvestigacixF3n Sanitaria Hospital 12 de Octubre (imas12), Madrid, SpainMiguel xC1ngel FernxE1ndez-Moreno\" - \"AuthorsSandra ValleYou can also search for this author in _Sonia AlcalxE1You can also search for this author in _Laura Martin-HijanoYou can also search for this author in _Pablo Cabezas-SxE1inzYou can also search for this author in _Diego NavarroYou can also search for this author in _Edurne Ramos MuxF1ozYou can also search for this author in _Lourdes YusteYou can also search for this author in _Kanishka TiwaryYou can also search for this author in _Karolin WalterYou can also search for this author in _Laura Ruiz-CaxF1asYou can also search for this author in _Marta Alonso-NoceloYou can also search for this author in _Juan A. RubioloYou can also search for this author in _Emilio GonzxE1 lez-ArnayYou can also search for this author in _Christopher HeeschenYou can also search for this author in _Laura Garcia-BermejoYou can also search for this author in _Patrick C. HermannYou can also search for this author in _Laura SxE1nchezYou can also search for this author in _Patricia SanchoYou can also search for this author in _Miguel xC1ngel FernxE1ndez-MorenoYou can also search for this author in _Bruno Sainz JrYou can also search for this author in _\" - Contributions - S.V. acquired, analyzed, and interpreted all data as well as developed the study concept and drafted the manuscript; S.A., L.M.-H., D.N., and M.A.-N. performed in vitro experiments and analyzed data; L.Y. and L.R.-C. performed in vivo studies; J.A.R. performed TEM experiments; P.C.-S. and L.S. designed and performed the zebrafish studies; E.R.M. and L.G.-B. performed the Seahorse studies and IHCs; K.W., K.T., and P.C.H. performed, designed, and analyzed the Nanog-YNL studies; E.G.-A. is our in-house pathologist and assisted with in vivo histological analyses; P.C.H. and C.H. produced the 185scd cell line. P.S. provided expertise on CSC metabolism and assisted in the execution and analysis of key in vitro experiments; M.A.F.-M. and B.S. developed the study concept, obtained funding, interpreted the data, and drafted/edited the manuscript; and all authors edited the manuscript. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License ().\" - \"Moreover, the expression of u03B1-TAT1 was downregulated in the hearts of Lmna mice compared with those of control mice at 6 months of age (Fig._). Considering the important role of microtubule mechanics in cardiomyocytes, we examined whether the K40 acetylation of u03B1-tubulin, which promotes microtubule stability and resistance to mechanical breakage, was altered in cardiomyopathy caused by mutations in LMNA as a result of the decreased u03B1-TAT1 expression. A comparison of cardiac protein extracts from WT and Lmna mice by immunoblot revealed the downregulation of u03B1-tubulin K40 acetylation in the hearts of mice with Lmna mutations (Fig._). We confirmed this result via immunoblot and immunofluorescence in cardiomyocytes isolated from WT and Lmna mice (Fig._). To determine the role of u03B1-tubulin K40 acetylation in human disease, we next analyzed cardiac tissue from explanted hearts from patients with cardiomyopathy caused by mutations in LMNA following a heart transplant. We observed that the K40 acetylation of u03B1-tubulin was decreased in the LMNA-mutated heart tissue compared with control heart tissue (Fig._). Similar u03B1-tubulin K40 acetylation profile was observed in immunoblot and immunofluorescence assays of cardiomyocytes derived from patient-specific human iPSCs (carrying LMNA mutations (Fig._). These observations indicated that the acetylation of u03B1-tubulin was abnormally downregulated in cardiomyopathy caused by mutations in LMNA. Further, our data support the hypothesis that decreased u03B1-TAT1 expression is responsible for the reduced levels of tubulin acetylation in models expressing mutated Lamin A.\" - \"To confirm that the abnormal u03B1-tubulin K40 acetylation profile was the consequence of the LMNA mutations and not a secondary effect of cardiac disease, we next used C2-WT and C2-H222P cells. We confirmed that the acetylation level of u03B1 -tubulin was lower in cells expressing mutated p.H222P Lamin A than in cells expressing WT Lamin A (Fig._). Histone deacetylase 6 (HDAC6) is the major deacetylase responsible for removing the acetyl group of u03B1-tubulin. We did not report any modulation in HDAC6 expression neither in the Lmna-mutated heart tissue compared with control heart tissue from mice nor in C2-H222P cells compared with C2-WT cells (Fig._). Together, these data indicate that A-type lamin variants negatively regulate the acetylation of u03B1-tubulin in heart tissues and in cultured cells, mainly through u03B1-TAT1 expression.\" - \"u03B1-Tubulin acetylation mediates cardiac function\" - \"To further evaluate the importance of the effects of u03B1-tubulin K40 acetylation on cardiac function in vivo, we studied mice bearing a targeted deletion of Atat1. The cardiac tissues of homozygous Atat1 knockout mice (Atat1) did not exhibit detectable levels of acetylated u03B1-tubulin as compared with WT mice (Fig._). We next analyzed Atat1 mice by echocardiography to test their cardiac function. Compared to WT mice, Atat1 mice had significantly increased left ventricular end-diastolic and end-systolic diameters, demonstrating left ventricle dilation, starting as early as 3 months of age (Table_, Fig._). We further found significantly decreased fractional shortening in 3-month-old Atat1 mice (Table_, Fig._). Our results revealed a previously unnoticed cardiac dilation phenotype of Atat1 mice.\" - \"Fig. 5: u03B1-Tubulin acetylation mediates cardiac function in Atat1 knock-out mice and in mice expressing cardiomyopathy-causing mutant A-type lamins.a, c, e Top: Immunoblots showing the cardiac protein expression of acetylated and total u03B1-tubulin a in 6-month-old male Atat1 (WT) mice (nu2009=u20093) and Atat1 knockout mice (Atat1) (nu2009=u20095) c in 5-month-old male WT mice (nu2009=u20093) and WT mice transduced with AAV expressing HDAC6 (nu2009 =u20093), and c in 3-month-old male Lmna (H222P) mice treated with tubastatin A for one month (nu2009=u20094) compared with mice treated with DMSO (nu2009 =u20094). a, c, e Bottom: bar graph showing the quantification of acetylated u03B1-tubulin normalized to total u03B1-tubulin (meanu2009xB1u2009SD). b, d, f Bar graphs showing the left ventricular end-diastolic diameters (LVEDd), end-systolic diameters (LVESd) and fraction shortening (FS) (meanu2009xB1u2009 SEM) b in Atat1 (WT) mice (nu2009=u200910) and Atat1 knockout mice (Atat1) (nu2009 =u200911) from 3 to 6 months of age, d in 5-month-old male WT mice (nu2009=u2009 3) and WT mice transduced with AAV expressing HDAC6 (nu2009=u20093) and f in 3-month-old male Lmna (H222P) mice treated for one month with tubastatin A (nu2009 =u20096) compared to mice treated with DMSO (nu2009=u20095). g Immunofluorescence staining of acetylated u03B1-tubulin and u03B1-actinin in cardiomyocytes derived from iPS cells from a patient carrying the LMNA p.H222P mutation, 40 days post-differentiation, treated (lower panel) or not (upper panel) with 3u2009xB5M of tubastatin A for 24u2009h. u03B1-Actinin is shown as cardiomyocyte differentiation marker. h Contraction frequency of cardiomyocytes derived from iPS cells from a patient carrying the LMNA p.H222P mutation, 40 days post-differentiation, treated or not with 3u2009xB5M of tubastatin A for 24u2009h (nu2009=u20093 technical replicates) (meanu2009xB1u2009SD). Statistics: for a, c, e unpaired two-tailed t-test; for b, d, f, h one-way ANOVA followed by Tukeys multiple comparison test. Source data are provided as a Source Data file.\" - Table 1 Echocardiographic data for Atat1 (WT) and Atat1 (Atat1 KO) mice at 3; 4 and 5 months of age - \"Given that the deacetylation of u03B1-tubulin is driven by HDAC6, and to confirm the role of u03B1-tubulin K40 acetylation on cardiac function, we studied by echocardiography the cardiac function of WT mice transduced with or without an AAV vector expressing HDAC6 (AAV-HDAC6). Compared with WT mice, mice transduced with AAV-HDAC6 had decreased acetylation levels of u03B1-tubulin (Fig._). This is ensuing significantly increased left ventricular end-systolic diameters and decreased fractional shortening (Table_, Fig._). Taken together, these data highlight the importance of u03B1-tubulin K40 acetylation for cardiac function.\" - Table 2 Echocardiographic data for wild-type (WT) mice transduced or not with AAV expressing HDAC6 (AAV-HDAC6) at 2, 3, 4 and 5 months of age - \"We next assessed whether increasing the acetylation levels in Lmna mice with the HDAC6 inhibitor tubastatin A could improve the cardiac function in these animals. We treated 2-month-old Lmna mice with tubastatin A for 1 month and then performed echocardiography. Tubastatin A treatment increased the level of u03B1-tubulin acetylation in hearts of Lmna mice as compared with DMSO treatment (Fig._) without altering other HDAC expression or acetylation of histone H3 (Fig._). Moreover, the left ventricular end-systolic diameter was significantly lowered and fractional shortening was significantly improved in Lmna mice treated with tubastatin A compared with DMSO treated (Table_, Fig._). In vitro, treatment with tubastatin A of cardiomyocytes derived from iPS (iPS-CM) from a patient with p.H222P LMNA mutation was able to increase u03B1-tubulin acetylation (Fig._ ) and decrease contraction frequency (Fig._) compared with untreated cells. Tubastatin A treatment was also able to ameliorate the contraction profile of iPS-CM carrying LMNA p.H222P mutation (Fig._), suggesting an increased contraction force. This reflects the previously reported nonreciprocal relationship between force and contraction frequency in cardiac muscle cells. These data demonstrate that u03B1 -tubulin acetylation plays key role in the development of cardiac dysfunction in mice with cardiomyopathy caused by mutations in LMNA.\" - Table 3 Echocardiographic data for Lmna (H222P) mice treated with tubastatin A or DMSO at 2, 2.5, and 3 months of age - \"One of the common cellular phenotypes observed in cardiomyopathy caused by mutations in LMNA is an abnormal elongation of nuclei in cardiomyocytes. Mean length of cardiomyocyte nuclei in hearts of Lmna mice treated with tubastatin A was significantly longer, as observed in untreated Lmna mice, than in WT mice (Fig._ and ). Nuclei of cardiomyocytes in hearts of Atat1 mice, as well as WT mice transduced with AAV-HDAC6, had an overall shape that was u201Croundedu201D, similar to WT mice (Fig._ and ). This suggests that u03B1-tubulin K40 acetylation does not have a role in the shape of the nuclei.\" - \"u03B1-Tubulin acetylation mediates Cx43 localization in cellular and mouse models expressing cardiomyopathy-causing mutant A-type lamins\" - \"The development of cardiomyopathy caused by mutations in LMNA was previously correlated with the remodeling of the gap junction protein Cx43 and characterized by a loss of localization at intercalated discs and redistribution to the lateral sides of cardiomyocyte membranes (i.e., u201Clateralizationu201D) in the hearts of Lmna mice. The trafficking of Cx43 and its correct localization at intercalated discs in the heart is regulated in part by the microtubule network. We, therefore, suspect that the changes in u03B1-tubulin acetylation might explain the Cx43 phenotype in cardiomyopathy caused by mutations in LMNA.\" - \"First, immunofluorescence analyses confirmed that Cx43 underwent extensive remodeling in isolated cardiomyocytes (Fig._) and whole hearts (Fig._) from Lmna mice at an earlier stage (3 months) of disease. To further study the mechanisms underlying this Cx43 remodeling, we used C2-H222P and C2-WT cells. We first assessed the localization of Cx43 in C2-H222P and C2-WT cells. While Cx43 was localized at the cell-to-cell junction in C2-WT cells, it was mostly localized in the cytoplasm of C2-H222P cells (Fig._). Similar alterations were observed when C2-H222P cells were cultured in 3D conditions (i.e., as spheroids; Fig._).\" - \"Fig. 6: u03B1-Tubulin acetylation mediates Cx43 localization in cellular and mouse models expressing cardiomyopathy-causing mutant A-type lamins.a, b Immunofluorescence staining of Cx43 and u03B1-actinin in a isolated adult cardiomyocytes and b heart cross-sections from 3-month-old male WT mice and Lmna (H222P) mice. The arrows indicate the lateralization of Cx43. c Immunofluorescence staining of Cx43 and N-cadherin in C2-WT and C2-H222P cells. d Immunofluorescence staining of Cx43 in C2-WT and C2-H222P cells organized in 3D spheroids. e, f Top: immunoblot showing expression of acetylated and total u03B1-tubulin in e C2C12 cells treated or not with siRNA targeting Atat1 and f in C2-H222P cells treated or not with different doses of tubastatin A. GAPDH was used as a loading control. e, f Bottom: representative immunofluorescence staining of Cx43 and acetylated u03B1-tubulin in e C2-H222P cells treated or not with a siRNA targeting Atat1 and f in C2-H222P cells treated or not with tubastatin A. The insets show a higher magnification. g Immunofluorescence staining of Cx43 and u03B1-actinin of heart cross-sections from 6-month-old male Atat1 (WT) mice and Atat1 knockout mice (Atat1) (top panel); from 5-month-old male WT mice transduced or not with AAV expressing HDAC6 (middle panel) and from 3-month-old male Lmna (H222P) mice treated with tubastatin A compared with mice treated with DMSO (bottom panel). h Bar graph showing the quantification of lateralized Cx43 staining (nu2009=u20094 mice per condition; meanu2009xB1u2009SD). i Left: representative Sirius red staining of heart cross-sections from 6-month-old male Atat1 (WT) mice, Atat1 knockout mice (Atat1), and 3-month-old male Lmna (H222P) mice treated or not with tubastatin A. Right: bar graph showing the quantification of Sirius red-stained area (nu2009=u2009 4 mice per condition; meanu2009xB1u2009SD). For au2013g, nuclei counterstained with DAPI are shown and a representative of three independent repeats is shown. Statistics: for h and i, one-way ANOVA followed by Tukeys multiple comparison test. Source data are provided as a Source Data file.\" - \"Considering that the levels of K40-acetylated u03B1-tubulin were decreased in the presence of mutated A-type lamin, we next asked whether altering the u03B1 -tubulin acetylation levels could affect Cx43 localization in cells. To test the effect of decreased tubulin acetylation, we knocked down Atat1 in C2C12 cells using siRNA (Fig._, top panel), which resulted in the mislocalization of Cx43 (Fig._, bottom panel). To further confirm that the acetylation of u03B1-tubulin was critical for Cx43 localization, we then treated C2-H222P cells with tubastatin A, which increased the levels of acetylated u03B1-tubulin in a dose-dependent manner (Fig._, top panel) and restored the normal localization of Cx43 (Fig._ , bottom panel). Together, these results demonstrate that the acetylation of u03B1 -tubulin is essential for the correct localization of Cx43 in muscle cells.\" - \"The trafficking of Cx43 along microtubules is driven in part by motor proteins from the kinesin-1 family. It has also been shown that u03B1-tubulin acetylation positively influences motor protein binding to microtubules and has a role in intracellular trafficking. To determine if active, microtubule-based transport is important for Cx43 localization, we treated cells with adenosine 5 -(u03B2,u03B3-imido)triphosphate (AMP-PNP), a non-hydrolysable analog of ATP that inhibits motility of kinesin motor proteins that drive intracellular transport. Following AMP-PNP treatment, Cx43 was misplaced within the cytoplasm of C2-WT myoblasts (Fig._) and cardiomyocytes isolated from WT mice (Fig._). To test the role of kinesin in this process more directly, we transfected C2-WT myoblasts with siRNA against Kif5B, the kinesin motor suggested to be responsible for Cx43 trafficking (Fig._). In cells depleted of Kif5B, Cx43 was localized in the cytosol rather than at cell-cell junctions (Fig._). This demonstrates an important role of active microtubule-based transport in the trafficking and localization of Cx43 to cell-cell junctions. This process is perturbed in the presence of mutated A-type lamins. To confirm the relevance of this mechanism in vivo, we performed immunostaining of Cx43 of cardiac sections. We observed Cx43 remodeling and redistribution to the lateral sides in Atat1 mouse cardiomyocytes, similar to the phenotype observed in Lmna hearts (Fig._ top panel, ). Similarly, Cx43 was relocated to the lateral sides in cardiomyocytes from WT mice transduced with an AAV-HDAC6 (Fig._ middle panel, ). Strikingly, Cx43 localization at intercalated discs in the hearts of Lmna mice was restored by treatment with tubastatin A (Fig._ bottom panel, ). This demonstrates that u03B1-tubulin acetylation plays a key role in the trafficking of Cx43 to intercalated discs.\" - \"Given that gap junctions are required to ensure a coordinated contraction and pumping activity in heart, we anticipated to prevent conduction defects (prolonged QRS complex duration) in tubastatin A-treated Lmna mice. One-month tubastatin A treatment failed to shorten QRS duration in 3-month-old mice (Table_, Fig._ ). Conversely, we expected to find conduction defects in Atat1 mice as well as in AAV-HDAC6 transduced WT mice. Surprisingly, we did not report any conduction defects (Table_, Fig._). This could be explained in part by the fact that C57BL/6J genetic background mice are resistant to arrhythmia. Also, abnormal tissue architecture due to increased fibrosis may have synergistic effects together with lateralization of Cx43 and lead to conduction slowing. We then performed Sirius red staining of collagen fibrils on heart sections of Atat1 and Lmna mice treated or not with tubastatin A. As previously reported, Lmna mice displayed significantly increased cardiac fibrosis as compared with WT mice (Fig._). Treatment with tubastatin A failed to prevent collagen fibrils accumulation (Fig._). On the opposite, heart sections from Atat1 were similar to WT (Fig._ ), meaning that lack of acetylated u03B1-tubulin does not trigger cardiac fibrosis.\"- - \"Dilated cardiomyopathy caused by mutations in LMNA is associated with both actin and microtubule network alterations. We aimed to decipher the mechanisms underlying the possible interplay between these two cytoskeletal components in this pathological context. Here, we uncovered a mechanism through which actin-microtubule crosstalk plays a major physiological role in cardiac function (illustrated in Fig._ ). Specifically, we found that cofilin-1 phosphorylation on T25 regulates the MRTF-A/SRF axis in cells expressing LMNA mutations. In this mechanism, phospho(T25)-cofilin-1 binds to MRTF-A, preventing its translocation to the nucleus, and thus blunts the transcription factor activity of SRF. This in turn leads to the loss of u03B1-tubulin K40 acetylation due to the downregulation of u03B1-TAT1 expression and thereby disrupts the localization of Cx43 in cardiomyocytes, participating in the development of cardiomyopathy. Further, we showed that elevation of u03B1 -tubulin acetylation levels with tubastatin A restored the correct localization of Cx43 at intercalated discs and improved cardiac function in a mouse model of cardiomyopathy caused by LMNA mutations.\" - \"Fig. 7: Schematic representation of the actin-microtubule interplay mediated by phospho(T25)-cofilin-1 via MRTF-A/SRF signaling and u03B1-tubulin acetylation in LMNA-cardiomyopathy.Created with BioRender.com.\" - \"Cofilin-1 is an essential protein that maintains the myofilament architecture needed for the mechanical properties of sarcomeres, cell motility, and nuclear architecture. Cofilin-1 phosphorylation controls the interactions of cofilin-1 and actin and thus the mechanics of the actin regulatory machinery. The phosphorylation of cofilin-1 on Ser3 by TESK and LIMK was previously described to markedly attenuates cofilin-1/actin interactions, while dephosphorylation at this site restores the affinity of cofilin-1 for actin. Cofilin phosphocycling at this residue clearly drives a diverse variety of actin-dependent processes, ranging from actin-based motility to the pathogenesis of neurodegenerative diseases. Here, we uncover a novel role of cofilin-1 through phosphorylation on Thr25 by ERK1/2 (kinase) and dephosphorylation by PDXP (phosphatase). The control of cofilin-1 phosphorylation on Thr25, in addition to its effect on actin dynamics, directly regulates MRTF-A/SRF signaling activity in cardiac cells and an imbalance in this regulation ultimately leads to cardiac disease associated with mutations in LMNA. Our results also broaden our knowledge of diseases related to cofilin-1, as we demonstrate that cofilin-1 participates in the pathogenesis of left ventricular dysfunction through the transcriptional control of ATAT1 driving u03B1-tubulin acetylation.\" - \"Previous work showed that activation of ERK1/2 signaling induces actin depolymerization via cofilin-1 phosphorylation at T25 and promotes the disassembly of F-actin filaments into G-actin monomers. We now show that phospho(T25)-cofilin-1 directly binds to MRTF-A and sequesters it in the cytoplasm, thereby inhibiting SRF transcriptional activity. MRTF-A physically associates with SRF to synergistically activate the transcription of a subset of CArG box-containing genes, including those encoding actin microfilament effectors such as cofilin-1. SRF plays a crucial role in cardiac maturation and function, and the cardiac-specific deletion of SRF in adult mice results in cardiomyopathy. We show that the expression of the acetylating enzyme u03B1-TAT1, which drives the acetylation of u03B1-tubulin, is controlled by the activity of SRF and plays a role in cardiac function. Our findings provide a novel link between impaired SRF signaling and the perturbation of cellular functions that lead to cardiac defects in cardiomyopathy caused by mutations in LMNA. Given that SRF transcriptional activity is dependent on actin cytoskeleton dynamics, it is becoming increasingly clear that the two cytoskeletal systems work together in core cellular processes and that their functional dynamics are often intimately interwoven. Recent research shows that actin and microtubule networks engage in a variety of types of crosstalk that are important for biological processes such as cell migration, axonal specification, cell polarity, or cell division. Our work shows that changes in the regulation of a single factor, cofilin-1, trigger a cascade of molecular events impacting both actin and microtubule dynamics, further highlighting that the cardiac cytoskeleton is not a set of individual elements but rather a cooperative system in which key elements function synergistically in a precise and highly complementary manner.\" - \"Altered Cx43 localization and disorganized gap junction coupling contribute to the pathogenesis of cardiomyopathy caused by mutations in LMNA. Tubulin acetylation is known to influence the binding of motor proteins to microtubules and to play a role in intracellular Cx43 trafficking. Changes in the localization and regulation of Cx43 have been described in many forms of cardiac diseases including heart failure. However, the molecular mechanisms of gap junction remodeling are not yet known, and their elucidation is of paramount importance for the development of therapies aimed at improving gap junction coupling during disease. Microtubules have a well-studied role in the trafficking of Cx43 to the plasma membrane, and the implication of the microtubule network in heart disease is solidly established. Recent findings highlighted the role of tubulin detyrosination in cardiomyopathy, as a perturbation of this post-translational modification was shown to correlate with functional decline in the heart. Here, we demonstrated that the remodeling of cardiac Cx43 in cardiomyopathy caused by mutations in LMNA occurred due to the decreased acetylation of u03B1-tubulin, thus establishing this tubulin post-translational modification as a cause of the cardiac disease phenotype. Post-translational modifications can play important roles in controlling the stability and function of microtubules. Most of tubulin post-translational modifications (e.g., detyrosination, polyglutamylation, and polyglycylation) alter residues within the C-terminal tail of tubulin that extends from the surface of the microtubule. However, u03B1-tubulin K40 acetylation is the unique tubulin post-translational modification that localizes to the inside of the microtubule. Acetylation of u03B1-tubulin K40 was found to be stable, long-lived microtubule subpopulations. However, it still remains controversial whether the relationship between u03B1 -tubulin K40 acetylation and microtubule stabilization is correlative or causative. How decreased tubulin acetylation alters the overall architecture of the microtubule cytoskeleton and disturbs Cx43 targeting intercalated discs remains to be explored. However, acetylated u03B1-tubulin stabilizes microtubules and a decreased acetylation could alter microtubules longitudinal alignment within cardiomyocytes that in turn, might affect Cx43 localization (illustrated in Fig._). Moreover, our observations could have broader implications for other physiologically important intracellular trafficking events in cardiomyocytes. For example, voltage-gated potassium channels, which are involved in the excitability of cardiac muscle cells and modulate contraction and relaxation, are transported along microtubules. It will thus be exciting to study the trafficking of other channels in cardiomyopathy caused by mutations in LMNA, which may help to identify novel therapeutic targets for cardiac disorders that arise from the deficient delivery of ion channels to the plasma membrane.\" - \"We showed that rescuing cardiac Cx43 localization with HDAC6 inhibitor is a straightforward therapeutic strategy. Aberrant tubulin acetylation profiles are observed in a broad spectrum of diseases, which has led to using HDAC6 inhibition as a therapeutic approach in preclinical studies. General HDAC inhibitors lead to the deacetylation of histones, modify gene transcription, and increase the acetylation of many non-histone proteins. HDAC6 does not act on histones and its major substrate is u03B1-tubulin. Acetylated u03B1-tubulin occurs on polymerized microtubules and affects microtubule dynamics and stability. Hence, the finding that u03B1 -tubulin acetylation in the heart is involved in cardiomyopathy, and that specific inhibition of HDAC6 has beneficial effects, opens up new therapeutic possibilities. Furthermore, while u03B1-tubulin is the main target of HDAC6, this latter has other substrates including cortactin, ERK1, and Smad3 that have been shown to be dysregulated in LMNA-cardiomyopathy. It has also been reported that HDAC6 inhibition has a cardioprotective effect through autophagy induction, a mechanism also affected in LMNA-cardiomyopathy. A synergistic effect through u03B1-tubulin, cortactin, ERK, and Smad signaling and autophagy induction might be highly beneficial in LMNA-cardiomyopathy.\" - \"Most of the promising HDAC inhibitors target multiple HDAC isoforms. These pan-HDAC inhibitors are often driving several undesirable side effects, which limits the clinical utility of these molecules. Selective HDAC6 inhibitor tubastatin A may exhibit reduced side effects compared to the pan-HDAC inhibitors. However, tubastatin A belongs to the hydroxamate class of molecules and displays mutagenicity/genotoxicity and poor oral delivery, thus hindering its development for clinical use. Therefore, the development of a highly potent and selective HDAC6 inhibitor with good bioavailability and without genotoxicity is highly desired. Access to such HDAC6 inhibitor would enable us to study the effects of HDAC6 inhibition in cardiomyopathy caused by mutations in LMNA in preclinical in vivo models for translation to clinical trials. Among HDAC6 inhibitors, only ricolinostat (ACY-1215, ricolinostat) and citarinostat (ACY-241, citarinostat) are currently evaluated in clinical trials for several forms of cancer. Given that HDAC6 inhibitors have lower adverse effects than pan-HDAC inhibitors, such compounds may be important in the array of therapeutic approaches for cardiomyopathy. This finding is notable because despite the growing evidence that microtubule-targeting strategies can be employed to slow or even halt the progression of inherited cardiomyopathies, clinical translation has thus far been hindered by the lack of available practical therapeutic targets. Our recent work showed that increasing u03B1-tubulin K40 acetylation by augmenting microtubule stability using paclitaxel, could also be potentially translatable into therapy for cardiomyopathy caused by mutations in LMNA. It has been shown that paclitaxel can be safely administered in patients with cardiac dysfunction. Other microtubule-stabilizing agents have been described and should also benefit clinical development for cardiac diseases. ERK1/2 inhibitors may also be promising therapies, as treatment of lamin mutant mice with ERK1/2 signaling inhibitors can reduce cardiac and skeletal phenotypes; this may be attributed at least in part to the effect on MRTF-A/SRF signaling. Although the effects of the long-term use of the aforementioned compounds are currently unknown, these findings encourage further approaches to target impaired MRTF-A/SRF signaling and thereby ameliorate devastating cardiac diseases associated with LMNA mutations. Overall, our work provides an adequate basis for testing these therapeutic approaches for patients with cardiomyopathy caused by mutations in LMNA.\"- - Ethical statement - \"This research complies with all relevant ethical regulations for the boards/committees and institutions that approved the study protocols. Sections of explanted hearts from human subjects with LMNA mutations were obtained without identifiers from Myobank-AFM de llnstitut de Myologie. Myobank-AFM received approval from the French Ministry of Health and from the Committee for Protection of Patients to share tissues and cells of human origin for scientific purposes, ensuring the donors anonymity, respect of their will, and consent according to the legislation. Animal experimentation was performed in agreement with the guidelines from Directive 2010/63/EU of the European Parliament regarding the protection of animals used for scientific purposes. All animal experiments were approved by the French Ministry of Higher Education and Research at the Center for Research in Myology for the care and use of experimental animals.\" - C2C12 cell culture and reagents - \"The generation of C2-WT and C2-H222P cells was described previously. Cells were cultured in proliferative Dulbeccos modified Eagles medium (DMEM) supplemented with 10% fetal bovine serum (Life Technologies) and 0.1% gentamycin (Invitrogen). Transient transfection experiments were performed using Lipofectamine 2000 (Invitrogen) according to the manufacturers instructions. Briefly, cells seeded at 3u2009xD7u200910 cells per well in six-well plates were transfected for 24u2009h. Plasmids encoding cofilin-1 T25A, T25D, and V20A were previously described in7. Scrambled siRNA and siRNAs silencing PDXP, SSH1, PP2B, and Atat1 were purchased from Santa Cruz Biotechnology. Working concentrations of 50u2009 u03BCM selumetinib were prepared from stocks diluted in DMSO. Cells were incubated with selumetinib for 15u2009h. Untransformed C2C12 cells (ATCC, #CRL-1772) were seeded at 3u2009xD7u200910 cells per well in six-well plates and were transfected using Lipofectamine 3000 (Invitrogen) with 1u2009xB5g of plasmids encoding MRTF-A (#19838, Addgene), MRTFu0394100 (#27176, Addgene) or SRF-VP16 for 48u2009h. SRF-VP16 plasmid was kindly provided by Dr. Sotiropoulos (UniversitxE9 de Paris).\" - Human induced pluripotent stem cell culture and differentiation - \"The hiPSC carrying LMNA p.H222P, p.S143P, and p.R190W mutations lines were generated from patients\\' primary cells (Table_) by Sendai virus infection and characterized as previously described. A signed informed consent was obtained from all the individuals participating and the study was approved by the review boards of Humanitas Research Hospital (ID:1215), and Tampere Pirkanmaa Hospital District (R08070) to establish, culture, and differentiate hiPSC lines. The hiPSC lines were cultured in mTeSR1 medium (Stemcell Technologies) supplemented with 5u2009 u03BCM StemMACS Y27632 (Miltenyi Biotec) at 37u2009xB0C and 5% CO2 on Matrigel hESC-Qualified Matrix (Corning Life Sciences), and the medium was refreshed daily (without Y27632). Upon reaching ~80% confluence, the cells were passaged in clumps by scraping with a pipette tip. After at least two passages, cells were differentiated using a 2D monolayer differentiation protocol and maintained in a 5% CO2/air environment. Briefly, the cells were detached using ReLeSR and replated onto Matrigel-coated 12-well plates in mTeSR1 medium. At 85% cell confluence, the cells were treated for 2 days with 6u2009xB5M CHIR99021 (Abcam) in RPMIu2009 +u2009B27 supplement without insulin. On day 2, the medium was replaced with RPMIu2009+u2009B-27 medium without insulin or CHIR99021. On days 3u20135, the cells were treated with 5u2009xB5M IWR1 (Sigma). On days 5u20136, the cells were removed from the medium containing IWR-1 and placed in RPMIu2009 +u2009B27 medium without insulin. From day 7 onwards, the cells were cultured in RPMIu2009+u2009B27 medium with insulin, and the medium was changed every two days. When replating for future use, iPSC-CMs were dissociated with 0.25% trypsin-EDTA into a single-cell suspension and seeded onto Matrigel-coated plates or micropatterned dished (4D cell). Then, the cells were washed with phosphate-buffered saline (PBS), lysed with protein extraction buffer (Cell Signaling), and frozen at u221280u2009xB0C. iPSC-CMs were treated with tubastatin A (SelleckChem) dissolved in DMSO at 3u2009xB5M for 24u2009h.\" - Mouse strains, husbandry, and treatments - \"Wild-type C57/Bl6, Lmna and Atat1 male mice were fed standard chow and water ad libitum and housed in a disease-free barrier facility under a 12u2009h/12u2009 h light/dark cycle, controlled temperature of 22u2009xB0C, and 55% humidity. All animal experiments were approved by the French Ministry of Higher Education and Research at the Center for Research in Myology for the Care and Use of Experimental Animals. The animal experiments were performed according to the guidelines from Directive 2010/63/EU of the European Parliament regarding the protection of animals used for scientific purposes. Tubastatin A (SelleckChem) dissolved in DMSO was delivered at a dose of 10u2009mg/kg daily to Lmna mice. Tubastatin A and placebo (DMSO) were administered to mice by intraperitoneal injection beginning at 2 months of age and continuing for one month.\" - Isolation of mouse cardiomyocytes - \"WT and Lmna mice were anesthetized via intraperitoneal injection of sodium pentobarbital (50u2009mg/kg), and ventricular cardiomyocytes were isolated as described previously. In brief, mouse hearts were removed and placed in ice-cold oxygenated Tyrode s solution. The aorta was cannulated above the aortic valve and perfused with Tyrodes solution containing 0.1u2009mmol/l EGTA for 2u2009min. An enzyme mixture containing 1u2009g/liter collagenase Type II (Worthington Biochemical Corporation) in Tyrodes solution supplemented with 0.1u2009mmol/l CaCl2 was then perfused until the aortic valve was digested. The heart was removed, cut into small pieces, and gently shaken in enzyme solution supplemented with 2u2009g/liter BSA for 2u2009min at 37u2009xB0C to disperse individual myocytes. The myocytes were then filtered through a 250-xB5m nylon mesh and centrifuged at 200u2009xD7u2009g for 2u2009min. A pellet containing myocytes was resuspended in Tyrodes solution supplemented with 0.5u2009mmol/l CaCl2 and 2u2009 g/liter BSA and then recentrifuged. The final pellet was resuspended in a storage buffer comprising Tyrodes solution supplemented with 1u2009mmol/l CaCl2. Isolated cardiomyocytes were used immediately, either protein were extracted for western blotting or cells were fixed for immunostaining.\" - Human heart tissue - \"Sections of explanted hearts from human subjects with LMNA mutations were obtained without identifiers from Myobank-AFM de llnstitut de Myologie. Myobank-AFM received approval from the French Ministry of Health and from the Committee for the Protection of Patients to share tissues and cells of human origin for scientific purposes, ensuring the donors anonymity, respect of their volition, and consent according to the legislation. The subjects were a 23-year-old man with cardiomyopathy associated with muscular dystrophy and LMNA p.delK261 mutation, a 53-year-old man with cardiomyopathy and LMNA p.E33D mutation, and a 47-year-old woman with cardiomyopathy and LMNA p.R60G mutation. Control human heart samples were obtained from a 57-year-old man with an intracranial bleed, a 15-year-old woman who died of a drug overdose, and a 46-year-old man who died from end-stage liver disease.\" - Protein extraction and immunoblotting - \"Total proteins were isolated from cultured cells, cardiac sections, or isolated cardiomyocytes with extraction buffer (Cell Signaling) containing protease inhibitors (25u2009mg/ml aprotinin, 10u2009mg/ml leupeptin, 1u2009mM 4-[2-aminoethyl]-benzene sulfonyl fluoride hydrochloride and 2u2009mM Na3VO4). The lysates were sonicated (3 pulses of 10u2009s at 30% amplitude). Cytosolic and nuclear fractions were prepared using NE-PER Nuclear and Cytosolic Extraction Reagents (Thermo Fisher Scientific) according to the manufacturers instructions. Sample protein content was determined by the bicinchoninic acid protein assay (Thermo Fisher Scientific). Protein extracts (10u201320u2009xB5g) were analyzed by SDSu2013 PAGE on a 10% gel and transferred onto nitrocellulose membranes (Invitrogen). After washing with Tris-buffered saline containing 1% Tween 20 (TBS-T), the membranes were blocked with 5% bovine serum albumin (BSA) in TBS-T for 1u2009 h at room temperature and then incubated with the appropriate antibody overnight at 4u2009xB0C. Membranes were incubated with fluorescent-conjugated anti-mouse or anti-rabbit secondary antibodies (BioRad) 1u2009h at RT. Antibody detection was imaged using the ChemiDoc Imaging System and ImageLab software (BioRad). Quantification was performed using FIJI/ImageJ software.\" - Immunofluorescence microscopy - \"For immunofluorescence microscopy, C2C12 and iPS-CM cells were grown on coverslips, washed with PBS, and fixed with 4% paraformaldehyde in PBS for 10u2009min. The cells were permeabilized with 0.2% Triton X-100 diluted in PBS for 10u2009 min, and nonspecific signals were blocked with 0.2% Triton X-100 and 5% BSA for 45u2009min. The samples were then incubated for 1u2009h with the primary antibody in PBS supplemented with 0.1% Triton X-100 and 1% BSA at room temperature. Cells and slides were then mounted in Vectashield mounting medium with 4,6-diamidino-2-phenylindole (DAPI) (Vector Laboratories).\" - \"Frozen cardiac tissues cut into 8u2009u03BCm-thick cryosections, were fixed (15u2009min, 4% paraformaldehyde in PBS at room temperature), permeabilized (10u2009min, 0.5% Triton X-100 in PBS), and blocked (1u2009h, PBS with 0.3% Triton X-100, 5% BSA). The sections were incubated with primary antibodies (in PBS with 0.1% Triton X-100 and 1% BSA) overnight at 4u2009xB0C and then washed with PBS. The sections were then incubated with the appropriate secondary antibodies for 1u2009h before being washed with PBS, and slides were mounted in Vectashield mounting medium with DAPI (Vector Laboratories).\" - \"Immunofluorescence microscopy was performed using an Axiophot microscope (Carl Zeiss) or a 3i Marianas spinning disk confocal with a Yokogawa CSU-W1 scanning unit on an inverted Zeiss AxioObserver Z1 microscope controlled by SlideBook 6 software (Intelligent Imaging Innovations GmbH, GxF6ttingen, Germany). A xD763/1.4 oil objective was utilized, and images were acquired with an ORCA Flash4 sCMOS camera (Hamamatsu Photonics, Hamamatsu, Japan). Most of the images were digitally deconvolved using Autodeblur v9.1 (Autoquant) deconvolution software and processed using Adobe Photoshop 6.0 (Adobe Systems). A Nikon Ti2 microscope equipped with a motorized stage and coupled with Prime 95B Scientific CMOS (sCMOS) camera at xD7100 with Oil Immersion objective was used for some of the acquisitions. The microscope was controlled by MetaMorph 7.10 software (Molecular Devices) with a pixel resolution of 0.11u2009xB5m/px at 16-bit. The images were processed by FIJI/ImageJ software.\" - F/G actin ratio measurements - \"The ratio of F-actin to G-actin was determined using the G-actin/F-actin in vivo assay kit (Cytoskeleton) according to the manufacturers instructions. Briefly, 2u2009mg of protein from cells or frozen heart tissues were homogenized in lysis and F-actin stabilization buffer and centrifuged at 350u2009xD7u2009 g for 5u2009min to remove unbroken cells. F-actin was separated from G-actin by centrifugation at 100,000u2009xD7u2009g for 60u2009min at 37u2009xB0 C. The F-actin-containing pellet was resuspended in F-actin depolymerizing buffer at a volume equivalent to the G-actin-containing supernatant volume. The resuspended F-actin pellet was kept on ice for 60u2009min and was gently mixed every 15u2009 min to dissociate F-actin. Proteins in equivalent volumes (10u2009u03BCl) of supernatant and pellet were separated by SDSu2013PAGE and subjected to immunoblot analysis using an anti-pan actin antibody supplied in the kit. F/G actin ratio was quantified using ImageJ software.\" - Proximity ligation analysis - \"The Duolink in situ proximity ligation assay (PLA) was performed according to the manufacturers protocol (Sigma-Aldrich). Briefly, C2-WT and C2-H222P cells were plated on glass coverslips. The cells were washed, blocked, and then incubated with anti-MRTF-A (1:25; Santa Cruz Biotechnology), anti-SRF (1:25; Santa Cruz Biotechnology), or anti-phospho(T25)-cofilin-1 (1:10; Genscript) primary antibodies overnight at 4u2009xB0C. After washing, the cells were incubated with the Duolink PLA probes MINUS and PLUS for 1u2009h at 37u2009 xB0C. A Duolink in situ detection kit was used for ligation and amplification, and nuclei were stained with Hoechst.\" - Immunoprecipitation - \"Cells were lysed in 0.5u2009ml of lysis buffer (50u2009mM Trisu2013HCl [pH 7.5], 0.15u2009M NaCl, 1u2009mM EDTA, 1% NP-40), precleared with 20u2009xB5 l of washed protein Gu2013Sepharose 4 fast-flow (GE Healthcare) and incubated with 20u2009xB5g of a specific antibody overnight at 4u2009xB0C. Next, the washed beads (30u2009xB5l) were added and incubated for 2u2009h at 4u2009 xB0C. Pelleted beads were collected in sample buffer (0.25u2009M Trisu2013 HCl [pH 7.5], 8% SDS, 40% glycerol, 20% u03B2-mercaptoethanol) and subjected to SDSu2013PAGE and immunoblotting. For control reactions, we used rabbit immunoglobulin G1.\" - Luciferase assays - \"The SRF reporter construct (3DA-luciferase: 3 CArGs upstream of the luciferase gene) was described previously. The pGL4.3.4[luc2P]/SRF-RE/Hygro vector (E1350; Promega) contains a sequence of five consensuses CArG boxes in tandem separated by nucleotide spacers followed by a minimal promoter upstream of a modified firefly luciferase gene. This vector was used to generate the plasmid ATAT1 CArGs1u20134 as described in ref. . The DNA constructs (50u2009ng) were transfected into C2C12 cells at 70% confluency together with 1.5u2009ng of a Renilla luciferase reporter plasmid, which was used to normalize for transfection efficiency, using Lipofectamine 2000. Forty-eight hours after transfection, the activities of firefly and Renilla luciferase were measured using a dual-luciferase reporter assay system (Promega) according to the manufacturers instructions on a TECAN SPARK 10u2009M microplate reader. SparkControl software was used to quantify luminescence. Each experiment was performed in triplicate.\" - RNA isolation and reverse-transcription qPCR - \"Total RNA was extracted from C2C12 or mouse heart using RNeasy isolation kit (Qiagen) according to the manufacturers instructions. The adequacy and integrity of the extracted RNA were determined with the 2100 Bioanalyzer system (Agilent) according to the manufacturers instructions. cDNA was synthesized using the SuperScript III first-strand synthesis system according to the manufacturer s instructions (Invitrogen). Real-time qPCR was performed with SYBR Green I Master mix (Roche) on the LightCycler 480 instrument and analyzed with LightCycler 480 software (Roche). The primers utilized in this study are listed in Table_ .\" - Adeno-associated virus (AAV) delivery - \"AAV delivery was performed in three-month-old mice by retro-orbital injection using a 30-G needle, at the dose of 5u2009xD7u200910 viral genomes/kg in 100u2009xB5l. Phosphate-buffered saline; PBS 1X was used as a placebo. Mice were anesthetized with an intraperitoneal injection of xylazine (10u2009mg/kg)/ketamine (100u2009mg/kg) cocktail and placed on a heating pad at 28u2009xB0C during the intervention. Wild-type males were injected with AAV-rh10-Cfl1, AAV-rh10-Cfl1(p.T25A) at 3 months of age. AAV vectors of serotype rh10 (AAV-rh10-Cfl1, AAV-rh10-Cfl1(p.T25A)), carrying sequence of the wild-type cofilin-1 gene or cofilin-1 c.73A>G under the control of the cytomegalovirus immediate/early promoter was prepared by the triple transfection method in HEK293T cells (ATCC #CRL-1573) as previously described.\" - Echocardiography - \"Mice were anesthetized with 0.75% isoflurane in O2 and placed on a heating pad (28u2009xB0C). Transthoracic echocardiography was performed using a Vivid 7 Dimension/Vivid7 PRO ultrasound with an 11u2009MHz transducer applied to the chest wall. Cardiac ventricular dimensions and fractional shortening were measured in 2D mode and M-mode three times for the number of animals indicated. A u2018blinded echocardiographer, unaware of the genotype and the treatment, performed the examinations.\" - Electrocardiography - Electrocardiograms were recorded from mice using the non-invasive ecgTUNNEL (Emka Technologies) with minimal filtering. Waveforms were recorded using Iox Software and intervals were measured manually with ECG Auto software. The electrocardiographer was blinded to the mouse genotype. - Microarray processing - \"We used GeneChip Mouse Genome 430 2.0 Arrays (Affymetrix), which contained 698,000 probes that covered 35,240 transcripts from the RefSeq database. RNA was extracted as described above and cDNA synthesis, cRNA synthesis, and labeling were performed as described in the Affymetrix GeneChip Expression Analysis Technical Manual. Hybridization, washing, staining, and scanning of the arrays were performed at the GeneChip Core Facility of Cochin Hospital (GENOMIC). Image files were obtained with Affymetrix GeneChip software and subjected to robust multichip analysis using Affymetrix microarray u201C.celu201D image files and GeneTraffic 3.0 software (Stratagene). The robust multichip analysis involved 3 steps: background correction, quantile normalization, and probe set summary. Genes were identified as differentially expressed if they showed at least a 2-fold difference in expression independent of the absolute signal intensity and met a false discovery rate threshold of qu2009<u20090.05 as determined by a 2-tailed Students t-test. Data generated have been deposited in the Gene Expression Omnibus database under accession code \" - Analysis of functional groups of genes - \"Gene expression changes related to functional groups were analyzed using the class score method in ermineJ (version 2.1.12; ) to provide statistical confidence values for the functional groupings. The algorithm utilizes the log-transformed 2-tailed Students t-test P values of genes that are members of a single Gene Ontology class as input and estimates the probability of the set of q values occurring by chance. Significant Gene Ontology terms were identified based on a false discovery rate of 0.05. For automated functional annotation and the classification of genes of interest based on Gene Ontology terms, we used the DAVID database ().\" - Three-dimensional C2C12 cultivation (spheroid model) - \"C2C12 cells were dissociated and resuspended in a culture medium at a concentration of 125,000 or 62,500 cells per ml. In total, 200u2009xB5l of the cell suspension was plated into each well of 96-well polystyrene round bottom plastic plates (Nunc) coated with a 1% agarose/water solution as previously described. The cells were cultured at 37u2009xB0C in a 5% CO2 atmosphere for 24u2009h and then fixed with 4% paraformaldehyde for immunocytochemistry.\" - Histological analysis - \"Frozen hearts were cut into 8-u03BCm-thick sections and stained with Sirius red. Briefly, heart sections were fixed in 4% formaldehyde for 10u2009min, rinsed in EtOH 100%, dried 20u2009min, and stained in 0.3% Sirius red solution for 1u2009h. Then sections were put in acetic acid 0.5% for 5u2009min twice, in EtOH 100% for 5u2009min three times, and finally in xylene for 10u2009 min twice. Collagen fibrils were detected in red whereas cytoplasm remained yellow.\" - MuscleMotion analysis - \"MUSCLEMOTION ImageJ macro was used as described in the users guide to quantify cardiomyocytes derived from iPS contractions.\" - Antibodies - \"The primary antibodies used were anti-cofilin1 (Cell Signaling), custom-made anti-phospho(T25)-cofilin-1 (GenScript, dilution 1:50), anti-phospho(Ser3)-cofilin-1 (Cell Signaling), anti-phosphoERK1/2 antibody (Cell Signaling), anti-ERK1/2 (Cell Signaling), anti-MRTF-A (Santa Cruz Biotechnology), anti-Cx43 (Abcam), anti-u03B1-tubulin (Abcam), anti-acetylated u03B1-tubulin (Sigma), anti-Flag (Sigma), anti-GAPDH (Abcam), anti-N-cadherin (Cell Signaling), anti-Cx43 (Abcam), anti-actin (Cytoskeleton), anti-cardiac troponin T (Cell Signaling), anti-HDAC6 (Cell signaling), and anti-u03B1-actinin sarcomeric (Sigma). The secondary antibodies for immunofluorescence were Alexa Fluor 488-conjugated goat anti-rabbit IgG, Alexa Fluor 568-conjugated goat anti-mouse IgG, and Alexa Fluor 488-conjugated donkey anti-goat IgG (Life Technologies). For immunoblotting, fluorescent-conjugated StarBright Blue 520 anti-mouse or StarBright Blue 700 anti-rabbit secondary antibodies (BioRad) were used. All the antibodies were used at dilutions recommended by the manufacturer.\" - Statistics - \"Statistical analyses were performed using GraphPad Prism software. Statistical significance between groups of mice subjected to echocardiography was analyzed with a corrected parametric test (Welchs t-test), with Pu2009<u2009 0.05 being considered significant. To validate the results of the echocardiographic analyses, we performed a nonparametric test (Wilcoxonu2013Mannu2013Whitney test). For all other experiments, either a two-tailed Students t-test or one-way ANOVA was used, with Pu2009<u20090.05 being considered significant. Values are presented as the meanu2009xB1u2009standard deviation (SD), and the sample sizes are indicated in the figure and table legends.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - \"All data generated or analyzed in this study are provided in the Source Data file/Supplementary Information file. The Affymetrix microarray data generated in this study have been deposited in the Gene Expression Omnibus database under accession code ._ are provided with this paper.\" - \\' \\'- - \"Fatkin, D. et al. Missense mutations in the rod domain of the lamin A/C gene as causes of dilated cardiomyopathy and conduction-system disease. N. Engl. J. Med. 341, 1715u20131724 (1999)._ _ _ Taylor, M. R. G. et al. Natural history of dilated cardiomyopathy due to lamin A/C gene mutations. J. Am. Coll. Cardiol. 41, 771u2013780 (2003)._ _ _ Muchir, A., Shan, J., Bonne, G., Lehnart, S. E. & Worman, H. J. Inhibition of extracellular signal-regulated kinase signaling to prevent cardiomyopathy caused by mutation in the gene encoding A-type lamins. Hum. Mol. Genet. 18, 241u2013247 (2009)._ _ _ Wu, W., Muchir, A., Shan, J., Bonne, G. & Worman, H. J. Mitogen-activated protein kinase inhibitors improve heart function and prevent fibrosis in cardiomyopathy caused by mutation in lamin A/C gene. Circulation 123, 53u201361 (2011)._ _ _ Wu, W., Iwata, S., Homma, S., Worman, H. J. & Muchir, A. Depletion of extracellular signal-regulated kinase 1 in mice with cardiomyopathy caused by lamin A/C gene mutation partially prevents pathology before isoenzyme activation. Hum. Mol. Genet. 23, 1u201311 (2014)._ _ Wu, W. et al. Macrocyclic MEK1/2 inhibitor with efficacy in a mouse model of cardiomyopathy caused by lamin A/C gene mutation. Bioorg. Med. Chem. 25, 1004u20131013 (2017)._ _ _ Chatzifrangkeskou, M. et al. Cofilin-1 phosphorylation catalyzed by ERK1/2 alters cardiac actin dynamics in dilated cardiomyopathy caused by lamin A/C gene mutation. Hum. Mol. Genet 27, 3060u20133078 (2018)._ _ _ Ho, C. Y., Jaalouk, D. E., Vartiainen, M. K. & Lammerding, J. Lamin A/C and emerin regulate MKL1-SRF activity by modulating actin dynamics. Nature 497, 507u2013511 (2013)._ _ _ _ Macquart, C. et al. Microtubule cytoskeleton regulates connexin 43 localization and cardiac conduction in cardiomyopathy caused by mutation in A-type lamins gene. Hum. Mol. Genet. 28, 4043u20134052 (2019)._ _ Borin, D. et al. Altered microtubule structure, hemichannel localization and beating activity in cardiomyocytes expressing pathologic nuclear lamin A/C. Heliyon 6, e03175 (2020)._ _ Dogterom, M. & Koenderink, G. H. Actin-microtubule crosstalk in cell biology. Nat. Rev. Mol. Cell Biol. 20, 38u201354 (2019)._ _ _ Clippinger, S. R. et al. Disrupted mechanobiology links the molecular and cellular phenotypes in familial dilated cardiomyopathy. Proc. Natl Acad. Sci. USA 116, 17831u2013 17840 (2019)._ _ _ _ Trembley, M. A. et al. Mechanosensitive gene regulation by myocardin-related transcription factors is required for cardiomyocyte integrity in load-induced ventricular hypertrophy. Circulation 138, 1864u20131878 (2018)._ _ _ Guo, Y. et al. Sarcomeres regulate murine cardiomyocyte maturation through MRTF-SRF signaling. Proc. Natl Acad. Sci. USA 118, e2008861118 (2021)._ _ _ Mseka, T., Bamburg, J. R. & Cramer, L. P. ADF/cofilin family proteins control formation of oriented actin-filament bundles in the cell body to trigger fibroblast polarization. J. Cell Sci. 120, 4332u20134344 (2007)._ _ _ Yang, N. et al. Cofilin phosphorylation by LIM-kinase 1 and its role in Rac-mediated actin reorganization. Nature 393, 809u2013812 (1998)._ _ _ _ Wang, D.-Z. et al. Potentiation of serum response factor activity by a family of myocardin-related transcription factors. Proc. Natl Acad. Sci. USA 99, 14855u201314860 (2002)._ _ _ _ Cen, B. et al. Megakaryoblastic leukemia 1, a potent transcriptional coactivator for serum response factor (SRF), is required for serum induction of SRF target genes. Mol. CellBiol. 23, 6597u20136608 (2003)._ _ Miralles, F., Posern, G., Zaromytidou, A.-I. & Treisman, R. Actin dynamics control SRF activity by regulation of its coactivator MAL. Cell 113, 329u2013342 (2003)._ _ _ Han, Z., Li, X., Wu, J. & Olson, E. N. A myocardin-related transcription factor regulates activity of serum response factor in Drosophila. Proc. Natl Acad. Sci. USA 101, 12567u201312572 (2004)._ _ _ _ Kuwahara, K., Barrientos, T., Pipes, G. C. T., Li, S. & Olson, E. N. Muscle-specific signaling mechanism that links actin dynamics to serum response factor. Mol. Cell. Biol. 25, 3173u20133181 (2005)._ _ _ Posern, G. & Treisman, R. Actin together: serum response factor, its cofactors and the link to signal transduction. Trends Cell Biol. 16, 588u2013596 (2006)._ _ _ Parlakian, Ara et al. Temporally controlled onset of dilated cardiomyopathy through disruption of the SRF gene in adult heart. Circulation 112, 2930u20132939 (2005)._ _ _ Olson, E. N. & Nordheim, A. Linking actin dynamics and gene transcription to drive cellular motile functions. Nat. Rev. Mol. Cell Biol. 11, 353u2013365 (2010)._ _ _ Grimes, K. M., Prasad, V. & McNamara, J. W. Supporting the heart: functions of the cardiomyocytes non-sarcomeric cytoskeleton. J. Mol. Cell. Cardiol. 131, 187u2013196 (2019)._ _ _ Prosser, B. L., Ward, C. W. & Lederer, W. J. X-ROS signaling: rapid mechano-chemo transduction in heart. Science 333, 1440u20131445 (2011)._ _ _ _ Prosser, B. L., Khairallah, R. J., Ziman, A. P., Ward, C. W. & Lederer, W. J. X-ROS signaling in the heart and skeletal muscle: stretch-dependent local ROS regulates [Ca]i. J. Mol. Cell. Cardiol. 58, 172u2013181 (2013)._ _ _ Verhey, K. J. & Gaertig, J. The tubulin code. Cell Cycle 6, 2152u20132160 (2007)._ _ _ Hammond, J. W., Cai, D. & Verhey, K. J. Tubulin modifications and their cellular functions. Curr. Opin. Cell Biol. 20, 71u201376 (2008)._ _ _ Magiera, M. M., Singh, P., Gadadhar, S. & Janke, C. Tubulin posttranslational modifications and emerging links to human disease. Cell 173, 1323u20131327 (2018)._ _ _ Janke, C. & Magiera, M. M. The tubulin code and its role in controlling microtubule properties and functions. Nat. Rev. Mol. Cell Biol. 21, 307u2013326 (2020)._ _ _ Kerr, J. P. et al. Detyrosinated microtubules modulate mechanotransduction in heart and skeletal muscle. Nat. Commun. 6, 8526 (2015)._ _ _ _ Robison, P. et al. Detyrosinated microtubules buckle and bear load in contracting cardiomyocytes. Science 352, aaf0659 (2016)._ _ Chen, C. Y. et al. Suppression of detyrosinated microtubules improves cardiomyocyte function in human heart failure. Nat. Med. 24, 1225u2013 1233 (2018)._ _ _ Li, T. et al. Histone deacetylase 6 in cancer. J. Hematol. Oncol. 11, 111 (2018)._ _ McMahon, D. K. et al. C2C12 cells: biophysical, biochemical, and immunocytochemical properties. Am. J. Physiol. 266, C1795u20131802 (1994)._ _ _ Zebedin, E. et al. C2C12 skeletal muscle cells adopt cardiac-like sodium current properties in a cardiac cell environment. Am. J. Physiol. Heart Circ. Physiol. 292, H439u2013450 (2007)._ _ _ Arimura, T. et al. Mouse model carrying H222P-Lmna mutation develops muscular dystrophy and dilated cardiomyopathy similar to human striated muscle laminopathies. Hum. Mol. Genet. 14, 155u2013169 (2005)._ _ _ Muchir, A. et al. Activation of MAPK pathways links LMNA mutations to cardiomyopathy in Emeryu2013Dreifuss muscular dystrophy. J. Clin. Investig. 117, 1282u2013 1293 (2007)._ _ _ Choi, J. C. et al. Dual specificity phosphatase 4 mediates cardiomyopathy caused by lamin A/C (LMNA) gene mutation. J. Biol. Chem. 287, 40513u201340524 (2012)._ _ _ Antoku, S. et al. ERK1/2 phosphorylation of FHOD connects signaling and nuclear positioning alternations in cardiac laminopathy. Dev. Cell 51, 602u2013 616.e12 (2019)._ _ _ Vignier, N. et al. Rescue of biosynthesis of nicotinamide adenine dinucleotide protects the heart in cardiomyopathy caused by lamin A/C gene mutation. Hum. Mol. Genet. 27, 3870u20133880 (2018)._ _ _ Vignier, N. et al. The non-muscle ADF/cofilin-1 controls sarcomeric actin filament integrity and force production in striated muscle laminopathies. Cell Rep. 36, 109601 (2021)._ _ _ Huang, T. Y., DerMardirossian, C. & Bokoch, G. M. Cofilin phosphatases and regulation of actin dynamics. Curr. Opin. Cell Biol. 18, 26u201331 (2006)._ _ _ Moriyama, K., Iida, K. & Yahara, I. Phosphorylation of Ser-3 of cofilin regulates its essential function on actin. Genes Cells 1, 73u201386 (1996)._ _ _ Balza, R. O. & Misra, R. P. Role of the serum response factor in regulating contractile apparatus gene expression and sarcomeric integrity in cardiomyocytes. J. Biol. Chem. 281, 6498u20136510 (2006)._ _ _ Vignier, N., Mougenot, N., Bonne, G. & Muchir, A. Effect of genetic background on the cardiac phenotype in a mouse model of Emeryu2013Dreifuss muscular dystrophy. Biochem. Biophys. Rep. 19, 100664 (2019)._ Sun, Q. et al. Defining the mammalian CArGome. Genome Res. 16, 197u2013207 (2006)._ _ _ FernxE1ndez-Barrera, J. et al. The actin-MRTF-SRF transcriptional circuit controls tubulin acetylation via u03B1-TAT1 gene expression. J. Cell Biol. 217, 929u2013944 (2018)._ _ Akella, J. S. et al. MEC-17 is an alpha-tubulin acetyltransferase. Nature 467, 218u2013222 (2010)._ _ _ _ Shida, T., Cueva, J. G., Xu, Z., Goodman, M. B. & Nachury, M. V. The major alpha-tubulin K40 acetyltransferase alphaTAT1 promotes rapid ciliogenesis and efficient mechanosensation. Proc. Natl Acad. Sci. USA 107, 21517u201321522 (2010)._ _ _ _ Hubbert, C. et al. HDAC6 is a microtubule-associated deacetylase. Nature 417, 455u2013458 (2002)._ _ _ _ Zhang, Y. et al. HDAC-6 interacts with and deacetylates tubulin and microtubules in vivo. EMBO J. 22, 1168u20131179 (2003)._ _ _ Kalebic, N. et al. u03B1TAT1 is the major u03B1-tubulin acetyltransferase in mice. Nat. Commun. 4, 1962 (2013)._ _ _ Butler, K. V. et al. Rational design and simple chemistry yield a superior, neuroprotective HDAC6 inhibitor, tubastatin A. J. Am. Chem. Soc. 132, 10842u201310846 (2010)._ _ _ Liu, J., Sun, N., Bruce, M. A., Wu, J. C. & Butte, M. J. Atomic force mechanobiology of pluripotent stem cell-derived cardiomyocytes. PLoS ONE 7, e37559 (2012)._ _ _ _ Burridge, P. W. et al. Multi-cellular interactions sustain long-term contractility of human pluripotent stem cell-derived cardiomyocytes. Am. J. Transl. Res. 6, 724u2013 735 (2014)._ Pesl, M. et al. Atomic force microscopy combined with human pluripotent stem cell derived cardiomyocytes for biomechanical sensing. Biosens. Bioelectron. 85, 751u2013757 (2016)._ _ _ Nikolova, V. et al. Defects in nuclear structure and function promote dilated cardiomyopathy in lamin A/C-deficient mice. J. Clin. Investig. 113, 357u2013369 (2004)._ _ _ Chandar, S. et al. Effects of mechanical stress and carvedilol in lamin A/C-deficient dilated cardiomyopathy. Circ. Res. 106, 573u2013582 (2010)._ _ _ Le Dour, C. et al. Decreased WNT/u03B2-catenin signalling contributes to the pathogenesis of dilated cardiomyopathy caused by mutations in the lamin a/C gene. Hum. Mol. Genet. 26, 333u2013343 (2017)._ Shaw, R. M. et al. Microtubule plus-end-tracking proteins target gap junctions directly from the cell interior to adherens junctions. Cell 128, 547u2013560 (2007)._ _ _ Smyth, J. W. et al. Limited forward trafficking of connexin 43 reduces cell-cell coupling in stressed human and mouse myocardium. J. Clin. Investig. 120, 266u2013279 (2010)._ _ _ Chkourko, H. S. et al. Remodeling of mechanical junctions and of microtubule-associated proteins accompany cardiac connexin43 lateralization. Heart Rhythm 9, 1133u2013 1140.e6 (2012)._ _ Reed, N. A. et al. Microtubule acetylation promotes kinesin-1 binding and transport. Curr. Biol. 16, 2166u20132172 (2006)._ _ _ Beyer, E. C., Paul, D. L. & Goodenough, D. A. Connexin43: a protein from rat heart homologous to a gap junction protein from liver. J. Cell Biol. 105, 2621u20132629 (1987)._ _ _ Revel, J. P. & Karnovsky, M. J. Hexagonal array of subunits in intercellular junctions of the mouse heart and liver. J. Cell Biol. 33, C7u2013C12 (1967)._ _ _ Yeager, M. & Gilula, N. B. Membrane topology and quaternary structure of cardiac gap junction ion channels. J. Mol. Biol. 223, 929u2013948 (1992)._ _ _ Waldeyer, C. et al. Regional, age-dependent, and genotype-dependent differences in ventricular action potential duration and activation time in 410 Langendorff-perfused mouse hearts. Basic Res. Cardiol. 104, 523u2013533 (2009)._ _ Barnabei, M. S., Palpant, N. J. & Metzger, J. M. Influence of genetic background on ex vivo and in vivo cardiac function in several commonly used inbred mouse strains. Physiol. Genom. 42A, 103u2013113 (2010)._ _ _ Jelinek, M., Wallach, C., Ehmke, H. & Schwoerer, A. P. Genetic background dominates the susceptibility to ventricular arrhythmias in a murine model of u03B2-adrenergic stimulation. Sci. Rep. 8, 2312 (2018)._ _ _ Bravo-Cordero, J. J., Magalhaes, M. A. O., Eddy, R. J., Hodgson, L. & Condeelis, J. Functions of cofilin in cell locomotion and invasion. Nat. Rev. Mol. Cell Biol. 14, 405u2013415 (2013)._ _ _ Kanellos, G. et al. ADF and Cofilin1 control actin stress fibers, nuclear integrity, and cell survival. Cell Rep. 13, 1949u20131964 (2015)._ _ _ Kanellos, G. & Frame, M. C. Cellular functions of the ADF/cofilin family at a glance. J. Cell Sci. 129, 3211u20133218 (2016)._ _ Arber, S. et al. Regulation of actin dynamics through phosphorylation of cofilin by LIM-kinase. Nature 393, 805u2013809 (1998)._ _ _ _ Toshima, J. et al. Cofilin phosphorylation by protein kinase testicular protein kinase 1 and its role in integrin-mediated actin reorganization and focal adhesion formation. Mol. Biol. Cell 12, 1131u2013 1145 (2001)._ _ _ Bamburg, J. R., McGough, A. & Ono, S. Putting a new twist on actin: ADF/cofilins modulate actin dynamics. Trends Cell Biol. 9, 364u2013370 (1999)._ _ _ Pollard, T. D. & Borisy, G. G. Cellular motility driven by assembly and disassembly of actin filaments. Cell 112, 453u2013465 (2003)._ _ _ Minamide, L. S., Striegl, A. M., Boyle, J. A., Meberg, P. J. & Bamburg, J. R. Neurodegenerative stimuli induce persistent ADF/cofilin-actin rods that disrupt distal neurite function. Nat. Cell Biol. 2, 628u2013636 (2000)._ _ _ Fulga, T. A. et al. Abnormal bundling and accumulation of F-actin mediates tau-induced neuronal degeneration in vivo. Nat. Cell Biol. 9, 139u2013148 (2007)._ _ _ Du, K. L. et al. Megakaryoblastic leukemia factor-1 transduces cytoskeletal signals and induces smooth muscle cell differentiation from undifferentiated embryonic stem cells. J. Biol. Chem. 279, 17578u201317586 (2004)._ _ _ Vartiainen, M. K., Guettler, S., Larijani, B. & Treisman, R. Nuclear actin regulates dynamic subcellular localization and activity of the SRF cofactor MAL. Science 316, 1749u20131752 (2007)._ _ _ _ Treisman, R. Identification and purification of a polypeptide that binds to the c-fos serum response element. EMBO J. 6, 2711u20132717 (1987)._ _ _ Miano, J. M., Long, X. & Fujiwara, K. Serum response factor: master regulator of the actin cytoskeleton and contractile apparatus. Am. J. Physiol. Cell Physiol. 292, C70u201381 (2007)._ _ _ Guo, Y. et al. Hierarchical and stage-specific regulation of murine cardiomyocyte maturation by serum response factor. Nat. Commun. 9, 3837 (2018)._ _ _ Niu, Z. et al. Conditional mutagenesis of the murine serum response factor gene blocks cardiogenesis and the transcription of downstream gene targets. J. Biol. Chem. 280, 32531u201332538 (2005)._ _ _ Dompierre, J. P. et al. Histone deacetylase 6 inhibition compensates for the transport deficit in Huntingtons disease by increasing tubulin acetylation. J. Neurosci. 27, 3571u20133583 (2007)._ _ _ Jongsma, H. J. & Wilders, R. Gap junctions in cardiovascular disease. Circ. Res. 86, 1193u20131197 (2000)._ _ _ Severs, N. J. et al. Gap junction alterations in human cardiac disease. Cardiovasc. Res. 62, 368u2013377 (2004)._ _ _ _ Fontes, M. S. C., van Veen, T. A. B., de Bakker, J. M. T. & van Rijen, H. V. M. Functional consequences of abnormal Cx43 expression in the heart. Biochim. Biophys. Acta 1818, 2020u20132029 (2012)._ _ _ Jordan, K. et al. Trafficking, assembly, and function of a connexin43-green fluorescent protein chimera in live mammalian cells. Mol. Biol. Cell 10, 2033u20132050 (1999)._ _ _ Giepmans, B. N. et al. Gap junction protein connexin-43 interacts directly with microtubules. Curr. Biol. 11, 1364u20131368 (2001)._ _ _ Howes, S. C., Alushin, G. M., Shida, T., Nachury, M. V. & Nogales, E. Effects of tubulin acetylation and tubulin acetyltransferase binding on microtubule structure. Mol. Biol. Cell 25, 257u2013266 (2014)._ _ Janke, C. & Montagnac, G. Causes and consequences of microtubule acetylation. Curr. Biol. 27, R1287u2013R1292 (2017)._ _ _ Choi, W. S. et al. Kv1.5 surface expression is modulated by retrograde trafficking of newly endocytosed channels by the dynein motor. Circ. Res. 97, 363u2013371 (2005)._ _ _ Loewen, M. E. et al. Shared requirement for dynein function and intact microtubule cytoskeleton for normal surface expression of cardiac potassium channels. Am. J. Physiol. Heart Circ. Physiol. 296, H71u2013 83 (2009)._ _ _ Zadeh, A. D. et al. Kif5b is an essential forward trafficking motor for the Kv1.5 cardiac potassium channel. J. Physiol. 587, 4565u20134574 (2009)._ _ _ McLendon, P. M. et al. Tubulin hyperacetylation is adaptive in cardiac proteotoxicity by promoting autophagy. Proc. Natl Acad. Sci. USA 111, E5178u20135186 (2014)._ _ _ Nekooki-Machida, Y. & Hagiwara, H. Role of tubulin acetylation in cellular functions and diseases. Med. Mol. Morphol. 53, 191u2013 197 (2020)._ _ _ Li, L. & Yang, X.-J. Tubulin acetylation: responsible enzymes, biological functions and human diseases. Cell. Mol. Life Sci. 72, 4237u20134255 (2015)._ _ _ Smith, A. S. T. et al. HDAC6 inhibition corrects electrophysiological and axonal transport deficits in a human stem cell-based model of Charcotu2013Marie-Tooth disease (Type 2D). Adv. Biol. 6, e2101308 (2022)._ _ Onishi, T. et al. A novel orally active HDAC6 inhibitor T-518 shows a therapeutic potential for Alzheimers disease and tauopathy in mice. Sci. Rep. 11, 15423 (2021)._ _ _ Bertels, Z. et al. Neuronal complexity is attenuated in preclinical models of migraine and restored by HDAC6 inhibition. Elife 10, e63076 (2021)._ _ _ Zheng, Y. et al. Inhibition of histone deacetylase 6 by Tubastatin A attenuates the progress of osteoarthritis via improving mitochondrial function. Am. J. Pathol. 190, 2376u20132386 (2020)._ _ _ Picci, C. et al. HDAC6 inhibition promotes u03B1 -tubulin acetylation and ameliorates CMT2A peripheral neuropathy in mice. Exp. Neurol. 328, 113281 (2020)._ _ _ Fischer, A., Sananbenesi, F., Mungenast, A. & Tsai, L.-H. Targeting the correct HDAC(s) to treat cognitive disorders. Trends Pharmacol. Sci. 31, 605u2013617 (2010)._ _ _ Kim, E. et al. Histone and non-histone targets of dietary deacetylase inhibitors. Curr. Top. Med. Chem. 16, 714u2013731 (2015)._ _ Boyault, C., Sadoul, K., Pabion, M. & Khochbin, S. HDAC6, at the crossroads between cytoskeleton and cell signaling by acetylation and ubiquitination. Oncogene 26, 5468u20135476 (2007)._ _ _ Pulya, S. et al. HDAC6 as privileged target in drug discovery: a perspective. Pharmacol. Res. 163, 105274 (2021)._ _ _ Chatzifrangkeskou, M., Kah, D., Lange, J. R., Goldmann, W. H. & Muchir, A. Mutated lamin A modulates stiffness in muscle cells. Biochem. Biophys. Res. Commun. 529, 861u2013867 (2020)._ _ _ Osseni, A. et al. Pharmacological inhibition of HDAC6 improves muscle phenotypes in dystrophin-deficient mice by downregulating TGF-u03B2 via Smad3 acetylation. Nat. Commun. 13, 7108 (2022)._ _ Chatzifrangkeskou, M. et al. ERK1/2 directly acts on CTGF/CCN2 expression to mediate myocardial fibrosis in cardiomyopathy caused by mutations in the lamin A/C gene. Hum. Mol. Genet. 25, 2220u20132233 (2016)._ _ _ Choi, J. C. et al. Temsirolimus activates autophagy and ameliorates cardiomyopathy caused by lamin A/C gene mutation. Sci. Transl. Med. 4, 144ra102 (2012)._ _ Shen, S. & Kozikowski, A. P. Why hydroxamates may not be the best histone deacetylase inhibitors-what some may have forgotten or would rather forget? Chem. Med. Chem. 11, 15u201321 (2016)._ _ _ Cosenza, M. & Pozzi, S. The therapeutic strategy of HDAC6 inhibitors in lymphoproliferative disease. Int. J. Mol. Sci. 19, E2337 (2018)._ _ Gollerkeri, A., Harrold, L., Rose, M., Jain, D. & Burtness, B. A. Use of paclitaxel in patients with pre-existing cardiomyopathy: a review of our experience. Int. J. Cancer 93, 139u2013141 (2001)._ _ _ Zhao, Y., Mu, X. & Du, G. Microtubule-stabilizing agents: new drug discovery and cancer therapy. Pharmacol. Ther. 162, 134u2013143 (2016)._ _ _ Shah, D. et al. Modeling of LMNA-related dilated cardiomyopathy using human induced pluripotent stem cells. Cells 8, 594 (2019)._ _ _ Ruiz-Hurtado, G. et al. Reconciling depressed Ca sparks occurrence with enhanced RyR2 activity in failing mice cardiomyocytes. J. Gen. Physiol. 146, 295u2013306 (2015)._ _ _ Biferi, M. G. et al. A new AAV10-U7-mediated gene therapy prolongs survival and restores function in an ALS mouse model. Mol. Ther. 25, 2038u20132052 (2017)._ _ _ Garzoni, L. R. et al. Fibrosis and hypertrophy induced by Trypanosoma cruzi in a three-dimensional cardiomyocyte-culture system. J. Infect. Dis. 197, 906u2013915 (2008)._ _ Sala, L. et al. MUSCLEMOTION: a versatile open software tool to quantify cardiomyocyte and cardiac muscle contraction in vitro and in vivo. Circ. Res. 122, e5u2013 e16 (2018)._ _ _ \"- - \"The work of A.M. was supported by the Institut National de la SantxE9 et de la Recherche MxE9dicale, Sorbonne UniversitxE9, and the Association FranxE7 aise contre les Myopathies, the French National Research Agency (ANR, award ANR-19-CE17-0013), MyoPharm and a grant (PER-SU) provided by Sorbonne UniversitxE9. C.J. received support from ANR-10-IDEX-0001-02, the LabEx CellNScale ANR-11-LBX-0038, the French National Research Agency (ANR, award ANR-17-CE13-0021), and the Fondation pour la Recherche Medicale (FRM, grant DEQ20170336756). P.T. was supported by research funding provided by the Academy of Finland. We thank the genomic facility at Institut Cochin, Paris, for Affymetrix data acquisition. We thank Dr. Ben Yaou (Institute of Myology Paris) and Dr. Eschenhagen (University Medical Center Hamburg-Eppendorf) for providing the iPSCs carrying the LMNA p.H222P mutation. We thank Dr. Dal Ferro (University of Trieste) for providing the iPSCs carrying the LMNA p.R190W mutation. We thank Dr. Alonso (Universidad AutxF3noma de Madrid) for providing the ATAT1 CArGs 1u20134 construct and Dr. Sadoul (UniversitxE9 Grenoble Alpes) for providing the ATAT knockout mouse strain. We thank Dr. Sotiropoulos (UniversitxE9 de Paris) for providing the SRF-VP16 plasmid. We thank Zoheir Guesmia for his help with imaging and quantification. Parts of Figs._ and were drawn using pictures from Servier Medical Art. Servier Medical Art by Servier is licensed under a Creative Commons Attribution 3.0 Unported License. Fig. was created thanks to BioRender.com.\"- - Author notes - \\'These authors contributed equally: Caroline Le Dour, Maria Chatzifrangkeskou, Coline Macquart.\\' - Authors and Affiliations - \"Centre de recherche en Myologie, U974 SU-INSERM, 75013, Paris, FranceCaroline Le Dour,_Maria Chatzifrangkeskou,_Coline Macquart,_CxE9cile Peccate,_Bruno Cadot,_DxE9borah Cardoso_&_Antoine MuchirInstitut Curie, UniversitxE9 PSL, CNRS UMR3348, 91401, Orsay, FranceMaria M. Magiera_&_Carsten JankeUniversitxE9 Paris-Saclay, CNRS UMR3348, 91401, Orsay, FranceMaria M. Magiera_&_Carsten JankeUniversitxE9 de Paris, Paris Cardiovascular Research Center PARCC, INSERM, 75015, Paris, FranceCharlxE8ne Jouve_&_Jean-SxE9bastien HulotInstitute of Biomedicine and FICAN West Cancer Centre, University of Turku, 20520, Turku, FinlandLaura Virtanen_&_Pekka TaimenHeart and Lung Centre, Helsinki University Central Hospital, University of Helsinki, 00290, Helsinki, FinlandTiina HelixF6 Faculty of Medicine and Health Technology and BioMediTech Institute, Tampere University, 33520, Tampere, FinlandKatriina Aalto-SetxE4lxE4Institute of Genetic and Biomedical Research (IRGB), UOS of Milan, National Research Council of Italy, Milan, ItalySilvia Crasto_&_Elisa Di PasqualeHumanitas Clinical and Research Center-IRCCS, 20089, Rozzano (MI), ItalySilvia Crasto_&_Elisa Di PasqualeSorbonne UniversitxE9 , INSERM, UMS28 PhxE9notypage du petit animal, Paris, 75013, FranceNathalie MougenotFundaxE7 xE3o Oswaldo Cruz, Instituto Oswaldo Cruz, LaboratxF3rio de Biologia Estrutural, Rio de Janeiro, BrasilDaniel AdesseDepartment of Pathology, Turku University Hospital, 20520, Turku, FinlandPekka Taimen\" - \"AuthorsCaroline Le DourYou can also search for this author in _Maria ChatzifrangkeskouYou can also search for this author in _Coline MacquartYou can also search for this author in _Maria M. MagieraYou can also search for this author in _CxE9cile PeccateYou can also search for this author in _CharlxE8ne JouveYou can also search for this author in _Laura VirtanenYou can also search for this author in _Tiina HelixF6You can also search for this author in _Katriina Aalto-SetxE4lxE4You can also search for this author in _Silvia CrastoYou can also search for this author in _Bruno CadotYou can also search for this author in _DxE9borah CardosoYou can also search for this author in _Nathalie MougenotYou can also search for this author in _Daniel AdesseYou can also search for this author in _Elisa Di PasqualeYou can also search for this author in _Jean-SxE9bastien HulotYou can also search for this author in _Pekka TaimenYou can also search for this author in _Carsten JankeYou can also search for this author in _Antoine MuchirYou can also search for this author in _\" - Contributions - \"Conceptualization, A.M.; investigation, C.L.D., M.C., C.M., C.P., B.C., D.C., and D.A.; ATAT1 mouse model, M.M.M. and C.J.; iPS cells, C.J., L.V., T.H., K.A.-S., S.C., E.D.P., P.T., and J.-S.H.; cardiac function assessment, N.M. and A.M.; writingu2014original draft, A.M.; writingu2014review and editing, C.L.D., P.T., M.C., and C.J.; funding acquisition, A.M.; supervision, A.M.\" - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \"Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work._ are available.\" - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License ().\" - \"Next, we assessed the in vivo tumour-initiating potential of limiting dilutions (500,000, 50,000 and 5,000 cells) of A375M2 and A375P cells subcutaneously injected into immunodeficient NOD/SCID/IL2Ru03B3u2212/u2212 (NSG) mice and using extreme limiting dilution analysis (ELDA). Amoeboid A375M2 cells were more efficient in tumour initiation, with a significant difference in tumour-initiating frequency (TIF) (Fig._), and showed increased tumour growth in all conditions compared to A375P cells (Fig._). Enrichment in rounded cells (Fig._) and Myosin II activity, as measured by phosphorylated MLC2 (p-MLC2) levels (Fig._), were observed at the invasive front (IF) of A375M2 tumours compared with tumour body (TB), while decreased cell rounding and Myosin II levels were found in A375P tumours (Fig._). Importantly, we also observed an increase in amoeboid features in IFs of A375P tumours compared to TBs, although less pronounced (Fig._). To further investigate the heterogeneity of Myosin II levels within the tumours, Myosin II activity was scored from 0 (low) to 3 (very high) based on p-MLC2 intensity. A375M2 tumours showed an increase in cells with very high Myosin II in the IF (Supplementary Fig._). High ki-67 levels have been associated with the aggressiveness of cutaneous melanoma. Although no differences were observed in cell numbers in vitro after 7 days in culture (Supplementary Fig._ ), A375M2 tumours showed a higher proliferation index in vivo, as evidenced by ki-67 staining (Supplementary Fig._). Interestingly, IFs of all tumours were enriched in ki-67 proliferative cells. These data suggest that amoeboid cells with intrinsically high Myosin II activity are also proliferative and promote tumour initiation in vivo.\" - \"We next investigated in vitro self-renewal capacity of melanoma cells in low adherent conditions. We introduced another pair of melanoma cell lines, WM983B (metastatic, rounded-amoeboid and high Myosin II) and WM983A (primary tumour, elongated and low Myosin II) derived from the same patient. Using these two models, we performed serial sphere passages of elongated melanoma cells with low levels of Myosin II (A375P and WM983A) (Supplementary Fig._). Serial passaging resulted in cells with increased melanosphere formation abilities over time (Fig._). Although tumour-initiating cells are described to be in a slow proliferative state, sub-populations of proliferating stem cells have also been found in some tumours. Immunohistochemical analysis of melanospheres revealed an increase in Myosin II activity (Fig._) and higher percentage of_ki-67 positive cells (Fig._) with increasing passage number. Morphology of cells from adherent conditions and of dissociated single cells from serially passaged spheres was also assessed on collagen I matrices to recapitulate the dermal environment. Importantly, serial passages resulted in an enrichment of rounded cells (Fig._) with high Myosin II levels (Fig._) and increased blebbing (Fig._). Increased self-renewal ability was, therefore, associated with increased amoeboid features. Although the enrichments were less pronounced, similar results were obtained when serial sphere passages were performed in cells in an already amoeboid phenotype (A375M2 and WM983B) (Supplementary Fig._). Moreover, MLC2-GFP was transduced into WM983A cells (Supplementary Fig._) and this induced increased melanosphere formation, increased cell rounding and increased Myosin II activity (Supplementary Fig._ ).\" - Overall, these data show that amoeboid cells are more tumourigenic and sustain self-renewal and tumour initiation in melanoma. - EMT genes regulated by ROCK1/2 control amoeboid invasive features - \"Melanoma is a non-epithelial tumour, hence acquisition of invasive features is not considered a canonical EMT. Nevertheless, EMT gene expression has been associated with the acquisition of stem cell-like properties. ssGSEA analysis in our signature for amoeboid melanoma cells revealed that amoeboid A375M2 cells were enriched in both EMT and metastasis-related gene_signatures (Fig._). Of note, we also found that amoeboid cells express genes from NC and invasive signatures associated with drug resistance (Figs._a, ). The generated amoeboid transcriptomes were obtained after 24u2009h of ROCK1/2-Myosin II inhibition. To assess more direct effects of ROCK1/2 inhibition, we performed an EMT-directed qPCR array in A375M2 and WM1361 melanoma cells (BRAF or NRAS-driven melanoma, respectively, to account for the main melanoma oncogenic drivers) treated with ROCKi for only 4u2009 h. At this time point, loss of cell rounding and decreased Myosin II activity were already observed (Fig._). Interestingly, ROCKi decreased expression of 72% and 59% of EMT-related genes in A375M2 and WM1361 cells, respectively (Fig._ and Supplementary Fig._). ROCKi treatment_resulted in significantly reduced expression of key EMT regulators, such as SNAI family members and the mesenchymal markers N-cadherin, Vimentin and Fibronectin (Fig._). These data show that ROCK1/2 are positive regulators of EMT-related gene expression, independently of melanoma oncogenic background.\" - \"Fig. 2: EMT genes regulated by ROCK1/2 control amoeboid invasive features.a Heatmap displaying enrichment scores for differentially expressed EMT and metastasis-related signatures in amoeboid A375M2 cells compared to A375P cells or to A375M2 cells treated with ROCKi (H1152 and Y27632) or blebbistatin using ssGSEA. b Representative phase-contrast images of A375M2 cells on top of collagen I matrix (left) and immunoblots of p-MLC2 in A375M2 and WM1361 cells (right) after 4u2009h treatment with ROCKi (H1152) (nu2009=u20093). Scale bar, 50u2009u03BCm. c Fold regulation of EMT-related gene expression from EMT-directed qPCR array in A375M2 and WM1361 cells treated with ROCKi (H1152) for 4u2009h (nu2009=u20094). d Representative confocal images of p-MLC2 and F-actin staining in WM1361 cells on collagen I matrix after WNT11 knockdown (nu2009=u20093). Scale bar, 20u2009u03BCm. e Quantification of cell morphology (>280 cells pooled from nu2009=u20093) and f p-MLC2 immunofluorescence signal normalized by cell area (>85 cells pooled from nu2009=u20093) in WM1361 cells on collagen I matrix after depletion of indicated genes. g Representative confocal images (left) and quantification (right) of adhesion of WM1361 cells to a monolayer of keratinocytes after depletion of indicated genes (nu2009=u20095 for SERPINE1, nu2009=u20094 for TCF4, nu2009=u20093 for WNT11, WNT5B, AHNAK and CAV2). Scale bar, 20u2009u03BC m. h Representative confocal images (left) and quantification (right) of 3D invasion index through a collagen I matrix of WM1361 cells after depletion of indicated genes (nu2009=u20093). Scale bar, 50u2009u03BCm. i Summary table of amoeboid functional assays on WM1361 cells after SERPINE1, WNT11, WNT5B, AHNAK, TCF4 and CAV2 knockdown. u201C+u201D sign indicates a significant phenotype of the corresponding gene knockdown. c, g, h Graphs show meanu2009xB1u2009s.e.m. e, f Box limits show 25th and 75th percentiles, the horizontal line shows the median, and whiskers show minimum and maximum range of values. bu2013h n means number of independent biological experiments. c Two-tailed t-test with Benjamini, Krieger and Yekutieli correction for multiple comparisons. e, f Kruskalu2013 Wallis test with Benjamini, Krieger and Yekutieli correction. g, h One-way ANOVA with Benjamini, Krieger and Yekutieli correction. For all graphs, *pu2009<u2009 0.05, **pu2009<u20090.01, ***pu2009<u20090.001, ****pu2009<u20090.0001. The exact significant p values for *p, **p and ***p are provided in Supplementary Table_.\" - \"To identify key signalling pathways associated with this gene signature, network enrichment analysis was performed. The most significantly enriched network was centred on Wnt and TGFu03B2 signalling, while the second one encompassed NF-u03BA B and STAT3 signalling (Supplementary Fig._). We selected genes from these top networks with no prior link to amoeboid migration (WNT11, WNT5B, AHNAK, TCF4 and CAV2) and investigated if they could regulate amoeboid features. As positive control, SERPINE1 was selected as a gene from our top network previously described to support amoeboid behaviour. Amoeboid cells are characterised by rounded morphology and high Myosin II activity, so we first measured these two features on WM1361 cells grown on collagen I matrices. Reducing expression of SERPINE1, WNT11, WNT5B, AHNAK and TCF4 led to both decreased cell rounding and decreased Myosin II activity (Fig._). During melanoma progression, melanoma cells detach from the surrounding keratinocytes in order to invade the dermis. Amoeboid melanoma cells adhere less to keratinocytes and migrate more efficiently through collagen I compared with elongated cells with lower Myosin II. We found that SERPINE1, WNT11, WNT5B and CAV2 depletion increased adhesion of WM1361 cells to keratinocytes (Fig._), while depletion of SERPINE1, WNT11, WNT5B and AHNAK reduced 3D invasion through collagen I matrices (Fig._). In summary, WNT11, WNT5B and SERPINE1 (our positive control) had an impact in all assays (Fig._). We next validated the role of non-canonical Wnt ligands in regulating amoeboid features found in our screens. Using three different RNAi to reduce WNT11 expression, we measured decreased cell rounding, Myosin II activity and reduced invasion (Supplementary Fig._) in both A375M2 and WM1361 cells. Similar results were obtained after WNT5B knockdown (Supplementary Fig._).\" - These results show that non-canonical Wnt ligands support melanoma amoeboid invasion independently of oncogenic background. - Non-canonical Wnt ligands support melanosphere formation and amoeboid behaviour - \"Non-canonical Wnt ligands are regulated by ROCK1/2 and in return these ligands control amoeboid features, generating a positive feedback loop (Fig._). ROCK1/2-Myosin II has multiple cellular functions including cell proliferation, migration or invasion, while EMT has been associated with increased stem cell-like properties. We first explored whether ROCK1/2-Myosin II sustains in vitro tumour initiation by blocking the entire transcriptional programme using different ROCKi (H1152 and GSK269962A). Expression of stem cell-related genes (Fig._ and Supplementary Fig._), cell rounding and p-MLC2 levels (Supplementary Fig._) were decreased after treatment. Interestingly, using several amoeboid cell lines, one single treatment with ROCKi impaired melanosphere formation (Fig._ and Supplementary Fig._). Of note, in vitro 2D cell viability was reduced after 7 days (Fig._ and Supplementary Fig._). Similar results were obtained blocking Myosin II directly with blebbistatin (Fig._). These effects were further confirmed to be ROCK1/2-Myosin II dependent, since specific silencing of both ROCK genes (ROCK1/2) or both MLC2 genes (MYL9 and MYL12B) reduced melanosphere formation in A375M2 and WM1361 cells (Fig._ and Supplementary Fig._). 2D cell viability was decreased after 7 days (Fig._), while no effects were measured after 3 days (Fig._). These data show that we can transiently reduce Myosin II levels without affecting melanoma cell viability but Myosin II long-term depletion compromises cell survival.\" - \"Fig. 3: Non-canonical Wnt ligands support melanosphere formation and amoeboid behaviour.a mRNA expression of stem cell-related markers by qRT-PCR in A375M2 cells treated with ROCKi (GSK269962A) for 24u2009h compared to control A375M2 cells (nu2009=u20094 for ALCAM, ALDH1A3, ALDH1A1; nu2009=u20093 for CD44, OCT4, JARID1B, SOX2). b Schematic of treatment (top), representative phase-contrast images (bottom left) and quantification of sphere formation index (bottom right) and c cell viability of A375M2 cells and A375M2 cells treated with one dose of ROCKi (H1152 or GSK269962A) or blebbistatin (nu2009=u20093). Scale bar, 250u2009u03BCm. du2013f After ROCK1/2, MYL9 or MYL12B knockdown in A375M2 and WM1361 cells, quantification of d sphere formation index (nu2009=u2009 4 for A375M2, nu2009=u20093 for WM1361) and e, f cell viability after_e 7 days (nu2009=u20093 for A375M2, nu2009=u20094 for WM1361) or f 3 days (nu2009 =u20093) of_cell seeding. gu2013j After WNT11 or WNT5B knockdown in A375M2 and WM1361 cells, g, h representative phase-contrast images (left) and quantification of sphere formation index (right) (nu2009=u20093) and i, j cell viability (nu2009=u20094 for A375M2, nu2009=u20093 for WM1361). Scale bar, 250u2009 u03BCm. k Representative phase-contrast images (left) and quantification of cell morphology (right) of A375P cells after 24u2009h of WNT11 or WNT5B stimulation (>225 cells pooled from nu2009=u20093). Scale bar, 100u2009u03BCm. au2013 j Graphs show meanu2009xB1u2009s.e.m. k Box limits show 25th and 75th percentiles, the horizontal line shows the median, and whiskers show minimum and maximum range of values. au2013k n means number of independent biological experiments. au2013c, h, j Two-tailed t-test. du2013g, i One-way ANOVA with Dunnett post-hoc test. k Kruskalu2013Wallis with Dunns multiple comparison test. For all graphs, *pu2009<u20090.05, **pu2009<u20090.01, ***pu2009<u20090.001, ****pu2009 <u20090.0001. The exact significant p values for *p, **p and ***p are provided in Supplementary Table_.\" - \"We, therefore, next explored whether ROCK1/2-Myosin II sustains tumour initiation via regulation of Wnt ligands and independently of other functions. Interestingly, reducing levels of WNT11 and WNT5B resulted in reduced melanosphere formation (Fig._), while in vitro 2D cell viability was unaffected after 7 days (Fig._ ). These data suggest that Wnt ligands support both amoeboid features and specifically in vitro melanosphere formation. On the other hand, Wnt signals can be produced by tumour cells as well as by stromal cells in the tumour microenvironment. Elongated A375P cells treated with either WNT11 or WNT5B ligands increased cell rounding (Fig._). These results indicate that melanoma cells are capable of regulating amoeboid features in response to non-canonical Wnt ligands in an autocrine and paracrine manner.\" - FZD7 downstream of WNT11 supports melanosphere formation and amoeboid invasion via DAAM1 - \"Non-canonical Wnt ligands can signal through different Fzd receptors and co-receptors. Using our microarray data for amoeboid melanoma cells, we found that most of the receptors for WNT11 and WNT5B were overexpressed in amoeboid A375M2 cells (Fig._). Since amoeboid cell content is enriched in advanced stages of melanoma progression, we used publicly available data to investigate receptor expression for non-canonical Wnt ligands in primary and metastatic human melanomas (Fig._ ). In particular, WNT11-cognate receptor FZD7 and its co-receptor RYK were both consistently upregulated in metastatic melanomas in most of the studies. Importantly, FZD7 silencing resulted in the strongest reduction of melanosphere formation (Fig._), while depletion of either FZD7 or RYK did not affect in vitro 2D cell viability (Fig._). FZD7 silencing also resulted in loss of cell rounding (Fig._ ) and decreased Myosin II activity (Fig._ and Supplementary Fig._). Moreover, FZD7 depletion impaired invasion through 3D collagen I matrix (Fig._). FZD7 effects were further validated using two different shRNAs (Supplementary Fig._ ). These data suggest that FZD7 is an important regulator of amoeboid cell invasion and melanosphere formation.\" - \"Fig. 4: FZD7 downstream of WNT11 supports melanosphere formation and amoeboid invasion via DAAM1.a, b Heatmap representing log2 fold change in expression of non-canonical Wnt receptors in a A375M2 cells compared to A375P cells or A375M2 cells treated with ROCKi (H1152 and Y27632) and blebbistatin and in b metastatic versus primary melanoma samples from indicated studies from GEO and TCGA databases. cu2013f After FZD7 or RYK knockdown in A375M2 and WM1361 cells, quantification of c sphere formation index (nu2009=u20094 for A375M2, nu2009 =u20093 for WM1361), d cell viability (nu2009=u20095 for A375M2, nu2009=u2009 3 for WM1361), e cell morphology (>300 cells pooled from nu2009=u20093) and f p-MLC2 levels (nu2009=u20093 for A375M2, nu2009=u20094 for WM1361). g Quantification of 3D invasion index through a collagen I matrix after FZD7 knockdown in A375M2 (nu2009=u20095) and WM1361 cells (nu2009=u20093). hu2013j After PLCB1 or DAAM1 knockdown in A375M2 and WM1361 cells, quantification of h cell morphology (>250 cells pooled from nu2009=u20093), i sphere formation index (nu2009=u20094) and j cell viability (nu2009=u20096 for A375M2, nu2009 =u20093 for WM1361). k Quantification of 3D invasion index through a collagen I matrix after DAAM1 knockdown in A375M2 (nu2009=u20094) and WM1361 cells (nu2009=u20093). l Representative phase-contrast images (left) and quantification of cell morphology (right) of A375P cells on collagen I matrix after WNT11 stimulation and PLCB1 or DAAM1 knockdown (>400 cells pooled from nu2009=u20095). Scale bar, 100u2009u03BCm. m, n Representative immunoblots (top) and quantification (bottom) of m RhoA-GTP in pulldown samples and total RhoA in total lysate (nu2009 =u20093) and n p-MLC2 levels (nu2009=u20094) in A375P cells after WNT11 stimulation and DAAM1 knockdown. c, d, f, g, iu2013k, m, n Graphs show meanu2009xB1u2009 s.e.m. e, h, l Box limits show 25th and 75th percentiles, the horizontal line shows the median, and whiskers show minimum and maximum range of values. cu2013 n n means number of independent biological experiments. a, g, k Two-tailed t-test. b Two-tailed t-test with Welchs correction. c, d, f, i, j One-way ANOVA with Dunnett post-hoc test. e, h Kruskalu2013Wallis with Dunns multiple comparison test. l Kruskalu2013Wallis test with Benjamini, Krieger and Yekutieli correction. m, n One-way ANOVA with Tukey post-hoc test. For all graphs, *pu2009 <u20090.05, **pu2009<u20090.01, ***pu2009<u20090.001, ****pu2009<u2009 0.0001. The exact significant p values for *p, **p and ***p are provided in Supplementary Table_.\" - \"After binding to their cognate receptors, non-canonical Wnt ligands can signal through two key molecular platforms: Dvl-PLCB mediated calcium signalling and Dvl-DAAM1-Rho signalling. To establish specificity, we investigated the effects of PLCB1 and DAAM1 depletion. PLCB1 was chosen among its different isoforms as it is upregulated in melanoma metastasis (Supplementary Fig._). Either PLCB1 or DAAM1 silencing decreased basal levels of cell rounding and p-MLC2 (Fig._ and Supplementary Fig._), indicating that both pathways are able to control basal Myosin II activity. However, only DAAM1 but not PCLB1 depletion impaired melanosphere formation in both A375M2 and WM1361 cells (Fig._) without affecting 2D cell viability (Fig._). In addition, DAAM1 silencing decreased invasion through 3D collagen I matrix (Fig._). DAAM1 effects were further validated using two different shRNAs (Supplementary Fig._). Moreover, expression of stem cell-related markers was decreased in shDAAM1 cells (Supplementary Fig._). These results suggest an important role of DAAM1 controlling amoeboid invasion and tumour-initiating potential.\" - \"We have shown that WNT11 can promote amoeboid features in melanoma cells in a paracrine manner (Fig._). Crucially, we found that the increase in cell rounding measured after WNT11 stimulation was no longer observed in DAAM1 depleted cells, while PLCB1 depleted cells still responded to the ligand (Fig._ and Supplementary Fig._). Furthermore, WNT11 stimulated RhoA activity and downstream Myosin II levels in melanoma cells, but this effect was lost when DAAM1 was silenced (Fig._ ).\" - \"Altogether, these results show that WNT11/5B-FZD7-DAAM1 pathway control melanosphere formation_and_amoeboid invasive behaviour by activation of a specific pool of RhoA-ROCK1/2-Myosin II.\" - FZD7-DAAM1-RhoA-ROCK1/2 supports tumour initiation and metastasis in vivo - \"We next tested whether the FZD7-DAAM1-RhoA-ROCK1/2-Myosin II axis might affect melanoma in vivo. First, A375M2 cells were pre-treated with different ROCKi (H1152 and GSK269962A) ex vivo for 5 days. Relative number of cells were reduced (Supplementary Fig._). Nevertheless, only viable cells were subsequently injected subcutaneously into immunodeficient NSG mice in equal numbers, without any further treatment of animals. 18 days post-injection, ROCKi pre-treated A375M2-derived tumours showed over 60% reduction in tumour weight (Fig._). Importantly, even if these tumours had never received any treatment in vivo, we measured reduced cell rounding, p-MLC2 levels and ki-67 positive cells in ROCKi pre-treated tumours (Fig._). Similarly, ROCKi pre-treated B16F10 melanoma cells (Supplementary Fig._) injected subcutaneously into immunocompetent C57BL/6J mice showed reduced tumour growth (Supplementary Fig._) and decreased amoeboid cell number in ROCKi pre-treated group (Supplementary Fig._). These results show that ROCK1/2 supports the intrinsic abilities of both human and mouse melanoma cells to form tumours by supporting amoeboid features.\" - \"Fig. 5: FZD7-DAAM1-RhoA-ROCK1/2 supports tumour initiation and metastasis in vivo.a Schematic of experiment (left) and tumour weight (right) of ROCKi (H1152 or GSK269962A) pre-treated A375M2 cells 18 days post-subcutaneous injection into NSG mice (nu2009=u20095 mice for control and H1152, nu2009=u20094 for GSK269962A). bu2013d Quantification of b melanoma cell shape score, c H-score of p-MLC2 staining and d ki-67 positive cells in A375M2 tumours from (a). e Limiting dilution assay estimating TIF (top) and tumour volume (bottom) of shControl and shFZD7 WM1361 cells when injected at different dilutions (500,000, 50,000 and 5,000 cells) into NSG mice (Number of tumours per condition indicated in table). TIF was determined using ELDA. fu2013h Representative images (top) and quantification (bottom) of f melanoma cell shape score, g H-score of p-MLC2 staining and h ki-67 positive cells in TB and IF of shControl (nu2009=u2009 16) and shFZD7 (nu2009=u20099) derived tumours from 50,000 cells condition from (e). Scale bar, 100u2009u03BCm; inset, 25u2009u03BCm. i Limiting dilution assay estimating TIF (top) and tumour volume (bottom) of shControl and shDAAM1 WM1361 cells when injected at different dilutions (500,000, 50,000 and 5,000 cells) into NSG mice (Number of tumours per condition indicated in table). TIF was determined using ELDA. ju2013l Representative images (top) and quantification (bottom) of j melanoma cell shape score, k H-score of p-MLC2 staining and l ki-67 positive cells in TB and IF of shControl (nu2009=u200910) and shDAAM1 (nu2009=u20095) derived tumours from 50,000 cells condition from (i). Scale bar, 100u2009u03BCm; inset, 25u2009u03BCm. m Representative confocal images (left) and percentage of fluorescence area (right) of mouse lungs 24u2009 h (nu2009=u20096 mice) and 3 weeks (nu2009=u20095 mice) after tail vein injection into NSG mice of WM1361 cells expressing shControl and shDAAM1. Scale bar, 50u2009 u03BCm. a, m Graphs show meanu2009xB1u2009s.e.m. bu2013l Box limits show 25th and 75th percentiles, the horizontal line shows the median, and whiskers show minimum and maximum range of values. au2013d One-way ANOVA with Dunnett post-hoc test. e, i Two-tailed Mannu2013Whitney test. fu2013h, ju2013l One-way ANOVA with Tukey post-hoc test. m Two-tailed t-test. For all graphs, *pu2009 <u20090.05, **pu2009<u20090.01, ***pu2009<u20090.001, ****pu2009<u2009 0.0001. The exact significant p values for *p, **p and ***p are provided in Supplementary Table_. Schematics in this figure were created using Servier Medical Art templates licensed under a Creative Commons Attribution 3.0 Unported License ().\" - \"To examine the functional role FZD7 and DAAM1 might play in tumour initiation in vivo, limiting dilutions of either shFZD7 or shDAAM1 WM1361 cells were injected subcutaneously into NSG mice. Reduced FZD7 levels (Supplementary Fig._) led to a decrease in tumour growth when 50,000 and 5,000 cells were injected, while no changes were observed with a saturating concentration of 500,000 cells (Fig._ ). Importantly, there was a significant reduction in TIF using ELDA (Fig._). An enrichment in rounded cells (Fig._) and p-MLC2 levels (Fig._ and Supplementary 5i) was observed at the IF of shControl tumours compared with TBs, while loss of cell rounding and reduced Myosin II activity were measured in shFZD7 derived tumours (Fig._). shFZD7 tumours showed a decrease in cells with very high Myosin II in the IF (Supplementary Fig._). Although there were no differences in cell numbers in vitro, shFZD7 tumours in vivo displayed a reduction of the fraction of cells positive for ki-67 proliferative marker (Fig._). These data demonstrate that FZD7 loss results in a reduction of the amoeboid proliferative cell fraction in the tumour, leading to reduced tumour initiation.\" - \"We next assayed the in vivo tumour-initiating capacity of limiting dilutions of shDAAM1 WM1361 cells. DAAM1 depletion (Supplementary Fig._) yielded a reduction in TIF and a decrease in tumour growth at limiting dilutions of 50,000 and 5,000 cells (Fig._). shDAAM1 tumours displayed pronounced loss of cell rounding (Fig._ ), Myosin II activity (Fig._ and Supplementary Fig._) and ki-67 positive cells (Fig._). The in vivo tumour-initiating capacity of DAAM1 in melanoma was further validated using an additional melanoma cell line A375M2 (Supplementary Fig._ ). DAAM1 depletion led to a reduction in TIF and in A375M2 tumour growth (Supplementary Fig._). These results confirm that FZD7-DAAM1-RhoA-ROCK1/2 promote tumour initiation by sustaining melanoma amoeboid behaviour.\" - \"Cells at the IF of tumours are those strategically positioned to leave the primary tumour and disseminate to distant sites. The lung is one of the main sites to which melanoma metastasizes and high Myosin II levels in amoeboid cells promote lung seeding and colonization. To test the role of DAAM1 in both processes, we used an experimental metastasis model. Similar cell numbers lodged in the lung vasculature 30u2009min post-tail vein injection (Supplementary Fig._ ). However, shDAAM1 melanoma cells were less efficient in colonising the lung after 24u2009h (Fig._). Strikingly, the area of metastatic lesions 3 weeks after was reduced when DAAM1 was depleted (Fig._). These results indicate that DAAM1 is important for early stages of lung colonization and for the ability to outgrow at later stages. Our data further suggest that loss of amoeboid features has an impact both in early lung seeding and later in metastatic outgrowth.\" - Amoeboid behaviour enhances tumour formation, tumour progression and metastasis in vivo - \"We have shown that the ROCK1/2-Myosin II amoeboid phenotype sustained by FZD7-DAAM1 signalling supports tumour formation and dissemination. Using an orthotopic spontaneous melanoma metastasis model, we next recapitulated the steps of melanoma progression. Injection of 4599-BRAF melanoma cells into the dermis of NSG mice resulted in the formation of orthotopic tumours. Mice were then systemically treated with ROCKi (Fig._) since to our knowledge no drugs were available for the effective/specific inhibition of non-canonical FZD7-DAAM1 signalling in vivo. We observed three defined areas in control tumours: TB, IF and a further area of local invasion into the dermis (invading cells or distal invasive front (DIF)) (Fig._) closely recapitulating the human disease. All invading cells at the DIF displayed rounded-amoeboid features: 56% of them exhibited the highest Myosin II activity and 60.4% of invading cells were positive for the proliferative marker ki-67 (Fig._). Specifically, co-staining of p-MLC2 score 3 and ki-67 was found in 30.2% of invading cells at the DIFs (Fig._). These data suggest that amoeboid invading cells are also proliferative. Cell rounding (Fig._), overall Myosin II activity (Fig._) and ki-67 positive cells (Fig._) were progressively increased from TB to IF to DIF in control tumours. Cells with very high Myosin II were abundant in the IF and further enriched in the DIF (Fig._ and Supplementary Fig._). Importantly, ROCKi led to loss of cell rounding and decreased Myosin II activity at IFs and DIFs (Fig._), while a reduction of ki-67 positive cells was observed in all areas of ROCKi treated tumours (Fig._). These data suggest that ROCKi impairs amoeboid features and proliferation in vivo.\" - \"Fig. 6: Amoeboid behaviour enhances tumour formation, tumour progression and metastasis in vivo.a Schematic of in vivo experiment with 4599 cells injected intradermally into NSG mice treated with ROCKi (25u2009mg/kg GSK269962A) or control (5% DMSO) (nu2009=u20095 mice for control and nu2009=u20094 for ROCKi). b Representative H&E images showing areas of TB, IF and local area of invasion into the dermis (distal invasive front (DIF)) of primary tumours derived from (a). Dashed lines represent the boundary between tumour areas. Scale bar, 200u2009u03BCm. c Representative images (left) and quantification (right) of invading cells with co-staining of p-MLC2 (red) and ki-67 (green) at the DIF of control tumours from (a). Scale bar, 200u2009u03BCm. Yellow arrows indicate ki-67 positive invading cells with p-MLC2 score 3 intensity. du2013 f Representative images (top) and quantification (bottom) of d melanoma cell shape score, e H-score and percentage of cells with score intensity 0u20133 for p-MLC2 staining and f ki-67 positive cells in primary tumours from (a). Scale bar, 100u2009u03BCm; inset, 25u2009u03BCm. g GSEA plots showing enrichment of proliferative and invasive gene signatures from Verfaillie study in A375M2 cells compared to A375M2 cells treated with ROCKi (H1152 and Y27632) and blebbistatin or to A375P cells. NES, normalized enrichment score; FDR, false discovery rate. h Tumour growth curves of 4599 cells from (a). i Representative images (left) and QuPath mark-up (right) of mCherry staining and j quantification of invading cells into the dermis in primary tumours derived from (a). Dashed lines represent the boundary between IF and DIF. Scale bar, 50u2009u03BCm. k Representative images (left) and quantification of tumour area (right) of spontaneous lung metastasis from (a). Scale bar, 1u2009mm. l, m Representative images of mouse lungs (left) and quantification of tumour area (right) at the indicated times after tail vein injection of 4599 cells l pre-treated with ROCKi for 24u2009 h or m without any pre-treatment into NSG mice. Animals were systemically treated with ROCKi (nu2009=u20096 mice/group). Scale bar, 2u2009mm; inset, 500u2009 u03BCm. c Graphs show meanu2009xB1u2009s.e.m. du2013f, ju2013m Box limits show 25th and 75th percentiles, the horizontal line shows the median, and whiskers show minimum and maximum range of values. du2013f One-way ANOVA with Benjamini, Krieger and Yekutieli correction. h, j Two-tailed t-test. ku2013m Two-tailed Mannu2013Whitney test. For all graphs, *pu2009<u20090.05, **pu2009<u2009 0.01, ***pu2009<u20090.001, ****pu2009<u20090.0001. The exact significant p values for *p, **p and ***p are provided in Supplementary Table_. Schematics in this figure were created using Servier Medical Art templates licensed under a Creative Commons Attribution 3.0 Unported License ().\" - \"Phenotype switching from proliferative to invasive states has been implicated in melanoma progression. However, our transcriptional analysis showed that amoeboid A375M2 cells were enriched in both proliferative and invasive genes (Fig._ and Supplementary Fig._). Moreover, enrichment in Myosin II activity and ki-67 positive cells was found in IFs of A375M2 tumours (Fig._ and Supplementary Fig._), suggesting that the amoeboid phenotype sustained by ROCK1/2-Myosin II could, in principle, support tumour formation and tumour dissemination. Remarkably, in accordance with its role in regulating both proliferative and invasive genes, ROCKi caused a reduction in primary tumour growth in 4599 tumours (Fig._) and resulted in loss of melanoma cells leaving the primary tumour and invading into adjacent tissue (Fig._). In line with reduced local invasion, spontaneous lung metastasis was reduced dramatically in ROCKi treated mice (Fig._). Our results suggest that ROCK1/2-Myosin II activity in amoeboid melanoma cells controls both tumour growth and spread.\" - \"In the previous experimental setting, as both primary tumour growth and local invasion were reduced, it was difficult to separate these effects from those on metastasis. We next evaluated the impact of ROCKi in melanoma metastatic colonisation and outgrowth using tail vein injection assays. 4599 cells were pre-treated with ROCKi for 24u2009h prior to intravenous injection and continued treatment in vivo for 6 days. Cell survival in the blood stream was not altered as similar cell numbers lodged in the lungs 30u2009min post-tail vein injection (Supplementary Fig._). However, ROCKi reduced lung metastatic establishment (Fig._). Reduced lung metastasis was also measured when 4599 cells -without any pre-treatment- were intravenously injected, while 4u2009h after injection mice received ROCKi systemic treatment for 12 days (Fig._).\" - Our data confirm that ROCK1/2-Myosin II activity is not only important for tumour formation and invasion but also for early colonization and later metastatic growth. This suggests that ROCK1/2-Myosin II activity plays a crucial role in all steps of melanoma progression. - Analysis of the invasive front of human primary melanomas - \"To understand if amoeboid cells in the IF have increased tumour-initiating features in the human clinical setting, we analysed matched TBs and IFs of 53 human primary melanomas (Supplementary Table_). Cell morphology and p-MLC2 levels were assessed as amoeboid markers; ki-67 staining as a proliferative marker; WNT11, WNT5B and DAAM1 expression as non-canonical Wnt markers; and ALDH1A1, ALDH1A3, CD44, NANOG and OCT4 were evaluated as cancer_stem cell-related markers. In accordance with previous assessments in melanoma patients, we could measure that IFs of human melanomas were enriched in rounded-amoeboid cells with high Myosin II activity (Fig._ and Supplementary Fig._). Importantly, as observed in our tumour models (Figs._, , ), cells with very high Myosin II were predominant at human IFs. Moreover, an increase in the percentage of ki-67 positive cells was also found at IFs (Fig._). Interestingly, cells with higher levels of DAAM1, WNT11 and WNT5B were also enriched in the IFs (Fig._). Cells with score 3 level of DAAM1 were predominant in these areas (Supplementary Fig._). Importantly, we found an enrichment in ALDH1A1, CD44 and NANOG expression at these IFs (Fig._ ), while ALDH1A3 and OCT4 did not change (Supplementary Fig._). We provided regional information showing high expression of amoeboid/non-canonical Wnt/stem cell markers in the IF area of human primary tumours (Supplementary Fig._). Interestingly, principal component analysis of all differentially expressed markers clearly separated TB and IF areas in primary melanomas (Fig._). Amongst all markers investigated, ALDH1A1 enrichment was highest, showing a median 13-fold increase in IFs. Strikingly, high ALDH1A1 protein expression in the IF appeared to confer worse prognosis and shorter disease-free survival in our cohort of melanoma patients (Fig._). ALDH1A1 is a functional marker involved in self-renewal and proliferation with key anti-oxidative roles. Invasive melanoma cells with high Myosin II harbour lower levels of reactive oxygen species (ROS), while antioxidants promote melanoma invasion and metastasis. Moreover, higher ROS levels reduce Myosin II activity. Silencing ALDH1A1 led to decreased melanosphere formation (Supplementary Fig._). Interestingly, ALDH1A1 depletion resulted in loss of cell rounding and lower p-MLC2 levels_(Supplementary Fig._), while ROS levels, as measured by FACS, were increased (Supplementary Fig._). Our data suggest that ALDH1A1 supports amoeboid tumour-initiating features via regulation of ROS metabolism. Overall, these data illustrate that the IFs of human primary melanomas are tumour areas enriched in amoeboid, proliferative, non-canonical Wnt and cancer_stem cell-related markers.\" - \"Fig. 7: Analysis of the invasive front of human primary melanomas.au2013i Representative images (left) and quantification (right) of a melanoma cell shape score, b H-score of p-MLC2 staining, c ki-67 positive cells, H-score of d DAAM1, e WNT11, f WNT5B, g ALDH1A1, h CD44 and i NANOG staining in matched TB and IF from primary melanomas. j Principal component analysis (PCA) based on the expression of amoeboid (cell shape and p-MLC2), proliferative (ki-67), non-canonical Wnt pathway (WNT11, WNT5B and DAAM1) and cancer_stem cell-related (ALDH1A1, CD44 and NANOG) markers assessed by immunohistochemistry in matched TB and IF from primary melanomas. Percentage of variation explained by each component is given in the axis labels. k, l Kaplanu2013Meier survival curves of k overall survival and l disease-free survival according to ALDH1A1 protein expression in the IF from our cohort of primary melanomas. ALDH1A1 expression was categorized as low or high using the median expression. au2013l 53 primary melanomas. a, eu2013i Scale bar, 100u2009 u03BCm; inset, 50u2009u03BCm. bu2013d Scale bar, 300u2009u03BCm. au2013 i Box limits show 25th and 75th percentiles, the horizontal line shows the median, and whiskers show the minimum and maximum range of values. a, b, du2013f Two-tailed paired t-test. c, gu2013i Two-tailed Wilcoxon test. j Scatter plot showing principal components 1 (PC1) and 2 (PC2). k, l Log-rank test. Human schematic in this figure was created using Servier Medical Art templates licensed under a Creative Commons Attribution 3.0 Unported License ().\" - Analysis of the invasive front of human metastatic melanomas - \"To further investigate if melanoma cells retain this phenotype during the course of secondary tumour formation, 45 matched TBs and IFs from human melanoma metastases were assessed (Supplementary Table_). Metastatic melanomas retained the same regional distribution identified in primary tumours; that is, amoeboid (Fig._ and Supplementary Fig._) and proliferative cells (Fig._) were prominent in the IF areas, and that IFs also had an enrichment in non-canonical Wnt markers (Fig._ and Supplementary Fig._) and stem cell-related markers (Fig._ and Supplementary Fig._). Importantly, the overall levels of all markers were higher in metastasis. In fact, TBs from metastatic lesions were more amoeboid, more proliferative and had a higher basal level of some non-canonical Wnt and stem cell-related markers (Supplementary Fig._) when compared to primary tumour TBs. As a result, the increased expression of all markers in metastatic IFs was not as pronounced as in the primary tumour setting. These results show that the amoeboid tumour-initiating phenotype found in primary IFs is recapitulated and further enhanced in metastatic melanoma specimens. Overall, our data show that amoeboid cell invasive/proliferative behaviour is selected during melanoma progression.\" - \"Fig. 8: Analysis of the invasive front of human metastatic melanomas.au2013 i Representative images (left) and quantification (right) of a melanoma cell shape score, b H-score of p-MLC2 staining, c ki-67 positive cells, H-score of d DAAM1, e WNT11, f WNT5B, g ALDH1A1, h CD44 and i NANOG staining in matched TB and IF from melanoma metastases. j Model summarizing the findings of this study. The IF of human primary melanomas is enriched in cells with rounded-amoeboid morphology, high levels of Myosin II, proliferative, non-canonical Wnt and cancer_stem cell-related markers. This phenotype is recapitulated and further enriched in melanoma metastasis. WNT11/5B activate FZD7 and DAAM1 to control Rho activity and then ROCK1/2-Myosin II levels. All these signalling components have an impact on amoeboid features and tumour initiation in melanoma both in vitro and in vivo. Specifically, DAAM1 plays an essential role in tumour and metastasis initiation and subsequent metastatic outgrowth by sustaining the amoeboid phenotype. au2013 i 45 metastatic melanomas. a, eu2013i Scale bar, 100u2009u03BCm; inset, 50u2009 u03BCm. bu2013d Scale bar, 300u2009u03BCm. au2013i Box limits show 25th and 75th percentiles, the horizontal line shows the median, and whiskers show minimum and maximum range of values. a, b, du2013f Two-tailed paired t-test. c, gu2013 i Two-tailed Wilcoxon test. Human schematic in this figure was created using Servier Medical Art templates licensed under a Creative Commons Attribution 3.0 Unported License ().\"- - Melanoma is a highly aggressive tumour that is able to metastasize from very early stages of the disease. The high migratory ability of melanoma cells has been explained in part by its NC origin. Signalling pathways involved in NC development and melanoma progression are closely related. Here, we provide evidence that amoeboid melanoma cells display increased tumour-initiating abilities. We show that rounded-amoeboid cells with high Myosin II activity and high levels of ki-67 are associated with increased tumour initiation capacity in vitro and in vivo. ROCK1/2-Myosin II pathway is a key regulator of cell proliferation, migration, invasion and metastatic behaviour. We have recently reported that Myosin II activity drives therapy resistance in melanoma, a mechanism closely related with tumour-initiating cell properties. NC stem cell transcriptional states play a critical role in driving relapse and drug resistance in melanoma. Moreover, analysis of our transcriptional signature for amoeboid melanoma cells reveals that amoeboid cells are enriched in NC and invasive signatures associated with drug resistance. On the other hand, tumour-initiating cells are supposed to represent a small cell population in a slow proliferative state. However, it has been described that they can proliferate vigorously and tumour-initiating cell frequencies up to 25% have been reported. Ki-67 can affect clonogenic stem cell properties, and sub-populations of ki-67 proliferating stem cells have also been found in some tumours. - \"Importantly, we also find that amoeboid melanoma cells express EMT-related genes driven by ROCK1/2. Rho-ROCK1/2 signalling has been shown in other models to regulate actin cytoskeleton rearrangements during EMT while promoting transcriptional changes through activating p38 signalling. Interestingly, ROCK1/2 regulates TGFu03B2 expression in our EMT array and TGFu03B2 signalling is one of the top networks associated with the EMT gene signature regulated by ROCK1/2. We have also shown that amoeboid cells secrete high levels of TGFu03B2 and its secretion is ROCK1/2 dependent. TGFu03B2 is a key driver of EMT by inducing transcription of several mesenchymal genes and increasing the activity of EMT transcription factors. This suggests TGFu03B2 as one of the possible mechanisms by which ROCKs modulate a global EMT gene program and, on the other hand, amoeboid invasion.\" - \"Among these EMT-related genes regulated by ROCKs, we find Wnt ligands. Such genes, in turn, drive amoeboid features. Specifically, we find that the WNT11/5B-FZD7-DAAM1 signalling axis has an impact on tumour initiation by controlling the amoeboid phenotype. WNT11-DAAM1 signalling has been described to play an essential role in NC migration, NC formation and NC specification during early embryonic development. WNT5A through Ca and PKC signalling has been described to play a role in melanoma progression, while other Wnt ligands have not been studied in detail. We show that WNT11 and WNT5B control amoeboid self-renewal and invasive behaviour. Moreover, melanoma cells could exploit paracrine Wnt signalling from stroma, sustaining the amoeboid phenotype, as long as they express the correct receptor repertoire. We find that FZD7 receptor downstream of WNT11/WNT5B ligands is upregulated during melanoma progression and we show this receptor sustains amoeboid behaviour. Limiting-dilution analysis revealed how FZD7 supports in vivo tumour-initiating abilities and tumour growth in melanoma, consistent with previous reports in other tumours. Importantly, our work has unravelled a crucial role for DAAM1 in promoting both tumour and metastasis initiation and later metastatic outgrowth in melanoma by regulating amoeboid features. DAAM1 is a formin-homology protein that plays a central role in translating signals from Wnt ligands to the actin cytoskeleton. Unlike other formins, DAAM1 activation is specifically mediated by its binding with Dvl upon Wnt stimulation rather than by its interaction with active Rho GTPases. Such mechanisms have been described during development, but not during cancer initiation or progression. Therefore, we highlight here yet another example of cancer cells hijacking a developmental program. On the other hand, DAAM1 promotes RhoA activation via recruitment of PDZ-RhoGEF. In accordance, using the Human Protein Atlas, we find that this PDZ-RhoGEF is highly expressed in melanoma compared to other cancers (Supplementary Fig._). We propose here that non-canonical WNT11/5B-FZD7-DAAM1 axis crucially and specifically controls tumour initiation potential in melanoma by promoting ROCK1/2-Myosin II driven amoeboid proliferative and invasive phenotype (Fig._).\" - \"Furthermore, we provide evidence that amoeboid cells are selected during several steps of melanoma progression. In the current study, we use potent and selective ROCKi to demonstrate that ROCK1/2-Myosin II axis is an important regulator of tumour growth and metastasis in melanoma. We have reported that amoeboid melanoma cells support tumour growth by reprogramming the immune microenvironment. Furthermore, the tumour microenvironment can regulate ROCK2-Myosin II to support tumour growth. We demonstrate here that in melanoma ROCK1/2-Myosin II also plays an important pro-tumourigenic role via cell autonomous mechanisms. Importantly, amoeboid melanoma cells express genes from both proliferative and invasive signatures. We show the in vivo existence of amoeboid invading cells that are positive for the ki-67 proliferative marker and harbour very high levels of Myosin II. Such a population is also impaired when blocking the whole pathway with ROCKi. In a recent study, a subpopulation of melanoma cells has been identified that simultaneously displays proliferative and invasive properties as a result of high transcriptional TGFu03B2 signalling activation. Since their gene ontology analysis show an enrichment in amoeboid features, it is tempting to speculate that this is the same population of melanoma cells.\" - \"In this study, we demonstrate that the amoeboid proliferative and invasive phenotype promoted by non-canonical WNT-DAAM1 signalling is required for tumour initiation in melanoma and for both initial and late metastatic stages. We suggest that amoeboid melanoma cells sustain the expression of an EMT and stem cell-related gene signature to successfully metastasize. In agreement with this concept, assessment of patient biopsies revealed that IFs of human primary melanomas are enriched in amoeboid, proliferative, non-canonical Wnt and cancer_stem cell-related markers. Functional stem cells that drive tumour expansion and proliferative cells have been described to reside at the tumour edge of some tumours. Moreover, ki-67 index and mitotic rate are indicative of neoplastic progression and appear as prognostic factors in melanoma. Furthermore, within this amoeboid tumour-initiating phenotype, we identify ALDH1A1, a key gene for tumour-initiating abilities. Importantly, loss of ALDH1A1 results in loss of the amoeboid phenotype and vice versa. Since ALDH1A1 has been implicated in tumour-initiating capacity in melanoma and has been associated with drug resistance, we propose ALDH1A1 as a potential prognostic marker in melanoma patients. Importantly, we find metastatic melanoma lesions to recapitulate the behaviour of primary tumours and show an enhancement of the amoeboid tumour-initiating pattern.\" - Overall, we have unveiled how WNT11B/5B-FZD7-DAAM1 in amoeboid cells supports tumour-initiating properties, while also promoting invasion. Since amoeboid cells are prominent at the edge of tumours, our work sheds some light into the early ability of melanomas to disseminate and grow at distant sites. We propose that the IFs of melanoma tumours are areas with important prognostic features that pathologists should carefully evaluate. Furthermore, after surgical melanoma removal, we suggest inhibiting non-canonical Wnt signalling to eradicate aggressive amoeboid behaviour at the edge of tumours.- - Cell culture - \"Cells were grown at 37u2009xB0C and 10% CO2 in DMEM (A375M2, A375P, B16F10, 4599, HEK293T and HaCaT) or RPMI (WM1361, WM983B and WM983A) supplemented with 10% FBS and 1% penicillin/streptomycin (all from Gibco). A375M2 and A375P cells were from Prof. Richard Hynes (HHMI, MIT, USA), WM1361 cells were from Prof. Richard Marais (Cancer Research UK Manchester Institute, UK), WM983B and WM983A were purchased from Coriell Institute, B16F10 cells were from Dr. Hector Peinado (CNIO, Spain), 4599 cells were from Dr. Amine Sadok (Institute of Cancer Research, UK) and Prof. Richard Marais, HEK293T cells were from Dr. Jeremy Carlton (The Francis Crick Institute, UK), and HaCaT cells were from Dr. Ester Martin-Villar (IIBM, UAM, Spain). A375M2, A375P, WM983B, WM983A and 4599 were authenticated using short tandem repeat DNA profiling. All cell lines were routinely tested for mycoplasma contamination. All cell lines were kept in culture for a maximum of three to four passages and cell phenotypes were verified routinely.\" - Sphere formation assay - \"The sphere-forming assay was performed as previously described. Briefly, individual cells were plated in 100u2009xB5l of Sphere Media (DMEM/F12 supplemented with 1:50 B27 (Invitrogen), 20u2009ng/ml EGF (PHG0315, Invitrogen), 20u2009ng/ml FGF (PHG0026, Invitrogen), 4u2009mg/ml heparin (H3149, Sigma) and 1% penicillin/streptomycin) in ultra-low-attachment 96-well plates (Costar) at a density of 1,000 cells/well and kept at 37u2009xB0C and 10% CO2. For serial passages, melanospheres were collected by centrifugation followed by enzymatic (trypsin treatment for 10u2009 min at 37u2009xB0C) and mechanical dissociation using 25G needles. The resulting single-cell suspension was counted and re-plated at the original density. In all assays, melanosphere formation was quantified after 7 days. The Sphere Formation Index (SFI) considered both, number and size of the spheres, and was calculated by summing the area of all spheres formed, divided by the number of single cells initially plated. In each biological experiment, 2u20133 technical replicates were performed and average SFI considered. Quantitative analysis of images was performed using ImageJ.\" - Cell culture on thick layers of collagen I - \"Bovine collagen I (no. 5005-B; PureCol, Advanced BioMatrix) thick layer was prepared at 1.7u2009mg/ml as previously described. After polymerization (4u2009h), cells were seeded on top of collagen in medium containing 10% FBS, allowed to adhere for 16u2009h and media was changed to 1% FBS with corresponding treatments (where appropriate). Cells were analysed 24u2009h later. At the end of the experiment, gels were fixed with 4% formaldehyde and imaged or lysates were collected.\" - Analysis of cell morphology - \"Cell morphology was quantified on phase-contrast images of cells cultured on top of bovine collagen I matrices using ImageJ as previously described. Cell morphology was assessed using the morphology descriptor tool u201Croundnessu201D after manually drawing around the cell shape. Roundness index is calculated as 4u2009xD7u2009area/u03C0u2009xD7u2009major_axis_length. Values closer to 1 represent rounded morphology; values closer to 0 represent more spread and/or spindle-shaped cells.\" - Drug treatments - \"Contractility inhibitors and concentrations used in this study were: 5u2009u03BC M H1152 (stock resuspended in water; #555550, Calbiochem), 1u2009xB5M or 5u2009 xB5M GSK269962A (stock resuspended in DMSO; #1167, Axon) and 25u2009xB5M blebbistatin (stock resuspended in 95% DMSO; #203390, Calbiochem). Time of treatment was as specified in each experiment. WNT11 (200u2009ng/ml; 6179-WN-010, R&D Systems,) and WNT5B (500u2009ng/ml; 7347-WN-025, R&D Systems) were added in serum-free media for 24u2009h.\" - Transfection and RNAi - \"2u2009xD7u200910 cells/well were seeded on 6-well plates and transfected the next day with 20u2009nM siGenome SmartPool (SP) or On-Targetplus (OT) siRNA oligonucleotides, using Optimem-I and Lipofectamine 2000 (Invitrogen). For sphere formation assays and cell viability assays, cells were re-transfected after 24u2009h. In all assays, 48u2009h after transfection, cells were harvested and equal numbers were re-seeded for subsequent analyses. All siRNA sequences were from Dharmacon (Lafayette, USA) and are listed in Supplementary Table_ . In all siRNA experiments, Non-Targeting siRNA was used as control.\" - Lentivirus generation and infection of melanoma cells - \"4u2009xD7u200910 HEK293T cells on 6-well plates were transfected with lentiviral vector (1u2009u03BCg) along with packaging vectors (p-MD2.VSVg (0.4u2009u03BC g) and pu03948.91 (1u2009u03BCg)) using Optimem-I and Lipofectamine 2000 (Invitrogen). Supernatants with lentiviruses were collected 48u2009h after transfection, spun down and filtered (0.45u2009u03BCm). For lentiviral transduction, 2u2009 xD7u200910 melanoma cells/well were seeded in 6-well plates and infected with lentiviruses. Stable cells were selected with 1u2009u03BCg/ml puromycin (Life Technologies). FZD7 and DAAM1 silencing was achieved using pGIPZ Lentiviral shRNA constructs expressing TurboGFP from Dharmacon (Lafayette, USA): shFZD7 #1 (V3LHS_368263 5u2032-AGAGCACGCTGAAGACGCC-3u2032), shFZD7 #2 (V3LHS_368264 5u2032-CGTGTTTCATGATGGTGCG-3u2032), shDAAM1 #1 (V3LHS_339679 5u2032-TTGTGATTGGTCTCTCTCT-3u2032 ), shDAAM1 #2 (V3LHS_339678 5u2032-TTAGATTGAGAACACTGGG-3u2032), Non-silencing shControl (5u2032-ATCTCGCTTGGGCGAGAGTAAG-3u2032). EGFP-fused rat wild-type MCL2 plasmid was obtained from Dr. Tohru Takaki and Prof Erik Sahai.\" - Immunoblotting and antibodies - \"Cells were lysed in Laemmli sample buffer, boiled for 5u2009min, sonicated for 15u2009s and spun down. Lysates were fractionated using 10% or 12% SDS-polyacrylamide gel electrophoresis and transferred onto PVDF filters (0.45u2009xB5m, Immobilon). ECL or Prime ECL detection Systems (GE Healthcare) with HRP-conjugated secondary antibodies (GE Healthcare) were used for detection. Bands were quantified using Image J. Levels of phospho-proteins were calculated after correction to total levels of the relevant protein. Antibodies: pThr18/Ser19-MLC2 (1:750, #3674), MLC2 (1:750, #3672) and RhoA (1:1000, #2117) from Cell Signalling Technology; GFP (1:10000, sc-8334) from Santa Cruz Biotechnology; GAPDH (1:10000, MAB374) from Millipore.\" - RhoA-GTP pulldown assay - \"RhoA-GTP pulldown assays were performed as described. 2u2009xD7u200910 cells/well were seeded on 6-well plates, serum starved and treated with WNT11 as indicated above. Cells were lysed in lysis buffer containing 50u2009mM Tris pH 7.4, 10% glycerol, 1% NP40, 5u2009mM MgCl2, 100u2009mM NaCl, 1u2009mM DTT and EDTA free protease inhibitor and spun down. A small proportion of protein lysates were separated for determination of total RhoA levels. The remaining protein lysate was incubated with glutathione S-transferase (GST)-conjugated with Rhotekin RBD beads for 1u2009h. Beads were collected by centrifugation, washed and resuspended in loading buffer. All samples were boiled for 5u2009min and resolved by SDS-polyacrylamide gel electrophoresis. RhoA was detected by immunoblot.\" - Confocal fluorescence microscopy and image quantification - \\'Cells were seeded on top of a collagen I matrix and immunostained as described. Cells were fixed with 4% formaldehyde, permeabilised with 0.3% Triton X-100, blocked with 5% bovine serum albumin (BSA), and stained with primary antibody pSer19-MLC2 (1:200, #3671, Cell Signalling), which was detected with secondary Alexa Fluor 488 or 647 antibodies (Life Technologies). F-actin was stained using Alexa Fluor 546-phalloidin (Life Technologies) and nuclei with Hoechst 33258 (Life Technologies). Antibodies were diluted in 5% BSA-PBS. Imaging was carried out on Zeiss LSM 510 Meta confocal microscope with Plan-Apochromat 40x/1.2 NA (water) objective lenses and Zeiss LSM 710 confocal microscope with Plan-Apochromat 40x/1.3 Oil or a Plan-Apochromat 63x/1.4 NA (oil) objective lenses (Carl Zeiss) and Zen software. Images were analysed using ImageJ. p-MLC2 fluorescence signal was quantified calculating the pixel intensity in single cells relative to the cell area.\\' - Adhesion assays to keratinocyte monolayer - \"HaCaT keratinocytes were seeded in a 96-well plate at 10 cells/well and, 24u2009 h later, melanoma cells stained with CMFDA green dye (10u2009xB5M, Life Technologies) were seeded on top at 10 cells/well. After 2u2009h at 37u2009xB0C, cells were imaged. Plates were then washed twice with PBS and re-imaged. Percentage of adhering cells was calculated as the number of cells imaged after vs before washing.\" - 3D invasion assays - \"Cells were suspended in serum-free bovine collagen I at 2.3u2009mg/ml to a final concentration of 1u2009xD7u200910 cells per 100u2009u03BCl of matrix, seeded on 96-well plates and spun down to the bottom of the well. After matrix polymerization, 10% FBS-containing media was added on top of the matrix, allowing the cells to invade upwards for 24u2009h as previously described. Then, plates were fixed in 4% formaldehyde, stained with 5u2009xB5g/ml Hoechst 33258 (Life Technologies) and imaged on Zeiss LSM 510 or Zeiss LSM 710 confocal microscopes (Carl Zeiss) with Zen software. Confocal z-slices were collected from each well at the bottom of the well and at 50u2009xB5m. The 3D invasion index was calculated as number of invading cells at 50u2009xB5m divided by the total number of cells.\" - Intracellular measurement of ROS - \"Cells were harvested and incubated with 5u2009u03BCM CellROX Green Reagent (Life Technologies) at 37u2009xB0C for 30u2009min in the dark. Then, cells were diluted with FACS buffer (PBSu2212, 1% BSA, 2u2009mM EDTA, 0.1% NaN3) and fluorescence was analysed by flow cytometry using a BD LSRFortessau2122 system (Flow Cytometry Core, Barts Cancer Institute) and FlowJo software (Supplementary Fig._). Cells were first gated on FSC-A vs SSC-A to eliminate cell debris. Then, discrimination of doublets was performed on FSC-A vs FSC-H and SSC-A vs SSC-W. Only viable and single cells were used for the analysis. CellROX green positivity was analysed using mean fluorescence intensity.\" - Cell viability assay - \"Cells stably expressing shRNAs or cells after 48u2009h transfection with siRNAs were seeded in 6-well plates (25,000 cells/well). For ROCKi experiments, 25,000 cells were seeded in 6-well plates, allowed to adhere and treatment was performed as indicated for each experiment. Cell viability was assessed after 7 days in all experiments, while for MYL9 and MYL12B siRNA experiments that it was assessed after 3 and 7 days. Then, cells were fixed with 1% formaldehyde and stained with 0.25% crystal violet. After air-drying, crystal violet stain was dissolved in 10% acetic acid. Samples from each well were transferred to a 96-well plate and the absorbance was measured at 595u2009nm on a microplate reader as an indirect measure of cell number. Results were normalized to the value of initially plated cells and presented as percentage of cells in each sample relative to control.\" - Quantitative real time one-step PCR and qPCR EMT array - \"Total RNA was isolated using TriZol (Life technologies) and RNeasy Mini Kit (Qiagen). qPCR primers and Brilliant III Ultra-Fast SYBR Green QRT-PCR Master Mix (Agilent Technologies) with 50u2009ng RNA were used following the manufacturer s instructions. GAPDH was used as loading control. The following qPCR primers from Sino Biological were used: WNT11 (HP101586), WNT5B (HP101843), ALDH1A1 (HP100088), OCT4 (HP101756). The following QuantiTect Primers from Qiagen were used: GAPDH (QT00079247), ROCK1 (QT00034972), ROCK2 (QT00011165), MYL9 (QT00072268), MYL12B (QT00075264), FZD7 (QT00213850), RYK (QT00047292), PLCB1 (QT00038689), DAAM1 (QT00005579), ALDH1A3 (QT00077588), NANOG (QT01025850), JARID1B (QT00060648), CD44 (QT00073549), SOX2 (QT00237601), ALCAM (QT00026824).\" - \"Epithelial to mesenchymal transition (EMT) RT2 Profiler PCR Array (330231, PAHS-090Z, Qiagen) was performed according to manufacturers instructions. For that, 2u2009xB5g of total RNA were reverse-transcribed to cDNA using RT2 First Strand Kit (Qiagen) and subjected to qPCR EMT array. Reactions were run on a QuantStudio 7 Flex Real-Time PCR System (ThermoFisher Scientific). PCR array was analysed using the web-based tool () supplied by the manufacturer and gene expression was normalized to housekeeping genes.\" - Analysis of gene expression data from human databases - From the public database GEO (Gene Expression Omnibus), Xu GSE8401 (31 primary and 52 metastatic melanomas), Riker GSE7553 (14 primary and 40 metastatic melanomas) and Kabbarah GSE46517 (31 primary and 73 metastatic melanomas) series were extracted. Data were normalized using GenePattern platform (). - Gene expression data and clinical information of human melanoma samples (70 primary and 319 metastatic melanomas) from The Cancer Genome Atlas (TCGA) database were downloaded from Firehose (). Only TCGA samples with no neo-adjuvant treatment prior to tumour resection were considered. - Gene enrichment analyses - \"Normalized gene expression microarray data of amoeboid melanoma cells (GSE23764) was analysed by comparing amoeboid A375M2 cells to more elongated and less contractile A375P cells or to A375M2 cells treated with ROCK1/2 inhibitors (H1152 and Y27632) or blebbistatin. Gene sets for proliferative and invasive signatures from Verfaille and Hoek studies; stem cell-like signatures and EMT processes were downloaded and analysed using Gene Set Enrichment Analysis (GSEA) software () with the specific settings: permutations-1,000, permutation type-gene set, metric for ranking genes-t-test. For single-sample Gene Set Enrichment Analysis (ssGSEA), significantly enriched stem cell-like/EMT gene sets in amoeboid A375M2 cells were considered according to p-value < 0.05 and FDRu2009<u20090.25 in at least 3 of the 4 comparisons performed. To calculate the gene-signature score in each sample, we used ssGSEA Projection Software from GenePattern platform (). Heatmaps were generated using MeV_4_9_0 software ().\" - Network enrichment analysis of significantly downregulated genes in at least one cell line also downregulated or with no expression changes in the other cell line from qPCR EMT array was performed using Ingenuity Pathway Analysis (Qiagen). - Animal studies - \"All animals were maintained under specific pathogen-free conditions and handled in accordance with the Institutional Committees on Animal Welfare of the UK Home Office (The Home Office Animals Scientific Procedures Act, 1986). All animal experiments were approved by the Ethical Review Process Committees at Barts Cancer Institute, Kings College London and The Francis Crick Institute and carried out under licences from the Home Office, UK. All mice were obtained from Charles River UK. Mice used were 6u201312 weeks old. Animals were housed in groups of 4u20135 mice per cage with access to food and water ad libitum. Mice were maintained on an alternating 12u2009h light-dark cycle, with controlled room temperature (21u2009xB1u20091u2009xB0C) and relative humidity (40u2013 60 %).\" - \"For tumour initiation assays with pre-treated A375M2 and B16F10 cells, prior to injection cells were pre-treated in vitro with ROCK1/2 inhibitors (5u2009 xB5M H1152, 1u2009xB5M or 5u2009xB5M GSK269962A) for 5 days, with fresh drug added every day. Then 1u2009xD7u200910 A375M2 cells or 2u2009xD7u2009 10 B16F10 cells were mixed in 100u2009xB5l of Growth Factor Reduced Matrigel (356230, Corning), and injected subcutaneously into female NOD/SCID/IL2Ru03B3 u2212 (NSG) mice (A375M2) or C57BL/6u2009J mice (B16F10). Mice were not treated during the course of the experiment.\" - \"For limiting dilution assays, 5u2009xD7u200910, 5u2009xD7u200910, 1u2009 xD7u200910_or 5u2009xD7u200910 cells (A375M2/A375P cells, stable shFZD7#1/shDAAM1#1/shControl WM1361 cells and stable shDAAM1#1/shControl A375M2 cells) were mixed in 100u2009 xB5l of Growth Factor Reduced Matrigel (356230, Corning) and injected subcutaneously into female NSG mice. Tumour growth was assessed 3 weeks after injection. The tumour-initiating frequency (TIF) was estimated by ELDA using the bottom 20 percentile of tumour size in each experiment as the limiting threshold for positive growth. Only tumours above this threshold were considered.\" - \"For the orthotopic experiment using 4599 cells, 2u2009xD7u200910 4599 cells stably expressing hNIS-mCherry (4599.hNIS-mCherry) in 30u2009xB5l PBS were injected intradermally into male NSG mice. Tumours were allowed to establish and treatment started at day 11 when mean tumour volume was 20 mm. Drug administration (25u2009mg/kg GSK269962A dissolved in 5% DMSO or vehicle 5% DMSO) was performed daily by oral gavage.\" - \"For all the above tumour models, tumour size was monitored and determined by caliper measurements and calculated as tumour volume (mm)u2009=u2009lengthu2009 xD7u2009widthu2009xD7u2009heightu2009xD7u20090.52. At the end of experiments, prior to dissection tumours were measured; or dissected tumours were weighed. Tumours were formalin-fixed and paraffin-embedded (FFPE) for immunohistochemical analysis.\" - \"For experimental metastasis assays, 1u2009xD7u200910 stable shDAAM1 and shControl WM1361 cells in 100u2009xB5l PBS were injected into the tail vein of NSG mice (matched male and female). Mice were sacrificed after 30u2009min (to show that equal cell numbers reach the lung), 24u2009h and 3 weeks, and then lungs were washed, fixed in 4% formaldehyde for 16u2009h and examined for fluorescent signal (GFP from shRNA constructs) under a Zeiss LSM 510 Meta confocal microscope (Carl Zeiss). 20u201325 images per mouse lung were analysed, and lung retention was represented as percentage of fluorescence area per mouse lung.\" - \"For experimental metastasis assays using 4599 cells, 4u2009xD7u200910 cells pre-treated in vitro with 5u2009xB5M GSK269962A for 24u2009h were injected in 200u2009xB5l PBS into the tail vein of NSG female mice. Mice were treated daily (25u2009mg/kg GSK269962A dissolved in 5% DMSO or vehicle 5% DMSO) by oral gavage. Mice were sacrificed after 30u2009min and 6 days, and then lungs were washed, fixed and examined by immunohistochemistry. Similarly, 4u2009xD7 u200910 4599 cells without any pre-treatment in 200u2009xB5l PBS were injected into the tail vein of NSG female mice and animals were treated daily (25u2009 mg/kg GSK269962A dissolved in 5% DMSO or vehicle 5% DMSO) by oral gavage for 12 days.\" - Immunohistochemistry - Case selection - \"Two tissue microarrays, including FFPE biopsies of 53 human primary melanomas and 45 metastases (Supplementary Tables_ and ) were included in the case series. Each biopsy was represented by two cores (1u2009mm diameter) from the tumour body (TB) and two cores from the invasive front (IF) areas. The IF was defined as melanoma cells with at least 50% contact with the matrix as previously described. Tumours were classified following the most recent World Health Organization criteria. Tumour samples were processed by IRBLleida (PT17/0015/0027) and HUB-ICO-IDIBELL (PT17/0015/0024) Biobanks integrated in the Spanish National Biobank Network and Xarxa de Bancs de Tumors de Catalunya following standard operating procedures with the appropriate approvals from the Ethics and Scientific Committee. Samples were collected with specific informed consent, in accordance with the Helsinki Declaration.\" - Whole sections from subcutaneous tumours (A375M2/A375P tumours, shControl/shFZD7/shDAAM1 WM1361 tumours and shControl/shDAAM1 A375M2 tumours from limiting dilution assays), from intradermal tumours (4599 cells) and mice lungs from tail vein metastasis assays (4599 cells) were included. - \"Melanospheres from serial sphere passages were also included. Agar pellets were generated with spheres from every passage. Spheres were collected gently and spun down at 300u2009xD7u2009g for 10u2009min. Pellets were washed twice with PBS and embedded in 50u2009xB5l 3% agar. Finally, agar pellets were embedded in paraffin using standards methods for immunohistochemical analysis.\" - Experimental procedure - \"All tissue samples were FFPE, sectioned (3u20134u2009u03BCm-thick) and dried for 1u2009h at 65u2009xB0C. Next, tissue samples were subjected to deparaffinization, rehydration and heat-induced epitope retrieval using a Biocare Decloaking Chamber (DC2012) at 110u2009xB0C for 6u2009min in the corresponding unmasking solution (Supplementary Table_). The endogenous peroxidase was blocked with dual endogenous enzyme-blocking reagent (#S2003, Agilent) for 10u2009min. Reagents were incubated at room temperature in a humidified slide chamber. Detailed information about the antibodies, dilutions and conditions used for the immunohistochemical stainings are specified in Supplementary Table_. All the stainings were counterstained with Haematoxylin.\" - Imaging and scoring - \"Morphologic analysis of tumour tissues was performed on haematoxylin- and eosin-stained (H&E) sections as previously described. Cell shape was graded from 0 to 3 (0u2009 =u2009round, 1u2009=u2009ovoid, 2u2009=u2009elongated and 3u2009=u2009 spindly) and a score was assigned as follows: cell shape scoreu2009=u2009((percentage of cells [%] shape 0u2009xD7u20090)u2009+u2009(% shape 1u2009xD7u2009 1)u2009+u2009(% shape 2u2009xD7u20092)u2009+u2009(% shape 3u2009xD7u2009 3)), with values ranging from 0 (all cells round) to 300 (all cells spindle). Using consecutive sections, IHC staining for all the markers was graded semiquantitatively by considering the percentage and intensity of the staining. A histologic score (H-score) was obtained from each sample, and values ranged from 0 (no immunoreaction) to 300 (maximum immunoreactivity). The score was obtained by applying the following formula, H-scoreu2009=u20091u2009xD7u2009(% light staining)u2009+u2009 2 u2009xD7u2009(% moderate staining)u2009+u20093u2009xD7u2009(% strong staining). WNT11, WNT5B, CD44, NANOG, OCT4, ALDH1A1 and ALDH1A3 stainings were scored blind and a staining H-score was provided for each section. Representative pictures according to mean H-score were imaged using iScope microscope (IS.1159EPLi, Euromex) and ImageFocus 4.0 (Euromex) software. For p-MLC2, ki-67, FZD7 and DAAM1 stainings, whole section images (WSI) were scanned using NanoZoomer S210 slide scanner (Hamamatsu). Staining quantification was performed with QuPath 0.1.2 software. Positive cell detection was performed, and three different thresholds were applied according to the intensity scores (0, 1, 2 and 3). Next, software was trained creating random tree classification algorithms combined with intensity information in order to differentiate tumour cells from necrosis, stroma and immune cells. Then, IHC staining was graded semiquantitatively by considering the percentage and intensity of the staining, and H-score values were calculated as described above. Furthermore, to show spatial correlation between different markers (ALDH1A1, CD44, p-MLC2, DAAM1 and ki-67), hotspots maps were created using the optical density mean (OD mean)u2014Chanel 2: VIP stain intensity in QuPath. For co-localization analysis, images for p-MLC2 and ki-67 markers were aligned in FIJI v1.52p using TrackEM2 module. Next, colour deconvolution was performed using AEC-Haematoxylin vectors and a composite was created using channel-2 (red) for each staining. The composite was adjusted inverting the LUT for each marker and was_given a pseudocolour. mCherry staining was performed for quantification of invading cells in primary tumours from experiments using 4599 cells. WSI were scanned using NanoZoomer S210 slide scanner (Hamamatsu). mCherry analysis was performed using QuPath software: an invasive area was delimited underneath the tumour invasive front and positive cells were counted using the positive cell detection plug-in. Assessment of lung colonization assays with 4599 cells was performed on H&E WSI of lungs using QuPath software. Several supervised annotations were performed in each study identifying tumour, normal tissue and whitespace areas. Next, pixel classifier was performed in the whole-image for all the cases using random-tree classification and low resolution.\" - Statistical analysis - \"Unpaired t test, Mannu2013Whitneys test, paired t-test, Wilcoxon test, one-way ANOVA with Dunnett or Tukey post hoc test or Benjamini, Krieger and Yekutieli correction, and Kruskalu2013Wallis with Dunns multiple comparison test were performed using GraphPad Prism (GraphPad Software, Inc). Principal Component Analysis, survival curves estimation based on the Kaplanu2013Meier method and log-rank test were performed using SPSS Statistics (IBM). Limiting dilution assays were analysed using ELDA software (). Data were plotted as graphs showing meanu2009xB1u2009standard error of the mean (s.e.m.); or boxplots where box limits show 25th and 75th percentiles, the horizontal line shows the median, and whiskers show minimum and maximum range of values as indicated in figure legends. P values were calculated using two-tailed tests. P values of less than 0.05 were considered statistically significant. In Figure legends, u201Cnu201D means number of independent biological experiments unless otherwise stated.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - \"Gene expression datasets re-analysed in this study are available from NCBI GEO under accession numbers: GSE23764, GSE8401, GSE7553 and GSE46517. Uncropped immunoblot scans for representative images displayed in the figures are shown in Source data. All other relevant data supporting the findings of this study are available within the article and its and from the corresponding author on reasonable request._ are provided with this paper.\" - \\' \\'- - \\' \\' - No custom codes were used in this study. All codes are indicated in the appropriate Methods sections and references. - \\' \\'- - \"Wang, J. X., Fukunaga-Kalabis, M. & Herlyn, M. Crosstalk in skin: melanocytes, keratinocytes, stem cells, and melanoma. J. Cell Commun. Signal 10, 191u2013 196 (2016)._ _ _ _ Liu, J., Fukunaga-Kalabis, M., Li, L. & Herlyn, M. Developmental pathways activated in melanocytes and melanoma. Arch. Biochem Biophys. 563, 13u201321 (2014)._ _ _ _ Mayor, R. & Theveneau, E. The neural crest. Development 140, 2247u20132251 (2013)._ _ _ _ Rambow, F. et al. Toward minimal residual disease-directed therapy in melanoma. Cell 174, 843u2013855 e819 (2018)._ _ _ _ Somasundaram, R., Villanueva, J. & Herlyn, M. Intratumoral heterogeneity as a therapy resistance mechanism: role of melanoma subpopulations. Adv. Pharm. 65, 335u2013359 (2012)._ _ _ Grzywa, T. M., Paskal, W. & Wlodarski, P. K. Intratumor and intertumor heterogeneity in melanoma. Transl. Oncol. 10, 956u2013 975 (2017)._ _ _ _ Cantelli, G. et al. TGF-beta-induced transcription sustains amoeboid melanoma migration and dissemination. Curr. Biol. 25, 2899u20132914 (2015)._ _ _ _ _ Herraiz C., et al. Reactivation of p53 by a cytoskeletal sensor to control the balance between DNA damage and tumor dissemination. J. Natl Cancer. Inst. 108, djv289 (2015).Friedl, P. Prespecification and plasticity: shifting mechanisms of cell migration. Curr. Opin. Cell Biol. 16, 14u201323 (2004)._ _ _ _ Sanz-Moreno, V. et al. ROCK and JAK1 signaling cooperate to control actomyosin contractility in tumor cells and stroma. Cancer Cell 20, 229u2013245 (2011)._ _ _ _ Sahai, E. & Marshall, C. J. Differing modes of tumour cell invasion have distinct requirements for Rho/ROCK signalling and extracellular proteolysis. Nat. Cell Biol. 5, 711u2013 719 (2003)._ _ _ _ Orgaz, J. L. et al. Diverse matrix metalloproteinase functions regulate cancer amoeboid migration. Nat. Commun. 5, 4255 (2014)._ _ _ _ _ Tozluoglu, M. et al. Matrix geometry determines optimal cancer cell migration strategy and modulates response to interventions. Nat. Cell Biol. 15, 751u2013762 (2013)._ _ _ _ Sanz-Moreno, V. et al. Rac activation and inactivation control plasticity of tumor cell movement. Cell 135, 510u2013 523 (2008)._ _ _ _ Georgouli, M. et al. Regional activation of myosin II in cancer cells drives tumor progression via a secretory cross-talk with the immune microenvironment. Cell 176, 757u2013774 e723 (2019)._ _ _ _ _ Pinner, S. & Sahai, E. Imaging amoeboid cancer cell motility in vivo. J. Microsc. 231, 441u2013445 (2008)._ _ _ _ _ Zhang, Y. & Weinberg, R. A. Epithelial-to-mesenchymal transition in cancer: complexity and opportunities. Front. Med. 12, 361u2013373 (2018)._ _ _ _ Hoek, K. S. et al. Metastatic potential of melanomas defined by specific gene expression profiles with no BRAF signature. Pigment. Cell Res. 19, 290u2013 302 (2006)._ _ _ _ Verfaillie, A. et al. Decoding the regulatory landscape of melanoma reveals TEADS as regulators of the invasive cell state. Nat. Commun. 6, 6683 (2015)._ _ _ _ _ Hoek, K. S. et al. In vivo switching of human melanoma cells between proliferative and invasive states. Cancer Res. 68, 650u2013656 (2008)._ _ _ _ Kemper, K., de Goeje, P. L., Peeper, D. S. & van Amerongen, R. Phenotype switching: tumor cell plasticity as a resistance mechanism and target for therapy. Cancer Res. 74, 5937u20135941 (2014)._ _ _ _ Hu, Y. & Smyth, G. K. ELDA: extreme limiting dilution analysis for comparing depleted and enriched populations in stem cell and other assays. J. Immunol. Methods 347, 70u201378 (2009)._ _ _ _ Pierard, G. E. Cell proliferation in cutaneous malignant melanoma: relationship with neoplastic progression. ISRN Dermatol. 2012, 828146 (2012)._ _ _ _ _ Orgaz, J. L. et al. Myosin II Reactivation and Cytoskeletal Remodeling as a Hallmark and a Vulnerability in Melanoma Therapy Resistance. Cancer Cell 37, 85u2013 103 e109 (2020)._ _ _ _ _ Rambow, F., Marine, J. C. & Goding, C. R. Melanoma plasticity and phenotypic diversity: therapeutic barriers and opportunities. Genes Dev. 33, 1295u20131318 (2019)._ _ _ _ _ Parte, S. C., Batra, S. K. & Kakar, S. S. Characterization of stem cell and cancer stem cell populations in ovary and ovarian tumors. J. Ovarian Res. 11, 69 (2018)._ _ _ _ _ Vandamme, N. & Berx, G. Melanoma cells revive an embryonic transcriptional network to dictate phenotypic heterogeneity. Front Oncol. 4, 352 (2014)._ _ _ _ Cartier-Michaud, A. et al. Matrix-bound PAI-1 supports cell blebbing via RhoA/ROCK1 signaling. PLoS ONE 7, e32204 (2012)._ _ _ _ _ _ Kumper, S. et al. Rho-associated kinase (ROCK) function is essential for cell cycle progression, senescence and tumorigenesis. Elife 5, e12994 (2016)._ _ _ Rodriguez-Hernandez I., Cantelli G., Bruce F. & Sanz-Moreno V. Rho, ROCK and actomyosin contractility in metastasis as drug targets. F1000Res. (2016).Medjkane, S., Perez-Sanchez, C., Gaggioli, C., Sahai, E. & Treisman, R. Myocardin-related transcription factors and SRF are required for cytoskeletal dynamics and experimental metastasis. Nat. Cell Biol. 11, 257u2013 268 (2009)._ _ _ _ _ Luga, V. et al. Exosomes mediate stromal mobilization of autocrine Wnt-PCP signaling in breast cancer cell migration. Cell 151, 1542u20131556 (2012)._ _ _ _ Kabbarah, O. et al. Integrative genome comparison of primary and metastatic melanomas. PLoS ONE 5, e10770 (2010)._ _ _ _ _ _ Riker, A. I. et al. The gene expression profiles of primary and metastatic melanoma yields a transition point of tumor progression and metastasis. BMC Med. Genomics 1, 13 (2008)._ _ _ _ _ Xu, L. et al. Gene expression changes in an animal melanoma model correlate with aggressiveness of human melanoma metastases. Mol. Cancer Res. 6, 760u2013769 (2008)._ _ _ _ _ De, A. Wnt/Ca2+ signaling pathway: a brief overview. Acta Biochim. Biophys. Sin. (Shanghai) 43, 745u2013756 (2011)._ _ _ Habas, R., Kato, Y. & He, X. Wnt/Frizzled activation of Rho regulates vertebrate gastrulation and requires a novel Formin homology protein Daam1. Cell 107, 843u2013854 (2001)._ _ _ _ Liu, W. et al. Mechanism of activation of the Formin protein Daam1. Proc. Natl Acad. Sci. USA 105, 210u2013215 (2008)._ _ _ _ _ Miller, A. J. & Mihm, M. C. Jr. Melanoma. N. Engl. J. Med. 355, 51u2013 65 (2006)._ _ _ _ Marcato, P., Dean, C. A., Giacomantonio, C. A. & Lee, P. W. Aldehyde dehydrogenase: its role as a cancer stem cell marker comes down to the specific isoform. Cell Cycle 10, 1378u2013 1384 (2011)._ _ _ _ Piskounova, E. et al. Oxidative stress inhibits distant metastasis by human melanoma cells. Nature 527, 186u2013191 (2015)._ _ _ _ _ _ Le Gal, K. et al. Antioxidants can increase melanoma metastasis in mice. Sci. Transl. Med. 7, 308re308 (2015)._ Roesch, A. et al. Overcoming intrinsic multidrug resistance in melanoma by blocking the mitochondrial respiratory chain of slow-cycling JARID1B(high) cells. Cancer Cell 23, 811u2013 825 (2013)._ _ _ _ Batlle, E. & Clevers, H. Cancer stem cells revisited. Nat. Med. 23, 1124u20131134 (2017)._ _ _ _ Quintana, E. et al. Efficient tumour formation by single human melanoma cells. Nature 456, 593u2013598 (2008)._ _ _ _ _ _ Cidado, J. et al. Ki-67 is required for maintenance of cancer stem cells but not cell proliferation. Oncotarget 7, 6281u20136293 (2016)._ _ _ _ Lamouille, S., Xu, J. & Derynck, R. Molecular mechanisms of epithelial-mesenchymal transition. Nat. Rev. Mol. Cell Biol. 15, 178u2013196 (2014)._ _ _ _ _ Zhou Y. et al. ROCK2 confers acquired gemcitabine resistance in pancreatic cancer cells by upregulating transcription factor ZEB1. Cancers (Basel) 11, 1881 (2019).Chen, X. et al. Supervillin promotes epithelial-mesenchymal transition and metastasis of hepatocellular carcinoma in hypoxia via activation of the RhoA/ROCK-ERK/p38 pathway. J. Exp. Clin. Cancer Res. 37, 128 (2018)._ _ _ _ _ Cantelli, G., Crosas-Molist, E., Georgouli, M. & Sanz-Moreno, V. TGFBeta-induced transcription in cancer. Semin Cancer Biol. 42, 60u201369 (2017)._ _ _ _ _ De Calisto, J., Araya, C., Marchant, L., Riaz, C. F. & Mayor, R. Essential role of non-canonical Wnt signalling in neural crest migration. Development 132, 2587u20132597 (2005)._ _ _ _ Matthews, H. K. et al. Directional migration of neural crest cells in vivo is regulated by Syndecan-4/Rac1 and non-canonical Wnt signaling/RhoA. Development 135, 1771u20131780 (2008)._ _ _ _ Carmona-Fontaine, C. et al. Contact inhibition of locomotion in vivo controls neural crest directional migration. Nature 456, 957u2013961 (2008)._ _ _ _ _ _ Ossipova, O. & Sokol, S. Y. Neural crest specification by noncanonical Wnt signaling and PAR-1. Development 138, 5441u20135450 (2011)._ _ _ _ _ Ossipova, O., Kerney, R., Saint-Jeannet, J. P. & Sokol, S. Y. Regulation of neural crest development by the formin family protein Daam1. Genesis 56, e23108 (2018)._ _ _ _ _ Webster M. R., Kugel C. & Weeraratna A. T. The Wnts of change: how Wnts regulate phenotype switching in melanoma. Biochim. Biophys. Acta 1856, 244-251 (2015).Asad, M. et al. FZD7 drives in vitro aggressiveness in Stem-A subtype of ovarian cancer via regulation of non-canonical Wnt/PCP pathway. Cell Death Dis. 5, e1346 (2014)._ _ _ _ _ Nishimura, T., Honda, H. & Takeichi, M. Planar cell polarity links axes of spatial dynamics in neural-tube closure. Cell 149, 1084u20131097 (2012)._ _ _ _ Wong P. P. et al. Cancer burden is controlled by mural cell-beta3-integrin regulated crosstalk with tumor cells. Cell 181,1346u2013 1363.e21(2020).Tuncer E., et al. SMAD signaling promotes melanoma metastasis independently of phenotype switching. J Clin Invest 129, 2702u20132716 (2019).Lenos, K. J. et al. Stem cell functionality is microenvironmentally defined during tumour expansion and therapy response in colon cancer. Nat. Cell Biol. 20, 1193u2013 1202 (2018)._ _ _ _ _ Ipenburg N. A., et al. The prognostic value of tumor mitotic rate in children and adolescents with cutaneous melanoma: a retrospective cohort study. J Am Acad Dermatol 82, 910u2013919 (2019).Thompson, J. F. et al. Prognostic significance of mitotic rate in localized primary cutaneous melanoma: an analysis of patients in the multi-institutional American Joint Committee on Cancer melanoma staging database. J. Clin. Oncol. 29, 2199u20132205 (2011)._ _ _ _ Yue, L. et al. Targeting ALDH1 to decrease tumorigenicity, growth and metastasis of human melanoma. Melanoma Res. 25, 138u2013148 (2015)._ _ _ _ Clark, D. W. & Palle, K. Aldehyde dehydrogenases in cancer stem cells: potential as therapeutic targets. Ann. Transl. Med. 4, 518 (2016)._ _ _ _ _ Del Pozo Martin, Y. et al. Mesenchymal cancer cell-stroma crosstalk promotes niche activation, epithelial reversion, and metastatic colonization. Cell Rep. 13, 2456u2013 2469 (2015)._ _ _ _ _ Takaki, T. et al. Actomyosin drives cancer cell nuclear dysmorphia and threatens genome stability. Nat. Commun. 8, 16013 (2017)._ _ _ _ _ Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl Acad. Sci. USA 102, 15545u201315550 (2005)._ _ _ _ _ _ Barbie, D. A. et al. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. Nature 462, 108u2013112 (2009)._ _ _ _ _ _ Bankhead, P. et al. QuPath: open source software for digital pathology image analysis. Sci. Rep. 7, 16878 (2017)._ _ _ _ _ _ \"- - \"This work was supported by Cancer Research UK (CRUK) C33043/A12065 and C33043/A24478 (V.S.-M., I.R.-H., O.M., L.K., G.C. B.F. and J.L.O.); Royal Society RG110591 (V.S.-M.); Barts Charity (V.S.-M., I.R.-H., O.M., J.M. and J.L.O.); Fundacion Alfonso Martin Escudero and Marie Sklodowska-Curie Action, grant agreement No 659022 (I.R.-H.); MRC C97993H (G.C.); The Harry J. Lloyd Charitable Trust (J.L.O. and V.S.-M.); Francis Crick Institute core funding from CRUK FC001112, MRC FC001112 and the Wellcome Trust FC001112 (I.M. and A.P.); CRUK C48390/A21153, CRUK/EPSRC and Wellcome Trust/EPSRC WT 203148/Z/16/Z (G.O.F.); NIHR BRC at Guys and St Thomas NHS Foundation Trust and KCL IS-BRC-1215u201320006, CRUK C30122/A11527 and C30122/A15774, MRC MR/L023091/1, CRUK/NIHR in England/DoH for Scotland, Wales and Northern Ireland ECMC C10355/A15587 (S.N.K.); ISCIII/FEDER u201C Una manera de hacer Europau201D FIS-PI1500711 and PI18/00573 (R.M.M.); CIBERONC CB16/12/0023 (R.M.M and X.M.-G). Views expressed are those of the author(s) and not necessarily those of the NHS, the NIHR, or the Department of Health.\"- - Author notes - \"Gaia CantelliPresent address: Department of Medicine, Division of Hematologic Malignancies and Cellular Therapy, Duke University, Durham, NC, USAJose L. OrgazPresent address: Instituto de Investigaciones Biomedicas u2018Alberto Sols, CSIC-UAM, 28029, Madrid, Spain\" - Authors and Affiliations - \"Barts Cancer Institute, Queen Mary University of London, John Vane Science Building, Charterhouse Square, London, EC1M 6BQ, UKIrene Rodriguez-Hernandez,_Oscar Maiques,_ Leonie Kohlhammer,_Anna Perdrix-Rosell,_Joanne Monger,_Jose L. Orgaz_&_Victoria Sanz-MorenoRandall Division of Cell and Molecular Biophysics, New Hunt s House, Guys Campus, Kings College London, London, SE1 1UL, UKIrene Rodriguez-Hernandez,_Oscar Maiques,_Leonie Kohlhammer,_Gaia Cantelli,_Anna Perdrix-Rosell,_Bruce Fanshawe,_Jose L. Orgaz_&_Victoria Sanz-MorenoTumour Host Interaction Laboratory, The Francis Crick Institute, 1 Midland Rd, London, NW1 1AT, UKAnna Perdrix-Rosell,_Victoria L. Bridgeman_&_Ilaria MalanchiDepartment of Imaging Chemistry and Biology, School of Biomedical Engineering and Imaging Sciences, Kings College London, London, SE1 7EH, UKBruce Fanshawe_&_ Gilbert O. FruhwirthSt Johns Institute of Dermatology, School of Basic & Medical Biosciences, Kings College London and NIHR Biomedical Research Centre at Guys and St Thomas Hospitals and Kings College London, London, SE1 9RT, UKSophia N. KaragiannisDepartment of Pathology, Hospital Universitari de Bellvitge, IDIBELL, lHospitalet de Llobregat, 08908, Barcelona, SpainRosa M. PeninDepartment of Dermatology, Hospital Universitari de Bellvitge, IDIBELL, lHospitalet de Llobregat, 08908, Barcelona, SpainJoaquim MarcolvalDepartment of Dermatology, Hospital Universitari Arnau de Vilanova, University of Lleida, IRB LleidaI, CIBERONC, 25198, Lleida, SpainRosa M. MartiDepartment of Pathology and Molecular Genetics, Hospital Universitari Arnau de Vilanova, University of Lleida, IRB Lleida, CIBERONC, 25198, Lleida, SpainXavier Matias-Guiu\" - \"AuthorsIrene Rodriguez-HernandezYou can also search for this author in _Oscar MaiquesYou can also search for this author in _Leonie KohlhammerYou can also search for this author in _Gaia CantelliYou can also search for this author in _Anna Perdrix-RosellYou can also search for this author in _Joanne MongerYou can also search for this author in _Bruce FanshaweYou can also search for this author in _Victoria L. BridgemanYou can also search for this author in _Sophia N. KaragiannisYou can also search for this author in _Rosa M. PeninYou can also search for this author in _Joaquim MarcolvalYou can also search for this author in _Rosa M. MartiYou can also search for this author in _Xavier Matias-GuiuYou can also search for this author in _Gilbert O. FruhwirthYou can also search for this author in _Jose L. OrgazYou can also search for this author in _Ilaria MalanchiYou can also search for this author in _Victoria Sanz-MorenoYou can also search for this author in _\" - Contributions - V.S.-M. was principal investigator, designed the study, supervised experiments, and wrote the paper; I.R.-H. designed the study, performed most experiments, and wrote the paper. O.M., L.K., G.C., A.P., J.M., B.F., V.L.B. and J.L.O. performed experiments; R.M.P., J.M., R.M.M. and X.M.-G. provided human tissue samples; S.N.K. supervised the in vivo experiment performed by J.L.O.; G.O.F. supervised in vivo experiments performed by B.F.; I.M. designed the study and supervised in vivo experiments performed by A.P. and V.L.B. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks Vito Rebecca, Lukas Sommer and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License ().\" - Bone formation by osteogenic microgels in critical-sized defect - \"The ability of microgels to regenerate bone was then evaluated in an established murine full-thickness critical-sized cranial defect model. A 3_mm wide nonsuture-associated parietal bone defect on an adult mouse (>u200912_weeks old) is shown to not heal spontaneously during the lifetime of the animal. Pure gelatin microgels and gelatin microgels with 0.25% chitosan, 1% HA were fabricated, seeded with murine MSC (1.0u2009xD7u200910 cells/200 xB5L of microgels), and cultured under osteogenic conditions for 12_days. A fibrin carrier gel was then used to concentrate the microgels and contain them within the critical-sized defect in mice. Animals that received acellular fibrin carrier gel alone served as control. Gross examination of the 3D reconstructed microCT images showed new bone formation within the defect resulting in 0%,u2009~u200965% andu2009 >u200995% defect closure by week 12 in control, microgel, and microgel with HA groups, respectively (Fig._A). Bone morphometric analysis showed a steady increase in new bone volume and bone mineral density (BMD) of the construct over 12_ weeks in both microgel conditions (Fig._B). At week 12, a total of 0.25u2009 xB1u20090.18 mm of new bone volume and 0.24u2009xB1u20090.12_mg of HA/mm mineral density was seen in microgels (Fig._B). A significantly higher total new bone volume of 0.7u2009xB1u20090.28 mm and a mineral density of 0.34u2009 xB1u20090.09_mg of HA/mm were seen in HA microgels. No noticeable new bone formation was seen in control conditions. A detailed statistical comparison of bone morphometry parameters between conditions is listed in Table .\" - \"Figure 10Murine critical-sized calvarial defect. (A) 3D reconstructed microCT images of the mice crania over time, showing the defect being bridged. Control (CNT) animals received acellular fibrin carrier gel, the second group received gelatin microgels seeded with MSC, and the last group received gelatin microgels with HA seeded with MSC. (B) Bone morphometric analysis of the newly formed bone shows a steady increase in new bone volume and mineral density (BMD) in conditions containing MSC-seeded microgels. nu2009=u20094 for each condition.\" - \"Table 2 Holm-Sidak comparison of bone morphometry parameters between conditions. (Controlu2009=u2009No microgel control, Gu2009+u2009MSCu2009=u2009Gelatin microgels seeded with MSC, GCHu2009+u2009MSCu2009=u2009Gelatin microgels with chitosan and HA seeded with MSC).\" - \"Histological analysis of the demineralized skulls shows control condition treatment with no bridging, while the gelatin microgels seeded with MSC showing partial bridging with collagen and elastic fibers (Fig._). The gelatin-chitosan-HA microgels exhibited the highest bridging with the osteoid formation and collagenous matrix deposition (Fig._). Together, the osteogenic microgels enabled MSC to bridge the critical-sized defect with mineralized collagen-rich osteoid bone.\" - \"Figure 11Gelatin microgels with MSC regenerate bone in a critical-sized calvarial defect. Movats Pentachrome staining of the demineralized bone sections shows healing of the critical-sized defect in various treatment conditions. A yellow stain indicates collagen fibers, and a black stain indicates elastic fibers. (Left) The control condition showed no bridging, while the gelatin microgels seeded with MSC showed partial bridging with collagen and elastic fibers. The gelatin-chitosan-HA microgels exhibited the highest bridging with the osteoid formation and collagenous matrix deposition. (Right) Magnified images of the defect area of corresponding treatments shown on the left.\"- - The hypo-immunogenicity of MSC and their ability to self-renew and differentiate into bone, cartilage, and adipose tissues have made them an attractive adult source for tissue engineering. The MSC are also known for their immunomodulatory and angiogenic factor secretion, further enriching the regenerative environment. Here we investigated the application of an off-the-shelf nanoporous microgel for MSC-based bone tissue engineering. - Tunable microgels for natural materials - \"The microgels primary constituent is gelatin, a well-characterized material consisting of water-soluble polypeptides derived from partially denatured collagen. The polypeptides have integrin-binding sites as parent collagen but are less immunogenic due to the removal of aromatic groups. Gelatin has a solu2013gel transition near 30_xB0C. Hence, to stabilize them in physiological conditions, it is necessary to crosslink the matrix, which also helps integrate other materials such as chitosan into the matrix. The method and extent of the crosslinking immensely influence the mechanical properties, swelling ratio, cytotoxicity, antigenicity, and rate of degradation of the hydrogel. Chemical crosslinkers such as formaldehyde, glutaraldehyde, and glyceraldehyde are commonly used to stabilize gelatin matrix and enhance resistance to thermal degradation, but their cytotoxicity is a significant concern. We used a naturally occurring crosslinking agent, genipin, obtained from the fruits of Gardenia jasminoides. They are significantly less cytotoxic and genotoxic with an intravenous LD50 of 153_mg/kg, which is tenfold higher than glutaraldehyde (mouse LD50_15.4_mg/kg i.v.). The genipin-crosslinked gelatin is enzymatically degradable, and the rate is concentration dependant. Further, genipin crosslinking allows controlling the degree of crosslinking through concentration and incubation time. Further, the swelling ratio and the polymer density of the microgels vary with the formulation. When gelatin and HA are added to the formulation, the resulting microgel exhibits a higher swelling ratio and polymer density than homogenous gelatin microgels, presumably by increasing the amorphous nature of the matrix.\" - \"We tested the propeller and parallel-plate impeller (aka flat-blade turbine) for emulsification to fabricate the microgels. Both are well-suited for the 100 cS (0.1_kg/m-s) PDMS used, but they create axial and radial flow patterns, respectively. The parallel-plate impeller had a bimodal size distribution due to solid body rotation and limited liquid mixing above and below the impeller. The parallel-plate impeller is efficient in power consumption and dispersion, but enhancements such as baffles may be necessary to avoid solid body rotation and a large central surface vortex. On the other hand, the axial flow pattern created by the propeller impeller yielded microgels with sizes showing normal distribution and a linear decrease in mean and median values with an increase in mixing speed. Further, the desired size range of the microgels was achieved with less turbulence in the mixing vessel. Hence the propeller impeller yielded the best outcomes for the microgel fabrication and was therefore chosen for the rest of the studies.\" - Microgels create a conducive environment for osteogenesis and vascularization - \"The material constituents and physical parameters of the microgel greatly influence the osteogenicity of the microenvironment. Tissue softness is a u201Ccolligativeu201D property as the water content of the tissue is inversely proportional to the tissue elasticity and collagen content. The average bone matrix hydration is around 40% compared to that of the brain, which is around 90%. The nanoporous structure and the high polymer density of the microgels arising from the highly cross-linked polypeptides (~u200990% degree of crosslinking) enable the matrix to hold large amounts of bound water and swellu2009~u2009fivefold in volume. The modest water content and high collagen density give rise to the stiffness (0.1u201330_MPa) and rigidity of the bone, which subsequently regulate bone function. Even during fractures, mechanobiological studies point to the mechanics of the callus (specific stiffness range of 0.1u20131_MPa) to be a crucial promotor of osteogenic differentiation of progenitor cells and neovascularization. Similarly, a stiffness ofu2009~u2009100_kPa is shown to regulate endothelial progenitor cell differentiation and vascularization. Hence, the measured stiffness of hydrated microgels (<u2009200_kPa) falls in a range that is conducive to both osteogenesis and vasculogenesis.\" - Further, in the body, MSC undergo osteogenesis when attached on the surface of the stiff bone with the soft marrow above. The 2D surfaces appropriately mimic the process of MSC on the bone surface and differentiation into osteoblasts. The microgels with a high polymer density, moderate stiffness, and modest hydration show a similar ability to support MSC attachment, proliferation, and osteogenesis. To enhance their osteogenicity, we tailored the formulation to mirror the native bone matrix, which increases osteogenic differentiation in MSC. When we supplemented the microgel matrix with chitosan and HA, it enhanced MSC differentiation even while retaining the properties of the pure gelatin microgels. Apart from material composition, the osteogenicity of the microgels can also be attributed to their geometry. A micron-scale curvature is shown to naturally induce cytoskeletal stress, which leads to the osteogenic differentiation of MSC via RhoA/ROCK pathways. - Dual-phase constructs promote synergism - Aside from the parenchymal cells, vessel growth and blood flow are vital in determining bone repair and osteogenesis. Distinct capillary subtypes in the bone are known to secrete angiocrine and osteogenic factors and maintain osteoprogenitors. Vessel growth and osteogenesis are coupled through molecular crosstalk between endothelial cells and osteoblasts. To harness this synergistic association, we mixed the osteogenic microgels with vasculogenic microgels containing endothelial cells and pericytes. Typically, cultured endothelial cells quickly lose their ability to adopt tissue-specific features required for organ development and regeneration. Although, this adaptability may be revived with a suitable blend of matrix proteins. We show that endothelial cells seeded in gelatin-only microgels retain their ability to sprout and form vascular plexi. But they might need an appropriate pericyte-like cell for providing the growth factor as robust sprouting was absent when endothelial cells were seeded alone. Similarly, the endothelial cells failed to sprout when the fibroblasts were seeded in the embedding matrix instead of the microgel (data not shown). Seemingly the pericytes also function better when provided with the support of the microgel, although more mechanistic studies are warranted to elucidate these effects. When we combined the vasculogenic microgels with osteogenic ones, they sustained endothelial sprouting while supporting the osteogenic phenotype. Further, the dual-phase constructs provide the flexibility to optimize formulations and culture mature the components independently without limiting each other. At the same time, they can be easily combined when necessary to form a single homogenous construct that can promote vascularized tissue regeneration. These dual-phasic constructs are valuable, especially to heal complex fractures that are often ischemic and require concurrent vascularization. - Injectable microgels promote bone regeneration - Minimally invasive applications can reduce surgery-related complications, including infection and long recovery times. Further, the microscale modules allow them to be injectable and conformally fill complex defects. When we injected our microgels using a hypodermic needle of varying sizes, we saw no change in compression force or clogging of microgels. The shear-induced damage to the cells attached to the surface is also negligible. These injectable microgels are an off-the-shelf alternative to the existing bone grafts. When combined with autologous blood, plasma, or bone marrow, their very high swelling ratios can enable them to quickly soak in growth factors while attracting adherent mesenchymal progenitor cells onto their surface. In addition, carrier gels such as fibrin glue can also be used to concentrate the microgels in the wound site. - \"To validate the ability of microgels in regenerating bone, we used a murine calvarial defect model. Although our calvarial model generates a critical-sized defect, it does not create an ischemic wound. So, we validated the osteogenic gelatin microgels alone without a vasculogenic component. As defined by the FDA, a nonunion is a fracture that persists for a minimum of nine months without any signs of healing for three months. We show that our implanted microgels supported robust bone regeneration and showed continual progress with 65u201395% defect closure by week 12, depending on the microgel formulation. As hypothesized, the addition of chitosan and HA to the gelatin matrix significantly accelerated the new bone formation and defect closure. The histological data also show that some of the microgels are present at the implant site by week 12. This is consistent with our in vitro degradation analysis, which shows a moderation degradation rate by macrophages. In addition, histological analysis of explants confirmed the effectiveness of the osteogenic microgel in regenerating bone with a rich collagenous matrix indicative of robust osteogenic differentiation. Overall, our data demonstrate the therapeutic ability of the osteogenic microgels, and further studies on an ischemic defect model are warranted to validate concurrent vascularization.\"- - Together, our work shows that microgels can be used to augment the differentiated function of MSC and endothelial cells to fabricate a vascularized bone construct. The microgel matrix is customizable to mimic tissue-specific functions while allowing minimally invasive delivery. Microgels support robust bone regeneration and healing of critical-sized defects, as evident from our mouse studies. Our work lays the foundation to establish principles of designing multiphasic scaffolds with tissue-specific biophysical and biochemical properties for regenerating vascularized and interfacial tissues.- - \\' \\' - All data generated or analyzed during this study are included in this published article. - \\' \\'- - \"Henkel, J. et al. Bone regeneration based on tissue engineering conceptionsu2014 a 21st century perspective. Bone Res. 1, 216u2013248 (2013)._ _ _ _ _ Calori, G. M., Albisetti, W., Agus, A., Iori, S. & Tagliabue, L. Risk factors contributing to fracture non-unions. Injury 38(Suppl 2), S11-18 (2007)._ _ _ Roberts, T. T. & Rosenbaum, A. J. Bone grafts, bone substitutes and orthobiologics. Organogenesis 8, 114u2013124 (2012)._ _ _ _ Dimitriou, R., Mataliotakis, G. I., Angoules, A. G., Kanakaris, N. K. & Giannoudis, P. V. Complications following autologous bone graft harvesting from the iliac crest and using the RIA: A systematic review. Injury 42(Suppl 2), S3-15 (2011)._ _ _ Arrington, E. D., Smith, W. J., Chambers, H. G., Bucknell, A. L. & Davino, N. A. Complications of iliac crest bone graft harvesting. Clin. Orthop. Relat. Res. 329, 300u2013309 (1996)._ _ Oryan, A., Alidadi, S., Moshiri, A. & Maffulli, N. Bone regenerative medicine: classic options, novel strategies, and future directions. J. Orthop. Surg. Res. 9, 18 (2014)._ _ _ _ Janicki, P. & Schmidmaier, G. What should be the characteristics of the ideal bone graft substitute? Combining scaffolds with growth factors and/or stem cells. Injury 42, S77u2013S81 (2011)._ _ _ Chatterjea, A., Meijer, G., van Blitterswijk, C. & de Boer, J. Clinical application of human mesenchymal stromal cells for bone tissue engineering. Stem cells international 2010, 215625 (2010)._ _ _ _ Khademhosseini, A., Langer, R., Borenstein, J. & Vacanti, J. P. Microscale technologies for tissue engineering and biology. Proc. Natl. Acad. Sci. U.S.A. 103, 2480u2013 2487 (2006)._ _ _ _ _ _ Annamalai, R. T., Armant, D. R. & Matthew, H. W. T. A glycosaminoglycan based, modular tissue scaffold system for rapid assembly of perfusable, high cell density, engineered tissues. PLoS ONE 9, e84287 (2014)._ _ _ Chang, B., Ahuja, N., Ma, C. & Liu, X. Injectable scaffolds: Preparation and application in dental and craniofacial regeneration. Mater. Sci. Eng. R. Rep. 111, 1u201326 (2017)._ _ _ _ Liu, M. et al. Injectable hydrogels for cartilage and bone tissue engineering. Bone Res. 5, 17014 (2017)._ _ _ _ _ Dreifke, M. B., Ebraheim, N. A. & Jayasuriya, A. C. Investigation of potential injectable polymeric biomaterials for bone regeneration. J. Biomed. Mater. Res., Part A 101, 2436u20132447 (2013)._ _ Annamalai, R. T. et al. Injectable osteogenic microtissues containing mesenchymal stromal cells conformally fill and repair critical-size defects. Biomaterials 208, 32u201344 (2019)._ _ _ _ _ Bicer, M., Cottrell, G. S. & Widera, D. Impact of 3D cell culture on bone regeneration potential of mesenchymal stromal cells. Stem Cell Res. Ther. 12, 31 (2021)._ _ _ _ Grayson, W. L. et al. Stromal cells and stem cells in clinical bone regeneration. Nat. Rev. Endocrinol. 11, 140u2013150 (2015)._ _ _ _ _ Engler, A. J., Sen, S., Sweeney, H. L. & Discher, D. E. Matrix elasticity directs stem cell lineage specification. Cell 126, 677u2013689 (2006)._ _ _ _ McBeath, R., Pirone, D. M., Nelson, C. M., Bhadriraju, K. & Chen, C. S. Cell shape, cytoskeletal tension, and RhoA regulate stem cell lineage commitment. Dev. Cell 6, 483u2013 495 (2004)._ _ _ _ Ruiz, S. A. & Chen, C. S. Emergence of patterned stem cell differentiation within multicellular structures. Stem Cells 26, 2921u20132927 (2008)._ _ _ Huang, L., Abdalla, A. M. E., Xiao, L. & Yang, G. Biopolymer-based microcarriers for three-dimensional cell culture and engineered tissue formation. Int. J. Mol. Sci. 21, 1895-1916 (2020)._ _ _ _ Offeddu, G. S., Ashworth, J. C., Cameron, R. E. & Oyen, M. L. Structural determinants of hydration, mechanics and fluid flow in freeze-dried collagen scaffolds. Acta Biomater. 41, 193u2013203 (2016)._ _ _ _ Cohen, B.A. Point of view: How should novelty be valued in science? eLife 6, e28699 (2017)._ _ _ _ BarrxE8re, F., Mahmood, T. A., de Groot, K. & van Blitterswijk, C. A. Advanced biomaterials for skeletal tissue regeneration: Instructive and smart functions. Mater. Sci. Eng. R. Rep. 59, 38u201371 (2008)._ _ Bello, A. B., Kim, D., Kim, D., Park, H. & Lee, S. H. Engineering and functionalization of gelatin biomaterials: From cell culture to medical applications. Tissue Eng. Part B Rev. 26, 164u2013180 (2020)._ _ _ _ Chicatun, F., Griffanti, G., McKee, M. D. & Nazhat, S. N. In Biomedical Composites 2nd edition. (ed. L. Ambrosio) 163u2013198 (Woodhead Publishing, 2017).Dubey, D. K. & Tomar, V. Role of the nanoscale interfacial arrangement in mechanical strength of tropocollagenu2013hydroxyapatite-based hard biomaterials. Acta Biomater. 5, 2704u20132716 (2009)._ _ _ _ Kilroy, G., Dietrich, M., Wu, X., Gimble, J. M. & Floyd, Z. E. In Adipose-Derived Stem Cells 137u2013146 (Springer, 2018).Levi, B. et al. Human adipose derived stromal cells heal critical size mouse calvarial defects. PLoS ONE 5, e11177 (2010)._ _ _ _ _ Park, E., Hart, M. L., Rolauffs, B., Stegemann, J. P. & Annamalai, T. R. Bioresponsive microspheres for on-demand delivery of anti-inflammatory cytokines for articular cartilage repair. J. Biomed. Mater. Res. Part A 108, 722u2013733 (2020)._ _ _ Liang, J., Teng, F., Chou, T.-M. & Libera, M. Measuring microgel swell ratio by cryo-SEM. Polymer 116, 1u20134 (2017)._ _ _ Sandin, J. N., Aryal, S. P., Wilkop, T., Richards, C. I. & Grady, M. E. Near simultaneous laser scanning confocal and atomic force microscopy (conpokal) on live cells. J. Vis. Exp. 162, e61433 (2020)._ Lin, D. C., Dimitriadis, E. K. & Horkay, F. Robust strategies for automated AFM force curve analysisu2014I. Non-adhesive indentation of soft, inhomogeneous materials. J. Biomech. Eng. 129, 430u2013440 (2007)._ _ _ Kilkenny, C., Browne, W. J., Cuthill, I. C., Emerson, M. & Altman, D. G. Improving bioscience research reporting: The ARRIVE guidelines for reporting animal research. PLoS Biol. 8, e1000412 (2010)._ _ _ _ Aalami, O. O. et al. Applications of a mouse model of calvarial healing: differences in regenerative abilities of juveniles and adults. Plast. Reconstruct. Surg. 114, 713-720 (2004)._ _ Caplan, A. I. Adult mesenchymal stem cells for tissue engineering versus regenerative medicine. J. Cell. Physiol. 213, 341u2013347 (2007)._ _ _ _ Samorezov, J. E., Headley, E. B., Everett, C. R. & Alsberg, E. Sustained presentation of BMP-2 enhances osteogenic differentiation of human adipose-derived stem cells in gelatin hydrogels. J. Biomed. Mater. Res. Part A 104, 1387u20131397 (2016)._ _ _ Solorio, L. D., Vieregge, E. L., Dhami, C. D., Dang, P. N. & Alsberg, E. Engineered cartilage via self-assembled hMSC sheets with incorporated biodegradable gelatin microspheres releasing transforming growth factor-u03B21. J. Controll. Rel. 158, 224u2013 232 (2012)._ _ _ Lou, X. & Chirila, T. V. Swelling behavior and mechanical properties of chemically cross-linked gelatin gels for biomedical use. J. Biomater. Appl. 14, 184u2013191 (1999)._ _ _ _ Lynn, A. K., Yannas, I. V. & Bonfield, W. Antigenicity and immunogenicity of collagen. J. Biomed. Mater. Res. B Appl. Biomater. 71B, 343u2013354 (2004)._ _ _ Nomori, H., Horio, H., Morinaga, S. & Suemasu, K. Gelatin-resorcinol-formaldehyde-glutaraldehyde glue for sealing pulmonary air leaks during thoracoscopic operation. Ann. Thorac. Surg. 67, 212u2013216 (1999)._ _ _ _ Sisson, K., Zhang, C., Farach-Carson, M. C., Chase, D. B. & Rabolt, J. F. Evaluation of cross-linking methods for electrospun gelatin on cell growth and viability. Biomacromol 10, 1675u20131680 (2009)._ _ _ Vandelli, M. A., Rivasi, F., Guerra, P., Forni, F. & Arletti, R. Gelatin microspheres crosslinked with D, L-glyceraldehyde as a potential drug delivery system: preparation, characterisation, in vitro and in vivo studies. Int. J. Pharm. 215, 175u2013 184 (2001)._ _ _ _ Tsai, C. C., Huang, R. N., Sung, H. W. & Liang, H. C. In vitro evaluation of the genotoxicity of a naturally occurring crosslinking agent (genipin) for biologic tissue fixation. J. Biomed. Mater. Res. 52, 58u201365 (2000)._ _ _ _ Harada, M., Tenmyo, N., Aburada, M. & Endo, T. Phagmacological studies of gardeniae fructus. I. Effect of geniposide and genipin on the biliary excretion, the gastric juice secretion, and the gastric contraction, and other pharmacological actions (authors transl). Yakugaku Zasshi 94, 157u2013 162 (1974)._ _ _ _ Annamalai, R. T. et al. Harnessing macrophage-mediated degradation of gelatin microspheres for spatiotemporal control of BMP2 release. Biomaterials 161, 216u2013227 (2018)._ _ _ _ _ Coker, A. K. Modeling of Chemical Kinetics and Reactor Design. (eds. A.K. Coker & C.A. Kayode) 552u2013662 (Gulf Professional Publishing, Woburn, 2001).Smith, L. R., Cho, S. & Discher, D. E. Stem cell differentiation is regulated by extracellular matrix mechanics. Physiology (Bethesda) 33, 16u2013 25 (2018)._ Ghiasi, M. S., Chen, J., Vaziri, A., Rodriguez, E. K. & Nazarian, A. Bone fracture healing in mechanobiological modeling: A review of principles and methods. Bone Rep. 6, 87u2013100 (2017)._ _ _ _ Xue, C. et al. Matrix stiffness regulates arteriovenous differentiation of endothelial progenitor cells during vasculogenesis in nude mice. Cell Prolif. 52, e12557u2013e12557 (2019)._ _ _ Huebsch, N. et al. Harnessing traction-mediated manipulation of the cell/matrix interface to control stem-cell fate. Nat. Mater. 9, 518u2013526 (2010)._ _ _ _ _ _ Mathews, S., Gupta, P., Bhonde, R. & Totey, S. Chitosan enhances mineralization during osteoblast differentiation of human bone marrow-derived mesenchymal stem cells, by upregulating the associated genes. Cell Prolif. 44, 537u2013549 (2011)._ _ _ _ _ Sridharan, R., Genoud, K. J., Kelly, D. J. & OBrien, F. J. Hydroxyapatite particle shape and size influence MSC osteogenesis by directing the macrophage phenotype in collagen-hydroxyapatite scaffolds. ACS Appl. Bio Mater. 3, 7562u20137574 (2020)._ _ _ _ Wang, Y.-K. et al. Bone morphogenetic protein-2-induced signaling and osteogenesis is regulated by cell shape, RhoA/ROCK, and cytoskeletal tension. Stem Cells Dev 21, 1176u20131186 (2012)._ _ _ _ Arnsdorf, E. J., Tummala, P., Kwon, R. Y. & Jacobs, C. R. Mechanically induced osteogenic differentiationu2014the role of RhoA, ROCKII and cytoskeletal dynamics. J. Cell Sci. 122, 546u2013553 (2009)._ _ _ _ _ Kusumbe, A. P., Ramasamy, S. K. & Adams, R. H. Coupling of angiogenesis and osteogenesis by a specific vessel subtype in bone. Nature 507, 323u2013328 (2014)._ _ _ _ _ _ Ramasamy, S. K., Kusumbe, A. P., Wang, L. & Adams, R. H. Endothelial Notch activity promotes angiogenesis and osteogenesis in bone. Nature 507, 376u2013 380 (2014)._ _ _ _ _ _ Palikuqi, B. et al. Adaptable haemodynamic endothelial cells for organogenesis and tumorigenesis. Nature 585, 426u2013432 (2020)._ _ _ _ _ _ Cunningham, B. P., Brazina, S., Morshed, S. & Miclau, T. 3rd. Fracture healing: A review of clinical, imaging and laboratory diagnostic options. Injury 48(Suppl 1), S69-s75 (2017)._ _ _ Patrick, M. D. & Annamalai, T. R. Licensing microgels prolong the immunomodulatory phenotype of mesenchymal stromal cells. Front. Immunol. 13, 987032 (2022)._ _ _ _ _ \"- - \\'Research reported in this publication was supported in part by the National Institute of Arthritis and Musculoskeletal and Skin Diseases (NIAMS) Award Number R21AR078447, National Institute of General Medical Sciences of the National Institutes of Health under Award Numbers P20GM130456 and P20GM103436-20 (KY IDeA Networks of Biomedical Research Excellence), National Center for Research Resources and the National Center for Advancing Translational Sciences of the National Institutes of Health under Award Number UL1TR001998, and Orthopedic Trauma Association (OTA, Grant Number: 6889). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health or other grant funding agencies.\\'- - Author notes - \\'These authors contributed equally: Matthew D. Patrick and Jeremy F. Keys.\\' - Authors and Affiliations - \"Department of Biomedical Engineering, University of Kentucky, 760 Press Avenue, 138 Healthy Kentucky Research Building, Lexington, KY, 40536, USAMatthew D. Patrick,_Jeremy F. Keys,_Harshini Suresh Kumar_&_Ramkumar T. Annamalai\" - \"AuthorsMatthew D. PatrickYou can also search for this author in _Jeremy F. KeysYou can also search for this author in _Harshini Suresh KumarYou can also search for this author in _Ramkumar T. AnnamalaiYou can also search for this author in _\" - Contributions - M.D.P., J.F.K., and R.T.A. wrote the main manuscript text and prepared figures -. H.S.K. prepared Figs. and . All authors reviewed the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License ().\" - Endogenous NESPRIN proteins are clients of the reductase TMX4 in cellula - \"The results shown so far led us to explore the possibility that conditions of cellular stress activate an enzyme-driven pathway that remodels the NE. This would promote/allow a substantial increase of the distance between ONM and INM upon reduction of the intermolecular disulfide bond linking the perinuclear domains of SUN and NESPRIN proteins, thus disassembling the LINC complexes. In our model (Fig._), the dissociation of SUN proteins from NESPRIN proteins would allow the increase of the width between the two membranes to the values measured in cells exposed to the pharmacologic treatment (Fig._, Supplementary Fig._).\" - \"More than 20 oxidoreductases of the protein disulfide isomerase (PDI) superfamily populate the ER. The intracellular distribution of a few of them, i.e., endogenous PDI, ERp57, ERp72 and ectopically expressed TMX3, TMX4, TMX5 are shown as examples (Fig._, respectively). The type I ER membrane proteins TMX3, TMX4 and TMX5 also decorate the NE, which is continuous with the ER membrane (Fig._ and insets).\" - \"Fig. 7: Distribution of PDI family members in the ER and in the NE.a CLSM analysis to monitor the subcellular distribution of endogenous PDI in MEF. Scale bars: 10u2009xB5m. b Same as (a) for ERp57. c Same as (a) for ERp72. d Same as (a) for ectopically expressed TMX3-V5. e Same as (a) for ectopically expressed TMX4-V5. f Same as (a) for ectopically expressed TMX5-V5. g RT-TEM micrograph showing immunogold labeling of endogenous TMX4 in the ER (arrowheads) and in the ONM of MEF (arrows). h RT-TEM micrograph showing immunogold labeling of endogenous TEX264 in the ER (arrowheads), in the ONM and INM of MEF (arrows). The CLSM experiments were performed once.\" - \"The biological functions of TMX3, TMX4 and TMX5 remain to be established. However, some information is available in the literature. TMX3 is characterized by a canonical cysteine-glycine-histidine-cysteine (CGHC) active site sequence and acts as an oxidase in vitro. TMX4 has a non-canonical cysteine-proline-serine-cysteine (CPSC) active site sequence. The proline residue at position 2 destabilizes the disulfide state and favors the di-thiol reduced form of the active site. Consistently, TMX4 acts as reductase in vitro and is therefore a good candidate as the enzyme involved in LINC complexes disassembly upon reduction of the intermolecular disulfide bonds that covalently link NESPRIN and SUN proteins. Moreover, TMX4 has a peculiar enrichment in the NE (Fig._ and Inset). Previous analyses by indirect immunofluorescence led to postulate a distribution of TMX4 in the ER and in the INM. Our analyses by immunoelectron microscopy confirm the localization of endogenous TMX4 in the ER (arrowheads in Fig._) but reveal that TMX4 distributes in the ONM (arrows in Fig._). As a control, TEX264, a protein involved in ER-phagy and in DNA repair distributes in the ER, in the INM and in the ONM (Fig._). Finally, TMX5 is an unconventional member of the PDI superfamily, which has a cysteine-arginine-phenylalanine-serine (CRFS) catalytic site and is likely to engage clients in long-living disulfide-bonded complexes.\" - \"To assess the client specificity of TMX3, TMX4 and TMX5 in cellula, we generated mutant forms of the enzymes, where the last cysteine residue of the TMX s CXXC catalytic sites has been mutated to alanine. This mutation stabilizes the mixed disulfide that oxidoreductases establish with client proteins. Consequently, client proteins remain disulfide-bonded to the oxidoreductase and can be identified upon co-immunoprecipitation and mass spectrometry analyses. Briefly, V5-tagged versions of TMX3 and its trapping mutant TMX3C56A, TMX4 and its trapping mutant TMX4C67A and of TMX5, a natural trapping mutant, were individually expressed in HEK293 cells. After immunoisolation from cell lysates with anti-V5 antibodies, the immunocomplexes containing TMX3-V5 (Fig._, lanes 2 and 8), TMX3C56A-V5 (lanes 3 and 9), TMX4-V5 (lanes 4 and 10), TMX4C67A-V5 (lanes 5 and 11) or TMX5-V5 (lanes 6 and 12) were separated in non-reducing/reducing SDS-polyacrylamide gel, which was subsequently silver stained (Fig._). TMX3C56A, TMX4C67A and TMX5 are trapped in mixed disulfides with several cellular polypeptides (Fig._ , lanes 3, 5 and 6 respectively, red rectangles). The disulfide-bonded complexes are disassembled and disappear from this region of the gel when the immunocomplexes are run in the gel under reducing conditions (Fig._, lanes 9, 11 and 12, yellow rectangles). As expected, the disulfide-bonded complexes are significantly less abundant in the lanes loaded with immunoisolates of cells expressing the active forms of TMX3 and TMX4, which rapidly release their clients (Fig._, lanes 2 and 4, black rectangles).\" - \"Fig. 8: Endogenous clients of TMX3, TMX4 and TMX5.a Anti V5-antibody was used to immunoisolate from HEK293 cell lysates V5-tagged versions of TMX3 (lane 2), TMX3C56A (lane 3), TMX4 (lanes 4), TMX4C67A (lanes 5) and TMX5 (lanes 6). Part of the immunoisolates has been separate in non-reducing/reducing gel (lanes 1u20136 and 7u201312, respectively). The gel has been silver stained. Uncropped gel in Supplementary Fig._. The polypeptide bands in the black and red rectangles are disulfide-bonded complexes (mixed disulfides) associated with the respective TMX protein. The complexes disappear (yellow boxes), i.e., are disassembled, when the samples are run under reducing conditions (lanes 7u201312). Part of the immunoisolates has been processed for mass spectrometry (see u201CMethodsu201D section) to determine the composition of the mixed disulfides and identify the endogenous proteins trapped in mixed disulfides with the given TMX protein. b Graphical representation of Supplementary Table_. Red dots show NESPRIN proteins.\" - \"The cellular proteins captured in disulfide-bonded complexes with TMX3C56A, TMX4C67A and TMX5 were identified by mass spectrometry (Supplementary Table_, Fig._ ). Notably for the work presented in this submission, these analyses reveal endogenous NESPRIN proteins (i.e., NESPRIN2 (SYNE2) and NESPRIN1 (SYNE2)) as major TMX4 clients (Supplementary Table_, Fig._, red). NESPRIN proteins were not captured by the TMX1C59A trapping mutant that we investigated in a previous study, nor by the TMX3C56A trapping mutant (Supplementary Table_, Fig._), or by TMX5 (Supplementary Table_, Fig._) confirming the different clients subset for the four membrane-associated PDI family members.\" - \"Notably, despite the higher expression level in HEK293 cells of SUN1 (33.3 normalized transcripts per million (nTPM)) and SUN2 (54.7 nTPM) compared to NESPRIN1 (4 nTPM), NESPRIN2 (3.6 nTPM) and NESPRIN3 (1 nTPM), SUN proteins were not trapped in mixed disulfides, showing that SUN proteins are not clients of the TMX4 reductase. All in all, TMX4 has reductase activity, distributes in the ONM (Fig._) and endogenous NESPRIN proteins are TMX4 clients in living cells (Supplementary Table_, Fig._). Altogether, these results support a possible involvement of TMX4 in the modulation of the LINC complex via its reductase activity during ER stress.\" - \"TMX4 forms mixed disulfides with NESPRIN3u03B1\" - \"The resolution of an inter-molecular disulfide bond occurs upon nucleophilic attack of a reductase catalytic cysteine to a cysteine residue of one of the partners. Our data imply that TMX4 attacks (i.e., forms mixed disulfides with) NESPRIN proteins preferentially. To confirm that TMX4 preferentially engages NESPRIN proteins in mixed disulfides, HEK293 cells were mock-transfected (Fig._ , lanes 1, 5, 9), co-transfected with HALO-NESPRIN3u03B1 and GFP-SUN1 (Fig._ , lanes 2, 6, 10), with HALO-NESPRIN3u03B1, GFP-SUN1 and TMX4C67A-V5 (Fig._ , lanes 3, 7, 11), or with HALO-NESPRIN3u03B1, GFP-SUN1 and TMX4-V5 (Fig._, lanes 4, 8, 12). Cell lysates were separated in reducing SDS-polyacrylamide gels and processed for western blotting to verify the expression of the polypeptides. The PVDF membranes were probed with antibodies to the V5 epitope (Fig._, lanes 1u20134) to confirm the expression of TMX4C67A-V5 (TMX4*, lane 3) and TMX4-V5 (TMX4, lane 4). The ectopic expression of GFP-SUN1 was confirmed with antibodies to GFP (Fig._, lanes 6u20138). Likewise, the ectopic expression of HALO-NESPRIN3u03B1 was confirmed with antibodies to HALO (Fig._, lanes 10u201312).\" - \"Fig. 9: In cellula assessment of TMX4 engagement in mixed disulfides with NESPRIN3u03B1 .a Lysates of cells mock-transfected (lanes 1, 5, 9), expressing GFP-SUN1 and HALO-NESPRIN3u03B1, TMX4C67A-V5 (lanes 2, 6, 10), GFP-SUN1 and HALO-NESPRIN3u03B1 (lanes 3, 7, 11), or TMX4-V5 GFP-SUN1 and HALO-NESPRIN3u03B1 (lanes 4, 8, 12) were separated in a reducing gel and transferred on PVDF membranes. Expression of the ectopic proteins was confirmed by WB with anti-V5 (lanes 1u20134), anti-GFP (lanes 5u20138), or anti-HALO antibodies (lanes 9u201312). Uncropped blots in Supplementary Fig._. b The presence of TMX proteins (lanes 1u20133), NESPRIN3u03B1 (lanes 4u20136), or SUN1 (lanes 7u20139) in mixed disulfides (MD) with TMX4C67A-V5 (lanes 2, 5, 8) or TMX4-V5 (lanes 3, 6, 9) immunoisolated from cell lysates with anti-V5 antibodies was monitored under non-reducing conditions by western blot with anti-V5, anti-HALO, or anti-GFP antibodies, respectively. c. The engagement of TMX proteins (lanes 10u201312), or NESPRIN3u03B1 (lanes 13u201315) in MD with HALO-NESPRIN3u03B1 immunoisolated from cell lysates with anti-HALO antibodies was monitored under non-reducing conditions by western blot with anti-V5 and anti-HALO antibodies, respectively. d The presence of TMX4 proteins in MD with GFP-SUN1 (lanes 1u20133), or with HALO-NESPRIN3u03B1 (lanes 4u2013 6) immunoisolated from cell lysates has been assessed by WB with anti-V5 antibodies under non-reducing conditions (or reducing conditions, lanes 7u201312). WB is representative of at least three independent experiments.\" - \"The engagement of TMX4C67A-V5 and TMX4-V5 in mixed disulfides was monitored by separation of the complexes immunoisolated from cell lysates with anti-V5 antibody in non-reducing gels and subsequent western blotting with anti-V5 antibody (Fig._ lanes 1u20133). These analyses confirm the enrichment of mixed disulfides (MD) by the trapping mutant version of TMX4 (lanes 2 vs. 3, Fig._). The V5-immunoreactive polypeptides shown with asterisks (Fig._, lanes 2, 3) are also detected when the same PVDF membrane is probed with the anti-HALO antibody (Fig._, circles in lanes 5, 6), revealing the TMX4 (lanes 2, 3) and the NESPRIN3u03B1 components (lanes 5, 6) of TMX4-V5-SS-HALO-NESPRIN3u03B1 mixed disulfides, respectively. In contrast, the MD immunoisolated from cell lysates with the anti-V5 antibody is not recognized by the anti-GFP antibody (Fig._, lanes 7u20139). These results confirm the data in cellula, which identify endogenous NESPRIN proteins, but not SUN proteins, as TMX4 clients (Fig._ and Supplementary Table_).\" - \"TMX4-V5-SS-HALO-NESPRIN3u03B1 mixed disulfides are also immunoisolated from cell lysates with the anti-HALO antibody (Fig._, lanes 10u201315). In this case, the TMX4 component of the mixed disulfide is revealed by western blotting with the V5 antibody (polypeptide with asterisk in Fig._, lanes 11, 12). The NESPRIN3u03B1 component of the mixed disulfides is revealed with the anti-HALO antibodies (Fig._, lanes 14, 15, circles). In all cases, specificity is supported by the enhanced presence of disulfide-bonded complexes when these are trapped upon mutation of the TMX4 catalytic site (Fig._, lanes 2 vs. 3 and 5 vs. 6, Fig. c, lanes 11 vs. 12 and 14 vs. 15).\" - \"To further support the preference of TMX4 to engage NESPRIN proteins, GFP-SUN1 (Fig._, IP:GFP, lanes 1u20133 and 7u20139) or HALO-NESPRIN3u03B1 (Fig._ , IP:HALO, lanes 4u20136 and 10u201312) were immunoisolated from cell lysates with anti-GFP or anti-HALO antibodies, respectively. Western blotting with anti-V5 antibodies reveals the co-precipitation of GFP-SUN with a low amount of monomeric TMX4 (Fig._, lanes 2, 3, arrows, and 8, 9) and the absence of GFP-SUN1-SS-HALO-NESPRIN3u03B1 mixed disulfides (Fig._, lanes 2, 3). Instead, TMX4 abundantly co-precipitates with HALO-NESPRIN3u03B1 (Fig._, lanes 5, 6 and 11, 12). Notably, most of the TMX4-associated immunoreactivity that co-precipitates with NESPRIN3u03B1 is in mixed disulfides (TMX4-V5-SS-HALO-NESPRIN3u03B1), which run between the 130 and 175u2009kDa (MD, Fig._, lane 5). MD disappears upon sample reduction to reveal the TMX4-V5 component upon WB with the anti-V5 antibody (Fig._, lane 11). These results confirm the analyses in cellula, where endogenous NESPRIN proteins show up as clients of the TMX4 reductase (Fig._ and Supplementary Table_)\" - TMX4 is dispensable for SEC62-driven recov-ER-phagy, but intervenes in SEC62-regulated ONM-phagy - \"To explore the role of TMX4 in NE dynamics, its intracellular level was reduced upon RNA interference. The stress/recovery protocol shown in Figs._ and was reproduced in MEF with normal levels of TMX4 upon transfection with a scrambled small interfering RNA (Supplementary Fig._, siSCR) and in MEF, where the level of TMX4 was reduced by 65% both at the mRNA (Supplementary Fig._, siTMX4) and at the protein level (Supplementary Fig._, siTMX4) upon transfection with a small interfering RNA targeting the TMX4 transcripts. In MEF transfected with siSCR, both SEC62 (Fig._ and Inset 1, Fig. c, siSCR) and HALO-NESPRIN3u03B1 (Fig._ and Inset 2, Fig. c, siSCR) are delivered within LAMP1-positive endolysosomes during recovery from ER stress. Silencing of TMX4 expression does not affect lysosomal delivery of SEC62 (Fig._, and Inset 1, Fig. c, siSCR vs. siTMX4) but inhibits lysosomal delivery of HALO-NESPRIN3u03B1 (Fig._, and Inset 2, Fig._, siSCR vs. siTMX4). This observation hints at a mechanistic difference between lysosomal clearance of ER subdomains and lysosomal clearance of ONM portions during recovery from ER stress. Both pathways involve the LC3 lipidation machinery but not autophagosome biogenesis (refs. for ER subdomains, Fig._ for ONM portions) and rely on activation of the autophagy receptor SEC62 (ref. for ER subdomains, Fig._ for ONM portions). However, only the lysosomal delivery of ONM, which requires the physical separation of the ONM from the INM, relies on TMX4 intervention (Fig._). The failure to re-activate the ONM-phagy pathway by back-transfecting siRNA-resistant TMX4 in cells where the expression of the endogenous TMX4 is silenced led us to assess the consequences of TMX4 overexpression in cells recovering from acute CPA-induced stress. Strikingly, in wild-type MEF expressing recombinant TMX4, lysosomal delivery of SEC62 progresses normally (Fig._ and Inset 1), but lysosomal delivery of NESPRIN3u03B1 is substantially inhibited (Fig._ and Inset 2). Thus, silencing or overexpression of TMX4 does not affect SEC62-driven recov-ER-phagy. However, both silencing and overexpression of TMX4 substantially inhibit SEC62-driven ONM-phagy.\" - \"Fig. 10: Consequences of TMX4 silencing on NE dynamics.a CLSM analyses of endogenous SEC62 (Inset1) and HALO-NESPRIN3u03B1-positive ONM subdomains (Inset2) delivery within LAMP1-positive endolysosomes in MEF recovering from ER stress, 12u2009 h after interruption of the pharmacologic treatment with CPA and exposed to 50u2009 nM BafA1. Scale bars: 10u2009u03BCm. b Same as (a), in cells where TMX4 expression has been silenced by RNA interference. Also, refer to Supplementary Fig._, . c Quantification of (a) and (b) in mock-treated cells and in cells with reduced expression of TMX4. nu2009=u200934 and 45 cells for mock-treated and TMX4 knockdown cells, respectively. Nu2009=u20093 independent experiments, meanu2009 xB1u2009SEM; unpaired, two-tailed t-test, ns. Pu2009=u20090.3073 for SEC62. nu2009=u200934 and 45 cells for mock-treated and with TMX4 knockdown cells, respectively. Nu2009=u20093 independent experiments, meanu2009xB1u2009 SEM; unpaired, two-tailed t-test, ****Pu2009<u20090.0001 for NESPRIN3u03B1 . d Same as (a) in wild-type MEF overexpressing TMX4. Representative of three independent experiments. Scale bars: 10u2009u03BCm. e RT-TEM micrographs of NE of two different CRISPRTMX4 MEF at steady state. Scale bars: 500u2009nm. f Same as (d) 2 different CRISPRTMX4 MEF during pharmacologically-induced ER stress.\" - ER swelling is not transmitted to the NE in cells lacking TMX4 - \"Next, we generated TMX4-KO MEF by CRISPR-Cas9 genome editing (Supplementary Fig._ ). CRISPRTMX4 cells are unstable and must be examined at low passages, implying an important role of TMX4 in the maintenance of cultured cells fitness and viability. Nevertheless, the NE of cells lacking TMX4 looks normal at steady state (Fig._ and insets, to be compared with Fig._). Cells lacking TMX4 do respond to CPA-induced ER stress as shown by the upregulation of the conventional ER stress marker BiP/GRP78 at the transcript (Supplementary Fig._) and at the protein level (Supplementary Fig._). However, in contrast to wild type cells (Figs._, bu2013d, , Supplementary Fig._), the NE of CRISPRTMX4 cells does not significantly change morphology upon pharmacologic treatment with CPA (Fig._ ). This confirms the role of the reductase TMX4 during the enlargement of PNS width resulting from perturbation of ER homeostasis.\"- - \"Perturbation of ER homeostasis activates UPR that enhances protein and lipid synthesis to expand ER membranes and volume. In this work, analyses of the NE ultrastructure by RT and cryo-EM reveal that the expansion of the mammalian ER lumen is transmitted to the contiguous PNS, whose width, at least in adherent cells and in cells characterized by increased mechanical tension is maintained below the 50u2009nm by LINC complexes formed by NESPRIN proteins in the ONM covalently linked, via intermolecular disulfide bonds, with SUN proteins in the INM. During pharmacologic perturbation of ER homeostasis, LINC complexes are disassembled, and the ONM forms bulges, which are not observed in cells lacking the ONM-membrane bound reductase TMX4. By trapping endogenous and ectopically expressed NESPRIN proteins (but not SUN proteins) in mixed disulfides with TMX4, we show that the active site cysteine of TMX4 acts upon the NESPRIN cysteine residue involved in the NESPRIN-S-S-SUN intermolecular disulfide bond to disassemble the LINC complexes. For some ER-resident oxidoreductases, client specificity is well-established. For example, the BiP-associated P5 oxidoreductase assists the oxidative folding of clients of the ER chaperone BiP, the calnexin/calreticulin-associated ERp57 assists the folding of glycoproteins that engage the lectin chaperones calnexin and calreticulin. In other cases, substrate-specificity may rely on the tissue-specific expression of the redox enzyme and its clients, as in the case of the testis-specific oxidoreductase PDILT. For TMX4, the reason for the clear preference for NESPRIN proteins compared to the more abundant SUN proteins is unclear. The localization of the reductase in the ONM, which we established by immunogold EM, might play a role. Also unclear is how UPR induction may trigger the reductase activity of TMX4 required to disassemble LINC complexes. The recently reported modulation of TMX4 by Zdhhc6-driven palmitoylation offers an interesting possibility to activate TMX4 on demand. It remains to be established if the TMX4-driven mechanisms that disassemble LINC complexes and expand the PNS upon perturbation of ER homeostasis also regulate nuclear envelope budding (NEB) events upon perturbation of protein folding in yeast or the NPC-independent nucleoplasm to cytoplasm transport of macromolecules (e.g., ribonucleoprotein particles), two cellular processes that require enlargement of the distance between the ONM and the INM. Future studies will also examine if the LINC complex disassembly machinery is hijacked by viruses such as Herpesviridae, whose capsid is produced in the nucleoplasm and must transit through the PNS for nuclear egress.\" - As previously reported for ER swelling, our results show that also the increase of the PNS width is reversible, and a few hours after interruption of the pharmacologic treatment, the physiologic ultrastructure of the NE is restored. This happens upon selective vesiculation of the ONM and direct capture of excess ONM by degradative endolysosomes via piecemeal micro-ONM-phagy. Like the recov-ER-phagy pathways that restore physiologic ER size and activity at the end of acute perturbations of ER homeostasis, what we here define as ONM-phagy involves the autophagy receptor SEC62 and the LC3 lipidation machinery. This is supported by genetic and morphometric analyses, which also show the dispensability of autophagosome biogenesis and intervention. The significant difference between ER and NE dynamics during perturbation of cellular homeostasis is that only the latter involves the reductase TMX4, which is activated during the pharmacologic treatment to transmit the stress-induced ER enlargement to the NE.- - Antibodies, expression plasmids - \"Commercial antibodies used in this study: Lamp1 (DSHB Hybridoma Product 1D4B deposited by J. T. August, Immunofluorescence (IF) 1:50), V5 (Western blot (WB) 1:5000 Thermo Fisher Scientific), GFP (WB 1:1000 IEM: 1:50 Abcam), TMX4 (WB 1:1000, IEM:1:20 Proteintech), TEX264 (IEM: 1:50 Novus), LC3B (IF: 1:50 Sigma), GAPDH (WB 1:30000 Millipore), KDEL (WB 1:1000 Stressgen), HaloTag (WB 1:1000 Promega), PDI (WB 1:100 Stressgen), Erp72 (WB 1:100 Stressgen). Alexau2013 conjugated secondary antibodies were purchased from Thermo Fisher Scientific (anti-rat 647, anti-rabbit 488), Invitrogen (anti-rabbit 568, anti-rabbit 405) and Jackson Immunoresearch (anti-mouse 488, anti-rabbit 657) (IF 1:300). The HRP-conjugated secondary antibodies were purchased from SouthernBiotech (anti-mouse, WB 1:20000) and Protein A HRP-conjugated were purchased from Invitrogen (WB 1:20000). The secondary Nanogold Fab anti-rabbit (H+L) and anti-mouse (H+L) antibodies were purchased from Nanoprobes (IEM: 1:100).\" - \"The HALO-Trap or GFP-Trap Agarose beads and V5u2010conjugated beads were purchased from ChromoTek and Sigma, respectively. The HaloTag ligands JF646 or tetram-ethylrhodamine (TMR) were purchased from Promega. Antibody against SEC62 (IF 1:100, IEM:1:50), CNX (WB 1:2000), FAM134B (WB 1:1000) and ERp57 (WB 1:100) were kind gifts from R. Zimmermann, A. Helenius, M. Miyazaki and T. Wileman, respectively. Plasmid encoding GFP-RAB7, GFP-SUN1, GFP-NESPRIN3u03B1 are kind gifts from T. Johansen, H. J. Worman and A. Sonnenberg, respectively. NESPRIN3u03B1 was subcloned in a pcDNA3.1 expression plasmid with the addition of an N-terminal HALO-Tag. The trapping mutants of TMX4 (TMX4C67A-V5), TMX3 (TMX3C53A-V5) and TMX5-V5 WT were synthesized by GenScript. The TMX3-V5 WT and TMX4-V5 WT were obtained from trapping mutants by site-directed mutagenesis.\" - Cell culture, transient transfection, CRISPR/Cas9 genome editing, small interfering RNA silencing - \"MEF, HEK293 and NIH/3T3 cells were grown in DMEM with 10% fetal bovine serum (FBS) at 37u2009xB0C and 5% CO2. Transient transfections were carried out using the JetPrime transfection reagent (PolyPlus) in accordance with the manufacturer s protocol. Atg5 KO and Atg14 KO MEF were kind gifts from N. Mizushima and T. Saitoh, respectively. CRISPRMOCK, CRISPRSEC62 and CRISPRFAM134B were generated by CRISPR/Cas9 genome editing in our lab as described in ref. . RNA interferences were performed in MEF plated at 50u201360% confluence. Cells were transfected with scrambled small interfering RNA or small interfering RNA (siRNA) to silence TMX4 expression (5u2032-GCAUGGUGUUCUUACGUUUtt-3u2032, 100u2009nM per dish, Silencer Select Pre-designed, Ambion). Cells were processed for immunofluorescence or for biochemical analyses after 48u2009h of transfection.\" - \"TMX4-KO MEF cells were generated as follows: the guideRNAu2010Cas9 plasmids, lentiCRISPRv2u2010puro system (Addgene52961) was obtained from Addgene (). Cas9 target design tools were used to generate guide sequences (). All protocols and information can be found at the website . The guide RNA target sequences were synthesized by Microsynth AG. Two annealed oligonucleotides (5u2032-CACCGCTCGCAGCGGCAGCGGCCG-3u2032 , 5u2032-AAACCGGCCGCTGCCGCTGCGAGC-3u2032 for murine TMX4 were inserted into the lentiCRISPRv2u2010puro vector using the BsmBI restriction site. LentiV2- gRNA vector was transfected in MEF cells with JetPrime (Polyplus) according to the manufacturers instructions. Cells were cultured in DMEM supplemented with 10% FBS. After 2 days of transfection, the medium was supplemented with 2u2009u03BCg/ml puromycin (Invitrogen). Puromycinu2010resistant clones were picked after 10 days and gene KO was verified by WB.\" - \"RNA extraction, polymerase chain reaction with reverse transcription (RTu2013 PCR)\" - \"The extraction of RNA from MEF was performed with the GenElute Mammalian Total RNA Miniprep Kit (Sigma) according to the manufacturers instructions. One microgram of RNA was used for cDNA synthesis with dNTPs (Kapa Biosystems), oligo(dT) and the SuperScript II reverse transcriptase (Thermo Fisher Scientific) according to the instructions of the manufacturer. For each qRT-PCR reaction, 10u2009u03BCl of Power SYBR Green PCR Master Mix (Bimake), 0.4u2009u03BC l of reference ROX dye and 7.6u2009u03BCl of milliQ sterile water were added to 1u2009u03BCl cDNA together with 1u2009u03BCl of 10u2009u03BCM forward and reverse primer mix (mTMX4 (I) fwd: (5u22123): TTG AGT GGC CGC TTC TTT GT rev: (5u22123): CCA GAC ATC GTT AGA GAG GCT; mTMX4 (II) fwd: (5u22123): CAT CCT GCC AGC AGA CTG ATT rev: (5u2212 3): GGC GGA ATA TCC CAT CTT TTG C), (mBiP fwd: (5u22123): GAGTTCTTCAATGGCAAGGA rev: (5u22123): CCAGTCAGATCAAATGTACCC) for the transcript of interest in 96-well reaction plate (MicroAmp Fast Optical 96-Well Reaction Plate with Barcode (0.1u2009ml), Applied Biosystems). The plate was vortexed and centrifuged. Samples were loaded as triplicates. Quantitative real-time PCR was performed using QuantStudiou2122 3 Real-Time PCR System. The housekeeping gene actin was used as reference. Data were analyzed using the QuantStudiou2122 Design & Analysis Software v1.5.5.\" - Protocol to induce transient ER stress - \"To induce transient ER stress, cultured cells were exposed for 12u2009h to 10u2009 xB5M CPA in DMEM, 10% FBS. For the recovery condition, CPA (Sigma) was washed out and incubation was prolonged in DMEM, 10% FBS up to 48u2009h. Cells were processed for biochemical or imaging analyses (as in refs. ).\" - Cell lysis - \"HEK293 cells plated on poly-L-lysine coated dishes were washed in cold phosphate-buffered saline (PBS) containing 20u2009mM N-ethylmaleimide (NEM) and were lysed in RIPA buffer (1% Triton X-100, 0.1% SDS, 0.5% sodium deoxycholate in HBS, pH 7.4, 20u2009mM NEM and protease inhibitors) and then collected by scraping, maintained for 20u2009min on ice and then post-nuclear supernatants (PNS) were collected by centrifugation at 10,000xD7g for 10u2009min.\" - Proteins immunoprecipitation and Western blot - \"The nuclear lysates were incubated with HALO-Trap Agarose beads for 2u2009h at 4u2009xB0C to isolate HALO-NESPRIN3. After immunoprecipitation, samples were washed in 1u2009ml HBS 1x pH 7.4, 0.5% Triton two times. Beads were dried out and resuspended in sample buffer without DTT at 95u2009xB0C for 5u2009 min. Then, half of the sample was reduced by the addition of 100u2009mM DTT. Samples were then boiled again for 5u2009min at 95u2009xB0C and loaded on 8% SDS-PAGE (polyacrylamide gel electrophoresis) for reducing and non-reducing protein separation. Proteins were transferred to polyvinylidene difluoride (PVDF) membranes with the Trans-Blot Turbo Transfer System (Bio-Rad). Membranes were blocked for 10u2009min with 8% (w/v) non-fat dry milk (Bio-Rad) in Tris-buffered saline (TBS)-Tween 1% and stained with primary antibodies (listed in u201C Methodsu201D) overnight, and for 45u2009min with HRP-conjugated secondary antibodies or Protein A HRP-conjugated. Membranes were developed using the Western bright ECL or Western bright QUANTUM (Witec) and signals were acquired with the FusionFX7 VILBER Witec using Fusion FX7 Edge software. Membrane stripping was performed using Re-Blot Plus Strong Solution (Millipore) following the manufacturer s instructions before repetition of the protocol for the detection of other antigens.\" - Silver staining - \"Polyacrylamide gels were fixed in a 40% EtOH, 10% Acetic Acid solution for 1u2013 4u2009h at room temperature, rinsed two times for 20u2009min in 30% EtOH and then soaked for 20u2009min in H2O. After 1u2009min in 0.02% Na2S2O3 and three washes in H2O, the gels were incubated for 20u2009min in 0.2% AgNO3 and then rinsed three times in H2O. Development was performed with a freshly prepared developing solution containing 3% Na2CO3 and 0.05% formaldehyde. Development was stopped by rinsing the gels in H2O and then in 5% Acetic Acid.\" - Confocal laser scanning microscopy (CLSM) - \"MEF were seeded on alcian blue-treated glass coverslips and transiently transfected with JetPrime reagent according to the manufacturers protocol. Cells were then treated with 10u2009u03BCM CPA for 12u2009h and, on CPA removal, incubated for 12u2009h with 50u2009nM BafA1 (Calbiochem), or DMSO (Sigma) and with 100u2009 nM TMR HaloTag ligand (Promega). Following two PBS washes, cells were fixed with 3.7% formaldehyde diluted in PBS for 20u2009min at RT. Cells were permeabilized with 0.05% saponin, 10% goat serum, 10u2009mM HEPES and 15u2009mM glycine (PS) for 15u2009min. Cells were incubated with the primary antibodies diluted 1:100 in PS for 120u2009min, washed two times in PS, and then incubated with Alexa Fluor-conjugated secondary antibodies diluted 1:300 in PS for 45u2009 min. Cells were rinsed three times with PS and once with water. Afterward, cells were mounted with Vectashield (Vector Laboratories) with 4u2032,6-diamidino-2-phenylindole (DAPI).\" - \"Leica TCS SP5u2009 or Leica Stellaris SP8 microscope with a Leica HCX PL APO lambda blue 63.0 xD7 1.40 OIL UV objective was used to acquire confocal images. The acquisition software Leica LAS X was used.\" - \"The quantifications of HALO-NESPRIN3u03B1, GFP-SUN1, SUN2, SEC62, SEC62-GFP or SEC62LIR-GFP positive lysosomes per cell were executed with LysoQuant, an unbiased and automated deep learning tool for fluorescent image quantification, which is freely available (). Image processing was also performed with Fiji/ImageJ and Adobe Photoshop.\" - Live imaging - \"NIH3T3 cells were seeded on glass bottom Mattek 35u2009mm dishes (MatTek #1.5 coverslips) and transiently transfected with JetPrime reagent according to the manufacturers protocol for GFP-RAB7 and HALO-NESPRIN3. Cells were treated with 10u2009u03BCM CPA for 12u2009h and, on CPA removal, incubated for 5u2009 h with 100u2009nM BafA1, and with 100u2009nM Janelia Flour 646 HaloTag ligand (Promega).\" - \"Movies were recorded with a Leica TCS SP5 confocalu2009microscope with a Leica HCX PL APO lambda blue 63.0 xD7 1.40 OIL UV objective was used to acquire confocal images at 37u2009xB0C, 5% CO2. Excitation was performed with 488u2009nm and 633u2009nm laser lines and fluorescence light was collected in ranges 493u2013 625u2009nm and 637u2013791u2009nm respectively with pinhole 1u2009AU. The pictures were taken every 1u2009s with a pixel size of 63u2009nm and line average 3. Acquisitions were then cleaned with a median filter and movie processing was performed using Fiji/ImageJ.\" - Mass spectrometry - \"HEK293 cells were transiently mock-transfected (pcDNA3.1) or transfected with TMX3-V5, TMX3C53A-V5, TMX4-V5, TMX4C64A-V5 and TMX5-V5. Sixteen h after transient transfection, cells were lysed in RIPA buffer and the PNS (post-nuclear supernatant) was collected. The PNS was double immunoprecipitated using anti-V5 conjugated beads. After washing, the beads used for immunoprecipitation were extracted by boiling for 5u2009min in 60u2009xB5l of sample buffer. The supernatant was split into two aliquots, one of which was reduced by adding DTT to a final concentration of 10u2009mM and incubating for 5u2009min at RT. Reduced and nonreduced samples were separated in a gel that was silver stained (Fig._) or migrated on separate 10% polyacrylamide gels for a total distance of 4.0u2009 cm. After Coomassie-blue staining, gel lanes loaded with the trapping variants of the three TMX proteins were excised in 4 bands from the loading point down to the 50u2009kDa marker and digested in-gel as described. Extracted tryptic peptides were dried and resuspended in 0.05% trifluoroacetic acid, 2% (v/v) acetonitrile.\" - LC-MS/MS and data analysis - \"Data-dependent LC-MS/MS analysis of extracted peptide mixtures after digestion with trypsin was carried out on a Q-Exactive Plus mass spectrometer (Thermo Fisher Scientific) interfaced to a nanocapillary HPLC (Dionex RSLC 3000). Peptides were separated on a custom-packed C18 reversed-phase column (75u2009u03BC m ID xD7 45u2009cm, 1.8u2009u03BCm particles, Reprosil Pur, Dr. Maisch), using a gradient from 4 to 76% acetonitrile in 0.1% formic acid (total time: 65u2009 min). Full MS survey scans were performed at 70,000 resolution. In data-dependent acquisition controlled by Xcalibur software (Tune 2.9, Thermo Fisher Scientific), the 10 most intense multiply charged precursor ions detected in the full MS survey scan were selected for higher energy collision-induced dissociation (HCD, normalized collision energy NCEu2009=u200927%) and analysis in the orbitrap at 17500 resolution. The window for precursor isolation was of 1.5u2009 m/z units around the precursor and selected fragments were excluded for 60u2009 s from further analysis. Collections of peptide tandem mass spectra were searched using Mascot 2.6.2 (Matrix Science, London, UK) against the January 2019 release of the SWISSPROT database limited to human taxonomy (20762 sequences), together with a custom-built database containing common contaminants. Carbamidomethyl (Cys) was defined as fixed modification while oxidation (Met) and N-terminal protein acetylation were specified as variable modifications. The software Scaffold (version 4.9.0, Proteome Software Inc.) was used to validate MS/MS-based peptide (minimum 90% probability) and protein (min 95% probability) identifications, perform dataset alignment as well as parsimony analysis to discriminate homologous hits. Identifications were filtered for a maximum FDR (false discovery rate) of 1% against a decoy database. Analysis was performed once and then confirmed by in vitro analysis shown in Fig._.\" - Statistical analyses and reproducibility - \"The legend of the figures indicates the number (n) of cells analyzed and the number N of independent experiments (biological replicates). Statistical analyses were performed using GraphPad Prism 9 software. Unpaired, two-tailed t-test were used to assess statistical significance. P-values are given in the figure legends; ns. Pu2009>u20090.05; *Pu2009<u20090.05; **Pu2009<u20090.01; ***Pu2009<u20090.001; ****Pu2009<u20090.0001.\" - \"Experiments for RT-TEM (Figs._, , and ) were repeated at least two times, up to five times, by four different operators. Experiment for 24u2009h recovery was also performed with consistent results (not shown in the paper). For Fig._ results of one experiment were analyzed in RT-TEM. Data were confirmed by RNA interference (Fig._). Additional, independent experiments were analyzed by CET.\" - \\'Cryo-electron tomography (CET): Grid preparation\\' - \"MEF were seeded on R2/2 holey carbon on gold grids (Quantifoil or Protochips) coated with fibronectin in a glass bottom dish (Mattek or Ibidi) and incubated for 12u2009h. For the stress condition, cells were then incubated for 12u2009 h with 10u2009xB5M CPA in DMEM+10% FBS. For the recovery condition, CPA was washed out and cells were incubated in DMEM+10% FBS for another 5u2009h. Grids were mounted to a manual plunger, blotted from the back for ~10u2009s and plunged into liquid ethane.\" - \\'CET: lamellae preparation\\' - \"Lamellae were prepared using an Aquilos FIB-SEM system (Thermo Fisher Scientific). Grids were sputtered with an initial platinum coat (10u2009s) followed by a 10u2009s gas injection system (GIS) to add an extra protective layer of organometallic platinum. Samples were tilted to an angle of 15xB0 to 22xB0 and 11-xB5m-wide lamellae were prepared. The milling process was performed with an ion beam of 30u2009kV energy in three steps: (1) 500 pA, gap 3u2009xB5m with expansion joints, (2) 300 pA, gap 1u2009xB5m, (3) 100 pA, gap 500u2009nm. Lamellae were finally polished at 30u201350 pA with a gap of 200u2009nm.\" - \\'CET: data acquisition\\' - \"A total of 66 tilt series were acquired on a Talos Arctica (Thermo Fisher Scientific) operated at an acceleration voltage of 200u2009kV and equipped with a K2 summit direct electron detector and 20u2009eV slit energy filter (Gatan). Images were recorded in movies of 5u20138 frames at a target defocus of 4u20136u2009u03BC m and an object pixel size of 2.17u2009xC5. Tilt series were acquired in SerialEM using a grouped dose-symmetric tilt scheme covering a range of xB160xB0 with a pre-tilt of xB110xB0 and an angular increment of 3xB0. The cumulative dose of a series did not exceed 80 e-/xC5. On lower quality lamellae from CPA recovery condition, lower magnification tilt series were also acquired at an object pixel size of 4.47u2009xC5, for membrane segmentation and illustration purposes (not used for distance measurements nor filament segmentation).\" - \\'CET: tomogram reconstruction\\' - \"Movie files of individual projection images were motion-corrected with MotionCor2 and combined into stacks of tilt series using a Matlab script. The combined stacks were dose corrected in Matlab and aligned using patch tracking in IMOD. Full tomograms were reconstructed by weighted back projection at a pixel size of 13.06u2009xC5. Ice thickness was determined manually and was found to be <200u2009nm for all tomograms.\" - \\'CET: ONM to INM distance measurements\\' - A density threshold mask was applied in Avizo software (Thermo Fisher Scientific) to highlight only stronger densities. ONM and INM were then traced for each tomogram with the brush tool. The exported labels were submitted to surface morphometrics to create a triangular mesh and measure the intra-membrane distance along the surfaces. Results were plotted in Python using matplotlib. - \\'CET: filament segmentation\\' - For each dataset, the highest quality tomogram, corresponding to the thinner lamellae, was used for the analysis of the filaments in the perinuclear region. Deconvolution was applied using tom_deconv in Matlab () and the resulting tomograms were imported into Avizo software (Thermo Fisher Scientific). A density threshold mask was applied to highlight only stronger densities to guide subsequent segmentation steps. Assisted by the density threshold mask, the continuous pieces of density between ONM and INM visible in the perinuclear space in each z slice were traced with the brush tool. The resulting labels were exported for visualization in UCSF Chimera. - Room temperature-transmission electron microscopy (RT-TEM) - \"MEFs were seeded on alcian blue-coated glass coverslips and fixed in double-strength fixative into the media (4% PFA EM grade, 5% GA in Na-cacodylate buffer 0.1u2009 M, pH 7.4) for 20u2009min at RT. After removing the mixture, cells were incubated with single-strength fixative (2% PFA, 2.5% GA in Na-cacodylate buffer 0.1u2009 M, pH 7.4) for 3u2009h at RT. After several washes in cacodylate buffer, cells were post-fixed in 1% osmium tetroxide (OsO4), 1.5% potassium ferricyanide (K4[Fe(CN)6]) in 0.1u2009M Na-cacodylate buffer for 1u2009h on ice, washed with distilled water (dH2O) and enbloc stained with 0.5% uranyl acetate in dH2O overnight at 4u2009xB0C in the dark. Samples were rinsed in dH2O, dehydrated with increasing concentrations of ethanol, embedded in Epon resin and cured in an oven at 60u2009 xB0C for 48u2009h. Ultrathin sections (70u201390u2009nm) were collected using an ultramicrotome (UC7, Leica microsystem, Vienna, Austria), stained with uranyl acetate and Satos lead solutions and observed in a Transmission Electron Microscope Talos L120C (FEI, Thermo Fisher Scientific) operating at 120u2009 kV. Images were acquired with a Ceta CCD camera (FEI, Thermo Fisher Scientific). Image analysis was performed using Microscopy Image Browser.\" - \\'RT-TEM: immunogold electron microscopy\\' - \"Cells were fixed in 4% PFA EM grade and 0.2u2009M HEPES buffer for 1u2009h at RT or in Periodate-lysine-paraformaldehyde (PLP) for 2u2009h at RT. After three washes in PBS, cells were incubated 10u2009min with 50u2009mM glycine and blocked 1u2009h in blocking buffer (0.2% bovine serum albumin, 5% goat serum, 50u2009mM NH4Cl, 0.1% saponin, 20u2009mM PO4 buffer, 150u2009mM NaCl). Staining with primary antibodies and nanogold-labeled secondary antibodies (Nanoprobes) were performed in a blocking buffer at RT. Cells were fixed for 30u2009min in 1% GA and nanogold was enlarged with gold enhancement solution (Nanoprobes) according to the manufacturers instructions. Cells were post-fixed with OsO4 and processed as described for conventional EM. Images were acquired with Talos L120C TEM (FEI, Thermo Fisher Scientific) operating at 120u2009kV. Images were acquired with a Ceta CCD camera (FEI, Thermo Fisher Scientific) using Velox 3.6.0 (FEI, Thermo Fisher Scientific).\" - RT-electron tomography (RT-ET) - \"For electron tomography, 130u2013150u2009nm thick sections were collected on formvar-coated copper slot grids and gold fiducials (10u2009nm) were applied on both surfaces of the grids. The samples were imaged in a 120u2009kV Talos L120C TEM (FEI, Thermo Fisher Scientific). Single or dual tilted image series (+60/u221260) were acquired using Tomography 4.0 software (FEI, Thermo Fisher Scientific). Tilted series alignment and tomography reconstruction were done with the IMOD software package. Segmentation and 3D visualization were done with Microscopy Image Browser or IMOD software packages.\" - \\'RT-TEM: measurement of INM-ONM distance\\' - ONM and INM were manually segmented in Microscopy Image Browser (MIB). Using the cell wall thickness plugin of MIB, the distance between the INM and the ONM was measured. Results were plotted in GraphaPad PRISM 9. - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - \"The source data are provided with this paper as a Source Data file. The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier and ._ are provided with this paper.\" - \\' \\'- - \"Brown, R. Observations on the organs and mode of fecundation in Orchideae and Asclepiadeae. Trans. Linnean Soc. 16, 684u2013745 (1833)._ _ Callan, H. G. & Tomlin, S. G. Experimental studies on amphibian oocyte nuclei. I. Investigation of the structure of the nuclear membrane by means of the electron microscope. Proc. R. Soc. Lond. B Biol. Sci. 137, 367u2013378 (1950)._ _ _ _ _ Afzelius, B. A. The ultrastructure of the nuclear membrane of the sea urchin oocyte as studied with the electron microscope. Exp. Cell Res. 8, 147u2013158 (1955)._ _ _ _ Dalton, A. J. & Felix, M. D. Cytologic and cytochemical characteristics of the Golgi substance of epithelial cells of the epididymis in situ, in homogenates and after isolation. Am. J. Anat. 94, 171u2013207 (1954)._ _ _ _ SjxF6strand, F. S. & Rhodin, J. The ultrastructure of the proximal convoluted tubules of the mouse kidney as revealed by high resolution electron microscopy. Exp. Cell Res. 4, 426u2013456 (1953)._ _ Watson, M. L. The nuclear envelope; its structure and relation to cytoplasmic membranes. J. Biophys. Biochem. Cytol. 1, 257u2013 270 (1955)._ _ _ _ _ Ungricht, R. & Kutay, U. Mechanisms and functions of nuclear envelope remodelling. Nat. Rev. Mol. Cell Biol. 18, 229u2013245 (2017)._ _ _ _ Kalukula, Y., Stephens, A. D., Lammerding, J. & Gabriele, S. Mechanics and functional consequences of nuclear deformations. Nat. Rev. Mol. Cell Biol. 23, 583u2013602 (2022)._ _ _ _ _ Dultz, E. et al. Systematic kinetic analysis of mitotic dis- and reassembly of the nuclear pore in living cells. J. Cell Biol. 180, 857u2013865 (2008)._ _ _ _ _ Lin, D. H. & Hoelz, A. The structure of the nuclear pore complex (an update). Annu. Rev. Biochem. 88, 725u2013783 (2019)._ _ _ _ _ Srivastava, N. et al. Nuclear fragility, blaming the blebs. Curr. Opin. Cell Biol. 70, 100u2013 108 (2021)._ _ _ _ Malhas, A., Goulbourne, C. & Vaux, D. J. The nucleoplasmic reticulum: form and function. Trends Cell Biol. 21, 362u2013373 (2011)._ _ _ _ Guilluy, C. et al. Isolated nuclei adapt to force and reveal a mechanotransduction pathway in the nucleus. Nat. Cell Biol. 16, 376u2013381 (2014)._ _ _ _ _ Sosa, B. A., Rothballer, A., Kutay, U. & Schwartz, T. U. LINC complexes form by binding of three KASH peptides to domain interfaces of trimeric SUN proteins. Cell 149, 1035u20131047 (2012)._ _ _ _ _ Padmakumar, V. C. et al. The inner nuclear membrane protein Sun1 mediates the anchorage of Nesprin-2 to the nuclear envelope. J. Cell Sci. 118, 3419u20133430 (2005)._ _ _ _ Cain, N. E. et al. Conserved SUN-KASH interfaces mediate LINC complex-dependent nuclear movement and positioning. Curr. Biol. 28, 3086u20133097 (2018)._ _ _ _ _ Piccus, R. & Brayson, D. The nuclear envelope: LINCing tissue mechanics to genome regulation in cardiac and skeletal muscle. Biol. Lett. 16, 1u20139 (2020)._ _ Lombardi, M. L. et al. The interaction between nesprins and sun proteins at the nuclear envelope is critical for force transmission between the nucleus and cytoskeleton. J. Biol. Chem. 286, 26743u201326753 (2011)._ _ _ _ _ Haque, F. et al. SUN1 interacts with nuclear lamin A and cytoplasmic nesprins to provide a physical connection between the nuclear lamina and the cytoskeleton. Mol. Cell. Biol. 26, 3738u20133751 (2006)._ _ _ _ _ Crisp, M. et al. Coupling of the nucleus and cytoplasm: role of the LINC complex. J. Cell Biol. 172, 41u2013 53 (2006)._ _ _ _ _ Sosa, B. A., Kutay, U. & Schwartz, T. U. Structural insights into LINC complexes. Curr. Opin. Struct. Biol. 23, 285u2013291 (2013)._ _ _ _ _ Cain, N. E., Tapley, E. C., McDonald, K. L., Cain, B. M. & Starr, D. A. The SUN protein UNC-84 is required only in force-bearing cells to maintain nuclear envelope architecture. J. Cell Biol. 206, 163u2013172 (2014)._ _ _ _ _ Cain, N. E. & Starr, D. A. SUN1 proteins and nuclear envelope spacing. Nucleus 6, 2u20137 (2015)._ _ _ _ Walter, P. & Ron, D. The unfolded protein response: from stress pathway to homeostatic regulation. Science 334, 1081u20131086 (2011)._ _ _ _ _ Reggiori, F. & Molinari, M. ER-phagy: mechanisms, regulation and diseases connected to the lysosomal clearance of the endoplasmic reticulum. Physiol. Rev. 102, 1393u20131448 (2022)._ _ _ _ _ Fumagalli, F. et al. Translocon component Sec62 acts in endoplasmic reticulum turnover during stress recovery. Nat. Cell Biol. 18, 1173u20131184 (2016)._ _ _ _ Loi, M. & Molinari, M. Mechanistic insights in recov-ER-phagy: micro-ER-phagy to recover from stress. Autophagy 16, 385u2013386 (2020)._ _ _ _ Loi, M., Raimondi, A., Morone, D. & Molinari, M. ESCRT-III-driven piecemeal micro-ER-phagy remodels the ER during recovery from ER stress. Nat. Commun. 10, 5058 (2019)._ _ _ _ _ Schuck, S., Prinz, W. A., Thorn, K. S., Voss, C. & Walter, P. Membrane expansion alleviates endoplasmic reticulum stress independently of the unfolded protein response. J. Cell Biol. 187, 525u2013536 (2009)._ _ _ _ _ OHara, L. et al. Control of phospholipid synthesis by phosphorylation of the yeast lipin Pah1p/Smp2p Mg2+-dependent phosphatidate phosphatase. J. Biol. Chem. 281, 34537u201334548 (2006)._ _ _ Campbell, J. L. et al. Yeast nuclear envelope subdomains with distinct abilities to resist membrane expansion. Mol. Biol. Cell 17, 1768u2013 1778 (2006)._ _ _ _ _ Panagaki, D. et al. Nuclear envelope budding is a response to cellular stress. Proc. Natl Acad. Sci. USA. 118, 1u201311 (2021)._ _ Barger, S. R., Penfield, L. & Bahmanyar, S. Coupling lipid synthesis with nuclear envelope remodeling. Trends Biochem. Sci. 47, 52u201365 (2022)._ _ _ _ Granzow, H. et al. Egress of alphaherpesviruses: comparative ultrastructural study. J. Virol. 75, 3675u20133684 (2001)._ _ _ _ _ Hagen, C. et al. Structural basis of vesicle formation at the inner nuclear membrane. Cell 163, 1692u20131701 (2015)._ _ _ _ _ Speese, S. D. et al. Nuclear envelope budding enables large ribonucleoprotein particle export during synaptic Wnt signaling. Cell 149, 832u2013846 (2012)._ _ _ _ _ Rigort, A. et al. Focused ion beam micromachining of eukaryotic cells for cryoelectron tomography. Proc. Natl Acad. Sci. USA. 109, 4449u20134454 (2012)._ _ _ _ _ _ Voeltz, G. K. & Prinz, W. A. Sheets, ribbons and tubulesu2014how organelles get their shape. Nat. Rev. Mol. Cell Biol. 8, 258u2013264 (2007)._ _ _ _ Hetzer, M. W. The nuclear envelope. CSH Perspect. Biol. 2, 1u201316 (2010)._ Pirot, P. et al. Global profiling of genes modified by endoplasmic reticulum stress in pancreatic beta cells reveals the early degradation of insulin mRNAs. Diabetologia 50, 1006u20131014 (2007)._ _ _ _ Takatsuki, A. & Tamura, G. Effect of tunicamycin on synthesis of macromolecules in cultures of chick embryo fibroblasts infected with Newcastle Disease Virus. J. Antibiot. 24, 785u2013794 (1971)._ _ _ Bergmann, T. J. et al. Chemical stresses fail to mimic the unfolded protein response resulting from luminal load with unfolded polypeptides. J. Biol. Chem. 293, 5600u20135612 (2018)._ _ _ _ _ Braakman, I., Helenius, J. & Helenius, A. Manipulating disulfide bond formation and protein folding in the endoplasmic reticulum. EMBO J. 11, 1717u20131722 (1992)._ _ _ _ _ Lytton, J., Westlin, M. & Hanley, M. R. Thapsigargin inhibits the sarcoplasmic or endoplasmic reticulum Ca-ATPase family of calcium pumps. J. Biol. Chem. 266, 17067u201317071 (1991)._ _ _ _ Belevich, I., Joensuu, M., Kumar, D., Vihinen, H. & Jokitalo, E. Microscopy Image Browser: a platform for segmentation and analysis of multidimensional datasets. PLoS Biol. 14, e1002340 (2016)._ _ _ _ Bolender, R. P. & Weibel, E. R. A morphometric study of the removal of phenobarbital-induced membranes from hepatocytes after cessation of treatment. J. Cell Biol. 56, 746u2013 761 (1973)._ _ _ _ _ Feldman, D., Swarm, R. L. & Becker, J. Elimination of excess smooth endoplasmic reticulum after phenobarbital administration. J. Histochem. Cytochem. 28, 997u20131006 (1980)._ _ _ _ Staubli, W., Hess, R. & Weibel, E. R. Correlated morphometric and biochemical studies on the liver cell. II. Effects of phenobarbital on rat hepatocytes. J. Cell Biol. 42, 92u2013112 (1969)._ _ _ _ _ Wilhelmsen, K. et al. Nesprin-3, a novel outer nuclear membrane protein, associates with the cytoskeletal linker protein plectin. J. Cell Biol. 171, 799u2013810 (2005)._ _ _ _ _ Ketema, M. et al. Requirements for the localization of nesprin-3 at the nuclear envelope and its interaction with plectin. J. Cell Sci. 120, 3384u20133394 (2007)._ _ _ _ England, C. G., Luo, H. M. & Cai, W. B. HaloTag technology: a versatile platform for biomedical applications. Bioconjug. Chem. 26, 975u2013986 (2015)._ _ _ _ _ Morone, D., Marazza, A., Bergmann, T. J. & Molinari, M. Deep learning approach for quantification of organelles and misfolded polypeptide delivery within degradative compartments. Mol. Biol. Cell 31, 1512u20131524 (2020)._ _ _ _ _ Hodzic, D. M., Yeater, D. B., Bengtsson, L., Otto, H. & Stahl, P. D. Sun2 is a novel mammalian inner nuclear membrane protein. J. Biol. Chem. 279, 25805u201325812 (2004)._ _ _ _ Lamark, T. & Johansen, T. Mechanisms of selective autophagy. Annu. Rev. Cell Dev. Biol 37, 1u201327 (2021)._ _ Kirkin, V. & Rogov, V. V. A diversity of selective autophagy receptors determines the specificity of the autophagy pathway. Mol. Cell 76, 268u2013285 (2019)._ _ _ _ Kuma, A. et al. The role of autophagy during the early neonatal starvation period. Nature 432, 1032u20131036 (2004)._ _ _ _ _ Sun, Q. M. et al. Identification of Barkor as a mammalian autophagy-specific factor for Beclin 1 and class III phosphatidylinositol 3-kinase. Proc. Natl Acad. Sci. USA. 105, 19211u201319216 (2008)._ _ _ _ _ _ Zhong, Y. et al. Distinct regulation of autophagic activity by Atg14L and Rubicon associated with Beclin 1-phosphatidylinositol-3-kinase complex. Nat. Cell Biol. 11, 468u2013476 (2009)._ _ _ _ _ Matsunaga, K. et al. Two Beclin 1-binding proteins, Atg14L and Rubicon, reciprocally regulate autophagy at different stages. Nat. Cell Biol. 11, 385u2013396 (2009)._ _ _ _ Itakura, E., Kishi, C., Inoue, K. & Mizushima, N. Beclin 1 forms two distinct phosphatidylinositol 3-kinase complexes with mammalian Atg14 and UVRAG. Mol. Biol. Cell 19, 5360u2013 5372 (2008)._ _ _ _ _ Diao, J. et al. ATG14 promotes membrane tethering and fusion of autophagosomes to endolysosomes. Nature 520, 563u2013566 (2015)._ _ _ _ _ _ Khaminets, A. et al. Regulation of endoplasmic reticulum turnover by selective autophagy. Nature 522, 354u2013358 (2015)._ _ _ _ _ Galligan, J. J. & Petersen, D. R. The human protein disulfide isomerase gene family. Hum. Genomics 6, 1u201315 (2012)._ _ Guerra, C. & Molinari, M. Thioredoxin-related transmembrane proteins: TMX1 and little brothers TMX2, TMX3, TMX4 and TMX5. Cells 9, 1u2013 11 (2020)._ _ Haugstetter, J. et al. Structure-function analysis of the endoplasmic reticulum oxidoreductase TMX3 reveals interdomain stabilization of the N-terminal redox-active domain. J. Biol. Chem. 282, 33859u2013 33867 (2007)._ _ _ _ Hatahet, F. & Ruddock, L. W. Protein disulfide isomerase: a critical evaluation of its function in disulfide bond formation. Antioxid. Redox Signal. 11, 2807u20132850 (2009)._ _ _ _ Sugiura, Y. et al. Novel thioredoxin-related transmembrane protein TMX4 has reductase activity. J. Biol. Chem. 285, 7135u2013 7142 (2010)._ _ _ _ _ Cheng, L. C. et al. Identification of new transmembrane proteins concentrated at the nuclear envelope using organellar proteomics of mesenchymal cells. Nucleus 10, 126u2013 143 (2019)._ _ _ _ Chino, H., Hatta, T., Natsume, T. & Mizushima, N. Intrinsically disordered protein TEX264 mediates ER-phagy. Mol. Cell 74, 909u2013921 (2019)._ _ _ _ An, H. et al. Tex264 is an endoplasmic reticulum-resident ATG8-interacting protein critical for ER remodeling during nutrient stress. Mol. Cell 74, 891u2013 908 (2019)._ _ _ _ _ Fielden, J. et al. TEX264 coordinates p97- and SPRTN-mediated resolution of topoisomerase 1-DNA adducts. Nat. Commun. 11, 1274 (2020)._ _ _ _ _ _ Fielden, J., Popovic, M. & Ramadan, K. TEX264 at the intersection of autophagy and DNA repair. Autophagy 18, 40u201349 (2021).Fass, D. & Thorpe, C. Chemistry and enzymology of disulfide cross-linking in proteins. Chem. Rev. 118, 1169u20131198 (2018)._ _ _ _ Jessop, C. E., Watkins, R. H., Simmons, J. J., Tasab, M. & Bulleid, N. J. Protein disulphide isomerase family members show distinct substrate specificity: P5 is targeted to BiP client proteins. J. Cell Sci. 122, 4287u20134295 (2009)._ _ _ _ _ Pisoni, G. B., Ruddock, L. W., Bulleid, N. & Molinari, M. Division of labor among oxidoreductases: TMX1 preferentially acts on transmembrane polypeptides. Mol. Biol. Cell 26, 3390u20133400 (2015)._ _ _ _ _ The Human Protein Atlas. (2022).Elliott, J. G., Oliver, J. D. & High, S. The thiol-dependent reductase ERp57 interacts specifically with N-glycosylated integral membrane proteins. J. Biol. Chem. 272, 13849u201313855 (1997)._ _ _ _ Molinari, M. & Helenius, A. Glycoproteins form mixed disulphides with oxidoreductases during folding in living cells. Nature 402, 90u201393 (1999)._ _ _ _ _ van Lith, M., Hartigan, N., Hatch, J. & Benham, A. M. PDILT, a divergent testis-specific protein disulfide isomerase with a non-classical SXXC motif that engages in disulfide-dependent interactions in the endoplasmic reticulum. J. Biol. Chem. 280, 1376u20131383 (2005)._ _ _ Van Lith, M. et al. A developmentally regulated chaperone complex for the endoplasmic reticulum of male haploid germ cells. Mol. Biol. Cell 18, 2795u20132804 (2007)._ _ _ _ Cheng, L. C. et al. Comparative membrane proteomics reveals diverse cell regulators concentrated at the nuclear envelope. Preprint at bioRxiv (2023).Kageyama, S. et al. The LC3 recruitment mechanism is separate from Atg9L1-dependent membrane formation in the autophagic response against Salmonella. Mol. Biol. Cell 22, 2290u20132300 (2011)._ _ _ _ _ Shevchenko, A., Wilm, M., Vorm, O. & Mann, M. Mass spectrometric sequencing of proteins silver-stained polyacrylamide gels. Anal. Chem. 68, 850u2013858 (1996)._ _ _ _ Wilm, M. et al. Femtomole sequencing of proteins from polyacrylamide gels by nano-electrospray mass spectrometry. Nature 379, 466u2013469 (1996)._ _ _ _ _ Keller, A., Nesvizhskii, A. I., Kolker, E. & Aebersold, R. Empirical statistical model to estimate the accuracy of peptide identifications made by MS/MS and database search. Anal. Chem. 74, 5383u20135392 (2002)._ _ _ _ Nesvizhskii, A. I., Keller, A., Kolker, E. & Aebersold, R. A statistical model for identifying proteins by tandem mass spectrometry. Anal. Chem. 75, 4646u20134658 (2003)._ _ _ _ Mastronarde, D. N. Automated electron microscope tomography using robust prediction of specimen movements. J. Struct. Biol. 152, 36u201351 (2005)._ _ _ Hagen, W. J. H., Wan, W. & Briggs, J. A. G. Implementation of a cryo-electron tomography tilt-scheme optimized for high resolution subtomogram averaging. J. Struct. Biol. 197, 191u2013198 (2017)._ _ _ _ Zheng, S. Q. et al. MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nat. Methods 14, 331u2013332 (2017)._ _ _ _ _ Grant, T. & Grigorieff, N. Measuring the optimal exposure for single particle cryo-EM using a 2.6 A reconstruction of rotavirus VP6. Elife 4, e06980 (2015)._ _ _ _ Kremer, J. R., Mastronarde, D. N. & McIntosh, J. R. Computer visualization of three-dimensional image data using IMOD. J. Struct. Biol. 116, 71u201376 (1996)._ _ _ _ Barad, B. A., Medina, M., Fuentes, D., Wiseman, R. L. & Grotjahn, D. A. A surface morphometrics toolkit to quantify organellar membrane ultrastructure using cryo-electron tomography. Preprint at bioRxiv (2022).Pettersen, E. F. et al. UCSF Chimera-a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605u20131612 (2004)._ _ _ _ Mastronarde, D. N. Dual-axis tomography: an approach with alignment methods that preserve resolution. J. Struct. Biol. 120, 343u2013 352 (1997)._ _ _ _ \"- - \"We thank the members of Molinaris laboratory for the discussions and critical reading of the manuscript. We also thank Euro-BioImaging () for providing access to imaging technologies and services via the Italian Node (ALEMBIC, Milan-Italy), Mihajlo Vanevic, Benjamin A. Barad, Miguel R. Leung and Gonzalo Obal for help with data processing and Andreas F. Sonnen for initial help with FIB milling. We are grateful to Stuart C. Howes and Menno Bergmeijer for cryo-EM support as well as to Mariska GrxF6llers-Mulderij for cell culture support. M.M. is supported by the ALPHA-1 Foundation Research Grant (ID: 681136), the Foundation for Research on Neurodegenerative Diseases, the Swiss National Science Foundation (SNF, 310030_184827/2), the Eurostar (E! 113321u2013CHAPERONE), Innosuisse (35449.1 IP-LS), and the Comel and Gelu Foundations. The work was also supported by the European Research Council under the European Unions Horizon 2020 Program (ERC Consolidator Grant Agreement 724425-BENDER) and the Nederlandse Organisatie voor Wetenschappelijke Onderzoek (Vici 724.016.001 to F.F. and Veni 212.152 to J.F.).\"- - Author notes - \"These authors contributed equally: Marika K. Kuciu0144ska, Juliette Fedry.\" - Authors and Affiliations - \"UniversitxE0 della Svizzera italiana (USI), Faculty of Biomedical Sciences, Institute for Research in Biomedicine, CH-6500, Bellinzona, SwitzerlandMarika K. Kuciu0144ska,_Carmela Galli,_Diego Morone,_Andrea Raimondi,_Tatiana SoldxE0_&_Maurizio MolinariDepartment of Biology, Swiss Federal Institute of Technology, CH-8093, Zurich, SwitzerlandMarika K. Kuciu0144skaStructural Biochemistry, Bijvoet Center for Biomolecular Research, Utrecht University, 3584 CG, Utrecht, The NetherlandsJuliette Fedry_&_Friedrich FxF6rsterGraduate School for Cellular and Biomedical Sciences, University of Bern, CH-3000, Bern, SwitzerlandDiego MoroneExperimental Imaging Center, San Raffaele Scientific Institute, I-20132, Milan, ItalyAndrea RaimondiSchool of Life Sciences, xC9 cole Polytechnique FxE9dxE9rale de Lausanne, CH-1015, Lausanne, SwitzerlandMaurizio Molinari\" - \"AuthorsMarika K. Kuciu0144skaYou can also search for this author in _Juliette FedryYou can also search for this author in _Carmela GalliYou can also search for this author in _Diego MoroneYou can also search for this author in _Andrea RaimondiYou can also search for this author in _Tatiana SoldxE0You can also search for this author in _Friedrich FxF6rsterYou can also search for this author in _Maurizio MolinariYou can also search for this author in _\" - Contributions - M.K.K. and J.F. performed the experiments, and the biochemical, imaging, transcriptional and quantitative analyses; C.G. and T.S. prepared DNA constructs and performed qPCR and biochemical analyses; D.M. assisted in CSLM analyses and set up and adapted the LysoQuant program. A.R. performed the analyses with the RT-TEM. J.F. and F.F. performed the FIB-CET analyses. M.M. designed the study, analyzed data, and wrote the manuscript. All the authors discussed the results and the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License ().\"- - In recent years, subsets of human RA-FLS have been extensively characterized leading to the discovery of pathological FLS subsets. Particularly, the PDPNTHY1CD34 sublining layer FLS subset is robustly linked to RA pathology (Fig. ). The SF has been proposed as an alternative source of RA FLS. Following the recent advances in the understanding and subclassification of tRA-FLS, a characterization of sfRA-FLS is needed to facilitate the understanding of results generated by these cells and potentially supplement the use of tRA-FLS. - Figure 4Similarities of pathological FLS harvested from the sublining layer or synovial fluid in RA. (a) A selection of subtype specific surface markers with corresponding dominated cytokine secretion. In general characterized by an upregulation of inflammatory pathways (transcriptomic profiling). (b) FLS harvested from the synovial fluid of flaring RA patients passaged in vitro displaying similar surface markers constantly or inducible, compatible cytokine secretion and increased joint destruction. Created with BioRender.com. - \"Interestingly, after passaging, the sfRA-FLS cultures consisted of primarily PDPNTHY1CD34 FLS. This subset of pathological FLS is present in the synovial fluid at passage 0 but at this passage only constitutes a small fraction of all sfRA-FLS. Hence, the aggressive and proliferative nature of these cells causes them to outgrow other sfRA-FLS subsets during in vitro cultivation. At passage 4, we observed a homogenous culture of around 90% pathological sfRA-FLS. These results show, that the majority of passage 4 sfRA-FLS are undistinguishable from the pathogenic subset of tRA-FLS, specifically with respect to receptor expression and inflammatory cytokine production (Fig. ). Hence, the present study supports that synovial fluid, harvested as part of routine treatment, can be used to acquire and expand pathological FLS utilizing a trypsinization passaging protocol. These PDPNTHY1CD34 sfRA-FLS secrete multiple proinflammatory cytokines and display a uniform basal expression of NFu03BAB pathway-related proteins across different RA donors.\" - \"These results also underline the persisting pathogenic, inflammatory and aggressive phenotype of sfRA-FLS conserved through passaging in the majority of chronic RA patients with active disease even despite ongoing treatments. In addition to an increased production of proinflammatory cytokines, HLA-DR expression were increased on sfRA-FLS after co-culture with either immune cells or addition of exogenous INFu03B3. This demonstrates that sfRA-FLS have the capacity to engage in direct antigen presentation, supporting the findings by Tran et al..\" - \"Further, sfRA-FLS have the ability to invade cartilage at the site of inflammation but also to migrate to and invate human cartilage at a distant site, similar to tRA-FLS previously described by LefxE9vre et al.. This potential could be confirmed recently with a mixed population of adherent SFMC, however, our data suggest that this capacity is mainly, if not solely, mediated by the pathological sfRA-FLS subset. Whether this cartilage invasiveness is direct or through a secondary activation of the surrounding tissue remains to be clarified. However, that the sfRA-FLS without positional identity could potentially inherit both proinflammatory and tissue-destructive properties, as reported recently by Kevin Wei et al.. Therefore, the distinction between lining and sublining layer FLS in RA may not be fixed and could potentially change due to tissue localization, cytokine milieu or extracellular matrix proteins, a plasticity known from synovial macrophages neighbouring FLS in the lining and sublining layer.\" - In the present study, the majority of samples isolated from active RA patients gave rise to proliferating sfRA-FLS cultures. The fact that a fraction of RA SFMC could not form these proliferating cultures may reflect differences in the cellular composition of the inflamed joint. Notably, we detected no differences between patient laboratory parameters or treatment with respect to their capabilities to form proliferating sfRA-FLS cultures (Table ). Despite previous results showing that the treatments had different efficacy in sfRA-FLS cultures. Additionally, FLS from two RA donors showed phenotypic and functional similarities with the sfRA-FLS. To our knowledge, this subset characterization of FLS is the first showing similarities between two human RA cell lines, harvested from either synovial tissue or synovial fluid, and sublining pathological PDPNTHY1CD34 FLS. - Table 2 Patient characteristics. - The present study also describes how sfRA-FLS, resembling pathological FLS, can be obtained and purified from patients with active RA. A limitation to the use of these sfRA-FLS, is that SF is primarily collected from a selected group of RA patients with large joint involvement. Further, SF can usually only be obtained from flaring patients. Thus, sfRA-FLS isolation is mostly relevant for studies on chronic flaring RA patients with large joint involvement. However, the cells from these patients, namely the treatment non-responders, are of particular interest, since a large proportion of this group has previously been shown to have fibroblast-dominated local pathology. Further, after passage 4, sfRA-FLS have proliferated to an extent that allows for the establishment and analysis of autologous co-culture models eliminating the potential confounding factors that may arise from allogenic culturing. - It would be of interest to further compare cells acquired from synovial biopsies with synovial fluid samples in future studies to clarify the potential of sfRA-FLS as a minimal invasive way of stratifying patients to a more personalized treatment. Moreover, whether sfRA-FLS share properties with the recently described non-adherent preinflammatory mesenchymal (PRIME) cells found in increased numbers systemically in RA patients preceding disease flares would also be of key interest to address in future projects. - The present phenotypical and functional characterization of sfRA-FLS, acquired and activated at the site of pathology, will facilitate an improved understanding of the results generated by the use of these cells in various ex vivo and in vivo FLS models. In combination with FLS cell lines, these FLS models may increase our understanding of the role and transformation of pathological FLS in RA. The FLS purification approach and later cultivation may be applicable to a wide range of other inflammatory joint diseases including osteoarthritis, in which SF are accessible.- - Patients and samples - \"A cross-sectional, paired set of PBMC and SFMC were obtained from patients with chronic RA (nu2009=u20099), with at least one swollen joint, at the outpatient clinic at Aarhus University Hospital, at the time of therapeutic arthrocentesis. Out of these nine RA patients, 7 donors were able to provide proliferating and viable FLS cultures at passage 4. Blood and synovial fluid were collected in EDTA tubes. Patient characteristics are described in Table . Immortalized RA FLS (FLS) were kindly provided by Prof. Hermann Eibel.\" - Ethics - All human samples were obtained after informed written consent according to the Declaration of Helsinki and approved by the Local Ethics Committee (The Central Denmark Region committee on health research ethics, project number 20121329; ethics committee Giessen, Germany, 66/08 and 74/05) and the Danish Data Protection Agency. All animal experiments were carried out following the guidelines of the German Animal Welfare Act. - Cell isolation and cultivation - \"PBMC and SFMC were isolated from blood and synovial fluid, respectively, by conventional Ficoll-Paque (GE HealthCare, Chicago, IL, USA) density gradient centrifugation and cryopreserved at -150_xB0C until time of analysis.\" - \"PBMC were cultivated in RPMI (Lonza, Walkersville, MA, USA), 10% fetal calf serum (FCS), 1% penicillin (Lonza), 1% streptomycin (Lonza), 1% HEPES (Gibco, Thermo Fisher Scientific, Waltham, MA, USA) and 1% glutaMAX (Gibco) (later termed RPMI full medium) in a humidified 37_xB0C CO2 incubator. SFMC, sfRA-FLS, FLS and normal human dermal fibroblasts (NHDF) were cultivated in DMEM (Lonza) full medium.\" - \"NHDF originated from foreskin of healthy individual donors undergoing circumcision. The foreskin biopsy was obtained at the Department of Urology at Aarhus University Hospital, Skejby, Denmark and primary foreskin fibroblasts were isolated from the tissue. Immediately after collection, the biopsies were placed in a 100_ mm Petri dish and rinsed in 75% ethanol. Any subcutaneous fat or capillaries were removed and the tissue was cut into pieces of 5u2009xD7u20095_mm to dissociate epidermis and dermis. The tissue pieces were incubated overnight in 10% RPMI medium supplemented with 2.4 U/ml Dispase II (Sigma-Aldrich). To isolate fibroblasts, the biopsies were then incubated in 10% RPMI supplemented with 2_mg/ml collagenase type IV-S (Sigma-Aldrich) and 200 U/ml DNase I (Sigma-Aldrich) for 3u20136_ h with occasional mixing. The digestion was inactivated with FCS and the tissue was disrupted by vortexing for 30_s. Then the cell suspension was filtered through a 70u2013100_xB5m nylon cell strainer and centrifuged at 200xg for 5_min. Supernatant was discarded and the cells were washed before subsequent culture in DMEM with 10% FCS, 1% L-glutamine and 1% penicillin and 1% streptomycin and cryopreserved at -150_xB0C.\" - Synovial fluid derived RA-FLS cultivation - \"SfRA-FLS were isolated by culturing SFMC for 48_h in DMEM full medium whereupon non-attached cells were washed away with PBS and remaining cells were cultured in a humidified 37_xB0C CO2 incubator in DMEM full medium. When cells wereu2009 >u200970% confluent they were washed in PBS and subsequently passaged using trypsinu2013 EDTA treatment (Gibco), as previously described by Nielsen et al.. SfRA-FLS were used for analysis at passage 4 for ex vivo experiments and up to passage 7 for in vivo experiments.\" - Cytokine stimulation of sfRA-FLS and FLS - \"SfRA-FLS and FLS were seeded (nu2009=u200920,000) in a Nuncu2122 48-well plate (Thermo Fisher Scientific) and cultured untilu2009>u200970% confluent. Next the fibroblasts were stimulated with 10_ng/mL TNFu03B1 (Peprotech, Rocky Hill, NJ, USA) or 10_ng/mL IFNu03B3 (Peprotech) in fresh media for 48_h before media were removed and safed at -20_xB0C and cells were harvested for flow cytometry analysis as previously described.\" - Autologous sfRA-FLS and PBMC co-cultures - \"Autologous RA PBMC (nu2009=u20090.5u2009xD7u200910) were seeded in a Nuncu2122 48-well plate (Thermo Fisher Scientific) and activated with DynabeadsxAE Human T-Activator CD3/CD28 (Thermo Fisher Scientific) in a 1:2 bead to cell ratio for 48_h in RPMI full medium supplemented with 15_xB5g/mL gentamicin in a humidified 37_xB0C CO2 incubator. Beads and supernatants were removed after 48_h of activation according to the instruction provided by the manufacture and cells were resuspended in fresh DMEM full medium supplemented with 15_xB5 g/mL gentamicin and transferred to wells withu2009>u200970% confluent autologous sfRA-FLS and cultured for additionally 48_h. PBMC monocultures were cultured in wells without sfRA-FLS for 48_h following 48_h activation.\" - Light microscopy - Cells were examined by bright-field microscopy using Olympus IX71 and pictures were obtained using Leica DFC350F. - Western blot - \"Cells from 3 different sfRA-FLS donors (nu2009=u20091.5u2009xD7u200910) were lysed in respectively 75_xB5l and 25_xB5l Pierce RIPA lysis buffer (Thermo Scientific), supplemented with 10_mM NaF, 1X complete protease cocktail inhibitor (Roche) and 0.25_IU_mL benzonase (Sigma). Whole-cell lysates were denatured for 5_min at 95_xB0C in the presence of Laemmli Sample Buffer, 2X (Sigma) at a dilution of 1:2 before loading on gel. Separation was done by 10% SDS-PAGE gel electrophoresis (Criterion TGX gels, Bio-Rad). Gel was run for 1.5_h at 90_V and subsequently transferred onto a PVDF membrane using a Trans-Blot-Turbo transfer system for 7_min. Membrane was blocked for 1_ h at room temperature with 5% skim milk (Sigma-Aldrich) in TBS with 0.05% Tween-20 (TBST). The membrane was cut into smaller pieces and incubated overnight at 4_xB0C with any of the following specific primary antibodies in TBST with 5% BSA: anti-NFu03BAB p65 (Cell Signaling #8242, 1:1000), anti-phospho-NFu03BA B p65 (Cell Signaling #3303, 1:1000), anti-NFu03BAB p100/p52 (Cell Signaling #3017, 1:1000), anti-Iu03BABu03B1 (Cell Signaling #4814, 1:1000) and anti-Vinculin (Sigma-Aldrich #V9131, 1:7500) used as loading control. After three washes in TBST, secondary antibodies, peroxidase-conjugated F(ab)2 donkey anti-mouse IgG (Hu2009+u2009L) (1:10,000) or peroxidase-conjugated F(ab)2 donkey anti-rabbit IgG (Hu2009+u2009L) (1:10,000) (Jackson Immuno Research) were added in TBST with 5% skim milk for 1_h at room temperature. After three additional washes in TBST, the membranes were exposed on an Image Quant LAS4000 mini imager (GE Healthcare) using Clarity Western ECL Blotting Substrate (Bio-Rad) or the SuperSignal West Femto maximum sensitivity substrate (Thermo Scientific). Protein levels were quantified by densitometry using the online Image J software.\" - Flow cytometry - \"Fibroblasts were blocked with 0.1_mg/mL mouse IgG (Jackson ImmunoReseach) and 0.1_mg/mL rat IgG (Jackson ImmunoReseach) for 15_min and stained with CD34-PerCP-eFlour710 (Clone: 4H11, Thermo Fisher Scientific), CD45-APC-Cy7 (Clone: HI3, Biolegend, San Diego, CA, USA), THY-1-PE-Cy7 (Clone: 5E10, Biolegend), Podoplanin-PE (Clone: NZ-1.3, eBioscience, Thermo Fisher Scientific), ICAM-1-BV421 (Clone: HA58, BD Bioscience, Franklin Lakes, NJ, USA), HLA-DR-BV650 (Clone: L243, Biolegend) and fixable LIVE/DEAD nIR Dead Cell Stain Kit (Life Technologies, Thermo Fisher Scientific). Data were acquired on NovoCyte Quanteon (ACEA Bioscience inc., San Diego, CA, USA) and processed in FlowJo (FlowJo software version 10.5.3).\" - ELISA and mesoscale - \"Supernatants from monocultures of NHDF, sfRA-FLS, PBMC, FLS and sfRA-FLSu2009 +u2009PBMC co-cultures were collected and stored at u2212u200920_xB0C until analysis. MCP-1 and IFNu03B3 levels in supernatants were assessed by ELISA MAXu2122 Deluxe Set Human MCP-1/CCL2 or IFNu03B3 (Biolegend) and levels of IL-1u03B2, -2, -4, -6, -8, -10, -12-p70, -13 and TNFu03B1 were assessed by V-PLEX Proinflammatory Panel 1 Human Kit (Meso Scale Diagnostics, Rockwille, MD, USA). Experiments were performed according to the manufacturers protocol and samples were diluted 1:2.\" - SCID mouse model of RA - \"Female immunodeficient Crl-scidBR mice (Charles River, Germany) were housed under standardized conditions under pathogen-free conditions with water and food ad libitum. Animals underwent inverse-wrap implantation. In brief, subcutaneous implantation of human sfRA-FLS together with cartilage (intact cartilage areas from osteoarthritis patients) in a Gealfoam carrier matrix (Pfizer, USA) was performed at the primary ipsilateral side. Contralaterally, cartilage without sfRA-FLS was implanted. Implants were removed after 45_days, snap frozen and 5_xB5m sections H/E-stained for scoring. Invasion score: 0u2009=u2009no invasion, 1u2009=u2009visible invasion (2 cell depths), 2u2009=u2009invasion (5 cell depths), 3u2009=u2009deep invasion (>u200910 cell depths). The animal study and evaluation was performed in compliance with the ARRIVE guidelines. The outcome measure was the fibroblast-mediated cartilage invasion and scoring of invasion was performed by blinded scorers. All animals were allocated to the respective groups by randomisation and there were no animals excluded from the study.\" - Statistics - \"Statistical analyses and graphs were done using GraphPad Prism 7 for Mac (GraphPad Software). Normally distributed data are represented as meanu2009xB1u2009 SEM and were analyzed by Students paired t-test. Non parametric data are represented as median [25%;75% percentile] and were analyzed by Wilcoxon signed-rank test. Data were transformed to ratios by dividing the value of the samples with the value of the matched control. A two-sided p valueu2009<u20090.05 was considered statistically significant.\" - Ethical approval and consent to participate - All samples were obtained after informed written consent according to the Declaration of Helsinki and approved by the Local Ethics Committee (The Central Denmark Region committee on health research ethics, project number 20121329; ethics committee Giessen, Germany, 66/08 and 74/05) and the Danish Data Protection Agency. Animal experiments were approved by the local committee of the regional council in Hesse, Germany (15-B2/275).- - SfRA-FLS closely resemble the pathological sublining layer FLS subset in terms of surface protein expression, cytokine production and cross-talk potential. Further, sfRA-FLS are comparable to tRA-FLS, specifically with respect to their capability to invade cartilage at implantation sites but also to spread tissue destruction to distant locations. Collectively, models using sfRA-FLS can serve as an alternative source of pathological FLS in RA.- - \\' \\' - The datasets used and/or analysed during the current study are available from the corresponding author on reasonable request. - \\' \\'- - \"BSA:Bovine serum albuminCD:Cluster of differentiationDAS28CRP:Disease activity score 28 based on C-reactive proteinELISA:Enzyme-linked immunosorbent assayFCS:Fetal calf serumFLS:Fibroblast-like synoviocytesHLA-DR: Human leukocyte antigen- DR isotypeICAM-1:Intercellular adhesion molecule 1IFNu03B3:Interferon gammaIg:ImmunoglobulinIL:InterleukinFLS: Immortalized RA human synoviocytesMCP-1:Monocyte chemoattractant protein 1 MTX:MethotrexatNF-u03BAB:Nuclear factor kappa-light-chain-enhancer of activated B cellsNHDF:Normal human dermal fibroblastsPB:Peripheral bloodPBMC: Peripheral blood mononuclear cellsPBS:Phosphate-buffered salinePDPN:Podoplanin PRIME:Preinflammatory mesenchymalRA:Rheumatoid arthritisRA-FLS:Rheumatoid arthritis fibroblast-like synoviocytesRF:Rheumatoid factorSF:Synovial fluidSFMC:Synovial fluid mononuclear cellsSfRA-FLS:Synovial fluid-derived fibroblast-like synoviocytesTBS:Tris-Hcl buffered salineTBST:Tris-Hcl buffered saline with 0.05% Tween-20THY1:Thymocyte differentiation antigen-1 TNFu03B1:Tumor necrosis factor alphaTNFi:Tumor necrosis factor alpha inhibitor tRA-FLS:Tissue-derived fibroblast-like synoviocytesUT:Untreated\"- - \"Silman, A. J. & Pearson, J. E. Epidemiology and genetics of rheumatoid arthritis. Arthr. Res. 4, S265u2013S272 (2002)._ _ Smolen, J. S. et al. Rheumatoid arthritis. Nat. Rev. Dis. Primers 4, 1u201323 (2018)._ _ Nygaard, G. & Firestein, G. S. Restoring synovial homeostasis in rheumatoid arthritis by targeting fibroblast-like synoviocytes. Nat. Rev. Rheumatol. 16, 316u2013333 (2020)._ _ Bartok, B. & Firestein, G. S. Fibroblast-like synoviocytes: Key effector cells in rheumatoid arthritis. Immunol. Rev. 233, 233u2013255 (2010)._ _ _ Bugatti, S., Bozzalla Cassione, E., De Stefano, L. & Manzo, A. Established rheumatoid arthritis. The pathogenic aspects. Best Pract. Res. Clin. Rheumatol. 33, 101478 (2020)._ _ Xia, Z. B. et al. Inhibition of NF-u03BAB signaling pathway induces apoptosis and suppresses proliferation and angiogenesis of human fibroblast-like synovial cells in rheumatoid arthritis. Medicine (United States) 97, e10920 (2018)._ Neidhart, M. et al. Functional characterization of adherent synovial fluid cells in rheumatoid arthritis: Destructive potential in vitro and in vivo. Arthr. Rheum. 48, 1873u2013 1880 (2003)._ _ Whitaker, J. W. et al. An imprinted rheumatoid arthritis methylome signature reflects pathogenic phenotype. Genome Med. 5, 40 (2013)._ _ _ Neumann, E. et al. Cell culture and passaging alters gene expression pattern and proliferation rate in rheumatoid arthritis synovial fibroblasts. Arthr. Res. Ther. 12, R83 (2010)._ _ Mizoguchi, F. et al. Functionally distinct disease-associated fibroblast subsets in rheumatoid arthritis. Nat. Commun. 9, 1u201311 (2018)._ _ _ _ Zhang, F. et al. Defining inflammatory cell states in rheumatoid arthritis joint synovial tissues by integrating single-cell transcriptomics and mass cytometry. Nat. Immunol. 20, 928u2013942 (2019)._ _ Cunningham, C. et al. Characterisation of Rheumatoid and Psoriatic Arthritis Synovial Fibroblasts - ACR Meeting Abstracts. ACR Convergence 2020 (2020).Buckley, C. D. et al. Fibroblasts regulate the switch from acute resolving to chronic persistent inflammation. Trends Immunol. 22, 199u2013 204 (2001)._ _ _ Tran, C. N. et al. Presentation of arthritogenic peptide to antigen-specific T cells by fibroblast-like synoviocytes. Arthr. Rheum. 56, 1497u20131506 (2007)._ _ _ Krzesicki, R. F. et al. T lymphocyte adhesion to human synovial fibroblasts. Role of cytokines and the interaction between intercellular adhesion molecule 1 and CD11a/CD18. Arthr. Rheum. 34, 1245u20131253 (1991)._ _ _ Miranda-CarxFAs, M.-E., Balsa, A., Benito-Miguel, M., PxE9 rez de Ayala, C. & MartxEDn-Mola, E. IL-15 and the initiation of cell contact-dependent synovial fibroblast-T lymphocyte cross-talk in rheumatoid arthritis: Effect of methotrexate. J. Immunol. 173, 1463u20131476 (2004)._ _ Wei, K. et al. Notch signalling drives synovial fibroblast identity and arthritis pathology. Nature (2020)._ _ _ _ Croft, A. P. et al. Distinct fibroblast subsets drive inflammation and damage in arthritis. Nature 570, 246u2013251 (2019)._ _ _ _ Huang, W. et al. Parallel comparison of fibroblast-like synoviocytes from the surgically removed hyperplastic synovial tissues of rheumatoid arthritis and osteoarthritis patients. BMC Musculoskelet. Disord. (2019)._ _ _ _ Stebulis, J. A., Rossetti, R. G., Atez, F. J. & Zurier, R. B. Fibroblast-like synovial cells derived from synovial fluid. J. Rheumatol. 32, 301u2013306 (2005)._ _ _ Tamariz, E. & Grinnell, F. Modulation of fibroblast morphology and adhesion during collagen matrix remodeling. Mol. Biol. Cell 13, 3915u20133929 (2002)._ _ _ Haas, C., Aicher, W. K., Dinkel, A., Peter, H. H. & Eibel, H. Characterization of SV40T antigen immortalized human synovial fibroblasts: Maintained expression patterns of EGR-1, HLA-DR and some surface receptors. Rheumatol. Int. 16, 241u2013247 (1997)._ _ _ LefxE8vre, S. et al. Synovial fibroblasts spread rheumatoid arthritis to unaffected joints. Nat. Med. 15, 1414u20131420 (2009)._ _ Wakabayashi, K., Isozaki, T., Ohta, S. & Kasama, T. Subsets of Synovial Fluid Derived Fibroblast-like Synoviocytes in Rheumatoid Arthritis - ACR Meeting Abstracts. ACR Meeting Abstracts (2020).Croft, A. P. et al. Rheumatoid synovial fibroblasts differentiate into distinct subsets in the presence of cytokines and cartilage. Arthr. Res. Ther. 18, 270 (2016)._ _ Buckley, C. D. Macrophages form a protective cellular barrier in joints. Nature 572, 590u2013592 (2019)._ _ _ _ Nerviani, A. et al. A pauci-immune synovial pathotype predicts inadequate response to TNFu03B1-blockade in rheumatoid arthritis patients. Front. Immunol. 11, 845 (2020)._ _ _ Orange, D. E. et al. RNA identification of PRIME cells predicting rheumatoid arthritis flares. N. Engl. J. Med. 383, 218u2013228 (2020)._ _ _ Humby, F. et al. Synovial cellular and molecular signatures stratify clinical response to csDMARD therapy and predict radiographic progression in early rheumatoid arthritis patients. Ann. Rheum. Dis. 78, 761u2013772 (2019)._ _ _ Nielsen, M. A. et al. Responses to cytokine inhibitors associated with cellular composition in models of immune-mediated inflammatory arthritis. ACR Open Rheumatol. 2, 3u201310 (2020)._ _ Stougaard, J., Lomholt, S., Ommen, P., Kelsen, J. & Kragstrup, T. W. The antifibrotic drug pirfenidone inhibits spondyloarthritis fibroblast-like synoviocytes and osteoblasts in vitro. BMC Rheumatol. 2, 33 (2018)._ _ Judex, M. et al. u2018Inverse wrap: An improved implantation technique for virus-transduced synovial fibroblasts in the SCID mouse model for rheumatoid arthritis. Mod. Rheumatol. 11, 145u2013150 (2001)._ _ _ \"- - \"We thank Karin SkovgxE5rd SxF8rensen (Department of Biomedicine, Aarhus University) and Simona Benninghoff (Department of Rheumatology and Clinical Immunology, University Hospital Freiburg) for technical assistant and the FACS Core Facility (Aarhus University, Denmark) for technical assistant regarding Flow cytometry. We thank medical doctors and nurses at the Department of Rheumatology and Department of Urology, Aarhus University Hospital for helping to collect the patient samples. Additionally, please contact corresponding author for data requests. We kindly acknowledge the generous grants from Aarhus University Research Foundation and the Danish Rheumatism Association.\"- - MAN was supported by Aarhus University Research Foundation (NOVA, AUFF-E-2016-9-27) and the Danish Rheumatoid Association (R188-A6589).- - Authors and Affiliations - \"Department of Biomedicine, Aarhus University, HxF8egh-Guldbergs Gade 10, 8000, Aarhus, DenmarkDitte KxF8ster,_Johanne Hovgaard Egedal,_SxF8ren Lomholt,_ Malene Hvid,_Martin R. Jakobsen,_Bent Deleuran,_Tue Wenzel Kragstrup_&_Morten Aagaard NielsenDepartment of Clinical Medicine, Aarhus University, Aarhus, DenmarkMalene HvidDepartment of Internal Medicine and Rheumatology, Justus-Liebig-University Giessen, Campus Kerckhoff, Bad Nauheim, GermanyUlf MxFCller-Ladner_&_Elena NeumannDepartment of Rheumatology and Clinical Immunology and Center for Chronic Immunodeficiency, Medical Center and Faculty of Medicine, University of Freiburg, Freiburg, GermanyHermann EibelDepartment of Rheumatology, Aarhus University Hospital, Aarhus, DenmarkBent Deleuran,_Tue Wenzel Kragstrup_&_Morten Aagaard Nielsen\" - \"AuthorsDitte KxF8sterYou can also search for this author in _Johanne Hovgaard EgedalYou can also search for this author in _SxF8ren LomholtYou can also search for this author in _Malene HvidYou can also search for this author in _Martin R. JakobsenYou can also search for this author in _Ulf MxFCller-LadnerYou can also search for this author in _Hermann EibelYou can also search for this author in _Bent DeleuranYou can also search for this author in _Tue Wenzel KragstrupYou can also search for this author in _Elena NeumannYou can also search for this author in _Morten Aagaard NielsenYou can also search for this author in _\" - Contributions - D.K. helped to collect the samples, performed experiments, analyzed data, and wrote the manuscript. J.H.E., T.W., S.L., H.E., E.N., and M.A.N., helped to collect the samples, performed experiments and analyzed data. M.A.N., T.W. and E.N. conceived the project, supervised the work, analyzed data, and co-wrote the manuscript. All authors discussed the results and commented on the final manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License ().\\'- - Current immunotherapies have improved the life of patients with lung cancer. However, combined therapeutic strategies are needed to extend the success of this therapy. One problem is nutrition which can be imbalanced and thus might support the tumor growth. We recently reported our investigation on the role of blood glucose in our cohort of patients with LUAD. Other studies also demonstrated that glucose restriction in the diet resulted in decreased tumor development in a murine model of lung cancer. Following up this observation, we asked about the role of lipids in the same cohort of patients with LUAD. In this study we surprisingly did not find any difference in the level of lipids such as triglycerides and cholesterol in the serum from patients with LUAD as compared to control subjects analyzed. However, our cohort of patients with lung cancer had a higher BMI as compared to control subjects. Moreover, by comparing the BMI with the triglycerides we found a direct correlation between these two values in the tumor patients but not in the control subjects. These data support a contribution of the triglycerides to the BMI in lung cancer. Obesity is suspected to be a risk for cancer. However, it is not clear if obesity per se is linked to the tumor growth. In our cohort we could not find any correlation between triglycerides and tumor diameter (Supplementary Fig. a,b). To continue our investigation, we analyzed HDL cholesterol and BMI and found an inverse correlation both in control and in patients with lung cancer. Moreover, when we correlated the total blood cholesterol with the tumor diameter of the patients with lung cancer, we also found an indirect correlation. Altogether this would suggest a worse prognosis in patients with high triglycerides as compared to patients with high blood cholesterol. - \"Blood cholesterol is one of the main risks for ischemic heart disease and ischemic stroke._However, it is important to analyze the lipid fractions including HDL and non-HDL (LDL) cholesterol. These fractions have been classified as good and bad-fat, respectively, to indicate their effect on cardiovascular disease and are influenced by nutritional and life style components and medications. Consistently, we found an indirect correlation between BMI_and serum levels of HDL-Cholesterol, in our patients. Moreover, we found an increased overall survival in patients with increased serum HDL cholesterol, whereas no difference in overall survival was found considering the LDL cholesterol level in serum of these patients. Furthermore, consistent with data from the literature, the highest risk of cardiovascular diseases was found in patients with higher inflammation marker CRP and lower HDL cholesterol. Moreover, we found that total serum cholesterol inversely correlated with the tumor diameter as well as with serum CRP and overall survival.\" - \"We next reasoned that the blood cholesterol would not reflect the cholesterol accumulating in the lung tumor environment where tumor cell growth is supported by intracellular storage of cholesterol. To extend this investigation, we isolated mRNA from the tumoral region and tumor free region (control region) of the lung of our LUAD patients and measured HMGCR and ACAT1 mRNA, two enzymes responsible of cholesterol accumulation into the tumor cell endosomes. In this investigation, we found increased mRNA levels of both HMGCR and ACAT1 in the tumoral region of the lung as compared to the control region in the lung of patients with LUAD. Finally, we found that the blood cholesterol directly correlated with ACAT1 mRNA in the control region but not in the tumoral region_of the lung of patients with LUAD. Considering that the blood cholesterol level was not different between controls and LUAD patients, our data indicate that both HMGCR and ACAT1 mRNA expression in lung_tumor contribute to increased intracellular_cholesterol accumulation that does not correlate with cholesterol egression from the_lung tumor cells. To demonstrate this point we measured the transporter molecule ABCA1 responsible for the cholesterol transport outside the cells and found it downregulated in the tumoral region. Thus, this study demonstrates that tumor progression is strictly dependent by the cancer cells increased cholesterol metabolism and decreased cholesterol egression from tumor cells rather than the absolute level of cholesterol in the blood.\" - \"Moreover, we have identified HMGCR and ACAT1 expression in the tumor region as possible markers for tumor progression and ABCA1 as anti-cancer gene in LUAD. These genes can easily be measured at the time of lung surgery. In conclusion, our work adds knowledge to recent literature indicating that tumor progression is associated with cancer cell softening in which cancer cells have plasma membrane enriched in cholesterol, a condition that makes tumor cells resistant to the cytotoxic effect of T cells. This_indicates that, non-HDL cholesterol reduction in the diet would also increase the effect of current immunotherapy. Further, detailed analysis on fraction of cholesterol accumulation in cancer cells of these patients should be performed in the future to improve current immunotherapy for lung cancer.\"- - \\' \\' - Moreover, the raw data analysed during this study are included in the supplementary file in the tables. - \\' \\'- - \"Dyba, T. et al. The European cancer burden in 2020: Incidence and mortality estimates for 40 countries and 25 major cancers. Eur. J. Cancer 157, 308u2013347 (2021)._ _ _ _ Siegel, R. L. et al. Cancer statistics, 2023. CA Cancer J. Clin. 73(1), 17u201348 (2023)._ _ _ Tang, M. et al. Levels of pretreatment blood lipids are prognostic factors in advanced NSCLC patients treated with anlotinib. Lipids Health Dis. 20(1), 165 (2021)._ _ _ _ _ _ Gahler, A. et al. Glucose-restricted diet regulates the tumor immune microenvironment and prevents tumor growth in lung adenocarcinoma. Front. Oncol. 12, 873293 (2022)._ _ _ _ Lu, Z. et al. Tumor factors stimulate lysosomal degradation of tumor antigens and undermine their cross-presentation in lung cancer. Nat. Commun. 13(1), 6623 (2022)._ _ _ _ _ _ _ Tang, Q. et al. Enhanced CHOLESTEROL biosynthesis promotes breast cancer metastasis via modulating CCDC25 expression and neutrophil extracellular traps formation. Sci. Rep. 12(1), 17350 (2022)._ _ _ _ _ _ Pan, Z. et al. Cholesterol promotes EGFR-TKIs resistance in NSCLC by inducing EGFR/Src/Erk/SP1 signaling-mediated ERRalpha re-expression. Mol. Cancer 21(1), 77 (2022)._ _ _ _ _ _ Karayama, M. et al. Increased serum cholesterol and long-chain fatty acid levels are associated with the efficacy of nivolumab in patients with non-small cell lung cancer. Cancer Immunol. Immunother. 71(1), 203u2013217 (2022)._ _ _ _ Gu, J. et al. Cholesterol homeostasis and cancer: A new perspective on the low-density lipoprotein receptor. Cell Oncol. (Dordr.) 45(5), 709u2013728 (2022)._ _ _ _ _ Shao, J. et al. Prognostic significance of the preoperative controlled nutritional status score in lung cancer patients undergoing surgical resection. Nutr. Cancer 73(11u201312), 2211u20132218 (2021)._ _ _ Liu, W. et al. Dysregulated cholesterol homeostasis results in resistance to ferroptosis increasing tumorigenicity and metastasis in cancer. Nat. Commun. 12(1), 5103 (2021)._ _ _ _ _ _ Li, Z. et al. Single-cell lipidomics with high structural specificity by mass spectrometry. Nat. Commun. 12(1), 2869 (2021)._ _ _ _ _ _ Hoppstadter, J. et al. Dysregulation of cholesterol homeostasis in human lung cancer tissue and tumour-associated macrophages. EBioMedicine 72, 103578 (2021)._ _ _ _ Pan, J. et al. Potentiation of Kras peptide cancer vaccine by avasimibe, a cholesterol modulator. EBioMedicine 49, 72u201381 (2019)._ _ _ _ _ Bi, M. et al. Effect of inhibiting ACAT-1 expression on the growth and metastasis of Lewis lung carcinoma. Oncol. Lett. 18(2), 1548u20131556 (2019)._ _ _ _ _ Huang, B., Song, B. L. & Xu, C. Cholesterol metabolism in cancer: Mechanisms and therapeutic opportunities. Nat. Metab. 2(2), 132u2013141 (2020)._ _ _ Ueno, G. et al. ACAT-1-regulated cholesteryl ester accumulation modulates gemcitabine resistance in biliary tract cancer. Ann. Surg. Oncol. 29(5), 2899u20132909 (2022)._ _ _ _ Lei, K. et al. Cancer-cell stiffening via cholesterol depletion enhances adoptive T-cell immunotherapy. Nat. Biomed. Eng. 5(12), 1411u20131425 (2021)._ _ _ _ _ Wu, K. et al. Roles of ABCA1 in cancer. Oncol. Lett. 24(4), 349 (2022)._ _ _ _ _ _ Phillips, M. C. Molecular mechanisms of cellular cholesterol efflux. J. Biol. Chem. 289(35), 24020u201324029 (2014)._ _ _ _ _ Liu, T. et al. Clinical significance of kinetics of low-density lipoprotein cholesterol and its prognostic value in limited stage small cell lung cancer patients. Cancer Control 28, 10732748211028256 (2021)._ _ _ _ Lin, X. et al. Blood lipids profile and lung cancer risk in a meta-analysis of prospective cohort studies. J. Clin. Lipidol. 11(4), 1073u20131081 (2017)._ _ _ Shin, J. et al. Anticancer potential of lipophilic constituents of eleven shellfish species commonly consumed in Korea. Antioxidants 10(10), 1629 (2021)._ _ _ _ _ Lee, S. S. et al. Avasimibe encapsulated in human serum albumin blocks cholesterol esterification for selective cancer treatment. ACS Nano 9(3), 2420u20132432 (2015)._ _ _ _ _ _ Travis, W. D. et al. The 2015 World Health Organization classification of lung tumors: Impact of genetic, clinical and radiologic advances since the 2004 classification. J. Thorac. Oncol. 10(9), 1243u20131260 (2015)._ _ _ Rosen, R. D. & Sapra, A. TNM Classification. In StatPearls Publishing Copyright xA9 2022 (StatPearls Publishing LLC, 2022).Jacobs, N. J. & Vandemark, P. J. The purification and properties of the alpha-glycerophosphate-oxidizing enzyme of Streptococcus faecalis 10C1. Arch. Biochem. Biophys. 88, 250u2013 255 (1960)._ _ _ _ Koditschek, L. K. & Umbreit, W. W. Alpha-glycerophosphate oxidase in Streptococcus faecium F 24. J. Bacteriol. 98(3), 1063u20131068 (1969)._ _ _ _ _ Allain, C. C. et al. Enzymatic determination of total serum cholesterol. Clin. Chem. 20(4), 470u2013475 (1974)._ _ _ _ Miki, Y. A homogeneous assay for the selective measurement of ldl-cholesterol in serum: enzymatic selective protection method. Clin. Lab. 45, 398u2013401 (1999)._ SimxF3, J. M. et al. Evaluation of a fully-automated particle-enhanced turbidimetric immunoassay for the measurement of plasma lipoprotein(a). Population-based reference values in an area with low incidence of cardiovascular disease. Clin. Biochem. 36(2), 129u201334 (2003)._ _ _ Weir, C. B. & Jan, A. BMI Classification Percentile And Cut Off Points (StatPearls, 2022)._ Li, Q. et al. The distribution of cardiovascular-related comorbidities in different adult-onset cancers and related risk factors: Analysis of 10 year retrospective data. Front. Cardiovasc. Med. 8, 695454 (2021)._ _ _ _ _ Cerqueira, N. M. et al. cholesterol biosynthesis: A mechanistic overview. Biochemistry 55(39), 5483u20135506 (2016)._ _ _ _ Mayengbam, S. S. et al. Influence of cholesterol on cancer progression and therapy. Transl. Oncol. 14(6), 101043 (2021)._ _ _ _ _ Antalis, C. J. et al. High ACAT1 expression in estrogen receptor negative basal-like breast cancer cells is associated with LDL-induced proliferation. Breast Cancer Res. Treat. 122(3), 661u2013670 (2010)._ _ _ _ Pilotto, S. et al. Nutritional support in lung cancer: Time to combine immunonutrition with immunotherapy?. Nutrition 98, 111637 (2022)._ _ _ Pilikidou, M. et al. Lung cancer, treatment and nutritional status. Mol. Clin. Oncol. 15(6), 248 (2021)._ _ _ _ _ Lien, E. C. et al. Low glycaemic diets alter lipid metabolism to influence tumour growth. Nature 599(7884), 302u2013307 (2021)._ _ _ _ _ _ Ma, C. et al. Prognostic significance of preoperative serum triglycerides and high-density lipoproteins cholesterol in patients with non-small cell lung cancer: A retrospective study. Lipids Health Dis. 20(1), 69 (2021)._ _ _ _ _ Lv, Y. et al. High BMI and low HDL-C predict the chemotherapy-related hepatic dysfunction in Chinese advanced NSCLC patients. Cancer Biomark. 16(1), 89u201397 (2016)._ _ _ _ _ Ridker, P. M. et al. C-reactive protein and other markers of inflammation in the prediction of cardiovascular disease in women. N. Engl. J. Med. 342(12), 836u2013843 (2000)._ _ _ _ \"- - \"The authors thank the teams at the Molecular Pneumology, Institute of Pathology and Laboratory of Clinic Medicine, Friedrich-Alexander-UniversitxE4t (FAU) Erlangen-NxFCrnberg, UniversitxE4tsklinikum Erlangen. This work was supported by the Molecular Pneumology Department in Erlangen. We_acknowledge financial support by Deutsche Forschungsgemeinschaft Project_FI 817/5-3 awarded to S.F._ and Friedrich-Alexander-UniversitxE4t Erlangen-NxFCrnberg within the funding programme \"Open Access Publication Funding\". The present work was performed in fulfilment of the requirements for obtaining the degree u201CDr. med. dent.u201D for Philipp Hartmann.\"- - Open Access funding enabled and organized by Projekt DEAL.- - Authors and Affiliations - \"Department of Molecular Pneumology, Friedrich-Alexander-UniversitxE4t (FAU) Erlangen-NxFCrnberg, UniversitxE4tsklinikum Erlangen, 91052, Erlangen, GermanyPhilipp Hartmann,_Katja Hohenberger,_Patrick Tausche,_Sonja Trump,_Susanne Mittler_ &_Susetta FinottoDepartment of Thoracic Surgery, Friedrich-Alexander-UniversitxE4 t (FAU) Erlangen-NxFCrnberg, UniversitxE4tsklinikum Erlangen, 91052, Erlangen, GermanyDenis I. Trufa_&_Horia SirbuInstitute of Pathology, Friedrich-Alexander-UniversitxE4 t (FAU) Erlangen-NxFCrnberg, UniversitxE4tsklinikum Erlangen, 91052, Erlangen, GermanyCarol I. Geppert,_Ralf J. Rieker_&_Arndt HartmannLaboratory of Clinic Medicine, Friedrich-Alexander-UniversitxE4t (FAU) Erlangen-NxFCrnberg, UniversitxE4 tsklinikum Erlangen, 91052, Erlangen, GermanyOliver SchieweckComprehensive Cancer Center Erlangen-EMN (CCC ER-EMN), Erlangen, GermanyHoria Sirbu,_Arndt Hartmann_ &_Susetta FinottoLaboratories of Cellular and Molecular Lung Immunology, Department of Molecular Pneumology, Friedrich-Alexander-UniversitxE4t Erlangen-NxFCrnberg, HartmannstraxDFe 14, 91052, Erlangen, GermanySusetta Finotto\" - \"AuthorsPhilipp HartmannYou can also search for this author in _Denis I. TrufaYou can also search for this author in _Katja HohenbergerYou can also search for this author in _Patrick TauscheYou can also search for this author in _Sonja TrumpYou can also search for this author in _Susanne MittlerYou can also search for this author in _Carol I. GeppertYou can also search for this author in _Ralf J. RiekerYou can also search for this author in _Oliver SchieweckYou can also search for this author in _Horia SirbuYou can also search for this author in _Arndt HartmannYou can also search for this author in _Susetta FinottoYou can also search for this author in _\" - Contributions - \"P.H.: literature search, data analysis, figures, tables, data interpretation, writing, revision. D.I.T.: data collection. K.H.: Data collection, data analysis,_ figures, editing,_revision. P.T.: Data collection, data analysis, editing. S.T.: Data collection,_revision. S.M.: Data collection. C.I.G.: Data analysis. R.J.R.:Data analysis. O.S.: Data analysis. H.S.: Data collection. A.H.: Data analysis. S.F.: study design, literature search, figures, data interpretation, writing,_revision.\" - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License ().- - Animal Model - \"Eight-week-old male C57BL/6u2009J and TLR4 (Toll-like Receptor 4 - B6.B10ScN-Tlr4lps-del/JthJ - JAX stock #007227) mice were obtained from Jackson Laboratory, for validation and phenotypic characterization of TLR4 see Fig._. Mice were housed on a 12u2009 h light/dark schedule and had free access to water and food. Mice were divided into four different groups: Wild Type (WT); WT tumor-bearing (WT TB); TLR4 and TLR4 tumor-bearing (TLR4 TB). For chronic cold challenge, WT and TLR4 mice were single-caged and housed at room temperature (23u2009xB0C) and cold exposure (6u2009xB0C) for six days (nu2009=u20095 per genotype and per condition) on a 12u2009h light/12u2009h dark cycle with access to a standard chow diet. All of the studies performed were approved by the Ethical Committee for Animal Research from the University of Mogi das Cruzes approved all the adopted procedures, which were carried out in accordance with the ethical principles stated by the Brazilian College of Animal Experimentation - Protocol n. 009/2013.\" - Tumor Model and Collection of Tissues - \"Lewis Lung Carcinoma (LLC) tumor cells were used for inducing cancer cachexia syndrome. LLC was injected subcutaneously into the right flanks (200u2009u03BC l LLC cells u22123.5u2009xD7u200910). Non-tumor bearing control mice received Phosphate-buffered saline (PBS) only. Development of cachexia was monitored by tumor size and body weight for 28 days. The impact of tumor growth and development of cachexia on animal welfare were considered, with appropriate measures to monitor and alleviate suffering implementation according to adopted endpoint criteria. Overnight-fasted mice were euthanized by decapitation without anesthesia, serum was collect and centrifuged (400u2009g, 15u2009min, 4u2009xB0C) and stored at u221280u2009xB0C for later analysis. After euthanasia, subcutaneous adipose tissue (scAT) was carefully dissected and weighed. All the mice were checked if they are positive for the presence of metastasis in the lungs, as well as other visceral organs. If there is a presence of metastasis, the animals are excluded from the experimental group (exclusion_criteria). Body weight was measured daily, at the same time, over the 28-day experimental period on a precision scale (OhausxAE).\" - Atorvastatin Treatment - \"For the inhibition of the TLR4 receptor via pharmacological action, Atorvastatin (CitalacorxAE) treatment was performed, currently described as a selective antagonist for TLR4. Only WT TB mice were used for this procedure. The animals were divided into two experimental groups: WT TBu2009+u2009PBS and WT TBu2009 +u2009Atorvastatin (TBu2009+u2009ATOR). The animals that received Atorvastatin were treated with a concentration of 10u2009mg/kg/day by orogastric gavage during the 27-day of protocol. The same protocol was performed for the animals that underwent the survival experiment.\" - Histological analysis - \"scAT was fixed in HistoChoicexAE MB (Amresco), pH 7.4 for 3u2009h. Fixed tissues were dehydrated and 8-u03BCm sections were used. H&E and Picrosirius staining were performed according to standard procedures (Sigma-Aldrich). The sections were analyzed with a Leica microscope (DM 750). Morphometric aspects were measured by Imagen Pro-Plus 6.0.\" - Immunohistochemistry - \"Immunohistochemistry of scAT was carried out with sections fixed by HistoChoicexAE MB (Amresco) at pH 7.4 for 3u2009hours. Fixed tissues were dehydrated and 5-u03BC m sections were used. Dehydrated tissue was blocked using a two different blocking buffers, a buffer solution of endogenous peroxidase activity with 0.3% H2O2 in methanol and buffer solution of free protein binding sites with 5% normal goat serum. The following primary antibodies were used: UCP1 (ab10983), TNFu03B1 (ab6671), CD68 (ab955) and CD3 (ab16669). We employed the polymer-peroxidase method, using Histofine Simple Stain MAX-PO (Nichirei Biosciences) and Sigma Fast 3,3-diaminobenzidine as substrate (Sigma-Aldrich). Sections were counter-stained with hematoxylin.\" - RNA isolation and qRT-PCR - \"Total RNA was isolated from scAT using QIAzol Lysis Reagent Protocol (QIAGEN) following the manufacturers instructions. cDNA was synthesized from 2u2009 u03BCg of total RNA using iScript cDNA Synthesis Kit (BioRad). RPL19 served as control for internal reference gene. Primer sequences used for qRT-PCR analyses were listed in Table_. Analyses of qRT-PCR products were performed with the Prism 7500 SDS software (Thermo Fisher Scientific). Relative quantification of mRNA amount was obtained by the by 2 method.\" - Western Blot - \"For protein expression analyses, tissues were homogenized in lysis buffer (20u2009 mM HEPES, 150u2009mM NaCl, 2u2009mM EDTA, 1% Triton X-100, 0.1% SDS, 10% glycerol, 0.5% sodium deoxycholate) supplemented with Halt protease and phosphatase inhibitors (Thermo Pierce). Samples from tissue lysates were then resolved by SDS-PAGE, and immunoblots were performed using standard protocols. Membranes were blotted with the following antibodies: HSL (ab45422); phospho-HSL Ser660 (Cell Sig. #4126); ATGL (Cell Sig. #2138); P38 MAPK (Cell Sig. #8690); phospho P38 MAPK (Cell Sig. #9211); phospho-AMPK (Thr172 - Cell Sig. 2531); AMPK (Cell Sig. 2532); phospho-PKA substrate (Cell Sig. #9624); Adiponectin (ab62551) and UCP1 (ab10983), all overnight. The membranes were incubated with the goat anti-rabbit IgG conjugated to horseradish peroxidase secondary antibody (7074 - Cell Signaling Technology) for 1u2009h at room temperature and were detected by ECL Prime (Amersham). For the control of protein loading and transfer, we used Ponceau staining of the membranes. Quantification of the antigenu2013antibody complex was performed by the ImageJ Analysis Software.\" - SVF isolation and Flow cytometry analysis - \"Stroma vascular fraction (SVF) from scAT were excised and minced in PBS with calcium chloride and 0.5% BSA. Collagenase II (Sigma-Aldrich) was added to 1u2009 mg/ml and incubated at 37u2009xB0C for 30u2009minutes with shaking. The cell suspension was filtered through a 100 u03BCm filter and then spin down at 300u2009 g for 5u2009minutes to separate floating adipocytes from the SVF pellet. Cells were incubated with Fc Block (BD Biosciences) prior to staining with conjugated antibodies for 15u2009minutes at 4u2009xB0C. SVF were purified and stained with the following antibody panel: ATMu03D5s were defined as CD68F4/80 subpopulations. M1 and M2 ATMu03D5s were defined as CD68F4/80CD11cCD206 and CD68F4/80CD11cCD206, respectively. Cells were analyzed using a BD Accuri (BD biosciences). The data was analysis was performed using FlowJo.\" - Measurement of plasma cytokines - \\'Plasma was collected using plasma separators (BD Biosciences). Cytokine levels were determined using a Milliplex assay kit (Milliplex MAP mouse cytokine/chemokine magnetic bead panel, #MCYTOMAG-70K - Millipore). Data were analyzed using the Bio-Plex manager software (Bio-Rad Laboratories).\\' - Lipolysis assay - \"Isolated adipocytes were suspended in 40u2009ml of Krebs/Ringer/phosphate buffer (pH 7.4, with 1% BSA) in 0.6u2009ml microtubes with 20u2009ml of adenosine deaminase (0.2 U/ml) and were incubated at 37u2009xB0C for 5u2009min to degrade endogenously released adenosine, which is a potent inhibitor of lipolysis. The cells were then incubated for 1u2009h at 37u2009xB0C either in the presence (stimulated) or absence (basal) of isoproterenol (10M, u03B2-adrenergic agonist) in a final volume of 200u2009u03BCL. At the end of incubation, the reaction mixture was blocked by moving the tubes to a cold-water bath followed by centrifugation at 3500u2009g for 5u2009min at 4u2009xB0C. The infranatant was carefully transferred to microtubes containing 150 u03BCL of silicone oil and re-centrifuged at 3500u2009g for 2u2009min. The concentration in the medium of released glycerol (nmol/10 cells) was quantified with a commercially available reagent (Sigma-Aldrich), measured according to the manufacturers instructions. The lipolysis response was assessed by the ratio of glycerol released relative to the basal condition. For additional analyses of the possible effect of ATOR on inhibiting lipolysis, 3T3-L1 cells were stimulated during 1u2009h, 6u2009h and 24u2009h with lipopolysaccharide (LPS, 100u2009ng/mL). The concentration in the medium of released glycerol was quantified with a commercially available reagent (Sigma-Aldrich), measured according to the manufacturers instructions.\" - Cell culture - \"3T3-L1 pre-adipocytes were plated at 1u2009xD7u200910 in 24-well culture plates and cultured in DMEM (Dulbeccos Modified Eagles Medium) with 4500u2009 mg glucose supplemented with 10% bovine serum and 2% penicillin with streptomycin at pH 7.4. The cells were maintained at 37u2009xB0C with 5% carbon dioxide (CO2) so as not to reach complete confluence until they were induced to differentiate. Preadipocytes were brought to complete confluence (day-2) and after two days of confluence, the culture medium was replaced by differentiation inducer medium, consisting of DMEM, supplemented with 10% fetal bovine serum (FBS), 1u2009u03BC M dexamethasone, 0.5u2009mM 3-isobutyl-1-methylxanthine (IBMX) and 1.67u2009 u03BCM bovine insulin. From the second day of differentiation, the cells were maintained in culture medium containing only 0.83u2009u03BCM insulin and 10% FBS which was changed every 48u2009hours for eight days.\" - Adipose Tissue lipid analysis - \"The total scAT lipids were extracted using the method of Folch. Transesterification of the lipid extracts was performed with acetyl chloride (5% CH3COCl in methanol) and fatty acids (FAs) composition was determined as methyl esters using a Varian Model 3900 Gas Chromatograph (GC) coupled to flame ionization detection (FID). FAs were identified by comparing the retention time using three known standards of fatty acid methyl esters (FAME). The FAMEs were analyzed in a capillary column (CP Wax 52 CB), 0.25u2009mm in thickness, 0.25-u03BCm inside diameter, and 30u2009m in length. Hydrogen was used as a carrier gas at a linear velocity of 22u2009cm/s. The temperature program was 170u2009xB0C for 1u2009min followed by a 2.5u2009xB0C/min ramp to 240u2009xB0C, and a final hold time of 5u2009 min. Injector and FID temperatures were 250 and 260u2009xB0C, respectively. The profiles of individual FAs were calculated using an automatic integrator and presented as percentages of total FAs according to the peak areas. Data are presented as palmitic acid profile (%) of total lipids.\" - Non-Esterified Fatty Acids (NEFA) - \"50u2009u03BCL of serum was collected from the different groups and the levels of NEFA was quantified using a kit following the manufacturers recommendations (MAK044, Sigma).\" - Statistical analysis - \"Data were analyzed in GraphPad Prism 7 (GraphPad Software). The statistical significance of the differences in the means of experimental groups was determined by Student s t-test or and 2-way ANOVA, followed by Tukeys post hoc comparison tests. The data are presented as meansu2009xB1u2009SEM. P values u2264 0.05 were considered significant.\"- - \"Fearon, K., Arends, J. & Baracos, V. Understanding the mechanisms and treatment options in cancer cachexia. Nat Rev Clin Oncol 10, 90u201399, (2013)._ _ _ _ Fearon, K. Cachexia: Treat wasting illness on multiple fronts. Nature 529, 156, (2016)._ _ _ _ _ Tsoli, M., Swarbrick, M. M. & Robertson, G. R. Lipolytic and thermogenic depletion of adipose tissue in cancer cachexia. Semin Cell Dev Biol 54, 68u201381, (2016)._ _ _ _ Batista, M. L. Jr. et al. Cachexia-associated adipose tissue morphological rearrangement in gastrointestinal cancer patients. J Cachexia Sarcopenia Muscle 7, 37u2013 47, (2016)._ _ _ Henriques, F. S. et al. Early suppression of adipocyte lipid turnover induces immunometabolic modulation in cancer cachexia syndrome. FASEB J 31, 1976u20131986, (2017)._ _ _ _ Batista, M. L. Jr. et al. Adipose tissue-derived factors as potential biomarkers in cachectic cancer patients. Cytokine 61, 532u2013 539, (2013)._ _ _ _ Petruzzelli, M. et al. A switch from white to brown fat increases energy expenditure in cancer-associated cachexia. Cell Metab 20, 433u2013447, (2014)._ _ _ _ Aoyagi, T., Terracina, K. P., Raza, A., Matsubara, H. & Takabe, K. Cancer cachexia, mechanism and treatment. World J Gastrointest Oncol 7, 17u2013 29, (2015)._ _ _ _ Beluzi, M. et al. Pioglitazone treatment increases survival and prevents body weight loss in tumor-bearing animals: possible anti-cachectic effect. PLoS One 10, e0122660, (2015)._ _ _ _ _ Zhang, G. et al. Toll-like receptor 4 mediates Lewis lung carcinoma-induced muscle wasting via coordinate activation of protein degradation pathways. Sci Rep 7, 2273, (2017)._ _ _ _ _ _ Cannon, T. Y. et al. The effect of altered Toll-like receptor 4 signaling on cancer cachexia. Arch Otolaryngol Head Neck Surg 133, 1263u20131269, (2007)._ _ _ Beutler, B. A. TLRs and innate immunity. Blood 113, 1399u20131407, (2009)._ _ _ _ _ Lu, Y. C., Yeh, W. C. & Ohashi, P. S. LPS/TLR4 signal transduction pathway. Cytokine 42, 145u2013151, (2008)._ _ _ Hoshino, K. et al. Cutting edge: Toll-like receptor 4 (TLR4)-deficient mice are hyporesponsive to lipopolysaccharide: evidence for TLR4 as the Lps gene product. J Immunol 162, 3749u20133752 (1999)._ _ _ Gao, W., Xiong, Y., Li, Q. & Yang, H. Inhibition of Toll-Like Receptor Signaling as a Promising Therapy for Inflammatory Diseases: A Journey from Molecular to Nano Therapeutics. Front Physiol 8, 508, (2017)._ _ _ _ Chansrichavala, P., Chantharaksri, U., Sritara, P., Ngaosuwankul, N. & Chaiyaroj, S. C. Atorvastatin affects TLR4 clustering via lipid raft modulation. Int Immunopharmacol 10, 892u2013899, (2010)._ _ _ _ Leon, C. G., Tory, R., Jia, J., Sivak, O. & Wasan, K. M. Discovery and development of toll-like receptor 4 (TLR4) antagonists: a new paradigm for treating sepsis and other diseases. Pharm Res 25, 1751u2013 1761, (2008)._ _ _ _ _ Baracos, V. E., Martin, L., Korc, M., Guttridge, D. C. & Fearon, K. C. H. Cancer-associated cachexia. Nat Rev Dis Primers 4, 17105, (2018)._ _ _ Fearon, K. et al. Definition and classification of cancer cachexia: an international consensus. Lancet Oncol 12, 489u2013495, (2011)._ _ Porporato, P. E. Understanding cachexia as a cancer metabolism syndrome. Oncogenesis 5, e200, (2016)._ _ _ _ _ Bossola, M. et al. Increased muscle proteasome activity correlates with disease severity in gastric cancer patients. Ann Surg 237, 384u2013389, (2003)._ _ _ _ Konishi, M., Ishida, J., von Haehling, S., Anker, S. D. & Springer, J. Nutrition in cachexia: from bench to bedside. J Cachexia Sarcopenia Muscle 7, 107u2013109, (2016)._ _ _ _ Rohm, M. et al. An AMP-activated protein kinase-stabilizing peptide ameliorates adipose tissue wasting in cancer cachexia in mice. Nat Med 22, 1120u20131130, (2016)._ _ _ _ Vila, I. K. et al. Immune cell Toll-like receptor 4 mediates the development of obesity- and endotoxemia-associated adipose tissue fibrosis. Cell Rep 7, 1116u20131129, (2014)._ _ _ _ Cancello, R. et al. Reduction of macrophage infiltration and chemoattractant gene expression changes in white adipose tissue of morbidly obese subjects after surgery-induced weight loss. Diabetes 54, 2277u20132286 (2005)._ _ _ Weisberg, S. P. et al. Obesity is associated with macrophage accumulation in adipose tissue. J Clin Invest 112, 1796u20131808, (2003)._ _ _ _ _ _ Aoyama, T., Paik, Y. H. & Seki, E. Toll-like receptor signaling and liver fibrosis. Gastroenterol Res Pract 2010, (2010).Campbell, M. T. et al. Toll-like receptor 4: a novel signaling pathway during renal fibrogenesis. J Surg Res 168, e61u201369, (2011)._ _ _ _ Seki, E. et al. TLR4 enhances TGF-beta signaling and hepatic fibrosis. Nat Med 13, 1324u20131332, (2007)._ _ _ _ Surmi, B. K. & Hasty, A. H. Macrophage infiltration into adipose tissue: initiation, propagation and remodeling. Future Lipidol 3, 545u2013556, (2008)._ _ _ _ _ Fukawa, T. et al. Excessive fatty acid oxidation induces muscle atrophy in cancer cachexia. Nat Med 22, 666u2013671, (2016)._ _ _ _ Arner, P. & Langin, D. Lipolysis in lipid turnover, cancer cachexia, and obesity-induced insulin resistance. Trends Endocrinol Metab 25, 255u2013262, (2014)._ _ _ _ Agustsson, T. et al. Mechanism of increased lipolysis in cancer cachexia. Cancer Res 67, 5531u20135537, (2007)._ _ _ _ Pang, S., Tang, H., Zhuo, S., Zang, Y. Q. & Le, Y. Regulation of fasting fuel metabolism by toll-like receptor 4. Diabetes 59, 3041u20133048, (2010)._ _ _ _ _ Kir, S. et al. Tumour-derived PTH-related protein triggers adipose tissue browning and cancer cachexia. Nature 513, 100u2013104, (2014)._ _ _ _ _ _ Cao, W., Medvedev, A. V., Daniel, K. W. & Collins, S. beta-Adrenergic activation of p38 MAP kinase in adipocytes: cAMP induction of the uncoupling protein 1 (UCP1) gene requires p38 MAP kinase. J Biol Chem 276, 27077u201327082, (2001)._ _ _ _ Chio, C. C., Chang, Y. H., Hsu, Y. W., Chi, K. H. & Lin, W. W. PKA-dependent activation of PKC, p38 MAPK and IKK in macrophage: implication in the induction of inducible nitric oxide synthase and interleukin-6 by dibutyryl cAMP. Cell Signal 16, 565u2013575 (2004)._ _ _ Kir, S. et al. PTH/PTHrP Receptor Mediates Cachexia in Models of Kidney Failure and Cancer. Cell Metab 23, 315u2013323, (2016)._ _ _ _ Bahrami, A., Parsamanesh, N., Atkin, S. L., Banach, M. & Sahebkar, A. Effect of statins on toll-like receptors: a new insight to pleiotropic effects. Pharmacol Res 135, 230u2013238, (2018)._ _ _ _ Chansrichavala, P., Chantharaksri, U., Sritara, P. & Chaiyaroj, S. C. Atorvastatin attenuates TLR4-mediated NF-kappaB activation in a MyD88-dependent pathway. Asian Pac J Allergy Immunol 27, 49u201357 (2009)._ _ _ Fang, D. et al. Atorvastatin suppresses Toll-like receptor 4 expression and NF-kappaB activation in rabbit atherosclerotic plaques. Eur Rev Med Pharmacol Sci 18, 242u2013246 (2014)._ _ _ Hodgkinson, C. P. & Ye, S. Statins inhibit toll-like receptor 4-mediated lipopolysaccharide signaling and cytokine expression. Pharmacogenet Genomics 18, 803u2013813, (2008)._ _ _ _ Iwata, A. et al. Inhibitory effect of statins on inflammatory cytokine production from human bronchial epithelial cells. Clin Exp Immunol 168, 234u2013240, (2012)._ _ _ _ _ Chen, X. et al. Mechanistic Study of Inhibitory Effects of Atorvastatin and Docetaxel in Combination on Prostate Cancer. Cancer Genomics Proteomics 13, 151u2013160 (2016)._ _ _ _ Peng, S. et al. Atorvastatin Inhibits Inflammatory Response, Attenuates Lipid Deposition, and Improves the Stability of Vulnerable Atherosclerotic Plaques by Modulating Autophagy. Front Pharmacol 9, 438, (2018)._ _ _ _ Tsoli, M. & Robertson, G. Cancer cachexia: malignant inflammation, tumorkines, and metabolic mayhem. Trends Endocrinol Metab 24, 174u2013183, (2013)._ _ _ _ Onesti, J. K. & Guttridge, D. C. Inflammation based regulation of cancer cachexia. Biomed Res Int 2014, 168407, (2014)._ _ _ _ Paster, E. V., Villines, K. A. & Hickman, D. L. Endpoints for mouse abdominal tumor models: refinement of current criteria. Comp Med 59, 234u2013241 (2009)._ _ _ _ Folch, J., Lees, M. & Sloane Stanley, G. H. A simple method for the isolation and purification of total lipides from animal tissues. J Biol Chem 226, 497u2013509 (1957)._ _ \"- - \"We thank all members of the Laboratory of Adipose Tissue Biology for helpful discussions and critical reading of the manuscript. This work was supported by SxE3o Paulo Research Foundation (FAPESP) Grants: 2010/51078-1, 2015/19259-0 and CNPq 311966/2015-2 to M.L.B.Jr_and from Instituto Serrapilheira/Serra-1708-15285 to A.B. The contents of this article are solely the responsibility of the authors and do not necessarily represent the official views of FAPESP.\"- - Authors and Affiliations - \"Integrated Group of Biotechnology, Laboratory of Adipose Tissue Biology, University of Mogi das Cruzes, SxE3o Paulo, BrazilFelipe Henriques,_Magno A. Lopes,_ Felipe O. Franco,_Pamela Knobl,_Kaltinaitis B. Santos,_Luana L. Bueno,_Victor A. Correa_&_Miguel L. Batista Jr.Program in Molecular Medicine, University of Massachusetts Medical School, Worcester, Massachusetts, USAFelipe Henriques,_ Alexander H. Bedard_&_Adilson GuilhermeDepartment of Pathology, Federal University of Minas Gerais, Minas, Gerais, BrazilAlexander BirbrairDepartment of Physiological Sciences, State University of MaringxE1, ParanxE1, BrazilSidney B. PeresDepartment of Biochemistry, Boston University School of Medicine, Boston, Massachusetts, USAStephen R. Farmer\" - \"AuthorsFelipe HenriquesYou can also search for this author in _Magno A. LopesYou can also search for this author in _Felipe O. FrancoYou can also search for this author in _Pamela KnoblYou can also search for this author in _Kaltinaitis B. SantosYou can also search for this author in _Luana L. BuenoYou can also search for this author in _Victor A. CorreaYou can also search for this author in _Alexander H. BedardYou can also search for this author in _Adilson GuilhermeYou can also search for this author in _Alexander BirbrairYou can also search for this author in _Sidney B. PeresYou can also search for this author in _Stephen R. FarmerYou can also search for this author in _Miguel L. Batista Jr.You can also search for this author in _\" - Contributions - F.H., S.B.P., S.R.F. and M.L.B.Jr. conceived the study; F.H., M.L.A., F.O.F., P.K., L.L.B.,V.A.C., A.H.B. and K.B.S. performed the research; F.H., A.G., A.B., S.B.P., S.R.F., M.L.A., F.O.F. and M.L.B.Jr. analyzed the data; F.H. and M.L.B.Jr. wrote the paper. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing Interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License ().- - Mechanical and optical properties of PEG-collagen hydrogels - \"The PEG-collagen hydrogels fabricated by mixing neutralized collagen and 4-arm PEG-NHS showed higher storage modulus than loss modulus for both 0.4 and 0.8% concentrations. The storage and loss moduli did not vary as a function of the frequency, and we confirmed the completion of gelation (Fig._A). The storage modulus of 0.4 and 0.8% PEG-collagen hydrogels were 221u2009xB1u20099 and 123u2009xB1u20098_Pa, respectively. The storage modulus of 0.4% PEG-collagen hydrogel was higher than 0.8% PEG-collagen hydrogel. In this study, we chose the 0.8% concentration of 4-arm PEG-NHS for followed experiments. The optical properties of PEG-collagen and nonu2010crosslinked collagen hydrogels were analyzed to determine if the crosslinked hydrogels were suitable for use in the cornea. The chemically crosslinked PEG-collagen hydrogel was shown to be transparent which was comparable with the nonu2010crosslinked collagen hydrogel (Fig._B).\" - Figure 2(A) Dynamic moduli of 0.4% and 0.8% 4-arm polyethylene glycol (PEG)-collagen as a function of frequency. (B) Photograph showing the clarity of 0.8% 4-arm PEG-collagen, non-crosslinked collagen, and collagen with neutralization solution. Both crosslinked and non-crosslinked (physical) collagens showed fair transparency. - Clinical examinations - \"There were no signs of abnormalities on the ocular surface before surgery. Following anterior keratectomy, a round corneal defect, 5_mm in diameter, was detected at the center of the cornea, which was stained after fluorescein dyeing with blue light. All corneas subjected to keratectomy lost their transparency moderately. After 1_week, the corneal defect in the keratectomy-only group remained as evidenced by fluorescence staining. However, the keratectomy with 0.8% 4-arm PEG-collagen-hydrogel-treated group showed faster wound healing in the defect site than the keratectomy-only group (Fig._A). All corneas treated with PEG-collagen hydrogel showed no evidence of inflammation, such as conjunctival infection, stromal infiltration, chamber flare, keratic precipitation, and corneal neovascularization, over 1u20134_weeks of follow-up. IOP in the keratectomy-only group was 11.1u2009 xB1u20090.9, 9.0u2009xB1u20090.8, and 9.0u2009xB1u20090.7_mmHg on the day of surgery, 1_week after surgery, and 4_weeks after surgery, respectively. IOP in the keratectomy with PEG-collagen hydrogel-treated group was 11.1u2009 xB1u20090.8, 10.0u2009xB1u20090.7, and 9.4u2009xB1u20090.5_mmHg on the day of surgery, 1_week after surgery, and 4_weeks after surgery, respectively (Fig._B). Statistical analysis showed no differences between the groups at each time point (pu2009=u20090.56, pu2009=u20090.60, and pu2009=u2009 0.22, respectively). The corneal opacity grades at 1_week and 4_weeks following surgery were 2.6u2009xB1u20090.5 and 2.6u2009xB1u20090.5 in the keratectomy-only group and 2.3u2009xB1u20090.5 and 2.0u2009xB1u20090.7 in the keratectomy with PEG-collagen hydrogel-treated group, respectively (Fig._C). Corneal haze was observed in both keratectomy-only and keratectomy with PEG-collagen hydrogel-treated groups, with no significant differences at all time points (pu2009=u20090.13 and pu2009=u20090.22, respectively).\" - \"Figure 3Representative anterior photographs (A), intraocular pressure (B), and corneal clarity grading by slit-lamp examination (C) of an in vivo rabbit keratectomy model without treatment and with polyethylene glycol-collagen hydrogel application immediately after keratectomy as well as 1_week and 4_weeks after surgery.\" - \"Results of AS-OCT of the normal cornea revealed the outermost tear film as a high-intensity thin layer of the cornea, and the low-intensity thin layer beneath it represented the epithelium. A demarcation line between keratectomy and corneal scarring was observed, which appeared as areas of stromal hyperreflectivity that were more prominent in the keratectomy-only group than in the keratectomy with PEG-collagen hydrogel-treated group. Corneal thickness was 301.3u2009xB1 u200915.8 u03BCm in the keratectomy-only group and 320.4u2009xB1u200922.1 u03BCm in the PEG-collagen hydrogel-treated group and normal corneas, however, there was no statistical differences (Fig._A). Pentacam images of both groups showed irregular astigmatism on the corneal surface (Fig._B).\" - \"Figure 4(A) Anterior segment optical coherence tomography (AS-OCT) cross-sectional images on the horizontal meridian at the center of the cornea, and (B) corneal imaging of axial curvature and corneal thickness measured using a rotating Scheimpflug camera after 4_weeks of surgery in the keratectomy-only and keratectomy with polyethylene glycol (PEG)-collagen hydrogel-treated group. AS-OCT revealed higher density (arrow), suggesting scarring at the center of the epithelial layer in the untreated keratectomy-subjected cornea than the keratectomy-subjected cornea treated with PEG-collagen, which showed high focal density area at the paracenter of the cornea. The regular anterior corneal surface and regular corneal thickness were increased in the keratectomy with PEG-collagen hydrogel-treated group compared with the keratectomy-only group.\" - Histological findings - \"In the keratectomy-only group, H&E-stained specimens showed an epithelial defect after 1_week of surgery, and fibrotic cells were stacked on the center of the cornea at 4_weeks following surgery, with the epithelial cells failing to migrate and proliferate from the periphery. In contrast, the cornea treated with PEG-collagen hydrogel maintained PEG-collagen at the stromal defect site after 1_week of surgery, and the epithelium formed a multilayer over the hydrogel after 4_ weeks of surgery, and cellular infiltrates were seen in PEG-collagen hydrogel (Fig._). Immunofluorescence confocal imaging after 1_week also found PEG-collagen hydrogel underneath the migrated epithelium, suggesting that the injected PEG-collagen was able to support the reepithelialization (Fig._A).\" - \"Figure 5Light microscopy (hematoxylin and eosin staining) images of normal rabbit cornea and cornea subjected to keratectomy with or without polyethylene glycol (PEG)-collagen hydrogel application were obtained after 1 and 4_weeks of follow-up after surgery. The keratectomy-only group did not show complete epithelialization during the 4_weeks of follow-up, whereas the keratectomy with PEG-collagen hydrogel-treated group showed multilayered epithelial cell migration and proliferation. Cellular infiltrates were found in PEG-collagen hydrogel (*), but no significant inflammatory cell infiltration was noted.\" - \"Figure 6Fluorescent confocal microscopy images of rabbit keratectomy cornea without treatment and with polyethylene glycol (PEG)-collagen hydrogel application. (A) Cy5 stained with PEG-collagen hydrogel (magenta) was maintained underneath a multilayered, migrated epithelial layer stained with CK3 (green) after 1_ week of keratectomy. (B) Immunohistochemical staining of u03B1-smooth muscle actin (u03B1-SMA), vimentin, and desmin markers for myofibroblasts in the keratectomy-subjected rabbit cornea over 1_week and 4_weeks. Note the high density of u03B1-SMA (red) and vimentin (green) in the anterior stroma of rabbit corneas at 1_week after keratectomy alone. After 4_weeks, u03B1-SMA (red) expression remained in the epithelium in the keratectomy-only group, whereas low u03B1-SMA expression was found in the keratectomy with PEG-collagen hydrogel-treated group. Desmin (red) was expressed in the epithelium of the keratectomy with PEG-collagen hydrogel-treated group after 1 to 4_weeks.\" - \"Individual staining of u03B1-SMA, which is a conventional marker for myofibroblast associated with corneal haze, was performed in each group, and the double staining of vimentin and desmin was performed to further analyze the corneal wound healing process after keratectomy with or without PEG-collagen hydrogel application. Strong expression of u03B1-SMA (red) and vimentin (green) in the anterior stroma was detected in the keratectomy-only group after 1_week of surgery, and u03B1 -SMA remained on the epithelium and vimentin in the anterior stroma after 4_ weeks of surgery (Fig._B).\"- - \"The human cornea has two unique characteristics. First, the cornea forms the anterior barrier and protects the inner portion of the eye from the external environment. Second, the cornea is the single most powerful focusing element of the eye providing approximately two-thirds of the eyes refractive power. Thus, the following are highly prioritized when considering corneal alternatives: fair transparency with regular anterior surface, adequate strength to overcome intraocular and blinking pressure, and biocompatibility to improve integration. Corneal stroma consists of highly organized collagen in arranged fibrils (collagen types I, V, and VI) with uniform spacing and alignment, which results in the transparent characteristic of the cornea. Collagen is a natural polymer that is a major component of the ECM and has been used as a scaffold in clinical applications.\" - \"The currently available biomaterials used as corneal adhesives cannot be used as stromal substitutes. For instance, the PEG-based materials ReSure (Ocular Therapeutix Inc., MA, USA) or OcuSeal (Beaver-Visitec International, MA, USA) cannot be used as filling biomaterials due to fast and uncontrollable polymerization, low adhesiveness, and lack of the mechanical properties required for regeneration. Preclinical pilot study was demonstrated using human cadaveric donor cornea from ECM microparticles have been used; however, immunologic reactions may be problematic. The newly introduced GelCORE, a gelatin-based adhesive, also showed in vivo biocompatibility and high tissue adhesion for 14_days. A longer duration of observation is needed regarding zoonotic disease transfer or immunologic reactions, and the replacement of gelatin by host collagen should be monitored. Very recently, the performance of fully synthetic collagen-like peptide (CLP)-PEG-fibrinogen LiQD was equivalent to that of syngeneic grafts in vivo, suggesting a role as an alternative to corneal transplantation. We chose a 4ARM PEG collagen formulation with 8% PEF due to its superior transparency both when cells are present and absent in vitro. These PEG-based hydrogels were expected to be able to firmly adhere to the stromal bed due to the reactivity of the NHS moieties with primary amines in the host stromal collagen.\" - Crosslinked collagens are used as hydrogel, sponge, or film and tailored to the shape of the defect site. Recently, three-dimensional bioprinting has enabled the adjustment of the size and depth of the defect. We used liquid in situ-forming hydrogel of collagen type I crosslinked via multifunctional PEG-NHS instead of solid alternative to treat corneal wounds. Liquid hydrogels would have advantages over solid materials in terms of being easier to handle and having no need for being sutured or surgery. However, the controllable gelation time and initiation method, including the use of UV, visible light, or chemicals, need to be considered for purposes of safety and clinical approach. In addition, the overflow of liquid hydrogels from the site of the defect, if not carefully controlled, may prevent epithelial migration over the hydrogel. Filling the defect would make it easy to individualize the various morphologies of corneal defects, and negating the irregular astigmatism by providing a smooth refracting surface is expected to improve vision even with the persistent corneal opacity of scarring. In vitro, we found that the underlying letter was readable in both 0.4% and 0.8% of PEG-collagen hydrogel. However, corneal haze was found in both the keratectomy-only group and the PDG-collagen hydrogel-treated group, in vivo. Histological findings revealed that the PEG-collagen hydrogel, after its application into the corneal stromal defect site, was observed underneath a multilayered epithelial layer, which suggested that PEG-collagen hydrogel maintained in the stromal defect and epithelial migration was not inhibited by hydrogels. We further continued the in vivo observation and performed clinical assessments, including slit-lamp examination, IOP evaluation, AS-OCT, and other tests using Pentacam to analyze the biocompatibility of in situ-forming PEG-collagen hydrogels in vivo. - \"Corneal mechanical strength and structural integrity originate mainly from the collagen present in the stroma. The mechanical properties of non-crosslinked collagen are difficult to control, which can pose a challenge. Therefore, we combined a natural polymer, collagen, and a synthetic polymer, PEG, to achieve adequate strength and control degradation. A prominent advantage of PEG is its ability to control the physical and chemical properties in crosslinked networks. Moreover, PEG is FDA-approved for use in clinical settings as an ocular sealant. Fernandes-Cunha et al. have previously reported that collagen crosslinked with 3-arm PEG at various concentrations had a higher storage modulus than non-crosslinked collagen. In the current study, we found that PEG-collagen hydrogel was maintained stably in the rabbit cornea up to 4_weeks after application. Our in vivo study showed that PEG-collagen hydrogel exhibited sufficient strength and elasticity to endure external (blinking) and internal (IOP) forces.\" - \"Results of the clinical analysis of AS-OCT revealed that the corneal thickness was restored to its normal range in the PEG-collagen-treated rabbit cornea, whereas this thickness decreased in the defective rabbit cornea without treatment. Thus, PEG-collagen hydrogel may play the role of temporary filler until the ECM is repaired. In photorefractive keratectomy, where the epithelium with underlying Bowmans layer and anterior stroma is removed, stromal remodeling takes several months for completion. During the wound healing process, aberrant ECM remodeling can lead to permanent corneal scar. In a previous study in rabbits where superficial stroma was manually removed like in our study, it was shown that the wound completely epithelialized by day 7 and stomal remodeling continued until days 14u201328; the findings were comparable to those of the human keratorefractive study. We hypothesize that PEG-collagen hydrogel effectively and stably fills the stromal defect and enables rapid epithelial coverage at the beginning. This gain time for normal stromal wound healing process involves keratocyte, proinflammatory, and profibrotic cytokine production and ECM-degrading enzyme activity. A deeper stromal defect filled with PEG-collagen is warranted to examine the stromal remodeling including the donor keratocytes grow into the matrix and replace the donor collagen. We stained the intermediate filament markersu2014vimentin and desminu2014and u03B1-SMA to analyze the dynamics of their expression during myofibroblast differentiation after keratectomy with or without PEG-collagen hydrogel application. The normal cornea expresses epithelial u03B1-SMA, vimentin, and desmin; however, anterior stromal staining of the markers has shown that they play a role in corneal scarring. Detailed molecular studies investigating the effect of PEG-collagen hydrogel on the corneal wound healing process need to be conducted.\" - \"Clinical examination following injectable PEG-collagen filling at the stromal site showed no significant adverse effects, such as corneal infiltration, anterior chamber reaction, and hypersensitivity. The immune privilege of the cornea enables the implantation of various therapeutic apertures in the corneal stroma, such as the corneal inlay and intracorneal segment ring. Due to the ubiquitous presence of PEG in cosmetic, cleaning, and personal care products, people can develop antibodies against the polymer. These antibodies can provoke hypersensitivity reactions and accelerate biodegradation, decreasing the therapeutic efficacy of PEG. There is a possibility that the PEG-collagen filling can also be used safely considering its marked association with the immune system. In the long term, further observations are needed to evaluate the effect of PEG-collagen on the absorption of disorganized collagen fibers and other matrix materials secreted by myofibroblasts. Limitation of this study is that we did not analyzed time dependent changes of underlying stroma and endothelium and overlying epithelium of the PEG-collagen, and only observed the 1_week and 4_weeks of results of the treated cornea. Moreover, the fate or role of treat PEG-collagen in the wound healing process is not clearly solved. Nevertheless, we have shown the clinical efficacy of the PEG-collagen hydrogel in vivo and found that the hydrogel filling at the defect site was stable and safe for 4_weeks after surgery. Although the PEG-collagen treated cornea did not show a perfectly smooth and transparent surface, the cornea curvature and thickness were tolerable and were maintained during the follow-up period. In our rabbit model, no steroids, antibiotics, or lubricant eyedrops were instilled after PEG-collagen injection. In the clinical setting, it may be possible for meticulous treatment during wound healing to restore transparency, and customized contact lenses or excimer laser surgery may lessen the astigmatism. Although further studies with longer duration are warranted, we demonstrated that the PEG-collagen hydrogel could be applied without major issues regarding cytotoxicity or foreign body reactions as a corneal stromal defect wound repair matrix.\"- - Materials - \"Unless otherwise noted, all chemicals and solvents were of analytical grade and used as advised by the manufacturer. Four-arm PEG succinimidyl glutarate was purchased from JenKem Technology (Texas, USA). Collagenase, dimethyl sulfoxide, fluorescamine, sodium hydroxide solution (1.0_N), bovine serum albumin, fibronectin, cholera subunit A, insulin, Triton-X, and Cell Counting Kit-8 reagent were purchased from Sigma-Aldrich (St. Louis, MO, USA). Phosphate-buffered saline (PBS; 10u2009 xD7u2009; pH 7.4), Slide-A-Lyzer Dialysis Kit (3.5_kDa MWCO), collagen I bovine protein solution (5_mg/mL), collagenase, Dulbeccos Modified Eagle Medium/Nutrient Mixture F-12 with 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid, insulin-transferrin-selenium, Dulbeccos PBS, antibioticu2013antimycotic, Live/Dead viability/cytotoxicity staining kit, and paraformaldehyde (PFA) were purchased from Thermo Fisher Scientific (Massachusetts, USA).\" - Collagen crosslinking by 4-arm PEG-NHS - \"The pH of type I bovine collagen was neutralized using a solution of 1_M sodium hydroxide solution, deionized (DI) water, and 10u2009xD7u2009PBS in a 3:57:20 ratio, as previously described. Briefly, the collagen solution (5_mg/mL) was mixed with the neutralization solution in a 3:2 ratio, such that the final concentration of collagen was 3_mg/mL. Since the collagen crosslinking occurs with the primary amine group of lysine by NHS ester reaction, the number of lysine residues is important to determine the concentration of each chemical. Lysine constitutes 2.5u20133.0% of collagen, amounting to a total of 75u2013100 lysine residues in the triple helix. Moreover, 3_mg of collagen corresponds to 7.5u2009xD7 u200910u20137.5u2009xD7u200910_mol of primary amine. The 4-arm PEG-NHS has four NHS groups per molecule; the molar equivalent amount of NHS groups is 1.875_ mg per 3_mg of primary amine group in collagen. Thus, 18.75 xB5L of 100_mg/mL of 4-arm PEG-NHS provided the equivalent amount of NHS groups. Four-arm PEG-NHS was dissolved in PBS at a concentration of 100_mg/mL, and the solution was subsequently mixed with the neutralized collagen solution at the desired concentration (0.8% w/v) for 30_s at room temperature. Subsequently, the collagen gels were incubated for 10u201315_min at 37_xB0C for the completion of gelation.\" - Physical properties of PEG-collagen hydrogel - \"The mechanical properties of the collagen hydrogels were evaluated using an MCR 302 rheometer (Anton Paar, Germany). For the PEG-collagen hydrogels, different concentrations of 0.4 and 0.8% of 4-arm PEG-NHS solution was added to the neutralized collagen solution. Then the solution was mixed with a pipette. To ensure complete gelation, the resultant PEG-collagen hydrogels were incubated at 37_xB0C for 2_h, and then frequency sweeps from 0.1 to 10_Hz with a fixed 1% strain were performed.\" - Animals and surgery - \"New Zealand White rabbits (male, 2.5u20133.0_kg) were purchased from DooYeol Biotech. In total, 20 rabbits were subjected to keratectomy, and 10 rabbits were randomly assigned to each group (0.8% 4-arm PEG-collagen hydrogel PEG and keratectomy only). Animals were anaesthetized using an intramuscular injection of Zoletilu2122 50 (15_mg/kg, Virbac, Carros, France) and xylazine hydrochloride (5_mg/kg, Bayer, Leverkusen, Germany). The right eye of each rabbit was chosen for surgery, and the contralateral eye served as the normal control. Three drops of alcaine (Alcon Laboratories, Inc., Ft. Worth, TX, USA) were applied to the cornea before surgery. Manual superficial stromal keratectomy was performed as previously described. Briefly, a 5-mm-diameter keratectomy groove of consistent depth was created by a biopsy punch (Miltex, Inc. York, USA). Subsequently, a 27-gauge needle was used to inject air into the stroma to avoid corneal perforation, and the anterior stroma was manually dissected. Finally, using the freehand technique, approximately 20u201330% of the anterior stroma was removed using a crescent blade. Following anterior lamellar dissection, the defect was filled with 0.8% 4-arm PEG-collagen-Alexa 647 hydrogels. After in situ-forming collagen hydrogel treatment, 3_min of drying time was allowed for complete gelation, and tarsorrhaphy was performed. Half of each group (Nu2009=u20095 eyes), which was chosen randomly, was evaluated 1_week after surgery, while the other half was assessed 4_weeks postoperatively. One week and four weeks after the surgery, the animals were evaluated before being sacrificed. After euthanizing, the corneas were analyzed using light and confocal microscopy. All animal protocols were reviewed and approved by the Institutional Animal Care and Use Committee of the Catholic University of Korea (Institutional Review Board No: YEO-2019u2013 015-01FA). This study was conducted in accordance with all applicable regulations and guidelines, including the ARRIVE 2.0 guidelines.\" - Clinical examination - \"Postoperative examinations were performed at 1 and 4_weeks after surgery. Slit-lamp biomicroscopy (Topcon Medical Systems Inc., USA) was performed to evaluate the anterior corneal surface and chamber, and images of the cornea were acquired using a digital camera (Sony Rx10M4, Japan) to compare its clarity and vascularity. Corneal clarity was graded manually from being completely transparent (0) to completely opaque (4) by the authors YHC, MJH, and KSN. Intraocular pressure (IOP) measurements were first performed using TonoVet (Helsinki, Finland). Because there was no calibration mode available for rabbits, a calibration mode for dogs was chosen, as previously introduced. The central groove of the tonometer was kept in a horizontal position, and the distance was maintained at 4u2013 8_mm between the tip of the probe and cornea. The measurement button was slightly pressed to enable the tip of the probe to hit the central cornea. IOP was measured before noon in each period to eliminate diurnal variations. Six consecutive measurements were recorded, and the resulting average IOP was reported.\" - \"Anterior segment optical coherence tomography (AS-OCT) (Visante OCT, Carl Zeiss Meditec, Inc., USA) was performed at week 4 postoperatively. Background illumination in the room was made constant by completely closing the door and minimizing room illumination. One anterior segment scan image centered over the pupil was acquired along the horizontal axis (between 0xB0 and 180xB0). The scan direction was aligned until a full corneal reflex was achieved. Corneal topography and pachymetry were performed using a rotating Scheimpflug camera (Pentacam; Oculus, Wetzlar, Germany) 4_weeks postoperatively.\" - Histology - \"After the completion of the study (on 1 and 4_weeks), tissues that required to be examined were isolated. The cornea was carefully separated from the eye without damaging it and washed using 1u2009xD7u2009PBS. Tissues were fixed using 4% PFA solution overnight at 4_xB0C and then washed in 1u2009xD7u2009 PBS. The tissue segments were dehydrated in graded ethanol and embedded in paraffin, and 3u20134_xB5m-thick sections were cut using a microtome. All the sections were subjected to hematoxylin and eosin (H&E) and immunofluorescence staining for histopathological analysis.\" - H&E and immunofluorescence staining - \"For H&E staining, tissue Sects._(3u20134_xB5m) were immersed in xylene (three times for 5_min); 100% (two times), 95%, 85%, and 70% ethanol for 1_min each; DI water for 1_min; and hematoxylin (BBC Biochemical) and eosin (BBC Biochemical) solutions for 10 to 20_s before washing with pure xylene.\" - \"For immunofluorescence staining, tissue slides were rehydrated by immersing in xylene (three times for 5_min); 100% (two times), 95%, 85%, and 70% ethanol for 5_min each; and DI water for 5_min. The antigen retrieval buffer (citrate buffer, pH 6.0) was preheated to 92u201395_xB0C. Slides were immersed in the preheated solution for 5u201310_min. The tissues were encircled with a hydrophobic barrier using a barrier pen. Tissues were further blocked with 2.5% Normal Horse Serum (Vector, Burlingame, CA, USA) for 30u201360_min.\" - \"The slides were washed and incubated with primary antibodies: mouse monoclonal antibody against CK3 (ab68260, Abcam, Cambridge, UK) diluted 1:50, mouse monoclonal antibody against u03B1-smooth muscle actin (u03B1-SMA) (ab7817, Abcam, Cambridge, UK) diluted 1:50, anti-vimentin antibody (ab45939, Abcam, Cambridge, UK) diluted 1:100, and anti-desmin monoclonal antibody (MA1-2215, Invitrogen, Carlsbad, CA, USA) diluted 1:100 at 4_xB0C overnight. The next day, after washing with 1u2009xD7u2009PBS (three times), the sections were incubated with goat anti-rabbit Alexa Fluor 488-conjugated secondary antibody (A-11008, Invitrogen, Carlsbad, CA, USA) or goat anti-mouse Alexa Fluor 546-conjugated secondary antibody (A-11003, Invitrogen, Carlsbad, CA, USA) at room temperature for 2_h in the dark. The slides were then mounted with a mounting medium containing 4u2032,6-diamidino-2-phenylindole (Vector). For negative controls, non-immune serum was used instead of the specific primary antibody. The corneal sections were observed and imaged with a confocal microscope (ZEISS LSM 880, Carl Zeiss AG, Oberkochen, Germany).\" - Statistical analyses - \"All data are expressed as the meanu2009xB1u2009standard deviation. Each experiment was performed at least three times, unless otherwise indicated. Statistical evaluation was performed using two-way analysis of variance. Results with pu2009 <u20090.05 were considered statistically significant. Statistical analysis was performed using the statistical software GraphPad Prism 7.\"- - \"Flaxman, S. R. et al. Global causes of blindness and distance vision impairment 1990u20132020: a systematic review and meta-analysis. Lancet Glob. Health 5, e1221u2013e1234 (2017)._ _ Fini, M. E. & Stramer, B. M. How the cornea heals: cornea-specific repair mechanisms affecting surgical outcomes. Cornea 24, S2-s11 (2005)._ _ Matthyssen, S., Van den Bogerd, B., Dhubhghaill, S. N., Koppen, C. & Zakaria, N. Corneal regeneration: A review of stromal replacements. Acta Biomater. 69, 31u201341 (2018)._ _ _ Sarnicola, E., Sarnicola, C., Cheung, A. Y., Holland, E. J. & Sarnicola, V. Surgical corneal anatomy in deep anterior lamellar keratoplasty: Suggestion of new acronyms. Cornea 38, 515u2013522 (2019)._ _ Gain, P. et al. Global survey of corneal transplantation and eye banking. JAMA Ophthalmol. 134, 167u2013173 (2016)._ _ McTiernan, C. D. et al. LiQD Cornea: Pro-regeneration collagen mimetics as patches and alternatives to corneal transplantation. Sci. Adv. 17, eaba2187 (2020)._ _ _ ShirzaeiSani, E. et al. Sutureless repair of corneal injuries using naturally derived bioadhesive hydrogels. Sci. Adv. 20, eaav1281 (2019)._ _ _ Brunette, I. et al. Alternatives to eye bank native tissue for corneal stromal replacement. Prog. Retin. Eye Res. 59, 97u2013130 (2017)._ _ Ghezzi, C. E., Rnjak-Kovacina, J. & Kaplan, D. L. Corneal tissue engineering: recent advances and future perspectives. Tissue Eng. Part B Rev. 21, 278u2013287 (2015)._ _ Chowdhury, S. R. et al. Collagen type I: A versatile biomaterial. Adv. Exp. Med. Biol. 1077, 389u2013414 (2018)._ _ _ Hassell, J. R. & Birk, D. E. The molecular basis of corneal transparency. Exp. Eye Res. 91, 326u2013335 (2010)._ _ _ Depalle, B., Qin, Z., Shefelbine, S. J. & Buehler, M. J. Influence of cross-link structure, density and mechanical properties in the mesoscale deformation mechanisms of collagen fibrils. J. Mech. Behav. Biomed. Mater. 52, 1u201313 (2015)._ _ Viguet-Carrin, S., Garnero, P. & Delmas, P. D. The role of collagen in bone strength. Osteoporos. Int. 17, 319u2013336 (2006)._ _ _ Irawan, V., Sung, T. C., Higuchi, A. & Ikoma, T. Collagen scaffolds in cartilage tissue engineering and relevant approaches for future development. Tissue Eng. Regen. Med. 15, 673u2013697 (2018)._ _ _ Coenen, A. M. J., Bernaerts, K. V., Harings, J. A. W., Jockenhoevel, S. & Ghazanfari, S. Elastic materials for tissue engineering applications: Natural, synthetic, and hybrid polymers. Acta Biomater. 79, 60u201382 (2018)._ _ _ Chaudhari, A. A. et al. Future Prospects for Scaffolding Methods and Biomaterials in Skin Tissue Engineering: A Review. Int. J. Mol. Sci. 17 (2016).Samarawickrama, C. et al. Collagen-based fillers as alternatives to cyanoacrylate glue for the sealing of large corneal perforations. Cornea 37, 609u2013616 (2018)._ _ Fernandes-Cunha, G. M. et al. In situ-forming collagen hydrogel crosslinked via multi-functional PEG as a matrix therapy for corneal defects. Sci. Rep. 10, 16671 (2020)._ _ _ _ Meek, K. M. & Knupp, C. Corneal structure and transparency. Prog. Retin. Eye Res. 49, 1u201316 (2015)._ _ Park, H. C., Champakalakshmi, R., Panengad, P. P., Raghunath, M. & Mehta, J. S. Tissue adhesives in ocular surgery. Expert Rev. Ophthalmol. 6, 631u2013655 (2011)._ _ Uy, H. S. & Kenyon, K. R. Surgical outcomes after application of a liquid adhesive ocular bandage to clear corneal incisions during cataract surgery. J. Cataract Refract. Surg. 39, 1668u20131674 (2013)._ _ Chandru, A. et al. Human cadaveric donor cornea derived extra cellular matrix microparticles for minimally invasive healing/regeneration of corneal wounds. Biomolecules 2, 532 (2021)._ _ Davidenko, N. et al. Control of crosslinking for tailoring collagen-based scaffolds stability and mechanics. Acta Biomater. 25, 131u2013142 (2015)._ _ _ Isaacson, A., Swioklo, S. & Connon, C. J. 3D bioprinting of a corneal stroma equivalent. Exp. Eye Res. 173, 188u2013193 (2018)._ _ _ Lee, A. et al. 3D bioprinting of collagen to rebuild components of the human heart. Science 365, 482u2013487 (2019)._ _ _ _ Na, K. S. et al. Effect of mesenchymal stromal cells encapsulated within polyethylene glycol-collagen hydrogels formed in situ on alkali-burned corneas in an ex vivo organ culture model. Cytotherapy (2021).Cressey, A., Jacobs, D. S., Remington, C. & Carrasquillo, K. G. Improvement of chronic corneal opacity in ocular surface disease with prosthetic replacement of the ocular surface ecosystem (PROSE) treatment. Am. J. Ophthalmol. Case. Rep. 10, 108u2013113 (2018)._ _ Crabb, R. A., Chau, E. P., Evans, M. C., Barocas, V. H. & Hubel, A. Biomechanical and microstructural characteristics of a collagen film-based corneal stroma equivalent. Tissue Eng. 12, 1565u2013 1575 (2006)._ _ Schmidt, C. E. & Baier, J. M. Acellular vascular tissues: natural biomaterials for tissue repair and tissue engineering. Biomaterials 21, 2215u20132231 (2000)._ _ _ Sargeant, T. D., Desai, A. P., Banerjee, S., Agawu, A. & Stopek, J. B. An in situ forming collagen-PEG hydrogel for tissue regeneration. Acta Biomater. 8, 124u2013132 (2012)._ _ _ Fagerholm, P. Wound healing after photorefractive keratectomy. J. Cataract Refract. Surg. 26, 432u2013447 (2000)._ _ _ Ljubimov, A. V. & Saghizadeh, M. Progress in corneal wound healing. Prog. Retin. Eye Res. 49, 17u201345 (2015)._ _ _ Joshi, V. P., Vaishnavi, K. S., Ojha, S. K., Singh, V. & Basu, S. A reliable animal model of corneal stromal opacity: Development and validation using in vivo imaging. Ocul Surf. 18, 681u2013688 (2020)._ _ Chaurasia, S. S., Kaur, H., de Medeiros, F. W., Smith, S. D. & Wilson, S. E. Dynamics of the expression of intermediate filaments vimentin and desmin during myofibroblast differentiation after corneal injury. Exp. Eye Res. 89, 133u2013139 (2009)._ _ _ Giacomin, N. T. et al. Intracorneal Ring Segments Implantation for Corneal Ectasia. J. Refract. Surg. 32, 829u2013839 (2016)._ _ Cursiefen, C. Immune privilege and angiogenic privilege of the cornea. Chem. Immunol. Allergy 92, 50u201357 (2007)._ _ _ Freire Haddad, H., Burke, J. A., Scott, E. A. & Ameer, G. A. Clinical Relevance of Pre-Existing and Treatment-Induced Anti-Poly(Ethylene Glycol) Antibodies. Regen. Eng. Transl. Med, 1u201311 (2021).Lee, H. J., Fernandes-Cunha, G. M., Na, K. S., Hull, S. M. & Myung, D. Bio-Orthogonally Crosslinked, In Situ Forming Corneal Stromal Tissue Substitute. Adv. Healthc. Mater. 7, e1800560 (2018).Percie du Sert, N. et al. The ARRIVE guidelines 20: Updated guidelines for reporting animal research. PLOS Biol. 18, 3000410 (2020)._ _ Ma, D. et al. Repeatability, reproducibility and agreement of intraocular pressure measurement in rabbits by the TonoVet and Tono-Pen. Sci. Rep. 6, 35187 (2016)._ _ _ _ \"- - \"This work was supported by a grant from the National Research Foundation of Korea (2019R1F1A1061421), a grant from the Catholic University Yeouido St. Mary s Hospital in the program year of 2020 and a grant of translational R&D project through institute for Bio-Medical convergence, Incheon St. Marys Hospital, The Catholic University of Korea.\"- - Authors and Affiliations - \"Department of Pediatrics, Incheon St. Marys Hospital, College of Medicine, The Catholic University of Korea, Incheon, 21431, Republic of KoreaYoon Hong ChunDepartment of Ophthalmology, Yeouido St. Marys Hospital, College of Medicine, The Catholic University of Korea, 10, 63-ro, Yeongdeungpo-gu, Seoul, 07345, Republic of KoreaSun-Kyoung Park,_Eun Jeong Kim_&_Kyung-Sun NaChemical and Biological Engineering, Gachon University, Seongnamu2010si, Gyeonggiu2010 do, 13120, Republic of KoreaHyun Jong LeeDepartment of Chemical and Biomolecular Engineering, Yonsei University, Seodaemun-gu, Seoul, 03722, Republic of KoreaHyewon Kim_&_Won-Gun KohOphthalmology, Byers Eye Institute at Stanford University School of Medicine, Palo Alto, CA, 94303, USAGabriella Fernandes Cunha_&_ David Myung\" - \"AuthorsYoon Hong ChunYou can also search for this author in _Sun-Kyoung ParkYou can also search for this author in _Eun Jeong KimYou can also search for this author in _Hyun Jong LeeYou can also search for this author in _Hyewon KimYou can also search for this author in _Won-Gun KohYou can also search for this author in _Gabriella Fernandes CunhaYou can also search for this author in _David MyungYou can also search for this author in _Kyung-Sun NaYou can also search for this author in _\" - Contributions - \\'Study conception and design: H.L., D.M., and K.N.; analysis and interpretation of the data: Y.C., H.L., W.K., S.P., D.M., and K.N.; acquisition of data: Y.C., E.K., H.L., and H.K.; revising or reviewing the manuscript: Y.C., G.C., H.L., H.K., W.K., D.M., and K.N. All authors were involved in drafting the article or revising it critically for important intellectual content and approved the final version.\\' - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License(). The mice in the images in Figs._a, were obtained from Biorender (). The image in Fig._ was assembled in Powerpoint, Microsoft 365.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - Source data are provided with this paper. All the data supporting our findings are available in the article, the supplementary files, or the Source data file. - \\' \\'- - \"Hoxhaj, G. & Manning, B. D. The PI3K-AKT network at the interface of oncogenic signalling and cancer metabolism. Nat. Rev. Cancer 20, 74u201388 (2020)._ _ _ Tanner, L. B. et al. Four key steps control glycolytic flux in mammalian cells. Cell Syst. 7, 49u201362.e8 (2018)._ _ _ Dai, Z., Shestov, A. A., Lai, L. & Locasale, J. W. A flux balance of glucose metabolism clarifies the requirements of the warburg effect. Biophys. J. 111, 1088u20131100 (2016)._ _ _ _ Vander Heiden, M. G. & DeBerardinis, R. J. Understanding the intersections between metabolism and cancer biology. Cell 168, 657u2013669 (2017)._ _ Locasale, J. W. et al. Phosphoglycerate dehydrogenase diverts glycolytic flux and contributes to oncogenesis. Nat. Genet. 43, 869u2013874 (2011)._ _ _ Villa, E., Ali, E. S., Sahu, U. & Ben-Sahra, I. Cancer cells tune the signaling pathways to empower de novo synthesis of nucleotides. Cancers (Basel) 11, (2019).Villa, E. et al. mTORC1 stimulates cell growth through SAM synthesis and m6A mRNA-dependent control of protein synthesis. Mol. Cell 81, 2076u20132093.e9 (2021)._ _ _ Ben-Sahra, I., Howell, J. J., Asara, J. M. & Manning, B. D. Stimulation of de novo pyrimidine synthesis by growth signaling through mTOR and S6K1. Science 339, 1323u20131328 (2013)._ _ _ _ Ben-Sahra, I., Hoxhaj, G., Ricoult, S. J. H., Asara, J. M. & Manning, B. D. mTORC1 induces purine synthesis through control of the mitochondrial tetrahydrofolate cycle. Science 351, 728u2013733 (2016)._ _ _ _ Hoxhaj, G. et al. The mTORC1 signaling network senses changes in cellular purine nucleotide levels. Cell Rep. 21, 1331u20131346 (2017)._ _ _ Emmanuel, N. et al. Purine nucleotide availability regulates mTORC1 activity through the Rheb GTPase. Cell Rep. 19, 2665u20132680 (2017)._ _ _ Valvezan, A. J. et al. mTORC1 couples nucleotide synthesis to nucleotide demand resulting in a targetable metabolic vulnerability. Cancer Cell 32, 624u2013638.e5 (2017)._ _ _ Liberti, M. V. et al. A predictive model for selective targeting of the warburg effect through GAPDH inhibition with a natural product. Cell Metab. 26, 648u2013659.e8 (2017)._ _ _ Chaneton, B. et al. Serine is a natural ligand and allosteric activator of pyruvate kinase M2. Nature 491, 458u2013 462 (2012)._ _ _ _ Dayton, T. L., Jacks, T. & Vander Heiden, M. G. PKM2, cancer metabolism, and the road ahead. EMBO Rep. 17, 1721u20131730 (2016)._ _ _ Nandi, S. & Dey, M. Biochemical and structural insights into how amino acids regulate pyruvate kinase muscle isoform 2. J. Biol. Chem. 295, 5390u20135403 (2020)._ _ _ Anastasiou, D. et al. Pyruvate kinase M2 activators promote tetramer formation and suppress tumorigenesis. Nat. Chem. Biol. 8, 839u2013 847 (2012)._ _ _ Kung, C. et al. Small molecule activation of PKM2 in cancer cells induces serine auxotrophy. Chem. Biol. 19, 1187u20131198 (2012)._ _ _ Meiser, J. et al. Increased formate overflow is a hallmark of oxidative cancer. Nat. Commun. 9, 1368 (2018)._ _ _ Ducker, G. S. et al. Human SHMT inhibitors reveal defective glycine import as a targetable metabolic vulnerability of diffuse large B-cell lymphoma. Proc. Natl Acad. Sci. USA 114, 11404u2013 11409 (2017)._ _ _ Pollard, T. D. & Cooper, J. A. Actin, a central player in cell shape and movement. Science 326, 1208u2013 1212 (2009)._ _ _ _ Fares, J., Fares, M. Y., Khachfe, H. H., Salhab, H. A. & Fares, Y. Molecular principles of metastasis: a hallmark of cancer revisited. Signal Transduct. Target. Ther. 5, 28 (2020)._ _ Nieto, M. A., Huang, R. Y.-J., Jackson, R. A. & Thiery, J. P. EMT: 2016. Cell 166, 21u201345 (2016)._ _ _ Cook, D. P. & Vanderhyden, B. C. Context specificity of the EMT transcriptional response. Nat. Commun. 11, 2142 (2020)._ _ _ _ Hanahan, D. Hallmarks of cancer: new dimensions. Cancer Discov. 12, 31u2013 46 (2022)._ _ _ Bergers, G. & Fendt, S.-M. The metabolism of cancer cells during metastasis. Nat. Rev. Cancer 21, 162u2013180 (2021)._ _ _ Ye, J. et al. Pyruvate kinase M2 promotes de novo serine synthesis to sustain mTORC1 activity and cell proliferation. Proc. Natl Acad. Sci. USA 109, 6904u20136909 (2012)._ _ _ _ DeBerardinis, R. J. & Chandel, N. S. We need to talk about the Warburg effect. Nat. Metab. 2, 127u2013129 (2020)._ _ Luengo, A., Gui, D. Y. & Vander Heiden, M. G. Targeting metabolism for cancer therapy. Cell Chem. Biol. 24, 1161u20131180 (2017).Friedl, P. & Alexander, S. Cancer invasion and the microenvironment: plasticity and reciprocity. Cell 147, 992u20131009 (2011)._ _ _ Kiweler, N. et al. Mitochondria preserve an autarkic one-carbon cycle to confer growth-independent cancer cell migration and metastasis. Nat. Commun. (2022).Ngo, B. et al. Limited environmental serine and glycine confer brain metastasis sensitivity to PHGDH inhibition. Cancer Discov. 10, 1352u20131373 (2020)._ _ _ Rinaldi, G. et al. In vivo evidence for serine biosynthesis-defined sensitivity of lung metastasis, but not of primary breast tumors, to mTORC1 inhibition. Mol. Cell 81, 386u2013397.e7 (2021)._ _ _ Israelsen, W. J. et al. PKM2 isoform-specific deletion reveals a differential requirement for pyruvate kinase in tumor cells. Cell 155, 397u2013409 (2013)._ _ _ Ubellacker, J. M. et al. Lymph protects metastasizing melanoma cells from ferroptosis. Nature 585, 113u2013118 (2020)._ _ _ _ Tasdogan, A. et al. Metabolic heterogeneity confers differences in melanoma metastatic potential. Nature 577, 115u2013120 (2020)._ _ _ _ Hoxhaj, G. et al. Direct stimulation of NADP+ synthesis through Akt-mediated phosphorylation of NAD kinase. Science 363, 1088u20131092 (2019)._ _ _ _ Hoxhaj, G. et al. The E3 ubiquitin ligase ZNRF2 is a substrate of mTORC1 and regulates its activation by amino acids. eLife 5, (2016).Tran, D. H. et al. Mitochondrial NADP+ is essential for proline biosynthesis during cell growth. Nat. Metab. 3, 571u2013585 (2021)._ _ _ Ali, E. S. et al. ERK2 phosphorylates PFAS to mediate posttranslational control of de novo purine synthesis. Mol. Cell 78, 1178u20131191.e6 (2020)._ _ _ Pacold, M. E. et al. A PHGDH inhibitor reveals coordination of serine synthesis and one-carbon unit fate. Nat. Chem. Biol. 12, 452u2013458 (2016)._ _ _ Kesavan, R. et al. Gentiana lutea exerts anti-atherosclerotic effects by preventing endothelial inflammation and smooth muscle cell migration. Nutr. Metab. Cardiovasc. Dis. 26, 293u2013301 (2016)._ _ _ \"- - \\'We thank Prof. Matthew Vander Heiden (Massachusetts Institute of Technology, Cambridge, MA) for the PKM2 wild-type and knockout MEFs and materials, Marc Morgan and Ali Shilatifard for providing us with the guide RNA targeting human P53, Min Yuan for assistance with mass spectrometry, and Vijayashree Ramesh for assistance with tail-vein injections. This research was supported by grants from the NIH: R01GM143236 (G.H.), R35-CA197459 (B.D.M.), P01-CA120964 (B.D.M. and J.A.), R00-CA194192, R01-GM135587, R01GM143334 (I.B.-S.); a Welch foundation award (I-2067-20210327(G.H.)); a TS Alliance Research Grants Program award (885252 (G.H.)), a LAM Foundation Career Development and Established Investigator Awards (LAM0127C01-18, LAM0151E01-22) (I.B.-S.); a TSC Alliance postdoctoral fellowship (SP0057487) (E.V.); and a Rothberg Courage Award from the TSC Alliance (B.D.M.). A.T. was supported by an Emmy Noether Award from the German Research Foundation (DFG, 467788900) and the Ministry of Culture and Science of the State of North Rhine-Westphalia (NRW-Nachwuchsgruppenprogramm). G.H. is recipient of a CPRIT Scholar (CPRIT; RR190087) and a V Scholar (V2021-019) awards.\\'- - Author notes - \\'These authors contributed equally: Mona Hoseini Soflaee, Rushendhiran Kesavan, Umakant Sahu.\\' - Authors and Affiliations - \"Childrens Medical Center Research Institute, University of Texas Southwestern Medical Center, 5323 Harry Hines Boulevard, Dallas, TX, 75390, USAMona Hoseini Soflaee,_Rushendhiran Kesavan,_Feng Cai,_Diem H. Tran,_Hieu S. Vu,_Halie Rion,_Sherwin Kelekar,_Misty Martin,_Thomas P. Mathews_&_Gerta HoxhajDepartment of Biochemistry and Molecular Genetics, Feinberg School of Medicine, Northwestern University, Chicago, IL, 60611, USAUmakant Sahu,_Elodie Villa,_Zied Djabari,_ Eunus S. Ali,_Brendan P. OHara_&_Issam Ben-SahraRobert H. Lurie Comprehensive Cancer Center, Northwestern University, Chicago, IL, 60611, USAUmakant Sahu,_ Elodie Villa,_Zied Djabari,_Eunus S. Ali,_Brendan P. OHara_&_Issam Ben-SahraDepartment of Dermatology, University Hospital Essen & German Cancer Consortium, Partner Site, Essen, GermanyAlpaslan TasdoganDepartment of Molecular Metabolism, Harvard T. H. Chan School of Public Health, Boston, MA, 02115, USAJames Hughes Hallett_&_Brendan D. ManningMetabolomics Core Facility, Robert H. Lurie Comprehensive Cancer Center, Northwestern University, Chicago, IL, 60611, USAPeng GaoMass Spectrometry Core, Beth Israel Deaconess Medical Center, Department of Medicine, Harvard Medical School, Boston, MA, 02115, USAJohn M. Asara\" - \"AuthorsMona Hoseini SoflaeeYou can also search for this author in _Rushendhiran KesavanYou can also search for this author in _Umakant SahuYou can also search for this author in _Alpaslan TasdoganYou can also search for this author in _Elodie VillaYou can also search for this author in _Zied DjabariYou can also search for this author in _Feng CaiYou can also search for this author in _Diem H. TranYou can also search for this author in _Hieu S. VuYou can also search for this author in _Eunus S. AliYou can also search for this author in _Halie RionYou can also search for this author in _Brendan P. OHaraYou can also search for this author in _Sherwin KelekarYou can also search for this author in _James Hughes HallettYou can also search for this author in _Misty MartinYou can also search for this author in _Thomas P. MathewsYou can also search for this author in _Peng GaoYou can also search for this author in _John M. AsaraYou can also search for this author in _Brendan D. ManningYou can also search for this author in _Issam Ben-SahraYou can also search for this author in _Gerta HoxhajYou can also search for this author in _\" - Contributions - B.D.M., I.B.-S., and G.H. conceptualized, directed the study, and wrote the manuscript. M.H.S., R.K., and U.S. performed and analyzed the data. A.T., E.V., Z.D., D.H.T., E.S.A., H.R., B.P.O., S.K., and J.H.H. assisted with molecular, cell biology, and animal experiments, F.C., H.S.V., M.M., T.P.M., P.G., and J.M.A. performed LC-MS/MS analyses and assisted with mass-spectrometry-related methods. I.B.-S. and G.H. prepared the manuscript. - Corresponding authors - \"Correspondence to , or .\"- - \\' \\' - Competing interests - \\' \\' - B.D.M. is a shareholder and scientific advisory board member of Navitor Pharmaceuticals. The remaining authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \"Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work._ are available.\" - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License).\" - Pre-culture time affects extent of vascular morphogenesis in vitro - \"We next assessed if pre-culturing microbeads prior to embedding them caused any differences in the overall extent of the vessel-like networks formed in vitro. Microbeads containing HUVEC and NHLF were embedded in fibrin hydrogels after 1 (Fig._a), 3 (Fig._b), 5 (Fig._c), and 7 (Fig._d) days of suspension pre-culture, and subsequently cultured for an additional 7_days within the hydrogels. Each tissue construct was imaged in its entirety, and the fractional area occupied by endothelial tube-like structures was quantified. Extended periods of pre-culture beyond 3_days prior to embedding resulted in reduced coverage of the tubular structures within the fibrin hydrogel, indicative of a less extensive vessel-like network (Fig._e). Microbeads pre-cultured for 3_days produced significantly more extensive vascular networks compared to microbeads pre-cultured for 7_ days (pu2009<u20090.05), and we therefore selected this pre-culture condition for subsequent in vivo studies.\" - \"Figure 2Pre-culture time affects vascular distribution in vitro. Microbeads were pre-cultured in suspension for (a) 1_day, (b) 3_days, (c) 5_days, or (d) 7_days, and subsequently embedded in fibrin hydrogels for an additional 7_ days. Endothelial sprouts (red) were stained with UEA-I. Images on the right for each pair show regions of the hydrogel that were magnified and adjusted with a Kirsch filter to facilitate edge detection and quantification. (e) Quantification of the fractional area of each fibrin construct occupied by tubular structures as a metric of distribution showed that 3_days of pre-culture time led to the most extensive vascular networks. The symbol (*) on the graph indicates values were statistically different (pu2009u2264u20090.05). Error bars indicateu2009 xB1u2009SD.\" - Cellular microbeads direct vascular morphogenesis in vivo - \"Pre-cultured microbeads were then evaluated for their ability to form functional microvasculature in vivo. Microbeads containing HUVEC and NHLF pre-cultured in suspension for 3_days (D3 microbeads) were injected in an acellular fibrin precursor solution into subcutaneous pockets on the dorsal surface of SCID mice (Fig._a). Their ability to form functional microvasculature was compared to D0 microbeads (encapsulating HUVEC and NHLF without pre-culture), acellular microbeads, or cellular hydrogels (HUVEC and NHLF delivered in a fibrin precursor solution, which gels in situ). All implants containing human cells showed evidence of vascular morphogenesis after implantation for 3 and 7_days (Fig._b, Supplementary Fig. ). HUVEC were identified in explanted tissue constructs via immunohistochemical (IHC) staining of human CD31 (Fig._c, brown). Host erythrocytes within the lumens of these hCD31+ neovessels confirmed the formation of functional anastomoses with host vasculature. Murine vessels (white arrows, negative for hCD31) were sometimes visible in sections counter-stained with hematoxylin. We also occasionally observed structures that simultaneously consisted of both human endothelial cells (in brown) and murine endothelial cells (not stained) in the same structure, suggestive of chimeric vessels (Fig._c, right panel, dashed rectangle). Representative images of hCD31+ sections of the four experimental groups at both days 3 and 7 and two different magnifications are shown (Fig._d). Quantification of the vessel and perfused vessel densities was achieved by counting the number of hCD31+ and erythrocyte-perfused hCD31+ structures with lumens, respectively. No significant differences in the average numbers of total hCD31+ vessels or perfused hCD31+ vessels per mm were observed across any of the implants containing cells at day 3 (Fig._e, f) and day 7 (Fig._g, h).\" - \"Figure 3Cell-laden fibrin microbeads catalyze the formation of functional microvasculature in vivo. (a) Implants were injected into the subcutaneous space on the dorsal surface of SCID mice. (b) Implants were retrieved, fixed, processed, embedded, and then stained with hematoxylin and eosin. Representative images of H&E-stained sections show vessel formation and cell infiltration in D0 microbeads (first column), D3 microbeads (second column), acellular microbeads (third column), and cellular hydrogels (fourth column) for 3_days (top row) and 7_days (bottom row). Dashed lines highlight the clusters of microbeads located within the implant. Arrows indicate representative vessels, black asterisks indicate representative regions where host erythrocytes are clearly present, and white asterisks indicate representative individual microbeads. (c) Implants were also IHC-stained for hCD31 (dark brown) to confirm the human origin of the neovessels. Implants containing D3 microbeads evaluated in vivo for 3 (left image) and 7 (middle and right image) days showed evidence of inosculation with host vessels. The white arrow in the day 7 micrograph (middle image) identifies a perfused mouse vessel (hCD31-), while white asterisks highlight hCD31+ vessels with red blood cells. The dashed black rectangle suggests a chimeric vessel formed by both mouse and human cells (right image). The magnified inset demonstrates the presence of host erythrocytes within the vessel. (d) Vessel structures containing human endothelial cells (hCD31+) were identified in all implants except for those containing acellular microbeads. Shown are representative 20xD7 and 40xD7 images of implants stained for hCD31 after 3 and 7_days in vivo. Quantification of both total and perfused human EC-derived vessel density for each condition after (e,f) 3_days and (g,h) 7_days was performed. No significant differences were observed in vessel and perfused vessel density between any of the experimental groups containing human endothelial cells after 3 or 7_days in vivo. Individual data points on graphs represent a single implant quantified per condition. Error bars indicateu2009 xB1u2009SD. Portions of this figure were created using images modified from Servier Medical Art (Servier, , licensed under a Creative Commons Attribution 3.0 Unported License).\" - Cellular microbeads form extensive functional vascular networks in vivo - \"Implants containing cellular microbeads appeared larger in volume upon extraction from the mice. To evaluate this observation, implants were histologically sectioned through their entire volume, each section (6_xB5m) was stained for H&E, and then the section with the maximum area (approximating the center of the implant region) was identified by quantifying stitched images of the entire implant region (Fig._a). Using this approach, the implant area (in mm) and the total number of vessels in the maximum implant area were quantified in tissues explanted after 3 and 7_days in vivo (Fig._bu2013i). While trends in the data suggest implants containing cellular microbeads had larger average cross-sectional areas than both acellular microbeads and cellular hydrogels on day 3 and 7, these differences were not statistically significant (Fig._b, c). With the exception of D0 microbeads, the average implant cross-sectional area of all conditions decreased from day 3 to day 7 in vivo.\" - \"Figure 4Cellular microbeads form extensive microvasculature in vivo. (a) Representative images of the cross-sectional areas of hCD31-stained tissues are shown for all 3 of the cell-containing experimental groups. D0 microbeads (left column), D3 microbeads (middle column), and cellular hydrogels (right column) were implanted for 3 (top row) and 7_days (bottom row) prior to their excision (black scale baru2009=u2009500_xB5m). Implant sizes were quantified by measuring the cross-sectional area of the implant region using H&E staining after (b) 3 and (c) 7_days in vivo. No significant differences in implant a rea were found between any of the experimental groups after 3 or 7_days. The total number of hCD31+ (d,g) vessels and (e,h) perfused vessel in the entire implant region was quantified after 3 and 7_days in vivo. No significant differences were observed between any of the cell-laden experimental groups after either time point. Percent of perfused hCD31+ vessels in the implant region after (f) 3 and (i) 7_days. After 3_days in vivo, D3 microbeads had a significantly greater percent of perfused vessel compared to D0 microbeads. After an additional 4_ days in vivo, no significant differences were observed between any of the cellular conditions. Asterisks indicate statistically significant differences (pu2009 u2264u20090.05) between groups. Individual data points on graphs represent a single implant quantified per condition. Error bars indicateu2009xB1u2009 SD.\" - \"We then multiplied these implant areas by the average vessel density to calculate the total numbers of vessels within the implant groups, and found all cellular implants contained comparable total numbers of hCD31+ vessels after 3_days in vivo (Fig._d, e). D0 microbeads still showed minimal evidence of vessel perfusion on day 3 even when considering total vessel coverage throughout the entire implant (Fig._e, f). For all conditions, the total numbers of hCD31+ vessels and hCD31+ perfused vessels increased from day 3 to day 7 (Fig._g, h); however, this increase was notably smaller in cellular hydrogels, which only showed a 1.2-fold increase in total number of vessels from day 3 to day 7 in vivo compared to a 3.31-fold and 2.44-fold increase in D0 and D3 microbeads, respectively. Implants containing D0 and D3 microbeads also exhibited greater increases in the total numbers of perfused vessels (233.9-fold and 5.1-fold, respectively) from day 3 to day 7 than cellular hydrogels (only 1.8-fold). Furthermore, of the total number of hCD31+ vessels formed by day 7 (Fig._i), 91.8% of vessels in D3 microbead implants were perfused, compared to only 54.8% and 64% in D0 microbeads and cellular hydrogels, respectively.\" - \"Cellular implants attain u03B1-SMA expression in vivo\" - \"IHC staining of smooth muscle alpha-actin (u03B1-SMA), a pericyte marker, was also performed to determine the extent of vessel maturation across the different experimental groups. At the day 3 time point, u03B1-SMA expression (brown stain with black arrows) was sparse in all four experimental groups, even when images were magnified to better visualize stained structures (Fig._a). Most positive u03B1-SMA staining observed within the implant region at this early time point was diffuse and not necessarily localized subjacent to vessel structures. Positive u03B1-SMA staining around vasculature was readily observed in the host tissue localized outside of the implant region, displayed at the top left corner of the cellular hydrogel image in Fig._a (magnified in Fig._b). After an additional 4_days in vivo, cellular conditions all displayed stronger u03B1-SMA staining compared to the day 3 time point (Fig._c). Implants containing acellular microbeads exhibited minimal u03B1-SMA+ staining within the implant region (Fig._c). Quantification of vessels stained positive for u03B1-SMA showed the extent of vessel maturation differed significantly only between cellular conditions and acellular microbeads (Fig._du2013f). Specifically, D3 microbead and cellular hydrogel groups both had a significantly higher total numbers of u03B1-SMA+ vessels than acellular microbeads (pu2009=u20090.04 for both conditions) (Fig._ d). Quantification of the total number of perfused u03B1-SMA+ vessels revealed the D3 microbead group had the highest numbers of these mature vessels, although the only significant difference was observed compared to the acellular microbead group (pu2009=u20090.05) (Fig._e). D0 and D3 microbead conditions also had a significantly higher percent of u03B1-SMA+ vessels perfused compared to acellular microbeads after 7_days in vivo (Fig._f).\" - \"Figure 5Implants containing cell-laden microbeads contained u03B1-SMA supported vessels. (a) D0 microbeads (first column), D3 microbeads (second column), cellular hydrogels (third column), and acellular microbeads (fourth column) that were injected and kept in vivo for 3_days displayed minimal evidence of u03B1-SMA+ staining. Images acquired via 10xD7 (top row) and 40xD7 (bottom row) objectives highlight u03B1-SMA staining in brown (black arrows). (b) Vessels within the surrounding host tissue were positive for u03B1-SMA as shown by 40xD7 image from animal tissue near 3-day cellular hydrogel implant. (c) After 7_days, expression of u03B1-SMA positive structures was higher in all conditions. White scale baru2009=u2009100_xB5m, and black scale baru2009=u200925_xB5m. Quantification of total number of (d) u03B1-SMA+ and (e) perfused u03B1-SMA+ vessels in the implant region after 7_days in vivo. D3 microbeads had significantly higher u03B1-SMA+ vessel and perfused vessel densities, and total number of perfused u03B1-SMA+ vessels than acellular microbeads. D3 microbeads and cellular hydrogels both had a higher total number of u03B1-SMA+ vessels than acellular microbeads. (f) Percent of perfused u03B1-SMA+ vessels in the implant region after 7_days. Both D0 and D3 microbeads had a significantly greater percent of perfused vessel compared to acellular microbeads. Asterisks indicate statistically significant differences (pu2009u2264u20090.05) between groups. Individual data points on graphs represent a single implant quantified per condition. Error bars indicateu2009xB1u2009SD.\" - Volume preservation of cellular microbeads may influence maximum implant area and vascular network distribution - \"The observed differences in implant morphologies suggested the cellular microbead formulations may yield larger implants in part because of their ability to better preserve implant volume over time. While the cellular hydrogel group exhibited higher hCD31+ vessel densities, the total numbers of hCD31+ vessels across the implants trended highest in the cellular microbead groups after 7_days in vivo, suggesting microbead formulations may be particularly useful in creating a more distributed vascular network throughout the entirety of an implant. To assess this directly, implant volumes were measured in model constructs in vitro the day after fabrication via 3D high-resolution ultrasound imaging (Fig._a). Fibrin-based constructs containing D3 microbeads did not significantly change volume within 24_h. By contrast, the volumes of constructs containing D0 microbeads or uniformly distributed cells (u201Ccellular hydrogelsu201D) significantly compacted, attaining volumes that were 17% and 47% less, respectively, than those containing D3 microbeads after only 24_h of incubation in vitro. All measured volumes were statistically different from one another (pu2009u2264u20090.05, Fig._ b). Further, we investigated if differences observed between microbead conditions could be attributed to cell invasion from microbeads into the surround fibrin hydrogel, leading to matrix remodeling and implant compaction. Phase contrast images qualitatively showed fewer cells migrated out of D3 microbeads than D0 microbeads in the surrounding fibrin matrix, especially at early time points (Fig._c). This observation is clearer when microbeads are left in the fibrin hydrogels for 3_days rather than 7_days since there is less time for cells to remodel the matrix. Collectively, these data reveal that constructs containing cellular microbeads exhibited less compaction, potentially due to delayed cell invasion and matrix remodeling, relative to cellular hydrogels, which may influence maximum implant area and vascular network distribution observed in vivo.\" - \"Figure 6Constructs with D3 microbeads do not compact after 1_day of in vitro culture. (a) 3D ultrasound images of model fibrin implants containing D0 microbeads (top row), D3 microbeads (middle row), or uniformly suspended cells (cellular hydrogels, bottom row). Black lines and arrows in the images on the right-hand side show the bottom of the implant; white vertical lines are 3_mm scale bars. (b) Relative implant volumes were quantified and normalized to the model implants containing D3 microbeads, whose volumes were constant after 24_h of in vitro culture. Statistically significant differences (pu2009u2264u20090.05) are indicated by matched symbols. Constructs containing D3 microbeads were of significantly larger volume than those containing D0 microbeads or the cellular hydrogels. (c) Cells deployed more slowly from D3 microbeads when embedded in fibrin hydrogels. Bright-field images of D0 (top row) and D3 (bottom row) microbeads embedded in fibrin hydrogels for 3 (left column) and 7_days (right column). Scale baru2009 =u2009500_xB5m. Error bars indicateu2009xB1u2009SD.\"- - \"Creating functional microvasculature remains a major challenge to engineering functional tissue replacements and restoring function in many ischemic pathologies. In this proof-of-concept study, we examined an approach to engineer microvasculature in vivo based on sub-millimeter vascularized tissue modules (microbeads) that can be injected via syringe in a minimally invasive manner. Similar approaches based on cell spheroids, microgels containing cells, and modular microtissues with an exterior coating of endothelial cells have all shown microvascular potential in vitro and in vivo. Inspired by these and other previous studies, our approach here preserves many of the attractive features of macroscale pre-vascularized tissue constructs, including the flexibility to be fabricated with a range of materials and cell types and the ability to support the complex morphogenetic programs necessary to recreate microvasculature that can inosculate with host vasculature following delivery in situ. While other studies have investigated the use of scaffold-free multicellular aggregates for vascularization applications, the use of ECM proteins for cell encapsulation provides an immediate 3D matrix conducive to remodeling during vascular assembly, which may be beneficial for more rapidly producing robust vascular networks throughout microbeads prior to implantation in vivo. Fibrin was chosen as the material for our microbeads for applications in tissue vascularization because it improves cell survival during injection, promotes wound healing and endothelial morphogenesis, and is cleared by the FDA for use as a surgical sealant in humans. Additionally, fibrins microstructure and polymerization rate can be readily modified.\" - \"Consistent with our prior study, we first demonstrated here that HUVEC co-encapsulated with NHLF in fibrin microbeads were capable of sprouting into a surrounding fibrin matrix and forming interconnected networks of vessel-like structures. We then assessed the value added by pre-culture prior to either embedding the microbeads in a model tissue in vitro or injecting them in vivo. Cells encapsulated in microbeads held in static suspension culture without embedding in an additional matrix for up to 7_days were able to form primitive vessel-like structures within each bead. Once embedded, endothelial sprouts from adjacent pre-cultured microbeads appeared to form connections with each other within the surrounding fibrin matrix. The distribution of the microvasculature nucleated by the microbeads embedded in larger model tissues (quantified by the area fraction occupied by endothelial tubules) was significantly improved when microbeads were pre-cultured for 3_days prior to embedding. Longer pre-culture times led to less extensive and more heterogeneous networks, perhaps due to microbead aggregation during pre-culture, which may have resulted in less uniform distribution of microbeads in the matrix. The observed formation of vessel-like structures within and between microbeads is consistent with other studies using endothelial and stromal supportive cells encapsulated in microgels of varying matrices. Other studies have found longer pre-culture times (up to 14_days) beneficial, especially when compared to non-pre-cultured controls; however, optimal pre-culture time is influenced by cell type, cell density, and material properties of the microgels.\" - \"The in vivo vascularization potential of these microbeads pre-cultured for 3_ days (D3 microbeads) was then compared to D0 microbeads (containing HUVEC and NHLF but no period of pre-culture), acellular microbeads, and cellular hydrogels (HUVEC and NHLF delivered in a fibrin precursor solution, which gels in situ). All four experimental groups were delivered as injectable formulations, with microbeads being delivered within an acellular fibrin precursor solution, into subcutaneous pockets on the dorsal surface of SCID mice. Pre-culturing microbeads for 3_days prior to delivery did not significantly accelerate the formation of anastomoses with host vessels in this particular animal model, as all three groups containing cells showed evidence of hCD31+ vessel formation by day 3. Further, at this time point D3 microbeads and cellular hydrogels also showed evidence of functional perfusion (i.e., the presence of erythrocytes). D0 microbeads showed little evidence of perfusion by day 3, perhaps indicating that the fabrication process of the microbeads may result in the encapsulated cells needing additional time to equilibrate to the new matrix prior to vessel morphogenesis, causing a delay in their ability to anastomose with host vasculature. After 7_days in vivo, all cellular implants contained perfused hCD31+ vessels. All three cellular conditions showed an increase in total number of vessels and perfused vessels from day 3 to day 7 in vivo, however, increases observed in cellular hydrogels were much lower than those of cellular microbeads. Though cellular hydrogels had greater average hCD31+ vessel densities, the total number of hCD31+ vessels throughout the entire implant region trended higher in D0 and D3 microbead conditions after 7_days, likely due to the larger sizes of cellular microbead implants. The total number of perfused hCD31+ vessels also trended higher for the D3 microbead group than D0 microbeads and cellular hydrogels. Interestingly, in a similar study utilizing pre-cultured multicellular alginate microgels implanted in a fibrin plug, non-pre-cultured (D1) microgels failed to assemble into vascular networks in the subcutaneous space after 7_days. Pre-cultured (D14) microgels showed some evidence of vessel perfusion after 7_days, but required an additional 7_days in vivo to show significant evidence of vessel perfusion throughout the implant.\" - \"Further, tissue sections were also stained for u03B1-SMA to characterize pericyte coverage and vessel maturity within the implants. Mature vessels were only found in the surrounding mouse tissue on day 3, but by day 7 all cellular implants exhibited higher levels of u03B1-SMA expression. Acellular microbeads had minimal u03B1-SMA expression throughout the implant even at the day 7 time point indicating minimal infiltration of host vasculature. All cellular conditions had comparable u03B1-SMA+ vessel densities and total number of u03B1-SMA+ vessels after 7_ days in vivo, but only D3 microbeads had a significantly higher average u03B1 -SMA+ vessel and perfused vessel density compared to acellular microbeads. D3 microbeads were also the only group to have a significantly higher total number of perfused u03B1-SMA+ vessels compared to acellular microbeads. This could be a result of pro-angiogenic growth factor sequestration to the fibrin matrix of microbeads during pre-culture. These growth factors could aid in the recruitment of mature host vessels into the implant region upon delivery of pre-cultured microbeads in vivo, leading to greater expression of u03B1-SMA throughout the implant. Further experiments are needed to more definitively determine if the u03B1-SMA+ mural cells observed around the vessels were due to host vasculature infiltration or differentiation of the NHLF into bona fide pericytes, although previous evidence from our group suggests NHLF only express a subset of pericyte markers, at least in our in vitro models.\" - \"The larger size of implants containing cellular microbeads translated to trends of higher numbers of total vessels across the entire implant volume, likely resulting in a more extensive vascular distribution. As observed differences in implant area were not statistically significantly, possibly due to the inherent variability in sample collection and histological processing, we used ultrasound to quantify the volumes of model implants cultured in vitro. A phenomenon similar to the compaction observed in vivo was also observed in these in vitro constructs. Constructs containing D3 microbeads did not compact, while those with D0 microbeads compacted approximately 20%. By contrast, the cellular hydrogel constructs, in which the cells were initially dispersed uniformly, compactedu2009~u2009 50% in just 24_h. Furthermore, constructs containing D3 microbeads contained higher density features (presumably the cell-laden beads) based on the higher attenuation displayed in the ultrasound images relative to the other constructs. Cells encapsulated in the D3 microbeads also took longer to infiltrate the surrounding fibrin hydrogel than the ones residing in the D0 microbeads, perhaps due to increased local ECM density. The volume preservation of constructs containing cellular microbeads, combined with the delayed deployment of cells from these beads, may underscore the utility of cellular microbead formulations to create more distributed microvascular networks.\" - In summary, this study demonstrated that fibrin microbeads containing cellular building blocks of microvasculature readily vascularize a fibrin construct in subcutaneous tissue and support a high degree of vascularity comparable to cellular hydrogels. This finding demonstrates that the fabrication and subsequent pre-culture of cellular microbeads do not have deleterious effects on the ability of these microbeads to form functional vasculature upon implantation in vivo. While other studies have investigated the use of cells spheroids and microgels for vascularization applications, few have utilized ECM protein-based microbeads for the encapsulation of multiple cell types to create discrete units for vascularization. Though differences observed between experimental groups were often not significant, the data suggest that cellular microbeads may have advantageous properties over cells uniformly distributed throughout bulk hydrogels, such as implant volume preservation and enhanced vascular network distribution. Pre-culturing microbeads may accelerate vessel inosculation with host vasculature and maturation characterized by an increase in the total numbers of pericyte-invested perfused microvessels. However, while the subcutaneous model is straightforward and commonly used to evaluate vascularization strategies in vivo, it does not mimic the harsh ischemic conditions typical of many clinical conditions. This proof-of-concept study therefore lays the groundwork for higher-powered animal studies in more physiologically-relevant models of ischemia, where cell survival may be more challenging due to the lack of oxygen and nutrients. Whereas encapsulated cells might not survive long enough to form vessels de novo, primitive microvasculature deployed from injectable pre-cultured microbeads may better withstand harsh ischemic environments and nucleate a distributed vascular network that can rapidly inosculate with host vessels to support parenchymal cells for regenerative applications.- - Cell culture - \"Human umbilical vein endothelial cells (HUVEC) were either purchased from a commercial source (Lonza, Inc., Walkersville, MD) or isolated from umbilical cords from the University of Michigan Mott Childrens Hospital as previously described. [Umbilical cords were obtained by a process considered exempt by the University of Michigans Institutional Review Board (notice of determination dated August 21, 2014) because the tissue is normally discarded, and no identifying information is provided to the researchers who receive the cords.] These two different HUVEC sources were evaluated in vitro to demonstrate the robustness of the system, regardless of endothelial source, as shown in the past. HUVEC isolated from fresh cords were used for in vitro studies and the commercial HUVEC were used for the in vivo studies. This was due to limited availability of fresh cords for HUVEC isolation. Purchasing HUVEC from a commercial source ensured all experimental condition contained HUVEC from a single source to prevent variability in in vivo data due to cell sourcing. Regardless of source, HUVEC were grown in endothelial growth media (EGM-2, Lonza), and used from passages 4u20137. Normal human lung fibroblasts (NHLF) were also purchased (Lonza), cultured in Media 199 (M199, Life Technologies, Grand Island, NY) with 10% fetal bovine serum (FBS), and used from passages 9u201314 for in vitro and in vivo experiments.\" - Fibrin microbead production - \"Fibrin microbeads were made in a water-in-oil emulsification process, as previously described. Prior to starting the emulsification process, 75_mL of 100 cSt polydimethylsiloxane (PDMS) oil (Clearco Products Co. Inc., Bensalem, PA) was placed into a 100_ mL beaker and left on ice. Cells were pelleted in a 1:1 HUVEC:NHLF ratio in a conical tube at the initial cell density of 2u2009xD7u200910_total cells/mL (in vitro studies) or 5u2009xD7u200910_total cells/mL (in vivo studies). Lower cell concentrations were used for the in vitro studies to facilitate quantification of vessel-like structures. Each batch of microbeads (3_mL total volume) was made by mixing 765_xB5L of serum-free endothelial growth media (SFEGM-2), 300_xB5L of fetal bovine serum (FBS, 10% final), 60_xB5L of 50_U/mL thrombin (1_U/mL final), and 1875_xB5L of fibrinogen stock solution (2.5_mg/ml final clottable protein) with the HUVEC-NHLF pellet. The cell and protein mixture was then added to the PDMS bath, which was kept on ice at 0_xB0C. Bovine fibrinogen (FGN, Sigma Aldrich, St. Louis, MO) stock solution was made by mixing protein with SFEGM-2 at 37_xB0C. The cell-fibrin solution was mixed at 600 RPM for 5_min at 0_xB0C to allow for emulsion, and then mixed for 25_min at 37_ xB0C to allow for gelation. The microbeads and PDMS solution were collected and separated with the addition of phosphate buffer saline (PBS) with 100_ppm of L101 surfactant (BASF, Florham Park, NJ) for inversion mixing, followed by 4u2013 5 centrifugation steps (200_g for 5_min/each). After each centrifugation step, PDMS was removed from the fibrin microbeads. The microbeads were then re-suspended with EGM-2 and placed in vented 15_mL conical tubes with filters (CELLTREAT Scientific Products, Shirley, MA) prior to starting any experimental procedures. Medium was changed the day after microbead fabrication then every other day after.\" - Embedding of microbeads in fibrin hydrogels - \"Fibrin microbeads were embedded in fibrin hydrogels immediately, 1, 3, 5, or 7 days after their fabrication process. One tenth of a 3 mL microbead stock was utilized to make 3 fibrin hydrogels, with u223Cu2009100 u03BCL of microbeads per hydrogel. Constructs were made by mixing the microbead pellet thoroughly with 382.5 u03BCL of SFEGM-2, 150 u03BCL of FBS (10% final), 30 u03BCL of 50 U/mL thrombin (1 U/mL final), and 937.5 u03BCL of fibrinogen stock solution (2.5 mg/mL final clottable protein). 500 u03BCL of the microbead-protein mixture was added per well of a standard 24-well culture plate, and left at room temperature for 5 min before being placed in the incubator for 25 min at 37 xB0C. EGM-2 (1 mL/well) was added to each hydrogel after the complete gelation process. Medium was replaced the next day then every other day after for the duration of the culture. In some experiments, microbeads were labeled with FITC-fibrinogen prior to embedding as previously shown.\" - Endothelial cell staining, imaging, and quantification - \"Prior to staining, microbeads alone or embedded in fibrin hydrogels were fixed in zinc-buffered formalin (Z-Fix, Anatech, Battle Creek, MI). After 10_min, the fixative was removed and the samples were washed 2xD7 with PBS. The endothelial cell specific lectin Ulex Europaeus Agglutinin I (UEA-I, Vector Laboratories, Burlingame, CA) was utilized to stain the HUVEC. A staining solution consisting of 1% BSA, 20_xB5g/mL rhodamine-labeled UEA-I, and 10_nM DAPI in PBS was incubated with microbead samples and left for 45_min at room temperature. Samples were then washed with PBS 2u20134xD7 prior to imaging. Hydrogels were taken out of the wells of a 24-well plate, placed on a microscope slide (Superfrost Plus Microscope Slides, Fisher Scientific, Pittsburgh, PA), and then overlaid with a microscope cover glass (Vista Vision Cover Glass, VWR International, LLC, Radnor, PA). The scan slide tool in the Olympus software (IX2-BSW, version 01.07, Olympus, Center Valley, PA) and an optical microscope (Olympus IX81, Olympus) was used to take single plain fluorescent images of endothelial networks formed by fibrin microbeads embedded in fibrin hydrogels. To evaluate microbead characteristics, both after continuous suspension culture and culture in a larger fibrin hydrogel, z-stacks of areas of interest were acquired and flattened to produce maximum intensity projection images. ImageJ (National Institutes of Health, Bethesda, MD) was utilized to merge fluorescent and bright-field images. Prior to quantification, all endothelial images were cropped and processed using the Kirsch filter to detect edges of endothelial sprouts. The processing and imaging settings were kept constant for all conditions. The Angiogenesis Tube Formation module in Metamorph Premier imaging software (, version 7.8.6.0, Molecular Devices, Sunnyvale, CA) was then used to quantify the area of each fibrin hydrogel occupied by endothelial tubules as a metric of microvascular network coverage. At least 12 individual hydrogels per pre-culture condition were quantified to assess differences between experimental groups.\" - Sample preparation for subcutaneous injection - \"Subcutaneous implants consisted of 2u2009xD7u200910_cells (HUVEC:NHLF in a nominal 1:1 ratio) in 500_xB5L of a fibrin precursor solutions for all 3 experimental groups containing cells. To ensure the same numbers of cells were injected into each mouse across different microbead preparations, a method was developed to quantify the number of cells within a given volume of microbeads after fabrication (as opposed to relying only on the theoretical starting cell concentration, as is more typical), and the volume of cell-laden beads injected into each mouse was adjusted in order to ensure consistent delivery of 2u2009 xD7u200910_cells per implant. Briefly, microbead pellets from each batch were re-suspended in 1_mL of EGM-2 and transferred into vented 15_mL conical tubes with filters (CELLTREAT) and either processed immediately or cultured for 1_ day. The total volume of microbeads and media was noted, and then 100_xB5L was transferred to a 96-well plate. An equal volume (100_xB5L) of a solution containing the fibrinolytic enzyme nattokinase (NSK-SD, Japan Bio Science Laboratory Co., Ltd) at a concentration of 50_FU/mL (fibrin degradation units) in D-PBS (calcium- and magnesium-free) containing 1_mM EDTA was then added on top and mixed. The plate was incubated at 37_xB0C for 30_min to allow the microbeads to degrade. Following fibrinolysis, cells were counted using a hemocytometer, and the microbead volume preparations adjusted to yield 2.4u2009xD7u200910_total cells, assuming half HUVEC and half NHLF. Samples for all 4 experimental groups (control microbeads, pre-cultured microbeads, acellular microbeads, or cellular hydrogels) were then centrifuged at 200_g for 1u20135_min to form cell or microbead pellets, supernatant removed, and the pellets resuspended in 600_xB5L of fibrin hydrogel precursor solution comprising 270_xB5L of SFEGM-2, 60_xB5L of FBS (10% final), 12_ xB5L of thrombin (1_U/mL final), and 258_xB5L of fibrinogen stock solution (2.5_mg/mL final clottable protein). Each solution was rapidly mixed, transferred into a 1-mL syringe fitted with a BD PrecisionGlide 20G needle, and 500_xB5 L of the 600_xB5L solution was subsequently injected into the animal to deliver 2u2009xD7u200910_total cells per implant.\" - Subcutaneous injections - \"All animal procedures were compliant with the NIH Guide for Care and Use of Laboratory Animals and approved by the University of Michigans Institutional Animal Care and Use Committee (IACUC). Male C.B-17 SCID mice, 6u20138_weeks of age, (Taconic Labs, Hudson, NY) were acclimated foru2009u2265u200972_h prior to surgery. An anesthetic/analgesic drug mixture of ketamine (80u2013120_ mg/kg), xylazine (5u201310_mg/kg), and buprenorphine (0.05u20130.01_mg/kg) was administered to each animal via intraperitoneal injection. Ophthalmic ointment (Puralube Vet Ointment, Dechra, Overland Park, KS) was added to the eyes of each mouse. Dorsal lumbar flanks were shaved and depilatory agent (Nair, Fisher Scientific, Pittsburg, PA) was applied to remove remaining hair. The injection sites were sterilized by wiping the flanks 3xD7 with alternating ethanol and Betadine (Thermo Fisher Scientific, Fremont, CA). A sterile surgical field was created to place the syringes, needles, and forceps on prior to injection. The samples were prepared, mixed, and injected subcutaneously on the dorsal flanks of the mouse with sterile gloves. The needle was left in the injection site for 30_s to allow for initial gelation of the solution prior to removal. Mice were allowed to recover from the anesthesia before being placed in their normal housing. The following four experimental groups were evaluated: (1) Acellular microbeads, (2) cellular hydrogels, (3) D0 microbeads (no pre-culture), and (4) D3 microbeads (pre-cultured for 3_days post-fabrication). Microbeads in all conditions were delivered in 500_xB5L of acellular fibrin precursor solution which gelled in situ. Bilateral implants were injected per animal, one on each flank in a randomized fashion. A total of 20 mice (nu2009=u20095 implants/condition, 2 implants/mouse) were used for this study. A second dose of buprenorphine (0.05u2013 0.01_mg/kg) was administered to each animal 12_h after subcutaneous injections. Animals were monitored every day post-surgery.\" - Implant retrieval and post-processing - \"After 3 or 7_days, animals were euthanized. Implants were located, removed with scissors and forceps, placed immediately in 20_mL glass scintillation vials with Z-fix, and subsequently fixed for 24_h at 4_xB0C. In some cases, implants could not be reliably located in the subcutaneous space after 3 or 7_days, presumably due to rapid fibrinolysis, and therefore not all formulations resulted in implants that could be retrieved for sample processing. Implants that could not be reliably retrieved were not included in analyses. After fixation, implants were washed with PBS 3u20134xD7 (5_min/wash), submerged in 70% ethanol, and stored at 4_xB0C until further processing. Samples were then placed in pink cassettes (Unisette Tissue Cassettes, Simport, Canada), embedded in paraffin in a KD-BMII tissue embedding center (IHC World, Ellicott City, MD), and sectioned through their entire volume with a Thermo Scientific HM 325 rotary microtome (6_xB5m sections) for further analysis.\" - Hematoxylin and eosin (H&E) staining - \"Sections were stained with Mayers hematoxylin (Electron Microscopy Sciences, Hatfield, PA) and eosin Y (Sigma). Slides were dewaxed with xylene twice (5_ min/wash), and then transferred to 100%, 95%, 70% ethanol, and deionized water baths (3_min/wash, two baths per ethanol concentration). Slides were submerged in hematoxylin bath for 15_min, and then rinsed with tap water for an additional 15_min. Slides were then placed in 95% ethanol for 30_s, followed by their immersion in eosin for 1_min. Slides were subsequently transferred into 95% ethanol bath for 1_min, and 2 separate 100% ethanol baths (1_min/bath). Samples were cleared by submerging them into 2 xylene baths (3_min/wash). Toluene mounting solution was added to each slide prior to placing the coverslips. Slides were left to dry overnight prior to imaging.\" - \"Human CD31 (hCD31) and alpha-smooth muscle actin (u03B1-SMA) staining\" - \"The largest cross-sectional area of each implant (approximating the center of the implant region) within the explanted tissue sections was first identified with H&E staining. The subsequent serial section of each implant was then deparaffinized with xylene and rehydrated through a series of graded ethanol washes and ending with water. Slides were placed in antigen retrieval solution (Dako: Agilent, Santa Clara, CA) and placed in steamer (95u201399_xB0C) for 35_min. The antigen retrieval solution with the samples was then removed and slides equilibrated to room temperature. Slides were rinsed 3xD7 with tris-buffered saline (TBS-T), 2_min/wash. The area around the tissue was marked with an ImmEdge pen (Vector Laboratories, Inc., Burlingame, CA). The Dako EnVision System-HRP (DAB) kit (Dako) was utilized for hCD31 and u03B1-SMA staining. Slides were rinsed with TBS-T (3 times, 2_min/wash) before any solution from the Dako kit was added to the samples. First, peroxidase blocking solution was added to each tissue for 5_min. For hCD31 labeling, a mouse anti-human CD31 monclonal antibody (Dako, 1:50 dilution in TBS-T) was used as the primary antibody, incubated at 4_xB0 C for 16_h. For u03B1-SMA labeling, a mouse anti-alpha smooth muscle actin monoclonal antibody (1A4 (asm-1)) (Invitrogen: Thermo Fisher Scientific) was used as the primary antibody, incubated at room temperature for 2_h. The HRP-labeled polymer solution was added to each sample and left for 30_min. Samples were then kept with DAB+ substrate-chromogen buffer solution for 5_min. At least one sample from each experimental condition was subjected to the entire staining protocol without primary antibody as a negative control to confirm the specificity of the staining. Most samples were counter-stained with hematoxylin and/or eosin. Slides were then washed with 95% ethanol, 100% ethanol, and xylene. Toluene mounting medium was added prior to covering the samples with coverslips.\" - Vessel quantification - \"Bright-field images (4xD7u2009and 20xD7) of each complete implant stained for H&E, hCD31, and u03B1-SMA were taken with an inverted Nikon microscope (Nikon Instruments Inc., Melville, NJ). Auto stitch software, developed by Brown and Lowe, was utilized to create complete images of each implant from the 4xD7 images taken. The total cross-sectional area (mm) of each implant was quantified using ImageJ after first defining the perimeter of the implant using H&E staining. For hCD31+ vessel quantification, five 20xD7 hCD31-stained images from each implant were quantified to determine the average vessel density (# of vessels/mm). The cross-sectional area (mm) was then multiplied by the average vascular density (# of vessels/mm) to determine the total number of hCD31+ vessels found within each implant at days 3 and 7. For u03B1-SMA+ vessel quantification, 20xD7 images were taken of the entire implant and all images were quantified to determine the total number of vessels in the implant region. Vessels were defined by the presence of a lumen surrounded by a complete brown rim of positive hCD31 or u03B1-SMA staining, depending on the quantification. If host erythrocytes were present in the lumens, vessels were considered perfused.\" - Measurement of implant volumes - \"The volumes of model implants fabricated in a manner identical to real implants were evaluated via ultrasound. Instead of injecting constructs into mice, the same volume (500_xB5L) delivered in vivo was pipetted into the transparent caps of 5_mL Eppendorf Tubes that had been placed in wells of 6-well plates. Samples were then placed in the incubator for 25u201330_min at 37_xB0C to allow complete gelation. To prevent samples from drying, 6_mL of EGM-2 was added to each construct, and constructs were then flipped upside-down and left in the incubator overnight. High frequency ultrasound imaging was employed using the Vevo 770 (VisualSonics Inc., Toronto, Canada) with the same parameters described previously to image cellular implants and quantify their volumes. The implant volumes were normalized to the volume of the pre-cultured condition, which was found to be close to 500_xB5L, the expected initial implant volume. Implant volumes for each test condition were averaged over three implants.\" - Statistical analysis - \"Statistical analyses for multiple comparisons for in vitro experiments were performed using one-way ANOVA followed by Tukeys post hoc test and in vivo experiments were performed using Brown-Forsythe and Welch ANOVA followed by Dunnett s post hoc test using GraphPad Prism version 8.4.2 for Windows, GraphPad Software, San Diego, California USA, . Data are reported as meanu2009xB1u2009standard deviation (SD). Values of pu2009u2264u20090.05 were considered statistically significant.\"- - \\' \\' - The datasets generated during and/or analyzed during the current study are available from the corresponding author on reasonable request. - \\' \\'- - \"Falluji, N. & Mukherjee, D. Critical and acute limb ischemia: an overview. Angiology 65, 137u2013146 (2014)._ _ _ Sanguily, J., Martinsen, B., Igyarto, Z. & Pham, M. Reducing amputation rates in critical limb ischemia patients via a limb salvage program: a retrospective analysis. Vasc. Dis. Manag. 13, E112u2013E119 (2016)._ Yost, M. Critical Limb Ischemia Volume I: United States Epidemiology (The SAGE Group, San Francisco, 2010)._ Raval, Z. & Losordo, D. W. Cell therapy of peripheral arterial disease: From experimental findings to clinical trials. Circ. Res. 112, 1288u2013 1302 (2013)._ _ _ _ Ferrara, N. & Alitalo, K. Clinical applications of angiogenic growth factors and their inhibitors. Nat. Med. 5, 1359u20131364 (1999)._ _ _ _ Rivard, A. & Isner, J. M. Angiogenesis and vasculogenesis in treatment of cardiovascular disease. Mol. Med. 4, 429u2013440 (1998)._ _ _ _ _ Simons, M. & Ware, J. A. Therapeutic angiogenesis in cardiovascular disease. Nat. Rev. Drug Discov. 2, 863u2013871 (2003)._ _ _ _ Yancopoulos, G. D. et al. Vascular-specific growth factors and blood vessel formation. Nature 407, 242u2013248 (2000)._ _ _ _ Losordo, D. W. & Dimmeler, S. Therapeutic angiogenesis and vasculogenesis for ischemic disease: part I: angiogenic cytokines. Circulation 109, 2487u20132491 (2004)._ _ _ Losordo, D. W. & Dimmeler, S. Therapeutic angiogenesis and vasculogenesis for ischemic disease: part II: cell-based therapies. Circulation 109, 2692u20132697 (2004)._ _ _ Cooke, J. P. & Losordo, D. W. Modulating the vascular response to limb ischemia: angiogenic and cell therapies. Circ. Res. 116, 1561u20131578 (2015)._ _ _ _ _ Rufaihah, A. J. et al. Endothelial cells derived from human iPSCS increase capillary density and improve perfusion in a mouse model of peripheral arterial disease. Arterioscler. Thromb. Vasc. Biol. 31, e72u2013e79 (2011)._ _ _ _ Gupta, R. & Losordo, D. W. Cell therapy for critical limb ischemia moving forward one step at a time. Circ. Cardiovasc. Interv. 4, 2u20135 (2011)._ _ _ _ Chen, X. et al. Prevascularization of a fibrin-based tissue construct accelerates the formation of functional anastomosis with host vasculature. Tissue Eng. A 15, 1363u20131371 (2009)._ _ _ Perry, L., Merdler, U., Elishaev, M. & Levenberg, S. Enhanced host neovascularization of prevascularized engineered muscle following transplantation into immunocompetent versus immunocompromised mice. Cells 8, 1472 (2019)._ _ _ _ Levenberg, S. et al. Engineering vascularized skeletal muscle tissue. Nat. Biotechnol. 23, 879u2013884 (2005)._ _ _ _ Stevens, K. R. et al. Physiological function and transplantation of scaffold-free and vascularized human cardiac muscle tissue. Proc. Natl. Acad. Sci. 106, 16568u2013 16573 (2009)._ _ _ _ _ _ Kang, K. T., Allen, P. & Bischoff, J. Bioengineered human vascular networks transplanted into secondary mice reconnect with the host vasculature and re-establish perfusion. Blood 118, 6718u20136721 (2011)._ _ _ _ _ Stevens, M. M. et al. In vivo engineering of organs: the bone bioreactor. Proc. Natl. Acad. Sci. 102, 11450u201311455 (2005)._ _ _ _ _ _ Dean, D. M. & Morgan, J. R. Cytoskeletal-mediated tension modulates the directed self-assembly of microtissues. Tissue Eng. A 14, 1989u20131997 (2008)._ _ _ Dean, D. M., Rago, A. P. & Morgan, J. R. Fibroblast elongation and dendritic extensions in constrained versus unconstrained microtissues. Cell Motil. Cytoskelet. 66, 129u2013141 (2009)._ _ _ Kelm, J. M. et al. Design of custom-shaped vascularized tissues using microtissue spheroids as minimal building units. Tissue Eng. 12, 2151u20132160 (2006)._ _ _ McGuigan, A. P. & Sefton, M. V. Vascularized organoid engineered by modular assembly enables blood perfusion. Proc. Natl. Acad. Sci. 103, 11461u201311466 (2006)._ _ _ _ _ _ Youssef, J., Nurse, A. K., Freund, L. B. & Morgan, J. R. Quantification of the forces driving self-assembly of three-dimensional microtissues. Proc. Natl. Acad. Sci. 108, 6993u20136998 (2011)._ _ _ _ _ _ Christman, K. L. et al. Injectable fibrin scaffold improves cell transplant survival, reduces infarct expansion, and induces neovasculature formation in ischemic myocardium. J. Am. Coll. Cardiol. 44, 654u2013660 (2004)._ _ _ _ Ceccarelli, J. & Putnam, A. J. Sculpting the blank slate: how fibrins support of vascularization can inspire biomaterial design. Acta Biomater. 10, 1515u20131523 (2014)._ _ _ _ Rioja, A. Y., Annamalai, R. T., Paris, S., Putnam, A. J. & Stegemann, J. P. Endothelial sprouting and network formation in collagen-and fibrin-based modular microbeads. Acta Biomater. 1, 33u201341 (2016)._ _ _ Melero-Martin, J. M. et al. Engineering robust and functional vascular networks in vivo with human adult and cord blood-derived progenitor cells. Circ. Res. 103, 194u2013202 (2008)._ _ _ _ _ Grainger, S. J., Carrion, B., Ceccarelli, J. & Putnam, A. J. Stromal cell identity influences the in vivo functionality of engineered capillary networks formed by co-delivery of endothelial cells and stromal cells. Tissue Eng. A 19, 1209u20131222 (2013)._ _ _ Rao, R. R., Peterson, A. W., Ceccarelli, J., Putnam, A. J. & Stegemann, J. P. Matrix composition regulates three-dimensional network formation by endothelial cells and mesenchymal stem cells in collagen/fibrin materials. Angiogenesis 15, 253u2013 264 (2012)._ _ _ _ _ Ghajar, C. M. et al. Mesenchymal cells stimulate capillary morphogenesis via distinct proteolytic mechanisms. Exp. Cell Res. 316, 813u2013825 (2010)._ _ _ _ _ Peterson, A. W. et al. Vasculogenesis and angiogenesis in modular collagenu2013fibrin microtissues. Biomater. Sci. 2, 1497u20131508 (2014)._ _ _ _ _ Annamalai, R. T., Rioja, A. Y., Putnam, A. J. & Stegemann, J. P. Vascular network formation by human microvascular endothelial cells in modular fibrin microtissues. ACS Biomater. Sci. Eng. 2, 1914u20131925 (2016)._ _ _ Alajati, A. et al. Spheroid-based engineering of a human vasculature in mice. Nat. Methods 5, 439u2013445 (2008)._ _ _ _ Laib, A. M. et al. Spheroid-based human endothelial cell microvessel formation in vivo. Nat. Protoc. 4, 1202u20131215 (2009)._ _ _ _ Korff, T., Kimmina, S., Martiny-Baron, G. & Augustin, H. G. Blood vessel maturation in a 3-dimensional spheroidal coculture model: direct contact with smooth muscle cells regulates endothelial cell quiescence and abrogates VEGF responsiveness. FASEB 15, 447u2013457 (2001)._ _ _ De Moor, L. et al. High-throughput fabrication of vascularized spheroids for bioprinting. Biofabrication 10, 035009 (2018)._ _ _ _ _ Vorwald, C. E., Joshee, S. & Leach, J. K. Spatial localization of endothelial cells in heterotypic spheroids influences Notch signaling. J. Mol. Med. 98, 425u2013435 (2020)._ _ _ _ Murphy, K. C., Whitehead, J., Zhou, D., Ho, S. S. & Leach, J. K. Engineering fibrin hydrogels to promote the wound healing potential of mesenchymal stem cell spheroids. Acta Biomater. 64, 176u2013186 (2017)._ _ _ _ _ Bhang, S. H. et al. Three-dimensional cell grafting enhances the angiogenic efficacy of human umbilical vein endothelial cells. Tissue Eng. A 18, 310u2013319 (2012)._ _ _ Roux, B. M. et al. Preformed vascular networks survive and enhance vascularization in critical sized cranial defects. Tissue Eng. A 24, 1603u20131615 (2018)._ _ _ Gorkun, A. A. et al. Angiogenic potential of spheroids from umbilical cord and adipose-derived multipotent mesenchymal stromal cells within fibrin gel. Biomed. Mater. 13, 044108 (2018)._ _ _ _ _ Torres, A. L. et al. Guiding morphogenesis in cell-instructive microgels for therapeutic angiogenesis. Biomaterials 154, 34u2013 47 (2018)._ _ _ _ Torres, A. L. et al. Microvascular engineering: dynamic changes in microgel-entrapped vascular cells correlates with higher vasculogenic/angiogenic potential. Biomaterials 228, 119554 (2020)._ _ _ _ Du, P. et al. An injectable, self-assembled multicellular microsphere with the incorporation of fibroblast-derived extracellular matrix for therapeutic angiogenesis. Mater. Sci. Eng. C 113, 110961 (2020)._ _ _ Rioja, A. Y., Daley, E. L. H., Habif, J. C., Putnam, A. J. & Stegemann, J. P. Distributed vasculogenesis from modular agarose-hydroxyapatite-fibrinogen microbeads. Acta Biomater. 55, 144u2013152 (2017)._ _ _ _ _ Ghajar, C. M., Blevins, K. S., Hughes, C. C. W., George, S. C. & Putnam, A. J. Mesenchymal stem cells enhance angiogenesis in mechanically viable prevascularized tissues via early matrix metalloproteinase upregulation. Tissue Eng. 12, 2875u20132888 (2006)._ _ _ _ Ghajar, C. M. et al. The effect of matrix density on the regulation of 3-D capillary morphogenesis. Biophys. J. 94, 1930u20131941 (2008)._ _ _ _ _ _ Carrion, B., Janson, I. A., Kong, Y. P. & Putnam, A. J. A safe and efficient method to retrieve mesenchymal stem cells from three-dimensional fibrin gels. Tissue Eng. C 20, 252u2013263 (2014)._ _ _ Brown, M. & Lowe, D. G. Automatic panoramic image stitching using invariant features. Int. J. Comput. Vis. 74, 59u201373 (2007)._ _ Gudur, M. S. R. et al. Noninvasive quantification of in vitro osteoblastic differentiation in 3D engineered tissue constructs using spectral ultrasound imaging. PLoS ONE 9, e85749 (2014)._ _ _ _ _ _ \"- - Research reported in this publication was supported by the National Heart, Lung, and Blood Institute of the National Institutes of Health under Award Numbers R01-HL085339 (to AJP) and R01-HL118259 (to JPS and AJP). NEF was partially supported by the Tissue Engineering and Regeneration Training Program at the University of Michigan (T32-DE007057). AYR was partially supported by the Training Program in Translational Cardiovascular Research and Entrepreneurship (T32-HL125242). JRB was partially supported by the Cellular Biotechnology Training Program at the University of Michigan (T32-GM008353). JAB was partially supported by the Kidney Research Training Program (T32-DK007378). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.- - Author notes - \\'These authors contributed equally: Nicole E. Friend and Ana Y. Rioja.\\' - Authors and Affiliations - \"Department of Biomedical Engineering, University of Michigan, Ann Arbor, USANicole E. Friend,_Ana Y. Rioja,_Yen P. Kong,_Xiaowei Hong,_Julia C. Habif,_Cheri X. Deng,_Jan P. Stegemann_&_Andrew J. PutnamDivision of Nephrology, Department of Internal Medicine, University of Michigan, Ann Arbor, USAJeffrey A. BeamishDepartment of Chemical Engineering, University of Michigan, Ann Arbor, USAJonathan R. Bezenah_ &_Andrew J. Putnam\" - \"AuthorsNicole E. FriendYou can also search for this author in _Ana Y. RiojaYou can also search for this author in _Yen P. KongYou can also search for this author in _Jeffrey A. BeamishYou can also search for this author in _Xiaowei HongYou can also search for this author in _Julia C. HabifYou can also search for this author in _Jonathan R. BezenahYou can also search for this author in _Cheri X. DengYou can also search for this author in _Jan P. StegemannYou can also search for this author in _Andrew J. PutnamYou can also search for this author in _\" - Contributions - A.Y.R., C.X.D., J.P.S., and A.J.P. designed the study. A.Y.R., N.E.F., Y.P.K., J.A.B., X.H., J.C.H., and J.R.B. performed experiments, processed samples, and quantified data. A.Y.R., N.E.F., J.P.S., and A.J.P. analyzed and interpreted data and wrote/edited the manuscript. All authors reviewed and approved this manuscript. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License, and from RCSB Protein Data Bank (doi: 10.2210/pdb1q3v/pdb) using the 3D View tool.\"- - Preparation and characterization of light-triggered Cre-UCNPs - \"Core-shell, lanthanide-doped NaYF4:Tm,Yb UCNPs (Fig._) were selected for the design of an NIR activatable editing system and synthesized according to established protocols. Yb acted as the sensitizer, enabling efficient NIR absorption in the range of 900-1000u2009nm, while Tm acted as the activator ion, enabling blue- to UV-shifted emissions. Incorporation of the NaYF4 shell improved the fluorescence intensity of the core UCNPs by minimizing quenching effects caused by interactions of dopant ions with the solvent. Powder X-ray diffraction confirmed that the UCNPs were highly crystalline and hexagonal in phase (Supplementary Figs._). The UCNPs also exhibited characteristic upconversion emission peaks at 345, 362, 450, and 475u2009nm after activation with a 980u2009nm NIR laser (Supplementary Fig._). The conversion yield of NIR to blue light was ~2u2013 4% (Supplementary Fig._).\" - \"To allow the chemical immobilization of biomolecules, the surface of the UCNPs was coated with a mixture of amino- and azido-containing organosilanes, as silica-coated UCNPs have shown good biocompatibility with brain tissue in vivo. The resulting N3/NH2-UCNPs showed an upconversion spectrum similar to that of nonsilanized samples (Supplementary Fig._), and an average diameter of 25.6u2009xB1u2009 1.1u2009nm, as determined by transmission electron microscopy (TEM) analyses (Supplementary Fig._). The presence of amine and azide groups was confirmed by Fourier transform infrared (FTIR) spectroscopy (Supplementary Fig._), fluorescamine quantification (~830 surface amine groups per UCNP) and u03B6-potential measurements (Supplementary Fig._). The UCNP surface amine groups reacted with the hydroxyl groups in HCQ (Supplementary Fig._). HCQ is a clinically approved drug used to treat malaria and lymphoma, and it has lysosomotropic properties contributing to the endosomal escape of NPs. However, the effect of HCQ on the endosomal escape of protein cargo is largely unknown. Approximately 60 HCQ molecules were conjugated to each UCNP, as determined by absorbance measurements. The UCNP surface azide groups were reacted with the alkyne group of the PCL, which contained a terminal carboxyl group enabling the reaction with the amine groups of Cre recombinase (Fig._ and Supplementary Fig._). The resultant formulation displayed an average diameter of 28.0u2009xB1u20092.8u2009nm as determined by scanning TEM analyses (Fig._ and Supplementary Fig._), an upconversion spectrum that overlaps the absorbance of the PCL (Fig._) and approximately 9.5u2009xB5 g of Cre per mg of UCNP, corresponding to an average total of 4 proteins per UCNP (hereafter termed as Cre-UCNP). According to dynamic laser scattering studies, the Cre-UCNPs were negatively charged when suspended in culture medium and relatively stable for at least 48u2009h (Supplementary Fig._). Importantly, the Cre-UCNP formulation released approximately 58% of the immobilized protein after exposure to an NIR laser (980u2009nm, 785u2009mW/cm) for 2u2009min (Fig._).\" - \"Fig. 2: Cellular uptake and intracellular trafficking of Cre-UCNPs.a Representative STEM image of Cre-UCNPs, obtained from a total of 5 images. Scale bar = 100u2009 nm. b Luminescence emission spectrum of UCNPs (red) and absorption spectra of PCL (black). c Protein release from Cre-UCNPs (50u2009u03BCg/mL in PBS) was quantified by SDS-PAGE after activation by a NIR laser (785u2009mW/cm) for a specific time. Results are expressed as Mean xB1 SEM (n =u20093 technical replicates). d Cellular uptake. Quantification of yttrium (Y) by ICP-MS in fibroblasts exposed to different concentrations of Cre-UCNPs for a specific time, washed, harvested and finally freeze-dried. The delivered dose was normalized per cell number. Results are expressed as Mean xB1 SEM (nu2009=u20093 independent experiments, being the value of each independent experiment the average of three technical replicates). e Cellular uptake mechanism. Uptake of Cre-UCNPs in fibroblasts after silencing key regulators of clathrin-mediated endocytosis (CLTC and LDLR), caveolin-mediated endocytosis (CAV1), GEEC-CCLIC pathways (CDC42), macropinocytosis (RAC1 and CTBP1) and scavenger receptors (SCARA3) with siRNAs. Results were expressed as Mean xB1 SEM (nu2009=u20093 independent experiments) and analyzed by one-way ANOVA followed by Tukeys multiple comparisons test: (*), Pu2009 =u20090.0229; (**), Pu2009=u20090.0076. f Intracellular trafficking. Co-localization of Cre-UCNPs (stained with anti-Cre antibody) with intracellular vesicles labeled for LAMP1 after 5u2009min exposure. Results are expressed as Mean xB1 SEM (nu2009=u200917u201318 images obtained from 3 technical replicates in 3 independent runs) and analyzed by a two-tailed t-test: (*), Pu2009=u20090.0285. g Endolysosomal disruption and formation of galectin-9 foci. Representative confocal microscopy images of fibroblasts 24u2009h after incubation with Cre-UCNP (with or without immobilized HCQ on their surface) or soluble HCQ (26u2009xB5g/mL; 60u2009 xB5M) for 16u2009h. Scale bars = 20 u03BCm. h Mean area of galectin-9 foci larger than 2u2009xB5m and i total number of galectin-9 foci per cell. Results are expressed as Mean xB1 SEM (nu2009=u20092 independent experiments, each experiment with 2 technical replicates; 5-8 confocal images were acquired for each replicate). Statistical analysis was performed by one-way ANOVA followed by Tukeys multiple comparisons test: in h, (**), Pu2009=u20090.0084; (***), Pu2009=u20090.0001; in i, (**), Pu2009=u20090.0028; (***), Pu2009 =u20090.0008.\" - Internalization, cytotoxicity and intracellular trafficking of light-triggered UCNPs - \"The uptake of Cre-UCNPs by a fibroblast Cre reporter cell line was determined by inductive coupled plasma mass spectrometry (ICP-MS) (Fig._). The cellular internalization of the Cre-UCNPs was relatively rapid (approximately 2-4u2009 h) and was dependent on the NP concentration and presence of the HCQ (Fig._ and Supplementary Fig._). To evaluate the possible cytotoxic effects of the Cre-UCNPs and NIR laser irradiation (980u2009nm, 785u2009mW/cm), cells were exposed to a range of concentrations of Cre-UCNPs (from 25 to 500u2009u03BCg/mL) and various light pulse durations (Supplementary Figs._). No measurable cytotoxicity effect was observed for concentrations less than or equal to 50u2009u03BC g/mL and irradiation times less than 30u2009min. Under these conditions, irradiated cells showed no heat shock response, which was monitored by heat shock protein 70 staining (Supplementary Fig._), and no double-stranded DNA breaks were found, as determined by assessment of the phosphorylation of a histone H2A.X variant (Supplementary Fig._).\" - \"To identify the pathway through which our negatively charged Cre-UCNP penetrated cells, we exposed fibroblasts to fluorescently labeled Cre-UCNPs in the presence of chemical or siRNA-mediated endocytosis inhibitors (Fig._ and Supplementary Fig._) and assessed particle internalization via flow cytometry. Our chemical and siRNA inhibition results showed that the main internalization mechanism was mediated by clathrin-mediated endocytosis and, to a lesser degree, by macropinocytosis. These cellular uptake pathways were also supported by TEM analyses (Supplementary Fig._).\" - \"To evaluate the intracellular trafficking of the Cre-UCNPs, cells were exposed to Cre-UCNPs (50u2009u03BCg/mL) and monitored by confocal microscopy (Fig._ ) to determine the staining of Cre and lysosome-associated membrane protein 1 (LAMP1). After 5u2009min of exposure to NPs, the cells were characterized, and the co-localization of Cre-UCNPs with stained LAMP1 foci was reduced compared to that of the Cre-UCNPs without HCQ. Our results further showed that cells exposed to Cre-UCNPs with HCQ had a lower number of LAMP1-positive vesicles than those exposed to Cre-UCNPs without HCQ, which may indicate disrupted lysosome biogenesis (Supplementary Fig._). The decrease in the colocalization of Cre-UCNPs with LAMP1 may be explained by the fact that HCQ impairs autophagosome fusion with lysosomes, and thus preventing the accumulation of NPs in LAMP1-positive vesicles, and/or by the disruption of the endolysosomal membrane compartment. To determine whether Cre-UCNPs with HCQ induced endolysosomal compartment disruption, we monitored galectin-9 by immunofluorescence, which is a very sensitive sensor of membrane damage and has been used to demonstrate endosomal escape of therapeutic molecules, including chloroquine. Indeed, the cells exposed to Cre-UCNPs with HCQ showed an increased number of galectin-9 foci compared to those treated with Cre-UCNPs without HCQ (Fig._). Moreover, the large foci size found in the cells treated with Cre-UCNPs with HCQ further suggested the disruption of the endolysosomal compartment. These results also aligned with the results obtained by TEM analyses (Supplementary Fig._). In contrast to Cre-UCNPs without HCQ, Cre-UCNPs with HCQ showed low colocalization within the endolysosomal compartment 4u2009h after transfection. Overall, our results indicated that Cre-UCNPs were taken up by cells mostly via clathrin-mediated endocytosis and that the NPs rapidly escaped the endolysosomal compartment due to the presence of HCQ.\" - Remote activation of Cre-UCNPs - \"To evaluate the intracellular delivery of Cre recombinase by Cre-UCNPs, fibroblast Cre reporter cells containing a LoxP-RFP-Stop-LoxP-GFP sequence cassette (Fig._ ) were exposed to the NPs for 4u2009h, washed to remove the noninternalized Cre-UCNPs and then activated by an NIR laser for 3 cycles of 5u2009min exposure. The recombination process was monitored by high-content microscopy (Fig._ and Supplementary Fig._) and flow cytometry (Fig._). In both analyses, the cells showed approximately 100% recombination after 30-40u2009h, even when cell transfection was performed in medium supplemented with 10% fetal bovine serum (Supplementary Fig._). Importantly, when Cre-UCNPs without HCQ were evaluated, the recombination efficiency was found to be largely absent even when the NPs had been internalized by the cells (Fig._ and Supplementary Fig._). In our experiments, 2.5u2009nM of Cre (i.e., 10u2009u03BCg/mL Cre-UCNP) was sufficient to achieve recombination in 50% of the cells, which is a concentration approximately 125-fold lower than that used by the commercial transfection agent RNAiMAX, to achieve similar recombination efficiency (Fig._). Our results compare favorably with other protein transfection agents, ranging from 4- to 100-fold enhanced activity compared to Cre enzyme engineered with a polyanionic protein, Cre complexed with cationic lipids, and Cre delivered by small transduction compounds after hypertonic shock or cell-penetrating peptide TAT dimers. Importantly, NIR activation was necessary to enable gene editing from intracellular Cre-UCNPs (Fig._), which could be triggered up to 8u2009h after transfection, albeit with a reduction in their activity (Supplementary Fig._). Furthermore, this nanoformulation enabled gene editing with single-cell resolution when multiphoton microscopy was used for NIR activation (Fig._).\" - \"Fig. 3: In vitro activation of Cre conjugated into UCNPs.a Reporter system for Cre activity. b Representative high-content microscopy images of the cell recombination process at time zero and 48u2009h after transfection, obtained from 3 independent experiments. Fibroblast reporter cells were transfected with Cre-UCNPs (50u2009 xB5g/mL) for 4u2009h, washed, and activated by a NIR laser (980u2009nm; 785u2009 mW/cm; 15u2009min). Scale bar = 100 u03BCm. c Quantification of cell recombination. GFP fluorescence intensity was quantified and normalized to the background. Results are Mean xB1 SEM (nu2009=u20093 independent experiments). Statistical analysis was performed by one-way ANOVA followed by Dunnetts multiple comparisons: (*), Pu2009=u20090.041. d Cell recombination monitored by flow cytometry. Representative scatter plots of cells treated with Cre-UCNPs at 40u2009 h post-transfection. Fibroblast reporter cells were transfected with Cre-UCNPs without (50u2009xB5g/mL) or with HCQ (50u2009xB5g/mL; named as Cre-UCNPs) for 4u2009h, washed, and activated by a NIR laser. Percentage of GFP cells was calculated based in the gates defined for untreated cells (below 3%; dark blue). e Quantification of cell recombination by flow cytometry. Results are Mean xB1 SEM (3 independent experiments; each experiment with 1u20133 technical replicates). In Cre-UCNPs (50u2009xB5g/mL) the concentrations of immobilized Cre and HCQ were 12.5u2009nM (0.48u2009xB5g/mL) and 0.22u2009xB5M, respectively. Therefore, as a control, cells were treated with a solution containing the same concentrations of Cre and HCQ. In the experimental group of HCQ alone, the concentration was 100u2009xB5M. Cell recombination efficiency was calculated based in the percentage of GFP cells. Statistical analysis was performed by one-way ANOVA test followed by a Bonferronis multiple comparisons test: (****), Pu2009 <u20090.0001. f Cell recombination efficiency by flow cytometry after cell transfection with Cre-UCNPs (with and without HCQ) and Cre using a commercial transfection agent (lipofectamine RNAiMAX). Results are Mean xB1 SEM (nu2009=u20093 independent experiments). g Spatial control in cell recombination. Fibroblast reporter cells were transfected with Cre-UCNPs (50u2009xB5g/mL) for 4u2009h, washed and a single cell was activated in a multiphoton microscope by the NIR laser (marked by an arrowhead). Representative image of recombination at 48u2009h post-transfection from 3 independent experiments. Scale bar = 40u2009xB5m.\" - \"To evaluate NIR laser penetration, an initial in vitro assay was performed using murine tissue varying in thickness and composition placed above culture dishes containing a fibroblast Cre reporter line (Supplementary Fig._). We determined that a 980u2009nm laser at 780u2009mW/cm power delivered through the whole mouse brain (thickness: u223C6u2009mm) led to the generation of 1.4u2009 mW/cm of blue light (Supplementary Fig._), which was sufficient to activate up to 50% of the Cre-UCNPs located within the Cre reporter cells (Supplementary Fig._). Next, an in vivo experiment was performed to evaluate NIR laser light penetration through live tissue. In this case, fibroblast Cre reporter cells (RFP-labeled without recombination) were preincubated with Cre-UCNPs and injected intramuscularly into the mouse hindlimb (approximately 4u2009mm deep). We chose intramuscular injection since we could readily trace and characterize the implanted cells. Immediately after the injection, the Cre-UCNPs were activated with NIR light, the muscles were harvested after 60u2009h, the number of GFP-positive cells was determined, and the GFP intensity was quantified (Supplementary Fig._ ). Implanted cells activated with an NIR laser had a higher number of GFP-positive cells than cells without activation. Collectively, our results indicated that we had designed a formulation that can be activated through biological tissues up to a few millimeters deep after in vivo administration.\" - Gene editing of SVZ cells mediated by Cre-UCNPs - \"Brain delivery of gene-editing protein formulations offers the potential to interrogate brain circuits and potentiate in vivo regenerative processes. Viral delivery of Cre-expressing constructs by stereotaxic administration into the brain or by crossing a cre-driver mouse line with a cre-dependent reporter mouse line has been performed; however, these may suffer limitations in terms of spatial resolution and biocompatibility issues (see below). Alternatively, in vivo gene editing of the inner ear and brain cells has been demonstrated by the delivery of Cre recombinase fused with a polyanionic protein and complexed with cationic lipids, or by encapsulation in bioreducible lipid NPs. However, these formulations have limited transfection efficiency (see above); they require the fusion of Cre with a negatively charged protein and they do not enable spatial control over gene editing. To demonstrate the potential for gene editing of the Cre-UCNPs in brain cells, we chose to target SVZ cells since these cells are an important in the adult mammalian neurogenic niche. Cells were isolated from Rosa26-YFP neonatal transgenic mice, which contain a LoxP-Stop-LoxP-YFP sequence cassette inserted into the Gt(ROSA)26Sor locus (Fig._). SVZ cells were cultured in the presence of Cre-UCNPs for 4u2009h, washed to remove the noninternalized NPs, exposed or not exposed to an NIR laser, and cultured for 4 additional days in fresh culture medium to allow cell recombination (Fig._). The transfected cells showed similar nuclear condensation at all the concentrations tested, indicating low cytotoxicity; however, higher concentrations of Cre-UCNPs affected SVZ cell proliferation (Supplementary Figs._). Cellular uptake of the Cre-UCNPs was demonstrated by ICP-MS analyses as well as Cre immunolabeling (Supplementary Figs._). Cell recombination was evaluated by the quantification of YFP-positive cells, using a high-content microscope (Fig._). Notably, Cre-UCNPs mediated gene editing with higher efficiency than both soluble Cre (even with HCQ) and adeno-associated transfection with Cre (AAV5-Cre; used in this study as a control). These results are in line with previous reports showing limited AAV5 efficiency in mediating gene delivery into some stem cells. Next, we investigated the tropism of our formulation by characterizing the phenotype of the resulting YFP-positive cells (Fig._). YFP-positive cells were found to be equally distributed within more immature cells (Nestin and NestinGFAP cells) and neuronal differentiated cells (NeuN cells), whereas fewer GFAP-positive astrocytes were found.\" - \"Fig. 4: Gene editing at SVZ region mediated by Cre-UCNPs.a In vivo reporter system for Cre activity. The administration of Cre-UCNPs in the SVZ region of R26YFP mice, followed by the NIR laser exposure leads to the excision of a stop codon and consequent expression of YFP fluorescence reporter protein. b Schematic representation of the in vitro protocol to study the gene edition of SVZ cells by Cre-UCNPs. c Quantification of YFP-positive cells in SVZ cells transfected with: (i) Cre-UCNPs (200u2009xB5g/mL) followed by NIR laser activation, (ii) AAV or with (iii) soluble Cre in the presence of HCQ (concentrations equivalent to the cargo of 200u2009xB5g/mL Cre-UCNPs). Results are Mean xB1 SEM (2-3 independent experiments, each experiment with 2 technical replicates). d Colocalization of YFP with markers of SVZ cell sub-populations. YFP positive cells (green) were immunolabelled for Nestin (red), GFAP (gray) and NeuN (red). Representative images from 3 independent experiments. Scale bar = 20 u03BCm. e Distribution of YFP-positive cells within SVZ cell sub-populations. Results are Mean xB1 SEM (nu2009=u20093 independent experiments, each one is average of two technical replicates). In (c) and (e), statistical analyses were performed by one-way ANOVA test followed by a TukeyxB4s multiple comparison test: (*), Pu2009=u2009 0.0244; (**), Pu2009=u20090.0052; (****), Pu2009<u20090.0001. f Schematic representation of the animal protocol used to monitor in vivo gene editing in the SVZ region, 4 weeks after stereotaxic injection of Cre-UCNPs. g Representative confocal microscopy images of the SVZ region, from nu2009=u20092u20133 animals. Cells labelled in yellow (stained by an anti-YFP antibody) were edited by Cre recombinase. Scale bar = 50 u03BCm. h Percentage of anti-YFP stained positive cells. i Quantification of anti-YFP fluorescence intensity in SVZ region tissues. Results are Mean xB1 SEM (nu2009=u20092-3 animals per condition, 2 injections per animal, 4 photos per injection; intensity normalized to ROI). Data in h and i were analyzed by an unpaired two-tailed t test: (**), Pu2009=u20090.0027 (***), Pu2009=u20090.0002.\" - \"To further demonstrate the potential of Cre-UCNPs to control endogenous SVZ cells, NPs were administered to the SVZ region of Rosa26-YFP mice by stereotaxic injection. One hour after administration, a subset of animals was sacrificed to assess the intracellular accumulation of Cre-UCNPs by TEM analyses (Supplementary Fig._ ) and some of the animals were stimulated with or without exterior NIR light (Fig._ ). One month following activation, the YFP signal was confirmed by immunostaining (Fig._), and the fluorescence intensity and the percentage of YFP-positive cells were quantified in the region of interest (Fig._). Our results showed Cre-UCNP-mediated gene editing in the SVZ region with various SVZ cell subpopulations, indicating recombination (Supplementary Fig._). Altogether, our results showed successful gene editing of SVZ cells.\" - Coupling gene edition, mediated by Cre-UCNPs, with optogenetics - \"Optogenetics is based on optical stimulation and genetics to achieve cell-type-specific modulation of electrical and biochemical neural activity, often with high temporal resolution. To achieve this effect, light-activated channels, such as microbial opsins, are artificially introduced into the genome of the cells. The most widely used method to incorporate opsins into the regions of interest is using AAV vectors for transport and delivery. However, this strategy has limitations since the virus often leaks from the region of interest to other locations, leading to misinterpretation of the biological activity, and gene delivery is relatively low in some brain cells, such as NSCs or microglia. Cre-UCNPs might be used to overcome these limitations since NPs offer greater spatial resolution when injected in vivo and have broader applicability since they are taken up by SVZ cells (see previous section) as well as microglia. In addition, a recent demonstration showing that UCNPs can act as optogenetic actuators of transcranial NIR light to stimulate deep brain neurons suggested new opportunities for the combination of gene editing with optogenetics. Therefore, to evaluate in vitro cellular activity with optogenetics after gene editing by Cre-UCNPs, we transfected primary cortical neurons with AAV5 carrying Cre-dependent ChR2, followed by a second transfection with Cre-UCNPs and NIR laser stimulation to release Cre and initiate the recombination process (Fig._). The concentration of Cre-UCNPs (100u2009 u03BCg/mL) was selected based on a compromise between cellular uptake capacity and NP cytotoxicity (Supplementary Figs._). Three days after irradiation, 25.6% of the cells were positive for YFP and ChR2 (Fig._, Supplementary Fig._ and ). This percentage is close to the 31.7% observed for cells transfected with AAV encoding YFP (not requiring Cre-mediated recombination to express YFP). The activation of ChR2 was demonstrated by whole-cell patch-clamp experiments (Fig._). Approximately 300 pA of electric current was generated in cortical neurons after blue light activation (power density: 4.2u2009mW/cm) of ChR2 (Fig._). The electrophysiological traces presented a characteristic initial peak followed by a steady-state current. As expected, the amplitude of the generated currents was correlated with the blue light power (Fig._). However, while blue-upconverted light generated after NIR irradiation of UCNPs can potentially be used for inducing ChR2 activity, we were unable to specifically dissect the activation of ChR2 due to the depolarization of other direct NIR-mediated membrane channels (independent of ChR2 expression) in cortical neurons (Supplementary Fig._).\" - \"Fig. 5: In vitro gene editing of cortical neurons by Cre-UCNPs followed by cellular activity measurements with optogenetics.a AAV.dflox.ChR2-YFP cassette used for Cre-mediated expression of ChR2 in cortical neurons. Cre-UCNPs delivery to cells followed by NIR laser activation leads to the inversion of the double-floxed inverted open reading frame and results in the expression of ChR2-YFP fusion protein. b Schematic representation of the experimental setup with cortical neurons. After isolation, cells were cultured for 7 days before treatments. Viral particles (AAV5.dflox.ChR2-YFP) were added to the medium for 3 days. Cells were then exposed to Cre-UCNPs (100u2009xB5g/mL) for 4u2009h, activated by NIR laser (980u2009nm; 785u2009mW/cm; 3 cycles of 5u2009min irradiation) and allowed to recombine for 3 days. Neurons were activated by a blue light for electrophysiology measurements. c Flow cytometry analyses of ChR2-YFP expression in neuronal cultures. Cells were transfected with Cre-UCNPs and activated or not with a NIR laser. Positive cells were gated based in the basal expression of ChR2-YFP in nontreated cortical neurons (dark blue). d Quantification of recombination by flow cytometry. Results are Mean xB1 SEM (nu2009=u20093). Statistical analyses were performed by one-way ANOVA followed by Bonferroni s multiple comparisons test: (****), Pu2009<u20090.0001. e Representative traces of ChR2-mediated currents, showing dependency on the intensity of blue light. Scale bars = 100 pA, 50u2009ms. Darker trace shows the dependence of Cre-UCNP recombination on the generation of photocurrents. f Voltage-clamp currents generated from different blue-light stimulus. Results are Mean xB1 SEM (nu2009=u2009 3 independent experiments).\" - \"To evaluate in vivo brain cell activity with optogenetics after gene editing, Cre-UCNPs were injected into the VTA; this region is located deep in the brain (approximately 4.0u2009mm from the bregma) and presents well-characterized anatomy and function, containing dopaminergic neurons that have been implicated in reward mechanisms. To express ChR2, AAV5.DIO.ChR2.YFP particles (Supplementary Table_) and Cre-UCNPs were injected simultaneously into the VTA via stereotaxic administration in the VTA (Fig._). The concentration of UCNPs in the brain decreased over time, up to 7 days post_injection, as confirmed by ICP-MS analyses (Fig._). No significant microglia activation was observed in the VTA region on day 3 post_administration (Supplementary Fig._).\" - \"Fig. 6: In vivo gene editing with Cre-UCNPs followed by functional analyses of the edited cells by optogenetics.a Schematic representation of the in vivo setup. AAV5.dflox.ChR2-YFP was injected into the VTA together with Cre-UCNPs (30u2009 xB5g). Following stereotaxic injection, the target site was irradiated by a NIR laser (980u2009nm, 425u2009mW/cm for 3 cycles of 5u2009min) to induce the release of Cre from the NPs. After 4 weeks, animals were subjected to a CPP test, where the animals preference for one of two regions with contrasting visual and tactile cues in a cage was monitored for 3 days using an automated mouse tracking system. On day 4, whenever the mouse crossed into the non-preferred region of the cage, the VTA was activated with blue light; on the final test day, the animals were re-introduced into the cage to assess changes in preferences. b Quantification of Cre-UCNPs accumulated in the brain of mice was performed in dried tissue by ICP-MS at days 0, 3, and 7 post stereotaxic administration. Results are Mean xB1 SEM (nu2009=u20094u20135 animals). c Local of injection of Cre-UCNPs. d Expression of ChR2-YFP (green) in the TH-positive region (red) of the VTA, as evaluated by immunofluorescence. Of note the differences in fluorescence between left and right VTA (nu2009=u20096 animals). e CPP test to evaluate behavioral changes in preference (fold change in the time spend on the non-preferred chamber before and after blue light stimulation). Results are Mean xB1 SEM, obtained from nu2009=u20093 (CRE-UCNPs), or 6 (Negative control and CRE-UCNPu2009 +u2009NIR) animals. Statistical analysis was performed by one-way ANOVA followed by Tukeys multiple comparisons test: (*), Pu2009=u20090.0115. f Expression of c-Fos positive nuclei in the VTA, following stimulation with blue light. Results are expressed as Mean xB1 SEM (2u20135 animals per condition; for each animal, c-Fos was quantified in 2u20133 regions in VTA slices, 4-13 fluorescence images per slice) and analyzed by unpaired two-tailed t test: (*), Pu2009=u2009 0.024. g Representative confocal images of a mouse treated with Cre-UCNPs showing the expression of c-Fos (gray) and ChR2-YFP (green) in the TH positive region (red) of the VTA, from nu2009=u20095 animals. Scale bar = 20u2009u03BCm.\" - \"Cre-UCNPs successfully released Cre after NIR activation and thus induced ChR2 expression in the VTA dopaminergic neurons, as confirmed by the expression of YFP and tyrosine hydroxylase (TH) in the injected region (Fig._). Importantly, ChR2 expression was markedly restricted to the VTA region after gene editing with the Cre-UCNPs (Supplementary Fig._), in contrast to the pattern observed with AAVs. To evaluate the impact of the selective expression of ChR2 in the VTA region, we monitored mouse behavior after blue light-mediated depolarizations of the VTA using a conditioned place preference (CPP) behavioral test. This conditioning test is widely used to determine the effects that specific objects or experiences exert on reward or aversion behaviors in laboratory animals. In our setup, a standard home cage was modified with two areas that might elicit contrasting visual (black vs. white) and tactile (sandpaper vs. grid) reactions (Fig._). The first part of the trial (Pretest) consisted of 3 days of habituation to determine endogenous preferences for a zone in the arena. Next, on the fourth day (Conditioning), we used a closed-loop stimulation interface to optogenetically activate the VTA whenever the animal crossed into the unpreferred zone. A final probe trial (Test) was performed on the fifth day to assess any preference changes. Our results showed that the animals exhibited changed preferences after optical stimulation compared to that exhibited by the control animals, confirming an optogenetic induced reward-based conditioning (Fig._). We further validated our hypothesis that VTA dopaminergic neurons were activated following ChR2 stimulation by evaluating c-Fos expression (Fig._). After Cre-mediated recombination, ChR2-expressing mice exhibited more c-Fos-positive cells in the VTA region than control mice (with no ChR2 expression).\" - \"Despite the demonstration of in vivo gene editing and optogenetic actuation in freely moving animals, one of the main limitations of the presented strategy is the requirement of an invasive administration route. Therefore, we investigated the potential of using Cre-UCNPs to perform targeted gene editing through a non-invasive route, specifically intranasal instillation (Supplementary Fig._ ). For this purpose, R26tdTomato mice were subjected to 3 intranasal administrations of Cre-UCNPs on 3 different days (i.e. days 1, 2 and 3). Each day, drops of NP suspensions were administered gradually into both nasal cavities (Supplementary Fig._). The animals were divided into two groups: the Cre-UCNP group received NPs (120u2009xB5g per administration), and the control group received only a saline solution.\" - \"First, we investigated the distribution of the Cre-UCNPs 4u2009h after a single intranasal administration by ICP-MS analyses (Supplementary Fig._). Previous studies have demonstrated that nanomaterials administered through the nasal route can reach the brain. Nose-to-brain transport has been reported for several nanomaterials, including exosomes and iron oxide nanoparticles, among others, and can be primarily described by two mechanisms: intracellular (via transcytosis) and extracellular. The extracellular pathway is the faster route and includes the passage of the NPs by passive transport through tight junctions into the brain parenchyma. The intracellular pathway includes endocytosis of the NPs into olfactory and trigeminal nerve branches followed by axonal transport into brain. Our results show that most of the NPs were in the nasal cavity, followed by the olfactory bulb (Supplementary Fig._). Because Cre-UCNPs reached the cortex and hippocampus within a few hours, it is possible that the NPs reached these parts of the brain by extracellular transport.\" - \"Next, we tested whether Cre-UCNPs induced gene editing with unilateral specificity by exposing only one of the brain hemispheres to NIR light (the other was covered to prevent laser activation; 980u2009nm; 425u2009mW/cm; 3 cycles of 5u2009 min each) at 1u2009h after administration (Supplementary Fig._). Our results showed recombined cells in the irradiated brain hemisphere, particularly in the olfactory bulb and cortex (Supplementary Fig._). The area of tdTomato-positive cells in the olfactory bulb of the irradiated hemisphere was greater than that in the nonirradiated hemisphere (Supplementary Fig._). Overall, our results demonstrated the utility of Cre-UCNPs for on-demand gene editing and the possibility to manipulate edited cellular circuits with optogenetic activation.\"- - We have developed a delivery system for gene-editing enzymes that is capable of efficient endolysosomal escape and on-demand release into deep tissues with high spatial resolution upon exposure to transcranial NIR light. This formulation enabled transient expression of Cre recombinase with 125-fold higher efficiency than commercially available lipofectamine and did not require further protein engineering to facilitate cellular uptake. We have demonstrated the utility of the formulation in vivo with the achievement of gene editing in neurogenic niches of the mouse brain, as well as the activation of mechanisms of reward and reinforcement in the VTA using NIR activation to genetically unlock an optogenetic actuator (ChR2), followed by its activation by blue light. The formulation reported here might be used in the treatment of reward-pathway dysfunctions that are highly relevant in psychiatric (e.g., substance-use, affective, eating and obsessive-compulsive disorders) and neurodevelopmental disorders (e.g. schizophrenia, attention-deficit/hyperactivity disorder and autism spectrum disorders), and genetic syndromes (i.e, fragile X syndrome, Prader-Willi syndrome, Angelman syndrome, and Rett syndrome). In addition, our formulation might be of particular relevance for gene editing in the context of the eye. For example, optogenetic modulation of the retina with the AAV-mediated expression of channelrhodopsin and subsequent visual stimulation for the treatment of retinitis pigmentosa is currently being assessed in a clinical trial (NCT03326336). Moreover, considering the large number of Cre/lox mouse lines available, it is anticipated that the formulation developed in this work might be useful for researchers to map specific neuronal circuits in vivo, as well as to image and track specific cell populations. Finally, UCNPs accumulated in the brain may be used as optogenetic actuators of transcranial NIR light to stimulate deep brain neurons, as has been recently demonstrated. - \"The efficiency of our formulation relied on the incorporation of low amounts of HCQ chemically conjugated to the UCNPs (below 1 u03BCM) that induce endolysosomal escape. Previously, it has been shown that soluble chloroquine is one of the most effective agents to disrupt the endolysosomal compartment. The molecule induced the disruption of endolysosomes that were positive for late endosomal markers including CD62/Rab7 (u223C30%) and Rab9 (u223C23%), and the lysosomal marker LAMP1 (u223C45%). Our results indicated that, in the absence of light activation, Cre-UCNPs with HCQ, but not Cre-UCNPs without HCQ, efficiently escaped the endosomal compartment at levels similar to those of soluble HCQ. To demonstrate this escape, we have used galectin-9 protein as a reporter, since it has been shown to respond rapidly and with high sensitivity to endosomal membrane disruption. This protein is recruited to damaged endosomes during lipid NP or transfection agent-mediated endosomal escape. Our results showed that very small amounts of immobilized HCQ in NPs can be as effective as high concentrations of soluble HCQ (60-fold higher). It is possible that the immobilized HCQ molecules acted synergistically in the endosomal membrane, maximizing its effect. Importantly, we anticipate that the strategy described here to facilitate endosomal escape based on immobilized HCQ can be easily incorporated into other delivery systems for protein or noncoding RNA delivery.\" - \"The results reported here demonstrate our capacity to control the activity of genes with precise spatial control, using a Cre-based NP formulation that can be externally triggered using NIR light. First, the capacity of Cre-UCNPs to induce in vitro cell recombination, with spatial resolution, was demonstrated at single-cell level (Fig._). To the best of our knowledge, this level of control has not been demonstrated in the past using NIR light-triggerable NP formulations. Second, we showed the capacity of Cre-UCNPs to induce in vivo cell recombination, with spatial resolution at the hemisphere level (Supplementary Fig._). In this case, the spatial control in gene editing was controlled by the spot of the light and not defined by the point of injection (as demonstrated in the SVZ and VTA studies). In the near future, it might be possible to align the laser with a stereotaxic equipment to activate a particular region of the brain. For applications requiring deeper penetration of the light (more than 4u2009mm), a craniotomy (a clinical procedure that is used in humans to implant drug delivery systems in the brain) may be required. Third, our results further showed that the spatial control obtained by Cre-UCNPs in the VTA region was higher than that obtained with an AAV5 virus (Supplementary Fig._). After injection into the target area of the VTA, the AAV virus spread throughout the tissue with ChR2 expression appearing at distances of 3.5u2009mm from the injection site, while with the Cre-UCNPs, the expression of ChR2 was 1.5u2009mm from the injection site.\" - Our results demonstrated that the activity of the edited cells can be monitored by optogenetics using blue light as a trigger. Although not tested in the current study, the modulation of brain cell activity may be performed by chemogenetics rather than optogenetics. In chemogenetic applications, cells are transfected with DREADD receptors that are activated by synthetic ligands that do not require invasive administration. In addition, it might be possible to incorporate transcription factor genes into the Cre-LoxP construct, in addition to ChR2, to investigate the impact of these factors in brain cells at the functional level. Importantly, using behavioral tests, we showed that the level of gene editing in the brain was sufficient for a functional output. - Our formulation may represent a combined alternative with reduced safety problems in the rhodopsin delivery strategy and efficient NIR-mediated stimulation. A widely used approach to study gene function in vivo is Cre/loxP recombination through viral delivery of Cre-expressing constructs. However, even though the widespread use of this technique is an indication of its usefulness, several studies have reported nonspecific and potentially noxious effects of Cre recombinase in mice in various organs, including the brain. In addition, a recent study demonstrated that local AAV-mediated Cre expression in the substantia nigra of wild-type mice induced a massive decrease in neuronal populations, greatly impacting the nigrostriatal dopaminergic pathway, culminating in an anatomical and functional phenotype resembling models of dopaminergic degeneration. Another method to achieve Cre/loxP recombination is through Cre-driver mouse lines, where optogenetic tools, delivered via Cre-inducible viral constructs, are expressed only in defined neurons expressing Cre recombinase. However, the use of these mouse lines has drawbacks. For example, nonspecific expression of Cre recombinase may occur outside the target cell or brain region due to promiscuous activity of the driving promoter. In addition, Cre-driver lines can present inherent physiological problems, such as altered metabolic phenotypes, impacting the body length and body weight of the mice. These findings underline the necessity for in-depth scrutiny of Cre toxicity and spatial localization in the brain, and further affirm our formulation as a possible tool to mitigate the side effects of existing techniques. - \"During the preparation of this manuscript, NIR-activatable NPs able to release Cre recombinase with spatiotemporal control were described; however, the formulation showed low gene editing (<20% of cell recombination), and the in vivo activity of the formulation was not demonstrated. Other recent studies have also documented the optical regulation of gene-editing formulations based on NIR. For example, an upconversion-activated CRISPR-Cas9 NP has been reported for the editing of cancer cells transplanted into skin tissue. However, the study did not: (i) show the spatial resolution of the technology in vitro or in vivo (to evaluate the functional impact of their formulation, the authors injected the NPs in the tumor site, irradiated the tissue, and measured the reduction in tumor size), (ii) demonstrate gene editing with spatial resolution using a noninvasive route for the administration of the NPs, and (iii) did not evaluate the cellular activity of the edited cells by optogenetics. Another study has reported a nanoparticle formulation for programmable genome editing in the second near-infrared (NIR-II) optical window. The formulation was composed of a cationic polymer-coated gold nanorod and a Cas9 plasmid driven by the heat-inducible promoter HSP70. The authors have shown the capacity to regulate Cas9 activity by heat-induced gene expression. Although this system may show greater tissue-penetration than NPs that respond to the first near-infrared optical window (such as Cre-UCNPs), it suffered from some limitations such as: (i) it was based in a plasmid and not in a protein (in contrast to the Cre-UCNPs), (ii) the long-term effects of the intracellular fluctuation of the temperature on cell activity remain to be determined, and (iii) the transfection efficiency was_only 3-fold higher than that of lipofectamine, while in Cre-UCNPs the transfection efficiency was more than 100-fold higher than that of a lipofectamine-based transfection agent (RNAiMax). Overall, the formulation reported here opens new avenues for in vivo spatial control of gene expression, which can be complemented with optogenetics to measure the activity of the edited cells.\" - Future studies should be directed to improving the accumulation of Cre-UCNPs in the brain after intranasal instillation to achieve high cell recombination. Additionally, while the Cre enzyme was used in this study as a proof of concept in this study for active protein delivery, we foresee that the designed NP formulation can be useful for the delivery of other gene editing proteins, such as CRISPR-Cas9, as well as transcription factors for cell reprogramming. In the case of Cas9, our formulation might offer specific advantages to correct mutations in SVZ cells since they have been recently associated with the development of glioblastoma.- - Ethical statement - \\'All animal experiments were carried out in line with the ARRIVE guidelines, with prior ethical approval from the Portuguese DGAV, under project license 0421/000/000/2020, and from the ethics committee of the Center for Neuroscience and Cell Biology, University of Coimbra (Ref: ORBEA_262_2020/25052020), in accordance with Portuguese Decree-Law no. 113/2013 and the European Directive 2010/63/EU regarding animal use in research.\\' - \\'Preparation of UCNPs: synthesis of the nanocrystal\\' - \"The UCNPs (NaYF4:Tm 0.5u2009mol%, Yb 30u2009mol%) were synthesized according to a protocol published elsewhere with minor modifications. The Yb acts as the sensitizer, while the Tm acts as the activator ion. Tm ions were selected as an activator in order to enable a blue to UV-shifted emission, while Yb ions was selected because it has only two multiplets, which energy gap between the ground state and excited state offers efficient absorption in the range of 900u2009 u2212u20091000u2009nm. For the synthesis of nanocrystals, thulium(III) acetate hydrate (Sigma-Aldrich, 1.7u2009mg, 0.015u2009mmol), ytterbium(III) acetate hydrate (Sigma-Aldrich, 38.1u2009mg, 0.9u2009mmol), and yttrium(III) acetate hydrate (Sigma-Aldrich, 55.5u2009mg, 2.1u2009mmol) were dissolved in a mixture of 1-octadecene (45u2009mL, Acros Organics) and oleic acid (Alfa Aesar, 18u2009 mL). The resulting solution was slowly heated to 120u2009xB0C under vacuum with magnetic stirring for 30u2009min to remove residual acetic acid, water and oxygen during which time the flask was purged periodically with nitrogen gas. The resulting clear solution had a slight yellow colour indicating the formation of lanthanide oleate complexes. The temperature was then lowered to 50u2009 xB0C and the reaction flask placed under a gentle flow of nitrogen gas. A solution of ammonium fluoride (12u2009mmol) and sodium hydroxide (7.5u2009mmol) dissolved in methanol (30u2009mL) was prepared via sonication. This methanol solution was added to the previous reaction flask and the resulting cloudy mixture was stirred for 30u2009min at 50u2009xB0C. The reaction temperature was then increased to 70u2009xB0C and the methanol distilled off from the reaction mixture. Subsequently, the reaction temperature was increased to 300u2009xB0 C (in a period of 2-3u2009min) and maintained at this temperature for 90u2009 min under the nitrogen gas flow. During this time, the reaction mixture became progressively clearer, having a yellowish-brown colour. The mixture was allowed to cool to room temperature and the UCNPs were precipitated by the addition of acetone and isolated via centrifugation (3000u2009rpm, 1157u2009g, 6u2009 min). The resulting pellet was dispersed in a minimal amount of cyclohexane and precipitated with excess anhydrous ethanol. The UCNPs were isolated via centrifugation (3000u2009rpm, 1157u2009g, 6u2009min) and then dispersed in cyclohexane (30u2009 mL). Next, a NaYF4 shell was grown in the previous UCNPs. The dopant was yttrium within a sodium fluoride (NaF4) matrix. The sodium fluoride matrix was chosen since it possesses a close lattice match to the dopant ions and has low phonon energy. The incorporation of the NaYF4 shell is vital since it improves the intensity of the core UCNPs by minimizing the effect of the quenching due to interactions of dopant ions with the solvent. In brief, 71.8u2009mg (2.7u2009 mmol) of yttrium(III) acetate hydrate was dissolved in a mixture of 1-octadecene (45u2009mL, Acros Organics) and oleic acid (18u2009mL). The temperature was increased to 120u2009xB0C for 30u2009min. Next, the temperature was decreased to 80u2009xB0C to introduce the NaYF4:Tm, Yb seeds in cyclohexane. The temperature was increased to 110u2009xB0C to remove the cyclohexane solvent. The temperature was decreased to 50u2009xB0C for the addition of NaF4 matrix (a solution of ammonium fluoride (10.5u2009mmol) and sodium hydroxide (6.6u2009mmol) dissolved in methanol (30u2009mL) was prepared via sonication) for 30u2009min. The temperature was increased to 70u2009xB0C to remove the methanol and then increased to 300u2009xB0C for 90u2009min. The nanocrystals were precipitated by the addition of acetone and isolated via centrifugation. The resulting pellet was dispersed in a minimal amount of cyclohexane and precipitated with excess anhydrous ethanol. The nanocrystals were isolated via centrifugation and then dispersed in cyclohexane (20u2009mL).\" - \\'Preparation of UCNPs: reaction of UCNPs with organosilanes containing terminal N3 and NH2 groups\\' - \"Initially, ligand exchange was performed to replace the hydrophobic oleic acid on UCNP surface by citric acid to obtain core-shell UCNPs that can be dispersed in water. The core-shell UCNPs coated with oleic acid (580u2009mg) were dispersed in a mixture of 1,2-dichlorobenzene and N, N-dimethylformamide (DMF) (80u2009 mL, 1:1, v/v, Sigma-Aldrich). The mixture was stirred at 100u02DAC for approximately 24u2009h after the addition of citric acid (0.5u2009g, Sigma-Aldrich). The hydrophilic particles were precipitated by the addition of ethyl ether. The UCNPs were washed by dispersion in acetone and centrifugation three times (4000u2009 rpm, 2057u2009g, 6u2009min). Finally, the UCNPs were dispersed in water (30u2009 mL). The suspension of core-shell UCNPs (30u2009mg, 200u2009xB5L) was added to methanol (15u2009mL), followed by the addition of a solution of APTES (6u2009 xB5L) and N3PTES (12u2009xB5L, Supplementary Fig._) in methanol (15u2009 mL) and stirred for 12u2009h at room temperature. The NP suspension was then added to glycerol (30u2009mL) to prevent the aggregation of the NPs during the evaporation of methanol and water (by rotary evaporator respectively, at 40 and 80u02DAC) and the dispersion was condensed in vacuum at 100 u2212110u02DA C for 2u2009h. Flocculated APTES/N3PTES-modified UCNPs were when washed with water/acetone (3:7, v/v, 100u2009mL, three times), being in each separated from the solvent by centrifugation (6000u2009rpm, 4629u2009g, 6u2009min). Finally, the modified UCNPs (N3/NH2-UCNPs) were suspended in water (1.5u2009 mL).\" - \\'Preparation of UCNPs: modification of N3/NH2-UCNPs with hydroxychloroquine\\' - \"Hydroxychloroquine sulfate (27u2009mg, 62.5 xB5mol), was suspended in a 1,1u201D -carbonyldiimidazole solution (CDI, 67u2009mg/mL in THF; 750 u03BCL) and the slurry taken to 40u02DAC under argon atmosphere and gentle stirring. After 5u2009h, the solid fraction was collected, washed with THF to remove excess of CDI, and then transferred to a new flask to which was added N3/NH2-UCNPs (1.25u2009mL of 14.5u2009mg/mL of NPs in THF). The slurry was gently stirred at room temperature for 24u2009h. The modified HCQ-N3/NH2-UCNPs were isolated via centrifugation and washed three times in water and finally suspended in 1u2009mL of water.\" - \\'Preparation of UCNPs: modification of HCQ-N3/NH2-UCNPs with PCL\\' - \"Initially, a solution of copper (II) sulphate pentahydrate (5.8u2009xB5L, 25u2009 mM) was reduced to copper (I) by addition of sodium ascorbate (29u2009xB5L, 20u2009mM). This reduced copper (I) solution was added to a suspension of HCQ-N3/NH2-UCNPs (12u2009mg) in water/isopropanol (1.25u2009mL, 3:2, v/v) mixed with a solution of PCL (0.2u2009mg, 0.58 xB5mol, dissolved in 250u2009xB5L of isopropanol, Supplementary Figs._ and ), and gently stirred for 1u2009h, in order to react the azide groups of HCQ-N3/NH2-UCNPs (Supplementary Fig._) with the alkyne group of PCL (Supplementary Fig._) via azide-alkyne cycloaddition. The PCL-HCQ-N3/NH2-UCNPs were isolated via centrifugation (6.000u2009rpm, 4629u2009g, 6u2009min) and washed three times in water and finally dispersed in water (500u2009xB5L).\" - \\'Preparation of UCNPs: loading of Cre onto the surface of the PCL-HCQ-N3/NH2-UCNPs\\' - \"PCL-HCQ-N3/NH2-UCNPs (1u2009mg) were dispersed in 100u2009mM MES buffer pH 6.4 (660u2009xB5L). A solution of N-hydroxysulfosuccinimide sodium salt (sulfo-NHS) and N-(3-dimethylaminopropyl)-N-ethylcarbodiimide hydrochloride (EDC) was added (100u2009xB5L, 2.7u2009mM and 2.24u2009mM, respectively) and gently stirred for 15u2009min on an orbital shaker at the speed of 250u2009rpm. Cre recombinase (108u2009xB5g, 1.84 nmol) was gently added to the suspension and left to react for 20u2009h at 4u02DAC. The Cre-PCL-HCQ-N3/NH2-UCNPs (from now on designated as Cre-UCNPs) were isolated via centrifugation (6.000u2009 rpm, 3824u2009g, 6u2009min) and washed three times in 50u2009mM Tris-buffer pH 7.5 and finally dispersed in 50u2009mM Tris-buffer pH 7.5 (100u2009xB5 L).\" - Cell culture - \"SC-1 mouse fetal embryo fibroblasts and the corresponding Cre reporter cell line transduced with the reporter allele SFr-II (from now on termed as Cre reporter fibroblasts) (kindly donated by Dr. Carol Stocking, University of Hamburg, Germany) were grown in DMEM supplemented with fetal bovine serum (FBS, 10%) and PenStrep (0.5%), at 37 u02DAC in a fully humidified air containing 5% CO2. Both cells were passaged after reaching 80% confluency. The cell line has an expression cassette having the RFP gene in-frame to the ATG initiation codon with its adjacent loxP sequence (Fig._). Translation of RFP is terminated at a UAG stop codon followed by a second loxP sequence. The ORF of a humanized eGFP, lacking an initiation codon, is downstream of the distal loxP site. Following Cre-mediated excision of RFP, eGFP codons are placed in-frame of the proximal ATG-LoxP sequence and are consequently translated.\" - In vitro Cre-UCNP-mediated DNA recombination - \"Cre reporter fibroblasts (5xD710 cells/well) were seeded onto a 96-well plate and left to adhere overnight. Cells were incubated with Cre-UCNPs (50u2009u03BC g/mL) for 4u2009h, washed with PBS, and left to grow in complete medium for another 48u2009h. Cre-mediated DNA recombination was followed over time for a period of 48u2009h by direct analysis of GFP expression using the IN Cell Analyzer 2200 cell imaging system (GE Healthcare Life Sciences) equipped with an inverted 20x PlanFluor objective/0.45 numerical aperture. GFP expression was detected using an excitation filter 475/28 and an emission filter 511.5/23. The mean green intensity in the cell cytoplasm was quantified using the IN Cell Developer software. Flow cytometry was used to quantify the percentage of GFP-positive cells. For this purpose, cells after 40u2009h of Cre-UCNP incubation were dissociated with trypsin (0.1%, w/v, in PBS), centrifuged, resuspended in PBS and finally characterized by flow cytometry.\" - NP cellular uptake by inductive coupled plasma mass spectrometry (ICP-MS) analyses - \"Cre reporter fibroblasts (5xD710 cells/well) were plated in 6 well plates and left to adhere overnight. Then the cells were incubated with Cre-UCNPs (25 or 50u2009xB5g/mL) between 2 and 8u2009h, washed with PBS to remove noninternalized NPs, dissociated with trypsin (0.1%, w/v, in PBS), centrifuged and counted. The samples were freeze-dried, followed by the digestion with an aqueous nitric acid solution (9.9u2009mL, 2% w/v).\" - \"SVZ cells (see section Animal testing: isolation of SVZ cells described in_ ) were seeded in 24-well plates at a density of 10 cells/well, whereas cortical neurons (see section Preparation of neuronal cultures described in_) were seeded in 12-well plates at a density of 1.8xD710 cells/well. Cells were incubated with Cre-UCNPs (100, 200 and 300u2009u03BCg/mL) for 4u2009h. After transfections, cells were washed with PBS to remove non-internalized NPs, dissociated with trypsin (0.1%, w/v, in PBS), centrifuged and counted. The samples were then freeze-dried, followed by the digestion with an aqueous nitric acid solution (9.9u2009mL, 2% w/v), and analyzed by ICP-MS for the quantification of internalized yttrium. The NP cellular uptake was also monitored in brain mice. Animals were subjected to a stereotaxic procedure for injection of Cre-UCNPs (30u2009u03BC g) in the VTA (detailed protocol in section Animal testing: VTA administration). Immediately after injection, animals were perfused with PBS. The brains were excised, weighed and processed as before for ICP-MS analyses.\" - NP cellular uptake and intracellular trafficking by confocal microscopy - \"Intracellular trafficking of Cre-UCNPs was assessed on Cre-reported fibroblasts. Cells (4xD710 cells/well) were treated with Cre-UCNPs (50u2009u03BCg/mL) with or without hydroxychloroquine for different time points: 5u2009min, 1u2009 h, 4u2009h or 24u2009h. After incubation, cells were washed extensively with PBS with the exception of one experimental group that was washed with culture medium and maintained for additional 24u2009h prior to fixation. Cells were fixed with 4% (v/v) paraformaldehyde for 15u2009min at room temperature and washed three times with PBS. Cells were permeabilized with 0.3% (w/v) Triton X-100. The detergent was removed by washing three times with PBS. Before staining, unspecific binding sites were blocked with 1% (w/v) BSA and 0.3u2009M glycine for 1u2009h. Cells were then stained for 1u2009h with primary antibodies (Supplementary Table_): anti-Cre recombinase for the detection of Cre-UCNPs and their cargo (1:3000 MAB3120, Merck Milllipore), anti-EEA1 as a marker of early endosomes (1:100 clone: C45B10, Cell Signaling) and anti-LAMP1 (1:1000 ab24170, Abcam) to stain lysosomes. Primary antibodies were detected with an anti-mouse IgG Alexa Fluor 488 (Life Technologies) or an anti-rabbit IgG Cy3 conjugate (Jackson ImmunoResearch). Confocal fluorescent images were acquired using a Laser scanning confocal microscope (LSM 710, Zeiss), equipped with a 40x objective/1.4 numerical aperture oil PlanApochromat immersion lens. Alexa Fluor 488 was detected using the 488u2009nm laser line of an Ar laser (25u2009mW nominal output) and an LP 505 filter. The Cy3 signal was detected using a 561u2009nm HeNe laser (1u2009 mW) and an LP 560 filter. The pinhole aperture was set to 1 Airy unit. Image acquisition was performed using the Zen Black 2012 software. Images were analyzed in ImageJ, and the colocalization was determined by calculating the Manders colocalization coefficient between Cre-UCNPs and EEA1 or LAMP-1.\" - \"For studies with galectin-9, Cre-reported fibroblasts (2xD710 cells/well in IBIDI xB5-Slide 8 Well) were treated with Cre-UCNPs (25u2009u03BCg/mL) with or without hydroxychloroquine for 4u2009h, then the cells were washed with culture medium and maintained for additional 20u2009h prior to fixation. As a positive control, cells were treated with soluble hydroxychloroquine (60u2009 u03BCM, 26u2009u03BCg/mL) for 16u2009h. Cells were fixed with 4% (v/v) paraformaldehyde for 15u2009min at room temperature and washed three times with PBS. Cells were permeabilized with 0.3% (w/v) Triton X-100 for 15u2009min, and finally washed three times with PBS. Before staining, unspecific binding sites were blocked with 1% (w/v) BSA and 0.3u2009M glycine for 1u2009h. Cells were then incubated for 1u2009h with primary antibody rat anti-galectin 9 (1:100 cat. no. 137901, BioLegend). The antibody was removed by washing three times with PBS. The secondary antibody anti-rat IgG Alexa Fluor 488 (Life Technologies) were incubated by 1u2009h and the nuclei stained with DAPI (Sigma) for 5u2009min at RT. The galectin-9 foci were imaged using a Zeiss LSM 710 confocal microscope equipped with a Apochromat 40x/1.4 objective. The foci number and area were assessed by the Analyze Particles built-in function in FIJI.\" - \"For the studies with Lysotracker, cells (4xD710 cells/well) were first incubated with Lysotracker Red (Life Technologies; 50u2009nM) for 30u2009min prior to Cre-UCNPs treatment. NPs with or without hydroxychloroquine (50u2009xB5g/mL) were exposed to cells for different time-points: 5u2009min, 10u2009min, 15u2009 min and 1u2009h. After incubation, cells were washed extensively with PBS and paraformaldehyde (4% (v/v)). Cell membrane was stained with WGA (Invitrogen), nuclei stained with DAPI (Sigma), and the slides were mounted with mounting medium (Dako). Co-localization was examined using a Zeiss LSM 710 confocal microscope equipped with a Apochromat 40x/1.4 objective. The acquired images were analyzed by classical co-localization tools in FIJI. Vesicle number and area was assessed by the Analyze Particles built-in function in FIJI.\" - NP cellular uptake pathways - \"Cre reporter fibroblasts (4xD710 cells/well) cells were seeded on a 24 well plate and left to adhere overnight. Cells were inhibited by one of the following chemicals during 30u2009min before adding a suspension of Dylight 488-labelled Cre-UCNPs: EIPA (100u2009u03BCM), dynasor (5u2009u03BCM), dansylcadaverine (10u2009u03BCM), cytochalasin D (1u2009u03BCM), nocodazole (3u2009u03BC M), filipin III (100u2009u03BCM) and polyinosinic acid (100u2009u03BCg/mL). After 30u2009min, cells were incubated with Dylight 488-labelled Cre-UCNPs (25u2009u03BCg/mL) for 4u2009h in the presence of the inhibitor. After 4u2009 h, cells were washed one time with cold trypan blue solution, re-washed three times with cold PBS, dissociated with trypsin (0.1%, w/v, in PBS), centrifuged and resuspended in PBS for flow cytometry analyses. To validate the inhibitory activity of dynasor we performed uptake studies of FITC-labelled transferrin, known to selectively enter cells via clatherin-mediated endocytosis. Fibroblasts (4xD710 cells/well) were seeded on a 24 well plate and treated or not with dynasor (30u2009min pre-incubation), followed by addition of FITC-labelled transferrin (1u2009u03BCg/mL, Life Technologies). After 8u2009min, cells were evaluated as before by flow cytometry.\" - \"The NP uptake mechanism was evaluated on fibroblasts by silencing key regulators of CME (CLTC and LDLR), caveolin-mediated endocytosis (CAV1), GEEC-CCLIC pathways (CDC42), macropinocytosis (RAC1 and CTBP1) and scavenger receptors (SCARA3) with siRNAs (Thermo Fisher). Silencing was performed on 24-well plates with 4xD710 cells/well and confirmed by RT-qPCR. Total RNA was extracted using TRIzol (Invitrogen) and purified using the RNeasy Mini kit (Qiagen). Total RNA concentration and purity were analyzed using the NanoDrop spectrophotometer for quality control. First-strand cDNA was generated from 1u2009u03BCg of RNA using the High Capacity cDNA Reverse Transcription kit (Applied Biosystems). The cDNA synthesis reaction followed a protocol of 10u2009min at 25u2009xB0C, followed by 2u2009h at 37u2009xB0C, and 5u2009min at 85u2009xB0C, executed by a CFX96 real-time PCR system (BioRad). Quantitative PCR was performed in the CFX96 RT-PCR system using the Power SYBR Green PCR Master Mix (Applied Biosystems). The reaction protocol started with an activation step at 50u2009xB0C for 2u2009min, followed by 2u2009min at 95u2009xB0C, and 40 cycles of amplification (denaturation at 95u2009xB0C for 15u2009s, then annealing and elongation at 60u2009xB0 C for 1u2009min). Minimal cycle threshold (Ct) values for each gene were calculated from 3 technical replicates. Quantification of target genes (Supplementary Table_ ) was performed using the Livak method (2), comparing the u0394Ct values obtained between each gene and the reference GAPDH gene to the negative control (No inhibitor).\" - \"Cells in antibiotic-free complete medium were transfected for 24u2009h with lipofectamine RNAiMAX (1.5u2009u03BCL, Life Technologies) complexed with siRNA (100u2009nM). Next, cells were incubated with Dylight 488-labelled Cre-UCNPs (25u2009u03BCg/mL) for 4u2009h, washed, with cold trypan blue solution, re-washed three times with cold PBS, dissociated with trypsin (0.1%, w/v, in PBS), centrifuged and resuspended in PBS for flow cytometry analyses. Non-transfected cells or cells transfected with lipofectamine complexed with scramble siRNA (MOCK) were used as controls.\" - Animals - \"R26YFP mice, in a C57BL/6u2009J background, were obtained as kind gift from Dr. Henrique Veiga-Fernandes (IMM, Lisbon). R26YFP mice were bred and maintained at Faculty of Medicine at the University of Coimbra animal facility. Male C57/BL6 and R26tdTomato mice were purchased from Charles River. All animals were housed at the animal house of the Faculty of Medicine at the University of Coimbra, in groups of 3 to 4 per cage, with food and water provided ad libitum and maintained in a 12u2009h light/dark cycle in temperature- and humidity-controlled rooms. For optogenetic studies, mice were single isolated after surgically intervened. When arriving, animals were acclimatized to the animal house and daily routines for at least one week after which they were handled twice a week. When needed, animal identification was performed by subcutaneous injection of green and/or black dyes in the paws. No randomization was used to allocate animals to experimental groups. All the animals used in the study were male with 7 to 20 weeks of age and were sacrificed by cervical dislocation at the end of each experiment.\" - \\'Animal testing: SVZ administration\\' - \"R26YFP mice (7 to 20-week-old) were anesthetized with isoflurane (1.5u20132.0%), carprofen (1u2009mg/kg) via intraperitoneal injection and buprenorphine (0.1u2009 mg/kg) via subcutaneous injection. Animals were placed on a heating pad maintained at 37u2009xB0C and allowed to reach a stable plane of anesthesia, which was periodically checked through examination of breathing rate and a toe-pinch test. The anesthetized mouse was then aligned on a Digital New Standard Stereotaxic frame (Stoelting), the skull was shaved, disinfected and craniotomies were performed with minimal damage to brain tissue. Mouse stereotaxic surgery target: sub ventricular zone (0.74u2009mm anterioposterior, +/u2212 0.75u2009mm mediolateral, u2212 3, u22122.7, u22122.3 and u22122 mm dorsoventral). All stereotaxic coordinates were relative to bregma. Cre-UCNPs (30u2009xB5g, 690 nL) were administered using a Nanoliter 2010 (World Precision Instruments).\" - \\'Animal testing: NIR activation of UCNPs\\' - \"Approximately 60u2009min after administrations, the injected area (above the craniotomy site) was irradiated with a NIR laser (980u2009nm; 425u2009mW/cm; 3 cycles of 5u2009min).\" - \\'Animal testing: preparation of implantable optical ferrules\\' - \"Custom-made implantable optical ferrules were constructed from 2.5u2009mm long 270 u03BCm xD8 ceramic stick ferrules (Thorlabs) based on a previously described design. Briefly, a 200 u03BCm xD8 0.39 NA optical fiber (Thorlabs) was stripped of its protective coating and cleaved. Next, cyanoacrylate adhesive (Loctite) was applied to the concave end of the ferrule through which the cleaved fiber segment was threaded. After wiping off the excess glue, the ferrules were allowed to cure for 30u2009min at room temperature. The ferrule connector was polished using a polishing disk and increasingly fine grades of polishing paper (Thorlabs), with frequent inspection to ensure transmission quality. Once polished, the free end of the fiber was scored and cleaved to 5u2009mm in length.\" - \\'Animal testing: optical fiber implant\\' - \"For conditioned place preference experiments, mice were then implanted with implantable optical fibers, who were attached to the stereotaxic arm and tilted 10xB0 outwards in the rostro-caudal axis. Two anchor stainless steel, sterile screws (M1.2) were implanted until contact with the dura and further fixated with VersaGlue PX-40 (Harvest Dental). Mouse stereotaxic surgery targets: ventral tegmental area (u2212 3.30u2009mm anterioposterior, + 0.50u2009mm mediolateral, u2212 3.8u2009mm dorsoventral). The craniotomy window was kept well hydrated with 0.9% (w/v) NaCl throughout all the implant procedure. Dental cement (3u2009M) or Maxcem Elite (Kerr) was used to secure the optical implants in place. After verification that the implant is completely secured, a dental acrylic mixture was applied (Scheu Dental) to further secure the implant.\" - \\'Animal testing: VTA administration\\' - \"AAV5.CMV.PI.Cre.rBG (AAV5.Cre) and AAV5.EF1a.DIO.hChR2(H134R)-eYFP.WPRE.hGH (AAV5.DIO.ChR2.YFP) were acquired from Penn Vector Core. Male C57/BL6 mice (8 to 20-week-old) were anesthetized with isoflurane (1.5u20132.0%), carprofen (1u2009mg/kg) via intraperitoneal injection, and buprenorphine (0.1u2009mg/kg) via subcutaneous injection. Animals were placed on a heating pad maintained at 37u2009xB0C and allowed to reach a stable plane of anesthesia, which was periodically checked through examination of breathing rate and a toe-pinch test. Their eyes were covered with Altalube ointment (Altaire Pharmaceuticals) to prevent corneal drying. The anesthetized mouse was then aligned on a Digital New Standard Stereotaxic frame (Stoelting) and craniotomies were performed with minimal damage to brain tissue. All stereotaxic coordinates were relative to bregma.\" - \"For viral infusion, animals were injected with AAV5.DIO.ChR2.YFP alone (negative control) or AAV5.DIO.ChR2.YFPu2009+u2009Cre-UCNPs (30u2009xB5g, 690 nL) (test group) at the VTA (u22123.3u2009mm anterioposterior, +0.5u2009mm mediolateral, u22124.1, u22124.0 and u22123.9u2009mm dorsoventral). Animals were infused with a Nanoliter 2010 (total volume of 1.4u2009xB5L) tilted 10xB0 outwards in the rostro-caudal axis. After infusion, the injector was left at the injection site for 10u2009min and then slowly withdrawn.\" - \\'Animal testing: behavioral analyses\\' - \"The custom-made conditioned place preference apparatus consisted of a rectangular cage with a left white chamber (17u2009cm xD7 12.5u2009cm) with a mesh grid floor, and a right black chamber (17u2009cm xD7 12.5u2009cm) with a sandstone floor grid. All behavioral video data was acquired with a Logitech C922 Pro Webcam and coded in Bonsai, a free open-source visual programming environment (). The conditioned place preference test consisted of 5 days. Days 1 to 3 consisted of a preconditioning test that measured the time mice spent in each chamber to determine their preference (1 trial/day, 15u2009min/trial). On day 4, after 3 consecutive days with mice preferring one chamber instead of the other, the mice were allowed to freely move around the cage and were stimulated in the non-preferred side of the cage (trial duration: 30u2009min). On day 5, the mice were placed back into the chamber to assess preference for the stimulation and non-stimulation chambers (trial duration: 15u2009min).\" - \\'Animal testing: light stimulation for in vivo assays\\' - \"Optic fibers were attached through an FC/PC adaptor to a 465u2009nm LED module (Plexon), and light pulses were generated through a stimulator (Plexon). For all in vivo experimental protocols, 10u2009Hz blue light pulses over 3u2009 min were delivered (S9). Optic fiber light intensity was measured at the tip using a light sensor (Thorlabs) and ranged from 5-6u2009mW.\" - \\'Animal testing: light stimulation for c-Fos induction\\' - \"Mice were anesthetized, 90u2009min after blue light stimulation, with chloral hydrate then perfused with 1xD7 PBS followed by 4% (v/v) paraformaldehyde.\" - \\'Animal testing: intranasal administration\\' - \"R26tdTomato mice were subjected to intranasal administrations of the Cre-UCNPs (1u2009mg/mL) over the time of three days. The animals were anesthetized with 2.5% isoflurane, mice in the supine position were intranasally administered using a micropipette. The animals were administered with 40u2009xB5L of Cre-UCNPs equally divided for each nostril in each day. One hour after administration, the mice were anesthetized with isoflurane and one of the brain hemispheres was covered with aluminum foil before irradiating the other half with NIR light for 3 cycles of 5u2009min.\" - \\'Animal testing: immunohistochemistry\\' - \"Tissues were harvested and fixed overnight (or post-fixed for perfused samples) with 4% (v/v) paraformaldehyde then cryo-protected in sucrose [30% (w/v) in PBS]. Gastrocnemius muscles from the hind limb of mice were imbedded in OCT compound, snap-frozen and sectioned on a Cryostast (Leica) into 10u2009xB5 m thick sections. Brains were sectioned (coronal plane sections) on a Cryostat (Leica) or on Vibrotome (Leica) into 50-u03BCm thick sections. For Cryostat sectioning brains were snap-frozen before cutting. Sections were first treated with blocking solution (0.3% Triton X-100, 10% goat serum in PBS) and incubated with the primary antibody (Supplementary Table_) in blocking solution for 8u2009 h to 48u2009h at 4u2009xB0C. Sections were washed with 1xD7 PBS and incubated in the secondary antibody at room temperature for 1u2009h to 24u2009h. Finally, sections were washed three times in 1xD7 PBS, stained with the DNA binding fluorescence probe DAPI (1u2009u03BCg/ml) and mounted on glass slides in a Prolong gold anti-fade medium (Thermo Fischer). Tile-scan pictures of the all section were obtained in the IN Cell Analyzer 2200 Cell Imaging System using 10x PlanApo objective/0.45 numerical aperture. A stitching program was done using FIJI software. Confocal fluorescent images were acquired using Laser scanning confocal microscope (LSM 710, Zeiss using an oil immersion Apochromat 40x/1.4 objective). Image analysis and quantification were done using FIJI software.\" - Statistics & reproducibility - \"Statistical analyses were performed with GraphPad Prism software (version 8). Statistical significance was analyzed using two-tailed unpaired Student s t test between two different groups. For multiple comparisons, a one-way ANOVA was performed followed by the appropriate multiple comparisons test, specified in the respective figure legend. Results were considered significant when pu2009 <u20090.05. Data are shown as meanu2009xB1u2009SEM unless other specification. No statistical method was used to predetermine sample size.\" - Additional methods - \"Further experimental details are available as_ in the_.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - \"The authors declare that the data supporting the findings of this study are available within the paper and its_. All the figures have associated source data, which are provided with this paper._ are provided with this paper.\" - \\' \\'- - \"Zhang, W. et al. Optogenetic control with a photocleavable protein, PhoCl. Nat. Methods 14, 391u2013394 (2017)._ _ _ _ Kawano, F., Okazaki, R., Yazawa, M. & Sato, M. A photoactivatable Creu2013 loxP recombination system for optogenetic genome engineering. Nat. Chem. Biol. 12, 1059 (2016)._ _ _ _ Kennedy, M. J. et al. Rapid blue-light-mediated induction of protein interactions in living cells. Nat. Methods 7, 973u2013975 (2010)._ _ _ _ _ Jain, P. K. et al. Development of Light-Activated CRISPR Using Guide RNAs with Photocleavable Protectors. Angew. Chem. Int Ed. Engl. 55, 12440u2013 12444 (2016)._ _ _ _ _ Polstein, L. R. & Gersbach, C. A. A light-inducible CRISPR-Cas9 system for control of endogenous gene activation. Nat. Chem. Biol. 11, 198u2013200 (2015)._ _ _ _ _ Nihongaki, Y., Kawano, F., Nakajima, T. & Sato, M. Photoactivatable CRISPR-Cas9 for optogenetic genome editing. Nat. Biotechnol. 33, 755u2013760 (2015)._ _ _ _ Conniot, J., Talebian, S., Simoes, S., Ferreira, L. & Conde, J. Revisiting gene delivery to the brain: silencing and editing. Biomater. Sci. 9, 1065u20131087 (2021)._ _ _ _ Taslimi, A. et al. Optimized second-generation CRY2-CIB dimerizers and photoactivatable Cre recombinase. Nat. Chem. Biol. 12, 425u2013430 (2016)._ _ _ _ _ Jung, H. et al. Noninvasive optical activation of Flp recombinase for genetic manipulation in deep mouse brain regions. Nat. Commun. 10, 314 (2019)._ _ _ _ _ _ Schindler, S. E. et al. Photo-activatable Cre recombinase regulates gene expression in vivo. Sci. Rep. 5, 13627 (2015)._ _ _ _ _ Edwards, W. F., Young, D. D. & Deiters, A. Light-Activated Cre Recombinase as a Tool for the Spatial and Temporal Control of Gene Function in Mammalian Cells. Acs Chem. Biol. 4, 441u2013 445 (2009)._ _ _ _ Morales, D. P. et al. Light-Triggered Genome Editing: Cre Recombinase Mediated Gene Editing with Near-Infrared Light. Small 14, e1800543 (2018)._ _ _ _ Idris, N. M. et al. In vivo photodynamic therapy using upconversion nanoparticles as remote-controlled nanotransducers. Nat. Med. 18, 1580 (2012)._ _ _ _ Jayakumar, M. K. G., Idris, N. M. & Zhang, Y. Remote activation of biomolecules in deep tissues using near-infrared-to-UV upconversion nanotransducers. Proc. Natl Acad. Sci. 109, 8483u20138488 (2012)._ _ _ _ _ _ Li, J. et al. Near-infrared light-triggered release of small molecules for controlled differentiation and long-term tracking of stem cells in vivo using upconversion nanoparticles. Biomaterials 110, 1u201310 (2016)._ _ _ _ _ Chen, S. et al. Near-infrared deep brain stimulation via upconversion nanoparticle-mediated optogenetics. Science 359, 679u2013684 (2018)._ _ _ _ _ Lin, X. et al. Core-Shell-Shell Upconversion Nanoparticles with Enhanced Emission for Wireless Optogenetic Inhibition. Nano Lett. 18, 948u2013 956 (2018)._ _ _ _ _ Atasoy, D., Aponte, Y., Su, H. H. & Sternson, S. M. A FLEX Switch Targets Channelrhodopsin-2 to Multiple Cell Types for Imaging and Long-Range Circuit Mapping. J. Neurosci. 28, 7025u20137030 (2008)._ _ _ _ _ Qian, H. S. & Zhang, Y. Synthesis of hexagonal-phase core-shell NaYF4 nanocrystals with tunable upconversion fluorescence. Langmuir 24, 12123u201312125 (2008)._ _ _ _ Sedlmeier, A., Achatz, D. E., Fischer, L. H., Gorris, H. H. & Wolfbeis, O. S. Photon upconverting nanoparticles for luminescent sensing of temperature. Nanoscale 4, 7090u20137096 (2012)._ _ _ _ _ Perche, F. et al. Hydroxychloroquine-conjugated gold nanoparticles for improved siRNA activity. Biomaterials 90, 62u201371 (2016)._ _ _ _ Mauthe, M. et al. Chloroquine inhibits autophagic flux by decreasing autophagosome-lysosome fusion. Autophagy 14, 1435u20131455 (2018)._ _ _ _ _ Paulo, C. S. O., Lino, M. M., Matos, A. A. & Ferreira, L. S. Differential internalization of amphotericin B u2013 Conjugated nanoparticles in human cells and the expression of heat shock protein 70. Biomaterials 34, 5281u20135293 (2013)._ _ _ _ Bewersdorf, J., Bennett, B. T. & Knight, K. L. H2AX chromatin structures and their response to DNA damage revealed by 4Pi microscopy. Proc. Natl Acad. Sci. 103, 18137u201318142 (2006)._ _ _ _ _ _ Du Rietz, H., Hedlund, H., Wilhelmson, S., Nordenfelt, P. & Wittrup, A. Imaging small molecule-induced endosomal escape of siRNA. Nat. Commun. 11, 1809 (2020)._ _ _ _ _ _ Zuris, J. A. et al. Cationic lipid-mediated delivery of proteins enables efficient protein-based genome editing in vitro and in vivo. Nat. Biotechnol. 33, 73u201380 (2015)._ _ _ _ Wang, M. et al. Efficient delivery of genome-editing proteins using bioreducible lipid nanoparticles. Proc. Natl Acad. Sci. USA 113, 2868u2013 2873 (2016)._ _ _ _ _ _ D Astolfo, D. S. et al. Efficient intracellular delivery of native proteins. Cell 161, 674u2013690 (2015)._ _ _ _ Erazo-Oliveras, A. et al. Protein delivery into live cells by incubation with an endosomolytic agent. Nat. Methods 11, 861u2013867 (2014)._ _ _ _ _ Rees, H. A. et al. Improving the DNA specificity and applicability of base editing through protein engineering and protein delivery. Nat. Commun. 8, 15790 (2017)._ _ _ _ _ _ Lois, C. & Alvarez-Buylla, A. Proliferating subventricular zone cells in the adult mammalian forebrain can differentiate into neurons and glia. Proc. Natl Acad. Sci. USA 90, 2074u20132077 (1993)._ _ _ _ _ _ Luskin, M. B. Restricted proliferation and migration of postnatally generated neurons derived from the forebrain subventricular zone. Neuron 11, 173u2013189 (1993)._ _ _ _ Srinivas, S. et al. Cre reporter strains produced by targeted insertion of EYFP and ECFP into the ROSA26 locus. BMC Dev. Biol. 1, 4 (2001)._ _ _ _ _ Jang, J.-H. et al. An Evolved Adeno-associated Viral Variant Enhances Gene Delivery and Gene Targeting in Neural Stem Cells. Mol. Ther. 19, 667u2013675 (2011)._ _ _ _ _ Hughes, S. M., Moussavi-Harami, F., Sauter, S. L. & Davidson, B. L. Viral-mediated gene transfer to mouse primary neural progenitor cells. Mol. Ther. 5, 16u2013 24 (2002)._ _ _ _ Deisseroth, K. et al. Next-generation optical technologies for illuminating genetically targeted brain circuits. J. Neurosci. 26, 10380u201310386 (2006)._ _ _ _ _ Zhang, F. et al. Optogenetics in Freely Moving Mammals: Dopamine and Reward. Cold Spring Harbor Protocols 2015, pdb.top086330 (2015).Zingg, B. et al. AAV-Mediated Anterograde Transsynaptic Tagging: Mapping Corticocollicular Input-Defined Neural Pathways for Defense Behaviors. Neuron 93, 33u201347 (2017)._ _ _ _ Rosario, A. M. et al. Microglia-specific targeting by novel capsid-modified AAV6 vectors. Mol. Ther. Methods Clin. Dev. 3, 16026 (2016)._ _ _ _ _ Tan, K. R. et al. GABA Neurons of the VTA Drive Conditioned Place Aversion. Neuron 73, 1173u20131183 (2012)._ _ _ _ _ Cunningham, C. L., Gremel, C. M. & Groblewski, P. A. Drug-induced conditioned place preference and aversion in mice. Nat. Protoc. 1, 1662 (2006)._ _ _ _ Bullitt, E. Expression of C-fos-like protein as a marker for neuronal activity following noxious stimulation in the rat. J. Comp. Neurol. 296, 517u2013530 (1990)._ _ _ _ Guo, S. et al. Intranasal Delivery of Mesenchymal Stem Cell Derived Exosomes Loaded with Phosphatase and Tensin Homolog siRNA Repairs Complete Spinal Cord Injury. ACS Nano 13, 10015u201310028 (2019)._ _ _ _ Sukumar, U. K. et al. Intranasal delivery of targeted polyfunctional gold-iron oxide nanoparticles loaded with therapeutic microRNAs for combined theranostic multimodality imaging and presensitization of glioblastoma to temozolomide. Biomaterials 218, 119342 (2019)._ _ _ _ _ Illum, L. Is nose-to-brain transport of drugs in man a reality? J. Pharm. Pharmacol. 56, 3u201317 (2004)._ _ _ _ Baik, J. H. Stress and the dopaminergic reward system. Exp. Mol. Med 52, 1879u2013 1890 (2020)._ _ _ _ _ Russo, S. J. & Nestler, E. J. The brain reward circuitry in mood disorders. Nat. Rev. Neurosci. 14, 609u2013625 (2013)._ _ _ _ Dichter, G. S., Damiano, C. A. & Allen, J. A. Reward circuitry dysfunction in psychiatric and neurodevelopmental disorders and genetic syndromes: animal models and clinical findings. J. Neurodev. Disord. 4, 19 (2012)._ _ _ _ Madisen, L. et al. A robust and high-throughput Cre reporting and characterization system for the whole mouse brain. Nat. Neurosci. 13, 133u2013 140 (2010)._ _ _ _ Miyazaki, T. et al. Large Timescale Interrogation of Neuronal Function by Fiberless Optogenetics Using Lanthanide Micro-particles. Cell Rep. 26, 1033u20131043 e1035 (2019)._ _ _ _ Pan, Y. et al. Near-infrared upconversion-activated CRISPR-Cas9 system: A remote-controlled gene editing platform. Sci. Adv. 5, eaav7199 (2019)._ _ _ _ _ _ Chen, X., Chen, Y., Xin, H., Wan, T. & Ping, Y. Near-infrared optogenetic engineering of photothermal nanoCRISPR for programmable genome editing. Proc. Natl Acad. Sci. USA 117, 2395u20132405 (2020)._ _ _ _ _ Chowdhary, S. A., Ryken, T. & Newton, H. B. Survival outcomes and safety of carmustine wafers in the treatment of high-grade gliomas: a meta-analysis. J. Neurooncol 122, 367u2013382 (2015)._ _ _ _ Gomez, J. L. et al. Chemogenetics revealed: DREADD occupancy and activation via converted clozapine. Science 357, 503u2013507 (2017)._ _ _ _ _ _ Forni, P. E. et al. High levels of Cre expression in neuronal progenitors cause defects in brain development leading to microencephaly and hydrocephaly. J. Neurosci. 26, 9593u20139602 (2006)._ _ _ _ _ Nakajima, M. et al. Psychiatric disorder-related abnormal behavior and habenulointerpeduncular pathway defects in Wnt1-cre and Wnt1-GAL4 double transgenic mice. J. Neurochem 124, 241u2013 249 (2013)._ _ _ _ Rezai Amin, S. et al. Viral vector-mediated Cre recombinase expression in substantia nigra induces lesions of the nigrostriatal pathway associated with perturbations of dopamine-related behaviors and hallmarks of programmed cell death. J. Neurochem 150, 330u2013 340 (2019)._ _ _ _ Abram, C. L., Roberge, G. L., Hu, Y. & Lowell, C. A. Comparative analysis of the efficiency and specificity of myeloid-Cre deleting strains using ROSA-EYFP reporter mice. J. Immunol. Methods 408, 89u2013100 (2014)._ _ _ _ _ Bittner-Eddy, P. D., Fischer, L. A. & Costalonga, M. Cre-loxP Reporter Mouse Reveals Stochastic Activity of the Foxp3 Promoter. Front Immunol. 10, 2228 (2019)._ _ _ _ _ Galichet, C., Lovell-Badge, R. & Rizzoti, K. Nestin-Cre mice are affected by hypopituitarism, which is not due to significant activity of the transgene in the pituitary gland. PLoS One 5, e11443 (2010)._ _ _ _ _ _ Lee, B. et al. Nanoparticle delivery of CRISPR into the brain rescues a mouse model of fragile X syndrome from exaggerated repetitive behaviours. Nat. Biomed. Eng. 2, 497u2013507 (2018)._ _ _ _ _ Lee, J. H. et al. Human glioblastoma arises from subventricular zone cells with low-level driver mutations. Nature 560, 243u2013247 (2018)._ _ _ _ _ Lattuada, M. & Hatton, T. A. Functionalization of monodisperse magnetic nanoparticles. Langmuir 23, 2158u20132168 (2007)._ _ _ _ Will, E. et al. Unmodified Cre recombinase crosses the membrane. Nucleic Acids Res. 30, e59 (2002)._ _ _ _ Sparta, D. R. et al. Construction of implantable optical fibers for long-term optogenetic manipulation of neural circuits. Nat. Protoc. 7, 12u201323 (2011)._ _ _ _ _ \"- - \"L.F. acknowledges the funding by COMPETE2020 and FCT programs [POCI-01-0145-FEDER-016390 (acronym: CANCEL STEM) and PTDC/NAN-MAT/28060/2017 (acronym: BrainEdition), POCI-01-0145-FEDER-016682], Portugal 2020-COMPETE funding through u201CPrograma Operacional Regional do Centrou201D CENTRO2020 (CENTRO-01-0145-FEDER-000014) and Projects Interreg 2IQBioNeuro (u201CImpulso de una red de Iu2009+u2009 i en quxEDmica biolxF3gica para diagnxF3stico y tratamiento de enfermedades neurolxF3gicasu201D) and NeuroAtlantic (u201CAn Atlantic innovation platform on diagnosis and treatment of neurological diseases and agingu201D, ref: EAPA_791/2018), and EC projects ERAatUC (Ref:669088) and RESETageing (Ref:952266). C.R. and S.P. acknowledge the FCT fellowships (SFRH/BD/52337/2013; SFRH/BPD/96048/2013). J.P. acknowledges funding from BIAL Foundation (BIAL 266/2016). A.F.R. is grateful to the EU Horizon 2020 programme for funding under grant agreement no. 101003413. The authors would like to thank Claudio Franco and Catarina Fonseca from IMM for the multiphoton microscopy studies.\"- - Authors and Affiliations - \"CNC-Center for Neurosciences and Cell Biology, University of Coimbra, Coimbra, PortugalCatarina Rebelo,_Tiago Reis,_Joana Guedes,_Artur Filipe Rodrigues,_ Susana SimxF5es,_JoxE3o PexE7a,_SxF3nia L. C. Pinho_&_Lino FerreiraFaculty of Medicine, University of Coimbra, Coimbra, PortugalCatarina Rebelo_&_Lino FerreiraHealth Sciences Research Centre (CICS-UBI), University of Beira Interior, CovilhxE3, PortugalClxE1udia Saraiva_&_Liliana BernardinoDepartment of Life Sciences, University of Coimbra, Coimbra, PortugalJoxE3o PexE7a\" - \"AuthorsCatarina RebeloYou can also search for this author in _Tiago ReisYou can also search for this author in _Joana GuedesYou can also search for this author in _ClxE1udia SaraivaYou can also search for this author in _Artur Filipe RodriguesYou can also search for this author in _Susana SimxF5esYou can also search for this author in _Liliana BernardinoYou can also search for this author in _JoxE3o PexE7aYou can also search for this author in _SxF3nia L. C. PinhoYou can also search for this author in _Lino FerreiraYou can also search for this author in _\" - Contributions - C.R., S.P. and L.F. designed the study, did the literature search and wrote the manuscript. C.R. conducted the study. C.R., S.P., T.R., C.S., A.F.R, S.S. and L.B collected the in vitro data. C.R., S.P., T.R., J.G., A.F.R, J.P and L.F. conducted and analyzed in vivo data. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - \"The authors declare the following competing interests: Lino Ferreira, Sonia Pinho and Catarina Rebelo are applicants of the patent entitled u201CLight triggerable gene editing formulation, in vivo editing method, and uses thereofu201D, PCT/IB2020/054856. The patent described a formulation for in vivo gene editing with spatial control. The remaining authors declare no competing interests.\" - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks Guosong Hong, Balint Kiraly and other anonymous Reviewer(s) for the peer review of this work. Peer review Reports are available. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License. - Progelator sterilization and formulation for in vivo analysis - \"HPLC purified peptides were dialyzed with 1u2009kDa molecular weight cutoff tubing into milliQ H2O, sterile filtered through a 0.2u2009xB5m PES filter, and lyophilized to a powder. Peptide was reconstituted prior to injection with sterile pH 9 H2O to form a clear solution (11.11u2009mM), then diluted further with sterile 10xD7 DPBS to a final concentration of 10u2009mM peptide in 1xD7 DPBS (pH 7.4). Solutions for in vivo analysis contained 5u2009mol% rhodamine-labeled progelator (0.5u2009mM labeledu2009+u20099.5u2009mM unlabeled) for fluorescence imaging purposes.\" - Transmission electron microscopy - \"Formvar/Carbon-coated 400 mesh Cu grids (Ted Paella, Inc.) were glow discharged for 90u2009s and spotted with 4u20135u2009xB5L peptide sample (100u2009 xB5M) and set for 5u2009min. Grids were washed with distilled H2O (10 drops), stained with 1% w/w uranyl acetate (3 drops), and wicked dry with filter paper. TEM images were acquired on an FEI Tecnai G2 Sphera at 200u2009kV.\" - Experimental CD - \"Peptides were dissolved at a final concentration of 125u2009xB5M (labeled peptides) or 500u2009xB5M (unlabeled peptides) in 10u2009mM Tris buffer or H2O (for TCEP reduction only) at pH 7.4 to reduce signal interference seen with DPBS at lower wavelengths. UVu2013Vis CD was measured on a Jasco J-810 Spectropolarimeter to evaluate the secondary structure of peptide samples. Measurements were taken using the following settings: wavelength rangeu2009=u2009190u2013260u2009 nm, scanning speedu2009=u200950u2009nmu2009min, response timeu2009=u2009 2u2009s, data pitchu2009=u20091u2009nm, band widthu2009=u20091u2009nm, accumulationsu2009=u20093, pathlengthu2009=u20091u2009mm, and temperatureu2009 =u200937u2009xB0C. Spectra are presented as an average of all accumulations. Sample voltage was monitored to verify the signal intensity did not exceed 800u2009 mV within the wavelength range. Spectra were converted from units of millidegrees (mdeg) to molar ellipticity [u03B8] using the following equation:\" - \"$$left[ theta right] = frac{{{mathrm{mdeg}} ast M}}{{C ast L ast 10}},$$ (1) \" - \"where mdeg is the measured absorbance of circularly polarized light, C is concentration in gu2009L, M is the mean residual weight in gu2009mol, and L is the cell pathlength in cm.\" - Computational modeling with FibPredictor - \"FibPredictor was utilized as a commercially available software for generating native-like amyloid fibril structures. This software was chosen for our study due to its unique ability to reliably model all classes of amyloid fibrils starting from sequence-only input. The algorithm combines u03B2-sheet model building, u03B2-sheet replication, and symmetry operations with side-chain prediction and statistical scoring functions for computational predictions on amyloid fibril structures of self-assembling peptides utilized in this study. From each random model generated, a SCRWL internal scoring function was used by SCWRL4 to predict the energetically lowest side-chain orientations. This scoring function was used to calculate the minimal total energy of the entire model and output coordinates for the structure as a PDB file (see_Supplementary Data) with predicted side chains using the same residue numbering and chain identifiers as the input structure. To identify the most energetically favorable candidate structures in the ensemble, both Amb_3b (computationally more efficient) and GOAP (more accurate) statistical scoring functions were used to calculate total energies of the protein structure. Reported energies are unitless as they are empirically derived scores, and thus it is not advisable to interpret results as kcalu2009mol. Normalized GOAP scores are useful for comparing fibrils of different sizes. Three generations of modeling were performed with literature-recommended parameters shown in Supplementary Table_. Results of top model for each peptide sequence from the third-generation analysis are reported in Supplementary Table_. Corresponding structures and raw PDB files are provided in the main text Fig._ and as_Supplementary Data. Output PDB files, containing the atomic coordinates of the chromophores, from top models were used to generate theoretical CD spectra with DichroCalc.\" - Theoretical CD spectra with DichroCalc - \"DichroCalc was used to calculate CD spectra using predicted PDB from FibPredictor. This web interface predicts secondary structure type using a variety of matrix method parameters, which have been derived from ab initio calculations. These parameters include the peptide chromophore in the far-UV, charge-transfer between neighboring peptide groups in the deep-UV, and aromatic side-chain chromophores with transitions in the near-UV. Full analysis CD spectra were output in units of molar ellipticity [u03B8] vs wavelength.\" - In vitro enzyme cleavage of cyclic peptide progelator - \"Unless otherwise stated, enzyme cleavage experiments were performed on progelator at a final concentration of 500u2009xB5M in 1xD7 enzyme cleavage buffers. MMP-9 cleavages were performed at 1:1000 enzyme/substrate molar ratio in 1xD7 buffer (50u2009mM Tris-HCl, 200u2009mM NaCl, 5u2009mM CaCl2, 1u2009mM ZnCl2, pH 7.5) for 5u2009h. Elastase cleavages were performed at 1:250 enzyme/substrate molar ratio in 1xD7 buffer (100u2009mM Tris-HCl, 0.2u2009mM NaN3, pH 8.0) and thermolysin (50u2009mM Tris-HCl, 0.5u2009mM CaCl2) for 5u2009h. Thermolysin cleavages were performed at 1:4500 enzyme/substrate molar ratio in 1xD7 buffer (1xD7 DPBS, pH 7.4) for 15u2009min. Analysis of unlabeled cyclic peptide progelator cleavage by CD (Fig._) was conducted with serials additions of thermolysin (4u2009xD7u20091:18,000 enzyme/substrate) over a period of 73u2009h in 10u2009 mM Tris buffer. Control samples utilized denatured enzymes under the same conditions. MMP and elastase were heat denatured with 10u2009min of incubation at 65u2009 xB0C. Thermolysin was inactivated by incubation with 10% (v/v) EDTA (0.5u2009 M, pH 8.0).\" - Rheological characterization - \"All peptide samples were prepared at 10u2009mM peptide, except for those used to assess the effect of SAP functionalization with respect to weight (15u2009 mgu2009mL). Viscous and viscoelastic properties were assessed using a stress-controlled rheometer (TA Instruments AR-G2) equipped with a Peltier plate to control temperature and a 20u2009mm diameter parallel plate geometry. Unless otherwise stated all measurements were taken at an angular frequency of 2.5u2009radu2009s, strain of 0.5%, and temperature of 37u2009xB0C. Measurements were performed with a gap height of 1000u2009xB5m and repeated three times (except for step-strain and time course measurements) to ensure reproducibility. To prevent water evaporation, mineral oil was wrapped around the edge of the geometry at the airu2013sample interface. For viscoelastic measurements the apparatus was used in oscillatory mode. To ensure the measurements were made in the linear viscoelastic regime, strain sweeps were performed between 0.05 and 50% strain and showed no variation in storage (Gu2032) and loss (Gu2033) moduli up to a strain of 0.5%. The dynamic moduli of the hydrogels were measured as a function of frequency in the range 0.25u2013100u2009radu2009s at a strain of 0.5%. Continuous step-strain oscillations were used to monitor hydrogel healing through disruption (3u2009min, 100% strain) and recovery (15u2009min, 0.5% strain) cycles (nu2009=u20093). For viscosity measurements the apparatus was used in steady-state flow mode. The viscosity of the samples was measured as a function of shear rate in the range 0.1u2013 10u2009s (5% tolerance). For time course measurements of enzyme activation, a stock of cyclic peptide (1u2009mL, 10u2009mM, 1xD7 DPBS, pH 7.4) was prepared and incubated at 37u2009xB0C. A sample from this stock was applied to the rheometer for storage and loss moduli measurements (2.5u2009radu2009s, 0.5% strain) over a 35u201340u2009min period. This sample was removed and replaced with an identical sample mixed via pipetting for 5u2009s with thermolysin (1:4500 enz/peptide) followed by immediate addition to the rheometer. Delay time between enzyme addition and rheology measurements was ~30u2009s.\" - In vitro catheter injections and effect on assembly - \"Peptide solutions (0.6u20130.8u2009mL) were prepared at 10u2009mM in 1xD7 DPBS (pH 7.4) and loaded into a 1u2009mL Leur Lock syringe attached to a syringe pump set to a flow rate of 0.6u2009mLu2009min. Peptide was injected through the 27u2009G inner nitinol tubing of a MyoStar catheter that was immersed in a 37u2009xB0C water bath.\" - Hemocompatibility analysis - \"For all measurements, peptide stocks in 1xD7 DPBS were prepared such that the final concentrations (v/v) in blood or plasma were 1:10, 1:100, 1:500,1:1000, 1:5000, 1:10,000, and 1:20,000 (peptide/fluid volume). Controls with collagen as a procoagulant initial platelet-adhesive surface (0.095u2009mgu2009mL or 1:240 dilution) or with calcium chelated by sodium citrate utilized 1xD7 DPBS as a vehicle to ensure consistent blood dilution in all experiments. Peptide stocks, whole human blood, isolated RBCs, and PPP were warmed to 37u2009xB0 C immediately prior to use. Plate reader measurements were conducted on an EnSpire Multimode Plate Reader with 96-well tissue culture plates (TCP). All blood studies were done in compliance with the Northwestern University Bloodborne Pathogens Program. Unspun human whole blood, Na citrate, (BSC, LS 2402453A), was processed under US FDA registration #2577632 and pre-screened for viral contaminants (Title 21-CFR PART 610.40).\" - ACT with whole human blood - \"A Hemochron 801 instrument calibrated with an electronic system verification (ESV) device was used to measure ACT of whole human blood. ACTs were determined using recalcified citrated whole human blood to minimize variability in starting time points for clotting in all assays. To each Hemochron P214 tube with glass beads was added 4u2009xB5L CaCl2 (1.1u2009M) and 36u2009xB5L peptide stock or additive (12.2xD7 final blood concentration). Samples were mixed thoroughly for 30u2009s to soak the glass beads and incubated for 30u2009s at 37u2009 xB0C. Citrated whole human blood (400u2009xB5L) was then added (tu2009=u2009 0u2009s), mixed by hand for 10u2009s, and added to the instrument. Time points at which the magnet was displaced by clot formation were recorded by the instrument. Collagen (0.095u2009mgu2009mL) was used as a positive control to decrease clotting time. Vehicle (1xD7 DPBS) served as a standard for blood without additive. Samples without calcium, serving as the negative control, exceeded instrument maximum time range (>1500u2009s). Each experiment was performed nu2009=u2009 6 times with averages and standard error of the mean (SEM) plotted.\" - Whole blood hemostasis kinetics - \"Changes to time-dependent hemostasis were monitored using a non-activated whole blood clotting assay. Briefly, 100u2009xB5L of citrated whole human was mixed with 10u2009xB5L of peptide or collagen stock (11.5xD7 final blood concentration) and 5u2009xB5L of CaCl2 (230u2009mM). Collagen (0.095u2009mgu2009mL) and glass coverslips were used as positive controls to increase clotting rates from an additive and contact initiated perspective, respectively. Samples without calcium, where no clot was observed at any timepoint, served as negative controls. Aliquots (100u2009xB5L) were transferred to a 12-well TCP, covered, and incubated at room temperature for 5, 15, 30, and 45u2009min. Each sample was prepared in triplicate. At the end of each timepoint, RBCs not caught in the thrombus were lysed by gently adding 3u2009mL distilled H2O and incubating for 5u2009 min. Each well was sampled (200u2009xB5L), taking care not to disturb the clot, and transferred to a 96-well plate for analysis. Released hemoglobin from lysis was detected by measuring the absorbance at 405u2009nm (height 4.0u2009mm, 100 flashes) using a plate reader. Extent of clotting is inversely proportional to measured absorbance. Averages (nu2009=u20093) and SEM are plotted.\" - Hemolysis assay - \"RBCs were isolated by centrifugation of 40u2009mL citrated whole human blood at 500u2009xD7u2009g for 5u2009min, followed by gentle washes with 150u2009 mM NaCl (xD71) and 1xD7 DPBS (xD73). Isolated RBCs were diluted 1:50 into 1xD7 DPBS (pH 7.4) and lightly agitated to prevent settling. Briefly, 190u2009xB5 L of dilute RBCs and 10u2009xB5L of peptide stock or additive (20xD7 final concentration) were mixed and added to a 96-well plate (nu2009=u20094 repeats), covered, and incubated at 37u2009xB0C for 1u2009h. Plates were centrifuged at 500u2009xD7u2009g to pellet intact RBCs using a centrifuge equipped with a microplate rotor. Supernatant (100u2009xB5L) was transferred to a new 96-well plate, taking care not to disturb the pellet. Absorbance of the supernatant was measured at 540u2009nm to detect released hemoglobin. To calculate % hemolysis, absorbances were corrected for background absorbance from untreated vehicle and then normalized to 1% Triton X-100-treated RBCs to represent 100% hemolysis. %Hemolysis was calculated according to the following equation:\" - \"$${mathrm{% Hemolysis}} = frac{{left( {{mathrm{Abs}}_{{mathrm{experiemental}}}} right) - left( {{mathrm{Abs}}_{{mathrm{negative}},{mathrm{control}}}} right)}}{{left( {{mathrm{Abs}}_{{mathrm{positive}},{mathrm{control}}}} right) - left( {{mathrm{Abs}}_{{mathrm{negative}},{mathrm{control}}}} right)}} times 100,$$ (2) \" - \"where Absexperiemental is the average well absorbance pertaining to RBCs pertaining to the additive sample being analyzed, Absnegative control is the average well absorbance for RBCs incubated with vehicle (1xD7 DPBS), and Abspositive control is the average well absorbance for RBCs containing 1% Triton X-100.\" - Pro-thrombotic assays - \"PPP was isolated from citrated whole human blood through two rounds of centrifugation at 2000u2009xD7u2009g for 10u2009min and collection of the upper two-third layer. Briefly, 100u2009xB5L of PPP, 50u2009xB5L of peptide stock or additive (4xD7 final concentration), and 50u2009xB5L of CaCl2 (50u2009mM) were mixed and added to a 96-well plate and incubated at 37u2009xB0C for kinetics measurements. For each sample, nu2009=u20096 repeats were conducted. Coagulation profiles were obtained by measuring well absorbance at 405u2009nm at 30u2009s intervals for 60u2009min. Onset to clotting was detected as a sharp increase in sample turbidity.\" - In vivo studies - \"All animal experiments were conducted in accordance with the guidelines established by the Institutional Animal Care and Use Committee at the University of California, San Diego (UCSD), and the Association for the Assessment and Accreditation of Laboratory Animal Care and approved by the Institutional Animal Care and Use Committee at the UCSD (A3033-01). Female Sprague-Dawley rats (225u2013250u2009 g) were used in all studies.\" - Intramyocardial injections of peptide into infarcted rats - \"MI was performed via 35u2009min ischemiau2013reperfusion and intramyocardial injections were performed under isoflurane. Briefly, cyclic peptide (75u2009 xB5L, 0.75 xB5mol, 10u2009mM) was administered as a single injection into the infarct with a 27u2009G needle at 7 days post MI. Animals were euthanized with an overdose of pentobarbital (200u2009mgu2009kg) at 24u2009h post injection (nu2009=u20095).\" - Cryosectioning, histology, and immunofluorescence imaging - \"After euthanasia, hearts were resected, fresh frozen in TissueTek OCT, and cryosectioned for histological analysis. Slides were either stained with hematoxylin and eosin (H&E) to identify the infarct region or stained for immunofluorescence analysis. H&E slides were imaged on an Aperio ScanScope CS at xD720 magnification. For immunofluorescence, tissue sections were permeabilized in acetone (u221220u2009 xB0C) for 1.5u2009min and blocked with 2% bovine serum albumin and 0.3% Triton X-100 in 1xD7 PBS. Myocardium was labeled using mouse anti-rat u03B1-actinin antibody (1:800 dilution in blocking buffer, 1u2009h of incubation, Sigma A7811) and an Alexa Fluor 488 goat-anti-mouse IgG secondary antibody (1:500 dilution in blocking buffer, 30u2009min of incubation, Thermo Fisher A-11001). Nuclei were labeled using Hoechst 33342 (1:10,000 dilution in deionized water, 10u2009 min of incubation, Life Technologies). Tissue sections were mounted with Fluoromount (Sigma) and imaged using a Leica Ariol slide scanner with Ariol software at xD720 magnification.\" - Human cardiac progenitor cell culture - \"Human fetal CPCs were isolated from tissue harvested under standard informed consent procedures and prior approval of the ethics committee of the University Medical Center Utrecht. All cell studies were done in compliance with the UC San Diego Human Research Protections Program. CPCs were isolated from human fetal hearts for selection with Sca-1 magnetic bead sorting (Miltenyi Biotech, 130-091-176) and cultured on 0.1% gelatin in growth media. Growth media consists of 25% EGM-2 (Cambrex, CC-3156) supplemented with EGM-2 Single Quots (Cambrex, CC-4176), 10% fetal bovine serum, 1xD7 penicillin/streptomycin (Sigma, P4458), and 1xD7 minimum essential medium (MEM) non-essential amino acids (BioWhittaker, BE13-114E) in M199 (BioWhittaker, BE12-119F). Cells at passages 3u20135 after isolation were tested for mycoplasma, showing no contamination. Cells at passages 17u201321 were used for all experiments.\" - Cytotoxicity assays - \"Material acute cytotoxicity was tested using human CPCs. The cells were stained with calcein-AM cell viability dye (25u2009nm; eBioscience) for 30u2009min and 500,000 cells were resuspended in 50u2009xB5L of each peptide (10u2009 mM). After 15u2009min, 1u2009mL of growth media was added to the peptide-cell mixture and 50u2009xB5L was transferred into a 24-well plate for imaging. Five to six images were taken per condition using a fluorescence microscope (Carl Zeiss, Dublin, CA, USA). Cells resuspended in media only were used as a control. The percentage of viable cells was determined by counting the stained cells over the total number of cells. Three experiments were performed (nu2009 =u20098u201312 repeats in duplicate). Alternatively, CPCs were also cultured up to 4 days with or without peptides, and cytotoxicity was evaluated by an Alamar blue metabolic activity assay. In all, 3500 hCPCs were plated into each well of a 96-well plate in growth media. At 24u2009h after cell seeding, baseline metabolic activity was taken by incubating the cells in Alamar blue (1:10 in medium, Invitrogen) for 3u2009h and analyzing at 550/585u2009nm with a Synergyu2122 4 Multi-Mode Microplate Reader (Biotek). Peptide was added to cells and cultured for 4 days. Alamar blue assay was repeated after 24 and 96u2009h, and growth media was used as control (nu2009=u20093 repeats in quadruplicate). Metabolic activity reported as fold increase vs baseline values.\" - Statistical analysis - \"All statistical results are expressed as meanu2009xB1u2009SEM. Ordinary one-way analysis of variance (ANOVA) tests were used for multiple comparisons of the mean in each group with that of the standard. Tukey\\'s corrections with 95% confidence intervals and significance were used. Two-way ANOVA tests without matching were used for multiple comparisons of the mean in each group at each timepoint with that of the standard. Holmu2013Sidak test was used for multiple comparisons. Statistical significance was defined as follows: ns (pu2009>u20090.05), *pu2009 u2264u20090.05, **pu2009u2264u20090.01, ***pu2009u2264u20090.001, and ****pu2009u2264u20090.0001.\" - Reporting summary - \"Further information on experimental design is available in the_ linked to this article.\"- - \\' \\' - \"The authors declare that all data supporting the findings of this study are available within the article and its_ or from the corresponding authors on reasonable request.\" - \\' \\'- - \"Mozaffarian, D. et al. Heart disease and stroke statisticsu20142016 update. Circulation 133, 38u2013360 (2015).Richardson, W. J., Clarke, S. A., Quinn, T. A. & Holmes, J. W. Physiological implications of myocardial scar structure. Compr. Physiol. 5, 1877u20131909 (2015).Singelyn, J. M. et al. Naturally derived myocardial matrix as an injectable scaffold for cardiac tissue engineering. Biomaterials 30, 5409u20135416 (2009)._ _ _ Yu, J. et al. The effect of injected RGD modified alginate on angiogenesis and left ventricular function in a chronic rat infarct model. Biomaterials 30, 751u2013 756 (2009)._ _ _ Dorsey, S. M. et al. MRI evaluation of injectable hyaluronic acid-based hydrogel therapy to limit ventricular remodeling after myocardial infarction. Biomaterials 69, 65u201375 (2015)._ _ _ Lee, A. S. et al. Prolonged survival of transplanted stem cells after ischaemic injury via the slow release of pro-survival peptides from a collagen matrix. Nat. Biomed. Eng. 2, 104u2013113 (2018)._ _ Losi, P. et al. Tissue response to poly(ether)urethane-polydimethylsiloxane-fibrin composite scaffolds for controlled delivery of pro-angiogenic growth factors. Biomaterials 31, 5336u20135344 (2010)._ _ _ Prokoph, S. et al. Sustained delivery of SDF-1u03B1 from heparin-based hydrogels to attract circulating pro-angiogenic cells. Biomaterials 33, 4792u20134800 (2012)._ _ _ Layman, H. et al. The effect of the controlled release of basic fibroblast growth factor from ionic gelatin-based hydrogels on angiogenesis in a murine critical limb ischemic model. Biomaterials 28, 2646u2013 2654 (2007)._ _ _ Fujimoto, K. L. et al. Synthesis, characterization and therapeutic efficacy of a biodegradable, thermoresponsive hydrogel designed for application in chronic infarcted myocardium. Biomaterials 30, 4357u20134368 (2009)._ _ _ Bastings, M. M. C. et al. A fast pH-switchable and self-healing supramolecular hydrogel carrier for guided, local catheter injection in the infarcted myocardium. Adv. Healthc. Mater. 3, 70u201378 (2014)._ _ _ Wang, H. et al. Improved myocardial performance in infarcted rat heart by co-injection of basic fibroblast growth factor with temperature-responsive Chitosan hydrogel. J. Heart Lung. Transplant. 29, 881u2013887 (2010)._ _ _ Formiga, F. R. et al. Controlled delivery of fibroblast growth factor-1 and neuregulin-1 from biodegradable microparticles promotes cardiac repair in a rat myocardial infarction model through activation of endogenous regeneration. J. Control. Release 173, 132u2013139 (2014)._ _ _ Suarez, S. L. et al. Degradable acetalated dextran microparticles for tunable release of an engineered hepatocyte growth factor fragment. ACS Biomater. Sci. Eng. 2, 197u2013204 (2016)._ _ _ Ungerleider, J. L. & Christman, K. L. Concise review: injectable biomaterials for the treatment of myocardial infarction and peripheral artery disease: translational challenges and progress. Stem Cells Transl. Med. 3, 1090u20131099 (2014)._ _ _ Hernandez, M. J. & Christman, K. L. Designing acellular injectable biomaterial therapeutics for treating myocardial infarction and peripheral artery disease. JACC Basic Transl. Sci. 2, 212u2013226 (2017)._ _ Portnov, T., Shulimzon Tiberiu, R. & Zilberman, M. In Rev. Chem. Eng. Vol. 33 91 (2017).Akiyama, N., Yamamoto-Fukuda, T., Takahashi, H. & Koji, T. In situ tissue engineering with synthetic self-assembling peptide nanofiber scaffolds, PuraMatrix, for mucosal regeneration in the rat middle-ear. Int. J. Nanomed. 8, 2629u20132640 (2013)._ _ Davis, M. E. et al. Injectable self-assembling peptide nanofibers create intramyocardial microenvironments for endothelial cells. Circulation 111, 442u2013450 (2005)._ _ _ French, K. M., Somasuntharam, I. & Davis, M. E. Self-assembling peptide-based delivery of therapeutics for myocardial infarction. Adv. Drug Deliv. Rev. 96, 40u201353 (2016)._ _ _ Li, X. et al. Image-guided stem cells with functionalized self-assembling peptide nanofibers for treatment of acute myocardial infarction in a mouse model. Am. J. Transl. Res. 9, 3723u20133731 (2017)._ _ _ _ Yuan, X., He, B., Lv, Z. & Luo, S. Fabrication of self-assembling peptide nanofiber hydrogels for myocardial repair. RSC Adv. 4, 53801u201353811 (2014)._ _ _ Ravichandran, R., Venugopal, J. R., Sundarrajan, S., Mukherjee, S. & Ramakrishna, S. Minimally invasive cell-seeded biomaterial systems for injectable/epicardial implantation in ischemic heart disease. Int. J. Nanomed. 7, 5969u20135994 (2012)._ _ Chien, M.-P. et al. Enzyme-directed assembly of nanoparticles in tumors monitored by in vivo whole animal imaging and ex vivo super-resolution fluorescence imaging. J. Am. Chem. Soc. 135, 18710u201318713 (2013)._ _ _ Nguyen, M. M. et al. Enzyme-responsive nanoparticles for targeted accumulation and prolonged retention in heart tissue after myocardial infarction. Adv. Mater. 27, 5547u20135552 (2015)._ _ _ Callmann, C. E. et al. Therapeutic enzyme-responsive nanoparticles for targeted delivery and accumulation in tumors. Adv. Mater. 27, 4611u20134615 (2015)._ _ _ Kalafatovic, D. et al. MMP-9 triggered micelle-to-fibre transitions for slow release of doxorubicin. Biomater. Sci. 3, 246u2013249 (2015)._ _ _ Zhou, J. & Xu, B. Enzyme-instructed self-assembly: a multistep process for potential cancer therapy. Bioconjug. Chem. 26, 987u2013999 (2015)._ _ _ Abul-Haija, Y. M. & Ulijn, R. V. In Hydrogels in Cell-Based Therapies 112u2013134 (The Royal Society of Chemistry, 2014).Bowerman, C. J. & Nilsson, B. L. A reductive trigger for peptide self-assembly and hydrogelation. J. Am. Chem. Soc. 132, 9526u20139527 (2010)._ _ _ Spinale, F. G. Myocardial matrix remodeling and the matrix metalloproteinases: influence on cardiac form and function. Physiol. Rev. 87, 1285u20131342 (2007)._ _ _ Prabhu, S. D. & Frangogiannis, N. G. The biological basis for cardiac repair after myocardial infarction. Inflamm. Fibros. 119, 91u2013112 (2016)._ _ Vulesevic, B., Sirois, M. G., Allen, B., de Denus, S. & White, M. Subclinical Inflammation in heart failure: a neutrophil perspective. Can. J. Cardiol. 34, 717u2013725 (2018)._ _ Rad-Malekshahi, M., Lempsink, L., Amidi, M., Hennink, W. E. & Mastrobattista, E. Biomedical applications of self-assembling peptides. Bioconjug. Chem. 27, 3u201318 (2016)._ _ _ Koutsopoulos, S. Selfu2010 assembling peptide nanofiber hydrogels in tissue engineering and regenerative medicine: progress, design guidelines, and applications. J. Biomed. Mater. Res. A. 104, 1002u20131016 (2016)._ _ _ Betush, R. J., Urban, J. M. & Nilsson, B. L. Balancing hydrophobicity and sequence pattern to influence selfu2010assembly of amphipathic peptides. Pept. Sci. 110, e23099 (2018)._ _ Hamley, I. W. et al. Self-assembly of a model amphiphilic oligopeptide incorporating an arginine headgroup. Soft Matter 9, 4794u20134801 (2013)._ _ _ _ Gao, J. et al. Controlling self-assembling peptide hydrogel properties through network topology. Biomacromolecules 18, 826u2013834 (2017)._ _ _ Tripathi, J. K. et al. Variants of self-assembling peptide, KLD-12 that show both rapid fracture healing and antimicrobial properties. Biomaterials 56, 92u2013103 (2015)._ _ _ Sun, J. et al. Biocompatibility of KLD-12 peptide hydrogel as a scaffold in tissue engineering of intervertebral discs in rabbits. J. Huazhong Univ. Sci. Technol. [Med. Sci.] 30, 173u2013 177 (2010)._ _ _ Kisiday, J. et al. Self-assembling peptide hydrogel fosters chondrocyte extracellular matrix production and cell division: implications for cartilage tissue repair. Proc. Natl Acad. Sci. USA 99, 9996u201310001 (2002)._ _ _ _ Tabatabaei Ghomi, H., Topp, E. M. & Lill, M. A. Fibpredictor: a computational method for rapid prediction of amyloid fibril structures. J. Mol. Model. 22, 206 (2016)._ _ Zhang, S., Zhao, X. & Spirio, L. In Encyclopedia of Biomedical Polymers and Polymeric Biomaterials Vol. 11 (ed. Mishra, M.K.) 15 (CRC Press, Boca Raton, 2015).Roeters, S. J. et al. Evidence for intramolecular antiparallel beta-sheet structure in alpha-synuclein fibrils from a combination of two-dimensional infrared spectroscopy and atomic force microscopy. Sci. Rep. 7, 41051 (2017)._ _ _ _ Bulheller, B. M. & Hirst, J. D. DichroCalcu2014circular and linear dichroism online. Bioinformatics 25, 539u2013540 (2009)._ _ _ Khakshoor, O., Demeler, B. & Nowick, J. S. Macrocyclic u03B2-sheet peptides that mimic protein quaternary structure through intermolecular u03B2-sheet interactions. J. Am. Chem. Soc. 129, 5558u20135569 (2007)._ _ _ Montenegro, J., Ghadiri, M. R. & Granja, J. R. Ion channel models based on self-assembling cyclic peptide nanotubes. ACC Chem. Res. 46, 2955u20132965 (2013)._ _ _ Dinerman, J. L. et al. Increased neutrophil elastase release in unstable angina pectoris and acute myocardial infarction. J. Am. Coll. Cardiol. 15, 1559u20131563 (1990)._ _ _ van den Burg, B. & Eijsink, V. In Handbook of Proteolytic Enzymes (ed. Salvesen, G.) 540u2013 553 (Academic Press, London, Boston, 2013).Clark, R. J. et al. Engineering stable peptide toxins by means of backbone cyclization: stabilization of the u03B1 -conotoxin MII. Proc. Natl Acad. Sci. USA 102, 13767u201313772 (2005)._ _ _ _ Chen, Y. et al. Stabilization of peptides against proteolysis through disulfide-bridged conjugation with synthetic aromatics. Org. Biomol. Chem. 15, 1921u20131929 (2017)._ _ _ Tian, Y. F., Hudalla, G. A., Han, H. & Collier, J. H. Controllably degradable u03B2-sheet nanofibers and gels from self-assembling depsipeptides. Biomater. Sci. 1, 1037u20131045 (2013)._ _ _ van den Burg, B. & Eijsink, V. In Handbook of Proteolytic Enzymes Third edn (eds. Rawlings, N. D. & Salvesen, G.) 540u2013553 (Academic Press, London, Boston, 2013).Habibi, N., Kamaly, N., Memic, A. & Shafiee, H. Self-assembled peptide-based nanostructures: Smart nanomaterials toward targeted drug delivery. Nano Today 11, 41u201360 (2016)._ _ _ Grover, G. N., Braden, R. L. & Christman, K. L. Oxime cross-linked injectable hydrogels for catheter delivery. Adv. Mater. 25, 2937u20132942 (2013)._ _ _ Seif-Naraghi, S. B. et al. Safety and efficacy of an injectable extracellular matrix hydrogel for treating myocardial infarction. Sci. Transl. Med. 5, 173ra125 (2013)._ _ Horton, S. & Augustin, S. In Haemostasisu2014Methods and Protocols Vol. 992_ (ed. Paul Monagle) 155u2013167 (Humana Press, Springer Science, 2013).Autian, J. In Polymers in Medicine and Surgery. Polymer Science and Technology Vol. 8 (eds. Kronenthal, R. L., Oser, Z. & Martin, E.) 181u2013 203 (Springer, Boston, 1975).Hee, K. S. et al. Self-assembling peptide nanofibers coupled with neuropeptide substance P for bone tissue engineering. Tissue Eng. Part A 21, 1237u20131246 (2015)._ Zhu, J. & Marchant, R. E. Design properties of hydrogel tissue-engineering scaffolds. Expert Rev. Med. Devices 8, 607u2013626 (2011)._ _ Wang, R. M. & Christman, K. L. Decellularized myocardial matrix hydrogels: in basic research and preclinical studies. Adv. Drug Del. Rev. 96, 77u201382 (2016)._ _ Gawaz, M. Role of platelets in coronary thrombosis and reperfusion of ischemic myocardium. Cardiovasc. Res. 61, 498u2013511 (2004)._ _ _ Roberts, D. E., McNicol, A. & Bose, R. Mechanism of collagen activation in human platelets. J. Biol. Chem. 279, 19421u201319430 (2004)._ _ _ Kim, S. J. et al. Therapeutic effects of neuropeptide substance P coupled with self-assembled peptide nanofibers on the progression of osteoarthritis in a rat model. Biomaterials 74, 119u2013130 (2016)._ _ _ Lu, J. et al. A neurotrophic peptide-functionalized self-assembling peptide nanofiber hydrogel enhances rat sciatic nerve regeneration. Nano Res. 11, 4599u20134613 (2018)._ _ _ Zhang, N., Luo, Y., He, L., Zhou, L. & Wu, W. A self-assembly peptide nanofibrous scaffold reduces inflammatory response and promotes functional recovery in a mouse model of intracerebral hemorrhage. Nanomedicine 12, 1205u20131217 (2016)._ _ _ Ghomi, H. T., Thompson, J. J. & Lill, M. A. Are distance-dependent statistical potentials considering three interacting bodies superior to two-body statistical potentials for protein structure prediction? J. Bioinform. Comput. Biol. 12, 1450022 (2014)._ _ _ Zhou, H. & Skolnick, J. GOAP: a generalized orientation-dependent, all-atom statistical potential for protein structure prediction. Biophys. J. 101, 2043u2013 2052 (2011)._ _ _ _ Johnson, T. D. et al. Human versus porcine tissue sourcing for an injectable myocardial matrix hydrogel. Biomater. Sci. 2014, 60283D (2014)._ _ _ \"- - \"The authors would like to thank G. Policastro, J. Ungerleider, A. Luthi, and M. Spang for assistance with animal studies, R. Wang and M. Touve for assistance with microscopy, and N. Ng and J. Madan for aid with cryosectioning. Thanks to the Molinski Lab at UC San Diego for the use of their CD spectrometer. Thanks to the Ameer Lab at Northwestern University for the use of their Hemochron instrument, and Dr. Joost Sluijter at the University Medical Center Utrecht for providing CPCs. The authors are also grateful for the support of an NIH Director s Transformative Research Award (R01HL117326), part of the NIH Common Fund, the NHLBI (R01HL139001), and an AFORS MURI (FA9550-16-1-0150). A.S.C. would like to thank the NSF for a Graduate Research Fellowship (DGE-1144086).\"- - Authors and Affiliations - \"Department of Chemistry & Biochemistry, University of California, San Diego, La Jolla, CA, 92093, USAAndrea S. CarliniDepartment of Chemistry, Department of Materials Science & Engineering, Department of Biomedical Engineering, Simpson Querrey Institute for BioNanotechnology, International Institute for Nanotechnology, and Chemistry of Life Processes Institute, Northwestern University, Evanston, IL, 60208, USAAndrea S. Carlini_&_Nathan C. GianneschiDepartment of Bioengineering, Sanford Consortium for Regenerative Medicine, University of California, San Diego, La Jolla, CA, 92093, USARoberto Gaetani,_Rebecca L. Braden,_Colin Luo_ &_Karen L. Christman\" - \"AuthorsAndrea S. CarliniYou can also search for this author in _Roberto GaetaniYou can also search for this author in _Rebecca L. BradenYou can also search for this author in _Colin LuoYou can also search for this author in _Karen L. ChristmanYou can also search for this author in _Nathan C. GianneschiYou can also search for this author in _\" - Contributions - A.S.C., N.C.G., and K.L.C. designed the research and analyzed the data. A.S.C. performed synthetic, computational, biochemical, and animal experiments, and drafted the paper. R.L.B. and C.L. performed direct intramyocardial injections and myocardial infarction surgeries, respectively. R.G. performed cell viability assays and toxicity staining. N.C.G. and K.L.C. provided financial support. K.L.C. and N.C.G. directed the research. All authors edited the manuscript. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - K.L.C. is co-founder, board member, consultant, and holds equity interest in Ventrix, Inc. The other authors declare no competing interests. - \\' \\'- - \\'Journal peer review information: Nature Communications thanks Ke Cheng and the other anonymous reviewer(s) for their contribution to the peer review of this work.\\' - \"Publishers note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License. At P0, tdT BMECs were: (i) passaged when confluent (every week) as WBM cultures; (ii) sorted into Emcn and Emcn, plated as individual cultures and passaged once confluent (every week). e Total cell number of Emcn cells counted at the indicated passage (left); fold changes of cell numbers at the indicated passage compared to the previous one (right). Fresh WBM nu2009=u20098; P0 nu2009=u200910; P1 nu2009 =u20098; sorted P1 nu2009=u20093. Data are presented as the meanu2009xB1u2009 SEM. Statistics for all comparisons shown were determined using paired t-test, two-sided. Left graph: Fresh WBM vs. P0, pu2009=u20090.0008; P0 vs. P1, pu2009 =u20090.0193. Right graph: P0 vs. P1, pu2009=u20090.0027. * pu2009<u2009 0.05, **pu2009<u20090.01, ***pu2009<u20090.001. Source data are provided as a Source Data file. f Total cell number of Emcn cells counted at the indicated passage (left); fold changes of cell numbers at the indicated passage compared to the previous one (right). Fresh WBM nu2009=u20098; P0 nu2009=u200910; unsorted P1 and P2, sorted P1, P2, and P3 nu2009=u20098 (each group); unsorted P3 nu2009=u20097; unsorted P4 nu2009=u20096, sorted P4 nu2009=u20096. Data are presented as the meanu2009xB1u2009SEM. Statistics for all comparisons shown were determined using paired t-test, two-sided. Left graph: Fresh WBM vs. P0, pu2009<u20090.0001; P0 vs. P1, pu2009=u20090.0015; P1 vs. P2, pu2009 =u20090.0151; P0 vs. P1, pu2009=u20090.0005; P1 vs. P2, pu2009<u20090.0001. Right graph: P0 vs. P1, pu2009<u20090.0001; P1 vs. P2, pu2009=u20090.0027; P2 vs. P3, pu2009=u20090.0141; P1 vs. P2, pu2009=u20090.0045; P2 vs. P3, pu2009=u20090.0296. *pu2009<u20090.05, **pu2009<u20090.01, ***pu2009 <u20090.001, and ****pu2009<u20090.0001. Source data are provided as a Source Data file.\" - These results demonstrate that the growth of both EmcnSca-1 venous/sinusoidal-like and Emcn-Sca-1 arterial-like BMECs is preserved and supported by the WBM culture for the first two weeks. - WBM populations and their kinetics in long-term culture - \"Our results suggest that the presence of other cell lineages in culture is critical for preserving BMEC heterogeneity and function. Flow cytometry analysis of fresh BM at day 0 and of P0 WBM culture at day 14 revealed three major populations (Fig._): (i) CD45tdT (BMECs); ii) CD45tdT mesenchymal cells (MCs), and (iii) CD45tdT hematopoietic cells/macrophages (HC/Mu03A6). In the 2-weeks WBM culture, BMECs were enriched from ~0.01% to ~20% and MCs were enriched from ~1% to ~40%, whereas HC/Mu03A6 decreased from 98% to ~40% (Fig._). Lineage identity of these cell populations was confirmed by gene expression analysis of EC, HC/Mu03A6 , and MC markers (Supplementary Fig._).\" - \"Further immunophenotypic analysis showed that, at day 0, the HC fraction comprised CD3 T-cells, B220 B-cells, and CD11b myeloid cells; by day 5 of culture and beyond, CD3 and B220 cells were barely detected, and by day 14, all CD45tdT HC cells were constituted solely by adherent macrophages (Mu03A6) expressing the myeloid/macrophage markers CD14, CD11b, and F4/80 (Supplementary Fig._ ). The stromal population CD45tdT expressed the MC markers PDGFRu03B1 (CD140a), CD90, CD44, CD29, and Sca-1 (Supplementary Fig._).\" - \"After P0, WBM cultures could be serially passaged for several times. BMECs had their greatest expansion within the first two weeks of WBM culture (P0) and proliferated up to Passage 4 (P4), constituting a steady 30% of the entire WBM population from P1 to P4 (Fig._ and Supplementary Fig._). Mu03A6 decreased significantly from P0 to P1 (from 40% to 5-7%) and throughout serial passages (to_~2%; Fig._ and Supplementary Fig._) suggesting limited expansion in this culture system. Unlike Mu03A6, MCs expanded during the serial passages and dominated the WBM culture throughout P1-P4 representing ~75% of the culture at P4 (~62-fold total expansion at P4; Fig._ and Supplementary Fig._).\" - Emcn BMECs are endowed with high proliferative potential - \"We characterized the proliferative potential of Emcn and Emcn BMECs and assessed how their growth depended on WBM culture after P0. P0 WBM cultures were harvested at day 14 and BMECs were passaged using two approaches: (1) BMECs were replated and serially passaged as WBM; (2) BMECs were sorted into Emcn and Emcn fractions and each population was separately replated and serially passaged as homogeneous populations (Scheme in Fig._).\" - \"The greatest expansion of BMEC (both Emcn and Emcn) occurred within the first two weeks of WBM culture (P0) (Fig._). The absolute number of Emcn BMECs increased significantly (48-folds; Supplementary Table_) during P0, but they rapidly declined at P1 and were no longer detected at P2, regardless of whether Emcn cells were growing as WBM culture (Figs._e and ) or as sorted cells (Fig._ ). Emcn cells formed 2D-networks only at P0 (Supplementary Fig._). Of note, the exhaustion of Emcn cells in WBM from P0 to P2, significantly correlated with the decline of Mu03A6 (Pearson ru2009=u20090.997; pu2009=u20090.0002; Fig._ ).\" - \"Emcn cells increased greatly (~600-fold; Supplementary Table_) during P0 culture, and unlike the Emcn cells, they propagated in culture for up to four passages (Fig._). Although no statistically significant differences were observed between the proliferative rates of Emcn BMECs passaged as WBM culture or as sorted cells, sorted Emcn cells showed a trend for greater expansion from P1 to P4 (~20-fold) compared to Emcn cells passaged as WBM (~10-fold; Fig._ and Supplementary Fig._ ). We observed a negative correlation between MC and Emcn cells in WBM P1-P4 (Pearson ru2009=u2009u22120.944; pu2009=u20090.055; Fig._); thus, it is likely that the greater expansion of sorted Emcn cells is due to absence of an inhibitory effect exerted by MCs. The rates of expansion of Emcn BMECs were high from P0 to P1 (~5-fold) but gradually decreased during subsequent passages (Fig._, right panel. Emcn BMECs proliferated as a cobblestone monolayer from P1 to P4 (Supplementary Fig._) and could generate up to either ~23 million (WBM) or ~32 million (sorted) BMECs from an initial culture of ~10,000 cells/mouse. Altogether, the overall expansion of total CD45tdT BMECs from day 0 to P4 of WBM culture was greater than 5000-fold.\" - \"Mu03A6 promote preservation of Emcn and expansion of Emcn BMECs\" - \"The above studies provided us with three key results: (1) Mu03A6 decreased significantly from P0 to P1; (2) MC increased over time; and (3) Emcn BMECs became undetectable after P1 (Figs._eu2013g and ). The significant correlation between Emcn BMEC depletion, decrease in Mu03A6 numbers and expansion of MC prompted us to explore whether changes in populations ratio impacted BMECs output. Treatment of P0 WBM with additional Mu03A6 significantly increased the total number of tdT BMECs vs. control condition: twice as many Mu03A6 added to standard WBM culture resulted in a 2-fold further increase of P0 Emcn over the T0 baseline (~96-fold vs. 48-fold), particularly at lower cell densities (Fig._). A similar enhancement was seen for P0 Emcn, and for all BMEC, as quantified by confocal microscopy (Supplementary Fig._). Coculture using transwell inserts suggested that the effect of Mu03A6 is mediated mostly by soluble factors (Supplementary Fig._).\" - \"Fig. 5: Impact of HC/Mu03A6 and MC on BMEC in WBM culture.au2013d P0 WBM cultures were divided in three conditions: control and supplement at day 0 with additional HC/Mu0278 or MC. a, b Snapshot images at day 14 culture showing tdT BMECs in P0 WBM cultures that were supplemented with a HC/Mu03A6 or b MC, compared to control; two representative fields; Scale bar, 2000u2009u03BCm. nu2009=u2009 12. c Fold change in BMEC numbers: tdT (left), Emcn (middle) and Emcn (right) when P0 WBM culture is supplemented with HC/Mu0278 or MC, vs. culture control. Data are presented as the meanu2009xB1u2009SEM. Emcn cells expanded on_ average 48-fold in control cultures (set up in this graph at value 1), 96-fold when supplemented with HC/Mu03A6, and 24-fold when supplemented with MC. BMECu2009 +u2009Mu03A6, nu2009=u20098; BMECu2009+u2009MC, nu2009=u20096. nu2009 =u20093 independent experiments. Statistics for all comparisons shown were determined using unpaired t-test, two-sided. tdT Mu03A6, pu2009=u20090.0026; tdT MC, pu2009<u20090.0001; Emcn Mu03A6, pu2009<u20090.0001; Emcn MC, pu2009<u2009 0.0001; Emcn Mu03A6, pu2009<u20090.0001; Emcn MC, ***pu2009<u20090.001, and ****pu2009<u20090.0001. Source data are provided as a Source Data file. d Total cell number of Emcn and Emcn cells of P0 WBM culture with or without additional HC/Mu03A6 or MC at different WBM cell densities. Representative of nu2009 =u20093 independent experiments. Source data are provided as a Source Data file. e, f P1 WBM cultures were supplemented with additional HC/Mu03A6 and cultured for seven days: e total cell number and f fold changes of Emcn and Emcn cells supplemented with additional HC/Mu03A6 vs. control, at different cell densities; nu2009=u20093. Data are presented as the meanu2009xB1u2009SEM. Statistics for all comparisons shown were determined using unpaired t-test, two-sided. e Cell density 1, Emcn, pu2009=u20090.0020; cell density 1/4, Emcn, pu2009 =u20090.0006; cell density 1/8, Emcn, pu2009=u20090.0001; Emcn pu2009=u2009 ns at all cell densities. f Cell density 1, Emcn, pu2009=u20090.0006; cell density 1/4, Emcn, pu2009=u20090.0004; cell density 1/8, Emcn, pu2009<u20090.0001; Emcn pu2009=u2009ns at all cell densities. **pu2009<u20090.01, ***pu2009 <u20090.001, and ****pu2009<u20090.0001. Source data are provided as a Source Data file. g P0 WBM cultures were supplemented with recombinant M-CSF (50u2009 ng/mL) and cultured for fifteen days. Total cell number of Emcn and Emcn cells of P0 WBM culture with or without M-CSF; nu2009=u20093 independent cultures. Data are presented as the meanu2009xB1u2009SEM. Statistics were determined using unpaired t-test, two-sided *pu2009=u20090.0336.\" - \"In contrast, supplement of additional MCs was associated with a decrease in the total number of BMECs, both Emcn and Emcn BMECs (Fig._). Coculture with transwells suggested that the effect of MC is mediated mostly by cell-to-cell contact (Supplementary Fig._).\" - \"Given these results, we hypothesized that supplementary BM Mu03A6 could rescue the cultures of sorted BMECs (T0; Fig._) and/or the serial passages of WBM (Fig._), where the absence or decline in Mu03A6 numbers correlated with the absence of Emcn cells. BMECs directly sorted from fresh bone were examined upon treatment with supplementary Mu03A6, supplementary MCs, or a combination of both. Addition of Mu03A6 resulted in a greater expansion of sorted T0 BMECs compared to sorted T0 BMECs cultured alone, and rescued T0 from the inhibitory effect of MCs (Supplementary Fig._). The addition of Mu03A6 to the P1 culture increased the numbers of Emcn cells while just preserving the numbers of the Emcn population (Fig._). Next, we tested whether the addition of M-CSF impacted BMEC outputs. As shown in Fig._, M-CSF resulted in a ~5-fold increase of total BMEC (average 4-fold and 10-fold increase in Emcn and Emcn, respectively).\" - \"Collectively, these results indicate that Mu03A6 promote BMEC expansion and maintenance of their heterogeneity in ex vivo culture, whereas MCs exert an inhibitory effect.\" - Emcn BMECs contribute to angiogenesis and support HSPCs - \"Next, we assessed the ability of both Emcn and Emcn BMECs to engage in in vitro angiogenesis in Matrigel tube assay. Emcn BMECs (sorted at either P0 or P1) were not able to form any type of structure in Matrigel (Fig._ and Supplementary Movie_). In contrast, Emcn BMECs sorted at P0 formed robust cord-networks on Matrigel by 36u2009h, which remained stable up to 47u2009h (Fig._ and Supplementary Movie_). Interestingly, Emcn BMECs from P1, P2, and P3 exhibited a faster dynamic of cord-networks formation, by forming cords already at ~6u2009h and completing them between 12 and 24u2009h (Fig._, Supplementary Fig._, and Supplementary Movie_). Cord-networks that were formed by Emcn cells at P1, P2, and P3 appeared less defined and were less complex across passages, as determined by measurement of their cellular connectivity (Fig._, Supplementary Fig._, and Supplementary Movie_).\" - \"Fig. 6: Distinct functional properties of Emcnu2009u2009and Emcn BMEC.a, b Time lapse images of cells in Matrigel at the indicated time points: a Emcn and Emcn cells sorted from P0 WBM cultures; b Emcn cells from P1 (grown in culture as sorted cells), and Emcn cells from P1, P2, and P3 (grown in culture as sorted cells). Scale bar, 200u2009u03BCm. Representative of nu2009=u20098-12 independent experiments. c Quantification of cords formation and structure integrity by Emcn cells at P1 and P2. Line graph shows differences in growth patterns of cells at P1 and P2, indicated as a decrease in the contour area with passage (see details in Methods section). P1 nu2009=u200912; P2 nu2009=u20098. Data are presented as the meanu2009xB1u2009SEM. Source data are provided as a Source Data file. d Immunostaining of CD31 marker on Matrigel plugs. Matrigel was admixed with sorted P0 Emcn (left) or P0 Emcn (right) cells and implanted into the flanks of syngeneic mice for two months, prior collection and fixation. Note that only exogenous P0 Emcn cells (nucleus in red), but not P0 Emcn cells, can be observed in new blood vessels formed by endogenous ECs (nucleus not in red). Zoom-in image of white-dashed squares 1 and 2 are depicted on the right panel. Scale bar, 100u2009u03BCm. Representative of nu2009=u20093 experiments. e, f Monolayers of Emcn or Emcn cells sorted from P0 WBM culture were cocultured with sorted LSK (6000 cells/well) for seven days in the presence or absence of SCF (50u2009ng/ml). Bar graphs show total cell numbers at day 7 for e LSK cells and f LK and CD11bu2009u2009cells; nu2009=u20092. Data are presented as the meanu2009xB1u2009SEM. Source data are provided as a Source Data file.\" - \"To date, it has not been possible to determine whether different types of primary BMECs contribute differently to in vivo angiogenesis or have distinct ability to support HSCs. To assess how Emcn and Emcn cells contribute to angiogenesis, we performed in vivo Matrigel plug assays. Either Emcn or Emcn cells were sorted from P0 WBM at day14, admixed with Matrigel and injected subcutaneously into the flank of syngeneic mice. Two months later, plug vascularization was evaluated by CD31 immunostaining and tdT fluorescence. Unlike Emcn cells, which were not able to form any blood vessels, Emcn cells successfully participated in forming new blood vessels with endogenous ECs (Fig._). These results indicate that Emcn, but not Emcn BMECs, contribute to angiogenesis in vivo.\" - \"A critical function of ECs in the BM is to support HSCs in the vascular stem cell niche. This method provided the opportunity to compare the ability of Emcn and Emcn BMECs to sustain hematopoietic stem/progenitor cells (HSPC: lineageSca-1cKit, LSK) in vitro. Cocultures of sorted LSK cells over layers of sorted BMECs showed that Emcn but not Emcn BMEC, preserved LSK cells in the absence of stem cell factor (SCF). In the presence of SCF, Emcn BMECs maintained LSK cells at significantly higher numbers and promoted lower rates of myeloid differentiation (Fig._, f). Thus, in our model, sinusoidal-like Emcn BMECs exhibit a better ability to support HSC.\" - Molecular signature of Emcn and Emcn BMECs ex vivo - \"We performed bulk RNA-seq analysis of Emcn and Emcn BMEC sorted from P0 WBM. Their transcriptome was compared to the ones from Emcn cells sorted from lung cultures and BM MSC (Figure_). P0 Emcn BMECs showed higher expression of conventional EC identity genes, Cdh5 (VE-cadherin), Pecam1 (CD31) and Tek (Tie2), and expressed the most well-known venous EC marker gene Ephb4 and other venous/sinusoidal endothelial genes, such as Stab2, Flt4, Epor, Dnase1/3, and Angpt2 (Fig._ and Supplementary Fig._). These genes were within the top 5% of variable genes across all samples. P0 Emcn BMECs expressed higher levels of known arterial endothelial genes, such Serpinf1, Ace, Tmem100, and Ltbp4 and were enriched in genes expressed by mesenchymal cells such as Gsn, Runx2, and Pdgfrb (Fig._ and Supplementary Fig._). Among the differentially expressed genes (Fig._ , Supplementary Fig._, and Supplementary Table_), we noted that P0 Emcn BMECs expressed markers associated with endothelial tip cells, such as Apln, Cxcr4, Kdr, Dll4, Angpt2, Sox17, and Esm1 (Fig._), whereas P0 Emcn BMECs expressed markers associated with endothelial stalk cells, such as Eno2, Hk2, Meox2, Acss2, Aldh2, and Lgals3 (Fig._ and Supplementary Table_). The molecular differences between P0 Emcn and P0 Emcn cells were even more evident when we restricted the analysis to signatures of 38 arterial/sinusoidal genes and 35 tip/stalk genes (Fig._) that we compiled from the literature (Supplementary Table_). Thus, P0 Emcn BMECs exhibited a combination of sinusoidal- and tip-like signature, whereas P0 Emcn expressed a combination of arterial- and stalk-like signature. These patterns of gene expression were also confirmed by RT-PCR (Fig._). Of note, Emcn BMECs expressed higher levels of genes encoding for hematopoietic supportive molecules, such as Dll4, Dll1, Tgfb1, and Pdgfb (bottom panels of Supplementary Fig._).\" - \"Fig. 7: Molecular Signatures of Emcnu2009and Emcn BMEC.au2013c Bulk-RNA-seq of sorted populations at day 14 of culture: Emcn and Emcn BMEC from P0 WBM; Emcn EC from lung; MSC from BM. a Heatmap of the top 5% variable genes. b Heatmap of arterial-sinusoidal gene signature and c tip-stalk signature. Measurements are expressed in scaled log2 counts per million (CPM). d Real-time (RT-) PCR for the indicated genes in Emcn and Emcn BMEC from P0 WBM culture at day 1; nu2009=u20092. Source data are provided as a Source Data file. e Bulk-RNA-seq of Emcn BMECs at the different passages. Heatmap of the top 5% variable genes. f Principal Component Analysis (PCA) of BMEC P0 Emcn and Emcn and P1-P4 Emcn; lung Emcn, and MSC transcriptomes. g Analysis of the most differentially expressed genes in Emcn BMEC cells during passages. Source data are provided as a Source Data file.\" - \"To sample differences between ECs from the BM and ECs from other organs, we analyzed lung ECs. We cultured mouse primary lung ECs following standard procedures and found that they are predominantly constituted by Emcn cells. Comparison between BM- and lung-derived Emcn cells showed great similarity in their gene expression profile (Fig._). BMECs and lung ECs did not express genes highly expressed in MSC (Supplementary Fig._).\" - \"Analysis of Emcn BMECs transcriptomic profile during passages in vitro (Fig._ ) shows that the transition from P0 to P1 was the one marked by most significant changes whereas the transitions from P1 to P4 exhibited a uniform gene expression profile. Major changes in the transition between P0 and P1 include a significant increase in the senescence marker p16 (Cdkn2a; Fig._) and the downregulation of the Notch pathway (Fig._ and Supplementary Fig._). Principal Component Analysis (PCA) of the expression matrix for all samples (including lung and BM MSC) was used to visualize relationships between populations. The plot of the first two principal components revealed that the transcriptomes of P0 Emcn BMECs, P0 Emcn BMECs, and BM MSCs, clustered in distinct areas; P0 Emcn BMECs and Lung Emcn ECs clustered together; P1-P4 Emcn BMECs transcriptomes clustered together and far from both P0 Emcn and P0 Emcn BMECs (Fig._).\" - \"BMEC heterogeneity and identity are_preserved in culture\" - \"An important question is whether BMECs in culture depart from their native, in vivo characteristics. To address this point, we compared the transcriptomes of fresh and cultured BMEC by single-cell RNA-seq (scRNA-seq) (scheme in Fig._ ). First, we obtained single-cell transcriptomic data of 854 and 671 cells from two replicates, for a total of 1525 tdT cells, BMEC, freshly isolated from BM. The endothelial identity of cells marked by Tie2CreERT2 was substantiated by integrating our data with publicly available data on populations marked by perivascular cell-specific LepR-Cre and osteoblast specific Col2.3-Cre (Supplementary Fig._ ). By clustering cells using PCA and a nearest neighbor approach (as in Seurat), we identified five clusters (clusters 0 to 4) (Fig._). It is noteworthy that expression of Emcn clearly defined two populations within the fresh Tie2tdT cells: Emcn (clusters 1 and 3; nu2009=u2009323 and 249) and Emcn (cluster 0; nu2009=u2009615) (Fig._). Expression of Tek (Tie2) was evident in clusters 0, 1, and 3 but was below detection in cluster 2, which showed expression of the perivascular marker Lepr (nu2009=u2009284 cells), and in cluster 4, which exhibited expression of the hematopoietic marker Ptprc (CD45) (nu2009=u2009 54 cells; Fig._).\" - \"Fig. 8: Native BMEC identity is preserved in ex vivo culture.a Scheme of experimental design for freshly isolated BM tdT and P0 WBM cultured cells. Parts of the figure (the femur) were drawn by using pictures from Servier Medical Art by Servier, licensed under a Creative Commons Attribution 3.0 Unported License. b, c UMAP plot of single-cell gene expression profile from fresh cells. Clusters ranging from 0 to 4 identified by PCA and nearest neighbors are displayed in b using different colors. c Level of expression, reported as z-scaled log2 counts per million (CPM), of Emcn, Tek, Lepr, and Ptprc are shown in blue for each cluster in the UMAP visualization. d Gene expression profile for the sinusoidal/arterial gene signature. The levels of expression are reported as z-scaled log2 counts per million (CPM). e Integrated UMAP plot of the single-cell gene expression profile of fresh sorted P0tdT (Emcn and Emcn) (cyan) and cultured BMEC cells sorted at P0 (red). Numbers indicate clusters. f Level of Emcn expression, reported as z-scaled log2 counts per million (CPM), visualized in Integrated UMAP plot as in e. g, h Heatmaps of gene expression profiles in fresh and cultured BMECs using the arterial/sinusoidal signature (g) and the tip/stalk signature (h). Dendrograms above heatmaps show hierarchical clusters and the distance between the subpopulations transcriptomes. Results show differences between Emcn and Emcn and the linkage between freshly isolated and cultured cells of the respective groups. The levels of expression are reported as z-scaled log2 counts per million (CPM).\" - \"To define the cell types represented by each cluster, we visualized expression of gene signatures associated with arterial, sinusoidal, perivascular, and hematopoietic cells in a heatmap (Fig._). Cluster 0 (Emcn) highly expressed ~30% of the arterial markers represented in our compiled arterial/sinusoidal gene signature (Supplementary Table_), such as Serpinf1, Ace, Fbln2, Sparcl1, Ltbp4, Ly6a, and CD34 and did not express any of the sinusoidal associated genes (Fig._). Thus, we are referring to cluster 0 as arterial-like. Cluster 1 (Emcn) showed a 100% expression of the sinusoidal/venous markers presented in our signature, such as Ephb4, Flt4, Stab2, Sele, and others (Fig._); we are referring to cluster 1 as a sinusoidal-like. Intriguingly, cluster 3 (also Emcn) expressed ~30% of the sinusoidal markers and >80% of the arterial associated markers u2013 exhibiting a mixed but predominant arterial-like signature (Fig._); we are referring to cluster 3 as a u201C mixedu201D cluster. Distribution of representative arterial (Ly6a, Cd34, Sparcl1, Ltbp4, Podxl) and sinusoidal (Flt4, Stab2, Sele, Dnase1l3, Ephb4) markers among clusters is shown in Supplementary Fig._. Cluster 2 and 4 were tdT cells, likely contaminating populations from the sorting procedure and exhibited signatures for perivascular cells and hematopoietic genes, respectively (Fig._).\" - \"Next, we performed scRNA-seq of cultured tdT BMEC from P0 WBM culture (scheme in Fig._). Following two weeks of WBM culture (P0), Emcn and Emcn were sorted and admixed together at 1:1 proportion, in duplicate. The mixed populations were subjected to scRNA-seq, from which we recovered 6476 cells in total (2883 and 3593 cells from each replicate). The cultured P0 population (Emcn plus Emcn) was visualized in an integrated UMAP analysis together with fresh BM tdT cells, now defining 7 clusters (Fig._). Data showed clustering of cultured P0 cells with fresh BM tdT (cluster 5 and 0; cluster 6 and 1). UMAP visualization of Emcn expression (Fig._), indicated that fresh and cultured Emcn BMECs clustered together and fresh and cultured Emcn BMECs clustered together. Examination of arterial/sinusoidal signature gene expression from fresh and cultured tdT BMEC cells showed remarkably similar profiles (Fig._). We also determined the expression of genes defining the tip/stalk signature (Fig._). This approach further -discriminated the Emcn from Emcn BMECs. To correct for possible bias in the choice of gene subset, we identified the top 5 percentile of the most variable genes (Supplementary Fig._). The dendrogram structure above heatmaps (Fig._, and Supplementary Fig._) shows the linkage between fresh and cultivated cells of the respective groups (Emcn and Emcn) and the separation between Emcn and Emcn. Interestingly, we noted that a few genes flipped expression during the in vivo to in vitro transition (such as CD34, Ltbp4, and Il6st), and that Emcn BMEC acquired a stronger u201Ctipu201D signature and expression of some u201Carterialu201D genes, likely due to the pro-angiogenic conditions of the in vitro culture (Fig._ ). Surprisingly, there was no cultured BMEC cluster overlapping with fresh cluster 3, Emcn u201Cmixedu201D (Fig._), suggesting that this population is lost in culture.\" - \"Gene Set Enrichment Analysis (GSEA) in Emcn and Emcn BMEC revealed enrichment of pathways aligned with their distinct cell functions. Emcn BMEC are enriched in gene ontology (GO) pathways associated with cell migration, wound healing, sprouting and Rho proteins signals (Supplementary Fig._). Such pathways well describe their u201Ctipu201D behavior and the associated genes include u201C tipu201D markers such as Cxcr4, Angp2, Apln, Flt4, Delta ligands and Notch receptors (see Supplementary Table_), E2F genes and Cdc42, Rac, Rap, and Rho family members. Emcn BMECs are enriched in pathways associated with external incapsulating structure organization, extracellular matrix dissembling, cell adhesion and growth factors (Supplementary Fig._) that concord with their ability to form cords in Matrigel and their u201Cstalku201D behavior. Such pathways include most members of the Adam family of disintegrins and of the Mmp family of matrix metallopeptidase; hyaluronan synthases and collagens; Tgfu03B2 pathway, and Fgf family members, and arterial identity signature genes Efnb2, Sparcl, Caveolin, and il6st (see Supplementary Table_).\"- - \"Lack of robust methods to define ECs functions both in vivo and in vitro has limited the rigorous deployment of mouse models for mechanistic studies. Efforts to isolate and culture adult primary ECs from various organs by using immortalization strategies and genetic reprogramming have been constrained by the challenge of expanding ECs without losing their heterogeneity and their native, primary, characteristics. Here, we used a Tie2-CreERT2;Rosa26-tdTomato reporter mouse to isolate, profile, and cultivate primary ECs from different organs using an ex vivo cell culture strategy that preserves some key cell-to-cell interactions and conditions of their microenvironment in vivo and can harbor ECs with characteristics similar to primary cells in vivo.\" - \"In this study, we used BM as a paradigm. EC-specific Tie2 promoter-driven tdTomato expression was employed to track BMECs in a whole BM (WBM) cell culture system. This strategy allowed to demonstrate that primary BMECs can be cultured ex vivo without losing their heterogeneity and native molecular signatures. Notably, this approach enabled to culture sinusoidal and arterial BMECs and to map their functional properties, unveiling previously unrecognized features of these two populations (Summary in Fig._). WBM culture supports viability, proliferation, phenotype, and molecular profile of two BMEC populations that can be distinguished just by their distinct migratory behavior: one population forms 2D-networks and another forms cobblestone-like colonies. Expression of Endomucin (Emcn), a single transmembrane sialomucin expressed by capillaries and by venous but not arterial endothelium is detected only on BMECs forming 2D-networks. Immunophenotypic and transcriptomic analysis indicated that Emcn BMECs exhibit a sinusoidal-like profile whereas Emcn BMECs exhibit an arterial-like profile.\" - \"Fig. 9: Summary of WBM ex vivo culture workflow and BMEC distinct characteristics.ECs can be isolated from all organs via their Tie2Cre-driven tdTomato expression. In vivo, three major BMEC populations can be identified in the BM by scRNA-seq transcriptomic profile: Emcn sinusoidal-like, Emcn arterial-like and Emcn with mixed sinusoidal/arterial-like signature, which are no longer observed in ex vivo culture. BMECs require interactions with WBM to maintain their heterogeneity and their native molecular signatures ex vivo. Macrophages (Mu03A6) promote maintenance of Emcn sinusoidal-like BMEC and expansion of all BMECs, whereas mesenchymal cells (MC) have an inhibitory effect. Endomucin expression distinguishes BMEC in Emcn and Emcn populations exhibiting distinct behaviors, as summarized in the table shown at the bottom of the figure.\" - We observed that Emcn BMECs show limited life span and proliferation activity, whereas Emcn BMECs exhibit high proliferative potential and expand beyond four passages in culture. It is intriguing that Emcn cells, which exhibit a high capacity for cell migration and organization in 2D, cannot engage in in vitro angiogenesis, whereas Emcn cells, which do not form networks in 2D, have instead a great ability to form cord-networks. Emcn overexpression has been shown to promote migration, proliferation, and cords formation in human retinal ECs. Our discordant observations suggest that expression levels, tissue- and species-specificity are likely to influence the outcome of the studies. Interestingly, we observed that only Emcn, but not Emcn BMECs, participate in neo-angiogenesis in vivo, by joining endogenous EC to form blood vessels. Collectively, this ex vivo culture system highlights some mutually exclusive properties of sinusoidal-like Emcn and arterial-like Emcn cells and indicates that the ability of BMECs to form 2D-networks and to engage in in vitro or in vivo angiogenesis are processes regulated by independent mechanisms. - \"A key feature of this culture system, associated with its ability to sustain BMECs diversity ex vivo, is the presence of BM macrophages. Collectively, our experiments demonstrate a critical impact of macrophages on BMECs expansion and on the preservation of Emcn BMECs. This positive impact of BM macrophages is not surprising, given their known role in vascular repair and regeneration. Interestingly, it was unexpected that BM mesenchymal cells inhibited BMEC growth, given their positive role on HUVEC cultures. Thus, our work shows the importance of macrophages in supporting BMEC longevity, most likely by secreted factors, and unveil a cellu2013cell mediated inhibitory effect of mesenchymal cells, providing a platform to address specific molecular mechanisms in the future.\" - In the BM, EC are a critical component of the hematopoietic stem cell (HSC) niche. Attempts to recapitulate the interactions between BMECs and HSCs in vitro, particularly in mouse, have been challenged by the difficulty to isolate and maintain BMECs in vitro. This challenge has been partially circumvented by genetic manipulation approaches or by using EC from other organs. To date, primary BM-derived arteriolar and sinusoidal BMECs have never been directly compared for ability to support hematopoiesis. Here, we compared side-by-side the ability of sinusoidal-like Emcn and arterial-like Emcn BMECs to support HSPCs by using coculture experiments. We found that Emcn ECs proved to be superior in supporting maintenance of immunophenotypic HSPC in vitro. Not surprising, this property was associated with higher expression of Notch ligands, known to promote HSC maintenance. - \"A critical question in our studies was whether BMECs cultured ex vivo drift from their fresh counterpart losing their native identity. scRNA-seq analysis identified 3 major clusters in BMECs in vivo: (1) Emcn arterial-like ECs, (2) Emcn sinusoidal-like ECs, and (3) Emcn u201Cmixedu201D ECs, which expressed both arterial and sinusoidal genes. Hierarchical clustering analysis shows the linkage between fresh and cultivated cells of the respective groups (Emcn and Emcn), indicating a significant similarity in their in vivo and ex vivo transcriptome. Sinusoidal and arterial signatures are remarkably well conserved in Emcn and Emcn during ex vivo culture. Application of a u201Ctip/stalku201D signature, which characterizes the function of sprouting during vascular morphogenesis, further defines Emcn cells as u201C tip-likeu201D cells and Emcn cells as u201Cstalk-likeu201D cells, in both fresh and cultured cells. The observation of a u201Ctip-stalku201D signature in fresh cells from homeostatic vessel beds, which have no or little sprouting behavior, is intriguing. One speculation is that adult vessels at homeostasis maintain a baseline expression of a restricted group of u201Ctip/stalku201D genes u2013 perhaps memory of a sprouting occurred u2013 that can be poised to be induced to higher expression when a situation necessitating regeneration occurs. Despite their global transcriptomic similarity, in vivo and ex vivo BMEC are not identical. We observed that during ex vivo culture, the expression of sinusoidal and arterial markers became attenuated whereas the expression of u201Ctipu201D and u201Cstalku201D markers became accentuated in Emcn and Emcn cells, respectively. This data suggests that a u201Ctipu201D and u201Cstalku201D polarization of Emcn and Emcn BMEC occurs in culture, likely due to the angiogenic conditions of our ex vivo culture system that may artificially mimic a regenerative call.\" - \"Despite the lower representation of arterial-like Emcn ECs in fresh BM compared to sinusoidal-like Emcn ECs, and their limited proliferation in vivo, Emcn cells exhibit high proliferative potential in vitro, similarly to endothelial cell colony-forming units (ECFCs). It is possible that these cells, relative quiescent at steady state, can be activated during process of vessel regeneration in response to disease or injury and that the presence of angiogenic factors in culture triggers a similar proliferative/regenerative response in vitro. Interestingly, Emcn BMECs share similarity with the transcriptomic profile of an ECs population harvested from the melanoma tumor microenvironment, recently described by Khosrotehrani s group and named as Endothelial Vascular Progenitors (EVP). This elegant study showed that these EVP can give rise to more differentiated endothelial cells and contribute to neo-angiogenesis in vivo. However, in our study, we did not find evidence of Emcn generating Emcn ECs or contributing to in vivo angiogenesis. How the in vitro attributes uncovered here for Emcn BMECs relate to their in vivo properties remains elusive. We anticipate that our platform will help to elucidate the nature of these cells in future studies.\" - \"Here, we have observed that expression of Endomucin discriminates distinctively, and more accurately than Sca-1 expression, BMEC with sinusoidal- or arterial-associated genes. However, our approach has also unveiled another type of Emcn population, which expresses both arterial and sinusoidal genes. This rare population is observed only in vivo but it is no longer observed in ex vivo culture. This Emcn u201Cmixedu201D population, identified by scRNA-seq, exhibits a dual arterial/sinusoidal transcriptomic signature and may represent a transitional state from a common precursor or a potential ancestor of Emcn and Emcn population(s). Studies assessing the potential and functional characteristics of this subset, are warranted.\" - \\'In conclusion, we have developed a robust ex vivo method allowing to culture and compare two major EC subtypes in the BM: arterial-like and sinusoidal-like endothelial cells. Prospectively, this approach provides a valuable and amenable platform to perform mechanistic studies of processes like sprouting, angiogenesis, cell plasticity, and cellular crosstalk, which can be paired with drug screening and in vivo studies. Ultimately, this model represents a simple and effective tool enabling the use of primary ECs to study tumor angiogenesis and vascular regeneration in different pathologies and conditions.\\'- - Animals - Animal experiments were performed in compliance with the ethical regulations of the City of Hope Institutional Animal Care and Use Committee (IACUC). - \"Experiments were conducted in 4-6 months-old Tie2u2010CreERT2;Rosa26-tdTomato and Osx-Cre;Rosa26-tdTomato mice under the City of Hope IACUC 20023 approved protocol. We did not observe sex-associated differences in the isolation and characteristics of BM-derived EC. Thus, animals were used regardless of their sex.\" - Mouse generation - \"This specific Tie2-CreERT2 mouse strain was developed by Dr. Yi Zhengs laboratory. It has been generated by using the DNA fragment containing intron sequence of u03B2-globulin gene, Cre recombinase, mutated ligand binding domain of the estrogen receptor and SV40 polyA (klenow enzyme filled Stu I site- Xba I DNA fragment) excised from plasmid pCreER and inserted into the EcoRV-Xba I site of plasmid pBSTie2-2. The resulting plasmid was called pTie2P-CreER. The Sal I fragment containing Tie2 promoter and CreERT2 was excised from pTie2P-CreER and cloned upstream into the_SV40 polyA signal and the first intron sequence of pBSpolyATie2-10 to generate the transgenic construct. The final Tie2-CreER transgene, comprising the Tie2 promoter, u03B2-globulin intron sequence, CreER fusion gene, 2 polyadenylation signal (pA) fragments, and Tie2 enhancer sequence was excised from the vector and used for C57BL/6 pronuclear micro-injection to generate the transgenic mouse line.\" - \"Tie2u2010CreERT2;Rosa26-tdTomato mice were generated by crossbreeding Tie2u2010 CreERT2 mice with Rosa26u2010tdTomato mice (B6;129S6u2010Gt(ROSA)26Sor/J; Ai9 or Ai9 (RCL-tdT); Jackson Laboratory stock number: 007905). 25u2009u03BCg/g of Tamoxifen (Sigma, T5648) were injected into 2u20137-week-old Tie2u2010 CreERT2;Rosa26-tdTomato mice once a day for 5 days.\" - \"Osx-Cre;Rosa26-tdTomato mice were used to obtain bona-fide Osx-Cre-mediated tdT MSC for bulk RNA-seq. Osx-Cre mice were obtained from Dr. Ernestina Schipani. Osxu2010Cre;Rosa26-tdTomato mice were generated by crossbreeding Osxu2010 Cre mice with Rosa26u2010tdTomato mice (as above, Jackson Laboratory stock number: 007905).\" - Primary EC isolation and culture from tissues - \"Lungs, kidneys, hearts, spleens, livers, or SAT (subcutaneous adipose tissues) were harvested and washed with a copious amount of sterile ice-cold 1X PBS to remove the maximum amount of blood cells from the tissues. The washed tissues were minced using a scalpel blade or micro-dissection scissors in 60u2009mm sterile Petri plates under a sterile hood, incubated with a digestion buffer containing DMEM supplemented with 20u2009mM HEPES, 3u2009mg/mL collagenase I, BSA 1u2009mg/mL, and 0.1u2009mg/mL DNase I for 30u2009min at 37u2009xB0 C with gentle agitation (0,24u2009g). The digested tissues were filtered via a 70 u03BCm cell strainer. The supernatant was collected and centrifuged at 135u2009g for 5u2009min. Cell pellets were dissolved in Endothelial Cell Growth Medium-2 (EGM-2) BulletKit (Lonza, CC-3156 & CC-4176) including EBM-2 Basal medium, FBS 10u2009mL, Hydrocortisone 0.2u2009mL, hFGF-B 2u2009mL, VEGF 0.5u2009 mL, R3-IGF-1 0.5u2009mL, Ascorbic Acid 0.5u2009mL, hEGF 0.5u2009mL, GA-1000 0.5u2009mL, Heparin 0.5u2009mL, and 1% Penicillin/Streptomycin. Cell suspensions were plated on collagen type I-coated (Corning) or 0.1% gelatin (Millipore)-coated culture dishes and cultured at 37u2009xB0C with 5% CO2 in an incubator (Panasonic). One mouse usually generates one 10u2009cm dish for lung, kidney, and heart, and one 6u2009cm dish for SAT. After 2 days, half (v/v) medium was added. On the day 4u20135, half of cell suspension (v/v) was removed from the culture and replenished by fresh EGM-2 medium. Half (v/v) medium changes were performed every 2u20133 days for approximately 2 weeks until confluent.\" - Primary BMEC culture - \"Protocol to harvest fresh tdTomato ECs from mouse BM: Femurs and tibias isolated from Tie2-CreERT2;Rosa26-tdTomato mice were collected into sterile cold PBS with 2% FBS and gently crushed using a mortar and pestle. To avoid damaging the ECs, especially those ones localized just beneath the bones, crushing was performed briefly and very gently after reducing each bone into 2-3 pieces. Note that different strengths in crushing can lead to different results. Cell suspensions were then filtered into a 50u2009mL tube on ice using a 70u2009 u03BCm cell strainer and resuspended in 5u2009mL of EGM-2. The remaining crushed bone pieces were collected into a 15u2009mL tube and digested for 10u2009 min in 1u20132u2009mL/mouse bone digestion buffer (containing DMEM supplemented with 20u2009mM HEPES, 3u2009mg/mL collagenase I, BSA 1u2009mg/mL, and 0.1u2009 mg/mL DNase I) at 37u2009xB0C and with gentle agitation (0,24u2009g). The rack in the shaking incubator was adjusted in a such way that the 15u2009mL tubes containing the crushed bones were positioned at a 45 degrees angle. After 10u2009 min, the bone digested mixture was brought to a 10u2009mL volume in PBS containing 2% FBS, filtered with a 70u2009u03BCm cell strainer and added to the cell suspension previously prepared into the 50u2009mL tube. A second digestion was repeated on the bone pieces in 1u20132u2009mL of digestion media for 30u2009 min at 37u2009xB0C with stronger agitation (0,42u2009g) and at a 45 degrees angle. The cells suspension derived from the bone digestion was washed again in PBS with 2% FBS and added to the previous 50u2009mL tube after filtration through the same 70u2009u03BCm cell strainer on ice. The digested bone pieces were collected back into the mortar and further crushed in PBS with 2% FBS. This crushing was performed more vigorously to harvest all the hematopoietic cells from the bones. The supernatant was passed through the same 70u2009u03BC m cell strainer and pooled into the same 50u2009mL tube, on ice. PBS with 2% FBS was added through the same cell strainer to fill the 50u2009mL tube up to 50u2009mL. The whole cell suspension was centrifuged at 135u2009g for 5u2009 min and the cell pellet was resuspended in EGM-2 medium for counting.\" - \"On average, the number of BMECs (CD45Ter119tdT cells) was 1u2009xD7u200910 cells/mouse (range 8u2009xD7u200910u221211u2009xD7u200910; Supplementary Table_ and ). On average, the number of BMEC sorted per mouse was 5598 (Supplementary Table_).\" - \"WBM Culture: All cells obtained and pooled from crashed and digested bones (WBM cells), as described above, were resuspended in EGM-2 media, plated on collagen type I-coated (Corning), or 0.1% gelatin, (Millipore)-coated culture dishes at the density of 6u20138u2009xD7u200910/10u2009cm dish, and cultured at 37u2009xB0C with 5% CO2 in an incubator (Panasonic). Usually one mouse (2 femurs and tibias) was necessary for one 10u2009cm dish. After 2 days, half (v/v) medium was added. On the day 4u20135, half of cell suspension (v/v) was removed from the culture and replenished by fresh EGM-2 medium. Half (v/v) medium changes were performed every 2-3 days for approximately 2 weeks until confluent. At that point, the culture, consisting entirely of adherent cells, was gently trypsinized using 0.05% Trypsin/0.53u2009mM EDTA (Corning) and replated at a density of 1u2009xD7u200910/10u2009cm dish. Cultures were passaged every 1u20132 weeks at confluency. All passages and sorted cells from P0 were cultured for 1u20132 weeks until confluent with half medium changes every 2u20133 days. Note that results may vary if a different endothelium medium is used.\" - \"Cultures of sorted BM tdTomato+ BMECs: tdTomato BMECs (T0) sorted from fresh BM were directly seeded at the density of ~5000 cells/well in 24-well-plates and cultured in EGM-2 medium at 37u2009xB0C with 5% CO2 in an incubator (Panasonic) by changing half (v/v) medium every 2u20133 days for around 2 weeks until confluent.\" - \"Addition of supplementary HC/Mu03A6 or MC into P0 WBM culture: CD11b cells were isolated from fresh BM using EasySepu2122 Mouse FITC Positive Selection Kit II (STEMCELL). In each well of 12-well-plates ~2u2009xD7u200910 or ~1u2009 xD7u200910 WBM cells were supplemented with 4u2009xD7u200910 CD11b cells (equivalent to a final density 1/2 and 1/4, respectively). DAPICD45tdTomato cells (MC) were sorted from a P0 culture at day 14 and ~1.5u2009xD7u2009 10 were added to ~2u2009xD7u200910 or ~1u2009xD7u200910 WBM cells (equivalent to density 1/2 and 1/4, respectively). Cells in suspension were removed at day 2 and 4 of culture, in such a way that only adherent CD11b cells were selected. Cultured cells were detached at day 14 for cell counting and FACS analysis. Following CD11b cell supplement, the number of Mu03A6 in the WBM at day 14 was increased ~2-fold compared to standard WBC cultures, while the number of MC was enhanced ~4-fold. Wells at low density did not reach confluency. In experiments using a transwell, the same numbers of the isolated CD11b cells or MC were added to the transwell membranes posed over 12 well/plate wells seeded with WBM (as above) and cultured until day 14.\" - \"Addition of supplementary HC/Mu03A6 into P1 WBM culture: CD11b cells were isolated as described above and plated at a density of 1u2009xD7u200910/well in 6-well-plates. Cells in suspension were removed on the next day while the adherent CD11b cells were combined with 1.8u2009xD7u200910, 4.5u2009xD7u200910, or 2.25u2009 xD7u200910 adherent cells harvested from WBM P0 at day 14 (cell density 1, 1/4, and 1/8, respectively, as indicated in Fig._) and cultured for 2 weeks (P1).\" - \"HSPC coculture: DAPICD45tdTomatoEmcn and DAPICD45tdTomatoEmcn cells sorted from day 14 P0 culture were seeded at a density of 5u2009xD7u200910/well in 12-well-plates for approximately 3 days (until confluence). LSK cells (~6000 cells) sorted from fresh BM of adult wild-type mice (2u20135 months old) were added to the culture for 7 or 14 days in the absence or presence of SCF (50u2009ng/ml).\" - Primary BM mesenchymal cell culture - \"Femurs and tibias isolated from Osx-Cre;Rosa26-tdTomato mice were collected into sterile cold PBS with 2% FBS and gently crushed using a mortar and pestle. Cell suspensions were then filtered into a 50u2009mL tube on ice using a 70u2009 u03BCm cell strainer while the crushed bone pieces were incubated at 37u2009 xB0C for 40u2009min with gentle agitation (0,24u2009g) in 1u20132u2009mL/mouse of bone digestion buffer containing DMEM supplemented with 20u2009mM HEPES, 3u2009mg/mL collagenase I, BSA 1u2009mg/mL, and 0.1u2009mg/mL DNase I. The cells suspension derived from the bone digestion was washed again in PBS with 2% FBS and added to the previous 50u2009mL tube after filtration through the same 70u2009u03BCm cell strainer on ice. The digested bone pieces were collected back into the mortar and further crushed in PBS with 2% FBS. The supernatant was passed through the same 70u2009u03BCm cell strainer and gathered into the same 50u2009mL tube on ice. The whole cell suspension was centrifuged at 135u2009g for 5u2009min and the cell pellet was dissolved in Mesencult medium (MesenCultu2122 Expansion Kit -Mouse; Stem Cell Technologies; #05513) and plated on collagen type I-coated (Corning) or 0.1% gelatin (Millipore)-coated culture dishes at the density of 6u20138u2009xD7u200910/10u2009cm dish (usually 2 legs per mouse per 10u2009cm dish) and started to be cultured at 37u2009 xB0C with 5% CO2 in an incubator (Panasonic). After 2 days, half (v/v) medium was added. On the day 4u20135, half of cell suspension (v/v) was removed from the culture and replenished by fresh Mesencult medium. Half (v/v) medium changes were performed every 2-3 days for ~2 weeks until confluent. At that point, the culture, consisting entirely of adherent cells, was gently trypsinized using 0.05% Trypsin/0.53u2009mM EDTA (Corning) and DAPICD45tdTomato cells were sorted by FACSAria Fusion (BD Biosciences).\" - Flow cytometry and cell sorting - \"Fresh BMEC cells were harvested as described above. At day 14, P0 cultured cells were detached by Trypsin-EDTA (0.05% trypsin, 0.02% EDTA) for 5u2009min and further detached by pipetting. Cells were then suspended in PBS containing 2% FBS at 1u2009xD7u200910 cells/mL and stained for 30u2009min on ice with the following antibodies: APC-antiu2010CD45 plus APC-anti-Ter119 (to eliminate hematopoietic and erythroid cells), and Alexa-488-anti-Emcn. DAPICD45Ter119tdTomato cells from fresh BM and the following populations from P0 cultured cells: DAPICD45tdTomatoEmcn, DAPICD45tdTomatoEmcn, DAPICD45tdTomato, and DAPICD45tdTomato were sorted by FACSAria Fusion (BD Biosciences). Cell number yields of sorted fresh or cultured BMEC populations are summarized in Supplementary Tables_ and . Complete characterization of BMECs was performed by using the following antibodies: CD45, Ter119, CD31, CD144, CD140a, CD105, Sca-1, Emcn, Flk-1, CD14, F4/80, and CD11b. Cells were acquired by LSRII (BD Biosciences) and analyzed with FlowJo. For LSK sorting, DAPILinSca-1cKit cells from lineage-depleted fresh BM cells using EasySepu2122 Mouse Hematopoietic Progenitor Cell Isolation Kit (STEMCELL) were sorted by FACSAria Fusion (BD Biosciences). 1 u03BCL of each antibody was used per 1u2009 xD7u200910 cells. Detailed information of antibodies is shown in Supplementary Table_. Gating strategies are shown in Supplementary Fig._.\" - Immunofluorescence staining - \"Frozen sections. Bones were isolated and immediately fixed overnight in cold 4% paraformaldehyde/PBS at 4u2009xB0C. The following day, bones were washed 3 times for 5u2009min with cold PBS and decalcified in cold 0.5u2009M EDTA (pH 7.4u20137.6) at 4u2009xB0C for 24-48u2009h with gentle agitation. Subsequently, the decalcified bones were washed three times with cold PBS and cryoprotected in PBS containing 20% sucrose and 2% PVP at 4u2009xB0C for 24-48u2009h. Bones were then embedded in PBS containing 8% gelatin, 20% sucrose, and 2% PVP, incubated at RT for 30u2009min, and transferred to u221280u2009xB0C for storage. Frozen bones were cut into sections with a thickness of 70u2013100u2009u03BCm using a Leica CM3050-S cryostat and high profile cryotome blades. Frozen sections were dried at RT for 30u2009min, rehydrated with PBS at RT for 5u2009min, permeabilized by PBS containing 0.3% (v/v) Triton X-100 at RT for 20u2009min, incubated in the blocking solution made of PBS containing 10% goat or donkey serum, 5% BSA and 0.01% Triton X-100, at RT for 1u2009h. The slides were incubated with primary antibodies at 4u2009xB0C overnight, washed with PBS, and incubated with secondary antibodies at RT for 75u2009min. DAPI staining was performed at RT for 30u2009min after PBS washing. VECTASHIELD Antifade Mounting Medium (Vector Laboratories) was used for mounting. Images were captured by a confocal microscope LSM880 (Zeiss) and analyzed by a ZEN (Zeiss) software. For immunostaining, all primary antibodies for Emcn, CD31, Sma, VE-Cadherin, and Laminin were used at 1:100 dilution, and all secondary antibodies at 1:400 dilution. Detailed information of antibodies is shown in Supplementary Table_.\" - Time lapse microscopy and image analysis - \"A 3.5u2009cm culture dish for monitoring cell growth of P0 WBM culture was placed in the chamber of an inverted Axio Observer 7 microscope (Carl Zeiss) at 37u2009 xB0C and 5% CO2. Tiled images with z-slices were acquired as 20 per tile every 45u2009min for 8 days (day 5u201313 of P0 WBM culture). The acquired images were stitched and processes with orthogonal projections for maximum intensity using ZEN (Zeiss) software and included_as Supplementary Movie_u2013.\" - In vitro Matrigel tube (cord) formation assay and analysis - \"Each well of u03BC-Slide Angiogenesis (Ibidi) was coated with 10u2009u03BC L of Growth factor-reduced Matrigel (Corning) on ice and placed in the humidified chamber at 37u2009xB0C for 1u2009h to be polymerized. Cells were counted, suspended in the EGM-2 medium at a concentration of 3u2009xD7u200910/mL, and carefully applied 50 u03BCL into the Matrigel-coated well. The Lid-covered u03BC-Slide Angiogenesis was incubated at 37u2009xB0C and 5% CO2 for 24u2013 48u2009h. Tiled images with z-slices were acquired as 20 per tile every 45u2009 min, stitched, and processed with orthogonal projections for maximum intensity using ZEN (Zeiss) software and made as Supplementary Movie_.\" - Original images obtained by time lapse microscopy for all observation times underwent contrast-limited adaptive histogram equalization, cropping of the growth boundary, and gaussian blurring. Next, images were converted to binary and skeletonized through a process of repeated erosion operations until prior to disconnection. This allowed kinematic analysis of endothelial cell cords/tube formation by quantification of changes in cellular connectivity. The network metric analyzed for quantification is the distribution of areas enclosed by fully connected network cycles (i.e., areas contained in loops of connectivity). Alterations in connectivity, due to either less formation of complete enclosures in the matrix, or thickening in the matrix connections), result in a smaller measure of enclosed area and expressed as arbitrary units. This metric is accessible using the pre-packaged contour detection functionality of the open-source software OpenCV. - In vivo Matrigel plug assay - \"2u2009xD7u200910 BM ECs were mixed with 400u2013500u2009u03BCL of Standard Matrigel (Corning) on ice. The EC-containing Matrigel was slowly injected subcutaneously into the right-shaved flank of the mouse, which was anesthetized with 5% of inhalant isoflurane and kept under 3% isoflurane anesthesia during all procedure. The left side flank of the same mouse was used for Matrigel injection without any cell as internal control. After 2 months, the Matrigel plugs were collected, trimmed by removing connective and adipose tissues surrounding the plugs, briefly washed by PBS, subjected to imaging, embedded into OCT compound, and stored at u221280u2009xB0C for the preparation of frozen section.\" - RNA extraction and qRT-PCR - \"Total RNAs were extracted using Trizol (Invitrogen) according to the manufacturer s instructions. The RNAs were converted into cDNA using the iScript cDNA Synthesis Kit (Bio Rad). qRT-PCR was performed with FastStart Essential DNA Green Master (Roche) using Light Cycler 96 (Roche). Primer information is shown in Supplementary Table_.\" - Bulk RNA-seq and scRNA-seq - \"RNA-seq and scRNA-seq libraries were prepared as previously described. For RNA-seq, total RNAs were extracted using the Trizol reagent. RNA integrity and quantity were determined by using the RNA screentape on a 4200 TapeStation system (Agilent). Samples with RNA integrity number above 7 were used for library construction. The RNA-seq libraries were generated with the KAPA RNA HyperPrep Kit (Roche, no. KK8581). Libraries were sequenced on an Illumina HiSeq X10 (for the 1 batch of biological replica) and Illumina Hiseq2500 with Rapid mode (for the 2 and 3 batch of biological replica). For scRNA-seq, DAPICD45Ter119tdTomato sorted cells from the fresh (without culture) bone marrows of two 4-month-old Tie2-CreERT2;Rosa26-tdTomato mice were loaded on the Chromium controller (10XGenomics) based on the number provided by the sorting equipment. Libraries were prepared using the Chromium Single Cell 3u2032 Reagent Kits (v3): Single Cell 3u2032 GEM, Library & Gel Bead Kit v3 (PN-1000075), Chromium Chip B Single Cell Kit (PN-1000073) and i7 Multiplex Kit (PN-120262) (10x Genomics) and following the Chromium Single Cell 3u2032 Reagent Kits (v3) User Guide (manual part no. CG000183 Rev A). Libraries were run on an Illumina HiSeq 2500 as 150-bp paired-end reads, at 20,000 read pairs per cell.\" - \"Bulk RNAseq analysis: To assess the quality of the sequencing libraries, the default parameters of the nf-core RNASeq pipeline, v1.4.2were used. Briefly, trimmed reads were mapped to the GRCm38 reference using STAR and abundance of transcripts from the Ensembl annotation (v81) was estimated using Salmon. Each library was subjected to extensive quality control, including estimation of library complexity, sequence quality, read length and depth, among other metrics detailed in the pipeline repository (). Pseudocounts were merged into a matrix of counts per million per gene for each sample using the tximport R package. Heatmaps were drawn using the ComplexHeatmap package after transforming counts with the vst function from DESeq2; the same transformation was used to visualize samples in PCA space. Differential expression was assessed using the default parameters of DESeq2. We called differentially expressed genes using merged counts with a Benjamini-Hochberg false discovery rate cutoffu2009ofu20090.05 and absolute shrunken log2-fold change (LFC)u2009u2009>u2009u20091.5 between P0 Emcn and each of the remaining samples. Shrunken LFC was obtained using the ashr adaptive shrinkage estimator implemented in DESeq2.\" - \"Single-cell RNAseq analysis: Processing of the scRNAseq reads was performed using the 10XGenomics Cell Ranger pipeline (v5.0.0) and visualized in R with the Seurat package (v4.0.5). Briefly, FASTQ files were generated using cellranger mkfastq with default parameters and cellranger count was run using the reference genome and annotation supplied (mm10-2.1.0) for each sample, counting UMIs for each gene. The output for each sample was integrated using cellranger aggr with default options and imported into R using Seurat for further analysis. Data for perivascular (Lepr-expressing) and osteoblast (Col1a1-expressing) cells were downloaded from the Gene Expression Omnibus (GSE108891) and imported to R. We kept cells where mitochondrial transcripts were <20%, > 200 genes expressed, <5% ribosomal transcripts, and genes expressed > 3 cells. To control for batch effects across experiments and between laboratories, the top 2000 variable genes shared by all samples under consideration were identified using the SelectIntegrationFeatures function. Integration anchors were identified based on these genes using the FindIntegrationAnchors function, employing the log normalization method. Samples were then integrated using the IntegrateData function. Principal component analysis (PCA) and uniform manifold approximation and projection (UMAP) dimension reduction with the top 20 principal components were performed. A nearest-neighbor graph using the 20 dimensions of the PCA reduction was calculated using FindNeighbors, followed by clustering using FindClusters. Where fewer dimensions of the PCA were informative, as indicated by an u201Celbowu201D in a plot of standard deviation vs principal components, we adjusted the FindNeighbours function accordingly. We used the doHeatmap and FeaturePlot functions to visualize the results of the clustering procedure and z-scaled expression per cell. Marker genes for each cluster were identified by the FindAllMarkers function using the Wilcoxon rank sum test. Overrepresented KEGG pathways and GO biological process categories for the 100 markers with the greatest log2FC (FDRu2009<u20090.05) in each cluster were identified by enrichKEGG and enrichGO functions in the clusterProfiler package (v4.2). Cell cycles were classified with the cyclone function from the scran R package using the default parameters and the mouse gene pairs included in the package.\" - Statistical analysis - \"Statistical analyses were performed using the Students t test, non-parametric Wilcoxon test and Peason r correlation using GraphPad software. Data is shown as meanu2009xB1u2009SD. Symbols represent: *pu2009<u20090.05, **pu2009 <u20090.01, ***pu2009<u20090.001, and ****pu2009<u20090.0001.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - \"The raw data and processed scRNA-seq and bulk data generated in this study have been deposited in the Gene Expression Omnibus (GEO) database under the accession number , which can be assessed with no restriction. The source data underlying Figs._b, d-f, a-c, e-f, c-g, c, e-f, d, , and Supplementary Figs._, , -b, -f, are in the Source Data file provided with this paper._ are provided with this paper.\" - \\' \\'- - \"Kalucka, J. et al. Single-cell transcriptome atlas of murine endothelial cells. Cell 180, 764u2013779.e720 (2020)._ _ _ _ Nolan, D. J. et al. Molecular signatures of tissue-specific microvascular endothelial cell heterogeneity in organ maintenance and regeneration. Dev. Cell 26, 204u2013219 (2013)._ _ _ _ Morrison, S. J. & Scadden, D. T. The bone marrow niche for haematopoietic stem cells. Nature 505, 327u2013334 (2014)._ _ _ _ _ _ Kubota, Y., Takubo, K. & Suda, T. Bone marrow long label-retaining cells reside in the sinusoidal hypoxic niche. Biochem. Biophys. Res. Commun. 366, 335u2013339 (2008)._ _ _ _ Itkin, T. et al. Distinct bone marrow blood vessels differentially regulate haematopoiesis. Nature 532, 323u2013328 (2016)._ _ _ _ _ _ Kunisaki, Y. et al. Arteriolar niches maintain haematopoietic stem cell quiescence. Nature 502, 637u2013643 (2013)._ _ _ _ _ _ Bowers, E. et al. Granulocyte-derived TNFalpha promotes vascular and hematopoietic regeneration in the bone marrow. Nat. Med. 24, 95u2013102 (2018)._ _ _ _ Himburg, H. A. et al. Distinct bone marrow sources of pleiotrophin control hematopoietic stem cell maintenance and regeneration. Cell Stem Cell 23, 370u2013 381 e375 (2018)._ _ _ _ _ Kopp, H. G. et al. Tie2 activation contributes to hemangiogenic regeneration after myelosuppression. Blood 106, 505u2013513 (2005)._ _ _ _ _ Duarte, D. et al. Inhibition of endosteal vascular niche remodeling rescues hematopoietic stem cell loss in AML. Cell Stem Cell 22, 64u201377 e66 (2018)._ _ _ _ _ Pitt, L. A. et al. CXCL12-producing vascular endothelial niches control acute T cell leukemia maintenance. Cancer Cell 27, 755u2013768 (2015)._ _ _ _ _ Veiga, J. P., Costa, L. F., Sallan, S. E., Nadler, L. M. & Cardoso, A. A. Leukemia-stimulated bone marrow endothelium promotes leukemia cell survival. Exp. Hematol. 34, 610u2013621 (2006)._ _ _ _ Winkler, I. G. et al. Vascular niche E-selectin regulates hematopoietic stem cell dormancy, self renewal and chemoresistance. Nat. Med. 18, 1651u2013 1657 (2012)._ _ _ _ Hauser, S., Jung, F. & Pietzsch, J. Human endothelial cell models in biomaterial research. Trends Biotechnol. 35, 265u2013277 (2017)._ _ _ _ Conchinha, N. V. et al. Protocols for endothelial cell isolation from mouse tissues: brain, choroid, lung, and muscle. STAR Protoc. 2, 100508 (2021)._ _ _ _ _ Poulos, M. G. et al. Endothelial Jagged-1 is necessary for homeostatic and regenerative hematopoiesis. Cell Rep. 4, 1022u2013 1034 (2013)._ _ _ _ _ Almeida-Porada, G. & Ascensao, J. L. Isolation, characterization, and biologic features of bone marrow endothelial cells. J. Lab Clin. Med. 128, 399u2013407 (1996)._ _ _ _ Kobayashi, H. et al. Angiocrine factors from Akt-activated endothelial cells balance self-renewal and differentiation of haematopoietic stem cells. Nat. Cell Biol. 12, 1046u20131056 (2010)._ _ _ _ _ Palikuqi, B. et al. Adaptable haemodynamic endothelial cells for organogenesis and tumorigenesis. Nature 585, 426u2013432 (2020)._ _ _ _ _ _ Yoder, M. C. et al. Redefining endothelial progenitor cells via clonal analysis and hematopoietic stem/progenitor cell principals. Blood 109, 1801u20131809 (2007)._ _ _ _ _ Fadini, G. P., Losordo, D. & Dimmeler, S. Critical reevaluation of endothelial progenitor cell phenotypes for therapeutic and diagnostic use. Circ. Res. 110, 624u2013637 (2012)._ _ _ _ _ Morgan, S. M., Samulowitz, U., Darley, L., Simmons, D. L. & Vestweber, D. Biochemical characterization and molecular cloning of a novel endothelial-specific sialomucin. Blood 93, 165u2013 175 (1999)._ _ _ _ Kusumbe, A. P., Ramasamy, S. K. & Adams, R. H. Coupling of angiogenesis and osteogenesis by a specific vessel subtype in bone. Nature 507, 323u2013328 (2014)._ _ _ _ _ _ Leppanen, V. M., Saharinen, P. & Alitalo, K. Structural basis of Tie2 activation and Tie2/Tie1 heterodimerization. Proc. Natl Acad. Sci. USA 114, 4376u20134381 (2017)._ _ _ _ _ _ Hochstetler, C. L. et al. KRas(G12D) expression in the bone marrow vascular niche affects hematopoiesis with inflammatory signals. Exp. Hematol. 79, 3u201315 e14 (2019)._ _ _ _ _ Sabbagh, M. F. et al. Transcriptional and epigenomic landscapes of CNS and non-CNS vascular endothelial cells. eLife 7, e36187 (2018).Manconi, F., Markham, R. & Fraser, I. S. Culturing endothelial cells of microvascular origin. Methods Cell Sci. 22, 89u201399 (2000)._ _ _ _ DeCicco-Skinner, K. L. et al. Endothelial cell tube formation assay for the in vitro study of angiogenesis. J. Vis. Exp. 91, e51312 (2014).Arnaoutova, I. & Kleinman, H. K. In vitro angiogenesis: endothelial cell tube formation on gelled basement membrane extract. Nat. Protoc. 5, 628u2013635 (2010)._ _ _ _ Ponce, M. L. Tube formation: an in vitro matrigel angiogenesis assay. Methods Mol. Biol. 467, 183u2013188 (2009)._ _ _ _ Nowak-Sliwinska, P. et al. Consensus guidelines for the use and interpretation of angiogenesis assays. Angiogenesis 21, 425u2013532 (2018)._ _ _ _ Nombela-Arrieta, C. et al. Quantitative imaging of haematopoietic stem and progenitor cell localization and hypoxic status in the bone marrow microenvironment. Nat. Cell Biol. 15, 533u2013543 (2013)._ _ _ _ _ Ramalingam, P., Butler, J. M. & Poulos, M. G. Vascular regulation of hematopoietic stem cell homeostasis, regeneration, and aging. Curr. Stem Cell Rep. 7, 194u2013 203 (2021)._ _ _ _ Wang, J., Niu, N., Xu, S. & Jin, Z. G. A simple protocol for isolating mouse lung endothelial cells. Sci. Rep. 9, 1458 (2019)._ _ _ _ _ Tikhonova, A. N. et al. The bone marrow microenvironment at single-cell resolution. Nature 569, 222u2013228 (2019)._ _ _ _ _ _ Park-Windhol, C. et al. Endomucin inhibits VEGF-induced endothelial cell migration, growth, and morphogenesis by modulating VEGFR2 signaling. Sci. Rep. 7, 17138 (2017)._ _ _ _ _ Oishi, Y. & Manabe, I. Macrophages in inflammation, repair and regeneration. Int Immunol. 30, 511u2013528 (2018)._ _ _ _ Bohrnsen, F. & Schliephake, H. Supportive angiogenic and osteogenic differentiation of mesenchymal stromal cells and endothelial cells in monolayer and co-cultures. Int. J. Oral. Sci. 8, 223u2013230 (2016)._ _ _ _ Acar, M. et al. Deep imaging of bone marrow shows non-dividing stem cells are mainly perisinusoidal. Nature 526, 126u2013130 (2015)._ _ _ _ _ _ Kusumbe, A. P. et al. Age-dependent modulation of vascular niches for haematopoietic stem cells. Nature 532, 380u2013 384 (2016)._ _ _ _ _ _ Fernandez, L. et al. Tumor necrosis factor-alpha and endothelial cells modulate Notch signaling in the bone marrow microenvironment during inflammation. Exp. Hematol. 36, 545u2013 558 (2008)._ _ _ _ _ Weber, J. M. & Calvi, L. M. Notch signaling and the bone marrow hematopoietic stem cell niche. Bone 46, 281u2013285 (2010)._ _ _ _ Eilken, H. M. & Adams, R. H. Dynamics of endothelial cell behavior in sprouting angiogenesis. Curr. Opin. Cell Biol. 22, 617u2013625 (2010)._ _ _ _ Yoon, C. H. et al. Synergistic neovascularization by mixed transplantation of early endothelial progenitor cells and late outgrowth endothelial cells: the role of angiogenic cytokines and matrix metalloproteinases. Circulation 112, 1618u20131627 (2005)._ _ _ Wang, X. et al. Simultaneous isolation of mesenchymal stem cells and endothelial progenitor cells derived from murine bone marrow. Exp. Ther. Med 16, 5171u20135177 (2018)._ _ _ _ Donovan, P. et al. Endovascular progenitors infiltrate melanomas and differentiate towards a variety of vascular beds promoting tumor metastasis. Nat. Commun. 10, 18 (2019)._ _ _ _ _ _ Patel, J. et al. Functional definition of progenitors versus mature endothelial cells reveals key SoxF-dependent differentiation process. Circulation 135, 786u2013805 (2017)._ _ _ _ Nakashima, K. et al. The novel zinc finger-containing transcription factor osterix is required for osteoblast differentiation and bone formation. Cell 108, 17u201329 (2002)._ _ _ _ Nunez-Iglesias, J., Blanch, A. J., Looker, O., Dixon, M. W. & Tilley, L. A new Python library to analyse skeleton images confirms malaria parasite remodelling of the red blood cell membrane skeleton. PeerJ 6, e4312 (2018)._ _ _ _ Calandrelli, R. et al. Stress-induced RNA-chromatin interactions promote endothelial dysfunction. Nat. Commun. 11, 5211 (2020)._ _ _ _ _ _ Ewels, P. A. et al. The nf-core framework for community-curated bioinformatics pipelines. Nat. Biotechnol. 38, 276u2013278 (2020)._ _ _ _ Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15u201321 (2013)._ _ _ _ Patro, R., Duggal, G., Love, M. I., Irizarry, R. A. & Kingsford, C. Salmon provides fast and bias-aware quantification of transcript expression. Nat. Methods 14, 417u2013419 (2017)._ _ _ _ _ Soneson, C., Love, M. I. & Robinson, M. D. Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. F1000Res 4, 1521 (2015)._ _ _ Gu, Z., Eils, R. & Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinformatics 32, 2847u2013 2849 (2016)._ _ _ _ Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014)._ _ _ _ Stephens, M. False discovery rates: a new deal. Biostatistics 18, 275u2013294 (2017)._ _ _ Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573u20133587 e3529 (2021)._ _ _ _ _ Yu, G., Wang, L. G., Han, Y. & He, Q. Y. clusterProfiler: an R package for comparing biological themes among gene clusters. OMICS 16, 284u2013287 (2012)._ _ _ _ _ Lun, A. T., McCarthy, D. J. & Marioni, J. C. A step-by-step workflow for low-level analysis of single-cell RNA-seq data with Bioconductor. F1000Res 5, 2122 (2016)._ _ _ \"- - \\'This work was supported by grant NHLBI R01 DK097837 and HL141379 (N.C.), R01 HL145170 and a MCBC Pilot Project from City of Hope (Z.B.C.) and CIRM EDU4-12772 (HK). We thank the shared facility resources at City of Hope: Animal Resource Core, Flow Cytometry and Sorting Core, Microscopy and Imaging Core and Integrative Genomics Core supported by the NCI/NIH grant P30CA033572. We thank Dr. Ian Talisman for editing and Dr. Nora Heisterkamp for critical review of the manuscript.\\'- - Author notes - \\'Young-Woong KimPresent address: Center for Genome Engineering, Institute for Basic Science, Yuseong-gu, Daejeon, 34126, Republic of KoreaAlex B. BrummerPresent address: Department of Physics and Astronomy, College of Charleston, Charleston, SC, 29424, USA\\' - Authors and Affiliations - \"Department of Stem Cell Biology and Regenerative Medicine, Gehr Family Center for Leukemia Research, Beckman Research Institute, City of Hope, Duarte, CA, 91010, USAYoung-Woong Kim,_Greta Zara,_HyunJun Kang_&_Nadia CarlessoDepartment of Computational and Quantitative Medicine, Division of Mathematical Oncology, Beckman Research Institute, City of Hope, Duarte, CA, 91010, USASergio Branciamore,_ Alex B. Brummer_&_Russell RockneCenter for Gene Therapy, Beckman Research Institute, City of Hope, Duarte, CA, 91010, USADenis OMeally_&_Angelo A. CardosoDivision of Experimental Hematology and Cancer Biology, Cincinnati Childrens Hospital Medical Center, Cincinnati, OH, 45229, USAYuxin Feng_&_Yi ZhengDepartment of Neuroscience, Center for Brain Immunology and Glia (BIG), University of Virginia School of Medicine, Charlottesville, VA, 22908, USAChia-Yi KuanDepartment of Diabetes Complications and Metabolism, Beckman Research Institute, City of Hope, Duarte, CA, 91010, USAYingjun Luo_&_Zhen Bouman ChenLight Microscopy Core, Beckman Research Institute, City of Hope, Duarte, CA, 91010, USAMichael S. NelsonIrell and Manella Graduate School of Biological Sciences, Duarte, USAZhen Bouman Chen,_ Angelo A. Cardoso_&_Nadia Carlesso\" - \"AuthorsYoung-Woong KimYou can also search for this author in _Greta ZaraYou can also search for this author in _HyunJun KangYou can also search for this author in _Sergio BranciamoreYou can also search for this author in _Denis OMeallyYou can also search for this author in _Yuxin FengYou can also search for this author in _Chia-Yi KuanYou can also search for this author in _Yingjun LuoYou can also search for this author in _Michael S. NelsonYou can also search for this author in _Alex B. BrummerYou can also search for this author in _Russell RockneYou can also search for this author in _Zhen Bouman ChenYou can also search for this author in _Yi ZhengYou can also search for this author in _Angelo A. CardosoYou can also search for this author in _Nadia CarlessoYou can also search for this author in _\" - Contributions - Y-W.K. and N.C. conceived the study, interpreted results, wrote, reviewed, and approved the final version of the manuscript. Y.-W.K. designed and performed experiments and analyzed results. G.Z. performed complementary experiments, confirmed independently the culture methods and reviewed the manuscript. H.K. conducted experiments for the revision. S.B. and D.O. analyzed RNA-seq data. Y.F. performed preliminary experiments. C-YK generated the Tie2-CreERT2 construct for the mouse strain. M.N., A.B., and R.R. analyzed and quantified Matrigel networks. Y.L. and Z.B. prepared RNAs for RNA-seq and reviewed the manuscript. Y.Z. provided Tie2-CreERT2;Rosa26-tdTomato mice and preliminary data. A.C. interpreted results and reviewed the manuscript. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \" Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work._ are available.\" - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License. The authors acknowledge the Division for Medical Research Engineering, Nagoya University Graduate School of Medicine, for the usage of transmission electron microscopy.- - Author notes - \\'These authors contributed equally: Norihiro Imai and Yuki Ohsaki.\\' - Authors and Affiliations - \"Department of Gastroenterology and Hepatology, Nagoya University Graduate School of Medicine, 65 Tsurumai-cho, Showa-ku, Nagoya, Aichi, 466-8550, JapanNorihiro Imai,_Jingjing Zhang,_Fumitaka Mizuno,_Shinya Yokoyama,_Kenta Yamamoto,_ Takanori Ito,_Yoji Ishizu,_Takashi Honda,_Masatoshi Ishigami_&_Hiroki KawashimaDepartment of Anatomy (I), Sapporo Medical University, S1W17 Chuo-ku, Sapporo, Hokkaido, 060-8556, JapanYuki OhsakiDepartment of Anatomy and Molecular Cell Biology, Nagoya University Graduate School of Medicine, Nagoya, Aichi, JapanJinglei Cheng_ &_Hiroaki WakeDepartment of Emergency and Critical Care Medicine, Nagoya University Graduate School of Medicine, Nagoya, Aichi, JapanTaku Tanaka\" - \"AuthorsNorihiro ImaiYou can also search for this author in _Yuki OhsakiYou can also search for this author in _Jinglei ChengYou can also search for this author in _Jingjing ZhangYou can also search for this author in _Fumitaka MizunoYou can also search for this author in _Taku TanakaYou can also search for this author in _Shinya YokoyamaYou can also search for this author in _Kenta YamamotoYou can also search for this author in _Takanori ItoYou can also search for this author in _Yoji IshizuYou can also search for this author in _Takashi HondaYou can also search for this author in _Masatoshi IshigamiYou can also search for this author in _Hiroaki WakeYou can also search for this author in _Hiroki KawashimaYou can also search for this author in _\" - Contributions - \\'Concept and study design: N.I. and Y.O. Acquisition of data: N.I., J.C., J.Z., F.M., K.Y., T.I., Y.I., T.H., M.I., H.K., H.W., and Y.O. Writing manuscript: N.I. and Y.O. Statistical analysis: N.I.\\' - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " \"Creative Commons Attribution 3.0 Unported License. The images in this article are included in the article''s Creative Commons license, unless indicated otherwise in the image credit; if the image is not included under the Creative Commons license, users will need to obtain permission from the license holder in order to reproduce the image. \",\n",
       " \"Creative Commons Attribution 3.0 Unported License. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. \",\n",
       " 'Creative Commons Attribution 3.0 Unported License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License.\"- - Transient transfection assays and constructs - \"The C*05 and C*07 hybrid constructs were made by amplifyingu223C2.04 and 2.06u2009 kb genomic fragments of HLA-C*05:01:01:01 and HLA-C*07:02:01:03 respectively, which contained 776 or 766u2009bp of the respective HLA 5u2032UTR and exons 1u20133 up to a midpoint in intron 3, which was fused to au223C3.58u2009 kb fragment of the genomic H-2K gene, beginning at a midpoint in intron 3 and containing exons 4u20138 and the H-2K 3u2032UTR. For experiments with swapped promoters/5u2032UTR, additional hybrid constructs were made where the 5u2032 flanking region of the HLA genes was interchanged, such that the HLA-C*05:01:01:01 promoter/5u2032UTR was fused to the exons 1u20133 sequence of HLA-C*07:02:01:03, and vice versa. The region from the genomic H-2K gene was the same as described above. These hybrid constructs were transfected into 721.221 cells using optimized electroporation conditions (260u2009V, 1070u2009u03BCF, u221E resistance) using a Genepulser II (Bio-Rad). A limiting concentration of the pmax-GFP plasmid (Lonza) was co-transfected as a transfection control. The cells were collected 48u2009h post-transfection, and used for flow cytometry or RNA isolation/QPCR experiments.\" - Luciferase assays - \"The promoters/5u2032UTR of HLA-C*05:01:01:01 (776u2009bp upstream of the start codon) or HLA-C*07:02:01:03 (766u2009bp upstream of the start codon) genes were cloned into a luciferase containing pGL4.14 (Promega) basic promoter-less vector. HEK 293u2009T or 721.221 cells were transfected with the luciferase constructs containing the 5u2032UTR/promoter from either C*05, C*07 or no promoter, along with the co-transfection of the Renilla luciferase vector, pGL4.74 (Promega), using TransIT 2020 transfection reagent (for HEK 293T cells) or optimized electroporation conditions (for 721.221 cells). Cells were lysed after 24u2009h (HEK 293T) or 5u2009h (721.221) post-transfection, and firefly and renilla luciferase activities were measured using dual luciferase reporter assay system (Promega) and the Glomax multi detection system (Promega). The firefly luciferase activity was normalized relative to Renilla luciferase for each transfection, and the luciferase activity of each reporter construct was calculated as a fold change relative to the activity of pGL4.14-basic vector lacking a promoter.\" - Lentiviral expression assays - \"The C*05 and C*07 lentiviral expression constructs were made by amplifying the cDNA of HLA-C*05:01:01:01 and HLA-C*07:02:01:03 genes from exons 1u20138, and cloning it into the pHRsinUbEm expression plasmid (a gift from J.M. Boname/P.J. Lehner, University of Cambridge), with the inclusion of the HA tag at the N-terminus of C*05 and C*07, just after the signal peptide sequence. For the modified C*05 and C*07 lentiviral expression constructs, the cDNA of exons 1u20133 from the respective HLA genes was fused to the cDNA of exons 4u20138 of the murine H-2K gene, and cloned into the pHRsinUbEm expression plasmid, with inclusion of the N-terminal HA tag. The lentiviral HLA expression plasmids were co-transfected with the vesicular stomatitis virus-G envelope plasmid pMD2.G (Addgene) and packaging plasmid psPAX2 (Addgene), containing HIV-1 Gag and Rev, into HEK 293T cells to package lentiviral particles. Viral titres were determined by serial dilution and transduction of HEK 293T cells. 721.221 cells were transduced with the packaged lentivirus at a multiplicity of infection of 20, in the presence of polybrene (Santa Cruz Biotechnology), added at a final concentration of 8u2009 u03BCgu2009ml. Cells were collected 72u2009h post transduction and flow cytometry, RNA isolation/QPCR or immunoblotting experiments were performed.\" - Flow cytometry and antibodies - HLA-C cell surface expression was measured in transfected or transduced cells using alexa fluor 647 anti-human HLA-A, B, C antibody (clone W6/32, BioLegend, 311414, 1:20) or anti-HA.11 antibody (clone 16B12, Covance, MMS-101R, 1:250) along with allophycocyanin goat anti-mouse Ig (BD Biosciences, 550826, 1:50). HLA-C or HA staining was determined on GFP+ cells. For intracellular staining, cells were fixed and permeabilized using BD Cytofix/Cytoperm Fixation/Permeabilization kit (BD Biosciences), followed by staining using phycoerythrin anti-human HLA-A, B, C antibody (clone W6/32, BioLegend, 311406, 1:20) or anti-HA.11 antibody (clone 16B12, Covance, MMS-101R, 1:250) along with allophycocyanin goat anti-mouse Ig (BD Biosciences, 550826, 1:50). HLA staining using the same antibodies on non-transfected cells was used as a negative control. Data obtained were analysed using the FlowJo software (Tree Star). - RNA isolation and real-time quantitative PCR analysis - \\'Total RNA was isolated from cells using the RNeasy isolation kit (Qiagen), and cDNA prepared using the Quantitech Reverse Transcription kit (Qiagen). SYBR-green based quantitative PCR assays were designed and optimized for the H-2K gene (which formed a common region of the HLA-C constructs), GFP (for detecting Emerald GFP in lentiviral expression plasmids), Copgreen (for detecting GFP in pmaxGFP) and UBC. The sequences of the primers used for these assays were as follows:\\' - \"H2Kb: 5u2032-GAGCTGCAATAGTCACTGGAG-3u2032 and 5u2032-CCATCACTTTACAATCTGGGAGAG-3u2032 ;\" - \"GFP: 5u2032-AAGAACGGCATCAAGGTGAAC-3u2032 and 5u2032-CAGGTAGTGGTTGTCGGG-3u2032 ;\" - \"Copgreen: 5u2032-AGGACAGCGTGATCTTCACC-3u2032 and 5u2032-CTTGAAGTGCATGTGGCTGT-3u2032 ;\" - \"UBC: 5u2032-AACATCCAGAAAGAGTCCACC-3u2032 and 5u2032- CTTGTCTTGGATCTTTGCCTTG-3u2032 .\" - Standard curves for each of the assays were performed using serial dilutions of cDNA and amplification efficiencies were determined. Relative expression was expressed as 2, where dCt is the difference of the cycle threshold between the transcript of the gene of interest and the reference gene transcript. - Immunoblotting - \"Whole cell lysates were prepared in NP-40 lysis buffer containing 150u2009mM NaCl, 1% NP-40 (Igepal CA-630), 50u2009mM Tris pH 8.0, supplemented with protease inhibitors (Roche). Protein concentration of cell lysates was determined using Pierce BCA Protein Assay Kit (Thermo Scientific) and equal amount of protein was loaded onto denaturing polyacrylamide gels. HA-tagged HLA-C protein was visualized using anti-HA.11 (clone 16B12; Covance) and normalized using anti-GFP (Life technologies) and anti-GAPDH (clone 14C10; Cell Signaling Technology) antibodies. Membranes were stained with IRDye 800CW goat-anti-mouse IgG and IRDye 680LT goat-anti-rabbit IgG secondary antibodies and visualized and quantified using an Odyssey Infra-Red Imaging System (LI-COR Biosciences).\" - Statistical analysis - \"Statistical tests were performed using GraphPad Prism and non-parametric Mannu2013 Whitney U-tests were used for comparing two experimental groups, with a 5% significance level. For swapped promoter analyses, a Bonferroni correction for multiple testing was used; considering P=0.05 for four independent hypotheses, the significance threshold used for this analysis was P=0.0125.\" - Protein expression and thermal stability assays - \"Soluble class I heterodimers of HLA-C*05 and HLA-C*07 heavy chain and full-length u03B22-microglobulin (u03B22m) were expressed in Escherichia coli as inclusion bodies as previously described. Both HLA molecules were refolded with four peptides ITASRFKEL, SAEPVPLQL, RYRPGTVAL and KYFDEHYEY and thermal stability assay was performed. The fluorescent dye Sypro orange was used to monitor the protein unfolding. The thermal stability assay was performed in the Real Time Detection system (Corbett RotorGene 3000), originally designed for PCR. Each pHLA complex was in 10u2009mM Tris-HCl pH 8, 150u2009mM NaCl, at two concentrations (5 and 10u2009u03BCM) in duplicate, was heated from 25 to 95u2009xB0C with a heating rate of 1u2009xB0C per min. The fluorescence intensity was measured with excitation at 530u2009nm and emission at 555u2009nm. The Tm, or thermal melt point, represents the temperature for which 50% of the protein is unfolded.\" - Crystallization and structure determination - \"Crystals of the HLA-C*05-SAE were grown by the hanging-drop, vapour-diffusion method at 20u2009xB0C with a protein/reservoir drop ratio of 1:1, at a concentration of 3u2009mgu2009ml in 10u2009mM Tris-HCl pH 8, 150u2009mM NaCl using 1.8u2009 M Na malonate pH 7. The HLA-C*05:01-SAE crystals were flash frozen in liquid nitrogen. Crystals of HLA-C*0702-RYR were grown in 0.1 M HEPES pH 8.5, 2u2009mM ZnSO4 and 28% jeffamine ED-2001 (Hampton), using the same technique as for the HLA-C*05-SAE crystals. The HLA-C*07:02-RYR crystals were soaked in a mother liquid solution with the addition of 25% ethylene glycol prior to be flash frozen in liquid nitrogen. The data were collected on the MX1 beamline at the Australian Synchrotron, using the ADSC-Quantum 210 CCD detector (at 100u2009K). Data were processed using the XDS and scaled using SCALA software from the CCP4 suite. The structures were determined by molecular replacement using the PHASER program with the HLA-C*08 for the MHC model without the peptide (Protein Data Base accession number, 4NT6 (ref. )). Manual model building was conducted using the Coot software followed by maximum-likelihood refinement with the Buster program or Refmac form the CCP4 suite. The final models have been validated using the Protein Data Base validation website and the final refinement statistics are summarized in . All molecular graphics representations were created using PyMol.\" - Peptide elution - \"The HLA class-I negative 721.221 cells stably transfected with HLA-C*05:01 or HLA-C*07:02 were utilized to obtain the peptide repertoires in a previously described manner. In short, HLA class I were purified from a total 4 xD7 10 721.221 cells for each HLA-C using the pan class I antibody W6/32 immobilized and cross-linked to protein A resin. Captured HLA-peptide complexes were eluted with 10% acetic acid. The dissociated complexes were further separated by Reversed-phase HPLC to isolate and fractionate the bound peptides before analysis with a Q Exactive Hybrid Quadrupole-Orbitrap Mass Spectrometer (Thermo Scientific). Peptides were identified by database search using the human UniProtKB/SwissProt database (Feb 2016) with ProteinPilot V5.0 (SCIEX). A false discovery rate of 5% was applied and known contaminant peptides removed from the final list of peptide ligands. Peptides of 8u201314 amino acids in length were then used for analysis.\" - Phylogenetic analysis of HLA-C and chimpanzee MHC-C alleles - \"Nucleotide sequences for HLA-C and chimpanzee MHC-C alleles were obtained from the IMGT database. All chimpanzee sequences in the database were considered and a subset of human sequences was selected in each allele subclass. The region of the sequences aligned included exons 2 and 3 region (without the intron) and spanned 546u2009bp for the C*05 and C*07 sequences. Sequences were aligned with MUSCLE (v3.8.31) (ref. ) and the exons 2 and 3 sequences were extracted from the alignment. MrBayes (v3.2.5) (ref. ) was used to infer the phylogenetic relationship between sequences using the GTR model of nucleotide evolution with rate gamma distributed, running parameters used were nruns=3, nchains=4, ngen=2000000 and samplefreq=1000.\" - Heat-map comparison of regulatory elements in HLA-C alleles - \"Sequences homologous to HLA-C*05:01:01:01 and HLA-C*07:02:01:03 were identified in the NCBI nucleotide database using BLAST+ (v. 2.3.0) (ref. ). Sequences containing the promoter and 5u2032UTR, exons 2u20133, and 3u2032UTR regions were considered for the analysis. The region of the sequences aligned was as follows: promoter and 5u2032UTR region was u2212859u2009bp relative to the start codon of HLA-C*05 or u2212864u2009bp relative to the start codon of HLA-C*07; exons 2u2013 3 region (including intron) length was 798 bases for the alignment, which included 792u2009bp for C*05 and 796u2009bp for C*07; the 3u2032UTR region included 541u2009bp after the stop codon in C*05 and C*07. For each complete sequence, we determined the HLA-C allele by aligning the individual sequence in the HLA-C IMGT data set to the query sequence and assigning the most similar allele. In some cases, sequences that had the same allele typing but a different accession number were obtained, and are shown in for completion. For display on the heat map, one representative sequence of each allele type was chosen. Pairwise alignments were performed with the Needlemanu2013Wunsch algorithm in the EMBOSS package (v. 6.6.0.0) (ref. ). Multiple sequence alignments were performed for each regulatory region (promoter and 5u2032UTR, exons 2u20133, and 3u2032UTR) using MUSCLE (v3.8.31) (ref. ) and phylogenetic trees were generated using MrBayes (v3.2.5) (ref. ) with model parameters mentioned previously. To quantify the relative similarity between a HLA-C allele and HLA-C*05 and HLA-C*07 in a phylogenetic tree, we calculated a metric quantifying the phylogenetic distance between the query HLA allele and its relative distance to HLA-C*05 and HLA-C*07 in the specific tree. This was done in the following way: phylogenetic distances as branch lengths from the HLA-C allele to HLA-C*05 and to HLA-C*07 were extracted using the R package ape, the similarity metric was then calculated as the ratio of the distance of the query HLA-C sequence to HLA-C*05 divided by the sum of the distances of the query HLA-C sequence to HLA-C*05 and HLA-C*07.\" - Data availability - Structural information has been deposited in the Protein Data Bank under accession numbers 5VGD (HLA-C*05:01-SAE) and 5VGE (HLA-C*07:02-RYR). The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the data set identifier PXD006455. The data that support the findings of this study are available from the corresponding author on request.- - \\'How to cite this article: Kaur, G. et al. Structural and regulatory diversity shape HLA-C protein expression levels. Nat. Commun. 8, 15924 doi: 10.1038/ncomms15924 (2017).\\' - \"Publishers note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \"Welter, D. et al. The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. Nucleic Acids Res. 42, D1001u2013D1006 (2014)._ _ _ Parham, P. MHC class I molecules and KIRs in human history, health and survival. Nat. Rev. Immunol. 5, 201u2013214 (2005)._ _ _ Shiina, T., Hosomichi, K., Inoko, H. & Kulski, J. K. The HLA genomic loci map: expression, interaction, diversity and disease. J. Hum. Genet. 54, 15u201339 (2009)._ _ _ Apps, R. et al. Influence of HLA-C expression level on HIV control. Science 340, 87u201391 (2013)._ _ _ _ Petersdorf, E. W. et al. HLA-C expression levels define permissible mismatches in hematopoietic cell transplantation. Blood 124, 3996u20134003 (2014)._ _ _ Koeffler, H. P., Ranyard, J., Yelton, L., Billing, R. & Bohman, R. Gamma-interferon induces expression of the HLA-D antigens on normal and leukemic human myeloid cells. Proc. Natl Acad. Sci. USA 81, 4080u20134084 (1984)._ _ _ _ Bashirova, A. A., Martin, M. P., McVicar, D. W. & Carrington, M. The killer immunoglobulin-like receptor gene cluster: tuning the genome for defense. Annu. Rev. Genomics Hum. Genet. 7, 277u2013300 (2006)._ _ _ Snary, D., Barnstable, C. J., Bodmer, W. F. & Crumpton, M. J. Molecular structure of human histocompatibility antigens: the HLA-C series. Eur. J. Immunol. 7, 580u2013585 (1977)._ _ _ Zemmour, J. & Parham, P. Distinctive polymorphism at the HLA-C locus: implications for the expression of HLA-C. J. Exp. Med. 176, 937u2013950 (1992)._ _ _ Apps, R. et al. Relative expression levels of the HLA class-I proteins in normal and HIV-infected cells. J. Immunol. 194, 3594u2013 3600 (2015)._ _ _ Kulkarni, S. et al. Genetic interplay between HLA-C and MIR148A in HIV control and Crohn disease. Proc. Natl Acad. Sci. USA 110, 20705u201320710 (2013)._ _ _ _ Fellay, J. et al. A whole-genome association study of major determinants for host control of HIV-1. Science 317, 944u2013947 (2007)._ _ _ _ Kulkarni, S. et al. Differential microRNA regulation of HLA-C expression and its association with HIV control. Nature 472, 495u2013 498 (2011)._ _ _ _ Thomas, R. et al. HLA-C cell surface expression and control of HIV/AIDS correlate with a variant upstream of HLA-C. Nat. Genet. 41, 1290u20131294 (2009)._ _ _ Apps, R. et al. HIV-1 Vpu mediates HLA-C downregulation. Cell Host Microbe 19, 686u2013695 (2016)._ _ _ O\\'Huigin, C. et al. The molecular origin and consequences of escape from miRNA regulation by HLA-C alleles. Am. J. Hum. Genet. 89, 424u2013431 (2011)._ _ _ Pereyra, F. et al. The major genetic determinants of HIV-1 control affect HLA class I peptide presentation. Science 330, 1551u20131557 (2010)._ _ Gonzalez-Galarza, F. F. et al. Allele frequency net 2015 update: new features for HLA epitopes, KIR and disease and HLA adverse drug reaction associations. Nucleic Acids Res. 43, D784u2013D788 (2015)._ _ _ Borenstein, S. H., Graham, J., Zhang, X. L. & Chamberlain, J. W. CD8+ T cells are necessary for recognition of allelic, but not locus-mismatched or xeno-, HLA class I transplantation antigens. J. Immunol. 165, 2341u20132353 (2000)._ _ _ Hundhausen, C. et al. Allele-specific cytokine responses at the HLA-C locus: implications for psoriasis. J. Invest. Dermatol. 132, 635u2013641 (2012)._ _ _ Apps, R. et al. Human leucocyte antigen (HLA) expression of primary trophoblast cells and placental cell lines, determined using single antigen beads to characterize allotype specificities of anti-HLA antibodies. Immunology 127, 26u201339 (2009)._ _ _ Hilton, H. G. & Parham, P. Direct binding to antigen-coated beads refines the specificity and cross-reactivity of four monoclonal antibodies that recognize polymorphic epitopes of HLA class I molecules. Tissue Antigens 81, 212u2013220 (2013)._ _ _ Kim, E., Kwak, H. & Ahn, K. Cytosolic aminopeptidases influence MHC class I-mediated antigen presentation in an allele-dependent manner. J. Immunol. 183, 7379u20137387 (2009)._ _ _ Neisig, A., Melief, C. J. & Neefjes, J. Reduced cell surface expression of HLA-C molecules correlates with restricted peptide binding and stable TAP interaction. J. Immunol. 160, 171u2013179 (1998)._ _ _ Sibilio, L. et al. A single bottleneck in HLA-C assembly. J. Biol. Chem. 283, 1267u20131274 (2008)._ _ _ Addo, M. M. et al. The HIV-1 regulatory proteins Tat and Rev are frequently targeted by cytotoxic T lymphocytes derived from HIV-1-infected individuals. Proc. Natl Acad. Sci. USA 98, 1781u20131786 (2001)._ _ _ _ Vales-Gomez, M., Reyburn, H. T., Mandelboim, M. & Strominger, J. L. Kinetics of interaction of HLA-C ligands with natural killer cell inhibitory receptors. Immunity 9, 337u2013344 (1998)._ _ _ Rasmussen, M. et al. Uncovering the peptide-binding specificities of HLA-C: a general strategy to determine the specificity of any MHC class I molecule. J. Immunol. 193, 4790u20134802 (2014)._ _ _ Hofmann, S. et al. Rapid and sensitive identification of major histocompatibility complex class I-associated tumor peptides by Nano-LC MALDI MS/MS. Mol. Cell Proteomics 4, 1888u20131897 (2005)._ _ _ Falk, K. et al. Allele-specific peptide ligand motifs of HLA-C molecules. Proc. Natl Acad. Sci. USA 90, 12005u201312009 (1993)._ _ _ _ Miyadera, H. et al. density profiling reveals instability of autoimmunity-associated HLA. J. Clin. Invest. 125, 275u2013 291 (2015)._ _ Reits, E. A. et al. Radiation modulates the peptide repertoire, enhances MHC class I expression, and induces successful antitumor immunotherapy. J. Exp. Med. 203, 1259u20131271 (2006)._ _ _ Faroudi, M. et al. Lytic versus stimulatory synapse in cytotoxic T lymphocyte/target cell interaction: manifestation of a dual activation threshold. Proc. Natl Acad. Sci. USA 100, 14145u201314150 (2003)._ _ _ _ Thomas, R. et al. A novel variant marking HLA-DP expression levels predicts recovery from hepatitis B virus infection. J. Virol. 86, 6979u20136985 (2012)._ _ Cassidy, S. A., Cheent, K. S. & Khakoo, S. I. Effects of peptide on NK cell-mediated MHC I recognition. Front. Immunol. 5, 133 (2014)._ _ Blais, M. E., Dong, T. & Rowland-Jones, S. HLA-C as a mediator of natural killer and T-cell activation: spectator or key player? Immunology 133, 1u20137 (2011)._ _ _ Chappell, P. et al. Expression levels of MHC class I molecules are inversely correlated with promiscuity of peptide binding. Elife 4, e05345 (2015)._ _ Koch, M. et al. Structures of an MHC class I molecule from B21 chickens illustrate promiscuous peptide binding. Immunity 27, 885u2013899 (2007)._ _ _ Zhang, J. et al. Narrow groove and restricted anchors of MHC class I molecule BF2*0401 plus peptide transporter restriction can explain disease susceptibility of B4 chickens. J. Immunol. 189, 4478u20134487 (2012)._ _ _ Tregaskes, C. A. et al. Surface expression, peptide repertoire, and thermostability of chicken class I molecules correlate with peptide transporter specificity. Proc. Natl Acad. Sci. USA 113, 692u2013697 (2016)._ _ _ _ Obst, R., Armandola, E. A., Nijenhuis, M., Momburg, F. & Hammerling, G. J. TAP polymorphism does not influence transport of peptide variants in mice and humans. Eur. J. Immunol. 25, 2170u20132176 (1995)._ _ _ Auton, A. et al. A fine-scale chimpanzee genetic map from population sequencing. Science 336, 193u2013198 (2012)._ _ _ _ Gras, S. et al. The shaping of T cell receptor recognition by self-tolerance. Immunity 30, 193u2013203 (2009)._ _ _ Cowieson, N. P. et al. MX1: a bending-magnet crystallography beamline serving both chemical and macromolecular crystallography communities at the Australian Synchrotron. J. Synchrotron Radiat. 22, 187u2013190 (2015)._ _ _ Kabsch, W. Xds. Acta Crystallogr. D Biol. Crystallogr. 66, 125u2013 132 (2010)._ _ _ Evans, P. Scaling and assessment of data quality. Acta Crystallogr. D Biol. Crystallogr. 62, 72u201382 (2006)._ _ Collaborative Computational Project N. The CCP4 suite: programs for protein crystallography. Acta Crystallogr. D Biol. Crystallogr. 50, 760u2013763 (1994).Read, R. J. Pushing the boundaries of molecular replacement with maximum likelihood. Acta Crystallogr. D Biol. Crystallogr. 57, 1373u2013 1382 (2001)._ _ _ Choo, J. A., Liu, J., Toh, X., Grotenbreg, G. M. & Ren, E. C. The immunodominant influenza A virus M158-66 cytotoxic T lymphocyte epitope exhibits degenerate class I major histocompatibility complex restriction in humans. J. Virol. 88, 10613u201310623 (2014)._ _ Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Acta Crystallogr. D Biol. Crystallogr. 66, 486u2013 501 (2010)._ _ _ Vonrhein, C. et al. Data processing and analysis with the autoPROC toolbox. Acta Crystallogr. D Biol. Crystallogr. 67, 293u2013302 (2011)._ _ _ DeLano, W. L. The PyMOL Molecular Graphics System. (2002).Schittenhelm, R. B., Dudek, N. L., Croft, N. P., Ramarathinam, S. H. & Purcell, A. W. A comprehensive analysis of constitutive naturally processed and presented HLA-C*04:01 (Cw4)-specific peptides. Tissue Antigens 83, 174u2013179 (2014)._ _ _ Dudek, N. L., Croft, N. P., Schittenhelm, R. B., Ramarathinam, S. H. & Purcell, A. W. A systems approach to understand antigen presentation and the immune response. Methods Mol. Biol. 1394, 189u2013209 (2016)._ _ _ Robinson, J. et al. The IPD and IMGT/HLA database: allele variant databases. Nucleic Acids Res. 43, D423u2013D431 (2015)._ _ _ Edgar, R. C. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 32, 1792u20131797 (2004)._ _ _ Ronquist, F. et al. MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space. Syst. Biol. 61, 539u2013542 (2012)._ _ Camacho, C. et al. BLAST+: architecture and applications. BMC Bioinform. 10, 421 (2009)._ _ Rice, P., Longden, I. & Bleasby, A. EMBOSS: the European Molecular Biology Open Software Suite. Trends Genet. 16, 276u2013277 (2000)._ _ _ Paradis, E., Claude, J. & Strimmer, K. APE: analyses of Phylogenetics and Evolution in R language. Bioinformatics 20, 289u2013290 (2004)._ _ _ Berman, H. M. et al. The protein data bank. Nucleic Acids Res. 28, 235u2013242 (2000)._ _ _ _ Vizcaino, J. A. et al. 2016 update of the PRIDE database and its related tools. Nucleic Acids Res. 44, D447u2013D456 (2016)._ _ _ Dundas, J. et al. CASTp: computed atlas of surface topography of proteins with structural and topographical mapping of functionally annotated residues. Nucleic Acids Res. 34, W116u2013W118 (2006)._ _ _ \"- - \"We would like to acknowledge the flow cytometry facility at the WIMM, which is supported by the MRC HIU; MRC MHU (MC_UU_12009); NIHR Oxford BRC and John Fell Fund (131/030 and 101/517), the EPA fund (CF182 and CF170) and by the WIMM Strategic Alliance awards G0902418 and MC_UU_12025. We would also like to acknowledge use of the facilities and the assistance of Dr Ralf Schittenhelm at the Monash Biomedical Proteomics Facility and thank Prof. Simon J. Davis for his helpful comments. Work in the authors laboratories is supported by the UK and Danish Medical Research Councils, The Lundbeck Foundation, The Alan and Babette Sainsbury Charitable Fund, the Naomi Bramson Trust, the Clinical Neuroimmunology Fund, the Oxford Biomedical Research Centre, the Oak Foundation (L.F.), Wellcome Trust (100308/Z/12/Z and 106130/Z/14/Z, L.F.; 100956/Z/13/Z, G.M.), Australian National Health and Medical Research Council (NHMRC), Australian Research Council (ARC) (J.R., A.W.P.), ARC Laureate fellowship (J.R.), NHMRC Senior Research Fellowship (1044215, A.W.P.), ARC Future Fellowship (FT120100416, S.G.). This project has been funded in part with federal funds from the Frederick National Laboratory for Cancer Research, under Contract No. HHSN261200800001E. This research was supported in part by the Intramural Research Program of the NIH, Frederick National Lab, Center for Cancer Research. The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products, or organizations imply endorsement by the U.S. Government.\"- - Author notes - \\'Jamie Rossjohn and Lars Fugger: These authors jointly supervised this work\\' - Authors and Affiliations - \"MRC Human Immunology Unit, Weatherall Institute of Molecular Medicine, John Radcliffe Hospital, University of Oxford, Oxford, OX3 9DS, UKGurman Kaur,_Subita Balaram Kuttikkatte,_Kathrine E. Attfield_&_Lars FuggerInfection and Immunity Program and the Department of Biochemistry and Molecular Biology, Biomedicine Discovery Institute, Monash University, Clayton, 3800, Victoria, AustraliaStephanie Gras,_ Jesse I. Mobbs,_Julian P. Vivian,_Anthony W. Purcell_&_Jamie RossjohnAustralian Research Council Centre of Excellence in Advanced Molecular Imaging, Monash University, Clayton, 3800, Victoria, AustraliaStephanie Gras,_Julian P. Vivian_ &_Jamie RossjohnNuffield Department of Clinical Neurosciences, Division of Clinical Neurology, Oxford Centre for Neuroinflammation, John Radcliffe Hospital, University of Oxford, Oxford, OX3 9DS, UKAdrian Cortes,_Thomas Barber,_Calliope A. Dendrou_ &_Lars FuggerWellcome Trust Centre for Human Genetics, University of Oxford, Oxford, OX3 7BN, UKAdrian Cortes,_Calliope A. Dendrou_&_Gil McVeanDepartment of Clinical Medicine, Aarhus University Hospital, Aarhus, 8200N, DenmarkLise Torp Jensen_&_Lars FuggerCancer and Inflammation Program, Leidos Biomedical Research Inc., Frederick National Laboratory for Cancer Research, Frederick, 21702, Maryland, USAMary CarringtonThe Ragon Institute of MGH, MIT and Harvard, Cambridge, 02139, Massachusetts, USAMary CarringtonBig Data Institute, Li Ka Shing Centre for Health Information and Discovery, University of Oxford, OX3 7FZ, Oxford, UKGil McVeanInstitute of Infection and Immunity, Cardiff University, School of Medicine, Heath Park, CF14 4XN, Cardiff, UKJamie Rossjohn\" - \"AuthorsGurman KaurYou can also search for this author in _Stephanie GrasYou can also search for this author in _Jesse I. MobbsYou can also search for this author in _Julian P. VivianYou can also search for this author in _Adrian CortesYou can also search for this author in _Thomas BarberYou can also search for this author in _Subita Balaram KuttikkatteYou can also search for this author in _Lise Torp JensenYou can also search for this author in _Kathrine E. AttfieldYou can also search for this author in _Calliope A. DendrouYou can also search for this author in _Mary CarringtonYou can also search for this author in _Gil McVeanYou can also search for this author in _Anthony W. PurcellYou can also search for this author in _Jamie RossjohnYou can also search for this author in _Lars FuggerYou can also search for this author in _\" - Contributions - G.K. contributed to conception, coordination and design of the study, all experiments apart from the protein crystallization, peptide elution and structural work, and to data analyses, drafting and writing of the manuscript. S.G. contributed to all structural experiments including protein purifications, stability assays, crystallizations and structure determination, and to writing of the manuscript. J.I.M. contributed to structural experiments including protein purifications, crystallizations, structure determination and peptide repertoire analysis, and to writing of the manuscript. J.P.V contributed to structural experiments including protein purifications, crystallizations, structure determination and peptide repertoire analysis. A.C. performed all phylogenetic analyses of human and chimpanzee sequences, and contributed to writing of the manuscript. T.B. contributed to cloning of HLA-C constructs, cellular transfection experiments, flow cytometry, QPCR and data analyses. S.B.K. contributed to optimization of QPCR experiments. L.T.J. contributed to cloning of HLA-C genomic constructs. K.E.A. contributed to experimental design and manuscript editing. C.A.D. contributed to experimental design, data and analyses discussions, and to manuscript editing. M.C. provided intellectual input and contributed to manuscript editing. G.M. contributed to the conception and design of phylogenetic analyses and to writing of the manuscript. A.W.P. contributed to conception and design of the peptide repertoire analyses. J.R. contributed to the conception, coordination and design of the study and to writing of the manuscript. L.F. contributed to conception, coordination and design of the study, and drafting and writing of the manuscript. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing financial interests. - \\' \\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License.\\' - Discovery and replication of urate-associated CpGs - \"The discovery analysis identified 140 significant CpGs (pu2009<u20091.1Eu2013 7u2009=u20090.05/441,854 CpGs analyzed), of which 100 replicated (consistent effect direction, pu2009<u20090.05 in the replication analysis, and overall meta-analysispu2009<u20091.1Eu20137; Supplementary Data_ and Fig._). The estimated inflation factor of the meta-analysis combining discovery and replication was 1.06 (Supplementary Fig._). The study-specific results had a median inflation factor of 0.99 (min: 0.86, max: 1.06, Supplementary Data_), and were corrected prior to any meta-analysis when >1. The heterogeneity of most replicated CpGs was low to moderate: the median I from the meta-analysis of the 17 EA studies was 11% (25th, 75th percentile: 0.0, 31.3), from the five AA studies was 0% (25th, 75th percentile: 0.0, 35.1), and from the overall meta-analysis of the four ancestry groups was 30.2% (25th, 75th percentile: 0.0, 60.0, Supplementary Data_). Of the 32 CpGs that showed heterogeneity >50% in the meta-analysis of the four ancestry groups, 90% (29 CpGs) had effect estimates in the same direction across the ancestry groups (Supplementary Fig._). Ancestry-specific results of CpGs with pu2009<u20091.1Eu20137 are reported in Supplementary Data_ [EA], [AA], and [SA] if present, and with pu2009<u20091Eu20135 otherwise (Supplementary Data_ [SSA]).\" - \"Fig. 2: CpGs in the EWAS of serum urate from the combined meta-analysis of discovery and replication results (nu2009=u200917,996).The CpGs are ordered by their chromosomal position on the x-axis with their u2013log10(p value) of the association on the y-axis. CpGs with positive and negative effect estimates are plotted in the upper and lower panels, respectively. The dotted horizontal lines represent the level of significance corrected for multiple testing (two-sided pu2009 <u20091.1Eu20137). Black: replicated CpGs; brown: gene with replicated CpGs associated with gene expression in monocytes and significant causal effects for serum urate; blue: replicated CpGs with significant causal effects on serum urate or gout; red: replicated CpGs associated with gene expression in monocytes or whole blood. Gene names are displayed if the gene had at least one CpG with p valueu2009<u20091Eu201314. MR Mendelian randomization.\" - \"The follow-up analyses focused on the replicated CpGs from the overall meta-analysis, in which the CpG with the lowest p value was at SLC7A11 (cg06690548, pu2009 =u20092.18Eu201359). Among the ten CpGs with the lowest p values, two mapped to PHGDH (cg14476101, pu2009=u20095.87Eu201338; cg16246545, pu2009=u2009 1.77Eu201324), which encodes for 3-phosphoglycerate dehydrogenase, and five to SLC2A9 (cg21795255, cg20479063, cg11266682, cg00071950, cg13841979; pu2009 <u20091Eu201325), which encodes the urate transporter GLUT9 and is the locus with the largest effect size in GWAS of serum urate. The probe cg21795255 at SLC2A9 had a common SNP at the extension base of the DNA methylation probe (rs3796839, minor allele frequency of 46% in GnomAD) and was excluded from all follow-up analyses, which therefore included 99 replicated CpGs annotated to 81 genes. Of these, 24 genes contained replicated CpGs with significant findings from our follow-up analyses and are featured in Table_.\" - Table 1 Genes having replicated CpG(s) associated with gene expression in whole blood or monocytes, with significant causal effects for urate or gout, associated with cardiometabolic traits, or annotated as a GWAS locus for urate or gout. - Heritability of urate-associated CpGs and explained serum urate variance - \"DNA methylation can be influenced by genetics and environmental factors. Heritability of DNA methylation levels at a CpG estimates the proportion of variance in DNA methylation levels at the CpG that can be attributed to additive genetics; it does not refer to germline inheritance of DNA methylation. Across three datasets from populations of EA, the heritability estimates of the replicated CpGs varied (Supplementary Data_). The mean heritability estimates for the 99 replicated, urate-associated CpGs was higher than the mean heritability across all CpGs assessed in each of the three studies: 0.36 (urate-associated CpGs) vs. 0.16 (HM450K array) based on Hannon et al., 0.54 vs. 0.19 for van_Dongen et al., and 0.52 vs. 0.19 for McRae et al.. Among the seven replicated sites at SLC2A9, the mean heritability combining the three datasets ranged from 0.39 at cg03725404 to 0.93 at cg11266682. These observations suggest that the DNA methylation at some of the urate-associated CpGs may reflect the effect of common genetic variants on serum urate levels.\" - The proportion of serum urate variance explained by the replicated, urate-associated CpGs was estimated in a separate study sample using the coefficient of determination (R) obtained from linear regression (Methods). Compared to an age- and sex-adjusted model, the replicated CpGs explained an additional 11.6% of the serum urate variance. In a recent GWAS of serum urate based on data from 457,690 individuals, replicated genetic index variants explained 7.7% of the age- and sex-adjusted serum urate variance. The higher proportion of urate variance explained by the replicated CpGs as compared to common genetic variants may partly reflect environmental influences on serum urate levels. - Causal effects of CpGs on serum urate and gout - \"To evaluate whether the urate-associated CpGs might causally affect serum urate levels, we used two-sample MR analysis, where genetic variants associated with DNA methylation of a CpG (methylation quantitative trait locus, meQTL) in cis (<500u2009kb) were used as proxies or genetic instruments of the CpGs. Cis meQTLs were available for 27 of the 99 replicated CpGs from a meta-analysis of cohorts of EA in the Genetics of DNA Methylation Consortium (GoDMC, Methods). Combined with summary statistics of a GWAS of serum urate among EA individuals, we found evidence for significant causal effects of DNA methylation on serum urate levels at four CpGs at SLC2A9 (pu2009<u20091.9Eu20133u2009=u2009 0.05/27) based on the multiplicative random effect inverse-variance weighted method, our primary method (Methods, Table_). In sensitivity analysis, all significant causal effects were supported by three or more other methods that are robust to pleiotropy (Supplementary Data_). After additionally removing the meQTLs that were correlated with any of the five GWAS index SNPs at SLC2A9 among EA persons (Methods), cg13841979 was the only CpG with u22654 meQTLs for MR analysis and showed nominally significant causal effects on serum urate levels (pu2009 =u20092.25Eu20132, Supplementary Data_). This attenuation of causal effect is consistent with the observation that DNA methylation at cg13841979 mediated the effects of two urate GWAS index SNPs on serum urate, as reported below.\" - Table 2 CpGs with significant causal effects on serum urate or gout from Mendelian randomization analysis. - \"As expected, the causal effects of the four CpGs on serum urate were consistent with their effect direction from EWAS. At cg11266682, a promoter-associated CpGs, higher DNA methylation levels were associated with higher serum urate levels (effect size: 0.21u2009mg/dL per standard deviation [SD] of DNA methylation beta value, pu2009=u20098.8Eu201304). In contrast, for three CpGs further downstream of the transcription, higher DNA methylation levels were associated with lower serum urate levels (effect size: u22120.65 to u22120.46u2009mg/dL per SD of DNA methylation beta value, pu2009<u20092.1Eu20134). At all four CpGs, little evidence of pleiotropy, a threat to MR assumptions, was detected based on the Egger intercept test (all pu2009>u20090.14). The observed heterogeneity of the meQTL effects on serum urate levels (p heterogeneity <7.54Eu2013239; Table_ and Supplementary Figs._ to ) suggests complex genetic influences on DNA methylation in this region. Together, these observations at SLC2A9 are consistent with a causal effect of differential DNA methylation on serum urate levels.\" - \"The same 27 CpGs that were tested for causal effects on serum urate were also tested for their causal effects on gout. The CpG cg03725404 at SLC2A9 showed a significant causal effect on gout (pu2009<u20091.9Eu20133u2009=u2009 0.05/27, Table_). Consistent with its causal effect on lower serum urate, cg03725404 conferred lower odds for gout, with an odds ratio of 0.43 per SD of DNA methylation beta value (pu2009=u20092.98Eu20139). In sensitivity analysis, the causal effect of cg03725404 for gout was supported by three MR methods robust to pleiotropy (Supplementary Data_). The scatter plots of the effects of the meQTLs on DNA methylation and gout along with the regression slope from the MR methods showed consistency among MR methods (Supplementary Fig._). The meQTLs included in the MR analysis for serum urate and gout were independent of each other (Supplementary Figs._ and , respectively). Leave-one-out analysis showed that the significant causal effects of DNA methylation on serum urate and gout were not driven by any single meQTL at any of the CpGs (Supplementary Fig._ for serum urate and Supplementary Fig._ for gout). Forest plots of the effects of the meQTLs showed that the majority of the meQTLs supported the significant causal effects, despite the presence of heterogeneity (Supplementary Fig._ for serum urate and Supplementary Fig._ for gout). Together, these findings show robust results for the performed MR analyses and support a causal effect of differential DNA methylation at SLC2A9 on serum urate levels and gout risk. The reverse MR analysis that assessed the potential causal effects of serum urate on DNA methylation levels did not detect any significant findings among the 99 tested CpGs (Supplementary Note_).\" - CpGs at SLC2A9 mediate genetic effects - \"Given that four CpGs at SLC2A9 had significant causal effects on serum urate, we hypothesized that these causal effects may mediate genetic effects at this locus. We previously identified four independent intronic or intergenic independent index SNPs in two neighboring 1u2009Mb regions at SLC2A9 in a large-scale GWAS meta-analysis of serum urate of EA populations. Therefore, we first tested whether the four CpGs mediated the effects of any of four independent SNPs on serum urate among EA participants of the ARIC study (nu2009=u2009637). Two CpGs (cg11266682 and cg13841979) had significant mediating effects (pu2009<u2009 0.0125) for two of the independent index SNPs (rs10017305 and rs6825187; Supplementary Data_). We next assessed these findings in two additional studies, KORA (Cooperative Health Research in the Region of Augsburg, nu2009=u20091636) and SHIP (Study of Health in Pomerania, nu2009=u2009223), and observed significant evidence for DNA methylation as a mediator of the effects of these SNPs on serum urate levels. From the meta-analysis of all three studies, the mediating effects ranged from 21% by cg13841979 for rs10017305 (mediating effect: 0.05u2009mg/dL, pu2009 =u20094.7Eu201307) to 43% by cg11266682 for rs6825187 (mediating effect: 0.10u2009 mg/dL, pu2009=u20093.0Eu20135, Supplementary Data_).\" - Urate-associated CpGs, gene expression, and chromatin accessibility - \"An important function of DNA methylation is the regulation of gene expression. To determine whether the urate-associated CpGs were associated with gene expression in blood cells, we used summary statistics of DNA methylation and gene expression from monocytes and whole blood. Gene expression in monocytes is particularly relevant for SLC2A9, which is mainly expressed in monocytes and myeloid dendritic cells among the blood cell types (Supplementary Fig._). Of the 99 replicated CpGs, 26 CpGs were significantly associated with gene expression in cis in monocytes (pu2009<u20095Eu20134; Supplementary Data_). Some of the genes harbored multiple urate-associated CpGs (two at PHGDH, five at SLC2A9, six at SLC1A5). At PHGDH and SLC1A5, higher DNA methylation levels were associated with lower gene expression, with CpGs mapping to intron 1 or intron 2 of the gene, depending on the reference isoform (Fig._). The inverse association between DNA methylation levels in intron 1 and gene expression has been reported to be conserved across vertebrates and is found in multiple human tissues. Interestingly, at SLC2A9, the effect direction on gene expression for the five CpGs differed by genomic location (Fig._). The three CpGs annotated as promoter-associated and located at introns 1 or 2 of isoform 1 (cg02734326, cg00071950, cg11266682) showed inverse associations with gene expression (e.g., cg11266682: effect u22120.037, pu2009 =u20094.16Eu201315; Fig._ and Supplementary Data_). These three promoter-associated CpGs were associated with higher serum urate levels in EWAS (Fig._). In contrast, cg03725404, located further downstream in the direction of transcription, also showed inverse associations with gene expression (effect u22120.024, pu2009 =u20094.75Eu20136, Fig._) but was associated with lower serum urate levels in EWAS (Fig._). Both cg11266682 and cg03725404 were also shown to have significant causal effects on serum urate, as reported above. The different directions of association across CpGs in SLC2A9 indicate complex regulation. Assuming that DNA methylation affects gene expression levels, which in turn influence serum urate levels, the observation that higher DNA methylation levels at the three promoter-associated CpGs in SLC2A9 were associated with lower gene expression in monocytes and higher serum urate levels (Fig._) allows for the inference that SLC2A9 expression in monocytes due to differential methylation at these promoter-associated CpGs has an inverse relationship with serum urate levels. As depicted in Fig._, cg11266682 was associated with lower gene expression levels but showed a positive, causal effect on serum urate levels. In contrast, cg03725404, a CpG further downstream in the direction of transcription, was also associated with lower gene expression but had an inverse, causal effect on serum urate levels (Fig._).\" - \"Fig. 3: Associations of CpGs at PHGDH and SLC1A5 with serum urate levels.For both_PHGDH_(A)_and SLC1A5 (B), the upper part shows_chromosomal positions on the x-axis and the u2013log10(p value) on the y-axis, and the lower part_ shows the correlations between DNA methylation levels of the CpGs. The replicated CpGs are labeled. Promoter-associated annotation was based on the HM450K annotation file. The gene models were based on RefSeq curated genes. Associations of DNA methylation with monocyte gene expression were from Kennedy et al. BMC Genomics 2018 and with whole blood from the KORA study (Methods). The correlations between the CpGs were generated using DNA methylation data from 804 European American participants of the ARIC study. Gene models were based on Genbank RefSeq (Accession numbers. PHGDH, NM_006623; SLC1A5 isoform 1, NM_005628; isoform 2, NM_001145144; isoform 3 NM_001145145). assoc. associated.\" - \"Fig. 4: Associations of CpGs at SLC2A9 with serum urate levels and gene expression in monocytes.CpGs in the SLC2A9 region (A) with the effect size of DNA methylation on gene expression in monocytes at five replicated CpGs (B) and the effect size of serum urate on DNA methylation levels at these same five CpGs (C). All labeled CpGs were replicated. In the legend, MR indicates CpGs with a significant causal effect on serum urate levels based on Mendelian randomization analysis. Colors of the estimates in panels B and C match the color legend in panel A. Promoter-associated annotation was based on the Illumina HM450K annotation file. The gene models were based on RefSeq curated genes. The correlations between the CpGs were generated using DNA methylation data from 804 European Americans participants of the ARIC study. The association between DNA methylation and monocyte gene expression in panel B is based on Kennedy et al. BMC Genomics 2018 (nu2009=u20091202). The DNA methylation estimates in panel C are based on the meta-analysis combining discovery and replication cohorts in the present study (nu2009=u200917,996). Genbank RefSeq accession numbers: isoform 1 (NM_020041), isoform 2 (NM_001001290). MR Mendelian randomization, assoc. associated.\" - \\'Fig. 5: Conceptual figure summarizing patterns of relationships between DNA methylation at SLC2A9, its expression in monocytes, and serum urate levels.The figure focuses on the two CpGs with significant causal effects on serum urate and association with gene expression in monocytes. The inferred relationship between gene expression and serum urate levels is inverse for the promoter-associated CpG cg11266682 (A, orange shading), and concordant for CpG cg03725404 (B, blue shading). Solid compound arrows indicate both observed cross-sectional association and causal effect of DNA methylation on serum urate based on Mendelian randomization analysis. Solid arrows indicate observed cross-sectional associations. Dashed arrows indicate inferred relationships.\\' - \"For gene expression in whole blood, eight CpGs showed significant associations (Supplementary Data_). For all seven CpGs with significant associations with gene expression in both monocytes and whole blood (two in PHGDH, one in PARP3, two in UAP1L1, and two in SLC1A5), the directions of effect in the two gene expression datasets were consistent. CpGs at SLC2A9 were not associated with gene expression in whole blood.\" - \"There are two main transcripts of SLC2A9. Isoform 1 encodes the long version of the GLUT9 protein, which has been reported to localize to the basolateral membrane of renal proximal tubule cells, where it mediates urate reabsorption. We investigated whether the DNA methylation pattern of the urate-associated CpGs in the kidney may be similar to those observed in whole blood. Whole genome bisulfite sequencing (WGBS) of kidney samples (nu2009=u20095) showed lower DNA methylation levels at the urate-associated CpGs in the promoter-associated region of isoform 1, and the pattern of DNA methylation level aligned with gene expression levels from RNA-sequencing (Fig._). We further studied whether the urate-associated CpGs mapped into open chromatin regions, where transcription factor binding may occur. ATAC-seq data from primary human kidney tissue (nu2009 =u20093 each from micro-dissected cortex and medulla) showed that the three promoter-associated CpGs, andu2014to a lesser extent cg20479063u2014mapped to DNA sequences with high chromatin accessibility in the kidney (Fig._). Taken together, the data from the kidney samples suggest that the urate-associated CpGs at SLC2A9 are in a regulatory region with potential effects on the expression of SLC2A9.\" - \"Fig. 6: Whole genome bisulfite sequencing and ATAC-sequencing data of kidney tissue at SLC2A9.Whole genome bisulfite sequencing of kidney samples (nu2009 =u20095) showed that lower DNA methylation levels at CpGs localized to the promoter-associated region in isoform 1 on SLC2A9 and aligned with gene expression levels from RNA-sequencing (A). ATAC-sequencing data from primary human kidney tissue (nu2009=u20093 from micro-dissected cortex and medulla each) showed that the three promoter-associated CpGs, andu2014to a lesser extent cg20479063u2014mapped to DNA sequences with high chromatin accessibility in the kidney (B). Gene models were based on Genbank RefSeq (accession numbers: isoform 1, NM_020041; isoform 2, NM_001001290).\" - \"Lastly, we investigated differential gene expression in kidney tissue using two resources. Of the six urate-associated CpGs in SLC2A9 that were associated with differential SLC2A9 expression in monocytes, four did not show significant associations based on DNA methylation and gene expression levels quantified from micro-dissected human tubule tissue, and two did not meet quality control criteria (Methods, Supplementary Data_). In the future, investigation of specific SLC2A9 isoforms and/or of specific kidney cell types may shed light on the potential different effects of DNA methylation on gene expression levels in the kidney and monocyte. Second, we evaluated whether the genes implicated by the urate-associated CpGs from our study showed evidence for differential expression in kidney or intestine of a humanized mouse model of hyperuricemia (Methods). Of the genes nearest to the urate-associated CpGs, four genes were differentially expressed at false discovery rate (FDR)u2009<u20090.05 (ANKRD11, CEBPB, SLC7A11, and UAP1L1). SLC7A11 had lower expression in the intestine of mice with hyperuricemia (log2 fold change: u22121.34, FDRu2009=u20092.9Eu20132). ANKRD11 had lower expression in the kidney (log2 fold change: u22120.24, FDRu2009=u20095.6Eu20133; Supplementary Data_). These findings are consistent with the urate-associated CpGs being downstream of factors influencing serum urate levels in mice.\" - Enrichment of urate-associated CpGs in DNase I hypersensitive sites and transporters - \"The urate-associated CpGs were enriched in DNase I hypersensitive sites of all blood cells tested (FDR qu2009<u20090.01), with the strongest enrichment observed in CD3 cells (qu2009<u20091eu20139). Among histone modifications, H3K4me1, often found at active and primed enhancers, and H3K4me3, often found at active promoters, were enriched in all blood cells tested. While the former were most enriched in primary monocytes and primary hematopoietic stem cells, H3K4me3 showed the strongest enrichment in primary hematopoietic stem cells (Supplementary Figs._ and ). In contrast, H3K9me3 and H3K27me3, generally associated with heterochromatin, were not significantly enriched in any blood cells or tissues tested (Supplementary Figs._ and ). In addition, among transcription factor binding sites, the urate-associated CpGs were most enriched in POLR2A, the largest subunit of RNA polymerase II, the major enzyme synthesizing mRNA in eukaryotes (pu2009<u20091Eu201311, Supplementary Fig._). Together, the observed enrichments suggest a role of the urate-associated CpGs in transcriptional regulation.\" - \"We also assessed the enrichment of genes implicated by urate-associated CpGs in Gene Ontology (GO) terms, and the Kyoto Encyclopedia of Genes and Genomes (KEGG) and Reactome pathways (Methods). After correction for multiple testing, significant enrichment was observed for 55 terms (Supplementary Data_). The terms or pathways with the strongest enrichment were related to transmembrane transport of organic acids, carboxylic acids, amino acids, organic and inorganic anions, as well as processes in leukocytes and myeloid cells (Fig._). These terms suggest that differentially methylated CpGs capture information about small molecule transport in urate homeostasis.\" - \\'Fig. 7: GO terms and KEGG and Reactome pathways that were enriched for urate-associated CpGs.Significance level was set at false discovery rate <0.05. GO Gene Ontology, KEGG Kyoto Encyclopedia of Genes and Genomes.\\' - Urate-associated CpG sites and other traits - \"Our prior work on the genetics of serum urate revealed significant genetic correlations between serum urate and many cardiometabolic traits. Using published EWAS summary statistics, we investigated whether these relationships were also observed when studying epigenetic variation by examining the association between replicated urate-associated CpGs and other cardiometabolic traits (Methods, Supplementary Data_). Liver and kidney traits were also included due to the important role of these two organs in urate production and excretion. Most of these traits were included as covariates in the EWAS of serum urate with the goal of detecting urate-specific DNA methylation signals (Methods). Of the replicated urate-associated CpGs, 17 had significant associations with at least one cardiometabolic trait (Fig._ and Supplementary Data_). Interestingly, the directions of association were always consistent with the correlation between serum urate and these traits in epidemiological studies (Fig._). A few urate-associated CpGs showed associations with a particularly high number of cardiometabolic traits: cg14476101 annotated to PHGDH (six traits), cg06690548 annotated to SLC7A11 (seven traits), and cg02711608 annotated to SLC1A5 (six traits). Lower DNA methylation level were associated with higher serum urate levels, as were as higher levels of body mass index (BMI), blood pressure, triglycerides, liver enzymes, C-reactive protein (CRP), and incident diabetes in their respective EWAS (Fig._). These traits represent components of the metabolic syndrome or are strongly correlated with the components, as is serum urate. Given that the EWAS of the cardiometabolic traits adjusted for white blood cell composition in their analysis, the associations between DNA methylation and traits at the CpGs are independent of cell type composition. Together, these associations at these CpGs may reflect DNA methylation signatures of metabolic syndrome in blood. In contrast, none of the 99 replicated urate-associated CpGs were associated with fasting glucose and insulin among individuals without diabetes, who are less likely to be affected by the metabolic syndrome. The urate-associated CpGs were also not found among those associated with high-density and low-density cholesterol (HDL-C, LDL-C) and estimated glomerular filtration rate (eGFR). In addition, 4 of the 17 CpGs that were associated with serum urate and cardiometabolic traits (cg16246545 at PHGDH, cg19693031 at NBPF20/TXNIP, cg06690548 at SLC7A11, and cg06690548 at CPT1A) were associated with mostly lipid-related metabolites, quantified using nuclear magnetic resonance (Supplementary Data_). The associations of these four CpGs were largely observed with triglycerides in various lipid subfractions or lipid metabolites that are part of very low-density lipoprotein and had concordant effect directions as their associations with serum urate.\" - \\'Fig. 8: Urate-associated CpGs that were associated with cardiometabolic and kidney traits.All traits that were included in the lookup are shown, including those without reported associations with urate-associated CpGs. Blue: positive association between DNA methylation and trait levels; Red: inverse association between DNA methylation and trait levels. BMI body mass index, SBP systolic blood pressure, DBP diastolic blood pressure, TG triglycerides, HDL-C high-density lipoprotein cholesterol, LDL-C low-density lipoprotein cholesterol, inc. incident, GGT Gamma-glutamyl transferase, ALT alanine aminotransferase, CRP C-reactive protein, eGFR estimate glomerular filtration rate. Genetic correlations were reported in Tin et al. Nat Genet 2019. *Genetic correlation between serum urate and hypertension was 0.39. No EWAS of hypertension was available; SBP and DBP were used instead.\\'- - This large-scale EWAS, based upon data from up to 17,996 participants largely drawn from population-based studies, identified and replicated 99 CpGs, at which differential DNA methylation was significantly associated with serum urate levels. The genes implicated by these CpGs were strongly enriched in terms and/or pathways related to small molecule transport, including organic anions and amino acids. MR analyses supported significant causal effects of some CpGs on serum urate levels and gout at SLC2A9, the strongest GWAS locus for serum urate that encodes a major urate transport protein in the kidney. Mediation analyses supported that genetic variants at SLC2A9 affect serum urate levels partly via epigenetic mechanisms. Moreover, 17 urate-associated CpGs showed significant associations with numerous cardiometabolic traits, with the direction of association always consistent with the ones between serum urate and the respective cardiometabolic trait reported from observational studies. Differential DNA methylation at these sites may therefore reflect an epigenetic signature of cardiometabolic traits in whole blood. - Results from dedicated EWAS of serum urate levels have not been reported previously. One previous EWAS of blood metabolite levels in one of our contributing studies, KORA F4, examined urate as one of 649 metabolites, and reported significant associations between DNA methylation at cg00071950 in SLC2A9 and serum urate levels. This EWAS meta-analysis of measured serum urate levels lends further support to the reproducibility and validity of this prior finding through successful replication in different study populations, association with gene expression, and biological plausibility. - In addition to replicated associations between DNA methylation and serum urate, our study reveals significant causal effects of DNA methylation at some CpGs of SLC2A9 on serum urate and gout. Prior exposure to serum urate in its soluble or crystal form has been shown to heighten the proinflammatory response of myeloid cells in vitro and in animal models potentially through epigenetic mechanisms. This is also known as urate-induced training immunity. In our study, MR analysis did not identify significant causal effects of serum urate on DNA methylation, but it is conceivable that serum urate might act on other forms of epigenetic mechanisms, such as histone modification. - Serum urate levels show high genetic and observational correlations with kidney function measures and with cardiometabolic traits. Previous large EWAS of kidney function measures focused on eGFR in population-based studies or on changes in kidney function among patients with kidney disease. Despite the strong, inverse genetic and observational correlations between serum urate and eGFR and the well-established role of the kidney in serum urate homeostasis, the replicated urate-associated CpGs in our study did not show significant associations when assessed in EWAS of eGFR. In contrast, many urate-associated CpGs showed significant associations in results from EWAS of cardiometabolic traits, including triglycerides, blood pressure, BMI, liver enzymes, CRP, as well as diabetes. Many of these traits are either conditions that define the metabolic syndrome, or directly related to such conditions (high triglyceride levels, high blood pressure, high BMI, high blood glucose levels). There has been no robust evidence supporting causal effects of serum urate on cardiometabolic traits. Instead, our observations are consistent with shared gene regulatory programs resulting in a common DNA methylation signature of serum urate and metabolic syndrome in whole blood. - Our study also provides additional insights into a major transporter of serum urate. SLC2A9 has consistently been detected as the strongest locus in GWAS of serum urate with multiple independent signals, but the mechanism underlying the GWAS signal is largely unknown. The association between genetic variants at SLC2A9 and serum urate levels may be mediated by transcript- or tissue-specific regulatory mechanisms. SLC2A9 has two described isoforms whose gene products have equivalent urate transport activity but differ in the N-terminal cytosolic portion of the protein. SLC2A9 shows high complexity in human kidney tissue; the two isoforms of SLC2A9 have been reported to differ in cellular localization possibly affecting their function in urate homeostasis. Isoform 1 was found to be expressed at the basolateral side of the proximal tubule, while isoform 2 was expressed at the apical side of the collecting duct. The functional consequences of this complexity are not yet clear. In our study, we observed complex relations of DNA methylation levels at SLC2A9 with gene expression in monocytes and serum urate. Our findings support an inferred inverse relationship between DNA methylation-relatedSLC2A9 expression in monocytes and serum urate levels at some CpGs. An inverse relationship between GLUT9 expression in the kidney and serum urate would be difficult to explain. Loss-of-function mutations at SLC2A9 in humans result in type 2 familial renal hypouricemia (OMIM# 612076) supporting a role for GLUT9 as a high-capacity urate transporter in tubular urate reuptake from urine into blood. Therefore, a positive association between SLC2A9 gene expression in the kidney and serum urate levels would be expected. - However, SLC2A9 is expressed in a range of extra-renal tissues, with high abundance in the liver and lower abundance in the heart, lung, and intestine. In mouse hepatocytes, GLUT9 is expressed in the basolateral membrane, across which urate uptake from blood into the cell can occur. Therefore, the role of GLUT9-mediated transport of urate may potentially affect serum urate levels in different directions, depending on transcript isoform and tissue localization. For example, loss of function mutations in dogs and liver-specific inactivation of SLC2A9 in a mouse model result in hyperuricemia. In addition, basolateral localization of GLUT9 has also been reported in mouse enterocytes of the small intestine, and enterocyte-specificSlc2a9knock-out resulted in reduced intestinal urate excretion and hyperuricemia. A role for SLC2A9 in intestinal excretion would also be consistent with an inverse relationship between DNA methylation-relatedSLC2A9 expression and serum urate levels. Therefore, it is conceivable that the DNA methylation signals we detected at SLC2A9 in blood represent a signature for urate transport into enterocytes, hepatocytes, and/or other cell types, as a constituent of a urate secretion pathway, a mechanism that would lower serum urate levels. Roles for SLC2A9 in urate uptake into cells in both secretory and reabsorption pathways may also help explain some of the heterogeneity of associations of SNPs in the SLC2A9 locus. Regardless of the mechanism, the finding that urate-associated genetic variants at SLC2A9 could affect serum urate levels via epigenetic mechanisms may be of potential clinical relevance, as each SD higher DNA methylation at CpGs at SLC2A9 conferred up to 57% lower odds of gout, the most common form of inflammatory arthritis in adults. - Terms or pathways enriched for urate-associated CpGs were mainly related to transmembrane transport of organic acids, carboxylic acids, amino acids, and organic and inorganic anions. This does not only align with uric acid as an organic acid, but also with several genes encoding enzymes and transporters related to amino acid generation and movement across membranes. For example, PHGDH encodes 3-phosphoglycerate dehydrogenase, which catalyzes the oxidation of 3-phosphoglycerate to 3-phosphohydroxypyruvate, the committed step in the biosynthesis of L-serine. Inhibition of PHGDH alters nucleotide metabolism via disruption of mass balance within central carbon metabolism, leading to changes in the pentose phosphate pathway and the tri-carboxylic acid cycle. In addition, PHGDH also has a role in lipid metabolism. Knockdown of PHGDH in liver cell lines resulted in lower expression of LPL, the gene encoding lipoprotein lipase, and higher expression of LDLR and ABCA1, both important in lipid transport. The genes SLC1A5, SLC7A11, and SLC43A1 encode transport proteins for neutral amino acids, cysteine as well as glutamate, and for branched-chain amino acids, respectively. The CpGs in genes involved in the transport of small molecules such as amino acids, represented the CpGs with the strongest associations in this EWAS of serum urate. These CpGs were also associated with other cardiometabolic traits. Taken together, these results suggest that shared epigenetic regulatory mechanisms may have an important role in urate homeostasis. - Strengths of our study include its large sample size, the replication of findings in separate study samples, and the use of rigorous statistical methods. The agreement of findings based on multiple layers of complementary evidence and the biological plausibility of many of the results support the validity of our findings. These findings provide new insights into the epigenetic regulation of serum urate levels that are complementary to genetic association studies of serum urate. Potential limitations of our study relate to the quantification of DNA methylation from whole blood or from specific blood cell populations, which may not be representative of other important organs of urate homeostasis such as the kidney or liver. However, serum urate was also measured in blood, where its levels may be influenced by metabolic functions beyond simply reflecting purine metabolism. In addition, serum urate concentrations play an important role in gout, making blood an attractive target tissue for studies of urate homeostasis and gout. - In summary, meta-analyses of EWAS of serum urate levels and downstream characterization identify urate-associated CpGs that explain a substantial proportion of serum urate variance, mediate some of the genetic effects on serum urate levels, and show associations with several cardiometabolic traits consistent with the observed relationships between serum urate and these traits. These findings constitute complementary evidence to insights from GWAS of serum urate levels.- - Study samples and workflow - \"The discovery and replication analyses were conducted based on a pre-specified analysis plan. A script () was circulated to all participating cohorts (Supplementary Data_) to generate summary files that were inspected by a central analysis team. All studies were community-based or non-clinical populations. Each cohort conducted study-specific EWAS and uploaded epigenome-wide summary statistics. Before the meta-analysis, we conducted quality control of study-specific results, which included the assessment of the distributions of effect sizes, standard errors, and p values within each study and across studies. Supplementary Data_ reports study-specific methods, including the DNA methylation detection p value and processing pipeline. Supplementary Data_ reports the white blood cell proportions in each cohort, which were used as covariates to estimate associations between urate and DNA methylation independent of cell type composition. Study research protocols were approved by the respective ethics committees. All participants in all studies provided written informed consent.\" - DNA methylation quantification and quality control - \"Genomic DNA was extracted from peripheral blood. Levels of DNA methylation were quantified using the Infinium MethylationEPIC BeadChip array (EPIC, six studies), the Illumina Infinium HumanMethylation450K BeadChip array (HM450K, 17 studies), or the Illumina Infinium HumanMethylation27 BeadChip array (HM27K, one study). DNA methylation data preprocessing was performed according to individual study protocols, which included background correction, quantile normalization, probe filtering, sample filtering, SNP matching to the SNP control probe locations, outlier filtering, and assay type correction (Supplementary Data_).\" - Study-specific EWAS - \"Ancestry-specific association analyses were conducted within each study. Among the 24 contributing cohorts, 14 cohorts excluded participants who were on urate-lowering medications, whereas the other cohorts did not have this information available (Supplementary Data_). The methylation levels at each CpG probe were represented as beta values, which can be interpreted as the proportion of CpG sites that were methylated. The beta values were analyzed as the dependent variable with serum urate as the independent variable in a linear regression model adjusting for age, sex, BMI, current smoking, eGFR, HDL-C, systolic blood pressure (SBP), log-transformed levels of CRP and triglycerides, genetic principal components (PC), and estimated or measured blood cell type proportions in order to allow for detecting associations between serum urate and DNA methylation levels independent of differences in cell type composition. Additional study-specific technical covariates included control probe PCs, study center, processing batch, sentrix ID, and sentrix position, as applicable.\" - Discovery and replication meta-analysis - \"Studies were separated into discovery and replication cohorts by chronological order of contribution to this project (Supplementary Data_). Only CpG sites common to both the EPIC and HM450K were included in the meta-analysis (nu2009 =u2009441,854). CpG probes overlapping with SNPs were annotated using information from Illumina.\" - \\'Prior to meta-analysis, each set of study-specific results was adjusted for test statistic inflation using the BACON method, which was developed to control for bias and inflation in EWAS using an empirical null distribution approach and assumes that the test statistics are a mixture of three distributions: negative, positive and null associations. The inflation factor is estimated from the distribution of null association. We conducted inverse-variance weighted fixed effects meta-analysis using the R package metafor (version 2.1-0) for discovery, replication, and for combining the results of discovery and replication. CpG sites were excluded if their available sample size was less than half of the sample size in the respective meta-analyses, or if their I heterogeneity estimate was >95%.\\' - \"The statistical significance threshold for the discovery step was pu2009<u2009 1.1Eu20137 (=0.05/441,854). The replication criteria were: Pdiscoveryu2009<u2009 1.1Eu20137, Preplicationu2009<u20090.05 with consistent effect direction, and Pcombinedu2009<u20091.1Eu20137. For the replicated CpGs, we quantified study heterogeneity within an ancestry using the I from a meta-analysis including all studies within an ancestry, namely EA (17 studies) and AA (5 studies). We further quantified heterogeneity among the four ancestry groups using the I from a transethnic meta-analysis that combined the summary statistics of the meta-analyses of EA, and AA with those from the LOLIPOP study (SA), and the RODAM study (SSA). Given that there was only a single study each including participants with SA and SSA ancestry, the I from this transethnic meta-analysis might represent both ancestry and study heterogeneities.\" - Heritability of replicated CpG sites and explained urate variance - \\'We used three sources of DNA methylation heritability estimates from whole blood to characterize the replicated urate-associated CpGs: (a) a twin study of 1464 individuals from Great Britain [Hannon et al.], (b) a twin study of 2386 individuals from the Netherlands [van Dongen et al.], and (c) a family-based study of 614 individuals from Australia [McRae et al.]. Hannon et al. included 426 monozygotic (MZ) and 306 same-sex dizygotic (DZ) twin pairs to estimate the proportion of the variance of DNA methylation explained by additive genetic (A), shared environment (C), and unshared environmental (E) factors. Van Dongen et al. included 769 MZ and 424 DZ twin pairs for the ACE model analysis. McRae et al. studied 614 individuals from 117 families and used intraclass correlation analysis to partition the variance of DNA methylation into additive genetics and environmental components. DNA methylation levels of these three studies were quantified using the HM450K array.\\' - \\'The proportion of serum urate variance explained was calculated based on data from 1832 participants of the KORA-FF4 study with DNA methylation levels quantified using blood samples collected in 2014. Among the participants of the KORA-FF4 study, 988 were also participants of the KORA F4 cohort, one of the discovery cohorts, with DNA methylation levels quantified using blood samples collected in 2007, 7 years earlier than those in the KORA-FF4 study. Data were available for 97 of the 99 replicated, urate-associated CpGs (missing observations for cg08257009 and cg05201185). To enable comparison of the proportion of explained urate variance to the one obtained from age- and sex-adjusted GWAS summary statistics, two models were computed: a base model of the proportion of urate variance explained by age and sex, and an extended model that additionally included residuals from a regression of the CpGs on sex, age, and cell type composition. The proportion of serum urate variance explained by the replicated CpGs was then calculated as the difference in coefficient of determination (R) between the two models.\\' - \\'Forward Mendelian randomization (MR) analysis: causal effects of DNA methylation on serum urate and gout\\' - \\'To evaluate whether the urate-associated CpGs might have a causal role in influencing serum urate levels or gout, we conducted two-sample MR analyses using meQTLs as genetic instruments of the replicated CpGs. The random assignment of alleles during meiosis mimics treatment allocation in randomized controlled trials. Using meQTLs as proxies or genetic instruments of DNA methylation enables inferences on the causal effects of DNA methylation on serum urate or gout, provided that the meQTLs meet the instrumental variable assumptions: (1) the meQTLs are associated with DNA methylation, (2) the meQTLs are not associated with potential confounders of the association between DNA methylation and serum urate or gout, and (3) the meQTLs are only associated with serum urate or gout through DNA methylation, i.e., without pleiotropic effects. While assumptions 2 and 3 cannot be fully verified, we have addressed these assumptions in our selection criteria of meQTLs and analysis methods as detailed below.\\' - \"The meQTLs of the CpGs were selected two steps. In the first step, we addressed the three instrumental variable assumptions. To address assumption 1, we selected independent cis meQTLs as genetic instruments with strong association with DNA methylation (pu2009<u20095Eu20138). To address assumptions 2 and 3, we removed those with genome-wide significant association (pu2009<u20095Eu20138) with potential confounders, and with stronger associations with the outcome than with DNA methylation, which suggests pleiotropic effects of a meQTL or may lead to reverse causation. In this first step of instrument selection, we selected cis meQTLs for each CpG (within 500u2009kb of the CpG) from the meta-analysis of the GoDMC with MAFu2009>u20091% and pu2009<u20095eu20138 to avoid weak instrument bias. The meQTLs were further pruned based on the following criteria: (a) a clumping threshold of ru2009<u20090.05, (b) associated with potential confounders (BMI, current smoking, HDL-C, SBP, CRP, and triglycerides) at pu2009 <u20095Eu20138, (c) failed Steiger filtering, i.e., displayed stronger association with the outcome than the exposure, and (d) failed harmonization checks of the harmonise_data function in the TwoSampleMR package, which matches the effect allele of the SNPs associated with the exposure and the outcome and flags palindromic SNPs with allele frequency close to 0.5 as ambiguous. The sources of the GWAS summary statistics of the potential confounders are reported in Supplementary Data_. In the second step, to ensure that the meQTLs for each CpG were indeed conditionally independent, we conducted conditional analysis using Genome-wide Complex Trait Analysis (GCTA) to obtain the conditional p value for each meQTL controlling for other meQTLs of the same CpG (command: cojo-cond). As the reference panel, we used 13,558 randomly selected individuals of British descent, and 16,969,363 SNPs with MAFu2009>u20090.01% after quality control as reported previously. We retained only meQTLs with conditional pu2009<u20095Eu2013 8 as genetic instruments of the CpGs. The median numbers of meQTLs among the 92 replicated CpGs outside of SLC2A9 was 2 (25th, 75th percentile: 0, 5), whereas it was 20 (25th, 75th percentile: 8, 21) among the 7 CpGs at SLC2A9.\" - \"The genetic association summary statistics for meQTLs, serum urate, and gout were generated among EA individuals. The meQTL summary statistics were obtained from a meta-analysis of the GoDMC Consortium (max. nu2009=u200927,750 from 36 EA studies), which included some of the participating studies of the EWAS of serum urate. DNA methylation in GoDMC was measured in whole blood or cord blood using the HM450K or EPIC BeadChips. The 1000 Genomes reference panel was used for genotype imputation. The analysis in each cohort had two steps. First, residuals of DNA methylation beta values were generated by adjusting out age, sex, predicted white blood cell type proportions, predicted smoking status based on published results of smoking-associated DNA methylation, and genetic PCs. Next, these residuals were rank transformed and standardized and used for genetic association analysis with adjustment for relatedness in family-based studies. The SNP effect sizes can be interpreted as SD higher DNA methylation per allele. The meta-analysis was conducted in two phases. First, each GoDMC study analyzed the association between all SNPs and all CpGs, returning only associations with pu2009<u20091Eu20135. Next, these associations with pu2009<u20091eu2013 5 were combined to create a candidate list of meQTL associations, which included cis associations found in at least one dataset and trans associations in at least two datasets. The association statistics of these candidate meQTLs (nu2009 =u2009120,212,413) were obtained from all cohorts, and then combined using fixed-effect meta-analyses.\" - \"The GWAS summary statistic for serum urate (nu2009=u2009288,649) and gout (nu2009 =u2009692,537) were obtained from recent GWAS meta-analyses of EA, samples from the CKDGen Consortium. We required each CpG to have at least four cis (<500u2009 kb) meQTLs as genetic instruments. As the primary MR analysis method, we used the multiplicative random effect inverse-variance weighted method as recommended by the guidelines for MR investigation. We used the Egger intercept to evaluate potential pleiotropy, and Cochrans Q test for assessing heterogeneity. Sensitivity analysis using other MR methods that are robust to pleiotropy were conducted: MR Egger, simple mode, weighted median, and weighted mode. To assess whether a causal estimate may be largely driven by a single genetic instrument, we conducted leave-one-out analysis. To assess whether the causal effects of the CpGs at SLC2A9 on serum urate were partly due to GWAS signals at this locus, we conducted an additional sensitivity analysis excluding meQTLs with ru2009 >u20090.05 with any of five GWAS index SNPs at SLC2A9 (rs4447862, rs10017305, rs6825187, rs62286563, and rs73224492) among individuals of EA, reported in Tin et al.. The SNP rs4447862 had the lowest p value in the SLC2A9 region, and the other four SNPs were detected as genome-wide significant independent SNPs by the GCTA stepwise model selection command (cojo-slct). All MR analyses were conducted using the TwoSampleMR package. Calculations of the minimum detectable effect size in the forward MR analysis of DNA methylation on serum urate or gout supported that the MR analysis was well powered (Supplementary Note_).\" - \\'Reverse MR analysis: causal effect of serum urate on DNA methylation\\' - To evaluate the potential causal effects of serum urate on DNA methylation levels, we conducted two-sample reverse MR analysis. The genetic instruments for serum urate were 123 index SNPs from a recent GWAS meta-analysis of 288,649 EA participants. The associations between these SNPs and DNA methylation levels were generated among 3866 EA participants of the Framingham Heart Study, a sample independent of the EWAS of serum urate. The ARIC study was not included in the reverse MR because most meQTL data were from AA participants, whereas the urate summary statistics were based on data from EA individuals. The DNA methylation levels were quantified as beta values from the HM450K array. The analysis of the association between SNPs and DNA methylation levels adjusted for age, sex, the top 50 methylation PCs, predicted blood cell fractions, and used linear mixed model to account for family structure. The reverse MR did not use the GoDMC meQTL results employed in the forward MR analysis, because the GoDMC meQTL results only contained SNPs associated with DNA methylation levels below a certain significance level, as reported above. Given that the urate index SNPs were selected in separate 1-Mb regions across the genome, we did not conduct GCTA conditional analysis to reassess the independence among these index SNPs. Otherwise, the selection of the SNPs for serum urate, their harmonization, and MR methods for primary and sensitivity analysis were the same as those in the forward MR analysis. - Mediation analysis - \"Given that four CpGs at SLC2A9 showed significant causal effects for serum urate, we evaluated whether these CpGs may mediate the genetic effects of the EA index SNPs of serum urate at the SLC2A9 locus. As reported previously, four independent SNPs in two neighboring 1u2009Mb intervals in the SLC2A9 region were identified based on GCTA stepwise model selection. Mediation analyses of the four CpGs for the four index SNPs were first conducted among 637 EA ARIC participants, controlling for age, sex, study center, and ten genetic PCs using the mediation package version 4.5 in R. This mediation method uses simulation to estimate the average causal mediation effect in a potential outcome framework. The significance threshold was set at 0.0125 (0.05/4 independent index SNPs). For the significant index SNP-CpG pairs, mediation analyses were performed in the SHIP-Trend (nu2009 =u2009223) and KORA (nu2009=u20091636) cohorts. The mediation effects of each CpG were then combined using fixed-effect inverse-variance meta-analysis as implemented in metafor.\" - Association with gene expression and chromatin accessibility - We investigated the association of the urate-associated CpGs with gene expression in cis in monocytes and whole blood. Monocyte data were obtained from Kennedy et al., who assessed these associations among 1202 individuals from the Multi-Ethnic Study of Atherosclerosis. DNA methylation levels were quantified using the HM450K array, and gene expression levels were quantified using Illumina HumanHT-12 version 3.0 and 4.0 Expression BeadChips. The regression analysis used log-transformed expression levels as the dependent variable and DNA methylation beta values as the independent variable controlling for age, race, sex, and study site. - \"The association of DNA methylation levels with gene expression in whole blood was obtained from 713 participants of the KORA study. The DNA methylation levels were quantified using the HM450K array, and gene expression levels were quantified using Illumina HumanHT-12 v3 Expression BeadChips. The log2-transformed gene expression values were regressed on the DNA methylation beta values adjusted for sex and age. Prior to the analysis the technical factors as well as the blood cell type proportions were regressed out of the mRNA and DNA methylation levels, and their residuals were included in the final association model. Statistical significance for the association between urate-associated CpGs and gene expression was defined as pu2009<u20095Eu20134 (=0.05/99).\" - To evaluate whether urate-associated CpGs at SLC2A9 might be associated with gene expression in the kidney, we selected five CpGs at the SLC2A9 locus that were significantly associated with SLC2A9 gene expression in monocytes (cg03725404, cg20479063, cg02734326, cg00071950, cg11266682). The DNA methylation and gene expression levels were quantified from tubulo-interstitial kidney tissue of 314 persons. The analysis used gene expression (transcripts per million) as the dependent variable and DNA methylation beta values as the independent variable controlling for age, sex, five genetic PCs, prevalent diabetes and hypertension, BMI, batch factors, bisulfite conversion rate, RNA-sequencing batch, and RNA integrity number. - To explore the epigenetic and gene expression landscape in the SLC2A9 locus in kidney tissues, we looked up the DNA methylation profiles detected by WGBS and gene expression profiles from RNA-sequencing in five human normal kidneys (GEO accession GSE115098). The methylation beta values from WGBS and read counts from RNA-sequencing at the SLC2A9 locus (GRCh37/hg19, chr4:9884000-10067000) were converted to bigwig format. The bigwig files were imported into IGV genome browser for visualization. In addition, we investigated whether the urate-associated CpGs at SLC2A9 might be mapped into open chromatin regions in the kidney using data generated from ATAC-seq (Assay for Transposase-Accessible Chromatin using sequencing). Kidney cortex and medulla tissues were macro-dissected from uninvolved portions of tumor nephrectomy specimens and snap-frozen. ATAC-seq was performed by ActiveMotif (Carlsbad, California) and the subsequent data were mapped to GRCh38 and normalized by read depth. Peak calling was performed using MACS 2.1.0 and filtered against the ENCODE blacklist. Master list generation and set operations with the resulting bed files were performed using bedops. GRCh38 positions were mapped back to GRCh37/hg19 using the online liftOver tool () - Differential gene expression in humanized mouse model of hyperuricemia - \"To investigate whether the closest genes of the urate-associated CpGs might be differentially expressed in hyperuricemic condition, we obtained gene expression data from a humanized mouse model of hyperuricemia. The orthologous mouse variant to the human ABCG2 Q141K (ABCG2 Q140K) was knocked in to the endogenous Abcg2 locus using CRISPR/Cas9 on a C57BL/6J background, resulting in hyperuricemia in adult male Q140K+/+ mice as described previously. Animal studies were performed in adherence to the NIH Guide for the Care and Use of Laboratory Animals and approved by the University of Maryland School of Medicine Institutional Animal Care and Use Committee. Mice were housed in groups of two to five per cage on a 12:12u2009h light/dark cycle; with lights on at 6u2009a.m., only male mice were used for the analysis. Animal organs were harvested and preserved in RNALater (Sigma R0901) and gradually cooled from 4u2009xB0C to u221280u2009xB0C. RNA was isolated using the RNeasy Plus Mini Kit (Qiagen 74134) per the manufacturer s protocol. RNA was suspended in molecular biology grade water and then quantified with a CLARIOstar plate reader and stored at u221280u2009xB0C. Illumina Sequencing Libraries were prepared using manufacturers protocol for NEB Ultra II Directional RNA Library Prep kit with poly-A enrichment (NEB E7760). Samples were sequenced on four flowcell lanes of an Illumina HiSeq4000 75u2009bp paired end run. Five samples were sequenced in each flowcell lane. Samples were grouped per lane by tissue type. Sample quality assessment, RNA-Seq library preparation, and sequencing were performed by the Genomics Resource Center at the University of Maryland School of Medicine. RNA-Seq data was stored and analyzed on BasePair () for expression count (STAR) and differential expression (DESeq2) analyses. Mouse gene names were mapped to human gene names in HUGO Gene Nomenclature using the Mouse Genome Informatics database. FDR for differential expression was calculated using the Benjaminiu2013Hochberg method. We consider FDRu2009<u20090.05 as being significant.\" - Enrichment analyses - \"To inform the potential functional effects of the urate-associated CpGs, we assessed the enrichment of these CpGs in sites of DNase I or histone modification (H3K4me1, H3K4me3, H3K9me3, H3K27me3), gene sets based on GO terms and pathways in the KEGG and Reactome databases. The enrichment analyses of DNase I or histone modification were performed using a local version of eForge version 2.0 (experimentally derived Functional element Overlap analysis of ReGions from EWAS). Briefly, eForge used date sources from either the ENCODE (125 samples) or Roadmap Epigenomics (299 samples) projects generated by the Hotspot method. The overlaps between the input CpGs and the data sources were compared with those from 1000 random sets of CpGs with matching gene relationship and CpG island annotation. FDR was obtained based on the binomial distribution and the Benjaminiu2013Yekutieli method for multiple testing correction. Our input included 1018 urate-associated CpGs with pu2009<u20091Eu201305 in the combined meta-analysis of the discovery and replication cohorts. We performed 10,000 resampling runs with an active proximity filter and considered FDRu2009<u20090.01 as significant, i.e., enrichment >99 percentile.\" - \"Enrichment in gene sets or pathways was performed using the methylGSA package and R version 3.6.1. The enrichment test method (methylglm) was a functional class scoring method implemented using logistic regression accounting for the number of CpG sites per gene and the autosomal background that overlaps the HM450K and EPIC arrays. Gene sets or pathways with 100u2013500 genes were tested (default setting). We considered a gene set or pathway to be significantly enriched at FDRu2009<u20090.05 correcting for multiple testing within each database using the Benjamini and Hochberg method.\" - Relationship of urate-associated CpGs with cardiometabolic traits - \"Our prior work on the genetics of serum urate revealed significant genetic correlations between serum urate and many cardiometabolic traits and showed that genes mapping into urate-associated loci were highly enriched for expression in kidney and liver. Other than gout, cardiometabolic traits with high genetic correlations (ru2009>u20090.35) included triglycerides, HDL cholesterol, diabetes, fasting insulin, hypertension, and BMI. To gain insights on whether these relationships were also observed when studying DNA methylation in blood, we investigated whether the urate-associated CpGs were also identified in EWAS of the respective cardiometabolic traits as well as in EWAS of eGFR and liver enzymes. We also included CRP, given that CRP is a marker of inflammation, a component of metabolic syndrome. When more than one EWAS were available for a trait, EWAS were prioritized for the larger sample size. The primary focus was placed on the interpretation of the direction of association of DNA methylation with serum urate and the other traits. The CpGs included in the lookup were those replicated in the EWAS of urate and also considered significant in the EWAS of the other traits based on criteria of the respective study (Supplementary Data_). All studies included in this lookup controlled for age, sex, and white blood cell type proportions for detecting independent association between DNA methylation and the trait. More details are provided in Supplementary Data_. When a study included replication and provided meta-analysis results of discovery and replication, the effects from the meta-analysis combining discovery and replication were used.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\" - Disclaimer - The views expressed in this manuscript are those of the authors and do not necessarily represent the views of the National Heart, Lung, and Blood Institute, the National Institutes of Health, or the US Department of Health and Human Services.- - \\' \\' - The summary statistics from the meta-analysis are available from the CKDGen Consortium website (). Additional data and programming code that support the findings of this study such as code used to call on software are available from the authors upon request. - \\' \\'- - \\' \\' - The script for generating the phenotypes used in the EWAS is available via GitHub (). EWAS QC, meta-analysis, and postprocessing were implemented in R v4.0.1 using metafor v2.4.0, qqman v0.1.4, limma v3.42.2, openxlsx v4.1.5, car v3.0.8, bacon v1.16.0, mutoss v0.1.12, methylGSA v1.6.1, ggplot2 v3.3.3, SeSAMe v1.10.5, rmeta v3.0, mediation v4.5.0, TwoSampleMR R v0.5.6, and Genome-wide Complex Trait Analysis (GCTA) v1.93.2beta. - \\' \\'- - \"Kottgen, A. et al. Genome-wide association analyses identify 18 new loci associated with serum urate concentrations. Nat. Genet. 45, 145u2013154 (2013)._ _ _ Kanai, M. et al. Genetic analysis of quantitative traits in the Japanese population links cell types to complex human diseases. Nat. Genet. 50, 390u2013400 (2018)._ _ _ _ Tin, A. et al. Target genes, variants, tissues and transcriptional pathways influencing human serum urate levels. Nat. Genet. 51, 1459u20131474 (2019)._ _ _ _ _ Roddy, E. & Choi, H. K. Epidemiology of gout. Rheum. Dis. Clin. North Am. 40, 155u2013175 (2014)._ _ _ _ Choi, H. K., Atkinson, K., Karlson, E. W., Willett, W. & Curhan, G. Purine-rich foods, dairy and protein intake, and the risk of gout in men. N. Engl. J. Med. 350, 1093u20131103 (2004)._ _ _ _ Dalbeth, N. et al. Gout. Nat. Rev. Dis. Prim. 5, 69 (2019)._ _ _ Feig, D. I., Kang, D. H. & Johnson, R. J. Uric acid and cardiovascular risk. N. Engl. J. Med. 359, 1811u20131821 (2008)._ _ _ _ _ SundstrxF6m, J. et al. Relations of serum uric acid to longitudinal blood pressure tracking and hypertension incidence. Hypertension 45, 28u201333 (2005)._ _ _ Juraschek, S. P., Miller, E. R. III & Gelber, A. C. Body mass index, obesity, and prevalent gout in the United States in 1988-1994 and 2007-2010. Arthritis Care Res. (Hoboken) 65, 127u2013132 (2013)._ _ Emmerson, B. T., Nagel, S. L., Duffy, D. L. & Martin, N. G. Genetic control of the renal clearance of urate: a study of twins. Ann. Rheum. Dis. 51, 375u2013377 (1992)._ _ _ _ _ Rice, T. et al. Heterogeneity in the familial aggregation of fasting serum uric acid level in five North American populations: the Lipid Research Clinics Family Study. Am. J. Med. Genet. 36, 219u2013225 (1990)._ _ _ _ Major, T. J., Dalbeth, N., Stahl, E. A. & Merriman, T. R. An update on the genetics of hyperuricaemia and gout. Nat. Rev. Rheumatol. 14, 341u2013353 (2018)._ _ _ _ Mandal, A. K. & Mount, D. B. The molecular physiology of uric acid homeostasis. Annu. Rev. Physiol. 77, 323u2013345 (2015)._ _ _ _ Futagi, Y., Narumi, K., Furugen, A., Kobayashi, M. & Iseki, K. Molecular characterization of the orphan transporter SLC16A9, an extracellular pH- and Na+-sensitive creatine transporter. Biochem. Biophys. Res. Commun. 522, 539u2013544 (2020)._ _ _ _ Woodward, O. M. et al. Identification of a urate transporter, ABCG2, with a common functional polymorphism causing gout. Proc. Natl Acad. Sci. USA. 106, 10338u201310342 (2009)._ _ _ _ _ _ Vitart, V. et al. SLC2A9 is a newly identified urate transporter influencing serum urate concentration, urate excretion and gout. Nat. Genet. 40, 437u2013 442 (2008)._ _ _ _ Kooner, J. S. et al. Genome-wide association study in individuals of South Asian ancestry identifies six new type 2 diabetes susceptibility loci. Nat. Genet. 43, 984u2013989 (2011)._ _ _ _ _ Voight, B. F. et al. Twelve type 2 diabetes susceptibility loci identified through large-scale association analysis. Nat. Genet. 42, 579u2013589 (2010)._ _ _ _ _ Teslovich, T. M. et al. Biological, clinical and population relevance of 95 loci for blood lipids. Nature 466, 707u2013713 (2010)._ _ _ _ _ _ Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434u2013443 (2020)._ _ _ _ _ _ Kilpinen, H. & Dermitzakis, E. T. Genetic and epigenetic contribution to complex traits. Hum. Mol. Genet. 21, R24u2013R28 (2012)._ _ _ _ Hannon, E. et al. Characterizing genetic and environmental influences on variable DNA methylation using monozygotic and dizygotic twins. PLoS Genet. 14, e1007544u2013e1007544 (2018)._ _ _ _ van Dongen, J. et al. Genetic and environmental influences interact with age and sex in shaping the human methylome. Nat. Commun. 7, 11115 (2016)._ _ _ _ _ McRae, A. F. et al. Contribution of genetic variation to transgenerational inheritance of DNA methylation. Genome Biol. 15, R73 (2014)._ _ _ _ Suzuki, M. M. & Bird, A. DNA methylation landscapes: provocative insights from epigenomics. Nat. Rev. Genet 9, 465u2013476 (2008)._ _ _ _ Uhlen, M. et al. Proteomics. Tissue-based map of the human proteome. Science 347, 1260419 (2015)._ _ _ Kennedy, E. M. et al. An integrated -omics analysis of the epigenetic landscape of gene expression in human blood cells. BMC Genomics 19, 476 (2018)._ _ _ _ Anastasiadi, D., Esteve-Codina, A. & Piferrer, F. Consistent inverse correlation between DNA methylation of the first intron and gene expression across tissues and species. Epigenetics Chromatin 11, 37 (2018)._ _ _ _ Kimura, T., Takahashi, M., Yan, K. & Sakurai, H. Expression of SLC2A9 isoforms in the kidney and their localization in polarized epithelial cells. PLoS One 9, e84996 (2014)._ _ _ _ _ Local, A. et al. Identification of H3K4me1-associated proteins at mammalian enhancers. Nat. Genet. 50, 73u2013 82 (2018)._ _ _ _ Sharifi-Zarchi, A. et al. DNA methylation regulates discrimination of enhancers from promoters through a H3K4me1-H3K4me3 seesaw mechanism. BMC Genomics 18, 964 (2017)._ _ _ _ Young, M. D. et al. ChIP-seq analysis reveals distinct H3K27me3 profiles that correlate with transcriptional activity. Nucleic Acids Res. 39, 7415u20137427 (2011)._ _ _ _ _ Nicetto, D. et al. H3K9me3-heterochromatin loss at protein-coding genes enables developmental lineage specification. Science 363, 294u2013297 (2019)._ _ _ _ _ _ Maiuolo, J., Oppedisano, F., Gratteri, S., Muscoli, C. & Mollace, V. Regulation of uric acid metabolism and excretion. Int J. Cardiol. 213, 8u201314 (2016)._ _ _ Kodama, S. et al. Association between serum uric acid and development of type 2 diabetes. Diabetes Care 32, 1737u20131742 (2009)._ _ _ _ _ Krishnan, E., Kwoh, C. K., Schumacher, H. R. & Kuller, L. Hyperuricemia and incidence of hypertension among men without metabolic syndrome. Hypertension 49, 298u2013303 (2007)._ _ _ _ Choi, H. K. & Ford, E. S. Prevalence of the metabolic syndrome in individuals with hyperuricemia. Am. J. Med. 120, 442u2013447 (2007)._ _ _ Grundy Scott, M., Brewer, H. B., Cleeman James, I., Smith Sidney, C. & Lenfant, C. Definition of metabolic syndrome. Circulation 109, 433u2013438 (2004)._ _ _ _ Gomez-Alonso, M. D. C. et al. DNA methylation and lipid metabolism: an EWAS of 226 metabolic measures. Clin. Epigenetics 13, 7 (2021)._ _ _ _ _ Petersen, A.-K. et al. Epigenetics meets metabolomics: an epigenome-wide association study with blood serum metabolic traits. Hum. Mol. Genet. 23, 534u2013 545 (2014)._ _ _ _ Cabu0103u, G., Criu0219 an, T. O., KlxFCck, V., Popp, R. A. & Joosten, L. A. B. Urate-induced immune programming: consequences for gouty arthritis and hyperuricemia. Immunol. Rev. 294, 92u2013105 (2020)._ _ _ Criu0219an, T. O. et al. Soluble uric acid primes TLR-induced proinflammatory cytokine production by human primary cells via inhibition of IL-1Ra. Ann. Rheum. Dis. 75, 755u2013 762 (2016)._ _ _ Chu, A. Y. et al. Epigenome-wide association studies identify DNA methylation associated with kidney function. Nat. Commun. 8, 1286 (2017)._ _ _ _ _ Gluck, C. et al. Kidney cytosine methylation changes improve renal function decline estimation in patients with diabetic kidney disease. Nat. Commun. 10, 2461 (2019)._ _ _ _ _ Dekkers, K. F. et al. Blood lipids influence DNA methylation in circulating cells. Genome Biol. 17, 138 (2016)._ _ _ _ Richard, M. A. et al. DNA methylation analysis identifies loci for blood pressure regulation. Am. J. Hum. Genet. 101, 888u2013902 (2017)._ _ _ _ _ Wahl, S. et al. Epigenome-wide association study of body mass index, and the adverse outcomes of adiposity. Nature 541, 81u201386 (2017)._ _ _ _ _ Nano, J. et al. Epigenome-wide association study identifies methylation sites associated with liver enzymes and hepatic steatosis. Gastroenterology 153, 1096u20131106.e2 (2017)._ _ _ _ Ligthart, S. et al. DNA methylation signatures of chronic low-grade inflammation are associated with complex diseases. Genome Biol. 17, 255 (2016)._ _ _ _ Cardona, A. et al. Epigenome-wide association study of incident type 2 diabetes in a british population: EPIC-Norfolk Study. Diabetes 68, 2315u20132326 (2019)._ _ _ _ _ Li, X. et al. MR-PheWAS: exploring the causal effect of SUA level on multiple disease outcomes by using genetic instruments in UK Biobank. Ann. Rheum. Dis. 77, 1039u20131047 (2018).Li, X. et al. Serum uric acid levels and multiple health outcomes: umbrella review of evidence from observational studies, randomised controlled trials, and Mendelian randomisation studies. BMJ 357, j2376 (2017)._ _ _ _ Doring, A. et al. SLC2A9 influences uric acid concentrations with pronounced sex-specific effects. Nat. Genet. 40, 430u2013436 (2008)._ _ _ Yang, Q. et al. Multiple genetic loci influence serum urate levels and their relationship with gout and cardiovascular disease risk factors. Circ. Cardiovasc. Genet. 3, 523u2013530 (2010)._ _ _ _ _ Matsuo, H. et al. Genome-wide association study of clinically defined gout identifies multiple risk loci and its association with clinical subtypes. Ann. Rheum. Dis. 75, 652u2013659 (2016)._ _ _ _ Li, S. et al. The GLUT9 gene is associated with serum uric acid levels in Sardinia and Chianti cohorts. PLoS Genet. 3, e194 (2007)._ _ _ _ Tin, A. et al. Genome-wide association study for serum urate concentrations and gout among African Americans identifies genomic risk loci and a novel URAT1 loss-of-function allele. Hum. Mol. Genet. 20, 4056u2013 4068 (2011)._ _ _ _ _ Anzai, N. et al. Plasma urate level is directly regulated by a voltage-driven urate efflux transporter URATv1 (SLC2A9) in humans. J. Biol. Chem. 283, 26834u201326838 (2008)._ _ _ _ Phay, J. E., Hussain, H. B. & Moley, J. F. Cloning and expression analysis of a novel member of the facilitative glucose transporter family, SLC2A9 (GLUT9). Genomics 66, 217u2013220 (2000)._ _ _ _ Keembiyehetty, C. et al. Mouse glucose transporter 9 splice variants are expressed in adult liver and kidney and are up-regulated in diabetes. Mol. Endocrinol. 20, 686u2013697 (2006)._ _ _ _ Witkowska, K. et al. hGLUT9 as a novel urate transporter: its role in liver urate handling and functional study of SLC2A9 SNPs. FASEB J. 23, 797.4u2013797.4 (2009)._ _ Preitner, F. et al. Glut9 is a major regulator of urate homeostasis and its genetic inactivation induces hyperuricosuria and urate nephropathy. Proc. Natl Acad. Sci. USA. 106, 15501u201315506 (2009)._ _ _ _ _ _ Bannasch, D. et al. Mutations in the SLC2A9 gene cause hyperuricosuria and hyperuricemia in the dog. PLoS Genet. 4, e1000246 (2008)._ _ _ _ Hoque, K. M. et al. The ABCG2 Q141K hyperuricemia and gout associated variant illuminates the physiology of human urate excretion. Nat. Commun. 11, 2767 (2020)._ _ _ _ _ _ DeBosch, B. J., Kluth, O., Fujiwara, H., SchxFCrmann, A. & Moley, K. Early-onset metabolic syndrome in mice lacking the intestinal uric acid transporter SLC2A9. Nat. Commun. 5, 4642 (2014)._ _ _ _ _ Reid, M. A. et al. Serine synthesis through PHGDH coordinates nucleotide levels by maintaining central carbon metabolism. Nat. Commun. 9, 5442 (2018)._ _ _ _ _ _ Karabegoviu0107, I. et al. Epigenome-wide association meta-analysis of DNA methylation with coffee and tea consumption. Nat. Commun. 12, 2830 (2021)._ _ _ _ _ Kanai, Y. et al. The SLC1 high-affinity glutamate and neutral amino acid transporter family. Mol. Asp. Med. 34, 108u2013120 (2013)._ _ _ Fotiadis, D., Kanai, Y. & PalacxEDn, M. The SLC3 and SLC7 families of amino acid transporters. Mol. Asp. Med. 34, 139u2013 158 (2013)._ _ _ Bodoy, S., Fotiadis, D., Stoeger, C., Kanai, Y. & PalacxEDn, M. The small SLC43 family: facilitator system l amino acid transporters and the orphan EEG1. Mol. Asp. Med. 34, 638u2013645 (2013)._ _ _ Nigam, S. K. & Bhatnagar, V. The systems biology of uric acid transporters: the role of remote sensing and signaling. Curr. Opin. Nephrol. Hypertens. 27, 305u2013313 (2018)._ _ _ _ _ Takada, T. et al. ABCG2 dysfunction increases serum uric acid by decreased intestinal urate excretion. Nucleosides Nucleotides Nucleic Acids 33, 275u2013281 (2014)._ _ _ _ Yano, H., Tamura, Y., Kobayashi, K., Tanemoto, M. & Uchida, S. Uric acid transporter ABCG2 is increased in the intestine of the 5/6 nephrectomy rat model of chronic kidney disease. Clin. Exp. Nephrol. 18, 50u201355 (2014)._ _ _ _ Houseman, E. A. et al. DNA methylation arrays as surrogate measures of cell mixture distribution. BMC Bioinforma. 13, 86 (2012)._ _ van Iterson, M., van Zwet, E. W., Heijmans, B. T. & the, B. C. Controlling bias and inflation in epigenome- and transcriptome-wide association studies using the empirical null distribution. Genome Biol. 18, 19 (2017)._ _ _ _ Viechtbauer, W. Conducting Meta-Analyses in R with the metafor Package. J. Stat. Softw. 36, 1u201348 (2010)._ _ Burgess, S. et al. Guidelines for performing Mendelian randomization investigations. Wellcome Open Res. 4, 186 (2020).Burgess, S., Butterworth, A. & Thompson, S. G. Mendelian randomization analysis with multiple genetic variants using summarized data. Genet. Epidemiol. 37, 658u2013665 (2013)._ _ _ _ Hemani, G. et al. The MR-Base platform supports systematic causal inference across the human phenome. Elife 7, e34408 (2018).Hemani, G., Tilling, K. & Davey Smith, G. Orienting the causal relationship between imprecisely measured traits using GWAS summary data. PLoS Genet. 13, e1007081 (2017)._ _ _ _ Yang, J., Lee, S. H., Goddard, M. E. & Visscher, P. M. Genome-wide complex trait analysis (GCTA): methods, data analyses, and interpretations. Methods Mol. Biol. 1019, 215u2013 236 (2013)._ _ _ _ Min, J. L. et al. Genomic and phenotypic insights from an atlas of genetic effects on DNA methylation. Nat. Genet. 53, 1311u20131321 (2021)._ _ _ _ _ Bowden, J., Davey Smith, G. & Burgess, S. Mendelian randomization with invalid instruments: effect estimation and bias detection through Egger regression. Int. J. Epidemiol. 44, 512u2013525 (2015)._ _ _ _ Cochran, W. G. The comparison of percentages in matched samples. Biometrika 37, 256u2013266 (1950)._ _ _ _ _ _ Rees, J. M. B., Wood, A. M. & Burgess, S. Extending the MR-Egger method for multivariable Mendelian randomization to correct for both measured and unmeasured pleiotropy. Stat. Med. 36, 4705u20134718 (2017)._ _ _ _ _ Hartwig, F. P., Davey Smith, G. & Bowden, J. Robust inference in summary data Mendelian randomization via the zero modal pleiotropy assumption. Int. J. Epidemiol. 46, 1985u20131998 (2017)._ _ _ _ Bowden, J., Davey Smith, G., Haycock, P. C. & Burgess, S. Consistent estimation in Mendelian randomization with some invalid instruments using a weighted median estimator. Genet. Epidemiol. 40, 304u2013314 (2016)._ _ _ _ Yang, J., Lee, S. H., Goddard, M. E. & Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. Am. J. Hum. Genet. 88, 76u201382 (2011)._ _ _ _ _ Huan, T. et al. Genome-wide identification of DNA methylation QTLs in whole blood highlights pathways for cardiovascular disease. Nat. Commun. 10, 4267 (2019)._ _ _ _ _ Imai, K., Keele, L. & Yamamoto, T. Identification, inference and sensitivity analysis for causal mediation effects. Stat. Sci. 25, 51u201371 (2010)._ _ _ _ Park, J. et al. Functional methylome analysis of human diabetic kidney disease. JCI Insight 4, e128886 (2019).Ashburner, M. et al. Gene Ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat. Genet. 25, 25u201329 (2000)._ _ _ _ _ The Gene Ontology Consortium. The Gene Ontology Resource: 20 years and still GOing strong. Nucleic Acids Res. 47, D330u2013D338 (2019)._ _ Kanehisa, M. & Goto, S. KEGG: Kyoto Encyclopedia of Genes and Genomes. Nucleic Acids Res. 28, 27u201330 (2000)._ _ _ _ _ Fabregat, A. et al. The Reactome Pathway Knowledgebase. Nucleic Acids Res. 46, D649u2013D655 (2018)._ _ _ _ Breeze, C. E. et al. eFORGE: a tool for identifying cell type-specific signal in epigenomic data. Cell Rep. 17, 2137u20132150 (2016)._ _ _ _ _ Sabo, P. J. et al. Discovery of functional noncoding elements by digital analysis of chromatin structure. Proc. Natl Acad. Sci. USA. 101, 16837 (2004)._ _ _ _ _ _ ENCODE Project Consortium. The ENCODE (ENCyclopedia Of DNA Elements) Project. Science 306, 636u2013640 (2004).Kundaje, A. et al. Integrative analysis of 111 reference human epigenomes. Nature 518, 317u2013 330 (2015)._ _ _ _ _ Ren, X. & Kuan, P. F. methylGSA: a Bioconductor package and Shiny app for DNA methylation data length bias adjustment in gene set testing. Bioinformatics 35, 1958u20131959 (2019)._ _ _ _ Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. Series B Methodol. 57, 289u2013300. (1995).Schlosser, P., Teumer, A., Tin, A. & KxF6ttgen, A. Phenotype generation code: epigenome-wide association studies identify DNA methylation associated with kidney function and damage. genepi-freiburg/ckdgen-pheno-ewas. GitHub. (2021).\"- - Study-specific acknowledgements and funding sources are listed as Supplementary Note .- - Open Access funding enabled and organized by Projekt DEAL.- - Author notes - \"These authors contributed equally: Adrienne Tin, Pascal Schlosser.These authors jointly supervised this work: Alexander Teumer, Anna KxF6ttgen.A full list of members and their affiliations appears in the Supplementary Information.\" - Authors and Affiliations - \"Department of Medicine, University of Mississippi Medical Center, Jackson, 39216, MS, USAAdrienne Tin_&_Adolfo CorreaDepartment of Epidemiology, Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USAAdrienne Tin,_Pascal Schlosser,_ Josef Coresh,_Allan C. Gelber,_Christine Ladd-Acosta_&_Anna KxF6ttgenInstitute of Genetic Epidemiology, Faculty of Medicine and Medical Center, University of Freiburg, Freiburg, GermanyPascal Schlosser,_Anselm Hoppmann,_Franziska Grundner-Culemann_&_Anna KxF6ttgenResearch Unit Molecular Epidemiology, Helmholtz Zentrum MxFCnchen, German Research Center for Environmental Health, Neuherberg, D-85764, Bavaria, GermanyPamela R. Matias-Garcia,_Christian Gieger,_Brigitte KxFChnel_&_Melanie WaldenbergerInstitute of Epidemiology, Helmholtz Zentrum MxFCnchen, German Research Center for Environmental Health, Neuherberg, D-85764, Bavaria, GermanyPamela R. Matias-Garcia,_Christian Gieger,_Brigitte KxFC hnel,_Annette Peters_&_Melanie WaldenbergerTUM School of Medicine, Technical University of Munich, Munich, GermanyPamela R. Matias-GarciaDepartment of Epidemiology, University of Groningen, University Medical Center Groningen, Groningen, the NetherlandsChris H. L. Thio,_Eliza Fraszczyk_&_Harold SniederFramingham Heart Study, Framingham, MA, USARoby Joehanes,_Shih-Jen Hwang_&_Daniel LevyPopulation Sciences Branch, National Heart, Lung, and Blood Institute, National Institutes of Health, Bethesda, MD, USARoby Joehanes_&_Daniel LevyDepartment of Medicine and Genetics, University of Pennsylvania Perelman School of Medicine, Philadelphia, 19104, PA, USAHongbo Liu_&_Katalin SusztakProgram in Medical and Population Genetics, Broad Institute, Cambridge, MA, USAZhi YuCardiovascular Research Center, Massachusetts General Hospital, Boston, MA, USAZhi YuCenter for Genomic Medicine, Massachusetts General Hospital, Boston, MA, USAZhi YuDepartment of Psychiatry and Psychotherapy, University Medicine Greifswald, Greifswald, GermanyAntoine Weihs_&_Sahar GhasemiMRC Integrative Epidemiology Unit, University of Bristol, Bristol, UKJosine L. Min_&_Gibran HemaniPopulation Health Sciences, Bristol Medical School, University of Bristol, Bristol, UKJosine L. Min_&_Gibran HemaniDepartment of Physiology, University of Maryland School of Medicine, Baltimore, MD, USAVictoria L. Halperin Kuhns_&_Owen M. WoodwardCenter for Research on Genomics and Global Health, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD, USAAdebowale A. Adeyemo_&_Karlijn A. C. MeeksDepartment of Public and Occupational Health, Amsterdam Public Health Research Institute, Amsterdam University Medical Centers, University of Amsterdam, 1105 AZ, Amsterdam, the NetherlandsCharles Agyemang_&_Karlijn A. C. MeeksDepartment of Neurobiology, Care Sciences and Society (NVS), Family Medicine and Primary Care Unit, Karolinska Institutet, Huddinge, SwedenJohan xC4rnlxF6v_&_Christoph NowakSchool of Health and Social Studies, Dalarna University, Falun, SwedenJohan xC4rnlxF6 vPopulation Health Sciences, German Centre for Neurodegenerative Diseases (DZNE), Bonn, GermanyNasir A. Aziz,_Monique M. B. Breteler_&_Dan LiuDepartment of Neurology, Faculty of Medicine, University of Bonn, Bonn, GermanyNasir A. AzizLaboratory of Environmental Precision Health, Mailman School of Public Health, Columbia University, New York, NY, USAAndrea Baccarelli_&_Xu GaoCenter for Primary Care and Public Health (UnisantxE9), University of Lausanne, Lausanne, SwitzerlandMurielle Bochud,_Cristian Carmeli,_Tanguy Corre_&_Silvia StringhiniGerman Cancer Research Center (DKFZ), Division of Clinical Epidemiology and Aging Research, Heidelberg, GermanyHermann Brenner,_Xu012Bn GxE0o,_Ben SchxF6ttker_&_ Hannah R. StockerNetwork Aging Research, Heidelberg University, Heidelberg, GermanyHermann Brenner,_Ben SchxF6ttker_&_Hannah R. StockerDivision of Preventive Oncology, German Cancer Research Center (DKFZ) and National Center for Tumor Diseases (NCT), Heidelberg, GermanyHermann BrennerGerman Cancer Consortium, German Cancer Research Center (DKFZ), Heidelberg, GermanyHermann BrennerHuman Genetics Center, Department of Epidemiology, Human Genetics and Environmental Sciences, School of Public Health, The University of Texas Health Science Center at Houston, Houston, 77030, TX, USAJan Bressler_&_Megan L. GroveInstitute for Medical Biometry, Informatics and Epidemiology (IMBIE), Faculty of Medicine, University of Bonn, Bonn, GermanyMonique M. B. BretelerPopulation Health Laboratory, University of Fribourg, Fribourg, SwitzerlandCristian CarmeliDepartment of Epidemiology, Erasmus University Medical Center, Rotterdam, the NetherlandsLayal Chaker_ &_Mohsen GhanbariDepartment of Internal Medicine, Erasmus Medical Center, Rotterdam, the NetherlandsLayal Chaker_&_Joyce B. J. van MeursLothian Birth Cohorts Group, Department of Psychology, The University of Edinburgh, 7 George Square, Edinburgh, EH8 9JZ, UKSimon R. Cox_&_Sarah E. HarrisVth Department of Medicine, Medical Faculty Mannheim, Heidelberg University, Mannheim, GermanyGraciela E. Delgado,_ Marcus E. Kleber_&_Winfried MxE4rzDepartment of Nephrology and Hypertension, University of Erlangen-NxFCrnberg, Erlangen, GermanyKai-Uwe EckardtDepartment of Nephrology and Medical Intensive Care, CharitxE9 u2013 UniversitxE4tsmedizin Berlin, Berlin, GermanyKai-Uwe EckardtInstitute of Human Genetics, Friedrich-Alexander-UniversitxE4 tErlangen-NxFCrnberg, 91054, Erlangen, GermanyArif B. EkiciDepartment of Anatomy and Cell Biology, University Medicine Greifswald, Greifswald, GermanyKarlhans EndlichDZHK (German Centre for Cardiovascular Research), Partner Site Greifswald, Greifswald, GermanyKarlhans Endlich,_Sahar Ghasemi,_Matthias Nauck,_Uwe VxF6 lker_&_Alexander TeumerDepartment of Medicine, University of Washington, Seattle, 98101, WA, USAJames S. Floyd_&_Bruce M. PsatyDepartment of Epidemiology, University of Washington, Seattle, 98101, WA, USAJames S. Floyd,_Bruce M. Psaty_&_Alex P. ReinerCardiovascular Health Research Unit, University of Washington, Seattle, 98101, WA, USAJames S. Floyd,_Bruce M. Psaty,_Nona Sotoodehnia_&_Brenton R. SwensonDepartment of Occupational and Environmental Health Sciences, School of Public Health, Peking University, Beijing, ChinaXu GaoDepartment of Medicine, Johns Hopkins School of Medicine, Baltimore, MD, USAAllan C. GelberInstitute for Community Medicine, University Medicine Greifswald, Greifswald, GermanySahar Ghasemi_&_Alexander TeumerDepartment of Preventive Medicine, Northwestern University Feinberg School of Medicine, Chicago, IL, USAPhilip Greenland,_ Lifang Hou,_Donald M. Lloyd-Jones_&_Yinan ZhengDepartment of Clinical Genetics, Amsterdam Reproduction & Development Research Institute, Amsterdam University Medical Centres, University of Amsterdam, Amsterdam, the NetherlandsPeter Henneman_ &_Andrea VenemaInstitute for Clinical Diabetology, German Diabetes Center, Leibniz Center for Diabetes Research at Heinrich Heine University DxFCsseldorf, DxFC sseldorf, GermanyChristian HerderGerman Center for Diabetes Research, Munich-Neuherberg, GermanyChristian HerderDivision of Endocrinology and Diabetology, Medical Faculty, Heinrich Heine University DxFCsseldorf, DxFCsseldorf, GermanyChristian HerderDepartment of Human Genetics, David Geffen School of Medicine, University of California Los Angeles, Los Angeles, 90095, CA, USASteve Horvath_&_Ake T. LuBiostatistics, Fielding School of Public Health, UCLA, Los Angeles, CA, USASteve HorvathDepartment of Microbiology and Immunology, Faculty of Medicine and Health Technology, Tampere University, Tampere, 33014, FinlandMikko A. HurmeDivision of Intramural Research, Population Sciences Branch, National Heart, Lung, and Blood Institute, National Institutes of Health, Bethesda, MD, USAShih-Jen HwangDepartment of Epidemiology, School of Public Health, University of Michigan, Ann Arbor, 48109, MI, USASharon L. R. Kardia,_Scott M. Ratliff,_Jennifer A. Smith_&_Wei ZhaoEstonian Genome Centre, Institute of Genomics, University of Tartu, Tartu, EstoniaSilva Kasela_ &_Lili MilaniSYNLAB MVZ Humangenetik Mannheim, Mannheim, GermanyMarcus E. KleberDeutsches Herzzentrum MxFCnchen, Technische UniversitxE4t MxFCnchen, Munich, GermanyWolfgang KoenigDZHK (German Centre for Cardiovascular Research), Partner site Munich Heart Alliance, Munich, GermanyWolfgang Koenig_&_Melanie WaldenbergerInstitute of Epidemiology and Medical Biometry, University of Ulm, Ulm, GermanyWolfgang KoenigNational Heart and Lung Institute, Imperial College London, London, UKJaspal S. KoonerDepartment of Cardiology, Ealing Hospital, London North West Healthcare NHS Trust, Southall, UKJaspal S. KoonerImperial College Healthcare NHS Trust, London, UKJaspal S. KoonerInstitute of Genetic Epidemiology, Medical University of Innsbruck, Innsbruck, AustriaFlorian KronenbergDepartment of Clinical Chemistry, Faculty of Medicine and Health Technology, Tampere University, Tampere, FinlandTerho LehtimxE4ki_&_Pashupati P. MishraFinnish Cardiovascular Research Centre, Faculty of Medicine and Health Technology, Tampere University, Tampere, FinlandTerho LehtimxE4ki_&_Pashupati P. MishraDepartment of Clinical Chemistry, Fimlab Laboratories, Tampere, FinlandTerho LehtimxE4ki_&_Pashupati P. MishraDepartment of Medical Sciences, Uppsala University, Uppsala, SwedenLars Lind_&_Johan SundstrxF6mInstitute of Nutritional Sciences, Friedrich Schiller University Jena, Jena, GermanyStefan LorkowskiCompetence Cluster for Nutrition and Cardiovascular Health (nutriCARD) Halle-Jena-Leipzig, Jena, GermanyStefan Lorkowski_&_Winfried MxE4rzCentre for Genomic and Experimental Medicine, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, EH4 2XU, UKRiccardo E. Marioni_ &_Daniel L. McCartneySynlab Academy, SYNLAB Holding Deutschland GmbH, Mannheim and Augsburg, GermanyWinfried MxE4rzClinical Institute of Medical and Chemical Laboratory Diagnostics, Medical University of Graz, Graz, AustriaWinfried MxE4 rzInstitute of Clinical Chemistry and Laboratory Medicine, University Medicine Greifswald, Greifswald, GermanyMatthias NauckLudwig-Maximilians UniversitxE4 t MxFCnchen, Munich, GermanyAnnette PetersInstitute of Human Genetics, Klinikum rechts der Isar, Technische UniversitxE4t MxFCnchen, Munich, GermanyHolger Prokisch_&_Juliane WinkelmannDepartment of Computational Health, Institute of Neurogenomics, Helmholtz Zentrum MxFCnchen, Munich, GermanyHolger ProkischDepartment of Health Services, University of Washington, Seattle, 98101, WA, USABruce M. PsatyResearch Centre of Applied and Preventive Cardiovascular Medicine, University of Turku, Turku, FinlandOlli T. RaitakariDepartment of Clinical Physiology and Nuclear Medicine, Turku University Hospital, Turku, FinlandOlli T. RaitakariCentre for Population Health Research, University of Turku and Turku University Hospital, Turku, FinlandOlli T. RaitakariDepartment of Environmental Health, Harvard T.H. Chan School of Public Health, Boston, MA, USAJoel SchwartzDivision of Epidemiology and Community Health, School of Public Health, University of Minnesota, Minneapolis, MN, USASanaz SedaghatSurvey Research Center, Institute for Social Research, University of Michigan, Ann Arbor, MI, USAJennifer A. SmithThe George Institute for Global Health, University of New South Wales, Sydney, NSW, AustraliaJohan SundstrxF6mInstitute for Public Health Genetics, University of Washington, Seattle, WA, USABrenton R. SwensonDepartment of Endocrinology, University of Groningen, University Medical Center Groningen, Groningen, the NetherlandsJana V. van Vliet-Ostaptchouk_&_Bruce H. R. WolffenbuttelInterfaculty Institute for Genetics and Functional Genomics, University Medicine Greifswald, Greifswald, GermanyUwe VxF6lkerInstitute of Neurogenomics, Helmholtz Zentrum MxFCnchen, Munich, GermanyJuliane WinkelmannChair Neurogenetics, Klinikum rechts der Isar, Technische UniversitxE4t MxFCnchen, Munich, GermanyJuliane WinkelmannMunich Cluster for Systems Neurology (SyNergy), Munich, GermanyJuliane WinkelmannLee Kong Chian School of Medicine, Nanyang Technological University, Singapore, SingaporeMarie LohDepartment of Epidemiology and Biostatistics, Imperial College London, London, UKMarie LohDepartment of Laboratory Medicine and Pathology, University of Washington, Seattle, WA, USAShreeram AkileshDepartment of Population Medicine and Lifestyle Diseases Prevention, Medical University of Bialystok, Bialystok, PolandAlexander Teumer\" - \"AuthorsAdrienne TinYou can also search for this author in _Pascal SchlosserYou can also search for this author in _Pamela R. Matias-GarciaYou can also search for this author in _Chris H. L. ThioYou can also search for this author in _Roby JoehanesYou can also search for this author in _Hongbo LiuYou can also search for this author in _Zhi YuYou can also search for this author in _Antoine WeihsYou can also search for this author in _Anselm HoppmannYou can also search for this author in _Franziska Grundner-CulemannYou can also search for this author in _Josine L. MinYou can also search for this author in _Victoria L. Halperin KuhnsYou can also search for this author in _Adebowale A. AdeyemoYou can also search for this author in _Charles AgyemangYou can also search for this author in _Johan xC4rnlxF6 vYou can also search for this author in _Nasir A. AzizYou can also search for this author in _Andrea BaccarelliYou can also search for this author in _Murielle BochudYou can also search for this author in _Hermann BrennerYou can also search for this author in _Jan BresslerYou can also search for this author in _Monique M. B. BretelerYou can also search for this author in _Cristian CarmeliYou can also search for this author in _Layal ChakerYou can also search for this author in _Josef CoreshYou can also search for this author in _Tanguy CorreYou can also search for this author in _Adolfo CorreaYou can also search for this author in _Simon R. CoxYou can also search for this author in _Graciela E. DelgadoYou can also search for this author in _ Kai-Uwe EckardtYou can also search for this author in _Arif B. EkiciYou can also search for this author in _Karlhans EndlichYou can also search for this author in _James S. FloydYou can also search for this author in _Eliza FraszczykYou can also search for this author in _Xu GaoYou can also search for this author in _Xu012Bn GxE0oYou can also search for this author in _Allan C. GelberYou can also search for this author in _Mohsen GhanbariYou can also search for this author in _Sahar GhasemiYou can also search for this author in _Christian GiegerYou can also search for this author in _Philip GreenlandYou can also search for this author in _Megan L. GroveYou can also search for this author in _Sarah E. HarrisYou can also search for this author in _Gibran HemaniYou can also search for this author in _Peter HennemanYou can also search for this author in _Christian HerderYou can also search for this author in _Steve HorvathYou can also search for this author in _Lifang HouYou can also search for this author in _Mikko A. HurmeYou can also search for this author in _Shih-Jen HwangYou can also search for this author in _Sharon L. R. KardiaYou can also search for this author in _Silva KaselaYou can also search for this author in _Marcus E. KleberYou can also search for this author in _Wolfgang KoenigYou can also search for this author in _Jaspal S. KoonerYou can also search for this author in _Florian KronenbergYou can also search for this author in _Brigitte KxFChnelYou can also search for this author in _Christine Ladd-AcostaYou can also search for this author in _Terho LehtimxE4 kiYou can also search for this author in _Lars LindYou can also search for this author in _Dan LiuYou can also search for this author in _Donald M. Lloyd-JonesYou can also search for this author in _Stefan LorkowskiYou can also search for this author in _Ake T. LuYou can also search for this author in _Riccardo E. MarioniYou can also search for this author in _Winfried MxE4 rzYou can also search for this author in _Daniel L. McCartneyYou can also search for this author in _ Karlijn A. C. MeeksYou can also search for this author in _Lili MilaniYou can also search for this author in _Pashupati P. MishraYou can also search for this author in _Matthias NauckYou can also search for this author in _Christoph NowakYou can also search for this author in _Annette PetersYou can also search for this author in _Holger ProkischYou can also search for this author in _Bruce M. PsatyYou can also search for this author in _Olli T. RaitakariYou can also search for this author in _Scott M. RatliffYou can also search for this author in _Alex P. ReinerYou can also search for this author in _Ben SchxF6ttkerYou can also search for this author in _Joel SchwartzYou can also search for this author in _Sanaz SedaghatYou can also search for this author in _Jennifer A. SmithYou can also search for this author in _Nona SotoodehniaYou can also search for this author in _Hannah R. StockerYou can also search for this author in _Silvia StringhiniYou can also search for this author in _Johan SundstrxF6 mYou can also search for this author in _Brenton R. SwensonYou can also search for this author in _Joyce B. J. van MeursYou can also search for this author in _Jana V. van Vliet-OstaptchoukYou can also search for this author in _Andrea VenemaYou can also search for this author in _Uwe VxF6lkerYou can also search for this author in _Juliane WinkelmannYou can also search for this author in _Bruce H. R. WolffenbuttelYou can also search for this author in _Wei ZhaoYou can also search for this author in _Yinan ZhengYou can also search for this author in _Marie LohYou can also search for this author in _Harold SniederYou can also search for this author in _Melanie WaldenbergerYou can also search for this author in _Daniel LevyYou can also search for this author in _Shreeram AkileshYou can also search for this author in _Owen M. WoodwardYou can also search for this author in _Katalin SusztakYou can also search for this author in _Alexander TeumerYou can also search for this author in _Anna KxF6ttgenYou can also search for this author in _\" - Consortia - The Estonian Biobank Research TeamLili Milani - \"The Genetics of DNA Methylation ConsortiumJosine L. Min_&_Gibran Hemani\" - Contributions - \"Drafting of manuscript: Anna KxF6ttgen, Pascal Schlosser, Alexander Teumer, Chris H. L. Thio, Owen M. Woodward, Adrienne Tin. Bioinformatics: Shreeram Akilesh, Nasir A. Aziz, Xu Gao, Peter Henneman, Anselm Hoppmann, Roby Joehanes, Christine Ladd-Acosta, Hongbo Liu, Ake T. Lu, Pamela R. Matias-Garcia, Daniel L. McCartney, Karlijn A.C. Meeks, Christoph Nowak, Pascal Schlosser, Andrea Venema, Antoine Weihs. Critical review of manuscript: Adebowale A. Adeyemo, Charles Agyemang, Nasir A. Aziz, Andrea Baccarelli, Hermann Brenner, Jan Bressler, Monique M. B. Breteler, Cristian Carmeli, Layal Chaker, Josef Coresh, Adolfo Correa, Simon R. Cox, Graciela E. Delgado, Kai-Uwe Eckardt, Arif B. Ekici, Karlhans Endlich, James S. Floyd, Xu Gao, Allan C. Gelber, Mohsen Ghanbari, Christian Gieger, Philip Greenland, Megan L. Grove, Sarah E. Harris, Peter Henneman, Christian Herder, Lifang Hou, Silva Kasela, Marcus E. Kleber, Wolfgang Koenig, Jaspal S. Kooner, Florian Kronenberg, Anna KxF6ttgen, Christine Ladd-Acosta, Daniel Levy, Lars Lind, Marie Loh, Stefan Lorkowski, Riccardo E. Marioni, Pamela R. Matias-Garcia, Daniel L. McCartney, Karlijn A. C. Meeks, Lili Milani, Winfried MxE4rz, Matthias Nauck, Christoph Nowak, Holger Prokisch, Bruce M. Psaty, Alex P. Reiner, Pascal Schlosser, Joel Schwartz, Sanaz Sedaghat, Jennifer A. Smith, Nona Sotoodehnia, Hannah R. Stocker, Johan SundstrxF6m, Katalin Susztak, Alexander Teumer, Chris H. L. Thio, Adrienne Tin, Uwe VxF6lker, Melanie Waldenberger, Juliane Winkelmann, Yinan Zheng, Johan xC4rnlxF6v. Statistical methods and analysis: Nasir A. Aziz, Cristian Carmeli, Graciela E. Delgado, Eliza Fraszczyk, Xu Gao, Sahar Ghasemi, Franziska Grundner-Culemann, Gibran Hemani, Peter Henneman, Anselm Hoppmann, Steve Horvath, Mikko A. Hurme, Shih-Jen Hwang, Roby Joehanes, Silva Kasela, Marcus E. Kleber, Anna KxF6ttgen, Brigitte KxFChnel, Christine Ladd-Acosta, Terho LehtimxE4ki, Dan Liu, Hongbo Liu, Marie Loh, Ake T. Lu, Pamela R. Matias-Garcia, Daniel L. McCartney, Karlijn A. C. Meeks, Josine L. Min, Pashupati P. Mishra, Christoph Nowak, Scott M. Ratliff, Pascal Schlosser, Sanaz Sedaghat, Brenton R. Swenson, Alexander Teumer, Chris H. L. Thio, Adrienne Tin, Andrea Venema, Antoine Weihs, Zhi Yu, Wei Zhao, Yinan Zheng. Functional studies: Victoria L. Halperin Kuhns, Hongbo Liu, Katalin Susztak, Owen M. Woodward. Phenotyping: Charles Agyemang, Murielle Bochud, Hermann Brenner, Monique M. B. Breteler, Simon R. Cox, James S. Floyd, Sarah E. Harris, Christian Herder, Sharon L. R. Kardia, Silva Kasela, Wolfgang Koenig, Anna KxF6ttgen, Joyce B. J. van Meurs, Annette Peters, Bruce M. Psaty, Scott M. Ratliff, Johan xC4rnlxF6 v. Interpretation of results: Shreeram Akilesh, Nasir A. Aziz, Andrea Baccarelli, Hermann Brenner, Jan Bressler, Layal Chaker, James S. Floyd, Xu Gao, Allan C. Gelber, Wolfgang Koenig, Anna KxF6ttgen, Christine Ladd-Acosta, Daniel Levy, Pamela R. Matias-Garcia, Christoph Nowak, Bruce M. Psaty, Pascal Schlosser, Joel Schwartz, Ben SchxF6ttker, Nona Sotoodehnia, Katalin Susztak, Alexander Teumer, Chris H. L. Thio, Adrienne Tin, Owen M. Woodward. Study design of an individual contributing study: Adebowale A. Adeyemo, Charles Agyemang, Andrea Baccarelli, Murielle Bochud, Hermann Brenner, Monique M. B. Breteler, Layal Chaker, Adolfo Correa, Simon R. Cox, Mohsen Ghanbari, Christian Gieger, Gibran Hemani, Peter Henneman, Lifang Hou, Sharon L. R. Kardia, Jaspal S. Kooner, Daniel Levy, Ake T. Lu, Riccardo E. Marioni, Karlijn A.C. Meeks, Josine L. Min, Winfried MxE4rz, Annette Peters, Holger Prokisch, Bruce M. Psaty, Alex P. Reiner, Joel Schwartz, Jennifer A. Smith, Harold Snieder, Jana V. van Vliet-Ostaptchouk, Juliane Winkelmann, Bruce H.R. Wolffenbuttel, Johan xC4rnlxF6v. Subject recruitment: Charles Agyemang, Nasir A. Aziz, Andrea Baccarelli, Murielle Bochud, Hermann Brenner, Simon R. Cox, Jaspal S. Kooner, Lars Lind, Karlijn A. C. Meeks, Lili Milani, Matthias Nauck, Annette Peters, Joel Schwartz. Methylation assessment: Charles Agyemang, Andrea Baccarelli, Hermann Brenner, James S. Floyd, Xu Gao, Megan L. Grove, Xin GxE0o, Sarah E. Harris, Peter Henneman, Steve Horvath, Lifang Hou, Mikko A. Hurme, Silva Kasela, Marcus E. Kleber, Stefan Lorkowski, Daniel L. McCartney, Joyce B. J. van Meurs, Lili Milani, Jennifer A. Smith, Nona Sotoodehnia, Silvia Stringhini, Alexander Teumer, Adrienne Tin, Jana V. van Vliet-Ostaptchouk, Melanie Waldenberger, Juliane Winkelmann, Bruce H. R. Wolffenbuttel, Wei Zhao, Yinan Zheng. Management of an individual contributing study: Adebowale A. Adeyemo, Nasir A. Aziz, Andrea Baccarelli, Murielle Bochud, Hermann Brenner, Monique M. B. Breteler, Layal Chaker, Josef Coresh, Tanguy Corre, Adolfo Correa, Simon R. Cox, Christian Gieger, Megan L. Grove, Xin GxE0 o, Sarah E. Harris, Gibran Hemani, Peter Henneman, Lifang Hou, Mikko A. Hurme, Sharon L. R. Kardia, Marcus E. Kleber, Wolfgang Koenig, Jaspal S. Kooner, Anna KxF6ttgen, Terho LehtimxE4ki, Daniel Levy, Lars Lind, Donald M. Lloyd-Jones, Marie Loh, Riccardo E. Marioni, Karlijn A. C. Meeks, Joyce B. J. van Meurs, Lili Milani, Josine L. Min, Pashupati P. Mishra, Winfried MxE4rz, Matthias Nauck, Annette Peters, Bruce M. Psaty, Olli T. Raitakari, Alex P. Reiner, Joel Schwartz, Ben SchxF6ttker, Jennifer A. Smith, Harold Snieder, Hannah R. Stocker, Johan SundstrxF6m, Jana V. van Vliet-Ostaptchouk, Uwe VxF6lker, Melanie Waldenberger, Juliane Winkelmann, Bruce H. R. Wolffenbuttel.\" - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - \"J. xC4rnlxF6v has served on advisory boards for AstraZeneca and Boehringer Ingelheim, and has received lecturing fees from AstraZeneca and Novartis, all unrelated to the present project. J. Coresh has grants from NIH and is a consultant to healthy.io. J. S. Floyd has consulted for Shionogi Inc. C. Herder reports personal fees from Sanofi and Lilly and grant support from Sanofi outside the submitted work. M. E. Kleber is employed with SYNLAB Holding Deutschland GmbH. O. Woodward has grants from AstraZeneca outside the submitted work. W. Koenig reports personal fees from AstraZeneca, Novartis, Pfizer, The Medicines Company, DalCor, Kowa, Amgen, Corvidia, Daichii-Sankyo, Genentech, Novo Nordisk, Omeicos, Esperion, Berlin-Chemie, Sanofi, and Bristol-Myers Squibb and grants and non-financial support from Abbott, Roche Diagnostics, Beckmann, and Singulex outside the submitted work. S. Lorkowski reports grants and personal fees from Akcea Therapeutics Germany, and personal fees from amedes, AMGEN, Berlin-Chemie, Boehringer Ingelheim Pharma, Daiichi Sankyo, Lilly Deutschland, MSD Sharp & Dohme, Novo Nordisk Pharma, Roche Pharma, Sanofi-Aventis, Synlab Holding Deutschland, Unilever, and Upfield, all outside the submitted work. R. E. Marioni has received payment from Illumina for presentations. W. MxE4rz reports grants from Siemens Healthineers, grants and personal fees from Aegerion Pharmaceuticals, grants and personal fees from AMGEN, grants from Astrazeneca, grants and personal fees from Sanofi, grants and personal fees from Alexion Pharmaceuticals, grants and personal fees from BASF, grants and personal fees from Abbott Diagnostics, grants and personal fees from Numares AG, grants and personal fees from Berlin-Chemie, grants and personal fees from Akzea Therapeutics, grants from Bayer Vital GmbH, grants from bestbion dx GmbH, grants from Boehringer Ingelheim Pharma GmbH Co KG, grants from Immundiagnostik GmbH, grants from Merck Chemicals GmbH, grants from MSD Sharp and Dohme GmbH, grants from Novartis Pharma GmbH, grants from Olink Proteomics, other from Synlab Holding Deutschland GmbH, all outside the submitted work. B. Psaty serves on the Steering Committee of the Yale Open Data Access Project funded by Johnson & Johnson. J. SundstrxF6m reports ownership in companies providing services to Itrim, Amgen, Janssen, Novo Nordisk, Eli Lilly, Boehringer, Bayer, Pfizer, and AstraZeneca, outside the submitted work. B. KxFChnel is currently an employee of Regeneron Genetics Center. All other authors have nothing to declare.\" - \\' \\'- - Peer review information Nature Communications thanks William Cookson and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License.\\'- - Animals - \\'All experiments on animals were approved by the Geneva Canton ethical regulation authority (authorization: GE/202/17) and were conducted according to the Swiss law. Ten-week-old C57BL/6J male mice from Charles River Laboratories were housed under normal conditions and had ad libitum access to regular chow diet (CTRL: RM3, Special Diet Services (SDS), 11.5% fat, 61.57% carbohydrate, 26.93% protein) and water for one week of acclimation.\\' - Diets - \\'Methionine choline deficient (MCD: 21% fat, 63% carbohydrate, 16% protein) and atherogenic (ATH: 60% fat, 20% carbohydrate, 20% protein, 1.25% cholesterol and 0.5% cholic acid) diets were purchased from Research Diets Inc. (A02082002BR, D17052505). Mice were randomly distributed into three different groups (CTRL, MCD, ATH). Mice in the CTRL and ATH groups were fed with their respective diets for 12 weeks. As the MCD diet rapidly induces the liver manifestation of NASH as well as an important weight loss, mice in the MCD group were fed a CTRL diet for 5 weeks followed by 7 weeks of MCD diet.\\' - Tissue collection - \"Mice were fasted 4u2009hours (9a.m.u20131p.m.), briefly anesthetized with isoflurane and immediately sacrificed. Truncal blood was collected, and organs were weighed prior to fixation or cryopreservation in liquid nitrogen.\" - Histology - \"Liver, epididymal white adipose tissue (eWAT), inguinal white adipose tissue (iWAT), brown adipose tissue (BAT) and ileum were fixed overnight in 10% formalin and processed for histology. 5-u03BCm sections were rehydrated and either stained with H&E for gross morphology, with Sirius Red/Fast Green for fibrosis quantification or with Periodic Acid Schiff (PAS) for glycogen assessments. To quantify lipid deposition, unfixed pieces of liver were embedded in optimal cutting temperature (OCT) compound and flash frozen. 5-u03BCm sections were rapidly fixed in 10% formalin and stained with hematoxylin and Oil Red O solution.\" - Quantification and pathological score - Pictures were acquired using an Olympus VS120 microscope. Fibrosis, lipid and glycogen quantifications were performed with the area measurement tool of the ImageJ software. Ileal villus and crypt sizes were measured on H&E stained sections with the ImageJ software. The SAF score (Steatosis, Activity (ballooning and lobular inflammation) and Fibrosis) was evaluated by a pathologist (CdV) blinded to the diets according to Bedossa, et al.. - RT-qPCR - \"Total RNA was isolated from liver and ileum tissues using TRI Reagent Solution (ThermoFisher AM9738). 500u2009ng of total RNA were reverse transcribed (PrimeScript RT reagent kit, Takara) and mRNA levels were assessed by qPCR using a Light-Cycler 480 (Roche Diagnostics). Normalization was done using the ribosomal RPS29 gene. The list of primers can be found in Supplementary Table_.\" - Blood parameters - \"Blood samples were collected in EDTA-coated tubes and plasma was stored at u2212 80u2009xB0C. Plasma levels of glucose, AST, ALT, total cholesterol, free fatty acids and triglycerides were assessed using a Cobas c 111 analyzer and appropriate reagents (Roche Diagnostics). Insulin was measured using an ultrasensitive mouse insulin ELISA kit (Mercodia).\" - Lipidomic analysis - \"Lipid extraction and lipidomic analysis were performed as described in Loizides-Mangold, et al.. Lipid extracts were prepared using the MTBE protocol. In short, 15u2013 20u2009mg of pulverized liver tissue was resuspended in 100u2009xB5l H2O. 360u2009 u03BCl methanol and a mix of internal standards were added (400 pmol PC 12:0/12:0, 1000 pmol PE 17:0/14:1, 1000 pmol PI 17:0/14:1, 3300 pmol PS 17:0/14:1, 2500 pmol SM d18:1/12:0, 500 pmol Cer d18:1/17:0 and 100 pmol GlcCer d18:1/8:0) together with 1.2u2009ml of MTBE. Phase separation was induced by addition of 200u2009 xB5l MS-grade water and the organic phase was dried in a vacuum concentrator (CentriVap, Labconco). Lipids were dissolved in chloroform/methanol and divided into three aliquots. One aliquot was treated by alkaline hydrolysis to enrich for sphingolipids according to the method by Clarke and Dawson and the other two aliquots were used for glycerophospholipids and phosphorus assay, respectively. Mass spectrometry analysis for the identification and quantification of phospho- and sphingolipid species was performed on a TSQ Vantage Triple Stage Quadrupole Mass Spectrometer (Thermo Fisher Scientific) equipped with a robotic nanoflow ion source (Nanomate HD, Advion Biosciences), using multiple reaction monitoring (MRM). Each individual ion dissociation pathway was optimized with regard to collision energy. Dried lipid extracts were resuspended in 250u2009xB5l MS grade chloroform/methanol (1:1) and further diluted in either chloroform/methanol (1:2) plus 5u2009mM ammonium acetate (negative ion mode) or in chloroform/methanol/H2O (2:7:1) plus 5u2009mM ammonium acetate (positive ion mode). Lipid concentrations were calculated relative to the relevant internal standards and normalized to the phosphate content of each lipid extract to account for variability in the amount of starting material and to correct for sample loss during the extraction procedure.\" - Statistical analyses - \"GraphPad Prism version 7.02 software was used for statistical analyses of the data using one-way ANOVA, pu2009<u20090.05 was considered statistically significant. Graphs represent meanu2009+u2009standard error of the mean (SEM), if not stated differently. Clustered heat map was generated in R using the heatmap.2 function of the u201Cgplotsu201D package. Principal component analysis was performed using the PAST program using default parameters.\"- - \"Younossi, Z. M. et al. Global epidemiology of nonalcoholic fatty liver disease-Meta-analytic assessment of prevalence, incidence, and outcomes. Hepatology 64, 73u201384, (2016)._ _ _ Rafiq, N. et al. Long-term follow-up of patients with nonalcoholic fatty liver. Clinical gastroenterology and hepatology: the official clinical practice journal of the American Gastroenterological Association 7, 234u2013238, (2009)._ _ Soderberg, C. et al. Decreased survival of subjects with elevated liver function tests during a 28-year follow-up. Hepatology 51, 595u2013602, (2010)._ _ _ Chalasani, N. et al. The diagnosis and management of non-alcoholic fatty liver disease: practice Guideline by the American Association for the Study of Liver Diseases, American College of Gastroenterology, and the American Gastroenterological Association. Hepatology 55, 2005u20132023, (2012)._ _ _ Angulo, P. et al. Liver Fibrosis, but No Other Histologic Features, Is Associated With Long-term Outcomes of Patients With Nonalcoholic Fatty Liver Disease. Gastroenterology 149, 389u2013397 e310, (2015)._ _ _ Ekstedt, M. et al. Fibrosis stage is the strongest predictor for disease-specific mortality in NAFLD after up to 33 years of follow-up. Hepatology 61, 1547u20131554, (2015)._ _ _ _ Younossi, Z. M. et al. Pathologic criteria for nonalcoholic steatohepatitis: interprotocol agreement and ability to predict liver-related mortality. Hepatology 53, 1874u20131882, (2011)._ _ _ Tilg, H. & Moschen, A. R. Evolution of inflammation in nonalcoholic fatty liver disease: the multiple parallel hits hypothesis. Hepatology 52, 1836u20131846, (2010)._ _ _ _ Buzzetti, E., Pinzani, M. & Tsochatzis, E. A. The multiple-hit pathogenesis of non-alcoholic fatty liver disease (NAFLD). Metabolism 65, 1038u20131048, (2016)._ _ _ _ Magee, N., Zou, A. & Zhang, Y. Pathogenesis of Nonalcoholic Steatohepatitis: Interactions between Liver Parenchymal and Nonparenchymal Cells. Biomed Res Int 2016, 5170402, (2016)._ _ _ _ _ Tirosh, O. Hypoxic Signaling and Cholesterol Lipotoxicity in Fatty Liver Disease Progression. Oxid Med Cell Longev 2018, 2548154, (2018)._ _ _ _ _ Younossi, Z. et al. Global burden of NAFLD and NASH: trends, predictions, risk factors and prevention. Nat Rev Gastroenterol Hepatol 15, 11u201320, (2018)._ _ _ Farrell, G. et al. Mouse Models of Nonalcoholic Steatohepatitis: Toward Optimization of Their Relevance to Human Nonalcoholic Steatohepatitis. Hepatology 69, 2241u20132257, (2019)._ _ _ Delire, B., Starkel, P. & Leclercq, I. Animal Models for Fibrotic Liver Diseases: What We Have, What We Need, and What Is under Development. Journal of clinical and translational hepatology 3, 53u201366, (2015)._ _ _ _ Haczeyni, F. et al. Mouse models of non-alcoholic steatohepatitis: A reflection on recent literature. Journal of gastroenterology and hepatology 33, 1312u20131320, (2018)._ _ _ Machado, M. V. et al. Mouse models of diet-induced nonalcoholic steatohepatitis reproduce the heterogeneity of the human disease. PLoS One 10, e0127991, (2015)._ _ _ _ _ Larter, C. Z. et al. Hepatic free fatty acids accumulate in experimental steatohepatitis: role of adaptive pathways. J Hepatol 48, 638u2013647, (2008)._ _ _ _ Matsuzawa, N. et al. Lipid-induced oxidative stress causes steatohepatitis in mice fed an atherogenic diet. Hepatology 46, 1392u20131403, (2007)._ _ _ _ Rinella, M. E. et al. Mechanisms of hepatic steatosis in mice fed a lipogenic methionine choline-deficient diet. J Lipid Res 49, 1068u20131076, (2008)._ _ _ _ _ Rizki, G. et al. Mice fed a lipogenic methionine-choline-deficient diet develop hypermetabolism coincident with hepatic suppression of SCD-1. J Lipid Res 47, 2280u20132290, (2006)._ _ _ _ Rein-Fischboeck, L. et al. Alpha-syntrophin null mice are protected from non-alcoholic steatohepatitis in the methionine-choline-deficient diet model but not the atherogenic diet model. Biochimica et biophysica acta. Molecular and cell biology of lipids 1863, 526u2013537, (2018)._ _ _ _ Mari, M. et al. Mitochondrial free cholesterol loading sensitizes to TNF- and Fas-mediated steatohepatitis. Cell Metab 4, 185u2013198, (2006)._ _ _ _ Watanabe, M. et al. Bile acids induce energy expenditure by promoting intracellular thyroid hormone activation. Nature 439, 484u2013 489, (2006)._ _ _ _ _ Rinella, M. E. & Green, R. M. The methionine-choline deficient dietary model of steatohepatitis does not exhibit insulin resistance. Journal of Hepatology 40, 47u201351, (2004)._ _ _ _ Schattenberg, J. M., Wang, Y., Singh, R., Rigoli, R. M. & Czaja, M. J. Hepatocyte CYP2E1 overexpression and steatohepatitis lead to impaired hepatic insulin signaling. The Journal of biological chemistry 280, 9887u20139894, (2005)._ _ _ _ Leclercq, I. A., Lebrun, V. A., Starkel, P. & Horsmans, Y. J. Intrahepatic insulin resistance in a murine model of steatohepatitis: effect of PPARgamma agonist pioglitazone. Laboratory investigation; a journal of technical methods and pathology 87, 56u201365, (2007)._ _ _ _ Fraulob, J. C., Ogg-Diamantino, R., Fernandes-Santos, C., Aguila, M. B. & Mandarim-de-Lacerda, C. A. A Mouse Model of Metabolic Syndrome: Insulin Resistance, Fatty Liver and Non-Alcoholic Fatty Pancreas Disease (NAFPD) in C57BL/6 Mice Fed a High Fat Diet. Journal of clinical biochemistry and nutrition 46, 212u2013223, (2010)._ _ _ _ _ Massague, J. & Wotton, D. Transcriptional control by the TGF-beta/Smad signaling system. The EMBO journal 19, 1745u20131754, (2000)._ _ _ _ _ Zhang, L. et al. Smad2 protects against TGF-beta1/Smad3-mediated collagen synthesis in human hepatic stellate cells during hepatic fibrosis. Molecular and cellular biochemistry 400, 17u201328, (2015)._ _ _ _ Levy, M. & Futerman, A. H. Mammalian ceramide synthases. IUBMB Life 62, 347u2013356, (2010)._ _ _ _ _ Laviad, E. L. et al. Characterization of ceramide synthase 2: tissue distribution, substrate specificity, and inhibition by sphingosine 1-phosphate. The Journal of biological chemistry 283, 5677u20135684, (2008)._ _ _ _ Arndt, S. et al. Enhanced expression of BMP6 inhibits hepatic fibrosis in non-alcoholic fatty liver disease. Gut 64, 973u2013 981, (2015)._ _ _ _ Chen, B. L. et al. Exogenous bone morphogenetic protein-7 reduces hepatic fibrosis in Schistosoma japonicum-infected mice via transforming growth factor-beta/Smad signaling. World journal of gastroenterology 19, 1405u20131415, (2013)._ _ _ _ _ Wang, S. L. et al. Inhibitory effect of bone morphogenetic protein-7 on hepatic fibrosis in rats. International journal of clinical and experimental pathology 6, 897u2013903 (2013)._ _ _ Wang, L. P. et al. BMP-7 attenuates liver fibrosis via regulation of epidermal growth factor receptor. International journal of clinical and experimental pathology 7, 3537u20133547 (2014)._ _ _ _ Kinoshita, K. et al. Adenovirus-mediated expression of BMP-7 suppresses the development of liver fibrosis in rats. Gut 56, 706u2013714, (2007)._ _ _ _ Hao, Z. M., Cai, M., Lv, Y. F., Huang, Y. H. & Li, H. H. Oral administration of recombinant adeno-associated virus-mediated bone morphogenetic protein-7 suppresses CCl(4)-induced hepatic fibrosis in mice. Molecular therapy: the journal of the American Society of Gene Therapy 20, 2043u20132051, (2012)._ _ _ Regnier, M., Polizzi, A., Guillou, H. & Loiseau, N. Sphingolipid metabolism in non-alcoholic fatty liver diseases. Biochimie, (2018)._ _ _ Chavez, J. A. & Summers, S. A. A ceramide-centric view of insulin resistance. Cell Metab 15, 585u2013594, (2012)._ _ _ _ Montgomery, M. K. et al. Regulation of glucose homeostasis and insulin action by ceramide acyl-chain length: A beneficial role for very long-chain sphingolipid species. Biochim Biophys Acta 1861, 1828u2013 1839, (2016)._ _ _ _ Maceyka, M. & Spiegel, S. Sphingolipid metabolites in inflammatory disease. Nature 510, 58u201367, (2014)._ _ _ _ _ _ Siddique, M. M., Li, Y., Chaurasia, B., Kaddai, V. A. & Summers, S. A. Dihydroceramides: From Bit Players to Lead Actors. The Journal of biological chemistry 290, 15371u2013 15379, (2015)._ _ _ _ _ Mullen, T. D., Hannun, Y. A. & Obeid, L. M. Ceramide synthases at the centre of sphingolipid metabolism and biology. Biochem J 441, 789u2013802, (2012)._ _ _ _ Cinar, R. et al. Hepatic cannabinoid-1 receptors mediate diet-induced insulin resistance by increasing de novo synthesis of long-chain ceramides. Hepatology 59, 143u2013153, (2014)._ _ _ _ Kasumov, T. et al. Ceramide as a mediator of non-alcoholic Fatty liver disease and associated atherosclerosis. PLoS One 10, e0126910, (2015)._ _ _ _ _ Raichur, S. et al. CerS2 Haploinsufficiency Inhibits beta-Oxidation and Confers Susceptibility to Diet-Induced Steatohepatitis and Insulin Resistance. Cell Metab 20, 919, (2014)._ _ _ _ Turpin, S. M. et al. Obesity-induced CerS6-dependent C16:0 ceramide production promotes weight gain and glucose intolerance. Cell Metab 20, 678u2013686, (2014)._ _ _ _ Osawa, Y. et al. Roles for C16-ceramide and sphingosine 1-phosphate in regulating hepatocyte apoptosis in response to tumor necrosis factor-alpha. The Journal of biological chemistry 280, 27879u201327887, (2005)._ _ _ _ Yetukuri, L. et al. Bioinformatics strategies for lipidomics analysis: characterization of obesity related hepatic steatosis. BMC systems biology 1, 12, (2007)._ _ _ _ _ Yen, C. L., Mar, M. H. & Zeisel, S. H. Choline deficiency-induced apoptosis in PC12 cells is associated with diminished membrane phosphatidylcholine and sphingomyelin, accumulation of ceramide and diacylglycerol, and activation of a caspase. FASEB journal: official publication of the Federation of American Societies for Experimental Biology 13, 135u2013142 (1999)._ _ _ Yang, R. X. et al. Serum Monounsaturated Triacylglycerol Predicts Steatohepatitis in Patients with Non-alcoholic Fatty Liver Disease and Chronic Hepatitis B. Sci Rep 7, 10517, (2017)._ _ _ _ _ _ Anjani, K. et al. Circulating phospholipid profiling identifies portal contribution to NASH signature in obesity. J Hepatol 62, 905u2013912, (2015)._ _ _ _ Tiwari-Heckler, S., Gan-Schreier, H., Stremmel, W., Chamulitrat, W. & Pathil, A. Circulating Phospholipid Patterns in NAFLD Patients Associated with a Combination of Metabolic Risk Factors. Nutrients 10, (2018)._ _ _ Gorden, D. L. et al. Biomarkers of NAFLD progression: a lipidomics approach to an epidemic. J Lipid Res 56, 722u2013736, (2015)._ _ _ _ _ Luukkonen, P. K. et al. Hepatic ceramides dissociate steatosis and insulin resistance in patients with non-alcoholic fatty liver disease. J Hepatol 64, 1167u20131175, (2016)._ _ _ _ Wagner, M. et al. Liver receptor homolog-1 is a critical determinant of methyl-pool metabolism. Hepatology 63, 95u2013106, (2016)._ _ _ _ Li, Z. et al. The ratio of phosphatidylcholine to phosphatidylethanolamine influences membrane integrity and steatohepatitis. Cell Metab 3, 321u2013331, (2006)._ _ _ _ Puri, P. et al. A lipidomic analysis of nonalcoholic fatty liver disease. Hepatology 46, 1081u20131090, (2007)._ _ _ _ Zhao, Y. et al. Lack of phosphatidylethanolamine N-methyltransferase alters plasma VLDL phospholipids and attenuates atherosclerosis in mice. Arteriosclerosis, thrombosis, and vascular biology 29, 1349u20131355, (2009)._ _ _ _ Igolnikov, A. C. & Green, R. M. Mice heterozygous for the Mdr2 gene demonstrate decreased PEMT activity and diminished steatohepatitis on the MCD diet. J Hepatol 44, 586u2013592, (2006)._ _ _ _ Chiappini, F. et al. Metabolism dysregulation induces a specific lipid signature of nonalcoholic steatohepatitis in patients. Sci Rep 7, 46658, (2017)._ _ _ _ _ Fromenty, B., Berson, A. & Pessayre, D. Microvesicular steatosis and steatohepatitis: role of mitochondrial dysfunction and lipid peroxidation. Journal of Hepatology 26, 13u201322 (1997)._ _ _ _ de Aguiar Vallim, T. Q., Tarling, E. J. & Edwards, P. A. Pleiotropic roles of bile acids in metabolism. Cell Metab 17, 657u2013669, (2013)._ _ _ _ _ Somm, E. et al. beta-Klotho deficiency protects against obesity through a crosstalk between liver, microbiota, and brown adipose tissue. JCI Insight 2, (2017).Somm, E. et al. b-Klotho deficiency shifts the gut-liver bile acid axis and induces hepatic alterations in mice. American journal of physiology. Endocrinology and metabolism (2018).Tanaka, N., Matsubara, T., Krausz, K. W., Patterson, A. D. & Gonzalez, F. J. Disruption of phospholipid and bile acid homeostasis in mice with nonalcoholic steatohepatitis. Hepatology 56, 118u2013129, (2012)._ _ _ _ Halilbasic, E., Claudel, T. & Trauner, M. Bile acid transporters and regulatory nuclear receptors in the liver and beyond. J Hepatol 58, 155u2013168, (2013)._ _ _ _ _ Dawson, P. A., Lan, T. & Rao, A. Bile acid transporters. J Lipid Res 50, 2340u20132357, (2009)._ _ _ _ _ Park, T. S. et al. Inhibition of sphingomyelin synthesis reduces atherogenesis in apolipoprotein E-knockout mice. Circulation 110, 3465u20133471, (2004)._ _ _ _ Hojjati, M. R. et al. Effect of myriocin on plasma sphingolipid metabolism and atherosclerosis in apoE-deficient mice. The Journal of biological chemistry 280, 10284u2013 10289, (2005)._ _ _ _ Park, T. S. et al. Modulation of lipoprotein metabolism by inhibition of sphingomyelin synthesis in ApoE knockout mice. Atherosclerosis 189, 264u2013272, (2006)._ _ _ _ Yang, G. et al. Central role of ceramide biosynthesis in body weight regulation, energy metabolism, and the metabolic syndrome. Am J Physiol Endocrinol Metab 297, E211u2013224, (2009)._ _ _ _ _ Correnti, J. M., Juskeviciute, E., Swarup, A. & Hoek, J. B. Pharmacological ceramide reduction alleviates alcohol-induced steatosis and hepatomegaly in adiponectin knockout mice. Am J Physiol Gastrointest Liver Physiol 306, G959u2013973, (2014)._ _ _ _ _ Schneider, C. A., Rasband, W. S. & Eliceiri, K. W. NIH Image to ImageJ: 25 years of image analysis. Nature methods 9, 671u2013675 (2012)._ _ _ _ _ Bedossa, P. et al. Histopathological algorithm and scoring system for evaluation of liver lesions in morbidly obese patients. Hepatology 56, 1751u20131759, (2012)._ _ _ Loizides-Mangold, U. et al. Lipidomics reveals diurnal lipid oscillations in human skeletal muscle persisting in cellular myotubes cultured in vitro. Proc Natl Acad Sci USA 114, E8565u2013E8574, (2017)._ _ _ _ _ Matyash, V., Liebisch, G., Kurzchalia, T. V., Shevchenko, A. & Schwudke, D. Lipid extraction by methyl-tert-butyl ether for high-throughput lipidomics. J Lipid Res 49, 1137u20131146, (2008)._ _ _ _ _ Clarke, N. G. & Dawson, R. M. Alkaline O leads to N-transacylation. A new method for the quantitative deacylation of phospholipids. Biochem J 195, 301u2013306 (1981)._ _ _ _ _ Hammer xD8, H. D. & Ryan, P. D. PAST: Paleontological statistics software package for education and data analysis. Palaeontol Electronica 4, 9 (2001)._ \"- - We would like to thank Marie-Claude Brulhart Meynet and Florian Visentin for technical assistance as well as Prof. Jacques Phillippe and his team for practical support and valuable discussions. This work was funded by the Gottfried and Julia Bangerter-Rhyner Foundation, unrestricted grant from Novo Nordisk, the Foundation of the Swiss Diabetes Association, the Swisslife Foundation (F.R.J), the SNSF grants 31003A_166700/1 and CRSII3_154405 (C.D.), and by the Bo & Kerstin Hjelt diabetes Foundation (U.L.-M.).- - Authors and Affiliations - \"Service of Endocrinology, Diabetes, Hypertension and Nutrition, Department of Internal Medicine, Geneva University Hospitals/University of Geneva, Geneva, SwitzerlandSophie A. Montandon,_Emmanuel Somm,_Ursula Loizides-Mangold,_ Charna Dibner_&_FranxE7ois R. JornayvazDepartment of Cell Physiology and Metabolism, University of Geneva, Geneva, SwitzerlandUrsula Loizides-Mangold_&_Charna DibnerDiabetes Center, Faculty of Medicine, University of Geneva, Geneva, SwitzerlandUrsula Loizides-Mangold_&_Charna DibnerInstitute of Genetics and Genomics in Geneva (iGE3), University of Geneva, Geneva, SwitzerlandUrsula Loizides-Mangold_&_ Charna DibnerDivision of Clinical Pathology, Geneva University Hospitals, Geneva, SwitzerlandClaudio de Vito\" - \"AuthorsSophie A. MontandonYou can also search for this author in _Emmanuel SommYou can also search for this author in _Ursula Loizides-MangoldYou can also search for this author in _Claudio de VitoYou can also search for this author in _Charna DibnerYou can also search for this author in _FranxE7ois R. JornayvazYou can also search for this author in _\" - Contributions - S.A.M., E.S. and F.R.J. conceived the study, S.A.M., U.L.-M. and E.S. conducted the experiments, analyzed the data and wrote the manuscript, C.d.V. established the SAF scores. S.A.M., E.S., U.L.-M., C.d.V., C.D. and F.R.J. reviewed the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - Supplementary table S1, Supplementary figure S1, Supplementary figure S2, Supplementary figure S3, Supplementary figure S4- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License; .\" - \"Fig. 2: Rheological characterization of spinning dopes.a Viscosity changes as a function of shear rate when extending the curing time (tcuring) at 25u2009 xB0C. b Effect of the curing temperature (Tcuring) on the viscosity change. temp., temperature. c Effect of the silver ion concentration (MAg+) on the viscosity change. d Evolution of the index n (extracted using the Cross-power model) for spinning dopes with different tcuring, Tcuring, MAg+. e Shear stress as a function of shear rate for spinning dopes at different tcuring at 25u2009xB0C. f Mapping of the spinning dopes viscosity at different tcuring, Tcuring, and MAg+. Black dashed lines marked the desired viscosity range where a good spinnability can be readily obtained.\"- - Fiber formation under ambient conditions via the NVIPS spinning - \"To achieve an optimal spinnability at room temperature, we adjusted the molecular network structures by introducing [Ag(Nu2009u2261u2009Cu2009u2212u2009 )x] complexes to PAN solutions. In contrast to the control of PAN dopes with only Van der Waals force between neighboring chains (Fig._ top), PANSion dopes presented additional inter- and intra-chain interactions (i.e., entanglement and crosslinking) through [Ag(Nu2009u2261u2009Cu2009u2212u2009)x] complexes (Fig._ bottom). This statement was first verified by the additional peak at 399.7u2009eV of the X-ray photoelectron spectroscopy (XPS) results, which was assigned to the [Ag(Nu2009u2261u2009Cu2212)x] complexes for PANSion samples (Supplementary Fig._). Notably, a significant increase of the zero-shear viscosity was recorded for PANSion dopes (Fig._), which was ascribed to the entangled and crosslinked molecular chain network via [Ag(Nu2009u2261u2009Cu2212)x] complexes. Concomitantly, the unique PANSion network guaranteed the excellent ambient-condition extensibility to form a large-aspect-ratio thread (Fig._). In addition, the autonomous solidification of a precursor fiber was promptly triggered by the surrounding water vapor (75% RH) due to the NVIPS effect. As schemed in Fig._, a PANSion fiber can be easily obtained via our NVIPS spinning approach by manually drawing or adapting a continuous extrudingu2013stretching process (Supplementary Fig._ and Supplementary Movies_ and ). By comparison, drawing a PAN dope under ambient conditions always showed capillary failure or gravity drainage (Supplementary Fig._), because of the insufficient inter-/intra-chain interactions (even for a concentrated PAN dope with a comparable viscosity to PANSion dopes).\" - \"Fig. 3: PANSion fibers via the NVIPS spinning approach.a Images showing the spinnability difference of PAN and PANSion dopes (scale bar, 1u2009cm). b Fiber formation via the NVIPS spinning approach where solvent droplets hanging on a fiber and can be autonomously removed due to gravity effect (scale bar, 1u2009cm). c A fiber holding a dead-load 100u2009g (scale bar, 1u2009cm). d An OM image of a fiber knot (scale bar, 100u2009xB5m). e A fiber passing through a needle hole (scale bar, 1u2009cm). f FTIR spectra of PANSion fiber with different silver ion concentrations. g, h UV-Vis spectra of spinning dopes after thermal curing at different temperatures (g) and silver ion concentrations (h). Ctrl, control. iu2013k TEM images showing the in-situ reduced AgNPs at both low (i, scale bar, 10u2009nm) and high (j, k, scale bar, 5u2009nm) magnifications. l SEM image showing a PANSion fiber surface. mu2013o Cross-sectional morphologies of PANSion fiber via NVIPS spinning approach (m), PANSion fiber via wet spinning approach (n), and control PAN fiber via wet spinning approach (o). p Elements mapping of the PANSion fiber from the red dash-line box in m, including C, N, O, and Ag. All scale bars for lu2013p are 50u2009xB5m.\" - \"For the ambient-conditions fiber formation via NVIPS spinning, three key processes were identified from the view of both the fibers longitudinal section and cross section (Fig._). Owing to the good miscibility of water and DMF, a precursor gel fiber could maintain a continuous water vapor adsorption once it was drawn into air (process u2460: miscibility-controlled water inward-adsorption), eventually forming a DMF/water mixture on the fiber surface. Note that DMF/water mixture was a poor solvent for the PANSion solute, thus resulting in a thin liquid layer of DMF/water on the fiber surface. Meanwhile, the PANSion solute contracted inwardly to the good solvent of DMF (process u2461: solubility-controlled inward-contraction), gradually leading to a fiber diameter shrinkage with a concomitant solute concentration increase. Simultaneously, a chemical potential difference was established due to the different DMF concentrations between the fibers inside and surface (process u2462: concentration-gradient-controlled outward-diffusion), thus expelling the continuous outward-precipitation of DMF solvent. Eventually, the solute part of PANSion contracted to a solid fiber, while the solvent part (DMF) separated from the solute and formed liquid droplets (with water) hanging on the fiber. Note that the phase transition from a liquid-like gel fiber to a solid freestanding fiber was independent and autonomous under ambient conditions (75% RH, 24u2009xB0C). Just as visualized experimentally by an optical microscope (Fig. ), a precursor fiber was gradually solidified with diameter shrinkage and solvent outward-precipitation (Fig._ and Supplementary Movie_). The liquid phase eventually amalgamated into droplets hanging on the freestanding fiber (Fig._). By inclining the fiber during spinning, DMF/water droplets can be easily removed or recollected due to the gravity effect (Fig._ , Supplementary Figs._ and , and Supplementary Movie_).\" - Tuning ambient-condition spinnability - \"We noticed that the zero-shear viscosity is a good indicator to assess the ambient-conditions spinnability of PANSion dopes (Supplementary Fig._). Having this in mind, viscosity adjustment was realized by modulating the density of [Ag(Nu2009u2261u2009 Cu2009u2212u2009)x] complexes, which, as crosslinkers, can enhance inter- and intra-chain interactions, thus effectively increasing the zero-shear viscosity to the desired level. Additionally, due to the dynamic and robust nature of [Ag(Nu2009u2261u2009Cu2009u2212u2009)x] complexes, the PANSion dopes embraced an elastic chain network, which guaranteed the good extensibility under ambient conditions. That is to say, the entanglement/crosslinking-enhanced zero-shear viscosity via [Ag(Nu2009u2261u2009Cu2009u2212u2009)x] complexes is the key to achieving good ambient-conditions spinnability (Supplementary Fig._ ). It is worth pointing out that a PAN dope, with either low or high concentration, is not capable of forming a long and stable precursor thread under ambient conditions either due to the high local stress or capillarity instability (Supplementary Note_).\" - \"In our current study, the zero-shear viscosity of PANSion dopes was synergistically dictated by three factors: thermal curing time (tcuring), curing temperature (Tcuring), and silver ion concentration (MAg+). Thus, we characterized the zero-shear viscosity (({eta }_{0})) as a function of shear rate at different combinations of these parameters (Fig._ and Supplementary Fig._). For instance, when extending the curing time (Tcuring = 25u2009xB0C and MAg+ = 1.177u2009M), the zero-shear viscosity presented an increase by more than two orders of magnitude, from 753 to 29,573u2009mPaxB7s (Fig._). Importantly, when the zero-shear viscosity was leveled up to the optimal range, in which good ambient-conditions spinnability was achieved, the shear stress presented a non-linear dependence on the shear rate (green dot in Fig._ and Supplementary Fig._). This behavior was different from the near-linear slope of non-spinnable dopes (black and red dots in Fig._ ). In addition, the zero-shear viscosity was significantly affected by the curing temperature from 25 to 55u2009xB0C (MAg+ = 1.177u2009M, and tcuring is at a comparable level), as the differences shown in Fig._. For instance, to reach the optimal viscosity range of ~10u2009mPaxB7s, the required curing time for dopes cured at Tcuringu2009=u200940u2009xB0C was much shorter than that of dopes cured at Tcuringu2009=u200925u2009xB0C. It was further shortened to just 27u2009h for the case of Tcuringu2009=u200955u2009xB0C. Figure_ presented the comparison of the zero-shear viscosity for dopes cured at Tcuringu2009 =u200955u2009xB0C while varying the curing time and silver ion concentrations. Notably, the required curing time to reach the optimal zero-shear viscosity range was decreased at a higher silver ion concentration (Supplementary Table_ ).\" - \"To map the parameter zone of good ambient-condition spinnability regarding to the tcuring, Tcuring, and MAg+, the flow behavior of PANSion dopes was investigated. PANSion dopes only started to show a non-Newtonian behavior (i.e., shear thinning for our case) for a shear rate greater than 100u2009s. Once the factor of thermal curing (including tcuring and Tcuring) came in, the shear thinning effect became obvious at a much lower shear rate (i.e., <1u2009s for tcuringu2009=u2009 144 and 192u2009h, Fig._). The non-Newtonian behavior at various combinations of tcuring, Tcuring, and MAg+ can be explained by the flow behavior index, n, using Cross power-law model:\" - \"$$eta left(dot{gamma }right)=frac{{eta }_{0}}{1+{left(frac{{ eta }_{0}dot{gamma }}{{tau }^{*}}right)}^{1-{{{{{boldsymbol{n}}}}}}}}$$ (1) \" - \"where (dot{gamma }) is the shear rate (s), u03C4* is the critical shear stress (Pa) (i.e., the stress at the break in the viscosityu2013shear rate curve), and n is the flow behavior index. Generally, a Newtonian fluid presents a value of nu2009=u20091, while a shear thinning fluid is with nu2009<u2009 1 and shear thickening fluid is with nu2009>u20091. By fitting Eq. () to the viscosityu2013shear rate data, we obtained the power-law index values of n under different combinations of factors, as displayed in Fig._ (Supplementary Figs._ and ). The overall flow behavior of spinning dopes presented a decrease trend when the curing time was extended, regardless of Tcuring or MAg+. For instance, n was decreased from 0.828 to 0.421 for dopes cured at Tcuringu2009 =u200925u2009xB0C and MAg+ = 1.177u2009M. The minimum value of n (nmin) was in the range of 0.338u20130.421. Particularly, the value of n showed an abrupt increase once crossing nmin if further extending the curing time.\" - \"When nmin was reached, the corresponding zero-shear viscosity of spinning dopes was also in the optimal range for ambient-condition spinning. Although slightly crossing the value of nmin did not deteriorate the dope spinnability significantly, further extending the curing time should be avoided as the viscosity changed swiftly to a very high level (i.e., the gelation). Based on these discussions, we mapped the parameter zone of tcuring, Tcuring, and MAg+, as shown in Fig._ , to identify the optimal viscosity range, which was denoted by a dashed line (i.e., the spinnable viscosity edge). In other words, when the zero-shear viscosity of PANSion dopes was in the proximity of the spinnable edge, the ambient-condition spinning can be realized. Out of the spinnable edge, the dope will only result in unstable fiber formation due to insufficient entanglements or no spinnability of the gelation.\" - Silver-coordinated molecular networks and in-situ reduced AgNPs - \"Reaping the benefits of coordination complexes-enabled good spinnability and the NVIPS effect-induced autonomous fiber fibrillation, a large aspect-ratio precursor fiber can be easily drawn (Fig._), which went through phase transition instantly to a freestanding solid fiber under ambient conditions (Fig._). The obtained PANSion fiber was strong (i.e., capable of holding a dead-weight 100u2009 g, Fig._), flexible and stretchable (Supplementary Fig._), and knottable (a knot with a radius <50u2009xB5m, Fig._). This could be ascribed to the elastic molecular chain networks via [Ag(Nu2009u2261u2009Cu2009u2212u2009)x] complexes. Furthermore, the fiber diameter is variable (i.e., small enough to pass through a needle hole, Fig._) by adjusting the stretching rate during spinning. Next, we verified the existence of [Ag(Nu2009u2261u2009Cu2009u2212u2009)x] complexes (also see Supplementary Fig._). As the FTIR spectra results shown in Fig._ , a peak at 2242u2009cm (u2212Cu2009u2261u2009N) was recorded for the control PAN dopes. This characteristic peak was red-shifted with an additional shoulder peak at 2266u2009cm for PANSion dopes because of the chemical environment change of nitrile groups by silver ions. In addition, we observed the color variations of spinning dopes after thermal curing (Supplementary Fig._), which was attributed to the in-situ reduced AgNPs. Previous studies reported that DMF solvent could reduce metal ions to nanoparticles, especially when the reaction temperature was elevated. Indeed, we confirmed the existence of AgNPs based on UV-Vis spectra results which displayed a unique peak at 425u2009nm, resulting from the surface plasma resonance of AgNPs (Fig._). At a higher silver ion concentration, more AgNPs were reduced in situ (Fig._). Furthermore, these AgNPs, well dispersed in the spinning dopes, were directly observed by TEM images (Fig._). Note that these in situ reduced AgNPs not only facilitated to create conductive pathways for as-prepared PANSion fibers, but also enabled the structural stabilization of the precursor gel fibers/dopes at a colloidal scale.\" - Mechanical properties of PANSion fibers - \"Although both PAN and PANSion fibers can be obtained via a wet spinning (WS) approach, the mechanical properties and morphologies of these fibers (fibers@WS) were significantly different from those via NVIPS spinning approach (fibers@NVIPS). For instance, unlike the high stretchability of PANSion fibers@NVIPS, fibers@WS were very brittle and not stretchable (Supplementary Fig._ and Supplementary Movie_). Figure_ showed the surface morphology of a PANSion fiber@NVIPS, which was smooth and different from crumped/grooved structures of fibers@WS (Supplementary Fig._ and ). Moreover, the internal structures of fibers@NVIPS were solid, whereas fibers@WS were porous with many defects (Fig._). This was due to the unstable spinodal decomposition of precursor fibers via wet spinning. By contrast, the phase separation via NVIPS spinning approach was mild under ambient conditions, which facilitated the formation of a more morphology-controlled fiber. Particularly, the distribution of silver element was even in the whole cross-sectional area (Fig._ and Supplementary Fig._). These results affirmed the advantages of using NVIPS spinning approach to produce functional soft fibers over conventional spinning methods.\" - \"Tensile properties of PANSion fibers were investigated as the strain-stress curves plotted in Fig._. A pattern of the two-stage load-bearing structure was presented: a fast linear stress growth in the low-strain regime and a slow stress growth in the large strain regime. A strain u201Chardeningu201D effect was recorded when raising the ramp rate from 5 to 150u2009mm/min (Fig._). Such mechanical features were similar to typical glassy polymers. Furthermore, cyclic stress-strain tests indicated a negligible residual strain when consecutively increasing strains from 0 to 150%, or cyclically stretching-releasing at 50% or 100% strains (Supplementary Fig._). Besides, we examined the effect of Tcuring and MAg+ on tensile behaviors of fibers. For instance, when increasing the silver ion concentrations (Tcuring = 25u2009xB0C), PANSion fibers presented a tendency to decrease the tensile strength, yet an opposite trend towards increasing failure strain (Fig._). This behavior pattern was also true for fibers spun from dopes cured at Tcuringu2009 =u200955u2009xB0C (Fig._). Such mechanical behavior was possibly ascribed to the enhanced elasticity of PANSion molecular chain networks when the silver ion concentration was high. It should be noted that for fibers spun from dopes cured at Tcuringu2009=u200940u2009xB0C (Fig._), the failure strain presented a different behavior in contrast to fibers from dopes cured at Tcuringu2009 =u200925 or 55u2009xB0C. Given this, we further conducted statistical analysis on tensile strength and failure strain (Fig._). For tensile strength, statistical significance was confirmed for all investigated fibers (Fig._), with a minimum pu2009<u20090.01 under the one-factor ANOVA test. However, the influence of curing temperature (Tcuring) on the failure strain was mixed (Fig._). From this perspective, future work will be conducted further to elaborate on the relationship between the parameters of preparing the spinning dopes and mechanical properties.\" - \"Fig. 4: Mechanical properties of PANSion fibers.au2013c Effects of silver ion concentrations on the tensile properties of PANSion fibers using spinning dopes that were thermally cured at 25u2009xB0C (a), 40u2009xB0C (b), and 55u2009 xB0C (c). d The effect of the strain rate on the fiber tensile behavior. e, f Box plot (center line at the median, upper bound at 75th percentile, lower bound at 25th percentile, whiskers at the minimum and maximum values with outliers not counted; each dot represents one test, nu2009=u200910) of the fiber tensile strength (e) and failure strain (f). Statistics are calculated using a one-factor ANOVA test. p values for comparisons are shown: *pu2009<u20090.05; **pu2009 <u20090.01; ***pu2009<u20090.001; ns (non-significant) pu2009>u20090.1.\" - Self-sensing PANSion fibers - \"The piezoresistive effect was verified for a single PANSion fiber (Fig._) because the in situ reduced AgNPs established conductive pathways, reaching a decent electrical conductivity of 0.5u20132u2009S/m. Note that silver ions in spinning dopes were excessive. Therefore, ionic conductivity also contributed to the overall conductivity improvement (Supplementary Fig._). It is worth noting that fabricating a conductive and stretchable fiber with piezoresistive sensing capability is usually associated with multi-step processes, such as coating conductive nanomaterials (i.e., carbon nanotubes) on a stretchable fiber substrate, filling liquid metals into a stretchable fiber hose, pre-strain engineering, or conductive filler embedding, etc. For our PANSion fibers, heterogeneous functions of both mechanical and electrical features were accomplished in a single-step process via NVIPS spinning approach, thus achieving the intrinsic self-sensing by the PANSion fiber itself. It did not require any post-spinning processes to enable the piezoresistive effect with no potential issues of coating delamination, poor adhesion, stress concentration, and electronic/mechanical property mismatch.\" - \"Fig. 5: Self-sensing of PANSion fibers for body motion monitoring and high-frequency movement detection.a Piezoresistive effect of PANSion fibers, roughly showing three stages of sensitivity (S1, S2, and S3) in the strain range of 0u2013 200%. b 10 cycles of sensing signals at both small (1u20135%, top) and large (10u201350%, bottom) strains. c Sensing signals at various motion frequencies from 1u201350u2009Hz. du2013f Human body motion monitoring using the self-sensing fiber, including a sensing glove for finger bending monitoring (d) and a sensitive fabric band for wrist flexion (e) and elbow flexion (f) monitoring. g Sensing signals under a mechanical stimulus at 10u2009Hz for 2300 cycles. hu2013j Monitoring the high-frequency movement of a converter using a self-sensing fiber: h the SFFT (short-time Fourier transform) analysis, i the corresponding frequency spectra, and j the sensing signals when the converter was working at various frequencies.\" - \"Given its excellent stretchability and decent conductivity, we explored the potential of PANSion fibers as self-sensing devices to monitor mechanical stimuli. Firstly, a decent sensitivity of 1.48u20133.41 was achieved for strains from 0u2013 200% (Fig._). Secondly, the sensing signals were stable and repeatable at both small (1u20135%) and large (10u201350%) strains (Fig._). As shown in Fig._ , when sewing a PANSion fiber into a fabric, a self-sensing textile was enabled to successfully detect joint motions of the human body, including finger bending and wrist/elbow flexions. More impressively, we noticed that the PANSion fiber was highly responsive to high-frequency stimuli at relatively small strains (i.e., >50u2009Hz at ~5u201310% strain, Fig._ and Supplementary Fig._). This feature could be credited to the resilience of fibers when strains were low. In light of the ability to detect high-frequency stimuli, we applied it to successfully monitor the working status of an electrical-mechanical converter (Fig._). As shown in Fig._, stable piezoresistive signals were presented for a mechanical stimulus at 10u2009Hz for ~2300 cycles. By adjusting the supplied voltage to the converter, the connecting lever (where a fiber sensor was attached, Fig._) oscillated up and down at different frequencies (Supplementary Fig._ and Supplementary Movie_). For instance, the oscillation frequency of ~8.57u2009 Hz under a voltage of 9u2009V was successfully captured by the self-sensing fiber, which increased to 16.63u2009Hz under a 12u2009V voltage (Fig._).\" - Self-powered PANSion fibers - \"Benefiting from the excellent softness, stretchability, and conductivity, PANSion fibers can be configured into a self-powered sensing device or a mechanical energy harvesting system based on the triboelectric mechanism (Fig._). As illustrated in Fig._, a single-electrode triboelectric fiber was made by a sheath coating layer of poly(vinylidene fluoride-co-hexafluoropropylene) (PVDF-HFP) as the triboelectrification layer and a PANSion fiber as the core electrode (Supplementary Fig._). The working principle was illustrated in Fig._ (see details in Supplementary Note_), which is mainly based on the coupling of triboelectrification and electrostatic induction effects. As results shown in Fig._, a 2-cm-long triboelectric fiber displayed an open-circuit voltage (VOC) of 1.2u2009V, short-circuit current (ISC) of 12u2009nA, and a short-circuit charge quantity (QSC) of 0.5u2009 nC. The output voltage showed an upward tendency with the increasing load resistance due to Ohms law (Fig._). The peak instantaneous output power of the triboelectric fiber could reach 0.82u2009u03BCW/m at 0.4u2009Mu03A9. It also should be noted that the internal resistance of this triboelectric fiber was much lower than those of previously reported fiber-based TENGs (Supplementary Table_).\" - \"Fig. 6: Self-powered PANSion fibers based on triboelectric mechanism.a, b Schematic diagram (a) and working mechanism (b) of the triboelectric fiber in the single-electrode mode, inside a: cross-section of the triboelectric fiber (Scale bar: 100u2009 u03BCm). c Electrical outputs measured at a frequency of 1.5u2009Hz. d Voltage density and power density of triboelectric fiber at a series of external loads. e Electrical outputs of triboelectric fiber at various frequencies (1u2013 5u2009Hz). f Cyclic output voltage under continuous mechanical impact for 1000 cycles. g Digital photo of triboelectric fiber with various diameters. h Electrical outputs of triboelectric fibers with different diameters. i Output voltage corresponding to five different tapping materials. j Pressure sensitivity from the output voltage-pressure curve. The red line corresponds to the linear fitting. k The electrical signals from small, medium, and large finger tapping force. l The output voltage corresponding to motions from finger, wrist, and elbow for biomechanical energy harvesting.\" - \"When varying the frequency of mechanical impact from 1u20135u2009Hz (in the range of most human motions), the triboelectric fiber presented a stable output voltage (VOCu2009=u20091u2009V) and an increasing trend for short-circuit current (ISC) from 12u201322u2009nA (Fig._ and Supplementary Fig._). Besides, the output voltage exhibited no noticeable attenuation after 1000 cycles (Fig._ ), indicating its good stability and durability. Interestingly, we found that a smaller diameter triboelectric fiber (i.e., from 0.28u20130.72u2009mm, mainly determined by the size of the conductive PANSion fiber), will generate a higher output voltage (Fig._, Supplementary Note_, and Supplementary Fig._). This effect could be ascribed to the increased electrostatic induction effect due to the considerable deformation for a small-sized triboelectric fiber when being pressed with the same external force (Supplementary Fig._). Furthermore, when in contact with the triboelectric fiber, different materials may generate diverse electrical signals owing to the different electron affinity, which demonstrates the feasibility of the triboelectric fiber for material recognition (Fig._ and Supplementary Note_). Figure_ presented the pressure sensitivity (K1u2009 =u20091.48u2009mV/Pa) of the triboelectric fiber as a tactile sensor in the pressure range of less than 200u2009kPa, which is high enough for human motion-based pressure sensing applications (see details in Supplementary Note_). For instance, when increasing the tapping force by a finger, the output electrical signals became noticeably higher (Fig._). Taking these features into account, including superior flexibility and excellent sensing characteristics, the PANSion triboelectric fiber also could be adopted as a self-powering sensor for body motion monitoring (Fig._), which will play a significant role in the wearable electronics fields.\"- - \\'Current functional soft fiber formation is usually an extrusion-dominated process and requires thermal melting of polymer blends and multi-step post-extrusion processes, which are energy-intensive. Post-treatments or filler-embedding procedures to enable electrical functions further complicate the fiber formation process and bring about potential issues. In contrast, the natural silk fibrillation by insects under ambient conditions is produced at a much higher energy efficiency level (at least three orders of magnitude less in energy intensity) yet is empowered with unique functions in-situ (i.e., directional wettability). In light of mimicking the native silk spinning, we successfully demonstrated a facile and energy-efficient way to produce functional soft PANSion fibers via a single-step spinning process under ambient conditions. Our NVIPS spinning technique under ambient conditions is enabled by surmounting two issues that are present in existing spinning methods: (1) achieving good ambient-conditions spinnability via silver-based coordination complexes and (2) accomplishing the autonomous phase transition from a precursor gel fiber to a solid freestanding fiber due to the water vapor-enabled NVIPS effect.\\' - \"In summary, our functional soft PANSion fiber via NVIPS spinning approach does not require any post-spinning treatments, such as thermal heating, in-line UV curing, coagulation bath, nor exposure to hot steam. The required energy to spin PANSion fibers can be further reduced via a manual dry-spinning way. According to a recent study by Cruz et al., the energy required for our NVIPS spinning approach is estimated to be roughly two orders of magnitude less than commercial synthetic fibers (only shearing input is needed when using an extrusion process, Fig._). More importantly, electrical functionality is simultaneously obtained for PANSion fibers along with mechanical softness and stretchability. The merits of extraordinary energy efficiency and seamless mechanical-electrical function integration via a single-step process represent significant progress in producing functional soft fibers under ambient conditions. As a proof-of-concept, PANSion fibers were successfully tested for various applications because of the unified mechanical-electrical properties, including self-sensing fiber electronics to monitor mechanical stimuli and self-powering for sensing and energy harvesting. In short, the formation of PANSion fibers via our proposed NVIPS spinning approach resemblesu2014to some extentu2014the natural silk fibrillation by spiders. These results of recreating the spider silk fibrillation process under ambient conditions provide insight into developing facile and energy-saving spinning technologies to afford a more sustainable future with abundant functional soft fibers.\"- - Materials - \"Polyacrylonitrile (PAN, >99%, Mw 150,000), N,N-dimethylformamide (DMF, 99.8%), silver nitrate (AgNO3, ACS reagent, u226599.0%), and poly(vinylidene fluoride-co-hexafluoropropylene) (PVDF-co-HFP) were purchased from Sigma-Aldrich. All chemicals were used as received without any post-treatment.\" - Preparation of spinning dopes - \"PAN/DMF solutions were firstly prepared by just mixing the PAN powder and DMF solvent under magnetic stirring. Then AgNO3 salt was added to the PAN/DMF solutions to obtain the as-prepared PANSion dopes (see Supplementary Table_ for detailed dope compositions). Then thermal curing for a specific time at 25, 40, and 55u2009 xB0C was applied to enable a good ambient-condition spinnability for PANSion dopes. The same procedures were applied to prepare the control PAN dopes which were just pure PAN/DMF solutions.\" - Spinning PANSion fibers via the NVIPS spinning approach - \"Spinnable PANSion dopes can be used for PANSion fiber preparation under ambient conditions, either via a manual spinning way or using a syringe-based extrusion system (Supplementary Fig._). Typically, for manual spinning, one can obtain a fiber by stretching a gel ball of PANSion dopes (75% RH at 24u2009xB0C). Fiber diameter can be adjusted by controlling the stretching ratio. For continuous spinning, extruding a dope-filled syringe can result in fiber formation while continuously stretching and collecting the fiber on a bobbin. Fibers could be drawn continuously and uniformly with a length longer than two meters. Note that the PAN dopes are not suitable for spinning via NVIPS spinning approach. However, both as-prepared PANSion and PAN dopes can be used for producing fibers via a wet spinning approach (Supplementary Fig._).\" - Self-sensing and self-powered fiber devices - PANSion fibers were used as self-sensing strain sensors without any post treatments. They can be further integrated with commercial textiles, thus enabling sensitive wearables electronics. In this study, body motions of elbow and wrist flexion and finger bending were monitored by the sensitive textiles or gloves. Resistance changes of the self-sensing fiber sensors during the body motions were recorded by a multimeter (Keithley 6500). For self-powered sensing devices, a PANSion fiber as the electrode was uniformly coated with a PVDF-HFP layer and then dried at room temperature. - Characterization - \"Rheological properties of spinning dopes were characterized using a Rheometer (Anton Paar, MCR 302) with a parallel plate of u220525u2009mm in the shear rate range of 0.1 to 1000 1/s. A tensile test was performed on a universal mechanical testing machine at 50u2009mm/min. For cyclic tests, the fiber was first stretched to a designed strain and then rested for intervals of 0, 5, 10, 20, and 30u2009 min before the next stretching. UV-Vis (ultravioletu2013visible) spectra were recorded from 800 to 250u2009nm using an Agilent Cary 5000. FTIR (Fourier-transform infrared) spectra were recorded using a Thermo Nicolet NEXUS 470 spectrometer with a wavenumber from 4000u2009cm to 400u2009cm. AgNPs were characterized using Transmission electron microscopy (TEM, JEOL JEM 2010F). Field-emission scanning electron microscopy (FE-SEM, Zeiss Supra 300) with an accelerating voltage at 3u2009kV was used for fiber morphology characterization. X-ray photoelectron spectroscopy (XPS, Kratos Analytical, Axis Ultra DLD) was used to analyze the chemical changes induced by the formation of silver-based coordination complexes. Optical images were recorded using an optical microscope (OM), Nikon Eclipse LV1000, equipped with Digital Sight DS-U1. Resistance changes (u0394R/R0) of fibers when subjected to various strains were collected using a multimeter (Keithley 6500). Sensing data from 1 to 50u2009Hz were first collected using a linear motor as the frequency generator. The electric output performances of the self-powered PANSion fiber were characterized by a Keithley 6514 electrometer. This work has been obtained informed consent from all participants and we have complied with all relevant ethical regulations.\"- - \\' \\' - All the data supporting the findings in this study are available in the article and files or available from the corresponding authors upon request. - \\' \\'- - \"Loke, G. et al. Digital electronics in fibres enable fabric-based machine-learning inference. Nat. Commun. 12, 3317 (2021)._ _ _ _ _ _ Shi, X. et al. Large-area display textiles integrated with functional systems. Nature 591, 240u2013245 (2021)._ _ _ _ _ Meng, K. et al. A wireless textile-based sensor system for self-powered personalized health care. Matter 2, 896u2013907 (2020)._ _ Su, Y. et al. Muscle fibers inspired high-performance piezoelectric textiles for wearable physiological monitoring. Adv. Funct. Mater. 31, 2010962 (2021)._ _ _ Zhang, S. et al. Leveraging triboelectric nanogenerators for bioengineering. Matter 4, 845u2013887 (2021)._ _ _ Zhou, Z. et al. Sign-to-speech translation using machine-learning-assisted stretchable sensor arrays. Nat. Electron. 3, 571u2013 578 (2020)._ _ Zhang, S., Suresh, L., Yang, J., Zhang, X. & Tan, S. C. Augmenting sensor performance with machine learning towards smart wearable sensing electronic systems. Adv. Intell. Syst. 4, 2100194 (2022)._ _ Park, M. et al. Highly stretchable electric circuits from a composite material of silver nanoparticles and elastomeric fibres. Nat. Nanotechnol. 7, 803u2013809 (2012)._ _ _ _ _ Jung, H. J. et al. Self-planarization of high-performance graphene liquid crystalline fibers by hydration. ACS Cent. Sci. 6, 1105u20131114 (2020)._ _ _ _ _ Wu, R. & Kim, T. Review of microfluidic approaches for fabricating intelligent fiber devices: importance of shape characteristics. Lab Chip 21, 1217u20131240 (2021)._ _ _ _ Wang, B. et al. Flexible and stretchable metal oxide nanofiber networks for multimodal and monolithically integrated wearable electronics. Nat. Commun. 11, 2405 (2020)._ _ _ _ _ _ Chen, M. et al. Self-powered multifunctional sensing based on super-elastic fibers by soluble-core thermal drawing. Nat. Commun. 12, 1416 (2021)._ _ _ _ _ _ Kanik, M. et al. Strain-programmable fiber-based artificial muscle. Science 365, 145 (2019)._ _ _ _ _ _ Lee, J. et al. Highly sensitive multifilament fiber strain sensors with ultra-broad sensing range for textile electronics. ACS Nano 12, 4259u20134268 (2018)._ _ _ _ Zhang, M. et al. Printable smart pattern for multifunctional energy-management e-textile. Matter 1, 168u2013179 (2019)._ _ Yuk, H. et al. 3D printing of conducting polymers. Nat. Commun. 11, 1604 (2020)._ _ _ _ _ Zhang, S. et al. Biomimetic spinning of functional soft fibres via spontaneous phase separation. Nat. Electron. 6, 338 (2023).Ling, S. et al. Polymorphic regenerated silk fibers assembled through bioinspired spinning. Nat. Commun. 8, 1387 (2017)._ _ _ _ _ Pan, L. et al. A supertough electro-tendon based on spider silk composites. Nat. Commun. 11, 1332 (2020)._ _ _ _ _ _ Dou, Y. et al. Artificial spider silk from ion-doped and twisted core-sheath hydrogel fibres. Nat. Commun. 10, 5293 (2019)._ _ _ _ _ Loke, G., Yan, W., Khudiyev, T., Noel, G. & Fink, Y. Recent progress and perspectives of thermally drawn multimaterial fiber electronics. Adv. Mater. 32, 1904911 (2020)._ _ _ Wang, C. et al. Ultra-stretchable and fast self-healing ionic hydrogel in cryogenic environments for artificial nerve fiber. Adv. Mater. 34, 2105416 (2022)._ _ _ Cruz, M. V., Shah, D. U., Warner, N. C., McCune, J. A. & Scherman, O. A. Facile, energy-efficient microscale fibrillation of polyacrylamides under ambient conditions. Adv. Mater. 34, 2201577 (2022)._ _ _ Zhao, X. et al. Bioinspired ultra-stretchable and anti-freezing conductive hydrogel fibers with ordered and reversible polymer chain alignment. Nat. Commun. 9, 3579 (2018)._ _ _ _ _ Wu, Y. et al. Bioinspired supramolecular fibers drawn from a multiphase self-assembled hydrogel. Proc. Natl Acad. Sci. USA 114, 8163 (2017)._ _ _ _ _ _ Duan, X. et al. Large-scale spinning approach to engineering knittable hydrogel fiber for soft robots. ACS Nano 14, 14929u201314938 (2020)._ _ _ _ Yu, D. et al. Scalable synthesis of hierarchically structured carbon nanotubeu2013graphene fibres for capacitive energy storage. Nat. Nanotechnol. 9, 555u2013562 (2014)._ _ _ _ _ Wang, X. Q. et al. Macromolecule conformational shaping for extreme mechanical programming of polymorphic hydrogel fibers. Nat. Commun. 13, 3369 (2022)._ _ _ _ _ _ Gao, Z. et al. Graphene reinforced carbon fibers. Sci. Adv. 6, eaaz4191 (2020)._ _ _ _ _ _ Kim, S.-W., Kwon, S.-N. & Na, S.-I. Stretchable and electrically conductive polyurethane- silver/graphene composite fibers prepared by wet-spinning process. Compos. B Eng. 167, 573u2013 581 (2019)._ _ _ Ding, T. et al. Scalable thermoelectric fibers for multifunctional textile-electronics. Nat. Commun. 11, 6006 (2020)._ _ _ _ _ _ Schmuck, B. et al. High-yield production of a super-soluble miniature spidroin for biomimetic high-performance materials. Mater. Today 50, 16u201323 (2021)._ _ _ Holland, C., Vollrath, F., Ryan, A. J. & Mykhaylyk, O. O. Silk and synthetic polymers: reconciling 100 degrees of separation. Adv. Mater. 24, 105u2013109 (2012)._ _ _ _ Zhang, S. et al. In-situ-grown silveru2013polymer framework with coordination complexes for functional artificial tissues. Adv. Mater. (2023).Zhang, S., Hao, A., Liu, Z., Park, J. G. & Liang, R. A highly stretchable polyacrylonitrile elastomer with nanoreservoirs of lubricant using cyano-silver complexes. Nano Lett. 19, 3871u20133877 (2019)._ _ _ _ _ Zhang, S. et al. Silver-coordinated strong, compressible, and ultrafast self-recovery organogel with in-situ electrical conductivity improvement. Adv. Funct. Mater. 33, 2209129 (2023).Ismail, N. et al. Investigating the potential of membranes formed by the vapor induced phase separation process. J. Membr. Sci. 597, 117601 (2020)._ _ _ Matsuyama, H., Teramoto, M., Nakatani, R. & Maki, T. Membrane formation via phase separation induced by penetration of nonsolvent from vapor phase. I. Phase diagram and mass transfer process. J. Appl. Polym. Sci. 74, 159u2013170 (1999)._ _ _ Li, H. & Minus, M. L. On the formation of potential polymer-nanotube blends by liquid-solid phase separation. Polymer 131, 179u2013192 (2017)._ _ _ Cross, M. M. Relation between viscoelasticity and shear-thinning behaviour in liquids. Rheol. Acta 18, 609u2013614 (1979)._ _ _ Stanciu, N. V., Stan, F. & Fetecau, C. Experimental investigation of the melt shear viscosity, specific volume and thermal conductivity of low-density polyethylene/multi-walled carbon nanotube composites using capillary flow. Polymers 12, 1230 (2020)._ _ _ _ _ Pastoriza-Santos, I. & Liz-MarzxE1n, L. M. N,N-dimethylformamide as a reaction medium for metal nanoparticle synthesis. Adv. Funct. Mater. 19, 679u2013688 (2009)._ _ _ Tripathi, A., Rutkeviu010Dius, M., Bose, A., Rojas, O. J. & Khan, S. A. Experimental and predictive description of the morphology of wet-spun fibers. ACS Appl. Polym. Mater. 1, 1280u20131290 (2019)._ _ _ Meijer, H. E. H. & Govaert, L. E. Mechanical performance of polymer systems: the relation between structure and properties. Prog. Polym. Sci. 30, 915u2013938 (2005)._ _ _ Lee, S. et al. A metalu2010like conductive elastomer with a hierarchical wrinkled structure. Adv. Mater. 32, 1906460 (2020)._ _ _ _ Kim, S. H. et al. An ultrastretchable and self-healable nanocomposite conductor enabled by autonomously percolative electrical pathways. ACS Nano 13, 6531u20136539 (2019)._ _ _ _ Zhang, S. et al. Carbon nanotube reinforced strong carbon matrix composites. ACS Nano 14, 9282u20139319 (2020)._ _ _ _ Ning, C. et al. Flexible and stretchable fiber-shaped triboelectric nanogenerators for biomechanical monitoring and human-interactive sensing. Adv. Funct. Mater. 31, 2006679 (2021)._ _ _ Bai, Z. et al. Constructing highly tribopositive elastic yarn through interfacial design and assembly for efficient energy harvesting and human-interactive sensing. Nano Energy 94, 106956 (2022)._ _ _ Chen, J. et al. Flexible hierarchical helical yarn with broad strain range for self-powered motion signal monitoring and human-machine interactive. Nano Energy 80, 105446 (2021)._ _ _ Xie, L. et al. Spiral steel wire based fiber-shaped stretchable and tailorable triboelectric nanogenerator for wearable power source and active gesture sensor. Nano-Micro Lett. 11, 39 (2019)._ _ _ _ \"- - \\'S.Z. acknowledges the support from Yinnan Zhang for schematic drawings and data representations. Prof. Dongsheng Mao is acknowledged for help with writing and revision of the manuscript. This work was financially supported by the Ministry of Education, Singapore, Academic Research Fund, Tier 2, grant number: A-0005415-01-00 (S.C.T.) and the Young Talents Program in Forestry and Grassland Science and Technology Innovation from the Forestry and Grassland Bureau of China, grant number: 2019132612 (W.C.).\\'- - Author notes - \\'These authors contributed equally: Mengjuan Zhou, Mingyang Liu.\\' - Authors and Affiliations - \"Department of Materials Science and Engineering, National University of Singapore, Singapore, 117574, SingaporeSonglin Zhang,_Mengjuan Zhou,_Mingyang Liu,_ Hao Qu_&_Swee Ching TanDepartment of Electrical and Computer Engineering, Center for Intelligent Sensors and MEMS (CISM), NUS Graduate School for Integrative Sciences and Engineering, National University of Singapore, Singapore, 117583, SingaporeZi Hao GuoKey Laboratory of Bio-based Material Science and Technology, Ministry of Education, Northeast Forestry University, 150040, Harbin, P.R. ChinaWenshuai Chen\" - \"AuthorsSonglin ZhangYou can also search for this author in _Mengjuan ZhouYou can also search for this author in _Mingyang LiuYou can also search for this author in _Zi Hao GuoYou can also search for this author in _Hao QuYou can also search for this author in _Wenshuai ChenYou can also search for this author in _Swee Ching TanYou can also search for this author in _\" - Contributions - S.Z. conceived the idea and designed the research. S.C.T., W.C., and S.Z. supervised the project. S.Z., M.L., and M.Z. prepared the fiber samples. S.Z., with the assistance from M.L., M.Z., Z.H.G., and H.Q. conducted the material property characterization and data analysis. S.Z. wrote the original draft with input from all co-authors. All authors provided comments and agreed with the final version of the manuscript. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks Wubin Bai, Sang-Woo Kim, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License; .\" - \"Fig. 2: Th1/Th2 skew in response to SARS-CoV-2 saRNA LNP vaccine.a IgG1 and IgG2a responses in mice vaccinated with doses of LNP-formulated saRNA ranging from 0.01u201310u2009u03BCg of saRNA with nu2009=u20097 biologically independent animals, b Th1/Th2 skewing responses in mice vaccinated with doses of LNP-formulated saRNA ranging from 0.01u201310u2009u03BCg of saRNA with nu2009=u20097 biologically independent animals, and 10u2009u03BCg of electroporated pDNA (EP pDNA) with nu2009=u20098 biologically independent animals. The asterisk (*) indicates significance of pu2009<u20090.05 as determined by a Kruskalu2013Wallis test. Line and error bars indicated meanu2009xB1u2009SD.\" - saRNA LNPs induce neutralization of pseudo- and wild-type SARS-CoV-2 - \"We then sought to characterize how antibodies generated by immunization compared to those generated by a natural SARS-CoV-2 infection as far as capacity to neutralize both a SARS-CoV-2 pseudotyped and wild type virus (Fig._). We observed highly efficient pseudoviral neutralization that varied in a linear dose-dependent manner for the mice vaccinated with saRNA LNP, with IC50 values ranging from 5u2009xD7u200910 to 10 and wild type viral neutralization titers of 80 to 20480. The groups that received 10 or 1u2009u03BCg of saRNA LNP were significantly higher than the electroporated pDNA positive control group, both with pu2009 <u20090.0001 and RABV control for the wild type virus, with pu2009<u20090.0001 and 0.0005, respectively (two-way ANOVA adjusted for multiple comparisons). Comparison to the IC50 values of recovered COVID-19 patients, which had an average IC50 of 10, revealed that even the lowest dose of saRNA LNP (0.01u2009u03BC g) in mice induced higher SARS-CoV-2 neutralization than a natural infection in humans.\" - SARS-CoV-2 IgG correlates directly with viral neutralization - \"We then determined if there is a correlation between the quantity of SARS-CoV-2 specific IgG and SARS-CoV-2 neutralization IC50 for both vaccinated mice and patients who have recovered from COVID-19. Both mice and patients have positive correlations between antibody level and viral neutralization, with Ru2009=u2009 0.88 and 0.87 and pu2009<u20090.0001 and =0.0007 (linear regression), respectively, indicating that high antibody titers enable more efficient viral neutralization. Furthermore, we found that the SARS-CoV-2 specific IgG of LNP-vaccinated mice and recovered COVID-19 patients directly correlated with wild type virus neutralization titers, with Ru2009=u20090.88 and pu2009<u20090.0001 and Ru2009=u2009 0.93 and pu2009=u20090.0019 (linear regression), respectively. No antibody-dependent enhancement (ADE) was observed after vaccination with saRNA LNP (Supplementary Fig._). We also tested the sera of vaccinated mice and recovered patients against other pseudotyped viruses, including SARS-CoV, MERS-CoV and 229E-CoV (Supplementary Fig._), and observed slight neutralization of SARS-CoV by vaccinated mice sera, but otherwise no cross-reactivity.\" - Cellular and cytokine responses to saRNA LNP SARS-CoV-2 vaccine - \"We also characterized the cellular response and induction of systemic cytokines in response to vaccination with saRNA LNP (Fig._). We observed that splenocytes from vaccinated mice re-stimulated with a library of SARS-CoV-2 peptides yielded remarkably high IFN-u03B3 secretion as quantified by ELISpot (Fig._). The saRNA LNP groups that received 0.01u201310u2009u03BCg ranged from 1,000u2013 2,600 SFU per 10 splenocytes, and the 1 and 10u2009u03BCg groups were significantly higher than the EP pDNA positive control group, with pu2009=u20090.0016 and 0.0078 (Kruskalu2013Wallis test), respectively. The re-stimulated splenocyte secretions were also characterized with a panel of cytokines (Supplementary Fig._), with notable increases in GM-CSF, IL-10, IL-12, IL-17a, IL-21, IL-4, IL-5, IL-6, TNF-u03B1, IP-10, MIP-1u03B2 and RANTES.\" - \"Fig. 3: Cellular and secreted cytokine responses to a SARS-CoV-2 saRNA LNP vaccine.a Quantification of IFN-u03B3 splenocytes upon restimulation with SARS-CoV-2 peptides, expressed as spot forming units (SFU) per 10 cells with nu2009=u2009 7 biologically independent animals. Electroporated pDNA (EP pDNA) was used as a positive control while saRNA encoding the rabies glycoprotein (RABV) in pABOL was used as a negative control (RABV control). bu2013g Cytokine profile in sera of mice 4u2009h after vaccination with SARS-CoV-2 LNP vaccine with nu2009 =u20097 biologically independent animals. Remaining cytokines can be found in Supplementary Fig._. The asterisk (*) indicates significance of pu2009<u2009 0.05 as determined by a Kruskalu2013Wallis test. Line and error bars indicated meanu2009xB1u2009SD.\" - \"We further characterized the immune response by assessing the systemic cytokine response 4u2009h after injection with saRNA LNP (Fig._). The groups that received 10 and 1u2009u03BCg of saRNA LNP had enhanced levels of IL-6, MIP-1u03B2 , RANTEs, IFN-u03B2 and IP-10 in the sera compared to the RABV control group, indicating that the LNP formulation enables the immunogenicity of the saRNA. Data from the complete cytokine panel is presented in Supplementary Fig._.\"- - \"Here we characterized the immunogenicity of a SARS-CoV-2 saRNA LNP vaccine compared to the immune response of a natural infection in COVID-19 recovered patients. We observed that two saRNA LNP immunizations induced remarkably high SARS-CoV-2 specific IgG antibodies in mice, with quantities that were superior to both EP pDNA and natural infection in humans, that were able to efficiently neutralize a pseudotyped and wild-type_virus. We also observed that the saRNA LNP vaccine induces a robust cellular response, which is partially enabled by the potent LNP formulation.\" - \"We observed that the saRNA-encoded pre-fusion stabilized spike protein of SARS-CoV-2 used in these studies is highly immunogenic, yielding antibody titers >10u2009 ngu2009mL (Fig._), which is superior to what others have reported for subunit vaccines for the SARS, MERS and SARS-2 coronaviruses. Furthermore, we observed higher antibody titers, viral neutralization (IC50) and cellular response for LNP-formulated saRNA than electroporated pDNA, which we postulate is due to the potent LNP used in these studies, as previous comparisons between polyplex-formulated saRNA and EP pDNA have yielded similar immunogenicity. This is highly useful for translation as it means a potent LNP-formulated saRNA vaccine can be injected with a widely accepted syringe and needle, and does not require electroporation instrumentation, which we envision will enable more widespread vaccination to curb the spread of SARS-CoV-2.\" - The saRNA LNP vaccine presented in these studies elicited robust antibody and cellular responses, with a Th1 bias that we hypothesize will enable immunogenicity in humans. We did not observe any antibody-dependent enhancement (ADE) of SARS-CoV-2 in our in vitro studies, as has been observed for SARS and MERS, but the role of this phenomena in vaccine-induced immunity is not yet fully understood. Overall, we believe that these data inform the antigen design, formulation and preclinical evaluation of immunogenicity that will enable rapid translation of a SARS-CoV-2 vaccine to the clinic trials.- - Vectors - \"We used a plasmid vector to synthesize a self-amplifying RNA (saRNA) replicon, based on a Trinidad donkey Venezuelan equine encephalitis virus strain (VEEV) alphavirus genome (saRNA sequence is detailed in Supplementary Fig._, sequencing primers can be found in Supplementary Table_). The viral structural proteins driven from the sub-genomic promoter were replaced by the surface u2018spike glycoprotein of the novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2): GenBank accession number: QHD43416.1 with some modifications. The pre-fusion state of the spike glycoprotein was stabilized by proline substitutions of K968 and V969. We synthesized oligonucleotide fragments encoding the SARS-CoV-2 gene using GeneArt strings (Thermo Fisher Scientific) and assembled into the plasmid vector with NEB HiFi assembly (New England BioLabs). An expression plasmid expressing the same pre-fusion stabilized full length transmembrane protein used the pcDNA3.1 backbone and was directly synthesized and cloned into the vector by GeneArt (Thermo Fisher Scientific). A plasmid that expressed a soluble pre-fusion version was directly synthesized and cloned into the pcDNA3.1 backbone vector by GeneArt (Thermo Fisher Scientific). This soluble version ends at glutamine Q1208 of the pre-fusion modified QHD43416.1 gene sequence followed by a GGGGSGGGGS linker, a T4 fibritin (foldon) trimerization motif, a further GGGGSGGGGS linker, the Myc tag, a GSGSGS linker and finally an 8xHIS tag to enable purification of the soluble recombinant protein. The RABV control saRNA was based on the Pasteur strain: GenBank accession number: NP_056796.1 with the F318V amino acid substitution to reduce glycoprotein binding to the neurotrophin receptor (p75NTR), a natural ligand.\" - Recombinant soluble SARS-CoV-2 S expression and purification - \"The plasmid expressing the soluble pre-fusion version of SARS-CoV-2 S was used to produce the recombinant protein using the FreeStyleu2122 293 Expression System (Thermo Fisher Scientific), according to the manufacturers instructions. Conditioned medium was clarified by centrifugation and protein was sequentially purified by a HisTrap HP column and a HiPrep 16/60 Sephacryl S-300 HR size exclusion chromatography (SEC) column (both from GE Healthcare). Purified protein was first analyzed by Native-PAGE and Western blot, and then filtered through a 0.22u2009u03BCm membrane, aliquoted and stored at u221280u2009xB0C.\" - In vitro transcription of RNA - \"Self-amplifying RNA encoding the pre-fusion stabilized SARS-CoV-2 was produced using in vitro transcription. pDNA was transformed into E. coli (New England BioLabs, UK), cultured in 100u2009mL of Luria Broth (LB) with 100u2009u03BC gu2009mL carbenicillin (Sigma Aldrich, UK). Plasmid was purified using a Plasmid Plus MaxiPrep kit (QIAGEN, UK) and the concentration and purity was measured on a NanoDrop One (ThermoFisher, UK). pDNA was linearized using MluI for 3u2009 h at 37u2009xB0C. Uncapped in vitro RNA transcripts were produced using 1u2009 u03BCg of linearized DNA template in a MEGAScriptu2122 reaction (Ambion, UK) for 2u2009h at 37u2009xB0C, according to the manufacturers protocol. Transcripts were then purified by overnight LiCl precipitation at u221220u2009 xB0C, centrifuged at 14,000 RPM for 20u2009min at 4u2009xB0C to pellet, washed with 70% EtOH, centrifuged at 14,000 RPM for 5u2009min at 4u2009xB0C and resuspended in UltraPure H2O (Ambion, UK). Purified transcripts were capped using the ScriptCapu2122 Cap 1 Capping System Kit (CellScript, WI, USA) for 2u2009h at 37u2009xB0C, according to the manufacturers protocol. Capped transcripts were purified by LiCl precipitation as described above, resuspended in RNA storage buffer (10u2009mM HEPES, 0.1u2009mM EDTA, and 100u2009mgu2009 mL trehalose) and stored at u221280u2009xB0C until further use.\" - Cell culture & saRNA transfection - \"HEK293T/17 cells (ATCC) and Vero-E6 cells (African green monkey VERO C1008 [Vero 76, clone E6, Vero E6] (ATCCxAE CRL-1586u2122)) were cultured in complete Dulbeccos Modified Eagles Medium (DMEM) (Gibco, Thermo Fisher Scientific) containing 10% fetal bovine serum (FBS, Gibco, Thermo Fisher Scientific), 1% L-glutamine and 1% penicillin-streptomycin (Thermo Fisher Scientific). For Caco2 cells (ATCC) culture medium was modified to include 20% fetal bovine serum. All cells were cultured at 37u2009xB0C, 5% CO2. HEK293T/17 cells (ATCC) were plated in a 12-well plate at a density of 0.75u2009xD7u200910 cells per well 48u2009h prior to transfection. Lipofectamine MessengerMAX (Thermo Fisher Scientific) was used according to the manufacturers instructions for the transfection of SARS-CoV-2 saRNA.\" - Flow cytometry - \"Twenty-four hours post transfection, cells were harvested and resuspended in 1u2009mL of FACS buffer (PBSu2009+u20092.5% FBS) at a concentration of 1u2009 xD7u200910 cells/mL. One hundred microliters of the resuspended cells was added to a FACS tube and stained with 50u2009xB5L of Live/Dead Fixable Aqua Dead Cell Stain (Catalog #L34965, Thermo Fisher Scientific) at a 1:400 dilution on ice for 20u2009min. Cells were then washed with 2.5u2009mL of FACS buffer and centrifuged at 1750 RPM for 7u2009min. After centrifugation, cells were stained with 1u2009xB5g (1:25 dilution) of a SARS-CoV spike protein polyclonal antibody (Catalog #PA1-41165, Thermo Fisher Scientific) for 30u2009min on ice before washing with 2.5u2009mL of FACS buffer and centrifuging at 1750 RPM for 7u2009min. Cells were then stained with 0.4u2009xB5g (1:62.5 dilution) of FITC goat anti-rabbit IgG (Catalog #554020, BD Pharmigen) for 30u2009min on ice. After incubation, cells were washed with 2.5u2009mL of FACS buffer, centrifuged at 1750 RPM for 7u2009min and resuspended with 250u2009xB5L of PBS. Cells were fixed with 250u2009xB5L of 3% paraformaldehyde for a final concentration of 1.5%. Samples were analyzed on a LSRFortessa (BD Biosciences) with FACSDiva software (BD Biosciences). Data were analyzed using FlowJo Version 10 (FlowJo LLC).\" - Formulation of saRNA - \"saRNA was encapsulated in LNP using a self-assembly process in which an aqueous solution of saRNA at pHu2009=u20094.0 is rapidly mixed with an ethanolic lipid mixture. LNP used in this study were similar in composition to those described previously, which contain an ionizable cationic lipid (proprietary to Acuitas)/phosphatidylcholine/cholesterol/PEG-lipid. The proprietary lipid and LNP composition are described in US patent US10,221,127. They had a mean hydrodynamic diameter of ~75u2009nm with a polydispersity index of <0.1 as measured by dynamic light scattering using a Zetasizer Nano ZS (Malvern Instruments Ltd, Malvern, UK) instrument and an encapsulation efficiency of >90%.\" - \"RABV control group was formulated with 8u2009kDa pABOL at a ratio of polymer to RNA of 45:1 (w/w) using the titration method. RNA and pABOL were diluted in HEPES buffer (20u2009mM HEPES, 5 wt.% glucose in water, pH 7.4) at a volume ratio of 4:1 (RNA solution to pABOL solution). The polymer solution was added to a centrifuge tube and place on a stir plate with a stir bar and mixed at 1200 RPM. The RNA solution was then added to the polymer solution at a flow rate of 160u2009xB5Lu2009min.\" - Animals and immunizations - \"BALB/c mice aged 6u20138 weeks old were placed into groups of nu2009=u2009 7 or 8. Animals were handled and procedures were performed in accordance with the terms of a project license granted under the UK Home Office Animals (Scientific Procedures) Act 1986. All the procedures and protocols used in this study were approved by an animal ethical committee, the Animal Welfare and Ethical Review Body (AWERB). Groups of mice were injected intramuscularly (IM; quadriceps) with a 50u2009xB5L of vaccine saRNA formulations. For animals that were vaccinated with pDNA, 10u2009xB5g of pDNA was injected in 50u2009xB5L PBS followed by electroporation (EP) using 5-mm electrodes using an ECM 830 square-wave electroporation system (BTX) (pulses: 100u2009V of positive and negative polarity at 1 pulse s, 50u2009ms pulse). Animals were immunized at week 0, boosted with a second vaccination at week 4 and euthanized using a Schedule 1 method at week 6, at which time the spleens were removed and processed to single cells for use in assays. Serum samples were collected at two-week intervals.\" - Recovered COVID-19 patient samples - \"Serum samples were donated to the Communicable Diseases Research Tissue Bank, Section of Virology, Imperial College London, following written informed consent, by patients who had been infected with SARS-CoV-2. The tissue bank is approved by the National Research Ethics Service, South Central Committee Oxford C (Ref 15/SC/0089). All patients with suspected COVID-19 were attended to at the St. Marys Hospital in London, United Kingdom. All patients had previously tested PCR-positive during their hospital stay, but then were PCR-negative at the time of sampling.\" - \"IFN-u03B3 ELISpots\" - \"Assessment of the IFN-u03B3 T cell response was performed using the Mouse IFN-u03B3 ELISpot kit (Mabtech) following the manufacturers instructions. Briefly, anti-IFN-u03B3 pre-coated plates were blocked with DMEMu2009+u200910% FBS for at least 30u2009min, then cells were added at 2.5u2009xD7u200910 cells per well for negative control (media only) and SARS-CoV-2 peptide pools (15-mers overlapping by 11; JPT Peptides) (1u2009xB5gu2009mL) in 200u2009xB5L final volume per well. The positive control wells contained 5u2009xD7u200910 cells per well in 200u2009xB5L final volume per well with 5u2009xB5gu2009mL of ConA. Plates were incubated overnight at 5% CO2, 37u2009xB0C incubator and developed as per the manufacturers protocol. Once dried, plates were read using the AID ELISpot reader ELR03 and AID ELISpot READER software (Autoimmun Diagnostika GmbH).\" - Antigen-specific IgG ELISA - \"The antigen-specific IgG, IgG1 and IgG2a titres in mouse sera were assessed by a semi-quantative ELISA. MaxiSorp high binding ELISA plates (Nunc) were coated with 100u2009u03BCL per well of 1u2009u03BCgu2009mL recombinant SARS-CoV-2 protein with the pre-fusion stabilized conformation in PBS. For the standard IgG/IgG1/IgG2a, 3 columns on each plate were coated with 1:1000 dilution each of goat anti-mouse Kappa (Catalog #1050-01, Southern Biotech) and Lambda light chains (Catalog #1060-01, Southern Biotech). After overnight incubation at 4u2009 xB0C, the plates were washed 4 times with PBS-Tween 20 0.05% (v/v) and blocked for 1u2009h at 37u2009xB0C with 200u2009u03BCL per well blocking buffer (1% BSA (w/v) in PBS-Tween-20 0.05%(v/v)). The plates were then washed and the diluted samples or a 5-fold dilution series of the standard IgG (or IgG1 or IgG2) added using 50u2009u03BCL per well volume. Plates were incubated for 1u2009h at 37u2009xB0C, then washed and secondary antibody added at 1:2000 dilution in blocking buffer (100u2009u03BCL per well) using either anti-mouse IgG-HRP (Catalog #1030-05, Southern Biotech), anti-mouse IgG1-HRP (Catalog #1070-05, Southern Biotech) or anti-mouse IgG2a-HRP (Catalog #1080-05, Southern Biotech). After incubation and washes, plates were developed using 50u2009u03BCL per well SureBlue TMB (3,3u2032, 5,5u2032-tetramethylbenzidine) substrate and the reaction stopped after 5u2009min with 50u2009u03BCL per well stop solution (Insight Biotechnologies). The absorbance was read on a Versamax Spectrophotometer at 450u2009nm (BioTek Industries).\" - Pseudotype virus neutralization assay - \"A HIV-pseudotyped luciferase-reporter based system was used to assess the neutralization ability of sera from vaccinated animals and recovered patients against SARS-CoV, SARS-CoV-2, MERS-CoV and 229E-CoV. In brief, CoV S-pseudotyped viruses were produced by co-transfection of 293T/17 cells with a HIV-1 gag-pol plasmid (pCMV-u0394 8.91, a kind gift from Prof. Julian Ma, St Georges University of London), a firefly luciferase reporter plasmid (pCSFLW, a kind gift from Prof. Julian Ma, St Georges University of London) and a plasmid encoding the S protein of interest (pSARS-CoV-S, pSARS-CoV2-S, pMERS-CoV-S or p229E-CoV-S) at a ratio of 1:1.5:1. Virus-containing medium was clarified by centrifugation and filtered through a 0.45u2009u03BCm membrane 72u2009h after transfection, and subsequently aliquoted and stored at u221280u2009xB0C. For the neutralization assay, heat-inactivated sera were first serially diluted and incubated with virus for 1u2009h, and then the serum-virus mixture was transferred into wells pre-seeded Caco2 cells. After 48u2009h, cells were lysed, and luciferase activity was measured using Bright-Glo Luciferase Assay System (Promega). The IC50 neutralization was then calculated using GraphPad Prism (version 8.4). We analyzed the pseudovirus neutralization data titration curves looking for any antibody-dependent enhancement (ADE) of infection, to determine if we could observe an altered elevated baseline level in the antibody treated wells when compared to either the control or virus alone conditions.\" - Wild-type viral neutralization assay - \"The ability of sera from vaccinated animals to neutralize wild type SARS-CoV-2 virus was assessed by neutralization assay on Vero-E6 cells. SARS-CoV-2/England/IC19/2020 was isolated on Caco2 cells from a clinical sample collected from a patient admitted to St. Marys Hospital in London, United Kingdom. Heat-inactivated sera were serially diluted in assay diluent consisting of DMEM (Gibco, Thermo Fisher Scientific) with 1% penicillin-streptomycin (Thermo Fisher Scientific), 0.3% BSA fraction V (Thermo Fisher Scientific) and 0.25u2009xB5gu2009mL TPCK trypsin (Worthington). Serum dilutions were incubated with 100 TCID50 per well of SARS-CoV-2/England/IC19/2020 diluted in assay diluent for 1u2009h at RT and transferred to 96-well plates pre-seeded with Vero-E6 cells. Serum dilutions were performed in duplicate. Plates were incubated at 37u2009xB0C, 5% CO2 for 5 days before adding an equal volume of 2X crystal violet stain to wells for 1u2009h. Plates were washed, wells were scored for cytopathic effect and a neutralization titer calculated as the reciprocal of the highest serum dilution at which full virus neutralization occurred.\" - Cytokine measurement in splenocytes and sera - \"Splenoctyes isolated from each individual mouse were plated into round bottom 96 well plates (1u2009xD7u200910 per well in a 200u2009uL total volume) and cultured for 7 days with media alone, 5u2009xB5gu2009mL SARS-CoV-2 recombinant protein or 5u2009xB5gu2009mL ConA as a positive control. For the sera samples, mice were bled 4u2009h after injection with SARS-CoV-2 LNP vaccine or control RABV vaccine and sera were collected. The cytokine response in each well was quantified with a custom 25-plex ProcartaPlex Immunoassay (ThermoFisher Scientific, UK) on a Bio-Plex 200 System (Bio-Rad), according to the manufacturer s instructions.\" - Statistical analysis - \"Graphs and statistics were prepared in GraphPad Prism (version 8.4). Statistical differences were analyzed using either a two-way ANOVA adjusted for multiple comparisons or a Kruskalu2013Wallis test adjusted for multiple comparisons, with pu2009<u20090.05 used to indicate significance.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - \"The data that support the findings of this study are available within the article and its Supplementary Information files, or are available from the corresponding author upon reasonable request._ are provided with this paper.\" - \\' \\'- - \"Organization, W. H. Global COVID-19 statistics. (2020).Ksiazek, T. G. et al. A novel coronavirus associated with severe acute respiratory syndrome. N. Engl. J. Med. 348, 1953u20131966 (2003)._ _ _ Zaki, A. M., van Boheemen, S., Bestebroer, T. M., Osterhaus, A. D. M. E. & Fouchier, R. A. M. Isolation of a novel coronavirus from a man with pneumonia in Saudi Arabia. N. Engl. J. Med. 367, 1814u20131820 (2012)._ _ _ Al-Omari, A., Rabaan, A. A., Salih, S., Al-Tawfiq, J. A. & Memish, Z. A. MERS coronavirus outbreak: Implications for emerging viral infections. Diagnostic Microbiol. Infect. Dis. 93, 265u2013285 (2019)._ _ Kirchdoerfer, R. N. et al. Pre-fusion structure of a human coronavirus spike protein. Nature 531, 118u2013121 (2016)._ _ _ _ Perri, S. et al. An alphavirus replicon particle chimera derived from venezuelan equine encephalitis and sindbis viruses is a potent gene-based vaccine delivery vector. J. Virol. 77, 10394u201310403 (2003)._ _ _ Blakney, A. K., McKay, P. F., Yus, B. I., Aldon, Y. & Shattock, R. J. Inside out: optimization of lipid nanoparticle formulations for exterior complexation and in vivo delivery of saRNA. Gene Ther._26, 363u2013 372_(2019)._ _ _ Blakney, A. K. et al. Big is Beautiful: Enhanced saRNA delivery and immunogenicity by a higher molecular weight, bioreducible, cationic polymer. ACS Nano14, 5711u20135727_(2020)._ _ _ Geall, A. J. et al. Nonviral delivery of self-amplifying RNA vaccines. Proc. Natl Acad. Sci. USA 109, 14604u201314609 (2012)._ _ _ _ Vogel, A. B. et al. Self-amplifying RNA vaccines give equivalent protection against influenza to mRNA vaccines but at much lower doses. Mol. Ther. 26, 446u2013455 (2018)._ _ _ Garber, K. Alnylam launches era of RNAi drugs. Nat. Biotechnol. 36, 777u2013778 (2018)._ _ _ Kim, E. et al. Microneedle array delivered recombinant coronavirus vaccines: Immunogenicity and rapid translational development. EBioMedicine 55, 102743 (2020)._ _ Tetro, J. A. Is COVID-19 receiving ADE from other coronaviruses? Microbes Infect. 22, 72u201373 (2020)._ _ _ Wang, S.-F. et al. Antibody-dependent SARS coronavirus infection is mediated by antibodies against spike proteins. Biochem. Biophys. Res. Commun. 451, 208u2013214 (2014)._ _ _ Wu, F. et al. A new coronavirus associated with human respiratory disease in China. Nature 579, 265u2013269 (2020)._ _ _ _ Kirchdoerfer, R. N. et al. Stabilized coronavirus spikes are resistant to conformational changes induced by receptor recognition or proteolysis. Sci. Rep. 8, 15701u2013 15701 (2018)._ _ _ Pardi, N. et al. Expression kinetics of nucleoside-modified mRNA delivered in lipid nanoparticles to mice by various routes. J. Control. Release 217, 345u2013351_(2015)._ _ _ Maier, M. A. et al. Biodegradable lipids enabling rapidly eliminated lipid nanoparticles for systemic delivery of RNAi therapeutics. Mol. Ther. 21, 1570u20131578 (2013)._ _ _ Jayaraman, M. et al. Maximizing the potency of siRNA lipid nanoparticles for hepatic gene silencing in vivo. Angew. Chem. 51, 8529u20138533 (2012)._ _ _ Badamchi-Zadeh, A. et al. Intramuscular immunisation with chlamydial proteins induces Chlamydia trachomatis specific ocular antibodies. PLoS ONE 10, e0141209 (2015)._ _ Wright, E. et al. Investigating antibody neutralization of lyssaviruses using lentiviral pseudotypes: a cross-species comparison. J. Gen. Virol. 89, 2204u20132213 (2008)._ _ _ Hu, K. et al. CCL19 and CCL28 augment mucosal and systemic immune responses to HIV-1 gp140 by mobilizing responsive immunocytes into secondary lymph nodes and mucosal tissue. J. Immunol. 191, 1935 (2013)._ _ _ \"- - \"We gratefully acknowledge Graham Taylor, Graham Cooke, Rachael Quinlan, Charlotte Short, and Carolina Rosadas de Oliveira for providing the samples from recovered COVID-19 patients, and Jonathan Yeow for providing the polymers used in these studies. We thank Andrew Davidson and David Matthews, University of Bristol UK for supplying the VeroE6 cells used in this study. AKB is supported by a Marie Sku0142odowska Curie Individual Fellowship funded by the European Commission H2020 (No. 794059). P.F.M., K.H., K.S., C.R.B., K.P., P.R. and R.J.S. are funded by the Department of Health and Social Care using UK Aid funding and is managed by the Engineering and Physical Sciences Research Council (EPSRC, grant number: EP/R013764/1, note: the views expressed in this publication are those of the author(s) and not necessarily those of the Department of Health and Social Care). This work was supported in part by the NIHR Biomedical Research Centre of Imperial College Healthcare NHS Trust. We also acknowledge Dormeur Investment Services Ltd for providing funds to purchase equipment used in these studies.\"- - Authors and Affiliations - \"Department of Infectious Diseases, Imperial College London, Norfolk Place, London, W2 1PG, UKPaul F. McKay,_Kai Hu,_Anna K. Blakney,_Karnyart Samnuan,_Jonathan C. Brown,_Rebecca Penn,_Jie Zhou,_ClxE9ment R. Bouton,_Paul Rogers,_Krunal Polra,_Wendy S. Barclay_&_Robin J. ShattockAcuitas Therapeutics, Vancouver, BC, V6T 1Z3, CanadaPaulo J. C. Lin,_Christopher Barbosa_&_Ying K. Tam\" - \"AuthorsPaul F. McKayYou can also search for this author in _Kai HuYou can also search for this author in _Anna K. BlakneyYou can also search for this author in _Karnyart SamnuanYou can also search for this author in _Jonathan C. BrownYou can also search for this author in _Rebecca PennYou can also search for this author in _Jie ZhouYou can also search for this author in _ClxE9ment R. BoutonYou can also search for this author in _Paul RogersYou can also search for this author in _Krunal PolraYou can also search for this author in _Paulo J. C. LinYou can also search for this author in _Christopher BarbosaYou can also search for this author in _Ying K. TamYou can also search for this author in _Wendy S. BarclayYou can also search for this author in _Robin J. ShattockYou can also search for this author in _\" - Contributions - P.F.M. and R.J.S. conceptualized the antigen design and designed the studies. P.F.M., K.S. and K.H. designed and performed in vitro experiments. J.C.B., R.P., J.Z. and W.S.B. conceived of and performed in vitro wild type viral neutralization assays. P.F.M., A.K.B., K.H. and K.S. performed in vivo studies, aided by C.R.B., K.P., and P.R. P.J.C.L., C.B. and Y.K.T. designed and prepared the saRNA LNP. A.K.B. analysed the data and wrote the manuscript with help from P.F.M., K.H. and K.S., and constructive feedback and editing from R.P., J.B., C.R.B., K.P., P.R., W.B. and R.J.S. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - P.F.M. and R.J.S. are co-inventors on a patent resulting from this work. P.J.C.L., C.B. and Y.K.T. are employed by Acuitas Therapeutics. - \\' \\'- - Peer review information Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License; .\" - \"In order to test the role of endothelial GR in nondiabetic fibrosis, UUO in 8-week-old GR and control littermates was performed (Supplementary Fig._). There was no significant difference in renal fibrosis between contralateral kidneys of controls and GR mice. However, UUO kidneys from GR mice showed a greater relative area of fibrosis and greater collagen deposition when compared to UUO kidneys of littermate controls (Supplementary Fig._). Immunofluorescence staining revealed higher collagen I, u03B1SMA, and fibronectin deposition in the UUO kidneys of GR when compared to UUO kidneys of control littermates (Supplementary Fig._ ). Administration of the GR agonist dexamethasone abolished the renal fibrogenic phenotype in C57BL/6 mice subjected to UUO by day 10 (Supplementary Fig._). Dexamethasone treatment slightly suppressed the renal fibrosis in contralateral kidneys of C57BL/6 mice at day 15 but did not cause any remarkable alteration in nondiabetic control CD-1 mice (Supplementary Fig._).\" - Endothelial GR loss reprograms cytokine and chemokine homeostasis - \"Inflammation is a key factor during the fibroblast activation process in the kidneys of diabetic mice and disruption of cytokine and chemokine homeostasis contributes to the development of diabetic kidney disease. To investigate whether there were derangements in homeostasis in our model, cytokine analysis in the plasma of diabetic mice with severe fibrosis (diabetic CD-1) and the plasma of diabetic mice with less severe fibrosis (diabetic C57BL/6) was performed. Diabetic CD-1 mice demonstrated higher levels of plasma IL-1u03B2, IL-6, IL-10, IL-17, G-CSF, IFN-u03B3, TNF-u03B1, MCP-1, CCL3, and CCL4 levels, however, the CCL5 level was suppressed when compared to that of diabetic C57BL/6 mice (Supplementary Fig._). The same cytokines were also analyzed in the plasma from diabetic and nondiabetic GR mice and littermate controls and diabetic and nondiabetic DKO and Apoe controls. A similar pattern was observed in both genotypes in which IL-1u03B2, IL-6, IL-10, eotaxin, G-CSF, and CCL4 were significantly higher, while CCL5 was significantly lower, in the plasma of diabetic GR and diabetic DKO mice when compared to the plasma of their respective diabetic control littermates (Fig._ and Supplementary Fig._). Similarly, mRNA gene expression analysis demonstrated that the level of IL-1u03B2, IL-6, IL-10, IL-17, eotaxin, and CCL4 was significantly upregulated, while CCL5 was significantly downregulated, in the kidneys of diabetic GR and diabetic DKO mice when compared to the diabetic kidneys of their respective control littermates (Supplementary Fig._), indicating more EC inflammation in mice lacking endothelial GR.\" - \"Fig. 3: IL-6 neutralization rescues renal fibrosis in diabetes.a Plasma IL-6 levels in nondiabetic and diabetic mice were measured by cytokine array (Luminex). Nu2009=u20095 biologically independent mice/group. b Measurement of IL-6 in media of cultured endothelial cells isolated from the indicated groups. Endothelial cells were isolated and cultured from six biologically independent mice per group and were analyzed in triplicate. Data are meanu2009xB1u2009SEM. c IL-6 neutralization in diabetic GR and diabetic DKO mice. Representative images are shown; original magnification xD7300. Relative area of fibrosis (%) and relative collagen (%) were measured using the ImageJ program. Nu2009=u20097 biologically independent mice/group. Scale bar 50u2009u03BCm. Data are meanu2009xB1u2009 SEM. For all panels, one-way ANOVA with Tukey post-test was used for the analysis of statistical significance. *pu2009<u20090.05, **pu2009<u20090.01, ***pu2009 <u20090.001. Source data are provided in the source data file.\" - \"Stimulation of isolated kidney ECs from CD-1 mice with recombinant IL-1u03B2 , IL-6, TNFu03B1, and TGFu03B2 caused a significant increase in expression of the mesenchymal markers vimentin and u03B1SMA (Supplementary Fig._); neutralization of IL-6 using IL-6 neutralizing antibody could suppress vimentin expression in TGFu03B22-treated and untreated cells (Supplementary Fig._). To further investigate the role of IL-6 in diabetes-associated renal fibrosis, we injected (IP) IL-6 neutralizing antibody in nondiabetic and diabetic CD-1 mice. In nondiabetic mice, IL-6 neutralization did not cause any significant difference; however, in diabetic mice, IL-6 neutralization resulted in a significant reduction of fibrosis and collagen deposition (Supplementary Fig._). In addition, isolated cells from diabetic GR and DKO mice had higher IL-6 levels in culture media when compared to isolated cells from their respective diabetic control littermates (Fig._). When IL-6 neutralization was performed in nondiabetic and diabetic GR and DKO mice, it completely reversed the fibrogenic phenotype in diabetic animals when compared to respective IgG control-injected mice (Fig._). There was no significant effect observed in nondiabetic mice (Supplementary Fig._ ).\" - Canonical Wnt signaling is a new drug target for the action of endothelial GR - \"Given the recently described regulation of Wnt signaling by endothelial GR as well as the recognized role of Wnt signaling in renal fibrosis, we assessed the mRNA expression of Wnt-dependent genes and fibrogenic markers in ECs isolated from the kidneys of diabetic GR and diabetic DKO mice and their diabetic littermate controls. The expression of Wnt-dependent genes and fibrogenic markers was upregulated in kidneys of diabetic GR and diabetic DKO when compared to their respective controls. However, the kidneys of diabetic DKO mice showed the highest expression of both Wnt-dependent genes, such as axin2 and tcf, and fibrogenic markers, such as u03B1SMA and fibronectin (Supplementary Fig._). The expression of HIF1u03B1 and Snaill1 was upregulated, while PPARu03B1 expression was downregulated in ECs isolated from the kidneys of diabetic GR and diabetic DKO mice when compared to diabetic controls (Supplementary Fig._). In ECs isolated from cultured glomeruli, the expression of Wnt-dependent genes axin2 and tcf and fibronectin was upregulated in those from the kidneys of diabetic GR mice compared to littermate control mice and nondiabetic mice (Supplementary Fig._). Protein expression of u03B2 -catenin was also higher in the interstitial and glomerular compartments of diabetic GR and diabetic DKO compared to their respective controls (Supplementary Fig._ ). Western blotting demonstrated higher protein expression of vimentin, HIF1u03B1 , snail1, and active u03B2-catenin and decreased levels of PPARu03B1 and CPT1a in isolated ECs from diabetic GR when compared to diabetic control littermates (Fig._). Isolated ECs from diabetic DKO mice exhibited higher fibronectin, u03B1SMA, and active u03B2-catenin levels when compared to littermate Apoe and diabetic GR mice (Fig._).\" - \"Fig. 4: Upregulation of Wnt signaling and fibrogenic markers with loss of endothelial GR.a Western blot analysis of vimentin, HIF1u03B1, Snail1, active u03B2-catenin, CPT1a, and PPARu03B1 in isolated endothelial cells from the kidneys of nondiabetic and diabetic control and GR mice. Endothelial cells were isolated and cultured from six biologically independent mice/group. Densitometry calculations from six samples/group combined from three independent experiments are shown and are normalized to u03B2-actin or HSP90 as indicated. Representative blots are shown. Data are meanu2009xB1u2009SEM. b Western blot analysis of u03B1SMA, fibronectin, active u03B2-catenin and u03B2-catenin in isolated endothelial cells from the kidneys of diabetic control, GR, Apoe, and DKO mice. Endothelial cells were isolated and cultured from six biologically independent mice/group. Densitometry calculations from six samples/group combined from three independent experiments are shown and are normalized to u03B2-actin or HSP90 as indicated. Data are meanu2009xB1u2009SEM. Representative blots are shown. c Immunofluorescence analysis of u03B1SMA/CD31 and TGFu03B2R1/CD31 was performed in the kidneys of nondiabetic and diabetic control, GR, Apoe, and DKO mice. FITC-labeled u03B1 SMA, FITC-labeled TGFu03B2R1, rhodamine-labeled CD31 and DAPI (nuclei, blue) were used. Representative merged images are shown; original magnification xD7 300. Scale bar 50u2009u03BCm. Nu2009=u20097 biologically independent mice/group. Data are meanu2009xB1u2009SEM. For all panels, one-way ANOVA with Tukey post-test was used for the analysis of statistical significance. Significance *pu2009 <u20090.05, **pu2009<u20090.01, ***pu2009<u20090.001. Source data are provided in the source data file.\" - \"Using immunofluorescent co-staining, the same pattern was also observed, with diabetic GR and diabetic DKO mice demonstrating higher levels of u03B1SMA/CD31 and TGFu03B2R1/CD31 co-staining in the kidneys when compared to their respective controls, suggestive of EndMT (Fig._). There was no difference in the EC density of ECs from the kidneys of diabetic GR and diabetic DKO when compared to control ECs; however, EC permeability, as demonstrated by FITC-dextran staining, was higher in kidneys and isolated cultured ECs from diabetic GR and diabetic DKO when compared to their respective control littermates (Supplementary Fig._ ).\" - Inhibition of canonical Wnt signaling improves renal fibrosis - \"Diabetic CD-1 mouse kidneys displayed significantly higher expression of u03B2 -catenin, a marker of canonical Wnt signaling, compared to those from diabetic C57BL/6 mice as assessed by immunohistochemical staining (Supplementary Fig._ ). The kidneys of diabetic CD-1 mice also showed dramatically higher mRNA expression of Wnt-dependent genes and fibronectin when compared to those of the diabetic C57BL/6 mice and the nondiabetic controls of both genotypes (Supplementary Fig._ ). To determine whether inhibition of the Wnt signaling pathway could ameliorate the observed fibrosis, we utilized LGK974, a small molecule inhibitor of all secreted Wnts. Supplementary Fig._ depicts the schematic diagram showing the experimental protocol for LGK974 treatment in diabetic CD-1 and UUO mice. LGK974 greatly diminished the ECM deposition, relative area of fibrosis, collagen accumulation, and glomerulosclerosis in both models used (Supplementary Fig._). Wnt inhibition also substantially restored endothelial GR and suppressed the level of u03B2 -catenin in diabetic and UUO mice (Supplementary Fig._). In diabetic CD-1 mice, LGK974 also significantly suppressed elevated levels of IL-1u03B2, IL-6, IL-10, G-CSF, TNFu03B1, MCP-1, and CCL4 while increasing the level of CCL5 (Supplementary Fig._).\" - \\'Wnt inhibitor partially suppresses the fibrogenic phenotype in the kidneys of diabetic GR \\' - \"To determine whether Wnt inhibition could mitigate the renal fibrosis observed in diabetic GR mice, a cohort of animals was treated with LGK974. At the age of 8 weeks, control and GR mice were injected with STZ 50u2009mg/kg for 5 consecutive days. Sixteen weeks after injection, LGK974 was administered by oral gavage for 8 additional weeks (Fig._). At the time of sacrifice, there were no significant differences in body weight or blood glucose among the groups (Fig._ and Supplementary Fig._). However, a significant reduction in kidney weight was observed in the Wnt inhibitor-treated diabetic GR and diabetic control mice (Fig._). Wnt inhibition clearly improved the relative area of fibrosis, relative collagen deposition, and tubular damage in diabetic control mice; this effect was less pronounced, though still significant in diabetic GR mice (Fig._). Wnt inhibition did not result in any remarkable change in the renal fibrogenic phenotype or u03B2 -catenin level in nondiabetic control mice or nondiabetic GR when compared to their untreated nondiabetic controls (Supplementary Fig._). Wnt inhibition suppressed the expression of Snail1 and HIF1u03B1 and significantly increased the level of PPARu03B1 in the kidneys of diabetic GR (Supplementary Fig._ ). A similar effect was observed in isolated ECs (Supplementary Fig._). Wnt inhibition also significantly suppressed EndMT (CD31/u03B1SMA double-positive cells) in diabetic control mice; this effect was less pronounced in diabetic GR mice (Supplementary Fig._). However, Wnt inhibition significantly reduced the level of EMT (E-cadherin/u03B1 SMA double-positive cells) in control and diabetic GR mice (Supplementary Fig._ ).\" - \"Fig. 5: Wnt inhibitor partially abrogates renal fibrosis in diabetic GR and DKO mice.a Schematic diagram showing the treatment protocol of Wnt inhibition in diabetic control and GR mice. b Physiological parameters including body weight, blood glucose, and kidney weight/body weight were analyzed. Nu2009=u20096 biologically independent mice/group combined from two experiments. c Masson trichrome, Sirius red, and PAS staining as well as immunohistochemical analysis of u03B2-catenin levels in the kidneys of wnti-treated diabetic control and GR mice. Representative images are shown; original magnification xD7300. Relative area of fibrosis (%) and relative collagen deposition (%) were measured using the ImageJ program. Nu2009=u20098 biologically independent mice/group combined from three experiments were analyzed. Scale bar 50u2009u03BCm. Data are meanu2009 xB1u2009SEM. For all panels, one-way ANOVA with Tukey post-test was used for the analysis of statistical significance. Significance *pu2009<u20090.05, **pu2009<u20090.01, ***pu2009<u20090.001. Source data are provided as source data file. Components of this figure were created using Servier Medical Art templates, which are licensed under a Creative Commons Attribution 3.0 Unported License; .\" - Metabolic reprogramming by loss of endothelial GR accelerates renal fibrosis - \"It is increasingly recognized that defects in central metabolism contribute to kidney fibrosis. Defective FA metabolism in ECs leads to EndMT events. To investigate whether FA metabolism was deranged in our model, radiolabeled [C]palmitate uptake experiments in isolated ECs from mouse kidneys were performed. We observed that FA uptake was higher in isolated ECs from diabetic kidneys of the more fibrotic strain (diabetic CD-1) when compared to kidney ECs from the less-fibrotic strain (diabetic C57BL/6). Administration of the Wnt inhibitor suppressed FA uptake in ECs from CD-1 mice (Supplementary Fig._). Kidney ECs from both diabetic GR and DKO mice displayed higher FA uptake when compared to those of the diabetic control littermates (Fig._). FAO was also assessed by measuring the [CO2]release from radiolabeled [C]palmitate in cultured ECs isolated from kidneys. FAO was diminished in the isolated kidney ECs of diabetic CD-1 mice and Wnt inhibition was able to restore the level of FAO (Supplementary Fig._). Cultured kidney ECs from diabetic GR and diabetic DKO mice showed a diminished level of FAO when compared to their diabetic control littermates (Fig._). In addition, gene expression of FA transporter proteins 1 and 4 (FATP1 and FATP4) were unaltered in both nondiabetic and diabetic ECs isolated from kidneys of control and GR mice; however, the gene expression level of CD36, another FA transporter, was lower in diabetic mice of both genotypes when compared to nondiabetic controls (Supplementary Fig._).\" - \"Fig. 6: Metabolic reprogramming by loss of endothelial GR loss worsens diabetic kidney disease.a Radiolabeled C-palmitate uptake analysis in isolated endothelial cells from kidneys of nondiabetic and diabetic control, GR, Apoe, and DKO mice. CPM of each sample were counted. Data were normalized to u03BCg protein. Fold activity is presented. Endothelial cells isolated and cultured from six biologically independent mice/group were performed in triplicate and analyzed for C-palmitate uptake analysis. Data are meanu2009xB1u2009SEM. b Radiolabeled [C]palmitate oxidation and [CO2]release were measured in isolated endothelial cells from kidneys of nondiabetic and diabetic control, GR, Apoe, and DKO mice. CPM of each sample was counted. Data were normalized to u03BCg protein. Fold activity is presented. Endothelial cells isolated and cultured from six biologically independent mice/group were performed in triplicate and analyzed for C-palmitate uptake analysis. Data are meanu2009xB1u2009SEM. c Body weight, blood glucose, and kidney weight/body weight were measured at the end of these experiments. Nu2009=u20096 biologically independent mice from two experiments. Ex vivo radiolabeled [C]palmitate oxidation and [CO2]release were measured in kidneys. CPM of each sample was counted. Isolated endothelial cells from five biologically independent mice/group were analyzed in triplicate. Data are meanu2009xB1u2009 SEM. d Masson trichrome, Sirius red, and PAS staining were analyzed in the kidneys of control-, etomoxir-, and C75-treated diabetic mice. Representative images are shown; original magnification xD7300. Relative area of fibrosis (%) and relative collagen deposition (%) were measured using the ImageJ program. Nu2009 =u20098 biologically independent mice per/group from three independent experiments were analyzed. Scale bar 50u2009u03BCm. Data are meanu2009xB1u2009SEM. e Immunohistochemical analysis of CPT1a and u03B2-catenin. Representative images are shown; original magnification xD7300. Scale bar 50u2009u03BCm. Nu2009 =u20096 biologically independent mice per group from three independent experiments. For all panels one-way ANOVA with Tukey post-test was used for the analysis of statistical significance. Significance *pu2009<u20090.05, **pu2009<u2009 0.01, ***pu2009<u20090.001. Source data are provided in the source data file.\" - \"In the next set of experiments, diabetic CD-1 mice were treated with the FA synthase inhibitor C75, the FAO inhibitor etomoxir, the PPARu03B1 agonist fenofibrate, or the cholesterol-lowering drug simvastatin for 4 weeks. Fenofibrate and C75 ameliorated the fibrogenic phenotype, whereas etomoxir exacerbated the fibrosis. Simvastatin treatment did not cause significant suppression in the level of fibrosis (Supplementary Fig._). Fenofibrate and C75 restored the level of GR protein in CD31 positive cells, whereas etomoxir suppressed it and simvastatin did not show any substantial effect (Supplementary Fig._). Fenofibrate and C75 downregulated fibronectin and u03B1SMA mRNA levels, while etomoxir upregulated this mRNA and simvastatin did not cause any significant change in the gene expression level of fibronectin or u03B1SMA in the diabetic kidneys (Supplementary Fig._ ). These FA modulators did not cause any significant differences in the blood glucose levels (Supplementary Fig._). Etomoxir treatment caused significant suppression of FAO, as measured by [CO2]release, and CPT1a level, and induced the expression of u03B2-catenin, whereas C75 and fenofibrate increased the level of FAO, increased the expression of CPT1a, and suppressed the level of u03B2-catenin in diabetic CD-1 mice (Supplementary Fig._).\" - \"Etomoxir and C75 were also tested in nondiabetic and diabetic control littermates and GR mice (Fig._ and Supplementary Figs._ and ). There were no significant differences in body weight, blood glucose, or kidney weight in nondiabetic or diabetic control littermates and GR mice after treatment with etomoxir or C75 (Fig._ and Supplementary Fig._). Data from kidney ECs revealed that etomoxir caused significant suppression of FAO, and C75 restored FAO in nondiabetic control littermates (Supplementary Fig._). However, in kidney ECs from diabetic GR mice, etomoxir caused significant suppression of FAO which C75 was unable to rescue (Fig._). Etomoxir treatment accelerated the renal fibrogenic phenotype, suppressed the CPT1a level and increased the expression level of u03B2-catenin in the kidneys of diabetic control and diabetic GR mice. C75 treatment clearly abolished the renal fibrogenic phenotype, restored CPT1a, and completely diminished the level of u03B2-catenin in the kidneys of diabetic control mice. These effects were also observed in the GR mice, though to a lesser extent (Fig._). Etomoxir slightly increased the level of renal fibrosis in nondiabetic control and GR mice, which could be mitigated by C75 (Supplementary Fig._). Etomoxir decreased expression of CPT1a in diabetic control mice while also increasing expression of u03B2-catenin; C75 was able to substantially reverse both of these effects. A similar pattern was observed in diabetic GR mice, though again, to a lesser extent (Fig._).\" - GR loss-linked EndMT disrupts central metabolism and induces mesenchymal transformation in tubular epithelial cells - \"These in vivo data suggest that GR mice exhibit enhanced EndMT and EMT in their diabetic kidneys. Overexpression of GR suppressed the TGFu03B22-stimulated increase in u03B1SMA, collagen I, and u03B2-catenin expression in HUVECs (Supplementary Fig._). To test whether endothelial GR deficiency affects mesenchymal programs and causes defects in central metabolism in neighboring epithelial cells, we analyzed the effects of conditioned media (CM) from GR knockdown HUVECs on the mesenchymal phenotype of HK-2 cells (Fig._). CM from GR siRNA-transfected HUVECs decreased E-cadherin protein levels and increased u03B1SMA, TGFu03B2R1, and u03B2-catenin protein levels in HK-2 cells when compared to media from scrambled siRNA-transfected HUVECs (Fig._). CM treatment from GR siRNA-transfected HUVECs caused a significant reduction in the level of FAO, oxygen consumption rate (OCR), and cellular ATP level in HK-2 cells (Fig._). TECs incubated with CM from GR siRNA-treated HUVECs exhibited significantly downregulated mRNA expression of the FAO-responsive genes Cpt1a, Cpt2, Pparu03B1, and Pgc1u03B1 compared to TECs incubated with CM from control siRNA-treated HUVECs (Fig._).\" - \"Fig. 7: GR loss in endothelial cells reprograms central metabolism in renal tubular cells.a Conditioned media experimental design. HUVECs were transfected with scrambled or GR siRNA; after 6u2009h, the medium was changed and cells were incubated for 96u2009h. The subsequently harvested media was transferred to HK-2 cells. b Representative Western blotting analysis of E-cadherin, u03B1 SMA, TGFu03B2R1, and active u03B2-catenin expression. Nu2009=u20097 independent experiments. Representative blots from seven biologically independent samples/group were combined from four independent experiments. Densitometric analysis relative to u03B2-actin is shown. c Immunofluorescence analysis of E-cadherin and u03B1 -SMA expression in conditioned medium-treated TECs. For each slide, images of six different fields of view at xC5u2009~u2009400 magnification were evaluated. Nu2009=u20097 independent samples/group. Scale bar 30u2009u03BCm. d [C]palmitate oxidation measured by [CO2]release. CPM were counted and normalized to the protein in the well. Nu2009=u20098 independent samples/group. e Oxygen consumption rate (OCR) in conditioned medium-treated TECs; each data-point represents the mean of eight independent samples. Three independent experiments were performed. OCR was measured in a Seahorse XF96 analyzer. f Cellular ATP measurement. Eight independent samples were analyzed. g Relative mRNA levels determined by qRT-PCR of FAO regulators in conditioned media-treated TECs. Nu2009=u20097 independent samples per group performed in triplicate were analyzed. h Experimental design for conditioned media experiment. ECs from the kidneys of diabetic GR and diabetic control mice were cultured for 96u2009h. The subsequently harvested media was transferred to TECs from diabetic control mice. Nu2009=u20097/group. i Representative Western blotting analysis of u03B1SMA, TGFu03B2R1 active u03B2-catenin, E-cadherin, CPT1a, and PPARu03B1 expression. Representative blots from seven biologically independent samples/group combined from three independent experiments are shown. Densitometry calculations are normalized to u03B2-actin or HSP90 as indicated. j Relative mRNA levels determined by qRT-PCR of regulators of FAO in conditioned media-treated isolated TECs. Nu2009=u20096 independent replicates per group. Data are meanu2009xB1u2009SEM. Students t test (unpaired two tailed) was used for (bu2013f); (i); for (g, j), one-way ANOVA with Tukey post hoc test was used for the analysis of statistical significance. Significance *pu2009 <u20090.05, **pu2009<u20090.01, ***pu2009<u20090.001. Source data are provided in the source data file. Components of this figure were created using Servier Medical Art templates, which are licensed under a Creative Commons Attribution 3.0 Unported License; .\" - \"To confirm these in vitro results, primary ECs from diabetic control and diabetic GR mice were isolated to analyze the contribution of GR-deficient ECs on mesenchymal activation in TECs. CM from isolated cultured ECs from the kidneys of diabetic GR and diabetic control littermates was transferred to cultured TECs from diabetic control mice (Fig._). This CM treatment from GR-deficient cells caused significant suppression of E-cadherin, CPT1a, and PPARu03B1 and induced u03B1SMA, TGFu03B2 R1, and active u03B2-catenin protein levels in TECs (Fig._). Furthermore, CM from ECs from diabetic GR mice significantly downregulated mRNA expression of the FAO-responsive genes Cpt1a, Cpt2, and Pparu03B1 (Fig._).\"- - This study demonstrates the crucial role of endothelial GR in the regulation of fibrogenic processes in a mouse model of diabetic kidney disease. Our results demonstrate that endothelial GR regulates the mesenchymal trans differentiation process by influencing FA metabolism and control over canonical Wnt signaling in the kidneys of diabetic mice. It is clear from our data that GR loss is one of the catalysts of renal fibrosis in diabetes that leads to disruption of cytokine and chemokine homeostasis by up regulating canonical Wnt signaling. Ultimately, these processes alter the metabolic switch in favor of defective FA metabolism and associated mesenchymal activation in TECs. - \"Metabolic reprogramming in ECs is a critical event in the development of myofibroblast formation, proliferation, and fibrosis in diabetic kidneys. Our data suggest that GR deficiency is a critical step in the metabolic reprogramming of kidney ECs. While bilateral adrenalectomy suppressed corticosterone significantly in all mice studied, this global loss of systemic steroid signaling was not sufficient to alter the course of fibrosis in diabetic mice, suggesting that the tissue-specific effects of targeted loss of GR in the endothelium supersede the systemic effects; this phenomenon will require further study. The altered cytokine levels in the plasma of GR mice include elevated levels of pro-inflammatory cytokines (IL-1u03B2 , IL-6, and IL-17) and the anti-inflammatory cytokine IL-10. The role of IL-10 has not been fully investigated in renal fibrosis in diabetic kidney disease so far. However, our data demonstrate that IL-6 is a key inflammatory cytokine, which is elevated in states of endothelial GR suppression. The neutralization of IL-6 in diabetic mice completely reversed the renal fibrotic phenotype, suggesting a critical pro-fibrotic role of IL-6 in diabetes.\" - Recently, we demonstrated that loss of endothelial GR results in upregulation of canonical Wnt signaling. It is accepted that GR performs its anti-inflammatory actions by targeting the NFkB signaling pathway. However, GR targets also canonical Wnt signaling in ECs, which is independent of its classic target, NFkB. Inhibition of Wnt signaling in ECs may prove to be a valuable therapeutic opportunity for combatting diabetic kidney disease. The Wnt pathway is known to be an important contributor to renal fibrosis and activated canonical Wnt signaling contributes to the disruption of cytokine and chemokine homeostasis. Our data demonstrate that higher levels of GR-deficient-canonical Wnt signaling are associated with the induction of mesenchymal and fibrogenic markers. - \"To further test the therapeutic potential of Wnt inhibition, we used the small molecule_LGK974. Wnt inhibition clearly suppressed canonical Wnt signaling, substantially improved the fibrogenic phenotype in our mouse model of diabetic kidney disease and restored the endothelial GR level. These data suggest that tonic repression of canonical Wnt signaling in ECs is one mechanism by which GR performs its anti-fibrotic action. Notably, this effect was less evident in GR mice, possibly since Wnt inhibition was able to suppress EMT processes in other cell types (TECs) while it was unable to mitigate EndMT processes. Cumulatively, these data suggest that endothelial GR is a key anti-EndMT molecule.\" - Research investigating lipid metabolism in kidney cells is limited but gaining importance. Defects in central metabolism contribute to diabetic kidney disease and there are a few reports showing that altered cytokine levels can affect renal lipid metabolism in diabetic kidney disease. Clinical observations indicate a potential association between lipid levels and kidney disease, and lipid control appears to be important in the prevention and treatment of diabetic kidney disease. Here, we aimed to dissect the contribution of lipid metabolism in ECs to the regulation of diabetic kidney disease. The connection between atherosclerosis and glomerulosclerosis was suggested two decades ago by Diamond, who introduced the foam cell (lipid overloaded macrophage) as the pivotal culprit in both disease processes. The Study of Heart and Renal Protection study was a double-blind, placebo-controlled trial that tried to assess the safety and efficacy of reducing LDL cholesterol in more than 9000 patients, with or without diabetes, with chronic kidney disease. Of ~6000 patients who were not on dialysis at randomization, allocation to simvastatin therapy did not produce a significant reduction by any measure of renal disease progression. There are several clinical trials on statins and their effects on kidney outcome. These clinical data suggest that lipid lowering drugs improve cardiovascular function in diabetic patients, but do not necessarily improve the renal outcome. Large-scale clinical trials that are prospective, randomized, and controlled are still lacking. - Based upon our previous work, DKO mice show worsened atherosclerosis, compared to Apoe mice, which is not explained by differences in plasma lipid levels. This observation was the catalyst which led to the evaluation of the diabetic phenotype in these animals. It is clear from our data that hypercholesterolemia worsened the severity of renal fibrosis in GR mice, suggesting that hypercholesterolemia affects EC metabolism and contributes to renal fibrosis. However, similar to available clinical data, the cholesterol-lowering drug simvastatin did not ameliorate the severity of renal fibrosis in this mouse model of diabetic kidney disease. Interestingly, fibrates are a class of drugs that treat hypertriglyceridemia with residual elevation of non-HDL cholesterol. However, the role of fibrates in patients with diabetic kidney disease has yet to be determined. - We assessed the contribution of endothelial GR loss-linked Wnt activation and its association with defects in FA metabolism. Disruption of endothelial FA metabolism contributes to the activation of EndMT in diabetic kidneys. FAO activation caused remarkable suppression of fibrosis by restoring the endothelial GR level in diabetic mice. In contrast, FAO inhibition caused acceleration of fibrosis by diminishing the level of endothelial GR in diabetic control mice, suggesting that endothelial GR is a critical protein for the action of FAO modulators. Our data clearly suggest that the anti-fibrotic effect of the FAO activator C75 is dependent on endothelial GR, in turn, suggesting that ECs are required for the anti-fibrotic action of this drug. - \"When CM from GR-deplete ECs from diabetic GR mice were transferred to cultured TECs from diabetic control kidneys, we observed induction of mesenchymal markers, activation of TGFu03B2 and canonical Wnt signaling, and concomitant suppression of epithelial cell markers. These findings suggest that EndMT leads to the activation of EMT processes in diabetes. GR-deplete cells have higher levels of TGFu03B2 -smad3 and canonical Wnt signaling, which are associated with disrupted levels of plasma cytokines and suppressed FAO. The cumulative effects of these metabolic changes result in the activation of mesenchymal transformation in ECs, which appears to exert paracrine effects on neighboring TECs. The functional importance of GR protein in EC homeostasis is depicted graphically (Fig._).\" - \\'Fig. 8: Graphical representation of the role of GR in endothelial cell homeostasis.GR is crucial for endothelial cell homeostasis and its loss causes homeostatic disruption through release of the pro-inflammatory cytokine IL-6 and aberrant activation of Wnt signaling which lead to defective fatty acid oxidation and associated mesenchymal activation. Components of this figure were created using Servier Medical Art templates, which are licensed under a Creative Commons Attribution 3.0 Unported License; .\\' - GR agonists like dexamethasone activate GR signaling in all cell types; however, in diabetes, dexamethasone intervention is not preferred due to the severe and predictable exacerbation of hyperglycemia. Alternative approaches which can activate GR in a cell-specific manner need to be identified and may be useful for next generation therapy for cardiovascular dysfunction and renal fibrosis in DN. These data highlight the regulatory role of GR in ECs. However, it is unclear whether endothelial GR directly regulates FAO in mitochondria or indirectly regulates this process by affecting Wnt signaling. What is the potential impact of intracellular GR translocation on the metabolic shift in ECs? It will be interesting to study the role of upstream regulators of GR that might have a significant effect on disease phenotypes in the kidney. Further studies will be required to understand cell-specific metabolic communication in kidney disease pathogenesis. - In conclusion, our findings highlight the regulatory role of GR on EndMT in diabetic kidneys, mediated by control over canonical Wnt signaling and linked defective FA metabolism. This study provides insight into the biology of GR and its critical role in renal fibrosis and diabetes.- - Reagents and antibodies - \"Rabbit polyclonal anti-GR (SAB4501309), mouse monoclonal anti-u03B1SMA (Cat:A5228), and mouse monoclonal anti-u03B2-actin (AC-74) (A2228) antibodies were from Sigma (St Louis, MO). Anti-TGFu03B2R1 (ab31013), PPARu03B1 (ab215270), mouse monoclonal anti-vimentin (RV202) (ab8978), rabbit polyclonal anti-u03B1SMA (ACTA2) (ab5694), anti-HIF1u03B1 (ab516008), and goat polyclonal anti-Snail1 (ab53519) antibodies were purchased from Abcam (Cambridge, UK). Mouse anti-u03B2 -catenin antibody (610154) was purchased from BD Biosciences. CPT1a (12252), rabbit polyclonal anti-E-cadherin antibody (24E10) (3195), and rabbit non-phospho (active) u03B2-Catenin (8814) antibodies were purchased from Cell Signaling Technology (Danvers, MA, USA). Anti-HSP90 was purchased from BD Biosciences (610419). In vivo mouse IL-6 IgG neutralization antibody and control IgG antibodies were purchased from Bio X Cell. Fluorescence-, Alexa fluor 647-, and rhodamine-conjugated secondary antibodies were obtained from Jackson ImmunoResearch (West Grove, PA). TGFu03B22, IL-1u03B2, and recombinant TNFu03B1 and TGFu03B2 neutralizing antibodies were purchased from PeproTech (Rocky Hill, NJ). Etomoxir, C75, and Wnt inhibitor (LGK974) were purchased from Cayman Chemical (Ann Arbor MI).\" - Animal experimentation - \"All experiments were performed according to a protocol approved by the Institutional Animal Care and Use Committee at the Yale University School of Medicine and were in accordance with the National Institute of Health (NIH) Guidelines for the Care of Laboratory Animals. Mice were housed at an ambient temperature of 68u201379u2009xB0F with a humidity that ranged between 30 and 70%. They were exposed to 12-h lightu2013dark cycles. Mice lacking the endothelial GR (known as GR) and those lacking this receptor on the Apoe null background (DKO) were generated as previously described; these mice were on the C57BL/6 background. Diabetes was induced in 10-week-old male mice with five consecutive IP doses of STZ 50u2009mg/kg in 10u2009mmol/L citrate buffer (pH 4.5). Wnt inhibitor (LGK974) was provided to 16-week-STZ-treated diabetic mice using a dose of 5u2009 mg/kg at a frequency of six doses per week for 8 weeks. Etomoxir (20u2009mg/kg) and C75 (15u2009mg/kg) were dosed (IP) three times per week for 3 weeks in GR and control littermates.\" - \"A single IP dose of 200u2009mg/kg STZ was used to induce diabetes in CD-1 mice. In one experiment, male mice were randomized to one of the four groups 16 weeks after induction of diabetes: (1) untreated, (2) fenofibrate (100u2009mg/kg), (3) simvastatin (40u2009mg/kg), or (4) Wnt inhibitor (LGK974; 5u2009mg/kg). In each case, mice were treated for 4 weeks and compared to untreated diabetic CD-1 mice. In another experiment, male diabetic CD-1 mice were randomized to one of the three groups: (1) untreated (vehicle), (2) etomoxir (20u2009mg/kg), or (3) C75 (15u2009mg/kg); in each case mice were treated (IP) three times/week for a total 4 weeks.\" - \"IL-6 IgG and control IgG were injected IP three times/week for a total 4 weeks at a dose of 3u2009mg/kg in both nondiabetic and diabetic mice. All mice had free access to food and water during experiments. Blood was obtained by retro-orbital bleed during experiments. Blood glucose was measured by glucose-strips. Urine albumin levels were assayed using a Mouse Albumin ELISA Kit (Exocell, Philadelphia, PA).\" - \"Tissues and blood were harvested at the time of sacrifice. Some kidneys were minced and stored at u221280u2009xB0C for gene expression and protein analysis. Other kidneys were placed immediately in optimal cutting temperature compound for frozen sections or 4% paraformaldehyde for histologic staining.\" - Bilateral adrenalectomy - \"Sixteen-week-STZ-injected (24 weeks old) diabetic mice and nondiabetic CD-1 and C57BL/6 mice were used for adrenalectomy and sham operations. Animals were handled daily during the last week before experimentation to reduce stress responses. Buprenorphine was used as an analgesic. The first dose was administered 30u2009 min before surgery and then every 12u2009h for 72u2009h, at a dose of 0.05u2009 mg/kg subcutaneously. For the first 24u2009h after surgery, mice were given drinking water containing 0.9% saline to counter the effect of mineralocorticoid removal. Adrenalectomized animals were sacrificed 4 weeks after surgery. Blood was withdrawn in the morning between 9:00 and 10:00 a.m. Corticosterone levels were measured using an ELISA from Cayman Chemical (Ann Arbor MI).\" - Mouse model of unilateral ureteral obstruction (UUO) - \"UUO surgical procedure was performed as previously described. Briefly, mice were anesthetized with isoflurane (3u20135% for induction and 1u20133% for maintenance). Mice were shaved on the left side of the abdomen, a vertical incision was made through the skin with a scalpel, and the skin was retracted. A second incision was made through the peritoneum to expose the kidney. The left ureter was ligated twice 15u2009mm below the renal pelvis with surgical silk, and the ureter was then severed between the two ligatures. Then, the ligated kidney was placed gently back into its correct anatomical position, and sterile saline was added to replenish loss of fluid. The incisions were sutured and mice were individually caged. Buprenorphine was used as an analgesic. The first dose was administered 30u2009min before surgery and then every 12u2009h for 72u2009h, at a dose of 0.05u2009mg/kg subcutaneously. Mice were sacrificed and kidney and blood samples were harvested after perfusion with phosphate-buffered saline (PBS) at 10 days after UUO. Contralateral kidneys were used as a nonfibrotic control for all experiments using this model.\" - Blood pressure measurement - \"Measurements were taken using the tail cuff method according to the manufacturer s instructions. Briefly, mice were trained for 5 days before measurement of blood pressure. After mice were placed in the restraint platform, which was maintained at 33u201334u2009xB0C, the tail was placed through the optical sensor and the cuff compressed. The instrument automatically measured the blood pressure and repeated this ten times. Data are presented as the average of ten measurement cycles.\" - Lipid analysis - \"Mice were fasted for 12u201315u2009h and blood was collected by retro-orbital venous puncture. Whole blood was spun down and plasma stored at u221280u2009 xB0C. Total cholesterol and triglyceride levels were measured enzymatically by kits from Wako and Sigma, respectively, according to the manufacturer s instructions.\" - Morphological evaluation - \"A point-counting method was utilized to evaluate the relative area of the mesangial matrix. We analyzed PAS-stained glomeruli from each mouse using a digital microscope screen grid containing 540 (27u2009xD7u200920) points. Massons trichrome-stained images were evaluated by ImageJ software, and the fibrotic areas were estimated.\" - Sirius red staining - \"Deparaffinized sections were incubated with picrosirius red solution for 1u2009 h at room temperature. The slides were washed twice with acetic acid solution for 30u2009s per wash. The slides were then dehydrated in absolute alcohol three times, cleared in xylene, and mounted with a synthetic resin. Sirius red staining was analyzed using ImageJ software, and fibrotic areas were quantified.\" - Immunohistochemistry - \"Paraffin-embedded kidney sections (5u2009u03BCm thick) were deparaffinized and rehydrated (2u2009min in xylene, four times; 1u2009min in 100% ethanol, twice; 1u2009min in 95% ethanol; 45u2009s in 70% ethanol; and 1u2009min in distilled water), and the antigen was retrieved in a 10u2009mM citrate buffer pH 6 at 98u2009xB0C for 60u2009min. To block the endogenous peroxidase, all sections were incubated in 0.3% hydrogen peroxide for 10u2009min. The immunohistochemistry was performed using a Vectastain ABC Kit (Vector Laboratories, Burlingame, CA). Mouse anti-u03B2-catenin antibody (1:100) and CPT1a (Abnova; H00001374-DO1P; 1:100) antibodies were used. In the negative controls, the primary antibody was omitted and replaced with the blocking solution.\" - Immunofluorescence - \"Frozen kidney sections (5u2009u03BCm) were used for immunofluorescence; double-positive labeling with CD31/u03B1SMA (1:100/1:500), CD31/TGFu03B2R1(1:100/1:500), and E-cadherin/u03B1SMA (1:500/1:500) was measured. Briefly, frozen sections were dried and placed in acetone for 10u2009min at u221230u2009xB0C. Once the sections were dried, they were washed twice in PBS for 5u2009min and then blocked in 2% bovine serum albumin (BSA)/PBS for 30u2009min at room temperature. Thereafter, the sections were incubated in primary antibody (1:100) for 1u2009h and washed in PBS (5u2009min) three times. Next, the sections were incubated with the secondary antibodies for 30u2009min, washed with PBS three times (5u2009min each), and mounted with mounting medium with DAPI (Vector Laboratories, Burlingame, CA). The immuno-labeled sections were analyzed by fluorescence microscopy. For each mouse, original magnification of xD7400 pictures was obtained from six different areas, and quantification was performed.\" - EndMT and EMT detection - \"Frozen sections (5u2009xB5m) were used for the detection of EndMT and EMT. Cells undergoing EndMT were detected by double-positive labeling for CD31 (1:100) and u03B1SMA (1:500) and/or TGFu03B2R1 (1:500). Cells undergoing EMT were detected by double-positive labeling for E-cadherin (1:500) and u03B1SMA (1:500). Sections were analyzed and quantified by fluorescence microscopy.\" - Isolation of endothelial cells - \"ECs from the kidneys of nondiabetic and diabetic mice were isolated using a standardized kit (Miltenyl Biotech, USA) by following the manufacturers instructions. Briefly, kidneys were isolated and minced into small pieces. Using a series of enzymatic reactions by treating the tissue with trypsin and Collagenase type I solution, a single cell suspension was created. The pellet was dissolved with CD31 magnetic beads and the CD31-labeled cells were separated on a magnetic separator. The cells were further purified on a column. Cell number was counted by hemocytometer and cells were plated on 0.1% gelatin coated Petri dishes. Cell purity was measured by flow cytometry (BD FACSDiva) using PE-conjugated CD31 (BDB553373) (1:100) and FITC-conjugated CD45 (BDB553079) (1:100), both from Becton Dickinson (USA).\" - Isolation of kidney TECs - \"After sacrifice, kidneys from diabetic GR and control littermates were excised and perfused with (10u2009mL) followed by collagenase type II digestion (2u2009 mg/mL). After digestion, the cortical region of kidneys was used for further processing. The cortical region of kidneys was minced and further digested in collagenase buffer for an additional 5u2009min at 37u2009xB0C with rotation to release cells. Digested tissue and cell suspension were passed through a 70-u03BCm cell strainer, centrifuged at 50u2009xD7u2009g for 5u2009min, and washed in PBS for two rounds to collect TECs. Isolated TECs were seeded onto collagen-coated Petri dishes and cultured in renal epithelial cell medium (C-26130, PromoCell) supplemented with growth factors for TEC growth.\" - Cellular bioenergetic analysis - \"FAO-associated OCR was studied using extracellular flux analysis (Seahorse XFe96, Agilent Technologies). On the assay day, substrate-limited medium was replaced with Krebsu2013Henseleit buffer assay medium supplemented with 0.2% carnitine for 1u2009h at 37u2009xB0C without CO2. Finally, just before starting the assay, BSA or 200u2009mM palmitate-BSA FAO substrate was added. After the assay, protein was extracted from wells with 0.1% NP-40u2013PBS solution and quantified with a bicinchoninic acid protein assay (Thermo Fisher Scientific) for data normalization. OCR was determined by normalizing the measurements to cell counts by quantifying the Hoechst staining in each well.\" - ATP measurement - \"ATP content was determined using the ATP Colorimetric Assay kit (Biovision), following the manufacturers instructions.\" - RNA isolation and qPCR - \"Total RNA was isolated using standard Trizol protocol. RNA was reverse transcribed using the iScript cDNA Synthesis kit (Bio-Rad) and qPCR was performed on a Bio-Rad C1000 Touch thermal cycler using the resultant cDNA, as well as qPCR Master mix and gene-specific primers. The list of mouse primers used is in Table_.\" - \"Results were quantified using the deltau2013delta-cycle threshold (Ct) method(u0394 u0394Ct). All experiments were performed in triplicate and 18S was utilized as an internal control.\" - Western blot - \"Protein lysates were boiled in sodium dodecyl sulfate (SDS) sample buffer at 94u2009xB0C for 5u2009min. After centrifugation at 17,000u2009xD7u2009 g for 10u2009min at 4u2009xB0C, the supernatant was separated on 6u201312% SDS polyacrylamide gels, and blotted onto PVDF membranes (Immobilon, Bedford, MA) via the semidry method. After blocking with TBS (Tris buffered saline containing 0.05% Tween 20) containing 5% BSA, membranes were incubated with each primary antibody (GR: 1:1000; Anti-TGFu03B2R1: 1:500; anti-u03B1SMA: 1:500; anti-vimentin: 1:2000, anti-u03B2-catenin: 1:500, anti-u03B2-actin 1:10,000), in TBS containing 5% BSA at 4u2009xB0C overnight. Protein bands were visualized using the Odyssey Infrared Imaging System (LI-COR Biotechnology), and enhanced chemiluminescence detection system (Pierce Biotechnology, Rockland IL) using ImageQuant LAS 400 (GE Healthcare Life Sciences, Uppsala, Sweden). Densitometry was performed using ImageJ software (NIH).\" - In vitro experiments and siRNA transfection - \"HUVECs were used at passages four to eight and cultured in Endothelial Basal Medium-2 media with growth factors and 10% serum. Human GR-specific siRNA (Invitrogen) was used at a concentration of 100u2009nM for 48u2009h to effectively knockdown GR. Cells were treated with or without TGFu03B22 (10u2009ng/ml) for 48u2009 h and harvested for western blot analysis. Some transfected cells were treated with fenofibrate (1u2009u03BCM) and etomoxir (40u2009u03BCM) for 48u2009 h. In a second set of experiments, Human HK-2 cells were cultured in DMEM and Keratinocyte-SFM (1X) medium (Life Technologies Green Island NY). When the cells reached 70% confluence, CM from control siRNA and GR siRNA-transfected HUVECs was added to the HK-2 cell culture.\" - Fatty acid uptake - \"Cultured isolated kidney ECs were incubated with medium containing 2u2009u03BC Ci [C]-palmitate. [C]-palmitate uptake was measured by liquid scintillation counting.\" - Fatty acid oxidation - \"Cultured isolated kidney ECs were incubated with medium containing 0.75u2009 mmol/L palmitate (conjugated to 2% free FAu2013free BSA/[C] palmitate at 2u2009 u03BCCi/mL) for 2u2009h. 1u2009mL of the culture medium was transferred to a sealable tube, the cap of which housed a Whatman filter paper disc. CO2 trapped in the media was then released by acidification of media using 60% perchloric acid. Radioactivity that had become adsorbed onto the filter discs was then quantified by liquid scintillation counting.\" - Statistical analysis - \"All values are expressed as meansu2009xB1u2009SEM and analyzed using the statistical package for the GraphPad Prism 7 (GraphPad Software, Inc., La Jolla, CA). One-way ANOVA, followed by Tukeys test and two-way ANOVA (Figs._ and ), was employed to analyze significance when comparing multiple independent groups. The post hoc tests were run only if F achieved pu2009<u20090.05 and there was no significant variance in homogeneity. In each experiment, N represents the number of separate experiments (in vitro) and the number of mice (in vivo). Technical replicates were used to ensure the reliability of single values. Data analyses were blinded. The data were considered statistically significant at pu2009<u20090.05.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - \\' are provided with this paper. All relevant data are available from the authors upon reasonable request.\\' - \\' \\'- - \"Reidy, K., Kang, H. M., Hostetter, T. & Susztak, K. Molecular mechanisms of diabetic kidney disease. J. Clin. Investig. 124, 2333u20132340 (2014)._ _ _ _ _ Badal, S. S. & Danesh, F. R. New insights into molecular mechanisms of diabetic kidney disease. Am. J. Kidney Dis. 63, S63u2013S83 (2014)._ _ _ _ _ Breyer, M. D. & Susztak, K. The next generation of therapeutics for chronic kidney disease. Nat. Rev. Drug Discov. 15, 568u2013588 (2016)._ _ _ _ _ Humphreys, B. D. et al. Fate tracing reveals the pericyte and not epithelial origin of myofibroblasts in kidney fibrosis. Am. J. Pathol. 176, 85u201397 (2010)._ _ _ _ _ Kuppe, C. et al. Decoding myofibroblast origins in human kidney fibrosis. Nature (2020).LeBleu, V. S. et al. Origin and function of myofibroblasts in kidney fibrosis. Nat. Med. 19, 1047u20131053 (2013)._ _ _ _ _ Srivastava, S. P., Koya, D. & Kanasaki, K. MicroRNAs in kidney fibrosis and diabetic nephropathy: roles on EMT and EndMT. Biomed. Res. Int. 2013, 125469 (2013)._ _ _ _ _ Srivastava, S. P., Hedayat, F. A., Kanasaki, K. & Goodwin, J. E. microRNA crosstalk influences epithelial-to-mesenchymal, endothelial-to-mesenchymal, and macrophage-to-mesenchymal transitions in the kidney. Front. Pharm. 10, 904 (2019)._ _ _ Medici, D. Endothelial-mesenchymal transition in regenerative medicine. Stem Cells Int. 2016, 6962801 (2016)._ _ _ Eelen, G. et al. Endothelial cell metabolism. Physiol. Rev. 98, 3u2013 58 (2018)._ _ _ _ Theodorou, K. & Boon, R. A. Endothelial cell metabolism in atherosclerosis. Front. Cell Dev. Biol. 6, 82 (2018)._ _ _ _ Zhou, H. L. et al. Metabolic reprogramming by the S-nitroso-CoA reductase system protects against kidney injury. Nature 565, 96u2013100 (2019)._ _ _ _ _ Cantelmo, A. R. et al. Inhibition of the glycolytic activator PFKFB3 in endothelium induces tumor vessel normalization, impairs metastasis, and improves chemotherapy. Cancer Cell 30, 968u2013985 (2016)._ _ _ _ _ Cruys, B. et al. Glycolytic regulation of cell rearrangement in angiogenesis. Nat. Commun. 7, 12240 (2016)._ _ _ _ _ _ Schoors, S. et al. Fatty acid carbon is essential for dNTP synthesis in endothelial cells. Nature 520, 192u2013197 (2015)._ _ _ _ _ _ Xiong, J. et al. A metabolic basis for endothelial-to-mesenchymal transition. Mol. Cell 69, 689u2013698 (2018)._ _ _ _ _ Lovisa, S. & Kalluri, R. Fatty acid oxidation regulates the activation of endothelial-to-mesenchymal transition. Trends Mol. Med. 24, 432u2013434 (2018)._ _ _ _ Zeisberg, M. & Neilson, E. G. Mechanisms of tubulointerstitial fibrosis. J. Am. Soc. Nephrolo. 21, 1819u2013 1834 (2010)._ _ _ Zeisberg, E. M., Potenta, S. E., Sugimoto, H., Zeisberg, M. & Kalluri, R. Fibroblasts in kidney fibrosis emerge via endothelial-to-mesenchymal transition. J. Am. Soc. Nephrol. 19, 2282u2013 2287 (2008)._ _ _ _ Zeisberg, E. M. et al. Endothelial-to-mesenchymal transition contributes to cardiac fibrosis. Nat. Med. 13, 952u2013961 (2007)._ _ _ _ Li, J. et al. Endothelial FGFR1 (fibroblast growth factor receptor 1) deficiency contributes differential fibrogenic effects in kidney and heart of diabetic mice. Hypertension 76, 1935u20131944 (2020)._ _ _ _ Medici, D. & Kalluri, R. Endothelial-mesenchymal transition and its contribution to the emergence of stem cell phenotype. Semin. Cancer Biol. 22, 379u2013384 (2012)._ _ _ _ _ Goodwin, J. E., Feng, Y., Velazquez, H. & Sessa, W. C. Endothelial glucocorticoid receptor is required for protection against sepsis. Proc. Natl Acad. Sci. USA 110, 306u2013311 (2013)._ _ _ _ _ Goodwin, J. E., Feng, Y., Velazquez, H., Zhou, H. & Sessa, W. C. Loss of the endothelial glucocorticoid receptor prevents the therapeutic protection afforded by dexamethasone after LPS. PLoS ONE 9, e108126 (2014)._ _ _ _ _ _ Goodwin, J. E. et al. Endothelial glucocorticoid receptor suppresses atherogenesisu2013brief report. Arterioscler. Thromb. Vasc. Biol. 35, 779u2013782 (2015)._ _ _ _ _ Zhou, H. et al. Loss of the podocyte glucocorticoid receptor exacerbates proteinuria after injury. Sci. Rep. 7, 9833 (2017)._ _ _ _ _ Zhou, H. et al. Endothelial cellu2013glucocorticoid receptor interactions and regulation of Wnt signaling. JCI Insight 5 (2020).He, W. et al. Wnt/beta-catenin signaling promotes renal interstitial fibrosis. J. Am. Soc. Nephrol. 20, 765u2013776 (2009)._ _ _ _ _ Kanasaki, K. et al. Linagliptin-mediated DPP-4 inhibition ameliorates kidney fibrosis in streptozotocin-induced diabetic mice by inhibiting endothelial-to-mesenchymal transition in a therapeutic regimen. Diabetes 63, 2120u20132131 (2014)._ _ _ _ Srivastava, S. P. et al. Effect of antifibrotic MicroRNAs crosstalk on the action of N-acetyl-seryl-aspartyl-lysyl-proline in diabetes-related kidney fibrosis. Sci. Rep. 6, 29884 (2016)._ _ _ _ _ _ Shi, S. et al. Interactions of DPP-4 and integrin beta1 influences endothelial-to-mesenchymal transition. Kidney Int. 88, 479u2013489 (2015)._ _ _ _ Srivastava, S. P. et al. SIRT3 deficiency leads to induction of abnormal glycolysis in diabetic kidney with fibrosis. Cell Death Dis. 9, 997 (2018)._ _ _ _ _ Sugimoto, H., Grahovac, G., Zeisberg, M. & Kalluri, R. Renal fibrosis and glomerulosclerosis in a new mouse model of diabetic nephropathy and its regression by bone morphogenic protein-7 and advanced glycation end product inhibitors. Diabetes 56, 1825u20131833 (2007)._ _ _ _ Fielding, C. A. et al. Interleukin-6 signaling drives fibrosis in unresolved inflammation. Immunity 40, 40u201350 (2014)._ _ _ _ _ Apalset, E. M. et al. Interferon gamma (IFN-gamma)-mediated inflammation and the kynurenine pathway in relation to risk of hip fractures: the Hordaland Health Study. Osteoporos. Int. 25, 2067u2013 2075 (2014)._ _ _ _ _ Fathy, S. A., Mohamed, M. R., Ali, M. A. M., El-Helaly, A. E. & Alattar, A. T. Influence of IL-6, IL-10, IFN-gamma and TNF-alpha genetic variants on susceptibility to diabetic kidney disease in type 2 diabetes mellitus patients. Biomarkers 24, 43u2013 55 (2019)._ _ _ _ Owens, E. P. et al. Biomarkers and the role of mast cells as facilitators of inflammation and fibrosis in chronic kidney disease. Transl. Androl. Urol. 8, S175u2013S183 (2019)._ _ _ _ Hickey, F. B. & Martin, F. Role of the immune system in diabetic kidney disease. Curr. Diab. Rep. 18, 20 (2018)._ _ _ _ Zhang, L. S. & Lum, L. Delivery of the porcupine inhibitor WNT974 in mice. Methods Mol. Biol. 1481, 111u2013117 (2016)._ _ _ _ _ Kang, H. M. et al. Defective fatty acid oxidation in renal tubular epithelial cells has a key role in kidney fibrosis development. Nat. Med. 21, 37u201346 (2015)._ _ _ _ Lovisa, S. et al. Epithelial-to-mesenchymal transition induces cell cycle arrest and parenchymal damage in renal fibrosis. Nat. Med. 21, 998u20131009 (2015)._ _ _ _ _ Tran, M. T. et al. PGC1alpha drives NAD biosynthesis linking oxidative metabolism to renal protection. Nature 531, 528u2013532 (2016)._ _ _ _ _ _ Qi, W. et al. Pyruvate kinase M2 activation may protect against the progression of diabetic glomerular pathology and mitochondrial dysfunction. Nat. Med. 23, 753u2013762 (2017)._ _ _ _ _ Delgado, M. & Ganea, D. Inhibition of endotoxin-induced macrophage chemokine production by VIP and PACAP in vitro and in vivo. Arch. Physiol. Biochem. 109, 377u2013382 (2001)._ _ _ _ Kato, H. et al. Wnt/beta-catenin pathway in podocytes integrates cell adhesion, differentiation, and survival. J. Biol. Chem. 286, 26003u2013 26015 (2011)._ _ _ _ _ Zhou, Z. et al. MicroRNA-27a promotes podocyte injury via PPARgamma-mediated beta-catenin activation in diabetic nephropathy. Cell Death Dis. 8, e2658 (2017)._ _ _ _ _ Feng, Y., Liang, Y., Ren, J. & Dai, C. Canonical Wnt signaling promotes macrophage proliferation during kidney fibrosis. Kidney Dis. 4, 95u2013103 (2018)._ _ Wanner, C., Drechsler, C. & Krane, V. Lipid metabolism in chronic kidney disease: the role of statins in cardiovascular risk. J. Ren. Nutr. 17, 75u201378 (2007)._ _ _ Srivastava, S. P., Shi, S., Koya, D. & Kanasaki, K. Lipid mediators in diabetic nephropathy. Fibrogenesis Tissue Repair 7, 12 (2014)._ _ _ _ _ Herman-Edelstein, M., Scherzer, P., Tobar, A., Levi, M. & Gafter, U. Altered renal lipid metabolism and renal lipid accumulation in human diabetic nephropathy. J. Lipid Res. 55, 561u2013 572 (2014)._ _ _ _ _ Wahl, P., Ducasa, G. M. & Fornoni, A. Systemic and renal lipids in kidney disease development and progression. Am. J. Physiol. Renal. Physiol. 310, F433u2013F445 (2016)._ _ _ _ Kovesdy, C. P., Anderson, J. E. & Kalantar-Zadeh, K. Inverse association between lipid levels and mortality in men with chronic kidney disease who are not yet on dialysis: effects of case mix and the malnutrition-inflammation-cachexia syndrome. J. Am. Soc. Nephrol. 18, 304u2013311 (2007)._ _ _ _ Diamond, J. R. Analogous pathobiologic mechanisms in glomerulosclerosis and atherosclerosis. Kidney Int. Suppl. 31, S29u2013S34 (1991)._ _ _ Kwan, B. C., Kronenberg, F., Beddhu, S. & Cheung, A. K. Lipoprotein metabolism and lipid management in chronic kidney disease. J. Am. Soc. Nephrol. 18, 1246u20131261 (2007)._ _ _ _ Baigent, C. et al. First United Kingdom Heart and Renal Protection (UK-HARP-I) study: biochemical efficacy and safety of simvastatin and safety of low-dose aspirin in chronic kidney disease. Am. J. Kidney Dis. 45, 473u2013484 (2005)._ _ _ _ Sharp Collaborative Group. Study of Heart and Renal Protection (SHARP): randomized trial to assess the effects of lowering low-density lipoprotein cholesterol among 9,438 patients with chronic kidney disease. Am. Heart J. 160, 785u2013794 (2010)._ _ _ _ Kimura, S. et al. Randomized comparison of pitavastatin and pravastatin treatment on the reduction of urinary albumin in patients with type 2 diabetic nephropathy. Diabetes Obes. Metab. 14, 666u2013669 (2012)._ _ _ _ Abe, M. et al. Effects of lipid-lowering therapy with rosuvastatin on kidney function and oxidative stress in patients with diabetic nephropathy. J. Atheroscler. Thromb. 18, 1018u20131028 (2011)._ _ _ _ Rutter, M. K. et al. Protection Against Nephropathy in Diabetes with Atorvastatin (PANDA): a randomized double-blind placebo-controlled trial of high- vs. low-dose atorvastatin(1). Diabet. Med. 28, 100u2013108 (2011)._ _ _ _ Colhoun, H. M. et al. Effects of atorvastatin on kidney outcomes and cardiovascular disease in patients with diabetes: an analysis from the Collaborative Atorvastatin Diabetes Study (CARDS). Am. J. Kidney Dis. 54, 810u2013819 (2009)._ _ _ _ Coonrod, B. A. et al. Predictors of microalbuminuria in individuals with IDDM. Pittsburgh Epidemiology of Diabetes Complications Study. Diabetes Care 16, 1376u20131383 (1993)._ _ _ _ Ting, R. D. et al. Benefits and safety of long-term fenofibrate therapy in people with type 2 diabetes and renal impairment: the FIELD Study. Diabetes Care 35, 218u2013225 (2012)._ _ _ _ _ Li, J. et al. FGFR1 is critical for the anti-endothelial mesenchymal transition effect of N-acetyl-seryl-aspartyl-lysyl-proline via induction of the MAP4K4 pathway. Cell Death Dis. 8, e2965 (2017)._ _ _ _ _ Price, N. L. et al. Genetic deficiency or pharmacological inhibition of miR-33 protects from kidney fibrosis. JCI Insight 4 (2019).\"- - This work is supported by grants from the National Institutes of Health to A.D. (R01HL128406), C.F.-H. (R35HL135820), and J.G. (R01HL131952).- - Authors and Affiliations - \"Department of Pediatrics, Yale University School of Medicine New Haven, New Haven, CT, USASwayam Prakash Srivastava,_Han Zhou,_Bing Liu_&_Julie GoodwinVascular Biology and Therapeutics Program, Yale University School of Medicine New Haven, New Haven, CT, USASwayam Prakash Srivastava,_Han Zhou,_Ocean Setia,_Bing Liu,_Alan Dardik,_Carlos Fernandez-Hernando_&_Julie GoodwinDepartment of Diabetology and Endocrinology, Kanazawa Medical University, Uchinada, JapanSwayam Prakash Srivastava,_Keizo Kanasaki_&_Daisuke KoyaDepartment of Surgery, Yale University School of Medicine New Haven, New Haven, CT, USAOcean Setia_&_ Alan DardikDepartment of Surgery, VA Connecticut Healthcare System, West Haven, CT, USAAlan DardikDepartment of Comparative Medicine, Yale University School of Medicine New Haven, New Haven, CT, USACarlos Fernandez-HernandoProgram in Integrative Cell Signaling and Neurobiology of Metabolism (ICSNM), Yale University School of Medicine New Haven, New Haven, CT, USACarlos Fernandez-HernandoDepartment of Pathology, Yale University School of Medicine New Haven, New Haven, CT, USACarlos Fernandez-Hernando\" - \"AuthorsSwayam Prakash SrivastavaYou can also search for this author in _Han ZhouYou can also search for this author in _Ocean SetiaYou can also search for this author in _Bing LiuYou can also search for this author in _Keizo KanasakiYou can also search for this author in _Daisuke KoyaYou can also search for this author in _Alan DardikYou can also search for this author in _Carlos Fernandez-HernandoYou can also search for this author in _Julie GoodwinYou can also search for this author in _\" - Contributions - S.P.S. performed experiments and wrote the paper. H.Z. helped in genotyping the mice and was involved in validation of the data. O.S. performed surgical experiments in mice. B.L. performed cell isolation experiments. K.K., D.K., A.D., and C.F.-H. provided intellectual input. J.G. mentored, validated the data, made intellectual contributions, wrote the paper, and performed final editing of the paper. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks Claude Libert, Jianxing Ma and Katalin Susztak for their contribution to the peer review of this work. Peer reviewer reports are available. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported License; .\\' - Finally; glycogen has been proposed to play a role in keratinization. The main sites of expression of glycogen metabolism proteins are suprabulbar ORS, hair cuticle and precortex, and the accumulation of glucose in these regions may reflect HF compartments with the highest metabolic rates. Co-aligned expression of the enzymes involved in glycolysis (PGM1, LDHA, LDHB) in the cuticle and the pre-cortex, together with the transient glucose accumulation suggest that they be may be regions of enzyme activity directed towards rapid keratinization and adapted to quick metabolic changes. Finally, the ORS could probably act as a stable glycogen storage site used both for anagen maintenance and remodelling and regression in catagen. This later role is suggested by our data showing that inhibition of PYGL by GPI significantly reduced the number of HF entering catagen suggesting that glycogen breakdown is essential for catagen but also despite significant decrease in mRNA expression in catagen, protein levels for the enzymes did not change significantly. - In conclusion, we have shown that the HF ORS is a major site of glycogen synthesis and functional glycogen storage and is capable of gluconeogenesis. That inhibiting glycogen catabolism ex vivo promotes hair growth and delays catagen, prolonging anagen. This constitutes a promising target for future research with regards HF nutrition and nutrient sensing.- - Human HF collection and organ culture - \"HF units from occipital region were obtained from hair transplant patients and whole scalp skin samples from face lift patients following ethical approval granted by East London and The City Research Ethics Committee 1 (REC Alpha 09/H0704/40) and written informed patient consent, adhering to the Declaration of Helsinki Principles. All samples, slides and biological material were tracked and stored according to the u2018Human Tissue Act guidelines. Anagen VI HFs were microdissected and cultured as previously described. ORS-KC were isolated as previously described with a modification-freshly isolated ORS-KC were seeded in collagen I (Thermo Fisher, UK) coated flasks and cultured in Keratinocyte Serum Free Medium (Thermo Fisher, UK). Starvation was performed by incubation in SILAC RPMI 1640 Flex Medium (Thermo Fisher, UK). PYGL inhibitor (Cayman Chemicals, UK) and 2-NBDG (Thermo Fisher, UK) were diluted from stock DMSO solution in culture medium. Lactate (Sigma-Aldrich, UK) was diluted in culture media.\" - RNA extraction, qPCR analysis - \"Total RNA was extracted from ORS-KC and HFs using the Qiagen RNeasy Microkit (Qiagen, UK) following the manufacturers instructions. cDNA was synthesized using SuperScript VILO cDNA Synthesis Kit (Invitrogen, UK). qPCR was performed using Rotor-Gene SYBR Green PCR Kit (Qiagen, UK). Relative expression was determined against B2M-Beta-2-microglobulin.\" - Periodic Acid Schiff staining - \"Formalin-fixed paraffin-embedded (FFPE) sections were stained using Periodic Acid Schiff (PAS) Kit (Sigma-Aldrich, UK), according to manufacturers protocol. Negative control sections were digested using porcine u03B1-amylase (Sigma-Aldrich, UK). Sections were imaged using Nanozoomer slide scanner (Hamamatsu, Japan). PAS quantification was performed using ImageJ software, intensity of staining was calculated in the ORS of HFs and normalized to cell number.\" - Immunofluorescence - \"Antigen retrieval was performed on 5_xB5m FFPE tissue sections using Citrate buffer, primary antibodies (GYS1, PGM1, LDHB-Abcam, UK; PYGL, Glut1, PCK1-Proteintech, UK; MCT1-Santa Cruz, UK; CD34-BD Biosciences) were incubated overnight at 4_ xB0C followed by appropriate Alexa Fluor secondary antibodies (Thermo Fisher, UK) and DAPI for nuclear staining. Analysis of immunostaining were performed using DM5000B (Leica) and LSM880 (Zeiss) microscopes.\" - Glycogen concentration assay - \"Glycogen was quantified using a coupled enzyme colorimetric Glycogen Assay Kit (Sigma-Aldrich, UK) according to manufacturers protocol, normalized to protein concentration [Pierce BCA Protein Assay Kit (Thermo Fisher, UK)].\" - Western blotting - \"ORS-KCs and HFs homogenized in RIPA buffer containing protease and phosphorylase inhibitors (Sigma-Aldrich, UK) were separated using NuPAGE minigels (Invitrogen, UK) according to the manufacturers instructions then transferred to a Trans-Blot Turbo PVDF membrane (Bio-Rad, UK) and incubated overnight at 4_xB0 C with primary antibodies (u03B2-Actin, G6Pase, G6PD, LDHA-Santa Cruz, UK; GYS1, Glut1, PCK1, PYGL-Proteintech, UK; GAPDH, LDHB, PGM1-Abcam) and thereafter with appropriate secondary horseradish peroxidase-conjugated antibodies (Dako, Denmark). Membranes were imaged using Clarity ECL chemiluminescence solution (Bio-Rad, UK) and visualized using ChemiDoc XRS (Bio-Rad, UK).\" - Statistical analyses - \"Biological replicates and number of HFs used are indicated in the respective figure legend. Statistical was carried out using either two-tailed paired t-test, one-way ANOVA or two-way ANOVA with multiple comparisons. Data was expressed as meanu2009xB1 SEM and considered statistically significant at a P valueu2009 <u20090.05.\"- - \\' \\' - All relevant data are included in the manuscript. Used datasets are available from the corresponding author on reasonable request. - \\' \\'- - \"CTS:Connective tissue sheathG6Pase:Glucose-6-phosphataseGLUT1:Glucose transporter 1GPI:Glycogen phosphorylase inhibitorGYS1:Glycogen synthase 1HF:Hair follicleHM:Hair matrixIRS:Inner root sheathLDHB:Lactate dehydrogenase BORS:Outer root sheathORS-KCs:Outer root sheath keratinocytes PAS:Periodic Acid SchiffsPCK1:Phosphoenolpyruvate carboxykinase 1 PGM1:Phosphoglucomutase 1PPP:Pentose phosphate pathwayPYGL:Glycogen phosphorylase\"- - \"Purba, T. S. et al. Characterisation of cell cycle arrest and terminal differentiation in a maximally proliferative human epithelial tissue: Lessons from the human hair follicle matrix. Eur. J. Cell Biol. 96, 632u2013641 (2017)._ _ _ _ Vidali, S., Knuever, J., Lerchner, J., Giesen, M. & BxEDrxF3, T. Hypothalamicu2013pituitaryu2013thyroid axis hormones stimulate mitochondrial function and biogenesis in human hair follicles. J. Invest. Dermatol. 134, 33u201342 (2014)._ _ _ _ Lemasters, J. J. et al. Compartmentation of mitochondrial and oxidative metabolism in growing hair follicles: A ring of fire. J. Invest. Dermatol. 137, 1434u20131444 (2017)._ _ _ _ _ Unterlass, J. E. & Curtin, N. J. Warburg and Krebs and related effects in cancer. Expert Rev. Mol. Med. 21, (2019).Williams, R., Philpott, M. P. & Kealey, T. Metabolism of freshly isolated human hair follicles capable of hair elongation: A glutaminolytic, aerobic glycolytic tissue. J Investig. Dermatol. 100, 834u2013840 (1993)._ _ _ _ Chandel, N. S. Navigating metabolism. Cold Spring Harbor Laboratory Press (2015)._ _ Vaitheesvaran, B. et al. The Warburg effect: A balance of flux analysis. Metabolomics 11, 787u2013796 (2015)._ _ _ _ Heiden, M. G. Vander, Cantley, L. C. & Thompson, C. B. Understanding the warburg effect: The metabolic requirements of cell proliferation. Science (80-. ). 324, 1029u20131033 (2009).Liberti, M. V. & Locasale, J. W. The Warburg effect: How does it benefit cancer cells?. Trends Biochem. Sci. 41, 211u2013218 (2016)._ _ _ _ _ Flores, A. et al. Lactate dehydrogenase activity drives hair follicle stem cell activation. Nat. Cell Biol. 19, 1017u20131026 (2017)._ _ _ _ Kim, C. S. et al. Glutamine metabolism controls stem cell fate reversibility and long-term maintenance in the hair follicle. Cell Metab. 4, 629u2013642 (2020)._ _ _ Seheult, J., Fitzpatrick, G. & Boran, G. Lactic acidosis: An update. Clin. Chem. Lab. Med. 55, 322u2013333 (2017)._ _ _ _ Gladden, L. B. Lactate metabolism: A new paradigm for the third millennium. J. Physiol. 558, 5u201330 (2004)._ _ _ _ _ Adeva-Andany, M. et al. Comprehensive review on lactate metabolism in human health. Mitochondrion 17, 76u2013100 (2014)._ _ _ _ Wang, Z. & Dong, C. Gluconeogenesis in cancer: Function and regulation of PEPCK, FBPase, and G6Pase. Trends Cancer 5, 3u201345 (2019)._ _ _ Tayek, J. A. & Katz, J. Glucose production, recycling, Cori cycle, and gluconeogenesis in humans: Relationship to serum cortisol. Am. J. Physiol. Metab. 272, E476u2013E484 (1997)._ _ _ Adeva-Andany, M. M., GonzxE1lez-LucxE1n, M., Donapetry-GarcxED a, C., FernxE1ndez-FernxE1ndez, C. & Ameneiros-RodrxEDguez, E. Glycogen metabolism in humans. BBA Clin. 27, 85u2013100 (2016)._ _ Shulman, R. G. & Rothman, D. L. The, u201Cglycogen shuntu201D in exercising muscle: A role for glycogen in muscle energetics and fatigue. Proc. Natl. Acad. Sci. U. S. A. 98, 457u2013461 (2001)._ _ _ _ _ _ Favaro, E. et al. Glucose utilization via glycogen phosphorylase sustains proliferation and prevents premature senescence in cancer cells. Cell Metab. 16, 751u2013764 (2012)._ _ _ _ Zois, C. E. & Harris, A. L. Glycogen metabolism has a key role in the cancer microenvironment and provides new targets for cancer therapy. J. Mol. Med. 94, 137u2013154 (2016)._ _ _ _ Kowalik, M., Columbano, A. & Perra, A. Emerging role of the pentose phosphate pathway in hepatocellular carcinoma. Front. Oncol. 11, 87 (2017)._ _ Montagna, W., Chase, H. B. & Hamilton, J. B. The distribution of glycogen and lipids in human skin. J Invest Dermatol 17, 147u2013157 (1951)._ _ _ _ Falin, L. Glycogen in the epithelium of mucous membranes and skin and its significance. Acta Anat 46, 244u2013276 (1961)._ _ _ _ Montagna, W. & Parakkal, P. F. An Introduction to Skin. in The Structure & Function of Skin 1u201317 (1974). Shipman, M., Chase, H. B. & Montagna, W. Glycogen in skin of the mouse during cycles of hair growth. 88, 449u2013451 (1955)._ _ Williams, R., Philpott, M. P. & Kealey, T. Metabolism of freshly isolated human hair follicles capable of hair elongation: A glutaminolytic, aerobic glycolytic tissue. J. Invest. Dermatol. 100, 834u2013840 (1993)._ _ _ _ Philpott, M. P., Green, M. R. & Kealey, T. Human hair growth in vitro. J. Cell Sci. 97, 463u2013471 (1990)._ _ _ _ Langan, E. A., Philpott, M. P., Kloepper, J. E. & Paus, R. Human hair follicle organ culture: theory, application and perspectives. Exp. Dermatol. 24, 903u2013 911 (2015)._ _ _ Philpott, M. P. Culture of the human pilosebaceous unit, hair follicle and sebaceous gland. Exp. Dermatol. 27, 571u2013577 (2018)._ _ _ Braun-Falco, O. The Histochemistry of the Hair Follicle. in The Biology of Hair Growth 65u2013 90 (1958). Fu A., D. & Campbell-Thompson, M. Periodic acid-schiff staining with diastase. in Methods in Molecular Biology 145u2013149 (2017). Purba, T. S. et al. Human epithelial hair follicle stem cells and their progeny: Current state of knowledge, the widening gap in translational research and future challenges. BioEssays 36, 513u2013525 (2014)._ _ _ _ Kloepper, J. et al. Methods in hair research: How to objectively distinguish between anagen and catagen in human hair follicle organ culture. Exp Dermatol 19, 305u2013312 (2010)._ _ _ _ Eshmuminov, D. et al. An integrated perfusion machine preserves injured human livers for 1 week. Nat. Biotechnol. 38, 189u2013198 (2020)._ _ _ _ _ Oh, J. W. et al. A guide to studying human hair follicle cycling in vivo. J. Invest. Dermatol. 136, 34u201344 (2016)._ _ _ _ _ Geyfman, M., Plikus, M. V., Treffeisen, E., Andersen, B. & Paus, R. Resting no more: Re-defining telogen, the maintenance stage of the hair growth cycle. Biol. Rev. 90, 1179u20131196 (2015)._ _ _ Geyfman, M., Gordon, W., Paus, R. & Andersen, B. Identification of telogen markers underscores that telogen is far from a quiescent hair cycle phase. J. Invest. Dermatol. 132, 721u2013724 (2012)._ _ _ _ Sullivan, M. A. & Forbes, J. M. Glucose and glycogen in the diabetic kidney: Heroes or villains?. EBioMedicine 47, 590u2013597 (2019)._ _ _ _ Slack, J. M. W. Molecular biology of the cell. Principles of Tissue Engineering: Fourth Edition (2013)._ _ McBride, A., Ghilagaber, S., Nikolaev, A. & Hardie, D. G. The glycogen-binding domain on the AMPK u03B2 subunit allows the kinase to act as a glycogen sensor. Cell Metab. 9, 23u201334 (2009)._ _ _ _ _ Langbein, L. et al. A novel epithelial keratin, hK6irs1, is expressed differentially in all layers of the inner root sheath, including specialized huxley cells (FlxFCgelzellen) of the human hair follicle. J. Invest. Dermatol. 118, 789u2013799 (2002)._ _ _ _ Bae, E., Kim, H. E., Koh, E. & Kim, K. S. Phosphoglucomutase1 is necessary for sustained cell growth under repetitive glucose depletion. FEBS Lett. 588, 3074u20133080 (2014)._ _ _ _ Philpott, M. P. & Kealey, T. Metabolic studies on isolated hair follicles: Hair follicles engaged in aerobic glycolysis and do not demonstrate the glucose fatty acid cycle. J. Invest. Dermatol. 96, 875u2013879 (1991)._ _ _ _ Jones, G. E., Limat, A. & Hunziker, T. Cultivation of keratinocytes from the outer root sheath of human hair follicles. Human Cell Cult. Protoc. (2003)._ _ Lisztes, E. et al. Adenosine promotes human hair growth and inhibits catagen transition in vitro: Role of the outer root sheath keratinocytes. J. Invest. Dermatol. 140, 1085u20131088 (2020)._ _ _ _ Shulman, R. & Rothman, D. The glycogen shunt maintains glycolytic homeostasis and the Warburg effect in cancer. Trends Cancer 3, 761u2013767 (2017)._ _ _ _ Klabunde, T. et al. Acyl ureas as human liver glycogen phosphorylase inhibitors for the treatment of type 2 diabetes. J. Med. Chem. 48, 6178u2013 6193 (2005)._ _ _ _ Ellis, R. A. W. M. Histology and cytochemistry of human skin. XV. Sites of phosphorylase and amylo-1, 6-glucosidase activity. J. Histochem. Cytochem. 6, 201u2013207 (1958)._ _ _ _ Adachi, K. & Uno, H. Glucose metabolism of growing and resting human hair follicles. Am. J. Physiol. 215, 1234u20131239 (1968)._ _ _ _ Kloepper, J. et al. Immunophenotyping of the human bulge region: The quest to define useful in situ markers for human epithelial hair follicle stem cells and their niche. Exp. Dermatol. 17, 592u2013 609 (2008)._ _ _ Garza, L. A. et al. Bald scalp in men with androgenetic alopecia retains hair follicle stem cells but lacks CD200-rich and CD34-positive hair follicle progenitor cells. J. Invest. Dermatol. 121, 613u2013622 (2011)._ _ Dienel, G. A. Does shuttling of glycogen-derived lactate from astrocytes to neurons take place during neurotransmission and memory consolidation?. J. Neurosci. Res. 98, 863u2013882 (2019)._ _ _ Ma, R. et al. A Pck1-directed glycogen metabolic program regulates formation and maintenance of memory CD8+ T cells. Nat. Cell Biol. 20, 21u201327 (2018)._ _ _ _ Zamaraeva, M. et al. Cells die with increased cytosolic ATP during apoptosis: A bioluminescence study with intracellular luciferase. Cell Death Differ. 12, 1390u20131397 (2005)._ _ _ _ Hardie, D. G. AMP-activated protein kinase-an energy sensor that regulates all aspects of cell function. Genes Dev. 25, 1895u20131908 (2011)._ _ _ _ _ BorbxEDrxF3, I. Activation of transient receptor potential Vanilloid-3 inhibits human hair growth. J. Invest. Dermatol. 131, 1605u20131614 (2011)._ _ _ _ \"- - Research presented in this article was funded by a PhD grant from Giuliani S.p.A., Milano, Italy. The authors thank Farjo Hair Institute for supplying the samples, Dr Jan Soetaert from the Blizard Advanced Light Microscopy, Rebecca Carroll from the Blizard Institute Pathology Facility for assistance with imaging.- - Authors and Affiliations - \"Centre for Cell Biology and Cutaneous Research, Blizard Institute, Barts & The London School of Medicine and Dentistry, Queen Mary University of London, 4 Newark Street, London, E1 2AT, UKKatarzyna Figlak_&_Michael P. PhilpottFarjo Hair Institute, London, UKGreg WilliamsMonasterium Laboratory, MxFCnster, GermanyMarta Bertolini_&_Ralf PausDr. Phillip Frost Department of Dermatology and Cutaneous Surgery, University of Miami Miller School of Medicine, Miami, FL, USARalf PausDermatology Research Centre, University of Manchester, and NIHR Biomedical Research Centre, Manchester, UKRalf Paus\" - \"AuthorsKatarzyna FiglakYou can also search for this author in _Greg WilliamsYou can also search for this author in _Marta BertoliniYou can also search for this author in _Ralf PausYou can also search for this author in _Michael P. PhilpottYou can also search for this author in _\" - Contributions - \\'K.F.: project conceptualization, investigation, data curation, validation, formal analysis, methodology, writing and editing, figure preparation. G.W. -resources, editing. M.B.-resources, methodology, editing. R.P.-conceptualization, funding aquisition, editing. M.P.-conceptualization, data curation, validation, project administration, funding aquisition, supervision, writing and editing. All authors reviewed and approved the final manuscript.\\' - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - RP serves as a consultant for Giuliani and is the founder and managing owner of Monasterium Laboratory. Other authors do not have any conflict of interest. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported license (). - Under new RAMP regulations, whale entanglement risk is assessed using data on marine life concentrations before the California Dungeness crab fishery opening and continuously until the season ends. Together with increasing whale populations and changing ocean conditions, increased risk of entanglement and more stringent regulations suggest that fishery closures and delays may become more frequent in coming years. Prior to whale entanglement mitigation closures, a multi-month delayed opening caused by a marine heatwave-induced HAB in the 2016 season caused significant economic impacts to fishers and the wider community, requiring $25 million in federal disaster financial assistance. New regulations imposing closures due to whale entanglement risk could bring similarly negative consequences for coastal fishing communities, but these potential losses have not been measured. - The Dungeness crab fishery has long been considered a reliable and economically lucrative fishery for US West Coast fishing communities. Populations of Dungeness crab fluctuate markedly from year to year and are driven by density-dependent biological mechanisms as well as exogenous environmental perturbations. Sea surface temperature, sea surface height and upwelling are among ecosystem level physical forces driving Dungeness crab populations, which are common drivers amongst other species in the CCS such as salmon populations. Thus, the status of Dungeness crab fishery is linked to the status of other CCS commercial fisheries through their relative responses to common physical drivers. The fishery is also strongly embedded within US West Coast fishery networks, with a high proportion of all fishers gaining revenue from Dungeness crab fishing while also participating in a number of other fisheries and therefore understanding impacts to this fishery is relevant to the management of other fisheries through cross-participation in the CCS. Estimating losses from the Dungeness crab fishery resulting from past closures due to whale entanglement risk will benefit ongoing risk mitigation planning, and will help to inform potential impacts of future events. - \"Our objective is to estimate retrospectively any revenue losses that occurred within the past 2 California Dungeness crab fishing seasons as a result of fishery closures prompted by increased whale entanglement risk. Here, we estimate lost revenues by applying an economic impacts model designed to measure revenue loss at the individual vessel level, which we then aggregate to the fishery level for two fishing management areas of California. We use hurdle models of crab revenue that estimate both the probability of a vessel participating in the fishery and the expected ex-vessel revenue contingent on participation. Models were fit with historical landings data, from 2011 to 2018, which included estimates of pre-season Dungeness crab abundance and vessel characteristics including frequency of participation, fishing strategy concentration index (as measured by the Herfindahlu2013Hirschman Index (HHI)), vessel size, latitude of fishing, proportion of revenues from Dungeness Crab and a binary variable which indicated whether a vessel switched between management areas within a fishing season. Retrospective loss estimates are determined by comparing observed to predicted revenues at the vessel level for the 2018u20132019 and 2019u20132020 fishing seasons. With this approach, which assesses the cumulative revenues through the season, we seek to quantify whether delays or closures implemented for whale entanglement regulation affect an overall seasons revenues. In measuring revenues cumulatively throughout the season, associated losses include revenues lost due to other disturbances to the season, such as crab quality delays, harmful algal blooms and whale entanglement risk. Quantifying the cost of closures will allow socio-economic impacts to be considered alongside ecological concerns within the decision-making process.\"- - \"We quantified direct economic impacts arising from delayed opening and early closures prompted by whale entanglement risk mitigation as well as crab meat quality concerns for two fishing seasons (2018u20132019 and 2019u20132020, subsequently referred to as 2019 and 2020 seasons) in the California Dungeness crab fishery (Fig._). Although whale entanglements have declined on the US West Coast since peaking during the large marine heatwave in 2014u20132016 (Fig._b), several delayed opening and early closures of the fishery were enacted to minimize whale entanglement risk (Fig._b). This provided a unique opportunity to assess economic impacts in a rapidly changing fishery and management landscape.\" - \"Our models incorporated pre-season crab abundance, which was estimated for the two management areas (north management area, NMA; and central management area, CMA; Fig._). Pre-season abundance of legal sized male Dungeness crab was estimated at 4.54 and 1.72 thousand tons for the 2019 season and 2.75 and 4.55 thousand tons for the 2020 season for the NMA and CMA respectively (Fig._b,c). All covariates except for the mean percent of crab within a vessels total revenues during the baseline period (2011u20132018 seasons) had a significant effect (pu2009 <u20090.05) on the selection (participation) model in the NMA. In the CMA, all covariates other than mean latitude of catches had a significant effect on participation (Supplementary Table ). Fishing strategy concentration (HHI) as well as mean latitude of catches and vessel length had a negative relationship with participation in both regions, suggesting higher probability of participation by vessels with diverse strategies, fishing at lower latitudes within regions and with smaller vessels. (Supplementary Table ). Mean latitude of crab catches however was not a significant variable in participation in the CMA.\" - \"Figure 2(a) Location of the California Commercial Dungeness crab fishery management areas in the California Current Ecosystem and its location with the USA and estimates of pre-season abundance of legal sized male Dungeness crab by season for (b) Northern and (c) Central California using a hierarchical depletion estimator model from Richerson et al. (2020). This figure, including the map in 2a, was created in R Studio (Version 4.0.3). The basemap and 200_m isobath were obtained from the rnaturalearth package (Made with Natural Earth. Free vector and raster map data @ naturalearthdata.com).\" - \"In the revenue model, all covariates other than the mean percent of crab within total vessel revenues were significant for the NMA (Supplementary Table ). All covariates other than fishing strategy concentration and switching had a positive sign with crab revenues. Participating in fishing in both management areas within a given season had a negative impact on total crab revenues in the NMA. This switching behavior had a negative relationship but statistically non-significant relationship with crab revenues in the CMA. All other covariates were significant in the crab revenues model for the CMA. Of the significant covariates, mean latitude and the diversity index had a negative relationship with crab revenues while all others showed positive relationships with crab revenues. Deviance explained by crab models, calculated using an analogue for R, was 0.39 for the NMA model and 0.28 for the CMA model. The model performed well overall at predicting revenues at the fishery level but over-estimated revenues for vessels that did not participate in the two seasons studied and underestimated revenues for some high earning vessels (see Supplementary Fig._au2013d). The hurdle model includes revenue estimates of all vessels in the fishery, including those that did not fish in the seasons for which we predict revenues, to provide an accurate total revenue estimate for the entire fishery. This accounts for vessels that chose not to fish due to the closures. These values however are less useful in describing the mean losses at the vessel level across vessels that actively participated as the calculated mean is skewed by the extremes of the distribution. Therefore, when presenting revenue losses at the vessel level, we report the 25thu2013 75th percentile of predicted revenues to remove the disproportionate influence of the extremes of the skewed distribution on the average expected losses.\" - \"In the CMA, the model predicts that the total crab fishery revenue would have been $9.4 million higher in the 2019 season and $14.4 million higher in the 2020 season in the absence of closures and other disturbances (Fig._, Table ). In the NMA, model results suggest that fishery revenues would have been $0.3 million higher than observed revenues in 2019 and $3.9 million higher in 2020 (Fig._, Table ). In the 2020 season in the NMA no whale entanglement mitigation was enacted so losses represent other factors influencing revenues such as the impact of COVID-19 on markets (NMFS 2021).\" - Figure 3Observed and retrospectively predicted revenues at the fishery level for the Commercial Dungeness crab fishery in California, within the Northern and Central Management Areas. This figure was created in R Studio (Version 4.0.3). Points show out-of-sample predictions of Dungeness crab revenue and demonstrate possible model prediction error of predictions made for the 2019 and 2020 fishing seasons. - Table 1 Observed, predicted and loss estimate values (million $) for revenues from the Commercial Dungeness crab fishery in California. - At the vessel level in the CMA, based on the 25th to 75th percentile of predicted revenues, the model predicted that vessel level revenues would have been on average $7.8 (SD $39.57) thousand higher in the 2019 season and on average $17.70 (SD $58.04) thousand higher in the 2020 season. At the vessel level in the NMA, based on the 25th to 75th percentile, the model predicted that vessel level revenues would have been on average $1.22 (SD $62.00) thousand higher in the 2019 season and $9.73 (SD $46) thousand higher in the 2020 season. - \"As large and small vessels may be impacted differently by fishery interventions, and there is large variability in vessel level revenues across the fishery, we also investigated losses predicted for small (<u200940 ft) and large vessels (u2265u200940 ft, the commonly used length cutoff for vessels in this fishery) separately. Median estimated revenue losses were similar between small ($22 thousand in 2019, $14 thousand in 2020) and large ($21 thousand in 2019, $17 thousand in 2020) vessels in the NMA. However, estimated losses as a proportion of a vessels mean historical revenues were higher for small (median of 33% in 2019, and 24% in 2020) than large vessels (median of 19% in 2019, 16% in 2020) (Fig._).\" - \"Figure 4Density plots of estimated percentage ex-vessel revenue loss as a proportion of a vessels mean annual revenue (2011u20132018) by small and large vessels in the Northern and Central Management Areas for the 2019 and 2020 fishing seasons. Median revenue losses are shown with dashed lines. Predicted revenues from within the 25th to 75th percentile are included. Negative numbers on the X axis indicate no revenue loss. The X axis has been restricted to between u2013_100 and 100 for plotting purposes.\" - \"In the CMA, revenue losses were larger for large vessels (median of $27 thousand in 2019 and $39 thousand in 2020) than small vessels (median $18 in 2019 thousand and $33 thousand in 2020). However, as a proportion of a vessels mean historical revenues, small vessels were estimated to have incurred larger percentage losses (median of 25% and 45% in 2019 and 2020) than large vessels (median of 23% and 37%) (Fig._).\" - \"Predicted fishery revenues in historical baseline (non-closure) years (in-sample predictions) were on average $0.47 (SD 5.79) million higher for the NMA model and $0.61 (SD 8.35) million higher for the CMA model than observed revenues suggesting some model error or unaccounted for variability. With the exception of 2019 predictions in the NMA, our predictions of losses during closure years are larger than the average model error. This possible error should be considered in the case of small revenue losses (e.g. NMA 2019) and as a proportion of larger predicted revenue losses. Average prediction residuals (observed-predicted revenue) at the vessel level are shown in (Supplementary Fig._c). We also provide out-of-sample prediction results. These were computed by iteratively holding out one year of data, fitting the model to the remaining data, and then making predictions for the held-out year (Supplementary Table ). Out-of-sample predictions typically over-predicted revenue in the NMA and under-predicted revenue in the CMA, with overpredictions larger in magnitude (Supplementary Table ). In the CMA, out-of-sample prediction errors ranged from a $22 million loss to a $8 million gain. Out-of-sample prediction errors in the NMA were smaller, ranging from a $10 million loss to a $14 gain. The largest out-of-sample prediction error was during 2017 in the CMA. This exceeded our prediction of a closure loss for both the 2019 and 2020 seasons, indicating that our predictions are quite uncertain. As the 2019 and 2020 seasons were predicted u201Cout-of-sampleu201D, some overestimation of revenues, and thus of losses may have occurred (Supplementary Table ). However, our predictions in 2019 and 2020 do align well with revenue predictions based on a 5-year average (Supplementary Table ), as is commonly done to calculate disaster assistance, which builds additional trust in our prediction models.\"- - \"Whale entanglements in fishing gear threaten whale populations, seafood production and long-term sustainability of commercial fisheries. While multiple mitigation strategies to reduce entanglements exist, there has been minimal consideration of the economic impact of these strategies. Here, we estimated retrospective losses to ex-vessel revenues for one of Californias most lucrative fisheries. Overall, we found fishery closures decreased ex-vessel revenue, with results showing some uncertainty due to large model prediction error. Regional differences in losses revealed interesting trends in the capacity for the fishery to recoup costs. For example, in the NMA, relatively small losses at the fishery level were predicted ($0.3 million in total) for the 2019 season despite an early closure to the season due to whale entanglement risk.\" - NMA fishers collectively were able to meet predicted revenue for the season despite a shortening of the fishing 2019 season. In the 2020 season however, the NMA did not experience disturbances due to whale entanglements but larger ex-vessel losses (of $3.9 million) were predicted. This suggests that other disturbances such as a delay to the season due to crab meat quality, lost fishing opportunity related to the COVID-19 pandemic, or other unknown factors, had an influence on ex-vessel revenue during the 2020 season. While most of the 2020 season landings in the NMA occurred before COVID-19 arrived in the US, there is evidence that prices in latter part of the season may have been depressed due to loss of export markets for live crab. - \"In the CMA however, despite landing the majority of crab available during the 2019 season (see Fig._c), losses of $9.4 million were experienced across the fishery. While total fishery catch was not greatly reduced, closure to the fishery in the spring may be responsible for revenue losses through other mechanisms (e.g. price). In the 2020 season, whale entanglement risk substantially shortened the fishing season in the CMA, through a delay at the beginning of the season and an early closure in the spring. Estimated losses were largest ($14.4 million) during this season. It is likely that the COVID-19 pandemic was also responsible for some of this estimated loss in the CMA in the 2020 season. Our model did not control for impacts of the COVID-19 pandemic. However, price trends suggest that that price of Dungeness Crab in California was not affected until mid-March 2020, at which point the fishery had caught 92% of the seasons catch (see Supplementary File ). Prices then returned to normal levels in mid-May. If we apply extrapolated prices between mid-March and mid-May by replacing observed prices with linearly increasing prices by week, revenues would have been $753,754 higher in total across the fishery. This rough estimate suggests we can attribute 4.1% of overall estimated revenue losses during the 2020 season to COVID-19 impacts, with the caveat that we do not know what prices would have been in the absence of the pandemic. A counterfactual approach has been used to disentangle multiple stressors to infer causal impacts of management interventions elsewhere, however as these closures, and the COVID pandemic, potentially impacted all fishers in the California Dungeness crab fishery, there are no control groups available for comparison and therefore this approach would not be appropriate.\" - \"Closures and other disturbances appear to have been less impactful in the NMA and high price for Dungeness crab may have contributed to the ability of vessels operating in the NMA to withstand disturbances (Supplementary Fig._). Prices were particularly high during the summer portion of the season in 2020 during which time the CMA was closed to Dungeness crab fishing (Supplementary Fig._ ). The NMA did not experience closures due to whale entanglement during 2020 and was predicted to have lower than average pre-season abundance (lower catch potential) during 2020 (see Fig._.b), while the CMA was predicted to have high catch potential for 2020 (Fig._.c), therefore differences in management measures implemented, and seasons catch potential, also contributed to differences in losses estimated.\" - \"The CMA also experienced high prices, including decadal high prices for crab during the Novemberu2013December of the 2019 fishing season (Supplementary Fig._). However, losses observed overall across the two seasons suggest the fishery, unlike the NMA, did not get much overall benefit from the high price in 2019 or the high pre-season abundance of crab (i.e. catch potential) estimated for the 2020 season in the CMA. A number of factors may have contributed to a poor season in the CMA including catchability or biology of Dungeness crab as well as external factors such as the COVID-19 pandemic behavioral choice factors, for example deciding not to fish. Temporally shifting or reducing the opportunity for participation through closed periods due to whale entanglement risk may have exacerbated other impacts on revenues in the CMA which were not as impactful on revenues in the NMA.\" - The high variability in estimated economic impacts per vessel reported here demonstrates that closures did not affect all vessels equally, similarly to impacts observed following a climate related harmful algal bloom in the 2016 season which were variable by vessel size and between communities. The estimated losses we present at the fishery level in the NMA and CMA may therefore be underestimated, or overestimated, for particular groups of vessels within those management areas. This reflects the diverse nature of the Dungeness Crab fishery in behaviour and fishing strategy and highlights the importance of capturing impacts at finer scales than the fishery level alone. - Limitations to the estimation of closure impacts - A limitation of the hurdle model is that there are other latent factors influencing fishery participation and revenues that our model does not incorporate, particularly those determining fisher behavior such as fuel price, shipyard backlogs and market demand. A behavioral choice model, for example one that incorporates location or fishing alternative choice given a closure would be a potential method to better understand how spatial management strategies affect fisher behavior and is recommended as a future analysis to assess trade-offs involving socio-economic risk. Our results, reporting losses from Dungeness crab fishing revenue only, also do not account for the ability of some fishers to mitigate revenue losses by participating in other fisheries. Dungeness crab fishing is highly connected within west coast fishery participation networks. Thus, it is important to note that our results for the 2019 and 2020 seasons present only losses from Dungeness crab fishing and may overestimate total annual revenue losses by some vessels that are able to mitigate impacts with participation in other fisheries. - The model, predicting out-of-sample, over-estimated revenues in recent years suggesting that our predictions of revenues may also be over-predicted. An improved estimation at the vessel level, given some over-estimation of vessels that did not fish, could be investigated through a selection model approach rather than a two-part model approach. However, two-part models are most appropriate for estimation of conditional (actual) outcomes as was intended here rather than unconditional (potential) outcomes and they do not require separate drivers for the selection and estimation model, which we did not have available. When the impacts of policy interventions are difficult to disentangle from other impacts, approaches such as a counterfactual synthetic control approach could be used to separate the impacts of the policy alone. In this context, however, it is useful to report the cumulative impact of disturbances given that these disturbances (e.g., delays due to crab quality, harmful algal blooms) happen frequently and therefore the closures will rarely happen in isolation. - Whilst there are limitations to our approach, revenue predictions presented here offer more insight compared to predicting revenues based only on a 5-year average of total fishery revenues (Supplementary Table ) as is commonly conducted to calculate disaster assistance requirement, as our analysis includes an estimation of crab abundance as well as historical vessel level data in its estimation. Accounting for the influence of crab abundance is critical in this fishery given abundance is highly variable and the majority of fishable biomass is taken each year. Estimation of revenue at the individual vessel level allows for consideration of fishery heterogeneity (e.g., by vessel size). Revenues calculated on a 5-year average would suggest total California Commercial Dungeness crab fishery revenues would have been $10.62 million higher than observed in 2019 and $12.73 million higher than observed in 2020 (Supplementary Table ). Thus, revenues estimated on the 5-year average suggest that losses would have been $0.97 million higher than our model prediction across the fishery for 2019 and $5.56 million lower than our model prediction for 2020. Our predictions suggest that delays and closures due to whale entanglement mitigation and other disturbances in to the 2019 and 2020 seasons were similar to the impact of closures due to the HAB in the 2016 season, which were estimated at $13.6 million in losses from Dungeness Crab revenues across the fishery. - Economic cost of mitigation - \"Many strategies that prevent fishery interactions with marine mammals exist, including gear reductions or modifications, depth limitations and dynamic or seasonal time-area closures. Whilst the fishery does implement pro-active gear modification measures set out in the best practices guide, only two management intervention options were enacted in the 2019 and 2020 seasons to mitigate against entanglements of marine life with Dungeness crab gear; delays to the start of the crab season in the winter and early closures in spring due to overlap with whale distribution in fishing grounds. These delays and closures can have differential impacts on the fishery as the fishing season is not heterogeneously prosperous. An example is that closures during the holiday season (Novu2013Dec) when Dungeness crab is traditionally consumed can cause substantial lost revenue opportunity for fishers at a time when price and demand are highest. The fishery operates as a derby in which the majority of revenues are made in the first month of the fishery being open. The strong seasonal dynamics of the Dungeness crab fishery, largely driven by rapid depletion of legal sized crab, mean that the timing of management actions can have important impacts on fishing revenues. Across the fishery, based on observed vessel level revenues during the 2011u20132018 baseline period, vessels earned an average of 62.33% (SD 24.04) of annual ex-vessel revenue during the first month of the season (15th Novu201315th Dec for the CMA/1st Decu201331st Dec for the NMA). After April 1st, vessels on average earn 10.54% (SD 18.98) of annual ex-vessel revenue. This average, based only on vessels that historically have actively participate past April 1, (283 vessels in the NMA, 346 vessels in the CMA) rises to 20.36% (SD 13.37) of ex-vessel revenue. Thus, while the majority of the overall fisheries revenue is taken at the start of the season, an April 1 closure could still have a substantial impact on the revenues of active fishing vessels in the spring. Determination of economic risk for the fishery, at a minimum, should consider timing of closures in addition to total revenue losses, in order to quantify losses that will be felt at the individual vessel level. We suggest further research to investigate how closures affect different groups of fishers through stakeholder participation.\" - \"Socio-economic impacts from whale mitigation measures could permeate into communities further than our analysis (based on ex-vessel revenue only) conveys, and further investigation into these community level impacts is necessary to understand and sustain an equitable fishery supply chain even where there is no absolute revenue loss. Some of the communities influenced by whale entanglement mitigation in California rely heavily on ocean resources for employment, through fishing occupations but also through hospitality and tourism. Managing this issue in a way that minimizes the burden on resource dependent communities is strongly in line with the objectives set out in the UN Sustainable Development Goals (SDGs), especially SDG 14 (life below water) but also related goals such as human well-being, reducing inequality and reducing the impacts of climate change.\" - Management Implications - Balancing socio-economic impacts against whale entanglement risk is challenging given the legally protected status of whale populations. However, potential economic losses reported here should motivate the development of mitigation measures (through cooperative innovation between industry, researchers and managers) that allow fishery production to be optimized whilst ensuring successful whale protection. At present, entire management areas, which constitute large regions of the coast, are closed in response to whale entanglement risk in California. Investigating how to minimize the spatiotemporal footprint of closures, such as by defining high risk zones dynamically based on fine-scale information of whale density and fishing effort, could provide an alternative mitigation structure. This could better consider the economic and conservation trade-offs while still being sensitive to changing environmental conditions. The introduction of dynamic zone closures, often broadly referred to as dynamic ocean management, has been demonstrated to reduce risk whilst minimizing lost fishing opportunities, especially when environmental variability is high or species have a dynamic distribution. Moreover, analysis of policy instruments to reduce whale entanglements with the American lobster fishery on the US Northeast coast found that economic costs of risk reduction could be 20% lower when mitigation decisions considered fishing opportunity costs alongside non-monetary benefits (biological risk), compared to non-monetary benefits alone. This is promising for the implementation of such strategies in the California Current System. - The caveat of this strategy is that dynamic zone closures require spatially and temporally explicit information on whale density and fishing effort which can be costly to attain. The use of ropeless gear has also been suggested as an alternative whale entanglement mitigation measure that requires further research and development before being initiated as an alternative regulatory tool. The costs of monitoring or technical advancements however may outweigh the financial and societal cost of fishery closures. Revenue losses for Dungeness crab estimated here for the 2019 and 2020 seasons are on par with losses experienced during the HAB period. During the delays to the 2016 fishing season an estimated $26.1 million was lost from ex-vessel revenues from all species that crab fishers target, including $13.6 million from Dungeness crab alone, requiring $25 million in government aid. Whale mitigation under the RAMP regulation will potentially delay or close the fishery year after year with uncertain economic impact that cannot be sustainably resolved with government aid. Development of tools to mitigate against economic loss while achieving whale protection will be necessary to come to a sustainable solution. This can only be achieved by first including economic loss in risk assessments. Doing so may also provide balance to partnerships between fishery managers and fishers. - Regulators are obligated to protect Humpback whales, blue whales and Leatherback turtles using the best available science. In this fishery, current triggers to open and close are based on a range of factors, but thus ultimately depend on the number of whales present within a management region. Regulators have a number of alternative regulatory options available to them, which include depth restrictions, gear restrictions or modifications and fleet advisories, if they can offer the same level of whale protection. Yet, the RAMP process lacks the socio-economic information needed to consider the socio-economic risk of regulatory actions, and that of the alternatives, to the fishing community. Results presented here highlight that the economic effects and that risk to fishing communities should be considered when designing whale entanglement mitigation programs. Having this economic information will facilitate the ability of managers, as set out in the RAMP regulation (subsection d4), to consider the socio-economic impact if deciding between management measures that equivalently reduce entanglement risk. - \"We have used two fishing seasons as an example of the economic impacts of these new whale entanglement regulations which will be implemented each year going forward. Synthesis of ex-vessel revenues is not a complete picture of the socio-economic impacts of regulations, but it provides a starting point for protecting both whales and fishing communities. While reported whale entanglements remain higher than pre-2014 totals, reported whale entanglements in California have declined markedly in the years following the 2014u20132016 large marine heatwave (Fig._ b). This is a success for this fishery and attributed to increased awareness, development of best practices for fishing gear and the mitigation program to protect whales. We now need to be successful at protecting and mitigating the socio-economic impacts to fishery participants and the fishing communities they support.\"- - The fishery - The California Dungeness crab fishery is divided into two management areas representing north and central (NMA and CMA, respectively), that have different opening dates. The Commercial Dungeness Crab fishing season typically spans from 1st December until 15th July in the NMA and from 15th November to 30th June in the CMA. The fishing season is named by calendar year in which the season ends (e.g. the 2020 season begins in late 2019 and continues into 2020). Open dates are sometimes adjusted due to crab meat quality, occurrence of harmful algal blooms or due to whale entanglement risk. Season open and close dates for each management area were compiled from online news reports and information provided by CDFW, then cross-checked with landings information following Richerson et al., 2020. - \"The California commercial Dungeness crab fishery has an estimated 501 active participants based on permits, however the number of vessels that have actively landed Dungeness crab between 2011 and 2020 ranges from 385 to 470 vessels in one season. The fishery has seen a rise in revenues in the past decade relative to the 2000u20132010 period, following a sharp rise relative to the 1970u2013 2000 period. In recent years the Dungeness crab fishery in California has seen annual ex-vessel revenues in excess of $80 million and landings in excess of 12,000 tons. The abundance of pre-season legal size males has also increased greatly in the CMA, with 2000u20132016 estimates of abundance averaging nearly five times greater than 1970u20132000 estimates. Abundance of pre-season legal size males in the NMA has been relatively stable. The fishery takes on average 83% and 65% of legal sized male crabs each year in the NMA and CMA, with the majority being landed in the first six weeks of the season.\" - \"The 2019 season began on time in the CMA on 15 November 2018. However, opening was delayed in the NMA because of crab meat quality (i.e., low weight), and landings began on 22 January 2019. Both management areas were closed to fishing on 15 April due to whale entanglement risk. The season was therefore shortened in the NMA by 53_days (almost 8_weeks) due to a crab meat quality delay and a further 91_days (13_weeks) at the end of the season due to whale entanglement risk. There were additional zone specific delays due to domoic acid presence which delayed fishing up to 55_days at the beginning of the season in the Northern portion of the NMA. In the CMA, the season was shorted by 76_days (11_weeks) due to whale entanglement risk at the end of the 2019 season. The northern portion of the CMA also lost an additional 23_days at the beginning of the 2019 season due to domoic acid presence.\" - \"In the 2020 season, the NMA was again delayed due to crab meat quality until 31 December (30_days) but was not closed due to whale entanglement risk in the 2020 season. The CMA experienced both a delay at the beginning of the season until 15th December (30_days) due to persistence of humpback whales and Pacific leatherback turtles on fishing grounds and closed early on 15 May due to entanglement risk, 46_days before the official close date. The season was shortened in the CMA in the 2020 season by 76_days in total.\" - Fishing and economic data - \"Data from individual vessel fish tickets (landings records) were provided by the California Department of Fish and Wildlife (CDFW) for all fisheries catches within the years 1981u20132020. To include only fishing trips targeting this species, fish tickets recording catches of Dungeness crab were included in the analysis if gear specified u201Ccrab or lobster trapu201D or u201Centrappingu201D . Fifty-one duplicate fish tickets were removed and 357, 599 fish tickets reporting catches of Dungeness crab were included. Fish tickets specify the CDFW fishing block area (catch area code) where fishing took place and the port of landing. Fish tickets were assigned to the Northern or Central Management Area by CDFW fishing block (Fig._a). If the fishing block recorded spanned both management areas, or if fishing block information was missing, the latitude of the port of landing was used (Fig._a). The two management areas are divided at the Sonoma/Mendocino County line (the Central Management Area isu2009<u200938.77xB0 latitude) (Fig._a-c).\" - \"All fish tickets recorded in one day by one vessel within one management area were counted as an individual fishing trip. Annual crab revenue per vessel was aggregated from fish tickets by vessel ID. Fish tickets with invalid vessel IDs were removed prior to analysis. Only fish tickets with landing dates falling within open season dates for each management area were included when calculating within revenues within each respective management area.\" - Model application - We follow the method introduced by Holland and Leonard (2020), used to predict California commercial Dungeness crab revenues during the 2016 season, which was disrupted by a long harmful algal bloom, and apply it to the same fishery during the 2019 and 2020 fishing seasons, which were interrupted by whale entanglement mitigation measures. Retrospective crab revenue estimates were calculated using a linear Cragg hurdle model (Stata15 TM). The hurdle model jointly estimates probability of participation, si (selection model) in the fishery and multiplies it by estimated revenue, hi, (revenues model) to predict the annual revenue per vessel. - \"$${Y}_{i}= {s}_{i}{{h}_{i}}^{*}$$ (1) \" - \\'where si is determined by the participation model:\\' - \"$${s}_{i}= left{begin{array}{c}1 if {z}_{i}Y+ {epsilon }_{i}>0 0 ,otherwiseend{array}right.$$ (2) \" - \\'and the latent variable ({{h}_{i}}^{*}) is observed if ({s}_{i})=1:\\' - \"$${{h}_{i}}^{*}={x}_{i}beta +{v}_{i}$$ (3) \" - \"As per Holland and Leonard (2020), the selection model is a binary logit model of participation choice, while the revenue model is a linear model of annual revenue per vessel. The dependent variable for the selection model is participation, where the dependent variable is 1 if revenueu2009>u20090 and 0 otherwise. The dependent variable for the expected revenue model is observed annual vessel revenue from Dungeness crab. The model was fit with data from the 2011u2013 2018 seasons (excluding 2016) to estimate revenues in the 2019 and 2020 seasons. This 7_year time period was chosen as a representative baseline period of levels of production in the recent history of the fishery given a change point in the 2011 season, especially in the CMA (Fig._c), in which catches by the fishery increased substantially and have since been sustained. We exclude the 2016 season from our analysis as a 5-month closure due to a harmful algal bloom severely reduced revenues in that season. We increase the baseline to 7_years, from the 5_years used by Holland and Leonard (2020), to increase the robustness of the model given this disturbance within our time period of interest. Vessels that had fished for Dungeness crab in California at least once during the 2011u2013 2018 seasons were included.\" - \"Explanatory variables for the participation model (({z}_{i})) and the conditional expected revenue model (({x}_{i})) included mean revenue for the vessel the prior 7_years (Mean Crab Revenue), the mean percent of a vessels total revenue made up of Dungeness Crab in the prior 7_years (Mean Percent Crab), the mean latitude the vessels crab was landed in the prior 7_ years weighted by revenue (Mean Latitude), the number of years the vessel fished in the prior 7_years (Years Fished) and an average index of diversification of a vessels diversification also over the prior 7_years (Concentration Index). The Herfindahlu2013Hirschman Index (HHI) was used as the index of the vessels diversification strategy, ranging from 0u20131 where 0 is the highest diversity in catches (revenues) and 1 is the least diverse or concentrated (catches of just 1 species). We refer to this as a concentration index. These covariates averaged over the prior 7_years thus used u201Cfish ticketu201D data from 2004 to 2010 to produce averages for the 2011 season, 2005u20132011 to produce averages for the 2012 season and so on. Vessel length was included as a covariate as a proxy for characteristics such as vessel power and crew size which influence a vessels catch potential.\" - \"Following Holland and Leonard (2020), pre-season abundance estimates of Dungeness crab were included as an explanatory variable in both models. Retrospective abundance estimates using the method from Richerson et al.(2020), updated to the 2019/2020 season, were calculated using a hierarchical linear mixed-effects depletion estimator model. The model uses the trend in catch per unit effort (retained weight per trip) per vessel over the fishing season to estimate the biomass of legal sized male Dungeness crab at the beginning of the fishing season. Fish ticket data from 1981 onwards was used to estimate this model. An index of crab abundance separately for the NMA and CMA for each season was derived from the model (Fig._b,c) and included as an explanatory variable in the relevant hurdle model. Separate models were run for revenues in the Northern Management Area and Central Management Area of California due to differences in management in the two regions. An additional variable of u201Cswitchingu201D was added to the ex-vessel revenues part of the hurdle model to account for vessels that fish in both management areas in a given season but do not have equal opportunity for fishing in both areas due to fair start rules which require a 30_day waiting period before transiting to land crab in another zone. We included this as a proxy variable of 0u2009=u2009no switching behavior, 1u2009=u2009vessel switched between two management areas in that season. Deviance explained by the model was calculated via an analogue to an R value as (1-(SSresidual/SStotal)) where SSresidual is the sum of squared difference between observed and predicted vessel revenues and SStotal is the sum of square differences between observed and mean vessel revenues.\" - \"To estimate retrospective total fishery revenues that would have been observed in the absence of closures and other disturbances to the season, hurdle models predicted revenues for the 2019 and 2020 seasons for all vessels that participated in the fishery during the 2011u20132018 seasons. Values predicted from the hurdle models were aggregated to the sum fishery total for the NMA and CMA. Estimated revenue losses were then calculated by subtracting the observed sum total of revenues for the 2019 and 2020 seasons from the predicted revenues. The 2019 and 2020 seasons revenues were predicted out-of-sample and therefore we provide a comparison of in-sample vs out-of-sample prediction of all years included within the model using a hold one out approach. To do this we iteratively held out one year of data, fit the model to the remaining data, then made predictions for the held-out year (Supplementary Table ).\"- - \\' \\' - Data from individual vessel fish tickets (landings records) were provided by the California Department of Fish and Wildlife (CDFW). Vessel-level landings data are confidential and the raw data cannot be made public. Data is available by CDFW upon request to CDFW (Paulo Serpa, Paulo.Serpa@wildlife.ca.gov). - \\' \\'- - \"Komoroske, L. M. & Lewison, R. L. Addressing fisheries bycatch in a changing world. Front. Mar. Sci. 2, 1 (2015).Lewison, R. L. et al. Ingredients for addressing the challenges of fisheries bycatch. Bull. Mar. Sci. 87, 235u2013250 (2011)._ _ _ Kirby, D. S. & Ward, P. Standards for the effective management of fisheries bycatch. Mar. Policy 44, 419u2013426 (2014)._ _ Carroll, E. L. et al. Reestablishment of former wintering grounds by New Zealand southern right whales. Mar. Mamm. Sci. 30, 206u2013 220 (2014)._ _ Jackson, J. A. et al. Have whales returned to a historical hotspot of industrial whaling? The pattern of southern right whale Eubalaena australis recovery at South Georgia. Endanger Species Res 43, 323u2013339 (2020)._ _ Calambokidis, J. & Barlow, J. Updated abundance estimates for blue and humpback whales along the U.S. West Coast using data through 2018 (2020).Curtis, K. A. et al. Abundance of Humpback Whales (Megaptera Novaeangliae) wintering in Central America and Southern Mexico from a one-dimensional spatial capture-recapture model. (2022).Clapham, P. J., Young, S. B. & Brownell, R. L. J. Baleen whales: Conservation issues and the status of the most endangered populations. Mamm. Rev. 29, 35u201360 (1999)._ _ Read, A. J., Drinker, P. & Northridge, S. Bycatch of marine mammals in US and global fisheries. Conserv. Biol. 20, 163u2013169 (2006)._ _ van der Hoop, J. M. et al. Assessment of management to mitigate anthropogenic effects on large whales. Conserv. Biol. 27, 121u2013 133 (2013)._ _ Thomas, P. O., Reeves, R. R. & Brownell, R. L. Status of the worlds baleen whales. Mar. Mamm. Sci. 32, 682u2013 734 (2016)._ _ Bisack, K. D. & Magnusson, G. M. Spatial management to reduce entanglement risk to north Atlantic right whales in fishing gear: A case study of US Northeast Lobster Fishery 2002u20132009. Front. Mar. Sci. 8, 1 (2021)._ _ Moore, M. J. How we can all stop killing whales: A proposal to avoid whale entanglement in fishing gear. ICES J. Mar. Sci. 76, 781u2013786 (2019)._ _ Davies, K. T. A. & Brillant, S. W. Mass human-caused mortality spurs federal action to protect endangered North Atlantic right whales in Canada. Mar. Policy 104, 157u2013 162 (2019)._ _ Meyer-Gutbrod, E. L., Greene, C. H. & Davies, K. T. A. Marine species range shifts necessitate advanced policy planning. Oceanography 31, 19u201323 (2018)._ _ Nicholas Record, B. R. et al. Rapid Climate-Driven Circulation Changes Threaten Conservation of Endangered North Atlantic Right Whales. vol. 32.Santora, J. A. et al. Habitat compression and ecosystem shifts as potential links between marine heatwave and record whale entanglements. Nat. Commun. 11, 1 (2020)._ _ Ingman, K. et al. Modeling changes in baleen whale seasonal abundance, timing of migration, and environmental variables to explain the sudden rise in entanglements in California. PLoS ONE 16, 1 (2021)._ _ Koubrak, O., VanderZwaag, D. L. & Worm, B. Saving the North Atlantic right whale in a changing ocean: Gauging scientific and law and policy responses. Ocean Coast. Manag. 200, 1 (2021)._ _ Hazen, E. L. et al. WhaleWatch: A dynamic management tool for predicting blue whale density in the California Current. J. Appl. Ecol. 54, 1415u20131428 (2017)._ _ Hazen, E. L. et al. A dynamic ocean management tool to reduce bycatch and support sustainable fisheries. (2018).Myers, R. A. et al. Saving endangered whales at no cost. Current Biology vol. 7 (2006).Reeves, R., Rosa, C., George, J. C., Sheffield, G. & Moore, M. Implications of Arctic industrial growth and strategies to mitigate future vessel and fishing gear impacts on bowhead whales. Mar. Policy 36, 454u2013 462 (2012)._ _ Lebon, K. M. & Kelly, R. P. Evaluating alternatives to reduce whale entanglements in commercial Dungeness Crab fishing gear. Glob. Ecol. Conserv. 18, 1 (2019)._ Ocean Protection Council. West Coast Entanglement Science Workshop Final Report. (2020).Farmer, N. A., Gowan, T. A., Powell, J. R. & Zoodsma, B. J. Evaluation of alternatives to winter closure of black sea bass pot gear: Projected impacts on catch and risk of entanglement with North Atlantic right whales eubalaena glacialis. Mar. Coast. Fish. 8, 202u2013221 (2016)._ _ Bisack, K. D. & Magnusson, G. M. Measuring management success for protected species: Looking beyond biological outcomes. Front. Mar. Sci. 3, 1 (2016)._ _ Fuller, E. C., Samhouri, J. F., Stoll, J. S., Levin, S. A. & Watson, J. R. Characterizing fisheries connectivity in marine social-ecological systems. ICES J. Mar. Sci. 74, 2087u20132096. (2017).NOAA Fisheries. NOAA Landings Tracker. (2021).California Department of Fish and Wildlife. 2021 California Commercial Fishing Regulations Digest. (2021).NOAA Fisheries. California Current Integrated Ecosystem Assessment - Whale Entanglement Indicators. (2022).Dungeness Crab Fishing Gear Working Group. California Whale Entanglement Discussion Dungeness Crab Fishing Gear Working Group Charter. - (2020).California Natural Resources Agency Department of Fish and Wildlife. Risk Assessment Mitigation Program Regulations. (2021).CDFW. Best Practices Guide For Minimizing Marine Life Entanglement. (2021).Ritzman, J. et al. Economic and sociocultural impacts of fisheries closures in two fishing-dependent communities following the massive 2015 US West Coast harmful algal bloom. Harmful Algae 80, 35u201345 (2018)._ _ Moore, S. K. et al. Harmful algal blooms and coastal communities: Socioeconomic impacts and actions taken to cope with the 2015 US West Coast domoic acid event. Harmful Algae 96, 1 (2020)._ _ Moore, S. K. et al. An index of fisheries closures due to harmful algal blooms and a framework for identifying vulnerable fishing communities on the US West Coast. Mar. Policy 110, 1 (2019)._ _ Holland, D. S. & Leonard, J. Is a delay a disaster? Economic impacts of the delay of the california dungeness crab fishery due to a harmful algal bloom. Harmful Algae 98, (2020).Thompson, C. J. Californias commercial fisheries: 1981u20132012. Mar. Fish. Rev. 77, 48u201372 (2015)._ _ Miller, R. R. et al. Spatial valuation of California marine fisheries as an ecosystem service. Can. J. Fish. Aquat. Sci. 74, 1732u20131748 (2016)._ _ Richerson, K., Leonard, J. & Holland, D. S. Predicting the economic impacts of the 2017 West Coast salmon troll ocean fishery closure. Mar. Policy 95, 142u2013152 (2018)._ _ Higgins, K., Hastings, A., Sarvela, J. N. & Botsford, L. W. Stochastic dynamics and deterministic skeletons: Population behaviour of dungeness crab. Science 1979(276), 1431u20131434 (1997)._ _ _ Botsford, L. W. & Lawrence, C. A. Patterns of co-variability among California Current chinook salmon, coho salmon, Dungeness crab, and physical oceanographic conditions. Progress in Oceanography vol. 53 (2002).Frawley, T. H. et al. Changes to the structure and function of an albacore fishery reveal shifting social-ecological realities for Pacific Northwest fishermen. Fish. Fish. 22, 280u2013297 (2021)._ _ Fisher, M. C., Moore, S. K., Jardine, S. L., Watson, J. R. & Samhouri, J. F. Climate shock effects and mediation in fisheries. Proc. Natl. Acad. Sci. 118, 2014379117 (2021)._ _ Feist, B. E., Samhouri, J. F., Forney, K. A. & Saez, L. E. Footprints of fixed-gear fisheries in relation to rising whale entanglements on the U.S. West Coast. Fish. Manag. Ecol. 28, 283u2013294 (2021).National Marine Fisheries Service. U.S. Seafood Industry and For-Hire Sector Impacts from COVID-19: 2020 in Perspective. (2021).Davies, T. K., Mees, C. C. & Milner-Gulland, E. J. Use of a counterfactual approach to evaluate the effect of area closures on fishing location in a tropical tuna fishery. PLoS ONE 12, 1 (2017)._ _ Mao, J. & Jardine, S. L. Market impacts of a toxic algae event: The case of california dungeness crab. Mar. Resour. Econ. 35, 1u2013 20 (2020)._ _ C. Haynie, A. & Layton, D. An expected profit model for monetizing fishing location choices. J. Environ. Econ. Manage 59, 165u2013176 (2010).DxE9palle, M., ThxE9baud, O. & Sanchirico, J. N. Accounting for fleet heterogeneity in estimating the impacts of large-scale fishery closures. Mar. Resour. Econ. 35, 361u2013378 (2020)._ _ Scheld, A. M., Goldsmith, W. M., White, S., Small, H. J. & Musick, S. Quantifying the behavioral and economic effects of regulatory change in a recreational cobia fishery. Fish. Res. 224, 1 (2020)._ _ Jardine, S. L., Fisher, M., Moore, S. & Samhouri, J. Inequality in the economic impacts from climate shocks in fisheries: the case of harmful algal blooms. (2020).Leung, S. F. & Yu, S. On the choice between sample selection and two-part models. ELSEMER J. Econ. 72 (1996).Gilman, E. et al. Robbing Peter to pay Paul: replacing unintended cross-taxa conflicts with intentional tradeoffs by moving from piecemeal to integrated fisheries bycatch management. Rev. Fish Biol. Fish. 29, 93u2013 123. (2019).The Sustainable Development Goals Report.Maxwell, S. M. et al. Dynamic ocean management: Defining and conceptualizing real-time management of the ocean. Mar. Policy 58, 42u201350 (2015)._ _ Pons, M. et al. Trade-offs between bycatch and target catches in static versus dynamic fishery closures. Proc. Natl. Acad. Sci. Am. 119, 1 (2021)._ Smith, J. A. et al. Lost opportunity: Quantifying the dynamic economic impact of time-area fishery closures. J. Appl. Ecol. 57, 502u2013513 (2020)._ _ Myers, H. J. et al. Ropeless fishing to prevent large whale entanglements: Ropeless Consortium report. in Marine Policy vol. 107 (Elsevier Ltd, 2019).Oliver, E. C. J. et al. Projected marine heatwaves in the 21st century and the potential for ecological impact. Front Mar. Sci. 6, 1 (2019)._ _ _ Richerson, K., Punt, A. E. & Holland, D. S. Nearly a half century of high but sustainable exploitation in the Dungeness crab (Cancer magister) fishery. Fish Res. 226, 1 (2020)._ _ NOAA Fisheries. California Dungeness Crab Pot Fishery - MMPA List of Fisheries. (2020).R Studio Team. RStudio: Integrated Development for R. Preprint at (2020).\"- - We thank CDFW for providing the vessel-level landings data, particularly Pheng Chang and Christy Juhasz for providing information, and John Rakitan at UCSC for drafting the data sharing agreement between CDFW and UCSC.- - \"Funding was provided by NOAAs SWFSC Center for Ecosystem Science Committee and NOAAs Integrated Ecosystem Assessment Program.\"- - Authors and Affiliations - \"Institute of Marine Sciences, University of California, Santa Cruz, CA, USARachel Seary,_Desiree Tommasi,_Steven J. Bograd_&_Stephanie BrodieFisheries Ecology Division, Southwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration, Santa Cruz, CA, USARachel Seary_ &_Jarrod A. SantoraEnvironmental Research Division, Southwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration, Monterey, CA, USARachel Seary,_Steven J. Bograd_&_Stephanie BrodieDepartment of Applied Math, University of California, 1156 High Street, Santa Cruz, CA, 95064, USAJarrod A. SantoraFisheries Research Division, Southwest Fisheries Science Center, National Marine Fisheries Service, National Oceanic and Atmospheric Administration, San Diego, CA, USADesiree Tommasi_&_Andrew ThompsonFishery Resource Analysis and Monitoring Division, Northwest Fisheries Science Center, Newport Field Station, 2032 S.E. OSU Drive, Newport, OR, 97365, USAKate RichersonConservation Biology Division, Northwest Fisheries Science Center, 2725 Montlake Blvd E., Seattle, WA, 98112, USADan Holland\" - \"AuthorsRachel SearyYou can also search for this author in _Jarrod A. SantoraYou can also search for this author in _Desiree TommasiYou can also search for this author in _Andrew ThompsonYou can also search for this author in _Steven J. BogradYou can also search for this author in _Kate RichersonYou can also search for this author in _Stephanie BrodieYou can also search for this author in _Dan HollandYou can also search for this author in _\" - Contributions - \"Conceptualization: R.S., J.A.S., D.T., S.J.B., A.T. Formal analysis: R.S., D.H. Methodology: D.H., K.R., S.B. Original draft preparation: R.S., J.A.S. Writingu2014 review and editing: all authors. Project administration: J.A.S., D.T., S.J.B., A.T.\" - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 Unported license, no changes were made\" - Vertebral complexity and organization - \"Vertebral complexity (range, polarization, and irregularity) and organization (concentration, smoothness, and clustering) were measured on the whole vertebra dataset, as well as two data subsets: centrum-only and arch-only. Values were optimized on the phylogeny using maximum likelihood to qualitatively examine patterns of variation in synapsid evolution (Fig._). Four metrics of complexity (degree of vertebral column differentiation) and organization (distribution of vertebrae along the column) showed an overall increase in synapsids, whereas one decreased then increased, and one showed no pattern, as described below.\" - \"Fig. 3Variation in vertebral complexity and organization. Based on the whole vertebra dataset optimized onto the sampled synapsid phylogeny using maximum likelihood. Evolutionary patterns of complexity: Range (a), Polarization (b), Irregularity (c). Evolutionary patterns of organization: Concentration (d), Smoothness (e), Clustering (f). For abbreviations see Supplementary Table_ . T, Therapsida; Cy, Cynodontia; Ma, Mammalia; Th, Theria; Bo, Boreoeutheria\" - \"Range and polarization (the spread of the data) are both generally lower in basal synapsids than in non-mammalian cynodonts and crown mammals, suggesting an overall increase through synapsid evolution (Fig._). The u2018pelycosaur Edaphosaurus also displayed relatively high values, relating to its exaggerated u2018sail consisting of elongated neural spines; high values were not recovered for the sail-backed Dimetrodon, presumably because the neural spines were excluded for the examined specimen due to damage. Xenarthrans displayed particularly low values. Irregularity (variation between adjacent vertebrae) did not support the hypothesis of evolutionary increase. Unlike the other two complexity metrics, u2018pelycosaurs exhibited the most irregular columns, followed by a decrease in non-mammalian therapsids and cynodonts, and a subsequent increase in some mammals (Fig._). Overall, patterns of complexity were similar in the centrum-only and arch-only subsets (Supplementary Figs._ and ).\" - \"Both smoothness (similarity of adjacent vertebrae relative to non-adjacent vertebrae) and clustering (patchiness, or degree to which the distribution of the vertebrae deviates from a uniform distribution) increased in synapsids (Fig._, Supplementary Figs._ and ), indicating that mammals have vertebral columns that cluster, but are integrated between adjacent vertebrae. Concentration (tendency of vertebral measurements to clump near the mean shape) varied little among synapsids (Fig._ , Supplementary Figs._ and ) indicating proportional changes in both range and polarization.\" - Evolutionary models - \"To test different evolutionary scenarios for producing mammalian vertebral complexity and organization, each of the seven evolutionary models were fit to the metrics across 60 phylogenetic trees reflecting variation in fossil branch length estimates. Based on the median AICc, the OU3B (Functional shift) model performed the best for both complexity and organization across all 60 trees (Fig._, Table_), receiving strong support based on AICc weights (complexity: 0.89, organization: 1.0) and supporting stepwise evolution of these traits. The other models performed relatively poorly, particularly the BM and BMtrend models, indicating that long-term uniform trends do not provide a good fit for the data compared to other models tested here.\" - \"Fig. 4Comparison of model performance based on AICc. Models compared across 60 trees for vertebral complexity (a) and organization (b). Results based on whole vertebra dataset. For model definitions see Table_. Source data are provided as a Source Data file\" - Table 2 Relative fit and rank order of evolutionary models for the whole vertebra dataset - \"Optima values were estimated for OU models to provide information regarding evolutionary mode, and the median and confidence intervals were calculated by jackknifing (Supplementary Table_). For all OU analyses, optima (u03B8) lay within the range of the observed data, rates (u03C3) were relatively high, and phylogenetic half-lives were short (Supplementary Tables_u2013). These results fit expectations of peak-type dynamics characterized by evolution toward an optimum value, as opposed to trend-like dynamics which are characterized by low rates and optima outside of the observed range.\" - \"In the best-fitting OU3B (Functional shift) model, optima differences were assessed based on the jackknifed confidence intervals. Significant increases between non-cynodont synapsids and cynodonts were observed for all measures except irregularity, for which there was a significant decrease (Supplementary Table_). By contrast, significant increases were only observed for polarization and smoothness at the shift between non-boreoeutherian mammals and boreoeutherians. Results of between-group ANOVAs on the tip values confirm this result (Supplementary Table_ ). There was a significant effect of grouping on range, polarization, smoothness, and clustering. Post-hoc Tukey HSDs revealed that there were significant differences for all the significant variables at the first shift, but only for polarization and smoothness at the second shift. Similarly, absolute values of increase were larger at the first shift than the second (Supplementary Table_). Despite this result, additional analyses allowing a single shift at Cynodontia and a single shift at Boreoeutheria confirmed that the OU3B model fit the data better than either shift on its own (Supplementary Tables_ and ).\" - Monte-Carlo simulations - \"Simulations were performed to test for significant differences between the best-performing model and the other models given phylogenetic uncertainty and taxonomic sample (Fig._, Table_). Each test compares the distribution of likelihood ratios (LR) generated under both the best model (Fig._: blue curve) and alternate model (Fig._: red curve) and compares the observed likelihood ratio (Fig._ : ertical line) to the alternate model distribution. If the jackknifed confidence intervals (Fig._: dotted line) on the LR lie outside of the 95% confidence interval for the null distribution (Fig._: shaded area) there is strong support for rejecting the null model.\" - Fig. 5Hypothesis tests between the functional shift and other models. Based on 1000 simulations for complexity (a) and organization (b). Top figure is an example plot. Red curves represent simulated distribution of Likelihood ratio (LR) under the null model and blue curves represent the simulated distribution under the test model. Vertical line indicates median LR recovered from jackknifing the observed data, with dashed lines indicating their confidence intervals. The shaded box represents 97.5% of the simulated null distribution. H0 is rejected in favor of H1 if the observed LR falls outside the significance threshold based on simulation (shaded box). Source data are provided as a Source Data file - Table 3 Hypothesis tests of OU3B (H1) against various null models (H0) for the whole vertebra dataset - \"Simulations on the whole dataset strongly supported the OU3B (Functional shift) model, suggesting it is significantly preferred over all the other models, based on both the median observed LR (Fig._: black line) and its lower confidence interval (Fig._: dotted lines, Table_). This approach considers both the topology and subsampling of the phylogeny. Therefore, these results suggest that the OU3B model is supported given the taxonomic sampling available.\"- - \"Four out of six measures of complexity (range, polarization) and organization (smoothness, clustering) show an increasing trend in synapsids (Fig._), supporting previous qualitative and quantitative studies. Range and polarization (reflecting differentiation of vertebrae), smoothness (reflecting integration between adjacent vertebrae), and clustering (reflecting modularity of vertebral regions) were generally higher in non-mammalian cynodonts and mammals than u2018pelycosaurs (Fig._). These results indicate that the degree of overall vertebral variation within each column increases while simultaneously becoming more organized as vertebrae are restructured into more distinct regions (Fig._). Thus, increases in variance in synapsids occurred in a highly-structured way and was not simply u2018scaled up in mammals. Further, differentiation of vertebrae in synapsid evolution did not occur at random with respect to position along the vertebral column. For example, smoothness is higher in mammals, suggesting that adjacent vertebrae are on average more alike relative to the mean, but clustering also increases, meaning that the distribution of vertebrae is uneven (Fig._). Taken together these results suggest both more integration between vertebrae within regions coupled with increased differences between regions in mammals. One interesting caveat to this pattern is that it likely only applies to terrestrial mammals. Secondarily aquatic mammals such as whales and sirenians have highly uniform vertebrae, accompanied by loss of observable regions in some cases, associated with their derived ecology and mode of locomotion. Although this study focuses on terrestrial mammals to provide a relevant comparison to non-mammalian synapsids, exploring the relationship between complexity and ecology within aquatic mammals will provide an interesting line of future inquiry.\" - \"Fig. 6Comparative morphology of vertebral columns. a A u2018pelycosaur , b a non-mammalian cynodont, and c a Boreoeutherian, illustrated using Principal Coordinates analysis. The vertebral columns of basal synapsids are relatively uniform (low range and polarization) but highly irregular. Through synapsid evolution, range and polarization increase, while columns become much smoother and more organized into clusters\" - \"Irregularity, which marks variation between adjacent vertebrae, decreases in synapsids (Fig._). The most basal synapsids, the u2018pelycosaurs, have elevated irregularity confirming prior findings of high centrum irregularity in the group (Fig._). Based on a more limited sample, Felice and Angielczyk proposed that irregularity of the centrum in basal u2018pelycosaurs may reflect looser functional constraints between vertebrae due to the larger relative size and presumed functional role of the notochord. Ancestrally, the notochord forms a significant portion of the vertebral column in early synapsids, but the size and craniocaudal extent of the notochord is reduced through synapsid evolution, with only an embryological remnant in mammals. In accordance with Felice and Angielczyks hypothesis, vertebral irregularity is reduced in non-mammalian therapsids and cynodonts that have a reduced notochordal contribution. However, analyzing the data subsets reveals that this pattern is manifest not only in the centrum that interacts directly with the notochord, but also in the arch measures, which might be hypothesized to be more independent of notochordal variation (Supplementary Figs._ and ). This finding is supported by observations of alternating variation in neural spine height between adjacent vertebrae in some u2018pelycosaurs, also implying irregularity of arch structures in basal synapsids. Therefore, the notochord hypothesis may be insufficient to explain patterns of irregularity. Instead, there appears to be increasing between-vertebra constraints across the whole vertebra (centrum and arch) at the transition from u2018pelycosaurs to therapsids, possibly associated with concurrent changes in regionalization. Concentration did not display any consistent trend, suggesting that the overall distribution of the vertebrae relative to the mean does not change systematically through evolutionary time. This indicates that increasing within-column disparity is driven by morphological divergence throughout the vertebral column, not by evolution of extreme morphologies of individual vertebrae or regions.\" - \"Passive diffusion and long-term driven trends cannot explain increasing complexity and organization in this dataset. Stephen J. Gould colorfully suggested that increasing complexity could be explained by a u2018drunkards walk , whose path is hindered by a wall and therefore tends toward the opposite direction. A key prediction of this hypothesis is that increases and decreases in complexity are equally likely at any given point in time. We modeled this hypothesis using a Brownian motion process in which complexity and organization varied randomly in both magnitude and direction over any given interval, but it was rejected by both model-fit and simulation-based hypothesis tests (Tables_ and , Figs._ and ). The release and radiate model, which restricts Brownian motion to just the crown group after assuming an OU process in the stem group, resulted in a considerable improvement in model fit but was significantly less supported than models with multiple OU optima. These results hold in spite of the relatively limited sampling of complete vertebral columns available in the fossil record, as their significance is based on a Monte-Carlo simulation approach that accounts for sampling bias using subsampling during hypothesis testing (Methods). Therefore, a random walk is insufficient to explain the patterns of vertebral column complexity and organization in synapsid evolution.\" - \"Alternatively, a long-term active trend of increasing complexity may be driven by continuous selection for niche differentiation, increasing entropy, or the natural tendency for similar parts to diverge through evolution. Active trends predict not only an increase in the mean and maximum total complexity across groups through geologic time (e.g., in metazoans and arthropods) but also a tendency for values to increase within lineages and in parallel along branches of the phylogeny (e.g., in crustaceans). If an active trend is driving mammal vertebral evolution, crown mammals should exhibit greater vertebral complexity than their non-mammalian synapsid ancestors. Furthermore, increases in complexity should be observed along the stem lineage and within sub-groups. Two findings show that the data do not support this hypothesis. First, the BMtrend model was poorly supported by both AICc and Monte-Carlo simulations, receiving even less support than a pure BM model (Tables_ and ). Second, OU models that fit the data well-recovered parameters for their optima within the range of the observed data (Supplementary Table_). Trend-like dynamics are signified in OU models by low u03B1 and u03B8 outside of the range of the data, representing slow evolution to a far-off optimum.\" - \"Taken together, these data show that evolution of the mammalian vertebral column is not best explained by long-term actively-driven increases acting throughout the evolutionary history of synapsids. Our results instead support prior observations made within the mammalian crown group. In comparing extant mammals to a paired u2018underived sister taxon representing the ancestral condition, McShea found that increases and decreases in vertebral complexity occurred with equal frequency. Although this approach is somewhat limited in its estimation of lineage-wise evolution (the degree to which the u2018underived taxa may represent the ancestral condition is debatable), it nevertheless highlights the absence of strong within-lineage trends in mammals despite elevated values relative to other major groups (i.e., when compared to fish or lizards as in ref.u2009 ), implying discontinuous change in different clades and periods of geologic time.\" - \"Instead of long-term, uniform trends, the data presented here most strongly support stepwise evolution of synapsid vertebral complexity and organization, indicating that evolutionary patterns vary across groups and through time. Similar stepwise evolutionary patterns have also been described for body size in archosaurs, dinosaurs, and pterosaursu2014another trait that has traditionally been thought of as displaying a long-term active trend (u2018Copes rule).\" - \"Stepwise trait evolution could be the result of selection, the release of constraints, or a combination of both processes. In a constraint-driven process, stepwise patterns can be generated in a system in which there is an underlying long-term trend (such as increasing entropy) by the sequential addition or removal of functional or developmental constraints that limit the generation of complexity. In this case, the transition between steps is marked by the evolution of novel phenotypes or developmental mechanisms that provide new opportunities to generate more complex morphologies. In a selection-driven process, stepwise patterns are instead generated by adaptation to novel selective regimes or release of functional constraints, relating to transitions in behavior, ecology, or functional complexity. An example of this type of mechanism would be the u2018ratchet hypothesis proposed by Stebbins, in which strong selection for complex morphologies occurs during adaptive radiations into new ecological niches, producing an evolutionary u2018ratchet of increasing complexity in successive clades. These two frameworks provide the end points of a continuum of evolutionary scenarios in which the generation of or selection for phenotypes may cause heterogeneity in evolutionary patterns or may combine in various ways through the evolutionary history of a clade.\" - \"We tested the role of developmental constraint using the Developmental shift (OU3A) model, where the evolution of vertebral regions drives increasing complexity and organization. Modularity has been argued to be u201Cone of the most critical features underlying the evolution of large and complex animalsu201D, and a driver of phenotypic and taxonomic diversity. Conversely, strong integration of serially-homologous structures through their development and evolution may act to restrict the generation of novel morphologies that can produce vertebral complexity. In the vertebral column, the expression of regionalizing Hox genes mark boundaries between phenotypically distinctive vertebrae and has been implicated in driving modularity. Likewise, in arthropods the evolution of Hox regulation along the anteroposterior axis and within-limb hierarchies has been proposed to have driven diversification of segment types associated with evolutionary increases in limb complexity. Mammal vertebral columns are strongly modular, and our recent work suggests that regionalization increased during synapsid evolution, implying increasing modularity through time. The Developmental shift model provided a relatively good fit to the data, but it was significantly less supported than the preferred model for both complexity and organization. Therefore, the evolution of new regions does not seem to be the trigger for increasing vertebral complexity and organization.\" - Instead, our results support the Functional shift (OU3B) model, where optima shifts are associated with increasing heterogeneity of the vertebral column. Vertebral heterogeneity in mammals, or the morphological differentiation of the vertebral column among regions, has been linked to functional specialization of vertebral regions, as well as ecomorphological diversification and increased rates of evolution. Therefore, this result supports the independence of the generation of regions from their morphological and functional differentiation, mimicking patterns found in the diversification of limb appendages in insects. In other words, developmental modularity may be prerequisite for vertebral complexity, but an additional selective trigger may also be required to realize vertebral diversity. Optima values suggest that the evolution of the synapsid vertebral column is characterized by a major shift in complexity and organization at Cynodontia in the late Permian, followed by a smaller increase in the Cretaceous within the northern placental mammals, the Boreoeutheria. Based on the phylogenetic location of these shifts and their correlation with various physiological and functional traits, as outlined below, we hypothesize that adaptation for increasing aerobic capacity in synapsids likely drove this stepwise pattern. - Mammals are endothermic, meaning they can metabolically produce body heat, a trait associated with sustained activity and locomotor stamina via enhanced aerobic capacity. Whereas ectotherms such as lizards perform well over short bursts, mammals can maintain much higher activity levels over long periods. In terms of locomotion, mammals can maintain sustained running speeds of around eight times that of a comparable lizard and use their higher stamina to forage widely. Understanding the evolutionary history of mammal endothermy is complicated because many associated traits are intercorrelated and difficult to trace in the fossil record. However, aerobic capacity for enhanced activity levels and locomotor stamina is one hypothesized selective factor for increasing basal metabolic rate (BMR). Although the precise evolutionary timing of mammalian endothermy remains controversial, the shift to a more active lifestyle is linked with numerous musculoskeletal specializations, many of which can be identified in non-mammalian cynodonts (see below). - \"Carrier directly implicated the vertebral column in adaptation for aerobic capacity as part of a functional complex whose role was to circumvent an ancestral locomotor constraint in tetrapods u2013 conflicting use of the axial musculature in both locomotion and respiration (Carriers constraint)u2014thus allowing mammals to move and breathe at the same time. These traits include a muscular diaphragm to free the body wall of ventilatory requirements, axial modifications that stabilize against lateral motion or provide increased independence of the vertebral musculoskeletal system from that of the ribs, and asymmetric mammalian gaits with sagittal bending that can facilitate aspiration. Released from this respiratory constraint, the mammalian vertebral column may have been free to diversify, enabling specialization of particular regions for new functions (e.g., sagittal bending occurs only in the posterior dorsal joints), driving increases in vertebral complexity and organization.\" - The largest optima shift in complexity and organization (indicating attraction toward a higher value) was recovered at Cynodontia and is consistent with multiple lines of fossil evidence that, taken together, point towards increased aerobic capacity in this group. For example, the origin of the muscular diaphragm is a key marker of locomotory-respiratory decoupling and ventilatory efficiency. While pectoral region differentiation associated with forelimb postural shifts in early therapsids may have provided the impetus for thoracic wall muscularization, fixation of the cervical count at seven implies the muscularized diaphragm did not arise until within cynodonts (e.g., Thrinaxodon). The evolution of respiratory turbinates, a mammalian feature associated with increased lung ventilation and body temperature, is inferred from bony ridges in the nasal cavity in both therocephalians and cynodonts, likely convergently; and the Early Triassic cynodont Thrinaxodon possessed a wide mammal-like nasal passage suitable for housing fully-developed turbinates. Non-mammalian cynodonts are also among several synapsid groups to evolve a secondary palate, a bony plate that divides the oral and nasal cavities linked with increased ventilation rate, and oxygen isotope data provide strong evidence of elevated body temperature among advanced cynodonts. Thus, multiple lines of evidence point to elevated activity levels and aerobic capacity in non-mammalian cynodonts. - \"The second, subtler, shift recovered in Boreoeutheria likely reflects a similar relationship with aerobic capacity via the repeated evolution of highly-aerobic cursorial specialists within the northern placental mammals. Among mammals, Boreoeutheria contains most extant diversity (around 5000 species), including ecologically-specialized clades, as well as all living cursorial placentals (e.g., horses, antelope, rabbits). Both running speed and maximal metabolic rate correlate with BMR, and body temperature correlates with metatarsal to femur ratio, a proxy for cursoriality, suggesting that high-stamina cursorial locomotion is an important driver of aerobic capacity. Whereas southern placental mammals (i.e., members of Atlantogenata) are characterized by lower BMR, lower body temperatures, and high developmental lability, boreoeutherians exhibit high developmental stability and pulses of u2018supraendothermy (body temperature above 38 degrees) associated with the evolution of cursoriality in numerous families.\" - \"To further test the aerobic capacity hypothesis, we gathered data on crown mammal BMR and body temperatureu2014physiological proxies of aerobic capacityu2014 and examined their correlation with our vertebral complexity and organization measures. Multivariate regressions revealed highly-significant correlations between both physiological parameters, complexity (BMR: Pillais pu2009 =u2009<0.001; temp: Pillais pu2009=u20090.001) and organization (BMR: Pillais pu2009=u2009<0.001; temp: Pillais pu2009=u2009<0.001). When phylogenetic correlations between taxa were incorporated using Phylogenetic generalized least squares (PGLS), correlations with complexity remained significant (Supplementary Fig._, Supplementary Table_, BMR: pu2009=u20090.019, rsqu2009 =u20090.13; temp: pu2009=u20090.005, rsqu2009=u20090.22), whereas those with organization were marginally insignificant (BMR: pu2009=u20090.13, rsqu2009 =u20090.08; temp: pu2009=u20090.065, rsqu2009=u20090.15). Therefore, data from extant species support a link between aerobic capacity and vertebral complexity in synapsids.\" - Evolutionary increases in morphological complexity have been widely discussed in the literature, yet empirical data testing these patterns are scarce. Using a dataset of exceptionally-preserved fossils and extant specimens, we tested competing hypotheses for the evolution of vertebral complexity and organization in synapsids. We reject the hypotheses of random passive increases or long-term evolutionary trends for increasing complexity. Instead, our data support stepwise evolution of complexity and organization via multiple evolutionary optima, suggesting clade-specific adaptations influenced complexity through geologic time. Stepwise evolution may represent increasing modularity coupled with an active trend, selection for stepwise adaptive optima relating to the evolution of novel mammalian vertebral functions, or a combination of both processes. Our results suggest that shifts in complexity did not coincide with the evolution of morphological regions, implying that developmental constraints are not limiting the evolution of complexity in synapsids. Instead, our data support a large shift in complexity and organization occurring at Cynodontia, followed by a smaller shift in Northern placental mammals (Boreoeutheria). Timing of the inferred evolutionary shifts, combined with fossil evidence and the correlation of complexity with BMR and body temperature in mammals, support aerobic capacity as a selective driver of stepwise patterns in synapsids. Thus, selection for higher activity levels combined with the release of respiratory constraints in cynodonts may have provided the trigger required to achieve vertebral complexity, and the subsequent biomechanical and ecological diversification of the presacral column in mammals.- - Sample - \"The sample included a broad taxonomic range of extant mammals (nu2009=u2009 35) and exceptionally-preserved fossil non-mammalian synapsids (nu2009=u2009 15) selected to encompass major stages in synapsid evolution, including u2018 pelycosaurs, basal therapsids, and cynodonts (Fig._, Supplementary Table_ ). All specimens were adult based on size and epiphyseal fusion and extant taxa were chosen to span a broad range of terrestrial ecologies across the mammalian phylogeny. Highly derived aquatic or flying taxa were not included as such specialized morphologies are not central to the synapsid-mammal transition. Non-mammalian mammaliaforms were not included due to the lack of appropriately-preserved three-dimensional material but the effect of excluding these taxa was considered in the subsampling simulations described below. The vertebral columns of all fossil specimens were complete or nearly complete (over 85% mean completeness, see ref. ), with minimal distortion.\" - Measurements - \"Fifteen linear measurements from all post-axial, pre-sacral vertebrae were taken from Jones et al. (Supplementary Table_). The measurement protocol captured variation in the centrum, neural arch, zygapophyses, and muscular processes. Transverse processes were broadly defined to include cervical transverse processes, diapophyses, and lumbar transverse processes. Mammal vertebrae are composed of two major ossification centers, the centrum (formed from the ancestral tetrapod pleurocentrum) and the neural arch, which have been hypothesized to form evolutionary modules. To examine variation across these modules, the evolutionary patterns of complexity and organization were compared centrum-only and arch-only data subsets (Supplementary Table_). The pleurocentrum and neural arch are the dominant centers of ossification in non-mammalian synapsids as well. Intercentra are retained in u201Cpelycosaursu201D and some therapsids (e.g., refs.u2009), but they are very minor components of the vertebral column with simple morphologies, and we excluded them from our analyses. Missing data were recorded as NA and were excluded in complexity and organization calculations. Following the previous studies, data were logged prior to analysis to enable comparison of variance within each column across specimens of different sizes. All analyses were conducted in R.\" - Metrics of complexity and organization - For each specimen, three metrics of vertebral complexity (range, polarization, and irregularity) and three metrics of organization (concentration, smoothness, and clustering) were calculated. Complexity metrics measure the total differentiation of the vertebral column, whereas the organization metrics quantify relative distribution of the vertebrae. Metrics were calculated following McShea with the addition of the clustering metric, but applied in a multivariate fashion over all the logged linear data. - Range is the maximum spread of the data and was calculated as the maximum multivariate Euclidean distance (ED). The ED between vertebrae i and h is calculated over p linear measures as in Eq. (). - \"$$ED_{{mathrm{i}},{mathrm{h}}} = sqrt {mathop {sum }limits_{{mathrm{j}} = 1}^{mathrm{p}} (x_{{mathrm{i}},{mathrm{j}}} - x_{{mathrm{h}},{mathrm{j}}})^2}$$ (1) \" - where xi,j is the jth measurement taken from the ith vertebra and xh,j is the jth measurement from the hth vertebra. - Polarization is the average spread of the data from the mean and was calculated as twice the mean Euclidean distance from the mean shape, (bar x), calculated using Eq. (). - \"$${mathrm{Polarization}} = 2mathop {sum }limits_{{mathrm{i}} = 1}^N ED_{{mathrm{i}},bar x}/N$$ (2) \" - Irregularity is the average spread of the vertebrae from their neighbors and was calculated as the average Euclidean distance between adjacent vertebrae. - \"$${mathrm{Irregularity}} = mathop {sum }limits_{{mathrm{i}} = 1}^{N - 1} ED_{{mathrm{i}},{mathrm{i}} + 1}/(N - 1)$$ (3) \" - All three complexity metrics were scaled by the total number of linear measures, P, for each specimen. This step was unnecessary for the organization metrics because complexity measures were already corrected, and clustering is unitless. - Concentration measures the degree to which the vertebrae clump toward the middle of the distribution versus occupying extreme morphologies. - \"$${mathrm{Concentration}} = {mathrm{range}} - {mathrm{polarization}}$$ (4) \" - Smoothness of the gradient along the column reflects the degree of integration between adjacent vertebrae. - \"$${mathrm{Smoothness}} = {mathrm{polarization}} - {mathrm{irregularity}}$$ (5) \" - Clustering is the relative deviation of the data from a uniform distribution and was measured using the Hopkins Statistic, H. This compares the nearest neighbor distances of the data (xi), to nearest neighbor distances from a randomly-simulated uniform distribution (y1). - \"$$H = frac{{mathop {sum}nolimits_{{mathrm{i}} = 1}^n {x_{mathrm{i}}} }}{{mathop {sum}nolimits_{{mathrm{i}} = 1}^n {y_{mathrm{i}}} + mathop {sum}nolimits_{{mathrm{i}} = 1}^n {x_{mathrm{i}}} }}$$ (6) \" - \"where H measures the uniformity of the data. Thus clustering is defined as 1-H. It was calculated using u2018get_clust_tendency in the package u2018 factoextra. The function was modified to permit missing data but exclude missing values from distance calculations (for more information see stats::dist).\" - Sensitivity analyses - \"To understand the impact of missing data and presence/absence data on our results, we conducted two sensitivity analyses. First, the influence of missing data on the complexity and organization metrics was examined by simulating missing data. For three complete specimens (Mus musculus, Tachyglossus aculeteus, Thrinaxodon liorhinus), data were randomly removed and metrics recalculated across 100 iterations. Data were removed element-wise (missing measurement), row-wise (missing vertebrae) and column-wise (missing variable), at 10, 20, and 30% levels (Supplementary Note_, Supplementary Fig._).\" - \"In mammal vertebral columns, some structures may be variably present or absent on different vertebrae along the column. For example, transverse processes are commonly absent on posterior thoracic (post-diaphragmatic) vertebrae. These absent structures were coded as missing (NA) in the main analysis to facilitate calculation of the above complexity and organization metrics. To examine how absent structures impacted the metrics, a second sensitivity analysis was conducted to assess the potential effect of excluding these serial absent structures (Supplementary Note_). Absent structures were included in the sensitivity analysis using two approaches: data were scaled to mean centrum length and absent structures were coded as a small number (0.01) following (u2018AbsSmall dataset) or they were coded as zero and an arbitrary unit of one was added to the whole dataset (u2018AbsOne dataset) (Supplementary Note_, Supplementary Figs._ and , Supplementary Tables_ and ).\" - Phylogenetic trees - \"To ensure proper node-date estimation given the limited fossil sampling, a time-scaled phylogeny of the sampled taxa (sample tree: Fig._) was generated by subsampling a much larger synapsid phylogeny (u2018master tree: Supplementary Fig._ ). The master tree was constructed as a composite phylogeny from three sources. First, positions and branch lengths of the 35 sampled extant species were taken from u2018timetree.org, which constructs relationships based on comparisons of all published molecular phylogenies of the taxa. Second, 56 fossil mammaliaforms and crown mammals were added at positions suggested by Slater based on combined molecular and morphological data. Branch lengths of the mammaliaform topology were scaled to time based on occurrence data from the paleobiology database (), using the packages u2018velociraptr and u2018paleotree and the function u2018timePaleoPhy using a minimum branch length of 1ma. Finally, an extensive phylogeny of 353 non-mammalian synapsids was constructed from several phylogenetic analyses of individual synapsid clades (e.g., refs.u2009 , but see Supplementary Note_ for all sources). Time bins for terminal taxa at the resolution of geologic stage were gathered from Brocklehurst and other literature sources, and the paleobiology database (Supplementary Note_). A population of 60 phylogenetic trees was then generated from u2018bin_timePaleophy using the minimum branch length method, in which u2018true first and last occurrence dates are randomly sampled from within their known time bin. The time-scaled mammaliaform phylogeny was bound to the non-mammalian synapsid phylogeny at the common taxon Sinocondon rigneyi to produce 60 master trees with 444 tips. These master phylogenies were finally subsampled to produce 60 sample trees (50 tips) with varying stem node ages (e.g., Fig._). Variation of the metrics was visualized by mapping them onto the phylogeny using maximum likelihood with the u2018contMap function of u2018phytools.\" - Evolutionary modeling - \"Six different multivariate evolutionary models were fit to the complexity and organization data separately using the R package u2018mvMORPH (Table_ ). Shifts were specified on the phylogeny by creating a SIMMAP tree using u2018 paintSubTree function in the package phytools. Alpha and sigma matrices were unconstrained, allowing for trait correlation. Model input parameters were as follows:\" - \"1.BMu2014Brownian motion with no trend. Function: u2018mvBM, model: u2018BM1.2.BMtrendu2014Brownian motion with increasing mean. Function: u2018mvBM, model: u2018BM1, param: u2018trendu2009 =u2009T.3.OU1u2014Selection toward a single optimum from lower root value. Function: u2018mvOU, model:OU1, param: u2018 rootu2009=u2009T.4.Release and radiate (RR)u2014Constrained evolution (OU) followed by release (BM). Function: u2018mvSHIFT, model: u2018 RR.5.OU2u2014Shift of optima at base of mammals. Function: u2018 mvOU, model: u2018OUM, param: u2018rootu2009=u2009F. 6.OU3Au2014 Two shifts at Therapsida and Theria. Function: u2018mvOU , model: u2018OUM, param: u2018rootu2009=u2009F.7.OU3B u2014Two shifts at Cynodontia and Boreoeutheria. Function: u2018mvOU , model: u2018OUM, param: u2018rootu2009=u2009F. Given the support for this model, an additional analysis was also conducted that included two additional models with shifts occurring at Cynodontia and Boreoeutheria separately. \" - \\'Models were fit across the 60 sample trees and the median values taken. Fit of the models was first assessed using the Akaike Information Criterion, corrected for small sample sizes (AICc), from which relative Akaike weights of the various models could be compared. Significance of the differences between optima values was assessed using jackknifed confidence intervals (see below) and Analysis of Variance with post-hoc Tukey HSD on the tip data (r functions: aov, TukeyHSD).\\' - Monte-Carlo simulations - \\'Monte-Carlo simulations of likelihood ratios were used to conduct pairwise hypothesis tests between specific models. Due to the computational demands of the analysis, the best-fitting model based on AICc was included as the test model (H1) and compared iteratively to the other less-supported models (H0). The approach taken was based on that applied by Boettinger et al., but applied multivariately. It quantifies the power and significance of the fit of a test model (H1, in the main dataset:OU3B) over a null model (H0) by simulating data under both scenarios and comparing them. Data were simulated 1000 times under four scenarios: the H0 model was run with data generated under the H0 model; the H1 model was run with data generated under the H0 model; the H0 model was run with data generated under the H1 model; and the H1 model was run with data generated under the H1 model. From these simulated likelihood values, likelihood ratios were generated between the H0 and H1 models under both a H0 and H1 scenario. The distribution of these ratios was then compared to assess the relative power of the test, given the phylogeny. P-values were calculated by comparing the observed likelihood ratio between H0 and H1 models with the distribution of ratios generated under the H0 scenario.\\' - Measuring vertebral complexity and organization requires very high skeletal completeness. Therefore, sampling potential among fossil taxa is limited. The Monte-Carlo simulations were further modified to simultaneously examine the impact of down-sampling on model selection, bias due to phylogenetic structure, and the potential impact of uncertainty associated with the dating of nodes. Data for each of the 1000 simulations were simulated on the more comprehensive master tree (444 tips, see above) but were repeatedly down-sampled to the sample tree (50 tips) prior to model fitting. To incorporate phylogenetic uncertainty, the tree used for the simulations was drawn at random from the available population of 60 phylogenetic trees for each simulation. To examine the influence of sampling on the observed likelihood ratio and optima values, a median likelihood value and confidence intervals were calculated by jackknifing the taxa while sampling across the 60 phylogenies. - \"The function u2018pmc for Monte-Carlo simulation hypothesis testing of evolutionary models was modified to include multivariate data (using u2018 mvMORPH models), phylogenetic uncertainty, and down-sampling as described above in a new function u2018mv_pmc_multiPhylo available at github:_ .\" - Multivariate regressions with BMR and body temperature - \"To test for a link between increasing complexity and organization and aerobic capacity in crown mammals, multivariate regression with physiological indicators of aerobic capacity was used. Basal metabolic rate (BMR) and resting body temperature were gathered from the literature for species or genera overlapping with our study. BMR is correlated with maximal metabolic rate, heart-rate, ecology and locomotor performance in mammals, whereas body temperature is correlated with aerobic scope, ATP generation by mitochondria and muscle-power output, making these measures useful proxies for aerobic capacity and stamina. BMR is strongly related to body size, so size was removed prior to analysis by taking the residuals from a log-log regression against body mass from the literature, including a quadratic term as recommended in ref. . BMR and temperature data were available for 23 and 18 of the mammal species examined in this study, respectively. An additional four and two taxa respectively were available for different species within the same genus and so, to maximize sampling, the genus mean was used. Multivariate regressions were performed on the six complexity or organization metrics against BMR and body temperature and significance were tested using the Pillais trace statistic. To account for the impact of phylogenetic signal on the relationship between the variables, phylogenetic generalized least squares (PGLS) was conducted using the u2018ProcD.pgls function in the package u2018geomorph. This approach uses randomization of model residuals, permuted across the tips of the phylogeny, to test the significance of multivariate relationships in a non-parametric framework. Relationships were visualized using the regression scores, from the u2018plot.ProcD.lm function, that represent the component of variation most highly correlated with the independent variable.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - \"Data required to replicate this study are available in Dryad and include raw measurements, complexity measures, fossil time ranges, and phylogenies. Raw measurements: ; other data: . The source data underlying Figs._ and and Supplementary Figs._ and are provided as a Source Data file.\" - \\' \\'- - \\' \\' - \"R functions used for Monte-Carlo simulations (u2018mv_pmc_multiPhylo) and to calculate the Hopkins Statistic with missing data (u2018get_clust_tendency ) are available at github .\" - \\' \\'- - \"Lineweaver C. H. & Davies P. C. Ruse M. Complexity and the arrow of time (Cambridge University Press, 2013).McShea, D. W. Metazoan complexity and evolution: is there a trend? Evolution 50, 477u2013492 (1996)._ _ Carroll, S. B. Chance and necessity: the evolution of morphological complexity and diversity. Nature 409, 1102 (2001)._ _ _ _ _ McShea, D. W. Complexity and evolution: what everybody knows. Biol. Philos. 6, 303u2013324 (1991)._ _ Lukeu0161 , J., Archibald, J. M., Keeling, P. J., Doolittle, W. F. & Gray, M. W. How a neutral evolutionary ratchet can build cellular complexity. IUBMB Life 63, 528u2013 537 (2011)._ _ _ _ Gould S. J. Full house. Harvard University Press (2011).McShea D. W. Evolutionary change in the morphological complexity of the mammalian vertebral column. Evolution, 47, 730u2013740 (1993).Adamowicz, S. J., Purvis, A. & Wills, M. A. Increasing morphological complexity in multiple parallel lineages of the Crustacea. Proc. Natl Acad. Sci. USA 105, 4786u2013 4791 (2008)._ _ _ _ _ Bonner J. T. The evolution of complexity by means of natural selection (Princeton University Press, 1988).Pierce, S. E. Non-equilibrium thermodynamics: An alternate evolutionary hypothesis. Crossing Boundariesu2014 Interdiscip. J. 1, 49u201359 (2002)._ McShea D. W. & Brandon R. N. Biologys first law: the tendency for diversity and complexity to increase in evolutionary systems (University of Chicago Press, 2010).Alroy, J. Understanding the dynamics of trends within evolving lineages. Paleobiology 26, 319u2013329 (2000)._ _ Benson, R. B., Hunt, G., Carrano, M. T. & Campione, N. Copes rule and the adaptive landscape of dinosaur body size evolution. Palaeontology 61, 13u201348 (2018)._ _ Stebbins G. L. The basis of progressive evolution (UNC Press Books, 2014).McShea, D. W. Functional complexity in organisms: parts as proxies. Biol. Philos. 15, 641u2013668 (2000)._ _ McShea D. W. Evolution of Complexity. Evolutionary Developmental Biology: A Reference Guide, 1u201311 (2017).Marcot, J. D. & McShea, D. W. Increasing hierarchical complexity throughout the history of life: phylogenetic tests of trend mechanisms. Paleobiology 33, 182u2013200 (2007)._ _ Cisne, J. L. Evolution of the world fauna of aquatic freeu2010living arthropods. Evolution 28, 337u2013366 (1974)._ _ _ Valentine, J. W., Collins, A. G. & Meyer, C. P. Morphological complexity increase in metazoans. Paleobiology 20, 131u2013142 (1994)._ _ Sidor, C. A. Simplification as a trend in synapsid cranial evolution. Evolution 55, 1419u20131442 (2001)._ _ _ _ McShea, D. W. A metric for the study of evolutionary trends in the complexity of serial structures. Biol. J. Linn. Soc. 45, 39u201355 (1992)._ _ Carrier, D. R. The evolution of locomotor stamina in tetrapods: Circumventing a mechanical constraint. Paleobiology 13, 326u2013341 (1987)._ _ Jenkins F. A., Jr. The postcranial skeleton of African Cynodonts. Bull. Peabody Mus. Nat. Hist. 36, 1u2013216 (1971).Kemp T. S. Mammal-like Reptiles and the Origin of Mammals (Academic Press, 1982).Schilling, N. & Hackert, R. Sagittal spine movements of small therian mammals during asymmetrical gaits. J. Exp. Biol. 209, 3925u20133939 (2006)._ _ _ Buchholtz, E. A. et al. Fixed cervical count and the origin of the mammalian diaphragm. Evol. Dev. 14, 399u2013411 (2012)._ _ _ Angielczyk K. D. & Kammerer C. F. in Handobook of Zoology: Mammalia: Mammalian Evolution, Diversity, and Systematics (eds Zachos F.E. & Asher R.J.). De Gruyter (2018).Butler, M. A. & King, A. A. Phylogenetic comparative analysis: a modeling approach for adaptive evolution. Am. Nat. 164, 683u2013695 (2004)._ _ _ Slater, G. J. Phylogenetic evidence for a shift in the mode of mammalian body size evolution at the Cretaceous-Palaeogene boundary. Methods Ecol. Evol. 4, 734u2013744 (2013)._ _ Jones, K. E. et al. Fossils reveal the complex evolutionary history of the mammalian regionalized spine. Science 361, 1249u20131252 (2018)._ _ _ _ _ Kemp T. S. The Origin and Evolution of Mammals (Oxford University Press, 2005).Jenkins, F. A. Cynodont postcranial anatomy and prototherian level of mammalian organization. Evolution 24, 230u2013252 (1970)._ _ Crompton, A. & Jenkins, F. A. Jr Mammals from reptiles: a review of mammalian origins. Annu. Rev. Earth. Planet. Sci. 1, 131u2013155 (1973)._ _ _ Buchholtz, E. A. & Gee, J. K. Finding sacral: Developmental evolution of the axial skeleton of odontocetes (Cetacea). Evol. Dev. 19, 190u2013204 (2017)._ _ _ Buchholtz E. A. The Emergence of Whales (Springer, 1998).Felice R. N. & Angielczyk K. D. in Early Evolutionary History of the Synapsida (eds Kammerer C. F., Angielczyk K.D., & FrxF6bisch J.) (Springer, 2014).Urban, J. P. G., Roberts, S. & Ralphs, J. R. The nucleus of the intervertebral disc from development to degeneration. Am. Zool. 40, 53u201361 (2000)._ Sumida, S. S. Reinterpretation of vertebral structure in the Early Permian pelycosaur Varanosaurus acutirostris (Amniota, Synapsida). J. Vert. Paleontol. 9, 451u2013458 (1989)._ _ Benson, R. B., Frigot, R. A., Goswami, A., Andres, B. & Butler, R. J. Competition and constraint drove Copes rule in the evolution of giant flying reptiles. Nat. Commun. 5, 3567 (2014)._ _ _ _ _ Sookias, R. B., Butler, R. J. & Benson, R. B. Rise of dinosaurs reveals major body-size transitions are driven by passive processes of trait evolution. Proc. R. Soc. B 279, 2180u20132187 (2012)._ _ _ _ Wellik, D. M. Hox genes and vertebrate axial pattern. Curr. Top. Dev. Biol. 88, 257u2013 278 (2009)._ _ _ _ Randau, M. & Goswami, A. Morphological modularity in the vertebral column of Felidae (Mammalia, Carnivora). BMC Evol. Biol. 17, 133 (2017)._ _ _ _ Jones, K. E., Benitez, L., Angielczyk, K. D. & Pierce, S. E. Adaptation and constraint in the evolution of the mammalian backbone. BMC Evol. Biol. 18, 172 (2018)._ _ _ _ Oliver, J. D., Jones, K. E., Hautier, L., Loughry, W. J. & Pierce, S. E. Vertebral bending mechanics and xenarthrous morphology in the nine-banded armadillo (Dasypus novemcinctus). J. Exp. Biol. 219, 2991u20133002 (2016)._ _ _ Slijper, E. J. Comparative biologic-anatomical investigations on the vertebral column and spinal musculature of mammals. Verh. K. Ned. Akad. Wet. Afd. Natuurkd Tweede Reeks 42, 1u2013128 (1946)._ Schilling, N. Evolution of the axial system in craniates: Morphology and function of the perivertebral musculature. Front. Zool. 8, 4u201323 (2011)._ _ _ _ Townsend, H. G. G. & Leach, D. H. Relationship between intervertebral joint morphology and mobility in the equine thoracolumbar spine. Equine Vet. J. 16, 461u2013465 (1984)._ _ _ _ Randau, M., Cuff, A. R., Hutchinson, J. R., Pierce, S. E. & Goswami, A. Regional differentiation of felid vertebral column evolution: a study of 3D shape trajectories. Org. Divers Evol. 17, 305u2013319 (2017)._ _ Ronshaugen, M., McGinnis, N. & McGinnis, W. Hox protein mutation and macroevolution of the insect body plan. Nature 415, 914 (2002)._ _ _ _ Ruben J., Hillenius W., Kemp T., Quick D. & Chinsamy-Turan A. The evolution of mammalian endothermy. Forerunners of mammals: radiation, histology, biology, 273u2013286 (2012).Kemp, T. S. The origin of mammalian endothermy: a paradigm for the evolution of complex biological structure. Zool. J. Linn. Soc. 147, 473u2013488 (2006)._ _ Bennett, A. F. & Ruben, J. A. Endothermy and activity in vertebrates. Science 206, 649u2013654 (1979)._ _ _ _ _ Hopson, J. A. The role of foraging mode in the origin of therapsids: implications for the origin of mammalian endothermy. Fieldiana.: Life Earth Sci. 5, 126u2013148 (2012)._ _ Perry, S. F., Similowski, T., Klein, W. & Codd, J. R. The evolutionary origin of the mammalian diaphragm. Respir. Physiol. Neurobiol. 171, 1u201316 (2010)._ _ _ Crompton, A., Owerkowicz, T., Bhullar, B.-A. & Musinsky, C. Structure of the nasal region of non-mammalian cynodonts and mammaliaforms: speculations on the evolution of mammalian endothermy. J. Vert. Paleontol. 37, e1269116 (2017)._ _ Hillenius, W. J. Turbinates in therapsids: evidence for Late Permian origins of mammalian endothermy. Evolution 48, 207u2013229 (1994)._ _ _ Rey, K. et al. Oxygen isotopes suggest elevated thermometabolism within multiple Permo-Triassic therapsid clades. eLife 6, e28589 (2017)._ _ _ _ SxE1nchez-Villagra, M. R. Why are there fewer marsupials than placentals? On the relevance of geography and physiology to evolutionary patterns of mammalian diversity and disparity. J. Mamm. Evol. 20, 279u2013290 (2013)._ _ Lovegrove, B. G. Locomotor mode, maximum running speed, and basal metabolic rate in placental mammals. Physiol. Biochem. Zool. 77, 916u2013928 (2004)._ _ _ Lovegrove, B. G. The evolution of mammalian body temperature: the Cenozoic supraendothermic pulses. J. Comp. Physiol. B 182, 579u2013589 (2012)._ _ _ Clarke, A., Rothery, P. & Isaac, N. J. Scaling of basal metabolic rate with body mass and temperature in mammals. J. Anim. Ecol. 79, 610u2013619 (2010)._ _ _ Clarke, A. & Rothery, P. Scaling of body temperature in mammals and birds. Funct. Ecol. 22, 58u201367 (2008)._ Hautier, L. et al. Patterns of ossification in southern versus northern placental mammals. Evolution 67, 1994u20132010 (2013)._ _ _ Asher, R., Lin, K., Kardjilov, N. & Hautier, L. Variability and constraint in the mammalian vertebral column. J. Evol. Biol. 24, 1080u20131090 (2011)._ _ _ _ Jones K. E. et al. Data from: Fossils reveal the complex evolutionary history of the mammalian regionalized spine. Dryad Digital Repository (2018).Randau, M. & Goswami, A. Unravelling intravertebral integration, modularity and disparity in Felidae (Mammalia). Evol. Dev. 19, 85u201395 (2017)._ _ _ Romer, A. S. & Price, L. I. Review of the Pelycosauria. Geol. Soc. Am. Spec. Pap. 28, 1u2013538 (1940)._ Rubidge B. S., Govender R. & Romano M. The postcranial skeleton of the basal tapinocephalid dinocephalian Tapinocaninus pamelae (Synapsida: Therapsida) from the South African Karoo Supergroup. J. Syst .Palaeontol. 17, 17u201323 (2019).R. R Development Core Team. A Language and Environment for Statistical Computing, (2009).Kassambara A., Mundt F. factoextra: Extract and Visualize the Results of Multivariate Data Analyses. R package version 105, (2017).Head, J. J. & Polly, P. D. Evolution of the snake body form reveals homoplasy in amniote Hox gene function. Nature 520, 86u201389 (2015)._ _ _ _ _ Hedges, S. B., Dudley, J. & Kumar, S. TimeTree: a public knowledge-base of divergence times among organisms. Bioinformatics 22, 2971u20132972 (2006)._ _ _ _ Bapst, D. W. paleotree: an R package for paleontological and phylogenetic analyses of evolution. Methods Ecol. Evol. 3, 803u2013807 (2012)._ _ Brocklehurst, N., Reisz, R. R., Fernandez, V. & FrxF6bisch, J. A re-description of u2018Mycterosaurus smithae, an Early Permian eothyridid, and its impact on the phylogeny of pelycosaurian-grade synapsids. PLoS. ONE. 11, e0156810 (2016)._ _ _ _ _ Sidor, C. A. & Hopson, J. A. Ghost lineages and u201Cmammalnessu201D : assessing the temporal pattern of character acquisition in the Synapsida. Paleobiology 24, 254u2013273 (1998)._ Kammerer, C. F. Systematics of the Anteosauria (Therapsida: Dinocephalia). J. Syst. Palaeontol. 9, 261u2013304 (2011)._ _ Angielczyk, K. D. & Kammerer, C. F. The cranial morphology, phylogenetic position and biogeography of the upper Permian dicynodont Compsodon helmoedi van Hoepen (Therapsida, Anomodontia). Pap. Palaeontol. 3, 513u2013545 (2017)._ _ Kammerer, C. F. & Masyutin, V. Gorgonopsian therapsids (Nochnitsa gen. nov. and Viatkogorgon) from the Permian Kotelnich locality of Russia. PeerJ 6, e4954 (2018)._ _ _ _ Huttenlocker, A. K. & Smith, R. H. M. New whaitsioids (Therapsida: Therocephalia) from the Teekloof Formation of South Africa and therocephalian diversity during the end-Guadalupian extinction. PeerJ 5, e3868 (2017)._ _ _ _ Van den Brandt, M. J. & Abdala, F. Cranial morphology and phylogenetic analysis Cynosurus suppostus (Therapsida, Cynodontia) form the upper Permian of the Karoo Basin, South Africa. Palaeontol. Afr. 52, 201u2013221 (2018)._ Pavanatto, A. et al. A new Upper Triassic cynodont-bearing fossiliferous site from southern Brazil, with taphonomic remarks and description of a new traversodontid taxon. J. South Am. Earth Sci. 88, 179u2013 196 (2018)._ _ _ MartxEDnez, R. N., Fernandez, E. & Alcober, O. A. A new non-mammaliaform eucynodont from the Carnian-Nornian Ischigualasto Formation, northwestern Argentina. Rev. Bras. Paleontol. 16, 61u2013 76 (2013)._ _ Martinelli, A. G., Eltink, E., Da-Rosa, A. A. S. & Langer, M. C. A new cynodont from the Santa Maria formation, south Brazil, improves Late Triassic probainognathian diversity. Pap. Palaeontol. 3, 401u2013423 (2017)._ _ Brocklehurst N. The Early Evolution of the Synapsida (Vertebrata, Amniota) and the Quality of Their Fossil Record (Humbolt-UniversitxE4t zu Berlin, 2015).Revell, L. J. phytools: an R package for phylogenetic comparative biology (and other things). Methods Ecol. Evol. 3, 217u2013223 (2012)._ _ Clavel, J., Escarguel, G. & Merceron, G. mvMORPH: an R package for fitting multivariate evolutionary models to morphometric data. Methods Ecol. Evol. 6, 1311u20131319 (2015)._ _ Boettiger, C., Coop, G. & Ralph, P. Is your phylogeny informative? Measuring the power of comparative methods. Evolution 66, 2240u2013 2251 (2012)._ _ _ _ White, C. R. & Seymour, R. S. Does basal metabolic rate contain a useful signal? Mammalian BMR allometry and correlations with a selection of physiological, ecological, and life-history variables. Physiol. Biochem. Zool. 77, 929u2013941 (2004)._ _ _ Clarke, A. & PxF6rtner, H. O. Temperature, metabolic power and the evolution of endothermy. Biol. Rev. 85, 703u2013727 (2010)._ _ Adams, D. C. & Otarola-Castillo, E. Geomorph: An R package for the collection and analysis of geometric morphometric shape data. Methods Ecol. Evol. 4, 393u2013399 (2013)._ _ Adams, D. C. A method for assessing phylogenetic least squares models for shape and other highu2010dimensional multivariate data. Evolution 68, 2675u20132688 (2014)._ _ _ Adams, D. C. & Collyer, M. L. Permutation tests for phylogenetic comparative analyses of highu2010dimensional shape data: what you shuffle matters. Evolution 69, 823u2013829 (2015)._ _ _ Drake, A. G. & Klingenberg, C. P. Large-scale diversification of skull shape in domestic dogs: disparity and modularity. Am. Nat. 175, 289u2013 301 (2010)._ _ _ \"- - \\'Funding for this work was provided by NSF grants EAR-1524523 (to S.E.P.) and EAR-1524938 (to K.D.A.), an American Association of Anatomists Postdoctoral Fellowship (to K.E.J), and Harvard University (to S.E.P.). We thank colleagues who facilitated collections access: R. Asher, J. Chupasko, M. Omura, J. Rosado, J. Cundiff, C. Capobianco, B. Zipfel, D. Brinkman, C. Mehling, A. Millhouse, K. Smithson, R. Tykoski, I. Werneburg, B. Brainerd, K. Jakata, T. Kemp, A. Resetar. We also thank D. Polly and B. Dickson for helpful discussions and J.Clavel for help with mvMorph.\\'- - Authors and Affiliations - \"Museum of Comparative Zoology and Department of Organismic and Evolutionary Biology, Harvard University, 26 Oxford Street, Cambridge, MA, 02138, USAKatrina E. Jones_ &_Stephanie E. PierceIntegrative Research Center, Field Museum of Natural History, 1400 South Lake Shore Drive, Chicago, IL, 60605-2496, USAKenneth D. Angielczyk\" - \"AuthorsKatrina E. JonesYou can also search for this author in _Kenneth D. AngielczykYou can also search for this author in _Stephanie E. PierceYou can also search for this author in _\" - Contributions - K.E.J contributed to study design, data collection, data analysis, and manuscript preparation; K.D.A contributed to study design, data collection, and manuscript preparation; S.E.P. contributed to study design, data collection, and paper preparation. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks Eli Amson, Lars Schmitz and the other, anonymous, reviewer for their contribution to the peer review of this work. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 license and Betsisman under Creative Commons CC0 license\" - \"Glycerol accumulated in our engineered strains during incubation with d-galUA (Supplementary Fig._), likely due to the poor utilization of glycerol by BY4741. Suspecting slow glycerol utilization to be limiting to growth, we transferred the pRJP1444 plasmid into a strain optimized for glycerol utilization (CEN.PK113-1A-GUT1JL1UBR2CBS), yielding strain yRJP224, and assayed for growth on d-galUA media. Excitingly, we observed higher saturating densities for yRJP224 in d-galUA media compared to our precursor strains and yRJP223, a wild-type CEN.PK strain expressing the catabolic pathway (Supplementary Fig._). To assay the utilization of d-galUA and resulting glycerol by these strains on pectin-rich agricultural waste, we received Naval orange peels as a gift from an industrial juicing facility and prepared CPW hydrolysate as previously described. We assayed growth of strains yRJP223 and yRJP224 in CPW hydrolysate media starting from low starting densities (OD600u2009=u20090.1) over 90u2009h. We observed d-galUA consumption for both strains (3.0u2009xB1u20090.2u2009gu2009L d-galUA and 5.1u2009xB1 u20090.3u2009gu2009L for yRJP223 and yRJP224, respectively) and growth during the d-galUA utilization phase after glucose was depleted (Supplementary Fig._ ). We found that the glycerol utilization strain, yRJP224, did not accumulate a final glycerol titer and co-consumed glycerol with d-galUA, whereas the wild-type CEN.PK strain, yRJP223, accumulated 4.1u2009xB1u20090.3u2009gu2009L glycerol.\" - Biphasic utilization of d-glucose and d-galUA - \"During growth of our engineered S. cerevisiae strains in acidic CPW hydrolysate, we observed minimal consumption of d-galUA (0.2u2009xB1u20090.1u2009gu2009 L and 0.1u2009xB1u20090.2u2009gu2009L for yRJP223 and yRJP224, respectively) until d-glucose was consumed after 21u2009h (Supplementary Fig._). To confirm this biphasic consumption pattern for the two monosaccharides, we incubated high density (OD600u2009=u200975) cultures of our engineered BY4741 d-galUA catabolic strain (yRJP064) in CPW hydrolysate and monitored consumption_of d-glucose and d-galUA (Fig._). We again observed limited uptake of d-galUA until d-glucose was fully consumed at 90u2009min. Despite the previously reported co-uptake of d-glucose and d-galUA by biochemical assays, this observation of biphasic consumption of d-glucose before d-galUA suggests that native yeast uptake mechanisms for d-galUA are not sufficient for co-utilization with d-glucose and poses a considerable problem for efficient utilization of d-galUA in pectin-rich hydrolysates. We hypothesized that expression of a specific, high-flux d-galUA transporter in yeast would be required for co-consumption of d-glucose and d-galUA.\" - Identification of a high-flux d-galUA transporter - \"While we previously described the heterologous d-galUA transporter GAT-1 from Neurospora crassa as a bona fide d-galUA transporter with activity in S. cerevisiae, GAT-1 demonstrates high-affinity, low-flux transport kinetics. We hypothesized that homologs of GAT-1 from various fungi (Fig._) would have diverse kinetic properties and could allow for a more rapid uptake of d-galUA in the presence of d-glucose. As an initial screen for functional transporter expression, we C-terminally tagged GAT-1 transporter homologs with a red fluorescent protein (mRuby) for yeast expression or with GFP for complementation trials of the u0394 gat-1 strain in N. crassa. Clades 1 and 2 contain known or putative fungal d-galUA transporters and we sampled five and four GAT-1 homologs from these groups, respectively (Supplementary Table_; Supplementary Data_; Supplementary Methods). Clades 3 and 4 contain known or putative quinate family transporters (QutD) and two homologs from these clades were assayed from organisms exhibiting pectin catabolism phenotypes. We observed plasma membrane trafficking in S. cerevisiae from the T. reesei Clade 1 homolog and A. niger and T. reesei Clade 2 homologs (Supplementary Fig._) and at least partial complementation of the u0394gat-1 phenotype in N. crassa by these homologs (Supplementary Fig._). To assay transport activity, we integrated untagged transporters into our original d-galUA catabolic strain (yRJP064) and incubated these cultures (OD600u2009=u200910.0) in synthetic complete media with d-galUA at pH 5.5, such that uronic acid transport by native mechanisms was restricted and d-galUA consumption could only be achieved by expression of a functional transporter. To control for possible toxicity of these transporters, we additionally incubated cultures at pH 3.5 (Supplementary Fig._). We identified An14g04280, or GatA, from A. niger as a transporter that allowed for 11.9u2009g/L consumption of d-galUA at high pH in our catabolic strain during the course of the assay (Fig._), whereas strains expressing no transporter and inactive d-galUA transporters showed negligible consumption. We assayed the kinetics of GatA with tritium-labeled d-galUA and measured a Km of 340u2009u03BCM and Vmax of 12.1u2009nmolu2009minu2009mg protein (Fig._ ). This is an approximately 50-fold increase in Vmax compared to the previously reported GAT-1 expressing strain, which has a Km of 1u2009u03BCM and Vmax of 0.256u2009nmolu2009minu2009mg protein.\" - \"Fig. 2Heterologous transporters enable uptake and utilization of d-galUA by S. cerevisiae_at high pH. a Homologues of the eukaryotic d-galacturonic acid transporter, GAT-1, include members of the quinate transporter family (QutD). b Transporters (bolded in A) were expressed in the catabolic strain, yRJP064, and assayed for d-galUA transport and consumption at pH 5.5 (initial OD600u2009=u200910). c A strain expressing GatA from Aspergillus niger exhibits high-flux d-galUA transport in tritiated d-galUA uptake assays with a Km of 340u2009u03BCM and Vmax of 12.1u2009nmolu2009min u2009mg protein\" - Engineering co-consumption of d-galUA and d-glucose - \"As stated previously, hydrolysates of pectin-rich biomass contain high concentrations of both d-galUA and d-glucose. After identifying that consumption of d-galUA was limited by the presence of d-glucose in our engineered catabolic strain using native transport mechanisms in CPW hydrolysate, we sought to assay the co-consumption of d-galUA and d-glucose in the catabolic strain, yRJP064, expressing the high activity A. niger d-galUA transporter, GatA. We additionally assayed the constitutive expression of yeast galactose transporter Gal2p, which is reported to have an extremely broad substrate range, including d-xylose, l-arabinose, d-fucose, d-glucose, d-fructose, d-mannose, and d-ribose. We incubated high density cultures (OD600u2009=u200975) of yRJP064 alone, yRJP064 expressing Gal2p (yRJP054), and yRJP064 expressing GatA (yRJP093), in acidic media containing only d-galUA or both d-galUA and d-glucose. We again observed that d-galUA is consumed at low pH in the absence of d-glucose and that expression of either transporter improves d-galUA consumption (Fig._). This suggests that Gal2p can mediate d-galUA uptake at low pH. However, when d-glucose was present in the media we observed complete inhibition of d-galUA consumption in the catabolic strain without a transporter, as well as when Gal2p was overexpressed (Fig._ ). Strikingly, the catabolic strain co-expressing the GatA transporter (yRJP093) consumed d-galUA in the presence of d-glucose at a rate similar to that of d-galUA consumption in the media without d-glucose (Fig._). d-glucose was present during the entirety of the uptake assay and d-glucose consumption rates were comparable across all strains (Fig._). Thus, heterologous expression of GatA should enable import of d-galUA from raw pectin hydrolysates.\" - \"Fig. 3Expression of GatA enables co-consumption of d-glucose and d-galUA. d-galUA utilization strains expressing no transporter (yRJP064), Gal2p (yRJP054) or GatA (yRJP093) were incubated in pH 3.5 media containing 20u2009g/L d-galUA (a) as the sole carbon source or b, c d-galUA and d-glucose at high density (OD600u2009=u200975). d Hexose transporters are necessary for consumption of d-galUA at low pH and GatA rescues consumption of a hexose knockout strain. Wild-type and hexose knockout (HXT1-7, GAL2) d-galUA utilization strains expressing no transporter (yRJP193, yRJP195) or GatA (yRJP194, yRJP196) were incubated in pH 3.5 media containing 20u2009g/L d-galUA (OD600u2009=u200910). Error bars represent standard deviation of biological triplicates\" - \"Our observation that d-glucose inhibited d-galUA consumption in wild-type S. cerevisiae strongly suggested competing mechanisms of transport between these sugars via hexose transporters. To investigate the involvement of hexose transporters in the wild-type uptake of d-galUA in acidic media, we introduced the fungal catabolic pathway into a strain, KY73, unable to import d-glucose due to knockouts of HXT1-7 and GAL2 yielding yRJP195 and assayed this strain for uptake of d-galUA at pH 3.5. The partial hexose knockout strain exhibited over fourfold lower total consumption of d-galUA compared to the wild-type strain containing a full set of native hexose transporters (yRJP193) (Fig._). The expression of GatA in this partial hexose knockout strain (yRJP196) rescued the consumption d-galUA. Thus, d-galUA must compete with d-glucose for import via hexose transporters unless a d-galUA-specific transporter, such as GatA, is heterologously expressed. Interestingly, while expression of GatA rescued d-galUA transport in the partial hexose knockout strain, we did not observe rescue of growth on d-glucose (Supplementary Fig_), suggesting that GatA has specificity for d-galUA transport and does not transport sufficient d-glucose to support growth.\" - \"Previous tritiated d-galUA uptake assays observed wild-type levels of d-galUA uptake in a hexose knockout strain and did not observe inhibition of d-galUA uptake at low pH by d-glucose. We repeated this assay and observed_both rapid association of d-galUA with yeast cells at low pH and that this effect was uninhibited by d-glucose (Supplementary Fig._). The biochemical explanation for this discrepancy between the clear inhibition of d-galUA uptake in our consumption assays and the tritiated uptake assay results is yet to be determined.\" - Conversion of d-galUA from CPW into meso-galactaric acid - \"Having demonstrated co-consumption of d-glucose and d-galUA in defined medium by expression of GatA, we applied this transporter strategy toward the direct production of a useful specialty chemical, meso-galactaric acid, from CPW hydrolysate, as outlined in Fig._. As a platform chemical with applications as a biodegradable chelate, cosmetic, pharmaceutical conjugate, and a biopolymer precursor, meso-galactaric acid is a promising target for bioconversion of pectin-rich wastes. In our previous study, we showed that yeast expressing uronate dehydrogenase (UDH) can oxidize d-galUA into meso-galactaric acid at xB5g/L yields. Since both d-glucose and d-galUA are present in high quantities in CPW hydrolysate, we hypothesized that expression of the GatA transporter will allow conversion of d-galUA to meso-galactaric acid via UDH even in the presence of d-glucose, which should provide ATP production and redox balancing via glycolysis to achieve higher productivities. Conversely, a strain expressing only UDH should show limited meso-galactaric acid production in the presence of d-glucose.\" - \"Fig. 4Production of meso-galactaric acid from citrus peel by engineered S. cerevisiae. a Citrus peel is enzymatically depolymerized to yield an acidic hydrolysate containing d-galUA and d-glucose. GatA transports d-galUA intracellularly in the presence of d-glucose. UDH oxidizes d-galUA to meso-galactarate, which accumulates in the media and is precipitated by acidification of the broth. Meso-galactarate is used directly as a renewable chelate and pharmaceutical conjugate. Efficient chemical conversions of meso-galactaric acid yield nylon and plastic monomers, adipic acid and 2,5-furandicarboxylic acid (FDCA), for polymerization and broad commercial applications. b Strains co-expressing UDH and GatA (yRJP067, triangles) exhibit higher conversion of d-galUA to meso-galactaric acid in acidic orange peel hydrolysate media in the presence of glucose compared to strains expressing only UDH (yRJP003, circles). Batch feeding of d-glucose (c) at 0 and 48u2009 h in CPW hydrolysate cultures increases meso-galactaric acid yields when expressing GatA (orange triangles), but inhibits yields without the transporter (green circles). Concentrations are normalized for glucose batch feeding. Error bars represent standard deviation of biological triplicates. Image sourced from Petr Kratochvil and licensed under CC0 1.0 Universal Public Domain Dedication by author\" - \"We inoculated CPW hydrolysate alone (measured d-glucose concentration 26.8u2009 gu2009L) or supplemented with additional d-glucose (starting concentration of 100u2009gu2009L) with S. cerevisiae cultures (initial OD600u2009=u20095.0) expressing either only UDH genomically integrated (yRJP003) or UDH and GatA genomically integrated (yRJP067) (Fig._; Supplementary Table_). In the d-glucose supplemented cultures, once the initial d-glucose was depleted, we added an additional 100u2009gu2009L d-glucose to promote continued fermentation at 48u2009h. At 8u2009h of fermentation, while glucose was present in all conditions, the conversion of d-galUA to meso-galactaric acid was observed in the UDH and GatA co-expression strain to be 0.12u2009xB1u20090.01 and 0.25u2009xB1u2009 0.02u2009gu2009Lu2009h with hydrolysate alone or glucose added, respectively. In the absence of GatA expression, no meso-galactaric product was observed at these time points as predicted by the expected lack of d-galUA import in the presence of d-glucose. After the d-glucose consumption phase in unsupplemented hydrolysate, further incubation of the UDH strains with or without GatA led to production of similar final concentrations of meso-galactaric acid. Excitingly, we found that supplementation of d-glucose increased meso-galactaric acid production in the UDH strain expressing the GatA transporter, which reached titers of 8.0u2009 xB1u20090.6u2009gu2009L in 80u2009h compared to the same strain that was not supplemented with d-glucose, which reached 3.2u2009xB1u20090.1u2009 gu2009L in 80u2009h. In d-glucose-fed conditions, a strain expressing only UDH showed a lower titer of 2.2u2009xB1u20090.4u2009gu2009L, likely due to the inhibition of d-galUA uptake by d-glucose. For both d-glucose-fed strains, we observed ethanol accumulation as a coproduct from d-glucose consumption (Supplementary Fig._). We measured higher relative ATP ratios in the GatA and UDH co-expression strain cultured in the presence of d-galUA and d-glucose (100%u2009xB1u2009 2) compared to d-galUA alone (25%u2009xB1u20091). Furthermore, the presence of d-glucose in the culture increased the NAD/NADH ratio (1.5u2009xB1u2009 0.1) compared to d-galUA alone (0.9u2009xB1u20090.1) (Supplementary Table_ ; Supplementary Methods). This suggests the role of d-glucose in maintaining higher concentrations of ATP and enabling NADH oxidation, thereby regenerating NAD for meso-galactaric acid production. To further investigate the role of d-glucose in balancing the NADH produced from UDH activity with NADH consumed by glycerol formation, we constructed a set of strains expressing GatA and increasing expression of UDH. We fermented these strains in d-glucose and d-galUA media and measured final meso-galactaric acid and glycerol media concentrations. We found that as UDH expression increased, both meso-galactaric acid and glycerol increased, indicating that d-glucose provided a redox sink for NADH production through glycerol formation (Supplementary Fig._, Supplementary Table_). We additionally observed ethanol accumulation in our d-glucose-fed hydrolysate conditions as a coproduct from d-glucose consumption (Supplementary Fig._).\"- - Vast global economic and environmental resources are directed toward the growth, harvesting, and processing of pectin-rich crops, including citrus fruits, apples, grapes and agave for juicing, and sugar beets, a top 10 world crop by mass, for sugar production. Large quantities of peel and pulp are already consolidated at processing facilities, partially treated, and more easily hydrolyzed than lignocellulosic material, making them promising feedstocks for fermentations. Furthermore, they offer the potential to produce products with dramatically reduced greenhouse gas emissions compared to existing bioprocesses, such as corn-ethanol, where roughly half the greenhouse gas emissions occur during crop management and collection. In this work, we provide tools enabling Saccharomyces cerevisiae to be used as a platform host for the utilization and bioconversion of primary sugars in pectin-waste hydrolysate. - \"We have identified several areas of yeast strain engineering that warrant future investigation and may further improve catabolism of the d-galUA in these pectin-rich feedstocks. First, we identified the preference for a d-galUA reductase capable of NADH cofactor utilization for growth on d-galUA as the sole carbon source (Supplementary Fig._). This suggests balancing d-galUA catabolism with host redox requirements may be a vital design feature. Secondly, we showed that inclusion of the d-galUA utilization pathway in an engineered glycerol-utilizing strain improved growth on d-galUA media and led to consumption of glycerol produced by the heterologous fungal catabolic pathway (Supplementary Fig._). Transient accumulation of glycerol in this strain during growth on CPW media suggests further improvements in glycerol catabolism will likely improve growth and d-galUA utilization. Third, flux was improved by the fortuitous 60-fold improvement in l-galactonate dehydratase activity we achieved by tagging the LGD1 enzyme with the Venus fluorescent protein. Uncovering the mechanism for this improvement may aid in expression of other dehydratases and further engineering of these enzymes.\" - \"The ability to co-utilize d-galUA and d-glucose by expression of a d-galUA-specific transporter will be necessary for economically-viable fermentations of pectin-rich wastes to maintain productivities in the presence of d-glucose. Sugar co-utilization has been a long-standing problem for yeast fermentations of numerous complex feedstocks, most notably lignocellulosics containing d-glucose and d-xylose. As evidence of this importance, we observed a biphasic pattern of d-galUA utilization and conversion to meso-galactaric acid in strains lacking heterologous expression of the GatA transporter in the presence of d-glucose (Fig._). Furthermore, co-import of d-glucose along with d-galUA can maintain the cell in a productive metabolic state. Heterologous expression of GatA and D-glucose co-feeding increased ATP and raised NAD/NADH levels (Supplementary Table_). A low NAD/NADH ratio in the absence of d-glucose is expected from the activity of UDH, which should deplete intracellular NAD and produce NADH. The addition of d-glucose, however, provides substrate to re-oxidize NADH, primarily by forming the reduced product glycerol. Indeed, higher glycerol concentrations were observed in glucose-fed GatA/UDH strains compared to UDH alone, and glycerol yields positively correlated with meso-galactaric acid titers (Supplementary Fig._) suggesting coupling between UDH and glycerol-3-phosphate dehydrogenase in the glycerol formation pathway. Similar findings of increased glycerol formation have been reported in engineered yeast strains overexpressing formate dehydrogenase during co-metabolism of d-glucose and oxidation of formate to carbon dioxide. Future opportunities include coupling d-galUA oxidation by UDH to the production of other reduced coproducts, such as alcohols and polyols.\" - Use of GatA for co-import of d-galUA and d-glucose has broader implications for the use of d-galUA in redox balancing applications with imbalanced d-glucose metabolic pathways. As first proposed by Van Maris et al., a heterologous reductive pathway converting d-galUA to central carbon metabolites in S. cerevisiae could be used to balance overproduction of NAD(P)H. For example, NADH generated by assimilatory metabolism during anaerobic fermentations of d-glucose to ethanol are naturally balanced by the NADH-consuming production of glycerol, resulting in a lower ethanol yield. The use of d-galUA as an external electron acceptor in engineered S. cerevisiae could offset this glycerol production and improve ethanol yields. - \"In addition to utilization and use in redox balancing applications, d-galUA is a unique starting substrate for bioconversions for production of commercially relevant aldaric acids, aldonic acids, and ascorbic acid (vitamin C). Aldaric acids, meso-galactaric acid and d-glucaric acid, have garnered considerable academic and industrial attention for their use as chelates in a broad range of markets, including cosmetics and pharmaceuticals. This earned d-glucaric acid status as a top target molecule from biomass by the Department of Energy in 2004. Both aldarics offer more effective chelation compared to existing bio-derived chelates, such as citrate, and are promising substrates for production of polyhydroxypolyamides (hydroxylated nylons), which have applications as biodegradable polymers, films and adhesives. These applications position meso-galactaric acid to be commercially relevant. Meso-galactaric acid currently sells for up to 100u2009USD per kg in cosmetics markets and, given its favorable chelation properties, has the potential to sell at prices comparable to higher volume, biobased chelates such as methylglycindiacetic acid (MGDA, 3u20135u2009USD per kg). The downstream processing of meso-galactaric acid is especially attractive, as it can be precipitated from the fermentation broth by acidification (Fig._). Simple and high purity isolation of meso-galactaric acid may enable effective one-pot chemical conversions to the drop-in plastic resin and nylon-6,6 monomers, 2,5-furandicarboxylic acid (FDCA) and adipic acid, respectively. FDCA-based poly(ethylene furonate) (PEF) is currently being pursued as a drop-in replacement for conventional, petroleum-based poly(ethylene terephthalate) (PET) plastic and demonstrates superior gas barrier and mechanical properties. Excitingly, scaling of meso-galactaric production from citrus peel waste could allow for production of citrus-peel based PEF plastic bottles for citrus juice packaging and replace the reliance on petroleum-based plastics.\" - The S. cerevisiae technologies described here provide platforms for diverting carbon in pectin-rich peel and pulp wastes globally toward diverse renewable chemical and consumer product markets. Such repurposing of current waste streams has the potential to create rural jobs, de-risk the historically volatile juice and sugar markets by accessing more isolated markets, and concurrently provide sustainable feedstocks for fermentation.- - Strains and growth conditions - \"Single gene expression plasmids were transformed into chemically competent TG1 E. coli and multigene plasmids were transformed into TransforMax EPI300 (Epicenter) electrocompetent E. coli. Selections were performed on LB containing kanamycin (25u2009mgu2009L). S. cerevisiae strain BY4741 (MATa his3u03941 leu2u0394 0 met15u03940 ura3u03940) was used for experiments in this study and propagated at 30u2009xB0C. Yeast strain KY73 (hxt1u2206::HIS3::u2206hxt4 hxt5::LEU2 hxt2u2206::HIS3 hxt3u2206::LEU2::hxt6 hxt7::HIS3 gal2u2206::DR* ura3u2013 52 his3u201311,15 leu2u20133,112 MAL2 SUC2 GAL MEL) was used for hexose knockout studies and propagated in YPM (10u2009gu2009L Bacto Yeast Extract; 20u2009 gu2009L Bacto Peptone; 20u2009gu2009L maltose). Wild-type yeast cultures were grown in YPD (10u2009gu2009L Bacto Yeast Extract; 20u2009gu2009L Bacto Peptone; 20u2009gu2009L d-glucose). Lithium acetate transformation method was used to transform yeast with plasmids containing the respective auxotrophic markers. Selection was performed on synthetic dropout media (6.7u2009gu2009L Difco yeast nitrogen base w/o amino acids; 2u2009gu2009L synthetic defined amino acid mix minus the respective autotrophy, w/o yeast nitrogen base (US Biological); 20u2009gu2009L d-glucose or the respective carbon source; 20u2009gu2009 L BD Difco agar was used for plates). pH was adjusted when appropriate with NaOH or HCl. A value of 0.21u2009mgu2009ODu2009mLu2009+/u2212 0.02 dry cell weight was calculated for triplicate cultures of BY4741 grown to mid-log in d-glucose synthetic complete media. Navel orange peel from fruits harvested in Winter of 2017 was received as a generous gift from Ventura Coastal, LLC (California, USA). Enzymatic hydrolysate of orange peel was prepared as previously described, with modification. Peel solids diluted in water (500u2009gu2009 L) were hydrolyzed for 24u2009h at room temperature by addition of pectinase (Sigma 17389) and cellulase (Sigma C2730) added at 0.5u2009mg enzyme per grams of total peel solids. For growth assays, hydrolysate was filtered through a fine mesh strainer and centrifuged before addition of 6.7u2009gu2009L Difco yeast nitrogen base w/o amino acids, 2u2009gu2009L synthetic defined amino acid mix, w/o yeast nitrogen base (US Biological) and sterilized by filtration. For meso-galactaric acid production, 2u2009gu2009L Difco yeast nitrogen base w/o amino acids, 2u2009gu2009L synthetic defined amino acid mix, w/o yeast nitrogen base (US Biological) and 100u2009gu2009L d-glucose, where indicated, was added. Sterile water was added to normalize sugar concentrations among hydrolysate batches. Growth of catabolic pathway strains in CPW hydrolysate media was performed in 50u2009mL cultures in 250u2009mL baffled flasks with 30u2009xB0C at 18u2009 xD7u2009g shaking in a New Brunswicku2122 InnovaxAE 44 shaking incubator.\" - Plasmids and cloning - \"A hierarchical golden gate cloning scheme was used for assembling coding sequence part plasmids, yeast protein expression cassettes and multigene plasmids. All protein coding sequences were synthesized or PCR amplified to omit internal BsaI and BsmBI sites for use in golden gate cloning. The protein coding sequences for fungal pathway enzymes were codon optimized for S. cerevisiae and synthesized Integrated DNA technologies (Coralville, IA). The protein coding sequences for uronate dehydrogenase from Pseudomonas syringae pv. tomato str. DC3000 and GAT-1 homologues were PCR amplified from genomic DNA or cDNA of respective fungal hosts. Single gene yeast expression plasmids for fungal enzymes (pRJP010, pRJP912, pRJP913, pER046, pRJP138, pRJP157, pER047, pRJP042, pRJP914, and pZYM005) and transporters (pRJP869-pRJP871, pRJP918-pRJP923, and pRJP1312) were cloned by BsaI golden gate reaction with respective yeast promoter and terminator pairs (Supplementary Data_) into a backbone containing ColE1 bacterial replication origin, Kanamycin resistance for bacterial selection and the respective yeast selection marker. Transporters from Botrytis cinerea B05.10 and Aspergillus nidulans were PCR amplified from cDNA with overhangs homologous to a yeast expression cassette (pTDH3-tADH1) targeting the Leu2 locus. PCR products and homologous yeast expression fragments were co-transformed into yeast for assembly by homologous recombination and colonies were selected for on SC-Leu dropout plates and confirmed by colony PCR. Multigene yeast expression plasmids (pRJP707, pRJP734, and pRJP915-917) were assembled from single gene cassettes in BsmBI golden gate reactions with their respective bacterial and yeast origins of replication and selection markers (Supplementary Table_). Primers used for Neurospora complementation strains cloning were list in Supplementary Data_.\" - Phylogenetic analysis - Fungal homologues of GAT-1, (NCU06026, quinate permease), from representative ascomycete and basidiomycete species were identified by BLAST and retrieved from the NCBI database. Protein sequences were aligned by Phylogeny.fr and a phylogenetic tree constructed using the program FigTree (). Gal2p and Hxt2p from S. cerevisiae were used as out-groups. - Sugar consumption and conversion assays and HPLC conditions - \"Yeast strains were grown to OD600u2009=u20091.0 in YPD or selective media, harvested and washed twice in sterile water. Cells were resuspended in assay media at the stated OD in 3u2009mL cultures and incubated in 24-deep well culture blocks at 30u2009xB0C at 16u2009xD7u2009g in an INFORS HT Multitron shaker with air-permeable seals. For meso-galactaric acid production assay in CPW hydrolysate media, strains were grown in YPD media overnight to saturation before washing twice in sterile water, pelleted and suspended in respective CPW hydrolysate media. Fermentations were conducted in 250u2009mL baffled flasks with 30u2009 xB0C at 18u2009xD7u2009g shaking in a New Brunswicku2122 InnovaxAE 44 shaking incubator. Batch feeding of d-glucose was performed by adding either a super-saturated 1000u2009gu2009L d-glucose solution or sterile water. For determination of media sugar concentrations, culture aliquots were pelleted and the supernatant was transferred to GC/MS vials for sampling. Fermentation samples for meso-galactaric acid analysis were adjusted to pH 10 with NaOH to solubilize sugar acids before HPLC analysis. _Media samples were analyzed by refractive index on a Shimadzu LC20AD HPLC equipped with a Rezex RFQ-Fast Acid H (8%), LC Column (100u2009 xD7u20097.8u2009mm) run with 0.5u2009mL per min 0.01u2009N H2SO4 mobile phase at 65u2009xB0C. Sugar concentrations were determined by comparing HPLC traces to a standard curve.\" - (H)-Galacturonic acid transport kinetics in yeast - \"Transport assays as described in Benz et al., with modification. Yeast strains were grown to an OD600 of 1.0 in YPD media, washed three times with ice cold assay buffer 30u2009mM MES-NaOH and resuspended to an OD of 80.0. To start transport reactions, cells were added to various (H)-d-galUA concentrations and incubated at 30u2009xB0C. At respective time points, 100u2009u03BCL of cell mixture was layered over 100u2009u03BCL of silicone oil (Sigma 85419) and reactions were stopped by spinning cells through oil for 1u2009min at 17,000u2009 xD7u2009g. Each tube was frozen in a bath of ethanol and dry ice and tube-bottoms containing the cell pellets were clipped off into vials containing 5u2009mL of Ultima Gold scintillation fluid. Cell solutions were solubilized at room temperature overnight and counts per minute (CPM) determined in a Tri-Carb 2900TR scintillation counter. (H)-d-galUA was purchased from ViTrax, Inc. (Placentia, CA, USA), and had a specific activity of 25u2009Ci per mmol and a purity of >u200999%. Kinetic parameters were determined by measuring the linear rate of (H)-d-galUA uptake over 5u2009min for d-galUA concentrations between 50 and 1000u2009u03BCM. Vmax values were normalized by total protein content using a Bradford assay with bovine serum albumin as a standard.\"- - \\' \\' - \"A reporting summary for this Article is available as a Supplementary_Information file. The data that support the findings of this study are available from the corresponding author upon reasonable request.\" - \\' \\'- - \"Keasling, J. D. Synthetic biology for synthetic chemistry. Acs. Chem. Biol. 3, 64u201376 (2008)._ _ _ Aristidou, A. & PenttilxE4 , M. Metabolic engineering applications to renewable resource utilization. Curr. Opin. Biotechnol. 11, 187u2013198 (2000)._ _ _ Willke, T. & Vorlop, K. Biotransformation of glycerol into 1,3-propanediol. Eur. J. Lipid Sci. Technol. 110, 831u2013840 (2008)._ _ _ Rude, M. A. & Schirmer, A. New microbial fuels: a biotech perspective. Curr. Opin. Microbiol. 12, 274u2013281 (2009)._ _ _ Tao, L. & Aden, A. The economics of current and future biofuels. Vitr. Cell. & Dev. Biol. - Plant 45, 199u2013217 (2009)._ _ Tilman, D. et al. Beneficial biofuelsu2014the food, energy, and environment trilemma. Science 325, 270u2013271 (2009)._ _ _ Klein-Marcuschamer, D. & Blanch, H. W. Renewable fuels from biomass: technical hurdles and economic assessment of biological routes. AIChE J. 61, 2689u2013 2701 (2015)._ _ _ US Department of Agriculture. Foreign Agricultural Service. Citrus: world markets and trade. (2015).FAOSTAT. Production quantities by Country. Food and Agricultural Organization of the United Nations, Statics Division (2013).LxF3pez, AngelSiles, Li, J., Q. & Thompson, I. P. Biorefinery of waste orange peel. Crit. Rev. Biotechnol. 30, 63u2013 69 (2010)._ _ Linskens, H. F. & Jackson, J. F. Analysis of Plant Waste Materials (Springer Science & Business Media, 2013).Kennedy, M. et al. in Analysis of Plant Waste Materials (eds Linskens, H. F. & Jackson, J. F.) 75u2013119 (Springer, Berlin, Heidelberg, 1999).Edwards, M. C. & Doran-Peterson, J. Pectin-rich biomass as feedstock for fuel ethanol production. Appl. Microbiol. Biotechnol. 95, 565u2013575 (2012)._ _ _ Xiao, C. & Anderson, C. T. Roles of pectin in biomass yield and processing for biofuels. Front. Plant Sci. 4, 67 (2013)._ _ Doran, J. B., Cripe, J., Sutton, M. & Foster, B. Fermentations of pectin-rich biomass with recombinant bacteria to produce fuel ethanol. Appl. Biochem. Biotechnol. 84-86, 141u2013 152 (2000)._ _ _ Neves, M. F., Trombin, V. G., Lopes, F. F., Kalaki, R. & Milan, P. in The Orange Juice Business: A Brazilian Perspective (eds Neves, M. F., Trombin, V. G., Lopes, F. F., Kalaki, R. & Milan, P.) 92u2013 96 (Wageningen Academic Publishers, 2012).Mohnen, D. Pectin structure and biosynthesis. Curr. Opin. Plant. Biol. 11, 266u2013277 (2008)._ _ _ Martens-Uzunova, E. S. & Schaap, P. J. An evolutionary conserved d-galacturonic acid metabolic pathway operates across filamentous fungi capable of pectin degradation. Fungal Genet. Biol. 45, 1449u20131457 (2008)._ _ _ Kuivanen, J., M. Jasmin Wang, Y. & Richard, P. Engineering Aspergillus niger for galactaric acid production: elimination of galactaric acid catabolism by using RNA sequencing and CRISPR/Cas9. Microb. Cell Fact. 15, 210 (2016).Rivas, B., Torrado, A., Torre, P., Converti, A. & DomxEDnguez, J. M. Submerged citric acid fermentation on orange peel autohydrolysate. J. Agric. Food Chem. 56, 2380u2013 2387 (2008)._ _ _ Grohmann, K., Baldwin, E. A. & Buslig, B. S. Production of ethanol from enzymatically hydrolyzed orange peel by the yeast Saccharomyces cerevisiae. Appl. Biochem. Biotechnol. 45-46, 315u2013 327 (1994)._ _ _ Abbott, D. A., Zelle, R. M., Pronk, J. T. & van Maris, A. J. A. Metabolic engineering of Saccharomyces cerevisiae for production of carboxylic acids: current status and challenges. Fems. Yeast. Res. 9, 1123u20131136 (2009)._ _ _ Benz, J. P. et al. Identification and characterization of a galacturonic acid transporter from Neurospora crassa and its application for Saccharomyces cerevisiae fermentation processes. Biotechnol. Biofuels 7, 20 (2014)._ _ Huisjes, E. H. et al. Toward pectin fermentation by Saccharomyces cerevisiae: expression of the first two steps of a bacterial pathway for D-galacturonate metabolism. J. Biotechnol. 162, 303u2013310 (2012)._ _ _ Mohd Azhar, S. H. et al. Yeasts in sustainable bioethanol production: a review. Biochem. Biophys. Rep. 10, 52u201361 (2017)._ _ _ Biz, A. et al. The introduction of the fungal d-galacturonate pathway enables the consumption of d-galacturonic acid by Saccharomyces cerevisiae. Microb. Cell. Fact. 15, 144 (2016)._ _ Souffriau, B. Transport and metabolism of D-galacturonic acid in the yeast Saccharomyces cerevisiae. Doctoral thesis, Katholieke Universiteit Leuven, Belgium (2012).Motter, F. A. et al. Categorisation of sugar acid dehydratases in Aspergillus niger. Fungal Genet. Biol. 64, 67u201372 (2014)._ _ _ Souffriau, B., den Abt, T. & Thevelein, J. M. Evidence for rapid uptake of d-galacturonic acid in the yeast Saccharomyces cerevisiae by a channel-type transport system. FEBS Lett. 586, 2494u20132499 (2012)._ _ _ Young, E. M., Comer, A. D., Huang, H. & Alper, H. S. A molecular transporter engineering approach to improving xylose catabolism in Saccharomyces cerevisiae. Metab. Eng. 14, 401u2013411 (2012)._ _ _ Reider Apel, A. et al. Evolved hexose transporter enhances xylose uptake and glucose/xylose co-utilization in Saccharomyces cerevisiae. Sci. Rep. 6, 19512 (2016).Enquist-Newman, M. et al. Efficient ethanol production from brown macroalgae sugars by a synthetic yeast platform. Nature 505, 239u2013243 (2014)._ _ _ _ Richard, P. & Hilditch, S. D-galacturonic acid catabolism in microorganisms and its biotechnological relevance. Appl. Microbiol. Biotechnol. 82, 597u2013604 (2009)._ _ _ Ho, P.-W., Swinnen, S., Duitama, J. & Nevoigt, E. The sole introduction of two single-point mutations establishes glycerol utilization in Saccharomyces cerevisiae CEN.PK derivatives. Biotechnol. Biofuels 10, 10 (2017)._ _ Zhang, L. et al. Genome-wide analysis of pectate-induced gene expression in Botrytis cinerea: identification and functional analysis of putative d-galacturonate transporters. Fungal Genet. Biol. 72, 182u2013191 (2014)._ _ _ Sloothaak, J., Schilders, M., Schaap, P. J. & de Graaff, L. H. Overexpression of the Aspergillus niger GatA transporter leads to preferential use of D-galacturonic acid over D-xylose. AMB Express 4, 66 (2014)._ _ Whittington, H. A., Grant, S., Roberts, C. F., Lamb, H. & Hawkins, A. R. Identification and isolation of a putative permease gene in the quinic acid utilization (QUT) gene cluster of Aspergillus nidulans. Curr. Genet. 12, 135u2013139 (1987)._ _ _ Kowalczyk, J. E. et al. Genetic interaction of Aspergillus nidulans galR, xlnR and araR in regulating d-galactose and l-arabinose release and catabolism gene expression. PLoS ONE 10, e0143200 (2015)._ _ Biely, P. & SlxE1vikovxE1, E. New search for pectolytic yeasts. Folia Microbiol. 39, 485u2013488 (1994)._ _ _ Kou, S. C., Christensen, M. S. & Cirillo, V. P. Galactose transport in Saccharomyces cerevisiae. II. Characteristics of galactose uptake and exchange in galactokinaseless cells. J. Bacteriol. 103, 671u2013678 (1970)._ _ _ _ Young, E., Poucher, A., Comer, A., Bailey, A. & Alper, H. Functional survey for heterologous sugar transport proteins, using Saccharomyces cerevisiae as a host. Appl. Environ. Microbiol. 77, 3311u2013 3319 (2011)._ _ _ Kruckeberg, A. L., Ye, L., Berden, J. A. & van Dam, K. Functional expression, quantification and cellular localization of the Hxt2 hexose transporter of Saccharomyces cerevisiae tagged with the green fluorescent protein. Biochem. J. 339(Pt 2), 299u2013307 (1999)._ _ _ MehtixF6, T. et al. Production and applications of carbohydrate-derived sugar acids as generic biobased chemicals. Crit. Rev. Biotechnol. 36, 904u2013 916 (2016)._ _ Kaliyan, N., Vance Morey, R. & Tiffany, D. G. Reducing life cycle greenhouse gas emissions of corn ethanol by integrating biomass to produce heat and power at ethanol plants. Biomass Bioenergy 35, 1103u2013 1113 (2011)._ _ _ Farwick, A., Bruder, S., Schadeweg, V., Oreb, M. & Boles, E. Engineering of yeast hexose transporters to transport D-xylose without inhibition by D-glucose. Proc. Natl Acad. Sci. USA 111, 5159u2013 5164 (2014)._ _ _ _ Geertman, J.-M. A., van Dijken, J. P. & Pronk, J. T. Engineering NADH metabolism in Saccharomyces cerevisiae: formate as an electron donor for glycerol production by anaerobic, glucose-limited chemostat cultures. Fems. Yeast. Res. 6, 1193u20131203 (2006)._ _ _ van Maris, A. J. A. et al. Alcoholic fermentation of carbon sources in biomass hydrolysates by Saccharomyces cerevisiae: current status. Antonie. Van. Leeuwenhoek 90, 391u2013418 (2006)._ _ _ Mojzita, D. et al. Metabolic engineering of fungal strains for conversion of D-galacturonate to meso-galactarate. Appl. Environ. Microbiol. 76, 169u2013175 (2010)._ _ _ Kuivanen, J. et al. Engineering filamentous fungi for conversion of D-galacturonic acid to L-galactonic acid. Appl. Environ. Microbiol. 78, 8676u20138683 (2012)._ _ _ Mapson, L. W. & Isherwood, F. A. Biological synthesis of ascorbic acid: the conversion of derivatives of D-galacturonic acid into L-ascorbic acid by plant extracts. Biochem J 59, ixu2013x (1954)._ Agius, F. et al. Engineering increased vitamin C levels in plants by overexpression of a D-galacturonic acid reductase. Nat. Biotechnol. 21, 177u2013181 (2003)._ _ _ Kuivanen, J., PenttilxE4, M. & Richard, P. Metabolic engineering of the fungal D-galacturonate pathway for L-ascorbic acid production. Microb. Cell. Fact. 14, 2 (2015)._ _ Tan, S. Z., Manchester, S. & Prather, K. L. J. Controlling central carbon metabolism for improved pathway yields in Saccharomyces cerevisiae. ACS Synth. Biol. 5, 116u2013124 (2016)._ _ _ Gupta, A., Reizman, I. M. B., Reisch, C. R. & Prather, K. L. J. Dynamic regulation of metabolic flux in engineered bacteria using a pathway-independent quorum-sensing circuit. Nat. Biotechnol. 35, 273u2013279 (2017)._ _ _ Moon, T. S., Yoon, S.-H., Lanza, A. M., Roy-Mayhew, J. D. & Prather, K. L. J. Production of glucaric acid from a synthetic pathway in recombinant Escherichia coli. Appl. Environ. Microbiol. 75, 589u2013595 (2009)._ _ _ Werpy, T. & Petersen, G. Top Value Added Chemicals from Biomass: Volume Iu2014Results of Screening for Potential Candidates from Sugars and Synthesis Gas. (2004).Saladini, M., Ferrari, E. & Menabue, L. Co-ordination of transition metal ions by galactaric acid: a potentiometric and spectroscopic study. J. Inorg. Biochem. 92, 121u2013127 (2002)._ _ _ Ciriminna, R., Meneguzzo, F., Delisi, R. & Pagliaro, M. Citric acid: emerging applications of key biotechnology industrial product. Chem. Cent. J. 11, 22 (2017)._ _ Kiely, D. E., Chen, L. & Lin, T.-H. Simple preparation of hydroxylated nylonsu2014polyamides derived from aldaric acids. ACS Symposium Series 575, 149u2013158 (1994).Chen, L. & Kiely, D. E. Synthesis of stereoregular head, tail hydroxylated nylons derived from d-glucose. J. Org. Chem. 61, 5847u20135851 (1996)._ _ _ Morton, D. W. & Kiely, D. E. Evaluation of the film and adhesive properties of some block copolymer polyhydroxypolyamides from esterified aldaric acids and diamines. J. Appl. Polym. Sci. 77, 3085u20133092 (2000)._ _ _ Mucic acid - Zauba.com. (2018).Trilon M. - Zauba.com. (2018).Zhang, H. et al. Production of adipic acid from sugar beet residue by combined biological and chemical catalysis. ChemCatChem 8, 1500u20131506 (2016)._ _ _ Taguchi, Y., Oishi, A. & Iida, H. One-step synthesis of dibutyl furandicarboxylates from galactaric acid. Chem. Lett. 37, 50u2013 51 (2008)._ _ _ Shiramizu, M. & Toste, F. D. Expanding the scope of biomass-derived chemicals through tandem reactions based on oxorhenium-catalyzed deoxydehydration. Angew. Chem. Int. Ed. Engl. 52, 12905u201312909 (2013)._ _ _ Eerhart, A. J. J. E., Faaij, A. P. C. & Patel, M. K. Replacing fossil based PET with biobased PEF; process analysis, energy and GHG balance. Energy Environ. Sci. 5, 6407 (2012)._ _ _ Mao, Y., Kriegel, R. M. & Bucknall, D. G. The crystal structure of poly(ethylene furanoate). Polym. (Guildf.). 102, 308u2013314 (2016)._ _ _ Daniel Gietz, R. & Woods, R. A. Transformation of yeast by lithium acetate/single-stranded carrier DNA/polyethylene glycol method. Methods Enzymol. 350, 87u201396 (2002).Lee, M. E., DeLoache, W. C., Cervantes, B. & Dueber, J. E. A highly characterized yeast toolkit for modular, multipart assembly. ACS Synth. Biol. 4, 975u2013986 (2015)._ _ _ Dereeper, A. et al. Phylogeny.fr: robust phylogenetic analysis for the non-specialist. Nucleic Acids Res. 36, W465u2013W469 (2008)._ _ _ \"- - This work was supported by the Energy Bioscience Institute and National Science Foundation Graduate Research Fellowship. We would like to thank Ventura Coastal, LLC, for providing raw orange peel for hydrolysate experiments and C. Somerville for critical discussion and suggestions. We would like to additionally thank Professor E. Nevoigt for providing the CEN.PK113-1A GUT1JL1UBR2CBS strain for glycerol utilization studies and J. Skerker and K. Wetmore for technical assistance.- - Authors and Affiliations - \"Department of Molecular and Cell Biology, University of California, Berkeley, CA, 94720, USARyan J. ProtzkoDepartment of Chemistry, University of California, Berkeley, CA, 94720, USALuke N. LatimerDepartment of Bioengineering, University of California, Berkeley, CA, 94720, USAZe Martinho_&_John E. DueberDepartment of Biotechnology and Biomedicine, Technical University of Denmark, 2800, Kgs Lyngby, DenmarkElise de ReusHolzforschung MxFCnchen, TUM School of Life Sciences Weihenstephan, Technische UniversitxE4t MxFCnchen, Freising, GermanyTanja Seibert_&_J. Philipp BenzMax Planck Institute of Molecular Plant Physiology, Potsdam, GermanyTanja SeibertBiological Systems & Engineering Division, Lawrence Berkeley National Laboratory, Berkeley, CA, 94720, USAJohn E. Dueber\" - \"AuthorsRyan J. ProtzkoYou can also search for this author in _Luke N. LatimerYou can also search for this author in _Ze MartinhoYou can also search for this author in _Elise de ReusYou can also search for this author in _Tanja SeibertYou can also search for this author in _J. Philipp BenzYou can also search for this author in _John E. DueberYou can also search for this author in _\" - Contributions - R.J.P. and J.E.D. conceived and supervised overall project. R.J.P., Z.M., and E.d.R. performed cloning, enzyme activity screening and constructed yeast strains. R.J.P. performed growth assays, uptake assays, transporter kinetics and assembled GAT-1 phylogeny. R.J.P. and Z.M. performed microscopy on yeast strains for transporter localization. J.P.B. and T.S. performed microscopy on N. crassa and complementation assays for transporter localization and function. R.J.P. and L.N.L. prepared hydrolysates from CPW and performed bioconversion assays. R.J.P., L.N.L., and Z.M. performed HPLC analysis on fermentation samples. R.J.P., L.N.L., J.P.B., and J.E.D. wrote and revised manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - R.J.P. and J.E.D. are authors on a patent application filed by the University of California at Berkeley Intellectual Property & Industry Research Alliances office on this work. R.J.P., L.N.L., and J.E.D. have financial interest in ZestBio, Inc. The remaining authors declare no competing interests. - \\' \\'- - \"Publishers note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 unported license ().\"- - Author notes - \\'These authors contributed equally: Dan Zhao, Kerui Wu.\\' - Authors and Affiliations - \"Department of Cancer Biology, Wake Forest University School of Medicine, Winston Salem, NC, 27157, USADan Zhao,_Kerui Wu,_Sambad Sharma,_Fei Xing,_Shih-Ying Wu,_Abhishek Tyagi,_Ravindra Deshpande,_Ravi Singh_&_Kounosuke WatabeDepartment of Surgery, University of Texas Southwestern Medical Center, Dallas, TX, 75390, USADan ZhaoDivision of Pediatric Endocrinology and Diabetes, Department of Pediatric and Adolescent Medicine, Ulm University Medical Center, Ulm, GermanyMartin WabitschCancer Institute, University of Mississippi Medical Center, Jackson, MS, 39216, USAYin-Yuan Mo\" - \"AuthorsDan ZhaoYou can also search for this author in _Kerui WuYou can also search for this author in _Sambad SharmaYou can also search for this author in _Fei XingYou can also search for this author in _Shih-Ying WuYou can also search for this author in _Abhishek TyagiYou can also search for this author in _Ravindra DeshpandeYou can also search for this author in _Ravi SinghYou can also search for this author in _Martin WabitschYou can also search for this author in _Yin-Yuan MoYou can also search for this author in _Kounosuke WatabeYou can also search for this author in _\" - Contributions - \"D.Z., K.wu and K.W. designed the study and wrote the manuscript. D.Z. conducted the experiments and acquired, analyzed and interpreted the data, performed ANOVA test. Kwu performed functional assay to compare the ExoQuick and ultracentrifuge, lipid staining, IHC for perilipin-1, pHSL, flow cytometry, and patient characteristics analysis. K.Wu and F.X. assisted with bioinformatics data analysis and data interpretation. F.X. assisted with lipid staining and acquisition of fluorescence images. K.Wu, S.S., S.W. and A.T. helped with animal experiments, managing patients specimens and data interpretation. M..W and Y.Y.M. provided resources. D.Z., K.Wu, F.X., S.S., S.W., A.T., R.D., M.W., Y.Y.M. and K.W. reviewed and edited the manuscripts. K.W. supervised the study.\" - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks Mikhail Kolonin, Takahiro Ochiya and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 unported license ().\\' - \"Platelets are critically involved in the occurrence of ischemic events, and antiplatelet therapy is standard of care in AMI treatment. Although such therapy has proven to be a major achievement in cardiovascular medicine, its resounding success may have obscured some of the more discrete but nevertheless important platelet functions. In fact, activated platelets release a plethora of bioactive substances with manifold effects on neighboring cells and tissues that would be missing when platelet activation is inhibited. Here, we were fortunate to unveil platelet S1P as a hereto unknown cardioprotective function of the platelet secretome in the setting of AMI. In addition, we were able to show how important even a small amount of released S1P can be for cardioprotection (Supplementary Fig._ ). The Mfsd2b receptor seems to be particularly important here, as the lack of Mfsd2b results in 50-60-fold accumulation of S1P in platelets. Mice lacking this receptor showed significantly increased infarct size, due to missing S1P u201Cburstu201D during AMI. Many reports have shown that administration of exogenous S1P protects against ischemia/ reperfusion (I/R) injury in several organs and that interfering with the generation or degradation of endogenous S1P determines the extent of I/R injury. However, the dynamics and sources of circulating plasma S1P that are verifiably present during acute coronary syndrome and murine AMI have been largely unknown. In general, the reasons why certain patients have higher S1P levels than others are unclear. S1P levels vary widely between individuals. Reasons for this are for example ethnicity, obesity, and body fat percentage. Moreover, age and gender influence S1P. However, data is regarding this is contrary. Although many pathological conditions such as cardiovascular and infectious diseases, inflammation, sepsis and metabolic disorders are associated with altered homeostatic S1P concentrations in plasma, only few exhibit acute S1P bursts in plasma as in AMI. Indeed, a time- and ischemia severity-dependent increase of S1P has been described in angina pectoris and AMI. This is in line with a rapid event-associated time-limited burst of S1P release followed by normalization, e.g., due to the short half-life of S1P in plasma. Nevertheless, such transient S1P bursts are not an AMI epiphenomenon, but clearly have a biologically relevant meaning in STEMI patients: the height of the S1P burst was positively associated with survival and negatively with infarct size. Interestingly, only S1P concentrations at admission determined outcome. However, S1P levels on day 1 or day 5 did not. The reason for this is unclear but might be due to medication in STEMI patients. Especially, as we could show that P2Y12 inhibition interferes with S1P release from platelets. This needs to be explored in a prospective setting, but if successful, it may define a novel prognostic marker.\" - \\'The source of AMI-associated S1P bursts had not been identified until now. Here, we show that activated platelets are the main source in contrast to other hematopoietic cells or the heart itself. Thrombin activation of platelets during clotting has been demonstrated to cause S1P release in a COX-dependent manner. Vice versa, S1P synergizes with thrombin to induce the release of tissue factor from platelets. This has led to the ostensible notion that S1P may either be a promiscuous marker of platelet activation or, assuming causal participation, even deleterious in AMI. Our data suggest exactly the contrary: platelet-released S1P is beneficial during AMI by limiting myocardial damage. Unsurprisingly, this effect has been difficult to identify so far as it is concealed beneath the widely implemented antiplatelet therapy in AMI and the valid therapeutic paradigm behind it.\\' - \"However, our study reveals an exploitable therapeutic window as we have observed that not all antiplatelet drugs prevented S1P release to the same extent. In AMI patients, dual antiplatelet therapy with ASA and a P2Y12 inhibitor is currently recommended. While the P2Y12 inhibitor cangrelor blocked platelet S1P release and blunted cardioprotection by activated platelet supernatant, the GPIIb/IIIa inhibitor tirofiban preserved both S1P release and cardioprotection, whereas both drugs achieved efficient anti-aggregation. Hence, an important conclusion of our study is that an antiplatelet strategy that conserves platelet S1P release during AMI would be the most desirable one. In the clinical setting, GPIIb/IIIa inhibitors were shown to reduce thrombotic or ischemic complications post PCI and the incidence of major adverse cerebro- and cardiovascular events (MACCE). Furthermore, GPIIb/IIIa inhibitors are known to improve ST-resolution and reduce infarct size in the setting of AMI. However, they are currently only recommended for: (a) up-stream therapy in high-risk patients with high thrombus burden, (b) thrombotic complications, and (c) bail-out in slow or no-reflow conditions due to thrombus formation. The reasons for this u201Csecond lineu201D recommendation are primarily that, (a) they enhance the risk of bleeding (like any other antithrombotic substance) and (b) were not tested in presence of more potent P2Y12 inhibitors like prasugrel, ticagrelor or cangrelor. However, we have learned much since the GPIIb/IIIa inhibitors landmark trials. Especially peri-procedural bleeding was diminished substantially due to reduction of access site bleeding by choosing radial access. Additionally, P2Y12 medication prior to decision of percutaneous coronary intervention in AMI is not recommended, and deescalation of antithrombotic treatment early after AMI reduced the risk of bleeding without enhancing ischemic events. Certainly, much more has to be examined and pharmacologically pursued, if the paradigm we have defined hereu2014that of cardioprotection by platelet-derived bioactive molecules such as S1Pu2014matures and becomes clinically relevant. If and how GPIIb/IIIa antagonists could be employed to exploit the benefits of such concealed cardioprotection awaits clinical proof. However, a possible scenario that can be implemented right away with the current treatment options in order to investigate it could be, e.g., hard and early platelet inhibition by GPIIb/IIIa inhibition to tackle ischemic events followed by deescalated P2Y12 inhibition or single antiplatelet therapy. We believe that the recognition of this platelet-mediated intrinsic protection against tissue injury should be taken into consideration in the treatment of many settings where antiplatelet therapy is indicated but could be modified in an attempt to preserve and exploits its benefits. This may be an exciting therapeutic paradigm in platelet medicine that reaches far beyond the reduction of infarct size in AMI.\"- - Animals - \"Wildtype mice were purchased from Janvier Labs (Saint-Berthevin, France). Edg1 Cardio Cre mice (S1P1 /S1P1) were bred as previously published. SphK1- deficient mice were gifted by Richard Proia, NIH and Mfsd2b knockout mice were obtained from the MMRRC at UCDavis. Mice at the age of 12u201315 weeks at the beginning of experiments were used. All animals had free access to drinking water and standard diet ad libitum.\" - Generation of activated and non-activated supernatant (SNT) - \"Blood was taken from donor mice via retrobulbar plexus with a heparinized cannula. Afterwards, heparinized whole blood rested for 15u2009min at 37u2009xB0C. After first centrifugation (170u2009g, 5u2009min, room temperature (RT)) platelet rich plasma (PRP) was taken and incubated at 37u2009xB0C with ADP (final concentration 10u2009xB5M) or collagen (10u2009xB5g/ml, Probe & go Labor Labordiagnostica GmbH, Germany). After 3u2009min of incubation SNT+ was generated via second centrifugation (2000g, 5u2009min, RT). SNTu2212 was incubated with the same volume of saline solution. If required, inhibitors were added after first centrifugation and 2u2009min prior to ADP incubation. They were used with following final concentrations: 200u2009nM cangrelor (Can, 200u2009xB5M, Sigma Aldrich, St. Louis, Missouri), 50u2009nM tirofiban (Tiro, 50u2009nM, AggrastatxAE, Correvio).\" - Determination of S1P concentrations - \"Lipid extraction and liquid chromatography-mass spectrometry (LCMS; Mercku2013 Hitachi Elite LaChrom System) were performed to measure S1P levels in plasma.\" - \"Platelets were resuspended in Methanol (MeOH), followed by addition of 10u2009 xB5L internal standards (for S1P (d18:1): 10u2009pmol C17 S1P in MeOH; for S1P(d17:1): d7 S1P in MeOH, Avanti Polar Lipids Inc., Alabaster, AL). Samples were mixed, precipitated overnight at u221280u2009xB0C and centrifugated (5u2009min at 21,300u2009xD7u2009g, 4u2009xB0C). The supernatant was transferred into mass spectrometry sample vials and stored at u221280u2009xB0C until measurement.\" - \"Tissue samples (10u201320u2009mg were homogenized in a Stomacher Bag with 10u2009 xB5L internal standard solution and 500u2009xB5L MeOH. The homogenate was collected, another aliquot MeOH (500u2009xB5L) was added to the stomacher bag and, after rinsing, the solution was pooled with the first homogenate (total 1u2009mL). Samples were mixed, precipitated overnight at u221280u2009xB0C, and centrifuged (5u2009min at 21,300u2009xD7u2009g, 4u2009xB0C). The supernatant was transferred into mass spectrometry sample vials and stored at u221280u2009xB0C until measurement.\" - \"Chromatographic separation was performed on a LCMS-8050 triple-quadrupole mass spectrometer (Shimadzu Duisburg, Germany) interfaced with a Dual Ion Source and a Nexera X3 Front-End-System (Shimadzu Duisburg, Germany). HPLC is performed with a 2u2009xD7u200960u2009mm MultoHigh-C18 RP column with 3u2009xB5 m particle size at 40u2009xB0C. Mobile phases consisted of [A] 1% aq. HCO2H and [B] MeOH. The following LC gradient was used: Start at 10% [B], linear increase of [B] from 10% to 100% from 0u2009min to 3.0u2009min (B-Curveu2009=u2009 u22122), 100% [B] from 3.00u2009min to 8u2009min and equilibration from 8.01u2009 min to 10.00u2009min prior next injection. Flow rate was 400u2009xB5L/min and injection volume of all samples was 10u2009xB5L. MS settings were the following: Interface: ESI, Nebulizing Gas flow: 2u2009L/min, Heating Gas Flow: 10u2009 L/min, Interface Temperature: 300u2009xB0C, DL Temperature: 250u2009xB0C, Heat Block Temperature: 400u2009xB0C, Drying Gas Flow: 10u2009L/min. Data were collected using multiple reaction monitoring (MRM). Positive ionization was used for qualitative analysis and quantification. Standard curves were generated by measuring increased amounts (100 fmol to 50 pmol) of S1P (d18:1) and C17 S1P (d17:1) together with internal standard (C17 S1P (d17:1) for S1P (d18:1); d7 S1P (d18:1) for C17 S1P (d17:1), 0.3u2009xB5M final conc. in MeOH). Linearity of the standard curves and correlation coefficients were obtained by linear regression analysis. All MS analyses were performed using LabSolutions 5.99 SP2 and LabSolutions Insight (Shimadzu Duisburg, Germany).\" - Isolated mouse heart perfusion - \"Langendorff experiments were conducted to study the cardioprotective effects on local cardiac cells without influence of circulating blood cells. In brief, Krebs-Henseleit Buffer (KHB) including 118u2009mM NaCl, 4.7u2009mM KCl, 0.8u2009 mM MgSO4, 25u2009mM NaHCO3, 1.2u2009mM KH2PO4, 5u2009mM glucose, 110u2009 mM Na-pyruvate and 2.5u2009mM CaCl2 was warmed at 37u2009xB0C and gassed with carbogen circulating in the Langendorff- apparatus (Hugo Sachs, March-Hugstetten, Germany). Wildtype mice (C57BL/6u2009J) and Edg1 CardioCre+/u2212 mice (S1P1 /S1P1) were used for experiments. DMSO and S1P1 antibody W146 were applied 30u2009 min prior to experiment. Mice were anesthetized by intraperitoneal (i.p.) injection of ketamine (100u2009mg/kg; Pfizer Pharma PFE GmbH, Berlin, Germany) and xylazine (10u2009mg/kg; Bayer, Leverkusen, Germany) followed by i.p. injection of heparin (1000 IU/mouse; Rotexmedica, Trittau, Germany). After loss of reflexes (5u2013 7u2009min after injection), hearts were extracted, prepared rapidly, and placed in 4u2009xB0C cold KHB. The ascending aorta was cannulated and fixed by a node. Hearts were placed into the Langendorff system and retrogradely perfused with a constant KHB flow measured by ultrasound flow probe (Type MC1PRB-HSE for HSE-TTFM; Hugo Sachs, March-Hugstetten, Germany) at 100u2009mmHg. A small water-filled balloon adapted to a pressure transducer was inserted in the left ventricle (LV) for quantification of LV pressure gradient (DP) and for LV end-diastolic pressure (LVEDP). Hearts were paced at 600 beats per minute. After equilibration of 20u2009min, 20u2009seconds test ischemia was conducted to investigate flow reserve. Hearts that did not reach a minimum of 70% flow increase were excluded from further analyses. Afterwards, hearts recovered for further 5u2009min. Baseline parameters were assessed. Hearts that did not reach a minimum of 50u2009 mmHg DP were excluded as well. Afterwards, global flow was interrupted and preloading with murine platelet supernatant with and without prior activation (SNTu2212 and SNT+) was conducted via a syringe driver (0.4u2009ml/min; 400u2009xB5 l) followed by 40u2009min of global ischemia. Hearts were warmed and kept moist by a small KHB-containing basin surrounding the ischemic heart. Next, the flow was restarted for two hours of reperfusion with KHB. Hemodynamic parameters were assessed after 60u2009min. After 120u2009min, hearts were extracted from the cannula and frozen for one hour at u221220u2009xB0C. The frozen heart was cut into six slices and 2,3,5-triphenyltetrazolium chloride (TTC, Sigma Aldrich, St. Louis, Missouri) staining was conducted. Infarct size (IS) was calculated as percentage of the left ventricle volume.\" - Murine model of acute myocardial infarction (AMI) - \"Mice of different genotypes underwent ischemia/reperfusion surgery as a murine model of AMI. After intubation of the mice, surgery was conducted under 2u2009 vol% isoflurane anesthesia. AMI was induced via transient left anterior descending artery (LAD) ligation for 30u2009min. Typical ST-elevations were used for monitoring of ligation and resolution. Body temperature was controlled continuously. SNTu2212 or SNT+ respectively was injected intravenously 5u2009min prior to ischemia. This open-chest surgery was followed by 24u2009h, 5 days or 21 days of reperfusion. Afterwards, mice were sacrificed and organs were extracted for further analyses.\" - Euthanasia of mice - \"First, animals were anesthetized with ketamine (100u2009mg/kg; Pfizer Pharma PFE GmbH, Berlin, Germany) and xylazine (10u2009mg/kg; Bayer, Leverkusen, Germany) by intraperitoneal injection. After loss of reflexes, the thorax was opened and the heart was removed for further analysis.\" - Myocardial infarct size measurement - \"Infarct size was measured 24u2009h after AMI. Hearts were taken out of the sacrificed mice and surrounding tissue was removed in cold saline solution. Hearts were reperfused with cold saline solution via aorta to remove blood. Ligation of LAD was repeated in a permanent manner. Afterwards, the hearts were perfused with 1% Evans blue dye (Sigma Aldrich, St. Louis, Missouri), which separates non-ischemic myocardium from area at risk (AAR). Hearts were frozen at u2212 20u2009xB0C overnight and then cut into six 1u2009mm slices starting apical. The slices were incubated for 5u2009min with 1% TTC solution, which colors vital tissue. Infarct size/AAR-ratio and AAR/left ventricle (LV)-ratio was determined via computer-assisted planimetry.\" - Cardiac function measurement by echocardiography - \"Cardiac function was measured before as well as 24u2009h and 21 days after AMI by echocardiography. Mice were anesthetized with isoflurane and body temperature was kept at 37u201338u2009xB0C. The heart rate was monitored during examination. High resolution 18u201338u2009MHz ultrasound transducer (MS 400, VEVO 3100, VisualSonicsInc., Toronto, Canada) was used. To determine left ventricular function parameters, different angles of the heart were measured. Post-processing analyses were performed offline using commercial software (VevoLab 3.2.6., VisualSonicsInc., Toronto, Canada)\" - Immunohistochemical staining and quantification - \"Mice were sacrificed 24u2009h or 5 days after AMI and perfused with cold PBS. The hearts were removed and fixed in 4% paraformaldehyde (PFA, Alfa Aesar, Massachusetts, USA) overnight. Hearts were then dehydrated and embedded in paraffin. Hearts were cut in 5u2009xB5m sections using a microtome (Leica CM 3050u2009S, Wetzlar, Germany). The heart sections were stained for inflammation and apoptosis. Afterwards, they were deparaffinized and rehydrated and a heat-induced epitope retrieval_ (HIER) was done using citrate buffer (pH 6.0, ThermoFisher Massachusetts, USA). After the retrieval the sections were blocked with BloxAll-Blocking solution (Vector laboratories, Burlingame, USA). They were then blocked with 2.5 % goat serum and subsequently incubated with primary antibodies at 4xB0C overnight. The following primary antibodies were used: rat anti Ly6G (1:100, Abcam, Cambridge, UK) and rabbit anti cleaved Caspase-3 (1:400, Cell signaling, Massachusetts, USA). After that, the ImmPRESSxAE-Goat Anti-Rat IgG (Mouse Adsorbed) Polymer Kit (MP-7444, Vector Labs, Burlingame, Canada) for Ly6G staining and the ImmPRESSxAE -AP Goat Anti-Rabbit IgG Polymer Kit (MP-7451, Vector Labs, Burlingame, Canada) for Caspase-3 staining were used. The staining was visualized using the ImmPact DABxAE (SK-4105, Vector Labs, Burlingame, Canada) detection system. Sections were counterstained with hematoxylin Gill II (T864.1, Roth, Karlsruhe, Germany) and mounted with permanent mounting medium (H-5000, Vector laboratories, Burlingame, Canada). Images of the stained sections were done by using a Leica bright field microscope (DM4000M) with twentyfold magnification. Five fields of 100u2009 xB5m x 100u2009xB5m each were imaged of one heart section and analyzed using ImageJ software.\" - Patients and study design - \"We conducted a hypothesis-generating, prospective, monocentric, time-series, translational analysis in 127 patients with ST-elevation myocardial infarction (STEMI). An all-comers design was applied. Inclusion criteria were age u2265 18 years, occurrence of one of the above-mentioned events, and written informed consent. Exclusion criteria were malignant comorbidities and coagulopathies. Written informed consent was obtained from all participants. Blood sampling was conducted during ischemia as well as 12u2009h, and 5 days after intervention (Several important procedural details (culprit vessel, contrast medium, TIMI flow) can be found in Supplementary Table_). To assess infarct size, gadolinium-based contrast agent (Gadovist, Bayer Healthcare, Berlin, Germany, 0.2u2009mmol/kg) was used to detect late gadolinium enhancement in cardiac magnet resonance imaging 6 months after STEMI during clinical follow-up. Post-processing analyses were performed offline using commercial software (cmr42, Circle Cardiovascular Imaging Inc., Calgary, Alberta, Canada and Extended Workspace, Philips Healthcare, Hamburg, Germany). Infarct size was determined in percent from left ventricle volume. Evaluation of CMR was performed by an experienced cardiologist regarding CMR-imaging. Cardiovascular death was assessed during 12-month follow-up. After sampling, blood was centrifuged at 500u2009xD7u2009g for 10u2009min to gain plasma.\" - Light transmission aggregometry (LTA) - \"Blood from healthy individuals was obtained after written informed consent. First, centrifugation of citrate-anticoagulated whole blood was performed at 1200u2009 rpm for 10u2009min to generate platelet-rich plasma (PRP). Second, PRP was centrifuged at 13,200u2009rpm for 5u2009min to generate platelet-poor plasma (PPP) for calibration of the aggregometer (APACT 4004 LABiTecxAE). After 2u2009min of equilibration, induction of platelet aggregation was performed using adenosine diphosphate (ADP, 5u2009xB5M Sigma Aldrich, St. Louis, Missouri). If necessary, PRP was incubated with cangrelor (Can, 200u2009xB5M, Sigma Aldrich, St. Louis, Missouri) or tirofiban (Tiro, 50u2009nM, AggrastatxAE, Correvio) in time of equilibration. Results were given as maximum aggregation (MoA) in percent.\" - Adenosine triphosphate (ATP) release assay - \"For quantitative analysis of ATP release after platelet activation, bioluminescence analysis of ATP was performed. Briefly, an ATP Assay Kit (FLAA, Sigma Aldrich, St. Louis, Missouri) was used to determine concentrations of platelet-released ATP. An ATP calibration curve was performed on the same day as the related samples. Washed platelets were incubated with cangrelor (Can, 200u2009xB5M, Sigma Aldrich, St. Louis, Missouri) or tirofiban (Tiro, 50u2009nM, AggrastatxAE, Correvio) prior to activation via ADP (5u2009xB5M Sigma Aldrich, St. Louis, Missouri).\" - Determination of p-selectin expression via flow cytometry - \"Citrate-anticoagulated whole blood was centrifuged at 300u2009xD7u2009g for 10u2009min to gain PRP. 100u2009xB5l of PRP were incubated for 6u2009min at 37u2009xB0C. After another 6u2009min of stimulation with ADP (5u2009xB5 M) at 37u2009xB0C, the following antibodies were added and incubated for 30u2009 min at 37u2009xB0C: CD41/61-APC-Cy7 (Miltenyi Biotec, Bergisch Gladbach, Germany), CD42b-Pe-Cy7 (Invitrogen, Massachusetts, USA) and CD62P-BV421 (BD Biosciences, Franklin Lakes, USA). Before measurement with BD FacsVersexAE (BD Biosciences, Franklin Lakes, USA), the solution was diluted 1:100 in phosphate-buffered saline (PBS). Data were processed and analyzed using Flow-JoxAE (BD Biosciences, Franklin Lakes, USA)\" - Statistical analyses - \"For statistical analyses, IBM SPSSxA9-Software (New York, USA) and GraphPad-PrismxA9 statistical software (GraphPad software Inc, San Diego) were used. Normality of distribution was tested with Kolmogorovu2013Smirnov test, DAgostino-Pearson test, qq-plots, and histograms. Normally distributed continues variables were analyzed using t test; non-normally distributed variables using Mannu2013Whitney U test. Homogeneity of variance was tested with Levene test. In case of heteroscedastic data, Welch test was performed. Comparison of three or more groups was performed using ANOVA-analyses. If the assumption of homogeneity of variances was not met, indicated by a significant Levene test, a Kruskalu2013Wallis test was performed. Tukey correction was applied in multiple testing. Log-rank test for trend was implemented to test, whether different S1P concentrations are associated with incidence of cardiovascular mortality over time. Additionally, Pearson correlation was used to investigate a possible relation between S1P levels and infarct size. Normality of distribution of variables used for Pearson correlation was assessed with Kolmogorovu2013Smirnov test. In case of not normally distributed variables Spearman correlation was performed. P values <0.05 were considered significant.\" - \"To explore whether S1P concentration after myocardial infarction could improve the predictive power of GRACE score, we first computed the GRACE score for each patient in our sample. We then allocated values to each of the three S1P concentrations. We started with the values 0, 1, and 2 for high, medium and low S1P concentration, respectively, and added them to the GRACE score of patients. We then computed an ROC curve to explore whether the AUC improved. We used Python programing to try out each combination of values for high concentration in the range 0u2013 20, medium concentration in the range 1u201340 and low concentration in the range 2u201360. Each combination was added to the patients GRACE score and an ROC curve was computed. The combination of values yielding the highest AUC in the ROC analysis was selected.\" - Ethics committee approval - \"The study conformed to the Declaration of Helsinki and was approved by the University of DxFCsseldorf Ethics Committee. It is registered_under clinicaltrails.org with the ID u201CNCT03539133u201D.\" - \"All mice experiments were approved by Landesamt fxFCr Natur, Umwelt und Verbraucherschutz Nordrhein-Westfalen (LANUV) and in accordance with the European Convention for the Protection of Vertebrate Animals used for Experimental and other Scientific Purposes (Council of Europe Treaty Series No. 123) and 2010/63/EU.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - \"The data supporting this study can be found in the figures and supplementary information._ are provided with this paper.\" - \\' \\'- - \"Gawaz, M., Neumann, F. J., Ott, I., Schiessler, A. & SchxF6mig, A. Platelet function in acute myocardial infarction treated with direct angioplasty. Circulation 93, 229u2013237 (1996)._ _ _ _ GidlxF6 f, O. et al. Platelets activated during myocardial infarction release functional miRNA, which can be taken up by endothelial cells and regulate ICAM1 expression. Blood 121, 3908u20133917 (2013)._ _ _ Valgimigli, M. et al. 2017 ESC focused update on dual antiplatelet therapy in coronary artery disease developed in collaboration with EACTS: the Task Force for dual antiplatelet therapy in coronary artery disease of the European Society of Cardiology (ESC) and of the European Association for Cardio-Thoracic Surgery (EACTS). Eur. Heart J. 39, 213u2013260 (2018)._ _ _ Walsh, T. G., Metharom, P. & Berndt, M. C. The functional role of platelets in the regulation of angiogenesis. Platelets 26, 199u2013211 (2015)._ _ _ _ Parsons, M. E. M. et al. Platelet releasate proteome profiling reveals a core set of proteins with low variance between healthy adults. Proteomics 18, e1800219 (2018)._ _ _ Peng, B. et al. Identification of key lipids critical for platelet activation by comprehensive analysis of the platelet lipidome. Blood 132, e1u2013e12 (2018)._ _ _ _ _ Prabhu, S. D. & Frangogiannis, N. G. The biological basis for cardiac repair after myocardial infarction: from inflammation to fibrosis. Circ. Res. 119, 91u2013112 (2016)._ _ _ _ _ Yalta, K. et al. Late versus early myocardial remodeling after acute myocardial infarction: a comparative review on mechanistic insights and clinical implications. J. Cardiovasc. Pharmacol. Ther. 25, 15u201326 (2020)._ _ _ Walsh, T. G. & Poole, A. W. Do platelets promote cardiac recovery after myocardial infarction: roles beyond occlusive ischemic damage. Am. J. Physiol. Heart Circ. Physiol. 314, H1043u2013H1048 (2018)._ _ _ _ _ Halucha, K., Rak-Pasikowska, A. & Bil-Lula, I. Protective role of platelets in myocardial infarction and ischemia/reperfusion injury. Cardiol. Res. Pract. 2021, 5545416 (2021)._ _ _ _ Yabanoglu, S. et al. Platelet derived serotonin drives the activation of rat cardiac fibroblasts by 5-HT2A receptors. J. Mol. Cell Cardiol. 46, 518u2013525 (2009)._ _ _ _ Meyer, A. et al. Platelet TGF-beta1 contributions to plasma TGF-beta1, cardiac fibrosis, and systolic dysfunction in a mouse model of pressure overload. Blood 119, 1064u20131074 (2012)._ _ _ _ _ Mezger, M. et al. Platelets and immune responses during thromboinflammation. Front. Immunol. 10, 1731 (2019)._ _ _ _ _ van der Meijden, P. E. J. & Heemskerk, J. W. M. Platelet biology and functions: new concepts and clinical perspectives. Nat. Rev. Cardiol. 16, 166u2013 179 (2019)._ _ _ Martinod, K. & Deppermann, C. Immunothrombosis and thromboinflammation in host defense and disease. Platelets 32, 314u2013 324 (2021)._ _ _ _ Oikonomou, E. et al. A link between inflammation and thrombosis in atherosclerotic cardiovascular diseases: clinical and therapeutic implications. Atherosclerosis 309, 16u201326 (2020)._ _ _ _ Schanze, N., Bode, C. & Duerschmied, D. Platelet contributions to myocardial ischemia/reperfusion injury. Front. Immunol. 10, 1260 (2019)._ _ _ _ _ Yang, B. C. & Mehta, J. L. Platelet-derived adenosine contributes to the cardioprotective effects of platelets against ischemia-reperfusion injury in isolated rat heart. J. Cardiovasc. Pharmacol. 24, 779u2013785 (1994)._ _ _ _ Frangogiannis, N. G. et al. Critical role of endogenous thrombospondin-1 in preventing expansion of healing myocardial infarcts. Circulation 111, 2935u2013 2942 (2005)._ _ _ _ Serhan, C. N. & Sheppard, K. A. Lipoxin formation during human neutrophil-platelet interactions. Evidence for the transformation of leukotriene A4 by platelet 12-lipoxygenase in vitro. J. Clin. Invest. 85, 772u2013780 (1990)._ _ _ _ _ Abdulnour, R. E. et al. Maresin 1 biosynthesis during platelet-neutrophil interactions is organ-protective. Proc. Natl. Acad. Sci. USA 111, 16526u2013 16531 (2014)._ _ _ _ _ _ Basil, M. C. & Levy, B. D. Specialized pro-resolving mediators: endogenous regulators of infection and inflammation. Nat. Rev. Immunol. 16, 51u201367 (2016)._ _ _ _ Mehta, J. L., Yang, B. C., Strates, B. S. & Mehta, P. Role of TGF-beta1 in platelet-mediated cardioprotection during ischemia-reperfusion in isolated rat hearts. Growth Factors 16, 179u2013190 (1999)._ _ _ _ Chen, D. et al. Crosstalk between SDF-1/CXCR4 and SDF-1/CXCR7 in cardiac stem cell migration. Sci. Rep. 5, 16813 (2015)._ _ _ _ _ _ Ulrych, T. et al. Release of sphingosine-1-phosphate from human platelets is dependent on thromboxane formation. J. Thromb. Haemost. 9, 790u2013798 (2011)._ _ _ _ Hait, N. C., Oskeritzian, C. A., Paugh, S. W., Milstien, S. & Spiegel, S. Sphingosine kinases, sphingosine 1-phosphate, apoptosis and diseases. Biochim. Biophys. Acta 1758, 2016u20132026 (2006)._ _ _ _ Jozefczuk, E., Guzik, T. J. & Siedlinski, M. Significance of sphingosine-1-phosphate in cardiovascular physiology and pathology. Pharmacol. Res. 156, 104793 (2020)._ _ _ _ Sattler, K. J. et al. Sphingosine 1-phosphate levels in plasma and HDL are altered in coronary artery disease. Basic Res. Cardiol. 105, 821u2013 832 (2010)._ _ _ _ Polzin, A. et al. Aspirin inhibits release of platelet-derived sphingosine-1-phosphate in acute myocardial infarction. Int. J. Cardiol. 170, e23u2013e24 (2013)._ _ _ Knapp, M. et al. Plasma sphingosine-1-phosphate concentration is reduced in patients with myocardial infarction. Med. Sci. Monit. 15, CR490u2013 CR493 (2009)._ _ _ Egom, E. E. et al. Serum sphingolipids level as a novel potential marker for early detection of human myocardial ischaemic injury. Front. Physiol. 4, 130 (2013)._ _ _ _ _ Harrison, P. & Cramer, E. M. Platelet alpha-granules. Blood Rev. 7, 52u201362 (1993)._ _ _ _ Keul, P. et al. Sphingosine-1-phosphate receptor 1 regulates cardiac function by modulating Ca2+ sensitivity and Na+/H+ exchange and mediates protection by ischemic preconditioning. J. Am. Heart. Assoc. 5, (2016).Theilmeier, G. et al. High-density lipoproteins and their constituent, sphingosine-1-phosphate, directly protect the heart against ischemia/reperfusion injury in vivo via the S1P3 lysophospholipid receptor. Circulation 114, 1403u20131409 (2006)._ _ _ _ Levkau, B. HDL-S1P: cardiovascular functions, disease-associated alterations, and therapeutic applications. Front. Pharmacol. 6, 243 (2015)._ _ _ _ Davidson, S. M. et al. Circulating blood cells and extracellular vesicles in acute cardioprotection. Cardiovasc. Res. 115, 1156u20131166 (2019)._ _ _ _ Vu, T. M. et al. Mfsd2b is essential for the sphingosine-1-phosphate export in erythrocytes and platelets. Nature 550, 524u2013528 (2017)._ _ _ _ _ Chandrakanthan, M. et al. Deletion of Mfsd2b impairs thrombotic functions of platelets. Nat. Commun. 12, 2286 (2021)._ _ _ _ _ _ Nitzsche, A. et al. Endothelial S1P(1) signaling counteracts infarct expansion in ischemic stroke. Circ. Res. 128, 363u2013382 (2021)._ _ _ _ Buie, J. N. J. et al. Differences in plasma levels of long chain and very long chain ceramides between African Americans and whites: an observational study. PLoS One 14, e0216213 (2019)._ _ _ _ Kowalski, G. M., Carey, A. L., Selathurai, A., Kingwell, B. A. & Bruce, C. R. Plasma sphingosine-1-phosphate is elevated in obesity. PLoS One 8, e72449 (2013)._ _ _ _ _ _ Ito, S. et al. Increased plasma sphingosine-1-phosphate in obese individuals and its capacity to increase the expression of plasminogen activator inhibitor-1 in adipocytes. Coron Artery Dis 24, 642u2013650 (2013)._ _ _ Vozella, V. et al. Elevated plasma ceramide levels in post-menopausal women: a cross-sectional study. Aging (Albany NY) 11, 73u201388 (2019)._ _ _ _ Weir, J. M. et al. Plasma lipid profiling in a large population-based cohort. J. Lipid Res. 54, 2898u20132908 (2013)._ _ _ _ _ Hammad, S. M. et al. Blood sphingolipidomics in healthy humans: impact of sample collection methodology. J. Lipid Res. 51, 3074u20133087 (2010)._ _ _ _ _ NikkilxE4, J. et al. Gender-dependent progression of systemic metabolic states in early childhood. Mol. Syst. Biol. 4, 197 (2008)._ _ _ _ Bu0142ogowski, W. et al. Perioperative release of pro-regenerative biochemical signals from human renal allografts subjected to ischemia-reperfusion injury. Innate immun. 20, 126u2013132 (2014)._ _ _ Waeber, C. & Walther, T. Sphingosine-1-phosphate as a potential target for the treatment of myocardial infarction. Circ. J. 78, 795u2013802 (2014)._ _ _ _ Davaille, J. et al. Antiproliferative properties of sphingosine 1-phosphate in human hepatic myofibroblasts. J. Biol. Chem. 275, 34628u2013 34633 (2000)._ _ _ _ Takeya, H., Gabazza, E. C., Aoki, S., Ueno, H. & Suzuki, K. Synergistic effect of sphingosine 1-phosphate on thrombin-induced tissue factor expression in endothelial cells. Blood 102, 1693u20131700 (2003)._ _ _ _ Liu, H. et al. Sphingosine-1-phosphate modulates PAR1-mediated human platelet activation in a concentration-dependent biphasic manner. Sci. Rep. 11, 15308 (2021)._ _ _ _ _ Collet, J. P. et al. 2020 ESC Guidelines for the management of acute coronary syndromes in patients presenting without persistent ST-segment elevation. Eur. Heart J. 42, 1289u20131367 (2021)._ _ _ Neumann, F. J. et al. 2018 ESC/EACTS Guidelines on myocardial revascularization. Eur. Heart J. 40, 87u2013165 (2019)._ _ _ Valgimigli, M. et al. The additive value of tirofiban administered with the high-dose bolus in the prevention of ischemic complications during high-risk coronary angioplasty: the ADVANCE Trial. J. Am. Coll. Cardiol. 44, 14u201319 (2004)._ _ _ _ Effects of platelet glycoprotein IIb/IIIa blockade with tirofiban on adverse cardiac events in patients with unstable angina or acute myocardial infarction undergoing coronary angioplasty. The RESTORE Investigators. Randomized Efficacy Study of Tirofiban for Outcomes and REstenosis. Circulation 96, 1445u20131453 (1997)._ _ ten Berg, J. M. et al. Effect of early, pre-hospital initiation of high bolus dose tirofiban in patients with ST-segment elevation myocardial infarction on short- and long-term clinical outcome. J. Am. Coll. Cardiol. 55, 2446u20132455 (2010)._ _ _ Dannenberg, L. et al. Safety and efficacy of tirofiban in STEMI-patients. Int. J. Cardiol. 274, 35u201339 (2019)._ _ _ Smit, J. J. et al. Prehospital triple antiplatelet therapy in patients with acute ST elevation myocardial infarction leads to better platelet aggregation inhibition and clinical outcome than dual antiplatelet therapy. Heart 96, 1815u20131820 (2010)._ _ _ _ Kloner, R. A. Current state of clinical translation of cardioprotective agents for acute myocardial infarction. Circ. Res. 113, 451u2013 463 (2013)._ _ _ _ Steg, P. G. et al. ESC Guidelines for the management of acute myocardial infarction in patients presenting with ST-segment elevation. Eur. Heart J. 33, 2569u20132619 (2012)._ _ _ _ Ibanez, B. et al. 2017 ESC Guidelines for the management of acute myocardial infarction in patients presenting with ST-segment elevation: the Task Force for the management of acute myocardial infarction in patients presenting with ST-segment elevation of the European Society of Cardiology (ESC). Eur. Heart J. 39, 119u2013177 (2018)._ _ _ Diaz, J. F., Cardenal, R., Gomez-Manchero, A. & Sanchez-Gonzalez, C. Safety and efficacy of tirofiban as adjunctive therapy for patients with ST-elevation myocardial infarction: a comparison versus placebo and abciximab. Cardiovasc. Hematol. Agents Med. Chem. 9, 147u2013153 (2011)._ _ _ _ Geeganage, C., Wilcox, R. & Bath, P. M. Triple antiplatelet therapy for preventing vascular events: a systematic review and meta-analysis. BMC Med. 8, 36 (2010)._ _ _ _ Bhardwaj, B. et al. Radial versus femoral access for left main percutaneous coronary intervention: an analysis from the Veterans Affairs Clinical, Reporting, and Tracking Program. Catheter. Cardiovasc. Interv. (2021).Sibbing, D. et al. Guided de-escalation of antiplatelet treatment in patients with acute coronary syndrome undergoing percutaneous coronary intervention (TROPICAL-ACS): a randomised, open-label, multicentre trial. Lancet 390, 1747u20131757 (2017)._ _ _ _ Sattler, K. et al. Defects of high-density lipoproteins in coronary artery disease caused by low sphingosine-1-phosphate content: correction by sphingosine-1-phosphate-loading. J. Am. Coll. Cardiol. 66, 1470u20131485 (2015)._ _ _ _ Totzeck, M., Korste, S., Miinalainen, I., Hendgen-Cotta, U. B. & Rassaf, T. S-nitrosation of calpains is associated with cardioprotection in myocardial I/R injury. Nitric Oxide 67, 68u201374 (2017)._ _ _ _ Knoop, B. et al. Cardioprotection by very mild hypothermia in mice. Cardiovasc. Diagn. Ther. 9, 64u201367 (2019)._ _ _ _ Dannenberg, L. et al. MTX treatment does not improve outcome in mice with AMI. Pharmacology 106, 225u2013232 (2021)._ _ _ _ Mergental, H. et al. Development of clinical criteria for functional assessment to predict primary nonfunction of high-risk livers using normothermic machine perfusion. Liver Transpl. 24, 1453u20131469 (2018)._ _ _ _ \"- - \"The authors thank Stefanie Becher for experimental support. This work was supported by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation)_Grant No. 236177352_- SFB 1116, TP B11 to B.L. and A.P., TP B6 and B12 to M.K. and by the German Research Foundation (LE 940/7-1 to B.L. and PO 2247/2-1 to A.P). We acknowledge the support of the Susanne-Bunnenberg-Stiftung at the DxFCsseldorf Heart Center.\"- - Open Access funding enabled and organized by Projekt DEAL.- - Author notes - \\'These authors contributed equally: Amin Polzin, Lisa Dannenberg, Marcel Benkhoff.\\' - Authors and Affiliations - \"Department of Cardiology, Pulmonology, and Vascular Medicine, University Hospital DxFCsseldorf, Medical Faculty of the Heinrich Heine University DxFCsseldorf, DxFCsseldorf, GermanyAmin Polzin,_Lisa Dannenberg,_Marcel Benkhoff,_Maike Barcik,_Carolin Helten,_Philipp Mourikis,_Samantha Ahlbrecht,_Laura Wildeis,_ Justus Ziese,_Dorothee Zikeli,_Daniel Metzen,_Hao Hu,_Leonard Baensch,_Dragos Duse,_SxFCreyya Saffak,_Mareike Cramer,_Florian BxF6nner,_Tobias Zeus_ &_Malte KelmCARID, Cardiovascular Research Institute DxFCsseldorf, Medical Faculty and University Hospital, DxFCsseldorf, GermanyAmin Polzin,_Florian BxF6nner,_ Tobias Zeus_&_Malte KelmNational Heart and Lung Institute, Imperial College London, London, UKAmin PolzinInstitute of Molecular Medicine III, University Hospital DxFCsseldorf, Heinrich Heine University DxFCsseldorf, DxFCsseldorf, GermanyNathalie H. SchrxF6der,_Petra Keul,_Sarah Weske,_Philipp Wollnitzke,_ Dragos Duse_&_Bodo LevkauDepartment of Anesthesiology and Intensive Care, University Hospital Jena, Jena, GermanyTina MxFCller_&_Markus H. GrxE4ler\" - \"AuthorsAmin PolzinYou can also search for this author in _Lisa DannenbergYou can also search for this author in _Marcel BenkhoffYou can also search for this author in _Maike BarcikYou can also search for this author in _Carolin HeltenYou can also search for this author in _Philipp MourikisYou can also search for this author in _Samantha AhlbrechtYou can also search for this author in _Laura WildeisYou can also search for this author in _Justus ZieseYou can also search for this author in _Dorothee ZikeliYou can also search for this author in _Daniel MetzenYou can also search for this author in _Hao HuYou can also search for this author in _Leonard BaenschYou can also search for this author in _Nathalie H. SchrxF6derYou can also search for this author in _Petra KeulYou can also search for this author in _Sarah WeskeYou can also search for this author in _Philipp WollnitzkeYou can also search for this author in _Dragos DuseYou can also search for this author in _SxFCreyya SaffakYou can also search for this author in _Mareike CramerYou can also search for this author in _Florian BxF6nnerYou can also search for this author in _Tina MxFCllerYou can also search for this author in _Markus H. GrxE4 lerYou can also search for this author in _Tobias ZeusYou can also search for this author in _Malte KelmYou can also search for this author in _Bodo LevkauYou can also search for this author in _\" - Contributions - A.P., L.D. M.Be., and B.L. designed the study, analyzed and interpreted data, and wrote the manuscript with the input of M.K. D.M., P.W., M.C., and F.B. performed the human studies and statistical analysis. M.Ba, C.H., P.M, S.A., L.W., J.Z., D.Z., H.H., L.B., N.S., P.K., S.W., D.D., S.S., T.M., M.G., and T.Z. collected and analyzed data and revised the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \"Nature Communications thanks Christian Weber and Burkhard Kleuser for their contribution to the peer review of this work._ are available.\" - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 unported license.\" - \"The chemical composition of SHPFs is analyzed by Fourier transform infrared (FTIR) spectroscopy (Fig._). In the spectrum of KGM (black curve), the absorption bands at 3390 and 2876u2009cm are assigned to the stretching of u2013OH groups and Cu2013H of methyl, while the characteristic peaks of mannose appear at 875 and 804u2009cmu2009. The peaks of HPC (blue curve) at 1060 and 2971u2009 cm are assigned to the u2013CH2u2013Ou2013CH2 stretching and the u2013CH stretching of the methyl group, respectively. The spectrum of SHPF (red curve) contains the combination of all characteristic peaks of KGM and HPC, confirming their co-existence in the SHPF network. Meanwhile, a clear shift of u2013OH stretching (3428u2009cm) indicates the formation of a hydrogen bond between the u2013 OH groups of HPC (3452u2009cm) and the u2013OH groups of KGM (3390u2009cm).\" - \"X-ray diffraction (XRD) is used to evaluate the crystallinity of SHPFs (Fig._ ). The XRD pattern of pure KGM has a broad band at 20xB0 due to its amorphous nature (black curve). Two characteristic diffraction peaks of HPC (red curve) are recognized at 2(theta)u2009=u20099xB0 and 20xB0, which are attributed to the crystalline phase and amorphous phase of HPC, respectively. The diffraction peaks become flatter and broader in the blend film (blue curve), indicating the good miscibility of KGM and HPC. Note that the crystalline phase of HPC at 2u03B8u2009=u20099xB0 does not appear in KGM and HPC blend film because the amount of integrated HPC is low (1.0u2009wt%). After the integration of LiCl into the KGMu2009+u2009HPC gel network, no crystalline LiCl is detected (magenta curve) due to its relatively low concentration in the polymer matrix (Supplementary Figs._ and ). Since deliquescent salt powders (e.g., LiCl, CaCl2) will agglomerate during the hydration that can deteriorate the performance, this uniform distribution of LiCl in SHPFs is favorable for sorbent materials to achieve stable functionality. With the highly porous structure and excellent capability to hold LiCl and avoid the aggregation of salt particles, SHPFs are highly desired in AWH technologies.\" - AWH performance of SHPFs - \"The water vapor sorption is tested with a dynamic vapor sorption (DVS) system under constant humidified airflow. Water vapor from the humid air diffuses through the open pores (Fig._) of SHPF to its wall due to the reduced vapor pressure at the hydrogelu2013air interface caused by the hygroscopic LiCl. The sorption of water vapor occurs at the liquidu2013gas interface inside the SHPF. Then, the absorbed water quickly diffuses to the polymer network (KGM and HPC), leading to a volumetric expansion of the SHPF. The initial sorption tests on the pure KGM film show the rapid saturation of water uptake within 20u2009min under low RH (u226430% RH) due to its highly porous network (inset of Fig._), which sets a good basis as the sorbent framework. The addition of LiCl (7.3u2009 wt% Li) with high affinity to water vapor boosts the overall water uptake to 0.69u2009 gu2009g at 15% RH and 1.03u2009gu2009g at 30% RH (Fig._). It should be noted that further increasing the LiCl content in SHPFs could elevate the water uptake, but the sorption kinetics became much slower (Supplementary Fig._). In addition, introducing HPC in SHPFs only slightly lowers the water uptake but notably enhances the water release process, which will be discussed later. The measured water uptake of the optimized SHPF is 0.64u2009gu2009g at 15% RH, 0.96u2009gu2009 g at 30% RH, and 1.53u2009gu2009g at 60% RH (Fig._). The sorption time required to reach 80% of its saturated water uptake at 15, 30, and 60% RH is 67, 36, and 28u2009min, respectively, which outperforms most of the state-of-the-art sorbent-based AWH materials (Fig._). The calculated vapor sorption rate of the SHPF is 1.65u2009Lu2009kg h at 30% RH, which indicates 1u2009kg of SHPF can capture 1.65u2009L of water vapor in 1u2009h, compared favorably among the state-of-the-art sorbent materials (Fig._).\" - \"Fig. 3: Water vapor sorption-desorption performance of SHPFs.a Dynamic water vapor sorption process of KGM-Li, SHPF, and pure KGM (inset) at 25u2009xB0 C, 15% RH and 30% RH. b Dynamic water vapor sorption-desorption processes of the SHPF at different conditions. c Evaporation behaviors of hydrated LiCl, KGM-Li film, and SHPF. d Cycling performance of SHPFs, where sorption occurs at 15% RH (top: 100u2009min per cycle, 14 cycles per day) and 30% RH (bottom: 60u2009 min per cycle, 24 cycles per day). The desorption is performed at 7.5% RH, 60u2009 xB0C. e Comparison of water uptake and time required to reach 80% of the saturated water uptake with the state-of-the-art sorbent materials at ~30% RH. f Comparison of vapor sorption rate with the state-of-the-art sorbent materials at u223C 30% RH.\" - \"The captured water in the SHPF can be released by >70% within 10u2009min through mild heating at 60u2009xB0C under a wide range of RH (Fig._ and Supplementary Fig._). This mild heating temperature is realized by introducing the hydrophilic thermoresponsive HPC with optimized molecular weight, concentration, and pH of the initial precursor (Supplementary Figs._u2013 and Supplementary Table_ ). We further measured the evaporation behaviors of pure LiCl, KGM-Li film, and SHPF via differential scanning calorimetry (DSC). Compared to the evaporation peak of KGM-Li film at 53u2009xB0C, the evaporation peak of SHPF shifted to 44u2009xB0C (Fig._), which is consistent with the phase transition temperature of SHPF (Supplementary Fig._), indicating that a large portion of water in SHPF starts to evaporate at a lower temperature. In principle, the breakage of hydrogen bonds between HPC and water molecules occurs at ~45u2009xB0C. However, practically, a complete transformation of HPC happens at a higher temperature. As such, the desorption condition is set to be 60u2009xB0C in a closed chamber under atmospheric pressure. Furthermore, the SHPF presented a stable performance for 14 cycles per day at 15% RH (70u2009min capturing and 30u2009min releasing) and 24 cycles per day at 30% RH (30u2009min capturing and 30u2009min releasing), respectively (Fig._).\" - \"A water collection system is designed to further validate the ability of SHPFs to deliver freshwater from arid conditions (Fig._). First, a centimeter-scale SHPF (25u2009mmu2009xD7u200940u2009mm) is located on top of a flexible heating plate, of which the temperature can be controlled by an external power supply (Fig._). The condenser wall is tilted at ~45xB0 for fast transport of condensed water droplets downward the water collection channel (Fig._). Vertical heating plates are attached to other walls to slightly elevate their temperatures, so that water vapor can only be condensed at the cooler condensation surface to improve the water collection efficiency. It is worth noting that practically, the vertical heating plates are usually not necessary when the environmental temperature is low enough for vapor condensation. Thermocouples are attached to the backside of heating plates and condenser surface to monitor and control the temperature. This device is properly sealed with a rubber ring and steel binder clips during water collection experiments. The moisture capturing via the large SHPF is done in a homemade chamber at targeted RH prior to each water collection test (Supplementary Fig._). The average water collection of SHPF is 0.56u2009gu2009g after it captured moisture at ~15% RH, 0.82u2009 gu2009g for ~30% RH, and 1.31u2009gu2009g for ~60% RH (Fig._). The calculated average water collection efficiency (i.e., the ratio of water collection to water uptake) reaches 87%. A continuous cycling test of SHPF shows an average water collection at 0.53u2009gu2009g (~15% RH) and 0.83u2009gu2009g (~30% RH) with low Li residues (Fig._ and Supplementary Fig._), corresponding to 5.8u2009Lu2009kg and 13.3u2009Lu2009kg per day, which highlights the attractive water delivery capability of SHPF. Featured with high water uptake under low RHs at outdoor conditions (~5.5u2009Lu2009kg day at 10.6u201341.6% RH, estimated ~12u2009mLwater kgdevice, Supplementary Fig._), low cost of materials (Supplementary Table_ and Supplementary Fig._), facile and scalable synthesis, and overall environmental friendliness, SHPFs hold promise for future sustainable implementation.\" - \"Fig. 4: Water collection and stable cycling performance of SHPF.a Design scheme and b a photograph of the water collection device. Inset: photographs of condensed water droplets (top) and zoom-in sample (bottom). Scale bar: 1u2009cm. c Water uptake and collected water at different relative humidity. d Cycling performance of SHPF at 15% and 30% RH.\"- - \"We develop SHPFs as a new class of sorbent materials to harvest atmospheric water from arid climates (RHu2009<u200930%). The polymer matrix of SHPFs consists of scalable, inexpensive, non-toxic biomasses (KGM and HPC), which facilitate the hierarchically porous structure for enhanced sorption-desorption kinetics. In addition, the KGM/HPC network suppresses the aggregation of hygroscopic salt particles during the hydration process to enable remarkable water uptake with stable cycling performance. The captured water in SHPFs can be rapidly released by electric heating-assisted hydrophilic-hydrophobic switching of HPC chains, leading to 14u201324 cycles of operation with a water yield of up to 13.3u2009 Lu2009kg per day. Considering the thin thickness of SHPF, scaling SHPF into multi-layered sorbent beds or vertical sorbent arrays is expected to further increase the water production in a unit area with improved compactness. It is anticipated that SHPFs offer possibilities to fulfill the technological gap for the development of cost-effective and sustainable AWH systems, which will be critically demanded in many arid areas as well as water-stressed communities.\"- - Chemicals and materials - \\'HPC (average MW: ~80,000, 370,000 and 1000,000), sodium hydroxide (NaOH, ACS reagent), hydrochloric acid (HCl, ACS reagent) and lithium chloride (LiCl, 99%) were purchased from Sigma-Aldrich. Low-grade KGM was purchased from Modernist Pantry on Amazon. All chemicals were used without further purification.\\' - Fabrication procedures - \"In a typical fabrication, LiCl powder (0.32u20130.82u2009g) is added into 10u2009 mL HPC solution (0u20132.0u2009wt%) forming solution A. The pH of solution A can be tuned by NaOH or HCl solution. 0.44u2009g KGM powder is added into solution A and quickly cast into the petri dish after vortex. The gelation takes place within 2u2009min, and sit in room temperature for 15u2009min. Then, the film is placed in the fridge (u22124u2009xB0C) for 3u2009h followed by 15u2009 min freeze in liquid nitrogen. Last, the gel film is ready to use after 12u2009 h freeze-drying. The final LiCl concentration in SHPFs is characterized by TGA.\" - Characterizations - \"The SEM images were taken by FEI Quanta 650 ESEM to observe the morphology and microstructure of samples. The SHPF samples were freeze-dried for 24u2009h before taking SEM images. The FTIR spectra were conducted by the FTIR Spectrometer (Thermo Mattson, Infinity Gold FTIR) equipped with liquid nitrogen cooled narrow band mercury cadmium telluride detector, using an attenuated total reflection cell equipped with a Ge crystal. The TG analysis was conducted by the thermogravimetric analyzer (PerkinElmer, TGA 4000) with an airflow rate of 25u2009mLu2009min and a heating rate of 10u2009xB0Cu2009min. The sorption-desorption performance was measured by a DVS system (Surface Measurement Systems LTD, DVS Adventure). Samples are preheated at 90u2009xB0C, 0% RH for 60u2009min in the testing chamber for completed drying, and then placed at 25xB0C for 30u2009min stabilization. The desorption process is conducted at 60u2009xB0C, 4.5u221220% RH. The airflow of DVS is fixed at 200u2009mLu2009min for all samples as the standard setting. Since the diameter of the sample holder is 0.8u2009cm, for each measurement, three pieces of SHPF with a diameter of 0.7u2009cm are loaded to fully occupy the sample holder. The evaporation behaviors of gel films (open Al crucible) and their phase transition behaviors (hermetic Al crucible) are evaluated by a differential scanning calorimeter (TA instrument, DSC 250). The scan rate of the DSC test is fixed at 2u2009xB0C min for all samples. XRD profiles were obtained by an X-ray Diffractometer (Rigaku, Miniflex 600). The concentration of ions was tracked by inductively coupled plasma mass spectrometry (ICP-MS, Agilent 7500ce).\" - Water vapor sorption measurement - \"Before the water vapor sorption measurement, all samples are dried in a vacuum oven at 90u2009xB0C for at least 2u2009h. Completely dried SHPF samples were set in a homemade RH-controlled system (Supplementary Fig._). The RH in the testing chamber was stabilized to the desired value by a certain super-saturated solution of a specific salt. The RH of ~15% and ~30% were realized by the super-saturated solution of LiCl and CH3CO2K, respectively, with a 250u2009mLu2009min dry airflow rate. A hygrometer was employed to monitor the RH and temperature in the chamber. Then the weight change was obtained by microbalance. Commercial dehumidifier was used to assist the environmental RH during the testing and measuring steps. For cycling test with water collection measurement, the weight of samples is measured together with the sample holder to reduce experimental errors. To measure the evaporated water accurately, we use a pipette to carefully extract all the collected water and weigh it within a vial (the total mass deducts the mass of the empty vial).\"- - \\' \\' - All the data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary information. - \\' \\'- - \"Mekonnen, M. M. & Hoekstra, A. Y. Four billion people facing severe water scarcity. Sci. Adv. 2, e1500323 (2016)._ _ _ _ _ Zhao, F., Guo, Y., Zhou, X., Shi, W. & Yu, G. Materials for solar-powered water evaporation. Nat. Rev. Mater. 5, 388u2013401 (2020)._ _ _ Schneider, S. H. Encyclopedia of Weather and Climate (Oxford Univ. Press, New York, 1996).Lord, J. et al. Global potential for harvesting drinking water from air using solar energy. Nature 598, 611u2013617 (2021)._ _ _ _ _ Lu, H. et al. Materials engineering for atmospheric water harvesting: progress and perspectives. Adv. Mater. 34, 2110079 (2022)._ _ _ Klemm, O. et al. Fog as a fresh-water resource: overview and perspectives. AMBIO 41, 221u2013234 (2012)._ _ _ _ Wahlgren, R. V. Atmospheric water vapour processor designs for potable water production: a review. Water Res. 35, 1u2013 22 (2001)._ _ _ _ Bintein, P.-B., Lhuissier, H., Mongruel, A., Royon, L. & Beysens, D. Grooves accelerate dew shedding. Phys. Rev. Lett. 122, 098005 (2019)._ _ _ _ _ Climate Research Unit, Univ. of East Anglia. . (1999).Zhou, X., Lu, H., Zhao, F. & Yu, G. Atmospheric water harvesting: a review of material and structural designs. ACS Mater. Lett. 2, 671u2013684 (2020)._ _ _ Wei, X. et al. Hierarchically porous aluminosilicates as the water vapor adsorbents for dehumidification. Chem. Eng. J. 228, 1133u20131139 (2013)._ _ _ Wang, L. W., Wang, R. Z. & Oliveira, R. G. A review on adsorption working pairs for refrigeration. Renew. Sustain. Energ. Rev. 13, 518u2013534 (2009)._ _ _ Tereshchenko, A. G. Deliquescence: hygroscopicity of water-soluble crystalline solids. J. Pharm. Sci. 104, 3639u20133652 (2015)._ _ _ _ Entezari, A., Ejeian, M. & Wang, R. Z. Super atmospheric water harvesting hydrogel with alginate chains modified with binary salts. ACS Mater. Lett. 2, 471u2013477 (2020)._ _ _ Dai, M. et al. A nanostructured moisture absorbing gel for fast and large-scale passive dehumidification. Adv. Mater. 34, 2200865 (2022)._ _ _ Aleid, S. et al. Salting-in effect of zwitterionic polymer hydrogel facilitates atmospheric water harvesting. ACS Mater. Lett. 4, 511u2013520 (2022)._ _ _ Fathieh, F. et al. Practical water production from desert air. Sci. Adv. 4, eaat3198 (2018)._ _ _ _ _ _ Hanikel, N. et al. Rapid cycling and exceptional yield in a metal-organic framework water harvester. ACS Cent. Sci. 5, 1699u20131706 (2019)._ _ _ _ _ Kim, H. et al. Water harvesting from air with metal-organic frameworks powered by natural sunlight. Science 356, 430u2013434 (2017)._ _ _ _ _ Li, R. et al. Hybrid hydrogel with high water vapor harvesting capacity for deployable solar-driven atmospheric water generator. Environ. Sci. Technol. 52, 11367u2013 11377 (2018)._ _ _ _ _ Xu, J. et al. Ultrahigh solar-driven atmospheric water production enabled by scalable rapid-cycling water harvester with vertically aligned nanocomposite sorbent. Energy Environ. Sci. 14, 5979u20135994 (2021)._ _ _ Zhao, F. et al. Super moisture-absorbent gels for all-weather atmospheric water harvesting. Adv. Mater. 31, e1806446 (2019)._ _ _ _ Lu, X., Hu, Z. & Schwartz, J. Phase transition behavior of hydroxypropylcellulose under interpolymer complexation with poly(acrylic acid). Macromolecules 35, 9164u20139168 (2002)._ _ _ _ Zhang, C., Chen, J.-d & Yang, F.-q Konjac glucomannan, a promising polysaccharide for OCDDS. Carbohydr. Polym. 104, 175u2013181 (2014)._ _ _ _ Guo, Y. et al. Biomass-derived hybrid hydrogel evaporators for cost-effective solar water purification. Adv. Mater. 32, e1907061 (2020)._ _ _ _ Li, Z. et al. A novel biocompatible double network hydrogel consisting of konjac glucomannan with high mechanical strength and ability to be freely shaped. J. Mater. Chem. B 3, 1769u20131778 (2015)._ _ _ _ Xu, X., Li, B., Kennedy, J. F., Xie, B. J. & Huang, M. Characterization of konjac glucomannanu2013gellan gum blend films and their suitability for release of nisin incorporated therein. Carbohydr. Polym. 70, 192u2013197 (2007)._ _ _ Bhatt, N., Gupta, P. K. & Naithani, S. Hydroxypropyl cellulose from u03B1-cellulose isolated from Lantana camara with respect to DS and rheological behavior. Carbohydr. Polym. 86, 1519u20131524 (2011)._ _ _ Hu, L. et al. A novel strategy to maintain the long-term viscosity stability of konjac glucomannan hydrosol by using zinc ion. Food Hydrocoll. 108, 106000 (2020)._ _ _ El-Wakil, N. A., Kassem, N. F. & Hassan, M. L. Hydroxypropyl cellulose/rice straw oxidized cellulose nanocrystals nanocomposites and their use in paper coating. Ind. Crops Prod. 93, 186u2013192 (2016)._ _ _ Du, Y. et al. Development of antimicrobial packaging materials by incorporation of gallic acid into Ca crosslinking konjac glucomannan g-1ellan gum films. Int. J. Biol. Macromol. 137, 1076u20131085 (2019)._ _ _ _ DxEDaz-MarxEDn, C. D. et al. Kinetics of sorption in hygroscopic hydrogels. Nano Lett. 22, 1100u20131107 (2022)._ _ _ _ _ Li, R., Shi, Y., Wu, M., Hong, S. & Wang, P. Photovoltaic panel cooling by atmospheric water sorptionu2013 evaporation cycle. Nat. Sustain. 3, 636u2013643 (2020)._ _ Yilmaz, G. et al. Autonomous atmospheric water seeping MOF matrix. Sci. Adv. 6, eabc8605 (2020)._ _ _ _ _ _ Xu, J. et al. Efficient solar-driven water harvesting from arid air with metalu2013organic frameworks modified by hygroscopic salt. Angew. Chem. Int. Ed. 59, 5202u20135210 (2020)._ _ _ Qi, H. et al. An interfacial solar-driven atmospheric water generator based on a liquid sorbent with simultaneous adsorption-desorption. Adv. Mater. 31, e1903378 (2019)._ _ _ _ Yao, H. et al. Highly efficient clean water production from contaminated air with a wide humidity range. Adv. Mater. 32, e1905875 (2020)._ _ _ _ Li, R., Shi, Y., Wu, M., Hong, S. & Wang, P. Improving atmospheric water production yield: Enabling multiple water harvesting cycles with nano sorbent. Nano Energy 67, 104255 (2020)._ _ _ Kim, H. et al. Adsorption-based atmospheric water harvesting device for arid climates. Nat. Commun. 9, 1191 (2018)._ _ _ _ _ _ Ni, F. et al. Tillandsia-inspired hygroscopic photothermal organogels for efficient atmospheric water harvesting. Angew. Chem. Int. Ed. 59, 19237u201319246 (2020)._ _ _ Lei, C. et al. Polyzwitterionic hydrogels for efficient atmospheric water harvesting. Angew. Chem. Int. Ed. 61, e202200271 (2022)._ _ Nakamura, C. et al. Thermoresponsive, freezing-resistant smart windows with adjustable transition temperature made from hydroxypropyl cellulose and glycerol. Ind. Eng. Chem. Res. 58, 6424u20136428 (2019)._ _ _ Zhang, L. et al. Energy-saving smart windows with HPC/PAA hybrid hydrogels as thermochromic materials. ACS Appl. Energy Mater. 4, 9783u20139791 (2021)._ _ _ Rockland, L. B. Saturated salt solutions for static control of relative humidity between 5xB0 and 40u2009xB0C. Anal. Chem. 32, 1375u20131376 (1960)._ _ _ \"- - G.Y. acknowledges the financial support from the U.S. Defense Advanced Research Projects Agency (DARPA) Atmospheric Water Extraction (AWE) Program under Contract No. HR0011-20-C-0055.- - Author notes - \\'These authors contributed equally: Youhong Guo, Weixin Guan.\\' - Authors and Affiliations - \"Materials Science and Engineering Program and Walker Department of Mechanical Engineering, The University of Texas at Austin, Austin, TX, 78712, USAYouhong Guo,_Weixin Guan,_Chuxin Lei,_Hengyi Lu,_Wen Shi_&_Guihua Yu\" - \"AuthorsYouhong GuoYou can also search for this author in _Weixin GuanYou can also search for this author in _Chuxin LeiYou can also search for this author in _Hengyi LuYou can also search for this author in _Wen ShiYou can also search for this author in _Guihua YuYou can also search for this author in _\" - Contributions - G.Y. directed the project. G.Y. and Y.G. conceived the idea. Y.G. and W.G. fabricated samples, conducted experiments, and analyzed the data. C.L., H.L., and S.W. assisted in experiments. All the authors discussed the results and commented on the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks Ruzhu Wang and Jia Zhu for their contribution to the peer review of this work. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0 unported license.\" - Circulating miRNAs dysregulated in blood samples of patients who underwent on-pump CABG and that developed versus failed to develop VS during the preoperative. - \"We identified among the 754 screened miRNAs, eight dysregulated miRNAs in the whole blood of VASO versus NONVASO patients, as shown in a volcano plot, (Fig._ B). Six miRNAs were with increased levels (red dots): hsa-miR-548c-3p, hsa-miR-30d-5p, hsa-miR-199b-5p, hsa-miR-183-3p, hsa-miR-571, hsa-miR-383-5p and two were with decreased levels (blue dots): hsa-miR-1236-3p, hsa-miR-770-5p. Supplementary Table 1 (Table S1) shows the list of the dysregulated miRNAs, their symbol, miRbase ID, fold change, and p-values. As an aid to interpretation, we sorted the dysregulated miRNAs and represented them in grouped box plots (Fig._C). We performed paired Mannu2013Whitney non-parametric test and, for detection of outliers, the ROUT method (based on the False Discovery Rateu2014FDR). After these analyses we observed that seven out of eight miRNAs were statistically significant (Pu2009u2264u20090.05) comparing VASO versus NONVASO, hsa-miR-1236-3p didnxB4t pass the P -value cutoff after the Mannu2013Whitney non-parametric statistical analysis test.\" - Combination of miR-30d-5p and hsa-miR-770-5p in whole blood as potential predictor of VS - \"To explore the potential of miRNAs to predict VS before CABG, ROC curves were generated for single miRNAs and for all possible combinations between two miRNAs, to analyze their synergistic diagnostic value (Fig._Au2013C and Supplementary Table 2 (Table S2). ROC curves for each single miRNAs yielded the top 2 highest AUC values of 0.8333 [95% confidence interval (CI), 0.6622u20131.000; P-valueu2009 =u20090.0047] and 0.8178 (95% CI, 0.6640u20130.9715; P-valueu2009=u20090.0030) for hsa-miR-770-5p and hsa-miR-30d-5p, respectively (Supplementary Table 2). The combination of these two miRNAs yielded an AUC value of 0.9615 (95%CI, 0.973u2013 0.999; Pu2009<u20090.0001) with 84.6% sensitivity and 91.67% specificity in distinguishing patients from VASO from NONVASO groups (Fig._D) showed a superior biomarker power to that of a single miRNA. We also performed ROC curve analyses using the combination of the two miRNAs, hsa-miR-770-5p and hsa-miR-30d-5p with the values of higher intraoperative blood glucose (HIG) parameters and we observed an AUC value increase compared to the one from the combination of the two miRNAs (from 0.9615 to 0.974) as shown in Supplementary Fig._.\" - \"Figure 2(A, B) ROC curve for each single miRNAs yielded the top 2 highest AUC values of 0.8333 [95% confidence interval (CI), 0.6622u20131.000; P-valueu2009 =u20090.0047] and 0.8178 (95% CI, 0.6640u20130.9715; P-valueu2009=u20090.0030) for hsa-miR-770-5p and hsa-miR-30d-5p, respectively. (C) The combination of these two miRNAs yielded an AUC value of 0.9615 (95%CI, 0.973u20130.999; Pu2009 <u20090.0001) (D) The combination showed 84.6% sensitivity and 91.67% specificity in distinguishing patients from VASO from NONVASO groups.\" - MiRNA target prediction and gene set enrichment analysis reveals canonical pathways potentially modulated in patients developing VS - \"We performed computational analyses to identify all potential targets of the seven dysregulated circulating miRNAs in VASO patients. Supplementary Table 3 contain the list of 3699 targets of the seven miRNAs. We obtained the targets list from three different target prediction databases, considering those predicted as moderate, high, or experimentally validated as targets. The genes predicted to be targets of the dysregulated miRNAs in VASO patients were significantly enriched in canonical pathways related to immune response, cellular stress, and injury and cardiovascular signaling. Figure_A shows the top 15 canonical pathways most significantly enriched in stacked bar charts. The total number of molecules within each pathway is at the top of each plotted bar. In this analysis, we applied a Benjaminiu2013Hochberg corrected-log (P-value) cutoff of 1.3. The top enriched pathway was u201CApelin Liver Signaling Pathwayu201D with 14 out of 26 molecules within the pathway (53.8%) containing the higher number of potential targets of the dysregulated miRNAs. In the list of most enriched pathways, we also identified another four pathways related to the apelinergic system e.g., Apelin signaling pathways in cardiac fibroblast, muscle, pancreas and endothelial cells. Figure_B shows as a chord diagram the amount of connection between the four canonical pathways related to the apelin signaling pathways with several entities or nodes (gene/protein/miRNAs) within these pathways or indirectly connected with their nodes. The more interrelatedness and connections the node has more relevant it is and also represented by different colors. The size was proportional to the number of connections with the colors of the chords partially mixed with the colors of their connected entities. The IDs and names of all molecules shown in the chord diagram are listed in Supplementary Table 4.\" - \"Figure 3(A) Top 15 canonical pathways most significantly enriched represented as stacked bar charts. The total number of molecules within each pathway is at the top of each plotted bar. The log (p-values) were Benjaminiu2013Hochberg corrected with a cutoff of 1.3. The top enriched pathway was u201CApelin Liver Signaling Pathwayu201D with 14 out of 26 molecules within the pathway (53.8%) containing the higher number of targets of the dysregulated miRNAs. (B) Chord diagram representing the amount of connection between the four canonical pathways related to the Apelin Signaling Pathway: in the liver, endothelial, muscle/cardiomyocyte, cardiac fibroblasts, and pancreas, with several entities or nodes (gene/protein/miRNAs) within these pathways or indirectly connected with the pathway nodes. The more interrelatedness and connections the node has more relevant it is and also represented by different colors. The size was proportional to the number of connections. In this particular diagram, the colors of the chords partially mixed with the colors of their connected entities.\" - Key molecules from the apelin signaling pathways are the potential targets of the dysregulated circulating miRNAs in patients who evolved with VS. - \"A significant proportion of potential targets of the dysregulated miRNAs in VASO group belong to canonical apelin signaling pathways. Six miRNAs targeted the apelin canonical pathways, as Fig._Au2013E show e.g., the downregulated hsa-miR-770-5p and other five, upregulated, in VASO patients: hsa-miR-199a-5p, hsa-miR-30d-5p, hsa-miR-383-5p, hsa-miR-548c-3p, hsa-miR-571. This pathway is key in different organs/tissues/cells in the body e.g., hepatocytes (Fig._A), cardiomyocytes (Fig._B) and cardiac fibroblasts (Fig._C), pancreatic cells (Fig._D), endothelial cells, and vascular smooth muscle cells (Fig._E). Three nodes are present in all five pathways shown in Figs._Au2013E: Apelin (APLN), hsa-miR-199-5p and its predicted target Apelin receptor (APLNR). In hepatocytes (Fig._A), hsa-miR-199-5p has three predicted targets: the apelin receptor (APLNR), the collagen type 1 (COL5A3) and the glycogen synthase kinase 3 beta (GSK3b). Endothelin 1 (EDN1) and endothelin 3 (EDN3), are predicted targets of hsa-miR-383-5p and hsa-miR-571 respectively, both previously shown able to increase the expression of APLN. Angiotensin 2 (AGT) has an inhibitory effect in APLN. Hsa-miR-30d-5p has three predicted targets: insulin receptor substrate 1 (IRS1), collagen type I alpha chain 1 and 2 (COL1A1 and COL1A2). In cardiomyocytes (Fig._B), the hsa-miR-199-5p has three predicted targets: APLNR, the transcription factor hypoxia-inducible factor 1 subunit alpha (Hif1) and, the enzyme catalase (CAT) which is also a predicted target of hsa-miR-30d-5p. In cardiac fibroblasts (Fig._C), hsa-miR-199-5p and other three miRNAs: hsa-miR-30d-5p, hsa-miR-571 and hsa-miR-548c-5p have AMP-activated kinase (AMPK) as a target in common. These same four miRNAs have different targets within the Apelin signaling in pancreatic cells (Figs._D) in endothelial and vascular muscle cells (Fig._E) leading to a predicted increase in insulin resistance and decrease in vasodilatation, respectively. In endothelial cells, hsa-miR-770-5p is downregulated in the VASO group and has vascular cell adhesion molecule 1 (VCAM1) as a target, a cell surface sialoglycoprotein expressed by cytokine-activated endothelium.\" - \"Figure 4The Apelin signaling pathway represented in different organs/tissues/cells in the body: (A) hepatocytes, (B) cardiomyocytes, (C) cardiac fibroblasts, (D) pancreatic cells, (E) endothelial cells and vascular smooth muscle cells. Six miRNAs have potential targets in each apelin canonical pathway. Based on their expression, these molecules were colored in red and green when upregulated or downregulated, respectively. MAP enabled the visualization of the overall effect on the pathway producing results where molecules or functions that are predicted to be activated or inhibited are colored in shades of orange and blue, respectively. Figures_4A to E were partly generated using Servier Medical Art, provided by Servier, licensed under a Creative Commons Attribution 3.0 unported license.\"- - \"In the present caseu2013control study nested in a cohort of patients who developed VS we showed, for the first time, a possible link between miRNA and VS. Our results provide clues about miRNA signatures in this complication that affects 9u201344% of patients after cardiac surgery and with a mortality rate of 25%. Patients with VS develop a systemic inflammatory response leading to endothelial dysregulation, systemic vasodilation refractory to vasopressors, hypotension and progressive death. Despite the lack of information regarding miRNAs in VS, some inflammatory diseases, such as sepsis, systemic inflammatory response syndrome, and sepsis-related acute lung injury, that have a similar mechanism as VS, were studied and also show a correlation with dysregulation levels of hsa-miR-30d-5p in body fluids. Here we show that in comparison to healthy controls, the levels of miR-30d-5p are increased in whole blood of VS patients, suggesting a possible anti-inflammatory role for this miRNA. A study in patients with heart failure and desynchrony (HFDYS), showed that miR-30d appears to be regulated by heart wall stress, consistent with its increased expression in the lateral wall in a canine model of HFDYS, which appears to play a functional role in cardiomyocyte biology, mediating cardiac hypertrophy with a distinct molecular signature and protecting against TNF-u03B1u2013induced apoptosis and deleterious signaling. This information could suggest that miRNA-30d might also play a similar role in VS, regulating the inflammatory response and endothelial gene expression dysregulation. MiRNA-770 was previously shown to be associated in regulation of apoptosis and cell proliferation and in different disease states as in diabetes, dilated cardiomyopathy, and different types of cancers, such as non-small cell lung, breast and gastric cancer. Here, we show, by computational prediction analyses, that the dysregulated miRNAs have a potential role in controlling targets related to the apelinergic system, responsible for signaling events related to severe and refractory hypotension, a common outcome in patients who progress to VS. One of the potential targets of the dysregulated miRNAs, described here, is the receptor of apelin (APLNR), a protein secreted from endothelial cells and induced by angiopoietin 1 (Ang1) which functions as regulator of vascular tone and blood pressure. Apelin/APLNR are highly expressed in endothelial cells acting in the vasodilatation mechanism as well in the regulation of blood vessel diameter during angiogenesis. Ishida et al., demonstrated that the acute administration of apelin significantly decreased the systolic blood pressure in wild type mice, but was unable to change the systolic blood pressure in mice deficient for APLNR. Interestingly, the hypotensive effect of apelin was also observed in chronic hypertensive model animals. Studies in rats revellead that apelin/APLNR are localized within the endothelia of small arteries in several organs such as the liver, spleen, lung, pancreas, intestine, kidney, and adipose tissues. Apelin causes endothelium-dependent vasorelaxation by triggering the release of nitric oxide (NO), and this effect can be almost completely abolished in the presence of endothelial NO synthase (eNOS) inhibitor, suggesting that the apelin/APLNR pathway exerts a vasorelaxation effect via activation of the eNOS pathway. Although we obtained statistically significant results and we show here that the measured levels of miRNAs, hsa-miR-30d and hsa-miR-770-5p have a potential ability to distinguish patients who develop VS from those who didnxB4t, we acknowledge the main limitation of the present work, which is the small number of participants. However, the findings of this study offer new, potentially useful information to improve the prognostic outcome of VS and also opening vistas to a potential new clinical biomarker to help the management of patients referred to cardiac surgery. The computational analysis also points out to the importance in further investigate the miRNAs/apelin/APLNR signaling pathway in the refractory hypotension observed in patients who develop VS.\"- - \\' \\' - All data generated or analyzed during this study are included in this published article (and its Supplementary Information files). - \\' \\'- - \\'CABG: Coronary artery bypass grafting VS: Vasoplegic syndrome MiRNA: MicroRNA EuroScore II: European System for Cardiac Operative Risk Evaluation STS: Society of Thoracic Surgeons. APLNR: Apelin receptor COL5A3: Collagen type 1 GSK3b: Glycogen synthase kinase 3 beta EDN1: Endothelin 1 EDN3: Endothelin 3 AGT: Angiotensin 2 IRS1: Insulin receptor substrate 1 and 2 COL1A1: Collagen type I alpha chain 1 COL1A2: Collagen type I alpha chain 2 Hif1: Hypoxia-inducible factor 1 subunit alpha CAT: Catalase AMPK: AMP-activated kinase VCAM1: Vascular cell adhesion molecule 1 \\'- - \"Zhou, Z. et al. Association of post-operative systolic blood pressure variability with mortality after coronary artery bypass grafting. Front. Cardiovasc. Med. 8, 717073, (2021).Kalender, M. et al. Validation of EuroSCORE II risk model for coronary artery bypass surgery in high-risk patients. Kardiochir Torakochirurgia Pol 11, 252u2013256. (2014)._ _ Goetzenich, A. et al. EuroScore 2 for identification of patients for transapical aortic valve replacementu2013a single center retrospective in 206 patients. J. Cardiothorac. Surg. 7, 89. (2012)._ _ Petermichl, W. et al. The additional use of methylene blue has a decatecholaminisation effect on cardiac vasoplegic syndrome after cardiac surgery. J. Cardiothorac. Surg. 16, 205. (2021)._ _ Lewis, B. P., Burge, C. B. & Bartel, D. P. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets. Cell 120, 15u201320. (2005)._ _ _ Ambros, V. The functions of animal microRNAs. Nature 431, 350u2013 355. (2004)._ _ _ _ Ambros, V. & Chen, X. The regulation of genes and genomes by small RNAs. Development 134, 1635u2013 1641. (2007)._ _ _ Zhou, S. S. et al. miRNAS in cardiovascular diseases: potential biomarkers, therapeutic targets and challenges. Acta Pharmacol. Sin. 39, 1073u20131084. (2018)._ _ _ Bartel, D. P. Metazoan MicroRNAs. Cell 173, 20u201351. (2018)._ _ _ Rigaud, V. O. et al. Circulating miR-1 as a potential biomarker of doxorubicin-induced cardiotoxicity in breast cancer patients. Oncotarget 8, 6994u20137002, (2017).Lorenzo-Almoros, A. et al. Predictive and diagnostic biomarkers for gestational diabetes and its associated metabolic and cardiovascular diseases. Cardiovasc. Diabetol. 18, 140. (2019)._ _ _ James, A. & Amour, J. Vasopressin versus norepinephrine in patients with vasoplegic shock after cardiac surgery: A discussion of the level of evidence. Anesthesiology 128, 228. (2018)._ _ Evora, P. R. et al. Is the cyclic GMP system underestimated by intensive care and emergency teams?. Med. Hypotheses 69, 564u2013567. (2007)._ _ _ Formosa, A., Turgeon, P. & Dos Santos, C. C. Role of miRNA dysregulation in sepsis. Mol. Med. 28, 99. (2022)._ _ _ Caserta, S. et al. Circulating Plasma microRNAs can differentiate human sepsis and systemic inflammatory response syndrome (SIRS). Sci. Rep. 6, 28006. (2016)._ _ _ _ Jiao, Y. et al. Exosomal miR-30d-5p of neutrophils induces M1 macrophage polarization and primes macrophage pyroptosis in sepsis-related acute lung injury. Crit. Care 25, 356. (2021)._ _ Caserta, S. et al. Severity of systemic inflammatory response syndrome affects the blood levels of circulating inflammatory-relevant MicroRNAs. Front. Immunol. 8, 1977. (2017)._ _ _ Melman, Y. F. et al. Circulating MicroRNA-30d Is associated with response to cardiac resynchronization therapy in heart failure and regulates cardiomyocyte apoptosis: A translational pilot study. Circulation 131, 2202u20132216. (2015)._ _ _ Zhang, J. F. et al. MicroRNA-770 affects proliferation and cell cycle transition by directly targeting CDK8 in glioma. Cancer Cell Int. 18, 195. (2018)._ _ _ Wang, M., Wei, J., Ji, T. & Zang, K. miRNA-770-5p expression is upregulated in patients with type 2 diabetes and miRNA-770-5p knockdown protects pancreatic beta-cell function via targeting BAG5 expression. Exp. Ther. Med. 22, 664. (2021)._ _ _ Huang, K. et al. Integrated analysis of hub genes and miRNAs in dilated cardiomyopathy. Biomed. Res. Int. 2020, 8925420. (2020)._ _ _ _ Liu, J. et al. Tumor cell-derived exosomal miR-770 inhibits M2 macrophage polarization via targeting MAP3K1 to inhibit the invasion of non-small cell lung cancer cells. Front. Cell Dev. Biol. 9, 679658, (2021).Li, Y. et al. MiR-770 suppresses the chemo-resistance and metastasis of triple negative breast cancer via direct targeting of STMN1. Cell Death Dis. 9, 14. (2018)._ _ _ Chen, B., Ji, F., Wen, X. & Jin, Z. Circular RNA circ_ASAP2 promotes cell viability, migration, and invasion of gastric cancer cells by regulating the miR-770-5p/CDK6 axis. Int. J. Clin. Exp. Pathol. 13, 2806u20132819 (2020)._ _ Lambden, S., Creagh-Brown, B. C., Hunt, J., Summers, C. & Forni, L. G. Definitions and pathophysiology of vasoplegic shock. Crit. Care 22, 174. (2018)._ _ Hwangbo, C. et al. Endothelial APLNR regulates tissue fatty acid uptake and is essential for apelin\\'s glucose-lowering effects. Sci. Transl. Med. 9, (2017).Kidoya, H. et al. Spatial and temporal role of the apelin/APJ system in the caliber size regulation of blood vessels during angiogenesis. EMBO J. 27, 522u2013 534. (2008)._ _ _ Besserer-Offroy, E. et al. The hypotensive effect of activated apelin receptor is correlated with beta-arrestin recruitment. Pharmacol. Res. 131, 7u201316. (2018)._ _ Tatemoto, K. et al. The novel peptide apelin lowers blood pressure via a nitric oxide-dependent mechanism. Regul. Pept. 99, 87u201392. (2001)._ _ _ \"- - \"This work was supported by FundaxE7xE3o de Pesquisa do Estado de SxE3o Paulo (FAPESP 2016/23630u20138). LRPF is recipient of the National Council of Research-CNPq productivity grant in the field of Immunology and is supported by the Brazilian National Institute of Science and Technology for Vaccines (INCT-Vacinas/CNPq- Project APQ 03608u201317) and FundaxE7xE3o de Pesquisa do Estado de Minas Gerais (FAPEMIGu2014Project APQ-03551u201318).\"- - Author notes - \"Vagner Oliveira Carvalho RigaudPresent address: Center for Metabolic Disease Research, Temple University, Philadelphia, PA, 19122, USAThese authors contributed equally: Omar AsdrxFAbal Vilca Mejia and Renato Cesar de Souza.These authors jointly supervised this work: Ludmila Rodrigues Pinto Ferreira and FxE1bio Biscegli Jatene.\" - Authors and Affiliations - \"Departamento de Cardiopneumologia, Instituto do CoraxE7xE3o - InCor, Hospital das Clinicas HCFMUSP, Faculdade de Medicina, Universidade de Sao Paulo, SxE3 o Paulo, SP, 05403-900, BrazilOmar AsdrxFAbal Vilca Mejia,_Renato Cesar de Souza,_ Bianca Meneghini,_LuxEDs Roberto Palma Dallan,_Luiz Augusto Ferreira Lisboa,_ LuxEDs Alberto Oliveira Dallan_&_FxE1bio Biscegli JateneDepartment of Cardiovascular Surgery, Hospital Samaritano Paulista, SxE3o Paulo, SP, 01333-030, BrazilOmar AsdrxFAbal Vilca MejiaInstituto Estadual de Cardiologia Aloysio de Castro, Rio de Janeiro, RJ, 22261-03, BrazilRenato Cesar de SouzaMarinha Do Brasil, Hospital Naval MarcxEDlio Dias, Rio de Janeiro, RJ, 20725-090, BrazilRenato Cesar de SouzaLaboratxF3rio de Carboidratos E Radioimunoensaios (LIM-18), Faculdade de Medicina FMUSP, Universidade de SxE3o Paulo, SxE3o Paulo, SP, 01246-100, BrazilAritania S. SantosRNA Systems Biology Laboratory, Department of Morphology, Biological Sciences Institute, Federal University of Minas Gerais (UFMG), Av. AntxF4nio Carlos, 6627 - Campus Pampulha. Bloco J4, Sala 158, Belo Horizonte, MG, 31270-910, BrazilAna Carolina Carvalho Silva_&_Ludmila Rodrigues Pinto FerreiraLaboratory of Immunology, Heart Institute, InCor, School of Medicine, University of SxE3o Paulo, SxE3o Paulo, SP, 05403-900, BrazilVagner Oliveira Carvalho Rigaud,_Jorge Kalil_&_Edecio Cunha-NetoInstitute for Investigation in Immunology (Iii) INCT, SxE3o Paulo, BrazilJorge Kalil_&_Edecio Cunha-NetoNational Institute of Science and Technology for Vaccines (INCTV), Belo Horizonte, MG, BrazilLudmila Rodrigues Pinto FerreiraLaboratxF3rio CirxFArgico de Pesquisa Cardiovascular (LIM-11), Faculdade de Medicina, Instituto Do CoraxE7xE3o InCor, Hospital das ClxEDnicas HCFMUSP, Universidade de SxE3o Paulo, SxE3o Paulo, 05403-900, BrazilLuiz Felipe Pinho MoreiraDepartamento de Cirurgia, Faculdade de Medicina, Universidade Federal de GoixE1s (UFG), GoixE2nia, GO, BrazilGuilherme Visconde Brasil\" - \"AuthorsOmar AsdrxFAbal Vilca MejiaYou can also search for this author in _Renato Cesar de SouzaYou can also search for this author in _Aritania S. SantosYou can also search for this author in _Bianca MeneghiniYou can also search for this author in _Ana Carolina Carvalho SilvaYou can also search for this author in _Guilherme Visconde BrasilYou can also search for this author in _Vagner Oliveira Carvalho RigaudYou can also search for this author in _LuxEDs Roberto Palma DallanYou can also search for this author in _Luiz Felipe Pinho MoreiraYou can also search for this author in _Luiz Augusto Ferreira LisboaYou can also search for this author in _LuxEDs Alberto Oliveira DallanYou can also search for this author in _Jorge KalilYou can also search for this author in _Edecio Cunha-NetoYou can also search for this author in _ Ludmila Rodrigues Pinto FerreiraYou can also search for this author in _FxE1bio Biscegli JateneYou can also search for this author in _\" - Contributions - O.A.V.M., R.C.S., B.M. and L.R.P.F. wrote the main manuscript text. O.A.V.M. and R.C.S. designed and coordinated the clinical part of the study. L.R.P.F. and A.C.C.S. created all the figures of the article and did the statistical analysis of the data. L.R.P.F., E.C.N. and A.C.C.S. coordinated the laboratory part of the study. A.S.S., G.V.B., V.O.C.R., L.R.P.D. L.F.P.M., L.A.F.L., L.A.O.D., J.K., and F.B.J. were responsible for data collection of the patients and storage of the organic material. All authors reviewed the manuscript. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0. For the female avatar, the same model was used, but the hair was created separately by our lab. - \"The VR Chat Application was implemented as a Unity application running on a Windows PC and includes the body tracker, and experiment overlay. It includes the body tracker which uses the Kinect for Windows SDK 2.0 and the corresponding Unity plugin. The body model used in the study was AJ 3D Model by Mixamo (Adobe Systems). All Kinect joints from spine base and upward were mapped to the model depicted in Fig._. While the Kinect reports joint rotation, we found that this performed poorly on arm joints and therefore rotation data were only used for spine joints. Arm, hand and shoulder joint rotation was inferred using inverse kinematics. A detailed list of the software programs used in the current study is as follows:\" - SoftwareVersionUnity2018.1.6f1Kinect for Windows SDK2.0.1410.19000iOS on iPhone XS12.1protobuf3.1.0python3.7.4numpy1.16.4pandas0.24.2sklearn0.21.3 - Control panel - \"The control panel was implemented as a Unity application running on a Windows PC. It allows the Experimenter to monitor the tracking and connection status of all trackers. It was also used to configure, calibrate, start, record responses, pause, resume and conclude the experiment. A diagram of how the body and face tracking data were processed can be seen in Fig._ and a network diagram of the connections between the devices is available in Fig. ._\" - Figure 13Sequence chart of how body and face tracking data were processed. - Figure 14Network diagram of the connections between the devices. - \\'Appendix 3: Latency assessment for experimental setting\\' - System latency was computed based on the latency of the subsystems. The latency of each individual component is listed in the table below. ARKit provides a capture timestamp, which was used to measure capture delay throughout the study. As the Kinect lacks this feature, we relied on previous research by Waltemate and colleagues. We observed network latency and variance for the face trackers that we connected via wireless network. In order to achieve the required time synchronization between trackers, we timestamped a message when captured, sent, received and rendered, and use a time synchronization approach to calculate time offset and network delay. The rolling average and standard deviation of the calculated latencies were logged every second. We calculate the render delay as the difference between the time the data is received and the time when Unity completed rendering the frame. - \"While 100_ms is established as a safe latency that ensures user satisfaction in video conferencing, a more recent study suggests that latencies as high as 500_ms do not have a significant negative impact on likeability and naturalness._ Of note, there were no complaints regarding system performance during the pilot study with 40 participants, which is expected as our total latency was within the established target range. In addition to the approach taken in the present study, future studies may also benefit from conducting a video-based assessment in order to determine motion-to-photon latencies.\" - \"_Body trackingFace trackingSensor/capture delay98.8u2009xB1u200919.2_ms84.8u2009 xB1u20099.0_msNetwork stream latency<u20091_ms8.5u2009xB1u200933.9_msRender delay30.4u2009xB1u200910.9_msDisplay response delay8_ms*Total138.2u2009 xB1u200922.1_ms131.7u2009xB1u200936.7_ms*As reported by the display manufacturer.\" - \\'Appendix 4: Measures for social presence, interpersonal attraction, and impression accuracy (BFI-S)\\' - \\'Social presence \\' - How strongly do you agree or disagree with the following statements about your partner? - 1234567Strongly disagreeDisagreeSomewhat disagreeNeither agree nor disagreeSomewhat agreeAgreeStrongly agree - \\' 1. I felt that my partner was present. 2. I felt that my partner was aware of my presence. 3. I paid close attention to my partner. 4. My partner paid close attention to me. 5. I was influenced by my partner\\'\\'s emotions. 6. My partner was influenced by my emotions. 7. My thoughts were clear to my partner. 8. My partner\\'\\'s thoughts were clear to me. \\' - \\'Interpersonal attraction \\' - Task attraction - How strongly do you agree or disagree with the following statements about your experience? - 1234567Strongly disagreeDisagreeSomewhat disagreeNeither agree nor disagreeSomewhat agreeAgreeStrongly agree - \\' 1. I enjoyed completing the tasks with my partner. 2. I had fun completing the tasks with my partner. 3. I would like to interact with my partner again. 4. It was interesting to complete the tasks with my partner. \\' - Social attraction - How strongly do you agree or disagree with the following statements about your partner? - 1234567Strongly disagreeDisagreeSomewhat disagreeNeither agree nor disagreeSomewhat agreeAgreeStrongly agree - \\' 1. I like my partner. 2. I would get along well with my partner. 3. I would enjoy a casual conversation with my partner. 4. My partner is friendly. \\' - Impression Accuracy (Short 15-item big five inventory; BFI-S) - BFI-S observer version - \"You will now see a number of statements, each of which starts with, \"I see_ MY PARTNER_as someone whou2026\"._For each statement, indicate how much you agree or disagree with this._If you are unable to make a judgment, select \" Cannot make judgment\".\" - \"1234567_N/AStrongly disagreeDisagreeSomewhat disagreeNeither agree nor disagreeSomewhat agreeAgreeStrongly agreeCannot make judgment\" - BFI-S self version - \"You will now see a number of statements, each of which starts with, \"I see_ MYSELF as someone whou2026\"._For each statement, indicate how much you agree or disagree with this.\" - 1234567Strongly disagreeDisagreeSomewhat disagreeNeither agree nor disagreeSomewhat agreeAgreeStrongly agree - TraitItemsOpenness to experiencecomes up with new ideasvalues artistic experienceshas an active imaginationConscientiousnessdoes a thorough jobtends to be lazydoes things efficientlyExtroversionis talkativeis outgoingis reservedAgreeablenessis sometimes rude to othershas a forgiving natureis kindNeuroticismworries a lotgets nervous easilyremains calm in tense situations- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 3.0.\"- - \\' \\' - All data are available in the main text or supplementary materials. - \\' \\'- - \\' \\' - All the relevant code used to generate the results in this paper and the Supplementary Information is available upon request. - \\' \\'- - \"Mack, M. J. Minimally invasive and robotic surgery. JAMA 285, 568u2013572 (2001)._ _ _ _ Burgner-Kahrs, J., Rucker, D. C. & Choset, H. Continuum robots for medical applications: A survey. IEEE Trans. Robot. 31, 1261u20131280 (2015)._ _ Peters, B. S., Armijo, P. R., Krause, C., Choudhury, S. A. & Oleynikov, D. Review of emerging surgical robotic technology. Surg. Endosc. 32, 1636u20131655 (2018)._ _ _ Yang, G.-Z. et al. The grand challenges of Science Robotics. Sci. Robot. 3, eaar7650 (2018)._ _ _ Dupont, P. E. et al. A decade retrospective of medical robotics research from 2010 to 2020. Sci. Robot. 6, eabi8017 (2021)._ _ _ _ Wang, B. et al. Endoscopy-assisted magnetic navigation of biohybrid soft microrobots with rapid endoluminal delivery and imaging. Sci. Robot. 6, eabd2813 (2021)._ _ _ Saeidi, H. et al. Autonomous robotic laparoscopic surgery for intestinal anastomosis. Sci. Robot. 7, eabj2908 (2022)._ _ _ _ _ Gopesh, T. et al. Soft robotic steerable microcatheter for the endovascular treatment of cerebral disorders. Sci. Robot. 6, eabf0601 (2021)._ _ _ _ Suzuki, H. & Wood, R. J. Origami-inspired miniature manipulator for teleoperated microsurgery. Nat. Mach. Intell. 2, 437u2013446 (2020)._ _ Norton, J. C. et al. Intelligent magnetic manipulation for gastro intestinal ultrasound. Sci. Robot. 4, eaav7725 (2019)._ _ _ _ Mitros, Z., Sadati, S. M. H., Henry, R., Da Cruz, L. & Bergeles, C. From theoretical work to clinical translation: progress in concentric tube robots. Annu. Rev. Control. Robot. Auton. Syst. 5, 335u2013359 (2022)._ _ Li, J. & Pumera, M. 3D printing of functional microrobots. Chem. Soc. Rev. 50, 2794u20132838 (2021)._ _ _ _ AlcxE2 ntara, C. C. J. et al. Mechanically interlocked 3D multi-material micromachines. Nat. Commun. 11, 5957 (2020)._ _ _ _ _ Liu, Z. et al. Creating three-dimensional magnetic functional microdevices via molding-integrated direct laser writing. Nat. Commun. 13, 2016 (2022)._ _ _ _ _ _ Gu, H. et al. Artificial microtubules for rapid and collective transport of magnetic microcargoes. Nat. Mach. Intell. 4, 678u2013684 (2022)._ _ Kim, Y., Parada, G. A., Liu, S. & Zhao, X. Ferromagnetic soft continuum robots. Sci. Robot. 4, eaax7329 (2019)._ _ _ Edelmann, J., Petruska, A. J. & Nelson, B. J. Magnetic control of continuum devices. Int. J. Rob. Res. 36, 68u201385 (2017)._ _ Nelson, B. J., Kaliakatsos, I. K. & Abbott, J. J. Microrobots for minimally invasive medicine. Annu. Rev. Biomed. Eng. 12, 55u201385 (2010)._ _ _ _ Jeon, S. et al. A magnetically controlled soft microrobot steering a guidewire in a three-dimensional phantom vascular network. Soft Robot. 6, 54u201368 (2019)._ _ _ _ Hu, X., Chen, A., Luo, Y., Zhang, C. & Zhang, E. Steerable catheters for minimally invasive surgery: a review and future directions. Comput. Assist. Surg. 23, 21u201341 (2018)._ _ Rafii-Tari, H., Payne, C. J. & Yang, G.-Z. Current and emerging robot-assisted endovascular catheterization technologies: a review. Ann. Biomed. Eng. 42, 697u2013715 (2014)._ _ _ Au, S., Ko, K., Tsang, J. & Chan, Y. C. Robotic endovascular surgery. Asian Cardiovasc. Thorac. Ann. 22, 110u2013114 (2014)._ _ _ Runciman, M., Darzi, A. & Mylonas, G. P. Soft robotics in minimally invasive surgery. Soft Robot. 6, 423u2013443 (2019)._ _ _ _ Cianchetti, M. et al. Soft robotics technologies to address shortcomings in todays minimally invasive surgery: the STIFF-FLOP approach. Soft Robot. 1, 122u2013131 (2014)._ _ Cianchetti, M., Laschi, C., Menciassi, A. & Dario, P. Biomedical applications of soft robotics. Nat. Rev. Mater. 3, 143u2013153 (2018)._ _ _ Shepherd, R. F. Multigait soft robot. Proc. Natl Acad. Sci. USA 108, 20400u201320403 (2011)._ _ _ _ _ _ Knaian, A. N. et al. The Milli-Motein: A self-folding chain of programmable matter with a one-centimeter module pitch. In 2012 IEEE/RSJ International Conference on Intelligent Robots and Systems 1447u20131453 (2012).Ranzani, T., Gerboni, G., Cianchetti, M. & Menciassi, A. A bioinspired soft manipulator for minimally invasive surgery. BioinspirationBiomim. 10, 35008 (2015)._ _ Mao, G. et al. Soft electromagnetic actuators. Sci. Adv. 6, eabc0251 (2020)._ _ _ _ _ _ Tortora, G., Dario, P. & Menciassi, A. Array of robots augmenting the kinematics of endocavitary surgery. IEEE/ASME Trans. Mechatron. 19, 1821u20131829 (2014)._ _ Park, S. et al. Trocar-less instrumentation for laparoscopy: magnetic positioning of intra-abdominal camera and retractor. Ann. Surg. 245, 379 (2007)._ _ _ _ Liu, K., Hacker, F. & Daraio, C. Robotic surfaces with reversible, spatiotemporal control for shape morphing and object manipulation. Sci. Robot. 6, eabf5116 (2021)._ _ _ Hawkes, E. et al. Programmable matter by folding. Proc. Natl Acad. Sci. 107, 12441u201312445 (2010)._ _ _ _ _ _ Soto, F. et al. Smart materials for microrobots. Chem. Rev. 122, 5365u20135403 (2021)._ _ _ Medina-SxE1 nchez, M., Magdanz, V., Guix, M., Fomin, V. M. & Schmidt, O. G. Swimming microrobots: Soft, reconfigurable, and smart. Adv. Funct. Mater. 28, 1707228 (2018)._ _ Gu, H. et al. Magnetic cilia carpets with programmable metachronal waves. Nat. Commun. 11, 2637 (2020)._ _ _ _ _ _ Mattmann, M. et al. Thermoset shape memory polymer variable stiffness 4d robotic catheters. Adv. Sci. 9, 2103277 (2022)._ _ Lussi, J. et al. A submillimeter continuous variable stiffness catheter for compliance control. Adv. Sci. 8, 2101290 (2021)._ _ Chautems, C. et al. Magnetic continuum device with variable stiffness for minimally invasive surgery. Adv. Intell. Syst. 2, 1900086 (2020)._ _ Niu, R. et al. Magnetic handshake materials as a scale-invariant platform for programmed self-assembly. Proc. Natl Acad. Sci. 116, 24402u2013 24407 (2019)._ _ _ _ _ _ Gu, H., Boehler, Q., Ahmed, D. & Nelson, B. J. Magnetic quadrupole assemblies with arbitrary shapes and magnetizations. Sci. Robot. 4, eaax8977 (2019)._ _ _ Tunay, I. Modeling magnetic catheters in external fields. In The 26th Annual International Conference of the IEEE Engineering in Medicine and Biology Society vol. 1 2006u20132009 (2004).Peyron, Q. et al. Kinematic analysis of magnetic continuum robots using continuation method and bifurcation analysis. IEEE Robot. Autom. Lett. 3, 3646u20133653 (2018)._ _ Chautems, C., Lyttle, S., Boehler, Q. & Nelson, B. J. Design and evaluation of a steerable magnetic sheath for cardiac ablations. IEEE Robot. Autom. Lett. 3, 2123u20132128 (2018)._ _ Han, M. et al. Catheter-integrated soft multilayer electronic arrays for multiplexed sensing and actuation during cardiac surgery. Nat. Biomed. Eng. 4, 997u20131009 (2020)._ _ _ _ _ Rivkin, B. et al. Electronically integrated microcatheters based on self-assembling polymer films. Sci. Adv. 7, eabl5408 (2021)._ _ _ _ _ _ Rivkin, B. et al. Shape-controlled flexible microelectronics facilitated by integrated sensors and conductive polymer actuators. Adv. Intell. Syst. 3, 2000238 (2021)._ _ Singh, M. et al. Minimally invasive electroceutical catheter for endoluminal defect sealing. Sci. Adv. 7, eabf6855 (2021)._ _ _ _ _ Barbot, A., Tan, H., Power, M., Seichepine, F. & Yang, G.-Z. Floating magnetic microrobots for fiber functionalization. Sci. Robot. 4, eaax8336 (2019)._ _ _ Huang, T.-Y., Gu, H. & Nelson, B. J. Increasingly intelligent micromachines. Annu. Rev. Control. Robot. Auton. Syst. 5, 279u2013310 (2022)._ _ Martin, J. W. et al. Enabling the future of colonoscopy with intelligent and autonomous magnetic manipulation. Nat. Mach. Intell. 2, 595u2013606 (2020)._ _ _ _ \"- - \"We would like to thank Jung-Chew Tse for his assistance in 3D printing, Alex Mesot, Roland Dreyfus, and Michelle Mattille for their assistance in the foldable endoscope experiments, and E. Zuurmond for proofreading this article. This work was financially supported by the European Research Council Advanced Grantu2013 Soft MicroRobots (SOMBOT, number 743217) and the SNSF PostDoc. Mobility Grant (203203). M.K. acknowledges support from the Swiss National Science Foundation under Project No. 200021L_197017.\"- - Authors and Affiliations - \"Institute of Robotics and Intelligent Systems, ETH Zurich, Zurich, SwitzerlandHongri Gu,_Marino MxF6ckli,_Claas Ehmke,_Minsoo Kim,_Matthias Wieland,_Simon Moser,_Quentin Boehler_&_Bradley J. NelsonDepartment of Physics, University of Konstanz, Konstanz, GermanyHongri Gu_&_Clemens Bechinger\" - \"AuthorsHongri GuYou can also search for this author in _Marino MxF6ckliYou can also search for this author in _Claas EhmkeYou can also search for this author in _Minsoo KimYou can also search for this author in _Matthias WielandYou can also search for this author in _Simon MoserYou can also search for this author in _Clemens BechingerYou can also search for this author in _Quentin BoehlerYou can also search for this author in _Bradley J. NelsonYou can also search for this author in _\" - Contributions - H.G. and Q.B. conceived the idea and initiated the project. M.W., M.M., S.M., and H.G. designed and fabricated the MaSoChain samples. C.E. and M.M. designed the flexible PCBs. M.K. performed the force test during the assembly and disassembly processes. C.B. participated in discussions. H.G. wrote the manuscript with contributions from all authors. Q.B. and B.J.N. supervised this project. - Corresponding authors - \"Correspondence to , , or .\"- - \\' \\' - Competing interests - \\' \\' - All authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \"Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work._ are available.\" - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 Generic License has already been assigned to the Author Accepted Manuscript version that might arise from this submission. MJdV, SN, and TBH gratefully acknowledge funding from the Vaccine Impact Modelling Consortium (VIMC; ) based at Imperial College London, UK. This award is jointly funded by the UK Medical Research Council (MRC) and the UK Department for International Development (DFID) under the MRC/DFID Concordat agreement and is also part of the EDCTP2 programme supported by the European Union. SN is also supported by the NIHR Imperial Biomedical Research Centre (BRC; ). The authors would also like to thank Meg Doherty, Niklas Luhmann, and Fuqiang Cui at the World Health Organization for their contributions towards developing realistic scenarios for investigating the impact of possible COVID-19 pandemic disruptions on HBV disease burden.- - Authors and Affiliations - \"MRC Centre for Global Infectious Disease Analysis, School of Public Health, Imperial College London, London, UKMargaret J. de Villiers,_Shevanthi Nayagam_&_Timothy B. HallettSection of Hepatology & Gastroenterology, Department of Metabolism, Digestion & Reproduction, Imperial College London, London, UKShevanthi Nayagam\" - \"AuthorsMargaret J. de VilliersYou can also search for this author in _Shevanthi NayagamYou can also search for this author in _Timothy B. HallettYou can also search for this author in _\" - Contributions - SN and TBH secured funding. SN, TBH and MJdV constructed the disease transmission model. SN and TBH designed the study. MJdV wrote computer code, conducted the analyses and prepared results. MJdV wrote the first draft of the paper, and the other authors reviewed drafts. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - \\'The authors declare the following competing interests: This work was carried out as part of the Vaccine Impact Modelling Consortium (), but the views expressed are those of the authors and not necessarily those of the Consortium or its funders. The funders were given the opportunity to review this paper prior to publication, but the final decision on the content of the publication was taken by the authors. SN and TBH have received personal fees for technical consultation work with the World Health Organization on Hepatitis elimination, surveillance and monitoring, and for the preparation of reports for guidelines committees on the prevention of mother-to-child transmission. The other authors declare no competing interests.\\' - \\' \\'- - Peer review information Nature Communications thanks Anna McNaughton, David van de Vijver, Nina Weis and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International (CC BY 4.0) license).\" - \"Figure 2Temporal distribution of neutral PFASs.FTOHs and other neutral PFAS compounds measured in air at Ny-xC5lesund, in the Arctic from Sep. 2011 to Sep. 2012. The black line indicates average air temperatures.\" - \"Figure 3Atmospheric concentration of neutral PFASs.Graph indicating the concentration of the neutral PFASs in air collected from Ny-xC5lesund, the Arctic.\" - \"The average ratio of 8:2 to 10:2 to 6:2 to 12:2 FTOH in air obtained in this study was 4.8:0.9:1.0:0.4. The ratio can be considered as an indicator of long range atmospheric transport due to the longer atmospheric residence time of FTOH 8:2 (80u2005d) than that of FTOH 6:2 (50u2005d). Higher ratios were measured in the Arctic air compared with urban/-semi-urban areas, such as 1.1:0.2:1.0 for 8:2 to 10:2 to 6:2 FTOH in semi-urban location in Toronto, Canada and 1.8:0.6:1.0 in the southeast of Hamburg, Germany, while the ratios were similar to the reported at remote sites, such as in the Mount Bachelor Observatory, the Canadian Arctic and the North Sea. The results confirmed that the FTOHs in Arctic air were from long range atmospheric transport process.\" - \"To investigate the relationship between the variation of gas-particle partitioning of PFASs in the atmosphere and vapor pressure, the linear relationships between logKSP and logpxB0L for all the 45 air samples were estimated. The physicochemical calculator SPARC v4.6 (October 2011 release w4.6.1691u2013s4.6.1687) was employed to calculate the values of pxB0L for individual PFASs at different sampling temperatures (). Considering the pKa values of 6:2 and 8:2, FTA were below the environmentally relevant pH range (u22647) and they mainly exist as anions in aerosols as stated by Wang et al.. The corrected logKSP (neutral form) of 6:2 and 8:2 FTA were considerably lower than those of FTOHs, FOSEs and FOSAs with similar vapor pressures, thus, they were excluded in the regression calculation performed in this study. The comprehensive regression results show that logKSP correlates well with logpxB0L (). presents the slopes, intercepts, relationship coefficients (r) and significance levels (p) of all the 45 samples. The slopes and intercepts ranged from u22120.63 to u22120.15 (mean: u22120.37) and from u22122.57 to u22121.31 (mean: u22121.90), respectively. The results confirm that the gas-particle partitioning of neutral PFASs (e.g., FTOHs, FOSAs and FOSEs) followed the classical logKSP - logpxB0L relation for classic POPs.\" - Neutral PFASs in Arctic snow - \"Snow sampling was conducted on glaciers around Ny-xC5lesund during January and May 2012 (, ). The concentrations of neutral PFASs in the Arctic snow samples are summarized in and . The range of the total concentration was from 334 to 692u2005pg L with an average value of 523u2005pg L. To our knowledge, there are few reports on PFAS concentrations in snow, thus it is hard to compare these values with those from other regions. Similar to the situation in air, 8:2 FTOH is the dominant species in the Arctic snow, accounting for 45% of the total concentration. u03A3FTOHs in snow were between 218 and 507u2005pg L (mean: 369u2005pg L). The composition of the 12 PFASs in snow was different to that in air. Besides 8:2 FTOH; 10:2 FTOH, MeFOSE and 12:2 FTOH were the three most abundant species, representing 18%, 11% and 6.5% of the u03A3PFASs in snow, respectively (). The difference between the compositions in air and snow might be caused by different degradation processes for neutral PFASs in air and snow and different air-snow exchange potentials of individual chemicals. The average ratio of 8:2 to 10:2 to 6:2 to 12:2 FTOH was 56.2:22.5:1.0:8.1 in snow. However, to date, there has been few data such ratios in snow, thus it is hard to compare the variation between Arctic and urban areas. The ratios in snow are considerably higher than those in air and could be explained by the higher volatilization potential of 6:2 FTOH and by different degradation processes of FTOHs in air and snow.\" - \"Figure 4Neutral PFASs concentrations in Arctic snow.Graph indicating the concentration of the neutral PFASs in snow samples collected from Ny-xC5lesund, the Arctic.\" - Air-snow exchange fluxes of PFASs in the Arctic - \" shows the calculated net fluxes between air and snow for the 12 neutral PFASs from September 2011 to September 2012, whereby negative fluxes represent net deposition into snow and positive values are net volatilization fluxes into the atmosphere. It should be noted that the dominant trends of air-snow exchange were different among the various neutral PFASs species (). For FTOHs and FTAs, the total air-snow exchange fluxes were positive and the fluxes were negative for FOSEs. The highest positive and negative fluxes were observed for 8:2 FTOH (with a value of 118u2005pg mu2005d) and 6:2 FTOH (u22122.1u2005pg mu2005 d), respectively. However, 6:2 FTOH and FOSAs illustrated changing directions of air-snow exchange driven by varying concentration gradients between air and snow. Uncertainty of the parameters involved in the calculation of exchange fluxes would influence the flux values. To investigate the influence of the uncertainties in Cs, Ca and SSA on the exchange fluxes and direction, the uncertainties of Cs, Ca and SSA were assumed to be 100% and the exchange fluxes were calculated based on the Monte Carlo method using Software Oracle Crystal Ball 11.1. The cumulative frequency of Monte Carlo analysis (10 000 times) for the 12 PFAS compounds are presented in . The results indicate that the exchange direction did not change with 95% certainty for all neutral PFASs except for 6:2 FTOH and EtFOSA.\" - \"Figure 5Air-snow exchange fluxes.Estimated air-snow exchange fluxes (pg mu2005 d) for the 12 neutral PFASs. Negative fluxes represent net deposition into snow and positive values are net volatilization fluxes into the atmosphere.\" - \"The evident variability in air-snow exchange fluxes can be attributed to the varying KSA values of the different neutral PFAS species, which results in their potential of retaining in snow. KSA is a useful parameter expressing how much of the PFASs can be retained in snow under equilibrium conditions with air and is determined by a combination of snow and compound specific properties. As stated by Hansen et al., snowmelt and the increased temperature would lead to a decrease of KSA. Considering all collected snow samples in this study were fresh, the physical properties of snow can be considered as being constant, thus only the compound properties would influence the values of KSA. Daly and Wania found that volatilization from snow is not a significant for chemicals with logKOA values higher than 10 (or logKSA > 0.5) at 25xB0C. Accordingly, PFASs with a logKSA > 0.5 would is unlikely to be volatile in snow and as expected, FOSEs (logKSA > 0.5) have a negative exchange fluxes between air and snow ().\"- - Previous studies have confirmed that POPs in snow were mainly from the atmosphere and that snow has a high efficiency in scavenging vapor and particle-bound POPs out of the atmosphere due to their large surface area and the low temperatures promoting surface sorption. The positive FTOHs and FTAs fluxes observed in this study show the dynamic nature of their air-snow exchange, i.e., re-volatilization from the snow occurs rapidly following their wet deposition, which implies their strong volatilization potential. Compared with FTOHs, FTAs and FOSEs, the net air-snow exchange fluxes for FOSAs varied between deposition and volatilization (), which implies that FOSAs in air and snow are in dynamic equilibrium. - In theory, under conditions of increasing temperature and snowmelt, snow would lose capacity to hold sparingly water-soluble chemicals due to decreases in volume, Asnow and values of logKSA. As a result, chemicals accumulated in snow would have the potential for volatilization. Daly and Wania indicated that the air concentration of semi-volatile organic pollutants could be considerably influenced by snowmelt and a high concentration would be observed during or after snowmelt, especially in high latitude areas. A significant correlation in this study was observed between the ambient temperature and the atmospheric concentrations of FOSEs and FOSAs (r = 0.70, p < 0.01) (). However, the concentrations of FTOHs and FTAs did not show a significant correlation with the ambient temperature. The differences in behavior of the different chemicals are believed to be due to differences in their volatilization potential in snow as discussed above. Once FTOHs and FTAs have being scavenged into snow, they have strong volatilization potential to air. On the other hand, net volatilization of FOSEs and FOSAs out of snow occurs only under conditions of increasing temperature or snowmelt (and hence lowers KSA). As a result, air concentrations of FOSEs and FOSAs are significantly influenced by temperature. Hansen et al. found that the increase in atmospheric hexachlorocyclohexane isomers concentrations were more pronounced than those for fluorene, phenanthrene and PCB-28 toward the end of the winter in the Canadian High Arctic. In general, chemical fluxes between snow and air are highly dynamic. Those chemicals having higher logKSA values can be efficiently retained in snow, which could lead to an increase in their concentration in the atmosphere during or after snowmelt.- - Air and snow sampling - \"Air sampling was conducted on the German atmospheric observation platform located in Ny-xC5lesund (78xB055u2032N, 11xB056u2032E) (), Svalbard from September 27, 2011 to September 21, 2012 and a total of 45 air samples were collected. A glass fiber filter (GFF: diameter, 150u2005mm, pore size, 0.7u2005u03BC m) and a self-packed polyurethane foam (PUF)/XAD-2 cartridge (PUF: u03C65.0u2005 cm xD7 2.5u2005cm; 35u2005g XAD-2, particle size: 0.3u20131.0u2005mm) were simultaneously employed for collecting air samples (gas- and particle-phase) using a high-volume air sampler (). Each sample was collected at ~15u2005m h for 7 days to obtain a sample volume of ~2500u2005m. After sampling, cartridges and GFFs were separately sealed in air tight aluminum bags and stored at 4xB0 C and u221220xB0C prior to extraction, respectively. Field blanks consisted of GFFs and PUF/XAD-2 cartridges that were taken to the field site and handled in the same manner as real samples. Detailed information on the sampling dates, air volume, total suspended particulate and the average temperature for each sample are listed in .\" - \"Snow sampling was conducted around Ny-xC5lesund from January to May 2012 (). The sampling depth was 0u20135u2005cm and the samples were collected within 4u2005h after snowfall, thus the snow samples can be considered as fresh. Snow was collected using a pre-cleaned 40u2005L stainless steel barrel. After collection, the snow was allowed to melt within 24u2005h in the lab (at ~15xB0C). Around 5u20138u2005L melt water was extracted with a polymer resin cartridge (40u2005 g PAD-3 packed in a glass column) at a flow rate of ~300u2005mL min and was used for the determination of neutral PFAS. PAD-3 columns were stored at 4xB0C and the GFF filters were stored at u221220xB0C.\" - Sample extraction - \"Prior to extraction, internal standards mixture (IS, 2.5u2005ng) (C 6:2, 8:2 and 10:2 FTOH, MeFOSA D3, EtFOSA D5, MeFOSE D7 and EtFOSE D9) was spiked into a PUF/XAD-2 cartridge and GFFs, respectively. A modified Soxhlet apparatus (MX extractor) (), specially designed for the self-packed PUF/XAD-2 and PAD-3 column, was employed to extract PUF/XAD-2 and GFF for 16u2005h using dichloromethane (DCM). Extracts were evaporated to 5u2005mL with hexane as a keeper and 3u2005 g Na2SO4 was added to remove residual water. Samples were further reduced to 200u2005 u03BCL under a gentle stream of nitrogen and spiked with 1u2005ng of 9:1 FTOH as an injection standard.\" - Instrumental analysis - \"The 12 neutral PFASs were analyzed using an Agilent 6890 gas chromatograph - 5973 mass spectrometry that was equipped with a 60u2005m SUPELCO WAXxAE 10 column (60u2005m xD7 0.25u2005mm xD7 1.0u2005u03BCm). Measurements were made using a selective ion monitoring mode with positive chemical ionization (PCI). Methane was used as a reagent gas for PCI and helium (at a flow rate of 1.3u2005mL min) was used as carrier gas. The detailed instrumental parameters are presented in SI. The full names and other information of the 12 neutral PFASs determined in the present study are summarized in .\" - Quality Control - Detailed quality control processes and breakthrough tests for air samples were described elsewhere, Method detection limits (MDLs) for neutral PFASs in air and snow samples were derived from mean blank values plus three times the standard deviation (). All results were recovery corrected for neutral PFASs with corresponding internal standards. - Calculation method of air-snow exchange flux - \\'The air-snow exchange fluxes for the PFASs were calculated based on the modified Whitman two-film resistance model, the processes and methods are described in detail in the . Briefly, the air-snow exchange flux (Fgas, pg (s m)) was calculated as:\\' - \\'\\' - \\'where Cs (pg m) and Ca (pg m) are the concentrations of target compounds in snow and air, respectively, v is an exchange velocity (m s), Ksnow-air is the snow-air partition coefficient and can be expressed by the following equation:\\' - \\'\\' - \"where KSA is the snow interface-air partition coefficient, SSA is the specific surface area (m kg) and u03C1 (kg m) is the density of water.\" - \"As stated above, all snow samples collected in this study were fresh, thus, the snowpack can be considered to be homogeneous. and show that snow physical parameters are key factors influencing Fgas. In this study, snow physical parameters were used and/or calculated based on the empirical relationship developed by Legagneux et al.. The overall or summation solute hydrogen bond acidity (u03A3 u03B12) and the overall or summation solute hydrogen bond basicity (u03A3u03B2 2) were calculated according to Lyakurwa et al. and the solute gas-hexadecane partition coefficient (logL) was calculated using SPARC software.\"- - \"Houde, M., De Silva, A. O., Muir, D. C. & Letcher, R. J. Monitoring of perfluorinated compounds in aquatic biota: an updated review. Environ Sci Technol 45, 7962u2013 73 (2011)._ _ _ _ Kannan, K. Perfluoroalkyl and polyfluoroalkyl substances: current and future perspectives. Environ Chem 8, 333u20138 (2011)._ _ _ Rubarth, J., Dreyer, A., Guse, N., Einax, J. W. & Ebinghaus, R. Perfluorinated compounds in red-throated divers from the German Baltic Sea: new findings from their distribution in 10 different tissues. Environ Chem 8, 419u201328 (2011)._ _ _ Wang, Z., Cousins, I. T., Scheringer, M. & Hungerbuhler, K. Fluorinated alternatives to long-chain perfluoroalkyl carboxylic acids (PFCAs), perfluoroalkane sulfonic acids (PFSAs) and their potential precursors. Environ Int 60, 242u2013 8 (2013)._ _ _ Young, C. J. et al. Perfluorinated acids in Arctic snow: new evidence for atmospheric formation. Environ Sci Technol 41, 3455u201361 (2007)._ _ _ _ Ahrens, L., Shoeib, M., Vento, S. D., Codling, G. & Halsall, C. Polyfluoroalkyl compounds in the Canadian Arctic atmosphere. Environ Chem 8, 399u2013406 (2011)._ _ _ Wang, Z., MacLeod, M., Cousins, I. T., Scheringer, M. & Hungerbuhler, K. Using COSMOtherm to predict physicochemical properties of poly- and perfluorinated alkyl substances (PFASs). Environ Chem 8, 389u2013 98 (2011)._ _ _ Cousins, I. T., Kong, D. & Vestergren, R. Reconciling measurement and modelling studies of the sources and fate of perfluorinated carboxylates. Environ Chem 8, 339u201354 (2011)._ _ _ Reth, M. et al. Water-to-air transfer of perfluorinated carboxylates and sulfonates in a sea spray simulator. Environ Chem 8, 381u2013 8 (2011)._ _ _ Wang, Z. et al. Occurrence and gas/particle partitioning of PAHs in the atmosphere from the North Pacific to the Arctic Ocean. Atmos Environ 77, 640u20136 (2013)._ _ _ _ Martin, J. W., Ellis, D. A., Mabury, S. A., Hurley, M. D. & Wallington, T. J. Atmospheric chemistry of perfluoroalkanesulfonamides: kinetic and product studies of the OH radical and Cl atom initiated oxidation of N-ethyl perfluorobutanesulfonamide. Environ Sci Technol 40, 864u201372 (2006)._ _ _ _ Hurley, M. D. et al. Atmospheric Chemistry of Fluorinated Alcohols: Reaction with Cl Atoms and OH Radicals and Atmospheric Lifetimes. J Phys Chem A 108, 1973u20139 (2004)._ _ _ Ellis, D. A. et al. Degradation of fluorotelomer alcohols: a likely atmospheric source of perfluorinated carboxylic acids. Environ Sci Technol 38, 3316u201321 (2004)._ _ _ _ D\\'Eon, J. C., Hurley, M. D., Wallington, T. J. & Mabury, S. A. Atmospheric chemistry of N-methyl perfluorobutane sulfonamidoethanol, C4F9SO2N(CH3)CH2CH2OH: kinetics and mechanism of reaction with OH. Environ Sci Technol 40, 1862u20138 (2006)._ _ _ _ Piekarz, A. M., Primbs, T., Field, J. A., Barofsky, D. F. & Simonich, S. Semivolatile fluorinated organic compounds in Asian and western U.S. air masses. Environ Sci Technol 41, 8248u201355 (2007)._ _ _ _ Wania, F. A global mass balance analysis of the source of perfluorocarboxylic acids in the Arctic ocean. Environ Sci Technol 41, 4529u201335 (2007)._ _ _ _ Stemmler, I. & Lammel, G. Pathways of PFOA to the Arctic: variabilities and contributions of oceanic currents and atmospheric transport and chemistry sources. Atmos Chem Phys 10, 9965u2013 80 (2010)._ _ _ _ Daly, G. L. & Wania, F. Simulating the influence of snow on the fate of organic compounds. Environ Sci Technol 38, 4176u201386 (2004)._ _ _ _ Herbert, B. M., Halsall, C. J., Villa, S., Jones, K. C. & Kallenborn, R. Rapid changes in PCB and OC pesticide concentrations in arctic snow. Environ Sci Technol 39, 2998u20133005 (2005)._ _ _ _ Shoeib, M., Harner, T. & Vlahos, P. Perfluorinated chemicals in the arctic atmosphere. Environ Sci Technol 40, 7577u201383 (2006)._ _ _ _ Ahrens, L., Harner, T., Shoeib, M., Lane, D. A. & Murphy, J. G. Improved characterization of gas-particle partitioning for per- and polyfluoroalkyl substances in the atmosphere using annular diffusion denuder samplers. Environ Sci Technol 46, 7199u2013206 (2012)._ _ _ _ Dreyer, A. & Ebinghaus, R. Polyfluorinated compounds in ambient air from ship- and land-based measurements in northern Germany. Atmos Environ 43, 1527u2013 35 (2009)._ _ _ _ Xie, Z. et al. Neutral poly- and perfluoroalkyl substances in air and seawater of the North Sea. Environ Sci Pollut Res Int 20, 7988u20138000 (2013)._ _ _ Wang, Z. et al. Atmospheric concentrations and gas/particle partitioning of neutral poly- and perfluoroalkyl substances in northern German coast. Atmos Environ 95, 207u201313 (2014)._ _ _ _ Hansen, K. M., Halsall, C. J. & Christensen, J. H. A dynamic model to study the exchange of gas-phase persistent organic pollutants between air and a seasonal snowpack. Environ Sci Technol 40, 2644u201352 (2006)._ _ _ _ Legagneux, L., Cabanes, A. & DominxE9, F. Measurement of the specific surface area of 176 snow samples using methane adsorption at 77u2005K. J Geophys Res: Atmos 107, ACH 5-1u2013ACH 5-15 (2002).Lyakurwa, F. S., Yang, X., Li, X., Qiao, X. & Chen, J. Development of In Silico Models for Predicting LSER Molecular Parameters and for Acute Toxicity Prediction to Fathead Minnow (Pimephales Promelas). Chemosphere 108, 17u201325 (2014)._ _ _ _ \"- - \"The German Alfred Wegener Institute for Polar and Marine Research (AWI) and the French Polar Institute Paul Emile Victor (IPEV) are kindly acknowledged for allowing us carry out the ePOP project at the AWIPEV Research Base. We are very thankful to the AWIPEV base staff during 2011/2012, Rudolf Denkmann, Andrea GroxDF, Christophe Briere, Sebastien Barrault, Winfried Markert, Pierre LizxE9 e, Goulven Largouet (IPEV), for their support and assistance in air and snow sampling in Ny-xC5lesund. Roland Neuber (AWI) and Dominique Fleury (IPEV) are kindly acknowledged for their cooperation. We thank Siegrid Debatin and Marion Maturilli for providing meteorological data obtained by the Alfred Wegener Institute.\"- - Authors and Affiliations - \"Department for Environmental Chemistry, Helmholtz-Zentrum Geesthacht, Centre for Materials and Coastal Research, Institute of Coastal Research, Geesthacht, 21502, GermanyZhiyong Xie,_Zhen Wang,_Wenying Mi,_Axel MxF6ller,_Hendrik Wolschke_&_Ralf EbinghausNational Marine Environmental Monitoring Center, Dalian, 116023, ChinaZhen Wang\" - \"AuthorsZhiyong XieYou can also search for this author in _Zhen WangYou can also search for this author in _Wenying MiYou can also search for this author in _Axel MxF6llerYou can also search for this author in _Hendrik WolschkeYou can also search for this author in _Ralf EbinghausYou can also search for this author in _\" - Contributions - Z.X. and R.E. conceived the study; Z.X., A.M. and H.W. carried out the sampling; W.M. and Z.X. performed sample analysis; Z.W. and Z.X. conducted data management and wrote the manuscript with significant contributions from all co-authors.- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing financial interests. - \\' \\'- - Supplementary Information- - \\' \\' - \\'This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article\\'\\'s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder in order to reproduce the material. ',\n",
       " \"Creative Commons Attribution 4.0 International License (), which permits use, duplication, adaptation, distribution, and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. - ' ' - ''- - \",\n",
       " 'Creative Commons Attribution 4.0 International License (). - \\' \\'- - \\' \\' - The Python codes used in the DFN modelling and the diffusivity and thermodynamic factor analysis are available in the Zenodo database under a Creative Commons Attribution 4.0 International License (). - \\' \\'- - \"IEA. Global EV outlook 2022. (2022).Frith, J. T., Lacey, M. J. & Ulissi, U. A non-academic perspective on the future of lithium-based batteries. Nat. Commun. 14, 1u201317 (2023)._ _ Dhir, S., Wheeler, S., Capone, I. & Pasta, M. Outlook on K-ion batteries. Chem 6, 2442u20132460 (2020)._ _ _ Hosaka, T., Kubota, K., Hameed, A. S. & Komaba, S. Research development on K-ion batteries. Chem. Rev. 120, 6358u20136466 (2020)._ _ _ _ Min, X. et al. Potassium-ion batteries: outlook on present and future technologies. Energy Environ. Sci. 14, 2186 (2021)._ _ _ Hwang, J. Y., Myung, S. T. & Sun, Y. K. Sodium-ion batteries: present and future. Chem. Soc. Rev. 46, 3529u20133614 (2017)._ _ _ _ Tapia-Ruiz, N. et al. 2021 Roadmap for sodium-ion batteries. J. Phys. Energy 3, 031503 (2021)._ _ _ _ Xiao, A. W., Galatolo, G. & Pasta, M. The case for fluoride-ion batteries. Joule 5, 2823u20132844 (2021)._ _ _ Jian, Z., Luo, W. & Ji, X. Carbon electrodes for K-ion batteries. J. Am. Chem. Soc. 137, 11566u201311569 (2015)._ _ _ _ Komaba, S., Hasegawa, T., Dahbi, M. & Kubota, K. Potassium intercalation into graphite to realize high-voltage/high-power potassium-ion batteries and potassium-ion capacitors. Electrochem. Commun. 60, 172u2013175 (2015)._ _ _ Li, M., Feng, M., Luo, D. & Chen, Z. Fast charging Li-ion batteries for a new era of electric vehicles. Cell Rep. Phys. Sci. 1, 100212 (2020)._ _ _ Tomaszewska, A. et al. Lithium-ion battery fast charging: a review. eTransportation 1, 100011 (2019)._ _ Hosaka, T., Kubota, K., Kojima, H. & Komaba, S. Highly concentrated electrolyte solutions for 4 V class potassium-ion batteries. Chem. Commun. 54, 8387u2013 8390 (2018)._ _ _ Okoshi, M., Yamada, Y., Komaba, S., Yamada, A. & Nakai, H. Theoretical analysis of interactions between potassium ions and organic electrolyte solvents: a comparison with lithium, sodium, and magnesium ions. J. Electrochem. Soc. 164, A54u2013A60 (2017)._ _ _ Matsuda, Y. Behavior of some ions in mixed organic electrolytes of high energy density batteries. J. Electrochem. Soc. 128, 2552 (1981)._ _ _ _ Landesfeind, J. et al. Comparison of ionic transport properties of non-aqueous lithium and sodium hexafluorophosphate electrolytes. J. Electrochem. Soc. 168, 040538 (2021)._ _ _ _ Abe, T., Fukuda, H., Iriyama, Y. & Ogumi, Z. Solvated Li-ion transfer at interface between graphite and electrolyte. J. Electrochem. Soc. 151, A1120 (2004)._ _ _ Wang, A. A. et al. Review of parameterisation and a novel database (LiionDB) for continuum Li-ion battery models. Prog. Energy 4, 032004 (2022)._ _ _ Hou, T. & Monroe, C. W. Composition-dependent thermodynamic and mass-transport characterization of lithium hexafluorophosphate in propylene carbonate. Electrochim. Acta 332, 135085 (2020)._ _ _ Bergstrom, H. K., Fong, K. D. & McCloskey, B. D. Interfacial effects on transport coefficient measurements in Li-ion battery electrolytes. J. Electrochem. Soc. 168, 060543 (2021)._ _ _ _ Landesfeind, J. & Gasteiger, H. A. Temperature and concentration dependence of the ionic transport properties of lithium-ion battery electrolytes. J. Electrochem. Soc. 166, A3079u2013A3097 (2019)._ _ Nyman, A., Behm, M. & Lindbergh, G. Electrochemical characterisation and modelling of the mass transport phenomena in LiPF6-EC-EMC electrolyte. Electrochim. Acta 53, 6356u20136365 (2008)._ _ _ Valoen, L. O. & Reimers, J. N. Transport properties of LiPF6-based Li-ion battery electrolytes. J. Electrochem. Soc. 152, A882 (2005)._ _ _ Lundgren, H., Behm, M. & Lindbergh, G. Electrochemical characterization and temperature dependency of mass-transport properties of LiPF6 in EC:DEC. J. Electrochem. Soc. 162, A413u2013A420 (2015)._ _ _ Lundgren, H., Scheers, J., Behm, M. & Lindbergh, G. Characterization of the mass-transport phenomena in a superconcentrated LiTFSI:acetonitrile electrolyte. J. Electrochem. Soc. 162, A1334u2013A1340 (2015)._ _ _ SteinrxFCck, H.-G. et al. Concentration and velocity profiles in a polymeric lithium-ion battery electrolyte. Energy Environ. Sci. 13, 4312u2013 4321 (2020)._ _ Fawdon, J., Ihli, J., Mantia, F. L. & Pasta, M. Characterising lithium-ion electrolytes via operando Raman microspectroscopy. Nat. Commun. 12, 1u20139 (2021)._ _ Wang, A. A. et al. Potentiometric MRI of a superconcentrated lithium electrolyte: testing the irreversible thermodynamics approach. ACS Energy Lett. 6, 3086u20133095 (2021)._ _ _ _ _ Bazak, J. D., Allen, J. P., Krachkovskiy, S. A. & Goward, G. R. Mapping of lithium-ion battery electrolyte transport properties and limiting currents with in situ MRI. J. Electrochem. Soc. 167, 140518 (2020)._ _ _ _ Hosaka, T., Muratsubaki, S., Kubota, K., Onuma, H. & Komaba, S. Potassium metal as reliable reference electrodes of nonaqueous potassium cells. J. Phys. Chem. Lett. 10, 3296u20133300 (2019)._ _ _ _ Xiao, N., McCulloch, W. D. & Wu, Y. Reversible dendrite-free potassium plating and stripping electrochemistry for potassium secondary batteries. J. Am. Chem. Soc. 139, 9475u2013 9478 (2017)._ _ _ _ Shkrob, I. A., Marin, T. W., Zhu, Y. & Abraham, D. P. Why bis(fluorosulfonyl)imide is a u201Cmagic anionu201D for electrochemistry. J. Phys. Chem. C 118, 19661u201319671 (2014)._ _ _ Wang, H. et al. Liquid electrolyte: the nexus of practical lithium metal batteries. Joule 6, 588u2013616 (2022)._ _ _ Le Pham, P. N. et al. Potassium-ion batteries using KFSI in monoglyme electrolytes: implications of cation solvation on the K-graphite (Co-)intercalation mechanism. Energy Storage Mater. 45, 291u2013300 (2022)._ _ Aspinall, J., Armstrong, D. E. J. & Pasta, M. EBSD-coupled indentation: nanoscale mechanics of lithium metal. Mater. Today Energy 30, 101183 (2022)._ _ _ Hosaka, T. et al. 1,3,2-Dioxathiolane 2,2-dioxide as an electrolyte additive for K-metal cells. ACS Energy Lett. 4, 3643u20133649 (2021)._ _ Popovic, J. Reviewu2013Recent advances in understanding potassium metal anodes. J. Electrochem. Soc. 169, 030510 (2022)._ _ _ _ Caracciolo, L., Madec, L. & Martinez, H. XPS analysis of K-based reference compounds to allow reliable studies of solid electrolyte interphase in K-ion batteries. ACS Appl. Energy Mater. 4, 11693u201311699 (2021)._ _ _ Allgayer, F., Maibach, J. & Jeschull, F. Comparing the solid electrolyte interphases on graphite electrodes in K and Li half cells. ACS Appl. Energy Mater. 5, 1136u2013 1148 (2022)._ _ _ Diederichsen, K. M., McShane, E. J. & McCloskey, B. D. Promising routes to a high Li transference number electrolyte for lithium ion batteries. ACS Energy Lett. 2, 2563u20132575 (2017)._ _ _ Bruce, P. G. & Vincent, C. A. Steady state current flow in solid binary electrolyte cells. J. Electroanal. Chem. 225, 1u201317 (1987)._ _ _ Wang, A. A., Hou, T., Karanjavala, M. & Monroe, C. W. Shifting-reference concentration cells to refine composition-dependent transport characterization of binary lithium-ion electrolytes. Electrochim. Acta 358, 136688 (2020)._ _ _ Wang, A. A. et al. Current-driven solvent segregation in lithium-ion electrolytes. Cell Rep. Phys. Sci. 3, 101047 (2022)._ _ _ Zhang, P., Jin, H., Wang, T. & Wang, M. Insight into the effect of lithium-dendrite suppression by lithium bis(fluorosulfony)imide/1,2-dimethoxyethane electrolytes. Electrochim. Acta 277, 116u2013126 (2018)._ _ _ Atkins, P. W., De Paula, J. & Keeler, J. Atkins Physical Chemistry (Oxford University Press, 2017).Raccichini, R., Dibden, J. W., Brew, A., Owen, J. R. & GarcxED a-ArxE1ez, N. Ion speciation and transport properties of LiTFSI in 1,3-dioxolane solutions: a case study for Li-S battery applications. J. Phys. Chem. B 122, 267u2013274 (2018)._ _ _ _ Zhang, Q. et al. Boosting the potassium storage performance of alloy-based anode materials via electrolyte salt chemistry. Adv. Energy Mater. 8, 1703288 (2018)._ _ Wang, L. et al. TiS2 as a high performance potassium ion battery cathode in ether-based electrolyte. Energy Storage Mater. 12, 216u2013 222 (2018)._ _ _ Yim, C.-H. & Abu-Lebdeh, Y. A. Connection between phase diagram, structure and ion transport in liquid, aqueous electrolyte solutions of lithium chloride. J. Electrochem. Soc. 165, A547u2013 A556 (2018)._ _ _ Stewart, S. & Newman, J. Measuring the salt activity coefficient in lithium-battery electrolytes. J. Electrochem. Soc. 155, A458 (2008)._ _ _ Newman, J. & Thomas-Alyea, K. E. Electrochemical Systems 3rd edn (Wiley, 2004).Pitzer, K. S. Activity Coefficients in Electrolyte Solutions 2nd edn (CRC Press, 2018).Zhang, X. et al. Comprehensive insights into electrolytes and solid electrolyte interfaces in potassium-ion batteries. Energy Storage Mater. 38, 30u201349 (2021)._ _ _ Harned, H. S. & French, D. M. A conductance method for the determination of the diffusion coefficients of electrolytes. Ann. NY Acad. Sci. 46, 267u2013 284 (1945)._ _ _ _ _ Newman, J. & Chapman, T. W. Restricted diffusion in binary solutions. AIChE J. 19, 343u2013348 (1973)._ _ _ Ma, Y. et al. The measurement of a complete set of transport properties for a concentrated solid polymer electrolyte solution. J. Electrochem. Soc. 142, 1859u20131868 (1995)._ _ _ _ Ehrl, A., Landesfeind, J., Wall, W. A. & Gasteiger, H. A. Determination of transport parameters in liquid binary lithium ion battery electrolytes. J. Electrochem. Soc. 164, A826u2013A836 (2017)._ _ _ Doyle, M., Fuller, T. & Newman, J. Modeling of galvanostatic charge and discharge of the lithium/polymer/insertion cell. J. Electrochem. Soc. 140, 1526u2013 1533 (1993)._ _ _ _ Sulzer, V., Marquis, S. G., Timms, R., Robinson, M. & Chapman, S. J. Python battery mathematical modelling (PyBaMM). J. Open Res. Softw. 9, 1u20138 (2021)._ _ Asenbauer, J. et al. The success story of graphite as a lithium-ion anode material - fundamentals, remaining challenges, and recent developments including silicon (oxide) composites. Sustain. Energy Fuels 4, 5387u20135416 (2020)._ _ _ Xu, W., Welty, C., Peterson, M. R., Read, J. A. & Stadie, N. P. Exploring the limits of the rapid-charging performance of graphite as the anode in lithium-ion batteries. J. Electrochem. Soc. 169, 010531 (2022)._ _ _ _ Neu010Das, D. & Klapetek, P. Gwyddion: an open-source software for SPM data analysis. Open Phys. 10, 181u2013188 (2012)._ _ _ Fairley, N. et al. Systematic and collaborative approach to problem solving using X-ray photoelectron spectroscopy. Appl. Surf. Sci. Adv. 5, 100112 (2021)._ _ Liu, J. & Monroe, C. W. On the characterization of battery electrolytes with polarization cells. Electrochim. Acta 167, 357u2013363 (2015)._ _ _ Casteel, J. F. & Amis, E. S. Specific conductance of concentrated solutions of magnesium salts in water-ethanol system. J. Chem. Eng. Data 17, 55u2013 59 (1972)._ _ _ Thompson, S. D. & Newman, J. Differential diffusion coefficients of sodium polysulfide melts. J. Electrochem. Soc. 136, 3362u20133369 (1989)._ _ _ _ Andersson, J. A., Gillis, J., Horn, G., Rawlings, J. B. & Diehl, M. CasADi: a software framework for nonlinear optimization and optimal control. Math. Program. Comput. 11, 1u201336 (2019)._ _ _ _ Ai, W., Kraft, L., Sturm, J., Jossen, A. & Wu, B. Electrochemical thermal-mechanical modelling of stress inhomogeneity in lithium-ion pouch cells. J. Electrochem. Soc. 167, 013512 (2020)._ _ _ _ Xu, S., Chen, K.-H., Dasgupta, N. P., Siegel, J. B. & Stefanopoulou, A. G. Evolution of dead lithium growth in lithium metal batteries: experimentally validated model of the apparent capacity loss. J. Electrochem. Soc. 166, A3456u2013A3463 (2019)._ _ _ Li, L. et al. Understanding high-rate K-solvent co-intercalation in natural graphite for potassium-ion batteries. Angew. Chem. Int. Ed. 59, 12917u2013 12924 (2020)._ _ _ Liu, J. et al. Unraveling the potassium storage mechanism in graphite foam. Adv. Energy Mater. 9, 1900579 (2019)._ _ Fiore, M. et al. Paving the way toward highly efficient, high-energy potassium-ion batteries with ionic liquid electrolytes. Chem. Mater. 32, 7661 (2020)._ _ Dhir, S., Jagger, B., Maguire, A. & Pasta, M. Supporting data for u201CFundamental investigations on the ionic transport and thermodynamic properties of non-aqueous potassium-ion electrolytesu201D. zenodo (2023).\"- - The authors would like to acknowledge the financial support of the Henry Royce Institute (through UK Engineering and Physical Sciences Research Council grant EP/R010145/1) for capital equipment. S.D. appreciates the financial support from EPSRC and Shell. B.J. is grateful for the support of the Clarendon Fund Scholarships. A.M. acknowledges the support of The Faraday Institution (via grant number FITG-FUSE-102). We are grateful to Christopher Doerrer for his help with the CAD for the final diffusion cell, Isaac Capone for his earlier teaching in preparing K, Giulia Galatolo and Soochan Kim for their help with the schematic design, Marco Siniscalchi for his earlier electrode preparation, Robert Timms for answering questions on PyBaMM, Maximilian Schart for his useful thoughts and ideas, Jack Fawdon for his earlier mentoring on transport, and Peter Klusener for his useful feedback.- - Author notes - \\'These authors contributed equally: Shobhan Dhir, Ben Jagger.\\' - Authors and Affiliations - \"Department of Materials, University of Oxford, Oxford, OX1 3PH, UKShobhan Dhir,_ Ben Jagger,_Alen Maguire_&_Mauro Pasta\" - \"AuthorsShobhan DhirYou can also search for this author in _Ben JaggerYou can also search for this author in _Alen MaguireYou can also search for this author in _Mauro PastaYou can also search for this author in _\" - Contributions - S.D. and M.P. conceptualised the study and experiments; S.D. developed the electrolyte characterisation cells and K preparation protocol; S.D., B.J. and M.P. developed the potassium electrolyte characterisation methodology; S.D., B.J. and A.M. conducted the electrolyte characterisation experiments; B.J. conducted the XPS and AFM experiments; S.D. and B.J. conducted the analysis; S.D. conducted the modelling; S.D. wrote the original draft; S.D., B.J. and M.P. wrote, edited and revised the manuscript; M.P. supervised the study and provided frequent input in interpretation of results. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - M.P. is a scientific advisor to Project K Energy. The remaining authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License (). We evaluated the association between temperature and humidity using downloadable raster datasets from the TerraClimate database (), which have been recently described.\" - We averaged monthly temperatures and VPDs to calculate annual temperature, annual VPD, and standard deviation of annual VPD for each location over a 30-year time period. We then developed custom code in Python (v. 3.6) to analyze the correlation between these environmental variables using linear regression analyses from the statsmodels library. We sampled the spatial layers using the gdal functions in osgeo library to convert the spatial data into a large dataset (~60 gigabytes) for downstream analysis. Due to the large size of the dataset, we used the chunksize parameter in the pandas library to import portions of the global dataset for analysis. We then used the multiprocessing library to evaluate correlations for each site (meaning, a single pixel of roughly 4 km) across 30 independent central processing units. These individual datasets were then compiled together and sorted into global maps using a custom function in Python. These maps were then visualized in QGIS (v. 2.1). All script is available at . - Extraction and sequencing of RNA - \"We extracted total RNA from 12 individuals after the initial physiological measurements following the one-month acclimation period, but prior to any exposure to the treatments. These individuals are referred to as baseline individuals or samples. We then extracted total RNA from salamander skin tissue immediately following the final measurement of ri. Individual salamanders were removed from the flow-through system and immediately immersed in liquid nitrogen for 15u201320u2009min. After freezing, we used a flame-sterilized razor blade to shave off a thin layer of skin from the dorsal side of the salamander. We also extracted hearts from the frozen specimen using the flame-sterilized razor blade. The hearts were easily visible through the ventral side of the salamander because the ventral skin is translucent. Though heart samples were included in the de novo transcriptome, we did not evaluate gene expression differences because the analysis is beyond the scope of this study. The tissue samples were immediately immersed in TRIzol reagent (Item#: 15596026; Thermo Fisher Scientifics, 168 Third Avenue, Waltham, MA 02451) and blended using a motorized tissue homogenizer with sterilized pestle (Item#: UX-44468-25; Argos Technologies Inc., 625 E. Bunker Ct., Vernon Hills, IL 60061). Once fully homogenized, the tissue samples were stored at 80u2009 xB0C until downstream preparation in accordance with standard protocols.\" - \"We extracted total RNA using standard protocols for TRIzol reagent. Prior to library preparation, we purified each sample to remove contaminants (e.g., phenol, ethanol, etc.) using the RNeasy Mini Kit for total RNA (ID: 74136; QiagenxAE , 1001 Marshall St., Redwood City, CA 94063). In addition, we treated samples with DNase to limit the possibility of DNA contamination in tandem with the RNeasy Mini Kit. Prior to library preparation, we determined the concentration of total RNA using a Qubit Fluorometer (Thermo Fisher Scientific) to ensure each sample contained sufficient material for library preparation. We prepared each sample using the Illumina TruSeqxAE mRNA Stranded Kit (Product#: RS-122-2101, Illumina, Inc., 5200 Illumina Way, San Diego, CA 92122) by following standard protocols. Once again, we used the Qubit Fluorometer to quantify the amount of cDNA prior to sequencing.\" - \"We sequenced our samples using the Illumina HiSeq2500 platform at the Genomics Facility at Cornell University. We sequenced singe-end 100-bp reads for all libraries. We pooled 12 barcoded samples per lane to balance sequencing depth for each sample with the total number of samples (Nu2009=u200948 total samples across four lanes). We randomly assigned an individual sample to each sequencing lane with respect to treatment to avoid lane biases. In preparation for de novo transcriptome assembly, we trimmed Illumina-specific sequences and adapters using Trimmomatic (v. 0.36). We evaluated error probabilities of each read with ConDeTri (v. 2.3) and low-quality bases or sections of multiple bases were removed using default parameters. After trimming, reads were assessed using FastQC (version 0.11.5) to confirm that extraneous reads had been removed and quality scores were greater than 30 along the entire sequence length.\" - We assembled our de novo transcriptome using 827 million reads from 48 tissue samples (36 skin and 12 heart samples) using Trinity (v. 2.4.0) and standard protocols for strand specific analysis and normalization. We used TransDecoder (version 3.0.1) to reduce the transcriptome to coding regions from all transcript sequences. TransDecoder identified likely coding sequences as those with a minimum open reading frame length of 50 amino acids. We then assigned gene ontology terms, enzyme codes, and KEGG pathways to genes by blasting against all databases using the Blast2GO Cloud Blast Function. We blasted against all known organisms to ensure the highest chance at annotating each gene. The number of blast hits was increased from the default parameter to 20, and the mapping and annotation steps were run using default protocols. RSEM (version 1.3.0) estimated gene and gene isoform expression levels by aligning individual samples back to the transcriptome using Bowtie (version 1.2.1.1). To prepare for differential gene expression analysis, low-expressed genes were filtered out, retaining only those for which approximately one-third of genes had 10 or more counts. - Statistical analyses - \"We analyzed the physiological data using R (v.3.4.2) using linear regression analyses. We evaluated the change ri over the course of the experiment with mass and initial ri as covariates. Temperature and humidity were treated as factors. Change in ri was calculated by taking the difference of final ri and initial ri for each individual. We also evaluated interactions between covariates and factors to assess complex responses to temperature and humidity. Covariates were scaled and centered over zero, and we assessed collinearity between covariates using the vif function from the car package in R to examine variance inflation factors (VIF). We excluded variables and combinations of variables that exceeded the VIF threshold of 3 (ref. ). We then conducted a Type-II analysis of covariance using the car package to assess significance of covariates and factors. Finally, we report effect size using omega-squared (u03C9) for each significant effect using the sjstats package in R.\" - \"We analyzed the environmental data using non-linear, mixed effects models from the nlmer() function in the lme4 package in R with VPD as_the response variable and temperature as a covariate. We also examined the relationship between temperature and the variation in nightly VPD because physical expectations suggest VPDs should become more variable with warming air temperature. For our random effects, we nested an iButton within the year the recording was taken. The nested random effects accounted for abiotic differences between locations and years due to local topographic effects and interannual seasonality. We generated starting values for the non-linear simulations using the nlsLM() function in the minpack package in R. We assessed model performance by evaluating the confidence intervals of the parameter estimates and 95% confidence intervals of the non-linear models.\" - Differential gene expression - \"We conducted a differential gene expression (DGE) analysis using DESeq2 library to identify differentially expressed genes between treatments. We tested for DGE by comparing the gene expression between samples collected at the end of the experiment using the DESeqDataSetFromMatrix function in DESeq2. We explored the number of differentially expressed genes for each treatment using two significance thresholds (u03B1u2009=u20090.1 and 0.001) on gene-specific p-values adjusted for false discovery rate using the default Benjaminiu2013Hochberg procedure. Varying the significance threshold provided insight into broad differences in gene ontology under the most liberal threshold, whereas the more stringent threshold was used to identify candidate genes involved in plasticity.\" - Gene ontology enrichment - \"We conducted a gene ontology (GO) enrichment analysis on differentially expressed genes using the GOseq package in R (v. 3.4.2) to identify overrepresentation of certain GO terms. The GOseq package determines enrichment of specific GO terms by incorporating the probability a gene will be differentially expressed based upon length of the gene. The GOseq procedure uses a_random selection process to evaluate the GO category membership and enrichment by randomly sampling the given dataset to generate a suitable null distribution. Upon identifying the distribution, GOseq evaluates each GO term for under- and overrepresentation. We conducted GO term enrichment on the DGE analysis and the weighted gene co-expression analysis. For enrichment analyses, we removed GO terms with less than 10 and more than 500 genes within a category to reduce enrichment sensitivity and redundancy.\" - Weighted gene co-expression analysis - We identified suites of correlated gene networks using weighted gene co-expression network analysis (WGCNA) to improve our understanding of the gene networks and genes underlying acclimation of ri. These analyses were conducted independently of the DGE analysis. We used WGCNA to identify networks of genes (or modules) associated with plasticity of ri. We refer to this analysis as the skin resistance analysis. We also conducted an additional analysis to identify potential mechanisms underlying the relationship between mass and temperature within modules identified in the skin resistance analysis, referred to as the interaction analysis. We designed these analyses to identify the possible genetic changes underlying specific physiological responses from the experiments. - Skin resistance analysis - \"We began the skin resistance analysis using all of the samples from the experimental treatments (nu2009=u200932). We used the post trimmed and cleaned samples generated from RSEM (from above) to begin the analysis. We then removed any genes that had <10 counts across 28 samples, which resulted in a total of 9346 genes for downstream analyses. Gene counts were then normalized using the varianceStabilizingTransformation function in DESeq2. We ensured that the dataset did not have any missing data or zero-variance genes, and we removed one sample identified as an outlier based upon cluster analysis prior to constructing co-expression networks.\" - \"Gene co-expression networks were constructed by building pairwise Pearson correlations between each pair of genes using the blockwiseModules function in WGCNA. The resulting co-expression similarity values were transformed into an adjacency matrix using the soft thresholding approach, which favors stronger correlations over weaker correlations between genes. We selected a soft threshold of 4 to simultaneously maximize the model fit of the scale-free topology (Ru2009>u2009 0.80) and mean connectivity among genes (Supplementary Fig. ). Then, we defined a dendrogram of the gene networks using average linkage hierarchical clustering coupled with a gene dissimilarity matrix. Finally, we used the Dynamic Tree Cut approach to merge highly correlated modules using a height-cut of 0.25 (ref. ). These modules could then be used to understand important networks and genes underlying acclimation of ri.\" - \"Module construction can be leveraged to identify biologically meaningful genes or groups of genes underlying variation in a phenotype. We summarized expression for each module using a principle component analysis (PCA) to calculate eigengenes using the blockwiseModules function, which summarized the expression for each sample using the first principle component (PC1) for each module. We then used a Pearson correlation to test for associations between eigengene values and the change in ri for each module using the cor function in WGCNA. The resulting p-values were estimated using the corPalueStudent function in WGCNA. Modules significantly associated with the change in ri, referred to as skin resistance modules, also identified potential genes based upon intramodular connectivity. Hub genes are defined as having high intramodular connectivity (or module membership (MM)), and previous research has revealed important biological functions by assessing hub genes. We explored the function of hub genes in subsequent analyses based upon their connectivity score, statistical significance (u03B1u2009 <u20090.05) with a particular phenotype, and their association with enriched GO terms. These methods helped to discover potential genes underlying plasticity of water loss physiology.\" - Temperature and humidity might also represent important cues that explain variation within gene co-expression networks. We evaluated the importance of our treatments in affecting gene expression in skin resistance modules using regression analyses. We used the eigengene values from PC1 as our dependent variable, the change in ri as a covariate, and temperature and humidity as fixed effects. The model also included interactions with the change in ri and treatment to determine whether interactions played in important role in affecting gene expression within skin resistance modules. We conducted separate analyses for modules that were negatively and positively associated with ri. Then we used the car package to conduct a Type-II ANCOVA to determine whether the treatments influenced gene expression within skin resistance modules. - We identified potential genes underlying plasticity of ri using two methods. First, we evaluated GO term enrichment based upon the genes within modules significantly associated with ri. We reported enrichment for biological processes, cell components, and molecular function for any terms related to the regulation of water loss. These GO terms included lipid metabolism, skin barrier formation, vasoconstriction, angiogenesis, and lipid excretion. We then cross-referenced these GO terms with hub genes significantly associated with ri (as denoted by gene significance and intramodular connectivity) to identify potential genes underlying acclimation of ri. We also identified hub genes (from the WGCNA analysis) that significantly correlated with the change in ri that were also associated with GO terms related to water loss. - Interaction analysis - We conducted an additional analysis to explore the interaction between mass and temperature on acclimation of ri. For the interaction analysis, we analyzed the interaction between mass and temperature for all genes within the skin resistance modules. We used linear regression analyses with the transformed gene expression value as the dependent variable, mass as a covariate, and temperature as a factor. We also included the interaction between mass and temperature. Then we conducted Type-II ANCOVA using the car package to evaluate significance of the interaction for each gene by looping across each gene in R. Genes from the skin resistance modules that exhibited a significant interaction between temperature and mass are referred to as interaction genes. We assessed the interaction genes for their potential role in water loss regulation by evaluating GO terms associated with each known gene, similar to our previous procedure for identifying candidate genes. - Gene set enrichment analysis - We performed a Gene Set Enrichment Analysis (GSEA v.2.1.0) to identify predefined gene sets that showed significant, concordant differences in expression between temperature and humidity treatments. GSEA identifies gene sets that show functionally related, but potentially smaller, changes in expression. A custom GSEA database, with each set containing at least 15 genes, was created from GO terms and enzyme code annotations of the assembled transcriptome. We identified enriched or depleted gene sets under specific temperature and humidity conditions using a false discovery rate (FDR) of 0.1. We also determined gene overlap between WGNA, DGE, and GSEA analyses using the merge() function in R. - Reporting summary - Further information on research design is available in the linked to this article.- - \\' \\' - \\'We provided the data required to interpret, replicate, and build upon our findings. We provided the raw data on the Open Science Framework repository (). The data are referenced using the following identifier: . Figure requires the available data from . RNA-seq data is currently available for download on Genbank (BioProject: ).\\' - \\' \\'- - \\' \\' - The Open Science Framework repository () contains the annotated script and analysis for Fig. written in Python (v. 3.6). - \\' \\'- - \"West-Eberhard, M. J. Developmental Plasticity and Evolution (Oxford University Press Inc., 2003).Bedade, P., Mateus, A. R. & Keller, R. A. Evolution and molecular mechanisms of adaptive developmental plasticity. Mol. Ecol. 20, 1347u20131363 (2011)._ _ Dall, S. R. X., McNamara, J. M. & Leimar, O. Genes as cues: phenotypic integration of genetic and epigenetic information from a Darwinian perspective. Trends Ecol. Evol. 30, 327u2013333 (2015)._ _ _ Stein, L. R., Bukhari, S. A. & Bell, A. M. Personal and transgenerational cues are nonadditive at the phenotypic and molecular level. Nat. Ecol. Evol. 2, 1306u20131311 (2018)._ _ _ _ Via, S. et al. Adaptive phenotypic plasticity: consensus and controversy. Trends Ecol. Evol. 10, 212u2013217 (1995)._ _ _ _ Charmantier, A. et al. Adaptive phenotypic plasticity in response to climate change in a wild bird population. Science 320, 800u2013803 (2008)._ _ _ _ _ Ghalambor, C. K., McKay, J. K., Carroll, S. P. & Reznick, D. N. Adaptive versus non-adaptive phenotypic plasticity and the potential for contemporary adaptation in new environments. Funct. Ecol. 21, 394u2013407 (2007)._ _ Ghalambor, C. K. et al. Non-adaptive plasticity potentiates rapid adaptive evolution of gene expression in nature. Nature 525, 372u2013375 (2015)._ _ _ _ _ Pigliucci, M. Evolution of phenotypic plasticity: where are we going now? Trends Ecol. Evol. 20, 481u2013486 (2005)._ _ _ Bonte, D. et al. Costs of dispersal. Biol. Rev. 87, 290u2013312 (2012)._ _ _ Seebacher, F., White, C. R. & Franklin, C. E. Physiological plasticity increases resilience of ectothermic animals to climate change. Nat. Clim. Change 5, 61u201366 (2015)._ _ _ Somero, G. N. The physiology of climate change: how potentials for acclimatization and genetic adaptation will determine u2018 winners and u2018losers. J. Exp. Biol. 213, 912u2013920 (2010)._ _ _ _ Riddell, E. A., Odom, J. P., Damm, J. D. & Sears, M. W. Plasticity reveals hidden resistance to extinction under climate change in the global hotspot of salamander diversity. Sci. Adv. 4, eaar5471 (2018)._ _ _ _ _ Albright, T. P. et al. Mapping evaporative water loss in desert passerines reveals an expanding threat of lethal dehydration. Proc. Natl Acad. Sci. USA 114, 2283u20132288 (2017)._ _ _ _ Park Williams, A. et al. Temperature as a potent driver of regional forest drought stress and tree mortality. Nat. Clim. Change 3, 292u2013297 (2012)._ _ _ Jackson, S. T., Betancourt, J. L., Booth, R. K. & Gray, S. T. Ecology and the ratchet of events: climate variability, niche dimensions, and species distributions. Proc. Natl Acad. Sci. USA 106 (Suppl 2), 19685u201319692 (2009)._ _ _ _ _ Toledo-Ortiz, G. et al. The HY5-PIF regulatory module coordinates light and temperature control of photosynthetic gene transcription. PLOS Genet. 10, e1004416 (2014)._ _ _ _ Leighly, J. A note on evaporation. Ecology 18, 180u2013198 (1937)._ Stull, R. Meteorology for Scientists and Engineers (Brooks/Cole, 2000).Skelton, R. P., West, A. G. & Dawson, T. E. Predicting plant vulnerability to drought in biodiverse regions using functional traits. Proc. Natl Acad. Sci. USA 112, 5744u20135749 (2015)._ _ _ _ _ McMenamin, S. K., Hadly, E. A. & Wright, C. K. Climatic change and wetland desiccation cause amphibian decline in Yellowstone National Park. Proc. Natl Acad. Sci. USA 105, 16988u201316993 (2008)._ _ _ _ _ Dudley, S. A. Differing selection on plant physiological traits in response to environmental water availability: a test of adaptive hypotheses. Evolution 50, 92u2013102 (1996)._ _ _ Young, J. E., Christian, K. A., Donnellan, S., Tracy, C. R. & Parry, D. Comparative analysis of cutaneous evaporative water loss in frogs demonstrates correlation with ecological habits. Physiol. Bioch. Zool. 78, 847u2013856 (2005)._ _ Tieleman, B. I., Williams, J. B. & Bloomer, P. Adaptation of metabolism and evaporative water loss along an aridity gradient. Philos. T. Roy. Soc. B 270, 207u2013214 (2003)._ _ Braz Titon, J. & Gomes, F. R. Relation between water balance and climatic variables associated with the geographical distribution of anurans. PLoS ONE 10, e0140761u201319 (2015)._ _ _ _ Storz, J. F., Scott, G. R. & Cheviron, Z. A. Phenotypic plasticity and genetic adaptation to high-altitude hypoxia in vertebrates. J. Exp. Biol. 213, 4125u20134136 (2010)._ _ _ _ Cheviron, Z. A., Connaty, A. D., McClelland, G. B. & Storz, J. F. Functional genomics of adaptation to hypoxic cold-stress in high-altitude deer mice: transcriptomic plasticity and thermogenic performance. Evolution 68, 48u201362 (2014)._ _ _ Campbell-Staton, S. C. et al. Winter storms drive rapid phenotypic, regulatory, and genomic shifts in the green anole lizard. Science 357, 495u2013498 (2017)._ _ _ _ _ Corl, A. et al. The genetic basis of adaptation following plastic changes in coloration in a novel environment. Curr. Biol. (2018). _ _ _ Riddell, E. A., Apanovitch, E. K., Odom, J. P. & Sears, M. W. Physical calculations of resistance to water loss improve predictions of species range models. Ecol. Mono 87, 21u201333 (2017)._ _ Riddell, E. A., McPhail, J., Damm, J. D. & Sears, M. W. Trade-offs between water loss and gas exchange influence habitat suitability of a woodland salamander. Funct. Ecol. (2018). _ _ Lillywhite, H. B. Plasticity of the water barrier in vertebrate integument. Int. Congr. Ser. 1275, 283u2013290 (2004)._ _ Burggren, W. W. & Vitalis, T. Z. The interplay of cutaneous water loss, gas exchange and blood flow in the toad, Bufo woodhousei: adaptations in a terrestrially adapted amphibian. J. Exp. Biol. 208, 105u2013112 (2005)._ _ _ Abatzoglou, J. T., Dobrowski, S. Z., Parks, S. A. & Hegewisch, K. C. TerraClimate, a high-resolution global dataset of monthly climate and climatic water balance from 1958u20132015. Sci. Data 5, 170191 (2018)._ _ _ _ Peng, J. C. et al. Jarid2/Jumonji coordinates control of PRC2 enzymatic activity and target gene occupancy in pluripotent cells. Cell 139, 1290u20131302 (2009)._ _ _ _ Boeckel, J.-N. et al. Jumonji domain-containing protein 6 (Jmjd6) is required for angiogenic sprouting and regulates splicing of VEGF-receptor 1. Proc. Natl Acad. Sci. USA 108, 3276u20133281 (2011)._ _ _ _ _ Zhang, B. & Horvath, S. A general framework for weighted gene co-expression network analysis. Stat. Appl. Genet. Mol. B 4, Article17 (2005).Staton, S. C. C., Bare, A., Losos, J. B., Edwards, S. V. & Cheviron, Z. A. Physiological and regulatory underpinnings of geographic variation in reptilian cold tolerance across a latitudinal cline. Mol. Ecol. 27, 2243u2013 2255 (2018)._ _ Pirot, N., Delpech, H. & Deleuze, V. Lung endothelial barrier disruption in Lyl1-deficient mice. Am. J. Physiol. 306, L775u2013L785 (2014)._ _ Pirot, N. et al. LYL activity is required for the maturation of newly formed blood vessels in adulthood. Blood 115, 5270u20135279 (2010)._ _ _ _ Hou, J. et al. MicroRNAu2010185 inhibits angiogenesis in human microvascular endothelial cells through targeting stromal interaction molecule 1. Cell Biol. Int. 40, 318u2013328 (2016)._ _ _ _ Levine, R. L. et al. Activating mutation in the tyrosine kinase JAK2 in polycythemia vera, essential thrombocythemia, and myeloid metaplasia with myelofibrosis. Cancer Cell 7, 387u2013397 (2005)._ _ _ Risau, W. Mechanisms of angiogenesis. Nature 386, 671u2013674 (1997)._ _ _ _ Roth, G., Nishikawa, K. C. & Wake, D. B. Genome size, secondary simplification, and the evolution of the brain in salamanders. Brain Behav. Evol. 50, 50u201359 (1997)._ _ _ _ Roy, S., Gardiner, D. M. & Bryant, S. V. Vaccinia as a tool for functional analysis in regenerating limbs: ectopic expression of Shh. Devel. Biol. 218, 199u2013 205 (2000)._ _ _ Sakamaki, K. Regulation of endothelial cell death and its role in angiogenesis and vascular regression. Curr. Nuero. Res. 1, 305u2013315 (2004)._ _ Cramp, R. L., McPhee, R. K., Meyer, E. A., Ohmer, M. E. & Franklin, C. E. First line of defence: the role of sloughing in the regulation of cutaneous microbes in frogs. Cons. Physiol. 2, cou012 (2014)._ _ Russo, C. J. M., Ohmer, M. E. B., Cramp, R. L. & Franklin, C. E. A pathogenic skin fungus and sloughing exacerbate cutaneous water loss in amphibians. J. Exp. Biol. 221, jeb167445 (2018)._ _ _ DeWitt, T. J., Sih, A. & Wilson, D. S. Costs and limits of phenotypic plasticity. Trends Ecol. Evol. 13, 77u201381 (1998)._ _ _ _ Murren, C. J. et al. Constraints on the evolution of phenotypic plasticity: limits and costs of phenotype and plasticity. Heredity 115, 293u2013301 (2015)._ _ _ _ _ Garcia, A. M., Cox, R. M. & Williams, J. B. Phenotypic flexibility in cutaneous water loss and lipids of the stratum corneum in House Sparrows (Passer domesticus) following acclimation to high and low humidity. Physiol. Biochem. Zool. 81, 87u201396 (2015)._ _ Champagne, A. M., Allen, H. C. & Williams, J. B. Lipid composition and molecular interactions change with depth in the avian stratum corneum to regulate cutaneous water loss. J. Exp. Biol. 218, 3032u20133041 (2015)._ _ _ Champagne, A. M. et al. Lipid composition of the stratum corneum and cutaneous water loss in birds along an aridity gradient. J. Exp. Biol. 215, 4299u2013 4307 (2012)._ _ _ _ MuxF1oz-Garcia, A., Thomas, C. P., Keeney, D. S., Zheng, Y. & Brash, A. R. The importance of the lipoxygenase-hepoxilin pathway in the mammalian epidermal barrier. Mol. Cell Biol. Lipids 1841, 401u2013 408 (2014)._ _ Jobard, F. et al. Lipoxygenase-3 (ALOXE3) and 12(R)-lipoxygenase (ALOX12B) are mutated in non-bullous congenital ichthyosiform erythroderma (NCIE) linked to chromosome 17p13.1. Hum. Mol. Genet 11, 107u2013 113 (2002)._ _ _ _ Spotila, J. R. & Berman, E. N. Determination of skin resistance and the role of the skin in controlling water loss in amphibians and reptiles. Comp. Biochem. Physiol. A 55, 407u2013 411 (1976)._ _ _ _ Feder, M. E. Integrating the ecology and physiology of plethodontid salamanders. Herpetologica 39, 291u2013 310 (1983)._ Peterman, W. E., Connette, G. M., Semlitsch, R. D. & Eggert, L. S. Ecological resistance surfaces predict fine-scale genetic differentiation in a terrestrial woodland salamander. Mol. Ecol. 23, 2402u2013 2413 (2014)._ _ _ Lillywhite, H. B., Mittal, A. K. & Garg, T. K. Integumentary structure and its relationship to wiping behaviour in the common Indian tree frog, Polypedates maculatus. J. Zool. 243, 675u2013 687 (1997)._ _ Oufiero, C. E. & Van Sant, M. J. Variation and repeatability of cutaneous water loss and skin resistance in relation to temperature and diel variation in the lizard Sceloporus consobrinus. J. Comp. Physiol. B 188, 671u2013681 (2018)._ _ _ Duursma, R. A. et al. On the minimum leaf conductance: its role in models of plant water use, and ecological and environmental controls. New Phytol. 221, 693u2013705 (2019)._ _ _ Gunderson, A. R., Siegel, J. & Leal, M. Tests of the contribution of acclimation to geographic variation in water loss rates of the West Indian lizard Anolis cristatellus. J. Comp. Physiol. B 181, 965u2013972 (2011)._ _ _ _ Schuster, A.-C. et al. Effectiveness of cuticular transpiration barriers in a desert plant at controlling water loss at high temperatures. AoB Plants 8, plw027 (2016)._ _ _ _ Smith, E. K., ONeill, J. J., Gerson, A. R., McKechnie, A. E. & Wolf, B. O. Avian thermoregulation in the heat: resting metabolism, evaporative cooling, and heat tolerance in Sonoran Desert songbirds. J. Exp. Biol. 220, 3290u20133300 (2017)._ _ _ McKechnie, A. E. et al. Avian thermoregulation in the heat: efficient evaporative cooling allows for extreme heat tolerance in four southern Hemisphere columbids. J. Exp. Biol. 219, 2145u20132155 (2016)._ _ _ Chaves, M. M. et al. Controlling stomatal aperture in semi-arid regionsu2014 the dilemma of saving water or being cool? Plant Sci. 251, 54u201364 (2016)._ _ _ _ Tieleman, B. I. & Williams, J. B. Cutaneous and respiratory water loss in larks from arid and mesic environments. Physiol. l Biochem. Zool. 75, 590u2013599 (2002)._ _ Levis, N. A., Isdaner, A. J. & Pfennig, D. W. Morphological novelty emerges from pre-existing phenotypic plasticity. Nat. Ecol. Evol. 2, 1289u20131297 (2018)._ _ _ Oostra, V., Saastamoinen, M., Zwaan, B. J. & Whear, C. W. Strong phenotypic plasticity limits potential for evolutionary responses to climate change. Nat. Commun. 9, 1005 (2018)._ _ _ _ _ Riddell, E. A. & Sears, M. W. Geographic variation of resistance to water loss within two species of lungless salamanders: implications for activity. Ecosphere 6, Article 86 (2015)._ _ IPCC. Climate Change 2014: Synthesis Report. Contribution of Working Groups I, II and III to the Fifth Assessment Report of the Intergovernmental Panel on Climate Change (eds Core Writing Team, Pachauri, R. K. & Meyer, L. A.), 151 pp. (IPCC, Geneva, Switzerland, 2014).Riddell, E. A. & Sears, M. W. Physical calculations of resistance to water loss improve species range models: reply. Ecology 98, 2965u20132968 (2017)._ _ _ Spotila, J. R. Role of temperature and water in the ecology of lungless salamanders. Ecol. Mono 42, 95u2013125 (1972)._ _ Zuur, A. F., Ieno, E. N. & Elphick, C. S. A protocol for data exploration to avoid common statistical problems. Methods Ecol. Evol. 1, 3u201314 (2009)._ _ Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 31u201321 (2014)._ _ Young, M. D., Wakefield, M. J., Smyth, G. K. & Oshlack, A. Gene ontology analysis for RNA-seq: accounting for selection bias. Genome Biol. 11, R14 (2010)._ _ _ _ Ballouz, S., Pavlidis, P. & Gillis, J. Using predictive specificity to determine when gene set analysis is biologically meaningful. Nucleic Acids Res. 45, e20 (2017)._ _ Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl Acad. Sci. USA 102, 15545u201315550 (2005)._ _ _ _ \"- - \"We thank Miranda Gray for her support in the sequencing process, Damien Wilburn for his advice, and Molly Womack for her comments on the manuscript. I also thank the Grant-In-Aid program at the Highlands Biological Research Station, Clemson University, and the Biological Sciences Department at Clemson for financial support._We also thank the Biological Sciences Department and Clemson University Libraries for providing funds to publish the manuscript. This research was funded by the Doctoral Dissertation Improvement Grant (#1601485) through the National Science Foundations Division of Environmental Biology.\"- - Authors and Affiliations - \"Department of Biological Sciences, Clemson University, 132 Long Hall, Clemson, SC, 29631, USAEric A. Riddell,_Emma Y. Roback,_Christina E. Wells_&_Michael W. SearsBiology Department, Grinnell College, 1116 Eighth Ave, Grinnell, IA, 50112, USAEmma Y. RobackDepartment of Ecology and Evolutionary Biology, Cornell University, E145 Corson Hall, Ithaca, NY, 14853, USAKelly R. Zamudio\" - \"AuthorsEric A. RiddellYou can also search for this author in _Emma Y. RobackYou can also search for this author in _Christina E. WellsYou can also search for this author in _Kelly R. ZamudioYou can also search for this author in _Michael W. SearsYou can also search for this author in _\" - Contributions - E.A.R. conducted the experiments, extracted the RNA, prepared the cDNA libraries, performed the analyses, and wrote the manuscript. E.Y.R. and C.E.W. constructed the de novo transcriptome. C.E.W. conducted the GSEA analysis. K.R.Z., C.E.W. and M.W.S. helped to design the experiments and helped with the manuscript preparation. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\'Peer review information: Nature Communications thanks Shane Campbell-Staton and the other, anonymous, reviewers for their contribution to the peer review of this work. Peer reviewer reports are available.\\' - \"Publishers note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License ().\"- - \"With further research it should be possible to build eDNA profiles of tablets that reflect their journey into the blisters. The techniques could also be used for medicines in capsules and medicines and vaccines in vials/syringes. One key issue that confounds the results is that the genomic assemblages will reflect the aggregate eDNA of the tablet components. For example, artesunate is derived from Artemisia annua and tablets may include maize flour, both plant products. So the artesunate tablets may include eDNA from the plant ingredients, from the fields in which they were grown, the container in which it was transported, the air and water in the factory where the artesunate was manufactured, the plastic used to make the blisters and the people, whether involved or bystanders, who came close to the ingredients. Hence, each encapsulated tablet will contain a u2018pharmabiome representing the ingress of eDNA at different stages in the creation of that tablet and should be characteristic of those processes in those environments and at that time. As such, we would expect the pharmabiome to vary through time and space as the surrounding eDNA envelope varies. Despite the enormous global public health importance of medicine quality, there has been very limited research and funding to address this final common pathway to what patients consume. The ability of data from this novel technique to inform manufacturing origins will be dependent on the growing availability of accurate databases of the spectra of eDNA globally and how these vary through time and space. Further work is needed using the creation of simulated medicines in different environments to understand the accrual of eDNA at different stages of the creation of tablets and how these spectra differ between different environments and seasons.\"- - \\' \\' - The datasets generated and analysed during the current study are available in the Sequence Read Archive (accession number PRJNA860729). - \\' \\'- - \"World Health Organization. WHO global surveillance and monitoring system for substandard and falsified medical products (2017).Pisani, E. et al. Identifying market risk for substandard and falsified medicines: an analytic framework based on qualitative research in China, Indonesia Turkey and Romania. Wellcome Open Res. 4, 70 (2019)._ _ Newton, P. N. et al. Manslaughter by fake artesunate in Asiau2014Will Africa be next?. PLoS Med. 3, 6 (2006)._ Hall, K. A. et al. Characterization of counterfeit artesunate antimalarial tablets from southeast Asia. Am. J. Trop. Med. Hyg. 75(5), 804u2013 811 (2006)._ _ _ Nayyar, G. M. et al. Falsified and substandard drugs: Stopping the pandemic. Am. J. Trop. Med. Hyg. 100(5), 1058u20131065 (2019)._ _ _ Nayyar, G. M., Breman, J. G., Newton, P. N. & Herrington, J. Poor-quality antimalarial drugs in southeast Asia and sub-Saharan Africa. Lancet Infect. Dis. 12(6), 488u2013496 (2012)._ _ Tabernero, P., FernxE1ndez, F. M., Green, M., Guerin, P. J. & Newton, P. N. Mind the gaps-the epidemiology of poor-quality anti-malarials in the malarious world-analysis of the World Wide antimalarial resistance network database. Malar. J. 13(1), 139 (2014)._ _ Dondorp, A. M. et al. Fake antimalarials in Southeast Asia are a major impediment to malaria control: Multinational cross-sectional survey on the prevalence of fake antimalarials. Trop. Med. Int. Health 9, 1241u20131246 (2004)._ _ _ Newton, P. N., Caillet, C. & Guerin, P. J. A link between poor quality antimalarials and malaria drug resistance?. Expert Rev. Anti-Infect. Ther. 14(6), 531u2013533 (2016)._ _ _ Newton, P. N. et al. A collaborative epidemiological investigation into the criminal fake artesunate trade in South East Asia. PLoS Med. 5, 2 (2008)._ _ Allwood, J. S., Fierer, N. & Dunn, R. R. The future of environmental DNA in forensic science. Appl Environ Microbiol. 86(2), e01504u2013e01519 (2020)._ _ _ _ L. Burgoyne, D. Catcheside, K.P. Kirkbride, C. Pearman. Assessment of the utility of obtaining human profiles from drug seizures. Canberra: National Drug Law Enforcement Research Fund. Monograph Series; No. 42(2012).Bruijns, B., Tiggelaar, R. & Gardeniers, H. Massively parallel sequencing techniques for forensics: A review. Electrophoresis 39(21), 2642u2013 2654 (2018)._ _ _ Tridico, S. R., Murray, D. C., Addison, J., Kirkbride, K. P. & Bunce, M. Metagenomic analyses of bacteria on human hairs: A qualitative assessment for applications in forensic science. Investig. Genet. 5(1), 16 (2014)._ _ Williams, D. W. & Gibson, G. Individualization of pubic hair bacterial communities and the effects of storage time and temperature. Forensic Sci. Int. Genet. 26, 12u201320 (2017)._ _ _ Metcalf, J. L. et al. Microbiome tools for forensic science. Trends Biotechnol. 35(9), 814u2013823 (2017)._ _ _ Fierer, N. et al. Forensic identification using skin bacterial communities. Proc. Natl. Acad. Sci. U.S.A. 107(14), 6477u20136481 (2010)._ _ _ _ Schmedes, S. E., Woerner, A. E. & Budowle, B. Forensic human identification using skin microbiomes. Appl. Environ. Microbiol. 83(22), e01672-e1717 (2017)._ _ _ _ Young, J. M., Higgins, D. & Austin, J. J. Soil DNA: Advances in DNA technology offers a powerful new tool for forensic science. Geol. Soc. Spec. Publ. 492, SP492-2017u2013 351 (2019)._ Leontidou, K. et al. DNA metabarcoding of airborne pollen: New protocols for improved taxonomic identification of environmental samples. Aerobiologia 34(1), 63u201374 (2018)._ _ FlxF8 jgaard, C. et al. Predicting provenance of forensic soil samples: Linking soil to ecological habitats by metabarcoding and supervised classification. PLoS ONE 14, 7 (2019)._ _ Young, J. & Linacre, A. Massively parallel sequencing is unlocking the potential of environmental trace evidence. Forensic Sci. Int. Genet. 50, 102393 (2020)._ _ Bowers, R. M., McLetchie, S., Knight, R. & Fierer, N. Spatial variability in airborne bacterial communities across land-use types and their relationship to the bacterial communities of potential source environments. ISME J. 5(4), 601u2013612 (2011)._ _ _ Bowers, R. M. et al. Sources of bacteria in outdoor air across cities in the midwestern United States. Appl. Environ. Microbiol. 77(18), 6350u20136356 (2011)._ _ _ _ Behzad, H., Gojobori, T. & Mineta, K. Challenges and opportunities of airborne metagenomics. Genome Biol. Evol. 7(5), 1216u20131226 (2015)._ _ _ Roeselers, G. et al. Microbial biogeography of drinking water: Patterns in phylogenetic diversity across space and time. Environ. Microbiol. 17(7), 2505u20132514 (2015)._ _ Weyrich, L. S. et al. Laboratory contamination over time during low-biomass sample analysis. Mol. Ecol. Resour. 19(4), 982u2013996 (2019)._ _ _ Caporaso, J. G. et al. Ultra-high-throughput microbial community analysis on the Illumina HiSeq and MiSeq platforms. ISME J. 6(8), 1621u20131624 (2012)._ _ _ Stoeck, T. et al. Multiple marker parallel tag environmental DNA sequencing reveals a highly complex eukaryotic community in marine anoxic water. Mol. Ecol. 19, 21u201331 (2010)._ _ _ Amaral-Zettler, L. A., McCliment, E. A., Ducklow, H. W. & Huse, S. M. A method for studying protistan diversity using massively parallel sequencing of V9 hypervariable regions of small-subunit ribosomal RNA genes. PLoS ONE 4(7), e6372 (2009)._ _ _ Bolyen, E. et al. Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2. Nat. Biotechnol. 37(8), 852u2013857 (2019)._ _ _ Haak, W. et al. Ancient DNA from the first European farmers in 7500-year-old Neolithic sites. Science 310(5750), 1016u20131018 (2005)._ _ _ _ Young, J. M., Higgins, D. & Austin, J. Hybridization enrichment to improve forensic mitochondrial DNA analysis of highly degraded human remains. Front. Ecol. Evol. 7, 450 (2019)._ _ Andrews, R. M. et al. Reanalysis and revision of the Cambridge reference sequence for human mitochondrial DNA. Nat. Genet. 23(2), 147u2013247 (1999)._ _ _ Parson, W. & DxFCr, A. EMPOPu2014A forensic mtDNA database. Forensic Sci. Int. Genet. 1(2), 88u201392 (2007)._ _ Parson, W. et al. DNA commission of the international society for forensic genetics: Revised and extended guidelines for mitochondrial DNA typing. Forensic Sci. Int. Genet. 13, 134u2013142 (2014)._ _ _ Zimmermann, B. et al. Application of a west Eurasian-specific filter for quasi-median network analysis: Sharpening the blade for mtDNA error detection. Forensic Sci. Int. Genet. 5(2), 133u2013137 (2011)._ _ _ Huber, N., Parson, W. & DxFCr, A. Next generation database search algorithm for forensic mitogenome analyses. Forensic Sci. Int. Genet. 37, 204u2013214 (2018)._ _ _ ur Rasheed, H., Jawad, M., Nazir, S., Noreen, S. & Rakha, A. mtDNAmap: Geographic representation of mtDNA Haplogroups. Forensic Sci. Int. Genet. Suppl. Ser. 6, e516u2013e517 (2017)._ _ Rishishwar, L. & Jordan, I. K. Implications of human evolution and admixture for mitochondrial replacement therapy. BMC Genomics 18(1), 1u201311 (2017)._ _ Faircloth, B. C. & Glenn, T. C. Not all sequence tags are created equal: Designing and validating sequence identification tags robust to indels. PLoS ONE 7(8), e42543 (2012)._ _ _ _ Liu, X.-Y. & Zheng, R.-Y. New taxa of Ambomucor (Mucorales, Mucoromycotina) from China. Mycotaxon 130(1), 165u2013171 (2015)._ _ Zheng, R.-Y. & Liu, X.-Y. Ambomucor gen. & spp. nov. from China. Mycotaxon 126(1), 97u2013108 (2014)._ _ Xu, D., Song, W. & Hu, X. Notes on two marine ciliates from the Yellow Sea, China: Placus salinus and Strombidium apolatum (Protozoa, Ciliophora). J. Ocean Univ. China 4(2), 137u2013 144 (2005)._ _ _ Habtom, H. et al. Applying microbial biogeography in soil forensics. Forensic Sci. Int. Genet. 38, 195u2013203 (2019)._ _ _ Damaso, N. et al. Bioinformatics approach to assess the biogeographical patterns of soil communities: The utility for soil provenance. J. Forensic Sci. 63(4), 1033u20131042 (2018)._ _ _ Gunde-Cimerman, N. & Plemenitau0161, A. Ecology and molecular adaptations of the halophilic black yeast Hortaea werneckii. In Life in Extreme Environments 177u2013185 (Springer, 2006)._ Caballero, J. R. I. et al. Genome comparison and transcriptome analysis of the invasive brown root rot pathogen, Phellinus noxius, from different geographic regions reveals potential enzymes associated with degradation of different wood substrates. Fungal Biol. 124(2), 144u2013154 (2020)._ _ Li, X. & Li, Y. Quality comparison and analysis on white Flammulina velutipes grown with bottle lines in China. Edible Fungi China 33, 20u201324 (2014)._ _ Liu, X. B., Li, J. & Yang, Z. L. Genetic diversity and structure of core collection of winter mushroom (Flammulina velutipes) developed by genomic SSR markers. Hereditas 155(1), 3 (2018)._ _ Hollingsworth, P. M., Li, D.-Z., van der Bank, M. & Twyford, A. D. Telling plant species apart with DNA: From barcodes to genomes. Philos. Trans. R. Soc. Lond. B. Biol. Sci. 371(1702), 20150338 (2016)._ _ Kanokwongnuwut, P., Kirkbride, K. P. & Linacre, A. Detection of latent DNA. Forensic Sci. Int. Genet. 37, 95u2013101 (2018)._ _ _ Templeton, J. E., Taylor, D., Handt, O., Skuza, P. & Linacre, A. Direct PCR improves the recovery of DNA from various substrates. J. Forensic Sci. 60(6), 1558u2013 1562 (2015)._ _ _ De la Puente, M. et al. The Global AIMs Nano set: A 31-plex SNaPshot assay of ancestry-informative SNPs. Forensic Sci. Int. Genet. 22, 81u201388 (2016)._ _ Phillips, C. et al. MAPlex-A massively parallel sequencing ancestry analysis multiplex for Asia-Pacific populations. Forensic Sci. Int. Genet. 42, 213u2013226 (2019)._ _ _ Young, J. M., Power, D., Kanokwongnuwut, P. & Linacre, A. Ancestry and phenotype predictions from touch DNA using massively parallel sequencing. Int. J. Legal Med. 135(1), 81u201389 (2021)._ _ \"- - \"We are very grateful to the Director and staff of Mahosot Hospital, Director of the Department of Health Care, Ministry of Health and H.E. the Minister of Heath, Lao PDR, for their support for the work of the Lao-Oxford-Mahosot Hospital-Wellcome Trust Research Unit (LOMWRU) within Mahosot Hospital. We especially thank Arjen Dondorp, Mayfong Mayxay, and T. T. Hien for contributing samples and the late Dr Rattanaphone Phetsouvanh for her support for this work._This research was funded in whole, or in part, by the Wellcome Trust [202935/Z/16/Z]. For the purpose of Open Access, the author has applied a CC BY public copyright licence to any Author Accepted Manuscript version arising from this submission.\"- - Authors and Affiliations - \"College of Science and Engineering, Flinders University, Adelaide, South Australia, 5042, AustraliaJennifer M. Young,_Craig Liddicoat_&_Adrian LinacreSchool of Public Health, The University of Adelaide, Adelaide, South Australia, 5005, AustraliaCraig LiddicoatAdvanced DNA, Identification and Forensic Facility, School of Biological Sciences, University of Adelaide, Adelaide, South Australia, 5005, AustraliaKor-jent van DijkLao-Oxford-Mahosot-Hospital-Wellcome Trust Research Unit (LOMWRU), Microbiology Laboratory, Mahosot Hospital, Vientiane, Lao PDRPatricia Tabernero,_Celine Caillet_&_Paul N. NewtonCentre for Tropical Medicine and Global Health, Nuffield Department of Medicine, University of Oxford, Oxford, OX3 7LJ, UKPatricia Tabernero,_Celine Caillet_&_Paul N. NewtonInfectious Diseases Data Observatory, Nuffield Department of Medicine, University of Oxford, Oxford, OX3 7LJ, UKPatricia Tabernero,_Celine Caillet,_Nicholas J. White_ &_Paul N. NewtonPublic Health Unit, Faculty of Medicine, University of AlcalxE1 , 28871, AlcalxE1 de Henares, SpainPatricia TaberneroMahidol Oxford Research Unit, Faculty of Tropical Medicine, Mahidol University, Bangkok, 10400, ThailandNicholas J. White_&_Paul N. NewtonAustralian Centre for Ancient DNA, School of Biological Sciences, University of Adelaide, Adelaide, South Australia, 5005, AustraliaJeremy J. AustinFaculty of Infectious and Tropical Diseases, London School of Hygiene and Tropical Medicine, London, WC1E 7HT, UKPaul N. NewtonSchool of Public Health, Boston University, Boston, MA, 02118, USAPaul N. Newton\" - \"AuthorsJennifer M. YoungYou can also search for this author in _Craig LiddicoatYou can also search for this author in _Kor-jent van DijkYou can also search for this author in _Patricia TaberneroYou can also search for this author in _Celine CailletYou can also search for this author in _Nicholas J. WhiteYou can also search for this author in _Adrian LinacreYou can also search for this author in _Jeremy J. AustinYou can also search for this author in _Paul N. NewtonYou can also search for this author in _\" - Contributions - J.M.Y., P.T., C.C., N.J.W., A.L. and P.N.N. planned the study, P.T., C.C., P.N.N. led the collection of samples, J.M.Y., C.L., K.V.D., A.L., J.J.A. led the genomic and data analysis, J.M.Y. and P.N.N. wrote the first draft and all authors contributed and reviewed the final manuscipt. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License ).\" - \\' \\' - \"In this study, we sampled a 460-cm-long sediment profile, YC1 (Fig._). Using gas chromatography-mass spectrometry (GC-MS), we report on the detection and identification of alkyl amides as trimethylsilyl (TMS)-derivatives and present the results as depth profiles. We also discuss the occurrence of discrepancies between the profiles of the short and long chain amides, their possible sources, and the potential of alkyl amides as markers for archeological and paleoenvironmental studies.\"- - Identification of alkyl amides in sediment - \"The sediment samples from Yuchisi were freeze-dried, Soxhlet extracted, saponified, separated into alkane, alcohol and acid fractions, and then analyzed by GC-MS. The acid fraction of the samples had significant peaks with intense fragment ions at m/z 73, 116, 131, and 144, besides the normal TMS esters of fatty acids (key ion at m/z 117) (Fig._). The mass spectra of these peaks have a fragmentation pattern similar to those of fatty acid TMS esters, but a molecular weight 1u2009 Da less (Fig._). Interpretation of this fragmentation pattern suggests that these peaks are alkyl amides.\" - \\'Figure 2Distributions of lipid tracers in profile YC1: (a) alkyl amides in sample YC1-225 (m/z 116, as TMS); (b) fatty acids in sample YC1-225 (m/z 117, as TMS); (c) alkyl nitriles in sample YC1-225 (m/z 122, in the acid fraction); (d) alkyl nitriles in sample YC1-225 (m/z 122, in the alkanol fraction).\\' - \\' \\' - Figure 3Mass spectra of (a) docos-13-enamide-TMS derivative (standard), (b) docos-13-enoic acid-TMS ester, and (c) docos-13-enamide. - \\' \\' - \"The molecular weight of erucamide is 337, with key ions at m/z 59 and 72 in its mass spectrum. After silylation, the characteristic mass spectra of oleamide and erucamide had ions at m/z 73, 116, 131, and 144, and M at m/z 353 and 409, respectively (Fig._ and Supplemental Material, Fig._), corresponding to those of the peak series in the YC1 samples. Based on these data, we chose the m/z 116 key ion as characteristic of alkyl amide mono-TMS derivatives (e.g. Fig._ ). The retention times of the alkyl amides-TMS with two less carbons are very close to those of the fatty acids-TMS (e.g. the retention time of C16 amide is 40.33u2009min while the retention time of C18 acid is 40.40u2009min) (Fig._ ).\" - \"Alkyl nitriles were detected in the total extract, alcohol and acid fractions of the samples with the alkyl amides (Table_). The series mainly comprises two compounds, oleonitrile (C18:1) and erucanitrile (C22:1) (e.g., key ion m/z 122, Fig._ and SM-2), with the same carbon number maxima as the parent alkyl amides. Alkyl nitriles were proposed to form by dehydration of alkyl amides during pyrolysis. The occurrence of alkyl nitriles in environmental samples has been reported and their origin attributed to the thermal alteration (e.g., by burning) of organic matter containing alkyl amides. In the present study, based on a doping test with standards, the similar distributions of both nitriles and amides indicated that the nitriles formed from the dehydration of the corresponding alkyl amides. Therefore, the amounts of alkyl amides and corresponding nitriles were summed to represent total alkyl amides.\" - Table 1 Concentrations of total alkyl amides, fatty acids and alkyl nitriles in the sediments of profile YC1. - \\' \\' - Composition and distribution patterns of alkyl amides and fatty acids in profile YC1 - \"The composition and distribution patterns of alkyl amides and fatty acids are overall comparable in the sediments of YC1, and their concentrations are similar. Both series have even-to-odd carbon number preferences and range from C14 to C22 (Fig._). The alkyl amides have the major homologs of C16, C18:1, C18 and C22:1, and the fatty acids are mainly C16 and C18 (Fig._). The unsaturated C18 amides consist of three peaks (Fig._), the same as the unsaturated C18 acids. In addition, the C22:1 amide and C22:1 acid have the double bond position at C-13 for both. This suggests a precursor-product relationship among the C18:1 and C22:1 amides and acids, respectively. The concentrations of C14 and C20 alkyl amides were too low for detection. The homolog distributions of the alkyl amides, as in the case of the fatty acids, indicate a biogenic origin.\" - \"In profile YC1, unsaturated C18:1 amides are the dominant ones, followed by unsaturated C22:1 amide. The concentrations of the C18:1 and C22:1 amide range from 1.53 to 6.01u2009u03BCg/g and 0.25 to 4.83u2009u03BCg/g, respectively. The saturated C16 and C18 amides are less abundant with concentrations ranging from 0.61 to 1.73u2009u03BCg/g and from 0.20 to 1.39u2009u03BCg/g, respectively. The three shorter chain (C16, C18:1 and C18) alkyl amides exhibit generally similar concentration-versus-depth profiles, and three peaks are evident, at depths of 218u2013220u2009cm, 328u2013330u2009cm and 448u2013450u2009cm. Longer chain C22:1 amides exhibit a similar profile as the three shorter chain forms, except for a clear difference at the depths of 198u2013210u2009cm. The concentrations of the C16, C18 and C18:1 alkyl amides increase from 208u2013210u2009cm to 198u2013200u2009cm, while the C22:1 amide decreases from 208u2013210u2009 cm to 198u2013200u2009cm (Fig._).\" - \"Figure 4Concentration profiles of the alkyl amides (the sum of the alkyl amides and nitriles) (u03BCg/g) and the fatty acid ratio for C18:2/C18:0 in the Yuchisi sediments (YC1).\" - \\' \\'- - Alkyl amide profiles and the paleoclimate of Yuchisi - \"During several decades around 4,400u2009yr BP, remarkable climatic changes occurred worldwide, including drought, floods, dust storms, and possibly other climatically-related anomalies. Many major prehistoric cultures such as the Akkadian empire in Mesopotamia, the pyramid-constructing Old Kingdom civilization of Egypt, the Harappan 3B civilization of the Indus valley, and the Early Bronze III civilizations of Palestine, Greece, and Crete, reached peaks in development around 4,300u2009 yr BP, but then terminated abruptly before 4,200u2009yr BP, as a result of catastrophic drought and cooling. The archaeological evidence indicates that the sedimentary sequence at Yuchisi records two important Chinese Neolithic cultures: the interval of 450u2013214u2009cm corresponds to the late Dawenkou Culture (5,050u2013 4,400u2009yr B.P.), and the interval of 214u201392u2009cm to the Longshan Culture (4,400u20134,000u2009yr B.P.). Previous studies showed that biomarkers of the fatty acid ratio C18:2/C18:0, alkane ratio 2C31/(C27u2009+u2009C29), n-C18-ol and n-C30-ol, as well as pollen assemblages, exhibited dramatic changes at the depths of the transition between the two cultures (about 4,400u2009yr B.P.). This suggests that the cultural changes recorded in different regions may have been synchronous and were a response to climatic and environmental changes. Our previous investigation also suggested that dramatic changes in climate and environment within a short interval also contributed to the transition between the Dawenkou Culture and Longshan Culture at Yuchisi site. In addition, the results of the present study suggest that the observed changes in alkyl amides within profile YC1 were responses to climatic and cultural changes during this interval.\" - \"Erucamide was the dominant amide in some urban aerosols and was proposed as a tracer for emissions from biomass burning or cooking (erucic acid is, for example, the main acid in rapeseed oil) and hence the C22:1 amide in aerosols; in addition, shorter chain amides (C16, C18:1 and C18) dominate the amide composition of mummies and penguin droppings (Fig._). The short chain amides apparently are more strongly correlated with animal sources such as humans and seabirds, and the additional longer chain (erucic) amides are from vegetation sources associated with food preparation or the burning of plant biomass (possibly for heating in cold weather). Archeological evidence at Yuchisi site shows that agriculture was well developed during the Longshan and Dawenkou cultures. Black charcoal was found in many layers in the Yuchisi sediment profile, in addition to carbonized seeds. The different sources of the short and long chain alkyl amides may reflect their different responses to paleoclimatic or anthropogenic changes (the ratio of the short versus long chain alkyl amides may also reflect different aspects of paleoclimatic or anthropogenic change), leading to different concentration profiles in the record of YC1.\" - Potential of alkyl amides and nitriles as environmental tracers - \"Earlier reports documented the formation of alkyl amides under hydrothermal conditions and in aerosols from biomass burning, implicating higher temperature processes, or via amidation of esters with sedimentary ammonia derived from anaerobic degradation of organic matter in thermally-unaltered sediments. Simoneit et al. treated fatty acids plus aqueous (NH4)HCO3 in confined vessels at 300u2009xB0C for 3 days and detected alkyl amides and nitriles. They proposed that amides and nitriles are formed in the reactions of fatty acids with ammonia or ammonium salts (possibly from protein decomposition) during burning and combustion processes. These reactions might also occur during early diagenesis of organic detritus in soils and sediments. The similar distribution patterns of alkyl amides and alkanoic acids in our samples support this hypothesis. High levels of nitrogen in human feces and domestic animal dung at the site may have provided sufficient ammonia or ammonium salts to react with alkanoic acids and form alkyl amides within the sediment profile.\" - However, the formation mechanism and sources of the alkyl amides in these sediments remain unclear without further research. The sparsity of reports of sedimentary alkyl amides might be due to two reasons. First, the concentrations of alkyl amides are usually lower than those of fatty acids (e.g., in fresh penguin droppings or related ornithogenic sediments - Wang et al., unpublished results). Second, Cn alkyl amides have a GC retention time quite close to that of Cn+2 fatty acids, both as TMS derivatives. Hence, the effective separation of amides from fatty acids is needed for further study of alkyl amides. - \"An ideal tracer compound should be stable, have a simple source, and be environmentally significant. The presence of alkyl amides in the Yuchisi sediments dating to 5,050u2009yr B.P. indicates that they are stable over geological time. Compared to fatty acids which have complex sources, alkyl amides have relatively simple sources. For example, the sources of saturated fatty acids, especially shorter chains (C16 and C18), may be plants, algae, or even bacteria. The chemical composition and possible formation mechanisms suggest that alkyl amides are directly or indirectly related to higher temperature processes and/or protein-derived nitrogen such as ammonia or ammonium. In our sampling environment, this suggest a possible connection with human activities such as cooking and the presence of human/animal feces. Furthermore, in living organisms, unsaturated acids (e.g., C18:1) usually exhibit a higher concentration than the corresponding saturated acid (i.e., C18:0) and readily form amides. Therefore, alkyl amides may be more sensitive indicators of the dynamics of living organisms and potentially broaden the application of alkyl amides in other environments.\" - \"To the best of our knowledge, this is the first study of alkyl amides as TMS-derivatives in extracts of sediment samples which assesses their temporal distribution. The depth profiles of the alkyl amides in the Yuchisi sediments show that they are sensitive to paleoclimatic changes and thus have potential as tracers for paleoenvironmental studies. However, it is noteworthy that although their concentrations are comparable in the Yuchisi sediments, the concentrations of alkyl amides are much lower than those of fatty acids in other sediment samples investigated, such as fresh penguin droppings or related ornithogenic sediments in the Antarctic (Fig._). In addition, our understanding of the sources and formation of alkyl amides in sediment samples are still limited. The variations of alkyl amides concentrations in the Yuchisi sediments need further investigation to determine their precise paleoclimatic and paleoenvironmental significance. In addition, more investigations of sediment samples from different environmental settings and sedimentation conditions are required to fully evaluate the application of alkyl amides as environmental tracers.\"- - Sampling site - \"The sampling site at Yuchisi is located roughly at the center of a hill, and is 2u20133u2009m above ground level. The hill has dimensions of ~370u2009 m (east-west) and ~250u2009m (north-south). The site is bounded on its western and southern sides by minor unsurfaced roads, and on its eastern side by a modern artificial canal (Fig._). The remains of an ancient moat were found around the site. The moat extends 230~240u2009m north-south, 220u2009m east-west, and has a width of 25u201330u2009m, a depth of 4.5u2009m, and forms a circle with an outlet in the south-west corner; the architectural complex of the late Dawenkou Culture is surrounded by this moat. Along the north-south axis and in the center of the moat are three archeological excavation pits. Sediment profile (YC1) is from the south wall of the central excavation pit. It is 4.6u2009 m deep, and its cultural layers and ages have been studied in detail. From top to bottom, the pit was divided into 8 layers or strata (Fig._), and the boundaries of each stratum were assigned in the field under the direction of Prof. Wang Jihuai from the Archaeological Research Institute, Chinese Academy of Social Science, Beijing, China. 230 samples were obtained from the profile, at an interval of 2u2009cm. In this study, we analyzed 16 samples from the profile; the sampling depths were selected based upon previous TOC analysis and archeological exploration.\" - Sample preparation and analysis - \"Prior to analysis, the sediment samples were stored in precleaned brown glass jars. For organic analysis, freeze-dried sediment samples were Soxhlet extracted for 72u2009h with dichloromethane/methanol (DCM/MeOH, 2:1 vv). The extracts were concentrated by rotary evaporation and then saponified using 0.5u2009 M KOH/MeOH. Neutral lipids were partitioned out of the basic solution with hexane. The pH of the saponified extract was then brought to 2 by 6u2009N HCl and acidic lipids were extracted with 20% DCM in hexane. The acidic lipids, which contain the amides, were treated overnight with anhydrous Na2SO4 to remove water. Neutral lipids were further separated by (5% deactivated) silica gel column chromatography using solvents of increasing polarity from hexane to DCM. The acid (with amides) and alcohol fractions were treated with N,O-bistrimethylsilyltrifluoroacetamide (BSTFA) to form trimethylsilyl (TMS)-ester and ether derivatives, respectively, prior to gas chromatography-mass spectrometry (GC-MS) analysis.\" - \\'For identification of alkyl amides, oleamide (cis-9-octadecenamide, CAS: 301-02-0), erucamide (docos-13-enamide, CAS: 112-84-5) were purchased from Sigma-Aldrich. The standards of oleamide and erucamide were subjected to the experimental procedure and also doped on an extracted sediment sample for recovery tests. The standard test samples were Soxhlet extracted, saponified, and separated by silica gel column chromatography. Then, the samples were treated with BSTFA to form TMS derivatives and analyzed by GC-MS.\\' - \"The recovery tests of oleamide and erucamide from the extracted sediment sample using the same extraction and separation scheme were semi-quantitative (75u2013 88%). No alkyl amides were detected in the procedural blank sample. However, alkyl nitriles were generated (up to 30%) during the analytical procedure from the standards and natural amides in the total extract, alcohol and acid fractions. The mass spectra of alkyl nitriles have odd molecular ions, successive even CnH2n fragments, and key ions at m/z 70, 110 for saturated and m/z 69, 122 for unsaturated homologues, respectively. We used m/z 122 as the key ion for GC-MS detection of nitriles. The GC-MS data for the oleamide and erucamide standards contained the corresponding alkyl nitriles, generated during analysis from the amides. Thus, the concentrations of the total alkyl amides were calculated as the sum of the alkyl nitriles plus amides.\" - \"The silylated extract fractions were analyzed on an HP 5890 GC-MS system with a DB-5MS (50u2009mu2009xD7u20090.32 mm and 0.25 u03BCm film thickness) capillary column (J&W). Helium was used as the carrier gas. The MS was operated in electron impact mode at 70u2009eV ionization energy. The GC oven was programmed as follows: hold 2u2009min at 60u2009xB0C, increase to 150u2009xB0C at 10u2009xB0C/min, increase to 300u2009xB0C at 2.5u2009xB0C/min, hold 30u2009 min at 300u2009xB0C. The MS data was acquired in the full scan mode and processed with the Chemstation data system.\"- - \"Meyers, P. A. Applications of organic geochemistry to paleolimnological reconstructions: a summary of examples from the Laurentian Great Lakes. Org. Geochem.34, 261u2013 289 (2003)._ _ _ Abas, B. M. R. & Simoneit, B. R. T. Composition of extractable organic matter of air particles from Malaysia: Initial study. Atmos. Environ.30, 2779u20132793 (1996)._ _ _ _ Simoneit, B. R. T., Rushdi, A. I., Abas, M. R. B. & Didyk, B. M. Alkyl amides and nitriles as novel tracers for biomass burning. Environ. Sci. Technol.37, 16u201321 (2003)._ _ _ _ Didyk, B. M., Simoneit, B. R. T., Pezoa, A. L., Riveros, L. M. & Flores, A. A. Urban aerosol particles of Santiago, Chile: organic content and molecular characterization. Atmos. Environ.34, 1167u20131179 (2000)._ _ _ _ Schauer, J. J., Kleeman, M. J., Cass, G. R. & Simoneit, B. R. T. Measurement of emissions from air pollution sources. 1. C1 through C29 organic compounds from meat charbroiling. Environ. Sci. Technol. 33, 1566u20131577 (1999)._ _ _ _ Schauer, J. J., Kleeman, M. J., Cass, G. R. & Simoneit, B. R. T. Measurement of emissions from air pollution sources. 4. C1 through C27 organic compounds from cooking with seed oils. Environ. Sci. Technol.36, 567u2013575 (2002)._ _ _ _ _ Rogge, W. F., Hildemann, L. M., Mazurek, M. A., Cass, G. R. & Simoneit, B. R. T. Sources of fine organic aerosol: 1.- Charbroilers and meat cooking operations. Environ. Sci. Technol. 1112u2013 1125 (1991).Buckley, S. A., Stott, A. W. & Evershed, R. P. Studies of organic residues from ancient Egyptian mummies using high temperature-gas chromatography-mass spectrometry and sequential thermal desorption-gas chromatography-mass spectrometry and pyrolysis-gas chromatography-mass spectrometry. Analyst.124, 443u2013 452 (1999)._ _ _ _ _ Makou, M. C., Thompson, L. G., Montlucon, D. B. & Eglinton, T. I. High-sensitivity measurement of diverse vascular plant-derived biomarkers in high-altitude ice cores. Geophys. Res. Lett. 36, (2009).Derenne, S., Largeau, C. & Taulelle, F. Occurrence of non-hydrolysable amides in the macromolecular constituent of Scenedesmus quadricauda cell wall as revealed by N NMR: Origin of n-alkylnitriles in pyrolysates of ultralaminae-containing kerogens. Geochim. Cosmochim. Ac57, 851u2013857 (1993)._ _ _ _ Kaal, J., Wagnerb, S. & JaffxE9baInstituto, R. Molecular properties of ultrafiltered dissolved organic matter anddissolved black carbon in headwater streams as determined by pyrolysis-GC-MS. J. Anal. Appl. Pyrol. 118, 181u2013191 (2016)._ _ _ McKee, G. A. & Hatcher, P. G. Alkyl amides in two organic-rich anoxic sediments: A possible new abiotic route for N sequestration. Geochim. Cosmochim. Ac74, 6436u20136450 (2010)._ _ _ _ Nierop, K. G. J. & van Bergen, P. F. Clay and ammonium catalyzed reactions ofalkanols, alkanoic acids and esters under flash pyrolytic conditions. J. Anal. Appl. Pyrol.63, 197u2013208 (2002)._ _ _ Rushdi, A. I. & Simoneit, B. R. T. Condensation reactions and formation of amides, esters, and nitriles under hydrothermal conditions. Astrobiology4, 211u2013224 (2004)._ _ _ _ _ Wang, J. et al. The abruptclimate change near 4,400 yr BP on the cultural transition in Yuchisi, China and its global linkage. Sci. Rep. 6, (2016).Xu, L. et al. Prehistoric culture, climate and agriculture at yuchisi, Anhui province, China. Archaeometry53, 396u2013410 (2011)._ _ _ Wang, J. Primary Discussion on the Settlement of Yuchisi Site. Archaeology and cultural relics4, 20u201328 (2001)._ Booth, R. K. et al. A severe centennial-scale drought in midcontinental North America 4200 years ago and apparent global linkages. The Holocene15, 321u2013328 (2005)._ _ _ Wu, W. & Liu, T. Possible role of the Holocene Event 3 on the collapse of Neolithic Cultures around the Central Plain of China. Quatern. Int.117, 153u2013166 (2004)._ _ Marchant, R. & Hooghiemstra, H. Rapid environmental change in African and South American tropics around 4000 years before present: a review. Earth Sci. Rev.66, 217u2013260 (2004)._ _ _ Wang, J. The original agricultural situation of Huaibei 5000 ye BP-Information of Yuchisi Site. Agr. Archaeol. 1, 188u2013192 (in Chinese) (1998).\"- - This work was supported by the National Natural Science Foundation of China (grant no. 41772366), the State Oceanic Administration of China Special Fund Project (grant SOA201303), Chinese Projects for Investigations and Assessments of the Arctic and Antarctic (grants CHINARE 04-02 and 02-01) and by the China Scholarship Council (grant no. 201404180008).- - Authors and Affiliations - \"Key Laboratory of Global Change and Marine-Atmospheric Chemistry, Third Institute of Oceanography, State Oceanic Administration, Xiamen, 361005, Fujian, ChinaJianjun Wang_&_Liqi ChenInstitute of Polar Environment, University of Science and Technology of China, Hefei, 230026, Anhui, ChinaJianjun Wang,_Libin Xu_&_ Liguang SunDepartment of Chemistry, Oregon State University, Corvallis, OR, 97331, USABernd R. T. SimoneitGuangzhou Institute of Geochemistry, Chinese Academy of Sciences, Guangzhou, 510640, Guangdong, ChinaGuoying Sheng_&_Xinming WangNIH Chemical Genomics Center, National Institute of Health, Bethesda, MD, 20892, USAYuhong Wang\" - \"AuthorsJianjun WangYou can also search for this author in _Bernd R. T. SimoneitYou can also search for this author in _Guoying ShengYou can also search for this author in _Liqi ChenYou can also search for this author in _Libin XuYou can also search for this author in _Xinming WangYou can also search for this author in _Yuhong WangYou can also search for this author in _Liguang SunYou can also search for this author in _\" - Contributions - J.W., B.R.T.S. and L.S. designed and performed the experiment, analyzed the data and wrote the manuscript with contributions from all authors. L.C., L.B., X.W., Y.W. and L.S contributed during the experiment and in the data analysis. All the authors discussed the results. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing Interests - \\' \\' - The authors declare that they have no competing interests. - \\' \\'- - \\' Publisher\\'\\'s note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License []. - \\' \\'- - \\' \\' - The R-scripts used in the analysis of electrochemical data and the Topas refinement setting and results are available in the Zenodo database under a Creative Commons Attribution 4.0 International License []. - \\' \\'- - \"Weppner, W. & Huggins, R. A. Determination of the kinetic parameters of mixedu2010 conducting electrodes and application to the system Li3Sb. J. Electrochem. Soc. 124, 1569u20131578 (1977)._ _ _ _ Shaju, K. M., Subba Rao, G. V. & Chowdari, B. V. R. Influence of li-ion kinetics in the cathodic performance of layered Li(Ni1/3Co1/3Mn1/3)O2. J. Electrochem. Soc. 151, A1324 (2004)._ _ _ Dees, D. W., Kawauchi, S., Abraham, D. P. & Prakash, J. Analysis of the galvanostatic intermittent titration technique (GITT) as applied to a lithium-ion porous electrode. J. Power Sources 189, 263u2013268 (2009)._ _ _ _ Nickol, A. et al. GITT analysis of lithium insertion cathodes for determining the lithium diffusion coefficient at low temperature: challenges and pitfalls. J. Electrochem. Soc. 167, 090546 (2020)._ _ _ _ Chen, C.-H. et al. Development of experimental techniques for parameterization of multi-scale lithium-ion battery models. J. Electrochem. Soc. 167, 080534 (2020)._ _ _ _ Horner, J. S. et al. Electrochemical modeling of GITT measurements for improved solid-state diffusion coefficient evaluation. ACS Appl. Energy Mater. 4, 11460u201311469 (2021)._ _ _ Wen, C. J., Boukamp, B. A., Huggins, R. A. & Weppner, W. Thermodynamic and mass transport properties of u201CLiAlu201D. J. Electrochem. Soc. 126, 2258 (1979)._ _ _ _ Tang, X.-C., Pan, C.-Y., He, L.-P., Li, L.-Q. & Chen, Z.-Z. A novel technique based on the ratio of potentio-charge capacity to galvano-charge capacity (RPG) for determination of the diffusion coefficient of intercalary species within insertion-host materials: theories and experiments. Electrochim. Acta 49, 3113u20133119 (2004)._ _ _ Lacey, M. J., EdstrxF6m, K. & Brandell, D. Visualising the problems with balancing lithiumu2013sulfur batteries by u201Cmappingu201D internal resistance. Chem. Commun. 51, 16502u201316505 (2015)._ _ _ Lacey, M. J. Influence of the electrolyte on the internal resistance of lithiumu2212sulfur batteries studied with an intermittent current interruption method. ChemElectroChem 4, 1997u20132004 (2017)._ _ _ Chien, Y.-C., Menon, A. S., Brant, W. R., Brandell, D. & Lacey, M. J. Simultaneous monitoring of crystalline active materials and resistance evolution in lithiumu2013sulfur batteries. J. Am. Chem. Soc. 142, 1449u2013 1456 (2020)._ _ _ _ Aktekin, B. et al. Understanding the capacity loss in LiNi0.5Mn1.5O4u2013Li4Ti5O12 lithium-ion cells at ambient and elevated temperatures. J. Phys. Chem. C 122, 11234u201311248 (2018)._ _ _ HernxE1ndez, G. et al. Elimination of fluorination: the influence of fluorine-free electrolytes on the performance of LiNi1/3Mn1/3Co1/3O2/siliconu2013 graphite li-ion battery cells. ACS Sustain. Chem. Eng. 8, 10041u201310052 (2020)._ _ _ _ de Levie, R. On porous electrodes in electrolyte solutions. Electrochim. Acta 8, 751u2013780 (1963)._ _ Lasia, A. Impedance of porous electrodes. J. Electroanalyt. Chem. 397, 27u201333 (1995)._ _ _ Bard, A. J. & Faulkner, L. R. Electrochemical Methods: Fundamentals and Applications (Wiley, 2000).Lasia, A. Electrochemical Impedance Spectroscopy and Its Applications (Springer New York, 2014).Charbonneau, V., Lasia, A. & Brisard, G. Impedance studies of Li+ diffusion in nickel manganese cobalt oxide (NMC) during charge/discharge cycles. J. Electroanalyt. Chem. 875, 113944 (2020)._ _ _ Geng, Z., Chien, Y.-C., Lacey, M. J., Thiringer, T. & Brandell, D. Validity of solid-state Li+ diffusion coefficient estimation by electrochemical approaches for lithium-ion batteries. Electrochim. Acta 404, 139727 (2022)._ _ _ R Core Team. R: a language and environment for statistical computing. (2019).Schied, T. et al. Determining the diffusion coefficient of lithium insertion cathodes from GITT measurements: theoretical analysis for low temperatures. ChemPhysChem 11, 885u2013893 (2021)._ _ MxE4rker, K., Reeves, P. J., Xu, C., Griffith, K. J. & Grey, C. P. Evolution of structure and lithium dynamics in LiNi0.8Mn0.1Co0.1O2 (NMC811) cathodes during electrochemical cycling. Chem. Mater. 31, 2545u20132554 (2019)._ _ Delacourt, C., Ati, M. & Tarascon, J. M. Measurement of lithium diffusion coefficient in LiyFeSO4F. J. Electrochem. Soc. 158, A741 (2011)._ _ _ Chouchane, M., Primo, E. N. & Franco, A. A. Mesoscale effects in the extraction of the solid-state lithium diffusion coefficient values of battery active materials: physical insights from 3D modeling. J. Phys. Chem. Lett. 11, 2775u20132780 (2020)._ _ _ _ _ Verma, A. et al. Galvanostatic intermittent titration and performance based analysis of LiNi0.5Co0.2Mn0.3O2 cathode. J. Electrochem. Soc. 164, A3380u2013 A3392 (2017)._ _ _ Hu, J. et al. Evolution of the rate-limiting step: From thin film to thick Ni-rich cathodes. J. Power Sources 454, 227966 (2020)._ _ _ Strehle, B., Friedrich, F. & Gasteiger, H. A. A comparative study of structural changes during long-term cycling of NCM-811 at ambient and elevated temperatures. J. Electrochem. Soc. 168, 050512 (2021)._ _ _ _ Koleti, U. R., Dinh, T. Q. & Marco, J. A new on-line method for lithium plating detection in lithium-ion batteries. J. Power Sources 451, 227798 (2020)._ _ _ Li, J., Downie, L. E., Ma, L., Qiu, W. & Dahn, J. R. Study of the failure mechanisms of LiNi0.8Mn0.1Co0.1O2 cathode material for lithium ion batteries. J. Electrochem. Soc. 162, A1401u2013A1408 (2015)._ _ _ Liu, H. et al. Understanding the roles of tris(trimethylsilyl) phosphite (TMSPi) in LiNi0.8Mn0.1Co0.1O2 (NMC811)/siliconu2013graphite (Siu2013 Gr) lithiumu2010ion batteries. Adv. Mater. Interfaces 7, 2000277 (2020)._ _ _ Xu, C. et al. Bulk fatigue induced by surface reconstruction in layered Ni-rich cathodes for Li-ion batteries. Nat. Mater. 20, 84u201392 (2021)._ _ _ _ _ Xu, C., Reeves, P. J., Jacquet, Q. & Grey, C. P. Phase behavior during electrochemical cycling of Ni-rich cathode materials for Li-ion batteries. Adv. Energy Mater. 11, 1u201312 (2021)._ _ _ Friedrich, F. et al. Editors Choiceu2014Capacity fading mechanisms of NCM-811 cathodes in lithium-ion batteries studied by X-ray diffraction and other diagnostics. J. Electrochem. Soc. 166, A3760u2013A3774 (2019)._ _ _ Liu, S. An analytical solution to Li/Li+ insertion into a porous electrode. Solid State Ion 177, 53u201358 (2006)._ _ _ Subramanian, V. R., Ritter, J. A. & White, R. E. Approximate solutions for galvanostatic discharge of spherical particles. I. Constant diffusion coefficient. J. Electrochem. Soc. 148, E444 (2001)._ _ _ Subramanian, V. R. & White, R. E. New separation of variables method for composite electrodes with galvanostatic boundary conditions. J. Power Sources 96, 385u2013395 (2001)._ _ _ _ Elzhov, T. V., Mullen, K. M., Spiess, A.-N. & Bolker, B. minpack.lm: R interface to the Levenberg-Marquardt nonlinear least-squares algorithm found in MINPACK, plus support for bounds. (2016).Rietveld, H. M. Line profiles of neutron powder-diffraction peaks for structure refinement. Acta Crystallogr. 22, 151u2013152 (1967)._ _ _ Rietveld, H. M. A profile refinement method for nuclear and magnetic structures. J. Appl. Crystallogr. 2, 65u201371 (1969)._ _ _ Coelho, A. A. TOPAS and TOPAS-Academic: An optimization program integrating computer algebra and crystallographic objects written in C++. J. Appl. Crystallogr. 51, 210u2013218 (2018)._ _ _ Chien, Y.-C. et al. Supporting data for u2018Rapid determination of solid-state diffusion coefficients in Li-based batteries via intermittent current interruption method. zenodo (2021).\"- - The authors acknowledge STandUP for Energy consortium. H.L. thanks for the support from the projects (project numbers 42758-1, P2020-90112 and P2022-00055) funded by Swedish Energy Agency. A.S.M. is grateful to the Swedish Foundation for Strategic Research (SSF) for the financial support through the Swedish national graduate school in neutron scattering (SwedNess).- - Open access funding provided by Uppsala University.- - Author notes - \\'Yu-Chuan ChienPresent address: Breathe Battery Technologies, Office 7, 35-37 Ludgate Hill, London, EC4M 7JN, UKAshok S. MenonPresent address: WMG, University of Warwick, Coventry, CV4 7AL, UK\\' - Authors and Affiliations - \"Department of Chemistryu2014xC5ngstrxF6m Laboratory, Uppsala University, Box 538, LxE4gerhyddsvxE4gen 1, 751 21, Uppsala, SwedenYu-Chuan Chien,_Haidong Liu,_Ashok S. Menon,_William R. Brant_&_Daniel BrandellScania CV AB, 151 87, SxF6dertxE4lje, SwedenMatthew J. Lacey\" - \"AuthorsYu-Chuan ChienYou can also search for this author in _Haidong LiuYou can also search for this author in _Ashok S. MenonYou can also search for this author in _William R. BrantYou can also search for this author in _Daniel BrandellYou can also search for this author in _Matthew J. LaceyYou can also search for this author in _\" - Contributions - Y.-C.C. co-conceived the concept, designed the experiments, performed and analyzed the electrochemical measurements, and wrote the majority of the manuscript. H.L. fabricated the electrodes and electrochemical cells, assisted with data interpretation, and revised the manuscript. A.S.M. assisted with the operando X-ray diffraction experiment, analyzed the diffraction data, and wrote about the analysis and interpretation of diffraction data in the manuscript. W.R.B. supervised the analysis and interpretation of the diffraction data and revised the manuscript. D.B. acquired the funding, supervised the project, and revised the manuscript. M.J.L. co-conceived the concept, supervised the project, and revised the manuscript. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks anonymous reviewer(s) for their contribution to the peer review of this work. are available. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License from ref. Scale bar is 1u2009xB5m. b Top view of a diatom valve (scale bar is 1u2009xB5m) and (c) valve morphology (scale bar is 300u2009nm). b, c Two different examples of a diatom valve\" - The pores in diatom valves are of particular importance for both diatom survival in the environment, as well as potential industrial applications. The porous architecture of their exoskeleton enables attractive optical properties, such as light harvesting, confinement, and selective optical transmission. Moreover, despite high porosity, diatom silica exhibits remarkably high mechanical stability, which is important for filtration applications. Controllably changing the pores thus empowers an array of properties, and in turn applications of diatom systems for a wide variety of tasks. - Diatom frustules, including the pores, are assembled through the concerted action of dozens of proteins encoded in the diatom genome. These proteins modulate silica precipitation and sintering to generate specific structures. New techniques in multi-omic characterization have identified many genes that may be involved in biomineralization, based on coordinated expression during cell growth and division. Validation of these predictions, however, is challenging, requiring disrupting expression of each putative frustule formation gene followed by identification of any resulting changes in morphology. While in certain cases changes to frustules may be obvious, often structure variation requires more rigorous analysis. Specifically, the recognition of persistent (but a priori unknown) features is a significant challenge. Further complicating this is the fact that isogenic diatom populations have substantial natural variation in morphological properties, which will render the task more difficult with regards to generalization. Therefore, to controllably modify diatom frustule structure through targeted genetic modifications, a robust genotype to phenotype linking methodology needs to be developed. - \"One potential method that has been pivotal of late is the use of artificial neural networks (NNs) that can learn abstract features from large datasets, negating the need for hand-crafted features. These deep NNs have been instrumental in the recent dramatic improvement of automatic image classification and speech recognition, and have also been applied to scientific domains, such as fitting potential energy surfaces, fundamental understating of phase transitions, processing of the atomically resolved images as well as fitting multiparametric empirical models. Classical image processing approaches work, but data generation rates are replacing them with more automated unsupervised and supervised learning methods. Artificial NNs, however, excel at classification problems and can be very efficient. NNs employ a u201Clearning by exampleu201D approach that optimizes parameters by training on labeled data. Hence, if there are systematic differences between classes, and the network has presented those cases in the training sets, a NN can detect and label features in real/validation data. Given that diatoms exhibit a diverse set of frustule morphologies, a NN is a reasonable choice of a classification scheme to detect the genotype to phenotype translation.\" - In this work, we investigated wild type and genetically modified T. pseudonana to capture the interplay between the changing genotype and the expressed phenotype, as gene manipulation could enable these organisms to be used as a direct source of specifically tailored nanostructured and microstructured materials. We modified the genotype by knocking down genes we suspected to be involved with frustule formation and characterized the phenotype by scanning electron microscopy (SEM). We used image processing and machine learning classification algorithms (artificial NNs) to screen for genes that affect diatom phenotype and to distinguish diatoms with wild type and modified morphologies. With regard to inspected Thaps3_21880 modification, we demonstrated a NN that can identify wild and modified diatoms with 94% accuracy. To explain the apparent efficiency of NN-based classification, class activation maps (CAMs) were used to highlight the image regions used by the network. It was found that pores are the defining features separating wild-type diatoms from one specific knockdown strain. We then created a separate neural net to focus specifically on pores and to extract their parameters. This automated feature extraction process allowed us to correlate the genetic modification with diatom morphology. Our approach identifies the changes in valve structure that result from a given genetic modification, offering biological insight into the biomineralization process.- - Gene modification and testing - We identified protein Thaps3_21880 as a potential frustule biosynthetic enzyme based on structural features and coordinated expression during silicization. To perturb gene expression in T. pseudonana, we synthesized an antisense RNA containing the first 427 nucleotides of the associated gene and expressed this construct from a heterologous plasmid. Two independent clonal knockdown lines were selected for nourseothricin resistance, and then confirmed by polymerase chain reaction (PCR) (Supplementary materials Fig. and Methods section for details). - Identification of morphological changes - \"A successful gene knockdown does not necessarily result in any morphological change in the diatom frustule and, furthermore, in those cases where the morphology is altered, the precise change is difficult to predict. Therefore, a robust method is needed for determining whether a population of diatoms has a variant phenotype and, if so, the details of the variation. SEM was used to visualize the surface morphology of the modified and nonmodified samples, and a typical wild-type diatom is depicted in Fig. , with the details of the structure highlighted in Fig. (see Supplementary materials, Figs. and for more examples of wild diatoms and the example images of modified diatoms). The images reveal a complex feature-rich structure making it difficult for the untrained eye to separate the wild and the modified diatoms, let alone quantify physical changes. Therefore, we used image processing and machine learning strategies to automatically classify wild and modified phenotypes directly from images. Classical image processing techniques proved to be cumbersome in trying to process multiple images since subtle contrast variation required significant manual parameter adjustment and precluded automation. We, therefore, decided to use a NN. Labeled training sets of wild (labeled as zero) and modified (labeled as one) diatom images were prepared in the following way. Each image was spliced into subsets of 89u2009xD7u2009 89-pixel frames. Each frame underwent a series of geometric transformations, such as horizontal and vertical mirroring, rotation, and change in contrast to expand the training set. The NN architecture is shown in Fig. (see Methods section for details). The validation test for this NN showed 94% class identification accuracy. We also justified the NN by only training the net on images of wild diatoms and validated on the separate set containing wild diatoms. As a result, the accuracy was ~50%, which was expected.\" - \"Fig. 2a Schematic of an artificial NN to distinguish between wild and modified diatoms (NN 1); scale bar is 300u2009nm; bu2013e Class activation maps explaining operation of NN 1. b, d Regions of the diatom images that trigger wild activation class (AC); c, e Show regions of corresponding images that trigger modified AC. bu2013e Scale bar is 50u2009nm\" - \"This NN allowed us to automatically screen for changes in images between the modified and wild genotypes, and indirectly screen genes that modify the structure of diatoms. However, NNs are often thought of as a u201Cblack boxu201D approach to solving a problemu2014it is difficult to ascertain what features drive the NNs to make decisions. To gain insights into what image aspects made a significant contribution to classification, we modified this NN, as described in Methods section, in order to peer into the layers using CAMs. Figure shows the same frame of the diatom image, which corresponds to the modified class and was identified by the NN. Figure highlights the regions that trigger wild activation class (AC), while Fig. emphasizes the regions that, for the NN, indicate modified AC. As shown by the colorbar, a whiter color corresponds to higher AC intensity. Figure also displays an identical frame of another diatom image, corresponding to the wild class. CAMs revealed that the NN focuses on pores as one of the main features to distinguish wild and modified diatoms (for additional examples of CAMs for NN, please, see Fig. in Supplementary Materials). In this way, we identified a specific gene connected to the changes in pores of diatoms.\" - Automatic recognition of relevant features and extraction of their parameters - \"Identification of pores emphasizes the importance of pore-related parameters. Thus, a new NN 2 was generated and trained to focus on these image characteristics. First, pores were recognized in several raw images of diatoms. As a result, matching pairs of the original images and corresponding binary images containing only pores were generated and used as a training set for NN 2. Similar to previous NN, training images were cut into 46u2009xD7u200946-pixel frames, which underwent identical geometric and contrast treatment to enlarge the training set. It was important to perform this image segmentation before loading the frames into the neural net, as the contrast variation within a given image often precluded the use of typical filtering approaches. Each frame was processed by the NN 2 (the architecture is shown in Fig. ; see Methods section for details), built in a way that its output would be a pixel-wise classification map corresponding to the input image.\" - \"Fig. 3a Schematic of an artificial NN to recognize pores of the diatoms (NN 2). b Frame of an original diatom image that is loaded into the NN 2. c Pixel-wise classification map of the NN 2 corresponding to raw image (b). d Result of the application of the NN 2 to the image of a diatom (red circles are recognized pores); scale bar is 300u2009nm\" - \"Figure shows pixel-matching images, where Fig. is an original frame of a diatom image that is loaded into the NN 2, and Fig. is an output image of the NN 2. Obtained NN-processed image matches with the raw image and closely resolves the presented pores. Scanning across each image, the pores can be determined, as shown in Fig. , where the raw image is overlapped with the contours of NN-identified pores (red circles). Since the CAMs pointed to pores as the features NN 1 used to separate wild and modified diatoms, we wanted to provide a quantitative analysis of multiple pore-related parameters to capture this change. Using NN-processed images, we extracted the following parameters: density of pores, mean area of pores, and the percentage of area occupied by the pores relative to the total area of the valve captured in an image. Additionally, pore area distribution was extracted and fitted with a Gaussian distribution, which yielded two more parameters, Gaussian value xB5 and Gaussian sigma u03C2, producing a total of five parameters. To minimize the influence of outliers (such as small features that NN could recognize as pores), threshold for pore size was set to be larger than three pixels.\" - The values of four of these parameters were plotted against the values of the fifth parameter (mean area of pores) to test if different regions of the parameter space produce distinguishable types of behavior (Fig. ). Figure reveals that the best separation between wild and modified diatoms can be established through density of pores against the mean area of pores. Figure shows that the separation of these two classes of diatoms can also be shown through the plots of expected Gaussian value and Gaussian sigma as a function of mean area, respectively. Finally, no significant difference between classes is observed in relation to the ratio of pore area to total area parameter (Fig. ). - \"Fig. 4Multifactor analysis of pores. a Density of pores, b expected Gaussian value xB5, c Gaussian sigma u03C2, and d percentage of area occupied by the pores relative to the total area as functions of mean area of pores\" - \"It is also helpful to plot the extracted parameters together to visualize parameter intercorrelation. We explored class interrelationship using a multivariate data visualization tool called CrossVis. CrossVis represents an evolution of both the EDEN and MDX visual analysis systems (see Methods section for details). For clarity, the expected Gaussian value xB5 was removed since its values are identical to the mean area of pores (Fig. ). Parameter values are graphed as polylines in a parallel coordinate plot, and each vertical axis represents one of the pore-related parameters, as shown in Fig. . Parameters of modified diatoms are colored orange, and parameters of wild diatoms are blue. Vertical columns are arranged so the parameters showing better separation between classes are on the left-hand side of the Type column, and the parameters with lesser separation are on the right-hand side. Certain statistics are graphically represented in the interior boxes of each column. Namely, thin blue and orange stripes within each column capture the mean-centered standard deviation of a given parameter of wild and modified diatoms, respectively. Similar to Fig. , this statistical display shows that there is little separation between classes based on the ratio of the pore area to total area and Gaussian sigma u03C2, but diatoms can be clearly classified based on the mean area and density of pores. The distribution of each parameter is also displayed along the vertical axis as shaded histogram bins (light grey is for modified diatoms; dark gray is for wild diatoms).\" - Fig. 5Class interrelationship visualization using CrossVis v.2.1.2. Parameters of modified diatoms are colored orange, and parameters of wild diatoms have a blue color. Histogram bins show the distribution of each parameter, and they have a light grey color for modified diatoms, and a dark gray color for wild diatoms - Figure also facilitates a visual correlation between presented parameters shown with blue and orange lines connecting neighboring vertical axis. Namely, each line connects values of the parameters extracted from the same diatom image. On the one hand, some parameters, such as Gaussian sigma and the ratio of pore area to total area, do not seem to show any correlation between each other. On the other hand, looking at the links between density of pores and mean area of pores, it is observed that these two parameters are negatively correlated within each diatom class (higher density corresponds to smaller mean area and vice versa), and the links have opposite directions for wild and modified diatoms. This behavior suggests that higher density of pores in wild diatoms is compensated by the smaller area of these pores, which is opposite to the trend shown by modified diatoms. In turn, this trend explains why the ratio of pore area to total area is very similar for both classes. - This pore-based multifactor analysis confirmed that, while there is an overlap between parameters of wild and modified diatoms, the proposed and investigated algorithm allowed for their separation and established a correlation between realized gene modification and alteration of frustule morphology. Moreover, it was shown that the performed gene alteration mainly affects such pore parameters as their area and density, increasing the former and decreasing the latter, while keeping the ratio of pore area to total area of frustule constant. - In this work, we have illustrated how machine learning can aid in biomaterial research using diatoms. After gene knockdown in T. pseudonana diatoms, supervised machine learning was successfully applied to screen for genes that altered the structure of the diatoms and separate those with modified morphology. This yielded a process where morphological changes were connected to a specific gene alteration. In the case of Thaps3_21880 modification, one artificial NN was developed to distinguish wild and modified diatoms based on the SEM images of frustules, resulting in 94% accuracy. Then CAMs visualized physical changes that allowed the NNs to separate diatom strains, subsequently establishing a specific gene that controls pores. Another artificial NN was created to process image data, automatically recognize pores, and extract pore-related parameters. In turn, multifactor analysis and visualization of these parameters emphasized the alteration in density and area of pores. An important advantage of the presented approach is that it allows for automated screening of the genes that modify diatom morphology, and it enables recognition and analysis of features, such as pores, whose changes might be indistinguishable with the naked eye.- - Culture and growth conditions - \"T. pseudonana (CCMP1335) stock cultures were maintained in NEPC (North East Pacific Culture) medium (). The cultures were grown in a temperature controlled incubator at 16u2009xB0C under a 14u2009h:10u2009h (light:dark) photoperiod at 100u2009 u03BCmol photon m s of photosynthetically active radiation.\" - Genetic manipulations - \"An antisense construct containing the reverse complement of the first 427 nucleotides of Thaps3_21880 was synthesized de novo and cloned into pMHL009 immediately following the NAT coding sequence (GenScript, Piscataway, NJ). This construct was transformed into T. pseudonana by microparticle bombardment using a Bio-Rad Biolistic PDS-1000/He particle delivery system, as described previously. Briefly, exponentially grown cells were harvested and 1u2009xD7u200910 cells were plated onto an NEPC agar plate lacking antibiotics. A nuclear transformation was performed by bombarding plasmid coated tungsten beads (1.1u2009u03BCm diameter, Bio-Rad M-17) at 1100u2009lb/in under vacuum at a distance of 8u2009 cm onto cells plated on the agar plate. After bombardment, NEPC medium was added to plates, which wasincubated under a 14u2009h:10u2009h (light:dark period) for 24u2009h. After 24u2009h, the plasmid bombarded diatom cells were plated on NEPC bacto-agar plate with 100u2009xB5L/mL of the antibiotic nourseothricin. The resistant colonies were transferred into 24-well plates with 2u2009ml of NEPC medium. To confirm construct integration, cells from well plates were screened by PCR using primers NAT_SeqF (5u2032-AAGGTGTTCCCCGACGACGAATC-3u2032) and 21880_SeqR (5u2032-TATGAGCATGTCTTTGCCACTCAGAC-3u2032), as described previously.\" - Sample preparation for SEM imaging - \"One milliliter of diatom culture (2u2009xD7u200910u2009cells/ml) was collected by centrifugation at 5000u2009rpm for 5u2009min and rinsed once with deionized water. The diatom frustules were cleaned with 1u2009mL concentrated sulfuric acid and boiled in a water bath for 10u2009min. After cooling, we added 20u2009 mg KNO3, then boiled again for 10u2009min. Samples were then washed three times with deionized water using centrifugation. The diatom frustules were spotted on silicon wafer and air dried in hood for approximately 10 min. While these harsh cleaning methods may affect the fine structures of diatom pores, they were necessary to remove residual organic matter and allow clear imaging.\" - SEM imaging of diatoms - \"Dried diatoms on Si wafers were imaged with a Merlin Field Emission SEM (Carl Zeiss, Oberkochen, Germany) operating at a base pressure of 2.8u2009xD7u2009 10 mbar. Images were acquired using an in-lens detector with an accelerating voltage of 1u2009kV, probe current of 100 pA, cycle time of 14.4u2009s, 10 line averages, and resolution of 1024u2009xD7u2009768. Images were collected of diatom valves with a 2u20133u2009xB5m field of view, depending on the individual size of the diatom, to capture the arrays of ribs and pores. Diatoms in various orientations were observed with SEM and only those with valves that were enact, unwrinkled, and approximately parallel to the Si wafer were included in this study, such as in Fig. , c.\" - Computational methods - A series of images of wild and modified diatoms, that were used as training sets, were analyzed using Python 3.6 libraries, such as numpy, scipy, matplotlib, opencv, and scikit-image. Conversion from pixels to nanometers was based on image scale bar, which was programmed to be automatically recognized by the Python code. Both artificial NNs were created using Keras library with a TensorFlow backend in Python 3.6. - Training dataset for NN 1 - \"We have selected 29 images of wild diatoms and 29 images of modified ones. A random selection would first determine the type of the particular image to be added into training set, then the specific image of that type would be also randomly drawn. This ensures that the presence of both types of diatoms is equal. Otherwise, the classifier becomes biased to output the type numerically dominant in the dataset. Image augmentation was done in the following manner. All the images were rescaled using resize function from opencv to have the same pixel/nm value of 3. A 250u2009xD7u2009250 px cutaways were randomly selected from a randomly selected image. Vertical and horizontal mirroring were applied at random. The rotation of the cutaway was done using opencv functions getRotationMatrix2D and warpAffine. We have applied non-linear adjustment of contrast for the training set to incorporate possible artifacts from SEM images. Two cases with equal probabilities were created: e and (1u2009u2212u2009e) where a is a random value between 0 and 1 and I is the intensity in the pixel. The resulting arrays were normalized to [0,u20091]. We have intentionally avoided rescaling as we expected that size of the pores may be characteristic to the diatom type, while SEM contrast, rotation, and mirroring were expected to be varying within the type.\" - Artificial NN to distinguish between wild and modified diatoms (NN 1) - \"NN 1 architecture, shown in Fig. , consists of two convolutional layers with a max pooling layer in between, and then a dense (fully connected) layer. The max pooling layer was added to increase the u201Cconnectednessu201D of the convolutional filters. Rectified linear units (ReLUs) were used as activation functions in these layers. The last layer consists of a two-unit dense layer with softmax activation, so that the classification outputs would sum to one, respecting the choices available. It is this layer that provides the estimate of which class the input image belongs. Optimization of the network was performed with the Adam optimizer utilizing the cross-entropy metric, and NN was trained on a randomly selected subset of images, where 80% were a training set, and 20% were a validation set. The validation test, which predicts the class of the labeled images, showed that NN can determine the class of the diatom with 94% accuracy.\" - The performance of created neural net (NN 1) was additionally validated to confirm that the NN differentiates between wild and modified diatoms based on actual patterns in the morphology and not based on noise or artifacts. We took only images of wild diatoms, randomly split them into two training sets with different labels, and retrained NN 1. Because both sets consist only of images of wild diatoms, NN should not be able to learn from these training sets and separate them if it works properly. As a result, accuracy of the training was ca. 50%, which was expected and validated the performance of NN 1. - Modification of NN 1 to obtain CAMs - CAMs allowed determination of the image regions used by NN 1 to distinguish between wild type and modified diatoms. However, in order to apply the concept of CAMs, next-to-last layer of NN 1 had to be changed from dense layer to global average pooling layer. This modification did not affect the accuracy of NN 1, but allowed to peer within the layers of the NN and extract CAMs. - Artificial NN to recognize pores of the diatoms (NN 2) - NN 2 architecture, shown in Fig. , consists of a convolutional layer with a max pooling layer, followed by two convolutional layers, and an up-sampling layer. An up-sampling layer was added to bring the output image to a higher resolution that would match the resolution of the input image. ReLUs were used as activation functions in these layers. The raw images were normalized to [0, 1], and the thresholding value of 0.5 was used to generate binary images for the NN 2. The last layer consisted of a convolutional layer with only one filter and a sigmoid activation, so that its output would be a pixel-wise classification map corresponding to the input image. - Data visualization - \"Class interrelationship between diatom parameters was inspected using a multivariate data visualization tool called CrossVis. The CrossVis system is a visual analytics system, which combines interactive data visualization and statistical analytics techniques. The system represents an evolution of both the EDEN and MDX systems. CrossVis includes a number of enhancements to the popular parallel coordinate information visualization technique. As shown in Fig. , the parallel coordinates plot yields a two-dimensional plot of multivariate data by representing each N-dimensional tuple with points on N parallel axes, which are joined with a polyline. CrossVis extends the basic parallel coordinates plot with human directed interactions and graphical representations of summary statistics. Users can select parameter ranges of interest by using the mouse to drag a rectangular region on the vertical axes. As shown in Fig. , within the interior of each vertical axis, the mean and mean-centered standard deviation range (95% confidence interval) are shown, where the rectangular box is the range and the horizontal line intersecting the box is the mean value. Correlations between parameters (vertical axes) are apparent when two axes are side-by-side. If the polylines cross to form an X-shaped pattern, the two parameters are negatively correlatedu2014 as one parameter increases in value the other decreases (see the polyline crossings for the two leftmost axes in Fig. ). If the polylines have few crossings, the two parameters are positively correlatedu2014as one parameter increases in value, the other also increases. CrossVis also allows categorical parameters (see the middle Type axis in Fig. ) to be represented, which is an extension of the standard parallel coordinate plot. The display of categorical parameters enables study of the wild vs. modified classes in the current work. Users can select a class by clicking on the appropriate box for the categorical axis. For more details on the parallel coordinate-based techniques, the reader is urged to consult the related references.\" - \"Fig. 6The polyline in a parallel coordinates plot maps the N-dimensional data tuple C with coordinates (c1, c2,u2026, cN) to points on N parallel axes which are joined with a polyline whose N vertices are on the Xi-axis for iu2009=u2009 1,u2026, N\"- - \\' \\' - All data used in this manuscript are available from the authors on request. - \\' \\'- - \\' \\' - Full code is available from the authors on request. - \\' \\'- - 13 August 2019An amendment to this paper has been published and can be accessed via a link at the top of the paper.- - \"De Tommasi, E. et al. UV-shielding and wavelength conversion by centric diatom nanopatterned frustules. Sci. Rep. 8, 16285 (2018)._ _ Jeffryes, C., Solanki, R., Rangineni, Y., Wang, W., Chang, C.-H. & Rorrer, G. L. Electroluminescence and photoluminescence from nanostructured diatom frustules containing metabolically inserted germanium. Adv. Mater. 20, 2633u20132637 (2008)._ _ _ Bismuto, A., Setaro, A., Maddalena, P., De Stefano, L. & De Stefano, M. Marine diatoms as optical chemical sensors: a time-resolved study. Sens. Actuat B-Chem 130, 396u2013399 (2008)._ _ _ Gale, D. K., Gutu, T., Jiao, J., Chang, C.-H. & Rorrer, G. L. Photoluminescence detection of biomolecules by antibody-functionalized diatom biosilica. Adv. Funct. Mater. 19, 926u2013933 (2009)._ _ _ Selvaraj, V., Muthukumar, A., Nagamony, P. & Chinnuswamy, V. Detection of typhoid fever by diatom-based optical biosensor. Environ. Sci. Pollut. R 25, 20385u201320390 (2018)._ _ _ Delalat, B. et al. Targeted drug delivery using genetically engineered diatom biosilica. Nat. Commun. 6, 1u201311 (2015)._ _ Ghobara, M. et al. On Light and diatoms: a photonics and photobiology review. In: Diatoms: Fundamentals and Applications (eds Seckbach, J. & Gordon, R.) (Wiley-Scrivener, 2019).Hildebrand, M., Holton, G., Joy, D. C., Doktycz, M. J. & Allison, D. P. Diverse and conserved nano- and mesoscale structures of diatom silica revealed by atomic force microscopy. J. Microsc. 235, 172u2013187 (2009)._ _ _ Hildebrand, M. et al. Nanoscale control of silica morphology and three-dimensional structure during diatom cell wall formation. J. Mater. Res. 21, 2689u20132698 (2006)._ _ _ De Tommasi, E., Gielis, J. & Rogato, A. Diatom frustule morphogenesis and function: a multidisciplinary survey. Mar. Genom. 35, 1u201318 (2017)._ _ Pawolski, D., Heintze, C., Mey, I., Steinem, C. & KrxF6ger, N. Reconstituting the formation of hierarchically porous silica patterns using diatom biomolecules. J. Struct. Biol. 204, 64u201374 (2018)._ _ _ Aitken, Z. H., Luo, S., Reynolds, S. N., Thaulow, C. & Greer, J. R. Microstructure provides insights into evolutionary design and resilience of Coscinodiscus sp. frustule. Proc. Natl Acad. Sci. USA 113, 2017u20132022 (2016)._ _ _ Hamm, C. E. et al. Architecture and material properties of diatom shells provide effective mechanical protection. Nature 421, 841u2013843 (2003)._ _ _ Leon, S. D. & Markus, J. B. Influence of geometry on mechanical properties of bio-inspired silica-based hierarchical materials. Bioinspir. Biomim. 7, 036024 (2012)._ _ Hildebrand, M. & Lerch, S. J. L. Diatom silica biomineralization: parallel development of approaches and understanding. Semin. Cell Dev. Biol. 46, 27u201335 (2015)._ _ _ Shrestha, R. P. et al. Whole transcriptome analysis of the silicon response of the diatom Thalassiosira pseudonana. BMC Genomics 13, 499u2013515 (2012)._ _ _ Tesson, B., Lerch, S. J. L. & Hildebrand, M. Characterization of a new protein family associated with the silica deposition vesicle membrane enables genetic manipulation of diatom silica. Sci. Rep. 7, 13457 (2017)._ _ Amato, A., Orsini, L., DAlelio, D. & Montresor, M. Life cycle, size reduction patterns, and ultrastructure of the pennate planktonic diatom Pseudo-nitzschia delicatissima (Bacillariophyceae). J. Phycol. 41, 542u2013556 (2005)._ _ Hense, I. & Beckmann, A. A theoretical investigation of the diatom cell size reductionu2013restitution cycle. Ecol. Model. 317, 66u2013 82 (2015)._ _ Dawes, R., Thompson, D. L., Guo, Y., Wagner, A. F. & Minkoff, M. Interpolating moving least-squares methods for fitting potential energy surfaces: computing high-density potential energy surface data from low-density ab initio data points. J. Chem. Phys. 126, 184108 (2007)._ _ Handley, C. M. & Popelier, P. L. Potential energy surfaces fitted by artificial neural networks. J. Phys. Chem. A 114, 3371u20133383 (2010)._ _ _ Manzhos, S., Wang, X., Dawes, R. & Carrington, T. Jr. A nested molecule-independent neural network approach for high-quality potential fits. J. Phys. Chem. A 110, 5295u20135304 (2006)._ _ _ Blank, T. B., Brown, S. D., Calhoun, A. W. & Doren, D. J. Neural-network models of potential-energy surfaces. J. Chem. Phys. 103, 4129u2013 4137 (1995)._ _ _ Carrasquilla, J. & Melko, R. G. Machine learning phases of matter. Nat. Phys. 13, 431 (2017)._ _ _ van Nieuwenburg Evert, P. L., Liu, Y.-H. & Huber Sebastian, D. Learning phase transitions by confusion. Nat. Phys. 13, 435u2013439 (2017)._ _ Broecker, P., Carrasquilla, J., Melko, R. G. & Trebst, S. Machine learning quantum phases of matter beyond the fermion sign problem. Sci. Rep. 7, 8823 (2017)._ _ Ziatdinov, M. et al. Deep learning of atomically resolved scanning transmission electron microscopy images: chemical identification and tracking local transformations. ACS Nano 11, 12742u2013 12752 (2017)._ _ _ Ziatdinov, M., Maksov, A. & Kalinin, S. V. Learning surface molecular structures via machine vision. npj Comput. Mater. 3, 31 (2017)._ _ Yin, F., Mao, H. J., Hua, L., Guo, W. & Shu, M. S. Back propagation neural network modeling for warpage prediction and optimization of plastic products during injection molding. Mater. Des. 32, 1844u20131850 (2011)._ _ _ Yun, S. Y., Namkoong, S., Rho, J. H., Shin, S. W. & Choi, J. U. A performance evaluation of neural network models in traffic volume forecasting. Math. Comput. Model. 27, 293u2013 310 (1998)._ _ Gupta, V. K. et al. Prediction of capillary gas chromatographic retention times of fatty acid methyl esters in human blood using MLR, PLS and back-propagation artificial neural networks. Talanta 83, 1014u20131022 (2011)._ _ _ Lee, W. Y., Park, G. G., Yang, T. H., Yoon, Y. G. & Kim, C. S. Empirical modeling of polymer electrolyte membrane fuel cell performance using artificial neural networks. Int. J. Hydrog. Energ. 29, 961u2013966 (2004)._ _ _ Belianinov, A. et al. Big data and deep data in scanning and electron microscopies: deriving functionality from multidimensional data sets. Adv. Struct. Chem. Imaging 1, 1u201325 (2015)._ _ Hildebrand, M. Prospects of manipulating diatom silica nanostructure. J. Nanosci. Nanotechnol. 5, 146u2013157 (2005)._ _ _ Sandhage, K. H. et al. Merging biological self-assembly with synthetic chemical tailoring: the potential for 3-D genetically engineered micro/nano-devices (3-D GEMS). Int. J. Appl. Ceram. Technol. 2, 317u2013326 (2005)._ _ _ Burch, M. J. et al. Helium ion microscopy for imaging and quantifying porosity at the nanoscale. Anal. Chem. 90, 1370u2013 1375 (2017)._ _ Ronneberger, O., Fischer, P. & Brox, T. U-net: convolutional networks for biomedical image segmentation. In Proc. International Conference on Medical Image Computing and Computer-Assisted Intervention (eds Navab, N., Hornegger, J., Wells, W. M. & Frangi, A. F.). Vol. 9351, (Springer, Cham, 2015).Hopke, P. K. & Song, X.-H. Classification of single particles by neural networks based on the computer-controlled scanning electron microscopy data. Anal. Chim. Acta 348, 375u2013388 (1997)._ _ _ Zhou, B., Khosla, A., Lapedriza, A., Oliva, A. & Torralba, A. Learning deep features for discriminative localization. In Proc. CVPR 2921u20132929 (2016).Shrestha, R. P. & Hildebrand, M. Evidence for a regulatory role of idatom silicon transporters in cellular silicon responses. Eukaryot. Cell 14, 29u2013 40 (2015)._ _ Steed, C. A., Swan, J. E., Fitzpatrick, P. J. & Jankun-Kelly, T. J. A visual analytics approach for correlation, classification, and regression analysis. In: Innovative approaches of data visualization and visual analytics (eds Mao, L. H. & Weidong, H.) (IGI Global, Hershey, Pennsylvania, USA, 2014) pp. 25u201345.Steed, C. A. et al. Big data visual analytics for exploratory earth system simulation analysis. Comput. Geosci. 61, 71u201382 (2013)._ _ Inselberg, A. The plane with parallel coordinates. Vis. Comput. 1, 69u201391 (1985)._ _ Tesson, B. & Hildebrand, M. Dynamics of silica cell wall morphogenesis in the diatom Cyclotella cryptica: substructure formation and the role of microfilaments. J Struct Biol 169, 62u2013 74 (2010)._ _ _ Van der Walt, S. et al. scikit-image: image processing in Python. PeerJ 2, e453 (2014)._ _ Chollet, F. Keras: the python deep learning library (Astrophysics Source Code Library, 2018). .Javaheri, N. et al. Temperature affects the silicate morphology in a diatom. Sci. Rep. 5, 1u20139 (2015)._ _ \"- - The research was partially conducted at the Center for Nanophase Materials Sciences, which is a DOE Office of Science User Facility. Work by C.A.S. was sponsored by the Department of Energy under the Scientific Discovery through Advanced Computing RAPIDS project. Work by K.A.H. and P.K.U. was enabled through the Oak Ridge High School Math Thesis Program. The research by J.K.M., T.J.M., O.S.O., A.A.P., A.A.T., S.M. and M.H. was sponsored by the Laboratory Directed Research and Development Program of Oak Ridge National Laboratory, managed by UT-Battelle, LLC, for the U.S. Department of Energy. This paper has been authored by UT-Battelle, LLC, under Contract no. DE-AC0500OR22725 with the U.S. Department of Energy. The Department of Energy will provide public access to these results of federally sponsored research in accordance with the DOE Public Access Plan ().- - Author notes - \\'Deceased: Mark Hildebrand\\' - Authors and Affiliations - \"Center for Nanophase Materials Sciences, Oak Ridge National Laboratory, Oak Ridge, TN, 37831, USAArtem A. Trofimov,_Alison A. Pawlicki,_Nikolay Borodinov,_ Maxim A. Ziatdinov,_Anton V. Ievlev,_Alex Belianinov,_Rama Vasudevan_&_Olga S. OvchinnikovaEnvironmental Sciences Division, Oak Ridge National Laboratory, Oak Ridge, TN, 37830, USAShovon Mandal_&_Teresa J. MathewsScripps Institute of Oceanography, University of California, San Diego, La Jolla, CA, 92093, USAMark HildebrandComputer Science and Mathematics Division, Oak Ridge National Laboratory, Oak Ridge, TN, 37830, USAKatherine A. Hausladen,_Paulina K. Urbanowicz_&_ Chad A. SteedBiosciences Division, Oak Ridge National Laboratory, Oak Ridge, TN, 37830, USAJoshua K. Michener\" - \"AuthorsArtem A. TrofimovYou can also search for this author in _Alison A. PawlickiYou can also search for this author in _Nikolay BorodinovYou can also search for this author in _Shovon MandalYou can also search for this author in _Teresa J. MathewsYou can also search for this author in _Mark HildebrandYou can also search for this author in _Maxim A. ZiatdinovYou can also search for this author in _Katherine A. HausladenYou can also search for this author in _Paulina K. UrbanowiczYou can also search for this author in _Chad A. SteedYou can also search for this author in _Anton V. IevlevYou can also search for this author in _Alex BelianinovYou can also search for this author in _Joshua K. MichenerYou can also search for this author in _Rama VasudevanYou can also search for this author in _Olga S. OvchinnikovaYou can also search for this author in _\" - Contributions - A.A.T. and N.B. developed the image processing procedures, constructed and trained both the NNs. A.A.T. analyzed the data and contributed to SEM imaging. N.B. extracted the CAMs. A.A.P. established the SEM imaging procedure and performed the SEM measurements. S.M., T.J.M. and J.K.M. grew diatom cultures and performed gene modification. M.H. obtained knockdown strains for the modification of diatoms. M.A.Z. proposed use of CAMs to highlight regions of interest for NNs. K.A.H., P.K.U. and C.A.S. contributed to the image analysis and data visualization. R.V. contributed to the neural net approach. A.V.I., A.B. and O.S.O. supervised the project and analyzed the data. All authors discussed the results and co-wrote the paper. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License to include the pterosaur data.\" - \"The extant bat framework comprises eight species assigned to one of four guilds: carnivores; u2018harder invertebrate consumers; piscivores; and u2018 softest invertebrate (e.g. moths) consumers (Supplementary Fig._, Supplementary Data_ and and Supplementary Note_). PCA of the 15 texture parameters that significantly differ between guilds (Supplementary Data_ and ) separates guilds in a texture-dietary space defined by PC axes 1 and 2 (Supplementary Fig._ ). PC 1 positively correlates with proportions of total invertebrates, u2018 softest invertebrates, plant matter and dietary generalism, and negatively correlates with total vertebrates and tetrapods. PC 2 positively correlates with dietary proportions of tetrapods (Supplementary Fig._, Supplementary Data_ and Supplementary Note_).\" - Pterosaur dietary reconstructions - \"Projecting pterosaur data into the reptile texture-dietary space plots them within the bounds of the reptiles, and where pterosaur taxa are represented by multiple specimens, there is tendency for them to cluster together rather than exhibit a random distribution across the entire texture-dietary space (e.g. Darwinopterus, Dimorphodon) although some are more broadly distributed (e.g. Dorygnathus, Pterodactylus; Fig._; Supplementary Fig._ and Supplementary Note_). This non-random distribution is noteworthy, given that pterosaur data played no role in structuring the PCA. The independently derived texture-dietary space for bats gives similar results (Supplementary Fig._ and Supplementary Note_). In both analyses, PC 1 explains the majority of microwear variation between reptiles and bats (66.2% and 55% respectively) and is therefore the more informative axis for interpreting and constraining pterosaur diets (this does not negate the usefulness of PC 2 as this axis can provide additional constraints, see below). PC 1 approximates a spectrum from invertebrate dominated diets at one end to vertebrate dominated diets at the other. Pterosaur taxa with values similar to vertebrate consuming guilds in the reptile-based analysis also plot with vertebrate consuming guilds in the bat-based analysis. The same is true for those with PC 1 values similar to invertebrate consuming guilds (Fig._; Supplementary Figs._ and and Supplementary Note_). That these independent analyses produce similar results provides powerful evidence that the differences between pterosaur non-occlusal tooth textures are attributable to dietary differences. This allows us to test previous dietary hypotheses, provide more refined characterisation of the ecological roles that pterosaurs performed within food webs, and explore their dietary evolution. For simplicity we focus here on the reptile-based analysis (see Supplementary Note_ for details of the bat-based analysis).\" - \"Further support for the robustness of our approach comes from consilience with other, well-supported dietary interpretations. While decades of research have yielded little consensus regarding the diet of most pterosaurs, there are a few taxa for which multiple lines of evidence provide more secure hypotheses. For example, multiple specimens of Rhamphorhynchus are preserved in association with fish remains as gut contents, and their sharp conical teeth have been compared to extant piscivores, a dietary hypothesis with which most analyses agree. In our analysis, all but one of the 14 specimens overlap with reptile piscivores along PCs 1 and 2 (Fig._), and the distribution of Rhamphorhynchus corresponds more closely to the convex hull for piscivores than any other group (Fig._ ). Similarly, Istiodactylus has consistently been reconstructed as a piscivore or a carnivore based on its razor-edged, lancet-shaped, interlocking teeth, interpreted as an adaptation for defleshing carcasses. Our DMTA results place Istiodactylus at the extreme end of the vertebrate scale with respect to reptile-based PC 1, plotting closest to a number of reptile carnivore samples (Fig._). This strong concordance between multiple independent lines of evidence gives us confidence in our DMTA results and provides further, quantitative evidence that Rhamphorhynchus was a piscivore and that Istiodactylus was an obligate vertebrate consumer, most likely a carnivore.\" - \"In contrast to the few pterosaur taxa where dietary hypotheses are relatively uncontroversial, DMTA gives us insight into pterosaurs that have poorly constrained diets. Dimorphodon, for example, has been variably interpreted as carnivorous, piscivorous and insectivorous based on comparative anatomy of the skull with extant birds and crocodilians (see ref. and references therein). The high degree of overlap of Dimorphodon tooth textures with reptile carnivores and piscivores across PCs 1 and 2 (Fig._) suggests that the diet of Dimorphodon largely consisted of vertebrates, although some consumption of u2018softer invertebrates cannot be entirely ruled out. Lonchodraco and Serradraco are poorly known pterosaurs whose anatomy and ecology is largely speculative. Serradraco is located towards the extreme end of the reptile carnivore/piscivore scale and overlaps more strongly with reptile piscivores along PC 2 (Fig._). This suggests that Serradraco was predominantly piscivorous. Lonchodraco exhibits a very similar PC 2 value to Serradraco and a more positive value along PC 1 (Fig._) which together suggests the inclusion of some invertebrates within a largely piscivorous diet.\" - \"DMTA results challenge some previous dietary hypotheses. Austriadactylus, for example, has been interpreted as carnivorous based on functional analyses of the closing mechanics of its jaws. However, its position in the texture-dietary space, with high values for PC 1, suggests a diet dominated by invertebrates (Fig._). Coloborhynchus has been interpreted as piscivorous based on its tooth morphology and on theoretical modelling of possible fishing behaviours; in the texture-dietary space its PC 1 value is comparable to carnivores and piscivores but its positive PC 2 value suggests it also consumed a high proportion of u2018 softer invertebrates (Fig._). It is unlikely that Coloborhynchus was a specialist piscivore; our analysis points towards a broader dietary range than has previously been proposed.\" - Sympatric partitioning and ontogenetic dietary shifts - \"Our results also have a bearing on contentious hypotheses of ecological interaction and competition between taxa: did sympatric pterosaurs compete for or partition food resources? DMTA demonstrates niche partitioning between pterosaurs from the Solnhofen Limestones. Rhamphorhynchus and Pterodactylus, taxa with the largest sample sizes, exhibit separation along PC 1 (reptile PCA: tu2009=u2009u2212 2.431, d.f.u2009=u200918, Pu2009=u20090.0257; Fig._; see Supplementary Note_ for similar results in the bat-based analysis), but not along PC 2 (tu2009 =u2009u22121.303, d.f.u2009=u200918, Pu2009=u20090.2087). More negative PC 1 values for Rhamphorhynchus indicate piscivory, while Pterodactylus individuals exhibit a broader range of more positive PC 1 values, indicating invertebrate-dominated, and possibly more generalist diets (Fig._). This agrees with theoretical models of pterosaur feeding behaviours that suggest Pterodactylus had relatively strong bite forces which would have facilitated dietary generalism. It is difficult to determine unequivocal dietary preferences for Solnhofen_species represented by single specimens that fall in the centre of the texture-dietary space, but Scaphognathus and Germanodactylus may have consumed both vertebrates and invertebrates (Fig._), with evidence from the analysis of bat diets suggestive of more invertebrates (Supplementary Fig._). While this might indicate finer niche partitioning between Solnhofen pterosaurs, it is also likely that these species were not typical members of the community as they are relatively rare (<5 specimens in both cases).\" - \"Ontogenetic dietary shifts have been suggested for several pterosaurs but the hypothesis lacks evidential support. In our study, the dietary separation of Rhamphorhynchus specimens along PC 1 correlates with specimen size (reptile PCA; lower jaw length; rsu2009=u2009u22120.6044, Pu2009=u20090.0221; Fig._ and Supplementary Note_). PC 2 values do not correlate with size (rsu2009 =u2009u22120.3451, Pu2009=u20090.2269). Specimens that most likely represent highly immature individuals based on their size and degree of skeletal ossification exhibit positive PC 1 values, indicating invertebrate-dominated diets. Specimens that represent late-stage juveniles and adults exhibit decreasing PC 1 values and thus increasingly piscivorous diets (Fig._). Ontogenetic dietary shifts are common in extant reptiles where offspring usually feed themselves (e.g. ref. ), but are not exhibited by birds. DMTA thus provides direct evidence of ontogenetic dietary shifts in a pterosaur, and supports the hypothesis that pterosaur life-histories were more like those of reptiles than birds or bats.\" - \"Fig. 3: Rhamphorhynchus ontogenetic shifts in the reptile texture-dietary space.Regression of lower jaw length (nu2009=u200914), as a proxy for specimen size, plotted against PC 1 values from the extant reptile texture-dietary space. The solid grey line denotes the line of best fit and grey shading denotes the 95% confidence intervals. Skull diagrams represent scaled examples of an adult, juvenile and hatchling Rhamphorhynchus, with the grey dashed lines denoting their relative size. These diagrams illustrate the size disparity between sampled life-history stages and are not specimen reconstructions of the datum points they are associated with. See Supplementary Fig._ for Rhamphorhynchus ontogenetic shifts in the bat texture-dietary space and the u2018Methods\\' for the skull diagram sources.\" - Pterosaur dietary evolution - \"The implications of our results for pterosaur dietary evolution were explored in two ways. First, by independently projecting pruned, time-calibrated trees from three pterosaur phylogenies, LxFC et al., Andres and Myers, and Wang et al. into the first two axes of the reptile texture-dietary space, creating u2018 phylo-texture-dietary spaces\\' analogous to phylomorphospaces. Second, by mapping pterosaur PC scores from the first two reptile texture-dietary axes onto each tree. The results using the LxFC et al. phylogeny (Fig._, Supplementary Figs._ , and and Supplementary Data_) are presented here as this phylogeny exhibits higher stratigraphic congruence than alternatives (see Supplementary Figs._ and , Supplementary Data_ and Supplementary Note_ for results using the Andres and Myers phylogeny, and Supplementary Figs._ and , Supplementary Data_ and Supplementary Note_ for results using the Wang et al. phylogeny). PC 1 values of the pterosaur specimens included for dietary evolution reconstructions negatively correlate with stratigraphic age (rsu2009=u2009u22120.4859, Pu2009=u2009 0.048) and PC 2 values do not correlate with age (rsu2009=u20090.0552, Pu2009 =u20090.8333). There is a significant phylogenetic signal in pterosaur microwear along PC 1 (Ku2009=u20090.9368, Pu2009=u20090.031, u03BBu2009=u20090.9999, Pu2009=u20090.000143), but no signal along PC 2 (Ku2009=u20090.262, Pu2009 =u20090.775, u03BBu2009=u20090.201, Pu2009=u20090.4014). The ancestral PC 1 value estimate of Pterosauria (node 1, Supplementary Fig._) is 2.2 (Fig._ ), and the ancestral PC 2 estimate is 0.428 (Supplementary Fig._ and Supplementary Data_). This suggests that the ancestral pterosaur diet was invertebrate-dominated. Over time, pterosaurs move through phylo-texture-dietary space to occupy increasingly negative PC 1 values, reflecting a shift towards consumption of more vertebrates. This is most pronounced in members of Istiodactylidae and Lonchodectidae (nodes 13 and 16 respectively, Supplementary Fig._), which have estimated ancestral PC 1 values of u22122.137 and u22122.007, respectively (Supplementary Data_ ) and independently evolved into obligate vertebrate consumers (Fig._). Interestingly, the invertebrate-dominated diet of Pterodactylus and the possible inclusion of invertebrates in the diet of Lonchodraco are both secondarily derived (Fig._ ).\" - \"Fig. 4: Pterosaur dietary evolution.a Phylo-texture-dietary space of pterosaur microwear from projecting a time-calibrated, pruned tree from LxFC et al. onto the first two PC axes of the extant reptile texture-dietary space. b Ancestral character-state reconstruction of pterosaur dietary evolution from mapping pterosaur PC 1 values onto a time-calibrated, pruned tree from LxFC et al.. To account for ontogenetic changes in diet, only the largest specimen of respective pterosaur taxa, identified by lower jaw length, were included. Pterosaur symbols same as Fig._. Skull diagrams of well-preserved pterosaurs not to scale (see u2018 Methods\\' for sources). Phylo-texture-dietary space adapted from Fig._ of ref. () by Jordan Bestwick under a Creative Commons Attribution 4.0 International License to remove the extant reptile datum points and to include the pterosaur data and phylogeny.\" - These results provide quantitative evidence that pterosaurs initially evolved as invertebrate consumers before expanding into piscivorous and carnivorous niches. The causes of this shift towards vertebrate-dominated diets require further investigation, but might reflect ecological interactions with other taxa that radiated through the Mesozoic. Specifically, competition with birds, which first appeared in the Upper Jurassic and diversified in the Lower Cretaceous, has been invoked to explain the decline of small-bodied pterosaurs, but this hypothesis is controversial. DMTA provides an opportunity for testing hypotheses of competitive interaction upon which resolution of this ongoing debate will depend. - \"In summary, our analyses provide quantitative evidence of pterosaur diets, revealing that dietary preferences ranged across consumption of invertebrates, carnivory and piscivory. This has allowed us to explicitly constrain diets for some pterosaurs, enabling more precise characterisations of pterosaurs roles within Mesozoic food webs and providing insight into pterosaur niche partitioning and life-histories. Our study sets a benchmark for robust interpretation of extinct reptile diets through DMTA of non-occlusal tooth surfaces and highlights the potential of the approach to enhance our understanding of ancient ecosystems.\"- - Specimen material - \"Tooth microwear textures were sampled from 17 pterosaur genera. Eight bat species were sampled and the microwear texture data of 13 reptile species, comprising six crocodilians and seven monitor lizards, from ref. were included to serve as extant multivariate frameworks. Crocodilians and monitor lizards were chosen to provide the primary framework because their dentitions have been compared with those of pterosaurs, and they often forage on the margins of terrestrial and aquatic environments, comparable to hypothesised pterosaur foraging behaviours. Bats were chosen as an independent comparator framework because they are the only extant group of dentulous flying vertebrates, and their foraging behaviours have sometimes been extrapolated to pterosaurs (see ref. and references therein). Extant and fossil specimens were sampled from the Bayerische Staatsammlung fxFC r PalxE4ontologie und Geologie, Munich, Germany (BSPG); Field Museum of Natural History, Chicago, USA (FMNH); British Geological Survey, British Geology Collections, Keyworth, UK (GSM); Institute of Vertebrate Paleontology and Paleoanthropology, Beijing, China (IVPP); Grant Museum of Zoology, University College London, UK (LDUCZ); Museum fxFCr Naturkunde der Humboldt-UniversitxE4t Berlin, Berlin, Germany (MB); Natural History Museum, London, UK (NHMUK); University of Oxford Museum of Natural History, Oxford, UK (OUMNH); Paleontological Museum of Liaoning, Shenyang, China (PMOL); Staatliches Museum fxFCr Naturkunde, Karlsruhe, Germany (SMNK); Staatliches Museum fxFCr Naturkunde, Stuttgart, Germany (SMNS); Teylers Museum, Haarlem, Netherlands (TM); Royal Tyrrell Museum of Palaeontology, Drumheller, Canada (TMP); Florida Museum of Natural History, Gainesville, USA (UF); and the National Museum of Natural History, Smithsonian Institute, Washington DC, USA (USNM). See Supplementary Data_ for the complete specimen list.\" - Dietary guild assignments - \"Reptiles and bats were selected to include taxa with well-constrained dietary differences determined from stomach and/or faecal content studies (Supplementary Data_). Studies were chosen that met as many of the following criteria as possible: representative sample sizes, dietary compositions represented as volumetric data (or frequency data at an absolute minimum); and spatial proximity of the dietary study to the location(s) from which the museum specimens we analysed were collected (specimen provenance data, where known, is included in Supplementary Data_). Taxa that had not been subjected to ecological studies that provided volumetric or frequency breakdowns of diet were not sampled for study. Many reptiles exhibit ontogenetic shifts in diet, thus similarly sized specimens from extant species were sampled where possible to minimise effects of this confounding variable. Lower jaw lengths, measured in lateral view, were used as a proxy for overall body size. For extant reptiles, lower jaw lengths were measured from the anterior tip of the dentary to the posterior margin of the surangular. Bat jaw lengths were measured from the anterior to posterior tip of the dentary. Specimen availability enabled Crocodylus porosus to be sampled as two separate life-history groups for analysis. Specimens with lower jaw lengths of less than 50u2009cm were classified as juveniles; specimens with jaw lengths exceeding 50u2009cm were classified as adults.\" - \"Extant reptiles and bats were assigned to dietary guilds based on the relative u2018intractability (roughly equivalent to hardness) of prey as food. Guilds include carnivores (tetrapod consumers); piscivores (fish consumers); u2018harder invertebrate consumers (invertebrates with hard exoskeletons, e.g. beetles, crustaceans and shelled gastropods); u2018softer invertebrate consumers (invertebrates with less hard exoskeletons, e.g. crickets, grasshoppers, dragonflies, damselflies and ants); u2018softest invertebrate consumers (invertebrates with soft exoskeletons, e.g. invertebrate larvae, butterflies, moths, arachnids and millipedes); omnivores (combination of plant and animal matter). These guilds do not represent the entire dietary diversity of the extant clades, but were selected because of their ecological relevance to pterosaur diet. For example, many New World bat species are nectarivores, sanguivores (blood feeders) or obligate frugivores. These diets have never been proposed for dentulous pterosaurs. In reptiles, Crocodylus porosus adults (saltwater crocodile, nu2009=u20096), Varanus komodoensis (Komodo dragon, nu2009=u2009 4), Varanus nebulosus (clouded monitor, nu2009=u200911), Varanus rudicollis (roughneck monitor, nu2009=u20098), and Varanus salvator (Asian water monitor, nu2009=u20098) were assigned to the carnivore guild (total nu2009=u2009 37); Crocodylus acutus (American crocodile, nu2009=u20097) and Crocodylus porosus juveniles (nu2009=u20095) were assigned to the u2018harder invertebrate consumer guild (total nu2009=u200912); Varanus niloticus (Nile monitor, nu2009 =u20098) and Varanus prasinus (emerald tree monitor, nu2009=u20097) were assigned to the u2018softer invertebrate consumer guild (total nu2009=u2009 15); Varanus olivaceus (Grays monitor, nu2009=u20096) was assigned to the omnivore guild (total nu2009=u20096); and Alligator mississippiensis (American alligator, nu2009=u20098), Caiman crocodilus (spectacled caiman, nu2009=u2009 6), Crocodylus niloticus (Nile crocodile, nu2009=u20094), and Gavialis gangeticus (gharial, nu2009=u20097) were assigned to the piscivore guild (total nu2009 =u200925). In bats, Trachops cirrhosis (fringe-lipped bat, nu2009=u20098) and Vampyrum spectrum (spectral bat, nu2009=u20099) were assigned to the carnivore guild (total nu2009=u200917); Nyctalus noctula (noctule bat, nu2009=u2009 7) and Rhinolophus ferrumequinum (greater horseshoe bat, nu2009=u20098) were assigned to the u2018harder invertebrate consumer guild (total nu2009 =u200915); Myotis capaccini (long-fingered bat, nu2009=u20097), Myotis mystacinus (whiskered bat, nu2009=u20096), and Plecotus auritus (brown long-eared bat, nu2009=u20096) were assigned to the u2018softest invertebrate consumer guild (total nu2009=u200919); and Noctilio leporinus (greater bulldog bat, nu2009=u20098) was assigned to the piscivore guild (total nu2009=u20098). See Fig._ for an overview of how reptiles and bats were assigned to guilds. The Cr. acutus skull diagram in Fig._ was drawn from UF 54201, the G. gangeticus diagram from Wikimedia Commons under the Creative Commons Attribution-Share Alike 2.0 Generic license (), and the V. olivaceus diagram from UF 57207.\" - Sampling strategy - \"Pterosaur specimens were cleaned before sampling using an ethaline solvent gel, produced, and applied according to Williams and Doyle. Extant reptile and bat teeth from dry skeletal specimens were cleaned using 70% ethanol-soaked cotton swabs to remove dirt and consolidant. Microwear data were acquired from non-occlusal (non-chewing) labial surfaces, as close to the tooth apex as possible. In extant reptiles, the mesial-most dentary tooth was sampled; in pterosaurs, the mesial-most tooth of the premaxilla or dentary tooth, based on preservation quality, was sampled. No preference was given to the left or right tooth in extant reptiles and teeth were pooled in analyses. Wear facets that likely formed from toothu2013 tooth occlusion from the opening and closing of jaws, characterised by their vertical orientation, elliptical shape and parallel features, were not sampled. Bat microwear data were acquired from the non-occlusal labial surface of the canine in the dentary, as close to the apex as possible. Canines were sampled over premolars and molars because they represent closer functional analogues to reptile teeth since they are not used for chewing. No preference was given to the left or right canine. To test for ontogenetic dietary shifts in Rhamphorhynchus, lower jaws were measured in lateral view from the anterior tip of the dentary to the posterior margin of the surangular. Lower jaw length is a reliable proxy for overall specimen size as Rhamphorhynchus skulls exhibit strongly linear size relationships with respect to other anatomical structures, such as the humerus and wing finger, and we follow this analysis in our use of u2018adult , u2018juvenile and u2018hatchling. The example adult and hatchling Rhamphorhynchus skull diagrams in Fig._ were redrawn from Bennett, and the example juvenile diagram was traced from ref. under a Creative Commons Attribution 4.0 International License (). High fidelity moulds were taken of teeth using President Jet Regular Body polyvinylsiloxane (ColtxE8ne/Whaledent Ltd., Burgess Hill, West Sussex UK). Initial moulds taken from each specimen were discarded to remove any remaining dirt and all analyses were performed on second moulds. Casts were made from these moulds using EpoTek 320 LV Black epoxy resin mixed to the manufacturers instructions. Resin was cured for 24u2009h under 200u2009kPa (2 Bar/30u2009psi) of pressure (Protima Pressure Tank 10L) to improve casting quality. Small casts were mounted onto 12.7u2009mm SEM stubs using President Jet polyvinylsiloxane with the labial, non-occluding surfaces orientated dorsally to optimise data acquisition. All casts were sputter coated with gold for three minutes (SC650, Bio-Rad, Hercules, CA, USA) to optimise capture of surface texture data. Replicas produced using these methods are statistically indistinguishable from original tooth surfaces.\" - Surface texture data acquisition - \"Surface texture data acquisition follows standard laboratory protocols. Data were captured using an Alicona Infinite Focus microscope G4b (IFM; Alicona GmbH, Graz, Austria; software version 5.1), using a xD7100 objective lens, producing a field of view of 146u2009xD7u2009100u2009u03BCm. Lateral and vertical resolution were set at 440 and 20u2009nm, respectively. Casts were orientated so labial surfaces were perpendicular to the axis of the objective lens.\" - \"All 3D data files were processed using Alicona IFM software (version 5.1) to remove dirt particles from tooth surfaces and anomalous data points (spikes) by manual deletion. Data were levelled (subtraction of least-squares plane) to remove variation caused by differences in tooth surface orientation at the time of data capture. Files were exported as_.sur files and imported into Surfstand (software version 5.0.0 Centre for Precision Technologies, University of Huddersfield, West Yorkshire, UK). Scale-limited surfaces were generated through application of a fifth-order robust polynomial to remove gross tooth form and a robust Gaussian filter (wavelength u03BBcu2009=u20090.025u2009mm). ISO 25178-2 areal texture parameters were then generated from each scale-limited surface. Descriptions of ISO parameters can be found in Supplementary Data_ (from ref. ).\" - DMTA statistical analyses - \"Log-transformed texture data were used for analyses as some of the texture parameters were non-normally distributed (Shapiro-Wilk, Pu2009>u20090.05). The parameter Ssk was excluded from analysis as it contains negative values and thus cannot be log-transformed. To test the hypotheses that microwear differs between reptiles and bats from different dietary guilds, analysis of variance (ANOVA) with pairwise testing (Tukey HSD) was used for each texture parameter. Where homogeneity of variance tests revealed evidence of unequal variances, Welch analysis of variance was used. Principal component analysis (PCA) was used to explore texture parameters exhibiting significant differences between reptile and bat dietary guilds. ANOVA with pairwise testing was used on the PC axis 1 and 2 values of dietary guilds to determine whether guilds occupy different areas of multivariate space along these axes (see ref. for results for reptiles). To test the hypotheses that reptile and bat microwear differences are determined by dietary differences, we used Spearmans rank to test for correlations between PC axes 1 and 2 and dietary characteristics. Additional analyses were employed to independently test for subtle microtextural differences between dietary guilds. Texture parameters were ranked by guild based on the average value for the guild; matched pairs t-tests were used to compare the profiles of average parameter values between guilds (see ref. for results for reptiles, and Supplementary Tables_ and for results for bats).\" - \"A Benjaminiu2013Hochberg (Bu2013H) procedure was used to account for the possibility of inflated Type I error rates associated with multiple comparisons. The false discovery rate was set at 0.05. The Bu2013H procedure was not needed for the Tukey HSD tests as it already accounts for inflated Type I error rates.\" - \"Pterosaur microwear data were independently projected onto the axes of the reptile and bat analyses. Correlation of Rhamphorhynchus lower jaw lengths with PC 1 values were tested using Spearmans rank tests. All DMTA analyses were performed with JMP Pro 12 (SAS Institute, Cary, NC, USA) except for the Bu2013 H procedure, which used Microsoft Excel ().\" - Pterosaur dietary evolution - \"Pterosauria is defined as the most recent common ancestor of Preondactylus buffarinii and Quetzalcoatlus northropi and all its descendants. Several competing phylogenies exist for Pterosauria which show similar, but disputed, taxonomic contents of clades and grades of morphologically similar taxa. We therefore used three independent phylogenies, LxFC et al., Andres and Myers and Wang et al., to visualise reconstructed pterosaur diets from DMTA in evolutionary contexts. These phylogenies have extensive taxonomic sampling across Pterosauria and contain the majority of taxa in this study. See Supplementary Note_ for Andres and Myers and Wang et al. results. A time-calibrated tree was constructed from each phylogeny using the DatePhylo function of the R package strap, version 1.4 (ref. ) (R version 3.2.3 (ref. )). First and last appearance dates were obtained from Dean et al. and the Paleobiology Database (PBDB: ). Branches were scaled using the u2018equal method which eliminates zero-length branches by sharing duration from the more basal non-zero-length branches. Time-scaled trees were pruned in R to include only pterosaurs subjected to DMTA. Pruning has no effect on the topological relationships of remaining pterosaurs. Missing pterosaurs (i.e. species included in the DMTA dataset but not in the phylogenetic hypothesis) were included by hand prior to time-scaling based on information from the literature on their likely phylogenetic position (e.g. assignments to particular clades in published systematic assessments). To minimise confounding effects of ontogeny, only the largest specimen of each pterosaur taxa was included in the analysis, identified by lower jaw length in lateral view from the anterior tip of the dentary to the posterior margin of the surangular. Each tree was independently projected onto the first two PC axes of the reptile texture-dietary space to generate pterosaur phylo-texture-dietary spaces using the phylomorphospace function of the R package phytools, version 0.6-60 (ref. ). Phylogenetic signals in pterosaur microwear along PCs 1 and 2 were tested using the phylosig function in phytools. Pterosaur PC 1 and PC 2 values were compared to time from the root of the time-calibrated trees in millions of years to test for shifts through phylo-texture-dietary space over evolutionary time. Ancestral character-state reconstructions of pterosaur diets were performed by independently mapping pterosaur PC 1 and PC 2 values as continuous characters onto each pruned tree using the contMap function in phytools. Ancestral PC 1 and PC 2 values (and the variance and 95% confidence intervals) were calculated for each node within the three phylogenies. Pterosaur skull diagrams in Figs._ and u2013 and Supplementary Figs._ and u2013 were traced from ref. under a Creative Commons Attribution 4.0 International License () except Anhanguera, Austriadactylus, Boreopterus, Coloborhynchus and Jiangchangnathus, which were redrawn from refs. , respectively.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - All the data analysed in this study are publically available in Zenodo () and within the Supplementary Information files. - \\' \\'- - \\' \\' - \"The code used for the dietary evolution reconstructions can be found in the R file Supplementary Code_ and is available in Zenodo ().\" - \\' \\'- - \"Witton, M. P. Pterosaurs: Natural History, Evolution, Anatomy (Princeton Univ. Press, 2013).Barrett, P. M., Butler, R. J., Edwards, N. P. & Milner, A. R. Pterosaur distribution in time and space: an atlas. Zitteliana 28, 61u2013107 (2008)._ Dean, C. D., Mannion, P. D. & Butler, R. J. Preservational bias controls the fossil record of pterosaurs. Palaeontology 59, 225u2013247 (2016)._ _ _ _ Bestwick, J., Unwin, D. M., Butler, R. J., Henderson, D. M. & Purnell, M. A. Pterosaur dietary hypotheses: a review of ideas and approaches. Biol. Rev. 93, 2021u20132048 (2018)._ _ _ Wang, X. & Zhou, Z. Pterosaur assemblages of the Jehol Biota and their implication for the Early Cretaceous pterosaur radiation. Geol. J. 41, 405u2013418 (2006)._ _ Bennett, S. C. A statistical study of Rhamphorhynchus from the Solnhofen limestone of Germany: year-classes of a single large species. J. Paleontol. 69, 569u2013580 (1995)._ _ Prondvai, E., Stein, K., u0150si, A. & Sander, M. P. Life history of Rhamphorhynchus inferred from bone histology and the diversity of pterosaurian growth strategies. PLoS ONE 7, e31392 (2012)._ _ _ _ _ _ Bennett, S. C. New smallest specimen of the pterosaur Pteranodon and ontogenetic niches in pterosaurs. J. Vert. Paleontol. 92, 254u2013 271 (2017)._ _ Benson, R. B., Frigot, R. A., Goswami, A., Andres, B. & Butler, R. J. Competition and constraint drove Copes rule in the evolution of giant flying reptiles. Nat. Comm. 5, 3567 (2014)._ _ _ _ Nesbitt, S. J. The early evolution of archosaurs: relationships and the origin of major clades. Bull. Am. Mus. Nat. Hist. 352, 1u2013292 (2011)._ _ Witton, M. P. in New Perspectives on Pterosaur Palaeobiology (eds. Hone, D. W. E., Witton, M. P. & Martill, D. M.) Vol. 455, 7u201323 (Geol. Soc. Lond. Spec. Pub., 2018).u0150 si, A. Feeding-related characters in basal pterosaurs: implications for jaw mechanism, dental function and diet. Lethaia 44, 136u2013152 (2011)._ _ Zhou, C.-F., Gao, K.-Q., Yi, H., Xue, J., Li, Q. & Fox, R. C. Earliest filter-feeding pterosaur from the Jurassic of China and ecological evolution of Pterodactyloidea. R. Soc. Open Sci. 4, 160672 (2017)._ _ _ _ _ Ungar, P. S., Brown, C. A., Bergstrom, T. S. & Walker, A. Quantification of dental microwear by tandem scanning confocal microscopy and scale-sensitive fractal analysis. Scanning 25, 185u2013193 (2003)._ _ _ Daegling, D. J. et al. Viewpoints: feeding mechanics, diet and dietary adaptations in early hominins. Am. J. Phys. Anthropol. 151, 356u2013371 (2013)._ _ _ Purnell, M., Seehausen, O. & Galis, F. Quantitative three-dimensional microtextural analyses of tooth wear as a tool for dietary discrimination in fishes. J. R. Soc. Interface 9, 2225u20132233 (2012)._ _ _ _ Purnell, M. A. & Darras, L. P. G. 3D tooth microwear texture analysis in fishes as a test of dietary hypotheses of durophagy. Surf. Topogr. Metrol. Prop. 4, 014006 (2016)._ _ _ Bestwick, J., Unwin, D. M. & Purnell, M. A. Dietary differences in archosaur and lepidosaur reptiles revealed by dental microwear textural analysis. Sci. Rep. 9, 11691 (2019)._ _ _ _ _ _ ISO 25178-2. Geometrical Product Specifications (GPS)u2014Surface Texture: Arealu2014Part 2: Terms, Definitions and Surface Texture Parameters (International Organisation for Standardisation) 1u2013 42 (2012).Winkler, D. E., Schulz-Kornas, E., Kaiser, T. M. & TxFCtken, T. Dental microwear texture reflects dietary tendencies in extant Lepidosauria despite their limited use of oral food processing. Proc. R. Soc. B. 286, 20190544 (2019)._ _ _ _ Frey, E. & Tischlinger, H. The Late Jurassic pterosaur Rhamphorhynchus, a frequent victim of the ganoid fish Aspidorhynchus? PLoS ONE 7, e31945 (2012)._ _ _ _ _ _ Wellnhofer, P. Die Rhamphorhynchoidea (Pterosauria) der Oberjura-Plattenkalke SxFCddeutschlands, Teil III: PalxF6kologie und Stammesgeschichte. Palaeontogr. Abt. A 1u20133, 1u201330 (1975).Hone, D., Henderson, D. M., Therrien, F. & Habib, M. B. A specimen of Rhamphorhynchus with soft tissue preservation, stomach contents and a putative coprolite. PeerJ 3, e1191 (2015)._ _ _ _ Wellnhofer, P. The Illustrated Encyclopaedia of Pterosaurs (Salamander Books, 1991).Witton, M. P. New insights into the skull of Istiodactylus latidens (Ornithocheiroidea, Pterodactyloidea). PLoS ONE 7, e33170 (2012)._ _ _ _ _ _ Dalla Vecchia, F. M. in Anatomy, Phylogeny and Palaeobiology of Early Archosaurs and their Kin (eds. Nesbitt, S.J., Desojo, J.B. & Irmis, R.B.) 379, 119u2013155 (Geol. Soc. Lond. Spec. Pub., 2013).Veldmeijer, A. J., Signore, M. & Bucci, E. in Predation in Organismsu2014A Distinct Phenomenon (ed. Elewa, A.M.T.) 295u2013308 (Springer, 2007).Unwin, D. M. & Martill, D. M. in The Crato Fossil Beds of Brazil: Window into an Ancient World (eds. Martill, D. M., Bechly, G. & Loveridge, B.) 475u2013 524. (Cambridge Univ. Press, 2007).Henderson, D. M. in New Perspectives on Pterosaur Palaeobiology (eds. Hone, D. W. E., Witton, M. P. & Martill, D. M.) 455, 25u2013 44 (Geol. Soc. Lond. Spec. Pub., 2018).Unwin, D. M. The Pterosaurs from Deep Time (Pi Press, 2006).Hoffmann, R., Bestwick, J., Berndt, G., Fuchs, D. & Klug, C. Pterosaurs ate soft-bodies cephalopods (Coleoidea). Sci. Rep. 10, 1230 (2020)._ _ _ _ _ _ Platt, S. G., Thorbjarnarson, J. B., Rainwater, T. R. & Martin, D. R. Diet of the American Crocodile (Crocodylus acutus) in marine environments of Coastal Belize. J. Herpet 47, 1u201310 (2013)._ _ LxFC, J., KundrxE1t, M. & Shen, C. New material of the pterosaur Gladocephaloideus LxFC et al., 2012 from the Early Cretaceous of Liaoning Province, China, with comments on its systematic position. PLoS ONE 11, e0154888 (2016)._ _ _ _ _ Andres, B. & Myers, T. S. Lone star pterosaurs. Earth Env. Sci. Trans. R. Soc. Edinburgh. 103, 383u2013398 (2013)._ _ Wang, X. et al. New evidence from China for the nature of the pterosaur evolutionary transition. Sci. Rep. 7, 42763 (2017)._ _ _ _ _ _ Andres, B. Death and taxa: the shape of pterosaur evolution. In The 5th International Symposium on Pterosaurs (Univ. Portsmouth, UK, 2015).Wang, M., Wang, X., Wang, Y. & Zhou, Z. A new basal bird from China with implications for morphological diversity in early birds. Sci. Rep. 6, 19700 (2016)._ _ _ _ _ _ Losos, J. B. & Greene, H. W. Ecological and evolutionary implications of diet in monitor lizards. Biol. J. Linn. Soc. 35, 379u2013409 (1988)._ _ Nifong, J. C. et al. Animal-borne imaging reveals novel insights into the foraging behaviors and diel activity of a large-bodied apex predator, the American alligator (Alligator mississippiensis). PLoS ONE 9, e83953 (2014)._ _ _ _ _ _ Chatterjee, S. & Templin, R. J. Posture, Locomotion, and Paleoecology of Pterosaurs. Vol. 376, 1u201364 (Geol. Soc. Am. Spec. Pub, 2004)..Delany, M. F., Linda, S. B. & Moore, C. T. Diet and condition of American alligators in 4 Florida lakes. Proc. Annual Conference on Southeastern Association of Fish and Wildlife Agencies (eds Eversole, R. G., Wong, K. C., Chamberlain, M. & Rohwer, F.) Vol. 53, 375u2013389 (1999).Laverty, T. M. & Dobson, A. P. Dietary overlap between black caimans and spectacled caimans in the Peruvian Amazon. Herpetologica 69, 91u2013101 (2013)._ _ Thorbjarnarson, J. B. The status and ecology of the American crocodile in Haiti. Bull. Fla. Mus. Biol. Sci. 33, 1u201386 (1988)._ Wallace, K. M. & Leslie, A. J. Diet of the Nile Crocodile (Crocodylus niloticus) in the Okavango Delta. Botsw. J. Herpet. 42, 361u2013368 (2008)._ _ Sah, S. A. M. & Stuebing, R. B. Diet, growth and movements of juvenile crocodiles Crocodylus porsus Schneider in the Klias river, Sabah, Malaysia. J. Trop. Ecol. 12, 651u2013 662 (1996)._ _ Taylor, J. A. The foods and feeding habits of subadult Crocodylus porosus Schneider in Northern Australia. Aust. Wildl. Res. 6, 347u2013359 (1979)._ _ _ Hussain, S. A. Ecology of Gharial (Gavialis gangeticus) in National Chambal Sanctuary, India. MPhil thesis, Aligarh Muslim University (1991).Auffenberg, W. The Behavioral Ecology of the Komodo Monitor (Univ. Press, Florida, 1981)._ Dalhuijsen, K., Branch, W. R. & Alexander, G. J. A comparative analysis of the diets of Varanus albigularis and Varanus niloticus in South Africa. Afr. Zool. 49, 83u201393 (2014)._ _ Auffenberg, W. Gray s Monitor Lizard (Univ. Press, Florida, 1988)._ Greene, H. W. Diet and arboreality in the emerald monitor, Varanus prasinus, with comments on the study of adaptation. Fieldiana Zool. 31, 1u201312 (1986)._ Rahman, K. M. M., Rakhimov, I. I. & Khan, M. M. H. Activity budgets and dietary investigations of Varanus salvator (Reptilia: Varanidae) in Karamjal ecotourism spot of Bangladesh Sundarbans mangrove forest. Basic Appl. Herpet 31, 45u201356 (2017)._ Almenar, D., Aihartza, J., Goiti, U., Salsamendi, E. & Garin, I. Diet and prey selection in the trawling long-fingered bat. J. Zool. 274, 340u2013348 (2008)._ _ Berge, L. Resource Partitioning between the Cryptic Species Brandts bat (Myotis brandtii) and the Whiskered Bat (M. mystacinus) in the UK. PhD thesis, Univ. Bristol (2007).Brooke, A. P. Diet of the fishing bat, Noctilio leporinus (Chiroptera: Noctilionidae). J. Mammal. 75, 212u2013218 (1994)._ _ Jones, G. Flight peformance, echolocation and foraging behaviour in noctule bats Nyctalus noctula. J. Zool. Soc. Lond. 237, 303u2013312 (1995)._ _ Rydell, J. Food habits of northern (Eptesicus nilssoni) and brown long-eared (Plecotus auritus) bats in Sweden. Holarct. Ecol. 12, 16u2013 20 (1989)._ Flanders, F. & Jones, G. Roost use, ranging behavior, and diet of greater horseshoe bats (Rhinolophus ferrumequinum) using a transitional roost. J. Mammal. 90, 888u2013896 (2009)._ _ Humphrey, S. R., Bonaccorso, F. J. & Zinn, T. J. Guild structure of surface-gleaning bats in PanamxE1. Ecology 64, 284u2013294 (1983)._ _ Bonato, V., Facure, K. G. & Uidea, W. Food habits of bats of subfamily Vampyrinae in Brazil. J. Mammal. 85, 708u2013713 (2004)._ _ Fukuda, Y., Saafeld, K., Lindner, G. & Nichols, T. Estimation of total length from head length of saltwater crocodiles (Crocodylus porosus) in the Northern Territory, Australia. J. Herpetol. 47, 34u201340 (2013)._ _ Santana, S. E., Dumont, E. R. & Davis, J. L. Mechanics of bite force production and its relationship to diet in bats. Funct. Ecol. 24, 776u2013784 (2010)._ _ Williams, V. S. & Doyle, A. M. Cleaning fossil tooth surfaces for microwear analysis: use of solvent gels to remove resistant consolidant. Palaeontol. Electron. 13, 2T (2010)._ Schubert, B. W. & Ungar, P. S. Wear facets and enamel spalling in tyrannosaurid dinosaurs. Acta Palaeontol. Pol. 50, 93u201399 (2005)._ Goodall, R. H., Darras, L. P. & Purnell, M. A. Accuracy and precision of silicon based impression media for quantitative areal texture analysis. Sci. Rep. 5, 10800 (2015)._ _ _ _ _ Purnell, M. A., Crumpton, N., Gill, P. G., Jones, G. & Rayfield, E. J. Within-guild dietary discrimination from 3-D textural analysis of tooth microwear in insectivorous mammals. J. Zool. 291, 249u2013257 (2013)._ _ _ Schulz, E., Calandra, I. & Kaiser, T. M. Feeding ecology and chewing mechanics in hoofed mammals: 3D tribology of enamel wear. Wear 300, 169u2013179 (2013)._ _ _ Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. B. 57, 289u2013300 (1995)._ _ _ Purnell, M. A., Goodall, R. H., Thomson, S. & Matthews, C. J. D. Tooth microwear texture in odontocete whales: variation with tooth characteristics and implications for dietary analysis. Biosurf. Biotribol. 3, 184u2013195 (2017)._ _ McDonald, J. H. Handbook of Biological Sciences 3rd edn. (Sparky House Publishing, 2014).Bell, M. A. & Lloyd, G. T. strap: an R package for plotting phylogenies against stratigraphy and assessing their stratigraphic congruence. Palaeontology 58, 379u2013389 (2014)._ _ R Development Core Team. R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing, Vienna, Austria, 2010).Revell, L. J. Phytools: an R package for phylogenetic comparative biology (and other things). Methods Ecol. Evol. 3, 217u2013223 (2012)._ _ Bantim, R. A. M., Saraiva, A. A. F. & SayxE3o, J. M. Skull variation and the shape of the sagittal premaxillary crest in anhanguerid pterosaurs (Pterosauria, Pterodactyloidea) from the Araripe Basin, Northeast Brazil. Hist. Biol. 27, 656u2013664 (2014)._ _ Dalla Vecchia, F. M. The first Italian specimen of Austriadactylus cristatus (Diapsida, Pterosauria) from the Norian (Upper Triassic) of the carnic Prealps. Riv. Ital. Paleontol. S. 115, 291u2013304 (2009)._ Jiang, S.-X., Wang, X.-L., Meng, X. & Cheng, X. A new boreopterid pterosaur from the Lower Cretaceous of western Liaoning, China, with a reassessment of the phylogenetic relationships of the Boreopteridae. J. Paleontol. 88, 823u2013 828 (2014)._ _ Martill, D. M. First occurrence of the pterosaur Coloborhynchus (Pterosauria, Ornithocheiridae) from the Wessex Formation (Lower Cretaceous) of the Isle of Wight, England. Proc. Geol. Assoc. 126, 377u2013 380 (2015)._ _ Cheng, X., Wang, X., Jiang, S. & Kellner, A. W. A. A new scaphognathid pterosaur from western Liaoning. China Hist. Biol. 24, 101u2013111 (2012)._ _ Bestwick, J., Unwin, D. M., Butler, R. J. & Purnell, M. A. Dietary diversity and evolution of the earliest flying vertebrates revealed by dental microwear texture analysis. Data Sets. Zenodo (2020)._ _ \"- - Thanks to P. Campbell, M. Carnall, T. Davidson, E. Frey, S. Harris, J. Hay, S.-X. Jiang, D. Lunde, R. Miguez, O. Rauhut, A. Resetar, R. Schoch, T. Schossleitner, A. Schulp, C. Sheehy, L. Steel, B. Strilisky, A. Wynn and C.-F. Zhou for specimen access. Thanks to S. Gabbott and D. Henderson for comments. This work was funded by an NERC studentship awarded through the Central England NERC Training Alliance (CENTA; grant reference NE/L002493/1) and the University of Leicester, and by a Palaeontological Association Sylvester-Bradley Award (PA-SB201701) to J.B. J.B. was also supported by a Leverhulme Trust Research Project Grant (RPG-2019-364) during the completion of the project.- - Authors and Affiliations - \"Centre for Palaeobiology Research, School of Geography, Geology and the Environment, University of Leicester, Leicester, LE1 7RH, UKJordan Bestwick_&_Mark A. PurnellSchool of Geography, Earth and Environmental Sciences, University of Birmingham, Edgbaston, Birmingham, B15 2TT, UKJordan Bestwick_&_Richard J. ButlerCentre for Palaeobiology Research, School of Museum Studies, University of Leicester, Leicester, LE1 7RF, UKDavid M. Unwin\" - \"AuthorsJordan BestwickYou can also search for this author in _David M. UnwinYou can also search for this author in _Richard J. ButlerYou can also search for this author in _Mark A. PurnellYou can also search for this author in _\" - Contributions - M.A.P. and D.M.U. conceived the study. All authors contributed to the analytical design. J.B. collected the data. J.B. analysed the data with input from M.A.P. and R.J.B. J.B. wrote the paper with contributions from all authors. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks Ivan Calandra, Danielle Fraser and Eberhard Frey for their contribution to the peer review of this work. Peer reviewer reports are available. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License, ); (2) Pucci et al. (ref, free access available at ; (3) Villani et al. (ref, free access available at: ), water discharge and geochemical measure points, tracer release and measure points. (b) Hydrogeological cross section with projection of earthquakes from a 4_km wide stripe. The green dashed line represents the vertical deformation (VD) recorded by DInSAR data (Open access Creative Commons CC BY license, ; ). The bar above the cross section indicates the amount of horizontal deformation (HD) recorded by DInSAR data. (c) Longitudinal hydrogeological section along the central basin with projection of earthquakes from a 7_km wide stripe showing that apart from surface co-seismic ruptures, earthquakes propagated up to shallow depths, where groundwater is hosted, in a widespread area. The discharge measure points (yellow diamond) acronyms are reported in Table .\" - \"The tectonic setting deeply influences groundwater flow. The Sibillini Mountains Thrust (SMth) is the main contractional tectonic structure and separates the carbonate Sibillini Mountains domain from the eastern terrigenous turbiditic deposits (Terrigenous Units complex, TU, Fig._a). The two main normal fault systems of the area are the NNW-SSE trending SW-dipping Vettore Bove Fault System (VBFS) and the Nottoria Preci Fault system (NPFS) and their antithetic segments (Fig._) which possess a cumulated stratigraphic throw of up to 1000_m each.\" - \"The 2016u20132017 sequence included more than 9 events with Mwu2009>u2009 5.0. The mainshocks occurred on August 24th (Mwu2009=u20096.0) and October 30th, 2016 (Mwu2009=u20096.5). Several cou2010seismic surface ruptures were recorded after both mainshocks along the VBFS.\"- - \"We compared hydrogeological sections with earthquake locations, and DInSAR data to build up an integrated geological model of the study area (Fig._).\" - We collected discharge data and analysed their evolution after the seismic crisis, comparing the results with pre-seismic conditions, focusing our attention on single and streambed springs fed by the Basal aquifer (CM-Co and Mai). The hydrogeological monitoring was coupled with a geochemical survey of groundwater. Post-seismic tracer tests were performed, and their results were compared with those of similar tests run before the seismic crisis. - Geo-structural elaboration - \"The main contractional and extensional elements delimit three regional groundwater basins, hereafter named Eastern, Central and Western Basins (Fig._a), roughly corresponding to Basin 1, 2 and 3 proposed by Mastorillo et al.\" - The Sibillini thrust (SMth) delimits eastward the Eastern Basin. - \"The Fiegni Vettore Thrust (FVth) separates the Eastern from the Central Basin to the north. More southward, the uplift of the TEv aquiclude up to an elevation higher than the saturated zone (Fig._b) acts as a lithological groundwater divide, located slightly East of VBFS. Because of the northward axial plunging of the structure, the groundwater divide continues northward as a piezometric watershed, separating the western and the eastern flow-paths, both characterized, in the pre-seismic, by a predominant NNW flow component sub-parallel to the main tectonic lineaments.\" - The Nottoria Preci Fault system (NPFS) can be generally considered as the hydrogeological limit between the Central and the Western Basins, although it was observed that prior to the seismic sequence groundwater coming from the Central Basin contributed to the feeding of Sordo river in the Norcia plain. Groundwater flow in the investigated aquifer systems is difficult along E-W direction, while it is preferential along the strike of the main fault systems (SSE-NNW e.g.). - \"The Eastern Basin represents the footwall of VBFS, while the Central one corresponds to its hanging wall. The Western Basin develops at the hanging-wall of the NPFS (Fig._b).\" - \"Despite of the presence of internal groundwater divides and of local groundwater flow exchanges between the Basins, the hydraulic separation among them is mainly operated by the fault systems, with the elevation of saturated zone progressively decreasing westward (Fig._b), as indicated by the elevation of punctual and streambed springs (Table ).\" - \"During the 2016u20132017 seismic sequence both VBFS and its eastward dipping antithetic fault were activated, as indicated by the hypocentres distribution (Fig._b).\" - \"Figure_b,c show the earthquakes distribution along two perpendicular cross sections. Earthquakes occurred along the entire N-S elongation of the VBFS and propagated up to shallow depths, where groundwater is hosted, in a wide area also far from the sites where the co-seismic ruptures were mapped. Also events with Mwu2009 u2265u20094 were located at relatively shallow depths below the surface (Fig._ c).\" - \"DInSAR data were sampled along section Au2013Au2032 in Fig._b and show that, in response to the seismic sequence, major co-seismic extension in the order of 20u201325_cm and subsidence in the range of 90_cm was concentrated within the Central Basin and close to the VBFS. The activation of a fault antithetic to the VBFS favored co-seismic fracturing along with the Western and Central Basins boundary.\" - Hydrogeological monitoring - \"Discharge data of the main springs and streams located in the Eastern, Central and Western Basins, recorded until December 2020, are shown in Fig._. The measurement sites are reported in Fig._a. Table lists the measuring points shown in Fig._ a, their acronyms, and position.\" - Table 1 List of acronyms used to indicate measuring points, location and brief description of each. - \"Figure 2Daily rainfall in representative gauges and discharge of monitoring points. (a) Daily Rainfall (R) in Norcia and Capodacqua, respectively representative for systems located on the western and on the eastern slope. (b) Mean discharge (Q) of the main springs located in the Eastern Basin (VBFS footwall) at a daily (FoS) and monthly (SsS) scale. (c) Discharge (Q) of stream and rivers of the Central Basin (hanging wall of both VBFS and of its antithetic system). CqS is located on the Eastern slope while other measures (NeR1, NeR2, UsR, UsNeR) are taken on streams and rivers (streambed gaining springs) pertaining to the Upstream Nera River System, on the western slope. (d) Discharge (Q) of springs, streams and rivers of the Western Basin (hanging wall of VBFS and footwall of its antithetic). The red vertical bars indicate the main seismic shocks of August 24th 2016 (Mwu2009=u20096.0) and October 30th 2016 (Mwu2009=u20096.5). Location of the measures is reported in Fig._a.\" - Discharge data were used to evaluate the effects of the seismic sequence in terms of variations of groundwater volume released by local and streambed springs before and after the seismic crisis. Concerning streambed gaining springs, volume variations were evaluated in the most downstream section of each investigated system, so that data from UsNeR and SoR were used. - \"After a careful analysis of the hydrographs, a common early post-seismic period (October 29th 2016u2013December 12th 2017), hereafter named early post-seismic, was chosen as a reference for all springs. A further post-seismic period, hereafter called overall post-seismic and including the investigated post-seismic time span (October 29th 2016u2013December 30th 2020) was also considered. Groundwater volume variations were quantified as the difference between the amount of groundwater effectively released by the springs in these intervals and the amount they would have released in the same time-spans if their pre-seismic average discharge had been maintained. Pre-seismic average was meant as the mean discharge of each time series shown in Fig._, from the first available value until October 28th 2016.\" - Table shows the results of this analysis, which are comparable with those presented by Mastrorillo el al., obtained by using a slightly different data set and procedure. - \"Table 2 Variations of groundwater volume (u0394V) released by the main local and streambed springs in early and overall post-seismic reference periods._ Volumes are measured in million cubic meters_ (mu2009xD7u200910) to make them immediately comparable wit each other.\" - \"The overall discharge of the Eastern Basin, represented by Foce spring (FoS) and to a lesser extent by the smaller Sassospaccato spring (SsS), decreased dramatically after the seismic crisis (Fig._b).\" - \"Since at least November 2016 FoS, the main outflow from Basal aquifer of the Eastern Basin, suffered a continuous negative discharge trend, which lowered the discharge down to 0.1 m/s in late 2020 (Fig._b). A deficit of u2212_ 5u2009xD7u200910 m was estimated in FoS in the early post-seismic. During the overall post-seismic, the computed water deficit was u2212_42u2009xD7u2009 10 m.\" - \"The same analysis on SsS shows that in the early post-seismic the water deficit of this spring was about u2212_1u2009xD7u200910 m, while in the overall post-seismic it was about u2212_4u2009xD7u200910 m (Table ).\" - \"The immediate reaction to the earthquake sequence of Capodacqua spring (CqS, 840_m a.s.l.), located on the south-east boundary of the Central Basin, is unknown, due to a gap in daily discharge measurements (Fig._c). Nonetheless, it can be observed that, although there were no significant rainfall events (Fig._a), in late 2016 CqS discharge was about 0.2 m/s higher than before the mainshock (Fig._c). It was estimated that, in the early post-seismic, CqS released a water surplus of about 4u2009xD7u200910 m. A negative trend in both maximum and minimum discharge was recorded from late 2017 to 2020 (Fig._c), although rainfall in this interval was not particularly low. The negative trend forced the water company managing the spring, tapped for drinking purposes, to integrate its natural discharge by means of pumping wells. Therefore, it was not possible to evaluate, for overall post-seismic, the natural groundwater volume variation with respect to the pre-seismic.\" - \"Discrete monitoring data of the north-west area of the Central Basin (Figs. a and c) reveal that, after the seismic sequence, a significant discharge increase occurred in the streambed gaining springs of upstream Nera River catchment (sections NeR1, NeR2 and UsR, Fig._a). A relevant discharge increase also occurred downstream the confluence of the Ussita River in the Nera River (UsNeR section, Figs. a and c). The peak discharge in UsNeR was reached during January 2017. Between early 2017 and late 2019, despite a general negative trend of both minimum and maximum values, discharge in UsNeR, the most downstream section of the system, was higher or equal to pre-seismic values (Fig._c). The excess water drained by the Upstream Nera River system in the early post-seismic was as high as 90u2009 xD7u200910 m (Table ). The water surplus in the overall post-seismic was found to be still 90u2009xD7u200910 m (Table ), indicating that the total amount of water surplus was released in the first year after the mainshocks.\" - \"A similar general discharge increase was observed in the Western Basin springs and streams, where the Torbidone Spring (ToS, Fig._a) was reactivated after the October 30th mainshock. ToS, San Martino and Pratarella springs (SmS and PrS, Fig._a) and Sordo River (SoR, Fig._a) reached their discharge maximum in May 2017 (Fig._d), with a delay of about five months with respect to the Central Basin springs and rivers (Fig._c). Thereafter, all springs and rivers in Western Basin experienced a progressive discharge decrease, which led to the drying of ToS in summer 2020 while, despite of the gradual reduction, discharge of SoR was still higher than before the seismic crisis in December 2020 (Fig._ d).\" - \"The discharge increase of ToS, SmS and PrS does not match the overall discharge excess of the downstream receiving SoR between December 2016 and November 2017 (Fig._d). SoR water surplus in the early post-seismic was about 91u2009xD7 u200910 m (Table ). A total amount of 45u2009xD7u200910 m was released in the same interval by ToS, SmS and PrS, which contribute for about a half to the total water excess released by SoR. This indicates that the discharge increase recorded in SoR is also due to an increase in the contribution of streambed gaining springs. In the overall post-seismic SoR released a water surplus as high as 246u2009xD7u200910 m (Table ).\" - Geochemical monitoring - \"All sampled spring and river waters (Fig. ) show a Ca(Mg)-HCO3 composition typical of carbonate aquifers (Fig. ). Nonetheless, Electric Conductivity (EC, a proxy of salinity), SO4 and Mg concentrations show a significant variability, from 205 xB5S/cm to 660 xB5S/cm, fromu2009~u20092.7 tou2009~u2009155_mg/L and fromu2009~u20093.4 tou2009~u200919_mg/L respectively (Fig._). These variations occur both in space (Fig._a,b) and time (Fig._c,d), allowing to distinguish different water sub-types and their interactions. Higher salinity is generally associated to higher SO4 and Mg contents (Fig._a,b) and it can be related to a larger interaction with the dolomitic component of the CM-Co complex and with the evaporitic lithologies (TEv) stratigraphically below CM-Co (Fig._).\" - \"Figure 3Geochemical features of sampled waters. (a) Electrical conductivity vs. SO4 concentration. (b) SO4 vs. Mg concentration. (c) Time variation of SO4 concentration of ToS, SmS and PrS compared with ToS discharges (grey line). The average SO4 concentration of SmS (Januaryu2013February 2016) is also reported (dashed pale brown line). Symbols in all panels: Foce Spring (FoS, red dots), Sassospaccato Spring (SsS, orange dots),_Capodacqua Spring (CqS, purple dots), Ussita-Nera River system (UsNeR grey dots), San Martino Spring (SmS, pale brown dots), Pratarella Spring (PrS, brown dots) Torbidone Spring (ToS, green dots) and Sordo River system (SoR, blue dots). UsNeR includes data from both springs and river waters upstream of UsNeR; SoR includes data from different location along Sordo river (Fig. ).\" - \"The main springs of the Eastern Basin (FoS and SsS) are characterized by a peculiar composition with relatively high Mg content, not associated to high SO4 concentration and salinity (Fig._a,b). This can be due to the relatively shallow and short groundwater flow-path to FoS_and SsS, which are_located at high elevation (910_m a.s.l._and 1130 m a.s.l. respectively).\" - \"The water compositions of the Central Basin highlight two water types with similar salinities but different SO4 contents and SO4/Mg ratios (Fig._a,b). Water discharged in the North-Western sector (i.e., UsNeR) has slightly higher salinity, higher SO4 contents and remarkably higher SO4/Mg ratio respect to those of the South-Eastern sector (i.e., CqS). These characteristics are reasonably due to longer circulation paths of the springs of North-Western sector, consistent with their lower elevation (~u2009650_m a.s.l.) compared to CqS (~u2009840_m a.s.l.) and to a lithological variability within the basin.\" - \"The Western Basin waters generally have the highest salinities, but also show different geochemical characteristics among each other. It is possible to distinguish two water types. The first one, hereafter named SMP hydrotype and including water from SmS and PrS, is more saline and SO4-rich respect to the other, hereafter named TOR hydrotype, which is represented by water emerging from ToS. TOR hydrotype has intermediate SO4, Mg and salinity between the SMP hydrotype and the groundwater of the Central Basin (Fig._a,b). Groundwater from SoR, sampled downstream of ToS, SmS and PrS, shows an intermediate composition between the two hydrotypes, with SO4 and Mg contents decreasing downstream.\" - \"Groundwater of SmS, PrS and SoR, showed the most evident variations following the 2016u20132017 seismic sequence, with SO4 concentration that gradually increased since May 2017, while discharges of all springs in the Western Basin were simultaneously decreasing (Fig._c,d). On the contrary, these variations were not recorded in ToS, whose SO4 concentration remained practically steady (Fig._c) until the spring dried, in summer 2020. SmS gradually approached pre-seismic SO4 concentration (Fig._c) and SO4 content of both SmS and PrS became roughly stable in 2019. Conversely, the SoR positive trend in SO4 concentration continued until about half 2019 (Fig._d).\" - Tracer tests - \"Tracer tests were run during the pre- and post-seismic period. Tracer was released in the Mergani sinkhole (Castelluccio plain, Figs. a and ), located at an elevation of 1300_m a.s.l., about 350_m higher than FoS, the highest detection point (Table ). This implies that the tracer movement involves the epiphreatic zone, characterised by fissure and karst patterns, before reaching the water table. In the pre-seismic, groundwater traced flow was predominantly SSE-NNW oriented and mainly drained within the Central Basin, where the injection point is located (Fig._a). Only minor groundwater exchanges along Eastu2013West direction were observed with those from the Central to the Western Basin more relevant than those from the Central to the Eastern Basin.\" - \"Figure 4Tracer test results. The base map was obtained by elaborating the digital elevation model (DEM) from the TINITALY/01 DEM of the whole Italian territory (; Tarquini et al.; CC BY 4.0 License, ), by means of the Open Source package QGIs v 3.16 (). The map has been then edited with the Open Source software Inkscape v.1.2.1 (). (a) Distribution of tracer arrival at the sampling points before the seismic crisis. The yellow symbols indicate that tracer was detected. White symbols indicate that no tracer was detected. All numbered sampling points, but point 5, roughly correspond to water discharge measure points as follows: 1u2009=u2009FoS; 2u2009=u2009CqS; 3u2009=u2009SmS/PrS; 4u2009~u2009 SoR; 6 downstream of UsNeR; 7u2009=u2009NeR1; 8u2009=u2009UsR; 9u2009=u2009 ToS. (b) Distribution of tracer arrival at the sampling points after the seismic crisis. The green symbols indicate that tracer was detected with increased concentration with respect to pre-seismic conditions. The red symbols indicate that tracer was detected with decreased concentration with respect to pre-seismic conditions. Connections between tracer release and monitoring point are not meant as actual pathways to the sampling sites but as an evidence of an existing link between the two.\" - \"After the seismic sequence, groundwater flow-paths significantly changed. The predominantly SSE-NNW oriented flow direction persists, but most of the tracer was recorded in the Western Basin springs, west of NPFS, also in springs and streams where the tracer was not detected before the seismic crisis (Fig._ b). The hydraulic connection between Castelluccio plain and springs located southward within the Central Basin (CqS) is weakened in favour of a transfer from the injection point to springs and rivers located northward and westward. Tracer arrivals in the Eastern Basin decreased with respect to the pre-seismic conditions with only 0.9% of the total tracer amount collected in FoS. Specific rainfall events determine the arrival of the tracer to FoS in about ten days, confirming that the main tracer movements towards the eastern basin involve the vadose passages of the epikarstic zone. The transfer from the Central to the Eastern Basin is negligible with respect to the overall groundwater transfer among basins.\"- - \"Normal faults mainly act as low permeability boundaries in the inter-seismic period, contributing to the separation of groundwater flow among the three Basins, with flowpaths mainly directed along the fault strike (SSE-NNW), despite some local transfers existing along the opposite direction (Fig._a). The elevation of saturated zone is highest in the Mt. Vettore Massif, at the foot-wall of VBFS, and progressively decreases westward (Figs. b and b).\" - \\'Figure 5Conceptual scheme of hydrogeological modifications induced by the seismic sequence. The base map was obtained by elaborating the digital elevation model (DEM) from the TINITALY/01 DEM of the whole Italian territory (; Tarquini et al.; CC BY 4.0 License, ), by means of the Open Source package QGis v 3.16 (). The map has been then edited with the Open Source software Inkscape v.1.2.1 (). Surface ruptures data are from after: (1) Brozzetti et al. (ref Open access, Creative Commons Attribution 4.0 International License, ); (2) Pucci et al. (ref, free access available at ; (3) Villani et al. (ref free access available at: ). (a) Pre-seismic (dashed lines) and post-seismic (continuous lines) main flow-paths. Variations in line width represent increase or decrease of groundwater flow with respect to pre-seismic conditions. Different flow lines colours indicate different hydro-chemical characteristics. (b) Pre-seismic elevation of saturated zone in the three Basins. (c) Post-seismic modification of saturated zone elevation in Central Basin with potential migration of groundwater divide.\\' - \"The activation of VBFS and of its antithetic during the 2016u20132017 seismic sequence determined a co-seismic fracturing of the fault zones. This induced an increase in hydraulic conductivity and a consequent westward migration of groundwater, previously stored in the Basal aquifer of Vettore Massif (Eastern Basin), towards the Central Basin, at the VBFS hanging-wall (Fig._a).\" - \"Such groundwater migration caused a lowering of the saturated zone within the Vettore Massif, which affected both FoS and SsS discharge, determining an overall water deficit in the Eastern basin of about u2212_6u2009xD7u200910 m in the early post-seismic and as high as u2212_46u2009xD7u200910 m in the overall post-seismic (Fig._b and Table ). This last datum suggests that the westward directed groundwater migration through VBFS continued at least until the end of 2020, determining a progressively more severe groundwater deficit in the Eastern Basin.\" - \"The overall excess of groundwater released by the Central Basin is much higher than the water deficit of the Eastern Basin both in the in the early post-seismic (u0394Vu2009=u2009u2009+u200994.0u2009xD7u200910 m) and in the overall post-seismic (u0394Vu2009=u2009u2009+u200990.0u2009xD7u200910 m not including the CqS contribution), as shown in Table . This suggests that a great portion of the discharge increase of the Central Basin was due to an overall increase of bulk permeability and to a process of fractures cleaning, resulting from the intense co-seismic fracturing. This caused a faster groundwater drainage, concentrated in the early post-seismic, as also testified by the post-seismic increase of spring depletion coefficients observed by other authors and by the increased tracer concentration recorded in the upper Nera River (point 7 in Fig._) after the seismic crisis.\" - \"The progressive aquifer emptying induced the lowering of the saturated zone within the Central Basin (Fig._c) and a subsequent gradual decrease in hydraulic gradient along the SSE-NNW flow direction, which determined the discharge lowering recorded since early 2017 (Fig._b). A drop of the saturated zone can influence the piezometric divide between UsNeR and CqS, also due to the additional drainage towards the western basin. The hypothesis of a widening of the aquifer sub-basin drained by UsNeR (Fig._c) and of a consequent restriction of CqS recharge area will require further investigations.\" - \"The increase of bulk permeability is congruent with seismological data, as the projection of earthquakes distribution along the geological sections (Fig._ b,c) shows that shocks propagated up to the range of depths where groundwater is hosted, determining an increase of fracturing and hydraulic conductivity. Such increase is also consistent with DInSAR measurements showing that, after the 2016u20132017 seismic sequence, major subsidence and extension, coherent with a significant permeability increase and a process of fractures cleaning, occurred at hanging-wall of VBFS, in the area corresponding to Central Basin (Fig._b).\" - \"In the Western Basin, the reactivation of ToS, the significant discharge increase in SmS and PrS, and the rise of SoR discharge were simultaneously observed after the seismic events (Fig._d). The evolution of hydro-chemical parameters, tracer test results and seismic data suggest that discharge increase in these systems was mainly due to an additional groundwater transfer from the Central Basin (Fig._a). Such transfer contributed further to the above-mentioned emptying of the Central Basin aquifer.\" - \"The activation of the fault system antithetic to VBFS determined an increase in hydraulic conductivity of fault systems, previously limiting the flow exchange towards the Western Basin. This interpretation is strengthened by the presence of surface co-seismic fracturing along both the fault antithetic to VBFS and NPFS (Fig._a).\" - \"Such interpretation is also supported by geochemical data. As resumed in Fig._ c,d, since half 2017, a progressive increase of SO4 concentration, synchronous with the gradual discharge decrease of springs and rivers of_the Western Basin, was observed in SmS, PrS and SoR, and not detected in ToS. This observation suggests that the initial discharge increase observed in these systems after the mainshock was mainly due to the arrival of a TOR-type groundwater component, similar to waters of the Central Basin and less saline than SMP (Fig._a,b). TOR was the only hydro-type emerging from the ToS, which was dry before the mainshock. Conversely, in SmS, PrS and SoR the TOR hydrotype was mixed with the previously emerging SMP hydrotype. The newly arrived TOR component determined a dilution of SMP in SmS, PrS and SoR, with a consequent shift of water hydrochemistry towards lower SO4 concentration. Since May 2017, TOR-type water gradually decreased, so discharge of all systems diminished, and SO4 concentration of SmS, PrS and SoR increased. It is worth to note that SO4 concentration of the SmS reached values similar to that of pre-seismic period (i.e., February 2016; Fig._c) in early 2018.\" - \"The SO4 concentration in SmS and PrS became roughly stable in late 2018, indicating that the influence of TOR-type water on the geochemical characteristics of these springs was negligible at that time. On the contrary the positive trend in SO4 concentration of SoR was still ongoing in June 2019 (Fig._d) while discharge was decreasing. This observation suggests that a progressively decreasing water component from the Central Basin (TOR like type) still contributed to the feeding of SoR streambed springs, located at lower elevation with respect to SmS and PrS.\" - Therefore, the increase of discharge observed in the Western Basin is mainly due to an additional water transfer from the Central one, favoured by the enhanced permeability of the fault zones, rather than to an increase of the overall hydraulic conductivity. - \"The hypothesis of increased permeability across the faut zones, inducing an additional water transfer from the Central Basin, is also supported by tracer tests results. As shown in Fig._, after the seismic sequence the amount of tracer reaching the Western Basin increased. The tracer was detected at points 4, 5 and 9, where it was not found before the earthquakes, and its amount increased at points 3 and 6.\" - \"The limited effect of hydraulic conductivity increase on the discharge rise of the Western Basin is congruent with the less intense vertical and lateral co-seismic deformations with respect to the Central Basin (Fig._b).\" - \"The different mechanisms responsible for the discharge increases observed in the Central and Western Basins explain the 5-month time shift in discharge peaks after the mainshock (Fig._c,d). The groundwater transfer from the Central to the Western Basin determined an increase of water table elevation, a consequent increase of hydraulic gradients and a post-seismic discharge excess lasting longer than in the Central Basin so that, in the overall post-seismic, the excess of groundwater volume drained by the Western basin has been as high asu2009 +u2009246u2009xD7u200910 m (Table ).\" - \\'In summary, two different mechanisms are considered responsible for the observed hydrogeological changes:\\' - \"i.increase of permeability due to co-seismic rupturing along normal faults previously contributing to the separation of the_Basins and suddenly activated during the seismic sequence. The rise of hydraulic conductivity along the fault zones was the relevant cause of an increase of the westward directed groundwater transfer, determining the overall and long lasting discharge increase in the Western Basin and of the water deficit recorded in the Eastern Basin (Fig._ a).ii.the increase of hydraulic conductivity of the Central Basin, due to intense co-seismic fracturing and fractures cleaning of the subsided and extended block between VBFS and its antithetic, which mobilized groundwater resources which would have otherwise been released in significantly longer times. \" - Groundwater flowed partly along pre-existing paths parallel to the fault strike (Central Basin Springs) and partly along paths perpendicular to the fault strike (Westward directed flow to the Central and Western Basins). - It can be assessed that, in case of extensional seismic sequences, the more relevant hydraulic permeability increase due to co-seismic fracturing must be expected at the hanging-wall of the activated fault, particularly when an antithetic fault system is activated, determining a relevant subsidence and consequent extension of the block located between the two fault systems. - \"The increase of permeability along the_activated fault systems is also a key factor in modifying the groundwater transfer between hydrogeological basins mainly separated among each other by tectonic lineaments. Such transfer is driven by the pre-seismic piezometric distribution on both sides of the activated fault systems, with groundwater flowing from the higher to the lower saturated zone.\" - In the studied system, where two fault systems antithetic to each other were activated, the pre-seismic piezometric elevation coupled with the increase of the bulk permeability of the fault zones, determined the above-described westward groundwater transfer. - \"It is still unknown whether and when, after a transient-state_condition period, the former equilibrium will be restored, or a new steady-state condition will be reached in the investigated area. The potential scenarios depend not only on the recovery of pre-seismic conditions in terms of bulk permeability of aquifers and fault role, but also on the groundwater recharge rate expected during next years to re-fill the emptied aquifers, which is largely uncertain, due to climate change threats.\" - This study stresses the importance of multidisciplinary studies for a better understanding of the reaction of complex carbonate fractured system to seismic sequences. It is shown how such reactions strongly depend on the location of aquifers with respect to geological structures. Understanding these mechanisms is of great importance as many carbonate aquifers, which contribute substantially to freshwater supply in many regions of the world and particularly in Southern Europe, are located in seismically active areas.- - Elaboration of geological, seismic and DInSAR data - \"In order to understand the relationships between the subsurface geological setting and groundwater flow and how these may change across the seismic cycle we integrated different datasets, namely previous elaborated geological sections after, earthquake locations after and DInSAR. The geological sections are based on the hydrogeological map (Fig._a) which was validated with a set of both transversal and longitudinal sections to produce a consistent image of the subsurface. We choose two most representative geological sections, one transversal and one longitudinal, which, when compared with hydrogeological data, allow to infer the lithologies into which water is stored and flows towards both rivers and springs. Onto these sections we projected the distribution of earthquakes of the 2016u20132017 seismic sequence after, along a 4_km wide stripe. Data of the events with Mwu2009 u2265u20094 were extracted from the sequence itself to visualize the altimetric distribution of the moderate events and how they relate to the subsurface geology. Earthquake projection onto the geological sections was made using the tool u201C v.profileu201D, an add-on to the GRASS GIS package. With the same procedure we plotted the distribution of both vertical and horizontal co-seismic deformation. The results are shown in Fig._.\" - Hydrogeological monitoring - \"The acquired data include discharge measurements of both local springs and stream sections. Discharges were measured by discrete and continuous monitoring by using flowmeters (FlowTracker by SonTek) and calibrated multiparametric probes (model by STS s.r.l.). The measures were integrated with data from the continuous monitoring of Umbria and Marche Regions (Servizio Rischio Idrogeologico, Idraulico e Sismico, Difesa del Suolo and SIRMIP, Protezione Civile Regione Marche). The frequency of measurements ranged from a minimum of one per week to a maximum of one per 15_min. Data were compared with those available for the pre-seismic period.\" - \"The location of all discharge measuring points is shown Fig._a while discharge time series are shown in Fig._.\" - Geochemical sampling and analysis - The main groundwater discharge of the Sibillini Mountains Basal aquifer was sampled after the seismic crises, starting from the end of August 2016 up to September 2019, at both punctual springs and along the main rivers in correspondence of the main increases of their flow rate (i.e., at streambed springs). - \"Seventeen springs were sampled with a different frequency for a total of 96 samples. The spatial distribution of the sampling sites is shown in Fig._a. A larger spatial and temporal density of samples is available for the western sector of the basin. The complete dataset is reported in Supplementary material (Table ).\" - \"PH, electrical conductivity (EC) and HCO3 were determined directly in the field. The HCO3 concentration was determined by acid titration with 0.01_N HCl using methyl orange as indicator.\" - \"Chemical analyses of major elements were performed by the Laboratory of Perugia University. Calcium and Mg concentrations were determined by atomic absorption (AA) flame spectroscopy on samples filtered through 0.45_u03BCm membrane filters and then acidified directly in the field with 1% of 1:1 diluted HCl. Sodium and K were determined by atomic emission (AE) flame spectroscopy. AA and AE were performed using an Instrumentation Laboratory Spectrophotometer 951. Clorine and SO4 were determined by ion chromatography using a Dionex DX-120.\" - Tracer tests - Several long time and periodic artificial tracer tests were conducted before, during, and after the seismic period in the Sibillini Mountains Area. - \"The tracer release was performed by applying the sudden injection method in the MxE8rgani sinking stream, located at an elevation of about 1300_m a.s.l. (Fig._ a). The stream level was continuously measured by means of a hydrometric pressure transducer (CTD diver Eijkelkamp, accuracyu2009xB1u20090.5 cmH2O and resolution 0.2 cmH2O) compensated by atmospheric pressure, using an acquisition time interval of 10_min, and the discharge was determined by a rating-curve.\" - Four tracer tests performed from February 2016 to June 2020, by using alternatively Na-Fluorescein (C20H10Na2O5) and Tinopal CBS-X (C28H20Na2O6S2), are of interest for this work. Technical details about these tests are reported in .- - \\' \\' - Data generated or analysed during this study are included in this published article (and its Supplementary Information files) or are available from the corresponding author on reasonable request. - \\' \\'- - \"Manga, M., Brodsky, E. E. & Boone, M. Response of streamflow to multiple earthquakes. Geophys. Res. Lett. (2003)._ _ Manga, M. & Rowland, J. C. Response of Alum Rock springs to the October 30, 2007 Alum Rock earthquake and implications for the origin of increased discharge after earthquakes. Geofluids 9, 237u2013250. (2009)._ _ _ Montgomery, D. R. & Manga, M. Streamflow and water well responses to earthquakes. Science 300, 2047u20132049. (2003)._ _ _ _ Muirwood, R. & King, G. C. P. Hydrological signatures of earthquake strain. J. Geophys. Res-Solid Earth 98, 22035u201322068. (1993)._ _ Petitta, M. et al. Water-table and discharge changes associated with the 2016u20132017 seismic sequence in central Italy: Hydrogeological data and a conceptual model for fractured carbonate aquifers. Hydrogeol. J. 26, 1009u20131026. (2018)._ _ _ _ Rojstaczer, S., Wolf, S. & Michel, R. Permeability enhancement in the shallow crust as a cause of earthquake-induced hydrological changes. Nature 373, 237u2013239. (1995)._ _ _ _ Mastrorillo, L. et al. Sustained post-seismic effects on groundwater flow in fractured carbonate aquifers in Central Italy. Hydrol. Process. (2019)._ _ Valigi, D. et al. Earthquake-induced spring discharge modifications: The Pescara di Arquata spring reaction to the Augustu2013October 2016 Central Italy earthquakes. Water (2020)._ _ Valigi, D. et al. Springs discharge variations induced by strong earthquakes: The Mw 6.5 Norcia event (Italy, October 30th 2016). Rend. Online Soc. Geol. Ital. 47, 141u2013146. (2019)._ _ Barberio, M. D., Barbieri, M., Billi, A., Doglioni, C. & Petitta, M. Hydrogeochemical changes before and during the 2016 Amatrice-Norcia seismic sequence (central Italy). Sci. Rep. (2017)._ _ Chiodini, G. et al. Correlation between tectonic CO2 Earth degassing and seismicity is revealed by a 10-year record in the Apennines, Italy. Sci. Adv. (2020)._ _ Claesson, L. et al. Hydrogeochemical changes before and after a major earthquake. Geology 32, 641u2013644. (2004)._ _ _ _ Mohr, C. H., Manga, M., Wang, C. Y. & Korup, O. Regional changes in streamflow after a megathrust earthquake. Earth Planet Sci. Lett. 458, 418u2013428. (2017)._ _ _ _ Adinolfi Falcone, R. et al. Changes on groundwater flow and hydrochemistry of the Gran Sasso carbonate aquifer after 2009 LAquila earthquake. Ital. J. Geosci. 131, 459u2013474. (2012)._ _ Manga, M. Origin of postseismic streamflow changes inferred from baseflow recession and magnitude-distance relations. Geophys. Res. Lett. 28, 2133u20132136. (2001)._ _ _ Mohr, C. H., Manga, M., Wang, C. Y., Kirchner, J. W. & Bronstert, A. Shaking water out of soil. Geology 43, 207u2013210. (2015)._ _ _ Hosono, T. et al. Coseismic groundwater drawdown along crustal ruptures during the 2016 M-w 7.0 Kumamoto earthquake. Water Resour. Res. 55, 5891u20135903. (2019)._ _ _ Bense, V. F., Gleeson, T., Loveless, S. E., Bour, O. & Scibek, J. Fault zone hydrogeology. Earth-Sci. Rev. 127, 171u2013 192. (2013)._ _ _ Caine, J. S., Evans, J. P. & Forster, C. B. Fault zone architecture and permeability structure. Geology 24, 1025u20131028. (1996)._ _ _ Faulkner, D. R. et al. A review of recent developments concerning the structure, mechanics and fluid flow properties of fault zones. J. Struct. Geol. 32, 1557u20131575. (2010)._ _ _ Evans, J. P., Forster, C. B. & Goddard, J. V. Permeability of fault-related rocks, and implications for hydraulic structure of fault zones. J. Struct. Geol. 19, 1393u20131404. (1997)._ _ _ Fairley, J., Heffner, J. & Hinds, J. Geostatistical evaluation of permeability in an active fault zone. Geophys. Res. Lett. (2003)._ _ Goldscheider, N. & Neukum, C. Fold and fault control on the drainage pattern of a double-karst-aquifer system, Winterstaude, Austrian Alps. Acta Carsologica 39, 173u2013186. (2010)._ _ Keegan-Treloar, R. et al. Fault-controlled springs: A review. Earth-Sci. Rev. (2022)._ _ Medici, G., Smeraglia, L., Torabi, A. & Botter, C. Review of modeling approaches to groundwater flow in deformed carbonate aquifers. Groundwater 59, 334u2013351. (2021)._ _ _ Fronzi, D. et al. The role of faults in groundwater circulation before and after seismic events: Insights from tracers, water isotopes and geochemistry. Water (2021)._ _ Sanz, E., Pidal, I. M., Escavy, J. I. & de Ojeda, J. S. Hydrogeological changes along a fault zone caused by earthquakes in the Moncayo massif (Iberian Chain, Spain). Sustainability (2020)._ _ Xue, L., Brodsky, E. E., Erskine, J., Fulton, P. M. & Carter, R. A permeability and compliance contrast measured hydrogeologically on the San Andreas Fault. Geochem. Geophys. Geosyt. 17, 858u2013871. (2016)._ _ _ Sibson, R. H. Earthquake Prediction Maurice Ewing Series (eds Simpson, D. W. & Richards, P. G.) 593u2013603 (American Geophysical Union, 1981).Amoruso, A., Crescentini, L., Petitta, M., Rusi, S. & Tallini, M. Impact of the 6 April 2009 LAquila earthquake on groundwater flow in the Gran Sasso carbonate aquifer, Central Italy. Hydrol. Process. 25, 1754u2013 1764. (2011)._ _ _ Chiarabba, C., De Gori, P., Valoroso, L., Petitta, M. & Carminati, E. Large extensional earthquakes push-up terrific amount of fluids. Sci. Rep. 12, 14597. (2022)._ _ _ _ Console, F., Motti, A. & Pantaloni, M. Lintermittenza delle sorgenti del Torbidone nella Piana di Norcia: analisi delle fonti storiche a partire dal XIV secolo. Rend. Online Soc. Geol. It. 43, 36u201356. (2017)._ _ Esposito, E., Pece, R., Porfido, S. & Tranfaglia, G. Hydrological anomalies connected to earthquakes in southern Apennines (Italy). Nat. Hazard Earth Syst. 1, 137u2013144. (2001)._ _ _ Chiaraluce, L. et al. The 2016 Central Italy seismic sequence: A first look at the mainshocks, aftershocks, and source models. Seismol. Res. Lett. 88, 757u2013771. (2017)._ _ Improta, L. et al. Multi-segment rupture of the 2016 Amatrice-Visso-Norcia seismic sequence (central Italy) constrained by the first high-quality catalog of early aftershocks. Sci. Rep. 9, 6921 (2019)._ _ _ Di Matteo, L. et al. Effects of earthquakes on the discharge of groundwater systems: The case of the 2016 seismic sequence in the Central Apennines, Italy. J. Hydrol. (2020)._ _ Mammoliti, E. et al. A holistic approach to study groundwater-surface water modifications induced by strong earthquakes: The case of Campiano catchment (Central Italy). Hydrology (2022)._ _ Barchi, M. R. & Mirabella, F. The 1997u201398 Umbria-Marche earthquake sequence: u201CGeologicalu201D vs. u201Cseismologicalu201D faults. Tectonophysics 476, 170u2013179. (2009)._ _ _ Boncio, P. & Lavecchia, G. A structural model for active extension in Central Italy. J. Geodyn. 29, 233u2013244. (2000)._ _ Viaroli, S., Mirabella, F., Mastrorillo, L., Angelini, S. & Valigi, D. Fractured carbonate aquifers of Sibillini Mts. (Central Italy). J. Maps 17, 140u2013149. (2021)._ _ Tarquini S., Isola I., Favalli M. & Battistini A. TINITALY, a digital elevation model of Italy with a 10 meters cell size (Version 1.0) (Istituto Nazionale di Geofisica e Vulcanologia (INGV), 2007).Brozzetti, F. et al. High-resolution field mapping and analysis of the Augustu2013October 2016 coseismic surface faulting (Central Italy earthquakes): Slip distribution, parameterization, and comparison with global earthquakes. Tectonics 38, 417u2013439. (2019)._ _ _ Pucci, S. et al. Coseismic ruptures of the 24 August 2016, M-w 6.0 Amatrice earthquake (central Italy). Geophys. Res. Lett. 44, 2138u2013 2147. (2017)._ _ _ Villani, F. et al. Surface faulting of the 30 October 2016 M-w 6.5 Central Italy earthquake: Detailed analysis of a complex coseismic rupture. Tectonics 37, 3378u20133410. (2018)._ _ _ Valerio, E. et al. Ground deformation and source geometry of the 30 October 2016 M-w 6.5 Norcia earthquake (Central Italy) investigated through seismological data, DInSAR measurements, and numerical modelling. Remote Sens. (2018)._ _ Carboni, F. et al. Surface ruptures and off-fault deformation of the October 2016 central Italy earthquakes from DInSAR data. Sci. Rep. (2022)._ _ Brozzetti, F. & Lavecchia, G. Seismicity and related extensional stress field: The case of the Norcia seismic zone (Central Italy). Annales Tectonicae 8, 36u201357 (1994)._ Fronzi, D., Di Curzio, D., Rusi, S., Valigi, D. & Tazioli, A. Comparison between periodic tracer tests and time-series analysis to assess mid- and long-term recharge model changes due to multiple strong seismic events in carbonate aquifers. Water (2020)._ _ Nanni, T., Vivalda, P. M., Palpacelli, S., Marcellini, M. & Tazioli, A. Groundwater circulation and earthquake-related changes in hydrogeological karst environments: A case study of the Sibillini Mountains (central Italy) involving artificial tracers. Hydrogeol. J. 28, 2409u2013 2428. (2020)._ _ _ _ Neteler, M. & Mitasova, H. Open Source GIS: A GRASS GIS Approach 3rd edn. (Springer, 2008)._ _ \"- - \"This study has benefited from funding by Regione Umbria, Italy, provided by the Grant u201CInterventi di emergenza previsti dallOrdinanza del Capo del Dipartimento di Protezione Civile del 26 agosto 2016, n. 388u201D and from funding by AutoritxE0 di Bacino Distrettuale dellAppennino Centrale, provided by the Grant u201CReSTARTu2014Obiettivo 2. Riprogrammazione delle risorse idriche a causa degli eventi sismici indotti; attivitxE0 A7 e A8u2014 Ricerca finalizzata alla revisione dei modelli concettuali dei corpi idrici sotterranei e valutazione dei relativi bilanci a seguito degli effetti indotti dagli eventi sismici del 2016u20132017u201D. The authors kindly acknowledge Dr. Roberto Checcucci and Eng. Marco Stelluti of the u201CServizio Idrogeologico, Idraulico e Sismico, Difesa del Suolou201D sector of Regione Umbria, the former as hydrogeological supervisor and the latter for having provided rainfall data, taken discharge measurements and provided valuable logistic support. We also kindly acknowledge Federico Marchettoni and Gianluca Tinarello for their support in discharge measurements and C.I.I.P. Servizio Idrico Integrato for providing tapped spring discharge data. The authors thank the editor, Prof. Yanhui Dong, and two anonymous reviewers whose comments and suggestions greatly improved the manuscript.\"- - Authors and Affiliations - \"Department of Physics and Geology, University of Perugia, Via Pascoli s.n.c., 06123, Perugia, ItalyCostanza Cambi,_Francesco Mirabella,_Giulio Beddini,_ Carlo Cardellini_&_Daniela ValigiDepartment of Earth Sciences, Sapienza University of Rome, P.le A. Moro 5, 00185, Rome, ItalyMarco Petitta_&_Francesca BanzatoIstituto Nazionale di Geofisica e Vulcanologia, INGV, Sezione di Bologna, Via Donato Creti, 12, 40128, Bologna, ItalyCarlo CardelliniDepartment of Science and Matter Engineering, Environment and Urban Planning (SIMAU), UniversitxE0 Politecnica delle Marche, Via Brecce Bianche 12, 60131, Ancona, ItalyDavide Fronzi_&_ Alberto TazioliDepartment of Sciences, Roma Tre University, Largo San Leonardo Murialdo 1, 00146, Rome, ItalyLucia Mastrorillo\" - \"AuthorsCostanza CambiYou can also search for this author in _Francesco MirabellaYou can also search for this author in _Marco PetittaYou can also search for this author in _Francesca BanzatoYou can also search for this author in _Giulio BeddiniYou can also search for this author in _Carlo CardelliniYou can also search for this author in _Davide FronziYou can also search for this author in _Lucia MastrorilloYou can also search for this author in _Alberto TazioliYou can also search for this author in _Daniela ValigiYou can also search for this author in _\" - Contributions - Co.C., F.M., M.P. and D.V. designed and conceptualized the work; F.B., Co.C., D.F., L.M. and D.V. acquired and elaborated hydrogeological data; Ca.C.; G.B. acquired and processed hydrogeochemical data; Ca.C. interpreted geochemical data; D.F. and A.T. performed and interpreted tracer tests; F.M. acquired and elaborated geo-structural and seismological data; F.B., Co.C., Ca.C., D.F., L.M., F.M., M.P. and D.V. contributed to the data processing; Co.C., F.M., M.P. and D.V. synthetized results from all data; Co.C. and F.M. wrote the manuscript with significant contributions from M.P.; F.M., D.F. and Co.C. drew all figures with suggestions by all the Authors; D.V. coordinated the scientific project and was responsible of the fundings. All authors reviewed the manuscript and figures and approved their submission to Scientific Reports. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " \"Creative Commons Attribution 4.0 International License, whi..ch permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. \",\n",
       " \"Creative Commons Attribution 4.0 International License, which permits use, sharing, 'adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. \",\n",
       " \"Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author (s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. \",\n",
       " \"Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The im3. - ' ' - ages or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. \",\n",
       " \"Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article''s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article''s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. \",\n",
       " \"Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. \",\n",
       " 'Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " \"Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. \",\n",
       " 'Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not includesd in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commonslicense, unless indicated otherwise in a credit line to the material. If material is not included in the articlesCreative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articleu0393xC7xD6s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articleu0393xC7xD6s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articleu2032s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articleu2032s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articlexE2u20ACu2122s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articlexE2u20ACu2122s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License, which permitsuse, sharing, adaptation, distribution and reproduction in any medium or format,as long as you give appropriate credit to the original author(s) and the source,provide a link to the Creative Commons license, and indicate if changes weremade. The images or other third party material in this article are included inthe articles Creative Commons license, unless indicated otherwise in a creditline to the material. If material is not included in the articles CreativeCommons license and your intended use is not permitted by statutory regulationor exceeds the permitted use, you will need to obtain permission directly fromthe copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License, which permitsuse, sharing, adaptation, distribution and reproduction in any medium or format,as long as you give appropriate credit to the original author(s) and the source,provide a link to the Creative Commons license, and indicate if changes weremade. The images or other third party material in this article are included inthe articles Creative Commons license, unless indicated otherwise in acredit line to the material. If material is not included in thearticles Creative Commons license and your intended use is notpermitted by statutory regulation or exceeds the permitted use, you will need toobtain permission directly from the copyright holder. To view a copy of thislicense, visit .\" - \\' \\' - \\'\\'- - ',\n",
       " \"Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article''s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder in order to reproduce the material. \",\n",
       " \"Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article''s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. \",\n",
       " \"Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. \",\n",
       " 'Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the articles Creative Com-mons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is nosluded under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission f+rom the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the lice0nse holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution 4.0 International License. The images or other third party materialin this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " \"Creative Commons Attribution 4.0 International License. The imagesor other third party material in this article are included in the article''sCreative Commons license, unless indicated otherwise in the credit line; ifthe material is not included under the Creative Commons license, users willneed to obtain permission from the license holder in order to reproduce thematerial. \",\n",
       " 'Creative Commons Attribution 4.0 International License.\" - \"All events recorded by pressure sensors, turbidity loggers, and current meters deployed in the navigation channel and on the adjacent mudflats, were referenced to ships passing the measurement transect using the AIS information. Real time ship position data were also used to activate an automatic water sampling device remotely. The device was mounted on a floating platform close to the station 3 and set to collect 600u2009mL of water at 50u2009cm below the surface. This made it possible to obtain water samples during resuspension peaks created by ship wakes at times when we were not on site, and to collect samples for calibration of turbidity. The samples were used to calibrate turbidity sensors for the concentration of SPM determined as described in ref. . The grain size of SPM was determined by laser scattering using a LISST 100X particle size analyzer (Sequoia Scientific, USA) for field measurements and a Mastersizer 3000 granulometer (Malvern, UK), for laboratory determinations. The grain size ranges for the two instruments are 2u2013500u2009xB5m and 0.03u20131000u2009xB5m respectively.\" - The variations in the coastline morphology were determined using GIS techniques. High-resolution information on rapid changes was acquired using drone surveys. Changes over longer time intervals were extracted from aerial and satellite images available from WMS services (Web Map Services) of the National and Regional cartographic portals. The relevant methodology is described in ref. .- - Unusual ship wakes in the venice lagoon - The motion of a ship in finite depths generates a depression region of the water surface called drawdown or Bernoulli wake. In relatively shallow depths, this perturbation becomes evident as a region of lowered water level of nearly uniform depth. The presence of this phenomenon causes the drawdown effect. This effect is also called squat in inland waterways and navigation channels and is a well-known feature of sailing at depth Froude numbers larger than about 0.6 in shallow waters. This perturbation usually extends to a distance from the sailing line and may easily penetrate bays and harbors located adjacent to the sailing line and can cause unacceptable water level falls or damage. - \"Under specific conditions, the depression signal may spread far from the waterway. This happens especially if the channel is connected to shallow-water areas (e.g. tidal flats or water bodies similar to the Venice Lagoon) where the Bernoulli wake may excite strongly nonlinear long-living solitary waves of depression. These waves penetrate to a distance of more than 1u2009km onto the sub-tidal flat. They may undergo various transformations as they propagate, including shoaling-type effects and formation of bore-like phenomena. These processes may influence the magnitude of the phenomena and amplify the depression generated, and therefore increase the intensity of erosion of both channel margins and areas in the shallows.\" - \"Out of more than 3000 commercial vessels that entered the PIMZ port through the MMC in 2016, according to the official statistics, we monitored a sub-sample of all ship traffic over a period of 45 days. Wakes were recorded along a 1.5u2009 km long transect over the sub-tidal flat on the east side of the MMC from Aprilu2013 May 2016. This time interval contains a total of 615 passages of large commercial vessels with a length of more than 100u2009m. At the channel margin, depression wakes had an average height of 0.52u2009m. The deepest depression (2.45u2009 m) was excited by cargo vessel Xin Xia Men (length 280u2009m, width 40u2009 m) on 26 April 2016. As squat of this magnitude seems unreasonable, it is likely that the classic Bernoulli wake in the vicinity of the ship hull (usually a drawdown of just a few tens cm) is amplified by shoaling and/or bathymetry-driven focusing on the channel margin.\" - \"This depression is, in essence, a part of the near-wave field of ships motion (a strongly nonlinear forced wave) and thus generally cannot be described in terms of travelling waves. Its original propagation direction cannot be quantified from the dataset at our disposal. As a consequence, we were not able to determine its approach angle to the channel margin and thus the exact trajectory, shoaling and refraction properties. Our dataset reflects the properties of the resulting wave-like disturbance on the channel margin. The resulting disturbance (that possibly drives a shallow-water Riemann wave) propagates over the sub-tidal flat. As its propagation direction apparently does not match the orientation of the measurement line, the sensors, strictly speaking, detect the properties of different parts of the wave front. However, considering that the wave field is largely homogeneous along the navigation channel, it is safe to assume that our recordings adequately describe the changes to this depression wave over a long distance.\" - \"Even though the disturbance that travels into the lagoon may be a highly persistent entity, its height gradually decreases as its energy is dissipated by the interaction with the seabed (Fig._). The height of the depression wave reduces considerably as it propagates from the channel margin to the sub-tidal flat. Beyond a distance of about 800u2009m, the water surface dropdown in the depression maintains a more or less constant value of about 0.15u2009m (Fig._). This amplitude is almost independent of the magnitude of the initial perturbation, as is also suggested by simulations, and confirmed by other observations. The strong initial attenuation marks an intense dissipation of the wake energy within a certain range around the navigation channel.\" - Figure 3Instrument position along the investigated transect. Bathymetries acquired respectively in 2002 and 2016 are shown to emphasize the extent of erosion. (a) Propagation of the depression wave (as recorded by pressure sensors) generated by the passage of the ship Hellenic Spirit. (b) Time series of SPM concentration recorded at different distances from the channel after the passage of Hellenic Spirit. Both insets use the same colour code for single stations. - \"Therefore, the most intense resuspension of bottom sediments occurs along the margin of the waterway and in the sub-tidal flat area near the channel. The SPM concentration in the water column at 200u2009m from the channel reaches 1000u2009mg/l after the perturbation. The intensity of resuspension is still considerable at a distance of 400u2009m from the channel where peaks of SPM concentration up to 58u2009mg/l were measured. This is much higher than typical natural concentrations (Fig._). Resuspension remains moderate and the SPM concentration is not significantly altered further from the channel. Some of the sediment mobilized by each wake resettles soon after the event.\" - The pattern of sediment transport - The fate of the sediment fraction that remains in suspension depends on the lagoon-scale motions induced jointly by successive ship transits and tidal currents as described in ref. . In particular, wakes of ships that enter the port in quick succession (within minutes of each other) often prevent particles settling and promote the efficient removal of sediment in suspension by the currents. This additional motion of the following ship (that is apparently associated with the alongshore transport of water by a set of deep depressions) can occur when the SPM concentration is still high from previous ship passages and causes a stepwise movement of sediment from the tidal flat towards the channel and thus exacerbates the silting of the channel. - \"The complicated pattern of perturbations attributed to the water velocity associated with the passage of a ship in the channel can be to some extent interpreted by in situ measurements in the proximity of the sailing line. Data from one of these events, measured on the channel slope using both the ADCP (Fig._, panels au2013c) and electromagnetic current meters S4 (Fig._), shows the effects on the eastern bed margin during the passage of a cruise ship (MSC Musica, length 294u2009m, width 32u2009m) moving northward. The ADCP data set is limited to the first 6u2009m of the water column because of the intrinsic limitations related to side-lobe effects at the bottom. To complete the information, the near-bed (20u2009cm above the bottom) velocity from the S4 current meter is plotted in panel D.\" - Figure 4(a) Horizontal component of current velocity at the margin of the channel during the passage of MSC Musica (b) water movement direction, (c) vertical velocity (positive values indicate upward movements), (d) water velocity vector at the eastern margin of the channel bed during the passage of the MSC Musica. Vector lengths are proportional to the velocity magnitude. The red line in panels a, b and c indicates the water surface associated with the depression wave. - \"The typical values of tidal currents in the area are 0.15u2009m/s. When a wake approaches, the horizontal speed increases abruptly up to 1.50u2009m/s (Fig._ , at 40u2009s). The direction of the flow is opposite to the direction of the vessel (Dir u2248 180xB0, Fig._). The passage of the drawdown (Bernoulli wake) forces the water to move from the sub-tidal flat towards the channel (Fig._ ). This is evident after the velocity peak (about 60u2009s), when the current direction 225xB0 indicates the movement of water from the sub-tidal flat to the channel. The bottom layer also initially moves towards the ship (Fig._ at 20u201340u2009s) but later, the motion is directed perpendicularly to the channel (45u201360u2009s, red framed subset). Both phases of the perturbation (reverse flow in the channel and motion towards the channel) affect the whole depth range covered by the ADCP.\" - \"The vertical component of current velocity shows an initial upward movement (Fig._ , at 8u201310u2009s) with the largest values in the bottom layers. This feature probably reflects the arrival of a soliton-like elevation (so-called precursor soliton) ahead of the ship. A downward movement, that can be naturally associated with the depression around the ship, dominates at 50u201360u2009s. It is accompanied by flow from the sub-tidal flat to compensate for a local pressure drop in the channel. This flow has a component parallel to the channel as evidenced in the S4 data (Fig._).\" - \"Immediately after the depression wave has moved further, the current velocity returns to values close to background levels. The described dynamics suggest that sediment resuspended from the channel margin and from the sub-tidal flat in the vicinity of the channel will generally be transported towards the navigation channel. Even though the transport time during each passage is fairly short (~20u2009s) and the transport range is about ten metres, this stepwise process is repeated thousands of times each year. Also, it may be enhanced by the cumulative effect resulting from consecutive ship transits.\" - Morphological impacts - \"As the typical length of the depression along its propagation direction is >100u2009 m, this disturbance (if interpreted as a wave) is already a long wave in the navigation channel and is even more so when it travels over the tidal flat. It is therefore likely that the pressure sensors reliably reproduce the basic features of its shape even if some short-wave effects (occasionally occurring at its front and apparently reflecting the Kelvin wave system) may remain partly undetected.\" - \"The shape of the depression generated by the steadily sailing ship in such a channel, theoretically, should be a gradually lengthening and almost symmetric u201Cvalleyu201D with relatively steep front and end that are separated by more or less constant-level trough. This theoretical shape is specific to a weakly nonlinear framework and is apparently not created within the MMC as our data show that depression wakes on the channel margin are slightly asymmetric V-like features. The further evolution of such waves is dictated by the properties of nonlinear shallow-water equations. The depression wave is gradually modified due to the shoaling effect on the channel margins and over the shoals. It progressively loses its initial symmetry. The wave eventually transforms into a sawtooth-shape travelling Riemann wave with a very steep rear face (Fig._). The propagation of this steep wave front in natural conditions is usually accompanied with intensive energy dissipation, large horizontal water velocities and thus strong entrainment of the bottom sediment. The occurrence of many such events, essentially each large vessel, is reflected by morphological changes that affect the channel margins and the adjacent tidal and subtidal flats, marshlands and the artificial deposits of the reclaimed area.\" - \"The joint impact of the described processes substantially depends on the nature of the seabed, the availability of finer sediment and the extent of the shallow-water areas adjacent to the channel. The described motions of water and associated transport of sediment lead to gradual erosion of the channel margins and tidal flats over the range where the depression waves cause considerable resuspension (as shown in the comparison of 2002 and 2016 bathymetries in Fig._). As described in ref. , the entrained materials are partly transported away from the site by background tidal currents but are also largely transported towards the channel (Supplementary Fig._) that is subjected to progressive filling with variable intensities depending on the erodibility of the channel edges and the tidal flat sediments. Results of this process are visible on high-resolution bathymetric surveys performed with multibeam echosounders that show the presence of erosional features on the channel slope and dredging marks on the channel bottom where sediment accumulates.\" - \"An example of the changes that occurred in the period 2015u20132017 to the cross section of the MMC reported in Fig._ and Fig._ confirms the presence of the described pattern of changes. The western bank of the channel is narrow and consists of soft and easily erodible material. It is thus expected that under the hydrodynamic conditions caused by the ship wakes, this channel margin is able to be affected by considerable erosion over such a short time (Fig._).\" - \"Figure 5MMC Channel profiles from surveys of 2015 and 2017 showing considerable erosion on the western side. The western margin slope changed from 29xB0 to 23xB0.\" - \"The reclaimed land (areas A, B and D-E) to the west of the channel has been experiencing progressive shoreline retreat and over a period of 30 years underwent a rapid change which has no comparison in the recent morphological history of the lagoon. The largest transformation affected the area A (Fig._) that was reclaimed in 1963 with materials dredged from the MMC. Similarly, to the other reclaimed areas its shoreline was initially protected by a rip-rap revetment. In accord with the processes described above, the erosion is not uniform in space and time. The total shoreline retreat in 54 years varies from about 70 to 220u2009 m. The relevant regression rate (from 2 to 7u2009m/yr) significantly exceeds rates determined for reclaimed area B.\" - \"Figure 6Coastline evolution of reclaimed area A on the western margin of the MMC in the period 1987u20132017. The coloured curves outline the erosion of the shoreline from satellite and aerial images. Traces of a dismantled rip-rap protection are visible as dark patches at the bottom of the right-hand panel. The image of the reclaimed area A was extracted from aerial photos taken by the authors with an Unmanned Aerial Vehicle. The image with the coastline evolution of reclaimed area A was created using QuantumGIS software version 3.4.1. Open Source Geospatial Foundation Project. .\" - \"The relative roles of natural and ship-driven hydrodynamic activity on sediment transport and associated erosion and accumulation in the study area can be, to some extent, estimated based on the time during which the processes of different origin maintain elevated SPM concentrations. Time series of SPM concentration from a monitoring station positioned in the area of the lagoon are available on the website of a local institution (station MAP in Fig._) that is practically not affected by ship-induced depression waves. The SPM data set for this station covers a period of 10 years with a sampling frequency of 15u2009min. The recordings are validated for 90% of the total acquisition time. On an annual basis, the 95 and 99 percentiles of suspended particle concentration vary between 17.7 and 38.2u2009mg/l, and 30.6 and 110.1u2009mg/l, respectively (Supplementary Table_). The same percentile values calculated from all recordings over the whole 10-year period are 22.2 and 50.5u2009mg/l, respectively. These values are, by definition, representative of periods of high SPM concentration, that include summer algal blooms, and resuspension due to moderate wind events up to 10u2009m/s and major storm events. Therefore, only major resuspension events created by natural forcing, limited to 1% of the time, can approach or exceed the values typically found in the area impacted by the ship depression wakes at a distance of up to 0.5u2009km from the navigation channel. Ship wake induced high SPM concentrations are characterized by a long persistence (at least one hour) and occur on average 10 times a day. Therefore, extreme SPM concentrations are found in the vicinity of the navigation channel as long as there is ship traffic. Furthermore, in the area closer to the channel margin (up to a distance of 200u2009m), the SPM concentrations induced by ship wakes exceed the extreme natural values (expressed in terms of the 99 percentiles of naturally driven SPM concentrations) by at least one order of magnitude. It is therefore safe to conclude that the contribution of the natural forcing in the lagoon to the total erosion observed in the study site is of minor importance compared to the effects of ship traffic.\" - \\'Figure 7Longitudinal distribution of the quantity ({{rm{V}}}_{{rm{i}}}) (coloured dots) along the MMC and Lido channels. The position of the turbidity gauge MAP is indicated. The background map was created using Quantum GIS software version 3.4.1. Open Source Geospatial Foundation Project. starting from original data, provided by Magistrato alle Acque di Venezia, elaborated upon by Sarretta et al., as described in ref. . 10.1016/j.csr.2009.07.002. An open access version of the article can be downloaded from: 10.6084/m9.figshare.810481. This article is licensed under a Creative Commons Attribution 4.0 International License.\\' - Channel sections with the largest impact - The existing research suggests that none of the classic parameters (blocking coefficient and the depth-based Froude number) nor their combination are able to properly describe the sailing regimes under which extremely strong depression waves are produced. The deepest depressions were generated by medium-size ships that sailed at fairly moderate Froude numbers. A possible reason for this shortcoming is that none of these indicators combine the mutual relationships between the channel geometry and vessel speed. In other words, the separate use of these parameters without considering interactions does not enable the identification of the critical sectors along the waterways. - \"For this reason, we use the quantity ({V}_{i}=v/A) [ms] to identify areas where larger hydrodynamic effects can be expected. Even though it is a dimensional measure, it combines the ships speed and the basic site-specific feature of the channel (its cross-section area) in a simple manner, and these two are known to affect the formation of strong wakes. From the definition of ({V}_{i} ) it follows that its larger values correspond to higher speeds and smaller channel cross-sections and this quantity thus could be a reasonable (at least qualitative) proxy of the chances for the formation of large wakes in different sections of the channels.\" - \"The values of ({V}_{i}) (Fig._) along the two main waterways in the Venice Lagoon, MMC and Lido channel, were calculated, as described above, for each 200u2009m long section, for all ships longer than 100u2009m that entered the lagoon in 2016. Data for the Lido channel, which is still used as a main route for cruise ships, are presented for comparison. For the two waterways the MMC has the largest values of Vi in the middle section of this channel next to the reclamation areas A and B. Another short segment of large values of Vi is found at the turn of the channel in the south-eastern section of the waterway. The values of ({V}_{i}) are much smaller in the channel that leads from the Lido inlet to the historic center of Venice. This difference can be explained by a combination of strict speed restrictions for cruise ships (AIS velocities are normally below 6 knots) that enter the city center and the relatively large cross-section of this waterway.\" - \"The patterns of spatial variation in the quantity ({V}_{i}) explain, to a first approximation, the significant impacts of ship traffic found in the central part of the Venice Lagoon. The largest values of ({V}_{i}) are located in the MMC to the north of the study transect (Fig._). The east side of this part of the channel is protected by artificial shoal embankments. These structures were apparently designed to prevent ship wake propagation into the sub-tidal flat. High values of ({V}_{i}) also occur along reclaimed area B where previous studies revealed rapid shoreline erosion. Our study site for ship wake measurements was specifically located in a segment with large values of ({V}_{i}) and the frequent occurrence of large ship wakes.\" - The link between wave height and sailing regime - \"It is natural to assume that the formation of depression wakes is mostly related to the vessels size, speed and hull shape. The relevant relationship is neither simple nor straightforward. Figure_ shows the frequency of depression wake amplitudes for different vessel dimensions. The largest disturbance in terms of magnitude and frequency is caused by vessels with a length in the range 150u2013200u2009m. This is the most frequent type of vessels in the MMC, accounting for 46% of the total number of identified ships. Ships from this category also pass the measurement cross-section with the highest speed (Fig._). Specifically, 63% of such ships sail at 8u201310 knots. It is thus likely that these vessels have the largest influence on the channel, sub-tidal flats and adjacent wetlands. However, as the distribution of the size and type of vessels is expected to vary in the future (e.g., if larger cruise ships use the MMC instead of the Lido inlet), this situation may change.\" - Figure 8(a) Frequency distribution of depression wave height for vessels of different length, (b) Frequency distribution of vessel speed for vessels of different length. - In an attempt to identify practically usable criteria for the magnitude of potential impacts of vessel wakes in the study area, we analysed the relationships between the amplitude of depression waves and the typically considered physical parameters of ships. The height of the depression wake is normally related to the depth Froude number (Fr,)and blocking coefficient (C). For example, Schoellhamer suggested that the quantity (F{r}^{2.4}{C}^{1.6}) is the best predictor for the maximum height of ship waves. As this predictor has been derived for (almost linear) waves of elevation, it does not necessarily work for strongly nonlinear waves of depression in the Venice Lagoon. - \"Following this line of thinking, we attempted to find a reasonable predictor for the height (H) of depression waves as an empirical relationship that combines water displacement and the depth-based Froude number (Fr). For each ship that passed the measurement section during the acquisition period we calculated the value of (Fr) and (C) as described in the Methods section, using the real time AIS data of velocity, draft, beam and length of the ships. We tested different combinations of powers of (Fr) and (C ) as well as other geometrical parameters. Interestingly, an even better fit (in terms of a higher correlation coefficient ({R}^{2})) than the optimal combination of (F{r}^{p}{C}^{q},)was obtained when (C) was replaced by the displaced volume ({D}_{V}). The optimal fit with experimental measurements of the height of the depression wave ((H)) (Fig._) is\" - \"$$1000,Happrox 0.558,F{r}^{1.6}{D}_{v}^{0.8}-0.019$$ (1) \" - Figure 9Scatterplot of depression wake heights (H) and the quantity (0.001,{{Fr}}^{1.6}{{D}}_{{rm{v}}}^{0.8}). Red dots indicate cruise ship passages on 19 July 2015. - \"The scatter of the values on different sides of Eq. () is quite limited for wave heights lower than 1u2009m. The cloud of data points is much wider for larger wave events (that are normally generated by larger vessels). A part of the scatter may be attributable to different tide levels. The variations in sea level in the entire lagoon are only partially accounted for in calculations of (Fr ). Other obvious contributors to the scatter are the distance of the ship route from the channel axis and different shapes of hulls.\" - \"The red dots in Fig._ reflect cruise ships measured on one day (19 July 2015), after the monitoring period of cargo ships. The data points for these vessels (that have dimensions and velocities comparable with those of typical larger cargo vessels) are also relatively strongly scattered, probably for reasons similar to those discussed above However, Fig._ suggests that the impact of this type of vessel (with lengths close to 300u2009m) falls within the largest range of magnitudes, as most wakes are >1u2009m. Therefore, for a confined channel like the MMC, the vessel length may play a specific role in the formation of large wakes. For a given blocking coefficient, a longer ship would excite a longer perturbation around the ship that in turn may drive a deeper depression on the channel margin.\"- - It is well known that the high wake energy induced when a big ship navigates in coastal waters, of limited depth or in a restricted channel is one of the main causes of erosion of side banks of channels, nearby beaches, microtidal estuaries and tidal flats. Many ports worldwide, are experiencing problems related to vessel wakes that often are the main cause of sediment transport and erosion. They have been occurring and reported over many decades but have recently become much more evident and disturbing. This has happened partially because of the continued increase in the intensity of ship traffic and vessel size and partially because of the advances that have increased our capacity to detect and observe the consequences. - As mentioned above, these problems occur worldwide, including Charleston Harbor in the USA with an increased likelihood of shoreline erosion resulting from vessel wakes, the Stockholm archipelago where one of the concerns is to regenerate the fine-sediment habitats that have been lost due to the impact of ship wakes, and in Australia with issues related to the optimization of shipping channel capacities for ports. It is also widely recognised that wakes of some vessels may travel over long distances without substantial loss of energy and strongly impact shore sections many kilometres from the fairway, examples being Tallinn, Estonia and the Marlborough Sounds, New Zealand. - The Venice Lagoon is a classic example where ship traffic can substantially modify the status and evolution of the habitat. Ship wakes have the potential to strongly modify the cross section of the navigation channel and the adjacent shallow water areas. In the light of similar research in other parts of the world showing the influence of vessel wakes as a dominant factor in morphodynamics due their relatively high intensity with respect to the natural forcing, this investigation also allows us to make the same inference regarding the situation in the Venice Lagoon. However, retreat rates of protected shoreline sections under the influence of ship wakes as high as those found in Venice in this study have not been previously recorded. The shoreline retreat in certain areas along the MMC channel can locally up to two times exceed previously measured erosion rates. To date, the rate of erosion shows no decrease over time. - The ongoing rise in the number and size of vessels serving the Venice port almost certainly extends the erosion effects in the lagoon, and our findings associated with the specific nature of ship wakes reveal a connection between ship parameters, velocity and erosion. - The principally new feature of our understanding of processes in the Venice Lagoon as described in this paper, is the unusual driver, namely, strongly nonlinear long-living solitary depression waves that resemble Riemann waves. This hydrodynamic driver results in unusually strong vertical velocities in some sections of the wake (particularly the steep rear face) similar to depression-wave tsunamis. It is highly likely that this feature, together with a specific pattern of water motion generated by moving ships, is responsible for the morphological changes in large areas of the central lagoon basin. - \"Despite the relative uncertainty of estimates of the depression wave heights for larger vessels, Fig._ indicates that feasible management options to address the impacts of ships with given dimensions may exist. Firstly, vessel speed can be managed in order to limit depth-based Froude numbers. Secondly, the hull geometry of ships could be improved in the future.\" - Management in terms of the Froude number requires highly accurate bathymetry data, information about water level during the ship passage across the lagoon (according to tides) and real-time positioning. While the first two sets of information can be fairly easily retrieved or predicted, the management of ship speed is more complicated. Existing regulations for vessel traffic in the MMC industrial channel only establish general criteria based on safety issues and do not involve any options to minimise the impacts of ship traffic. The regulations also do not prescribe any speed limit along most of the channel, only prescribing a speed limit at the entrance to the industrial port. - \\'A rational regulatory approach that also enhances protection of the environment does not necessarily imply a contraction of port activities. Such regulation could lead to immediate positive effects due to: (i) less erosion and therefore savings in dredging costs; (ii) the ability to adopt ship-specific speed limits (that could allow certain low-impact vessels to move faster and foster future improvements in vessel design); (iii) an increase in the payload for certain types of ships. As a first approximation, express estimates based on Eq. () (or more refined versions) have the potential to drive changes in the regulations for ship traffic towards a more sustainable, safe and efficient vessel traffic management system. Such systems would also reduce both the cost and the impact of dredging.\\' - The results of our research suggest that progress in this direction has to become a priority if the current intention to redirect cruise ships into the MMC is implemented. Unless carefully and specifically managed, this scenario of combined cruise, commercial and industrial shipping will lead to increased negative impacts of ship traffic in parts of the Venice Lagoon - \"Figure_ clearly indicates that even based on a simplified parameter, the hydrodynamics and morphological impacts of increased ship traffic in the MMC on critical areas of the Venice Lagoon are likely to be very significant. The largest changes are expected in the areas that are not yet in morphological equilibrium with the stressors, such as the tidal flat in the vicinity of the MMC and the shorelines of the reclaimed areas. Figure_ also indicates that the projected transit of cruise ships in the MMC would considerably increase the impacts on the morphology of the channel and sub-tidal flats.\" - According to the statistics of the port of Venice, if all cruise ships are redirected through the MMC, about 500 additional vessels (i.e., 1000 transits) will be added to existing traffic in the waterway each year. If no action is taken to limit depression wave height, most cruise ships are expected to produce very large wakes. This rerouting, therefore, may lead to a significant increase in the erosion of areas that are already shown to be susceptible to natural and anthropogenic processes such as sea level rise and subsidence. - Strong impacts on the seabed may also have equally concerning side effects. The Venice Lagoon has been used as a port for many centuries and for a variety of other purposes. Additional intense erosion may adversely impact submerged historic and prehistoric cultural relics. As a result of industrial activities in the surrounding area, lagoon sediments are known to accumulate concentrations of contaminants that could be released into the environment by erosion and remobilization of sediments. The almost continuous resuspension of sediment in the area can redistribute toxic pollutants of industrial origin from the lagoon sediments and reclaimed areas. Moreover the pumping effect of ship wakes on ground water in contaminated aquifers in the industrial area in contact with the channel bed can cause the release of harmful contaminants and eventual effects on the biota. - Finally, it is important to underline the existence of many other similar situations worldwide where navigation channels cut through lagoons, estuaries or extensive nearshore areas that are surrounded by wide shallow water bodies. While the threats associated with the direct impact of the classic Kelvin and Bernoulli wakes and the remote impact of nonlinear waves of elevation and various wave packets have been widely recognised for a long time, the presence of strongly nonlinear solitary waves of depression in certain situations was only discovered a few years ago. As wave physics is universal, it is likely that such waves are often present in many water bodies all over the world. This highlights the need to reconsider the estimates of potential environmental impacts of ship traffic in all similar situations.- - \\' \\' - The datasets generated and/or analysed during the current study are available from the corresponding author on reasonable request. - \\' \\'- - \"Madden, T. A History of Venice: Queen of the Seas. Penguin, New York, 446 (2013).Orlando, E. Venezia e il mare nel medioevo. Il Mulino, Bologna, 213 (2014).Lessan, P., Combatti, B., Moretti, D. Mappa Generale delle Lagune di Venezia Archivio di Studi Adriatici ASA - ISMAR CNR Sede di Venezia. (1813u20131818?) BSA Project, .Lippe, E. Porto di Lido ed i suoi fondali: confronto dei rilievi batometrici eseguiti dal 1800 ad oggi. Studi di idraulica marittima e lagunare 93u2013 118 (1984).DxB4Alpaos, L. Levoluzione morfologica della Laguna di Venezia attraverso la lettura di alcune mappe storiche e delle sue carte idrografiche. Ist. Veneto di Sci. Lett. e Arti 1u201358 (2010).Gatto, P. il cordone litoraneo della laguna di Venezia e le cause del suo degrado in Istituto Veneto di Scienze Lettere ed Arti Rapporti e Studi, 9, 163u2013193 (1984).Costantini, M. Porto navi e traffici a Venezia 1700u20132000. Marsilio editori, Venezia, 158 (2004).Port of Veniceu2013Throughput Statistics. Final-December 2017. Available from, .Sorensen, R. M. Ship-generated waves. Adv. Hydrosci. 9, 49u201383 (1973)._ _ Wehausen, J. V. The wave resistance of ships. Adv. Appl. Mech. 13, 93u2013244 (1973)._ _ Davis, S. E. III., Allison, J. B., Driffill, M. J. & Zhang, S. J. Influence of vessel passages on tidal creek hydrodynamics at Aransas National Wildlife Refuge (Texas, United States): Implications on materials exchange. J. Coast. Res. 25(2), 359u2013365 (2009)._ _ Schoellhamer, D. H. Anthropogenic sediment resuspension mechanisms in a shallow microtidal estuary. Estuar. Coast. Shelf Sci. 43, 533u2013 548 (1996)._ _ _ Hofmann, H., Lorke, A. & Peeters, F. The relative importance of wind and ship waves in the littoral zone of a large lake. Limnol. Oceanogr. 53(1), 368u2013380 (2008)._ _ _ Hofmann, H., Lorke, A. & Peeters, F. Wind and ship wave-induced resuspension in the littoral zone of a large lake. Water Resources Res. 47, W09505, (2011)._ _ _ Houser, C. Relative importance of vessel-generated and wind waves to salt marsh erosion in a restricted fetch environment. J. Coast. Res. 26(2), 230u2013240 (2010)._ _ _ Anderson, F. E. Effect of wave-wash from personal watercraft on salt marsh channels. J. Coast. Res. Special Issue 37, 33u201349 (2002)._ _ Browne, J. P. Long-term erosional trends along channelized salt marsh edges. Estuar. Coasts 40(6), 1566u20131575, (2017)._ _ Bourne, J. Louisianas Vanishing Wetlands: Going, Going. Science 289(5486), 1860u20131863 (2000)._ _ _ _ Houser, C. Sediment resuspension by vessel-generated waves along the Savannah River, Georgia. J. Waterw. Port Coast. Ocean Eng.-ASCE 137(5), 246u2013 257, (2011)._ _ Jiang, T. Ship Waves in Shallow Water. Fortschritt-Berichte VDI, Reihe 12, VDI Verlag, DxFCsseldorf, 466 pp (2001).Brown, E. D. et al. Observations of a nonlinear solitary wave packet in the Kelvin wake of a ship. J. Fluid Mech. 204, 263u2013293 (1989)._ _ _ Gourlay, T. P. The supercritical bore produced by a high-speed ship in a channel. J. Fluid Mech. 434, 399u2013409 (2001)._ _ _ _ Soomere, T. Nonlinear components of ship wake waves. Appl. Mech. Rev. 60(3), 120u2013138 (2007)._ _ _ Parnell, K. E. & Kofoed-Hansen., H. Wakes from large high-speed ferries in confined coastal waters: Management approaches with examples from New Zealand and Denmark. Coastal Manage. 29, 217u2013237 (2001)._ _ Parnell, K. E. et al. Ship-induced solitary Riemann waves of depression in Venice Lagoon. Phys. Lett. A 379(6), 555u2013559, (2015)._ _ _ Parnell, K. E., Zaggia, L., Soomere, T., Lorenzetti, G. & Scarpa, G. M. Depression waves generated by large ships in the Venice Lagoon. J. Coast. Res. 75, 907u2013911 (2016)._ _ Rodin, A., Soomere, T., Parnell, K. E. & Zaggia, L. Numerical simulation of the propagation of ship-induced Riemann waves of depression into Venice Lagoon. Proc. Estonian Acad. Sci. 64 (1), 22u201335, 10.3176/proc.2015.1.04 (2015).Rapaglia, J., Zaggia, L., Ricklefs, K., Gelinas, M. & Bokuniewicz, H. Characteristics of ships depression waves and associated sediment resuspension in Venice Lagoon, Italy. J. Mar. Syst. 85, 45u201356 (2011)._ _ Rapaglia, J., Zaggia, L., Parnell, K., Lorenzetti, G. & Vafeidis, A. T. Ship-wake induced sediment remobilization: Effects and proposed management strategies for the Venice Lagoon. Ocean Coast. Manage. 110, 1u201311 (2015)._ _ Ostendorp, W., Peeters, F., Hofmann, H., Schlichtherle, H. & Brem, H. Erosion hazards and efficient preservation measures in prehistoric cultural layers in the littoral of Lake Constance (Germany, Switzerland). Conserv. Manage. Archaeol. Sites 18(1u20133), 217u2013229, (2016)._ _ Uliczka, K. & Kondziella, B. Ship-induced sediment transport in coastal waterways (SeST). 4th MASHCON u2013 International Conference on Ship Manoeuvring in Shallow and Confined Water with Special Focus on Ship Bottom Interaction. Karlsruhe: Bundesanstalt fxFCr Wasserbau, 2u20138 (2016).Nanson, G. C., von Krusenstierna, A. & Bryant, E. A. Experimental measurements of river-bank erosion caused by boat-generated waves on the Gordon River, Tasmania, Regulated Rivers, Research and Management 9, 1u201314 (1994)._ Gaskin, S. J., Pieterse, J., Al Shafie, A. & Lepage, S. Erosion of undisturbed clay samples from the banks of the St. Lawrence River. Can. J. Civ. Eng. 30, 585u2013595 (2003)._ _ McConchie, J. A. & Toleman, I. E. J. Boat wakes as a cause of riverbank erosion: A case study from the Waikato River. New Zealand. J. Hydrol. New Zealand 42, 163u2013179 (2003)._ Liedermann, M. et al. Typification of vessel-induced waves and their interaction with different bank types, including management implications for river restoration projects. Hydrobiologia. 729, 17u201331 (2014)._ _ Larson, M., AlmstrxF6m, B., GxF6ransson, G., Hanson, H. & Danielsson, P. Sediment movement induced by ship-generated waves in restricted waterways. Coastal Dynamics. 120, 300u2013311 (2017)._ Ravens, T. M. & Thomas, R. C. Ship wave-induced sedimentation of a tidal creek in Galveston Bay. J. Waterw. Port. Coast. Ocean. Eng. ASCE 134, 1u201341 (2008)._ _ U.S Army Corps of Engineers navigation mission. Charleston Harbor Post 45 Final Integrated Feasibility Report/Environmental Impact Statement (2015).Sarretta, A., Pillon, S., Molinaroli, E., Guerzoni, S. & Fontolan, G. Sediment budget in the Lagoon of Venice, Italy. Cont. Shelf Res. 30, 934u2013949 (2010)._ _ _ Molinaroli, E., Guerzoni, S., Sarretta, A., Masiol, M. & Pistolato, M. Thirty-year changes (1970 to 2000) in bathymetry and sediment texture recorded in the Lagoon of Venice sub-basins, Italy. Marine Geology 258(1u2013 4), 115u2013125 (2009)._ _ _ Molinaroli, E., Guerzoni, S., Sarretta, A., Cucco, A. & Umgiesser, G. Link between hydrology and sedimentology in the Lagoon of Venice, Italy. J. Mar. Syst. 68, 303u2013317 (2007)._ _ Carniello, L., Defina, A. & DAlpaos, L. Morphological evolution of the Venice lagoon: Evidence from the past and trend for the future. J. Geophys. Res. Earth Surf. 114, 1u201310 (2009)._ _ Zaggia, L. et al. Fast shoreline erosion induced by ship wakes in a coastal lagoon: Field evidence and remote sensing analysis. PLoS One 12, 1u201323 (2017)._ _ _ Trincardi, F. et al. The 1966 flooding of Venice: What time taught us for the future. Oceanography. 29(186), 178 (2016)._ Del Bello, L. Venice anti-flood gates could wreck lagoon ecosystem. Nature 564, 16 (2018)._ _ _ _ _ PIANC (The World Association for Waterborne Transport Infrastructure). Maritime navigation commission. Harbour approach channels design guidelines. PIANC report 121 (2014).Soomere, T., PxF5der, R., Rannat, K. & Kask, A. Profiles of waves from high-speed ferries in the coastal area. Proc. Estonian Acad. Sci. Eng. 11(3), 245u2013260 (2005)._ Akylas, T. On the excitation of long nonlinear water waves by a moving pressure distribution. J. Fluid Mech. 141, 455u2013466 (1984)._ _ _ _ _ Grimshaw, R. H. J. & Smyth, N. Resonant flow of a stratified flow over topography. J. Fluid Mech. 169, 429u2013 464 (1986)._ _ _ _ Torsvik, T., Didenkulova, I., Soomere, T. & Parnell, K. E. Variability in spatial patterns of long nonlinear waves from fast ferries in Tallinn Bay. Nonlinear Process. Geophys. 16, 351u2013 363 (2009)._ _ _ Naghdi, P. M. & Rubin, M. B. On the squat of a ship. J. Ship Res. 28, 107u2013117 (1984)._ Gourlay, T. Slender-body methods for predicting ship squat. Ocean Eng. 35, 191u2013200 (2008)._ _ Torsvik, T., Pedersen, G. & Dysthe, K. Waves generated by a pressure disturbance moving in a channel with a variable cross-sectional topography. J. Waterw. Port Coast. Ocean Eng.-ASCE 135(3), 120u2013123 (2009), 10.1061/(ASCE)0733-950X (2009)135:3(120).Forsman, B. From bow to beach. SSPA Highlights No 3, 4u20135 (2001)._ Balzerek, H. & Kozlowski, J. Ship-induced riverbank and harbour damage. Evidence for claims processing, Hydro International, September 2007, 2u2013 7 (2007)._ Whitham, G. B. Linear and nonlinear waves. Wiley, New York (1974).Didenkulova, I., Zahibo, N., Kurkin, A. A. & Pelinovsky, E. N. Steepness and spectrum of a nonlinearly deformed wave on shallow waters. Izvestiya Atmos. Ocean. Phys. 42, 773u2013776 (2006)._ _ Madricardo F. et al. High resolution multibeam and hydrodynamic datasets of tidal channels and inlets of the Venice Lagoon. Sci. 4 (2017).Corila Open data, .Townsend, K. et al. Reducing shoaling in the Texas GIWW (Gulf Intracoastal Waterway) and erosion of barrier islands along west Galveston Bay. Terra. Aqua 136, 20u201329 (2014)._ Kelpu0161aitu0117, L., Parnell, K. E. & Soomere, T. Energy pollution: the relative influence of wind-wave and vessel-wake energy in Tallinn Bay, the Baltic Sea. J. Coast. Res. Special Issue 56, 812u2013816 (2009)._ Soomere, T., Parnell, K. E. & Didenkulova, I. Implications of fast-ferry wakes for semi-sheltered beaches: a case study at Aegna Island, Baltic Sea. J. Coast. Res. Special Issue 56, 128u2013132 (2009)._ Mortensen, S.B. et al. An improved integrated approach for optimizing shipping channel capacity for Australian ports. Terra Aqua 21u201323 (2017).Parnell, K. E., Mcdonald, S. C. & Burke, E. Shoreline effects of vessel wakes, Marlborough Sounds, New Zealand. J. Coast. Res. Special Issue 50, 502u2013506 (2007)._ Ordinanza della Capitaneria di Porto (n.175/09 Ministero delle infrastrutture e dei trasporti Capitaneria di Porto-Guardia Costiera Venezia).Tosi, L., Teatini, P. & Strozzi, T. Natural versus anthropogenic subsidence of Venice. Sci. Rep. 3, 1u20139 (2013)._ _ Zonta, R. et al. Sediment chemical contamination of a shallow water area close to the industrial zone of Porto Marghera (Venice Lagoon, Italy). Mar. Pollut. Bull. 55, 529u2013542 (2007)._ _ _ _ Molinaroli, E. et al. Relationship of morpho-sedimentological variations to the fate of Hg- and Zn-polluted sediments in the contaminated site of Porto Marghera, Lagoon of Venice, Italy. J. Environ. Prot. 4, 37u201349 (2013)._ _ Teatini, P. et al. Hydrogeological effects of dredging navigable canals through lagoon shallows. A case study in Venice. Hydrol. Earth Syst. Sci. 21, 5627u20135646 (2017)._ _ _ Soomere, T. Nonlinear ship wake waves as a model of rogue waves and a source of danger to the coastal environment: a review. Oceanologia 48(S), 185u2013202 (2006)._ Sarretta, A. et al. Bathymetry of Lagoon of Venice, 2002. figshare, .\"- - This project is partially supported by the European Regional Development Fund program Mobilitas Plus, No 2014-2020.4.01.16-0024, the Estonian Research Council grant IUT333, and by an Italian Ministry of Education and Research grant. The authors wish to thank Loris Dametto of CNR-ISMAR for the support in the field activities. LANDSAT data are available from the U.S. Geological Survey. Sentinel2 data courtesy of ESA (European Space Agency).- - Authors and Affiliations - \"UniversitxE0 Ca Foscari, Dipartimento di Scienze Ambientali, Informatica e Statistica, Venezia, 30175, ItalyGian Marco Scarpa_&_Emanuela MolinaroliConsiglio Nazionale delle Ricerche, Istituto di Scienze Marine, Venezia, 30122, ItalyGian Marco Scarpa,_Luca Zaggia,_Giorgia ManfxE8_&_Giuliano LorenzettiTallinn University of Technology, Department of Cybernetics, Tallinn, EstoniaKevin Parnell_ &_Tarmo SoomereSacred Heart University, Department of Biology, Fairfield, CT, 06825, United States of AmericaJohn Rapaglia\" - \"AuthorsGian Marco ScarpaYou can also search for this author in _Luca ZaggiaYou can also search for this author in _Giorgia ManfxE8You can also search for this author in _Giuliano LorenzettiYou can also search for this author in _Kevin ParnellYou can also search for this author in _Tarmo SoomereYou can also search for this author in _John RapagliaYou can also search for this author in _Emanuela MolinaroliYou can also search for this author in _\" - Contributions - G.M.S. wrote large sections of the paper and was responsible for data curation, formal analysis and all investigations. G.L. and G.M. performed field and GIS investigations and methodology development. L.Z. and E.M. contributed to the conceptualization and writing of the paper and managed data validation, project supervision and funding. T.S. wrote parts of the manuscript that discuss wave properties and propagation. K.P., T.S., J.R. contributed to data validation, and handled review and editing of the manuscript. All authors discussed and commented on the methods, results and the content of the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International License.- - Author notes - Bannock James H. and Treat Neil D. contributed equally to this work. - Authors and Affiliations - \"Centre for Plastic Electronics, Imperial College London, London, SW7 2AZ, UKJames H. Bannock,_Neil D. Treat,_Natalie Stingelin,_Martin Heeney_&_John C. de MelloDepartment of Chemistry, Imperial College London, London, SW7 2AZ, UKJames H. Bannock,_Martin Heeney_&_John C. de MelloDepartment of Materials, Imperial College London, London, SW7 2AZ, UKNeil D. Treat_&_Natalie StingelinMaterials Research Laboratory, University of California Santa Barbara, Santa Barbara, 93117, California, USANeil D. Treat_&_Michael Chabinyc\" - \"AuthorsJames H. BannockYou can also search for this author in _Neil D. TreatYou can also search for this author in _Michael ChabinycYou can also search for this author in _Natalie StingelinYou can also search for this author in _Martin HeeneyYou can also search for this author in _John C. de MelloYou can also search for this author in _\" - Contributions - J.B. prepared and analysed the polymers. N.T. fabricated and characterised the OPV devices. J.B. and J.d.M. wrote the main manuscript text. J.B., N.T., M.C., N.S., M.H. and J.d.M. reviewed and made amendments to the manuscript.- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing financial interests. - \\' \\'- - \\'\\'- - \\' \\' - \"This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution 4.0 International License]. c Formation of silicene between graphene bilayers. [Reprinted figure with permission from ref. . Copyright (2019) American Chemical Society]. d Atomic structures of ripplocations with different numbers of extra units added on top MoS2 layer. [Adapted with permission from ref. . Copyright (2019) American Chemical Society]. e Histograms corresponding to averaged interlayer distance without and with K ions [Reprinted from ref. . Copyright (2019), with permission from Elsevier].\\' - \\' \\' - \"Silicene and phosphoreneu2014Silicene and black phosphorene (BP) are the silicon and phosphorus analogs of graphene, respectively. A favorable route for silicene formation has been proposed by employing the graphene bilayer as a template (Fig. ). It was found that vacancy defects reduce the thermal stability of silicene, where the critical temperature reduces by more than 30%. A P/H parameter set has been developed for monolayer BP to examine the mechanical, thermal, and chemical stability of both pristine and defective BP.\" - \\' \\' - \"Molybdenum disulfideu2014The ReaxFF potential for 2D MoS2 systems was introduced in ref. . This potential could precisely calculate the formation energy of ripplocations in MoS2 (Fig. ), which result from the slippage of the upper layer against the lower layer to relax the strain without breaking covalent bonds. This potential was also trained over the vacancy formation energies, diffusion barriers, bending rigidity, and kinetics. It has also been applied to design S vacancy defects on MoS2 surface. Reactive MD simulations have also been used to model the synthesis of MoS2 layers via sulfidation of MoO3 surfaces. These simulations identified a three-step reaction pathway as follows: (i) evolution of O2 and reduction of MoO3 surface; (ii) S2-assisted reduction and formation of SO/SO2; and (iii) sulfidation of the surface via Mo-S bond formation. The ReaxFF MD simulations have also been employed within a multiscale framework to capture thermal properties of MoS2. The predicted thermal conductivity of MoS2 structures with infinite length (37u2009xB1u20093u2009W/mK) falls within the range of experimental results (34.5u2013110u2009W/mK).\" - \\' \\' - \"MXenesu2014MD simulations with ReaxFF of MXenes revealed that intercalating with potassium cations drastically improves water stability and homogeneity of MXenes, and reduce the self-diffusion coefficient of water by two orders of magnitude. The dynamical response of MXenes with different surface terminations to intercalating ions was studied with DFT calculations and ReaxFF MD simulations. Figure compares the interlayer distance of Ti3C2(OH)2 MXene bilayer with and without K ions. Influence of metal ions intercalation on vibrational properties of water molecules trapped between MXene layers was studied using ReaxFF MD simulations. Widening of the interlayer gap may enable the penetration of molecular reactants such as urea, which decomposes readily and leads to the intercalation of ammonium cations. ReaxFF simulations were performed for defect formation and homoepitaxial growth of TiC layer on Ti3C2 MXene structures.\" - \\' \\' - REBO/AIREBO potentials - \\' \\' - The REBO potential (( {E_{ij}^{mathrm {REBO}}} )) is a combination of attractive (( {V_{ij}^A} )) and repulsive (( {V_{ij}^R} )) interactions with certain ratio (bij), (E_{ij}^{mathrm {REBO}} = V_{ij}^R + b_{ij}V_{ij}^A). The bond-order interaction ratio, bij, is dependent on the local coordination in the atomic environment. The REBO potential can define the conjugation of bonds using bij, and therefore the potential has been widely used for modeling hydrocarbon systems. The repulsive term is expressed by the Brenner equation, - \"$$V_{ij}^R = w_{ij}left( {r_{ij}} right)left[ {1 + frac{{Q_{ij}}}{{r_{ij}}}} right]A_{ij}e^{ - alpha _{ij}r_{ij}}.$$ (16) \" - \\' \\' - \"Here, the Qij, rij, and u03B1ij parameters depend on i and j. The bond weighting parameter wij(rij) depends on switching function S(t) as\" - \"$$w_{ij}left( {r_{ij}} right) = Sleft( {t_cleft( {r_{ij}} right)} right),$$ (17) \" - \"$$Sleft( t right) = Theta left( { - t} right) + Theta left( t right)Theta left( {1 - t} right)0.5left[ {1 + cos left( {pi t} right)} right],$$ (18) \" - \"$$t_cleft( {r_{ij}} right) = frac{{r_{ij} - r_{ij}^{min}}}{{r_{ij}^{max} - r_{ij}^{min}}}.$$ (19) \" - \\' \\' - The attractive interaction is expressed as - \"$$V_{ij}^A = - w_{ij}left( {r_{ij}} right)mathop {sum}limits_{n = 1}^3 {B_{ij}^{(n)}} e^{B_{ij}^{(n)}r_{ij}};$$ (20) \" - the bond-order interaction ratio bij is - \"$$b_{ij} = frac{1}{2}left[ {p_{ij}^{sigma pi } + p_{ji}^{sigma pi }} right] + pi _{ij}^{rc} + pi _{ij}^{dh}.$$ (21) \" - \\' \\' - \"Here, (p_{ij}^{sigma pi }) and (p_{ji}^{sigma pi }) are not necessarily equal as they depend on the penalty function (gi) of bond angle u03B8jik between the vector rij, and vector rki, i.e.,\" - \"$${p}_{ij}^{{upsigma uppi }} = left[ {1 + {sum limits_{{k} ne {i},{j}}} {{w}_{ik}} ({r}_{ik}){g}_{i}({cos} {uptheta}_{jik}){e}^{{uplambda }_{jik}} + {P}_{ij}} right]^{ - 1/2}.$$ (22) \" - \\' \\' - The AIREBO potential is the modification over the REBO potential, which also considers the torsional (( {E_{kijl}^{mathrm {TORSION}}} )) and LJ (( {E_{ij}^{lj}} )) interactions. The integration of pairwise interactions in AIREBO potential can be represented by the following equation - \"$$E = frac{1}{2}mathop {sum }limits_i mathop {sum }limits_{j ne i} left[ {E_{ij}^{mathrm {REBO}} + E_{ij}^{lj} + mathop {sum }limits_{k ne i,j} mathop {sum }limits_{l ne i,j,k} E_{kijl}^{mathrm {TORSION}}} right].$$ (23) \" - \\' \\' - The REBO potential can describe the breaking, formation, and hybridization of covalent bonds. The REBO and AIREBO potentials have been used to study the growth of 2D amorphous carbon, mechanical properties of MoS2, and even thermal stabilities of C60 2D nanostructures using atomic coordination, internal stress, and mass density analysis. The subplantation phenomenon during CVD, i.e., penetration of the substrate bulk carbon by carbon atoms, can be explained by this REBO potential. REBO potential is also suitable for modeling and analysis of TMD materials such as MoS2. The potential results in stabilized structure and good agreement of the structural and mechanical properties. It was also used for modeling the synthesis of diamane from graphene and its stability under different conditions. - \\' \\' - COMB3 potential - \\' \\' - The general formation of the COMB3 potential can be expressed as - \"$$U^{mathrm {tot}}left[ {left{ q right},left{ r right}} right] = U^{mathrm {es}}left[ {left{ q right},left{ r right }} right] + U^{mathrm {short}}left[ {left{ q right},left{ r right}} right] + U^{mathrm {vdW}}left[ {left{ q right}, left{ r right}} right] + U^{mathrm {corr}}left[ {left{ q right },{ r} } right]$$ (24) \" - \\' \\' - The total potential energy consists of electrostatic energy (left( {U^{mathrm{es}}left[ {left{ q right},left{ r right}} right]} right)), short-range interactions (left( {U^{mathrm {short}}left[ {left{ q right},left{ r right}} right]} right)), long-range vdW interaction (left( {U^{mathrm {vdW}}left[ {left{ q right},left{ r right}} right]} right)), and the correction terms. Here, q and r represent the charge and coordinate array of the elements. The electrostatic energies (left( {U^{mathrm {es}}left[ {left{ q right},left{ r right}} right]} right)) can be represented as - \"$$U^{mathrm {es}}left[ {left{ q right},left{ r right}} right] = U^{mathrm {self}}left[ {left{ q right},left{ r right }} right] + U^{qq}left[ {left{ q right},left{ r right}} right] + U^{qZ}left[ {left{ q right},left{ r right}} right] + U^{mathrm {polar}}left[ {left{ q right},{ r} } right]$$ (25) \" - \\' \\' - The self-interaction energy (left( {U^{mathrm {self}}left[ {left{ q right},left{ r right}} right]} right)) consists of field-effect and ionization energy, or the affinity energy. Charge-to-charge interaction (left( {U^{qq}left[ {left{ {q1s} right},left{ r right}} right]} right)) is associated with the charge density distribution functions. Nuclear-charge interaction (U^{qZ}left[ {left{ q right},left{ r right}} right]) depends on the coulombic interaction between charge density distributions in the system. Polar interaction energy (U^{mathrm {polar}}left[ {left{ q right},{ r} } right]) depends on the charge interactions and dipole distributions in response to the external electric field. - \\' \\' - The charge-dependent short-range interaction in Eq. () is associated with bond distance functions of the atom and the associated charge functions. This energy term can be expressed by - \"$$U^{mathrm {short}}[ {{ q },{ r }} ] = mathop {sum }limits_i mathop {sum }limits_{j > i} V_{ij}^{mathrm {bond}} = mathop {sum }limits_i mathop {sum }limits_{j > i} { {F_c( {r_{ij}} )[ {V^R( {r_{ij},q_i,q_j} )} ] - b_{ij}V^A(r_{ij},q_i,q_j)} }$$ (26) \" - \\' \\' - Here, (V_{ij}^{mathrm {bond}}) is the bond energy that is associated with pairwise attraction (left( {V^A(r_{ij},q_i,q_j)} right)) and pairwise repulsion energy ((V^R(r_{ij},q_i,q_j))). They exponentially decrease with the interatomic distance rij. Fc(rij) is the cutoff function. - \\' \\' - The long-range interaction energy is defined by the vdW interaction, which can be expressed by the LJ interaction formula - \"$$U^{mathrm {s}}left[ {left{ q right},left{ r right}} right] = mathop {sum }limits_i mathop {sum }limits_{j > i}^{NN} 4 varepsilon _{ij}^{mathrm {vdW}}left[ {left( {frac{{sigma _{ij}^{mathrm {vdW}}}}{{r_{ij}}}} right)^{12} - left( {frac{{sigma _{ij}^{mathrm {vdW}}}}{{r_{ij}}}} right)^6} right]$$ (27) \" - \\' \\' - Here, (sigma _{ij}^{mathrm {vdW}}) and (varepsilon _{ij}^{mathrm {vdW}}) are the equilibrium distance and interaction energy, respectively; rij is the cutoff radii where the interaction assumes zero. - \\' \\' - The correction factor in Eq. () adjusts the energy penalties associated with specified angles. It can be calculated by a sixth-order Legendre polynomial and bond bending term. - \"$$U^sleft[ {left{ q right},left{ r right}} right] = mathop {sum }limits_i mathop {sum }limits_{j > i}^N mathop {sum }limits_{k ne i}^N left{ {mathop {sum }limits_{n = 1}^6 K_{ijk}^{LP_n}left( {cos left( {theta _{ijk}} right)} right) + K_{ijk}^{BB}left[ { cos left( {theta _{ijk}} right) - cos left( {K_{ijk}^theta } right)} right]^2} right}$$ (28) \" - \\' \\' - \"Here, (K_{ijk}^theta) indicates the specific bond angles and u03B8ijk is the bond angle.\" - \\' \\' - The COMB3 potential has been introduced to study the CVD process for various 2D materials. The charge-optimized COMB3 potentials have been used to simulate the experimental CVD deposition and growth of graphene on metal substrates. Especially, the introduction of dihedral interaction terms in the COMB3 potential can capture the delocalized bonding and bond bending during the CVD process on a particular substrate. The wrinkle formation due to the size mismatch of graphene and Cu-substrate has been studied using COMB3 potential, which was comparable to experimental results. This potential was also used to understand the role of growth parameters such as absorption energy, migration barrier, and temperature in the Cu deposition on ZnO substrate. COMB3 potential has also been used to investigate the effect of surface hydroxylation of amorphous SiO2 substrate on heat conduction of supported graphene. - \\' \\'- - Phase-field model - At mesoscale, a simple and effective model for 2D materials growth is the classical Burton-Cabrera-Frank (BCF) theory, where the island growth is realized by the deposition of adatoms from the supersaturated gas atmosphere and the incorporation of diffusing atoms at the step edge. The direct numerical implementation of the BCF model requires tracking the evolving step edges and applying the boundary conditions due to the sharp-interface nature of the model; in contrast, phase-field diffuse-interface models avoid the explicit tracking of the interfaces. - \"The BCF-based phase-field models have been developed since the 1990s for a series of problems related to crystal growth, including collective step motions in a 1-D step train, spiral surface growth during thin-film epitaxy, and step flow under different kinetic regimes. More recently, the combined effects of edge diffusion, the Ehrlich-Schwoebel barrier, deposition, and desorption for epitaxial growth have been investigated in 2-D. Based on these studies, Meca et al. applied the model to the growth of 2D materials with anisotropic diffusion on substrates, using the epitaxial graphene growth on copper foil during CVD as an example, with experimental validations. The typical BCF-based phase-field model for 2D materials growth uses a smoothly varying order parameter u03D5 to distinguish the phases, e.g., u03D5u2009=u20091 in the 2D island, u03D5u2009=u2009 u22121 in the substrate, and u22121u2009<u2009u03D5u2009<u20091 at step edges. The free energy functional u2131 of the system has the general form of:\" - \"$${cal{F}} = mathop {int}limits_V {(f(phi )} - lambda ,u,g( phi ) + frac{1}{2}W(theta )^2|nabla phi |^2)dV$$ (29) \" - \"where the first term f(u03D5) is a double-well (or multi-well, e.g., see refs. ) function with global minima at u03D5u2009=u20091 and u03D5u2009=u2009 u22121; the second term describes the coupling with the reduced saturation field u, with g(u03D5) an interpolation function and u03BB the coupling coefficient; the third term is the gradient energy due to the inhomogeneous distribution of u03D5 at interfaces, where W(u03B8) is the angle-dependent interface thickness with (theta = arctan( {phi _y/phi _x})). u can be directly related to the adatom concentration c through (u = {mathrm{Omega }}( {c - c_{eq}^0} )), where (c_{eq}^0) is the equilibrium concentration for a flat interface and u03A9 is the atomic area of the solid phase. The evolution of u03D5 is governed by\" - \"$$begin{array}{*{20}{l}} {tau _phi frac{{partial phi }}{{partial t}}} hfill = hfill { - frac{{delta {cal{F}}}}{{delta phi }} = - f ^prime (phi ) + lambda u g ^{prime} (phi ) + nabla cdot left( {W(theta )^{2}nabla phi } right)} hfill {} hfill {} hfill -{partial _xleft( {W(theta ) cdot W ^{prime} (theta ) cdot partial _yphi } right) + partial _yleft( {W(theta ) cdot W ^{ prime} (theta ) cdot partial _xphi } right),} hfill end{array}$$ (30) \" - \"where u03C4u03D5 is the characteristic time. The evolution of the saturation field u is governed by\" - \"$$frac{{partial u}}{{partial t}} = Dnabla ^2u + F_{mathrm {d}} - frac{u}{{tau _s}} - frac{1}{2}frac{{partial phi }}{{partial t}}$$ (31) \" - \"where D is the diffusion coefficient, Fd is the effective deposition rate, and u03C4s is the characteristic time scale for atom desorption. The phase-field equations (Eqs. () and ()) can recover the sharp-interface BCF model in the limit of W(u03B8)u2009u2192u20090.\" - The applications of the phase-field models in 2D materials growth can be categorized into the following three aspects. (1) Reproducing and explaining experimentally observed growth morphologies. Examples include the reconstruction of the spiral growth of SnSe2, the verification of impurity-induced bilayered graphene growth, and the explanation to island shape change from quasi-hexagons to triangles and dendrite-like morphologies. (2) Investigating the effect of experimental parameters on growth morphologies. Taking the CVD growth of graphene as an example, the effects of flux of carbon species, deposition rates, and equilibrium saturation, and substrate orientation on graphene island morphologies have been investigated using phase-field simulations, which could provide useful guidance to the experimental control of the growth qualities. (3) Further development of the model to include the effect of additional physical/chemical processes during growth. In addition to the deposition, edge diffusion, and edge anisotropies, phase-field simulations have been extended to model multi-island interactions, tilt GB topology of 2D materials on substrates with topological cones, and to include the chemical reaction kinetics in the gas phase using a microkinetic model. These works improve the fidelity and applicable range of the phase-field model for 2D materials growth. - \"Nevertheless, a primary obstacle to applying phase-field models for computational synthesis of 2D materials is the lack of connection between the model parameters and the experimentally controllable CVD processing parameters, which is largely due to the difference in size scales of interest. Most of the existing phase-field simulations only focus on the growth of a few 2D islands (up to micrometer scale), which can hardly represent the effect of the macroscopic CVD parameters. One possible solution is to enable the u201Cmultiscaleu201D modeling approach, integrating the FEM (see the section on u201CMacroscale modelsu201D) to account for the effect of macroscopic CVD parameters on transport phenomena and the mesoscale phase-field model for 2D crystal growth within selected zones of the whole substrate. In particular, the steady-state velocity, concentration, and temperature profiles from FEM can be incorporated in the phase-field model, as it has been proposed and applied by Momeni et al. (see Fig. ). The incorporation of chemical reaction kinetics in the phase-field model can improve the accuracy of the predictions and expand its applicability. Phase-field models have also been applied to phase changes in 2D materials. Notable examples include the multidomain microstructure during H-T structure transformation in MoTe2 and the domain switching behaviors under external stimuli. A fast Fourier transform algorithm has been implemented to study the domain switching during bending of MoTe2 (see Fig. ).\" - \"Fig. 2: A coupled macroscale/phase-field simulation and experimental comparison for MoS2.a Initial precursor concentration distribution from FEM. b Phase-field simulation results for MoS2 morphology on substrate at tu2009=u20090.25u2009 h, the enlarged images of iu2013iii boxes are shown in du2013f. c Experimental results for deposition of MoS2 after deposition for tu2009=u20090.25u2009 h, the enlarged images of 1u20133 boxes are shown in g, h. Reprinted from ref. under Creative Commons Attribution 4.0 International License. The scale bar in a, b shows 400u2009u03BCm and the one in du2013f shows 400u2009u03BC m.\" - \"Fig. 3: Variant selection and transformation morphologies in MoTe2 under an applied in-plane strain (bar varepsilon _{ii}).a Phase diagram in (left( {bar varepsilon _{xx},bar varepsilon _{yy}} right)) space. Colored (hatched) regions correspond to one (two) phase regions. Solid lines (analytic) and filled data points (simulations) indicate the strain at which H and a given Tu2032 variant have equal energies. Dashed lines and open symbols similarly represent two-phase coexistence bounds. (u25B2, u25B3) and (u25CF, u25CB ) correspond to simulations with and without bending, respectively, and blue circles (blue solid circle and blue open circle) represent DFT results at 0. bu2013 d Images of the microstructures with increasing time/strain from left to right for strain paths bu2013d in phase diagram a. Strain values (left( {bar varepsilon _{xx},bar varepsilon _{yy}} right)) are given below each panel. e Corresponding maximum principal stress (GPa) maps for microstructures in d. All simulations are for 50u2009u03BCmu2009xD7u200950u2009u03BC m simulations. Adapted with permission from ref. . Copyright (2017) American Chemical Society.\" - Phase-field-crystal model - Different from the continuous mesoscale phase-field approach, the phase-field-crystal (PFC) approach describes the thermodynamics and dynamics of phase transformations through an atomically varying order parameter related to the atomic density field. With the application of the classical dynamic DFT, PFC can capture the atomistic-scale morphology and evolution dynamics, within diffusional time scales, which is usually challenging for conventional atomistic methods. Recently, PFC has been actively applied to investigate the defect formation and the atomic structure of interfaces, GBs, and triple junctions during the growth of 2D materials. The unique GB structure and collective domain dynamics in binary 2D materials, e.g., h-BN, was investigated (see Fig. ). - \"Fig. 4: PFC simulations of the grain growth and coalescence in h-BN.au2013f Grain coalescence and inversion domain dynamics. c Atomic site number N vs. t, showing two regimes Nu2009=u2009u22120.065tu2009+u20093.045u2009xD7 u200910 and Nu2009=u2009u22120.085tu2009+u20093.612u2009xD7u200910 via fitting. d, e Domain shrinking in the white boxed region of b. f Transient of three merging heart-shaped defects before their annihilation. gu2013i Time evolution of collective atomic displacements in the yellow boxed corner of d. Adapted figure with permission from ref. . Copyright (2017) by the American Physical Society.\" - Compared with the continuous phase-field approach, the advantage of PFC lies in resolving atomistic-scale interface structures and growth dynamics of 2D islands; however, due to the limitation of simulation size, interface dynamics under the effect of realistic CVD parameters are difficult to realize. Future directions of PFC simulations for 2D materials growth include the consideration of atomistic-scale island-substrate interactions and the integration with the larger-scale phase-field or FEM simulations. - Kinetic Monte Carlo - KMC is another mesoscale technique for predicting the growth morphology and kinetic mechanisms of 2D materials, where all possible kinetic events are listed in an event catalog, and their stochastic sequence are randomly selected based on their activation energies. Activation energies of the related kinetic processes are usually obtained from atomistic-scale calculations. KMC identifies the governing kinetic pathways and simulates the kinetic process. Details of its implementation can be found in ref. . - \"KMC has been extensively applied to simulate the growth of graphene, e.g., the evolution of vacancy complexes and formation of vacancy chains, the formation and kinetic effect of multi-member carbon ring complexes, etching of graphene GBs due to oxygen migration and reaction, and GB evolution following the Stone-Wales mechanism. Growth of nanoscale graphene islands from carbon monomer nuclei or pre-existing growth fronts has been captured, showing catalytic growth behaviors, anisotropic morphological patterns, temperature- and deposition-flux-rate-dependent size and shapes, inhomogeneous and nonlinear growth kinetics due to lattice mismatch, and geometry-determined growth mechanisms. The u201Ccoastlineu201D graphene morphology during sublimation and the step-flow growth of epitaxial graphene have also been reported. Based on KMC simulations for graphene growth on Cu (111) with and without hydrogen, the growth protocol was designed for bilayer growth and N-doped graphene growth.\" - \"Regarding TMDs, the KMC simulations for the growth of WSe2 monolayer on graphene have been used to develop the phase diagram of domain morphologies as a function of flux and precursor stoichiometry (Fig. ). It was found that the fast kink nucleation and propagation, rather than edge attachment and diffusion, could lead to ultrafast growth of monolayer WSe2. KMC simulations also guided CVD growth of large-scale WSe2 grains by controlling the three-stage adsorptionu2013 diffusionu2013attachment mechanisms. KMC model is also a key component in a more generalized mechanistic model for growth morphology predictions of 2D materials.\" - \"Fig. 5: KMC simulations of WSe2 growth.a Reaction energy diagram of the growth process based on DFT calculations. The simulation starts with adatoms. They react with each other to form the TMD domains. After bonded into the domain, the in-flake atoms are mobile through edge diffusion and vacancy diffusion. Each state is defined by its state energy, and each process is defined by its transition activation energy. Adapted from ref. under Creative Commons Attribution 4.0 International License. b Domain morphology diagram on the metal flux-C/M ratio plane at 973u2009K from KMC simulations. Five regions are identified: I: no growth, II: quasi-equilibrium compact domain, III: fractal, IV: dendrite, and V: semi-compact domain. The arrows show a proposed scenario to obtain high-quality domains with high growth rate. Adapted from ref. .\" - In summary, the KMC model can be a useful tool for investigating the kinetic pathways and morphologies during the growth of 2D materials. However, the probability-based nature of KMC makes it most suitable for cases where atomic fluctuations are high, i.e., the atomistic and nanoscale morphology and kinetics of 2D materials. For larger-scale simulations, due to the significantly increased system size and the disparate rates of different KMC events, a full-KMC is computationally expensive. As a compromise, multiscale KMC has been developed; simplifications should also be made to account for the key events that are most relevant to the large-scale kinetics. Under such situations, the phase-field approach could be a more efficient option. Meanwhile, the KMC model can provide the governing kinetic mechanisms for phase-field simulations to improve the validity and accuracy of the simulation.- - \\'Although the growth of 2D materials occurs at the nano- and mesoscale, it is controlled by physics and parameters that have a macroscopic nature, e.g., heat and mass transfer, furnace configuration, and gas-phase reactions. Thus, having a thorough understanding of the macroscale physics and processes is essential for controlling the growth of 2D materials and their synthesis by design. We can classify the macroscale models of the growth chamber into four groups: (i) experiment-based models, where rate equations and their constants used to describe the growth are determined from experiments; (ii) analytical models, where the governing equations are simplified and solved analytically; (iii) adaptive models, where a set of experiments are used to train the model; and (iv) multiphysics models, where the coupled system of governing equations at different length and temporal scales are solved numerically. Among these methods, the last group of models have key advantages, providing a profound understanding to the growth process, flexibility to apply to different growth conditions, and the ability to optimize the process.\\' - A practical macroscale model of the growth chamber should capture the critical governing physics, e.g., the heat and mass transports and chemical reactions. Setting up these models requires several key information that can be obtained from lower scale simulations or experiments. Identifying the gas-phase reactions, we may approximate some of the reaction parameters using classical theories, e.g., the collision rate to estimate the chemisorption rate of species or the group contribution methods to determine the diffusion coefficient. The other approximation is for low concentrations of reactive species, where the change in pressure and heat of reaction can be neglected. In the latter case, we may decouple the fluid and heat transfer of gaseous materials from the mass transfer and kinetics. In contrast, for high concentrations of reactive species in the gas phase, such as in metalorganic chemical vapor deposition (MOCVD), the coupled system of equations must be solved. A summary of main equations in macroscale models are presented below. - Gas flow - The Navier-Stokes equation governs the flow rates in the growth chamber, - \"$$rho frac{{partial {mathbf{u}}}}{{partial t}} + rho left( {{ mathbf{u}} cdot nabla } right){mathbf{u}} = nabla cdot left[ { - p{mathbf{I}} + mu left( {nabla {mathbf{u}} + left( {nabla { mathbf{u}}} right)^T} right)} right] + {mathbf{F}};quad quad rho nabla cdot {mathbf{u}} = 0,$$ (32) \" - \"where u is the velocity field, p is pressure, I is the unit matrix, u03BC is the dynamic viscosity, and F is the volumetric applied force, i.e., weight. Furthermore, the density u03C1 is a function of temperature and precursor concentration. In general, the concentration of precursor in the gas phase is low and u03C1 can be assumed to be only a function of temperature.\" - Heat transfer - The two main heat transfer mechanisms in the growth chamber are convection and conduction that respectively are - \"$$Q = rho C_Pfrac{{partial T}}{{partial t}} + rho C_P{mathbf{u}} cdot nabla T + nabla cdot q;,,,q = - knabla T,$$ (33i, ii) \" - where Q is the heat source, CP is the heat capacity at constant pressure, T is the temperature, q is the heat flux vector, and k is the thermal conductivity coefficient. - Mass transfer - Commonly, the concentration of precursor materials in the growth chamber is negligible. Thus, the equation governing the mass transfer can be formulated with the flow-assisted diffusion of dilute species - \"$$R = frac{{partial c}}{{partial t}} + nabla cdot left( { - Dnabla c} right) + {mathbf{u}} cdot nabla c;,,,N = - D,nabla c + {mathbf{u}}c,$$ (34i, ii) \" - \"where R is the source term for precursor, c is the precursor concentration, D is the diffusion coefficient, and N is the flux of the precursor. Equations ()u2013() must be solved for a set of initial and boundary conditions, as shown in Table .\" - Table 6 Common boundary conditions for the heat and mass transfer in macroscale models. - Chemical reaction - Consider a set of j reactions involving i species of the form (aA + bB + ldots rightleftharpoons xX + yY + ldots) with forward and reverse reaction rate constants (k_j^f) and (k_j^r), respectively. These reaction rates depend on the temperature via an Arrhenius expression, e.g., (k_j^f = A_j^fleft( {T/T_{{mathrm{ref}}}} right)^n{mathrm{exp}}left( { - E_j^f/R_gT} right)). Here, (A_j^f) denotes the frequency factor, n the temperature exponent, (E_j^f) the activation energy, and Rg the gas constant. According to the mass action law, the reaction rates rj can be described as, - \"$$r_j = k_j^fmathop {prod }limits_{i, in ,{mathrm{react}}} c_i^{ - v_{ij}} - k_j^rmathop {prod }limits_{i, in ,{mathrm{prod}}} c_i^{ - v_{ij}},$$ (35) \" - where ci is the concentration of i-th species and vij is the stoichiometric coefficient (negative for reactants and positive for products). The rate equations provide the information regarding the concentration of different species, which in combination with mass and heat balance equations form the complete system of equations. - Macroscale models provide insight into the growth mechanisms and flow regimes, and play a crucial role in determining proper furnace design and growth parameters. Figure shows the application of a macroscopic model of the flow and concentration diffusion in a vertical furnace with a side inlet to study the effect of the gap between the susceptor and substrate. It shows that the presence of the gap results in the formation of eddy currents close to the substrate edge, which disturbs precursor concentration and hinders the formation of 2D materials that is consistent with the experimental observations. These models have been successfully used in determining the mechanisms governing the growth of 2D materials. For example, it was revealed that the concentration gradient of the precursor is a critical factor in determining the growth morphology of 2D materials, where a low concentration gradient along the substrate leads to the formation of multigrain 2D films. In contrast, moderate planar concentration gradients lead to isolated islands of varying morphologies, and an out-of-plane gradient leads to the formation of standing 2D materials. Furthermore, macroscale models can be combined with lower scale models to form a multiscale platform with high fidelity, e.g., ref. . - \\'Fig. 6: Macroscale model of the flow and precursor concentration in a vertical MOCVD furnace.a Snapshots of the concentration profile within the growth chamber with a heated susceptor where the precursor enters from top left side of the furnace. It indicates the effect of buoyancy forces on inducing instabilities and back flow formation. b, c Magnified views of the flow and concentration profiles close to the susceptor/substrate. Eddy current forms when there is a gap between the susceptor and substrate which hinders formation of 2D materials.\\'- - Data-driven methods have been employed to satisfy the urgent need for theoretical development of 2D materials. The improvement of computational power has resulted in extensive collections of data and scientifically organized databases, which can be used by methods such as data mining and machine learning, to accelerate the computational design of new 2D materials. - Database of 2D materials - Compared with the databases of 3D bulk materials such as the Crystallographic Open Database, the Inorganic Crystal Structure Database, and the Cambridge Crystallographic Data Centre, the database focused on 2D materials is still under development. Several 2D materials databases have been attempted primarily based on a high-throughput approach coupled with first-principles and quantum theory to address this shortcoming. Besides, data-mining methods have also been applied to screen those bulk materials databases and identify possible 2D materials from 3D structures. These approaches can provide various information about 2D materials that have been or not been synthesized in experiments. For example, using a criterion to identify 2D materials, a database has been developed on the basis of experimental and DFT data. This database provides information including exfoliation energy, bandgap, work function, and elastic-constant values of various 2D materials, and also predicted hundreds of new 2D materials for the experimental synthesis. DFT and many-body perturbation theory have been used to generate the Computational 2D Materials Database (C2DB). The C2DB contains structural, thermodynamic, elastic, electronic, magnetic, and optical information of about two thousand 2D materials in more than 30 crystal structures. A few hundred new monolayer structures that were previously unknown, but potentially synthesizable were also identified. A data-mining method has also been used to screen the bulk materials from the open database and identified a few hundred 2D weakly bonded solid materials. The band gaps and point groups that determine piezoelectric and nonlinear optical properties in these materials were also provided. A search in the database of bulk materials was performed and DFT calculations were used to explore these materials for possible exfoliation, where more than one thousand materials have been identified. Information including electronic, vibrational, magnetic and topological properties of promising materials was also provided. Similarly, the crystal database was screened for layered motifs, and hundreds of stable layered materials were found that can be considered as candidates for the formation of 2D monolayers via exfoliation. - Machine learning method - Machine learning models trained using 2D material databases can be parameterized, validated, and therefore used for predictive exploration of novel 2D materials with desired properties. Machine learning methods can also aid the understanding of the complex correlations between structures and properties in 2D materials. A combination of machine learning with MD simulations and in-situ high-resolution transmission electron microscopy to explore TMDs has been performed, where machine learning provided information for structural optimization and evolution of defects to help understand the structural transition in 2D TMDs. A practical method to explore hybrid 2D materials was developed by coupling machine learning with DFT calculations. The structural and electronic properties of different hybrid 2D materials were provided and various parameters for vdW heterostructures were screened. A machine learning model with force-filed-inspired descriptors in material screening for complex systems has been introduced and used to discover exfoliated 2D-layered materials. An artificial neural network for titanium dioxide systems was trained based on the a DFT calculated database, where a novel quasi 2D titanium dioxide structure was revealed. Similarly, new 2D materials with high magnetic moments were found using a machine learning model trained by first-principles data. Machine learning methods have also been used to aid the development of force fields for classical simulations of materials. For instance, a force field for classical simulations of stanene was developed using a machine learning method trained by data sets from ab-initio results to calculate the mechanical and thermal properties of stanene. 2D materials without bulk layered counterparts are also being discovered, using genetic algorithms or particle swarm optimization, both using energy-based merit criteria and additional biases towards 2D sheets (see refs. ).- - \"This overview summarizes the state-of-the-art of modeling efforts on the growth of 2D-layered materials and outlines the advantages and limitations of computational models and methods at different length and time scales. The emphasis is on modeling, understanding, and predicting the thermodynamics and kinetics of 2D materials growth. The eventual goal is to develop computational tools and techniques to realize the u201Csynthesis by designu201D of 2D materials.\" - An ideal computational model should not only capture accurately the correct physics to allow robust designs but also be computationally efficient to be employed for active control of the growth process. The complex nature of the growth of 2D materials requires applications of a range of models, which operate at multiple length and temporal scales to capture the full range of the growth phenomena from macroscale flow and concentration profiles in the furnace to atomistic reactions leading to formation and growth of 2D materials at mesoscale. Modeling the full spectrum of mechanisms involved in the growth of 2D materials is challenging due to the limitations associated with each of the computational models at different length and temporal scales. - \"A high-fidelity prediction of 2D materials growth often requires an integration of methods applicable across different scales, ranging from electronic and atomistic, to mesoscale phenomenological, and large-scale continuum models. For example, chemical potentials of different phases and the migration energies of atomic species in different structural and chemical environment from electronic and atomistic models can be employed to parametrize mesoscale phase-field models for predicting the morphology and thus characteristics of as-grown 2D materials with the boundary conditions specified by the information obtained from large-scale continuum models such as temperature distribution in the growth chamber as presented in ref. . Furthermore, technical obstacles to in-situ experimental monitoring of growth hinder the validation of computational predictions. The current mathematical and numerical models are computationally expensive and commonly do not capture simultaneously all the physics involved in a growth process of 2D materials, limiting their widespread adoption in active control of 2D materials growth. One promising way to circumvent this challenge is to use machine learning models trained using experimental and computational databases of the 2D materials structure and growth conditions such as the ones available at the 2D Crystal Consortiumu2013 Materials Innovation Platform. Finally, the models and approaches applied in understanding the growth of 2D materials can be adapted to the growth of other materials that use the same synthesis techniques, e.g., CVD growth of thin films.\" - As a future perspective, we can envision two main strategies for the development of computational models and their applications to the design and synthesis of 2D materials, which are long-term challenging goals of the field. First is the open-loop design approach, where the growth chamber design and synthesis conditions are determined by performing a series of simulations mimicking the thermophysical conditions and reaction and growth kinetics that would be expected in experiments. In this approach, the models must have high fidelity and a comprehensive sensitivity analysis should be performed to obtain a robust design. The second approach is the closed-loop design that the developed models will be utilized to determine the states of the system to be passed to a controller which adjusts the growth conditions in real-time. In this approach, the computational efficiency of the models is key to the success of the design.- - \\' \\' - All data that was obtained during this project are available from the authors. - \\' \\'- - \"Henini, M. Principles of electronic materials and devices (second edition). Microelectron. J. 33, 681 (2002)._ Yi, M. & Shen, Z. A review on mechanical exfoliation for the scalable production of graphene. J. Mater. Chem. A 3, 11700u2013 11715 (2015)._ _ Coleman, J. N. et al. Two-dimensional nanosheets produced by liquid exfoliation of layered materials. Science 331, 568u2013571 (2011)._ _ Lim, Y. R. et al. Roll-to-roll production of layer-controlled molybdenum disulfide: a platform for 2D semiconductor-based industrial applications. Adv. Mater. 30, 1705270 (2018)._ McKee, G. S. B. & Vecchio, K. S. Thermogravimetric analysis of synthesis variation effects on CVD generated multiwalled carbon nanotubes. J. Phys. Chem. B 110, 1179u20131186 (2006)._ _ Ostadhossein, A. et al. ReaxFF reactive force-field study of molybdenum disulfide (MoS2). J. Phys. Chem. Lett. 8, 631u2013640 (2017)._ _ Hong, S. et al. Computational synthesis of MoS layers by reactive molecular dynamics simulations: initial sulfidation of MoO surfaces. Nano Lett. 17, 4866u20134872 (2017)._ _ Momeni, K., Ji, Y., Zhang, K., Robinson, J. A. & Chen, L.-Q. Multiscale Framew. Simul.-Guide. growth 2D Mater. npj 2D Mater. Appl. 2, 27 (2018)._ Zhang, X. et al. Defect-controlled nucleation and orientation of WSe2 on hBN: a route to single-crystal epitaxial monolayers. ACS Nano 13, 3341u20133352 (2019)._ _ Chen, W., Gui, X., Yang, L., Zhu, H. & Tang, Z. Wrinkling of two-dimensional materials: methods, properties and applications. Nanoscale Horiz. 4, 291u2013320 (2019)._ _ Momeni, K. & Levitas, V. I. Propagating phase interface with intermediate interfacial phase: phase field approach. Phys. Rev. B 89 (2014).Mehdipour, H. & Ostrikov, K. Kinetics of low-pressure, low-temperature graphene growth: toward single-layer, single-crystalline structure. ACS Nano 6, 10276u201310286 (2012)._ _ Jiang, J.-W., Wang, B.-S., Wang, J.-S. & Park, H. S. A review on the flexural mode of graphene: lattice dynamics, thermal conduction, thermal expansion, elasticity and nanomechanical resonance. J. Phys. Condens. Matter 27, 83001 (2015)._ Mounet, N. et al. Two-dimensional materials from high-throughput computational exfoliation of experimentally known compounds. Nat. Nanotechnol. 13, 246u2013252 (2018)._ _ Zhuang, H. L. & Hennig, R. G. Stability and magnetism of strongly correlated single-layer VS2. Phys. Rev. B 93, 54429 (2016)._ Cohen, A. J., Mori-SxE1nchez, P. & Yang, W. Insights into current limitations of density functional theory. Science 321, 792u2013794 (2008)._ _ Paul, J. T. et al. Computational methods for 2D materials: discovery, property characterization, and application design. J. Phys. Condens. Matter 29, 473001 (2017)._ _ Sun, Y. et al. Low-temperature solution synthesis of few-layer 1Tu2032-MoTe2 nanostructures exhibiting lattice compression. Angew. Chem. Int. Ed. 55, 2830u20132834 (2016)._ _ Kretschmer, S., Komsa, H.-P., BxF8ggild, P. & Krasheninnikov, A. V. Structural transformations in two-dimensional transition-metal dichalcogenide MoS2 under an electron beam: insights from first-principles calculations. J. Phys. Chem. Lett. 8, 3061u2013 3067 (2017)._ _ Duerloo, K.-A. N., Li, Y. & Reed, E. J. Structural phase transitions in two-dimensional Mo- and W-dichalcogenide monolayers. Nat. Commun. 5, 4214 (2014)._ _ Gan, L.-Y., Zhang, Q., Zhao, Y.-J., Cheng, Y. & SchwingenschlxF6gl, U. Order-disorder phase transitions in the two-dimensional semiconducting transition metal dichalcogenide alloys Mo1u2212xWxX2 (X = S, Se, and Te). Sci. Rep. 4, 6691 (2015)._ Komsa, H.-P. & Krasheninnikov, A. V. Two-dimensional transition metal dichalcogenide alloys: stability and electronic properties. J. Phys. Chem. Lett. 3, 3652u2013 3656 (2012)._ _ Dumcenco, D. O., Kobayashi, H., Liu, Z., Huang, Y.-S. & Suenaga, K. Visualization and quantification of transition metal atomic mixing in Mo1u2212xWxS2 single layers. Nat. Commun. 4, 1351 (2013)._ Azizi, A. et al. Spontaneous formation of atomically thin stripes in transition metal dichalcogenide monolayers. Nano Lett. 16, 6982u20136987 (2016)._ _ Grimme, S., Antony, J., Ehrlich, S. & Krieg, H. A consistent and accurate ab initio parametrization of density functional dispersion correction (DFT-D) for the 94 elements H-Pu. J. Chem. Phys. 132, 154104 (2010)._ Tkatchenko, A. & Scheffler, M. Accurate molecular van der Waals interactions from ground-state electron density and free-atom reference data. Phys. Rev. Lett. 102, 73005 (2009)._ Dion, M., Rydberg, H., SchrxF6der, E., Langreth, D. C. & Lundqvist, B. I. Van der Waals density functional for general geometries. Phys. Rev. Lett. 92, 246401 (2004)._ _ He, J., Hummer, K. & Franchini, C. Stacking effects on the electronic and optical properties of bilayer transition metal dichalcogenides MoS2, MoSe2, WS2, and WSe2. Phys. Rev. B 89, 75409 (2014)._ Dreyer, C. E., Janotti, A. & Van de Walle, C. G. Absolute surface energies of polar and nonpolar planes of GaN. Phys. Rev. B 89, 81305 (2014)._ Mathew, K. et al. MPInterfaces: A Materials Project based Python tool for high-throughput computational screening of interfacial systems. Comput. Mater. Sci. 122, 183u2013190 (2016)._ _ Henkelman, G., Uberuaga, B. P. & Jonsson, H. A climbing image nudged elastic band method for finding saddle points and minimum energy paths. J. Chem. Phys. 113, 9901u2013 9904 (2000)._ _ Eichfeld, S. M., Colon, V. O., Nie, Y., Cho, K. & Robinson, J. A. Controlling nucleation of monolayer WSe2 during metal-organic chemical vapor deposition growth. 2D Mater. 3, 25015 (2016)._ Tuckerman, M. E. Statistical mechanics: theory and molecular simulation (Oxford Univ. Press, 2010).Bucko, T. Ab initio calculations of free-energy reaction barriers. J. Phys. Condens. Matter 20, 64211 (2008)._ _ Sheppard, D., Terrell, R. & Henkelman, G. Optimization methods for finding minimum energy paths. J. Chem. Phys. 128, 134106 (2008)._ Misawa, M. et al. Reactivity of sulfur molecules on MoO3 (010) surface. J. Phys. Chem. Lett. 8, 6206u20136210 (2017)._ _ Torrie, G. M. & Valleau, J. P. Nonphysical sampling distributions in Monte Carlo free-energy estimation: umbrella sampling. J. Comput. Phys. 23, 187u2013199 (1977)._ Sun, H. et al. Theory of thin-film-mediated exfoliation of van der Waals bonded layered materials. Phys. Rev. Mater. 2, 94004 (2018)._ _ van Erp, T. S. & Bolhuis, P. G. Elaborating transition interface sampling methods. J. Comput. Phys. 205, 157u2013181 (2005)._ Laio, A. & Parrinello, M. Escaping free-energy minima. Proc. Natl Acad. Sci. USA 99, 12562u201312566 (2002)._ _ Darve, E. & Pohorille, A. Calculating free energies using average force. J. Chem. Phys. 115, 9169u20139183 (2001)._ _ Kaminski, J. W., Kratzer, P. & Ratsch, C. Towards a standardized setup for surface energy calculations. Phys. Rev. B 95, 85408 (2017)._ Liu, Y., Bhowmick, S. & Yakobson, B. I. BN white graphene with u2018colorful edges: the energies and morphology. Nano Lett. 11, 3113u20133116 (2011)._ _ Chetty, N. & Martin, R. M. First-principles energy density and its applications to selected polar surfaces. Phys. Rev. B 45, 6074u20136088 (1992)._ _ Artyukhov, V. I., Liu, Y. & Yakobson, B. I. Equilibrium at the edge and atomistic mechanisms of graphene growth. Proc. Natl Acad. Sci. USA 109, 15136u2013 15140 (2012)._ _ Li, X. et al. Edge-controlled growth and etching of two-dimensional GaSe monolayers. J. Am. Chem. Soc. 139, 482u2013 491 (2017)._ _ Rajan, A. G., Warner, J. H., Blankschtein, D. & Strano, M. S. Generalized mechanistic model for the chemical vapor deposition of 2D transition metal dichalcogenide monolayers. ACS Nano 10, 4330u20134344 (2016)._ Freysoldt, C. et al. First-principles calculations for point defects in solids. Rev. Mod. Phys. 86, 253u2013305 (2014)._ Stone, A. J. & Wales, D. J. Theoretical studies of icosahedral C60 and some related species. Chem. Phys. Lett. 128, 501u2013503 (1986)._ _ Komsa, H. P., Berseneva, N., Krasheninnikov, A. V. & Nieminen, R. M. Charged point defects in the flatland: Accurate formation energy calculations in twodimensional materials. Phys. Rev. X 4, 31044 (2014)._ Rao, R. et al. Dynamics of cleaning, passivating and doping monolayer MoS2 by controlled laser irradiation. 2D Mater. 6, 45031 (2019)._ _ Rastogi, P., Kumar, S., Bhowmick, S., Agarwal, A. & Chauhan, Y. S. Doping strategies for monolayer MoS2 via surface adsorption: a systematic study. J. Phys. Chem. C. 118, 30309u201330314 (2014)._ _ Weston, L., Wickramaratne, D., Mackoit, M., Alkauskas, A. & Van de Walle, C. G. Native point defects and impurities in hexagonal boron nitride. Phys. Rev. B 97, 214104 (2018)._ _ Walsh, A. & Zunger, A. Instilling defect tolerance in new compounds. Nat. Mater. 16, 964u2013967 (2017)._ _ Zhang, F. et al. Full orientation control of epitaxial MoS2 on hBN assisted by substrate defects. Phys. Rev. B 99, 155430 (2019)._ _ Lee, G.-D. et al. Diffusion, coalescence, and reconstruction of vacancy defects in graphene layers. Phys. Rev. Lett. 95, 205501 (2005)._ Kotakoski, J., Krasheninnikov, A. V. & Nordlund, K. Energetics, structure, and long-range interaction of vacancy-type defects in carbon nanotubes: atomistic simulations. Phys. Rev. B 74, 245420 (2006)._ Momeni, K., Attariani, H. & Lesar, R. A. Structural transformation in monolayer materials: A 2D to 1D transformation. Phys. Chem. Chem. Phys. 18, 19873u201319879 (2016)._ _ Zhou, W. et al. Intrinsic structural defects in monolayer molybdenum disulfide. Nano Lett. 13, 2615u20132622 (2013)._ _ Zou, X., Liu, Y. & Yakobson, B. I. Predicting dislocations and grain boundaries in two-dimensional metal-disulfides from the first principles. Nano Lett. 13, 253u2013258 (2013)._ _ Xu, F., Yu, H., Sadrzadeh, A. & Yakobson, B. I. Riemann surfaces of carbon as graphene nanosolenoids. Nano Lett. 16, 34u201339 (2016)._ _ Yu, Z. G., Zhang, Y.-W. & Yakobson, B. I. An anomalous formation pathway for dislocation-sulfur vacancy complexes in polycrystalline monolayer MoS2. Nano Lett. 15, 6855u20136861 (2015)._ _ Sha, Z. D. et al. Inverse pseudo Hall-Petch relation in polycrystalline graphene. Sci. Rep. 4, 5991 (2014)._ _ Grantab, R., Shenoy, V. B. & Ruoff, R. S. Anomalous strength characteristics of tilt grain boundaries in graphene. Science 330, 946u2013948 (2010)._ _ Zhang, Z., Yang, Y., Xu, F., Wang, L. & Yakobson, B. I. Unraveling the sinuous grain boundaries in graphene. Adv. Funct. Mater. 25, 367u2013373 (2015)._ _ Liu, Y., Xu, F., Zhang, Z., Penev, E. S. & Yakobson, B. I. Two-dimensional mono-elemental semiconductor with electronically inactive defects: the case of phosphorus. Nano Lett. 14, 6782u20136786 (2014)._ _ Jones, J. E. On the determination of molecular fields. II. From the equation of state of a gas. Proc. R. Soc. Lond. Ser. A Math. Phys. Eng. Sci. 106, 463u2013477 (1924)._ _ Wen, M., Carr, S., Fang, S., Kaxiras, E. & Tadmor, E. B. Dihedral-angle-corrected registry-dependent interlayer potential for multilayer graphene structures. Phys. Rev. B 98, 235404 (2018)._ _ Kolmogorov, A. N. & Crespi, V. H. Registry-dependent interlayer potential for graphitic systems. Phys. Rev. B 71, 235415 (2005)._ Naik, M. H., Maity, I., Maiti, P. K. & Jain, M. Kolmogorovu2013Crespi potential formultilayer transition-metal dichalcogenides: capturing structural transformations in MoirxE9 superlattices. J. Phys. Chem. C. 123, 9770u20139778 (2019)._ _ Rappe, A. K., Casewit, C. J., Colwell, K. S., Goddard, W. A. & Skiff, W. M. UFF, a full periodic table force field for molecular mechanics and molecular dynamics simulations. J. Am. Chem. Soc. 114, 10024u201310035 (1992)._ _ Stillinger, F. H. & Weber, T. A. Computer simulation of local order in condensed phases of silicon. Phys. Rev. B 31, 5262u20135271 (1985)._ _ Jiang, J.-W. Parametrization of Stillingeru2013Weber potential based on valence force field model: application to single-layer MoS2 and black phosphorus. Nanotechnology 26, 315706 (2015)._ Jiang, J.-W., Park, H. S. & Rabczuk, T. Molecular dynamics simulations of single-layer molybdenum disulphide (MoS2): Stillinger-Weber parametrization, mechanical properties, and thermal conductivity. J. Appl. Phys. 114, 64307u201364311 (2013)._ Jiang, J.-W., Qi, Z., Park, H. S. & Rabczuk, T. Elastic bending modulus of single-layer molybdenum disulfide (MoS2): finite thickness effect. Nanotechnology 24, 435705 (2013)._ Yapicioglu, H., Kandemir, A., Sevik, C., xC7au011Fu0131n, T. & Kinaci, A. Thermal transport properties of MoS2 and MoSe2 monolayers. Nanotechnology 27, 55703 (2016)._ Norouzzadeh, P. & Singh, D. J. Thermal conductivity of single-layer WSe2 by a Stillingeru2013Weber potential. Nanotechnology 28, 75708 (2017)._ Jiang, J.-W., Rabczuk, T. & Park, H. S. A Stillingeru2013Weber potential for single-layered black phosphorus, and the importance of cross-pucker interactions for a negative Poissons ratio and edge stress-induced bending. Nanoscale 7, 6059u20136068 (2015)._ _ Xu, W., Zhu, L., Cai, Y., Zhang, G. & Li, B. Direction dependent thermal conductivity of monolayer phosphorene: parameterization of Stillinger-Weber potential and molecular dynamics study. J. Appl. Phys. 117, 214308 (2015)._ Jiang, J.-W. & Zhou, Y.-P. In Parameterization of Stillinger-Weber Potential for Two-Dimensional Atomic Crystal Chapter 1 (IntechOpen, 2017). .Allinger, N. L., Yuh, Y. H. & Lii, J. H. Molecular mechanics. The MM3 force field for hydrocarbons. 1. J. Am. Chem. Soc. 111, 8551u20138566 (1989)._ _ Sun, H. COMPASS: an ab initio force-field optimized for condensed-phase applications overview with details on alkane and benzene compounds. J. Phys. Chem. B 102, 7338u20137364 (1998)._ _ Keating, P. N. Effect of invariance requirements on the elastic strain energy of crystals with application to the diamond structure. Phys. Rev. 145, 637u2013645 (1966)._ _ Allinger, N. L. Calculation of molecular structure and energy by force-field methods. Adv. Phys Org. Chem. 13, 1u201382 (1976).Allinger, N. L. Conformational analysis. 130. MM2. A hydrocarbon force field utilizing V1 and V2 torsional terms. J. Am. Chem. Soc. 99, 8127u20138134 (1977)._ _ Allinger, N. L., Chen, K., Katzenellenbogen, J. A., Wilson, S. R. & Anstead, G. M. Hyperconjugative effects on carbonu2014carbon bond lengths in molecular mechanics (MM4). J. Comput. Chem. 17, 747u2013755 (1996)._ _ Tu, Z. & Ou-Yang, Z. Single-walled and multiwalled carbon nanotubes viewed as elastic tubes with the effective Youngs moduli dependent on layer number. Phys. Rev. B 65, 233407 (2002)._ Chang, T. & Gao, H. Size-dependent elastic properties of a single-walled carbon nanotube via a molecular mechanics model. J. Mech. Phys. Solids 51, 1059u20131074 (2003)._ _ Tersoff, J. Empirical interatomic potential for carbon, with applications to amorphous carbon. Phys. Rev. Lett. 61, 2879u20132882 (1988)._ _ Brenner, D. W. et al. A second-generation reactive empirical bond order (REBO) potential energy expression for hydrocarbons. J. Phys. Condens. Matter 14, 783u2013802 (2002)._ _ Stuart, S. J., Tutein, A. B. & Harrison, J. A. A reactive potential for hydrocarbons with intermolecular interactions. J. Chem. Phys. 112, 6472u20136486 (2000)._ _ Liang, T., Devine, B., Phillpot, S. R. & Sinnott, S. B. Variable charge reactive potential for hydrocarbons to simulate organic-copper interactions. J. Phys. Chem. A 116, 7976u20137991 (2012)._ _ Van Duin, A. C. T., Dasgupta, S., Lorant, F. & Goddard, W. A. ReaxFF: a reactive force field for hydrocarbons. J. Phys. Chem. A 105, 9396u20139409 (2001)._ Liang, T. et al. Reactive potentials for advanced atomistic simulations. Annu. Rev. Mater. Res. 43, 109u2013129 (2013)._ _ Fonseca, A. F. et al. Grapheneu2013titanium interfaces from molecular dynamics simulations. ACS Appl. Mater. Interfaces 9, 33288u201333297 (2017)._ _ Mortazavi, B., Pereira, L. F. C., Jiang, J.-W. & Rabczuk, T. Modelling heat conduction in polycrystalline hexagonal boron-nitride films. Sci. Rep. 5, 13228 (2015)._ _ Mukhopadhyay, T., Mahata, A., Adhikari, S. & Asle Zaeem, M. Probing the shear modulus of two-dimensional multiplanar nanostructures and heterostructures. Nanoscale 10, 5280u20135294 (2018)._ _ Klaver, T. P. C., Zhu, S.-E., Sluiter, M. H. F. & Janssen, G. C. A. M. Molecular dynamics simulation of graphene on Cu (100) and (111) surfaces. Carbon N. Y. 82, 538u2013547 (2015)._ _ Hong, S. et al. Chemical vapor deposition synthesis of MoS2 layers from the direct sulfidation of MoO3 surfaces using reactive molecular dynamics simulations. J. Phys. Chem. C. 122, 7494u20137503 (2018)._ _ Tersoff, J. New empirical model for the structural properties of silicon. Phys. Rev. Lett. 56, 632u2013635 (1986)._ _ Tersoff, J. New empirical approach for the structure and energy of covalent systems. Phys. Rev. B 37, 6991u20137000 (1988)._ _ Jiang, J.-W. & Wang, J.-S. Manipulation of heat current by the interface between graphene and white graphene. Europhys. Lett. 96, 16003 (2011)._ Mahdizadeh, S. J. & Akhlamadi, G. Optimized Tersoff empirical potential for germanene. J. Mol. Graph. Model. 72, 1u20135 (2017)._ _ Tersoff, J. Modeling solid-state chemistry: interatomic potentials for multicomponent systems. Phys. Rev. B 39, 5566u20135568 (1989)._ _ Jiang, J.-W. & Wang, J.-S. Theoretical study of thermal conductivity in single-walled boron nitride nanotubes. Phys. Rev. B 84, 85439 (2011)._ Liu, B. et al. Thermal conductivity of silicene nanosheets and the effect of isotopic doping. J. Phys. D. Appl. Phys. 47, 165301 (2014)._ McGuigan, B. C., Pochet, P. & Johnson, H. T. Critical thickness for interface misfit dislocation formation in two-dimensional materials. Phys. Rev. B 93, 214103 (2016)._ Raju, M., van Duin, A. & Ihme, M. Phase transitions of ordered ice in graphene nanocapillaries and carbon nanotubes. Sci. Rep. 8, 3851 (2018)._ Huang, X., Yang, H., van Duin, A. C. T., Hsia, K. J. & Zhang, S. Chemomechanics control of tearing paths in graphene. Phys. Rev. B 85, 195453 (2012)._ Paupitz, R. et al. Graphene to fluorographene and fluorographane: a theoretical study. Nanotechnology 24, 35706 (2012)._ Neyts, E. C., van Duin, A. C. T. & Bogaerts, A. Formation of single layer graphene on nickel under far-from-equilibrium high flux conditions. Nanoscale 5, 7250u20137255 (2013)._ _ Achtyl, J. L. et al. Aqueous proton transfer across single-layer graphene. Nat. Commun. 6, 6539 (2015)._ _ Raju, M., Govindaraju, P. B., van Duin, A. C. T. & Ihme, M. Atomistic and continuum scale modeling of functionalized graphyne membranes for water desalination. Nanoscale 10, 3969u20133980 (2018)._ _ Berdiyorov, G. R., Neek-Amal, M., Peeters, F. M. & van Duin, A. C. T. Stabilized silicene within bilayer graphene: a proposal based on molecular dynamics and density-functional tight-binding calculations. Phys. Rev. B 89, 24107 (2014)._ Berdiyorov, G. R. & Peeters, F. M. Influence of vacancy defects on the thermal stability of silicene: a reactive molecular dynamics study. RSC Adv. 4, 1133u20131137 (2014)._ _ Kushima, A., Qian, X., Zhao, P., Zhang, S. & Li, J. Ripplocations in van der Waals layers. Nano Lett. 15, 1302u20131308 (2015)._ _ Yilmaz, D. E., Lotfi, R., Ashraf, C., Hong, S. & van Duin, A. C. T. Defect design of two-dimensional MoS2 structures by using a graphene layer and potato stamp concept. J. Phys. Chem. C. 122, 11911u201311917 (2018)._ _ Mortazavi, B. et al. Strong thermal transport along polycrystalline transition metal dichalcogenides revealed by multiscale modeling for MoS2. Appl. Mater. Today 7, 67u201376 (2017)._ Yan, R. et al. Thermal conductivity of monolayer molybdenum disulfide obtained from temperature-dependent Raman spectroscopy. ACS Nano 8, 986u2013993 (2014)._ _ Osti, N. C. et al. Effect of metal ion intercalation on the structure of MXene and water dynamics on its internal surfaces. ACS Appl. Mater. Interfaces 8, 8859u20138863 (2016)._ _ Berdiyorov, G. R. & Mahmoud, K. A. Effect of surface termination on ion intercalation selectivity of bilayer Ti3C2T2(T=F, O and OH) MXene. Appl. Surf. Sci. 416, 725u2013730 (2017)._ _ Osti, N. C. et al. Influence of metal ions intercalation on the vibrational dynamics of water confined between MXene layers. Phys. Rev. Mater. 1, 65406 (2017)._ Overbury, S. H. et al. Complexity of intercalation in MXenes: destabilization of urea by two-dimensional titanium carbide. J. Am. Chem. Soc. 140, 10305u201310314 (2018)._ _ Sang, X. et al. In situ atomistic insight into the growth mechanisms of single layer 2D transition metal carbides. Nat. Commun. 9, 2266 (2018)._ Wang, N. & Komvopoulos, K. The effect of deposition energy of energetic atoms on the growth and structure of ultrathin amorphous carbon films studied by molecular dynamics simulations. J. Phys. D. Appl. Phys. 47, 245303 (2014)._ Ma, T., Hu, Y.-Z., Wang, H. & Li, X. Microstructural and stress properties of ultrathin diamondlike carbon films during growth: Molecular dynamics simulations. Phys. Rev. B 75, 35425 (2007)._ Wang, X., Tabarraei, A. & Spearot, D. E. Fracture mechanics of monolayer molybdenum disulfide. Nanotechnology 26, 175703 (2015)._ Reddy, C. D., Gen Yu, Z. & Zhang, Y.-W. Two-dimensional van der Waals C60 molecular crystal. Sci. Rep. 5, 12221 (2015)._ _ Paul, S. & Momeni, K. Mechanochemistry of stable diamane and atomically thin diamond films synthesis from bi- and multilayer graphene: a computational study. J. Phys. Chem. C. 123, 15751u201315760 (2019)._ _ Edward, L.-J. J. & Sydney, C. On the forces between atoms and ions. Proc. R. Soc. A Math. Phys. Eng. Sci. 109, 584u2013597 (1925)._ Cheng, Y.-T. et al. Cu cluster deposition on ZnO(101xAF0): morphology and growth mode predicted from molecular dynamics simulations. Surf. Sci. 621, 109u2013116 (2014)._ _ Cui, L., Shi, S., Li, Z., Wei, G. & Du, X. Manipulating thermal conductance of supported graphene via surface hydroxylation of substrates. J. Phys. Chem. C. 122, 27689u201327695 (2018)._ _ Burton, W. K., Cabrera, N., Frank, F. C. & Francis, M. N. The growth of crystals and the equilibrium structure of their surfaces. Philos. Trans. R. Soc. Lond. Ser. A, Math. Phys. Sci. 243, 299u2013358 (1951)._ Steinbach, I. Phase-field model for microstructure evolution at the mesoscopic scale. Annu. Rev. Mater. Res. 43, 89u2013107 (2013)._ _ Liu, F. & Metiu, H. Stability and kinetics of step motion on crystal surfaces. Phys. Rev. E 49, 2601u20132616 (1994)._ _ Karma, A. & Plapp, M. Spiral surface growth without desorption. Phys. Rev. Lett. 81, 4444u20134447 (1998)._ _ Pierre-Louis, O. Phase field models for step flow. Phys. Rev. E 68, 21604 (2003)._ _ Hu, Z., Lowengrub, J. S., Wise, S. M. & Voigt, A. Phase-field modeling of epitaxial growth: applications to step trains and island dynamics. Phys. D. Nonlinear Phenom. 241, 77u2013 94 (2012)._ Meca, E., Lowengrub, J., Kim, H., Mattevi, C. & Shenoy, V. B. Epitaxial graphene growth and shape dynamics on copper: phase-field modeling and experiments. Nano Lett. 13, 5692u20135697 (2013)._ _ Meca, E., Shenoy, V. B. & Lowengrub, J. Phase-field modeling of two-dimensional crystal growth with anisotropic diffusion. Phys. Rev. E 88, 52409 (2013)._ Wu, J. et al. Spiral growth of SnSe2 crystals by chemical vapor deposition. Adv. Mater. Interfaces 3, 1600383 (2016)._ Li, J. et al. Impurity-induced formation of bilayered graphene on copper by chemical vapor deposition. Nano Res 9, 2803u20132810 (2016)._ _ Zhang, K. et al. Manganese doping of monolayer MoS2: the substrate is critical. Nano Lett. 15, 6586u20136591 (2015)._ _ Hao, Y. et al. The role of surface oxygen in the growth of large single-crystal graphene on copper. Science 342, 720u2013723 (2013)._ _ Luo, B. et al. Etching-controlled growth of graphene by chemical vapor deposition. Chem. Mater. 29, 1022u20131027 (2017)._ _ Artyukhov, V. I., Hu, Z., Zhang, Z. & Yakobson, B. I. Topochemistry of Bowtie- and star-shaped metal dichalcogenide nanoisland formation. Nano Lett. 16, 3696u20133702 (2016)._ _ Yu, H. et al. Tilt grain boundary topology induced by substrate topography. ACS Nano 11, 8612u20138618 (2017)._ _ Meca, E., Shenoy, V. B. & Lowengrub, J. H. H2-dependent attachment kinetics and shape evolution in chemical vapor deposition graphene growth. 2D Mater. 4, 31010 (2017)._ Berry, J., Zhou, S., Han, J., Srolovitz, D. J. & Haataja, M. P. Dynamic phase engineering of bendable transition metal dichalcogenide monolayers. Nano Lett. 17, 2473u2013 2481 (2017)._ _ Emmerich, H. et al. Phase-field-crystal models for condensed matter dynamics on atomic length and diffusive time scales: an overview. Adv. Phys. 61, 665u2013743 (2012)._ Elder, K. L. M., Seymour, M., Lee, M., Hilke, M. & Provatas, N. Two-component structural phase-field crystal models for graphene symmetries. Philos. Trans. R. Soc. A Math. Phys. Eng. Sci. 376, 20170211 (2018)._ Taha, D., Mkhonta, S. K., Elder, K. R. & Huang, Z.-F. Grain boundary structures and collective dynamics of inversion domains in binary two-dimensional materials. Phys. Rev. Lett. 118, 255501 (2017)._ Battaile, C. C. The kinetic Monte Carlo method: foundation, implementation, and application. Comput. Methods Appl. Mech. Eng. 197, 3386u20133398 (2008)._ Nie, Y. et al. A kinetic Monte Carlo simulation method of van der Waals epitaxy for atomistic nucleation-growth processes of transition metal dichalcogenides. Sci. Rep. 7, 2977 (2017)._ Trevethan, T., Latham, C. D., Heggie, M. I., Briddon, P. R. & Rayson, M. J. Vacancy diffusion and coalescence in graphene directed by defect strain fields. Nanoscale 6, 2978u20132986 (2014)._ _ Whitesides, R. & Frenklach, M. Effect of reaction kinetics on graphene-edge morphology and composition. Z. fur Physikalische Chem. 229, 597u2013614 (2015)._ _ Wang, B., Puzyrev, Y. S. & Pantelides, S. T. Enhanced chemical reactions of oxygen at grain boundaries in polycrystalline graphene. Polyhedron 64, 158u2013162 (2013)._ _ Zhuang, J., Zhao, R., Dong, J., Yan, T. & Ding, F. Evolution of domains and grain boundaries in graphene: a kinetic Monte Carlo simulation. Phys. Chem. Chem. Phys. 18, 2932u2013 2939 (2016)._ _ Taioli, S. Computational study of graphene growth on copper by first-principles and kinetic Monte Carlo calculations. J. Mol. Model. 20, 2260 (2014)._ Fan, L. et al. Topology evolution of graphene in chemical vapor deposition, a combined theoretical/experimental approach toward shape control of grapheme domains. Nanotechnology 23, 115605 (2012)._ Gaillard, P., Schoenhalz, A. L., Moskovkin, P., Lucas, S. & Henrard, L. Growth of nitrogen-doped graphene on copper: multiscale simulations. Surf. Sci. 644, 102u2013108 (2016)._ _ Wu, P. et al. Lattice mismatch induced nonlinear growth of graphene. J. Am. Chem. Soc. 134, 6045u20136051 (2012)._ _ Jiang, H. & Hou, Z. Large-scale epitaxial growth kinetics of graphene: a kinetic Monte Carlo study. J. Chem. Phys. 143, 84109 (2015)._ Huang, J. Y. et al. In situ observation of graphene sublimation and multi-layer edge reconstructions. Proc. Natl Acad. Sci. USA 106, 10103u201310108 (2009)._ _ Ming, F. & Zangwill, A. Model and simulations of the epitaxial growth of graphene on non-planar 6Hu2013SiC surfaces. J. Phys. D. Appl. Phys. 45, 154007 (2012)._ Qiu, Z., Li, P., Li, Z. & Yang, J. Atomistic simulations of graphene growth: from kinetics to mechanism. Acc. Chem. Res. 51, 728u2013735 (2018)._ _ Nie, Y. et al. First principles kinetic Monte Carlo study on the growth patterns of WSe2 monolayer. 2D Mater. 3, 25029 (2016)._ Chen, S. et al. Origin of ultrafast growth of monolayer WSe2 via chemical vapor deposition. npj Comput. Mater. 5, 28 (2019)._ Yue, R. et al. Nucleation and growth of WSe2: enabling large grain transition metal dichalcogenides. 2D Mater. 4, 45019 (2017)._ Safron, N. S. & Arnold, M. S. Experimentally determined model of atmospheric pressure CVD of graphene on Cu. J. Mater. Chem. C. 2, 744u2013755 (2014)._ _ Eversteyn, F. C., Severin, P. J. W., Brekel, C. H. Jvd & Peek, H. L. A stagnant layer model for the epitaxial growth of silicon from silane in a horizontal reactor. J. Electrochem. Soc. 117, 925u2013931 (1970)._ Chen, J., Wong, D. S. H., Jang, S. S. & Yang, S. L. Product and process development using artificial neuralu2010network model and information analysis. AIChE J. 44, 876u2013887 (1998)._ _ VilxE1 , R. A. et al. Bottom-up synthesis of vertically oriented two-dimensional materials. 2D Mater. 3, 41003 (2016)._ Lee, P. W., Omstead, T. R., McKenna, D. R. & Jensen, K. F. In situ mass spectroscopy studies of the decomposition of organometallic arsenic compounds in the presence of Ga(CH3)3 and Ga(C2H5)3. J. Cryst. Growth 93, 134u2013142 (1988)._ _ Theodoropoulos, C. et al. Kinetic and transport modeling of the metallorganic chemical vapor deposition of InP from trimethylindium and phosphine and comparison with experiments. J. Electrochem. Soc. 142, 2086u20132094 (1995)._ _ Poling, B. E., Prausnitz, J. M. & Oconnell, J. P. The Properties of Gases and Liquids, Vol. 5 (Mcgraw-Hill, New York, 2001).Zhang, F. et al. Controlled synthesis of 2D transition metal dichalcogenides: from vertical to planar MoS2. 2D Mater. 4, 25029 (2017)._ Grazulis, S. et al. Crystallography open database - an open-access collection of crystal structures. J. Appl. Crystallogr 42, 726u2013729 (2009)._ _ Belsky, A., Hellenbrandt, M., Karen, V. L. & Luksch, P. New developments in the Inorganic Crystal Structure Database (ICSD): accessibility in support of materials research and design. Acta Crystallogr. Sect. B 58, 364u2013369 (2002)._ Allen, F. H. et al. The Cambridge Crystallographic Data Centre: computer-based search, retrieval, analysis and display of information. Acta Crystallogr. Sect. B 35, 2331u20132339 (1979)._ Choudhary, K., Kalish, I., Beams, R. & Tavazza, F. High-throughput identification and characterization of two-dimensional materials using density functional theory. Sci. Rep. 7, 5179 (2017)._ Haastrup, S. et al. The Computational 2D Materials. Database.: high.-throughput modeling Discov. atomically thin Cryst. 2D Mater. 5, 42002 (2018)._ _ Cheon, G. et al. Data mining for new two- and one-dimensional weakly bonded solids and lattice-commensurate heterostructures. Nano Lett. 17, 1915u20131923 (2017)._ _ Ashton, M., Paul, J., Sinnott, S. B. & Hennig, R. G. Topology-scaling identification of layered solids and stable exfoliated 2D materials. Phys. Rev. Lett. 118, 106101 (2017)._ Patra, T. K. et al. Defect dynamics in 2-D MoS2 probed by using machine learning, atomistic simulations, and high-resolution microscopy. ACS Nano 12, 8006u20138016 (2018)._ _ Tawfik, S. A. et al. Efficient prediction of structural and electronic properties of hybrid 2D materials using complementary DFT and machine learning approaches. Adv. Theory Simul. 2, 1800128 (2019)._ Choudhary, K., DeCost, B. & Tavazza, F. Machine learning with force-field-inspired descriptors for materials: fast screening and mapping energy landscape. Phys. Rev. Mater. 2, 83801 (2018)._ _ Eivari, H. A. et al. Two-dimensional hexagonal sheet of TiO2. Chem. Mater. 29, 8594u20138603 (2017)._ _ Miyazato, I., Tanaka, Y. & Takahashi, K. Accelerating the discovery of hidden two-dimensional magnets using machine learning and first principle calculations. J. Phys. Condens. Matter 30, 06LT01 (2018)._ Cherukara, M. J. et al. Ab initio-based bond order potential to investigate low thermal conductivity of stanene nanostructures. J. Phys. Chem. Lett. 7, 3752u20133759 (2016)._ _ Revard, B. C., Tipton, W. W., Yesypenko, A. & Hennig, R. G. Grand-canonical evolutionary algorithm for the prediction of two-dimensional materials. Phys. Rev. B 93, 54117 (2016)._ Singh, A. K. et al. Genetic algorithm prediction of two-dimensional group-IV dioxides for dielectrics. Phys. Rev. B 95, 155426 (2017)._ Chmiela, S. et al. Machine learning of accurate energy-conserving molecular force fields. Sci. Adv. 3, e1603015 (2017)._ Botu, V., Batra, R., Chapman, J. & Ramprasad, R. Machine learning force fields: construction, validation, and outlook. J. Phys. Chem. C. 121, 511u2013522 (2017)._ _ Song, Z., Artyukhov, V. I., Yakobson, B. I. & Xu, Z. Pseudo Hallu2013Petch strength reduction in polycrystalline graphene. Nano Lett. 13, 1829u20131833 (2013)._ _ Okada, M. et al. Direct chemical vapor deposition growth of WS2 atomic layers on hexagonal boron nitride. ACS Nano 8, 8273u20138277 (2014)._ _ Kobayashi, Y. et al. Growth and optical properties of high-quality monolayer WS2 on graphite. ACS Nano 9, 4056u20134063 (2015)._ _ Huang, J. et al. Large-area synthesis of monolayer WSe2 on a SiO2/Si substrate and its device applications. Nanoscale 7, 4193u2013 4198 (2015)._ _ Chen, L. et al. Step-edge-guided nucleation and growth of aligned WSe2 on sapphire via a layer-over-layer growth mode. ACS Nano 9, 8368u20138375 (2015)._ _ Eichfeld, S. M. et al. Highly scalable, atomically thin WSe2 grown via metalu2013organic chemical vapor deposition. ACS Nano 9, 2080u20132087 (2015)._ _ Yanase, T. et al. Chemical vapor deposition of NbS2 from a chloride source with H2 Flow: orientation control of ultrathin crystals directly grown on SiO2/Si substrate and charge density wave transition. Cryst. Growth Des. 16, 4467u2013 4472 (2016)._ _ Friday, M., Angeles, L., Reyes-lillo, S. & Rabe, K. Bulletin of the American Physical Society APS March Meeting 2018. 2018 (2018).Hotta, T. et al. Molecular beam epitaxy growth of monolayer niobium diselenide flakes. Appl. Phys. Lett. 109, 133101 (2016)._ Efetov, D. K. et al. Specular interband Andreev reflections at van der Waals interfaces between graphene and NbSe2. Nat. Phys. 12, 328u2013332 (2016)._ _ Zhou, L. et al. Large-area synthesis of high-quality uniform few-layer MoTe2. J. Am. Chem. Soc. 137, 11892u201311895 (2015)._ _ Farahani, H., Rajabpour, A. & Reyhani, A. Interfacial thermal resistance between few-layer MoS2 and silica substrates: a molecular dynamics study. Comput. Mater. Sci. J. 142, 1u20136 (2018)._ _ Hong, Y., Ju, M. G., Zhang, J. & Zeng, X. C. Phonon thermal transport in a graphene/MoSe2 van der Waals heterobilayer. Phys. Chem. Chem. Phys. 20, 2637u20132645 (2018)._ _ Filippova, V. P., Kunavin, S. A. & Pugachev, M. S. Calculation of the parameters of the Lennard-Jones potential for pairs of identical atoms based on the properties of solid substances. Inorg. Mater. Appl. Res. 6, 1u20134 (2015)._ Al-Matar, A. K. & Rockstraw, D. A. A generating equation for mixing rules and two new mixing rules for interatomic potential energy parameters. J. Comp. Chem. 25, 660u2013668 (2004)._ _ Hiura, H., Miyazaki, T. & Kanayama, T. Formation of metal-encapsulating Si cage clusters. Phys. Rev. Lett. 86, 1733u20131736 (2001)._ _ Mundin, K. C. & Ellis, D. E. Stochastic classical molecular dynamics coupled to functional density theory: applications to large molecular systems. Braz. J. Phys. 29, 199u2013214 (1999)._ Morris, G. M. et al. van Der Waals Potential Energy (2003). Available at: (Accessed 23 March 2019).Lewis, L. J. & Mousseau, N. Tight-binding molecular-dynamics studies of defects and disorder in covalently bonded materials. Comput. Mater. Sci. 12, 210u2013241 (1998)._ _ Turner, G. W., Chushak, Y. G. & Bartell, L. S. Molecular dynamics investigation of transient nucleation in the freezing of clusters of SeF6. J. Phys. Chem. A 108, 1666u20131670 (2004)._ _ Monajjemi, M., Ketabi, S., Hashemian Zadeh, M. & Amiri, A. Simulation of DNA bases in water: comparison of the Monte Carlo algorithm with molecular mechanics force fields. Biochem 71, S1u2013S8 (2006)._ _ Landron, C. et al. Liquid alumina: detailed atomic coordination determined from neutron diffraction data using empirical potential structure refinement. Phys. Rev. Lett. 86, 4839u20134842 (2001)._ _ Bartell, L. S. & Xu, S. Molecular synamics examination of an anomalous phase of TeFe molecules. J. Phys. Chem. 95, 8939u20138941 (1991)._ _ \"- - \"K.M. and L.Q.C. are partially supported by the Hamer Professorship at Penn State, Louisiana Tech University, the National Science Foundation 2D Crystal Consortium u2013 Material Innovation Platform (2DCC-MIP) under NSF cooperative agreement DMR-1539916, and the NSF-CAREER under NSF cooperative agreement CBET-1943857. Y.Z.J. is supported by the I/UCRC Center for Atomically Thin Multifunctional Coatings (ATOMIC) seed project SP001-17 and the 2DCC-MIP. This project is also partly supported by DoE-ARPA-E OPEN, NASA-EPSCoR, Louisiana EPSCoR-OIA-1541079 (NSF(2018)-CIMMSeed-18 and NSF(2018)-CIMMSeed-19), and LEQSF(2015-18)-LaSPACE. J.W.J. is supported by the Recruitment Program of Global Youth Experts of China, the National Natural Science Foundation of China (NSFC) under Grant Number 11822206, and the Innovation Program of Shanghai Municipal Education Commission under Grant Number 2017-01-07-00-09-E00019. H.S.P. acknowledges the support of the Mechanical Engineering Department at Boston University. A.C.T.v.D., D.Y., and Y.S. acknowledge funding from the Fluid Interface Reactions, Structures and Transport (FIRST) Center, an Energy Frontier Research Center funded by the U.S. Department of Energy (DOE), Office of Science, Basic Energy Science for MXene force-field development and application work.\"- - Authors and Affiliations - \"Mechanical Engineering Department, Louisiana Tech University, Ruston, LA, 71272, USAKasra Momeni_&_Shiddartha PaulDepartment of Mechanical Engineering, University of Alabama, Tuscaloosa, AL, USAKasra MomeniInstitute for Micromanufacturing, Louisiana Tech University, Ruston, LA, USAKasra Momeni_&_Long-Qing ChenMaterials Research Institute, The Pennsylvania State University, University Park, PA, 16802, USAYanzhou Ji_&_Long-Qing ChenDepartment of Materials Science and Engineering, The Pennsylvania State University, University Park, PA, 16802, USAYanzhou Ji,_ Difan Zhang_&_Susan Sinnott2-Dimensional Crystal Consortium, The Pennsylvania State University, University Park, PA, 16802, USAYuanxi Wang_&_Vincent CrespiDepartment of Physics, The Pennsylvania State University, University Park, PA, 16802, USAYuanxi Wang_&_Vincent CrespiDepartment of Electrical Engineering, Iowa State University, Ames, IA, 50010, USASara NeshaniMechanical Engineering Department, The Pennsylvania State University, University Park, PA, 16802, USADundar E. Yilmaz,_Yun Kyung Shin_&_Adri van DuinShanghai Institute of Applied Mathematics and Mechanics, Shanghai Key Laboratory of Mechanics in Energy Engineering, Shanghai University, Shanghai, 200072, ChinaJin-Wu JiangDepartment of Mechanical Engineering, Boston University, Boston, MA, 02215, USAHarold S. ParkDepartment of Engineering Science and Mechanics, The Pennsylvania State University, University Park, PA, 16802, USALong-Qing ChenDepartment of Mathematics, The Pennsylvania State University, University Park, PA, 16802, USALong-Qing Chen\" - \"AuthorsKasra MomeniYou can also search for this author in _Yanzhou JiYou can also search for this author in _Yuanxi WangYou can also search for this author in _Shiddartha PaulYou can also search for this author in _Sara NeshaniYou can also search for this author in _Dundar E. YilmazYou can also search for this author in _Yun Kyung ShinYou can also search for this author in _Difan ZhangYou can also search for this author in _Jin-Wu JiangYou can also search for this author in _Harold S. ParkYou can also search for this author in _Susan SinnottYou can also search for this author in _Adri van DuinYou can also search for this author in _Vincent CrespiYou can also search for this author in _Long-Qing ChenYou can also search for this author in _\" - Contributions - All authors contributed to collecting the references and outlining the review paper. K.M. drafted the review. All authors participated in revising the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International Public License [CC BY 4.0]).\" - \\' \\' - \"Shirshov Massif was drilled at IODP Site U1346 (Fig._), for which pre-cruise seismic survey illustrated a thin sediment cover blanketing the flank of this volcanic edifice, thereby assuring the ease of access to the basaltic basement. Site selection at this peripheral area of the plateau, eventually, led to unprecedented opportunity to study the conspicuous sedimentary record that had not been encountered through the history of Shatsky Rise drilling. The unique sedimentary interval is from Cores 6u2009R (part) to 4u2009R and comprising, in ascending order, limestones rich in clastics and molluscan bioclasts (Unit IV), rhythmically-bedded siltstone with minor turbiditic sandstones (Unit III), and intermingled limestones and basalts (Unit II) (Fig._). Overall, it should represent a deepening-upward marine sequence, from more or less neritic sediments (Unit IV) to intermediate, gravitational clastic/calcareous deposits (Unit III/II) to deep, fully pelagic chalk/chert (Unit I). A notable find among macrofossils is the Desmoceratidae ammonite from near the base of Unit IV, just above the basaltic basement (Fig._ ; Supplementary Fig._). Microfossils are present through Unit IV to II, including moderately/poorly preserved nannofossils indicative of the Hauterivian Age, whereas planktonic foraminifera are notably lacking (for additional notes see Methods). Remarkably, and as detailed below, a variety of well-preserved benthic foraminifera and ostracods have been retrieved from a thin, semi-consolidated marly interval in the uppermost Unit IV.\" - \"Figure 2New rock and fossil records of Integrated Ocean Drilling Program (IODP) Site U1346, Shirshov Massif, northern Shatsky Rise. (a) Summary lithostratigraphy with symbols indicating key fossil occurrences (stars) and fresh volcanic glass (arrow; analyzed at single level). For age interpretation see Methods. mbsfu2014 meters below seafloor. (b) Ammonite (Desmoceratidae gen. & sp. indet.) from just above igneous basement; right lateral (right) and cross-section (left) views of incomplete specimen, with broken lines delineating inferred lost part of whorl (see also Supplementary Fig._). (c,d) Microscopic images of key representative genera of benthic foraminifera (c) and ostracods (d) from two samples 324-U1346A-4R-CC, 3u20134u2009cm and 7u20138u2009cm. Note paired magnified images of eye tubercle on Eucytherura, viewed from valve exterior (right) and interior (left).\" - \\' \\'- - \"The benthic foraminiferal and ostracod assemblages can be considered as reliable indicators of the depositional water depth of the Shirshov Massif sediments. The former comprises 10 genera of the Family Nodosariidae/Vaginulinidae (Astacolus, Citharina, Dentalina, Frondicularia, Laevidentalina, Lenticulina, Planularia, Psilocitharella, Saracenaria, Vaginulinopsis) (Fig._). The nodosariid/vaginulinid benthic foraminfiera occur mainly from the neritic setting and often as deep as the upper bathyal depth, yet such a monofamily assemblage without typical deep-water taxa is indicative of the neritic environment.\" - \"A narrower range of palaeo-water depth, probably in a coastal inner neritic setting shallower than 20u2009m, is estimated from the ostracod assemblage with 6 genera (Cytherelloidea, Eucytherura, Hemicytherura, Paracypris?, Procytherura? Semicytherura) of potentially up to 12 species (Fig._). Following are the key features that support reconstruction of such nearshore marine environment: (i) assemblage composition dominated by the Family Cytheruridae and with Cytherelloidea, resembling a Jurassic Argentine inner shelf biofacies; (ii) abundant occurrence of known seaweed-associated intertidal taxa, Semicytherura and Hemicytherura; (iii) generally low species-diversity; and (iv) absence of typical outer neritic or deeper taxa such as Cytheropteron, Argilloecia, trachyleberidids and thaerocytherids (their absence is indicative of shallow [<20u2009m] depth in specific examples of modern western Pacific coasts).\" - \"Additionally, and most importantly, we document the common presence of large eye tubercles on Eucytherura (Fig._). Eye tubercles, usually possessed by particular groups of post-Palaeozoic eurybathic ostracods, are larger in the shallow-marine counterparts, and are smaller with increasing water depth, eventually disappearing in habitats of middle bathyal or deeper depth. The development of eye tubercles up to ~50u2009u03BCm in diameter (like that shown in Fig._) is an independent line of functional morphological or palaeobiological evidence that can definitely demonstrate the shallow, euphotic environment. Ostracod water-depth reconstruction reinforced by such a known environmentally regulated functional determinant is highly effective rather than relying only on the aforementioned interpretation from comparative autoecology, especially with respect to extant taxa (modern analogue technique). That is, the latter approach is empirical and may be compromised by benthic habitat shifts that can occur through the course of macroevolution on a geologic time scale.\"- - \"With new definitive fossil evidence for near-sea level (<20u2009m at the shallowest) for deposition of Site U1346 sediments, the flat-topped acoustic basement of Shirshov Massifu2009 (Fig._) can be interpreted as a topographic expression of subaerial/wave erosion. Taking a closer look at, the summit flatness is distorted to a minor extent (and to a larger extent in another orthogonal seismic profile), but this feature is explained by later tectonic modifications that accompanied ubiquitous small-scale intrusions, namely the Ojin Rise Seamounts (Fig._). Meanwhile, it is noteworthy that a flat basement top is also seismically imaged for a part of Ori Massifu2009, and that cores just above the Ori Massif basement (IODP Site 1349) have been found to contain some sedimentary components indicating subaerial erosion as well as deposition near the sea level. Altogether, the Shatsky Rise volcanism is understood to have occurred subaerially in the northern region, and possibly in the central region as well, during the final phase of the plateau emplacement.\" - \"Figure 3NNWu2013SSE seismic profile of Shirshov Massif along TN037 ship track from ref. (see also Fig._), with sub-seafloor geological interpretation of igneous basement, Cretaceous chalk/chert, and Cenozoic ooze. mbslu2014meters below sea level.\" - \\' \\' - \"Interestingly, our micropalaeontological evidence for the plateau emergence does not exactly match the independent eruption-depth reconstructions at the same site by means of lava vesicularity and volcanic glass CO2-H2O concentrations (Fig._), both of which give somewhat deeper estimates (with errors) of <300u2009 m and 700u2013800u2009m below sea level, respectively. This discrepancy in estimated palaeo-depths between the sediment and basement sections would be accounted for by a temporarily accelerated final vertical tectonics, uplifting the young, initially submarine plateau to the elevation well above the sea level in a geologic instant.\" - \"Revelation of the early emergent phase of Shatsky Rise suggests that this oceanic LIP had a potential of effectively impacting early Cretaceous biogeochemical cycles via direct emissions of CO2, SO2, halogens, as well as vigorous weathering of basalts and volcaniclastics. Based on available Site U1346 age data (see Methods), though not definitive, the subaerial volcanism and summit weathering/erosion occurred during the Valanginian. This period is well-known for major carbon cycle perturbations, such as enhanced marine carbon burial, global u03B4C shift, and demise of carbonate platforms. The primary trigger of the Valanginian palaeoenvironmental events has been considered by many as the warming induced by the ParanxE1-Etendeka LIP volcanism. This view is supported by Pb isotopic fingerprinting, but unsupported by radiometric dating. Therefore, it may be worthwhile exploring the involvement of Shatsky Rise subaerial volcanism to the Valanginian events. If substantiated, it can explain some of the palaeoenvironmental episodes, including possible anoxic condition in the Pacific, at that time.\" - \"The definitive case history of emergence of Shatsky Rise adds confidence to the universality of this geologically outstanding phenomenon. Major Cretaceous oceanic plateaus of Ontong Java, Manihiki, Kerguelen, and Caribbean have already been known for the physical (fossil and/or rock), albeit patchy, evidence for their past uplifting more or less above the sea level; some developed discussions on their palaeoenvironmental consequences by assuming the massive subaerial volcanism of these plateaus. However, these attempts were not so convincing because none of those oceanic LIPs retain the topographic expressions of large-scale emergence due to later tectonic disturbance of the original plateau geomorphology. Therefore, our combined fossil-seismic evidence from Shatsky Rise is crucial in making reasonable inference, that significant early emergent phases would have been fundamental to Cretaceous oceanic plateaus. It will also help facilitate the study of surface interaction of solid-earth and biogeochemical processes in understanding Earths Cretaceous palaeoenvironmental evolution.\"- - Micropalaeontology - \"For separation of fossil foraminifera and ostracods, sediment samples (see Fig._ caption) were treated in dilute hydrogen peroxide, wet-sieved, dried, and examined under a stereomicroscope at >125u2009u03BCm fraction. Scanning electron microscopic imaging was performed at the SEM Lab, National Museum of Natural History, Smithsonian Institution. Detailed palaeontological description of the microfossils encountered will be reported elsewhere.\" - It is noteworthy that planktonic foraminifera are completely absent in the examined samples, and such observation may also be attributed to a very shallow bathymetric setting. However, during the early Early Cretaceous (Hauterivian and older), planktonic foraminifera were in the course of early evolution with restricted geographic distribution at epicontinental seas as well as small sizes and very low diversity. These palaeobiogeographic and evolutionary contexts more reasonably explain the non-existence of planktonic foraminifera of this age at Site U1346 and other Shatsky Rise sites in the then central Pacific. - Age model - \"Age assignments for the basement and sediment sections of Shirshov Massif are based on the following observations and assumptions. The age of the basement rocks is near the Berriasian/Valanginian boundary (~139u2009Ma), considering the seafloor magnetic lineation of Chron M13-14. Such dating based on magnetic lineation can be validated by recent Ar/Ar geochronology for Tamu Massif (ref.), which demonstrated consistency in age data between the seafloor magnetic anomalies and the actual radiometric measurements. A Hauterivian age for a part of the sedimentary interval is based on nannofossils obtained from Cores 5u2009R and 4R. Overlying chalk/chert interval of Cores 3u2009R to 1u2009W was not dated directly using microfossils (due to rotary core drilling that disintegrated softer chalk sediments, leaving chert nodules only), but it is presumably of Barremian age in light of seismic correlation between ODP Site 1207 (Fig._ ). Of note is the presence of the Desmoceratidae ammonite (total range: late Valanginianu2013 Maastrichtian) (Fig._; Supplementary Fig._) from the basal sedimentary unit of Core 6R-1, which can indicate, at least, that sediments as old as the early Valanginian do not exist. In summary, it is probable that the basement/sediment contact is represented by an unconformity due to subaerial erosion erasing several million years of the Valanginian interval.\"- - \"Coffin, M. F. & Eldholm, O. Large igneous provinces: Crustal structure, dimensions, and external consequences. Rev. Geophys.32, 1u201336 (1994)._ _ _ Bryan, S. E. & Ferrari, L. Large igneous provinces and silicic large igneous provinces: Progress in our understanding over the last 25 years. Geol. Soc. Am. Bull.125, 1053u20131078 (2013)._ _ _ _ Bond, D. P. G. & Wignall, P. B. Large igneous provinces and mass extinctions: An update. Geol. Soc. Am. Spec. Pap.505, 29u201355 (2014)._ Kerr, A. C. & Mahoney, J. J. Oceanic plateaus: Problematic plumes, potential paradigms. Chem. Geol.241, 332u2013353 (2007)._ _ _ _ Frey, F. A., Coffin, M. F., Wallace, P. J. & Weis, D. Leg 183 synthesis: Kerguelen Plateauu2013Broken Ridgeu2014a large igneous province. Proc. ODP Sci. Res. 183, (2003).Kuroda, J. et al. Contemporaneous massive subaerial volcanism and late cretaceous Oceanic Anoxic Event 2. Earth Planet. Sci. Lett.256, 211u2013223 (2007)._ _ _ _ van der Merwe, P. et al. Sourcing the iron in the naturally fertilised bloom around the Kerguelen Plateau: particulate trace metal dynamics. Biogeosciences 12, 739u2013755 (2015)._ _ _ _ Sager, W. W., Kim, J., Klaus, A., Nakanishi, M. & Khankishieva, L. M. Bathymetry of Shatsky Rise, northwest Pacific Ocean: Implications for ocean plateau development at a triple junction. J. Geophys. Res.104, 7557u20137576 (1999)._ _ _ Sager, W. W. et al. IODP Expedition 324: Ocean drilling at Shatsky Rise gives clues about oceanic plateau formation. Sci. Drill.12, 24u201331 (2011)._ _ Sager, W. W., Sano, T. & Geldmacher, J. Formation and evolution of Shatsky Rise oceanic plateau: Insights from IODP Expedition 324 and recent geophysical cruises. Earth-Sci. Rev.159, 306u2013 336 (2016)._ _ Shimizu, K., Shimizu, N., Sano, T., Matsubara, N. & Sager, W. W. Paleo-elevation and subsidence of ~145 Ma Shatsky Rise inferred from CO2 and H2O in fresh volcanic glass. Earth Planet. Sci. Lett.383, 37u2013 44 (2013)._ _ _ _ Expedition 324 Scientists. Site 1346. Proc. IODP324, (2010).Olsson, R. K. & Nyong, E. E. A paleoslope model for Campanian-lower Maastrichtian foraminifera of New Jersey and Delaware. J. Foraminifer. Res.14, 50u201368 (1984)._ _ Koutsoukos, E. A. M. & Hart, M. B. Cretaceous foraminiferal morphogroup distribution patterns, palaeocommunities and trophic structures: a case study from the Sergipe Basin, Brazil. Trans. R. Soc. Edinburgh Earth Sci.81, 221u2013246 (1990)._ _ Ballent, S. C. & Whatley, R. The composition of Argentinian Jurassic marine ostracod and foraminiferal faunas: Environment and zoogeography. Geobios33, 365u2013376 (2000)._ _ Ballent, S. C. & Whatley, R. C. Taxonomy and zoogeography of the Mesozoic cytherurid ostracoda from west-central Argentina. Palaeontology52, 193u2013218 (2009)._ _ Okubo, I. Six species of the subfamily Cytherurinae MxFC ller, 1984, in the Inland Sea, Japan (Ostracoda). Publ. Seto Mar. Biol. Lab. 25, 7u201326 (1980)._ _ Yamada, S., Tsukagoshi, A. & Ikeya, N. Taxonomy, morphology and speciation of the Semicytherura henryhowei group (Crustacea, Ostracoda). Hydrobiologia538, 243u2013265 (2005)._ _ Tanaka, G., Kaseda, Y. & Ikeya, N. Reclassification of the genus Hemicytherura (Crustacea, Ostracoda) from Japan and the surrounding regions. Bull. Gunma Mus. Nat. Hist.15, 19u201342 (2011)._ _ Hazel, J. E. Patterns of marine ostracode diversity in the Cape Hatteras, North Carolina, area. J. Paleontol.49, 731u2013744 (1975)._ Ikeya, N. & Cronin, T. M. Quantitative analysis of Ostracoda and water masses around Japan: application to Pliocene and Pleistocene paleoceanography. Micropaleontology39, 263u2013281 (1993)._ _ Yasuhara, M., Yoshikawa, S. & Nanayama, F. Reconstruction of the Holocene seismic history of a seabed fault using sea-level curves reconstructed by ostracode assemblages: Case study on the Median Tectonic Line in Iyo-nada Bay, western Japan. Palaeogeogr. Palaeoclimatol. Palaeoecol.222, 285u2013312 (2005)._ _ Yasuhara, M. & Seto, K. Holocene relative sea-level change in Hiroshima Bay, Japan: a semi-quantitative reconstruction based on ostracodes. Paleontol. Res.10, 99u2013116 (2006)._ _ Benson, R. H. Estimating greater paleodepths with ostracodes, especially in past thermospheric oceans. Palaeogeogr. Palaeoclimatol. Palaeoecol.48, 107u2013141 (1984)._ _ _ Puckett, T. M. Absolute paleobathymetry of Upper Cretaceous chalks based on ostracodesu2014Evidence from the Demopolis Chalk (Campanian and Maastrichtian) of the northern Gulf Coastal Plain. Geology19, 449u2013 452 (1991)._ _ _ Tanaka, G. Functional morphology and light-gathering ability of podocopid ostracod eyes and the palaeontological implications. Zool. J. Linn. Soc.147, 97u2013108 (2006)._ _ Klaus, A. & Sager, W. W. Data report: high-resolution site survey seismic reflection data for ODP Leg 198 drilling on Shatsky Rise, northwest Pacific. Proc. ODP Init. Rep. 198, (2002).Bornemann, A. & Mutterlose, J. Calcareous nannofossil and u03B4C records from the Early Cretaceous of the western Atlantic Ocean: Evidence for enhanced fertilization across the Berriasianu2013Valanginian transition. Palaios23, 821u2013832 (2008)._ _ _ Westermann, S. et al. The Valanginian u03B4C excursion may not be an expression of a global oceanic anoxic event. Earth Planet. Sci. Lett.290, 118u2013131 (2010)._ _ _ _ FxF6llmi, K. B. Early Cretaceous life, climate and anoxia. Cretac. Res.35, 230u2013257 (2012)._ _ Peate, D. W. Global dispersal of Pb by large-volume silicic eruptions in the ParanxE1-Etendeka large igneous province. Geology37, 1071u20131074 (2009)._ _ _ _ Thordarson, T. Accretionary-lapilli-bearing pyroclastic rocks at ODP Leg 192 Site 1184: a record of subaerial phreatomagmatic eruptions on the Ontong Java Plateau. Geol. Soc. Lond. Spec. Publ.229, 275u2013 306 (2004)._ _ _ Ai, H.-A., Stock, J. M., Clayton, R. & Luyendyk, B. Vertical tectonics of the High Plateau region, Manihiki Plateau, Western Pacific, from seismic stratigraphy. Mar. Geophys. Res.29, 13u2013 26 (2008)._ _ Caron, M. & Homewood, P. Evolution of early planktonic foraminifers. Mar. Micropaleontol. 7, 453u2013462 (1982/83).Geldmacher, J., van den Bogaard, P., Heydolph, K. & Hoernle, K. The age of Earths largest volcano: Tamu Massif on Shatsky Rise (northwest Pacific Ocean). Int. J. Earth Sci.103, 2351u20132357 (2014)._ _ _ Wright, C. W. Treatise on Invertebrate Paleontology, Part L, Mollusca 4 (Revised), Volume 4, Cretaceous Ammonoidea (GSA, Boulder, CO, and Univ. Kansas Press, Lawrence, KS, 1996).Smith, W. H. F. & Sandwell, D. T. Global seafloor topography from satellite altimetry and ship depth soundings. Science277, 1957u20131962 (1997)._ Becker, J. J. et al. Global bathymetry and elevation data at 30 arc seconds resolution: SRTM30_PLUS. Mar. Geod.32, 355u2013371 (2009)._ _ Ando, A. et al. An emerging palaeoceanographic u2018 missing link: multidisciplinary study of rarely recovered parts of deep-sea Santonianu2013Campanian transition from Shatsky Rise. J. Geol. Soc., London 170, 381u2013384 (2013)._ _ _ Shipboard Scientific Party. Site 1207. Proc. ODP Init. Rep. 198, (2002).\"- - This research used samples provided by the IODP. Funding for this research was partly provided by CDEX and JAMSTEC (IODP After Cruise Research Program) (to A.A.), Smithsonian Postdoctoral Fellowship and Smithsonian Marine Science Network Postdoctoral Fellowship, Research Grants Council of the Hong Kong Special Administrative Region, China (HKU 17303115, HKU 17306014) and the Seed Funding Programme for Basic Research of the University of Hong Kong (201611159053, 201511159075) (to M.Y.), and MEXT/JSPS KAKENHI Grant (JP16K17825) (to Y.I.). Permission to use the Global Bathymetry grid (Fig. 1) was granted by Office of Contract and Grant Administration, Scripps Institution of Oceanography, University of California, San Diego.- - Authors and Affiliations - School of Biological Sciences, Swire Institute of Marine Science, and Department of Earth Sciences, the University of Hong Kong, Kadoorie Biological Sciences Building, Pokfulam Road, Hong Kong, SAR, ChinaMoriaki YasuharaDepartment of Paleobiology, National Museum of Natural History, Smithsonian Institution, P.O. Box 37012, MRC 121, Washington, DC 20013-7012, USAAtsushi AndoBugWare, Inc., 1615 Village Square Blvd., Suite 8, Tallahassee, Florida, 32309, USAAtsushi AndoDepartment of Earth and Planetary Sciences, Hokkaido University, N10W8, Kita-ku, Sapporo, Hokkaido, 060-0810, JapanYasuhiro Iba - \"AuthorsMoriaki YasuharaYou can also search for this author in _Atsushi AndoYou can also search for this author in _Yasuhiro IbaYou can also search for this author in _\" - Contributions - A.A. carried out IODP Expedition 324 onboard sampling and designed research; M.Y., A.A. and Y.I. analyzed ostracods, foraminifera and ammonite, respectively; A.A. drafted the paper, and all authors contributed on discussion and interpretation in the paper. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing Interests - \\' \\' - The authors declare that they have no competing interests. - \\' \\'- - \\' Publisher\\'\\'s note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International cLicense, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International licence (CC-BY4.0) as a single PSICQUIC service that can be accessed and searched via the IMEx Consortium website (ww.imexconsortium.org) and any other resource implementing the IMEx PSICQUIC webservice. These include the IntAct website () and the mentha web resource, maintained by the MINT group. - Member databases may import some or all of the IMEx dataset back into their own resources. Partners may add other information to the IMEx dataset on their own websites or choose to only provide access to a subset of data that is of interest to their specific resource. For example, IID complements the IMEx data with interactions predicted by multiple machine learning and data mining algorithms, tissue and disease annotation context, while MatrixDB only provides IMEx data pertaining to extracellular matrix proteins and glycosaminoglycans. This model enables not only large interaction databases to contribute to the overall effort of IMEx but also allows annotation of molecular interactions by groups in data resources that do not maintain an interaction database (e.g. the UniProt Consortium). It also ensures that the dataset will continue to be maintained should funding be withdrawn or a resource decide to focus on a different area of research. For example, the data curated by the Microbial Protein Interaction Database (MPIDB), a former IMEx member, has been maintained and updated within the IntAct database since this resource ceased curation in 2013. The IntAct database additionally holds a considerable number of records curated by several of the IMEx Consortium members (primarily IntAct, MINT, DIP and UniProt), which pre-date the IMEx agreement. There is currently no concerted effort planned to bring this legacy data up to IMEx standards, although individual records may be re-curated on demand. - The Consortium benefits as a whole from the expertise of individual member databases. In addition to the extracellular matrix expertise of MatrixDB, HPIDB bring host-pathogen interaction experience and InnateDB focuses on the curation of molecular interactions involved in the innate immune system. The consortium structure enables a rapid response to new areas of biology, as demonstrated by the early release of a coronavirus interactome curated to full IMEx specifications based on pooled curation resources and the knowledge of individual members such as UniProtKB, which provided early access to protein sequence data.- - During the past 8 years, the IMEx curation model has been enriched and refined with new data types, methodologies and additional levels of detail. Every interaction curated by the IMEx Consortium reflects a piece of experimental evidence manually curated from a publication or directly submitted by a data producer. The IMEx Consortium adheres to a detailed curation model, which comprises all aspects of an interaction experiment. The IMEx record includes information on host organism (with details about the cell line or tissue in which the assay was performed), methods for interaction detection and participant identification, full details of the constructs (binding domains, effects of site-directed mutations, etc.), and further contextual information (e.g. any treatment of the host organism). All this information is mapped to CV terms, in particular those described by the HUPO PSI-MI CV (). Curated data records are linked to the source text from both figure legends and the main text body of the paper. This has enabled the use of these data to develop and assess deep learning approaches for text mining. - \"The Consortium agreement was originally restricted to the curation of PPIs, but this proved limiting to some resources and compromised the ability of users to fully understand biological processes. Therefore, the remit of the Consortium has more recently extended to cover proteinu2013protein complex, proteinu2013 small molecule, proteinu2013carbohydrate, proteinu2013nucleic acid and nucleic acidu2013nucleic acid interactions. Curation guidelines have been developed to enable the description of transcription factor-transcribed gene interactions. Furthermore, experimental techniques to capture ncRNAu2013protein and miRNAu2013 mRNA interactions have been included in the CVs and curation guidelines. The Consortium is currently also examining the structured description of downstream effects of molecular interactions, such as the up- or down-regulation of gene expression.\" - Proteins are curated at the sequence level, using the UniProtKB database as the reference resource for proteins and peptides. The use of UniProtKB enables the curator, for each publication, to accurately describe the level of detail provided about the proteins and to use identifiers for the unambiguous annotation of each protein interactor. For instance, a publication may only give enough detail for an interactor to be mapped to any or all of the protein isoform products of a specific gene, or more specifically to a single protein isoform, or to a post-translationally cleaved peptide chain. UniProtKB supplies appropriate identifiers for all of these, and in each case supplies the corresponding underlying sequence. Binding regions can be aligned to known protein domains, as described by InterPro. The effects of point mutations can be captured down to the amino acid level, using a CV to describe their effect on an interaction. To capture this level of detail, the use of a high-quality protein reference resource is essential. Reverse engineering protein to gene identifiers to enable network analysis of, for example, RNA-Seq data is a relatively trivial task but it is considerably more difficult, if not impossible, to map isoforms and binding domain data directly to a gene model or genomic sequence. Databases that curate PPI data directly to gene identifiers simply do not capture this wealth of information. - The detailed biocuration of binding domains, mutations and post-translational modifications (PTMs) requires that these coordinate-level mappings are kept synchronised with changes to the underlying protein sequence database. An update to a predictive gene model may result in a corresponding change to the protein sequence(s) derived from it. Interactions involving domains and/or residues of that protein sequence then require a corresponding update to ensure that the mapping to the updated sequence is correct. Update pipelines need to be run regularly, in line with the release cycle of the sequence database, namely every 8 weeks in the case of UniProtKB. This is a computationally complex set of procedures run at the EMBL-EBI on the entire IMEx content, ensuring its consistency and representing one advantage of maintaining this as a single dataset. Similarly, all CVs used to describe an aspect of an interaction are updated with every release. - Small molecule interactors, including carbohydrates and lipids, are mapped to the ChEBI database, protein complexes to the Complex Portal, mRNAs to Ensembl or ENA, ncRNAs to RNAcentral and genes to Ensembl. Whilst in many cases the database representation of these entities is largely more stable than that of proteins, update pipelines for each of these will be enhanced and improved with time. - For analyses to be meaningful, data quality and full representation of experimental detail are of paramount importance. This is particularly required when working in an area of high data complexity and heterogeneity. However, capturing these data poses a significant challenge for curators who need to be familiar with an ever-growing set of experimental techniques. In order to minimise curation errors, all IMEx records are double-checked by a second curator prior to release. A more recent innovation is cross-database checking, which ensures that curation standards remain consistent between member databases. This is reinforced by Consortium workshops where rule changes and extensions are agreed, and joint curation exercises undertaken.- - \"The IMEx data is constantly growing, with a new data release approximately every 8 weeks. Whilst there are interactions captured for a wide range of species, Table_ highlights the predominance of human PPI data. The significant fraction of human-other species interactions includes a considerable number of interactions tested with human proteins against close mammalian (primarily mouse) orthologues, but also hostu2013pathogen interactions between human and viral/bacterial proteins. Other model organisms, such as S. cerevisiae, E. coli or A. thaliana are also well represented in the dataset and a curation focus for IMEx partners.\" - \"Table 1 Number of binary proteinu2013protein interactions for selected model organisms present in the IMEx dataset, May 2020.\" - \"The human IMEx data set is dominated by PPIs. The number of interactions involving other molecule types is considerably lower (Table_), but is steadily increasing and will provide interesting extensions to biologically relevant networks.\" - \"Table 2 Number of binary molecular interactions for different interactor types present in the IMEx dataset, May 2020 (humanu2013human data only).\" - \"The curation performed by members of the IMEx Consortium represents a significant proportion of the interaction data found in publicly available databases. There is a degree of data redundancy with other databases that directly curate experimental data such as BioGRID, and the IMEx dataset has been imported by meta-databases such as STRING, mentha, IID and HiPPIE. Figure_ shows the overlap between unique human interacting pairs in all these different resources, plus legacy data curated by IMEx partners as an intersection plot. The greatest overlap exists between IMEx and BioGRID, which is well represented in the meta-databases considered, and between BioGRID and the rest of meta-databases. Figure_ also highlights that the number of interacting pairs in BioGRID is larger than the IMEx dataset. This is due to the less detailed curation model adopted by BioGRID, which allows for faster curation and inclusion of data sets that are not considered as indicating physical interactions by the stringent IMEx rules (e.g., co-fractionation studies).\" - \\'Fig. 1: Overlap between Unique human PPIs from different resources.The intersection plot shows several PPI resources as well as legacy data curated by IMEx partners. Individual resource sizes are displayed as horizontal bars on the lower left corner of the image. Intersection sizes are shown as individual bars, with those including IMEx data highlighted in grey. Specific resources involved in each intersection are identified with connected solid black circles under the vertical bars, with unconnected circles representing pairs that are found exclusively in the corresponding database. Shading in the background of the circles helps differentiate between primary databases (green) and meta-databases (orange). Only the experimentally-derived content of the IID and STRING databases was used, excluding all predicted or text-mined interaction data. Only the 20 largest of all possible intersections are shown for clarity.\\' - \"Figure_ compares the number of unique interacting pairs in the main model organisms (Homo sapiens, Mus musculus, Arabidopsis thaliana, Drosophila melanogaster, Caenorhabditis elegans, and Saccharomyces cerevisiae), highlighting also the type of studies from which the data was curated. Most of the interacting pairs hosted in primary databases come from high-throughput publications (>100 interacting pairs in an experiment or >100 interactors in an n-ary interaction). A comparison with BioGRID is included as this is the only other publicly available, manually curated interaction database.\" - \\'Fig. 2: Number of unique interacting pairs in selected model organisms present in interaction databases.Differently coloured bars indicate the type of studies from which the data were curated. Data were accessed in January 2020.\\' - \"Finally, interactions in IMEx involve most of the representative/canonical proteins present in the human, mouse and S. cerevisiae proteomes as represented in the reviewed UniProtKB/Swiss-Prot (Fig._). Lower coverage is found in other model organisms such as D. melanogaster and C. elegans, partially because smaller fractions of these proteomes are represented in UniProtKB/Swiss-Prot. Many mappings are to proteins in the unreviewed UniProtKB/TrEMBL section and do not necessarily translate into UniProtKB/SwissProt when reviewed by UniProt curators. This makes the estimate of proteome coverage more difficult.\" - \"Fig. 3: Coverage of UniProtKB model organism proteomes in interaction databases.Shown is the fraction of all proteome entries of key model organisms represented in UniProtKB/Swiss-Prot (manually reviewed) and UniProtKB/TrEMBL (computationally inferred) that are found in primary interaction databases_(IMEx, IMEx + legacy data and BioGRID).\"- - ComplexViewer - The ComplexViewer has been designed specifically to visualise detailed data annotated by the IMEx Consortium. Its capabilities include a visual representation of a range of biomolecules as interactor types (proteins, small molecules and nucleic acids), interactions with more than two participants (n-ary interactions), sequence features relevant to the interaction (e.g., binding domains) and stoichiometry information. The data is taken from a JavaScript Object Notation format MI-JSON, which can be generated from any PSI-MI compliant data source using the Java Molecular Interactions library (JAMI, see below). This tool has been incorporated into the Complex Portal, HumanMine () and YeastMine () data warehouses, and is freely available for implementation by additional resources (; ). - ProtVista - The UniProt team has developed ProtVista, an interactive tool for visualisation of a wide range of protein sequence features together in the same space. ProtVista is implemented using JavaScript and makes extensive use of D3 (), a library for producing dynamic, interactive data visualisations in web browsers. Work is currently underway to introduce an interaction track using the IMEx Consortium data, to visualise protein binding domains within the sequence of the protein represented in a UniProtKB entry, and to show the proteins to which it binds. - PSICQUIC - PSICQUIC is a webservice created to enable computational access to standards-compliant molecular interaction data resources. PSICQUIC defines a minimum set of standard SOAP (Simple Object Access Protocol) and REST (Representational state transfer) methods which accept a Molecular Interactions Query Language (MIQL) query as input and return molecular interaction information in one of the standard formats (PSI-XML2.5, MITAB). PSICQUIC enables the IMEx Consortium to make data available to the research community for rapid search and download. The current PSICQUIC implementation provides only limited access, however, to the wealth of data on molecule binding features, such as binding domains and the effect of site-directed mutations, and plans are in place to extend its capabilities. - JAMI - \"The recently published JAMI library has been developed to import and export molecular interaction data in a variety of formats and versions, providing a single change-resilient software component. Software and tools developed on top of the JAMI framework are able to integrate and support all versions of MITAB, PSI-MI XML, MI-JSON and XGMML. JAMIs model interfaces are abstracted from the underlying format, masking the requirements of each data format from developers. JAMI is capable of serving the full richness of IMEx data to tools and resources built on this framework. It has been implemented by the IntAct and Complex Portal databases in addition to the InterMine data warehouse.\" - RpsiXML - RpsiXML provides an R interface to PSI-MI XML2.5 files and can therefore be used to query the IMEx data. It provides a link between the detailed information held in these files and the diverse range of analytical tools provided in the Bioconductor software environment.- - Building high-quality networks for data analysis - Protein interaction networks are used by biologists to understand the interconnectivity of signalling systems inside and outside of the cellular environments, exploring both physiologically normal and disease states. Network-based analysis is a powerful technique for extracting biological insights from large datasets. It enables researchers to associate proteins of unknown functions to human-curated pathways or identify clusters of interacting molecules, which may participate in the same biological process or belong to the same physical complex. Network topology can suggest the biological properties and function of component molecules. Algorithms exist to analyse aspects of network structures (such as their scale-free, small world, geometric random or hierarchical nature), to calculate network metrics (e.g., centralities, shortest path, clustering coefficient and graphlets) or to investigate interaction motifs driving specific interactions between different biomolecules. The results of any network analysis will inevitably depend on the coverage and quality of the protein interaction network and the experimental conditions under which the interacting network was derived. IMEx data-derived networks have been incorporated into several clinical resources such as RD-Connect GPAP (), DISNOR, CancerGeneNET and OpenTargets, enabling researchers to investigate genomic data and the consequence of disease variants. - \"The detailed IMEx curation model enables data selection and filtering on many levels and thus can contribute to the building of high-quality and context-specific networks. At the simplest level, such filtering can be performed using the manually curated u2018interaction type described by a series of terms contained in the MI CV. For example, it is possible to filter for u2018direct interaction , or child terms, thus making specific smaller networks of the very highest quality. This approach was taken by Sacco et al. who queried IMEx data via the mentha interactome browser to build and analyse phosphatase sub-networks to identify new phosphatase substrates. Conversely, data types can be filtered out of a network. For example, computationally expanded data could be removed from a network using CV terms from the u2018curation content (MI:1045) branch which contains child terms such as u2018spoke expansion (MI:1060). IID supports filtering PPIs based on source, number of studies, number of bioassays, broad or detailed tissue, disease, cellular localisation and druggability. The MatrixDB resource supports filtering PPIs based on interaction detection method, gene expression and protein levels, disease, Gene Ontology terms, and UniProtKB keywords to build specific interactomes, e.g. tissue-specific basement membrane interactomes, and to define consensus interactomes composed of the interactions common to all basement membranes. Furthermore, it is possible to search the IntAct website or PSICQUIC webservice using the hierarchical structure of the ontology. For example, the term u2018lipid (CHEBI:18059) will identify all lipid-binding molecules even when the detailed annotation is to a cholesterol (CHEBI:16113) or phosphatidyl 3,4,5 inositol trisphosphate (CHEBI:16618) binding event.\" - \\'Whilst these simple filters have been possible since the first release of IMEx curated data, for the last 6 years IMEx data have been scored using an implementation of MIscore, thus enabling more sophisticated filtering. MIscore relies on the available annotation evidence associated with an interaction and represents the degree of confidence in the existence of a particular interaction. The scoring system takes three factors into account, and uses the CV terms added by the IMEx curators:\\' - \\' (1) How the interaction was observed (interaction detection method; MI:0001) (2) The type of interaction: e.g., direct interaction, physical association and colocalization. (interaction type; MI:0190) (3) The number of publications reporting a specific interaction \\' - \"The results are normalised on a 0u20131 scale.\" - \"Searching the IMEx dataset with the query u201Cintact-miscore:[x TO y]u201D enables the user to select data subsets by confidence score. At the time of writing, the authors recommend a MIscore range of 0.45u20131 to identify medium confidence interactions and 0.6u20131 for high confidence sets. These thresholds approximately correspond to interactions found with at least two distinct pieces of evidence (MIscoreu2009>u20090.45) or those found with three or more pieces of evidence, obtained with different methodologies (MIscoreu2009>u20090.6). The MIscore functionality is used by the Reactome pathways database Molecular Interaction overlay, which allows proteinu2013protein or proteinu2013small molecule interactions to be superimposed onto a pathway diagram. For example, Fig._ shows the tyrosine kinase ZAP70 (UniProtKB: P43403) in the Reactome TCR signaling pathway overlaid with 9 protein interactors imported from IMEx. The default setting provides fast access to a quarterly updated and locally hosted version of IMEx data with a MIscore threshold of 0.45, which selects ~50% of all interactions in IMEx. Interactions with a confidence level equal to or above this threshold will be visible in the viewport. The Reactome pathway analysis service also gives the user the option to set a more stringent MIscore filter using a slider feature, to select alternative interaction databases, and to perform an extended analysis that includes the IMEx interaction dataset served from the IntAct database.\" - \\'Fig. 4: Presentation of IMEx data in different resources.a, b IMEx data for the human ZAP70 protein (P43403) represented in a Reactome and b UniProt. c IMEx-derived binding domain data for the S. cerevisiae vesicular SNARE complex SSO1-SEC9-SNC1 (CPX-1365) used in the Complex Portal.\\' - \"Two studies independently applied data filtering to essentially the same network to investigate the biology of LRRK2, a protein linked to familial forms of Parkinson s disease, using the data made publicly available by IMEx partners. Porras et al. filtered the dataset using MIscore to generate a high-confidence sub-network, which was used to produce a draft list of high confidence LRRK2 interacting partners. Manzoni et al. filtered by the number of publications reporting an interaction and then performed Gene Ontology network analysis on the LRRK2 interactome with edges identified by 2 or more publications. Both groups showed that the LRRK2 network was associated with terms referring to transport, cellular organization, vesicles and the cytoskeleton. Experimental data have since shown LRRK2 to be associated with selected Rab GTPases, and are now also present in the IMEx dataset.\" - \"MIscore was designed to be customisable and a different version of the algorithm is used to identify binary interactions for export to the UniProtKB Interaction lines and to the Gene Ontology database. All binary interaction evidence in the IntAct database, including the data generated by spoke expansion of co-complex data, are clustered to produce a non-redundant set of protein pairs. Each binary pair is then scored using a variation of MIscore to give a simple addition of the accumulated value of a weighted score for the interaction detection method and the interaction type for each interaction evidence associated with that binary pair. Once the interactions have been scored, a threshold is established, below which the interaction is not exported to UniProtKB and to the Gene Ontology annotation files. The threshold is stringent enough to ensure no interaction is exported based on only a single piece of experimental evidence. Additional rules ensure that any protein pair scoring above the cut-off must also include at least one piece of evidence of a binary pair, excluding spoke expanded and co-localisation data, to be exported to UniProtKB and Gene Ontology annotation. The UniProtKB dataset is then displayed in the appropriate entries in an adjacency viewer (Fig._).\" - \"The IMEx dataset has also been used as a gold-standard training set in text-mining exercises such as the Biocreative competitions. These exercises make use of IMEx data partly because a linkage between figures and experiments has been maintained during the process of IMEx biocuration. Furthermore, in a subset of IMEx entries, sentences identifying interacting entities and the interaction detection method have been systematically captured in the u2018source-text annotation field. These sentences are highlighted for readers on abstracts and full text articles in Europe PMC via the SciLite tool.\" - Characterising protein isoforms and features - \"Most eukaryotic protein-coding genes transcribe more than one isoform. The different functions of isoforms are sometimes known or can be inferred (for example specific isoforms do/do not contain certain functional domains), but in many cases the biological significance of multiple isoforms derived from the same gene is not understood. However, the different interaction patterns of associated isoforms may provide an indication of their different biological functions by analysing their respective binding partners. In 2013, Talavera et al. published an editorial stating that u201Cit is crucial to the advance of basic and medical research that interactions are reported on an isoform-to-isoform basis and that databases switch to a similar approachu201D. The IMEx databases curate this information, whenever the data is made available by authors, making isoform comparisons possible. UniProtKB identifiers enable curators to differentiate between transcripts being identified at the isoform or canonical (reference sequence) level. Over 100,000 interactions in IMEx (~12% of IMEx data) contain specific isoform information, with more than 11,000 records containing specific isoformu2013isoform interactions. The UniProtKB database recently (release 2020_02) refactored the Interaction section of their records to improve the display of isoform data imported from IMEx. It is anticipated that the availability of such data will increase as protein identification techniques improve or as authors realise the value of such data and include this level of detail in publications.\" - IMEx also captures so-called negative interactions, which will be of increasing use in the future. These data largely pertain to isoform-specific interactors, and describe cases where certain isoforms of a gene bind to a bait protein, while other isoforms of the same gene do not bind to the same bait in the same assay system. IMEx curation rules mandate publication of the protein expression levels of the negative interactors to exclude poor protein expression as a reason for the lack of interaction. - To fully comprehend protein interactions, researchers frequently need to identify the sequence region to which a molecule binds and any modification to that sequence. Any change to an amino acid sequence has the potential to influence the molecules with which the protein interacts. The IMEx Consortium captures these variations, thereby supporting the analysis of their downstream effects as shown in the examples below. - Binding domains - \"One critical piece of information captured by the information-rich IMEx curation model is the minimum u2018sufficient binding region (MI:0442) or u2018 necessary binding region (MI:0429) of a protein derived from an interaction experiment. When a binding domain maps to a known protein domain, a cross-reference to the appropriate InterPro entry is added. Capture of data to this level of detail has enabled, for example, an improved understanding of the role of the SH2 domain including a classification of its target protein specificity and the identification of the WD40 domain as potentially being directly involved in ncRNA interactions. The binding regions captured by the IMEx Consortium have also been used for the precise mapping of binding domains within protein complexes in the EBI Complex Portal, e.g. in the Saccharomyces cerevisiae vesicular SNARE complex SSO1-SEC9-SNC1 (CPX-1365) formed by SNARE-SNARE domain binding (Fig._ ).\" - Post-translational modifications - \"PTMs, such as the phosphorylation of amino acid side chains, increase the complexity of the proteome and are essential for driving molecular interactions. These modifications can change PPIs by causing protein oligomerization and aggregation, binding to or dissociation from other proteins, protein conformational changes or local unfolding. The IMEx Consortium differentiates between a u2018prerequisite-ptm (MI:0638), which is required for an interaction to occur, and an u2018observed-ptm (MI:0925), which has been experimentally validated but not shown to be required for the interaction. For example, a phosphorylation event can introduce a charge in a hydrophobic environment, destabilising an interaction. This can be systematically described using CV terms such as u2018ptm disrupting an interaction (MI:1225).\" - Post-translational cleavage of a polypeptide is also a PTM yielding a mature protein or bioactive peptide chains, which may have interaction repertoires very different from those of the originating full-length transcript. Using the mature protein chain identifiers supplied by the UniProtKB database allows IMEx curators to accurately capture the form of the protein used in the assay. This is of particular importance for the annotation of the interactions of viral proteins where one gene in the viral genome may encode multiple proteins. The protein interactions of these post-processed protein and peptide chains cannot be meaningfully described when protein interactions are only captured at the gene level. - \"Reversible and transient PTMs transmit and amplify signals in a highly regulated manner by reversible site-specific modulation, and thus play a key role in signal transduction. PTMs are often the result of an enzyme acting on a substrate and the enzymeu2013substrate reaction can be taken as evidence of a direct interaction in the IMEx data model. The PTM resulting from this interaction is additionally captured, using the u2018resulting-PTM (MI:0639) term. Cell signalling resources, such as the SIGNOR database have used the relationships between enzymes and substrates from the IMEx dataset and the effects of resulting PTMs on interactions to derive causal interactions.\" - More recently, the IMEx Consortium has also started to capture the effects of chemical modifications of RNA molecules, several of which undergo specific nucleotide modifications during their maturation. - Point mutations - \"To understand how amino acid variations influence protein function and stability, researchers have for many years examined the effect of induced point mutations on protein interactions. These targeted changes to the amino acid sequence of a protein may mimic known sequence variants, remove post-translational modification sites, disrupt regions required for protein stability or alter the properties of protein binding domains. The IMEx Consortium has been collecting these data using CV terms to describe the observed effects such as u2018mutation decreasing interaction strength (MI:0116), u2018mutation increasing interaction rate (MI:1131) or u2018mutation causing an interaction (MI:2227). The curation rules have recently been extended to include deep mutational scanning data such as described by Woodsmith et al.. At the time of writing, this set consists of 58,000 point mutations representing 20,000 interaction evidences annotated with differentially reported effects. To make these data more readily available to the user community, the IMEx Consortium has recently concatenated this dataset and made it available in a tab-delimited format (FeatureTAB). This new data format includes details of the position and the amino acid change of the mutation, the interacting molecules and the effect of the mutation on the interaction.\" - \"The IMEx Consortium mutation-specific dataset is available to download at . The data have already been used to provide potential mechanisms of action for disease related amino acid variants, to investigate the destabilising effect of mutations on structural models of proteinu2013protein interfaces, and to benchmark tools predicting the effect of SNPs on protein function. In another recent study, both the protein interaction network from IID and the mutation-specific interaction dataset have been used to explain survival and treatment resistance in ovarian cancer. Patients with a TP53 p.Arg175His mutation have a poor prognosis, which may be explained by chemoresistance due to a disrupted TP53 interaction with the BCL2 complex as well as several additional mechanisms, as highlighted in Fig._.\" - \\'Fig. 5: Mutation-specific TP53 interaction network.Survival-related patient mutation data overlaid onto the mutation-specific TP53 network derived from interactions from IID (version 2020-05) and mutation-specific interaction data from IMEx. Survival-related mutations are shown on the network edges. It should be noted that many interactions are affected by many more mutations, but these are not necessarily linked to ovarian cancer. The network is visualised in NAViGaTOR ver 3.0.13. Node colour corresponds to GO molecular function, edge colour represents the effect of the mutation on the interaction.\\' - The IMEx database also collects information concerning the nucleotides involved in miRNA-mRNA interactions and the effect of nucleic acid substitutions on binding. These data are accurately mapped on the mRNA sequence, making them a valuable resource for modelling miRNA interactions and the regulation of gene expression at specific targets. To the best of our knowledge, the IMEx Consortium is the only group capturing this information. - Protein tags - Protein tags are chemical additions to a molecule to enable its identification and/or purification. They often consist of peptide sequences genetically attached onto a recombinant protein. As the nature and position of a tag may affect its interaction profile, this information must be recorded to enable researchers to fully comprehend the consequences of such changes. For example, a tag expressed as either an N- or C-terminal fusion of Ebola Virus VP24 protein identified 48 and 51 interacting human proteins, respectively, of which only 40 proteins are common to both fusions. - Contextualising protein interactions with metadata - As detailed above, the interaction repertoire of a protein depends on many factors, not least the cellular environment in which they are expressed. The detailed metadata now captured in an IMEx record have been significantly expanded since the work of the Consortium was first described. Most techniques for measuring interactions do not use native conditions and, when described by the author in the publication, these data are captured in a systematic way using ontologies and CV terms. In addition to full details on the host organism, experimental methodology and construct details, IMEx curators now routinely capture expression level, any full or partial purification of a molecule, and the method by which constructs are delivered or engineered into a cell or expression system. This information allows analysing the effects of environmental change on interaction patterns. For example, it has been noted that the Huh7.5 cell line differs from the 7.0 parent cells by a single mutation (p.Thr55Ile) in Retinoic acid-inducible gene I (RIG-I) protein, which impairs interferon signaling. Analyses by the MacCarthy group using data uniquely provided by IMEx suggest notable differences in the pattern of host proteins interacting with HCV proteins from these two different cell lines. Huh7 proteins interacting with proteins from HCV are involved in cancer, apoptosis, immune defence response and cell cycle functions, whilst the equivalent Huh7.5 proteins are enriched for protein folding, localisation, and transport. - \"Recent changes to both the database model and PSI-MI XML3.0 download format allow curators to capture dynamic interactions, such as changes in sub-network composition at different stages of the cell cycle or in response to changing concentrations of an agonist/antagonist, pH changes and other factors. It is also possible to describe the directionality of a reaction or binding event and the result of a directional binding event, such as an up- or down-regulation of the target s activity. Once sufficient data have been accumulated, they will be made available to users in a new tab-delimited format.\"- - \"Since first described in 2012, the IMEx Consortium has gained new member data resources and curation groups; these include the UniProt Consortium and the Functional Gene Annotation group at University College London. Some previous member resources u2015 such as MPIDB u2015 are no longer in existence, but the data remain in the IntAct database and are updated with each release of UniProt. The IMEx Consortium has released almost 900,000 interaction evidences to IMEx curation standards and continues to provide access to another 100,000 legacy binary evidences. It has expanded the IMEx dataset to include new interactor types, new methodologies and new data types such as dynamic interactions. The Consortium remains open to the participation of new partners, and will make access to the IntAct editorial tool, curation training and data quality control available on request. For detailed information on both IMEx membership and on data deposition, please see .\" - \"Data producers can contribute to the IMEx project in three key ways. First, by depositing interaction data with one of the Consortium partners as an integral part of the publication process. Second, by always clearly identifying all the constructs used in any interaction experiment, ideally by the addition of an accession number from a database such as UniProtKB or by making the species of origin of any clone very clear (for example u201CHuman hemagglutinin (HA)-tagged RRP1B (Q14684)u2026.u201D). Additional sequence detail will enable mapping to the correct isoform, when relevant. Third, data producers can request the curation of papers, particularly when these supply interactions missing from an interactome, or bring additional details to an interaction that is not already present in the dataset. Researchers seeking assistance with these requirements may contact intact-help@ebi.ac.uk.\" - The IMEx Consortium received EC funding to establish itself and has more recently received UK BBSRC and NIH grants. Currently, the consortium relies on localised, national funding and research grants to maintain resources. IMEx is the first Consortium to be recognised as an ELIXIR core resource, highlighting the importance this organisation places on database collaboration and data sharing. ELIXIR has supplied local funding to support member databases and to fund Consortia meetings. It is hoped that the recognition by ELIXIR will result in longer term funding for Consortium-wide activities. The Global Biodata Coalition is currently looking to extend this model of sustainable funding for core-data resources in the life sciences giving hope for the long-term future of key resources such as the IMEx Consortium.- - \"Das, J. et al. Exploring mechanisms of human disease through structurally resolved protein interactome networks. Mol. Biosyst. 10, 9u201317 (2014)._ _ _ _ _ Correia, F. B., Coelho, E. D., Oliveira, J. L. & Arrais, J. P. Handling noise in protein interaction networks. Biomed. Res. Int. 2019, 8984248 (2019)._ _ _ _ _ Luck, K., Sheynkman, G. M., Zhang, I. & Vidal, M. Proteome-scale human interactomics. Trends Biochem. Sci. 42, 342u2013354 (2017)._ _ _ _ _ von Mering, C. et al. Comparative assessment of large-scale data sets of protein-protein interactions. Nature 417, 399u2013 403 (2002)._ _ _ _ Venkatesan, K. et al. An empirical framework for binary interactome mapping. Nat. Methods 6, 83u2013 90 (2009)._ _ _ _ Choi, S. G. et al. Maximizing binary interactome mapping with a minimal number of assays. Nat. Commun. 10, 3907 (2019)._ _ _ _ _ _ Orchard, S. et al. Protein interaction data curation: the International Molecular Exchange (IMEx) consortium. Nat. Methods 9, 345u2013350 (2012). First description of the IMEx Consortium and detailing of the curation model._ _ _ _ _ Orchard, S. et al. The MIntAct projectu2013 IntAct as a common curation platform for 11 molecular interaction databases. Nucleic Acids Res. 42, D358u2013D363 (2014)._ _ _ _ Xenarios, I. et al. DIP: The Database of Interacting Proteins: 2001 update. Nucleic Acids Res. 29, 239u2013241 (2001)._ _ _ _ _ UniProt Consortium. UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res. 47, D506u2013D515 (2019)._ _ _ Clerc, O. et al. MatrixDB: integration of new data with a focus on glycosaminoglycan interactions. Nucleic Acids Res. 47, D376u2013D381 (2019)._ _ _ _ Breuer, K. et al. InnateDB: systems biology of innate immunity and beyondu2013recent updates and continuing curation. Nucleic Acids Res. 41, D1228u2013D1233 (2013)._ _ _ _ Ammari, M. G., Gresham, C. R., McCarthy, F. M. & Nanduri, B. HPIDB 2.0: a curated database for host-pathogen interactions. Database 2016, baw103 (2016).Pastrello, C., Kotlyar, M. & Jurisica, I. Informed use of protein-protein interaction data: a focus on the Integrated Interactions Database (IID). Methods Mol. Biol. 2074, 125u2013134 (2020)._ _ _ _ Drysdale, R. et al. The ELIXIR Core Data Resources: fundamental infrastructure for the life sciences. Bioinformatics 36, 2636u20132642 (2020)._ _ _ _ Orchard, S. & Hermjakob, H. Shared resources, shared costsu2013 leveraging biocuration resources. Database. (2015).Kerrien, S. et al. Broadening the horizonu2013level 2.5 of the HUPO-PSI format for molecular interactions. BMC Biol. 5, 44 (2007). Description of the formats, standards and controlled vocabularies which enable release of the IMEx data set in a single format which is re-usable by other resources._ _ _ _ _ Sivade Dumousseau, M. et al. Encompassing new use cases - level 3.0 of the HUPO-PSI format for molecular interactions. BMC Bioinform. 19, 134 (2018)._ _ _ Wilkinson, M. D. et al. The FAIR Guiding Principles for scientific data management and stewardship. Sci. Data 3, 160018 (2016)._ _ _ _ Aranda, B. et al. PSICQUIC and PSISCORE: accessing and scoring molecular interactions. Nat. Methods 8, 528u2013529 (2011)._ _ _ _ _ del-Toro, N. et al. A new reference implementation of the PSICQUIC web service. Nucleic Acids Res. 41, W601u2013 W606 (2013)._ _ _ _ Calderone, A., Castagnoli, L. & Cesareni, G. mentha: a resource for browsing integrated protein-interaction networks. Nat. Methods 10, 690u2013691 (2013)._ _ _ _ Kotlyar, M., Pastrello, C., Malik, Z. & Jurisica, I. IID 2018 update: context-specific physical protein-protein interactions in human, model organisms and domesticated species. Nucleic Acids Res. 47, D581u2013D589 (2019)._ _ _ _ Kotlyar, M., Pastrello, C., Sheahan, N. & Jurisica, I. Integrated interactions database: tissue-specific view of the human and model organism interactomes. Nucleic Acids Res. 44, D536u2013 D541 (2016)._ _ _ _ Launay, G., Salza, R., Multedo, D., Thierry-Mieg, N. & Ricard-Blum, S. MatrixDB, the extracellular matrix interaction database: updated content, a new navigator and expanded functionalities. Nucleic Acids Res. 43, D321u2013D327 (2015)._ _ _ _ Perfetto, L. et al. The IMEx Coronavirus interactome: an evolving map of Coronaviridae-Host molecular interactions. Database (Oxford). 2020:baaa096. (2020).Burns, G. A., Li, X. & Peng, N. Building deep learning models for evidence classification from the open access biomedical literature. Database 2019, baz034 (2019).Mitchell, A. L. et al. InterPro in 2019: improving coverage, classification and access to protein sequence annotations. Nucleic Acids Res. 47, D351u2013 D360 (2019)._ _ _ _ IMEx Consortium Curators. Capturing variation impact on molecular interactions in the IMEx Consortium mutations data set. Nat. Commun. 10, 10 (2019). IMEx dataset describing the effects of targeted mutations of the amino acid sequence of a protein on molecular interactions._ _ _ _ Hastings, J. et al. ChEBI in 2016: improved services and an expanding collection of metabolites. Nucleic Acids Res. 44, D1214u2013D1219 (2016)._ _ _ _ Meldal, B. H. M. et al. The complex portalu2013an encyclopaedia of macromolecular complexes. Nucleic Acids Res. 43, D479u2013D484 (2015)._ _ _ _ Yates, A. D. et al. Ensembl 2020. Nucleic Acids Res. 48, D682u2013D688 (2020)._ _ _ _ Amid, C. et al. The European Nucleotide Archive in 2019. Nucleic Acids Res. 48, D70u2013D76 (2020).The RNAcentral Consortium. RNAcentral: a hub of information for non-coding RNA sequences. Nucleic Acids Res. 47, D1250u2013 D1251 (2019)._ _ Oughtred, R. et al. The BioGRID interaction database: 2019 update. Nucleic Acids Res. 47, D529u2013D541 (2019)._ _ _ _ Szklarczyk, D. et al. STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Res. 47, D607u2013D613 (2019)._ _ _ _ Alanis-Lobato, G., Andrade-Navarro, M. A. & Schaefer, M. H. HIPPIE v2.0: enhancing meaningfulness and reliability of protein-protein interaction networks. Nucleic Acids Res. 45, D408u2013D414 (2017)._ _ _ _ Lex, A., Gehlenborg, N., Strobelt, H., Vuillemot, R. & Pfister, H. UpSet: Visualization of Intersecting Sets. IEEE Trans. Vis. Comput. Graph. 20, 1983u20131992 (2014)._ _ _ _ Combe, C. W. et al. ComplexViewer: visualization of curated macromolecular complexes. Bioinformatics 33, 3673u20133675 (2017)._ _ _ _ _ Sivade Dumousseau, M. et al. JAMI: a Java library for molecular interactions and data interoperability. BMC Bioinform. 19, 133 (2018)._ _ _ Smith, R. N. et al. InterMine: a flexible data warehouse system for the integration and analysis of heterogeneous biological data. Bioinformatics 28, 3163u20133165 (2012)._ _ _ _ _ Balakrishnan, R. et al. YeastMineu2013 an integrated data warehouse for Saccharomyces cerevisiae data as a multipurpose tool-kit. Database 2012, bar062 (2012)._ _ _ _ _ Watkins, X., Garcia, L. J., Pundir, S. & Martin, M. J. & UniProt Consortium. ProtVista: visualization of protein sequence annotations. Bioinformatics 33, 2040u20132041 (2017)._ _ _ _ _ Gentleman, R. C. et al. Bioconductor: open software development for computational biology and bioinformatics. Genome Biol. 5, R80 (2004)._ _ _ _ Rahmati, S. et al. pathDIP 4: an extended pathway annotations and enrichment analysis resource for human, model organisms and domesticated species. Nucleic Acids Res. 48, D479u2013D488 (2020)._ _ _ Lo Surdo, P. et al. DISNOR: a disease network open resource. Nucleic Acids Res. 46, D527u2013D534 (2018)._ _ _ _ Iannuccelli, M. et al. CancerGeneNet: linking driver genes to cancer hallmarks. Nucleic Acids Res. 48, D416u2013D421 (2020)._ _ _ _ Koscielny, G. et al. Open Targets: a platform for therapeutic target identification and validation. Nucleic Acids Res. 45, D985u2013D994 (2017)._ _ _ _ Sacco, F. et al. Combining affinity proteomics and network context to identify new phosphatase substrates and adapters in growth pathways. Front. Genet. 5, 115 (2014)._ _ _ _ _ Villaveces, J. M. et al. Merging and scoring molecular interactions utilising existing community standards: tools, use-cases and a case study. Database 2015, bau131 (2015).Croft, D. et al. The Reactome pathway knowledgebase. Nucleic Acids Res. 42, D472u2013D477 (2014)._ _ _ _ _ Porras, P. et al. A visual review of the interactome of LRRK2: Using deep-curated molecular interaction data to represent biology. Proteomics 15, 1390u20131404 (2015)._ _ _ _ _ Manzoni, C., Denny, P., Lovering, R. C. & Lewis, P. A. Computational analysis of the LRRK2 interactome. PeerJ 3, e778 (2015)._ _ _ _ _ Steger, M. et al. Phosphoproteomics reveals that Parkinsons disease kinase LRRK2 regulates a subset of Rab GTPases. Elife 5, e12813 (2016).Burns, G. A. P. C., Dasigi, P., de Waard, A. & Hovy, E. H. Automated detection of discourse segment and experimental types from the text of cancer pathway results sections. Database 2016, 1u201312 (2016)._ _ Levchenko, M. et al. Europe PMC in 2017. Nucleic Acids Res. 46, D1254u2013D1260 (2018)._ _ _ _ Talavera, D., Robertson, D. L. & Lovell, S. C. Alternative splicing and protein interaction data sets. Nat. Biotechnol. 31, 292u2013293 (2013). Publication arguing that isoform-to-isoform protein interactions should be reported to reflect the isoform-based specificity of interactions._ _ _ _ Tinti, M. et al. The SH2 domain interaction landscape. Cell Rep. 3, 1293u20131305 (2013)._ _ _ _ _ Panni, S., Prakash, A., Bateman, A. & Orchard, S. The yeast noncoding RNA interaction network. RNA 23, 1479u20131492 (2017)._ _ _ _ _ Lee, M. J. & Yaffe, M. B. Protein regulation in signal transduction. Cold Spring Harb. Perspect. Biol. 8, a005918 (2016).Licata, L. et al. SIGNOR 2.0, the SIGnaling Network Open Resource 2.0: 2019 update. Nucleic Acids Res. 48, D504u2013D510 (2020)._ _ _ Woodsmith, J. et al. Protein interaction perturbation profiling at amino-acid resolution. Nat. Methods 14, 1213u20131221 (2017)._ _ _ _ Mandilaras, V. et al. TP53 mutations in high grade serous ovarian cancer and impact on clinical outcomes: a comparison of next generation sequencing and bioinformatics analyses. Int. J. Gynecol. Cancer. (2019)._ _ _ Ferlini, C. et al. Bcl-2 down-regulation is a novel mechanism of paclitaxel resistance. Mol. Pharmacol. 64, 51u201358 (2003)._ _ _ _ Makhija, S., Taylor, D. D., Gibb, R. K. & GerxE7 el-Taylor, C. Taxol-induced bcl-2 phosphorylation in ovarian cancer cell monolayer and spheroids. Int. J. Oncol. 14, 515u2013521 (1999)._ _ _ Rajagopala, S. V., Hughes, K. T. & Uetz, P. Benchmarking yeast two-hybrid systems using the interactions of bacterial motility proteins. Proteomics 9, 5296u20135302 (2009)._ _ _ _ _ Ammari, M., McCarthy, F. & Nanduri, B. Leveraging experimental details for an improved understanding of host-pathogen interactome. Curr. Protoc. Bioinform. 61, 8.26.1u20138.26.12 (2018)._ _ _ GarcxED a-Dorival, I. et al. Elucidation of the Ebola virus VP24 cellular interactome and disruption of virus biology through targeted inhibition of host-cell protein function. J. Proteome Res. 13, 5120u20135135 (2014)._ _ _ _ Liu, H. M. & Gale, M. Hepatitis C virus evasion from RIG-I-dependent hepatic innate immunity. Gastroenterol. Res. Pract. 2010, 548390 (2010)._ _ _ Perfetto, L. et al. CausalTAB: the PSI-MITAB 2.8 updated format for signalling data representation and dissemination. Bioinformatics 35, 3779u20133785 (2019)._ _ _ _ _ Orchard, S. et al. The minimum information required for reporting a molecular interaction experiment (MIMIx). Nat. Biotechnol. 25, 894u2013898 (2007)._ _ _ _ Anderson, W. P. & Global Life Science Data Resources Working Group. Data management: a global coalition to sustain core data. Nature 543, 179 (2017)._ _ _ _ _ Brown, K. R. et al. NAViGaTOR: Network analysis, visualization and graphing Toronto. Bioinformatics 25, 3327u20133329 (2009)._ _ _ _ _ \"- - \"The IntAct team at EMBL-EBI received funding from EMBL core funding, Open Targets (grant agreements OTAR-044 and OTAR02-048) and the Wellcome Trust (Biomedical Resources grant INVAR #3367). This work by UniProt was supported by the National Eye Institute (NEI), National Human Genome Research Institute (NHGRI), National Heart, Lung, and Blood Institute (NHLBI), National Institute on Aging (NIA), National Institute of Allergy and Infectious Diseases (NIAID), National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK, National Institute of General Medical Sciences (NIGMS), National Cancer Institute (NCI) and National Institute of Mental Health (NIMH) of the National Institutes of Health under Award Number [U24HG007822] (the content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health) and EMBL core funding and Open Targets (grant agreements OTAR02-048). Support for IID was supplied in part by Ontario Research Fund (#34876), Natural Sciences Research Council (NSERC #203475) and Canada Foundation for Innovation (CFI #29272, #225404 and #33536). MatrixDB was supported by the Fondation pour la Recherche MxE9dicale (grant DBI20141231336 to S.R.B.) and by the Institut FranxE7ais de Bioinformatique (GlycoMatrixDB project- AAP IFB 2015 to S.R.B.). InnateDB was previously supported by Genome BC through the Pathogenomics of Innate Immunity (PI2) project and by the Foundation for the National Institutes of Health and the Canadian Institutes of Health Research under the Grand Challenges in Global Health Research Initiative [Grand Challenges ID: 419]. Further funding was also provided by AllerGen grants 12ASI1 and 12B&B2. D.J.L. is currently supported by an EMBL Australia Group Leader award. The Database of Interacting Proteins (DIP) was supported by National Institute of General Medical Sciences grant R01GM123126.\"- - Authors and Affiliations - \"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Campus, Hinxton, Cambridge, CB10 1SD, UKPablo Porras,_Elisabet Barrera,_Noemi del-Toro,_Margaret Duesbury,_Henning Hermjakob,_Birgit Meldal,_ Kalpana Paneerselvam,_Livia Perfetto,_Anjalia Shrivastava_&_Sandra OrchardSIB Swiss Institute of Bioinformatics, Centre Medical Universitaire, 1 rue Michel Servet, CH-1211, Geneva, SwitzerlandAlan BridgeUniversity of Rome Tor Vergata, Rome, ItalyGianni Cesareni,_Marta Iannuccelli_&_Luana LicataIRCCS Fondazione Santa Lucia, 00143, Rome, ItalyGianni CesareniUCLA-DOE Institute, University of California, Los Angeles, CA, 90095, USAMargaret Duesbury,_Negin Rahimzadeh,_ Prashansa Ratan,_Lukasz Salwinski_&_Gautam ShirodkarOsteoarthritis Research Program, Division of Orthopedic Surgery, Schroeder Arthritis Institute, and Krembil Research Institute, University Health Network, 60 Leonard Avenue, 5KD-407, Toronto, ON, M5T 0S8, CanadaIgor Jurisica,_Max Kotlyar_&_Chiara PastrelloDepartments of Medical Biophysics, and Computer Science, University of Toronto, Toronto, ON, CanadaIgor JurisicaInstitute of Neuroimmunology, Slovak Academy of Sciences, Bratislava, SlovakiaIgor JurisicaFunctional Gene Annotation, Preclinical and Fundamental Science, UCL Institute of Cardiovascular Science, University College London, London, WC1E 6JF, UKRuth C. LoveringComputational and Systems Biology Program, Precision Medicine Theme, South Australian Health and Medical Research Institute, Adelaide, SA, 5000, AustraliaDavid J. LynnCollege of Medicine and Public Health, Flinders University, Bedford Park, SA, 5042, AustraliaDavid J. LynnInstitute for Genomics, Biocomputing and Biotechnology, Mississippi State University, Starkville, MS, USABindu NanduriUniversitxE0 della Calabria, Dipartimento di Biologia, Ecologia e Scienze della Terra, Via Pietro Bucci Cubo 6/C, Rende, CS, ItalySimona PanniDepartment of Molecular, Cell and Developmental Biology, UCLA, Box 951606, Los Angeles, CA, 90095-1606, USAMatteo PellegriniICBMS, UMR 5246 University Lyon 1 - CNRS, Univ. Lyon, 69622, Villeurbanne, FranceSylvie Ricard-BlumOpen Targets, Wellcome Genome Campus, Hinxton, Cambridge, CB10 1SD, UKAnjalia Shrivastava\" - \"AuthorsPablo PorrasYou can also search for this author in _Elisabet BarreraYou can also search for this author in _Alan BridgeYou can also search for this author in _Noemi del-ToroYou can also search for this author in _Gianni CesareniYou can also search for this author in _Margaret DuesburyYou can also search for this author in _Henning HermjakobYou can also search for this author in _Marta IannuccelliYou can also search for this author in _Igor JurisicaYou can also search for this author in _Max KotlyarYou can also search for this author in _Luana LicataYou can also search for this author in _Ruth C. LoveringYou can also search for this author in _David J. LynnYou can also search for this author in _Birgit MeldalYou can also search for this author in _Bindu NanduriYou can also search for this author in _Kalpana PaneerselvamYou can also search for this author in _Simona PanniYou can also search for this author in _Chiara PastrelloYou can also search for this author in _Matteo PellegriniYou can also search for this author in _Livia PerfettoYou can also search for this author in _Negin RahimzadehYou can also search for this author in _Prashansa RatanYou can also search for this author in _Sylvie Ricard-BlumYou can also search for this author in _Lukasz SalwinskiYou can also search for this author in _Gautam ShirodkarYou can also search for this author in _Anjalia ShrivastavaYou can also search for this author in _Sandra OrchardYou can also search for this author in _\" - Contributions - S.O. and P.P. wrote this manuscript with revisions by A.B., H.H. D.J.L, I.J, L.L., R.C.L. and M.P.; P.P., A.B., M.D., G.C., M.I., L.L., M.K., B.M., B.N., C.P., K.P., L.P., S.P., N.R., P.R., S.R.-B., G.S. and S.O. contributed to the dataset generation; and E.B., N.d.-T., A.S. and L.S. contributed to its maintenance and release. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International license. The code used for generating the figures including the EMCD optimization routines is likewise available under the GNU public license 3.0. This code is presented in a human-readable format as the supplementary information to this manuscript. - \\' \\'- - \"Chao, W., Harteneck, B. D., Liddle, J. A., Anderson, E. H. & Attwood, D. T. Soft X-ray microscopy at a spatial resolution better than 15 nm. Nature 435, 1210u2013 1213, (2005)._ _ _ _ _ Wiesendanger, R. et al. Topographic and Magnetic-Sensitive Scanning Tunneling Microscope Study of Magnetite. Science 255, 583u2013586, (1992)._ _ _ _ _ Heinze, S. et al. Real-Space Imaging of Two-Dimensional Antiferromagnetism on the Atomic Scale. Science 288, 1805u20131808, (2000)._ _ _ _ _ Kaiser, U., Schwarz, A. & Wiesendanger, R. Magnetic exchange force microscopy with atomic resolution. Nature 446, 522u2013 525, (2007)._ _ _ _ _ Midgley, P. A. & Dunin-Borkowski, R. E. Electron tomography and holography in materials science. Nature Materials 8, 271u2013280, (2009)._ _ _ _ _ Schattschneider, P. et al. Detection of magnetic circular dichroism using a transmission electron microscope. Nature 441, 486u2013488, (2006)._ _ _ _ _ HxE9bert, C. & Schattschneider, P. A proposal for dichroic experiments in the electron microscope. Ultramicroscopy 96, 463u2013468, (2003)._ _ _ _ Schattschneider, P. et al. Detection of magnetic circular dichroism on the two-nanometer scale. Phys. Rev. B 78, 104413, (2008)._ _ _ _ Salafranca, J. et al. Surfactant Organic Molecules Restore Magnetism in Metal-Oxide Nanoparticle Surfaces. Nano Lett. 12, 2499u20132503, (2012)._ _ _ _ _ Thersleff, T. et al. Quantitative analysis of magnetic spin and orbital moments from an oxidized iron (1 1 0) surface using electron magnetic circular dichroism. Sci. Rep. 5, 13012, (2015)._ _ _ _ _ _ Schattschneider, P., Schaffer, B., Ennen, I. & Verbeeck, J. Mapping spin-polarized transitions with atomic resolution. Phys. Rev. B 85, 134422, (2012)._ _ _ _ Thersleff, T. et al. Towards sub-nanometer real-space observation of spin and orbital magnetism at the Fe/MgO interface. Sci. Rep. 7, 44802, (2017)._ _ _ _ _ _ Thersleff, T., Rusz, J., HjxF6rvarsson, B. & Leifer, K. Detection of magnetic circular dichroism with subnanometer convergent electron beams. Phys. Rev. B 94, 134430, (2016)._ _ _ Rusz, J. et al. Magnetic measurements with atomic-plane resolution. Nature Communications 7, 12672, (2016)._ _ _ _ _ _ Song, D., Rusz, J., Cai, J. & Zhu, J. Detection of electron magnetic circular dichroism signals under zone axial diffraction geometry. Ultramicroscopy 169, 44u201354, (2016)._ _ _ _ Spiegelberg, J., Song, D., Dunin-Borkowski, R. E., Zhu, J. & Rusz, J. Blind identification of magnetic signals in electron magnetic chiral dichroism using independent component analysis. Ultramicroscopy 195, 129u2013135, (2018)._ _ _ _ Rusz, J., Idrobo, J.-C. & Bhowmick, S. Achieving Atomic Resolution Magnetic Dichroism by Controlling the Phase Symmetry of an Electron Probe. Phys. Rev. Lett. 113, 145501, (2014)._ _ _ _ _ Idrobo, J. C. et al. Detecting magnetic ordering with atomic size electron probes. Advanced Structural and Chemical Imaging 2, 5, (2016)._ _ _ Wang, Z. et al. Atomic scale imaging of magnetic circular dichroism by achromatic electron microscopy. Nature Materials 17, 290, (2018)._ _ _ _ _ Negi, D. et al. Proposal for Measuring Magnetism with Patterned Apertures in a Transmission Electron Microscope. Phys. Rev. Lett. 122, 037201, (2019)._ _ _ _ _ _ Calmels, L. et al. Experimental application of sum rules for electron energy loss magnetic chiral dichroism. Phys. Rev. B 76, 060409, (2007)._ _ _ _ Rusz, J., Eriksson, O., NovxE1k, P. & Oppeneer, P. M. Sum rules for electron energy loss near edge spectra. Phys. Rev. B 76, 060408, (2007)._ _ _ _ Schattschneider, P. et al. Magnetic circular dichroism in EELS: Towards 10nm resolution. Ultramicroscopy 108, 433u2013438, (2008)._ _ _ _ Ali, H., Warnatz, T., Xie, L., HjxF6rvarsson, B. & Leifer, K. Quantitative EMCD by use of a double aperture for simultaneous acquisition of EELS. Ultramicroscopy 196, 192u2013196, (2018)._ _ _ _ Thersleff, T. et al. Single-pass STEM-EMCD on a zone axis using apatterned aperture: Data and methodology, (2019).de la PexF1a, F. et al. hyperspy/hyperspy: HyperSpy 1.3.1, (2018).Johnstone, D.N. et al. pyxem/pyxem: pyXem 0.7.1, (2019).SchxF6 nstrxF6m, L. Time Stamp Code (2018). This code is available upon request.Rusz, J., Oppeneer, P., Lidbaum, H., Rubino, S. & Leifer, K. Asymmetry of the two-beam geometry in EMCD experiments. J. Microsc. 237, 465u2013468, (2010)._ _ _ _ _ Song, D., Wang, Z. & Zhu, J. Effect of the asymmetry of dynamical electron diffraction on intensity of acquired EMCD signals. Ultramicroscopy 148, 42u201351, (2014)._ _ _ _ Loukya, B. et al. Effect of Bloch wave electron propagation and momentum-resolved signal detection on the quantitative and site-specific electron magnetic chiral dichroism of magnetic spinel oxide thin films. Phys. Rev. B 91, 134412, (2015)._ _ _ _ Schneider, S. et al. Magnetic properties of single nanomagnets: Electron energy-loss magnetic chiral dichroism on FePt nanoparticles. Ultramicroscopy 171, 186u2013194, (2016)._ _ _ _ Muto, S. et al. Quantitative characterization of nanoscale polycrystalline magnets with electron magnetic circular dichroism. Nat. Commun. 5, 3138, (2014)._ _ _ _ _ Hou, V. & SchxF6nstrxF6m, L. Timed Image Recorder (2009).Hou, V. Drift Tube Scan (2008).Mitchell, D.R. Drift Tube Scanning (2017).Bosman, M. & Keast, V. J. Optimizing EELS acquisition. Ultramicroscopy 108, 837u2013 846, (2008)._ _ _ _ Schattschneider, P. & Jonas, P. Iterative reduction of gain variations in parallel electron energy loss spectrometry. Ultramicroscopy 49, 179u2013188, (1993)._ _ _ Witjes, H., van den Brink, M., Melssen, W. & Buydens, L. Automatic correction of peak shifts in Raman spectra before PLS regression. Chemometrics and Intelligent Laboratory Systems 52, 105u2013116, (2000)._ _ _ Guizar-Sicairos, M., Thurman, S. T. & Fienup, J. R. Efficient subpixel image registration algorithms. Opt. Lett., OL 33, 156u2013 158, (2008)._ _ _ Witjes, H. et al. Automatic Correction for Phase Shifts, Frequency Shifts, and Lineshape Distortions across a Series of Single Resonance Lines in Large Spectral Data Sets. Journal of Magnetic Resonance 144, 35u201344, (2000)._ _ _ _ _ Witjes, H., Pepers, M., Melssen, W. J. & Buydens, L. M. C. Modelling phase shifts, peak shifts and peak width variations in spectral data sets: its value in multivariate data analysis. Analytica Chimica Acta 432, 113u2013124, (2001)._ _ _ \"- - T. T. acknowledges Wei Wan for suggesting to install the aperture device in the negative chamber of the TEM, Sebastian Schneider and Carolin Schmitz-Antoniak for the formative discussions on post-edge normalization routines, as well as Stef Smeets and Bin Wang for the fruitful scientific exchanges and inspiring discussions regarding open science publishing. L. S. acknowledges Jakob Paulin for valuable contributions to the design and build of the aperture holding system. T. T. and L. S. acknowledge funding from the Swedish Research Council (project nr. 2016-05113). C.-W. T. and T. T. acknowledge funding from the Swedish Strategic Research foundation (project nr. ITM17-0301). J. R. acknowledges funding from the Swedish Research Council (project nr. 2017-04026). Open access funding provided by Stockholm University.- - Authors and Affiliations - \"Stockholm University, Department of Materials and Environmental Chemistry, 10691, Stockholm, SwedenThomas Thersleff,_Linus SchxF6nstrxF6m_&_Cheuk-Wai TaiUppsala University, Department of Physics and Astronomy, Box 516, 75120, Uppsala, SwedenLinus SchxF6nstrxF6m_&_JxE1n RuszNagoya University, Institute of Materials and Systems for Sustainability, Nagoya, 464-8603, JapanShunsuke MutoForschungszentrum JxFClich GmbH, Peter GrxFCnberg Institut, D-52425, JxFClich, GermanyRoman Adam,_Daniel E. BxFCrgler_&_Claus M. Schneider\" - \"AuthorsThomas ThersleffYou can also search for this author in _Linus SchxF6nstrxF6mYou can also search for this author in _Cheuk-Wai TaiYou can also search for this author in _Roman AdamYou can also search for this author in _Daniel E. BxFCrglerYou can also search for this author in _Claus M. SchneiderYou can also search for this author in _Shunsuke MutoYou can also search for this author in _JxE1n RuszYou can also search for this author in _\" - Contributions - \"T.T. proposed the single-pass mirror symmetry geometry investigated in this manuscript, which was based off of the original idea for using a patterned aperture by J.R. T.T. and L.S. designed the experiment, the aperture plate and the aperture holding system together with valuable input from S.M. and C.-W. T.L.S. authored the hook-up script allowing for 2D-EELS and STEM-DP frame capture, and built the aperture holding system. T.T. acquired the data and performed the analysis, including authoring all of the relevant Matlab code (except where otherwise noted). R.A., D.B. and C.S. grew and prepared the iron sample used in this manuscript._ All authors contributed to the writing of the manuscript and the interpretation of the results.\" - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International)._ are provided with this paper.\" - \\' \\'- - \"Saparov, B. & Mitzi, D. B. Organicu2013inorganic perovskites: structural versatility for functional materials design. Chem. Rev. 116, 4558u20134596 (2016)._ _ _ _ Ogomi, Y. et al. CH3NH3SnxPb(1u2013 x)I3 perovskite solar cells covering up to 1060 nm. J. Phys. Chem. Lett. 5, 1004u2013 1011 (2014)._ _ _ _ Tsai, H. et al. High-efficiency two-dimensional Ruddlesdenu2013Popper perovskite solar cells. Nature 536, 312u2013 316 (2016)._ _ _ _ _ Lin, R. et al. Monolithic all-perovskite tandem solar cells with 24.8% efficiency exploiting comproportionation to suppress Sn(II) oxidation in precursor ink. Nat. Energy 4, 864u2013873 (2019)._ _ _ _ Zhao, Y. & Zhu, K. Organicu2013inorganic hybrid lead halide perovskites for optoelectronic and electronic applications. Chem. Soc. Rev. 45, 655u2013689 (2016)._ _ _ _ Bisquert, J. and Juarez-Perez, E. J. The Causes of Degradation of Perovskite Solar Cells. J. Phys. Chem. Lett. 10, 5889u2013 5891 (2019)._ _ _ _ Zhang, H., Nazeeruddin, M. K. & Choy, W. C. H. Perovskite photovoltaics: the significant role of ligands in film formation, passivation, and stability. Adv. Mater. 31, 1805702 (2019)._ _ _ Jiang, Q. et al. Surface passivation of perovskite film for efficient solar cells. Nat. Photonics 13, 460u2013466 (2019)._ _ _ _ Wang, R. et al. Constructive molecular configurations for surface-defect passivation of perovskite photovoltaics. Science 366, 1509u20131513 (2019)._ _ _ _ _ Zheng, X. et al. Managing grains and interfaces via ligand anchoring enables 22.3%u2013efficiency inverted perovskite solar cells. Nat. Energy 5, 131u2013 140 (2020)._ _ _ _ Tumen-Ulzii, G. et al. Detrimental effect of unreacted PbI2 on the long-term stability of perovskite solar cells. Adv. Mater. 32, 1905035 (2020).Yin, W.-J., Shi, T. & Yan, Y. Unusual defect physics in CH3NH3PbI3 perovskite solar cell absorber. Appl. Phys. Lett. 104, 063903 (2014)._ _ _ _ Krishna, A. et al. Defect passivation via the incorporation of tetrapropylammonium cation leading to stability enhancement in lead halide perovskite. Adv. Func. Mater. 30, 1909737 (2020)._ _ _ Ruiz-Preciado, M. A. et al. Supramolecular modulation of hybrid perovskite solar cells via bifunctional halogen bonding revealed by two-dimensional F solid-state NMR spectroscopy. J. Am. Chem. Soc. 142, 1645u20131654 (2020)._ _ _ _ Zhang, F. et al. Suppressing defects through the synergistic effect of a Lewis base and a Lewis acid for highly efficient and stable perovskite solar cells. Energy Environ. Sci. 11, 3480u20133490 (2018)._ _ _ Gratia, P. et al. The many faces of mixed ion perovskites: unraveling and understanding the crystallization process. ACS Energy Lett. 2, 2686u20132693 (2017)._ _ _ Saliba, M. et al. Cesium-containing triple cation perovskite solar cells: improved stability, reproducibility and high efficiency. Energy Environ. Sci. 9, 1989u20131997 (2016)._ _ _ _ _ Park, Y. H. et al. Inorganic rubidium cation as an enhancer for photovoltaic performance and moisture stability of HC(NH2)2PbI3 perovskite solar cells. Adv. Funct. Mater. 27, 1605988 (2017)._ _ _ Landini, D., Maia, A., Montanari, F. & Pirisi, F. M. Crown ethers as phase-transfer catalysts. A comparison of anionic activation in aqueousu2013 organic two-phase systems and in low polarity anhydrous solutions by perhydrodibenzo-18-crown-6, lipophilic quaternary salts, and cryptands. J. Chem. Soc., Perkin Trans. 2, 46u201351 (1980)._ _ Steed, J. W. First-and second-sphere coordination chemistry of alkali metal crown ether complexes. Coord. Chem. Rev. 215, 171u2013221 (2001)._ _ _ A Veldhuis, S. et al. Crown ethers enable room-temperature synthesis of CsPbBr3 quantum dots for light-emitting diodes. ACS Energy Lett. 3, 526u2013531 (2018)._ _ _ Yang, M. et al. Facile fabrication of large-grain CH3NH3PbI3-xBrx films for high-efficiency solar cells via CH3NH3Br-selective Ostwald ripening. Nat. Commun. 7, 12305 (2016)._ _ _ _ _ _ Ono, L. K., Qi, Y. & Liu, S. Progress toward stable lead halide perovskite solar cells. Joule 2, 1961u20131990 (2018)._ _ _ Zu, F. et al. Constructing the electronic structure of CH3NH3PbI3 and CH3NH3PbBr3 perovskite thin films from single-crystal band structure measurements. J. Phys. Chem. Lett. 10, 601u2013609 (2019)._ _ _ _ Greco, A. et al. Kinetics of ion-exchange reactions in hybrid organicu2212 inorganic perovskite thin films studied by in situ real-time Xu2010ray scattering. J. Phys. Chem. Lett. 9, 6750u20136754 (2018)._ _ _ _ Wiedemann, D. & Kohl, J. Invariom-model refinement and Hirshfeld surface analysis of well-ordered solvent-free dibenzo-21-crown-7. Acta Cryst. Sect. C 73, 654u2013659 (2017)._ _ _ Hanrahan, M. P., Men, L., Rosales, B. A., Vela, J. & Rossini, A. J. Sensitivity-enhanced Pb solid-state NMR spectroscopy for the rapid, non-destructive characterization of organolead halide perovskites. Chem. Mater. 30, 7005u20137015 (2018)._ _ _ Kubicki, D. J. et al. Phase segregation in Cs-, Rb- and K-doped mixed-cation (MA)x(FA)1u2013xPbI3 hybrid perovskites from solid-state NMR. J. Am. Chem. Soc. 139, 14173u201314180 (2017)._ _ _ _ _ Zhang, J. et al. Intermediate phase enhances inorganic perovskite and metal oxide interface for efficient photovoltaics. Joule 4, 222u2013 234 (2020)._ _ _ Bi, D. et al. Multifunctional molecular modulators for perovskite solar cells with over 20% efficiency and high operational stability. Nat. Commun. 9, 4482 (2018)._ _ _ _ _ _ Miliu0107, J. V. et al. Supramolecular engineering for formamidinium-based layered 2D perovskite solar cells: structural complexity and dynamics revealed by solid-state NMR spectroscopy. Adv. Energy Mater. 9, 1900284 (2019)._ _ _ Franssen, W. M. J., Bruijnaers, B. J., Portengen, V. H. L. & Kentgens, A. P. M. Dimethylammonium incorporation in lead acetate based MAPbI3 perovskite solar cells. ChemPhysChem 19, 3107u2013 3115 (2018)._ _ _ _ Soong, L. L., Leroi, G. E. & Popov, A. I. Cesium-133 NMR studies of crown ether-cesium(1+) complexes in high dielectric amide solvents. Inorg. Chem. 29, 1366u20131370 (1990)._ _ _ Franssen, W. M. J., van Es, S. G. D., Derviu015F ou011Flu, R., de Wijs, G. A. & Kentgens, A. P. M. Symmetry, dynamics, and defects in methylammonium lead halide perovskites. J. Phys. Chem. Lett. 8, 61u2013 66 (2017)._ _ _ _ Braga, D. et al. Supramolecular complexation of alkali cations through mechanochemical reactions between crystalline solids. Chem. Eur. J. 10, 3261u20133269 (2004)._ _ _ _ _ Giordano, F. et al. Enhanced electronic properties in mesoporous TiO2 via lithium doping for high-efficiency perovskite solar cells. Nat. Commun. 7, 10379 (2016)._ _ _ _ _ _ Zhu, H. et al. Tailored amphiphilic molecular mitigators for stable perovskite solar cells with 23.5% efficiency. Adv. Mater. 32, 1907757 (2020)._ _ _ Jeong, M. et al. Stable perovskite solar cells with efficiency exceeding 24.8% and 0.3-V voltage loss. Science 369, 1615u2013 1620 (2020)._ _ _ _ _ Wolff, C. M., Caprioglio, P., Stolterfoht, M. & Neher, D. Nonradiative recombination in perovskite solar cells: the role of interfaces. Adv. Mater. 31, 1902762 (2019)._ _ _ de Mello, J. C., Wittmann, H. F. & Friend, R. H. An improved experimental determination of external photoluminescence quantum efficiency. Adv. Mater. 9, 230u2013232 (1997)._ _ Tress, W. et al. Predicting the open-circuit voltage of CH3NH3PbI3 perovskite solar cells using electroluminescence and photovoltaic quantum efficiency spectra: the role of radiative and non-radiative recombination. Adv. Energy Mater. 5, 1400812 (2015)._ _ _ Tress, W. et al. Interpretation and evolution of open-circuit voltage, recombination, ideality factor and subgap defect states during reversible light-soaking and irreversible degradation of perovskite solar cells. Energy Environ. Sci. 11, 151u2013165 (2018)._ _ _ Perdew, J. P., Burke, K. & Ernzerhof, M. Generalized gradient approximation made simple. Phys. Rev. Lett. 77, 3865u20133868 (1996)._ _ _ _ _ Grimme, S. Semiempirical GGA-type density functional constructed with a long-range dispersion correction. J. Comput. Chem. 27, 1787u20131799 (2006)._ _ _ _ Giannozzi, P. et al. QUANTUM ESPRESSO: a modular and open-source software project for quantum simulations of materials. J. Phys. 21, 395502 (2009)._ Perdew, J. P. et al. Restoring the density-gradient expansion for exchange in solids and surfaces. Phys. Rev. Lett. 100, 136406 (2008)._ _ _ _ _ CPMD, Copyright IBM Corp 1990u20132015, Copyright MPI fxFCr FestkxF6rperforschung Stuttgart 1997u20132001.Adamo, C. & Barone, V. Toward reliable density functional methods without adjustable parameters: the PBE0 model. J. Chem. Phys. 110, 6158u20136170 (1999)._ _ _ _ \"- - \"The project leading to this application has received funding from the European Unions Horizon 2020 research and innovation programme under Grant agreements No. 785219, 881603, and 764047. J.V.M. acknowledges the SNSF PRIMA Grant No. 193174. U.R. acknowledges SNSF Grant No. 200020-185092, NCCR-MUST, and the SINERGIA interdisciplinary research program EPISODE for funding_ and_Swiss National Supercomputing Centre (CSCS) for computing time. L.E. is grateful for financial support from Swiss National Science Foundation Grant No. 200020-178860. L.M., A.H. (Alex), and F.S. acknowledge funding from the DFG. O.O. acknowledges funding from the Natural Sciences and Engineering Research Council of Canada. M.M. acknowledges the affiliation with the Scientific Computing Laboratory, Center for the Study of Complex Systems, Institute of Physics Belgrade, University of Belgrade, Serbia. A.H. (Anders) acknowledges the financial support from the Swiss National Science Foundation REquip program under the Grant number 183305. For the PL measurements the authors acknowledge the help of Wolfgang Tress and funding by the Swiss National Science Foundation (Ambizione Energy Grant no. 173617). A.K. acknowledges funding from the European Unions Horizon 2020 Research and Innovation program under the Marie Sku0142odowska-Curie Grant Agreement No. 843453. Z.D. and J.G. are grateful to Prof. Kay Severin and Prof. Anders Meibom for hosting and supporting his research at EPFL, respectively._The authors acknowledge the help of Mr._Shi Wang, Dr. M. D. Mensi, and Mr._Anand Agarwalla for the assistance in TOF-SIMS, UPS, and stability tests, respectively. We are also grateful to Prof. Kevin Sivula and Dr. Jun Ho Yum for KPFM measurement, Dr. Pierre Mettraux for XPS measurement, Dr. David Bi for XRD measurements, and Prof. Shien-Ping Feng for the assistance in the GIWAXS preliminary studies, respectively. We acknowledge DESY (Hamburg, Germany) for the provision of synchrotron facilities and we would like to thank Florian Bertram for assistance in using beamline P08 (PETRA III).\"- - Author notes - \"Anders HagfeldtPresent address: Department of Chemistry u2013 xC5ngstrxF6 m Laboratory, Uppsala University, Uppsala, Sweden\" - Authors and Affiliations - \"Laboratory of Photonics and Interfaces, Institute of Chemical Sciences and Engineering, xC9cole Polytechnique FxE9dxE9rale de Lausanne, Lausanne, SwitzerlandHong Zhang,_Felix Thomas Eickemeyer,_Zhiwen Zhou,_Olivier Ouellette,_Dan Ren,_ Tzu-Sen Su,_Shaik M. Zakeeruddin,_Jovana V. Miliu0107_&_Michael GrxE4tzelLaboratory of Computational Chemistry and Biochemistry, Institute of Chemical Sciences and Engineering, xC9cole Polytechnique FxE9dxE9rale de Lausanne, Lausanne, SwitzerlandMarko Mladenoviu0107,_Farzaneh Jahanbakhshi,_Paramvir Ahlawat_ &_Ursula RothlisbergerInstitut fxFCr Angewandte Physik, UniversitxE4t TxFC bingen, TxFCbingen, GermanyLena Merten,_Alexander Hinderhofer_&_Frank SchreiberLaboratory of Magnetic Resonance, Institute of Chemical Sciences and Engineering, xC9 cole Polytechnique FxE9dxE9rale de Lausanne, Lausanne, SwitzerlandMichael A. Hope,_Aditya Mishra_&_Lyndon EmsleyLaboratory of Photomolecular Science, Institute of Chemical Sciences and Engineering, xC9cole Polytechnique FxE9 dxE9rale de Lausanne, Lausanne, SwitzerlandAnurag Krishna,_Zaiwei Wang_&_ Anders HagfeldtLaboratory of Supramolecular Chemistry, Institute of Chemical Sciences and Engineering, xC9cole Polytechnique FxE9dxE9rale de Lausanne, Lausanne, SwitzerlandZhaowen DongLaboratory for Biological Geochemistry, xC9cole Polytechnique FxE9dxE9rale de Lausanne, Lausanne, SwitzerlandJinming GuoAdolphe Merkle Institute of the University of Fribourg in Switzerland, Fribourg, SwitzerlandJovana V. Miliu0107\" - \"AuthorsHong ZhangYou can also search for this author in _Felix Thomas EickemeyerYou can also search for this author in _Zhiwen ZhouYou can also search for this author in _Marko Mladenoviu0107You can also search for this author in _Farzaneh JahanbakhshiYou can also search for this author in _Lena MertenYou can also search for this author in _Alexander HinderhoferYou can also search for this author in _Michael A. HopeYou can also search for this author in _Olivier OuelletteYou can also search for this author in _Aditya MishraYou can also search for this author in _Paramvir AhlawatYou can also search for this author in _Dan RenYou can also search for this author in _Tzu-Sen SuYou can also search for this author in _Anurag KrishnaYou can also search for this author in _Zaiwei WangYou can also search for this author in _Zhaowen DongYou can also search for this author in _Jinming GuoYou can also search for this author in _Shaik M. ZakeeruddinYou can also search for this author in _Frank SchreiberYou can also search for this author in _Anders HagfeldtYou can also search for this author in _Lyndon EmsleyYou can also search for this author in _Ursula RothlisbergerYou can also search for this author in _Jovana V. Miliu0107 You can also search for this author in _Michael GrxE4 tzelYou can also search for this author in _\" - Contributions - M.G. supervised the study. H.Z. conceived the idea, designed the experiments, conducted the device fabrication and characterization. J.V.M. has conceptually contributed to the project while being involved in the mechanistic investigation, analysis, as well as project coordination. F.E. and O.O. performed the PL/TRPL characterizations and the analysis of the optoelectronic loss mechanisms. Z.Z. participated in device optimization and assisted in some sample preparations as well as GIWAXS preliminary studies. A.M., M.H., and L.E. performed the solid-state NMR studies and analysis. M.M., F.J., P.A., and U.R. were responsible for the molecular dynamics simulations and DFT calculations as well as the analysis. T.S.S. optimized the device fabrication. A.H. (Alex), L.M., and F.S. performed the GIWAXS measurements and the analysis. D.R. and Z.W. performed the SEM and EDS. Z.D. synthesized the cesium crown ether complex for ssNMR studies and performed H NMR characterization. A.K. assisted in device stability test and data analysis. J.G. performed the STEM/EDX measurement and analysis. S.M.Z. and A.H. (Anders) was involved in discussion and project coordination. H.Z. wrote the first manuscript draft, and all the authors contributed to the revision. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks the anonymous reviewer(s) for their contribution to the peer review of this work. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International, under the DOI 10.5281/zenodo.5547224. The data contains files in MATLAB and Bruker TopSpin 4 format. - \\' \\'- - \"Sweetlove, L. J. & Fernie, A. R. The role of dynamic enzyme assemblies and substrate channelling in metabolic regulation. Nat. Commun. 9, 2136 (2018)._ _ Holdgate, G. A., Meek, T. D. & Grimley, R. L. Mechanistic enzymology in drug discovery: a fresh perspective. Nat. Rev. Drug Disco. 17, 115u2013 132 (2018)._ _ _ Gebauer, D. & Wolf, S. E. Designing Solid Materials from Their Solute State: A Shift in Paradigms toward a Holistic Approach in Functional Materials Chemistry. J. Am. Chem. Soc. 141, 4490u2013 4504 (2019)._ _ _ Wang, Y. et al. The predominant role of collagen in the nucleation, growth, structure and orientation of bone apatite. Nat. Mater. 11, 724u2013733 (2012)._ _ _ Boehr, D. D., Nussinov, R. & Wright, P. E. The role of dynamic conformational ensembles in biomolecular recognition. Nat. Chem. Biol. 5, 789u2013796 (2009)._ _ _ Lee, M. K., Gal, M., Frydman, L. & Varani, G. Real-time multidimensional NMR follows RNA folding with second resolution. Proc. Natl Acad. Sci. USA 107, 9192u20139197 (2010)._ _ _ Gal, M., Mishkovsky, M. & Frydman, L. Real-time monitoring of chemical transformations by ultrafast 2D NMR spectroscopy. J. Am. Chem. Soc. 128, 951u2013 956 (2006)._ _ _ Dass, R., Kozminski, W. & Kazimierczuk, K. Analysis of complex reacting mixtures by time-resolved 2D NMR. Anal. Chem. 87, 1337u20131343 (2015)._ _ _ Miclet, E. et al. Toward Quantitative Measurements of Enzyme Kinetics by Dissolution Dynamic Nuclear Polarization. J. Phys. Chem. Lett. 5, 3290u20133295 (2014)._ _ _ Felli, I. C. & Brutscher, B. Recent advances in solution NMR: fast methods and heteronuclear direct detection. Chemphyschem 10, 1356u20131368 (2009)._ _ _ Rule, G. S. & Hitchens, T. K. Fundamentals of Protein NMR Spectroscopy. (Springer, 2006).Ardenkjaer-Larsen, J. H. et al. Increase in signal-to-noise ratio of >10,000 times in liquid-state NMR. Proc. Natl Acad. Sci. USA 100, 10158u201310163 (2003)._ _ _ Jannin, S., Dumez, J. N., Giraudeau, P. & Kurzbach, D. Application and methodology of dissolution dynamic nuclear polarization in physical, chemical and biological contexts. J. Magn. Reson 305, 41u201350 (2019)._ _ _ Kouu0159il, K., Kouu0159ilovxE1, H., Levitt, M. H. & Meier, B. Dissolution-Dynamic Nuclear Polarization with Rapid Transfer of a Polarized Solid. Nat. Commun. 10, 1733 (2019)._ _ Kurzbach, D. et al. Investigation of Intrinsically Disordered Proteins through Exchange with Hyperpolarized Water. Angew. Chem. Int Ed. Engl. 56, 389u2013 392 (2017)._ _ _ Kim, J., Liu, M. & Hilty, C. Modeling of Polarization Transfer Kinetics in Protein Hydration Using Hyperpolarized Water. J. Phys. Chem. B 121, 6492u20136498 (2017)._ _ _ Szekely, O., Olsen, G. L., Felli, I. C. & Frydman, L. High-resolution 2D NMR of disordered proteins enhanced by hyperpolarized water. Anal. Chem. 90, 6169u20136177 (2018).Sadet, A. et al. Hyperpolarized water enhances two-dimensional proton NMR correlations: a new approach for molecular interactions. J. Am. Chem. Soc. 141, 12448u201312452 (2019)._ _ _ Olsen, G., Markhasin, E., Szekely, O., Bretschneider, C. & Frydman, L. Optimizing water hyperpolarization and dissolution for sensitivity-enhanced 2D biomolecular NMR. J. Magn. Reson. 264, 49u201358 (2016)._ _ _ Ragavan, M., Iconaru, L. I., Park, C. G., Kriwacki, R. W. & Hilty, C. Real-Time Analysis of Folding upon Binding of a Disordered Protein by Using Dissolution DNP NMR Spectroscopy. Angew. Chem. Int. Ed. Engl. 56, 7070u20137073 (2017)._ _ _ Novakovic, M. et al. A 300-fold enhancement of imino nucleic acid resonances by hyperpolarized water provides a new window for probing RNA refolding by 1D and 2D NMR. Proc. Natl Acad. Sci. USA 117, 2449u20132455 (2020)._ _ _ Luchinat, E., Barbieri, L., Cremonini, M. & Banci, L. Protein in-cell NMR spectroscopy at 1.2 GHz. J. Biomol. Nmr 75, 97u2013107 (2021)._ _ _ Harris, T., Szekely, O. & Frydman, L. On the potential of hyperpolarized water in biomolecular NMR studies. J. Phys. Chem. B 118, 3281u20133290 (2014)._ _ _ Kaderavek, P., Ferrage, F., Bodenhausen, G. & Kurzbach, D. High-Resolution NMR of Folded Proteins in Hyperpolarized Physiological Solvents. Chem. Eur. J. 24, 13418u201313423 (2018)._ _ _ Kim, J., Mandal, R. & Hilty, C. Observation of Fast Two-Dimensional NMR Spectra during Protein Folding Using Polarization Transfer from Hyperpolarized Water. J. Phys. Chem. Lett. 10, 5463u20135467 (2019)._ _ _ Szekely, O., Olsen, G. L., Novakovic, M., Rosenzweig, R. & Frydman, L. Assessing Site-Specific Enhancements Imparted by Hyperpolarized Water in Folded and Unfolded Proteins by 2D HMQC NMR. J. Am. Chem. Soc. 142, 9267u20139284 (2020)._ _ _ Lipso, K. W., Bowen, S., Rybalko, O. & Ardenkjaer-Larsen, J. H. Large dose hyperpolarized water with dissolution-DNP at high magnetic field. J. Magn. Reson. 274, 65u201372 (2017)._ _ Kouno, H. et al. Triplet dynamic nuclear polarization of crystalline ice using water-soluble polarizing agents. Chem. Commun. (Camb.) 56, 3717u20133720 (2020)._ _ _ Olsen, G. L. et al. Sensitivity-enhanced three-dimensional and carbon-detected two-dimensional NMR of proteins using hyperpolarized water. J. Biomol. Nmr 74, 161u2013171 (2020)._ _ _ Schanda, P., Kupce, E. & Brutscher, B. SOFAST-HMQC experiments for recording two-dimensional heteronuclear correlation spectra of proteins within a few seconds. J. Biomol. Nmr 33, 199u2013211 (2005)._ _ _ Honegger, P. & Steinhauser, O. The protein-water nuclear Overhauser effect (NOE) as an indirect microscope for molecular surface mapping of interaction patterns. Phys. Chem. Chem. Phys. 22, 212u2013222 (2019)._ _ Tugarinov, V., Kanelis, V. & Kay, L. E. Isotope labeling strategies for the study of high-molecular-weight proteins by solution NMR spectroscopy. Nat. Protoc. 1, 749u2013754 (2006)._ _ _ Pellecchia, M., Sem, D. S. & Wuthrich, K. NMR in drug discovery. Nat. Rev. Drug Disco. 1, 211u2013219 (2002)._ _ _ Milani, J. et al. A magnetic tunnel to shelter hyperpolarized fluids. Rev. Sci. Instrum. 86, 024101-1u2013024101-8 (2015).Kress, T., Walrant, A., Bodenhausen, G. & Kurzbach, D. Long-Lived States in Hyperpolarized Deuterated Methyl Groups Reveal Weak Binding of Small Molecules to Proteins. J. Phys. Chem. Lett. 10, 1523u20131529 (2019)._ _ _ Delaglio, F. et al. Nmrpipe - a Multidimensional Spectral Processing System Based on Unix Pipes. J. Biomol. Nmr 6, 277u2013293 (1995)._ _ _ \"- - \"The project leading to this application received funding from the European Research Council (ERC) under the European Unions Horizon 2020 research and innovation programme (grant agreement 801936). This project was supported by the Austrian FWF (stand-alone grant no. P-33338). The authors thank Dr. Pavel Kadeu0159xE1 vek and Dr. Gregory L. Olsen for helpful discussions. Furthermore, the authors thank Prof. Geoffrey Bodenhausen for his support from the beginning of this research programme on.\"- - Authors and Affiliations - \"Faculty of Chemistry, Institute of Biological Chemistry, University Vienna, WxE4 hringer Str. 38, 1090, Vienna, AustriaMattia Negroni_&_Dennis Kurzbach\" - \"AuthorsMattia NegroniYou can also search for this author in _Dennis KurzbachYou can also search for this author in _\" - Contributions - M.N. and D.K. performed experiments. D.K. wrote the paper with the help of M.N. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Communications Chemistry thanks the anonymous reviewers for their contribution to the peer review of this work. Peer reviewer reports are available. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 International._Chao, Weng-si; Lee, An-Sheng; Tiedemann, Ralf; Lembke-Jene, Lester; Lamy, Frank (2022): XRF down-core scanning and bulk chemistry measurements of sediments from the high latitude sectors of Pacific Ocean. PANGAEA, . The analyzing results is included in the supplement.\" - \\' \\'- - \"Alpaydin, E. Introduction to Machine Learning (Adaptive Computation and Machine Learning Series) 3rd edn. (The MIT Press, 2014)._ _ Jordan, M. I. & Mitchell, T. M. Machine learning: Trends, perspectives, and prospects. Science 349, 255u2013260. (2015)._ _ _ _ _ _ Schulz, H. & Zabel, M. Marine Geochemistry 2nd, XIX edn, 574 (Springer, 2006)._ _ Archer, D. & Maier-Reimer, E. Effect of deep-sea sedimentary calcite preservation on atmospheric CO2 concentration. Nature 367, 260u2013263 (1994)._ _ _ Croudace, I. W., LxF6wemark, L., Tjallingii, R. & Zolitschka, B. Current perspectives on the capabilities of high resolution XRF core scanners. Quatern. Int. 514, 5u201315. (2019)._ _ Weltje, G. J. et al. In Micro-XRF Studies of Sediment Cores: Applications of a Non-destructive Tool for the Environmental Sciences (eds Croudace, I. W. & GuyRothwell, R.) 507u2013534 (Springer, 2015)._ _ Croudace, I. W., Rindby, A. & Rothwell, R. G. ITRAX: Description and evaluation of a new multi-function X-ray core scanner. Geol. Soc. Lond. Spec. Publ. 267, 51u201363 (2006)._ _ _ Tjallingii, R., RxF6hl, U., KxF6lling, M. & Bickert, T. Influence of the water content on X-ray fluorescence core-scanning measurements in soft marine sediments. Geochem. Geophys. Geosyst. 8, 25 (2007)._ _ BxF6ning, P., Bard, E. & Rose, J. Toward direct, micron-scale XRF elemental maps and quantitative profiles of wet marine sediments. Geochem. Geophys. Geosyst. 8, 5 (2007)._ _ Jansen, J., Van der Gaast, S., Koster, B. & Vaars, A. CORTEX, a shipboard XRF-scanner for element analyses in split sediment cores. Mar. Geol. 151, 143u2013153 (1998)._ _ _ Weltje, G. J. & Tjallingii, R. Calibration of XRF core scanners for quantitative geochemical logging of sediment cores: Theory and application. Earth Planet. Sci. Lett. 274, 423u2013438. (2008)._ _ _ Bolton, M. S. M. et al. Machine learning classifiers for attributing tephra to source volcanoes: An evaluation of methods for Alaska tephras. J. Quat. Sci. 35, 81u201392. (2020)._ _ xD6n, Z. B. & xD6zeren, M. S. Temperature and precipitation variability in eastern Anatolia: Results from independent component analysis of Lake Van sediment data spanning the last 250 kyr BP. Quatern. Int. 514, 119u2013129. (2019)._ _ Kaboth-Bahr, S. et al. Deciphering u223C 45,000 years of Arctic Ocean lithostratigraphic variability through multivariate statistical analysis. Quatern. Int. (2018)._ _ Abu-Mostafa, Y. S., Magdon-Ismail, M. & Lin, H.-T. Learning From Data: A Short Course (AMLBook, 2012)._ FxE9votte, C. & Idier, J. Algorithms for nonnegative matrix factorization with the u03B2-divergence. Neural Comput. 23, 2421u20132456 (2011)._ _ _ _ Cichocki, A. & Phan, A.-H. Fast local algorithms for large scale nonnegative matrix and tensor factorizations. IEICE Trans. Fundam. Electron. Commun. Comput. Sci. 92, 708u2013721 (2009)._ _ Clifford, G. Biomedical Signal and Image Processing (MIT Electrical Engineering and Computer Science, 2008)._ Chang, C.-C. & Lin, C.-J. LIBSVM: A library for support vector machines. ACM Trans. Intell. Syst. Technol. 2, 1u201327. (2011)._ _ Breiman, L. Machine Learning vol 45 5u201332 (Springer, 2001)._ Geurts, P., Ernst, D. & Wehenkel, L. Extremely randomized trees. Mach. Learn. 63, 3u201342 (2006)._ _ _ Pedregosa, F. et al. Scikit-learn: Machine learning in Python. J. Mach. Learn. Res. 12, 2825u20132830 (2011)._ _ _ MxFCller, A. C. & Guido, S. Introduction to Machine Learning with Python: A Guide for Data Scientists 1st edn. (OReilly Media, 2016)._ Bellon-Maurel, V., Fernandez-Ahumada, E., Palagos, B., Roger, J.-M. & McBratney, A. Critical review of chemometric indicators commonly used for assessing the quality of the prediction of soil attributes by NIR spectroscopy. Trends Anal. Chem. 29, 1073u20131081. (2010)._ _ _ Caley, T. et al. New Arabian Sea records help decipher orbital timing of Indo-Asian monsoon. Earth Planet. Sci. Lett. 308, 433u2013444 (2011)._ _ _ Ziegler, M., Jilbert, T., de Lange, G. J., Lourens, L. J. & Reichart, G. J. Bromine counts from XRF scanning as an estimate of the marine organic carbon content of sediment cores. Geochem. Geophys. Geosyst. 9, 25 (2008)._ _ McHugh, C. M. et al. The last reconnection of the Marmara Sea (Turkey) to the World Ocean: A paleoceanographic and paleoclimatic perspective. Mar. Geol. 255, 64u201382 (2008)._ _ Max, L. et al. Sea surface temperature variability and sea-ice extent in the subarctic northwest Pacific during the past 15,000 years. Paleoceanography (2012)._ _ Benz, V., Oliver, E., Rainer, G., Frank, L. & Ralf, T. Last Glacial Maximum sea surface temperature and sea-ice extent in the Pacific sector of the Southern Ocean. Quatern. Sci. Rev. 146, 216u2013237 (2016)._ _ Lamy, F. et al. Increased dust deposition in the Pacific Southern Ocean during glacial periods. Science 343, 403u2013407. (2014)._ _ _ _ Ullermann, J. et al. Pacific-Atlantic circumpolar deep water coupling during the last 500 ka. Paleoceanography 31, 639u2013650 (2016)._ _ Wang, W. et al. Dating North Pacific Abyssal sediments by geomagnetic paleointensity: Implications of magnetization carriers, plio-pleistocene climate change, and benthic redox conditions. Front. Earth Sci. 25, 577 (2021)._ NxFCrnberg, D. RV SONNE Fahrtbericht / Cruise Report SO264-SONNE-EMPEROR: The Plio/Pleistocene to Holocene development of the pelagic North Pacific from surface to depthu2014assessing its role for the global carbon budget and Earth u0301s climate, Suva (Fiji)-Yokohama (Japan), 30.6.-24.8.2018. 284 (GEOMAR Helmholtz-Zentrum fxFCr Ozeanforschung, Kiel, Germany, 2018).Chao, W.-S. et al. Glacial-interglacial variations in productivity and carbonate deposition in the Northwest Pacific during the last 500,000 years. Front. Earth Sci. (In prep.).McKinney, W. in The 9th Python in Science Conference. 56u201361.Millman, K. J. & Aivazis, M. Python for scientists and engineers. Comput. Sci. Eng. 13, 9u201312. (2011)._ _ Virtanen, P. et al. SciPy 1.0u2014fundamental algorithms for scientific computing in Python. Nat. Methods 17, 261u2013272. (2020)._ _ _ _ _ Harris, C. R. et al. Array programming with NumPy. Nature 585, 357u2013362. (2020)._ _ _ _ _ Thomas, K. et al. In Positioning and Power in Academic Publishing: Players, Agents and Agendas (eds F. Loizides & B. Schmidt) 87u201390 (2016).Waskom, M. L. seaborn: Statistical data visualization. J. Open Source Softw. 6, 3021. (2021)._ _ Hunter, J. D. Matplotlib: A 2D graphics environment. Comput. Sci. Eng. 9, 99u2013104. (2007)._ _ Wilkinson, M. D. et al. The FAIR guiding principles for scientific data management and stewardship. Sci. Data 3, 160018. (2016)._ _ _ _ \"- - \"We thank the crew and the science parties of different cruises for their contributions to core and sample acquisition on the respective expeditions. We are very grateful to Dr. Frank Lamy for providing data from the Southern Ocean. We also sincerely thank ValxE9a Schumacher, Susanne Wiebe, and Rita FrxF6hlking and student assistants at the AWI Marine Geology Laboratory in Bremerhaven for technical assistance with XRF-scanning, CaCO3 and TOC measurements. We also acknowledge Ben Marzeion and Timo Rothenpieler (Klimageographie, University of Bremen) for generously providing access and support to their high-performance computing power. We acknowledge financial support from the Wadden Sea Archive (WASA) project funded by the u2018NiedersxE4chsisches Vorab of the VolkswagenStiftung within the funding initiative u2018KxFCsten und Meeresforschung in Niedersachsen of the Ministry of Science and Culture of Lower Saxony, Germany (project VW ZN3197), the Ministry of Science and Technology of Taiwan (project number: MOST 110-2116-M-002-023), the AWI institutional funding through u201CPACES-IIu201D (2018-20)_and u201CChanging Earth - Sustaining our Futureu201D (ST 2.1, 2020-2022), the German Federal Ministry for Education and Research (BMBF) Grants no. 03G0264B u201CSO264-EMPERORu201D and 03F0785A u201CNOPAWACu201D, and the 2022 xD8 rsteds Green Energy Scholarship Program.\"- - Open Access funding enabled and organized by Projekt DEAL.- - Author notes - \\'These authors contributed equally: An-Sheng Lee and Weng-Si Chao.\\' - Authors and Affiliations - \"Institute of Geography, University of Bremen, Bremen, GermanyAn-Sheng Lee_&_ Bernd ZolitschkaDepartment of Geosciences and Research Center for Future Earth, National Taiwan University, Taipei, TaiwanAn-Sheng Lee_&_Sofia Ya Hsuan LiouAlfred-Wegener-Institut Helmholtz-Zentrum fxFCr Polar- und Meeresforschung, Bremerhaven, GermanyWeng-Si Chao,_Ralf Tiedemann_&_Lester Lembke-Jene\" - \"AuthorsAn-Sheng LeeYou can also search for this author in _Weng-Si ChaoYou can also search for this author in _Sofia Ya Hsuan LiouYou can also search for this author in _Ralf TiedemannYou can also search for this author in _Bernd ZolitschkaYou can also search for this author in _Lester Lembke-JeneYou can also search for this author in _\" - Contributions - \"A.S.L. and W.S.C.: conceptualization, methodology, investigation, data curation, writingu2014original draft. R.T., L.L.J., B.Z., and S.Y.H.L.: supervision, writingu2014review and editing, resources, funding acquisition.\" - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 License.Nanoscale photochromism by near-field optics. (a) Transparent and coloured states of diarylethene and their absorption spectra. (b) Local near-field light excitation at the surface of the photochromic material to generate photoisomerization at the nanometre scale. Complex pathways are developed. (c) Once th pattern is formed, the local optical excitation travels through the generated path. (d) The output position of the photons was observed using scanning near-field optical microscopy. - \"Furthermore, it should be emphasized that the formation of an autonomous pattern has been realized in forming a nanometre-scale ultrafine structure instead of pixel-wise addressing, as in conventional optical memory. To fully benefit from subwavelength-scale photonics, we utilized optical near-fields and locally induced photoisomerization of photochromic materials. Indeed, in a previous study, Nakagomi et al., demonstrated that a local optical near-field excitation on the surface of a photochromic crystal yields local photoisomerization on the scale of tens of nanometres. Initially, the entire material was in a coloured state (1c). Through an optical near-field excitation with a visible wavelength that was fixed at a spatial position on the crystal surface, the material was locally isomerized into a transparent 1o state. The locally induced transparency guides the incoming optical near-field into a neighbouring area in the subwavelength regime. Consequently, the local photoisomerization of the transparent state causes a succeeding chain reaction of local transparency, including bifurcations, leading to the formation of a complex pattern of transmitted photons on the opposite surface of the crystal (Fig._b).\" - \"Once the nanoscale transparency paths are formed, an incoming near-field excitation is transferred to the opposite side and scattered by the probe tip located at a particular position (Fig._c). Figure_d shows the experimentally observed photon count statistics in a 2_u03BCmu2009xD7u20092_u03BCm regime taken by a scanning near-field optical microscope (SNOM) with a spatial resolution of approximately 50_nm. We observed non-uniform distributions stemming from the complex structure formed in the photochromic materials.\" - Although the photon count pattern appears complicated, internal photoisomerization is triggered by the local excitation from a single fixed position. Therefore, photon statistics are not random but comprise certain spatial correlations. The origin of the complexity stems from the interplay between local photoisomerization by an optical near-field and its subsequent anisotropic deformation of the molecular size, leading to anisotropic mechanical strain. Such a structure is, metaphorically speaking, as if chaos provided complex profiles originating from simple common deterministic nonlinear dynamics. By exploiting these characteristics, Uchiyama et al. have successfully demonstrated the generation of Schubert polynomials using the photon count statistics combined with a simple photon detection protocol. Here, the experimentally observed optical near-field data was common for the previous studies and the present study. - However, Uchiyama et al. only described the production of Schubert polynomials; its mathematical properties, formulated as Schubert calculus, were not fully utilized. Furthermore, quantitative comparisons with other random numbers, such as uniformly distributed pseudorandom numbers, were not performed. Therefore, in the present study, we demonstrated order recognition using Schubert polynomials generated by the experimentally observed optical near-field pattern. - \"Here, the issue of order recognition concerns the following situation: there are N slot machines whose reward probabilities are given by Pi (iu2009=u2009 1, u2026, N). The task of order recognition is to realize the ranking of all Pi in an ascending or descending order. In this study, we proposed an order recognition strategy based on Schubert calculus. The primary advantage of the proposed strategy is that it never calculates the estimated reward probability (hat{P}_{i}) while evaluating the order; instead, the proposed method highlights the relative relation between two choices, which is determined by an inverse relationship in a Schubert polynomial. Furthermore, we quantitatively analyzed Schubert polynomials and their order recognition capabilities depending on the property of the source of the randomness. We showed that the number of singularities contained in the generated Schubert polynomials correlated with the order recognition capability.\" - \"In a general context, the contributions of the present study are based on Schubert calculus and complex optical near-field photon to physics-based computing from the following perspectives: the first contribution is the capability of producing versatile Schubert polynomials that inherit the correlation induced in the photochromic material by local excitation. The number of possible Schubert polynomials reaches the order of Avogadros number (10) or becomes extremely diverse when N reaches 24. Nevertheless, Schubert polynomials generated via the proposed physics-based protocol carry the spatial correlations formed in the subwavelength-scale material system.\" - Second is the newly designed ordering algorithm based on Schubert calculus, a solid theoretical foundation of an ordered structure. As mentioned earlier, the proposed algorithm consists of repeating the revision of orders between two choices, which mathematically corresponds to the so-called divided difference operation in Schubert calculus. The transformation of the divided difference operator into an order recognition strategy is the original contribution of the present study. - Third, through the analysis of the comparison among the spatially uniform, spatially highly skewed, and optical near-field patterns, we found that ordering performs well when the singularity in the given problem and that in the source patterns meet each other. In other words, excellent performance results are observed when the problem under study and the solver are in an adjoint relationship. Although much deeper insights are needed from future studies, these contributions of the present study shed light on the utilization of naturally constructed complex information inherent at the subwavelength scale for intelligent functions. Based on this fundamental nature, the proposed strategy may be extended to other applications in the future, not just for order recognition of slot machines or bandit problems examined in the present study. - \\'The remainder of the paper is organized as follows: first, we review the optical near-field statistics observed using a double-probe optical near-field microscope. We then review the generation of Schubert polynomials based on near-field observation coupled with a simple photon detection protocol. The order recognition strategy based on Schubert calculus is then introduced, including the demonstrations using the experimentally observed optical near-field pattern. We also quantitatively compare order recognition abilities with random number distributions, such as uniformly distributed random numbers.\\'- - \\'Preparation (1): observation of a complex optical near-field transmission pattern on the photochromic crystal\\' - \"We reviewed the results of previous studies in the present study. First, we summarized the complex photon path formation in photochromic materials. As mentioned above, the material was a photochromic single crystal made of diarylethene, with the molecular structure shown in Fig._a. Upon UV irradiation, the molecule was isomerized from a transparent isomer (1o) into a blue-coloured isomer (1c) while maintaining the crystal structure. Upon visible light irradiation, the material was converted back into a colourless or transparent isomer. The black and blue curves in Fig._a represent the absorption spectra of isomers 1o and 1c, respectively. The molecular lengths of the two carbon atoms at 4- and 4- positions in the phenyl rings of 1c and 1o were 1.39_nm and 1.41_nm, respectively, and the thicknesses of the molecules 1c and 1o were 0.39_nm and 0.49_nm, respectively, indicating that structural deformation accompanies photoisomerization.\" - \"The local optical excitation and its near-field optical transmission characterization were conducted using a double-probe scanning near-field optical microscope (Unisoku, USM-1300S), which has a metallic probe on one side and an apertured fibre-optic probe on the other side (Fig._c). With the sharpened gold-coated probe tip located near the surface of the diarylethene crystal with a thickness of 100_ u03BCm, local optical excitation was induced using light with the wavelength of 532 nm. Here, we utilized the local electric-field enhancement at the tip of the sharpened metallic probe. The near-field transmission was measured using an optical fibre probe with a spatial resolution of approximately 50_nm, which was connected to a single photon counting apparatus. The two-dimensional scanning area was 2_u03BCmu2009xD7u20092_u03BCm with a resolution of 256 pixelsu2009 xD7u2009256 pixels.\" - \"Figure_d shows the observed near-field photon count distribution obtained after adopting a two-dimensional Gaussian filter with a standard deviation of 6 pixels (approximately 50_nm), corresponding to the resolution limit of the system. We observed complex subwavelength structures with a representative scale of 100u2013200_nm. In particular, the local excitation by the metallic tip did not uniformly propagate through the material. The generation of such a complex pattern was attributed to the balance between the mechanical deformation of the photochromic material and photoisomerization. This suggests that local photoisomerization induces anisotropic deformation of the molecular size, leading to anisotropic mechanical strain, which causes subsequent photoisomerization in a non-uniform manner in the surrounding material.\" - \"In other words, a chain of spontaneous symmetry breakings was generated during local photoisomerization involving branching and selection; the near-fieldu2013 excited diarylethene was locally modified to be transparent. The developed transparent area was accompanied by an anisotropic mechanical strain in the adjacent area, leading to a suppressed or enhanced photoisomerization of the subsequent photons. Therefore, an anisotropic strain field from the adjacent locally photoisomerized areas provided an anisotropic spatial distribution of the quantum yield of photoisomerization within the crystal. Such chains of photoisomerization and strain-field formation led to nanometre-scale transparent paths, resulting in the generation of versatile patterns observed in Fig._d. After generating a transparent path, the two-dimensional near-field observation was conducted with the input light from the metallic tip at a single photon level, with photon energies lower than the limit to induce further photoisomerization. Diarylethene used in this study does not exhibit thermal isomerization from the closed- to open-ring isomer or vice versa; therefore, the constructed nanometre-scale pattern was preserved until complete resetting via strong UV or visible light irradiation was observed.\" - \"As reported by Nakagomi et al., the metal probe was illuminated using 532-nm laser light with an intensity of 1_u03BCm/cm. The number of photons detected by the optical near-field fibre probe was in the range of 10 counts per second (cps) from approximately 100-nm order spatial area. Although the principal contribution of the present study was constructing an order recognition strategy, we briefly discussed the photon utilization efficiency of the experiment described in the study by Nakagomi et al., which was not discussed in previous studies. Future studies with a detailed experimental discussion are warranted.\" - \"For simplicity, assuming that the local excitation occupies an approximately 100-nm square area and the tip-enhancement factor by the gold-coated probe is 100, the number of photons that are subjected to the nanometre-scale transparent paths formed in the photochromic crystal per second is approximately 30,000 or the input photon rate is 30,000 cps. Furthermore, assuming that the input photons can travel in the 5-u03BCm square area of the sample, the number of photons observed from a 100-nm square area is approximately 12 cps. Interestingly, this number is comparable to the experimentally observed photon counts by SNOM, meaning that, the intermediate loss is negligible. However, if the tip enhancement by the gold probe is estimated to be 1000, the output photon should be 120 cps, meaning that the loss is approximately 90%. As mentioned above, detailed characterization is warranted in future studies.\" - \\'Preparation (2): generation of the Schubert matrix\\' - \"A single photon travels through the generated complex transparent paths and is finally transmitted from a particular position to the opposite side. The spatial position of the photon detection differs from photon to photon, and Fig._d represents its statistics. Uchiyama et al. reported that, although the spatial positions of the photon observations were versatile, it was constrained by the transparent paths formed in the diarylethene crystal, which were originally specified by a fixed local excitation from the metallic probe. To transform the spatial near-field photon statistics into versatile information in a coherent manner, Uchiyama et al. incorporated the following simple photon detection protocol to generate Schubert polynomials, which were equivalent to what they called u201CSchubert matricesu201D, as introduced in the present study.\" - Schubert matrix - \"In a Schubert matrix, there is only a single element of 1 for all rows and columns, while all the other elements are 0. In other words, a Schubert matrix represents a permutation. the Schubert matrix has a one-to-one correspondence with the Schubert polynomial. See the u201Cu201D section for details of the definition of Schubert polynomials. Schubert matrices of the size Nu2009xD7u2009N were utilized for recognizing the order of N distinct slot machines. As given by Eqs. ()u2013(), a Schubert polynomial corresponding to an Nu2009xD7u2009 N Schubert matrix is a polynomial composed of the term (x_{1} ,x_{2} , ldots ,x_{N - 1}).\" - \\'Basic notions: diagram, inversion, Young tableau, and singularity\\' - \"Herein, we introduce a few basic notions to characterize Schubert matrices: diagram, inversion number, and singularity. An example of a Schubert matrix is a 4u2009 xD7u20094 matrix given by\" - \"$$S = left( {begin{array}{*{20}c} 0 & 1 & 0 & 0 0 & 0 & 0 & 1 0 & 0 & 1 & 0 1 & 0 & 0 & 0 end{array} } right),$$ (1) \" - \"which is illustrated in Fig._a with yellow and green elements denoting 1_ s and 0_s, respectively. Herein, we marked the elements that are located on the right and lower sides of the yellow elements, which are marked by thick black lines in Fig._b. Therein, the non-marked elements are called the diagram of the Schubert matrix, which in this case is given by\" - \"$$left( {begin{array}{*{20}c} * & 0 & 0 & 0 * & 0 & * & 0 * & 0 & 0 & 0 0 & 0 & 0 & 0 end{array} } right).$$ (2) \" - \"Figure 2Schematic of an example of the Schubert matrix. (a) The permutation of [2431] is represented by a matrix S in Eq._(). (b) The diagram of the matrix S is illustrated. The upper-left block consisting of (*) s (indicated by orange squares) is called the Young tableau, whereas isolated (*) blocks are called singularities.\" - \"The number of (*) s in the diagram is called the inversion number, which is 4 in the case of the present example. An array of (*) s located at the upper-left corner of the matrix is called Young tableau. Furthermore, in the diagram, let us connect (*) s into blocks if they are connected by any of their four neighbours: up, down, left, or right. We then count the number of blocks of connected (*) s, which is 2 in the case of Eq._(). Here, the number of singularities is defined by the number of such blocks without counting the Young tableau. In the case of Eq._(), there are two blocks of (*), with one of them being the Young tableau. Therefore, the singularity is 1.\" - How to generate a Schubert matrix - \"The generation of a Schubert matrix S is described in the study by Uchiyama et al.. Here, we introduced the procedure along with a case of generating a Schubert matrix with a size of 4u2009xD7u20094. As previously illustrated, a single photon was observed at a spatial position with the corresponding probability distribution (Fig._a). The details of the conversion from the experimentally observed photon statistics to selection probability are described in the previous study.\" - \"1.The probability distribution specified by the near-field observation was rescaled or coarse-grained into an Nu2009xD7u2009N grid. The probability at each point of the grid is denoted by Pi,j (iu2009=u20091,u2026, N, ju2009 =u20091,u2026, N) hereafter, where N is a natural number. In the example, N is equal to 4 (Fig._a). In the physical system shown in Fig._c, when sending a single photon, we observed the photon at one of the grid points. In this study, by using uniformly distributed pseudorandom numbers generated in a computer and pondered by the measured probabilities, we emulated the output photon observations. 2.We counted the number of photons detected at each grid element. Let the photon count at the grid point (i, j) be denoted by Ci,j (iu2009=u20091,u2026 , N, ju2009=u20091,u2026, N). When the photon count at a certain position Cm,n reaches the threshold value denoted by PT, we determined that the (m, n)-element of the Schubert matrix is given by 1; Sm,nu2009=u20091. In the example shown in Fig._b, S3,3u2009=u20091.3.Accordingly, the photon detection probability distribution Pi,j was updated for the rest of the loop. First, photon detection at the m-th row and the n-th column was configured to be zero, which are denoted by x marks in Fig._c, i.e., P3,ju2009=u20090 (ju2009=u20091,u2026, 4) and Pi,3u2009=u20090 (iu2009=u20091,u2026, 4). This reconfiguration was based on the definition of the Schubert matrix. Second, the probability distribution Pi,j was renormalized such that (sumnolimits_{i,j} {P_{i,j} } = 1) was maintained.4.Similar to Step (2) described above, when Cm,n reaches PT, we determined the (m, n)-element of the Schubert matrix to be 1, i.e., Sm,nu2009=u20091. In the example shown in Fig._ d, S2,4u2009=u20091.5.We repeat steps (2) and (3) for two times (Fig. eu2013h); S1,2u2009=u20091 is determined as shown in Fig._f, followed by S4,1u2009=u20091 in Fig._h.\" - \"Figure 3Adapted from Uchiyama et al., Sci. Rep. 10, 2710 (2020). Copyright 2020 Author(s), licensed under a Creative Commons Attribution 4.0 License.Generation of Schubert polynomials based on optical near-field observation. (a) The detection area is divided into Nu2009xD7u2009N elements. Here Nu2009=u20094. The photon detection probability is spatially irregular. (b) When a photon is detected at an element (i, j), the (i, j)-element of the Schubert matrix is given by 1. Here, (i, j)u2009=u2009(3,3). (c) The sensitivities of all the elements in the third row and column are zero. The probabilities are renormalized. (d) The next element of the Schubert matrix is determined. (eu2013h) The process described in (c) and (d) is repeated. (h) is the finally obtained Schubert matrix in this example.\" - \"The final determination of S1,2u2009=u20091 is immediately given after the previous step because only a single grid point has a non-zero photon observation probability. The resultant Schubert polynomial is (x_{1}^{2} x_{2} x_{3} + x_{1} x_{2}^{2} x_{3}); see the u201Cu201D section for information regarding Schubert polynomials.\" - \"The total number of different Schubert matrices of size Nu2009xD7u2009N is given by N!, meaning that the factorial order increase with the size of the observation elements. With Nu2009=u200932, the total number is more than 10. The ability to generate such an extremely diverse pattern from the same substrate via optical near-field processes was the primary topic investigated by Uchiyama et al..\" - \"The photon number threshold PT played an important role. A smaller PT value leads to diverse choices of 1_s in the generated Schubert matrix. Conversely, a larger PT value indicates that the element with the maximum photon detection probability most likely yields the value of 1 in the resulting Schubert matrix. In other words, the resultant Schubert matrices are more weakly and more strongly correlate with the source optical near-field pattern with smaller and larger PT values, respectively. These are called soft and hard correspondences between the near-field patterns and Schubert matrices, as described previously by Uchiyama et al..\" - Order recognition - In previous sections, we reviewed the background and fundamentals needed for the generation of Schubert matrices based on near-field photon statistics from a photochromic material. The primary focus of this study is to utilize these Schubert matrices for information processing; specifically, our main objective is to recognize the order or ranking of slot machines with an initially unknown reward probability. - Order recognition by the revision of the name and value - \"The key is to exploit the inversion relation presented in a Schubert matrix; here, we introduce a 4u2009xD7u20094 matrix, which is schematically illustrated in Fig._a. If the column number of the element with the value of 1 in the i-th row is smaller than that in the (iu2009+u20091)-th row, we define that these two elements are in a relation of inversion. Furthermore, this inversion is related to the concept of singularity, as described above. The diagram of the Schubert matrix (Fig._a) is shown in Fig._b.\" - Figure 4Order recognition principle based on Schubert polynomials. (a) Inversion relation (marked by a blue arrow) inherent in the Schubert matrix. (b) Young tableau (orange colour) and a singularity (red colour). (c) Illustration of an anti-diagonal matrix and (d) its diagram consists of a Young tableau with the maximum inversion number. (e) In the proposed ordering strategy, a matrix is characterized by the name and value axes, which correspond to the vertical and horizontal directions, respectively. In the name axis, the upper row indicates a high rank and in the value axis, the right column indicates a high value. (f) Our decision of machine selection is suggested by the Schubert matrix. We paid attention to the inversion relation of the given matrix, which is the first and the second row in (f). Let us assume that the names of the first and second rows be machines A and B, respectively. The inversion relation suggests a contrary situation between the name and value axes. (g) If the reward dispensed by machine B is greater than that by machine A, the association of the name axis is inaccurate; hence, the allocation of the names is swapped. (h) If the reward by machine A is larger than that by machine B, the association of the value axis is inaccurate; therefore, the allocation of the values is swapped. - \"Conversely, when the Schubert matrix is represented by an anti-diagonal matrix (Fig._c), there are no inverse relations. The diagram of an anti-diagonal matrix is shown in Fig._d, where the upper triangular elements and the Young tableau only contribute to forming the inversion number. The inversion number is maximized, and there is no singularity. The Schubert polynomial is represented by a single term (x_{1}^{3} x_{2}^{2} x_{3}^{{}}).\" - As described above, inversion relations and singularities indicate characteristic properties. The idea of the proposed ordering strategy based on Schubert calculus begins with associating information to the vertical (row) and the horizontal (column) axis of Schubert matrices. - \"Let us assume that there are four slot machines (M1, M2, M3, M4) and their reward probabilities are given by (P1, P2, P3, P4)u2009=u2009(0.4, 0.2, 0.8, 0.6), which are unknown to us. In this case, the ground truth order of the slot machines reward probability is M3, M4, M1, and M2 in descending order. We want to recognize this ground truth order by playing the slot machines.\" - \"The first critical point of the proposed order recognition strategy is to associate the vertical (row) and horizontal (column) axes of a matrix with the name and value, respectively. Specifically, the names of the slot machines are bound with the rows; the upper rows indicate a higher-ranked machine, while the lower rows indicate a lower-ranked machine. Meanwhile, the columns correspond to the reward probabilities, with the right column being the high reward probability. These associations are illustrated in Fig._e. In the pseudo-code shown below, these associations are represented by line 5.\" - \"A Schubert matrix proposes the relation between the name and value by the elements of 1. Assuming that the elements in the matrix along the name axis are perfectly arranged in the order of the ground truth, and the elements along the value axis are arranged in the order of the ground truth reward probabilities in the ascending order, then, the matrix results in an anti-diagonal matrix, as shown in Fig._c. Here, there are no inverse relations and no singularities, suggesting that, the matrix for the perfect order recognition is an anti-diagonal matrix. Therefore, transforming the initially unsure order regarding both the name and value axes to adapt the correlated Schubert matrices toward an anti-diagonal matrix corresponds to accomplishing the correct order recognition. The question now is how to realize such transformations.\" - We considered that Schubert matrices propose which machines should be checked to determine the correct order. In this study, two machines are selected for a single-slot machine play. This is similar to the multi-armed bandit (MAB) problem with multiple plays, although the aim of solving MAB problems and that in the present study are fundamentally different. The purpose of solving MAB problems is usually to minimize regret, whereas the objective of the present study is to recognize the order. - \"The machine selection was determined by the inverse relation in the Schubert matrix that provided the deviation from the anti-diagonal matrix. In the example shown in Fig._a, there is one inverse relation indicated by a solid blue arrow.\" - \"We now focus our attention on the inverse relation. In the example shown in Fig._ a, the inversion denotes the first and the second rows. Let us assume that the names of the first and second rows are machine A and machine B, respectively. Based on the definition of the vertical and horizontal axes of the matrix shown in Fig._e, the relation between machines A and B is contradictory (Fig._f). According to the name axis (vertical), machine A was considered a higher reward machine. However, the value axis (horizontal) indicated that the value of machine B was greater than that of machine A, suggesting that the relationship is contradictory.\" - To resolve such a contradiction, we selected these two machines and checked the betting results. - \"[CASE 1]: When the reward dispensed by machine A is greater than that of machine B:The order of the name axis is correct; machine A is above machine B. In contrast, the value axis is wrong; the value of machine A should be on the right-hand side of machine B. Therefore, we swapped the allocation of the values, leading to a situation depicted in Fig. g.[CASE 2]: When the reward dispensed by machine B is greater than that of machine A:The order of the value axis is correct; the value of machine B is on the right-hand side of machine A. Conversely, the name axis is wrong; machine B should be above machine A. Therefore, we swapped the allocation of the names, leading to a situation depicted in Fig. h.\" - The resulting arrangement is the same as shown in Fig. g and h for both cases; however, the reconfiguration of the axes is contrasting. - \"After finishing the reconfiguration using the current Schubert matrix, we continued the process by using the next Schubert matrix, which is shown in line 5 in the pseudo-code below (Su2190 Schubert matrix). Furthermore, the row and column of the new Schubert matrix were transformed by the name and value axes to date, which is represented by line 5 in the pseudo-code (Su2190 S[name,value]).\" - \\'The above-described principle is summarized as a pseudo-code by the following algorithm: variable t is incremented after a single loop is completed, as shown by line 18 in the pseudo-code. A single Schubert matrix is used in a single loop given by line 5.\\' - \\'\\' - Demonstration - \"We demonstrated the order recognition with 16 slot machines (M1u2013M16) with the ground truth reward probability setting of (P1u2013P16), as shown by the bar graph in Fig._a(i). The true order of the reward probability in the descending order is given by M9, M14, M8, M11, M2, M6, M3, M13, M1, M4, M12, M7, M5, M10, M16, and M15, as shown in Fig._a(ii).\" - Figure 5Demonstration of order recognition in 16 slot machine environments. (a) (i) Reward probability arrangement and (ii) its sorted representation. (b) The accumulated incidence pattern of the ranking-and-value matrix after the loop of (i) 1, (ii) 50, (iii) 100, (iv) 200, (v) 500, and (vi) 1000. (c) The time evolution of the estimated ranking of each slot machine. (d) The number of correctly ordered machines over time. - \"Initially, the association of the name axis starts with the linear order corresponding to the index of the machines, i.e., M1u2013M16. We used a total of 10,000 Schubert matrices consecutively. This whole procedure was repeated for Ru2009=u2009 100 trials. As the reward dispensed by each machine was probabilistic, a varying winu2013lose event occurred among R-times trials.\" - \"The images in Fig._b demonstrate how order recognition evolves as time elapses. Specifically, the horizontal axis indicates the number of Schubert matrices used, which is equal to the elapsed number of loops and is denoted by t using the notations shown in lines 3 and 18 of the Algorithm.\" - Here, the order of machine Mi at time t was evaluated by calculating how often the machine Mi is associated with the row number of the Schubert matrices. Meanwhile, we computed how the value of Mi was evaluated at time t by analyzing how its value was ranked in the horizontal direction. - \"The two-dimensional images in Fig._b show how the order recognition evolves by counting the accumulated incidences of the matrix (ranking and value), referred to as the ranking-and-value matrix hereafter, at tu2009=u20091, 50, 100, 200, 500, and 1000. If the order recognition is completed to perfection, the top-ranked machine owns the highest value, whereas the worst-ranked machine has the minimum value. Therefore, the ranking-and-value matrix exhibits higher incidences along the anti-diagonal elements.\" - \"As time elapses, the high incidence elements were gradually localized along the anti-diagonal axis, indicating successful order recognition. Immediately after the first round of the loop (tu2009=u20091), the ranking-and-value matrix was in an anti-diagonal arrangement (Fig._b(i)). This is because there was only a single incidence regarding both ranking and value axes at tu2009=u2009 1. As shown in Fig._b(ii), the ranking-and-value matrix shows a diverse incidence pattern at tu2009=u200950.\" - \"Figure_c visualizes the results in another manner, where the vertical axis represents the estimated order, while the horizontal axis shows the number of Schubert matrices used or the time dimension. Precisely, the estimated order of machine Mi at time t was calculated by the average of the estimated ranking of Mi by the time t. Furthermore, such an estimated order is averaged over 100 different trials. The 16 curves therein, shown with different colours, represent the time evolution of how machine Mi was ranked. We observed that the estimated order was drastically reconfigured during the first approximately 500 steps. When the time step was around 2000, the order recognition was accomplished.\" - \"Furthermore, Fig._d summarizes the number of correct order-recognized machines as a function of time, which equals 16 when the perfect order recognition is achieved. Indeed, after tu2009=u20092298, the estimated order agreed with the ground truth ranking. We also observed that the order of most machines (14 out of 16) was recognized at tu2009=u2009626. Therefore, the order recognition was almost completed at tu2009=u2009626, although the perfect recognition was accomplished at tu2009=u20092298.\" - \"The error bars of Fig._c indicate the standard deviation of the estimated order; when the ground truth reward probability difference between the slot machines was minute, the error bars of associated machines overlapped with each other. We observed such a situation regarding M15 and M16, where the reward probability difference was only approximately 0.02.\" - \"However, we can automatically grasp such uncertainty by the ranking-and-value matrix. Indeed, we observed a high incidence square area or a block matrix in the lower-left corner of the matrix shown in Fig._b(iv), meaning that the ranking of the 15th and 16th machines are comparable. This is a remarkable attribute of the proposed strategy in which the strong correlations inherent in the problem under study are automatically detected.\" - Furthermore, as mentioned above, the proposed order recognition strategy relies only on the two choices specified by Schubert matrices and their resulting outputs. As opposed to the former order recognition principles, the present study did not involve the notions of expected probabilities and confidence intervals.- - Comparison - \"Here, we discuss the differences in Schubert matrices and their order recognition performances among various random number generators, especially, in a spatially uniform probability distribution called u201Cuniformu201D (Fig._a) and a spatially strongly skewed probability pattern called u201Ccentreu201D (Fig._ b). The latter has a more significant probability in the square area located at the centre. The detailed setting of the u201Ccentreu201D is described in the u201Cu201D section. The near-field photon observation pattern (Fig._d) is simply referred to as u201Cnear-fieldu201D hereafter.\" - \"Figure 6Characterization of Schubert polynomials generated by the near-field pattern compared with the (a) uniformly distributed and (b) spatially strongly skewed (centred) patterns. (c) Inversion number distribution when Nu2009=u2009 8 by the near-field, uniform, and centre patterns. (d, e) Basic statistical analysis of the (d) inversion number and (e) singularity number as a function of the size of the matrix (N). (i) Average, (ii) median, and (iii) standard deviations are examined and no evident differences among the sources are observed. (f, g) When the incidence is normalized by the number of unique matrices, evident source dependencies are observed in both the (f) inversion number and (g) singularity number.\" - \"The motivation behind the comparison among the near-field, uniform, and centre patterns is described as follows: the spatial inhomogeneity of the optical near-field observed from the locally excited photochromic crystal is one of the most significant characteristics, as shown in Fig._d, which was examined in detail in a previous study. A contrasting counterpart is a spatially completely uniform distribution, from which we examined the difference between the uniform and near-field patterns.\" - The near-field pattern was not too spatially skewed, and a high-incident photon count area was spatially distributed. Based on these observations, we examined the case where the photon counts were strongly concentrated in a particular spatial area. By comparing such spatially extremely skewed cases and near-field patterns, we could examine the dependencies on spatial skewness. The centre pattern is an example of such a spatially skewed distribution. The size of the high-incident area and its position were also considered. We primarily focused on the centre pattern as a representative case in the present study. - Schubert matrix statistics - \"We generated Schubert matrices based on the near-field, uniform, and centre patterns with a PT value of 1. Figure_c shows the statistics of the inversion number when N is equal to 8. Herein, we generated 2u2009xD7u200910 Schubert matrices for each of the near-field, uniform, and centre patterns, for which inversion number statistics were depicted by the blue, green, and red curves in Fig._ c, respectively. Both the near-field and uniform patterns exhibited symmetric incidences with a peak value having an inversion number of 14. The centre pattern showed a local minimum incidence when the inversion number was 14 and exhibited fluctuations as a function of the inversion number. The fluctuation is discussed later.\" - \"Although the centre pattern deviated from the near-field and uniform patterns, the average, median, and standard deviation exhibited almost the same value, as demonstrated in Fig._d, when N is equal to 8. Therefore, there are no evident differences from the viewpoint of such statistical perspectives.\" - \"Furthermore, we examined the statistics of Schubert matrices with large sizes of up to Nu2009=u200924. The square, x, and circular marks in Fig._d(i) represent the average value of the inversion number of the generated Schubert matrices from the near-field, uniform, and centre patterns, respectively, as a function of the size of the matrix (N). The detailed setting of the generation of a Schubert matrix is described in the u201Cu201D section. Figures_d(ii) and 6d(iii) summarize the median and standard deviation of the inversion number. To summarize, we did not observe significant source-signal dependencies in these basic statistics for the inversion number from Fig._d.\" - \"Similarly, we examined the average (Fig._e(i)), median (Fig._e(ii)), and standard deviation (Fig._e(iii)) of the singularity of the generated Schubert matrices from the near-field, uniform, and centre patterns, respectively. There were no explicit source-signal dependencies for singularity.\" - \"In contrast, evident source-signal dependencies were observed when the uniqueness of Schubert matrices was considered. The blue, green, and red curves in Fig._ f(i) represent the frequency of Schubert matrices divided by the unique number of Schubert matrices as a function of their inversion number in the near-field, uniform, and centre patterns, respectively, when N was equal to 8. However, the uniform pattern yielded an almost identical frequency, regardless of the inversion number; the near-field pattern showed a peak frequency around the inversion number of 10. Similar trends were observed when N was equal to 9 and 10, as demonstrated in Fig._f(ii) and f(iii), respectively.\" - \"Similarly, the frequency of Schubert matrices divided by the unique number of Schubert matrices exhibited a similar tendency with respect to the number of singularities, as shown in Fig. g(i) (Nu2009=u20098), Fig. g(ii) (Nu2009 =u20099), and Fig. g(iii) (Nu2009=u200910). Although the uniform pattern displayed identical values for all singularities, the near-field pattern provided a more significant frequency as the singularity increased. The statistics were highly similar among these three source patterns. Therefore, the observation in Fig._ g suggests that versatile Schubert matrices are generated through small singularities via the near-field pattern. Regarding the centre pattern, the generated Schubert matrices were mostly similar to each other when the number of singularities was small.\" - \"The near-field pattern was based on the experimentally observed photon statistics provided by Uchiyama et al., as demonstrated in Fig._d. The original data were obtained with the size of 256u2009xD7u2009256 pixels, which was rescaled to form Nu2009xD7u2009N blocks when the size of the Schubert matrix was specified by Nu2009xD7u2009N. When the original size of 256 pixels was not indivisible by N, the photon counts on the boundary were divided proportionally.\" - \"The underlying mechanism of the oscillatory behaviour of the inversion number generated in the centre pattern observed in Fig. c and f is not understood yet. However, we considered that the following properties may be involved: The centre pattern induces a high probability of generating non-zero elements in the centre area. Therefore, based on the construction protocol of the Schubert matrix, it is highly unlikely to have non-zero elements in the centre areas north, south, west, and east sides. Hence, the resultant generated Schubert matrix inherits such a spatially highly skewed nature of the centre pattern.\" - Comparison of order recognition performances - \"We examined the order recognition performances depending on the source patterns regarding eight slot machine (M1u2013M8) environments. While the demonstrations showed so far suppose a probabilistic reward environment, here we assume the deterministic, different-valued rewards among (0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, and 0.9). The correspondence between M1 to M8 and the rewards are randomly configured; we evaluated Ku2009=u20098!u2009=u200940,320 types of unique correspondences between the machines and the rewards.\" - \"For a given reward environment, consecutive 250-time slot machine plays were conducted. Such plays were repeated for K-types of different reward environments. The correct order recognition rate (COR) at the time step t was defined by the number of times when the order of the reward was correctly recognized at the time step t among K-types of different reward environments divided by K. The blue, green, and red curves in Fig._a represent the time evolution of COR. We observed that the near-field pattern most promptly achieved a higher COR than others. On the contrary, the centre pattern could not conduct correct order recognition for certain reward environments because COR does not converge to unity.\" - \"Figure 7Order recognition performance evaluations with different source randomness using eight slot machine environments. (a) Time evolution of the COR rate for the near-field, uniform, and centre patterns. (b) The time steps exceed the COR value of 0.9 as a function of the photon number threshold (PT). (c) The breakdown of all permutations of Nu2009=u20098 by the number of singularities. (d) Comparison of order recognition performances with the problem settings having the same singularity value. When the singularity number is small than 2, the near-field pattern outperforms other sources, whereas, with the largest singularity, the centre pattern exhibits the promptest responses.\" - \"We then examined the order recognition ability as a function of the photon threshold PT. Figure_b characterizes the number of steps when COR exceeds the value of 0.9 as a function of photon threshold PT. The near-field pattern, denoted by the blue curve, achieved smaller steps than others. The uniform and centre patterns, marked by the green and red curves, respectively, did not show evident PT dependencies, which was understood from the fact that the uniform and centre patterns do not provide different spatial inhomogeneity by changing PT. The near-field pattern, marked by the blue curve, showed the minimum steps when PTu2009=u20098. However, the difference in the time steps by changing PT was not evident in this study, even in the case of a near-field pattern.\" - \"Finally, we examined order recognition from the viewpoint of singularities. There are a total of Ku2009=u200940,320 assignments of the rewards to the eight slot machines. Each assignment can be represented by a Schubert matrix. Therefore, we can derive the singularities of the given assignment. Figure_c shows the number of assignments, with the number being annotated with the bar graph, as a function of the number of singularities ranging from 0 to 6.\" - \"We evaluated the COR depending on the singularity number of the assignments, ranging from 0 to 5, as shown in Fig._d. When the singularity number was smaller than or equal to 2, the near-field pattern accomplished the fastest adaptation to CORu2009=u20091 or the perfect order recognition, whereas the COR by the centre pattern was limited below unity. Conversely, beyond the singularity number of 3, COR exhibited comparable performance irrelevant to the source-signal patterns. However, with the singularity number of 5, the centre pattern outperformed the others.\" - \"Interestingly, such a trend was contravariant to the trend of the frequency of the unique Schubert matrix analyzed in Fig._f. Therefore, the near-field pattern yields versatile Schubert matrices when there are few singularities and provides superior ordering performances for slot machine assignments with small singularities. With the centre pattern, on the other hand, the versatility of Schubert matrices is low when there are few singularities, and the order recognition ability is inferior when the singularity number of the problem is small. Overall, as shown in Fig._c, the dominant parts of all arrangements consisted of fewer singularity configurations; thus, on average, COR by near-field pattern predominantly exhibits higher values than the centre and uniform patterns, as summarized in Fig._ a. Furthermore, the time step exceeding the COR of 0.9 by the near-field pattern was smaller than that in the uniform and centre patterns, as shown in Fig._ b.\" - However, detailed insights into this phenomenon are warranted in future studies. For example, the time at which COR reached unity appears to be almost the same between the uniform and near-field patterns. Additionally, in the initial time, where the time was shorter than approximately 50, the centre pattern slightly outperformed the other two patterns. - Moreover, a deeper understanding of the underlying mechanism behind the relationship between the singularities of the source Schubert matrices and the order recognition is an exciting topic for future studies. For now, we believe that the abovementioned results demonstrate that the matching of the versatility of Schubert matrices with the number of singularities of the given problem plays a key role. Presumably, by subjecting a variety of different Schubert matrices matched with the singularity number of the given situation, the revision of the ordering proceeds efficiently. If a limited type of Schubert matrices is used, the order cannot be accurately recognized owing to insufficient comparisons. In a similar context, Okada et al., demonstrated that negative autocorrelation contained in a chaotic time series accelerates solving the two-armed bandit problems, and this mechanism was recently evaluated based on correlated random walks, which manifests the benefits gained by correlated randomness. - If one can know the singularity of the given problem as prior knowledge, the acceleration of the order recognition will be possible by engineering the source pattern that comprises appropriate singularities. For example, if the singularity number of the given problem is 5 or 6 only, the usage of the centre pattern is beneficial rather than the use of near-field or the uniform patterns. A clear understanding of related issues may be another interesting topic for future studies.- - We demonstrate an order recognition algorithm based on Schubert polynomials generated via irregular spatial distribution of photon transmission through a photochromic crystal photoisomerized by a local optical near-field excitation. The proposed strategy exploits the inversion relations and the singularities inherent in Schubert polynomials. A successful order recognition is demonstrated for 16 different reward probability slot machines. Furthermore, the performance of order recognition is quantitatively compared among the source Schubert matrices generated via near-field, spatially uniform, and strongly skewed patterns. We found that the singularity number contained in Schubert matrices and that of the given problems exhibit a clear correlation, suggesting that the order recognition will accelerate if the singularity number of the considered environment is presupposed. This study paves the way toward nanophotonic intelligent devices and systems by the interplay of complex natural processes and mathematical insights gained by Schubert calculus.- - Definition of Schubert polynomials - The Schubert polynomial (left{ {{varvec{S}}_{omega } } right} = left{ {{varvec{S}}_{omega } (x)} right} = left{ {{varvec{S}}_{omega } (x_{1} , ldots ,x_{N - 1} )} right}) in the symmetric group SN is defined through the divided difference operator (partial_{xy}) given by - \"$$partial_{xy} f = frac{f(x,y) - f(y,x)}{{x - y}}.$$ (3) \" - Let the polynomial of the permutation with the maximum inversion number be given by - \"$${varvec{S}}_{{omega_{0} }} = x_{1}^{N - 1} x_{2}^{N - 2} ldots x_{N - 1}^{{}} .$$ (4) \" - For a general permutation (omega), the polynomial is defined by - \"$${varvec{S}}_{omega } = partial_{{omega^{ - 1} omega_{0} }} {varvec{S}}_{{ omega_{0} }} .$$ (5) \" - \"In the case of the four-dimensional symmetric group (S_{4}), the permutation with the largest inversion number is (omega_{0} = [4321]), which is represented by the anti-diagonal matrix shown in Fig._c. Its inversion number is 6, as shown in Fig._d. The corresponding Schubert polynomial is ({varvec{S}}_{{ omega_{0} }} = {varvec{S}}_{4321} = x_{1}^{3} x_{2}^{2} x_{3}^{1}).\" - \"The Schubert polynomial of the permutation ([4231]) is given by the divided difference operation on ({varvec{S}}_{{omega_{0} }}) regarding x2 and x3, which corresponds to the interchange of the second and third rows in the Schubert matrix. Hence, ({{{varvec{S}}_{4231} = left( {x_{1}^{3} x_{2}^{2} x_{3}^{1} - x_{1}^{3} x_{3}^{2} x_{2}^{1} } right)} mathord{left/ { vphantom {{{varvec{S}}_{4231} = left( {x_{1}^{3} x_{2}^{2} x_{3}^{1} - x_{1}^{3} x_{3}^{2} x_{2}^{1} } right)} {(x_{2} - x_{3} )}}} right. kern-nulldelimiterspace} {(x_{2} - x_{3} )}} = x_{1}^{3} x_{2}^{{}} x_{3}^{{}}) is obtained. Similarly, the Schubert polynomial of [2431] is given by the divided difference of ({ varvec{S}}_{4231}) with respect to x1 and x2, which corresponds to the interchange of the first and second rows in the Schubert matrix. Therefore, ({{{varvec{S}}_{2431} = left( {x_{1}^{3} x_{2}^{{}} x_{3}^{{}} - x_{2}^{3} x_{1}^{{}} x_{3}^{{}} } right)} mathord{left/ {vphantom {{{varvec{S}}_{2431} = left( {x_{1}^{3} x_{2}^{{}} x_{3}^{{}} - x_{2}^{3} x_{1}^{{}} x_{3}^{{}} } right)} {(x_{1} - x_{2} )}}} right. kern-nulldelimiterspace} {(x_{1} - x_{2} )}} = x_{1}^{2} x_{2}^{{}} x_{3}^{{}} + x_{1}^{{}} x_{2}^{2} x_{3}^{{}}) is obtained, which is the Schubert polynomial corresponding to the Schubert matrix shown in Fig._ a. All Schubert polynomials, inversion numbers, and singularities of S4 are summarized in Table .\" - Table 1 Schubert polynomials of S4 and the corresponding inversion and singularity numbers. - Simulation details - \"In the performance comparison of the order recognition, we generated Schubert matrices from a spatially strongly skewed probability distribution called u201C centreu201D. Herein, we describe the details of this pattern. When the size of the matrix under study is Nu2009xD7u2009N, the area of the centre is the set of elements (i, j) assuming the integer numbers of (leftlceil {7N/16} rightrceil le i le leftlceil {9N/16} rightrceil) and ( leftlceil {7N/16} rightrceil le j le leftlceil {9N/16} right rceil), where (leftlceil * rightrceil) denotes the nearest integer greater than or equal to *. The detection probability of a pixel belonging to the centre was 100 times larger than that of the pixel outside the centre area.\" - \"We examined the statistics of the Schubert matrices regarding Nu2009=u2009 8, 9, 10, 11, 12, 16, 20, and 24. The number of generated Schubert matrices for each case is summarized in Table . As the total number of different Schubert matrices is N!, the number of generated matrices should be sufficiently high to investigate the statistical characteristics. However, the computational demands are too high, especially when N is larger than 11 in the present computing environment. For these reasons, the number of the generated matrices was limited to 2u2009 xD7u200910 when N is larger than or equal to 11.\" - Table 2 The number of generated matrices. - \"The computing environment was a workstation comprised of HPC Systems (CPU: Xeon Gold 6249R (20 cores, 3.1_GHz), RAM: 384_GB, OS: Ubuntu), and Matlab was the utilized software platform.\"- - \\' \\' - The datasets used and/or analyzed in the current study are available from the corresponding author upon reasonable request. - \\' \\'- - \"Kiyoto, N., Hakuta, S., Tani, T., Naya, M. & Kamada, K. Development of a near-infrared reflective film using disk-shaped silver nanoparticles. Fujifilm Res. Dev. 58, 55u201358 (2013)._ Geiregat, P., Van Thourhout, D. & Hens, Z. A bright future for colloidal quantum dot lasers. NPG Asia Mater. 11, 1u2013 8 (2019)._ _ _ Mohammadi, E. et al. Nanophotonic platforms for enhanced chiral sensing. ACS Photon. 5, 2669u20132675 (2018)._ _ _ Naruse, M., Tate, N., Aono, M. & Ohtsu, M. Information physics fundamentals of nanophotonics. Rep. Prog. Phys. 76, 056401 (2013)._ _ _ _ Ishihara, T., Shinya, A., Inoue, K., Nozaki, K. & Notomi, M. An integrated nanophotonic parallel adder. ACM J. Emerg. Tech. Com. 14, 1u201320 (2018)._ _ Mohammadi Estakhri, N., Edwards, B. & Engheta, N. Inverse-designed metastructures that solve equations. Science 363, 1333u20131338 (2019)._ _ _ _ _ _ _ Ma, W. et al. Deep learning for the design of photonic structures. Nat. Photon. 15, 77u201390 (2021)._ _ _ _ Zhang, Q., Yu, H., Barbiero, M., Wang, B. & Gu, M. Artificial neural networks enabled by nanophotonics. Light Sci. Appl. 8, 1u201314 (2019)._ _ Ohtsu, M., Kawazoe, T., Yatsui, T. & Naruse, M. Nanophotonics: Application of dressed photons to novel photonic devices and systems. IEEE J. Sel. Top. Quant. 14, 1404u20131417 (2008)._ _ _ Kawazoe, T. et al. Demonstration of nanophotonic NOT gate using near-field optically coupled quantum dots. Appl. Phys. B 84, 243u2013246 (2006)._ _ _ _ Aono, M. et al. Amoeba-inspired nanoarchitectonic computing: Solving intractable computational problems using nanoscale photoexcitation transfer dynamics. Langmuir 29, 7557u20137564 (2013)._ _ _ _ Naruse, M. et al. Decision making based on optical excitation transfer via near-field interactions between quantum dots. J. Appl. Phys. 116, 154303 (2014)._ _ _ Kawazoe, T. et al. Two-dimensional array of room-temperature nanophotonic logic gates using InAs quantum dots in mesa structures. Appl. Phys. B 103, 537u2013546 (2011)._ _ _ _ Zhou, L. et al. Self-assembly of highly efficient, broadband plasmonic absorbers for solar steam generation. Sci. Adv. 2, e1501227 (2016)._ _ _ _ _ Naruse, M. et al. Stochastic processes in light-assisted nanoparticle formation. Appl. Phys. Lett. 100, 193106 (2012)._ _ _ Irie, M. Diarylethene Molecular Photoswitches: Concepts and Functionalities (Wiley-VCH, Weinheim, 2021).Nakagomi, R. et al. Nano-optical functionality based on local photoisomerization in photochromic single crystal. Appl. Phys. A 124, 1u20137 (2018)._ _ _ _ Uchiyama, K. et al. Generation of Schubert polynomial series via nanometre-scale photoisomerization in photochromic single crystal and double-probe optical near-field measurements. Sci. Rep. 10, 2710 (2020)._ _ _ _ _ _ Fomin, S. & Kirillov, A. N. Combinatorial Bn-analogues of Schubert polynomials. Trans. Am. Math. Soc. 348, 3591u2013 3620 (1996)._ _ _ Billey, S. & Haiman, M. Schubert polynomials for the classical groups. J. Am. Math. Soc. 8, 443 (1995)._ _ _ _ Maeno, T. Schubert Polynomials (Sugakushobo, 2016). (In Japanese)Irie, M., Sakemura, K., Okinaka, M. & Uchida, K. Photochromism of dithienylethenes with electron-donating substituents. J. Org. Chem. 60, 8305u2013 8309 (1995)._ _ _ Irie, M., Fukaminato, T., Matsuda, K. & Kobatake, S. Photochromism of diarylethene molecules and crystals: Memories, switches, and actuators. Chem. Rev. 114, 12174u201312277 (2014)._ _ _ _ Kobatake, S., Yamada, T., Uchida, K., Kato, N. & Irie, M. Photochromism of 1, 2-Bis (2, 5-dimethyl-3-thienyl) perfluoro-cyclopentene in a single crystalline phase. J. Am. Chem. Soc. 121, 2380u20132386 (1999)._ _ _ Irie, M., Kobatake, S. & Horichi, M. Reversible surface morphology changes of a photochromic diarylethene single crystal by photoirradiation. Science 291, 1769u20131772 (2001)._ _ _ _ _ Hartschuh, A. Tip-enhanced near-field optical microscopy. Angew. Chem. Int. Ed. 47, 8178u20138191 (2008)._ _ _ Hatano, E. et al. Photosalient phenomena that mimic impatiens are observed in hollow crystals of diarylethene with a perfluorocyclohexene ring. Angew. Chem. Int. Ed. 56, 12576u201312580 (2017)._ _ _ Irie, M. Diarylethenes for memories and switches. Chem. Rev. 100, 1685u2013 1716 (2000)._ _ _ _ Shibata, K., Muto, K., Obatake, S. & Irie, M. Photocyclization/cycloreversion quantum yields of diarylethenes in single crystals. J. Phys. Chem. A 106, 209u2013214 (2002)._ _ _ Fulton, W. Young Tableaux: With Applications to Representation Theory and Geometry (Cambridge University Press, Cambridge, 1997)._ _ Komiyama, J., Honda, J. & Nakagawa, H. Optimal regret analysis of thompson sampling in stochastic multi-armed bandit problem with multiple plays. Int. Conf. Mach. Learn. 2015, 1152u20131161 (2015)._ Narisawa, N., Chauvet, N., Hasegawa, M. & Naruse, M. Arm order recognition in multi-armed bandit problem with laser chaos time series. Sci. Rep. 11, 4459 (2021)._ _ _ _ _ _ Okada, N., Hasegawa, M., Chauvet, N., Li, A. & Naruse, M. Analysis on effectiveness of surrogate data-based laser chaos decision maker. Complexity 2021, 8877660 (2021)._ _ Okada, N. et al. Theory of acceleration of decision-making by correlated times sequences. Complexity 2022, 5205580 (2022)._ _ \"- - This work was supported in part by the CREST project (JPMJCR17N2) funded by the Japan Science and Technology Agency and Grants-in-Aid for Scientific Research (A) (JP20H00233) and Grant-in-Aid for Challenging Research (Exploratory) (JP21K18710) funded by the Japan Society for the Promotion of Science.- - Authors and Affiliations - \"University of Yamanashi, 4-3-11 Takeda, Kofu, Yamanashi, 400-8511, JapanKazuharu Uchiyama_&_Hirokazu HoriDepartment of Information Physics and Computing, Graduate School of Information Science and Technology, The University of Tokyo, 7-3-1 Bunkyo-ku, Tokyo, 113-8656, JapanSota Nakajima,_Hirotsugu Suzui,_Nicolas Chauvet,_ Ryoichi Horisaki_&_Makoto NaruseNagahama Institute of Bio-Science and Technology, 1266 Tamura, Nagahama, Shiga, 526-0829, JapanHayato SaigoRyukoku University, 1-5 Yokotani, Oe-cho, Seta, Otsu, Shiga, 520-2194, JapanKingo Uchida\" - \"AuthorsKazuharu UchiyamaYou can also search for this author in _Sota NakajimaYou can also search for this author in _Hirotsugu SuzuiYou can also search for this author in _Nicolas ChauvetYou can also search for this author in _Hayato SaigoYou can also search for this author in _Ryoichi HorisakiYou can also search for this author in _Kingo UchidaYou can also search for this author in _Makoto NaruseYou can also search for this author in _Hirokazu HoriYou can also search for this author in _\" - Contributions - Ka.U., M.N., and H.H. directed the project and designed the system architecture. Ka.U. and S.N. conducted physical modelling and numerical performance evaluations. Ka.U., H.S. Ki.U., and H.H. examined the materials and near-field optics fundamentals. Ki.U. prepared the photochromic materials. Ka.U., M.N., H.S., and H.H. examined the mathematical fundamentals. Ka.U., M.N., Ki.U, and H.H. wrote the manuscript. All authors discussed the results and reviewed the manuscript. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution 4.0 Unported License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution 4.0; ). Color indicates marker expression intensity\" - \"To test the suitability of opt-SNE for applications beyond flow and mass cytometry, we analyzed a 1.3u2009xD7u200910 cell single_cell RNA-seq dataset of mouse embryonic brain cells published by 10x Genomics. We used pre-calculated PCA projections included in the dataset to generate opt-SNE maps that we compared with 10x_Genomics standard t-SNE embedding (Fig._). 10x_Genomics used EEu2009 =u20091000/total 4000 iterations of standard t-SNE while we used opt-SNE settings with u03B7u2009=u200997,959, EEu2009=u200966/total 885 iterations (Suppl. Fig._). Non-immune single cell transcriptomics data are more difficult to interpret with ground-truth classes since much fewer scRNA-seq markers can be easily interpreted for population identification. Therefore, we utilized both single gene classification and classification through the Louvain clustering algorithm using the Scanpy Python package to annotate the data. While opt-SNE and t-SNE both capture the macro-structure illustrated by gene overlays, several Louvain clusters (3, 18, 19, 25, 28) appear partially or completely overlapped by other clusters in standard t-SNE but not in the opt-SNE embedding. Therefore, opt-SNE allowed equivalent or superior resolution of single cell transcriptomics data as with standard t-SNE but with ~5xD7 smaller iteration time (885 iterations of opt-SNE vs. 4000 iterations of standard t-SNE).\" - \"HSNE (hierarchical SNE) is a t-SNE adaptation that was recently reported to facilitate analysis of large datasets by constructing a hierarchy of embeddings that can be explored from the overview of landmark populations up to a single_cell level or resolution. To directly compare HSNE and opt-SNE, we applied opt-SNE to the 5.2 million point dataset that was reported in HSNE analysis of mass cytometry data, and compared opt-SNE visualization to the full resolution level of HSNE embedding (Fig._, Suppl. Fig._). In the CD4 subset, opt-SNE visualization revealed two groups of CD4CD28 cells likely representing terminally differentiated memory CD4 T cells with different levels of CCR7 expression that may reflect distinct differentiation states of the two populations. While HSNE allowed identification of CD4CD28CCR7 cells, it was unable to visualize the_CD4CD28CCR7_cells as a single cluster and projected these cells sparsely in the CD4 island (Fig._ , left). Also, both algorithms projected heterogenous expression of CD56 in CD4CD28CCR7 cells, but HSNE embedding did not resolve separate CD56 and CD56 clusters within CD4CD28CCR7 cells, loosely mapping them to the poles of the single round cluster (Fig._, right). However, opt-SNE embedding visualized both the CD56+ and CD56u2212 clusters and the_disparate CD45RA expression within each cluster, revealing distinct phenotypes for the control and diseased subject groups (Fig._). Also, the_opt-SNE maps revealed that the CD56CD45RA cells in the cluster originate from several subjects with Crohns disease (Fig._). In summary, these results confirm that opt-SNE embedding provides superior visualization quality for complex cytometry data.\"- - \"Visual exploration of data drives hypothesis formation and/or serendipitous discoveries; therefore, t-SNE is an extremely valuable tool for data comprehension. It is often used to facilitate data perception when hypothesis generation is automated by robust computational methods. Comparison of t-SNE embeddings from multiple experimental conditions, timepoints, or subjects is invaluable to visualize sample-to-sample differences including disease hallmarks and longitudinal observations. t-SNE is also valuable for quality assessment of data, when abnormal clustering can_be traced back to sample preparation, data acquisition, and preprocessing artifacts. Therefore, batch embedding of multiple experimental points is essential for sample comparison and can only be enabled when t-SNE accommodates large datasets.\" - t-SNE was first introduced in cytometry research as a tool to visualize CyTOF data, since fluorescence-based high-parameter datasets were less common at that time. With recent advances in instrumentation and reagent availability, flow cytometry datasets with >20 parameters are quickly becoming prevalent and even standard in the field, yet the proper data assessment tools are lacking for general use. DNA-barcoded antibodies have been recently utilized to allow simultaneous protein-epitope and transcriptome measurements in single cells thus expanding the repertoire of traditional cytometry methods that could employ t-SNE as a staple method of data visualization and presentation. - \"One approach to large dataset t-SNE embedding is to model visualization with a subset of datapoints curated or randomly selected (as implemented in cytometry data analysis platforms such as FCS Express and Gemstone) from the nearest neighbor graph. Such techniques may fail to project extremely rare datapoints, as demonstrated by_the CD4CD28CCR7cells that were identified via use of opt-SNE, but not HSNE, in the van Unen et al dataset (Fig._). This T cell subset was significantly less abundant in subjects with severe inflammatory conditions including Crohn s disease as compared to controls (Suppl. Fig._). However, the cells expressing the_CD4CD28CCR7 phenotype, while also low in frequency in most PBMC samples (<1% of all CD4u2009cells), was likely successfully clustered by HSNE because two of the 11 Crohn disease subjects in the dataset showed an unusually high frequency of this population (40.7% and 31.0% of all CD4 cells), allowing it to be well represented in the kNN graph of the full dataset.\" - \"Several attempts to successfully apply t-SNE-like methods to massive datasets have been recently reported including aforenoted HSNE, LargeVis, and net-SNE. When applied to large datasets, these methods often require_and/or_benefit from considerable computational resources; for instance, the LargeVis study was performed on a 512u2009GB RAM, 32 core station. However, it is ideal for such data analysis to be performed successfully with the computational resources found in most laboratories. We benchmarked our opt-SNE modification of multicore t-SNE C++ implementation on a quad-core, 16GB RAM workstation although even further improvement in computation speed would occur if more cores were available. FIt-SNE, a recently published alternative to Barnes-Hut t-SNE that uses fast Fourier transform for much faster computation of repulsive forces approximation, renders opt-SNE even more feasible on personal computers. Notably, we have not observed differences in quality between embeddings generated with fast Fourier transform vs. Barnes-Hut approximations when opt-SNE parameter adjustments were used to compute the visualization (Suppl. Fig._). Therefore, we expect opt-SNE to be applicable for existing or future adaptations of t-SNE even if alternative methods of computation are utilized provided that they retain the core principles of t-SNE embedding. A promising approach that may be integrated with opt-SNE is the smart EE adjustment implemented in A-tSNE (approximated t-SNE) algorithm where EE is removed gradually and on a per-point basis. Cytosplore, a novel software platform that includes HSNE and A-tSNE, allows the analyst to interactively initiate the local refinement of the map, resulting_in a significant improvement in computation time.\" - \"Often compared to t-SNE, Uniform Manifold Approximation and Projection (UMAP) is a recently developed novel technique for dimension reduction that can be successfully applied to cytometry datasets. As has been the case for other groups, our comparison of t-SNE and opt-SNE to UMAP generated mixed results. Specifically, we found that UMAP parameter selection had dramatic effects on UMAP computation performance and embedding quality assessed by both human perception and 1NN classifier accuracy (Supp. Fig._). When benchmarked for resource use in the Omiq cloud, UMAP consumed up to 128u2009GB of RAM when embedding 5.2 million cell van Unen dataset, while opt-SNE and FIt-SNE peaked at 16u2009GB RAM consumption; the UMAP embedding took a comparable amount of time to complete as opt-SNE. With careful parameter selection, UMAP accuracy was approaching, but not surpassing, opt-SNE accuracy (Supp. Fig._).\" - \"Similar to other types of biological data, the structure of cytometry data is difficult to project due to its mixed nature, often comprised of cluster-like, manifold-like and/or hierarchical components. In this paper we propose multiple techniques that are essential for superior quality t-SNE data projection and are all germane to the fine-tuning of the early exaggeration phase of t-SNE embedding. EE facilitates cluster formation on a 2D plane and serves as a necessary compromise that allows clusters to escape the crowding effect. We designed an efficient measure to ensure that the cluster-like global structure of the data is fully revealed during the EE phase by monitoring the KLD output of the embedding in real time. As indicated by the lower KLD values of opt-SNE embedding where EE was limited to fewer iterations (Fig._), prolonged amplification of the attractive forces that drive tight cluster formation in EE may be detrimental for the manifold-like local data structure represented by signal distributions that are continuous with background signal. These u2018continuum expression molecules include proteins used to define classic immune cell subsets as well as markers indicating disease phenotypes, cell activation, and/or exhaustion state. Conversely, the non-exaggerated phase of t-SNE allows local data structures to be revealed. Therefore, cytometry data analysis would be missing valuable information if we reduced t-SNE applicability to finding only well separated clusters, especially since other techniques would perform this task better and faster. However, some workflows call for t-SNE pre-processing to facilitate extraction of cluster features from multidimensional data. In those cases, it may be helpful to adapt the opt-SNE toolkit to terminate the embedding calculation immediately at the EE stop iteration and re-assess the high-dimensional structure within each cluster. Alternatively, a u2018late exaggeration approach can be cautiously applied to create very tight clusters, although in our experience this approach was only marginally beneficial for global structure representation while detrimental for local structure (Suppl. Fig._). Interestingly, UMAP embeddings also presented a challenge to perceive both cluster data structure and continuum marker expression data structure within the same embedding, as_increasing the number of neighbors and decreasing the minimum distance parameters seemed to benefit the former but penalize the latter task (Suppl. Fig._ and). These results suggest that UMAP is highly promising but not yet_readily available to replace t-SNE as the_state-of-the-art method for cytometry data visualization.\" - It is advisable to note that certain data structures, such as cluster hierarchy, cannot be revealed with t-SNE. t-SNE accessibility in cytometry analysis software lead to its not infrequent misuse with cytometry data, evident when the cluster-like structure is not prominent in the map. Therefore, the features identified from t-SNE embedding in its current form should be verified with alternative methods when possible for confirmational purposes. Im et al also suggest that if a continuous manifold structure exists in the data, large perplexity values may cause artificial breaks (overclustering). The perplexity values commonly used in cytometry analysis are on the lower end of the suggested range for efficient clustering, as it is often advised to scale the number of nearest neighbors to the average cluster size however, if computationally feasible, higher perplexity values might facilitate feature preservation for markers whose expression is not bimodally distributed. To preserve global geometry, Kobak et al propose to perform the embedding in two steps and use the kNN-based extrapolation of a high-perplexity t-SNE of a subset of the large dataset as initialization to run the t-SNE of the whole dataset. - In summary, opt-SNE is a powerful optimization toolkit that subverts major limitations of t-SNE for use with cytometric datasets and thus enables novel data-driven findings in single cell research.- - Datasets - \"All datasets used in the study are summarized in Table_.\" - Data pre-processing - Singlet events from several data recordings were digitally concatenated and a randomly subsampled file of 1,000,000 mass or flow cytometry events was created and used for analyses of the mass41parameter and flow18parameter datasets. All observations from 27 recordings of flow cytometry data were concatenated to generate the flow20M dataset. All observations from 104 recordings of mass cytometry data were concatenated to generate the van Unen et al dataset. - All flow cytometry data were compensated with acquisition-defined compensation matrices. Prior to t-SNE analysis, all cytometry data were transformed using asinh (with cofactor 5 for mass cytometry data and with cofactor 150 for flow14parameter data) or biexponential (flow20M) transformation. Light scatter parameters were log-transformed. - The standard t-SNE configuration for cytometry applications - t-SNE computes low-dimensional coordinates of high-dimensional data resulting in similar and dissimilar data points in the raw data space placed proximally and at a distance, respectively, in the dimensionally reduced map. This map placement is achieved via t-SNE modeling the probabilities as a Gaussian distribution around each data point in the high-dimensional space and modeling the target distribution of pairwise similarities in the lower-dimensional space using Cauchy distribution (Student t-distribution with 1 degree of freedom). Then, the Kullback-Leibler Divergence (KLD) between the distributions is iteratively minimized via gradient descent. The gradient computation is essentially an N-body simulation problem with attractive forces (approximated to nearest neighbors using vantage-point trees) pulling similar points together and repulsive forces (approximated at each iteration using the Barnes-Hut algorithm) pushing dissimilar points apart. - \"An important part of t-SNE gradient descent computation is the early exaggeration (EE) that was proposed by van der Maaten and Hinton to battle the so called overcrowding artifact of embedding. With EE, all probabilities modeling distances in high-dimensional space are multiplied by a factor (early exaggeration factor, EEF, or u03B1) for the duration of the EE phase (typically 250, or 25% of the total number of iterations). EE coerces data to form tight and widely separated clusters in the map and is considered to enable the map to find a better global structure.\" - Multiple software platforms incorporate the BH-tSNE algorithm specifically for analysis of cytometry data, including commonly used cytometry analysis desktop packages (FlowJo, FCS Express), and the cloud-based analysis platform Cytobank. Also, implementations of t-SNE are available as open-source packages in popular programming languages such as R (rtsne) and Python (sci-kit learn). Most of implementations wrap or re-write original C++ code of Barnes-Hut t-SNE and produce comparable analysis results upon direct comparison. Here, we customized the standard t-SNE C++ code to implement the parameter adjustments described in this work and published this customization as an open source solution to enable the research community to use this optimized t-SNE algorithm. Also, the equivalent adjustments of t-SNE have been made available as a cloud application in Omiq and integrated into FlowJo and SeqGeq programs. - Data analysis - \"A desktop C++ Barnes-Hut implementation of t-SNE for Mac OS was used for t-SNE analyses with original implementation adapted to allow user input for the early exaggeration stop iteration, perplexity, total number of iterations, early exaggeration factor value, and learning rate value. A multicore modification of Barnes-Hut t-SNE was used as a source for building the opt-SNE package with user-accessible parameters similar to listed above. The Kullback-Leibler Divergence (KLD) value and t-SNE coordinates were reported during each iteration or as frequently as requested. All datasets were embedded in 2D space. Visually comparable t-SNE maps were generated with the same random seed values used when permutations for specific parameters were tested and compared; all experiments were repeated with several values of random seed. Benchmarking data were generated on an iMac personal station with Intel Core i7 quad-core i7 processor and 16u2009GB of RAM. For cross-validation, we utilized cloud-based Cytobank, cloud-based Omiq, FlowJo V10.3u201310.5 and FlowJo V9.9.6. R flowcore package, Cytobank and FlowJo were used to generate FCS files and graphical outputs from tabular data. Omiq platform was used for FlowSOM, FIt-SNE (Fast Fourier Transform-accelerated Interpolation-based t-SNE) and UMAP (Uniform Manifold Approximation and Projection) analyses. Also, the FIt-SNE plugin for FlowJo and FIt-SNE prototype in FlowJo were used to ensure FIt-SNE compatibility with opt-SNE. Logs of t-SNE runs were batch-processed with VBA scripts and analyzed with GraphPad Prism. Expert-guided (manual) analysis of cell populations was performed in FlowJo 10.3u201310.5 as described previously for specific datasets (Suppl. Figs._u2013) and used for map annotations as cluster classes.\" - We used SeqGeq 1.3 for scRNAseq analysis and leveraged PCA projections provided in the 10x Genomics dataset to calculate the t-SNE embedding and annotated it using marker genes for major cell types. Louvain cluster classification was adopted from the SCANPY data analysis study. - \"The quality of embedding was assessed by a 1-nearest neighbor (1NN) classifier based on previous reports on t-SNE accuracy evaluation. The classifier was written in R (version 3.5.0). For each invocation, the classifier was provided a complete embedding and an accompanying array of assigned class values for each observation in the embedding. Ten thousand cells were sampled from the embedding as a training set using uniform pseudorandom sampling with the u201Csampleu201D function from base R. Next, a set of 50,000 cells disjoint to the training set was sampled from the embedding as the test set. Each cell in the test set was then mapped to its single nearest neighbor within the test set and assigned the class of that neighbor using the u201Cknnu201D function of the u201CFNNu201D R package (version 1.1.3, available on CRAN). The number of correct class assignments in the test set was calculated on a total basis and also on a per-class basis using the supplied class values. This process was repeated five times for the provided embedding using different seeds for the sampling step. Replacement was allowed for pseudorandom sampling between the five internal repeat runs. The results were then presented as the average of these five repetitions. The seeds used for the five repetitions were kept consistent such that classification results between different embeddings of the same dataset tested the same cells.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - Open source multicore t-SNE C++ implementation and usage instructions are available at Cloud version of opt-SNE is available at . opt-SNE is natively supported within the FlowJo data analysis software version 10.6 and later, and the SeqGeq version 1.5 or later, both downloadable from . The code of modified van der Maaten BH-tSNE C++ implementation with user-accessible parameters is available upon request. - \\' \\'- - \\' \\' - Flow18parameters and mass41parameters datasets are available via FigShare at . Van Unen et al. dataset is available at . 10XGenomics 1.3M scRNA-seq dataset is available at . - \\' \\'- - \"van der Maaten, L. & Hinton, G. Visualizing data using t-SNE. J. Mach. Learn. Res. 9, 85 (2008)._ _ van der Maaten, L. Accelerating t-SNE using tree-based algorithms. J. Mach. Learn. Res. 15, 3221u20133245 (2014)._ _ _ Amir el, A. D. et al. viSNE enables visualization of high dimensional single-cell data and reveals phenotypic heterogeneity of leukemia. Nat. Biotechnol. 31, 545u2013552 (2013)._ _ Wong, M. T. et al. Mapping the diversity of follicular helper t cells in human blood and tonsils using high-dimensional mass cytometry analysis. Cell Rep. 11, 1822u20131833 (2015)._ _ _ Becher, B. et al. High-dimensional analysis of the murine myeloid cell system. Nat. Immunol. 15, 1181u20131189 (2014)._ _ _ Chen, H. et al. Cytofkit: a bioconductor package for an integrated mass cytometry data analysis pipeline. PLoS Comput. Biol. 12, e1005112 (2016)._ _ van Unen, V. et al. Visual analysis of mass cytometry data by hierarchical stochastic neighbour embedding reveals rare cell types. Nat. Commun. 8, 1740 (2017)._ _ _ Tang, J., Liu, J., Zhang, M. & Mei, Q. Proceedings of the 25th International Conference on World Wide Web 287u2013 297 (International World Wide Web Conferences Steering Committee, Montreal, Quebec, Canada, 2016).McInnes, L., Healy, J. & Melville, J. UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction. Preprint at arXiv:1802.03426 (2018).KratochvxED l, M. et al. SOM-based embedding improves efficiency of high-dimensional cytometry data analysis. Preprint at bioRxiv (2019).Van Gassen, S. et al. FlowSOM: using self-organizing maps for visualization and interpretation of cytometry data. Cytom. A 87, 636u2013645 (2015)._ _ Levine, J. H. et al. Data-driven phenotypic dissection of AML reveals progenitor-like cells that correlate with prognosis. Cell 162, 184u2013197 (2015)._ _ _ Donnenberg, A. D. & Donnenberg, V. S. Rare-event analysis in flow cytometry. Clin. Lab. Med. 27, 627u2013652 (2007)._ _ DiGiuseppe, J. A., Tadmor, M. D. & Peer, D. Detection of minimal residual disease in B lymphoblastic leukemia using viSNE. Cytom. Part B Clin. Cytom. 88, 294u2013304 (2015)._ _ _ Lin, L. et al. Identification and visualization of multidimensional antigen-specific T-cell populations in polychromatic cytometry data. Cytom. A 87, 675u2013682 (2015)._ _ Hirakawa, M. et al. Low-dose IL-2 selectively activates subsets of CD4(+) Tregs and NK cells. JCI Insight 1, e89278 (2016)._ _ Pezzotti, N., HxF6llt, T., Lelieveldt, B., Eisemann, E. & Vilanova, A. Hierarchical Stochastic Neighbor Embedding. Computer Graph. Forum 35, 21u2013 30 (2016)._ _ Linderman, G. C., Rachh, M., Hoskins, J. G., Steinerberger, S. & Kluger, Y. Fast interpolation-based t-SNE for improved visualization of single-cell RNA-seq data. Nat. Methods 16, 243u2013245 (2019)._ _ _ Wattenberg, M. VixE9gas, F., Johnson, I. How to use t-SNE effectively. Distill_1, e2 (2016).Weber, L. M. & Robinson, M. D. Comparison of clustering methods for high-dimensional single-cell flow and mass cytometry data. Cytom. Part A 89, 1084u20131096 (2016)._ _ _ Kobak, D. & Berens, P. The art of using t-SNE for single-cell transcriptomics. Nat. Commun 10 (2019).Jacobs, R. A. Increased rates of convergence through learning rate adaptation. Neural Netw. 1, 295u2013307 (1988)._ _ Linderman, G. C. & Steinerberger, S. Clustering with t-SNE, provably. Preprint at arXiv: 1706.02582 (2017).Ulyanov, D. Github. https://github.com/DmitryUlyanov/ (2016).Belkina, A. C. & Snyder-Cappione, J. E. OMIP-037: 16-color panel to measure inhibitory receptor signatures from multiple human immune cell subsets. Cytom. A 91, 175u2013179 (2017)._ _ Wolf, F. A., Angerer, P. & Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. Genome Biol. 19, 15 (2018)._ _ Mou, D., Espinosa, J., Lo, D. J. & Kirk, A. D. CD28 negative T cells: is their loss our gain? Am. J. Transplant. 14, 2460u20132466 (2014)._ _ _ Butler, A., Hoffman, P., Smibert, P., Papalexi, E. & Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Nat. Biotechnol. 36, 411 (2018)._ _ _ Wogsland, C. E. et al. Mass cytometry of follicular lymphoma tumors reveals intrinsic heterogeneity in proteins including HLA-DR and a deficit in nonmalignant plasmablast and germinal center B-Cell populations. Cytom. Part B Clin. Cytom. 92, 79u2013 87 (2017)._ _ _ Mazza, E. M. C. et al. Background fluorescence and spreading error are major contributors of variability in high-dimensional flow cytometry data visualization by t-distributed stochastic neighboring embedding. Cytom. Part A 93, 785u2013792 (2018)._ _ _ Staser, K. W., Eades, W., Choi, J., Karpova, D. & DiPersio, J. F. OMIP-042: 21-color flow cytometry to comprehensively immunophenotype major lymphocyte and myeloid subsets in human peripheral blood. Cytom. A 93, 186u2013189 (2018)._ _ Mair, F. & Prlic, M. OMIP-044: 28-color immunophenotyping of the human dendritic cell compartment. Cytom. A 93, 402u2013405 (2018)._ _ Nettey, L., Giles, A. J. & Chattopadhyay, P. K. OMIP-050: A 28-color/30-parameter fluorescence flow cytometry panel to enumerate and characterize cells expressing a wide array of immune checkpoint molecules. Cytometry A 93, 1094u20131096 (2018).Stoeckius, M. et al. Simultaneous epitope and transcriptome measurement in single cells. Nat. Methods 14, 865 (2017)._ _ _ Pezzotti, N. et al. Approximated and user steerable tSNE for progressive visual analytics. IEEE Trans. Vis. Comput. Graph 23, 1739u2013 1752 (2017)._ _ Cho, H., Berger, B. & Peng, J. Generalizable and scalable visualization of single-cell data using neural networks. Cell Syst. 7, 185u2013191 e184 (2018)._ _ _ Chan, D. M., Rao, R., Huang, F. & Canny, J. F._GPU accelerated t-distributed stochastic neighbor embedding. Journal of Parallel and Distributed Computing, 131, 1u2013 13 (2019).Linderman, G. C., Rachh, M., Hoskins, J. G., Steinerberger, S. & Kluger, Y. Efficient algorithms for t-distributed stochastic neighborhood embedding. Preprint at (2017).Finn, W. G., Carter, K. M., Raich, R., Stoolman, L. M. & Hero, A. O. Analysis of clinical flow cytometric immunophenotyping data by clustering on statistical manifolds: Treating flow cytometry data as high-dimensional objects. Cytom. Part B Clin. Cytom. 76B, 1u20137 (2008)._ Arora, S., Hu, W. & Kothari, P. K. in Proceedings of the 31st Conference On Learning Theory, Vol. 75. (eds. SxE9bastien, B., Vianney, P. & Philippe, R.) 1455u20131462 (PMLR, Proceedings of Machine Learning Research, 2018).Chattopadhyay, P. K. & Roederer, M. Cytometry: todays technology and tomorrows horizons. Methods 57, 251u2013258 (2012)._ _ _ Shekhar, K., Brodin, P., Davis, M. M. & Chakraborty, A. K. Automatic classification of cellular expression by nonlinear stochastic embedding (ACCENSE). Proc. Natl Acad. Sci. 111, 202 (2014)._ _ _ _ Amid, E. & Warmuth, M. K. A more globally accurate dimensionality reduction method using triplets. Preprint at (2018).Im, D. J., Verma, N. & Branson, K. Stochastic neighbor embedding under f-divergences. Preprint at (2018).Cao, Y. & Wang, L. Automatic Selection of t-SNE Perplexity. Preprint at (2017).Bendall, S. C. et al. Single-cell mass cytometry of differential immune and drug responses across a human hematopoietic continuum. Science 332, 687u2013696 (2011)._ _ _ _ Belkina, A. C. et al. Multivariate computational analysis of gamma delta t cell inhibitory receptor signatures reveals the divergence of healthy and ART-suppressed HIV+ aging. Front. Immunol. 9, 2783 (2018)._ _ _ Chen, T. J. & Kotecha, N. Cytobank: providing an analytics platform for community cytometry data analysis and collaboration. Curr. Top. Microbiol. Immunol. 377, 127u2013157 (2014)._ _ _ \"- - The authors would like to thank Yvan Saeys, Gary Kazantsev, Jonathan Irish, Dmitry Kobak, Allison Irvine and Katherine Drake for helpful discussions, and Geoff Kraker for technical assistance. We thank Sean Bendall from Stanford University and Vincent van Unen from Leiden University Medical Center for sharing their data, and Brian Tilton and Riley Pihl from BUSM Flow Cytometry Core Facility for assistance with data collection at BUSM. Anna C. Belkina is an ISAC (International Society for Advancement of Cytometry) SRL Emerging Leader and thanks the ISAC organization and members for continuous support and encouragement. Josef Spidlen is an ISAC Marylou Ingram Scholar.- - Authors and Affiliations - \"Department of Pathology and Laboratory Medicine, Boston University School of Medicine, Boston, MA, 02118, USAAnna C. BelkinaFlow Cytometry Core Facility, Boston University School of Medicine, Boston, MA, 02118, USAAnna C. Belkina_ &_Jennifer E. Snyder-CappioneOmiq, Inc, Santa Clara, CA, 95050, USAChristopher O. CiccolellaDepartment of Mathematics, Kansas State University, Manhattan, KS, 66506, USARina AnnoBD Life Sciencesu2013FlowJo, Ashland, OR, 97520, USARichard Halpert_&_Josef SpidlenDepartment of Microbiology, Boston University School of Medicine, Boston, MA, 02118, USAJennifer E. Snyder-Cappione\" - \"AuthorsAnna C. BelkinaYou can also search for this author in _Christopher O. CiccolellaYou can also search for this author in _Rina AnnoYou can also search for this author in _Richard HalpertYou can also search for this author in _Josef SpidlenYou can also search for this author in _Jennifer E. Snyder-CappioneYou can also search for this author in _\" - Contributions - A.C.B. has conceived the study. A.C.B. and C.O.C. developed the opt-SNE method. C.O.C. developed implementation in C++; and R.H. and J.S. developed implementation in FlowJo and SeqGeq. A.C.B. has analyzed flow and mass cytometry datasets. R.H. and J.S. have analyzed the scRNA-seq dataset. R.A. and J.S.C. provided conceptual input. A.C.B. wrote the paper. All authors discussed the results and commented on the paper. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - C.O.C. is a founder of Omiq, Inc. R.H. and J.S. are employees of Beckton Dickinson (BD); FlowJo is a subsidiary of BD. The remaining authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks El-ad David Amir and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " \"Creative Commons Attribution 4.0International License. The images or other third party material in this article areincluded in the article''s Creative Commons license, unless indicatedotherwise in the credit line; if the material is not included under the CreativeCommons license, users will need to obtain permission from the license holder inorder to reproduce the material. \",\n",
       " 'Creative Commons Attribution 4.0International License. The images or other third party material in this article areincluded in the articles Creative Commons license, unless indicatedotherwise in the credit line; if the material is not included under the CreativeCommons license, users will need to obtain permission from the license holder toreproduce the material. ',\n",
       " 'Creative Commons Attribution 4.0International License. The images or other third party material in this articleare included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution CC BY 3.0 IGO, Washington, DC, 2017)._ _ Eyre, D. W. et al. A Canida auris outbreak and its control in an intensive care setting. New Engl. J. Med. 379, 1322u20131331 (2018)._ _ _ Lockhart, S. R. et al. Simultaneous emergence of multidrug-resistant Candida auris on 3 continents confirmed by whole-genome sequencing and epidemiological analyses. Clin. Infect. Dis. 64, 134u2013140 (2017)._ _ _ _ Centers for Disease Control and Prevention. Antibiotic Resistance Threats in the United States (U.S. Department of Health and Human Services, 2019).Satoh, K. et al. Candida auris sp. nov., a novel ascomycetous yeast isolated from the external ear canal of an inpatient in a Japanese hospital. Microbiol. Immunol. 53, 41u201344 (2009)._ _ _ _ Rhodes, J. & Fisher, M. C. Global epidemiology of emerging Candida auris. Curr. Opin. Microbiol. 52, 84u201389 (2019)._ _ _ Chow, N. et al. Tracing the evolutionary history and global expansion of Candida auris using population genomic analyses. mBio 11, e03364u201319 (2020)._ _ _ _ _ Schelenz, S. et al. First hospital outbreak of the globally emerging Candida auris in a European hospital. Antimicrob. Resist. Infect. Control 5, 35 (2016).Ruiz-GaitxE1 n, A. et al. An outbreak due to Candida auris with prolonged colonisation and candidaemia in a tertiary care European hospital. Mycoses 61, 498u2013505 (2018)._ _ _ _ Robbins, N., Caplan, T. & Cowen, L. E. Molecular evolution of antifungal drug resistance. Annu. Rev. Microbiol. 71, 753u2013775 (2017)._ _ _ _ Revie, N. M., Iyer, K. R., Robbins, N. & Cowen, L. E. Antifungal drug resistance: evolution, mechanisms and impact. Curr. Opin. Microbiol. 45, 70u201376 (2018)._ _ _ _ _ Martel, C. M. et al. Identification and characterization of four azole-resistant ERG3 mutants of Candida albicans. Antimicrob. Agents Chemother. 54, 4527u20134533 (2010)._ _ _ _ _ Hoepfner, D. et al. An integrated approach for identification and target validation of antifungal compounds active against Erg11p. Antimicrob. Agents Chemother. 56, 4233u20134240 (2012)._ _ _ _ _ Flowers, S. A., ColxF3n, B., Whaley, S. G., Schuler, M. A. & David Rogers, P. Contribution of clinically derived mutations in ERG11 to azole resistance in Candida albicans. Antimicrob. Agents Chemother. 59, 450u2013460 (2015)._ _ _ _ Perea, S. et al. Prevalence of molecular mechanisms of resistance to azole antifungal agents in Candida albicans strains displaying high-level fluconazole resistance isolated from human immunodeficiency virus-infected patients. Antimicrob. Agents Chemother. 45, 2676u20132684 (2001)._ _ _ _ _ _ Healey, K. R. et al. Limited ERG11 mutations identified in isolates of Candida auris directly contribute to reduced azole susceptibility. Antimicrob. Agents Chemother. 62, e01427-18 (2018)._ _ _ _ Wasi, M. et al. ABC transporter genes show upregulated expression in drug-resistant clinical isolates of Candida auris: a genome-wide characterization of ATP-binding cassette (ABC) transporter genes. Front. Microbiol. 10, 1445 (2019)._ _ _ _ MuxF1oz, J. F. et al. Genomic insights into multidrug-resistance, mating and virulence in Candida auris and related emerging species. Nat. Commun. 9, 5346 (2018)._ _ _ _ _ _ Kean, R. et al. Transcriptome assembly and profiling of Candida auris reveals novel insights into biofilm-mediated resistance. mSphere 3, 1u201314 (2018)._ _ Rybak, J. M. et al. Abrogation of triazole resistance upon deletion of CDR1 in a clinical isolate of Candida auris. Antimicrob. Agents Chemother. 63, 1u20137 (2019)._ _ Rybak, J. M. et al. Mutations in_TAC1B: a novel genetic determinant of clinical fluconazole resistance in_Candida auris._mBio_11, e00365u201320 (2020)._ _ _ _ Eldesouky, H. E., Li, X., Abutaleb, N. S., Mohammad, H. & Seleem, M. N. Synergistic interactions of sulfamethoxazole and azole antifungal drugs against emerging multidrug-resistant Candida auris. Int. J. Antimicrob. Agents 52, 754u2013761 (2018)._ _ _ _ OBrien, B., Chaturvedi, S. & Chaturvedi, V. In vitro evaluation of antifungal drug combinations against multidrug-resistant Candida auris isolates from New York outbreak. Antimicrob. Agents Chemother. 64, e02195u201319 (2020)._ _ _ Nishikawa, J. L. et al. Inhibiting fungal multidrug resistance by disrupting an activator-Mediator interaction. Nature 530, 485u2013489 (2016)._ _ _ _ _ _ Brown, G. D. et al. Hidden killers: human fungal infections. Sci. Transl. Med. 4, 165rv13 (2012)._ _ _ _ Brown, L. E. et al. Discovery of new antimalarial chemotypes through chemical methodology and library development. Proc. Natl Acad. Sci. USA 108, 6775u20136780 (2011)._ _ _ _ _ _ Roemer, T. et al. Confronting the challenges of natural product-based antifungal discovery. Chem. Biol. 18, 148u2013164 (2011)._ _ _ _ Perfect, J. R. The antifungal pipeline: a reality check. Nat. Rev. Drug Discov. 16, 603u2013616 (2017)._ _ _ _ _ Iyer, K. R. et al. Translation inhibition by rocaglates activates a species- specific cell death program in the emerging fungal pathogen Candida auris. mBio 11, e03329u201319 (2020)._ _ _ _ _ Odds, F. C. Synergy, antagonism, and what the chequerboard puts between them. J. Antimicrob. Chemother. 52, 1 (2003)._ _ _ _ Spitzer, M., Robbins, N. & Wright, G. D. Combinatorial strategies for combating invasive fungal infections. Virulence 8, 169u2013 185 (2016)._ _ _ _ _ Iskar, M. et al. Drug-induced regulation of target expression. PLoS Comput. Biol. 6, e1000925 (2010)._ _ _ _ _ Henry, K. W., Nickels, J. T. & Edlind, T. D. Upregulation of ERG genes in Candida species by azoles and other sterol biosynthesis inhibitors. Antimicrob. Agents Chemother. 44, 2693u20132700 (2000)._ _ _ _ _ Ivnitski-Steele, I. et al. Identification of Nile red as a fluorescent substrate of the Candida albicans ATP-binding cassette transporters Cdr1p and Cdr2p and the major facilitator superfamily transporter Mdr1p. Anal. Biochem. 394, 87u201391 (2009)._ _ _ _ _ Kim, S. H. et al. Genetic analysis of Candida auris implicates Hsp90 in morphogenesis and azole tolerance and Cdr1 in azole resistance. mBio 10, e02529u201318 (2019)._ _ _ _ Prasad, R., Sharma, M. & Rawal, M. K. Functionally relevant residues of Cdr1p: a multidrug ABC transporter of human pathogenic Candida albicans. J. Amino Acids 2011, 1u201312 (2011)._ _ _ Copping, V. M. S. et al. Exposure of Candida albicans to antifungal agents affects expression of SAP2 and SAP9 secreted proteinase genes. J. Antimicrob. Chemother. 55, 645u2013 654 (2005)._ _ _ _ Mann, P. A. et al. Chemical genomics-based antifungal drug discovery: targeting glycosylphosphatidylinositol (GPI) precursor biosynthesis. ACS Infect. Dis. 1, 59u201372 (2015)._ _ _ _ Holmes, A. R. et al. Targeting efflux pumps to overcome antifungal drug resistance. Future Med. Chem. 8, 1485u20131501 (2016)._ _ _ _ _ White, T. C. Increased mRNA levels of ERG16, CDR, and MDR1 correlate with increases in azole resistance in Candida albicans isolates from a patient infected with human immunodeficiency virus. Antimicrob. Agents Chemother. 41, 1482u20131487 (1997)._ _ _ _ _ MorschhxE4user, J. et al. The transcription factor Mrr1p controls expression of the MDR1 efflux pump and mediates multidrug resistance in Candida albicans. PLoS Pathog. 3, e164 (2007)._ _ _ _ _ White, T. C. The presence of an R467K amino acid substitution and loss of allelic variation correlate with an azole-resistant lanosterol 14u03B1 demethylase in Candida albicans. Antimicrob. Agents Chemother. 41, 1488u20131494 (1997)._ _ _ _ _ White, T. C., Pfaller, M. A., Rinaldi, M., Smith, J. & Redding, S. Stable azole drug resistance associated with a substrain of Candida albicans from an HIVu2010infected patient. Oral Dis. 3, S102 (1997)._ _ _ Hill, J. A., OMeara, T. R. & Cowen, L. E. Fitness trade-offs associated with the evolution of resistance to antifungal drug combinations. Cell Rep. 10, 809u2013819 (2015)._ _ _ _ Popp, C. et al. Competitive fitness of fluconazole- resistant clinical Candida albicans strains. Antimicrob. Agents Chemother. 61, e00584u201317 (2017)._ _ _ _ _ Coste, A. T., Crittin, J., Bauser, C., Rohde, B. & Sanglard, D. Functional analysis of cis-and trans-acting elements of the Candida albicans CDR2 promoter with a novel promoter reporter system. Eukaryot. Cell 8, 1250u20131267 (2009)._ _ _ _ _ Caplan, T. et al. Overcoming fungal echinocandin resistance through inhibition of the non-essential stress kinase Yck2. Cell Chem. Biol. 27, 269u2013 282.e5 (2020)._ _ _ _ _ Liston, S. et al. Antifungal activity of gepinacin-scaffold GPI anchor biosynthesis inhibitors with improved metabolic stability. Antimicrob. Agents Chemother. 64, e00899u2013 20 (2020)._ _ _ _ Lepak, A. J., Zhao, M., Berkow, E. L., Lockhart, S. R. & Andes, D. R. Pharmacodynamic optimization for treatment of invasive Candida auris infection. Antimicrob. Agents Chemother. 61, e00791u201317 (2017)._ _ _ _ Ferrari, S., Sanguinetti, M., Torelli, R., Posteraro, B. & Sanglard, D. Contribution of CgPDR1-regulated genes in enhanced virulence of azole-resistant Candida glabrata. PLoS ONE 6, e17589 (2011)._ _ _ _ _ _ MorschhxE4user, J. The genetic basis of fluconazole resistance development in Candida albicans. Biochim. Biophys. Acta 1587, 240u2013248 (2002)._ _ _ Sharma, A., Gupta, V. & Pathania, R. Efflux pump inhibitors for bacterial pathogens: from bench to bedside. Indian J. Med. Res. 149, 129 (2019)._ _ _ _ Shriram, V., Khare, T., Bhagwat, R., Shukla, R. & Kumar, V. Inhibiting bacterial drug efflux pumps via phyto-therapeutics to combat threatening antimicrobial resistance. Front. Microbiol. 9, 2990 (2018)._ _ _ _ Castaldi, M. P., Troast, D. M. & Porco, J. Jr. Stereoselective synthesis of spirocyclic oxindoles via prins cyclizations. Org. Lett. 11, 3362u20133365 (2009)._ _ _ _ _ Morrison, B. L. et al. Oxyphenisatin acetate (NSC 59687) triggers a cell starvation response leading to autophagy, mitochondrial dysfunction, and autocrine TNFu03B1-mediated apoptosis. Cancer Med. 2, 687u2013700 (2013)._ _ _ _ _ Denoyelle, S. et al. Synthesis and SAR study of novel 3,3-diphenyl-1,3-dihydroindol-2-one derivatives as potent eIF2xB7GTPxB7Met-tRNAiMet ternary complex inhibitors. Eur. J. Med. Chem. 69, 537u2013553 (2013)._ _ _ _ Zheng, X. et al. Targeting multidrug-resistant ovarian cancer through estrogen receptor a dependent ATP depletion caused by hyperactivation of the unfolded protein response. Oncotarget 9, 14741u201314753 (2018)._ _ _ Neel, D. A. et al. 3,3-Bisaryloxindoles as mineralocorticoid receptor antagonists. Bioorg. Med. Chem. Lett. 15, 2553u20132557 (2005)._ _ _ _ Andreani, A. et al. New isatin derivatives with antioxidant activity. Eur. J. Med. Chem. 45, 1374u20131378 (2010)._ _ _ _ _ Prasad, R., De Wergifosse, P., Goffeau, A. & Balzi, E. Molecular cloning and characterization of a novel gene of Candida albicans, CDR1, conferring multiple resistance to drugs and antifungals. Curr. Genet. 27, 320u2013329 (1995)._ _ _ _ Celaj, A. et al. Highly combinatorial genetic interaction analysis reveals a multi-drug transporter influence network. Cell Syst. 10, 25u201338.e10 (2020)._ _ _ _ Tsao, S., Rahkhoodaee, F. & Raymond, M. Relative contributions of the Candida albicans ABC transporters Cdr1p and Cdr2p to clinical azole resistance. Antimicrob. Agents Chemother. 53, 1344u2013 1352 (2009)._ _ _ _ _ Lomovskaya, O. & Bostian, K. A. Practical applications and feasibility of efflux pump inhibitors in the clinic - a vision for applied use. Biochem. Pharmacol. 71, 910u2013 918 (2006)._ _ _ _ Shekhar-Guturja, T. et al. Dual action antifungal small molecule modulates multidrug efflux and TOR signaling. Nat. Chem. Biol. 12, 867u2013875 (2016)._ _ _ _ _ Tong, Y. et al. Beauvericin counteracted multi-drug resistant Candida albicans by blocking ABC transporters. Synth. Syst. Biotechnol. 1, 158u2013168 (2016)._ _ _ _ Robey, R. W. et al. Revisiting the role of ABC transporters in multidrug-resistant cancer. Nat. Rev. Cancer 18, 452u2013464 (2018)._ _ _ _ _ Wright, G. D. Antibiotic adjuvants: rescuing antibiotics from resistance. Trends Microbiol. 24, 862u2013871 (2016)._ _ _ _ Min, K., Ichikawa, Y., Woolford, C. A. & Mitchell, A. P. Candida albicans gene deletion with a transient CRISPR-Cas9 system. mSphere 1, 1u20139 (2016)._ _ _ Liu, Z. & Myers, L. C. Mediator tail module is required for Tac1-activated CDR1 expression and azole resistance in Candida albicans. Antimicrob. Agents Chemother. 61, 1u201320 (2017)._ Homann, O. R., Dea, J., Noble, S. M. & Johnson, A. D. A phenotypic profile of the Candida albicans regulatory network. PLoS Genet. 5, e1000783 (2009).Xie, J., Singh-Babak, S. & Cowen, L. Minimum inhibitory concentration (MIC) assay for antifungal drugs. Bio-Protocol 2, e252 (2012)._ _ \"- - \"We thank Dr. Anuradha Chowdhary for the C. auris strain VPCI 673/P/12, the CDC and Dr. Philipp Dufresne for the C. auris strain B12037, Dr. T.C. White for the C. albicans clinical isolates, and Dr. Lawrence Myers for providing that plasmid constructs used for strain construction in C. albicans. We also wish to thank Dr. Norman Lee (Boston University) for assistance with high-resolution mass spectrometry analysis. Finally, we thank members of the Cowen lab, past and present, for their constant support and valuable discussions. K.R.I. is supported by uFEFFan Ontario Graduate Scholarship. L.E.C. is supported by the Canadian Institutes of Health Research Foundation Grant (FDN-154288); L.E.C. is a Canada Research Chair (Tier 1) in Microbial Genomics & Infectious Disease and co-Director of the CIFAR Fungal Kingdom: Threats & Opportunities program. Work at the BU-CMD is supported by U01 TR002625, R24 GM111615 and R35 GM118173. J.F.M. and C.A.C. are supported by NIAID award U19AI110818 to the Broad Institute; C.A.C. is a CIFAR fellow in the Fungal Kingdom Program. J.R.N. is supported by the Canadian Institutes of Health Research Open Operating Grant (MOP-133636). A.S.I. and J.E. are supported by a NIAID award R01AI141202. D.R.A is supported by an NIH Grant R01AI073289.\"- - Authors and Affiliations - \"Department of Molecular Genetics, University of Toronto, Toronto, ON, CanadaKali R. Iyer,_Zhongle Liu,_Sang Hu Kim,_Luke Whitesell,_Nicole Robbins_&_Leah E. CowenDepartment of Chemistry and Center for Molecular Discovery (BU-CMD), Boston University, Boston, MA, USAKaddy Camara,_Richard Trilles,_John A. Porco Jr._&_Lauren E. BrownClark+Elbing LLP, Boston, MA, USAKaddy CamaraDepartment of Biochemistry, University of Toronto, Toronto, ON, CanadaMartin Daniel-Ivad,_ Sheila M. Pimentel-Elardo_&_Justin R. NodwellDepartment of Medicine, University of Wisconsin School of Medicine and Public Health, Madison, WI, USAJen L. Fossen,_ Karen Marchillo_&_David R. AndesDepartment of Medical Microbiology and Immunology, University of Wisconsin, Madison, WI, USAJen L. Fossen,_Karen Marchillo_&_ David R. AndesDivision of Infectious Disease, The Lundquist Institute for Biomedical Innovation Los Angeles Biomedical Research Institute at Harbor-University of California, Los Angeles (UCLA) Medical Center, Torrance, CA, USAShakti Singh,_ Ashraf S. Ibrahim_&_John E. Edwards Jr.Infectious Disease and Microbiome Program, Broad Institute of MIT and Harvard, Cambridge, MA, USAJosxE9 F. MuxF1oz_ &_Christina A. CuomoDepartment of Biochemistry, University of Texas Southwestern Medical School, Dallas, TX, USANoelle S. WilliamsDavid Geffen School of Medicine, UCLA, Los Angeles, CA, USAAshraf S. Ibrahim_&_John E. Edwards Jr.\" - \"AuthorsKali R. IyerYou can also search for this author in _Kaddy CamaraYou can also search for this author in _Martin Daniel-IvadYou can also search for this author in _Richard TrillesYou can also search for this author in _Sheila M. Pimentel-ElardoYou can also search for this author in _Jen L. FossenYou can also search for this author in _Karen MarchilloYou can also search for this author in _Zhongle LiuYou can also search for this author in _Shakti SinghYou can also search for this author in _JosxE9 F. MuxF1ozYou can also search for this author in _Sang Hu KimYou can also search for this author in _John A. Porco Jr.You can also search for this author in _Christina A. CuomoYou can also search for this author in _Noelle S. WilliamsYou can also search for this author in _Ashraf S. IbrahimYou can also search for this author in _John E. Edwards Jr.You can also search for this author in _David R. AndesYou can also search for this author in _Justin R. NodwellYou can also search for this author in _ Lauren E. BrownYou can also search for this author in _Luke WhitesellYou can also search for this author in _Nicole RobbinsYou can also search for this author in _Leah E. CowenYou can also search for this author in _\" - Contributions - K.R.I conceived the study, performed the majority of experiments, interpreted data, and wrote the initial manuscript. K.C. and R.T. characterized and synthesized the azoffluxin compound. M.D.I. and S.M.P.E. in the lab of J.R.N. performed LC-MS experiments and interpreted data. Z.L. constructed a strain and S.H.K. aided in screening the chemical library. J.F.M. and C.A.C. performed sequence analysis of C. auris isolates, identifying variants. L.E.B. and J.A.P provided insights into chemical scaffold and activity. N.S.W. performed the murine pharmacokinetic experiment and analysis. S.S, A.S.I., and J.E. performed or supervised the murine toxicity study and J.L.F., K.M., and D.R.A. performed the murine efficacy study. N.R., L.W., and L.E.C. helped conceive the study, design the experiments, and interpret data. All authors provided comments in editing the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - L.E.C. and L.W. are co-founders and shareholders in Bright Angel Therapeutics, a platform company for development of novel antifungal therapeutics. L.E.C. is a consultant for Boragen, a small-molecule development company focused on leveraging the unique chemical properties of boron chemistry for crop protection and animal health. The remaining authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution License (), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited).\" - \"Effect of u201CCa. L. asiaticusu201D infection on oxygen tension profile in ACP hemolymph\" - \"The average hemolymph oxygen tensions in u201CCa. L. asiaticusu201D-infected and -free ACPs, as shown in Fig._, were 35.61%u2009xB1u20091.26% and 67.13%u2009 xB1u20092.11%, respectively, a difference that was found to be statistically significant (two-tailed Students t-test, tu2009=u2009u221211.99, Pu2009 <u20090.0001). No linear relationship between oxygen tension and log(GE/insect) was observed.\" - \"Figure 5Effect of infection on oxygen tension in ACP hemolymph (%). (a) Levels of oxygen tension measured in infected and uninfected ACPs. (b) Oxygen tension (%) versus log(GE/insect) in infected ACPs(two-tailed Students t-test, tu2009=u2009u221211.99, Pu2009<u20090.0001). No linear regression was observed between oxygen tension and GE/insect (slopeu2009=u2009u22120.066, SEu2009 =u20090.055; linear regression: tu2009=u2009u22121.19, Pu2009=u20090.255, Ru2009=u20090.09). u201CCa. L. asiaticusu201D genomic equivalents (GE) per insect was calculated based on the standard curve, given that each u201CCa. L. asiaticusu201D has three identical copies of the 16S rRNA gene in its genome.\" - \"Oxygen tension reduction in the ACP body may result from two different mechanisms. By one mechanism, oxygen is consumed by u201CCa. L. asiaticus.u201D In a previous study, it was shown that Japanese beetles infected with Bacillus popilliae (B. popilliae) had a 15% lower oxygen level than uninfected beetles. It was hypothesized that the oxygen level reduction is due to oxygen consumption by B. popilliae. It is possible that, similar to Japanese beetles, in infected ACPs, oxygen reduction results from oxygen consumption by u201CCa. L. asiaticus,u201D suggesting that the bacterium requires oxygen for growth. By the other mechanism, oxygen tension is reduced via elevated respiration by the insect. In a previous study, it was observed that in infected ACPs, the respiration rate was increased by 42%, which was hypothesized to be due to induction of the TCA cycle. A proposed mechanism for this is outlined in Fig._. In addition, the oxygen tension reduction could be the reason that GABA accumulates inside the psyllid body. Stress from oxygen deficiency influences the accumulation of GABA. In other words, GAD enzymes become more activated in a reduced oxygen tension condition and, since the GAD reaction is proton-consuming, pH increases.\" - \"Determination of required pH and oxygen tension to culture u201CCa. L. asiaticusu201D in vitro\" - \"A brief discussion of potential metabolic pathways present in u201CCa. L. asiaticusu201D accompanied the initial report of the metagenomics sequencing-derived predicted genome of this important pathogen. The authors suggested that the u201CCa. L. asiaticusu201D had limited capacity for anaerobic respiration because it putatively contained no homologs for terminal stages in oxidative phosphorylation. However, the two enzymes that were listed as evidence for this (as key components of this well-known and important biochemical pathway) were polyphosphate kinase and inorganic diphosphatase, enzymes that are not actually involved in oxidative phosphorylation but instead are involved in phosphate homeostasis within the cell. Some homologs for potential terminal oxidases or electron transport chain intermediates, the cytochrome bc1 complex, cbb3-type cytochrome c oxidase, and the cytochrome bd complex were reported as missing from the genome sequence. However, all four known cytochrome O ubiquinol oxidases (which can be used in non-canonical oxidative phosphorylation) were found in the genome sequence. In another study five copies of nrdB, encoding the u03B2-subunit of ribonucleotide reductase (RNR), were reported to be encoded by the u201CCa. L. asiaticusu201D genome sequence. The nrdB gene encodes a key enzyme for converting ribonucleotides to deoxyribonucleotides, the precursors of DNA synthesis and repair. Based on nrdB classification, u201CCa. L. asiaticusu201D nrdB is exclusively oxygen-dependent, which suggests that u201CCa. L. asiaticusu201D has an aerobic lifestyle. Moreover, in the previous report by Parker et al., the authors successfully prolonged u201CCa. L. asiaticusu201D viability by supplementing the growth medium with citrus juice under aerobic conditions. Therefore, it is likely that u201CCa. L. asiaticusu201D is an aerobe.\" - \"One of the most reliable methods to determine the optimum pH for u201CCa. L. asiaticusu201D growth is to measure pH in u201CCa. L. asiaticusu201D vectors and hosts. In a previous study, the pH of the phloem sap of citrus trees was reported to be 6.04u2009xB1u20090.16, which is lower than the pH observed for the psyllid body in this study. This could be one of the reasons that u201C Ca. L. asiaticusu201D multiplies in microenvironments with lower pH (hemolymph) within the psyllid. In the study by Parker et al. the pH of the culture medium was around 3, much lower than that observed in the ACP abdomen. Low pH may have prevented u201CCa. L. asiaticusu201D replication under the conditions tested by Parker et al.. In future attempts to culture u201CCa. L. asiaticus,u201D it could be advantageous to adjust the pH of the culture medium to a more neutral range. Thus, the results of this study provide key insights into the natural growth environment of u201CCa. L. asiaticusu201D that will be helpful in guiding future attempts at its culture.\" - Midgut average pH and oxygen tension - \"We also calculated the average pH and oxygen tension of infected and u201CCa. L. asiaticusu201D-free ACP midgut. The pH of the midgut of u201CCa. L. asiaticusu201D -free ACPs (meanu2009=u20097.93, SEu2009=u20090.37) was not significantly different from that of the midgut of infected ACPs (meanu2009=u20098.31, SEu2009 =u20090.55; two-tailed Students t-test: tu2009=u20090.108, Pu2009=u2009 0.915) (Fig._). The levels of oxygen tension in the midgut of u201CCa. L. asiaticusu201D -free ACPs (meanu2009=u200997.91%, SEu2009=u20092.90) were also not significantly different from those in the midgut of infected ACPs (meanu2009=u200993.46% SEu2009=u20094.62; two-tailed Students t-test: tu2009=u2009u2212 0.758, Pu2009=u20090.455) (Fig._). It has been shown that the midgut pH can be affected by feeding conditions. Since the metabolites in infected versus u201CCa. L. asiaticusu201D-free citrus tree phloem sap are different, the midgut pH may change because of feeding on infected trees. Therefore, it is not clear whether u201CCa. L. asiaticusu201D infection can affect the pH of the midgut. To the best of our knowledge, there is no information available on the effect of psyllid feeding on midgut oxygen tension.\" - \"Figure 6Effects of u201CCa. L. asiaticusu201D infection on (a) pH and (b) oxygen tension in ACP midgut. There are no significant differences in pH (two-tailed Students t-test: tu2009=u20090.108, Pu2009=u20090.915) or oxygen tension (two-tailed Students t-test: tu2009=u2009u22120.758, Pu2009=u2009 0.455) between u201CCa. L. asiaticusu201D-free and infected ACPs.\" - Practical implications - \"In this study, we found that the average oxygen tension in the hemolymph of infected ACPs was significantly lower than that of u201CCa. L. asiaticusu201D-free ACPs (35.61u2009xB1u20091.26% vs. 67.13u2009xB1u20092.11%). Therefore, measurement of the oxygen tension in the hemolymph of ACPs can potentially predict the infection status of the ACPs. In addition, there is a statistically significant linear relationship between pH and infection level (genome equivalents per insect). Equation_ shows the relationship between pH and infection level in ACPs.\" - \"$${rm{pH}}={rm{2.95}},log ({rm{GE}}/{rm{insect}})-{rm{20.5}}$$ (1) \" - \"In this equation, GE is the genome equivalents of u201CCa. L. asiaticusu201D and pH is the average pH of the hemolymph. Based on this equation, it may be possible to predict the GE of u201CCa. L. asiaticusu201D in infected ACPs from the average hemolymph pH of the ACPs. However, we should note that these correlations may be different for ACPs of different ages, physiological conditions and symbiont populations.\"- - Psyllids - \"The ACP colony was originally established using field populations from Polk County, Florida. ACPs in the colony were reared continuously on curry leaf, Bergera koenigii, placed into plexiglas chambers and kept under a light:dark 16:8u2009 h photoperiod at 27u2009xB1u20092u2009xB0C and 60u2009xB1u20095% relative humidity. ACPs used for experiments were transferred to uninfected or u201C Ca. L. asiaticusu201D- infected Valencia sweet orange (C.u2009sinensis u2018 Valencia) in two separate growth rooms under a light:dark 14:10u2009h photoperiod at 27u201328u2009xB0C and 60u201365% relative humidity. Both colonies, as well as the citrus plants, were routinely monitored using PCR to confirm infection with u201CCa. L. asiaticus.u201D Adult u201CCa. L. asiaticusu201D- infected psyllids were sampled without categorizing by age or sex when the u201CCa. L. asiaticusu201D infection percentage rose above 50%.\" - Construction of pH microelectrodes - \"A pH microelectrode is a potentiometric microelectrode that measures pH based on the potential difference across a liquid ion exchange (LIX) membrane located on the tip of the microelectrode. The schematics of a pH microelectrode are shown in Fig._. The construction details are given in our previous publication. The microelectrode potential was measured against a separate Ag/AgCl reference electrode in various pH buffers (4, 7 and 10) to generate a calibration curve. The pH microelectrodes had tip diameters of less than 20u2009xB5m and response times of less than 3u2009seconds.\" - Figure 7Schematics of (a) pH and (b) oxygen microelectrodes. The pH microelectrode requires a reference electrode (not shown), which can be located near the microelectrode. (c) Measurement setup used to measure oxygen and pH profiles in ACPs. (d) ACP on agar plate before start of measurement. The wings of the ACP were fixed using small drops of agar. ACPs were laid on their backs, and the microelectrode penetrated their abdomens. The figure is not drawn to scale. - Construction of oxygen microelectrodes - \"The oxygen microelectrode is a needle-type microelectrode (Fig._). It detects the oxygen concentration at the tip in both liquid and gas phases. The microelectrode uses a gold cathode and an Ag/AgCl reference/counter electrode. Oxygen diffuses through a silicone rubber membrane and then is reduced at the gold cathode. The reduction current is proportional to the oxygen concentration in the vicinity of the microelectrode tip. The construction process is detailed in our previous publication. The oxygen microelectrodes we used had tip diameters of less than 20u2009xB5m and response times of less than 3u2009seconds. The gold cathode was polarized at u22120.8u2009V against Ag/AgCl to operate the oxygen microelectrode. The microelectrodes were calibrated using two-point calibration: in Na2SO3 solution (zero oxygen concentration) and in the air.\" - Measurement of pH and oxygen profiles - \"The setup and instruments that were used for microelectrode measurements are shown in Fig._. Oxygen profile measurements were performed on 16 infected and 14 u201CCa. L. asiaticusu201D-free ACP individuals.\" - \"Each ACP was used for a single measurement. For each measurement, a live ACP was fixed on its back on a 2.5% agar layer (Fig._). A measurement was completed in less than 5u2009min for each ACP. The oxygen tension measurement started a little above the abdomen and continued until the microelectrode emerged from the other side of the abdomen. The measurement stopped when the tip of the microelectrode reached the agar layer below the insect. A mercury-step stepper motor (PI M-230.10u2009 S) was used to control the movement of the microelectrode. This stepper motor was connected to a computer, and the relative distance of the microelectrode tip was determined with custom-made MicroprofilerxA9 software developed by our research group. Data were recorded in the computer using a data logger (Measurement ComputingxA9 USB-1608FS).\" - \"The pH profile measurements were performed in the same manner as the oxygen profile measurements except that we added a few drops of phosphate-buffered saline with pHu2009=u20097.5 (PBS, 100u2013300u2009xB5L) to cover the insect s surface. The microreference electrode was inserted into the PBS, and then the procedures for measuring pH with oxygen microelectrodes were followed. The potential between the reference electrode and the pH microelectrode was measured, and then the potential values were converted to pH values using the calibration curve for this microelectrode. For pH measurements, 9 infected and 13 u201C Ca. L. asiaticusu201D -free ACP individuals were tested.\" - \"DNA extraction and quantitative PCR for u201CCa. L. asiaticusu201D detection\" - \"A heating method described previously, but with several modifications, was used to prepare ACP DNA extracts. Individual ACPs were washed with washing buffer, and then each was put in a mortar containing 300u2009xB5L of extraction buffer (10u2009mM Tris and 1u2009mM EDTA, pH 8.0) and crushed completely using a pestle. The crude ACP extract solution was transferred to a Lysing Matrix E tube (MP Biomedicals) and vortexed for 10u2009min, followed by centrifugation (16100u2009xD7u2009g for 90u2009min), with the aqueous portion of each sample being transferred to a PCR tube. The samples were then heated at 80 xB0C for 7u2009min and then at 95u2009xB0C for 3u2009min, followed by centrifugation at 8000u2009xD7u2009g for 2u2009min. The aqueous portion of each sample was used as a DNA template in a final PCR reaction volume of 200u2009xB5l. Quantitative PCR using the following primer pair was performed to quantify u201C Ca. L. asiaticusu201D DNA in each sample: 5u2032-GGT TTT TAC CTA GAT GTT GGG TAC T-3u2032 and 5u2032-CTT CGC AAC CCA TTG TAA CC-3u2032. Both of the primer sequences are conserved in u201CCa. L. asiaticusu201D and were designed to amplify the Las-specific DNA fragment (140u2009bp) from its 16S rRNA gene. The reaction mixture components and target concentrations were a qPCR master mix (Power SYBR Green PCR Master Mix, Applied Biosystems, USA) with a final concentration of 1xD7, forward primer and reverse primers with a final concentration of 0.2u2009xB5M each. The DNA volume for each reaction mixture was 2u2009 xB5l. The cycling parameters used to run the quantitative PCR were 94u2009xB0 C for 2u2009min, followed by 40 cycles of 94u2009xB0C (denaturation) for 15u2009 s and 60u2009xB0C (annealing, extension, and red fluorescence) for 1u2009min.\" - \"The specific 16S-rRNA amplicon (1160u2009bp) for u201CCa. L. asiaticusu201D was obtained using PCR with high-fidelity DNA polymerase and the conventional primers (Ol1, Ol2c), purified and used as the molecular standard for qPCR. A 10-fold dilution series was prepared to create the standard curve. The genomic equivalents (GE) per insect was calculated based on the standard curve, given that each u201CCa. L. asiaticusu201D has three identical copies of the 16S rRNA gene in its genome.\" - Data analysis - \"Two-sample t-tests were used to determine whether there was a statistical difference in pH or oxygen tension between u201CCa. L. asiaticusu201D-free and infected ACPs. We used linear regression to explore the relationships between pH and genome equivalents/insect (GE/insect) values and between oxygen tension and genome equivalents/insect (GE/insect) values. All statistics were computed using Excel 2013.\" - Location of the microelectrode tip within the ACP body - \"Based on a sharp change in profile, the microelectrode tip is exposed to new conditions, which in turn indicates that it has entered a new environment. Based on the ACPs anatomy, the microelectrode tip passes through three different environments. Therefore, by comparing the profiles with ACP anatomy, it is possible to predict the location of the microelectrode tip.\" - \"When the microelectrode tip penetrates into the midgut, a hole is formed on the wall of the gut. This is expected to cause insignificant leakage of midgut lumen into the hemolymph, and because of the circulation of midgut lumen in the ACP body, this phenomenon may increase the pH of the hemolymph. Thus, to calculate the average pH of the hemolymph, the pH of the region that is highlighted with a star in Fig._ was used to make sure that gut lumen had not changed the pH of the hemolymph.\"- - \\' \\' - The authors confirm that the data supporting the findings of this study are available within the article. Raw data supporting the findings of this study are available from the corresponding author (HB) on request. - \\' \\'- - \"Killiny, N., Hijaz, F., Ebert, T. A. & Rogers, M. E. A plant bacterial pathogen manipulates its insect vectors energy metabolism. Appl. Environ. Microbiol. 83, e03005u201303016 (2017)._ _ _ Killiny, N. & Almeida, R. P. Host structural carbohydrate induces vector transmission of a bacterial plant pathogen. Proc Natl Acad Sci 106, 22416u201322420 (2009)._ _ _ _ Perilla-Henao, L. M. & Casteel, C. L. Vector-borne bacterial plant pathogens: interactions with hemipteran insects and plants. Front plant sci 7, 1163 (2016)._ _ Hall, D. G., Richardson, M. L., Ammar, E.-D. & Halbert, S. E. Asian citrus psyllid, Diaphorina citri, vector of citrus huanglongbing disease. Entomol. Exp. Appl. 146, 207u2013223, (2013)._ _ Bonani, J. P. et al. Characterization of electrical penetration graphs of the Asian citrus psyllid, Diaphorina citri, in sweet orange seedlings. Entomol. Exp. Appl. 134, 35u2013 49, (2010)._ _ Lai, K.-K., Davis-Richardson, A. G., Dias, R. & Triplett, E. W. Identification of the Genes Required for the Culture of Liberibacter crescens, the Closest Cultured Relative of the Liberibacter Plant. Pathogens. Front Microbial 7, 547, (2016)._ _ Ukuda-Hosokawa, R. et al. Infection density dynamics of the citrus greening bacterium Candidatus Liberibacter asiaticus in field populations of the psyllid Diaphorina citri, and its relevance to pathogen transmission efficiency to citrus plants. Appl. Environ. Microbiol. AEM, 00707u201300715 (2015)._ Pelz-Stelinski, K., Brlansky, R., Ebert, T. & Rogers, M. Transmission parameters for Candidatus Liberibacter asiaticus by Asian citrus psyllid (Hemiptera: Psyllidae). J. Econ. Entomol. 103, 1531u20131541 (2010)._ _ _ Ammar, E.-D., Ramos, J. E., Hall, D. G. & Dawson, W. O. & Shatters, R. G., Jr. Acquisition, Replication and Inoculation of Candidatus Liberibacter asiaticus following Various Acquisition Periods on Huanglongbing-Infected Citrus by Nymphs and Adults of the Asian Citrus Psyllid. PLoS One 11, e0159594, (2016)._ _ _ _ Ammar, E.-D., Shatters, R. G. & Hall, D. G. Localization of Candidatus Liberibacter asiaticus, Associated with Citrus Huanglongbing Disease, in its Psyllid Vector using Fluorescence in situ Hybridization. J. Phytopathol. 159, 726u2013734, (2011)._ _ _ Ammar, E.-D., Shatters, R. G., Lynch, C. & Hall, D. G. Detection and Relative Titer of Candidatus Liberibacter asiaticus in the Salivary Glands and Alimentary Canal of Diaphorina citri (Hemiptera: Psyllidae) Vector of Citrus Huanglongbing Disease. Ann. Entomol. Soc. Am. 104, 526u2013533, (2011)._ _ _ Ghanim, M., Fattah-Hosseini, S., Levy, A. & Cilia, M. Morphological abnormalities and cell death in the Asian citrus psyllid (Diaphorina citri) midgut associated with Candidatus Liberibacter asiaticus. Sci Rep 6 (2016).Richards, B., Blood, H. & Linford, M. Destructive outbreak of unknown potato disease in Utah. Plant dis rep 11, 93u201394 (1927)._ Killiny, N., Nehela, Y., Hijaz, F. & Vincent, C. I. A plant pathogenic bacterium exploits the tricarboxylic acid cycle metabolic pathway of its insect vector. Virulence, 00u201300 (2017).Malik, N. S., Perez, J. L., Kunta, M. & Olanya, M. Changes in polyphenol levels in Satsuma (Citrus unshiu) leaves in response to Asian Citrus Psyllid nfestation and water stress. The Open Agric J 9 (2015).Brune, A. & KxFChl, M. pH profiles of the extremely alkaline hindguts of soil-feeding termites (Isoptera: Termitidae) determined with microelectrodes. J. Insect Physiol. 42, 1121u20131127 (1996)._ _ _ Tartes, U., Vanatoa, A. & Kuusik, A. The insect abdomenu2014a heartbeat manager in insects? Comp Biochem Physiol A Mol Integr Physiol 133, 611u2013623 (2002)._ _ _ Toei, M., Saum, R. & Forgac, M. Regulation and isoform function of the V-ATPases. Biochemistry 49, 4715u20134723 (2010)._ _ _ Beyenbach, K. W. & Wieczorek, H. The V-type H+ ATPase: molecular structure and function, physiological roles and regulation. J. Exp. Biol. 209, 577u2013589 (2006)._ _ _ Weiner, B., Kwolek, W., Julian, G. S., Hall, H. & Jackson, R. Oxygen concentration in larval hemolymph of the Japanese beetle, Popillia japonica, infected with Bacillus popilliae. J. Invertebr. Pathol. 8, 308u2013 313 (1966)._ _ Mei, X. et al. Dual mechanisms regulating glutamate decarboxylases and accumulation of gamma-aminobutyric acid in tea (Camellia sinensis) leaves exposed to multiple stresses. Sci Rep 6, 23685 (2016)._ _ _ _ Duan, Y. et al. Complete genome sequence of citrus huanglongbing bacterium,u2018Candidatus Liberibacter asiaticus obtained through metagenomics. Mol. Plant-Microbe Interact. 22, 1011u20131020 (2009)._ _ _ Zheng, Z. et al. Unusual Five Copies and Dual Forms of nrdB in u201CCandidatus Liberibacter asiaticusu201D: Biological Implications and PCR Detection Application. Sci Rep 6, 39020 (2016)._ _ _ Parker, J. K. et al. Viability of u2018Candidatus Liberibacter asiaticus prolonged by addition of citrus juice to culture medium. Phytopathology 104, 15u201326 (2014)._ _ _ Hijaz, F. & Killiny, N. Collection and chemical composition of phloem sap from Citrus sinensis L. Osbeck (sweet orange). PLoS One 9, e101830 (2014)._ _ _ Nepomuceno, D. B. et al. pH control in the midgut of Aedes aegypti under different nutritional conditions. J. Exp. Biol. 220, 3355u20133362 (2017)._ _ Cevallosu2010Cevallos, J. M., GarcxEDau2010Torres, R., Etxeberria, E. & Reyesu2010Deu2010Corcuera, J. I. GCu2010MS analysis of headspace and liquid extracts for metabolomic differentiation of citrus huanglongbing and zinc deficiency in leaves of u2018Valencia sweet orange from commercial groves. Phytochem. Anal. 22, 236u2013246 (2011)._ _ Lewandowski, Z. & Beyenal, H. Fundamentals of biofilm research. (CRC press, 2013).Keremane, M. L. et al. A rapid field detection system for citrus huanglongbing associated u2018Candidatus Liberibacter asiaticus from the psyllid vector, Diaphorina citri Kuwayama and its implications in disease management. Crop Protect. 68, 41u201348 (2015)._ _ Jagoueix, S., BovxE9, J. M. & Garnier, M. PCR detection of the two xABCandidatusxBB liberobacter species associated with greening disease of citrus. Mol. Cell. Probes 10, 43u201350 (1996)._ _ _ \"- - This work is supported by the Citrus Disease Research and Extension Program of the Specialty Crop Research Initiative (grant no. 2016-70016-24824) of the USDA National Institute of Food and Agriculture.- - Authors and Affiliations - \"Gene and Linda Voiland School of Chemical Engineering and Bioengineering, Washington State University, Pullman, Washington, USABanafsheh Molki,_Phuc Thi Ha,_Abdelrhman Mohamed_&_Haluk BeyenalDepartment of Plant Pathology, Citrus Research and Education Center, IFAS, University of Florida, Lake Alfred, Florida, USANabil KillinyInstitute of Biological Chemistry, Washington State University, Pullman, Washington, USADavid R. GangPaul G. Allen School for Global Animal Health, Washington State University, Pullman, Washington, USAAnders Omsland\" - \"AuthorsBanafsheh MolkiYou can also search for this author in _Phuc Thi HaYou can also search for this author in _Abdelrhman MohamedYou can also search for this author in _Nabil KillinyYou can also search for this author in _David R. GangYou can also search for this author in _Anders OmslandYou can also search for this author in _Haluk BeyenalYou can also search for this author in _\" - Contributions - B.M. performed experiments with assistance from P.H.T., A.M., H.B. and N.K. All authors contributed to the experimental design and the writing and completion of the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution License (). Each color is associated with an elevation range defining a paleosurface which, in turn, is established based on the statistical analysis. See text for further explanation. (b) Cross-section showing the three marine terraces recognized in this area: the lower Latina and Minturno complexes (description of the associated deposits by is provided in the text), and the higher Campoverde complex recognized in this study.\" - \"Indeed, the area where these terraces are recognized corresponds to the coastal reach facing the Monti Sabatini and Colli Albani volcanic districts (Fig._ ), which were active 800u2009ka through 86u2009ka, and 600u2009ka though 36u2009 ka, respectively.\" - \"A Ar/Ar age constraint of 129u2009xB1u20091u2009ka allowed correlation of the 36u2009m terrace with the highstand of (Marine Isotopic Stage)_MIS 5e (125u2009ka), in good agreement with previous geochronologic constraints achieved on the sedimentary deposits forming a terrace at higher elevation, ranging 51u2013 56u2009m a.s.l., providing correlation with MIS 7. Based on the strong age constraint for the 56u2009m and 36u2009m terraces, and lacking direct geochronologic constraints for the lower terraces at 26u2009m and at 15u2009m, they have tentatively been correlated with MIS 5c (100u2009ka) and MIS 5a (80u2009ka), respectively, following the principle of a staircase geometry (e.g.).\" - \"In contrast with the widespread morphological evidence all along the coast to the north, the terrace at 36u2009m was not recognized southeast of Anzio, where related literature accounts for several sea-level indicators ranging from 10 to 6u2009m a.s.l., which are identified as the MIS 5e sea-level (and references therein). In the present work, we have performed a geomorphological study of the coastal area southeast of Anzio, and we have re-analyzed the faunal remains hosted at the Nettuno museum with the aim of providing biochronological constraints to the Campoverde paleosurface, in order to check its chronological relationships with the 36u2009m terrace north of Anzio, and with the sea-level indicators attributed to MIS 5e in this area.\" - Moreover, we have re-analyzed the lithic artifacts recovered along with the faunal remains in Campoverde, aiming to provide techno-typological and taphonomic information to support the paleontological and paleoenvironmental interpretations provided in this paper.- - \"The fossil remains and lithic artifacts originally recovered at Campoverde are housed in the Paleontological and Palethnological Collection of the u201CAntiquarium Comunaleu201D in Nettuno (Rome). The bone remains and the lithic artifacts were collected in two distinct areas not far from each other (CV1 and CV2) from the sediment accumulated on the ground level during excavation of the irrigation channel. For this reason, apart for a few specimens collected in situ, it was not possible to establish the exact stratigraphic level for most finds.\" - The mammal fossils from Campoverde have been reexamined and re-measured for the present work. - \\'Abbreviations: L.: length; B. breadth; MTD: transverse diameter of diaphysis; MAPD: antero-posterior diameter of diaphysis; DTD: distal transverse diameter; DAPD: distal antero-posterior diameter; DB/L index: distal transverse diameter/length * 100; MB/L index: transverse diameter of diaphysis/length * 100.\\' - The lithic collection from Campoverde is composed of 562 artifacts. The first sample of 245 artifacts was techno-typologically examined in the past, while the remaining lithic industries have been analysed for this paper.- - \"In the present study, paleosurfaces have been mapped following a well-established geomorphological approach, based on the identification of a set of flat surfaces characterized by topographic culminations with elevation ranging through a few meters around a mean value. Selected topographic culminations of the reconstructed paleosurfaces were detected on the 1:25.000 topographic maps of Italy edited by Istituto Geografico Militare (IGM) (sheets: 158 I SO, 158 I SE, 158 II NO, 158 II NE, 158 II SE, 158 III NO, 158 IV SE). They include all the hilltops (i.e., each elevation point within a closed, 5u2009m spaced contour line) and other quasi-equivalent points within almost closed contour lines bordering plateau-like sectors. Distribution of the topographic culminations has been statistically analyzed in order to verify the occurrence of discrete elevation intervals corresponding to peaks of concentration, which can be assumed as the mean value for each paleosurface. The identification of the paleosurfaces implies a combined approach that integrates statistically significant concentrations of elevations around a mean value, and the morphologic evidence for the concentration of these elevation points within a finite area, as detected in the maps.\" - \"Due to the copyright covering the topographic maps by IGM, we have used the 1:10.000 technical maps by Regione Lazio (), available under Creative Commons Attribution License (), as a topographic base in Fig._ of this paper.\"- - \"Results of the geomorphological study are shown in Fig._, where three very well-defined paleosurfaces at 11u201316, 20u201328, and 31u201340u2009m a.s.l., are depicted in the coastal sector southeast of Anzio. The topographic culminations identified on the 1:25.000 maps are reported with triangles of different colors. Each color is associated with an elevation range defining a paleosurface which, in turn, is established based on the statistical analysis. Elevation ranges for the paleosurfaces are represented by grey boxes of cumulated frequency. Blue color shading is used in Fig._ for the 31u201340u2009m paleosurface, while inner margins represented by solid colored lines are reported for the lower paleosurfaces, located closer to the coast.\" - \"The intermediate paleosurface is characterized by two separate peaks of concentration at 20u201322 and 25u201328u2009m a.s.l. (Fig._). Similarly, the lowest paleosurface has a minor peak at 11u2009m and the main peak at 15u2009m a.s.l., and the highest one has two peaks at 31 and 36u2009m a.s.l. We interpret this feature as reflecting a coastal set represented by a beach flat (lowest elevation peak) and a backbeach plateau including the coastal plain (highest elevation peak), separated by the beach ridge, as illustrated in Fig._.\" - \"Finally, the occurrence of a highest, less well preserved paleosurface is suggested by a series of topographic culminations ranging 51u201358u2009m a.s.l. in the northwestern portion of this area. Given its limited extent at the margin of the investigated coastal sector, a possible inner margin for this paleosurface is not reported.\"- - \"Remarkably, the two lowest order paleosurfaces identified by the geomorphological study, at 20u201328u2009m and 11u201316u2009m a.s.l., match well with two marine terraces identified by previous physiographic studies, termed the Latina complex and the Minturno complex, respectively (Fig._).\" - \"The Latina complex consists of a dissected level plateau, 20 to 25u2009m a.s.l., mainly composed of well-sorted fine augite-rich sands with rare intercalations of clay, representative of a brackish/marine environment. Borehole data in (Fig._ ) indicated that at a depth of 12u201315u2009m this homogeneous sand horizon is underlain by fossiliferous calcareous muds, of lagoonal origin, with peat intercalations. On the more level and less eroded, upper part of the Latina complex, the sands are covered by a layer of greenish clay to sandy clay loam, 1.5u20133.0u2009m in thickness.\" - \"The Minturno complex as been defined as consisting of an elongated ridge 8 to 15u2009m a.s.l., and of a posterior dissected level area, 11 to 16u2009m a.s.l. (Fig._). Textures of the deposits range from sandy clay loam to loamy coarse sand with sparse gravel, and they are interpreted as a beachridge system, overlying greenish lagoonal clay above which the beachridge migrated. These greenish, non-stratified clays also form the top of the deposits in the level area (Fig._ ). Outcrop data show that the clays lie conformably on loams and subsequently gravels, forming a fining upwards sequence overall.\" - \"Based on tentative correlation with known marine terrace sequences for Latium, a correspondence with the Tyrrhenian II stage (approximately 127u2009ka, corresponding to MIS 5e) was proposed for the Minturno complex in, whereas only a relative, older age was assigned to the Latina complex. This correlation is derived from the elevations at which the raised beach deposits containing the warm exotic gastropod Strombus bubonius occur in northern Latium, namely 2u20133u2009 m (STROMBUS III, ~90u2009ka), 10u201315u2009m (STROMBUS II, ~127u2009ka), and 18u201325u2009m a.s.l. (STROMBUS I, ~200u2009ka). Subsequently, based on the definition of characteristic aminozones (i.e., based on amino acid geochronology) previous interpretations of three distinct Strombus levels occurring along the Latium coast have been rejected, although correlation of the Minturno complex transgression with the MIS 5e interglacial of ca. 125u2009ka has been coinfirmed. The MIS 5e terrace has been successively identified at varying elevations in the area comprised between Monte Argentario and Rome, ranging over 20u2013 36u2009m a.s.l. due to differential tectonic uplift, and progressively descending to ~10u2009m in the stable sectors to the north and to the south, see Fig._ ). More recently, have provided a Ar/Ar age of 129u2009xB1u20091u2009ka on a primary volcanic layer intercalated in the deposits forming the 36u2009m terrace at Cava Rinaldi in the central coastal area (Fig._), demonstrating its correlation with MIS 5e. Moreover, the 36u2009m terrace has been reconstructed over a wide coastal reach, showing that it homogeneously extends between Ladispoli and Anzio (Fig._), revising previous inferences of varying elevation due to differential uplift rates along the coast. In contrast, it has also been shown that two lower, younger coastal terraces are present in this area, at an elevation of ca. 27 and ca. 16u2009m, which were previously interpreted as tectonically displaced portions of the MIS 5e terrace. No direct geochronologic constraint has been so far provided on these lower terraces, which have been tentatively correlated with MIS 5c and MIS 5a, respectively (Fig._u2009).\"- - \"Excavations to rectify the stream channel exposed a ca. 2.5u2009m high section at Campoverde, in the middle of the flat area at around 36u2009m a.s.l. (Figs_ and ). The trench cut exposed a fining-upwards sedimentary succession unconformably overlying the pyroclastic-flow deposit of Tufo Lionato (Lower Flow Unit of the Villa Senni Eruption Cycle, 365u2009xB1u20094u2009ka). A gravelly sand sediment of fluvial origin rich in reworked volcanics represents the basal coarse deposit of the sedimentary sequence, which grades upward into a loamy sand deposit, followed by whitish calcareous muds of a brackish/lagoonal environment. These features allow the sedimentary sequence at Campoverde to be interpreted as an aggradational succession deposited in response to sea-level rise during a glacial termination, according to the definition by. It is indeed a transgressive deposit which, by analogy with the Latina and Minturno complexes defined in this area, we name the Campoverde complex (Fig._).\" - Figure 3Outcrop of the deposits forming the Campoverde complex (photo by authors). See text for comments.- - \"The following taxa have been identified in the faunal assemblage of Campoverde (Table_).\" - Table 1 Summary of paleontological revisions to the Campoverde mammal assemblage. - Elephantidae - Palaeoloxodon antiquus - The presence of the straight-tusked elephant is indicated by several upper and lower molars, and numerous fragments of ribs and vertebrae, and short and long limb-bones, referable to young and adult individuals. - Mammuthus ex gr. chosaricus-primigenius - Bones and dental remains of Mammuthus also occur, characterized by the rather archaic structure of the molars, testified by the enamel thickness (unusual for M. primigenius) and by the rather broad lamellar frequency, typical of the trogontherii-chosaricus-primigenius transitional forms. - Ursidae - Ursus spelaeus - This species is represented by several fossil remains. The shape and size of the limb-bone fragments and especially of the upper canine are suggestive of this species rather than U. deningeri or U. arctos. - Canidae - Canis lupus - A mandibular fragment of modest size with a second lower molar, probably belonging to a female, testifies to the presence of this species at Campoverde. - Equidae - Equus ferus - Numerous teeth and limb-bone remains can be attributed, from morphology and large dimensions, to E. ferus. Upper molars and premolars display the plis caballin and symmetric and lengthened protocone; the lower cheek teeth show the lingual flexid of caballine-type and, as for the upper teeth, the plis caballin (with references). - Equus hydruntinus - \"Some upper teeth and a fragmentary metacarpus belong to this small-sized equid. Upper molars display interstylar faces with a flattened trend, simple pilasters and short protocone with the typical shape of u201Chorse hoofu201D.\" - Rhinoceratidae - Stephanorhinus sp - Many fragments of upper and lower teeth and some fragmented post-cranial bone remains of Rhinoceratidae can be referred generically to genus Stephanorhinus. Among these, one was probably worked to realize a bone tool. - A fragmentary upper premolar is brachydont and it shows a rather rough enamel, a marked cingulum and occlusial surface with contours similar to those of S. hemitoechus, and it can be prudently attributed to Stephanorhinus cf. S. hemitoechus. - Hippopotamidae - Hippopotamus cf. H. amphibius - A fragmented humerus can be attributed to Hippopotamus cf. H. amphibius from its small size, compatible with the dimensions of the species. - Cervidae - Megaloceros cf. M. giganteus - A few specimens, including two complete astragals, show the typical morphology of the Cervidae, and belong to a large cervid, probably the giant deer. - Cervus elaphus - \"The presence of red deer is indicated by several fragmented cranial and post-cranial bones, including some upper and lower teeth. A third lower molar belonging to an elderly individual display the very large third lobe almost completely combined with the other two. These morphological features and the dimensions (L. 28.90, B. 12.70u2009mm) are close to the modern forms of C. elaphus, which occurred in the peninsula at the onset of Late Pleistocene.\" - Dama dama ssp - \"Several cranial and post-cranial remains belong to the fallow deer. An upper molar shows a weak anterior and posterior cingulum, but does not display the presence of any entostyle (Fig._); a rather small lower molar (L. 17.80, B. 10.0u2009mm) displays an imperceptible anterior cingulum and, above all, the absence of the entostyle (Fig._). These morphological features permit attribution of these teeth to the recent form of modern fallow deer, D. dama dama. A particularly thin diaphysis of metatarsus and a proximal radius (Fig._) of reduced size can also be attributed to this subspecies.\" - \"Figure 4(a) Upper and (b) lower molar, and (c) proximal radius of Dama dama dama (photos by authors). (d) Scatter diagram_of major measurements (mm) of the fallow deer subspecies proximal radius.\" - \"Some post-cranial remains have the typical morphology of fallow deer, but the sizes are larger than the current D. dama dama and are unusual also for the Late Pleistocene fallow deer (e.g., metacarpus: MTD 21.5, MAPD 20.5u2009mm; tibia: DTD 36.0, DAPD 30.9u2009mm). Probably, these bones belong to the archaic form of modern fallow deer, D. dama tiberina (see), and can be prudently referred to D. dama cf. D. d. tiberina.\" - Capreolus capreolus - A couple of brachydont lower check-teeth with very small sizes can be referred to C. capreolus. - Bovidae - Bos primigenius - \"The aurochs is the most abundant taxon at Campoverde, as in the most of the Aurelian mammal assemblages. The measurements of many specimens reveal a population of large size and some skeletal elements are larger than those of B. primigenius from Avetrana (Taranto, Apulia), from a fossiliferous deposit referred to MIS 5. For example, a metacarpus 169u2009mm long and belonging to a male (DB/L and MB/L indicesu2009=u200935.53 and 20.82, respectively) was estimatedto have a withers height of 170.3u2009cm, among the largest recorded in Italy.\"- - \"The lithic industries studied in the late 1990s and early 2000s belong to a technocomplex with a high percentage of corticated blanks obtained from small pebbles (<5u2009 cm), a strong predominance of corticated and smooth striking platform, an absence of Levallois technique, and the presence of bipolar and centripetal technology of cores (Fig._u2009). Unretouched blanks are rare whilst carinated blanks are abundant. Retouched blanks are extremely small and are mainly composed of denticulates, often microlithic (42,8%), side scrapers (21,1%), borers (8,8%), other tools (7,5%), end scrapers (6,8%) and choppers (6,1%) (Fig._). These characteristics compare well with the lithic assemblage from Casal De Pazzi, therefore suggesting an_MIS 7_age.\" - \"Figure 5(a) Lithic industry from the first technocomplex (re-drawn here from original drawings by). (b) Artifacts displaying the brownish polished patina (authors image). (c) Ichnotraces by u201Cvermetidaeu201D, identified on few artifacts (authors image). (d) Artifacts displaying carbonate concretions (authors image).\" - A second collection of 317 implements from Campoverde has been recently analysed for this study. The main features are the preponderance of unretouched blanks (=179), retouched blanks (=70), cores (=26) and debris (=42). A large number of side scrapers (76%), notches and denticulates (13%) and rare borers (2%) were identified. The Levallois technique is rarely observed (3%). Cores are perfectly distributed among centripetal (23%), prepared platform (23%) and shapeless or exhausted (23%), while pseudoprismatic (11,5%), globular (8%) flaked pebbles (8%) and a single chopper core (3,5%) are identified. - \"The first assemblage displays diverse and interesting taphonomic evidence. A large number of artifacts exhibit a brownish polished patina (86%; Fig._), a small percentage (14%) does not display no patina at all, while sharp and rounded edges are present in both categories. Moreover, ichnotraces (Fig._ ) and encrustations (Fig._) are present on a small number of implements. The ichnotraces are probably attributable to encrusting aquatic organisms, probably vermitidae, and suggest that the area experienced marine conditions after the lithics were discarded. The encrustations seem to support a wet habitat, since these are identified as carbonate concretions with nodules and/or crystals of manganese (or iron), and strongly suggest that these specimens come from the whitish layer of carbonate lagoon mud. Notably, the second assemblage does not show patinas or encrustation of any type.\" - Taphonomic analyses suggest that different depositional dynamics characterise the industries composing this collection, which certainly reached the site from diverse source areas, likely at some distance for those implements with rounded edges. Otherwise from specimens with the brownish polished patina, which point to a prolonged emplacement within a peat deposit, those without patina were most probably discarded in the area in a period of time when it was characterized by a different environmental/geological framework. Some of the brownish specimens display both ichnotraces and carbonate encrustations. Remarkably, while the first assemblage features diverse taphonomy, suggesting extremely disparate and problematic depositional dynamics, the second assemblage presents an extended homogeneity indicating an almost contemporary deposition of the artifacts.- - \"[1] pointed out that the exact stratigraphic position within the sedimentary succession of the majority of the finds is unknown, and that many fossil bones show evidence of transportation, suggesting that several of these may be reworked. Nevertheless, these remains were considered as a single assemblage, referred it to middle-late Middle Pleistocene. In fact, most of the taxa found at Campoverde (such as Palaeoloxodon antiquus, Ursus spelaeus, Canis lupus, Equus ferus, Stephanorhinus hemitoechus, Hippopotamus amphibius, Bos primigenius, and Capreolus capreolus) have their First Occurrence (FO) in Italy in the Galerian, and are well represented in the Aurelian mammal assemblages, spanning a wide temporal range (MIS 15/13 through MIS 7, i.e.: 600u2013200u2009ka, see Fig._).\" - Figure 6Biochronological scheme showing the range and the FO (black arrows) of the principal taxa constituting the faunal assemblage of Campoverde. Comparison with the Oxygen Isotope timescale is also shown. - \"Similarly, Megaloceros giganteus is reported in Italy from MIS 9, in the Torre in Pietra Faunal Unit (FU), becoming quite frequent in the Late Pleistocene and disappearing at the end of the Last Glacial. Equus hydruntinus, which was firstly recorded in the Vitinia FU, MIS 8.5u20137, became more common during the Late Pleistocene, and became extinct during the middle Holocene. Also Cervus elaphus, which occurs for the first time in the Slivia FU, MIS 19, and is still living in Italy, does not offer biochronological information, because such evidence requires identification at the subspecies level only, and can be made mostly by comparison of the antlers, which are not reported among the fossil remains of Campoverde.\" - \"In contrast, discriminating elements are represented by Mammuthus and Dama dama. Indeed, if M. primigenius occurs in Italy in the Late Pleistocene, the transitional forms M. trogontherii-chosaricus-primigenius are recorded only from the late Galerian to middle-late Aurelian, in the Middle Pleistocene (Fig._). Among the modern forms of fallow deer, D. dama tiberina occurs only in the Vitinia FU, MIS 8.5u20137, while D. dama dama occurs at the beginning of Late Pleistocene, and was particularly abundant in central-southern Italy during the last interglacial, and then progressively decreased in abundance before extinction in MIS 2.\" - \"The presence at Campoverde of D. dama dama, which occurred only in the Late Pleistocene, and of Mammuthus ex gr. chosaricus-primigenius, recorded only in the Middle Pleistocene, clearly indicates two distinct mammal assemblages. The younger one is referred to the Late Pleistocene, and should be considered to reflect the age of deposition of the aggradational succession forming the Campoverde complex. This transgressive deposit forms a marine terrace, at an elevation ranging 31u201340u2009m, that we correlate with MIS 5e. An older faunal assemblage is referred to the late Middle Pleistocene; the occurrence of E. hydruntinus along with the possible occurrence of D. dama tiberina, constraints the time interval to the Vitinia FU, MIS 8.5u20137 (Fig._). We interpret it as evidencing the occurrence of reworked material within the basal coarse deposit, incorporating misplaced fossils eroded from the older deposits forming the higher terrace of MIS 7 age, whose remnants are recognized in the northwestern sector of the investigated area, at an elevation of 51u201358u2009m a.s.l. (Fig._).\" - \"We remark that amino acid geochronology for the sand deposits at an elevation of ~12u2009m a.s.l. in the Agro Pontino (Minturno complex) provided two groups of Alle/Ile ratios (site 15 in). One considered typical of MIS 5e (0.39, within the E aminozone), and another, dominant group indicative of a later stage 5 age (0.29, within the C/D aminozone). These data suggest the occurrence of reworked, older shells from the MIS 5e terrace conglobated within the deposit of a younger transgressive cycle, according to the correlation proposed in this paper with MIS 5a for the Minturno complex.\" - \"Consistently, the techno-typological study supports interpretation of the collection recovered at Campoverde as composed of two different technocomplexes, with the first assemblage highlighting a predominance of denticulates whilst the second is mainly composed of side scrapers. Taphonomic and techno-typological characteristics suggest a different chronological attribution of the assemblages which, however, can be generally assigned to the MIS 7u20135a timeframe.\"- - \"The correlation of the 36u2009m terrace with MIS 5e in the investigated area south of Anzio is consistent with the wider regional geomorphological setting, where this terrace is continuously recognized in the coastal area between Civitavecchia and Anzio, and is geochronologically constrained at 129u2009ka (see Fig._ ). Based on this straightforward correlation, the two lower terraces at ~26 and ~15u2009m in the Agro Pontino must be correlated with younger interstadials, namely MIS 5c and 5a, respectively, as already proposed for the equivalent terraces reconstructed in the central Latium coast by. As a consequence, previous correlation of the lowest terrace (Minturno complex) and of the related sea-level indicators between 12 and 10u2009m a.s.l. with MIS 5e appears questionable, and should be revised.\" - \"Notably, this area was previously considered tectonically stable and the sea-level elevation of ca. 9u2009m testified by tidal notches in the Circeo promontory has been considered a benchmark for estimation of the maximum amplitude of the sea-level oscillation during MIS 5e in the Mediterranean Sea (e.g.). We believe that the new data presented in this paper require reconsideration of the sea-level history during the Last Interglacial Stage for the central Mediterranean Sea so far conceived, as well as, provide new insights on the tectonic history of the Tyrrhenian Sea Margin of central Italy, evidencing a uniform uplift affecting the whole coastal reach of this region since 125u2009ka. However, no safe geochronologic constraint to this sea-level exists in the literature, and future work needs to be addressed aimed at providing direct dating of the associated deposits, as well as, of those associated with the 26u2009m and with the 36u2009m terraces in this area.\"- - \\' \\' - All data generated or analysed during this study are included in this published article. - \\' \\'- - \"Mazza, P., Moggi-Cecchi, J. & Rustioni, M. Vertebrate remains from Campo Verde, Latium, central Italy. Antropologia Contemporanea 15, 65u201383 (1992)._ Vianello, F. et al. Palaeolithic from the volcanoclastic deposits of the Agro Pontino (Latium, Central Italy). Quaternaria Nova 5, 7u201322 (1995)._ La Rosa, M. The earliest Paleolithic industries of the Agro Pontino (Latina, Central Italy). Papers from the EAA third Annual Meeting at Ravenna 1997. Volume I: Pre- and Protohistory. BAR International Series 717, 6u201310 (1998)._ La Rosa, M., Mazza, P. & Rustioni, M. The elephant remains from the Agro Pontino, Latina, Central Italy. Atti del 1xB0 Congresso Internazionale u201CLa Terra degli Elefantiu201D, Roma, CNR, 16-20 ottobre 2001, 205u2013208 (2001)._ Marra, F., Florindo, F., Anzidei, M. & Sepe, V. Paleo-surfaces of glacio-eustatically forced aggradational successions in the coastal area of Rome: assessing interplay between tectonics and sea-level during the last ten interglacials. Quaternary Science Reviews 148, 85u2013100, (2016)._ _ _ Westaway, R., Bridgland, D. & Mishra, S. Rheological differences between Archaean and younger crust can determine rates of Quaternary vertical motions revealed by fluvial geomorphology. Terra Nova 15, 287u2013298 (2003)._ _ _ Peccerillo, A. Plio-Quaternary volcanism in Italy. Petrology, Geochemistry, Geodynamics. Springer, Heidelberg (2005).Sottili, G. et al. - Geochronology of the most recent activity in the Sabatini Volcanic District, Roman Province, central Italy. Journ. Volc. and Geoth. Res. 196, 20u201330, (2010)._ _ _ _ Marra, F. et al. Major explosive activity in the Sabatini Volcanic District (central Italy) over the 800-390 ka interval: geochronological - geochemical overview and tephrostratigraphic implications. Quaternary Science Reviews 94, 74u2013101 (2014)._ _ _ Gaeta, M. et al. Paleozoic metasomatism at the origin of Mediterranean ultrapotassic magmas: constraints from time-dependent geochemistry of Colli Albani volcanic products (Central Italy). Lithos 244, 151u2013164, (2016)._ _ _ _ Marra, F. et al. A new age within MIS 7 for the Homo neanderthalensis of Saccopastore in the glacio-eustatically forced sedimentary successions of the Aniene River Valley, Rome. Quaternary Science Reviews 129, 260u2013274, (2015)._ _ _ Lisiecki, L. E. & Raymo, M. E. A Pliocene-Pleistocene stack of 57 globally distributed benthic u03B4O records. Paleoceanography 20(PA), 1003, (2005)._ _ _ Marra, F. et al. Independent Ar/Ar and 14C age constraints on the last five glacial terminations from the aggradational successions of the Tiber River, Rome (Italy). Earth Planet Sci Lett. 449, 105u2013117 (2016)._ _ _ _ Bridgland, D. R. & Westaway, R. Climatically controlled river terrace staircases: a worldwide Quaternary phenomenon. Geomorphology 98, 285u2013315 (2008)._ _ _ Ferranti, L. et al. Markers of the last interglacial sea-level high stand along the coast of Italy: tectonic implications. Quaternary International 145/146, 30u201354 (2006).Marra, F., Florindo, F. & Petronio, C. Quaternary fluvial terraces of the Tiber Valley: geochronologic and geometric constraints on the back-arc magmatism-related uplift in central Italy. Journal Scientific Reports 7, 2517, (2017)._ _ _ _ _ Sevink, J. et al. A sequence of marine terraces near Latina (Agro Pontino, central Italy). Catena 9, 361u2013378 (1982)._ _ Bigazzi, G., Bonadonna, F. P. & Iaccarino, S. Geochronological hypothesis on Plio-Pleistocene boundary in Latium region (Italy). Boll. Soc. Geol. It. 92, 391u2013422 (1973)._ _ Hearty, P. J., Miller, G. H., Stearns, C. E. & Szabo, B. J. Aminostratigraphy of Quaternary shorelines in the Mediterranean basin. Geological Society of America Bulletin 97, 850u2013858 (1986)._ _ _ Nisi, M. et al. Coastal deformation between the Versilia and the Garigliano plains (Italy) since the last interglacial stage. Journal of Quaternary Science 18(8), 709u2013 721 (2003)._ _ _ Marra, F., Karner, D. B., Freda, C., Gaeta, M. & Renne, P. R. Large mafic eruptions at the Alban Hills Volcanic District (Central Italy): chronostratigraphy, petrography and eruptive behavior. Journ. of Volc. and Geoth. Res. 179, 217u2013232, (2009)._ _ _ _ Kotsakis, T., Palombo, M. R. & Petronio, C. Mammuthus chosaricus e Cervus elaphus del Pleistocene superiore di Via Flaminia (Roma). Geol. Romana 17, 411u2013445 (1979)._ Foronova, I. V. Mammuthus intermedius (Proboscidea, Elephantidae) from the late Middle Pleistocene of the southern Western and Central Siberia, Russia: the problem of intermediate elements in the mammoth lineage. Russian Journal of Theriology 13, 71u2013 82 (2014)._ _ Lister, A. M. et al. The pattern and process of mammoth evolution in Eurasia. Quaternary International 126/128, 49u2013 64 (2005)._ _ _ Capasso Barbato, L., Cerilli, E. & Petronio, C. Differenze morfologiche e morfometriche nei crani di Ursus spelaeus e Ursus arctos. Il Quaternario 6(1), 67u201376 (1993)._ Petronio, C., Di Canzio, E. & Di Stefano, G. Morphological and biometrical differences in the limb bones of Ursus arctos and Ursus spelaeus and phylogenetical considerations on the two species. Palaeontographica Abt. A 269, 137u2013152 (2003)._ Salari, L. et al. The wolf from Grotta Mora Cavorso (Simbruini mountains, Latium) within the evolution of Canis lupus L., 1758 in the Quaternary of Italy. Palaeogeography, Palaeoclimatology, Palaeoecology 476, 90u2013105 (2017)._ _ _ Conti, N., Petronio, C. & Salari, L. The Late Pleistocene equids from u201CTana delle Ieneu201D (Ceglie Messapica, Brindisi, Southern Italy). Bollettino della SocietxE0 Paleontologica Italiana 49, 227u2013236 (2010)._ Pandolfi, L. & Marra, F. Rhinocerotidae (Mammalia, Perissodactyla) from the chrono-stratigraphically constrained Pleistocene deposits of the urban area of Rome (Central Italy). Geobios 48, 147u2013167 (2015)._ _ Pandolfi, L. & Petronio, C. A brief review of the occurrences of Pleistocene Hippopotamus (Mammalia, Hippopotamidae) in Italy. Geol. Croatica 68, 313u2013319 (2015)._ Di Stefano, G. & Petronio, C. A new Cervus elaphus of Middle Pleistocene age. Neues Jahrbuch fxFCr Geologie und PalxE4ontologie abhandlungen 190, 1u201318 (1993)._ Di Stefano, G. & Petronio, C. Origin and evolution of the European fallow deer (Dama, Pleistocene). N. Jb. Geol. PalxE4ont. Abh. 203(1), 57u2013 75 (1997)._ _ Di Stefano, G., Pandolfi, L., Petronio, C. & Salari, L. The morphometry and the occurrence of Cervus elaphus (Mammalia, Cervidae) from the Late Pleistocene of the Italian peninsula. Rivista Italiana di Paleontologia e Stratigrafia 121, 103u2013120 (2015)._ Di Stefano, G. Identification of fallow deer remains on the basis of its skeletal features: taxonomical considerations. Bollettino SocietxE0 Paleontologica Italiana 34, 323u2013331 (1995)._ Marra, F. et al. A review of the geologic sections and the faunal assemblages of Aurelian Mammal Age of Latium (Italy) in the light of a new chronostratigraphic framework. Quaternary Science Reviews 181, 173u2013199 (2018)._ _ _ Pandolfi, L., Petronio, C. & Salari, L. Catastrophic death assemblages from the Late Pleistocene of Italy: The case of Avetrana karst filling (Taranto, southern Italy). Riv. It. Paleont. Strat. 119, 109u2013124 (2013)._ Matolcsi, J. Historesche Erforschung der KxF6pergrxF6sse des Rindes auf Grund von ungarischen Knochen material. Zeitschrift fxFCr TierzxFCchtung und ZxFCchtungsbiologie 87, 89u2013137 (1969)._ _ Anzidei, A. P. & Gioia, P. The lithic industry at Rebibbia-Casal de Pazzi. Papers of the forth Conference of Italian Archaeology, New Developments in Italian Archaeology, Part one. Accordia Research Institute 156 (1990).Marra, F. et al. The Aggradational Successions of the Aniene River Valley in Rome: Age Constraints to Early Neanderthal Presence in Europe. PLoS ONE 12(1), e0170434, (2017)._ _ _ _ Gliozzi, E. et al. Biochronology of selected Mammals Molluscs and Ostracods from the Middle Pliocene to the Late Pleistocene in Italy The state of the art. Rivista Italiana di Paleontologia e Stratigrafia 103, 369u2013 388 (1997)._ Petronio, C., Bellucci, L., Martinetto, E., Pandolfi, L. & Salari, L. Biochronology and Palaeoenvironmental changes from the Middle Pliocene to the Late Pleistocene in Central Italy. Geodiversitas 33, 485u2013 517 (2011)._ _ Salari, L. & Masseti, M. Attardamenti olocenici di Equus hydruntinus Regalia, 1907 in Italia. In: Thun Hohenstein, U., Cangemi, M., Fiore, I. & De GrossiMazzorin, J. (Eds), Atti del 7xB0 Convegno Nazionale di Archeozoologia, Annali dellUniversitxE0 degli Studi di Ferrara. Museologia Scientifica e Naturalistica12, 313u2013320 (2016).Hearty, P. J. & Dai Pra, G. Aminostratigraphy of quaternary marine deposits in thew Lazio region of central Italy. Zeitschrift fur Geomoprph., Suppl. Bd. 62, 131u2013140 (1986)._ Antonioli, F., Silenzi, S., Vittori, E. & Villani, C. Sea level changes and tectonic stability: precise measurements in 3 coastlines of Italy considered stable during the last 125 ky. Physics and Chemistry of the Earth 24 (A), 337u2013342 (1999)._ _ _ Antonioli, F., Ferranti, L. & Kershaw, S. A glacial isostatic adjustment origin for double MIS 5.5 and Holocene marine notches in the coastline of Italy. Quaternary International 145, 19u201329 (2006)._ _ _ Antonioli, F. et al. Tidal notches in Mediterranean Sea: a comprehensive analysis. Quaternary Science Reviews 119, 66u201384 (2015)._ _ _ \"- - \"We are grateful to Serena De Francesco, Director at museum u201CAntiquarium Comunaleu201D in Nettuno, to Annalisa Zarattini and Micaela Angle of Soprintendenza Archeologia del Lazio e dellEtruria meridionale, for authorizing the study of the fossil remains and of the lithic industries, and the reproduction of photographs of the materials from Campoverde. We also thank Enrico Latini, discoverer of the Campoverde site, for precious information and field support. This research has been financially supported by the DTS-MIUR NextData project.\"- - Authors and Affiliations - \"Istituto Nazionale di Geofisica e Vulcanologia, Via di Vigna Murata 605, 00143, Rome, ItalyF. Marra_&_F. FlorindoUniversitxE0 di Roma u201CSapienzau201D , Dipartimento di Scienze della Terra, P.le Aldo Moro 5, 00185, Rome, ItalyC. Petronio,_G. Di Stefano_&_L. SalariFreelance, Tivoli, (Rome), ItalyP. CeruleoUniversitxE0 di Roma u201CTor Vergatau201D, Department of History, Humanities and Society, Via Columbia 1, 00133, Rome, ItalyM. Gatta_&_M. F. RolfoUniversity of York, Department of Archaeology, The Kings Manor, York, UKM. GattaEcomuseo dell Agro Pontino, Via G.B. Vico 15, Latina, ItalyM. La Rosa\" - \"AuthorsF. MarraYou can also search for this author in _C. PetronioYou can also search for this author in _P. CeruleoYou can also search for this author in _G. Di StefanoYou can also search for this author in _F. FlorindoYou can also search for this author in _M. GattaYou can also search for this author in _M. La RosaYou can also search for this author in _M. F. RolfoYou can also search for this author in _L. SalariYou can also search for this author in _\" - Contributions - F.M. designed the geologic study, performed the geomorphological study, and wrote the paper. C.P. performed the paleontological analyses and wrote the paleontological section. G.D.S. and L.S. performed the paleontological analyses and contributed to write the paleontological section. P.C., M.G., M.L.R. and M.F.R. performed the palethnological analyses and wrote the palethnological section. F.F. contributed to the writing of the text, discussed the results of the study and commented on the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing Interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution License .\\' - Plasmodium falciparum culture and growth inhibition assay - \"Plasmodium falciparum Dd2 (multidrug resistant), NF54 (chloroquine-sensitive) and 3D7 (chloroquine-sensitive/) strains were maintained at 37_xB0C in 5% CO2 using a modified Trager and Jensen method in fresh O+ human red blood cells at 3% hematocrit in RPMI culture media containing glutamax and NaHCO3 (Gibco, UK), supplemented with 25_mM HEPES (Gibco, UK), 1X hypoxanthine (Gibco, USA), 20_xB5g/mL gentamicin (Gibco, China), and 0.5% Albumax II (Gibco, USA). When needed, parasites were synchronized at the ring stage by sorbitol (5%) treatment and further cultivated through one complete cycle (48_h) prior to drug activity studies. Compounds in dimethyl sulfoxide (DMSO) were diluted in RPMI 1640 and mixed with parasite cultures (1% parasitemia and 1.5% hematocrit) in 96-well plates to a final drug concentration of 10_u03BCM for primary screening assays, or 10u20130.078_u03BCM for the doseu2013response assays. The final DMSO concentration per 100 u03BCL culture per well was 0.1%. Artemisinin at 1_xB5 M was used as positive drug control, while the solvent treated culture (0.1% DMSO) was used as negative drug control. Following 72_h incubation at 37_xB0C, parasite growth was assessed by a SYBR green I-based DNA quantification assay. Briefly, 80 xB5L of parasitized erythrocytes were transferred to a dark plate and 40 xB5L of SYBR green lysis buffer (3X) added to the plate. The plate was incubated in the dark for 30_min and fluorescence was measured using a Fluoroskan Ascent multi-well plate reader with excitation and emission wavelengths at 485 and 538_nm, respectively. Mean half-maximal effective concentrations (EC50 values) were derived by plotting percent growth inhibition against log drug concentration and fitting the response data to a variable slope sigmoidal curve-fit function using GraphPad Prism v5.0. EC50 values represent meansu2009xB1u2009 standard error from 2 independent assays. The inclusion of chloroquine, artemisinin or artesunate in the doseu2013response analyses served as experimental control.\" - Cytotoxicity analyses - \"Cytotoxicity of the compounds was carried out using the MTT viability assay method. Normal African Green Monkey Kidney Epithelial (Vero) and the Human Liposarcoma (SW872) cells were maintained in MEM supplemented with 10% FBS, 1% Glutamine and 1% penicillinu2013streptomycin. Confluent cells were trypsinized and seeded at a density of 2500 cells/well (total volume 90 u03BCL) in 96-well plates and incubated for 24_h prior to drug treatment. Two-fold serial dilutions of the compounds (100u20130.0488_u03BCM) were added to the plates and incubated under humidified conditions at 37_xB0C for 48_h. Viability of the cells was assessed by MTT (3-(4,5-dimethylthiazol-2-yl)-2,5-Diphenyltetrazolium Bromide) cell proliferation assay (Vybrant MTT Cell Proliferation Assay Kit V-1315) according to the manufacturers instructions. Absorbance of the formed formazan product was measured at 550_nm wavelength using a SUNRISE microtiter plate reader. Doseu2013response curves were plotted using GraphPad Prism v.5 and CC50 values were obtained. Selectivity indices (SIu2009=u2009CC50/EC50) were calculated as an indication of toxicity relative to the observed antiplasmodial activity.\" - Determination of cellular mode of action - \"The inhibitory effects of selected compounds and reference drugs on parasite development and merozoite egress were determined by quantitative light microscopy as described. Briefly, synchronized cultures (1.5% hematocrit and 3% parasitaemia) at different time-points (8, 30 and 42_h post invasion) were treated with either test compound or controls at 10_xB5M final concentration for 24_h under regular culture conditions.\" - Following treatment, Giemsa-stained thin smears were prepared and parasites were counted in 1,000 erythrocytes per treatment. Stage accumulation indices (stage proportion in test wells relative to same stage proportion in solvent control wells) were calculated and used to assess the in vitro effects of each compound on trophozoite and schizont development as well as merozoite egress. - Drug effect on hemoglobin degradation - \"To investigate drug effects on hemoglobin degradation, early rings (6u20138 hpi) or mid-trophozoite (20u201324 hpi) stage P. falciparum parasites (1.5% hematocrit, 5% parasitemia) were co-cultured with the test compound or experimental controls (0.1% DMSO, chloroquine, artemisinin and E64) at a final concentration of 10_xB5M. The cultures were incubated at 37_xB0C and 5% CO2 for 24_h after which the inhibitors were removed by centrifugation at 1800_rpm for 5_ min. The pellets were washed in an equal volume of 1X PBS and re-suspended in same volume of 0.1% saponin (in 1X PBS) for 3_min. Isolated parasites were pelleted at 2500_rpm for 5_min and washed twice in an_equal volume of 1X PBS. The resulting parasite pellets were lysed by treatment in 20_xB5L of 1% Triton X-100 (in 1X PBS) for 5_min. The hemoglobin content was measured at 550_nm using a nanodrop spectrophotometer. Increases in intra-parasite hemoglobin content were then calculated relative to the solvent-treated controls.\" - Drug effect on vacuolar lipid dynamics - The effect of the compounds on intracellular lipid dynamics was assessed by measurement of intracellular lipid content using Oil Red O. - \"Briefly, late trophozoite stage parasites (30 hpi) were treated with the compounds and controls for 24_h under regular culture conditions. Following treatment, inhibitors were removed by centrifugation at 2500_rpm for 5_min and pellets were washed using equal volumes of 1X PBS. The parasites were fixed at room temperature for 30_min with 4% paraformaldehyde and attached on poly-l-lysine coated coverslips at room temperature for 30_min. Three volumes of Oil Red O stock solution (5_mg/mL in 100% isopropanol) were diluted in two volumes of distilled water and filtered prior to use. The coverslips were then stained with diluted oil red O solution for 15_min at room temperature. Coverslips were washed thrice with 60% isopropanol and then distilled water, to remove excess dye. Thereafter, coverslips were air dried and mounted in DAPI Antifade reagent. Prepared slides were viewed under a 100xD7 oil immersion lens and images acquired.\" - \"Drugu2013drug combination analysis\" - \"Malaria parasites were continuously cultured and synchronized as earlier described prior to the assay. Drug interaction studies were performed as described. Briefly, two-fold drug dilutions were prepared using a variable potency ratio drug combination approach, starting at 5EC50A: 0EC50B to 0EC50A: 5EC50B in serum-free medium, where A and B represent the different partner molecules. The ring-stage parasitized erythrocytes (u223Cu200910 hpi) were diluted in complete medium to 1% parasitaemia and 1.5% hematocrit and 90 u03BCL was added in duplicate to 10 u03BCL of the drug dilution in a 96-well plate. The test was run for 72 h and terminated when the untreated parasites were at the early trophozoite stage of the second cycle. Parasite viability was assessed by the SYBR-Green I fluorescence-based assay as earlier described. Doseu2013response curves and EC50 values of each combination and drug alone were obtained using GraphPad Prism v.5.0 and Microsoft Excel was used to calculate mean EC50 values and the standard error of the mean.\" - The obtained EC50 values were used to calculate 50% fractional inhibitory concentrations (FIC50) and the combination indices (CI) were computed from the obtained FICs.- - \"Discovery of (xB1)-moxiquindole as an antiplasmodial compound\" - \"In an effort to discover antimalarial compounds that act on new cellular targets, a library of 44 hybrid compounds incorporating structural features of two well-known antimalarial scaffolds_(THIQ and oxindole) was tested against the CQ-sensitive 3D7 strain of P. falciparum. The SYBR green I-based DNA quantification assay was used to assess antiplasmodial activities and primary screening hits were defined as those inhibiting parasite growth by at least 50% when compared to the DMSO solvent control. Based on this criterion, 5 hits were identified (Fig._ ) and were subjected to doseu2013response studies on the same parasite strain. Compounds exhibiting complete doseu2013response curves with EC50 values less than 2_u03BCM were screened against the multidrug resistant Dd2 parasite strain as well as the chloroquine-sensitive strain NF54. Four of the primary hits (13c, 13d, 14h and 15c) did not produce complete doseu2013response curves within the 0.078_xB5M to 10_xB5M concentration range tested, and as such were considered inactive. One hit (14b, Fig._), subsequently referred to as (xB1)-moxiquindole, emerged as the most potent with EC50 values of 0.686_xB5M, 1.865_xB5M and 1.730_xB5M against PfNF54, Pf3D7 and PfDd2 strains, respectively (Table ).\" - \"Figure 3Identification of (xB1)-moxiquindole (14b) as most active Spiroindolone-Naphthylisoquinoline Hybrid. Dotplot indicating the ranks of each spiroindolone-naphthylisoquinoline hybrids tested in primary screens. The position of each assay well in the series (no drug, ART reference or test compound) is indicated on the x-axis.\" - Cytotoxicity testing of this compound against both Vero and SW872 cell lines, showed no evidence of toxicity as selectivity indices were greater than 50-fold (Table ), indicating in vitro safety against the tested cell lines. - \"Table 1 Antimalarial and cytotoxicity profiles of (xB1)-moxiquindole (MQN).\" - \"Fast-acting multistage activity of (xB1)-moxiquindole\" - \"The life cycle stage specific action of (xB1)-moxiquindole was determined by treatment of parasites at different time points followed by microscopy analysis of Giemsa-stained thin blood smears. Stage accumulation indices were calculated for each asexual parasite stage in the drug treated wells relative to the same stage in the negative control wells.\" - \"The results indicate that the test compound, like artemisinin and chloroquine, exhibits fast-acting inhibitory activities against ring and trophozoite development as revealed by more than five-fold ring and trophozoite accumulation indices (Table ). Treatment of parasites at the 42 hpi time-point resulted in a schizont accumulation index of 6.21, suggesting that (xB1)-moxiquindole, like the cysteine protease inhibitor E64 (l-trans-epoxysuccinyl-leucylamido-(4-guanidino), rapidly inhibits merozoite egress from the infected erythrocytes leading to accumulation of schizonts in culture. Similarly, artemisinin inhibited schizont rupture, while chloroquine had no effect, as confirmed by the appearance of ring stage parasites in the chloroquine treated cultures 24_h post-treatment at the late schizont stage (Fig._). Furthermore, unlike the chloroquine or artemisinin treated cultures, parasites exposed to (xB1)-moxiquindole showed characteristic morphological features of dying cells such as abnormal vacuolation (Fig._). Together, these findings suggest a novel mode of action for (xB1)-moxiquindole including the rapid induction of parasite vacuolation at all developmental stages tested.\" - \"Table 2 Stage-specific activities of (xB1)-moxiquindole determined by a Giemsa-based cytological profiling method.\" - \"Figure 4Representative images of Giemsa-stained thin smears showing drug effects on parasite life cycle progression when added at ring (6 hpi), trophozoite (30 hpi) or schizont (42 hpi). 0 hpt: treatment time point, 24 hpt: 24_h post-treatment. Arrow heads indicate unusual vacuolar structures in (xB1)-moxiquindole treated parasites.\" - \"(xB1)-Moxiquindole inhibits hemoglobin degradation in P. falciparum\" - \"To determine the effect of (xB1)-moxiquindole on hemoglobin uptake and break down in P. falciparum, tightly synchronized ring or trophozoite stage parasites were treated with test compounds for 24_h followed by parasite isolation and quantification of the intraparasitic hemoglobin contents. As shown in Fig._ , the hemoglobin content in (xB1)-moxiquindole-treated parasites was fourfold higher than the quantity in E-64 treated cultures when the test compound was added at the early ring stage, and 1.5-fold higher when added at the mid-trophozoite stage. Whereas artemisinin showed no detectable effect on hemoglobin accumulation in ring stage parasites, both artemisinin and chloroquine induced hemoglobin accumulation to the same degree as E-64 in trophozoite parasites. Together, these findings suggest that (xB1)-moxiquindole is a more effective accumulator of intra-parasitic hemoglobin in P. falciparum, an activity that might occur by inhibition of hemoglobin uptake and hydrolysis in the parasites food vacuole.\" - \"Figure 5Drug effect on hemoglobin uptake and hydrolysis. P. falciparum rings (A) or trophozoites (B) were treated for 24_h and the hemoglobin content of saponin-isolated parasites determined by spectrophotometry. Data show fold changes in hemoglobin content of drug-treated parasites relative to E-64 treated parasites.\" - \"(xB1)-Moxiquindole disrupts vacuolar lipid dynamics\" - \"Because cellular vacuolation may result from perturbation of intracellular lipid content, we investigated the effect of (xB1)-moxiquindole on vacuolar lipid dynamics using fluorescent Oil red O as lipid stain. As shown in Fig._, Oil red O distinctly stained intracellular spots in trophozoite and schizont stage parasites in untreated cultures as well as in artemisinin and E-64 treated cultures, but not in chloroquine and (xB1)-moxiquindole treated parasites. Together, these findings suggest that both chloroquine and (xB1)-moxiquindole may exhibit antiplasmodial activity by disrupting lipid deposition at intracellular sites proximal to the parasite digestive vacuole.\" - \"Figure 6Unusual effect of (xB1)-moxiquindole and chloroquine on vacuolar lipid dynamics. Parasites were treated at the late-trophozoite stage (30 hpi) for 24_h and the intracellular lipid deposits were stained with Oil red O. Arrow heads indicate intracellular sites with intense staining by Oil red O. No intense staining was observed in chloroquine and (xB1)-moxiquindole treated cultures, suggesting an effect on vacuolar lipid dynamics.\" - \"(xB1)-Moxiquindole interferes with chloroquine and artemisinin actions in vitro\" - \"Given the overlapping modes of action between (xB1)-moxiquindole and some known antimalarial compounds in terms of speed and targeted cellular processes, we investigated the interaction between these compounds at variable combination ratios. As presented in Table (cf isobolograms in Fig._), (xB1)-moxiquindole exhibited antagonistic interactions with both chloroquine (CI: 1.478) and artemisinin (CI: 1.440), similar to the antagonism observed between chloroquine and artemisinin (CI: 1.183). These results suggest possible interaction between (xB1)-moxiquindole and both chloroquine and artemisinin in terms of inhibited cellular processes or intracellular targets.\" - \"Table 3 Antagonistic interactions between (xB1)-moxiquindole and representative antimalarial compounds.\" - \"Figure 7Isobolograms of the in vitro interactions between moxiquindole with chloroquine or artemisinin at fixed ratio. (A) Control ART/ART; (B) Control CQ/ART; (C) MQN/CQ; (D) MQN/ART. CQ: Chloroquine; ART: Artemisinin; MQN: (xB1)-Moxiquindole. FIC50u2009=u2009Fractional Inhibitory Concentration 50. A concave curve represents a synergistic interaction, a convex is consistent with an antagonistic interaction and a straight line is consistent with an additive interaction. Axes are EC50s normalized to 1.\"- - \"The emergence and eventual spread of resistance to artemisinin-based monotherapy led to the adoption of artemisinin-based combination therapy (ACT) as the frontline treatment strategy for malaria. Unfortunately, there are increasing and worrying reports of delayed parasite clearance following the administration of artemisinin in combination with partner drugs, suggesting a slow but inexorable emergence of resistance. To address this global problem, new antimalarial drug candidates with novel mechanisms of action have to be identified and developed. The hybridization of privileged scaffolds offers a potentially useful strategy for the discovery of compounds with novel mechanisms of action. A useful example in this regard is the 2u2032,3u2032,4u2032,9u2032-tetrahydrospiro[indoline-3,1u2032-pyrido[3,4-b]indol]-2-one scaffold, constructed from two privileged scaffolds, 1,2,3,4-tetrahydro-u03B2 -carboline (THu03B2C) and oxindole (OX), which has given rise to the spiroindolones, exemplified by NITD609 (KAE609, cipargarmin). Compounds of this group constitute a novel class of antimalarials which act by inhibition of Pf Na ATPase, a mode of action that is hitherto unknown in any other oxindole or THu03B2C. Borrowing from this experience, our attention was drawn to the recently described 3u2032 ,4u2032-dihydro-2u2032H-spiro[indoline-3,1u2032-isoquinolin]-2-ones (DSIIQs) which are also constructed from two privileged scaffolds: oxindole (OX) and 1,2,3,4-tetrahydroisoquinoline (THIQ), and are therefore structurally similar to the spiroindolones and the THIQs such as the naphthylisoquinolines (NIQs). Screening of a small library of DSIIQs against CQ-sensitive Pf NF54 and Pf 3D7 strains, and multidrug resistant Pf Dd2 parasites resulted in the identification of a hit. The compound, subsequently referred to as (xB1)-moxiquindole, displayed potent inhibitory activity against all selected strains (EC50 Pf NF54u2009 =u20090.686_u03BCM; EC50 Pf 3D7u2009=u20091.87_xB5M; EC50 for Pf Dd2u2009 =u20091.73_xB5M), suggesting no cross-resistance with known antimalarials including pyrimethamine and chloroquine, and prompting further investigation of this compound as a representative of the DSIIQ structural class.\" - \"As a counter screen, the cytotoxicity of this compound on Normal African Green Monkey Kidney Epithelial (Vero) and the Human Liposarcoma (SW872) cell lines was evaluated using an MTT cell proliferation assay. The CC50 value of (xB1 )-moxiquindole against both cell lines wasu2009>u2009100_u03BCM, resulting in over 50-fold selectivity (Table ). Subsequently, the developmental stage specific action of (xB1)-moxiquindole was studied by light microscopy. Following the invasion of erythrocytes, malaria parasite merozoites attain maturity by passing through a series of developmental stages termed ring, trophozoite, schizont, and segmenter. Antimalarials exert their actions by interfering with one or more stages in the life cycle of the parasite; therefore, knowledge of the stage specific action of a test compound can provide useful insights into the mode of action. In the current investigation, the mode of action of (xB1)-moxiquindole was compared with two antimalarials in use (ART and CQ) and E64 (a cysteine protease inhibitor), by determining their inhibitory effects on the intraerythrocytic stages of the parasite, schizont rupture and merozoite invasion. In a previous study of all antimalarials then in use, only artemisinin displayed significant inhibition of schizont maturation but no compound inhibited merozoite invasion. Consistent with these reports, the present study found that artemisinin has a significant inhibitory effect on schizont maturation and eventual merozoite egress, as the schizonts appeared dead compared to the viable rings in the untreated controls. Chloroquine on the other hand, had no effect on schizont rupture as revealed by the presence of rings 24_h following drug treatment. The test compound rapidly arrested parasite development at the early ring, mid trophozoite, and schizont stages (Fig._). The presence of schizonts following a 24-h treatment with (xB1)-moxiquindole contrasts sharply with the presence of rings in the untreated controls and is indicative of an arrest in schizont rupture. Merozoite egress, which is preceded by schizont rupture, is a protease-dependent sequence of membrane permeabilization and eventual breakage; therefore it is reasonable to attribute the blockade of schizont rupture by (xB1)-moxiquindole to protease inhibition. Consistent with this suggestion, the cysteine protease inhibitor E64 was also found to arrest merozoite egress in the present study (Fig._) as expected from previous reports. Proteases have been implicated in a wide range of malaria parasite metabolic processes including hemoglobin degradation; therefore, additional investigations were carried out in order to gain a better understanding of the actions of the test compound. Indeed, the antimalarial activity of quinoline-based antimalarials is attributed to their interference with hemoglobin metabolism either by inhibiting the uptake of hemoglobin by the parasite, or by inhibiting degradation of hemoglobin, or by binding the toxic hematin and preventing hemozoin formation, all of which result in the eventual death of the parasite. In the present study, both ring and trophozoite stage parasites treated with (xB1)-moxiquindole were found to accumulate hemoglobin to a greater extent than parasites treated with either chloroquine or artemisinin (Fig._). Although hemoglobin degradation is seen during schizont development, a vast majority of degradation occurs during trophozoite_development. As expected, hemoglobin accumulation was higher in drug-treated_ring_stage parasites compared to drug-treated trophozoite stage parasites (Fig._). In addition, reports have shown that hemoglobin degradation is an ordered process initiated by aspartic-protease cleavages, followed by cysteine protease action. As such, the effect of cysteine protease inhibitors is minimal compared to the effect of aspartic protease inhibitors as far as hemoglobin degradation is concerned. Consistent with these reports, E64; a known cysteine protease inhibitor demonstrated a minimal effect on hemoglobin degradation compared to the test compound. These observations further support the view that (xB1)-moxiquindole is indeed acting as a protease inhibitor and a more effective accumulator of intra-parasitic hemoglobin; thus, depriving the organism of an essential source of substrates for energy metabolism.\" - \"Further support for this proposed mode of action is provided by the combination studies which paired (xB1)-moxiquindole with either chloroquine or artemisinin. As presented above, the interaction of the test compound with either antimalarial was antagonistic (Table ). The actions of both chloroquine and artemisinin depend on the presence of heme, a metabolite generated by protease catalyzed degradation of hemoglobin. Drug induced blockade of the latter process, with resulting depletion of heme levels, would therefore be expected to negatively affect the actions of these two compounds as observed in the present study. Similar observations have been reported in a study of the interaction between chloroquine and both aspartic and cysteine proteases using the same analytic methods.\" - \"Previous studies have showed that during intraerythrocytic development, the phospholipid content of the parasite increases significantly as the latter accumulates membranes needed for growth and division. Though some of the lipids are synthesized in the apicoplast, the bulk of the lipids are scavenged from the host. These lipids are metabolised to generate neutral lipid bodies which nucleate the formation of hemozoin in the parasite digestive (food) vacuole. Consequently, the lipid metabolic pathway is an important target in antimalarial drug discovery. In the continuing investigation of the mode of action of (xB1)-moxiquindole, the actions of this compound on lipid metabolism were compared with those of chloroquine, artemisinin and E64. Trophozoite stage parasites treated with either, artemisinin or E64 clearly revealed the presence of stained neutral lipid bodies indicating that these compounds have no effect on vacuolar lipid uptake. On the other hand, parasites treated with either chloroquine or the test compound were devoid of neutral lipid bodies clearly suggesting interference with parasite vacuolar lipid dynamics (Fig._). This supports previous findings that chloroquine and other quinoline-based compounds bind toxic ferriprotoporphyrin IX and inhibit dimerization.\" - \"In conclusion, preliminary investigation of (xB1)-moxiquindole, a representative compound of the recently described privileged scaffold hybrid 3u2032,4u2032 -dihydro-2u2032H-spiro[indoline-3,1u2032-isoquinolin]-2-one (DSIIQ), strongly suggests that the antimalarial activity of the compound can be attributed to (1) protease inhibition, which results in the blockade of hemoglobin degradation with subsequent starvation of the parasite, and (2) interference with parasite lipid dynamics. The DSIIQs are structurally related to both the spiroindolones and the NIQs. Previous investigations have attributed the antimalarial activity of NIQs to interference with hemoglobin metabolism. Based on the results discussed above, it would therefore appear that at least some of the actions of (xB1 )-moxiquindole are similar to those of the NIQs. Subsequent studies will compare the actions of this compound with those of the structurally related spiroindolones and concentrate on improving the antimalarial activity by modification of chemical structure. The available evidence suggests, nevertheless, that this new hybrid scaffold has the potential to yield an interesting group of antimalarial agents.\"- - \"World Health Organization. World malaria report 2019. Report No.: ISBN 978-92-4-156572-1.Heller, L. E. & Roepe, P. D. Artemisinin-based antimalarial drug therapy: molecular pharmacology and evolving resistance. Trop. Med. Infect. Dis. 4, 89. (2019)._ _ _ WHO. World malaria report 2018 (2018).Chanaki, A. et al. Dihydroartemisininu2013piperaquine resistance in Plasmodium falciparum malaria in Cambodia: a multisite prospective cohort study. Lancet Infect. Dis. 16, 357u2013365. (2016)._ _ _ Takala-Harrison, S. et al. Independent emergence of artemisinin resistance mutations among Plasmodium falciparum in Southeast Asia. J. Infect. Dis. 211, 670u2013679. (2014)._ _ _ _ _ Tun, K. M. et al. Spread of artemisinin-resistant Plasmodium falciparum in Myanmar: a cross-sectional survey of the K13 molecular marker. Lancet Infect. Dis. 15, 415u2013421. (2015)._ _ _ _ _ Evans, B. E. et al. Methods for drug discovery: development of potent, selective, orally effective cholecystokinin antagonists. J. Med. Chem. 31, 2235u20132246. (1988)._ _ _ _ Tibon, N. S., Ng, C. H. & Cheong, S. L. Current progress in antimalarial pharmacotherapy and multi-target drug discovery. Eur. J. Med. Chem. 188, 111983. (2020)._ _ _ _ Pedrosa, M. O. et al. Hybrid compounds as direct multitarget ligands: a review. Curr. Top. Med. Chem. 17, 1044u20131079. (2017)._ _ _ Rottmann, M. et al. Spiroindolones, a potent compound class for the treatment of malaria. Science 329, 1175u20131180. (2010)._ _ _ _ _ _ Turner, H. Spiroindolone NITD609 is a novel antimalarial drug that targets the P-type ATPase PfATP4. Future Med. Chem. 8, 227u2013238. (2016)._ _ _ _ Ibrahim, S. R. M. & Mohamed, G. A. Naphthylisoquinoline alkaloids potential drug leads. Fitoterapia 106, 194u2013225. (2015)._ _ _ _ Li, J. et al. Dioncophyllines C2, D2, and F and related Naphthylisoquinoline alkaloids from the congolese liana Ancistrocladus ileboensis with potent activities against Plasmodium falciparum and against multiple myeloma and leukemia cell lines. J. Natl. Prod. 80, 443u2013458. (2017)._ _ _ Fayez, S. et al. Ancistrobrevines E-J and related naphthylisoquinoline alkaloids from the West African liana Ancistrocladus abbreviatus with inhibitory activities against Plasmodium falciparum and PANC-1 human pancreatic cancer cells. Fitoterapia 131, 245u2013259. (2018)._ _ _ _ FranxE7ois, G. et al. Naphthylisoquinoline alkaloids against malaria: evaluation of the curative potentials of Dioncophylline C and Dioncopeltine A against Plasmodium berghei in vivo. Antimicrob. Agents Chemother. 41, 2533u20132539 (1997)._ _ FranxE7ois, G. et al. Retarded development of exoerythrocytic stages of the rodent malaria parasite Plasmodium berghei in human hepatoma cells by extracts from Dioncophyllaceae and Ancistrocladaceae species. Int. J. Parasitol. 27, 29u201332. (1997)._ _ _ FranxE7ois, G. et al. Differential sensitivity of erythrocytic stages of the rodent malaria parasite Plasmodium chabaudi chabaudi to dioncophylline B, a highly active naphthylisoquinoline alkaloid. Parasitol. Res. 85, 935u2013941 (1999)._ _ Lobe, M. M. M. & Efange, S. M. N. 3u2032,4u2032-Dihydro-2u2032H-spiro[indolin-3:1u2032-isoquinolin]-2-ones as potential anticancer agents: synthesis and preliminary screening. R. Soc. Open Sci. 7, 191316. (2020)._ _ _ _ _ _ Trager, W. & Jensen, J. B. Human malaria parasites in continuos culture. Science 193, 673u2013675. (1976)._ _ _ _ _ Smilkstein, M., Sriwilaijaroen, N., Kelly, J. X., Wilairat, P. & Riscoe, M. Simple and inexpensive fluorescence-based technique for high-throughput antimalarial drug screening. Antimicrob. Agents Chemother. 48, 1803u20131806. (2004)._ _ _ _ _ Lee, S. et al. Discovery of carbohybrid-based 2-aminopyrimidine analogues as a new class of rapid-acting antimalarial agents using image-based cytological profiling assay. J. Med. Chem. 57, 7425u20137434. (2014)._ _ _ _ Bracken, F. R. et al. 4-Nitro styrylquinoline is an antimalarial inhibiting multiple stages of Plasmodium falciparum asexual life cycle. Int. J. Parasitol. Drugs Drug Resist. 7, 120u2013129. (2017)._ _ Tarkang, P. A. et al. In vitro antiplasmodial activities and synergistic combinations of differential solvent extracts of the polyherbal product, nefang. Biomed. Res. Int. (2014)._ _ Thomas, J. A. et al. A protease cascade regulates release of the human malaria parasite Plasmodium falciparum from host red blood cells. Nat. Microbiol. 3, 447u2013455. (2018)._ _ _ _ _ Hale, V. L. et al. Parasitophorous vacuole poration precedes its rupture and rapid host erythrocyte cytoskeleton collapse in Plasmodium falciparum egress. PNAS 114, 3439u20133444. (2017)._ _ _ _ Wilson, D. W., Langer, C., Goodman, C. D., McFadden, G. I. & Beesona, J. G. Defining the timing of action of antimalarial drugs against Plasmodium falciparum. Antimicrob. Agents Chemother. 57, 1455u20131467. (2013)._ _ _ _ _ Lehmann, C. et al. Plasmodium falciparum dipeptidyl aminopeptidase 3 activity is important for efficient erythrocyte invasion by the malaria parasite. PLoS Pathog. 14, e1007031. (2018)._ _ _ _ Prasad, R. et al. Blocking Plasmodium falciparum development via dual inhibition of hemoglobin degradation and the ubiquitin proteasome system by MG132. PLoS ONE 8, e73530. (2013)._ _ _ _ _ _ Hoppe, H. C. et al. Antimalarial quinolines and artemisinin inhibit endocytosis in Plasmodium falciparum. Antimicrob. Agents Chemother. 48, 2370u20132378. (2004)._ _ _ _ _ Herraiz, T., GuillxE9n, H., GonzxE1lez-PexF1a, D. & ArxE1 n, V. J. Antimalarial quinoline drugs inhibit u03B2-hematin and increase free hemin catalyzing peroxidative reactions and inhibition of cysteine proteases. Sci. Rep. 9, 15398. (2019)._ _ _ _ _ _ Francis, S. E., Sullivan, D. J. & Goldberg, D. E. Hemoglobin metabolism in the malaria parasite Plasmodium falciparum. Annu. Rev. Microbiol. 51, 97u2013 123 (1997)._ _ _ Mungthin, M., Bray, P. G., Ridley, R. G. & Ward, S. A. Central role of hemoglobin degradation in mechanisms of action of 4-aminoquinolines, quinoline methanols, and phenanthrene methanols. Antimicrob. Agents Chemother. 42, 2973u20132977 (1998)._ _ _ Wijayanti, M. A. et al. Additive in vitro antiplasmodial effect of N-alkyl and N-benzyl-1, 10-phenanthroline derivatives and cysteine protease inhibitor E64. Malar. Res. Treat. (2010)._ _ _ _ Jackson, K. E. et al. Food vacuole-associated lipid bodies and heterogeneous lipid environments in the malaria parasite, Plasmodium falciparum. Mol. Microbiol. 54, 109u2013122. (2004)._ _ _ _ Fitch, C. D., Chen, Y. & Cai, G. Chloroquine-induced masking of a lipid that promotes ferriprotoporphyrin IX dimerization in malaria. J. Biol. Chem. 278, 22596u2013 22599 (2003)._ _ _ \"- - Funding for the study was provided by the Ministry of Higher Education under the Modernization of Research Scheme. We thank the H3D Drug Discovery & Development Centre, University of Cape Town, South Africa for screening the compounds for antimalarial activity on the PfNF54 strain.- - Authors and Affiliations - \"Department of Biochemistry and Molecular Biology, University of Buea, P.O. Box 63, Buea, CameroonNoella M. EfangeCentre Pasteur du Cameroon, YaoundxE9, CameroonNoella M. Efange,_Rodrigue Keumoe_&_Lawrence AyongDepartment of Chemistry, University of Buea, P.O. Box 63, Buea, CameroonMaloba M. M. Lobe_&_Simon M. N. Efange\" - \"AuthorsNoella M. EfangeYou can also search for this author in _Maloba M. M. LobeYou can also search for this author in _Rodrigue KeumoeYou can also search for this author in _Lawrence AyongYou can also search for this author in _Simon M. N. EfangeYou can also search for this author in _\" - Contributions - \\'Chemistry: S.M.N.E.: Designed the compounds; supervised the synthesis; drew chemical structures (Fig. ); wrote some portions of the manuscript and reviewed the completed manuscript. M.M.M.L.: Synthesized, purified and characterized the test compounds. Biology: N.M.E.: Performed some of screening for antimalarial activity; life cycle stage specificity experiments; cytotoxicity studies, hemoglobin accumulation, lipid dynamics and drug combination studies. Wrote part of the manuscript. R.K.: Performed some of the screening and stage specificity assays. L.A.: Designed and supervised the biological experiments; oversaw the data analysis and interpretation; Wrote part of the manuscript and reviewed the completed work.\\' - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution License 3.0 (https://creativecommons.org/licenses/by/3.0/).\\' - The specific codenitrification pathway is unknown, but fungi reportedly reduce NO2 to NO using NO2 reductase (encoded by the nirK gene) and then reduce NO to N2O using nitric oxide reductase (P450nor). The role of N2O in the codenitrification process and in N2 formation is not clear. While utilisation of N2O to form N2 has been suggested as a plausible codenitrification pathway, reports of N2O consumption during fungal production of N2 (commonly observed during bacterial denitrification) are lacking. Potentially bypassing reduction of N2O to form N2, in addition to formation of hybrid N2, sets codenitrification and anammox apart from classical denitrification. - \"Laboratory studies commonly report evidence of fungal denitrification or codenitrification when pure cultures are incubated under anaerobic or microaerophilic conditions with sterile media, consisting of carbon, NO2 and mineral salts. However, reduced metals in the medium, such as Fe(II), could provide electrons required for abiotic reduction of NO2 to N2O, commonly known as chemodenitrification. Chemodenitrification occurs through nitrosylation when reduced forms of inorganic N react with a metal centre to form N2O in the absence of oxygen. Nitrosylation may also drive abiotic formation of N2, given high concentrations of metal and NO2u2009. Wullstein and Gilmour mixed 10,000 ppm N as potassium nitrite (KNO2) with 5,000 ppm ferrous sulfate in an abiotic, anoxic reactor and recovered 15% of added N as N2 within 3 d. In this case, the N2 formed would have been denitrified from a single N source, KNO2. Alternatively, chemical formation of hybrid N2 through nitrosation is often ignored by biologists. Abiotic nitrosation of organic matter by NO2 in soil was first suggested by Nelson and Bremner when they recovered over 20% of added N (5 mmols NO2u2009g soil) as N2 for sterile soil at neutral pH in helium (He) and heliox (20% O2, 80% He) atmospheres. The isotopic composition of N2 was not reported, but they indicated soil organic matter was an important factor. Trimmer and Prudy showed that deep seawater samples amended with NO2 and NH4 produced more N2 when organic N (allylthiourea) was added. They suggested an alternative metabolic pathway to anammox but did not address the possibility of abiotic N2 formation. Babbin et al. also found organic N enhances anammox N2 production. A common thread among chemical, anammox and fungal denitrification studies is NO2. Thus, NO2 is a pivot-point for divergence in biological and chemical N trace gas production ().\" - Here, we pursue open questions raised by this early work regarding abiotic N trace gas production recently reviewed by Heil et al.. We aimed to investigate if previously unexplored sources of N2O and N2 could be contributing to reactive N removal and if N2O was an intermediate in the abiotic N2-production pathway. We questioned whether N2 reportedly due to fungal codenitrification in pure culture experiments was formed abiotically in the presence and absence of O2, given diverse inorganic and organic sources of N. New knowledge of abiotic N2O and N2 production would advance environmental N-removal research and applications and perhaps explain some mass balance discrepancies found in isotopic pairing studies. - \"To address these questions, we used a ubiquitous soil fungus, Bipolaris sorokiniana (Sacc.) Shoemaker [telemorph: Cochliobolus sativus (S. Ito & Kurib.) Drechsler ex Dastur], as our model. Previously, we found that B. sorokiniana used NO2 as the sole source for denitrification to N2O under anaerobic and microaerophilic conditions. We also found NO2 as the sole source for chemodenitrification, which accounted for 6u20138% of total N2O production. These results prompted further inquiry regarding fungal and chemical trace gas production of N2. We aimed to more explicitly evaluate abiotic and biotic N2O and N2 production by comparing live fungi with fungal necromass incubated under strictly sterile conditions, with and without O2. Pure culture experiments have demonstrated fungal and chemical N2O production, but reports are lacking that indicate abiotic nitrosation of organic compounds and formation of hybrid N2. We include necromass in the design because amino acids and other nucleophilic compounds from necromass could potentially, in the absence of live fungi, react with NO to form N2. Further, necromass would provide more surface area for chemical decomposition of NO2 to N2O (). This would serve as a test for abiotic formation of N2O and N2 in the presence of decomposing organic material.\" - \"We aimed to assess biotic and abiotic N2O and N2 production using established microbiological laboratory incubation methods (). We exposed live fungi and necromass to both inorganic and organic N sources [0.25u2009mmolu2009N as sodium nitrite (NaNO2) and 0.25u2009mmolu2009N as glutamine (C5H10N2O3); concentration of 20u2009mmolu2009L each] under two O2 conditions (anaerobic and 20% O2). For each treatment, there were replicate sets where N was not added to the media (No N control). We also included sterile media-only control vessels. Oxygen status was tightly controlled with airtight laboratory incubation vessels, where headspace was filled with either helium (anaerobic) or heliox (aerobic). Accumulation of headspace O2, CO2, N2O, and N2 were measured approximately every 6u2009 h with a customised, robotic gas chromatography system. Headspace O2 was monitored to confirmed anaerobiosis was maintained during the 30u2009h incubation. Slopes of the linear increases in N2O and N2 in the headspace of each vessel were calculated to determine production rates. We used ANOVAs to test if production of N2O and N2 by live fungi was similar to necromass, and if both groups responded similarly to O2. Production rates were calculated in units of u03BCmol N as N2O and N2 g fungal biomass h and in units of u03BCmol N as N2O and N2 h. We performed a second incubation (illustration of this incubation design not shown) using isotope pairing techniques to determine if N2O or N2O were produced abiotically from sterile medium amended with glutamine and NO2 in the presence or absence of O2. We used equal amounts of unlabelled glutamine and N-labelled NO2 to achieve 0.5u2009mmolu2009N and 1.0u2009mmolu2009N, as well as a medium that was not amended with N.\" - \\'Figure 2Experimental set-up to test how live fungi, necromass and media only incubated in an anoxic and 20% O2 atmosphere affects production of N2O, CO2, O2, and N2 following addition of both organic and inorganic forms of N to pure cultures under aseptic conditions.Terms are defined as: Glut, glutamine; NaNO2, sodium nitrite; He, helium; HeOx heliox.\\'- - \"We found both fungal states (live fungi and necromass) produced N2O and N2 but only for those samples amended with N. Linear production of N2O and N2, following N amendment, occurred quickly under completely anaerobic conditions and in a 20% O2 atmosphere (). Average [(standard deviation (SD)] rate of N2O produced by necromass was 0.012 (<0.001) u03BCmol N2O g biomass h under both aerobic and anaerobic conditions. Average N2O production by live fungi was 0.017 (<0.001) u03BCmol N2O g biomass h and 0.028 (0.003) u03BCmol N2O g biomass h, respectively, under aerobic and anaerobic conditions (). Rates of N2O production were significantly greater for live fungi incubated anaerobically (O2u2009xD7u2009fungal state interaction; pu2009<u20090.001), indicating biological production of N2O in the absence of O2. In a 20% O2 atmosphere, rates of N2O production were similar to necromass. Our previous study indicated B. sorokiniana produced N2O at similar rates when incubated anaerobically and in a 0.4% O2 atmosphere. Here, we used 20% O2, and we found no evidence of microbial denitrification under these high-O2 conditions.\" - \"Figure 3Kinetics of (a) N2O and (b) N2 production per g biomass over time following N addition at each time point as gases in the headspace accumulated. Average for each treatment are slightly staggered in time due to robotized measurement system, which is why comparisons were based on slopes over the entire incubation. Production rates of N2O and N2 were strongly affected by O2 status [O2u2009 xD7u2009fungal state (live or dead); pu2009<u20090.001] for those samples amended with N. Boxes represent average data; error bars, s.d.; nu2009=u2009 4.\" - \" provides a basis of comparison for N2O recovered in the headspace of sterile media using comparable units (u03BCmol N2O h). These data were not normalised per g of fungal biomass but illustrate accumulation of N2O in sterile medium relative to necromass and live fungi. Nitrous oxide production rates were greater for necromass, as compared to media only (). Necromass produced an average of 0.0034u2009u03BCmol N2O h at both O2 levels, as compared to 0.0011u2009u03BC mol N2O h for sterile media. Greater N2O for necromass suggests the presence of decaying biomass provided greater surface area for chemodenitrification ().\" - \"Figure 4Abiotic and biotic contributions to rates of (a) N2O and (b) N2 accumulated per hour following N addition under both aerobic and anaerobic conditions for sterile medium, necromass and live fungi. Boxes represent the 90th percentile data; error bars, s.d.; nu2009=u20094. Median values are the lines horizontally bisecting each box.\" - \"Rates of fungal N2O production observed here are lower than other reports in the literature. Maeda et al. found fungi carrying the nirK gene produced from 0.1 to 3.2 u03BCmol N2O g biomass h, and Rohe et al. reported fungal N2O production rates from 0.05 to 14 u03BCmols N2O h. In both cases, denitrification varied with fungal species and N amendment. Here, B. sorokiniana does not carry the nirK gene, which may influence NO2 reduction to NO. Data on chemical formation of N2O under oxic (20% O2) conditions are lacking, so these first results challenge the paradigm that chemodenitrification requires anoxia. Contrary to chemostat studies, we included fungal necromass, which contributed to abiotic N2O. Results call into question if all N2O produced in pure culture experiments is enzymatically mediated if cultures include live and dead fungal or bacterial biomass.\" - \"We found no evidence of biological N2 production in the presence or absence of necromass (). Lowest rates of N2 production were observed for live fungi incubated in a 20% O2 atmosphere (O2u2009xD7u2009fungal state interaction; pu2009 <u20090.001). Rates of N2 produced in the headspace of live fungi were 0.243 (0.039) u03BCmol N2 g biomass h and 0.424 (0.049) u03BCmol N2 g biomass h under aerobic and anaerobic conditions, respectively. Rates of N2 produced in the headspace of necromass were 0.415 (0.041) u03BCmol N2 g biomass h and 0.356 (0.047) u03BCmol N2 g biomass h under aerobic and anaerobic conditions, respectively. Our results indicate there is a strong abiotic component to measured rates of N2 production and challenge the assumption that fungal N2 production requires anoxic or microaerophilic conditions.\" - \"Most of the N2 accumulated in the headspace for live B. sorokiniana was commensurately found in the headspace of B. sorokiniana necromass (), which points to chemical N2 formation. We show chemical production of N2 (u03BCmol h) for sterile medium relative to necromass and live fungi at both O2 levels in . Overall average rates of N2 formation were 0.12 (0.039) u03BCmol N2 h for sterile medium, 0.11 (0.024) u03BCmol N2 h for necromass, and 0.08 (0.018) u03BCmol N2 h for live fungi. Others have reported much higher rates of anoxic chemical N2 formation (7u201332 u03BCmol N2 h) at neutral pH when high concentrations of reduced metals were reacted with high concentrations of NO2 and when high NO2 solutions (20u2009M) were added to autoclaved soil, but these did not report if hybrid N2 was formed or not. Comparable rates of abiotic N2 production for necromass and sterile medium caused us to question if abiotic N2, like N2O, could result from nitrosylation of inorganic N only or from nitrosation to form hybrid N2. Organic N has been found to increase biological production of N2 in oceanic studies, but our results () indicated there may also be abiotic processes that contribute strongly to N2 production. Consequently, we determined if both inorganic and organic N were used to produce N2 abiotically in a separate, isotope pairing experiment.\" - \"The headspace above sterile media incubated under oxic and anoxic conditions with N-labelled NaNO2 and unlabelled C5H10N2O3 indicated abiotic, formation of hybrid N2. Almost all (99.9%) of the N2 produced was N2 (). The N2 production was proportional to the mass of added N and approximately 3u20134% of the total N added was transformed to N2 under both oxic and anoxic conditions and at both levels of NO2 addition. Results demonstrate hybrid formation of N2 is not necessarily enzymatically mediated and does not required anoxia. Formation of abiotic, hybrid N2 by two distinct inorganic N molecules has not been ruled out here. Previous bodies of work that use formation of N2 as evidence of anammox and/or codenitrification need to be reviewed with respect to abiotic N2 production.\" - \"Table 1 Average (SD) of N2 and N2 recovered in the headspace following aerobic and anaerobic incubation of sterile media at three levels of N addition, where added N comprised 50% N from unlabelled C5H10N2O3 and 50% N from labelled N-NaNO2 (nu2009=u20095).\" - Linear increases in cumulative N2O and N2 over time under aerobic and anaerobic conditions for both live fungi and necromass are shown in . These data suggest that N2O was not consumed during the incubation and both gases were produced independently. This finding contrasts with bacterial denitrification, where a sharp rise in microbial production of N2O is followed by a rise in biological reduction of N2O to N2, and a drop in cumulative N2O production. Bipolaris sorokiniana denitrifies NO2 only and does not use glutamine to form N2O. If the pathway to N2 were through the intermediate N2O, we would expect not only N2O consumption but also accumulation of N2 in the isotopic pairing experiment. Like anammox, our abiotic N2 production results indicate N2O formation was bypassed in the pathway to N2 (). - \"We show compelling evidence that formation of N2 does not result from solely biotic nitrosation but also abiotic nitrosation. Abiotic nitrosation occurred both in the presence and absence of O2, and abiotic N2 was formed exclusively through hybridisation of inorganic and organic N sources and did not require the N2O intermediary. Experimental protocols, including N concentrations, were in accordance with fungal denitrification and codenitrification studies but in the absence of soils and sediments. In soil, NO accumulates when excess free NH3 inhibits bacterial NO2 oxidation, which is often a consequence of urea hydrolysis. Venterea et al. recovered 3u201360% of added urea N as NO2, suggesting high soil NO is likely following urea addition (depending upon conditions and application rate). Based on these data, a 1-kg bovine urine addition to soil (N concentration of 0.4u2009mmolu2009kg) could result in up to 0.24u2009mmol NO2. Here, we added 0.25u2009mmol NO2 to 10u2009ml of fungal culture, which is on the high end of this scale. While we would expect abiotic formation of hybrid N2 via nitrosation of organic N to be more likely in grazed or fertilised agroecosystems, further N2 investigations with soil and at lower NO2 levels are needed to bridge the gap between pure culture and environmental applications.\" - \"Other areas where NO2 could accumulate include laboratory incubations where antibiotic inhibitors are added to soil and used to partition N2 and N2O production into fungal and bacterial contributions. These data may be subject to artefacts if antibiotics repress NO2 oxidation, leading to NO2 accumulation and potential abiotic formation of N2 and/or N2O. Our report also calls into question if codenitrification alone accounts for N2 and N2O emissions following high doses of N as urea (approximately 9u2009mmolu2009g soil) and/or NO2 (166u2009mmolu2009L). If abiotic, hybrid N2 is formed under these conditions, our understanding of soil fungal codenitrification and N trace gas emissions would need to be re-examined. We offer new insight into chemodenitrification of NO2 to N2 and present an alternative u2018N2O bypass pathway. Finally, we conclude the potential for aerobic, abiotic removal of excess reactive N in terrestrial and aquatic ecosystems presents environmental mitigation research opportunities, particularly where transitory or chronic NO2 accumulation occurs.\"- - \"Fungal incubations were conducted as described in Phillips et al. (2016) with the same culture ICMP 6809 isolated as a pathogen of Hordeum distichon in New Zealand. () GenBank: KU194490. We used laboratory incubations and controlled O2 status to evaluate if rates of N2O, N2 and CO2 production rates vary with aerobic conditions, as commonly reported for bacteria. Experimental design was similar to previous work, with 4 replicates for each treatment (N with and without O2; no N with or without O2). Cultures were grown in medium similar to other fungal denitrification studies, consisting of 1% glucose, 0.2% peptone and inorganic salts. The only N source in this medium used for fungal growth was peptone. One day prior to the experiment, the growth medium was washed off cultures and replaced with the same medium that was identical except it was free of peptone and so contained no N. Using the peptone-free medium herein, two amendment solutions were prepared for fungal inoculation: (a) media without N and (b) 0.25u2009 mmolu2009N as NaNO2 and 0.25u2009mmolu2009N as C5H10N2O3. Nitrogen concentration was 0.04u2009mmolu2009N L. Necromass was obtained by heating live B. sorokiniana at 60u2009xB0C for 48u2009h. Cell death was confirmed by (a) microscopy and (b) lack of CO2 respiration over a 24u2009h period under aerobic and anaerobic conditions. Approximately 10u2009ml live fungi or fungal necromass were blindly pipetted by a second independent scientist into 0.125u2009L serum bottles that were then amended with either media that included 0.25u2009mmolu2009N as NaNO2 and 0.25u2009mmolu2009N as C5H10N2O3 or media without N and mixed gently. Additional bottles containing sterile media solutions (with and without N) only and without fungi were also prepared. Oxygen status was controlled with airtight laboratory incubation vessels for all necromass, live fungi and media only samples. Headspace of each sample was evacuated and filled with either He (anaerobic) or heliox [aerobic (80% He, 20% O2)] within 2 hr following inoculation. Headspace gases were quantified approximately every 6u2009h at 19u2009xB0C using a robotic gas chromatograph (GC) fitted with electron capture and thermal conductivity detectors according to McMillan et al.. Helium or heliox blank standards were included in each GC run. Cumulative production rates were calculated as slopes of the masses (u03BCmol) of N2O or N2 measured over time (h) per g of fungal biomass. Vessels incubated in He remained anoxic with the exception of necromass, where we observed 0.05% O2. Vessels incubated in heliox remained above 19% O2 with the exception of live fungi incubated without N, where 5% of the headspace O2 was consumed. As reported previously, addition of NO2 inhibited O2 consumption and CO2 respiration by live fungi. Fungal biomass was determined as the difference in mass with and without media after air-drying each vessel post-incubation. For comparisons with sterile media, rates were also calculated as slopes of the masses (u03BCmol) of N2O and N2 measured over time. Sterility was maintained and conditions remained constant, including pH (6.2u20136.9). Sterility was tested at the end of gas sampling by pipetting 200u2009u03BCL medium used in the experiment onto blood agar plates and incubating under aerobic, anaerobic (AnaeroGen, Thermo Scientific), and enriched CO2 (3.5u20139%; CO2Gen, Thermo Scientific) conditions. Further, medium was also pipetted onto yeast nutrient agar, brain heart infusion agar, and potato dextrose agar plates under aerobic conditions. No evidence of fungal or bacterial growth was observed following 3, 7, and 10 d incubations at 26u2009xB0C. Data were analysed to test for effects of O2 and live fungi on either N2O or N2 production rates with a generalised linear model. We analysed only those samples amended with N because samples without N did not produce N2O or N2. Log transformations were employed when data did not meet the assumptions of normality. Treatments variances met assumptions of homoscedasticity. All interactions were tested and remained in the model if significant.\" - \"Di-nitrogen isotopes were evaluated in a separate experiment to assess sources of N used in abiotic production of N2 by adding NaNO2 and C5H10N2O3 to sterile medium at neutral pH, as described above. We aimed to determine if N2 would be produced through combination of NO2 only (thus forming N2) or through combination of both C5H10N2O3 and NO2 (thus forming the hybrid N2) for N-enriched medium relative to no-N medium only. In this experiment, we used five replicates at three levels of N: (a) no-N, (b) 0.25u2009mmol NaNO2 and 0.25u2009mmol C5H10N2O3, and (c) 0.5u2009mmol NaNO2 and 0.5u2009mmol C5H10N2O3. We used the same concentration of N in each N treatment but doubled the mass of N added. Vials were prepared under aerobic and anaerobic conditions as described previously and accompanied by He blanks. The aerobic experiment was conducted separately from the anaerobic experiment, which obviated testing for effects of O2 on masses of N2 and N2. Headspace N2 and N2 were measured on a continuous-flow isotope ratio mass spectrometer (Thermo Finnigan Delta V, Thermo Scientific) in line with an automated gas bench interface (Thermo Gas Bench II). Precision of the isotopic analysis was <0.001atom%.\"- - \\'How to cite this article: Phillips, R. L. et al. Chemical formation of hybrid di-nitrogen calls fungal codenitrification into question. Sci. Rep. 6, 39077; doi: 10.1038/srep39077 (2016).\\' - \\'Publisher\\'\\'s note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\'- - \"Zumft, W. G. Cell biology and molecular basis of denitrification. Microbiol. Molec. Biol. Rev. 61, 533u2013616 (1997)._ _ Ravishankara, A. R., Daniel, J. S. & Portmann, R. W. Nitrous oxide (N2O): The dominant ozone-depleting substance emitted in the 21st century. Science 326, 123u2013125; doi: 10.1126/science.1176985 (2009)._ _ _ _ _ Philippot, L., Hallin, S. & Schloter, M. Ecology of denitrifying prokaryotes in agricultural soil. Adv. Agron. 96, 249; doi: 10.1016/S0065-2113(07)96003-4 (2007)._ _ _ Yang, H., Gandhi, H., Ostrom, N. E. & Hegg, E. L. Isotopic fractionation by a fungal P450 nitric oxide reductase during the production of N2O. Environ. Sci.Tech. 48, 10707u201310715; doi: 10.1021/es501912d (2014)._ _ _ Rohe, L. et al. Fungal oxygen exchange between denitrification intermediates and water. Rapid Comm. Mass Spectrom. 28, 377u2013 384; doi: 10.1002/rcm.6790 (2014)._ _ _ _ Maeda, K. et al. N2O production, a widespread trait in fungi. Sci. Rep. 5, 9697; doi: 10.1038/srep09697 (2015)._ _ _ _ _ Shoun, H., Fushinobu, S., Jiang, L., Kim, S. & Wakagi, T. Fungal denitrification and nitric oxide reductase cytochrome P450nor. Philosoph. Trans. Royal Soc. B: Biol. Sci. 367, 1186u20131194; doi: 10.1098/rstb.2011.0335 (2012)._ _ _ Spott, O., Russow, R. & Stange, C. F. Formation of hybrid N2O and hybrid N2 due to codenitrification: First review of a barely considered process of microbially mediated N-nitrosation. Soil Biology and Biochemistry 43, 1995u20132011; doi: 10.1016/j.soilbio.2011.06.014 (2011)._ _ _ Tanimoto, T., Hatano, K.-i., Kim, D.-h., Uchiyama, H. & Shoun, H. Co-denitrification by the denitrifying system of the fungus Fusarium oxysporum. FEMS Microbiol. Lett. 93, 177u2013180; doi: 10.1111/j.1574-6968.1992.tb05086.x (1992)._ _ _ Shoun, H., Kim, D.-H., Uchiyama, H. & Sugiyama, J. Denitrification by fungi. FEMS Microbiology Letters 94, 277u2013 281; doi: 10.1111/j.1574-6968.1992.tb05331.x (1992)._ _ _ Kuypers, M. M. M. et al. Massive nitrogen loss from the Benguela upwelling system through anaerobic ammonium oxidation. Proc. Nat. Acad. Sci., USA 102, 6478u20136483; doi: 10.1073/pnas.0502088102 (2005)._ _ _ _ Long, A., Heitman, J., Tobias, C., Phillips, R. & Song, B. Co-occurring anammox, denitrification, and codenitrification in agricultural soils. Appl. Environ. Microbiol. 79, 168u2013176; doi: 10.1128/aem.02520-12 (2013)._ _ _ _ _ Selbie, D. R. et al. Confirmation of co-denitrification in grazed grassland. Sci. Rep. 5, 17361; doi: 10.1038/srep17361 (2015)._ _ _ _ _ _ Laughlin, R. J. & Stevens, R. J. Evidence for fungal dominance of denitrification and codenitrification in a grassland soil. Soil Sci. Soc. Amer. J. 66, 1540u20131548 (2002)._ _ _ _ Jones, L. C., Peters, B., Lezama Pacheco, J. S., Casciotti, K. L. & Fendorf, S. Stable isotopes and iron oxide mineral products as markers of chemodenitrification. Environ. Sci. Tech. 49, 3444u20133452; doi: 10.1021/es504862x (2015)._ _ _ Phillips, R. L. et al. Fungal denitrification: Bipolaris sorokiniana exclusively denitrifies inorganic nitrogen in the presence and absence of oxygen. FEMS MIcrobiol. Lett. 363; doi: 10.1093/femsle/fnw007 (2016).Sutka, R. L., Adams, G. C., Ostrom, N. E. & Ostrom, P. H. Isotopologue fractionation during N2O production by fungal denitrification. Rapid Comm. Mass Spectrom. 22, 3989u20133996; doi: 10.1002/rcm.3820 (2008)._ _ _ _ Zhou, Z., Takaya, N., Sakairi, M. A. C. & Shoun, H. Oxygen requirement for denitrification by the fungus Fusarium oxysporum. Arch. Microbiol. 175, 19u201325; doi: 10.1007/s002030000231 (2001)._ _ _ _ Schmid, M. C. et al. Anaerobic ammonium-oxidizing bacteria in marine environments: Widespread occurrence but low diversity. Environ. Microbiol. 9, 1476u20131484 (2007)._ _ _ _ Butterbach-Bahl, K., Baggs, E. M., Dannenmann, M., Kiese, R. & Zechmeister-Boltenstern, S. Nitrous oxide emissions from soils: how well do we understand the processes and their controls? Phil. Trans.Royal Soc. B: Biol. Sci. 368, 20120122; doi: 10.1098/rstb.2013.0122 (2013)._ _ _ McMillan, A. M. S. et al. Can pH amendments in grazed pastures help reduce N2O emissions from denitrification? u2013 The effects of liming and urine addition on the completion of denitrification in fluvial and volcanic soils. Soil Biol. Biochem. 93, 90u2013104; doi: 10.1016/j.soilbio.2015.10.013 (2016)._ _ _ Firestone, M. K. & Tiedje, J. M. Temporal change in nitrous oxide and dinitrogen from denitrification following onset of anaerobiosis. Appl. Environ. Microbiol. 38, 673u2013679; doi: (1979)._ _ _ _ Moraghan, J. & Buresh, R. Chemical reduction of nitrite and nitrous oxide by ferrous iron. Soil Sci. Soc. Amer. J. 41, 47u201350 (1977)._ _ _ _ Kampschreur, M. J. et al. Emission of nitrous oxide and nitric oxide from a full-scale single-stage nitritation-anammox reactor. Water Sci. Tech. 60, 3211u20133217 (2009)._ _ _ Wullstein, L. H. & Gilmour, C. M. Non-enzymatic formation of nitrogen gas. Nature 210, 1150u20131151; doi: 10.1038/2101150a0 (1966)._ _ _ _ Nelson, D. W. & Bremner, J. M. Gaseous products of nitrite decomposition in soils. Soil Biol. Biochem. 2, 203u2013IN208; doi: 10.1016/0038-0717(70)90008-8 (1970).Trimmer, M. & Purdy, K. J. Evidence for the direct oxidation of organic nitrogen to N2 gas in the Arabian Sea. ISME J. 6, 1798u20131800; doi: 10.1038/ismej2012.18 (2012)._ _ _ _ _ Babbin, A. R., Keil, R. G., Devol, A. H. & Ward, B. B. Organic matter stoichiometry, flux and oxygen control nitrogen loss in the ocean. Science 344, 406u2013408; doi: 10.1126/science.1248364 (2014)._ _ _ _ _ Venterea, R. T. Nitrite-driven nitrous oxide production under aerobic soil conditions: kinetics and biochemical controls. Global Change Biology 13, 1u201312; doi: 10.1111/j.1365-2486.2007.01389.x (2007)._ _ Heil, J., Vereecken, H. & Bruggemann, N. A review of chemical reactions of nitrification intermediates and their role in nitrogen cycling and nitrogen trace gas formation in soil. Europ. J. Soil Sci. 67, 23u201339; doi: 10.1111/ejss.12306 (2016)._ _ _ Rohe, L. et al. Dual isotope and isotopomer signatures of nitrous oxide from fungal denitrification u2013 a pure culture study. Rapid Comm. Mass Spectrom. 28, 1893u20131903; doi: 10.1002/rcm.6975 (2014)._ _ _ _ Zhu, G. et al. Occurrence, activity and contribution of anammox in some freshwater extreme environments. Environ. Microbiol. Rep. (2015).Ward, B. B. et al. Denitrification as the dominant nitrogen loss process in the Arabian Sea. Nature 461, 78u201381; doi: 10.1038/nature08276 (2009)._ _ _ _ _ Venterea, R. T. et al. Ammonium sorption and ammonia inhibition of nitrite-oxidizing bacteria explain contrasting soil N2O production. Scientific Reports 5, 12153, doi: 10.1038/srep12153; (2015)._ _ _ _ _ _ Hoogendoorn, C. J., Betteridge, K., Costall, D. A. & Ledgard, S. F. Nitrogen concentration in the urine of cattle, sheep and deer grazing a common ryegrass/cocksfoot/white clover pasture. New Zealand J. Agri. Res. 53, 235u2013243; doi: 10.1080/00288233.2010.499899 (2010)._ _ _ Marx, D. H. & Bryan, W. C. Growth and ectomycorrhizal development of loblolly pine seedlings in fumigated soil infested with the fungal symbiont Pisolithus tinctorius. For. Sci. 21, 245u2013 254; doi: 10.1080/00288233.2010.499899 (1975)._ _ _ \"- - \"The authors thank the team at Landcare Research in New Zealand for supporting this idea. This work was partially funded by a USDA-NIFR grant [2014-67019-21614]; New Zealands Ministry of Business Innovation and Employment, Royal Society of New Zealand International Travel Programme; and New Zealand Ecosystems and Global Change Fund. The authors are grateful to Veronica Rollinson, Sujatha Senanayake, Duckchul Park, and Megan Peterson for technical assistance and Bill Schlesinger, Iris Anderson, Neha Jha, Mikki Eken, and Leah Kearns for their comments, support and editorial reviews.\"- - Authors and Affiliations - \"Landcare Research, Gerald Street, Lincoln, New ZealandRebecca L. Phillips,_ Andrew M. S. McMillan,_Gwen Grelet,_Bevan S. Weir_&_Thilak PalmadaDept. of Biological Sciences, Virginia Institute of Marine Science, Gloucester Point, Virginia, USABongkeun SongDept. of Marine Sciences, University of Connecticut, Groton, Connecticut, USACraig Tobias\" - \"AuthorsRebecca L. PhillipsYou can also search for this author in _Bongkeun SongYou can also search for this author in _Andrew M. S. McMillanYou can also search for this author in _Gwen GreletYou can also search for this author in _Bevan S. WeirYou can also search for this author in _Thilak PalmadaYou can also search for this author in _Craig TobiasYou can also search for this author in _\" - Contributions - R.L.P., B.S. and G.G. designed the experiments, T.P. performed lab experiments, A.M.S.M. analysed chromatographic data, C.T. performed isotope pairing experiments, B.S.W. and G.G. cultured, and verified fungal isolates, and R.L.P. and B.S. wrote the manuscript.- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing financial interests. - \\' \\'- - \\' \\' - \"This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution License CC BY 4.0.\" - Although simulations such as those reported in ref. remain computationally demanding, they are clearly reaching the stage of addressing important problems, way beyond proof-of-principle demonstrations. - Spectroscopy from first principles - \"Techniques enabling the calculation of spectroscopic properties of materials and molecules are critical components of a predictive simulation framework of complex systems. Through the calculation of response functions and electronically excited states, these tools may be used as computational probes of the atomic structure simulated with FPMD, or FPMD coupled with advanced sampling. Numerous excited state properties and spectroscopic signatures may not be accurately obtained from the solution of the Kohn-Sham equations, i.e., the basic equations of DFT used for the predictions of ground state properties. Recent developments of hybrid functionals have led to improved simulations of charged excitations for certain classes of materials, while the time-dependent formulation of DFT (TDDFT) has provided a computationally tractable method to compute neutral excitations. However, most hybrid functionals are not yet able to reliably treat broad classes of heterogeneous systems where materials with different dielectric properties are combined; on the other end, TDDFT within the adiabatic approximation to the exchangeu2212correlation functional may not be sufficiently accurate to describe certain types of excited states where charge transfer and excitonic effects are relevant.\" - \"Many-body perturbation theory (MBPT) provides a way of predicting excited state properties of molecules and materials, and holds the promise of overcoming the limitations, at least for certain materials, of hybrid DFT and TDDFT. Within MBPT, one can obtain charge excitation energies corresponding to photoemission and inverse photoemission spectra by evaluating so called self-energies (e.g., electron-electron or electron-phonon self-energies); furthermore, one can obtain neutral excitation energies corresponding to optical spectra by solving the Betheu2212Salpeter equation (BSE). Several methodological advances have been proposed to improve the efficiency of MBPT calculations, which have enabled the simulations of relatively large and complex systems, e.g., heterogeneous solids and liquids and nanoparticles with thousands of electrons. In particular, electronic structure calculations based on MBPT can be performed using a low-rank representation of density response functions, whose spectral decomposition is obtained through iterative diagonalization, thus avoiding the inversion and storage of large dielectric matrices. Such a diagonalization can be carried out without explicitly computing empty electronic states using density functional perturbation theory (DFPT). An implementation based on this strategy is at the core of the WEST code interfaced with the Quantum Espresso code, and it has been successfully applied to investigate the quasi-particle energies of numerous systems including defects in semiconductors, nanoparticles, aqueous solutions, and solid/liquid interfaces.\" - \"We have recently reformulated MBPT calculations of charged and neutral excitations by performing DFT calculations in finite electric fields, instead of using DFPT. The formulation has been implemented by coupling the Qbox and WEST codes using the client-server protocol. In particular, the variation of the charge density u03B4n in response to the applied perturbation u03B4V is computed by Qbox performing two self-consistent calculations: (left( {H pm delta V} right)left| {psi _i^ pm } rightrangle = varepsilon _i^ pm left| {psi _i^ pm } rightrangle), where H is the Kohn-Sham Hamiltonian. The variation of the charge difference may be obtained as (delta n = frac{1}{2}(n^ + - n^ - )), where (n^ pm = mathop {sum }nolimits_i left| {psi _i^ pm } right|^2). The finite-field approach allows for the straightforward calculation of response functions beyond the so called random phase approximation (RPA), which neglects the response of the perturbation arising from the exchange-correlation term; in addition the approach can be readily combined to DFT calculations using hybrid functionals for which analytical expressions of the functional derivative of the exchange correlation potential with respect to the charge density are not available. In our implementation, WEST is the driver program of multiple Qbox instances, which initially perform a wavefunction optimization step to compute the electronic ground state of the system. WEST generates the perturbation u03B4V, and the self-consistent calculations are solved by Qbox, which returns the linear variation of the charge density u03B4 n. Data for both u03B4V and u03B4n are exchanged by the two codes using an XML file that conforms to the annotated function3d.xsd XML schema (see ). Analogously to the case described in the previous section, the in vivo coupling between WEST and Qbox facilitates the execution of response commands by keeping in memory the electronic structure of the system.\" - Using the WEST-Qbox coupling, we have computed exciton binding energies of several molecules and absorption spectra of condensed systems, including water samples with hundreds of atoms. We emphasize that the coupling allows one to solve the BSE bypassing altogether the calculation of dielectric matrices and virtual states. In addition, the formulation implemented in ref. uses linear combinations of Bloch orbitals that are localized in appropriate regions of real space using the recursive bisection method, thus leading to substantial computational savings. We note that calculations beyond the random-phase approximation become straightforward in finite field, and the complexity and scaling of solving the BSE is the same when using local or hybrid-DFT calculations as starting points. The same finite field approach can also be used to perform GW calculations and obtain quasi-particle orbitals, through the evaluation of the eigenvectors of the irreducible density-density response function. Recently, a method based on machine learning was introduced to speed-up MBPT calculations. In particular, the data sets exchanged by WEST and Qbox for the calculation of absorption spectra were used to train a surrogate model of the dielectric screening computed in finite field. Interoperability between WEST and Tensorflow () enabled computational gains of up to two orders of magnitude for the calculation of absorption spectra of molecules and materials at finite temperature. - The ability to obtain the screened Coulomb interaction by solving DFT in finite electric field has made possible quantum embedding calculations. In the presence of specific localized electronic states, e.g., those associated with spin defects in semiconductors or insulators, one may obtain geometries and energies of excited states by performing DFT calculations with constrained occupation numbers. In addition, the effect of the environment around the defect may be incorporated in the calculation through the definition of an effective Hamiltonian including a screened Coulomb interaction. Using these two concepts, and computing the screened Coulomb interaction inclusive of exchange-correlation interactions through the WEST-Qbox coupling, we have built a quantum embedding theory (Fig. ) that goes beyond the constrained random phase approximation. The accuracy and effectiveness of this approach was demonstrated by investigating several spin defects in semiconductors that are of interest for quantum information technologies. Importantly, the proposed quantum embedding theory is able to capture strongly-correlated electronic states of active regions (defined by the localized electronic states of the defects), with the rest of the system described within DFT. An important consequence of the quantum embedding theory, as demonstrated in ref. , is that an effective Hamiltonian describing strong correlations within the active site may be defined and solved on classical as well as on quantum computers using ex vivo interoperability schemes with quantum chemistry and quantum computing software packages, e.g., PySCF and Qiskit (). Quantum computers promise, in principle, a much more favorable scaling as a function of system size than classical architectures although noise and quantum correction issues are still affecting the accuracy of results obtained on near-term quantum computers and substantial progress is needed before complex materials science simulations may be carried out on quantum computers. - \\'Fig. 4: Quantum embedding theory.In vivo coupling of the Qbox and WEST codes was used to build an embedding theory to study strongly correlated defects in semiconductors and insulators. Single particle wavefunctions and eigenvalues obtained from DFT calculations are used to compute the screening of the environment on a chosen active region of the material (defined, e.g., by the localized single particle states of a defect). The screening enters the definition of a second quantized Hamiltonian that may be diagonalized using configuration interaction techniques to obtain many body states of the defect.\\' - Transport from first principles - Similar to the computation of free-energy surfaces and spectroscopic properties, coupled interoperable codes may be used to compute transport properties, in particular, to evaluate quantities relevant to the description of electronic transport in the hopping regime, and to simulate heat transport in insulators. - \"The hopping regime of charge transfer between localized states in molecules and solids is the dominant charge transfer mechanism in many organic crystals, conducting polymers, in several metal oxides in the solid state, and in nanoparticle solids. In many instances, charge transfer in the hopping regime may be described using Marcus theory which predicts the charge transfer rate in terms of diabatic electronic coupling and reorganization energies, in addition to free-energy differences. Constrained density functional theory (CDFT) provides a robust framework for constructing diabatic states from first principles and for predicting their electronic coupling in molecules and solids (see Fig. ): constraint potentials are added to the Kohnu2010Sham Hamiltonian, and their strengths are optimized so as to obtain a desired localized charge on a given site. By using the client-server functionality of Qbox as described in the previous section, we recently implemented a Python package (PyCDFT) to perform singleu2010point selfu2010consistentu2010 field and geometry optimization calculations using CDFT. The operations specific to PyCDFT are decoupled from those carried out by Qbox, again presenting important advantages for the maintainability and reusability of the package.\" - \\'Fig. 5: Transport from first principles.Charge transfer excitation processes in assemblies of hydrogenated Si nanoparticles (panel a) and in the bulk and at the surface of crystalline BiVO4 (panel b), investigated using constrained density functional theory. Panel b is reproduced with permission from ref. , copyright (American Chemical Society, 2020).\\' - As a last example, we describe the simulation of heat transport, in particular the calculation of the ionic thermal conductivity. As in the case of advanced sampling, most calculations of the thermal conductivity, especially those for large systems, have been carried out using empirical interatomic potentials. However, first principles methods have been proposed in the recent literature, based on approximate solutions of the Boltzmann transport equation (BTE), on Green-Kubo (GK) formulations, and using nonequilibrium molecular dynamics (NEMD). Approximate solutions of the BTE can only be applied to solids. The method proposed in ref. is the only one available to perform first principles simulations based on GK. This elegant approach involves repeated solutions of the Sternheimer equation to calculate the heat current within DFPT. Addressing the need for an efficient and general quantum simulation framework for thermal properties of materials, we developed a method to simulate heat transport from first principles based on approach to equilibrium simulations (SAEMD). The method can be used for homogeneous and heterogeneous solids, and in the latest generalization also to liquids; it has the important advantage of requiring only the computation of MD trajectories and atomic forces, with no additional calculation of energy derivatives or direct calculations of currents. Recently, we carried out FPMD simulations of the thermal conductivity of a representative oxide, MgO, by using Qbox to compute DFT forces on atoms and LAMMPS to integrate the equation of motion and to apply the local thermostats required in the application of SAEMD. We successfully simulated systems with ~1000 atoms and using extrapolation techniques we obtained the thermal conductivity of MgO from first principles. However, our calculations highlighted the need of much larger simulation cells to obtain robustly converged results. Hence, a direction we are exploring is the use of deepMD to simulate thermal properties and work is in progress to apply this technique to liquid water.- - We described a strategy to couple interoperable software for quantum simulations of materials and we presented examples of calculations of properties of heterogeneous systems, including molecules, solids, liquids and surfaces. Interoperable software elements can be independently modified and maintained, thus facilitating rapid developments and release of new ideas and new features. Although we described specific codes in this article, the strategy presented here is general. It does not require the definition of universal format standards, and it is generalizable to include properties and systems beyond the examples we have discussed. For example, work is in progress to couple FPMD and calculation of excited state properties with machine learning techniques of dielectric screening, and to couple FPMD with quantum thermostats and calculations of excited state properties beyond DFT to evaluate electron phonon interaction in clusters, solids and amorphous systems containing light atoms. Additional work is in progress to evaluate activation energies in reactive systems and morphology changes in metallic clusters by coupling Qbox and SSAGES, and to evaluate coherence times of spin defects in solids by coupling DFT and spin Hamiltonian calculations. - Finally, we note that the ability to carry out coupled calculations will serve as a stepping stone to build hybrid strategies to use classical and quantum computers, especially near-term quantum computers. Different properties or different parts of a calculation may be performed on different architectures, depending on the scaling of the algorithm used. - We expect that the development of extensible codes that can be interfaced with each other to perform complex tasks will enable modular simulations of multiple properties of materials, thus accelerating the development of design strategies applicable to broad classes of systems and effectively broadening the scope of quantum simulations.- - The interoperable codes discussed in this work are Qbox, WEST, SSAGES and PyCDFT - Qbox is a first principles molecular dynamics code based on the plane-wave, pseudopotential formalism. The electronic structure is obtained using density functional theory (DFT) or hybrid-DFT. In addition to the simulation of thermodynamic and structural properties of materials, Qbox allows for the calculations of vibrational spectra, ionic conductivity, and heat transport coefficients. - WEST performs large-scale, many-body perturbation theory (MBPT) calculations providing electronic and optical spectroscopic characterization of complex materials. WEST uses the plane-wave, pseudopotential formalism and is interfaced with the Quantum Espresso code and with Qbox in client-server mode. In addition to the calculation of electron-electron, electron-phonon and electron-hole interactions, WEST enables calculations based on a quantum embedding theory. - SSAGES provides a unified, extensible framework to calculate reaction coordinates and reactive pathways, and free energies within molecular simulations. SSAGES offers a comprehensive suite of advanced methods for phase space sampling organized into a unified framework that can function as a wrapper, adding functionality to first-principles and classical molecular dynamics and Monte Carlo engines. - \"PyCDFT is a Python package to compute diabatic states using constrained density functional theory (CDFT). PyCDFT allows for both singleu2010point selfu2010 consistentu2010field calculations and geometry optimizations. PyCDFT is designed to interface with existing DFT codes to perform CDFT calculations where constraint potentials are added to the Kohnu2013Sham Hamiltonian.\"- - \\' \\' - The examples reported in this work use the (QSO) format to exchange datasets, which is based on the XML markup language. Data that support the findings discussed in this study are available through the Qresp portal (). - \\' \\'- - \\' \\' - \\'The examples reported in this work use open-source codes: Qbox (), WEST (), SSAGES ().\\' - \\' \\'- - 01 March 2021The link to the open-source code Qbox in the Code availability section was updated shortly after publication, changing from \"Qbox (http://qbox-code.org)\" to \"Qbox (http://qboxcode.org)\".- - \"Curtarolo, S. et al. AFLOWLIB.ORG: a distributed materials properties repository from high-throughput ab initio calculations. Comput. Mater. Sci. 58, 227u2013 235 (2012)._ _ _ Jain, A. et al. Commentary: The Materials Project: a materials genome approach to accelerating materials innovation. APL Mater. 1, 011002 (2013)._ _ Saal, J. E., Kirklin, S., Aykol, M., Meredig, B. & Wolverton, C. Materials design and discovery with high-throughput density functional theory: the open quantum materials database (OQMD). J. Mater. 65, 1501u20131509 (2013)._ _ Bhat, T. N., Bartolo, L. M., Kattner, U. R., Campbell, C. E. & Elliott, J. T. Strategy for extensible, evolving terminology for the materials genome initiative efforts. J. Mater. 67, 1866u20131875 (2015)._ Kalidindi, S. R. & De Graef, M. Materials data science: current status and future outlook. Ann. Rev. Mater. Res. 45, 171u2013193 (2015)._ _ _ Rajan, K. Materials informatics: the materials u201CGeneu201D and big data. Ann. Rev. Mater. Res. 45, 153u2013169 (2015)._ _ _ Blaiszik, B. et al. The materials data facility: data services to advance materials science research. J. Mater. 68, 2045u20132052 (2016)._ Jain, A., Persson, K. A. & Ceder, G. Research update: the materials genome initiative: data sharing and the impact of collaborative ab initio databases. APL Mater. 4, 053102 (2016)._ _ Thygesen, K. S. & Jacobsen, K. W. Making the most of materials computations. Science 354, 180 (2016)._ _ _ Bennett, C. H. & DiVincenzo, D. P. Quantum information and computation. Nature 404, 247u2013255 (2000)._ _ _ Koehl, W. F., Seo, H., Galli, G. & Awschalom, D. D. Designing defect spins for wafer-scale quantum technologies. MRS Bull. 40, 1146u20131153 (2015)._ _ _ Govoni, M. et al. Qresp, a tool for curating, discovering and exploring reproducible scientific papers. Sci. Data 6, 190002 (2019)._ _ Talirz, L. et al. Materials Cloud, a platform for open computational science. Sci. Data 7, 299 (2020)._ _ Gygi, F. Architecture of Qbox: a scalable first-principles molecular dynamics code. IBM J. Res. Dev. 52, 137u2013144 (2008)._ _ Sidky, H. et al. SSAGES: Software suite for advanced general ensemble simulations. J. Chem. Phys. 148, 044104 (2018)._ _ Govoni, M. & Galli, G. Large scale GW calculations. J. Chem. Theory Comput. 11, 2680u2013 2696 (2015)._ _ _ Plimpton, S. Fast parallel algorithms for short-range molecular dynamics. J. Comput. Phys. 117, 1u201319 (1995)._ _ _ Ma, H. et al. PyCDFT: A Python package for constrained density functional theory. J. Comput. Chem. 41, 1859u20131867 (2020)._ _ _ Barnes, T. A., Marin-Rimoldi, E., Ellis, S. & Crawford, T. D. The MolSSI Driver Interface Project: a framework for standardized, on-the-fly interoperability between computational molecular sciences codes. Comput. Phys. Commun. 261, 107688 (2021).Gygi, F., Duchemin, I., Donadio, D. & Galli, G. Practical algorithms to facilitate large-scale first-principles molecular dynamics. J. Phys. Conf. Ser. 180, 012074 (2009)._ _ Foster, I. Globus online: accelerating and democratizing science through cloud-based services. IEEE Internet Comput. 15, 70u201373 (2011)._ _ Car, R. & Parrinello, M. Unified approach for molecular dynamics and density-functional theory. Phys. Rev. Lett. 55, 2471u20132474 (1985)._ _ _ Marx, D. & Hutter, J. Ab Initio Molecular Dynamics: Basic Theory and Advanced Methods. (Cambridge University Press, 2009).Chodera, J. D. & NoxE9, F. Markov state models of biomolecular conformational dynamics.Curr. Opin. Struct. Biol. 25, 135u2013144 (2014)._ _ _ Miao, Y., Feixas, F., Eun, C. & McCammon, J. A. Accelerated molecular dynamics simulations of protein folding. J. Comput. Chem. 36, 1536u20131549 (2015)._ _ _ Helfferich, J., Lyubimov, I., Reid, D. & de Pablo, J. J. Inherent structure energy is a good indicator of molecular mobility in glasses. Soft Matter 12, 5898u20135904 (2016)._ _ _ Reid, D. R., Lyubimov, I., Ediger, M. D. & de Pablo, J. J. Age and structure of a model vapour-deposited glass. Nat. Commun. 7, 13062 (2016)._ _ _ Perilla, J. R. et al. Molecular dynamics simulations of large macromolecular complexes. Curr. Opin Struct. Biol. 31, 64u201374 (2015)._ _ _ Cheng, B. & Ceriotti, M. Bridging the gap between atomistic and macroscopic models of homogeneous nucleation. J. Chem. Phys. 146, 034106 (2017)._ _ Giberti, F., Salvalaglio, M. & Parrinello, M. Metadynamics studies of crystal nucleation. IUCrJ 2, 256u2013266 (2015)._ _ _ Giberti, F., Salvalaglio, M., Mazzotti, M. & Parrinello, M. 1,3,5-tris(4-bromophenyl)-benzene nucleation: from dimers to needle-like clusters. Cryst. Growth Des. 17, 4137u20134143 (2017)._ _ _ Zhang, C. et al. Dissociation of salts in water under pressure. Nat. Commun. 11, 3037 (2020)._ _ _ Lee, E. et al. Neural network sampling of the free energy landscape for nitrogen dissociation on ruthenium. ChemRxiv, (2020).Singh, S., Chopra, M. & Pablo, J. J. D. Density of statesu2013based molecular simulations. Annu. Rev. Chem. Biomol. Eng. 3, 369u2013394 (2012)._ _ Fu, H., Shao, X., Cai, W. & Chipot, C. Taming rugged free energy landscapes using an average force. Acc. Chem. Res. 52, 3254u20133264 (2019)._ _ _ Bussi, G. & Laio, A. Using metadynamics to explore complex free-energy landscapes. Nat. Rev. Phys. 2, 200u2013212 (2020)._ _ Torrie, G. M. & Valleau, J. P. Nonphysical sampling distributions in Monte Carlo free-energy estimation: Umbrella sampling. J. Comput. Phys. 23, 187 (1977)._ _ Laio, A. & Parrinello, M. Escaping free-energy minima. Proc. Natl Acad. Sci. USA 99, 12562 (2002)._ _ _ Dellago, C., Bolhuis, P. & Geissler, P. L. Transition path sampling. Adv. Chem. Phys. 123, 1 (2003)._ Darve, E., RodrxEDguez-GxF3mez, D. & Pohorille, A. Adaptive biasing force method for scalar and vector free energy calculations. J. Chem. Phys. 128, 144120 (2008)._ _ Barducci, A., Bussi, G. & Parrinello, M. Well-tempered metadynamics: a smoothly converging and tunable free-energy method. Phys. Rev. Lett. 100, 020603 (2008)._ _ Jorgensen, W. L., Maxwell, D. S. & Tirado-Rives, J. Development and testing of the OPLS all-atom force field on conformational energetics and properties of organic liquids. J. Am. Chem. Soc. 118, 11225 (1996)._ _ _ Hornak, V. et al. Comparison of multiple Amber force fields and development of improved protein backbone parameters. Proteins Struct. Funct. Genet. 65, 712 (2006)._ _ _ Marrink, S. J., Risselada, H. J., Yefimov, S., Tieleman, D. P. & De Vries, A. H. The MARTINI force field: coarse grained model for biomolecular simulations. J. Phys. Chem. B 111, 7812 (2007)._ _ _ Babin, V., Leforestier, C. & Paesani, F. Development of a u201Cfirst principlesu201D water potential with flexible monomers: Dimer potential energy surface, VRT spectrum, and second virial coefficient. J. Chem. Theory Comput. 9, 5395 (2013)._ _ _ Chen, C., Arntsen, C. & Voth, G. A. Development of reactive force fields using ab initio molecular dynamics simulation minimally biased to experimental data. J. Chem. Phys. 147, 161719 (2017)._ _ Henriques, J. & SkepxF6, M. Molecular dynamics simulations of intrinsically disordered proteins: on the accuracy of the TIP4P-D water model and the representativeness of protein disorder models. J. Chem. Theory Comput. 12, 3407 (2016)._ _ _ Lyubartsev, A. P. & Rabinovich, A. L. Force field development for lipid membrane simulations. Biochim. Biophys. Acta Biomembr. 1858, 2483 (2016)._ _ _ Sevgen, E. et al. Hierarchical coupling of first-principles molecular dynamics with advanced sampling methods. J. Chem. Theory Comput. 14, 2881u20132888 (2018)._ _ _ Gaiduk, A. P. et al. Photoelectron spectra of aqueous solutions from first principles. J. Am. Chem. Soc. 138, 6912u20136915 (2016)._ _ _ Pham, T. A. et al. Electronic structure of aqueous solutions: Bridging the gap between theory and experiments. Sci. Adv. 3, e1603210 (2017)._ _ Gerosa, M., Gygi, F., Govoni, M. & Galli, G. The role of defects and excess surface charges at finite temperature for optimizing oxide photoabsorbers. Nat. Mater. 17, 1122u20131127 (2018)._ _ _ Gaiduk, A. P., Gustafson, J., Gygi, F. & Galli, G. First-principles simulations of liquid water using a dielectric-dependent hybrid functional. J. Phys. Chem. Lett. 9, 3068u20133073 (2018)._ _ _ Dawson, W. & Gygi, F. Equilibration and analysis of first-principles molecular dynamics simulations of water. J. Chem. Phys. 148, 124501 (2018)._ _ Gaiduk, A. P., Pham, T. A., Govoni, M., Paesani, F. & Galli, G. Electron affinity of liquid water. Nat. Commun. 9, 247 (2018)._ _ LaCount, M. D. & Gygi, F. Ensemble first-principles molecular dynamics simulations of water using the SCAN meta-GGA density functional. J. Chem. Phys. 151, 164101 (2019)._ _ Harmon, K. J. et al. Validating first-principles molecular dynamics calculations of oxide/water interfaces with x-ray reflectivity data. Phys. Rev. Mater. 4, 113805 (2020)._ _ _ Carter, E. A., Ciccotti, G., Hynes, J. T. & Kapral, R. Constrained reaction coordinate dynamics for the simulation of rare events. Chem. Phys. Lett. 156, 472u2013477 (1989)._ _ _ Sprik, M. & Ciccotti, G. Free energy from constrained molecular dynamics. J. Chem. Phys. 109, 7737u20137744 (1998)._ _ _ Ciccotti, G. & Ferrario, M. Blue moon approach to rare events. Mol. Simula. 30, 787u2013793 (2004)._ _ _ Ciccotti, G., Kapral, R. & Vanden-Eijnden, E. Blue moon sampling, vectorial reaction coordinates, and unbiased constrained dynamics. Chem. Phys. Chem. 6, 1809u20131814 (2005)._ _ _ Barducci, A., Bonomi, M. & Parrinello, M. Metadynamics. WIREs Comput. Mol. Sci. 1, 826u2013843 (2011)._ _ _ Sutto, L., Marsili, S. & Gervasio, F. L. New advances in metadynamics. WIREs Comput. Mol. Sci. 2, 771u2013779 (2012)._ _ _ KxE4stner, J. Umbrella sampling. WIREs Comput. Mol. Sci. 1, 932u2013 942 (2011)._ _ Martou0148xE1k, R., Donadio, D., Oganov, A. R. & Parrinello, M. Crystal structure transformations in SiO2 from classical and ab initio metadynamics. Nat. Mater. 5, 623 (2006)._ _ Ensing, B., De Vivo, M., Liu, Z., Moore, P. & Klein, M. L. Metadynamics as a tool for exploring free energy landscapes of chemical reactions. Acc. Chem. Res. 39, 73 (2006)._ _ _ Guido, C. A., Pietrucci, F., Gallet, G. A. & Andreoni, W. The fate of a zwitterion in water from ab initio molecular dynamics: Monoethanolamine (MEA)-CO2. J. Chem. Theory Comput. 9, 28 (2013)._ _ _ Pietrucci, F. & Saitta, A. M. Formamide reaction network in gas phase and solution via a unified theoretical approach: Toward a reconciliation of different prebiotic scenarios. Proc. Natl Acad. Sci. USA 112, 15030 (2015)._ _ _ Herron, J. A., Morikawa, Y. & Mavrikakis, M. Ab initio molecular dynamics of solvation effects on reactivity at electrified interfaces. Proc. Natl Acad. Sci. USA 113, 201604590 (2016)._ _ Kubota, Y., Ohnuma, T. & Buu010Dko, T. Carbon dioxide capture in 2-aminoethanol aqueous solution from ab initio molecular dynamics simulations. J. Chem. Phys. 146, 094303 (2017)._ _ Perdew, J. P., Ernzerhof, M. & Burke, K. Rationale for mixing exact exchange with density functional approximations. J. Chem. Phys. 105, 9982u20139985 (1996)._ _ _ Martin, R. M. Electronic Structure: Basic Theory and Practical Methods. (Cambridge University Press, 2004).Adamo, C. & Barone, V. Toward reliable density functional methods without adjustable parameters: the PBE0 model. J. Chem. Phys. 110, 6158 (1999)._ _ _ Perdew, J. P., Burke, K. & Ernzerhof, M. Generalized gradient approximation made simple. Phys. Rev. Lett. 77, 3865 (1996)._ _ _ Pan, D., Spanu, L., Harrison, B., Sverjensky, D. A. & Galli, G. Dielectric properties of water under extreme conditions and transport of carbonates in the deep Earth. Proc. Natl Acad. Sci. USA 110, 6646 (2013)._ _ _ Pan, D., Govoni, M. & Galli, G. Communication: dielectric properties of condensed systems composed of fragments. J. Chem. Phys. 149, 051101 (2018)._ _ Sevgen, E., Guo, A. Z., Sidky, H., Whitmer, J. K. & de Pablo, J. J. Combined force-frequency sampling for simulation of systems having rugged free energy landscapes. J. Chem. Theory Comput. 16, 1448u2013 1455 (2020)._ _ _ Hohenberg, P. & Kohn, W. Inhomogeneous electron gas. Phys. Rev. 136, B864u2013B871 (1964)._ _ Kohn, W. & Sham, L. J. Self-consistent equations including exchange and correlation effects. Phys. Rev. 140, A1133u2013A1138 (1965)._ _ Becke, A. D. Perspective: fifty years of density-functional theory in chemical physics. J. Chem. Phys. 140, 18A301 (2014)._ _ Skone, J. H., Govoni, M. & Galli, G. Self-consistent hybrid functional for condensed systems. Phys. Rev. B 89, 195112 (2014)._ _ Skone, J. H., Govoni, M. & Galli, G. Nonempirical range-separated hybrid functionals for solids and molecules. Phys. Rev. B 93, 235106 (2016)._ _ Brawand, N. P., VxF6rxF6s, M., Govoni, M. & Galli, G. Generalization of dielectric-dependent hybrid functionals to finite systems. Phys. Rev. X 6, 041002 (2016)._ Brawand, N. P., Govoni, M., VxF6rxF6s, M. & Galli, G. Performance and self-consistency of the generalized dielectric dependent hybrid functional. J. Chem. Theory Comput. 13, 3318u2013 3325 (2017)._ _ _ Runge, E. & Gross, E. K. U. Density-functional theory for time-dependent systems. Phys. Rev. Lett. 52, 997u20131000 (1984)._ _ _ Casida, M. E. in Recent Advances in Density Functional Methods Vol. Volume 1 Recent Advances in Computational Chemistry 155-192 (WORLD SCIENTIFIC, 1995).Zheng, H., Govoni, M. & Galli, G. Dielectric-dependent hybrid functionals for heterogeneous materials. Phys. Rev. Mater. 3, 073803 (2019)._ _ _ Casida, M. E. & Huix-Rotllant, M. Progress in time-dependent density-functional theory. Annu. Rev. Phys. Chem. 63, 287u2013323 (2012)._ _ _ Onida, G., Reining, L. & Rubio, A. Electronic excitations: density-functional versus many-body Green s-function approaches. Rev. Mod. Phys. 74, 601u2013659 (2002)._ _ _ Deslippe, J. et al. BerkeleyGW: a massively parallel computer package for the calculation of the quasiparticle and optical properties of materials and nanostructures. Comput. Phys. Commun. 183, 1269u20131289 (2012)._ _ _ Sangalli, D. et al. Many-body perturbation theory calculations using the yambo code. J. Phys. Condens. Matter 31, 325902 (2019)._ _ _ Golze, D., Dvorak, M. & Rinke, P. The GW compendium: a practical guide to theoretical photoemission spectroscopy. Front. Chem. 7, (2019).Giustino, F. Electron-phonon interactions from first principles. Rev. Mod. Phys. 89, 015003 (2017)._ _ Nguyen, N. L., Ma, H., Govoni, M., Gygi, F. & Galli, G. Finite-field approach to solving the Bethe-Salpeter equation. Phys. Rev. Lett. 122, 237402 (2019)._ _ _ Salpeter, E. E. & Bethe, H. A. A relativistic equation for bound-state problems. Phys. Rev. 84, 1232u20131242 (1951)._ _ Albrecht, S., Reining, L., Del Sole, R. & Onida, G. Ab initio calculation of excitonic effects in the optical spectra of semiconductors. Phys. Rev. Lett. 80, 4510u2013 4513 (1998)._ _ _ Benedict, L. X., Shirley, E. L. & Bohn, R. B. Optical absorption of insulators and the electron-hole interaction: an ab initio calculation. Phys. Rev. Lett. 80, 4514u20134517 (1998)._ _ _ Rohlfing, M. & Louie, S. G. Electron-hole excitations in semiconductors and insulators. Phys. Rev. Lett. 81, 2312u20132315 (1998)._ _ _ Scherpelz, P., Govoni, M., Hamada, I. & Galli, G. Implementation and validation of fully relativistic GW calculations: spinu2013 orbit coupling in molecules, nanocrystals and solids. J. Chem. Theory Comput. 12, 3523u20133544 (2016)._ _ _ McAvoy, R. L., Govoni, M. & Galli, G. Coupling first-principles calculations of electronu2013 electron and electronu2013phonon scattering, and applications to carbon-based nanostructures. J. Chem. Theory Comput. 14, 6269u20136275 (2018)._ _ _ Yang, H., Govoni, M. & Galli, G. Improving the efficiency of G0W0 calculations with approximate spectral decompositions of dielectric matrices. J. Chem. Phys. 151, 224102 (2019)._ _ Umari, P., Stenuit, G. & Baroni, S. GW quasiparticle spectra from occupied states only. Phys. Rev. B 81, 115104 (2010)._ _ Giustino, F., Cohen, M. L. & Louie, S. G. GW method with the self-consistent Sternheimer equation. Phys. Rev. B 81, 115105 (2010).Nguyen, H. V., Pham, T. A., Rocca, D. & Galli, G. Improving accuracy and efficiency of calculations of photoemission spectra within the many-body perturbation theory. Phys. Rev. B 85, 081101 (2012).Wilson, H. F., Gygi, F. & Galli, G. Efficient iterative method for calculations of dielectric matrices. Phys. Rev. B 78, 113303 (2008).Wilson, H. F., Lu, D., Gygi, F. & Galli, G. Iterative calculations of dielectric eigenvalue spectra. Phys. Rev. B 79, 245106 (2009).Baroni, S., de Gironcoli, S., Dal Corso, A. & Giannozzi, P. Phonons and related crystal properties from density-functional perturbation theory. Rev. Mod. Phys. 73, 515u2013562 (2001)._ _ _ Giannozzi, P. et al. QUANTUM ESPRESSO: a modular and open-source software project for quantum simulations of materials. J. Phys. Condens. Matter 21, 395502 (2009).Govoni, M. & Galli, G. GW100: comparison of methods and accuracy of results obtained with the WEST code. J. Chem. Theory Comput. 14, 1895u20131909 (2018)._ _ _ Seo, H., Govoni, M. & Galli, G. Design of defect spins in piezoelectric aluminum nitride for solid-state hybrid quantum technologies. Sci. Rep. 6, 20803 (2016)._ _ _ Seo, H., Ma, H., Govoni, M. & Galli, G. Designing defect-based qubit candidates in wide-gap binary semiconductors for solid-state quantum technologies. Phys. Rev. Mater. 1, 075002 (2017)._ _ Smart, T. J., Wu, F., Govoni, M. & Ping, Y. Fundamental principles for calculating charged defect ionization energies in ultrathin two-dimensional materials. Phys. Rev. Mater. 2, 124002 (2018)._ _ _ Ma, H., Govoni, M., Gygi, F. & Galli, G. A finite-field approach for GW calculations beyond the random phase approximation. J. Chem. Theory Comput. 15, 154u2013164 (2019)._ _ _ Ma, H., Govoni, M., Gygi, F. & Galli, G. Correction to a finite-field approach for GW calculations beyond the random phase approximation. J. Chem. Theory Comput. 16, 2877u20132879 (2020)._ _ _ Dong, S., Govoni, M. & Galli, G. Machine learning dielectric screening for the simulation of excited state properties of molecules and materials. arXiv 2012, 12244 (2020)._ Ma, H., Govoni, M. & Galli, G. Quantum simulations of materials on near-term quantum computers. npj Comput. Mater. 6, 85 (2020)._ _ Ma, H., Sheng, N., Govoni, M. & Galli, G. First-principles studies of strongly correlated states in defect spin qubits in diamond. Phys. Chem. Chem. Phys. (2020).Aryasetiawan, F. et al. Frequency-dependent local interactions and low-energy effective models from electronic structure calculations. Phys. Rev. B 70, 195104 (2004)._ _ Miyake, T. & Aryasetiawan, F. Screened Coulomb interaction in the maximally localized Wannier basis. Phys. Rev. B 77, 085122 (2008)._ _ Vaugier, L., Jiang, H., Biermann, S. & Hubbard, U. and Hund exchange J in transition metal oxides: Screening versus localization trends from constrained random phase approximation. Phys. Rev. B 86, 165105 (2012)._ _ Sun, Q. et al. PySCF: the Python-based simulations of chemistry framework. WIREs Comput. Mol. Sci. 8, e1340 (2018)._ _ Kandala, A. et al. Error mitigation extends the computational reach of a noisy quantum processor. Nature 567, 491u2013 495 (2019)._ _ _ Yu, D., Wang, C., Wehrenberg, B. L. & Guyot-Sionnest, P. Variable range hopping conduction in semiconductor nanocrystal solids. Phys. Rev. Lett. 92, 216802 (2004)._ _ Talapin, D. V., Lee, J.-S., Kovalenko, M. V. & Shevchenko, E. V. Prospects of colloidal nanocrystals for electronic and optoelectronic applications. Chem. Rev. 110, 389u2013458 (2010)._ _ _ Guyot-Sionnest, P. Electrical transport in colloidal quantum dot films. J. Phys. Chem. Lett. 3, 1169u2013 1175 (2012)._ _ _ Lan, X. et al. Quantum dot solids showing state-resolved band-like transport. Nat. Mater. 19, 323u2013329 (2020)._ _ _ Marcus, R. A. On the theory of oxidationu2010 reduction reactions involving electron transfer. I. J. Chem. Phys. 24, 966u2013 978 (1956)._ _ _ Marcus, R. A. Electron transfer reactions in chemistry. Theory Exp. Rev. Mod. Phys. 65, 599u2013610 (1993)._ _ _ Nan, G., Yang, X., Wang, L., Shuai, Z. & Zhao, Y. Nuclear tunneling effects of charge transport in rubrene, tetracene, and pentacene. Phys. Rev. B 79, 115203 (2009)._ _ Kaduk, B., Kowalczyk, T. & Van Voorhis, T. Constrained density functional theory. Chem. Rev. 112, 321u2013370 (2012)._ _ _ Broido, D. A., Malorny, M., Birner, G., Mingo, N. & Stewart, D. A. Intrinsic lattice thermal conductivity of semiconductors from first principles. Appl. Phys. Lett. 91, 231922 (2007)._ _ Garg, J., Bonini, N., Kozinsky, B. & Marzari, N. Role of disorder and anharmonicity in the thermal conductivity of silicon-germanium alloys: a first-principles study. Phys. Rev. Lett. 106, 045901 (2011)._ _ Marcolongo, A., Umari, P. & Baroni, S. Microscopic theory and quantum simulation of atomic heat transport. Nat. Phys. 12, 80u201384 (2016)._ _ _ Carbogno, C., Ramprasad, R. & Scheffler, M. Ab initio Green-Kubo approach for the thermal conductivity of solids. Phys. Rev. Lett. 118, 175901 (2017)._ _ Stackhouse, S., Stixrude, L. & Karki, B. B. Thermal conductivity of periclase (MgO) from first principles. Phys. Rev. Lett. 104, 208501 (2010)._ _ Zhou, F., Nielson, W., Xia, Y. & Ozoliu0146u0161, V. Lattice anharmonicity and thermal conductivity from compressive sensing of first-principles calculations. Phys. Rev. Lett. 113, 185501 (2014)._ _ Puligheddu, M., Gygi, F. & Galli, G. First-principles simulations of heat transport. Phys. Rev. Mater. 1, 060802 (2017)._ _ Puligheddu, M. & Galli, G. Atomistic simulations of the thermal conductivity of liquids. Phys. Rev. Mater. 4, 053801 (2020)._ _ _ Zhang, L., Han, J., Wang, H., Car, R. & E, W. Deep potential molecular dynamics: a scalable model with the accuracy of quantum mechanics. Phys. Rev. Lett. 120, 143001 (2018)._ _ _ Seo, H. et al. Quantum decoherence dynamics of divacancy spins in silicon carbide. Nat. Commun. 7, 12935 (2016)._ _ _ Bourassa, A. et al. Entanglement and control of single nuclear spins in isotopically engineered silicon carbide. Nat. Mater., (2020).Ghosh, K., Ma, H., Gavini, V. & Galli, G. All-electron density functional calculations for electron and nuclear spin interactions in molecules and solids. Phys. Rev. Mater. 3, 043801 (2019)._ _ _ Ma, H. G. Marco; Galli, Giulia. PyZFS: a Python package for first-principles calculations of zero-field splitting tensors. J. Open Source Softw. 5, 2160 (2020)._ _ Wan, Q. & Galli, G. First-principles framework to compute sum-frequency generation vibrational spectra of semiconductors and insulators. Phys. Rev. Lett. 115, 246404 (2015)._ _ Wan, Q., Spanu, L., Galli, G. A. & Gygi, F. Raman spectra of liquid water from ab initio molecular dynamics: vibrational signatures of charge fluctuations in the hydrogen bond network. J. Chem. Theory Comput. 9, 4124u20134130 (2013)._ _ _ Rozsa, V., Pan, D., Giberti, F. & Galli, G. Ab initio spectroscopy and ionic conductivity of water under Earth mantle conditions. Proc. Natl Acad. Sci. USA 115, 6952 (2018)._ _ Wickstrom, L., Okur, A. & Simmerling, C. Evaluating the performance of the ff99SB force field based on NMR scalar coupling data. Biophys. J. 97, 853u2013856 (2009)._ _ _ Goldey, M. B., Brawand, N. P., VxF6rxF6s, M. & Galli, G. Charge transport in nanostructured materials: implementation and verification of constrained density functional theory. J. Chem. Theory Comput. 13, 2581u20132590 (2017)._ _ _ Seo, H., Ping, Y. & Galli, G. Role of point defects in enhancing the conductivity of BiVO4. Chem. Mater. 30, 7793u20137802 (2018)._ _ _ Wang, W. et al. The role of surface oxygen vacancies in BiVO4. Chem. Mater. 32, 2899u2013 2909 (2020)._ _ _ \"- - We thank S. Dong, E. Lee, H. Ma, M. Puligheddu, and W. Wang for fruitful discussions. This work was supported by MICCoM, as part of the Computational Materials Sciences Program funded by the U.S. Department of Energy, Office of Science, Basic Energy Sciences, Materials Sciences, and Engineering Division through Argonne National Laboratory. This research used resources of the National Energy Research Scientific Computing Center (NERSC), a DOE Office of Science User Facility supported by the Office of Science of the US Department of Energy under Contract No. DE-AC02-05CH11231, resources of the Argonne Leadership Computing Facility, which is a DOE Office of Science User Facility supported under Contract DE-AC02-06CH11357, and resources of the University of Chicago Research Computing Center.- - Authors and Affiliations - \"Materials Science Division and Center for Molecular Engineering, Argonne National Laboratory, Lemont, IL, 60439, USAMarco Govoni,_Juan de Pablo_&_Giulia GalliDepartment of Chemical and Biomolecular Engineering, University of Notre Dame, Notre Dame, IN, 46556, USAJonathan WhitmerPritzker School of Molecular Engineering, University of Chicago, Chicago, IL, 60637, USAJuan de Pablo_&_Giulia GalliDepartment of Computer Science, University of California, Davis, CA, 95616, USAFrancois GygiDepartment of Chemistry, University of Chicago, Chicago, IL, 60637, USAGiulia Galli\" - \"AuthorsMarco GovoniYou can also search for this author in _Jonathan WhitmerYou can also search for this author in _Juan de PabloYou can also search for this author in _Francois GygiYou can also search for this author in _Giulia GalliYou can also search for this author in _\" - Contributions - M.G. and G.G. designed the research. The various concepts discussed in the paper were developed and implemented by all authors. All authors contributed to the writing of the paper. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution License).\\' - \\'Fig. 8: DRT at different states of aging.a Baseline anode, b improved anode, c baseline cathode, d improved cathode, e baseline whole cell, f improved whole cell. (Reprinted from ref. . Copyright 2022, with permission from Elsevier).\\' - \"Solid state batteries have aroused a lot of attentions recently. The basic principles of EIS are also applied to all solid-state batteries. One unique feature of an all solid-state battery which results from the nature of a solid electrolyte is the charge carriers of the solid-state electrolyte. Unlike a liquid electrolyte, in which the electric current between electrodes is carried by ionic species, there may be electronic conductivity in a solid-state electrolyte. Although it is small, the electronic conductivity is responsible to the detrimental leakage current, which is equivalent to connect a large resistor between the cathode and anode. Theoretically, an electronic conductivity and ionic conductivity can be easily separated using both DC and AC. Experimentally it is hard to get a reproducible measurement. A simple ac impedance measurement and fitting using a carefully designed electrochemical cell can be advantageous. During the impedance measurements, an ion-blocking or electron-blocking or a redox electrode can be used. Ion-blocking electrodes can be metals e.g., Pt and Au or pyrolytic graphite, an ion-blocking electrode is inert and doesnt facilitate charge transfer at the interface. Electron-blocking electrodes are high ionic conductivity materials with negligible electronic conductivity. Therefore, the only ionic conductivity is measured. If a redox electrode is used e.g., Li metal, then both ionic and electronic conductivity are included in the impedance measurement.\"- - EIS has been used for many decades. The proliferations of the EIS application, especially its application in energy storages occurred in the past decade. The revolutionary advancements in computers and electronics have made EIS analysis very easy and reduced the cost of EIS instruments drastically. Historically, an AC Wheatstone bridge was used for impedance measurement. It took an experienced electrochemist hour to measure the impedance at one frequency, especially at a low frequency. Now, even relatively affordable electrochemical workstations come with a built-in impedance spectrometer. Instead of running each frequency individually, the multiple-frequency excitation can be multiplexed and superimposed on a bias DC signal. With the help of Fourier transformation and digital frequency response analyzer, all frequency can be measured simultaneously with automatic output of real and complex spectra. Now an EIS measurement can be done by pushing the enter key on the keyboard. The advancements of the technology have made the powerful EIS technique widely available not only to electrochemists but also to professionals in the areas of material study, engineering, production quality control etc. Like every advancement in technology, there are always negative impacts. Here are two major issues found in many publications. - \\' As mentioned earlier, EIS is a steady-state technique. The electrochemical system must be at an equilibrium state or at least the DC signal must be substantially smaller than the responding ac signals, when the impedance is measured. Almost all the electrodes used in energy storage are porous and the relaxation time is long. Using a modern impedance spectrometer, a set of impedance results can be produced. Even if the system is far from its equilibrium state, the obtained spectrum could be non-reproducible and misleading. There is a clear trend of using EIS very casually and lightly in energy storage research. Due to the increasing convenience and affordability of the instrument, researchers just feel the pressure to have a Nyquist plot in the manuscript. Without a careful conduction of the experiments, validating the impedance data points frequency by frequency, establishing equivalent circuits, fitting the experimental data based on the equivalent circuit, and error analysis, the results can be at the very least useless or even misleading. \\' - EIS is a powerful technique in electrochemical studies. It bridges the steady state and transient state so the kinetics of an electrode can be investigated at different stages of discharge or recharge. We hope it can be rightfully used to maximize the potential of the technique. We hope this discussion can provide a useful starting point for researchers in energy storage area, especially those who are not professional electrochemists. - The last but not the least, impedance techniques covered in this paper are for the investigation of electrochemical processes in a battery. In industries, impedance is also used as diagnostic tool for the state-of-health and state-of-charge. Those could be a single frequency impedance or discrete multi-frequency impedance. Since a large excitation signal is normally applied, the response would be non-linear. Most of the simulation models are semi-empirical and mostly proprietary.- - \"Qu, D. et al. Electrochemical impedance and its applications in energy-storage systems. Small Methods 2, 1700342 (2018)._ _ Chang, B.-Y. & Park, S.-M. Electrochemical impedance spectroscopy. Annu. Rev. Anal. Chem. 3, 207u2013229 (2010)._ _ _ Meddings, N. et al. Application of electrochemical impedance spectroscopy to commercial Li-ion cells: A review. J. Power Sources 480, 228742 (2020)._ _ _ Mc Carthy, K., Gullapalli, H., Ryan, K. M. & Kennedy, T. Reviewu2014 Use of impedance spectroscopy for the estimation of Li-ion battery state of charge, state of health and internal temperature. J. Electrochem. Soc. 168, 080517 (2021)._ _ Fan, C., ORegan, K., Li, L., Kendrick, E. & Widanage, W. D. Frequency domain non-linear characterization and analysis of lithium-ion battery electrodes. J. Energy Storage 36, 102371 (2021)._ _ Kramers, H. A. Die dispersion und absorption von RxF6ntgenstrahlen. Phys. Z. 30, 522u2013523 (1929)._ _ de L Kronig, R. On the theory of dispersion of X-rays. J. Opt. Soc. Am. 12, 547u2013557 (1926)._ _ Bode, H. W. Network Analysis and Feedback Amplifier Design (Van Nostrand, 1945).Qu, H. et al. Application of ac impedance as diagnostic toolu2014Low temperature electrolyte for a Li-ion battery. Electrochim. Acta 322, 134755 (2019)._ _ _ Ji, W., Qu, H., Zhang, X., Zheng, D. & Qu, D. Electrode architecture design to promote charge-transport kinetic in high-loading and high-energy lithium-based batteries. Small Methods 5, 2100518 (2021)._ _ _ Dawson, J. L. & John, D. G. Diffusion impedanceu2014An extended general analysis. J. Electroanal. Chem. Interf. Electrochem. 110, 37u201347 (1980)._ _ _ Qu, D. Application of a.c. impedance technique to the study of the proton diffusion process in the porous MnO2 electrode. Electrochim. Acta 48, 1675u20131684 (2003)._ _ _ Qu, D. The study of the proton diffusion process in the porous MnO2 electrode. Electrochim. Acta 49, 657u2013665 (2004)._ _ _ de Levie, R. On porous electrodes in electrolyte solutions: I. Capacitance effects. Electrochim. Acta 8, 751u2013780 (1963)._ _ de Levie, R. On porous electrodes in electrolyte solutionsu2014IV. Electrochim. Acta 9, 1231u2013 1245 (1964)._ _ de Levie, R. Advances in Electrochemistry and Electrochemical Engineering (Wiley-Interscience, 1967).Qu, D. High discharge rate alkaline battery. US6960409B2 (2005).Qu, D. Investigation of the porosity of electrolytic manganese dioxide and its performance as alkaline cathode material. J. Power Sources 156, 692u2013699 (2006)._ _ _ Gourdin, G., Meehan, A., Jiang, T., Smith, P. & Qu., D. Investigation of the impact of stacking pressure on a double-layer supercapacitor. J. Power Sources 196, 523u2013529 (2011)._ _ _ Qu, D. The ac impedance studies for porous MnO2 cathode by means of modified transmission line model. J. Power Sources 102, 270u2013276 (2001)._ _ _ Barnard, R., Baugh, L. M. & Randell, C. F. The impedance of alkaline manganese cells and their relationship to cell performance. I. Interpretation of the basic impedance spectrum. J. Appl. Electrochem. 17, 165u2013173 (1987)._ _ _ Bard, A. J. & Faulkner, L. R. Electrochemical Methods, Fundamentals and Applications 2nd edn (John Wiley & Sons, Inc., 2001).Ivers-Tiffee, E. & Weber, A. Evaluation of electrochemical impedance spectra by the distribution of relaxation times. J. Ceram. Soc. Japan 125, 193u2013201 (2017)._ _ _ Danzer, M. A. Generalized distribution of relaxation times analysis for the characterization of impedance spectra. Batteries 5, 53 (2019)._ _ _ Smirnova, A. L., Ellwood, K. R. & Crosbie, G. M. Application of Fourier-based transforms to impedance spectra of small-diameter tubular solid oxide fuel cells. J. Electrochem. Soc. 148, A610u2013A615 (2001)._ _ _ Qu, H., Zhang, X., Ji, W., Zheng, D. & Qu, D. Impedance investigation of the high temperature performance of the solid-electrolyte-interface of a wide temperature electrolyte. J. Colloid Interface Sci. 608, 3079u20133086 (2022)._ _ _ Huggins, R. A. Simple method to determine electronic and ionic components of the conductivity in mixed conductors. Ionics 8, 300u2013312 (2002)._ _ _ \"- - Authors and Affiliations - \"Department of Mechanical Engineering, University of Wisconsin Milwaukee, Milwaukee, WI, 53211, USADeyang Qu,_Weixiao Ji_&_Huainan Qu\" - \"AuthorsDeyang QuYou can also search for this author in _Weixiao JiYou can also search for this author in _Huainan QuYou can also search for this author in _\" - Contributions - \\'D.Q.: Conceptualization, manuscript structuring, and writing the first draft. W.J.: Contributing to conceptualization, reviewing and editing the manuscript, and graphing artworks. H.Q.: Participating, writing and editing, and obtaining copyright permission.\\' - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \"Communications Materials thanks Min Hong and the other, anonymous, reviewers for their contribution to the peer review of this work. Primary Handling Editors: Jie Xiao and John Plummer._ are available\" - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution License, which permits unrestricted use, distribution and reproduction in any medium, provided the original author and source are credited.\"- - \"Bernardo, M. E., Locatelli, F. & Fibbe, W. E. Mesenchymal stromal cells. Ann. NY Acad. Sci. 1176, 101u2013117 (2009)._ _ _ _ Kuhn, N. Z. & Tuan, R. S. Regulation of stemness and stem cell niche of mesenchymal stem cells: implications in tumorigenesis and metastasis. J. Cell. Physiol. 222, 268u2013277 (2010)._ _ _ Rogers, C. D., Moody, S. A. & Casey, E. S. Neural induction and factors that stabilize a neural fate. Birth Defects Res. C Embryo Today 87, 249u2013262 (2009)._ _ _ Wegner, M. From head to toes: the multiple facets of Sox proteins. Nucleic Acids Res. 27, 1409u20131420 (1999)._ _ _ _ Lefebvre, V., Dumitriu, B., Penzo-MxE9 ndez, A., Han, Y. & Pallavi, B. Control of cell fate and differentiation by Sry-related high-mobility-group box (Sox) transcription factors. Int. J. Biochem. Cell Biol. 39, 2195u20132214 (2007)._ _ _ Takahashi, K. & Yamanaka, S. Induction of pluripotent stem cells from mouse embryonic and adult fibroblast cultures by defined factors. Cell 126, 663u2013676 (2006)._ _ _ Sekido, R. & Lovell-Badge, R. Sex determination and SRY: down to a wink and a nudge? Trends Genet. 25, 19u201329 (2009)._ _ _ Lefebvre, V. & Smits, P. Transcriptional control of chondrocyte fate and differentiation. Birth Defects Res. C Embryo Today 75, 200u2013212 (2005)._ _ _ Francois, M., Koopman, P. & Beltrame, M. SoxF genes: key players in the development of the cardio-vascular system. Int. J. Biochem. Cell. Biol. 42, 445u2013448 (2010)._ _ _ Penzo-MxE9ndez, A. I. Critical roles for SoxC transcription factors in development and cancer. Int. J. Biochem. Cell Biol. 42, 425u2013 428 (2010)._ _ Dy, P. et al. The three SoxC proteinsu2014 Sox4, Sox11 and Sox12u2014exhibit overlapping expression patterns and molecular properties. Nucleic Acids Res. 36, 3101u20133117 (2008)._ _ _ Hoser, M. et al. Sox12 deletion in the mouse reveals nonreciprocal redundancy with the related Sox4 and Sox11 transcription factors. Mol. Cell Biol. 28, 4675u20134687 (2008)._ _ _ Hargrave, M. et al. Expression if the Sox11 gene in mouse embryos suggests roles in neuronal maturation and epithelio-mesenchymal induction. Dev. Dyn. 210, 79u201386 (1997)._ _ _ Sock, E. et al. Gene targeting reveals a widespread role for the high-mobility-group transcription factor Sox11 in tissue remodeling. Mol. Cell Biol. 24, 6635u20136644 (2004)._ _ _ Schilham, M. W. et al. Defects in cardiac outflow tract formation and pro-B-lymphocyte expansion in mice lacking Sox-4. Nature 380, 711u2013714 (1996)._ _ _ _ Schilham, M. W., Moerer, P., Cumano, A. & Clevers, H. C. Sox-4 facilitates thymocyte differentiation. Eur. J. Immunol. 27, 1292u2013 1295 (1997)._ _ _ Wilson, M. E. et al. The HMG box transcription factor Sox4 contributes to the development of the endocrine pancreas. Diabetes 54, 3402u20133409 (2005)._ _ _ Nissen-Meyer, L. S. et al. Osteopenia, decreased bone formation and impaired osteoblast development in Sox4 heterozygous mice. J. Cell Sci. 120, 2785u20132795 (2007)._ _ _ Bergsland, M., Werme, M., Malewicz, M., Perlmann, T. & Muhr, J. The establishment of neuronal properties is controlled by Sox4 and Sox11. Genes Dev. 20, 3475u20133486 (2006)._ _ _ Lee, C. J., Appleby, V. J., Orme, A. T., Chan, W. I. & Scotting, P. J. Differential expression of SOX4 and SOX11 in medulloblastoma. J. Neurooncol. 57, 201u2013 214 (2002)._ _ Weigle, B. et al. Highly specific overexpression of the transcription factor SOX11 in human malignant gliomas. Oncol. Rep. 13, 139u2013144 (2005)._ _ _ Hide, T. et al. Sox11 prevents tumorigenesis of glioma-initiating cells by inducing neuronal differentiation. Cancer Res. 69, 7953u20137959 (2009)._ _ _ Wang, X. et al. The subcellular Sox11 distribution pattern identifies subsets of mantle cell lymphoma: correlation to overall survival. Br. J. Haematol. 143, 248u2013 252 (2008)._ _ Liu, P. et al. Sex-determining region Y box 4 is a transforming oncogene in human prostate cancer cells. Cancer Res. 66, 4011u20134019 (2006)._ _ _ Andersen, C. L. et al. Dysregulation of the transcription factors SOX4, CBFB and SMARCC1 correlates with outcome of colorectal cancer. Br. J. Cancer 100, 511u2013523 (2009)._ _ _ Pramoonjago, P., Baras, A. S. & Moskaluk, C. A. Knockdown of Sox4 expression by RNAi induces apoptosis in ACC3 cells. Oncogene 25, 5626u20135639 (2006)._ _ _ de Bont, J. M. et al. Differential expression and prognostic significance of SOX genes in pediatric medulloblastoma and ependymoma identified by microarray analysis. Neuro. Oncol. 10, 648u2013660 (2008)._ _ _ Pan, X. et al. Induction of SOX4 by DNA damage is critical for p53 stabilization and function. Proc. Natl. Acad. Sci USA 106, 3788u20133793 (2009)._ _ _ _ Penzo-MxE9ndez, A., Dy, P., Pallavi, B. & Lefebvre, V. Generation of mice harboring a Sox4 conditional null allele. Genesis 45, 776u2013780 (2007)._ _ O\\'Gorman, S., Dagenais, N. A., Qian, M. & Marchuk, Y Protamine-Cre recombinase transgenes efficiently recombine target sequences in the male germ line of mice, but not in embryonic stem cells. Proc. Natl. Acad. Sci. USA 94, 14602u201314607 (1997)._ _ _ _ De Vries, W. N. et al. Expression of Cre recombinase in mouse oocytes: a means to study maternal effect genes. Genesis 26, 110u2013112 (2000)._ _ _ Ahn, K., Mishina, Y., Hanks, M. C., Behringer, R. R. & Crenshaw, E. B. III. BMPR-IA signaling is required for the formation of the apical ectodermal ridge and dorsal-ventral patterning of the limb. Development 128, 4449u20134461 (2001)._ _ _ Danielian, P. S., Muccino, D., Rowitch, D. H., Michael, S. K. & McMahon, A. P. Modification of gene activity in mouse embryos in utero by a tamoxifen-inducible form of Cre recombinase. Curr. Biol. 8, 1323u20131326 (1998)._ _ _ Mao, X., Fujiwara, Y. & Orkin, S. H. Improved reporter strain for monitoring Cre recombinase-mediated DNA excisions in mice. Proc. Natl. Acad. Sci. USA 96, 5037u20135042 (1999)._ _ _ _ Logan, M. et al. Expression of Cre Recombinase in the developing mouse limb bud driven by a Prxl enhancer. Genesis 33, 77u201380 (2002)._ _ _ Zhao, B., Lei, Q. -Y. & Guan, K. -L. The Hippo-YAP pathway: new connections between regulation of organ size and cancer. Curr. Opin. Cell Biol. 20, 638u2013646 (2008)._ _ _ Kaneko, K. J., Kohn, M. J., Liu, C. & DePamphilis, M. L. Transcription factor TEAD2 is involved in neural tube closure. Genesis 45, 577u2013587 (2007)._ _ _ Sawada, A. et al. Redundant roles of Tead1 and Tead2 in notochord development and the regulation of cell proliferation and survival. Mol. Cell. Biol. 28, 3177u20133189 (2008)._ _ _ Vassilev, A., Kaneko, K. J., Shu, H., Zhao, Y. & DePamphilis, M.L. TEAD/TEF transcription factors utilize the activation domain of YAP65, a Src/Yes-associated protein localized in the cytoplasm. Genes Dev. 15, 1229u20131241 (2001)._ _ _ Avilion, A. A. et al. Multipotent cell lineages in early mouse development depend on SOX2 function. Genes Dev. 17, 126u2013 140 (2003)._ _ _ Masui, S. et al. Pluripotency governed by Sox2 via regulation of Oct3/4 expression in mouse embryonic stem cells. Nat. Cell Biol. 9, 625u2013635 (2007)._ _ _ Ikushima, H. et al. Autocrine TGF-beta signaling maintains tumorigenicity of glioma-initiating cells through Sry-related HMG-box factors. Cell Stem Cell 5, 504u2013514 (2009)._ _ _ Hoser, M. et al. Prolonged glial expression of Sox4 in the CNS leads to architectural cerebellar defects and ataxia. J. Neurosci. 27, 5495u20135505 (2007)._ _ _ Potzner, M. R. et al. Prolonged Sox4 expression in oligodendrocytes interferes with normal myelination in the central nervous system. Mol. Cell. Biol. 27, 5316u20135326 (2007)._ _ _ Kondoh, H. & Kamachi, Y. SOX-partner code for cell specification: Regulatory target selection and underlying molecular mechanisms. Int. J. Biochem. Cell. Biol. 42, 391u2013399 (2010)._ _ _ Liao, Y. -L. et al. Identification of SOX4 target genes using phylogenetic footprinting-based prediction from expression microarrays suggests that overexpression of SOX4 potentiates metastasis in hepatocellular carcinoma. Oncogene 27, 5578u20135589 (2008)._ _ _ Cheah, S. S. & Behringer, R. R. Contemporary gene targeting strategies for the novice. Mol. Biotechnol. 19, 297u2013304 (2001)._ _ _ Meyers, E. N., Lewandoski, M. & Martin, G. R. An Fgf8 mutant allelic series generated by Cre- and Flp-mediated recombination. Nat. Genet. 18, 136u2013141 (1998)._ _ _ Hogan, B., Beddington, R., Costantini, F. & Lacy, E. Manipulating the Mouse Embryo: A Laboratory Manual 2nd edn. 373u2013375 (Cold Spring Harbor Laboratory Press, 1994).Albrecht, U., Eichele, G., Helms, J. A. & Lu, H. C. Visualization of Gene Expression Patterns by In Situ Hybridization. In Molecular and Cellular Methods in Developmental Toxicology (ed. Daston, G.P.) 23u201348 (CRC Press, Boca Raton, FL, 1997).Ducy, P. et al. A Cbfa1-dependent genetic pathway controls bone formation beyond embryonic development. Genes Dev. 13, 1025u20131036.Livak, K. J. & Schmittgen, T. D. Analysis of relative gene expression data using real-time quantitative PCR and the 2(u2212Delta Delta C(T)). Methods 25, 402u2013408 (2001)._ _ _ Kuschert, S., Rowitch, D. H., Haenig, B., McMahon, A. P. & Kispert, A. Characterization of Pax-2 regulatory sequences that direct transgene expression in the Wolffian duct and its derivatives. Dev. Biol. 229, 128u2013140 (2001)._ _ _ Ott, M. O., Bober, E., Lyons, G., Arnold, H. & Buckingham, M. Early expression of the myogenic regulatory gene, myf-5, in precursor cells of skeletal muscle in the mouse embryo. Development 111, 1097u20131107 (1991)._ _ _ Echelard, Y. et al. Sonic hedgehog, a member of a family of putative signaling molecules, is implicated in the regulation of CNS polarity. Cell 75, 1417u20131430 (1993)._ _ _ Sawada, A. et al. Tead proteins activate the Foxa2 enhancer in the node in cooperation with a second factor. Development 132, 4719u20134729 (2005)._ _ _ Caballero, R. et al. Combinatorial effects of splice variants modulate function of Aiolos. J. Cell Sci. 120, 2619u20132630 (2007)._ _ _ Han, Y. & Lefebvre, V. L-Sox5 and Sox6 drive expression of the aggrecan gene in cartilage by securing binding of Sox9 to a far-upstream enhancer. Mol. Cell Biol. 28, 4999u20135013 (2008)._ _ _ \"- - We thank H. Wang, P. Dy and A. Silvester for expert technical assistance; J. Jensen, T. Sakai, J.A. Drazba, P.W. Faber and C. Shemo for scientific and technical advice; R. Conlon, J. Martin Y. Mishina and H. Sasaki for in situ probes; and D. Driscoll, Y. Mishina, O. Reizes and A. Zhu for advice on the paper. This work was funded by NIH Grant AR54153 to V.L., DFG Grant SO251/3-1 to E.S. and an Arthritis Foundation postdoctoral fellowship to A.P.-M.- - Author notes - \"Pallavi Bhattaram and Alfredo Penzo-MxE9ndez: These authors contributed equally to this work.\" - Authors and Affiliations - \"Department of Cell Biology, and Orthopaedic and Rheumatologic Research Center, Cleveland Clinic Lerner Research Institute, Cleveland, 44195, Ohio, USAPallavi Bhattaram,_Alfredo Penzo-MxE9ndez_&_VxE9ronique LefebvreInstitut fxFC r Biochemie, Emil-Fischer-Zentrum, UniversitxE4t Erlangen-NxFCrnberg, 91054 Erlangen, GermanyElisabeth Sock_&_Michael WegnerDepartment of Cancer Biology, Cleveland Clinic Lerner Research Institute, Cleveland, 44195, Ohio, USAClemencia ColmenaresNational Institute of Child Health and Human Development, National Institutes of Health, Bethesda, 20892, Maryland, USAKotaro J. Kaneko,_Alex Vassilev_&_Melvin L. DePamphilis\" - \"AuthorsPallavi BhattaramYou can also search for this author in _Alfredo Penzo-MxE9ndezYou can also search for this author in _Elisabeth SockYou can also search for this author in _Clemencia ColmenaresYou can also search for this author in _Kotaro J. KanekoYou can also search for this author in _Alex VassilevYou can also search for this author in _Melvin L. DePamphilisYou can also search for this author in _Michael WegnerYou can also search for this author in _VxE9ronique LefebvreYou can also search for this author in _\" - Contributions - P.B., A.P.-M. and E.S. were responsible for the experiments and wrote the paper; K.J.K., A.V. and M.L.D. provided Tead2 mutant mice and retroviral vectors; C.C. contributed technical and scientific help; and E.S., M.W. and V.L. directed the project and wrote the paper. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing financial interests. - \\' \\'- - Supplementary Figures S1-S8 and Supplementary Tables S1-S4 (PDF 1548 kb)- - \\' \\' - \\'This work is licensed under a Creative Commons Attribution-NonCommercial-Share Alike 3.0 License. ',\n",
       " 'Creative Commons Attribution License, which permits unrestricted use, distribution and reproduction in any medium, provided the original author and source are credited.\"- - \\' \\' - Accessions - \\' \\' - ArrayExpress - \\' \\' - \\' \\'- - \\'13 May 2014A correction has been published and is appended to both the HTML and PDF versions of this paper. The error has not been fixed in the paper.13 May 2014Nature Communications 1: Article number: 11 (2010); Published: 12 April 2010; Updated: 13 May 2014. After reanalysing the original electron micrographs, as well as new ones, we discovered that, contrary to what was stated in the paper, the ciliary dilations in DCX-EMAP mutants do contain electron-dense material (arrow in new Fig.\\'- - \"Vogel, V. Mechanotransduction involving multimodular proteins: converting force into biochemical signals. Annu. Rev. Biophys. Biomol. Struct. 35, 459u2013 488 (2006)._ _ _ _ Gillespie, P. G. & Walker, R. G. Molecular basis of mechanosensory transduction. Nature 413, 194u2013 202 (2001)._ _ _ _ _ Vollrath, M. A., Kwan, K. Y. & Corey, D. P. The micromachinery of mechanotransduction in hair cells. Annu. Rev. Neurosci. 30, 339u2013365 (2007)._ _ _ _ _ Bounoutas, A. & Chalfie, M. Touch sensitivity in Caenorhabditis elegans. Pflugers Arch. 454, 691u2013702 (2007)._ _ _ _ Marshall, W. F. & Nonaka, S. Cilia: tuning in to the cell\\'s antenna. Curr. Biol. 16, R604u2013R614 (2006)._ _ _ _ Keil, T. A. Functional morphology of insect mechanoreceptors. Microsc. Res. Tech. 39, 506u2013531 (1997)._ _ _ _ Nadrowski, B., Albert, J. T. & GxF6pfert, M. C. Transducer-based force generation explains active process in Drosophila hearing. Curr. Biol. (2008).Kernan, M., Cowan, D. & Zuker, C. Genetic dissection of mechanosensory transduction: mechanoreception-defective mutations of Drosophila. Neuron 12, 1195u20131206 (1994)._ _ _ _ Eberl, D., Duyk, G. & Perrimon, N. A genetic screen for mutations that disrupt an auditory response in Drosophila melanogaster. Proc. Natl. Acad. Sci. USA 94, 14837u2013 14842 (1997)._ _ _ _ _ _ Suprenant, K. A., Dean, K., McKee, J. & Hake, S. EMAP, an echinoderm microtubule-associated protein found in microtubule-ribosome complexes. J. Cell Sci. 104, 445u2013 450 (1993)._ _ _ Li, Q. & Suprenant, K. Molecular characterization of the 77-kDa echinoderm microtubule-associated protein. Homology to the beta-transducin family. J. Biol. Chem. 269, 31777u201331784 (1994)._ _ _ Eichenmuller, B., Everley, P., Palange, J., Lepley, D. & Suprenant, K. A. The human EMAP-like protein-70 (ELP70) is a microtubule destabilizer that localizes to the mitotic apparatus. J. Biol. Chem. 277, 1301u2013 1309 (2002)._ _ _ _ Pollmann, M., Parwaresch, R., Adam-Klages, S., Kruse, M. L., Buck, F. & Heidebrecht, H. J. Human EML4, a novel member of the EMAP family, is essential for microtubule formation. Exp. Cell Res. 312, 3241u20133251 (2006)._ _ _ _ Soda, M., Choi, Y. L., Enomoto, M., Takada, S., Yamashita, Y. & Ishikawa, S. et al. Identification of the transforming EML4-ALK fusion gene in non-small-cell lung cancer. Nature 448, 561u2013566 (2007)._ _ _ _ _ Tegha-Dunghu, J., Neumann, B., Reber, S., Krause, R., Erfle, H. & Walter, T. et al. EML3 is a nuclear microtubule-binding protein required for the correct alignment of chromosomes in metaphase. J. Cell Sci. 121, 1718u2013 1726 (2008)._ _ _ _ Suprenant, K. et al. Conservation of the WD-repeat, microtubule-binding protein, EMAP, in sea urchins, humans, and the nematode C. elegans. Dev. Genes Evol. 210, 2u201310 (2000)._ _ _ _ O\\'Connor, V., Houtman, S. H., Zeeuw, C. I. D., Bliss, T. V. P. & French, P. J. Eml5, a novel WD40 domain protein expressed in rat brain. Gene 336, 127u2013137 (2004)._ _ _ _ Hueston, J. L. et al. The C. elegans EMAP-like protein, ELP-1 is required for normal touch sensation and associates with microtubules and adhesion complexes. BMC Dev. Biol. 8, 110 (2008)._ _ _ _ des Portes, V. et al. A novel CNS gene required for neuronal migration and involved in X-linked subcortical laminar heterotopia and lissencephaly syndrome. Cell 92, 51u201361 (1998)._ _ _ _ Gleeson, J., Allen, K., Fox, J. & Lamperti, E. Doublecortin, a brain-specific gene mutated in human X-linked lissencephaly and double cortex syndrome, encodes a putative signaling protein. Cell 92, 63u201372 (1998)._ _ _ _ Moores, C. A., Perderiset, M., Francis, F., Chelly, J., Houdusse, A. & Milligan, R. A. Mechanism of microtubule stabilization by doublecortin. Mol. Cell 14, 833u2013839 (2004)._ _ _ _ Moores, C. A. et al. Distinct roles of doublecortin modulating the microtubule cytoskeleton. EMBO J. 25, 4448u20134457 (2006)._ _ _ _ _ Thurm, U. Mechanoreceptors in the cuticle of the honey bee: fine structure and stimulus mechanism. Science 145, 1063u20131065 (1964)._ _ _ _ _ Field, L. H. & Matheson, T. in Advances in Insect Physiology 27, 1u2013228 (1998)._ _ Chan, W. P., Prete, F. & Dickinson, M. H. Visual input to the efferent control system of a fly\\'s \\'gyroscope\\'. Science 280, 289u2013292 (1998)._ _ _ _ _ Cole, E. S. & Palka, J. The pattern of campaniform sensilla on the wing and haltere of Drosophila melanogaster and several of its homeotic mutants. J Embryol Exper Morphol 71, 41u201361 (1982)._ _ Li, J. et al. Comparative genomics identifies a flagellar and basal body proteome that includes the BBS5 human disease gene. Cell 117, 541u2013552 (2004)._ _ _ _ Avidor-Reiss, T. et al. Decoding cilia function: defining specialized genes required for compartmentalized cilia biogenesis. Cell 117, 527u2013539 (2004)._ _ _ _ Walker, R. G., Willingham, A. T. & Zuker, C. S. A Drosophila mechanosensory transduction channel. Science 287, 2229u20132234 (2000)._ _ _ _ _ Lyne, R. et al. FlyMine: an integrated database for Drosophila and Anopheles genomics. Genome Biol. 8, R129 (2007)._ _ _ _ Tomancak, P. et al. Systematic determination of patterns of gene expression during Drosophila embryogenesis. Genome Biol. 3, RESEARCH0088 (2002)._ _ _ _ Liu, L. et al. Drosophila hygrosensation requires the TRP channels water witch and nanchung. Nature 450, 294u2013298 (2007)._ _ _ _ _ McDermott, B. M., Baucom, J. M. & Hudspeth, A. J. Analysis and functional evaluation of the hair-cell transcriptome. Proc. Natl. Acad. Sci. USA 104, 11820u2013 11825 (2007)._ _ _ _ _ _ Thisse, B. & Thisse, C. Fast release clones: a high throughput expression analysis. ZFIN Direct Data Submission (http://zfin.org) (2004).Thibault, S. T. et al. A complementary transposon tool kit for Drosophila melanogaster using P and piggyBac. Nat. Genet. 36, 283u2013287 (2004)._ _ _ _ Spradling, A., Stern, D., Kiss, I., Roote, J., Laverty, T. & Rubin, G. Gene disruptions using P transposable elements: an integral component of the Drosophila genome project. Proc. Natl. Acad. Sci. USA 92, 10824u201310830 (1995)._ _ _ _ _ _ Benzer, S. Genetic dissection of behavior. Sci. Am. 229, 24u201337 (1973)._ _ _ _ Albert, J. T., Nadrowski, B. & GxF6pfert, M. C. Mechanical signatures of transducer gating in the Drosophila ear. Curr. Biol. 17, 1000u20131006 (2007)._ _ _ _ GxF6 pfert, M. C. & Robert, D. Motion generation by Drosophila mechanosensory neurons. Proc. Natl. Acad. Sci. USA 100, 5514u20135519 (2003)._ _ _ _ _ GxF6pfert, M. C., Albert, J. T., Nadrowski, B. & Kamikouchi, A. Specification of auditory sensitivity by Drosophila TRP channels. Nat. Neurosci. 9, 999u20131000 (2006)._ _ _ Chu-Wang, I. W. & Axtell, R. C. Fine structure of the terminal organ of the house fly larva, Musca domestica L. Zeitschrift fxFCr Zellforschung und mikroskopische Anatomie (Vienna, Austria: 1948) 127, 287u2013305 (1972)._ _ _ Lee, E., Sivan-Loukianova, E., Eberl, D. F. & Kernan, M. J. An IFT-A protein is required to delimit functionally distinct zones in mechanosensory cilia. Curr. Biol. 18, 1899u20131906 (2008)._ _ _ _ _ Smith, T., Gaitatzes, C., Saxena, K. & Neer, E. The WD repeat: a common architecture for diverse functions. Trends Biochem. Sci. 24, 181u2013185 (1999)._ _ _ _ Kamikouchi, A. et al. The neural basis of Drosophila gravity-sensing and hearing. Nature 458, 165u2013171 (2009)._ _ _ _ _ Brand, A. H. & Perrimon, N. Targeted gene expression as a means of altering cell fates and generating dominant phenotypes. Development 118, 401u2013415 (1993)._ _ _ Wu, Z. J., Irizarry, R. A., Gentleman, R., Martinez-Murillo, F. & Spencer, F. A model-based background adjustment for oligonucleotide expression arrays. J. Am. Stat. Assoc. 99, 909u2013917 (2004)._ _ _ Barrett, T. et al. NCBI GEO: mining millions of expression profilesu2014database and tools. Nucleic. Acids Res. 33, D562u2013 D566 (2005)._ _ _ _ Albert, J. T., Nadrowski, B., Kamikouchi, A. & GxF6pfert, M. C. Mechanical tracing of protein function in the Drosophila ear. Nature Protocols(doi:10.1038/nprot.2006.364) (2006).GxF6 pfert, M. C., Humphris, A. D. L., Albert, J. T., Robert, D. & Hendrich, O. Power gain exhibited by motile mechanosensory neurons in Drosophila ears. Proc. Natl. Acad. Sci. USA 102, 325u2013330 (2005)._ _ _ _ Muller-Reichert, T., Srayko, M., Hyman, A. & Ou2032Toole, E. Correlative light and electron microscopy of early Caenorhabditis elegans embryos in mitosis. Methods Cell Biol. 79, 101u2013119 (2007)._ _ _ MxFCller-Reichert, T., Hohenberg, H., O\\'Toole, E. T. & McDonald, K. Cryoimmobilization and three-dimensional visualization of C. elegans ultrastructure. J. Microsc. 212, 71u201380 (2003)._ _ _ _ Bird, A. W. & Hyman, A. A. Building a spindle of the correct length in human cells requires the interaction between TPX2 and Aurora A. J. Cell Biol. 182, 289u2013300 (2008)._ _ _ _ _ \"- - \"We thank Yun Doo Chung for reagents and Jana MxE4ntler, Ingmar Riedel-Kruse and Patrik Rath for technical help. This work was supported by grants from EuroHear (contract LSHG-CT-2004-512063), Deafness Research UK, the Biotechnology and Biological Sciences Research Council UK (J.T.A), the Volkswagen Foundation (S.B., J.H. and M.C.G.), the BMBF Bernstein Network for Computational Neuroscience and from the German Science Foundation (DFG, GO 1092/1-1) (M.C.G.).\"- - Author notes - \"S. BechstedtPresent address: u2020Present addresses: Department of Biology, McGill University, Montreal, Canada, T. MxFCller-Reichert: Medical Theoretical Center, Dresden University of Technology, Dresden, Germany\" - Authors and Affiliations - \"Max Planck Institute of Molecular Cell Biology and Genetics, Dresden, GermanyS. Bechstedt,_T. MxFCller-Reichert_&_J. HowardUCL Ear Institute, London, UKJ.T. AlbertChair of Bioinformatics, Boku University Vienna, Vienna, AustriaD.P. KreilDepartment of Cellular Neurobiology, University of GxF6ttingen, GxF6ttingen, GermanyM.C. GxF6pfert\" - \"AuthorsS. BechstedtYou can also search for this author in _J.T. AlbertYou can also search for this author in _D.P. KreilYou can also search for this author in _T. MxFCller-ReichertYou can also search for this author in _M.C. GxF6pfertYou can also search for this author in _J. HowardYou can also search for this author in _\" - Contributions - S.B. designed and conducted most of the experiments, analysed the data and wrote the paper. J.T.A. and M.C.G. designed and conducted the experiments shown in and and participated in writing the paper. T.M.-R. was involved in experiments and data analysis shown in and . D.P.K. analysed the microarray data. J.H. conceived the project, and participated in designing the experiments and writing the paper, together with S.B. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing financial interests. - \\' \\'- - Supplementary Methods, Supplementary Figures S1-S3 and Supplementary Tables S1-S2 (PDF 593 kb)Candidate Gene List (PDF 187 kb)- - \\' \\' - \\'This work is licensed under a Creative Commons Attribution-NonCommercial-Share Alike 3.0 License. ',\n",
       " 'Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.\"- - \\' \\' - Accessions - \\' \\' - GenBank/EMBL/DDBJ - \\' \\' - \\' \\'- - \"Bradbury, J. W. & Vehrencamp, S. L. Principles of Animal Communication (Sinauer Associates, 1998).Tinbergen, N. \\'Derived\\' activities; their causation, biological significance, origin, and emancipation during evolution. Q. Rev. Biol. 27, 1u2013 23 (1952)._ _ _ Huxley, J. S. A discussion on ritualization of behaviour in animals and manu2013Introduction. Philos. Trans. R. Soc. Lond, B 251, 249u2013271 (1966)._ _ _ Lorenz, K. Z. Evolution of ritualization in the biological and cultural spheres. Philos. Trans. R. Soc. Lond, B 251, 273u2013284 (1966)._ _ _ Alexander, R. D. Evolutionary change in cricket acoustical communication. Evolution 16, 443u2013462 (1962)._ _ Elsner, N. A neuroethological approach to the phylogeny of leg stridulation in gomphocerine grasshoppers. in Neuroethology and Behavioral Physiology (eds Huber, F. & Markl, H.) 54u2013 68 (Springer, 1983).Schenkel, R. Zur Deutung der Balzleistungen einiger Phasianiden und Tetraoniden. Ornithol. Beobachter 53, 182u2013201 (1956)._ Yack, J. E., Smith, M. L. & Weatherhead, P. J. Caterpillar talk: acoustically mediated territoriality in larval Lepidoptera. Proc. Natl. Acad. Sci. USA 98, 11371u201311375 (2001)._ _ _ _ Nakajima, H. A contribution to the knowledge of the immature stages of Drepanidae occurring in Japan (Lepidoptera). Tinea 8, 167u2013184 + 20 pls (1970)._ Nakajima, H. Additions to the knowledge of the immature stages of Drepanidae occurring in Japan (Lepidoptera). Tinea 9, 253u2013256 + 4 pls (1972)._ Bowen, J. L., Mahony, S. J., Mason, A. C. & Yack, J. E. Vibration-mediated territoriality in the warty birch caterpillar Drepana bilineata. Physiol. Entomol. 33, 238u2013250 (2008)._ _ Scott, J. L., Matheson, S. M. & Yack, J. E. Variation on a theme: vibrational signaling in caterpillars of the rose hook-tip moth, Oreta rosea. J. Insect Sci. (in press).Stehr, F. W. Order Lepidoptera. In Immature Insects Vol. 1 (ed. Stehr, F.W.) 288u2013 596 (Kendall/Hunt Publishing Company, 1987).Scoble, M. J. The Lepidoptera: Form, Function and Diversity (Oxford University Press, 1995).Minet, J. Tentative reconstruction of the ditrysian phylogeny (Lepidoptera: Glossata). Entomol. Scand. 22, 69u2013 95 (1991)._ _ Randall, J. A. Evolution and function of drumming as communication in mammals. Am. Zool. 41, 1143u20131156 (2001)._ Lorenz, K. On Aggression (Harcourt, Brace & World, 1963).Rumph, J. A. & Turner, W. J. Alternative to critical point drying for soft-bodied insect larva. Ann. Entomol. Soc. Am. 91, 693u2013699 (1998)._ _ _ Belshaw, R. & Quicke, D. L. J. A molecular phylogeny of the Aphidiinae (Hymenoptera: Braconidae). Mol. Phylogenet. Evol. 7, 281u2013293 (1997)._ _ _ Regier, J. C. Protocols, concepts, and reagents for preparing DNA sequencing templates, version 12/4/08, (2008).Weller, S. J., Pashley, D. P., Martin, J. A. & Constable, J. L. Phylogeny of noctuoid moths and the utility of combining independent nuclear and mitochondrial genes. Syst. Biol. 43, 194u2013211 (1994)._ _ Katoh, K. MAFFT, version 6.717b, (2009).Akaike, H. Information theory as an extension of the maximum likelihood principle. in Second International Symposium on Information Theory (eds Petrov, B.N. & Csaki, F.) 267u2013281 (Akademiai Kiado, 1973).Posada, D. & Crandall, K. A. Modeltest: testing the model of DNA substitution. Bioinformatics 14, 817u2013818 (1998)._ _ _ Ronquist, F. & Huelsenbeck, J. P. MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics 19, 1572u20131574 (2003)._ _ _ Wigenbusch, J. C., Warren, D. L. & Swofford, D. L. AWTY: a system for graphical exploration of MCMC convergence in Bayesian phylogenetic inference, (2004).Maddison, W. P. & Maddison, D. R. Mesquite: a modular system for evolutionary analysis, version 2.71, (2009).Lewis, P. O. A likelihood approach to estimating phylogeny from discrete morphological character data. Syst. Biol. 50, 913u2013925 (2001)._ _ _ Pagel, M. D. & Meade, A. Bayesian analysis of correlated evolution of discrete characters by reversible-jump Markov chain Monte Carlo. Am. Nat. 6, 808u2013825 (2006)._ Pagel, M. D. Detecting correlated evolution on phylogenies: a general method for the comparative analysis of discrete characters. Philos. Trans. R. Soc. Lond, B 255, 37u2013 45 (1994)._ _ Blumstein, D. T., Daniel, J. C. & Evans, C. S. J Watcher, version 1.0, (2006).Sen, Y. C. & Lin, C. S. Larval morphology and host plants of Drepanidea (Lepidoptera: Drepanidae) in Southern Taiwan. Formos. Entomol. 22, 27u201342 (2002)._ Solis, A. & Styer, L. Revision and phylogenetic analysis of Accinctapubes solis (Pyralidae: Epipaschiinae) with a larval description of an avocado-feeding species. J. Lepid. Soc. 57, 121u2013136 (2003)._ Abraham, D., Ryrholm, N., Wittzell, H., Holloway, J. D., Scoble, M. J. & LxF6fstedt, C. Molecular phylogeny of the subfamilies in Geometridae (Geometroidea: Lepidoptera). Mol. Phylogenet. Evol. 20, 65u201377 (2001)._ _ _ Beccaloni, G. W., Scoble, M. J., Robinson, G. S. & Pitkin, B. (eds). The Global Lepidoptera Names Index (LepIndex). (2003).\"- - We thank H. Beck, Y.L. Chen, S. Corver, K. Eda, J. Miall, T. Muus, L. Scott, K. Silvonen, C.H. Wei and S. Wu for collecting moths; V. Bura, S. Matheson and R. Smith for their help with data collection and I. Hasenfuss for unpublished observations on Drepanidae caterpillars. We are deeply indebted to Charles and Kim Mitter for providing specimens from the University of Maryland LepTree frozen tissue collection. We thank C. Castaldo for help with statistical comparisons of trees in BayesTraits. This work was supported by NSERC Discovery Grants (J.E.Y., J.H.S. and M.L.S.), CFI (J.E.Y.), COA (S.-H.Y.), NSC (S.-H.Y.), Agriculture and Agri-Food Canada (J.H.S.) and an NSERC Postgraduate Scholarship (J.L.S.).- - Authors and Affiliations - \"Department of Biology, Carleton University, 1125 Colonel By Drive, Ottawa, K1S 5B6, Ontario, CanadaJaclyn L. Scott,_Abeer Sami,_Myron L. Smith_&_Jayne E. YackDepartment of Entomology, University of Maryland, 4112 Plant Sciences Building, College Park, 20742-4454, Maryland, USAAkito Y. KawaharaAgriculture and Agri-Food Canada, Canadian National Collection of Insects, Arachnids and Nematodes, 960 Carling Avenue, K.W. Neatby Building, Ottawa, K1A 0C6, Ontario, CanadaJeffrey H. SkevingtonDepartment of Biological Sciences, National Sun Yat-Sen University, Kaohsiung, 804, TaiwanShen-Horn Yen\" - \"AuthorsJaclyn L. ScottYou can also search for this author in _Akito Y. KawaharaYou can also search for this author in _Jeffrey H. SkevingtonYou can also search for this author in _Shen-Horn YenYou can also search for this author in _Abeer SamiYou can also search for this author in _Myron L. SmithYou can also search for this author in _Jayne E. YackYou can also search for this author in _\" - Contributions - J.L.S. and J.E.Y. developed concepts, designed the study, performed behavioural experiments and wrote the paper. J.E.Y. supervised the project. A.S., A.Y.K., J.H.S., J.L.S. and M.L.S. contributed to the molecular phylogeny by collecting and/or analysing data. S.-H.Y. contributed specimens and conceptual advice on comparative behaviour and morphology. All authors discussed results and commented on all stages of the paper. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing financial interests. - \\' \\'- - A territorial encounter over a partially constructed leaf shelter. A resident Drepana arcuata caterpillar defends its leaf shelter using ritualized vibratory signals. These vibrations are produced by dragging anal \\'oars\\' on the leaf surface and scraping and drumming mandibles. Sound recordings are made with a microphone held behind the leaf. (MOV 6093 kb)Walking movements in Tethea or, a species that possesses anal prolegs that are used for walking, but lacks modified setae used for producing vibrations (see movie 3). (MOV 2496 kb)Movements associated with anal scraping in Drepana arcuata. This species lacks anal prolegs, and instead uses modified setae to produce vibratory signals. (MOV 4108 kb)A territorial encounter in Tethea or recorded using a laser vibrometer. An intruder enters the leaf shelter comprising 2 leaves sewn together with silk, and the resident repels the intruder by scraping its mandibles, pushing, hitting, and biting. (MOV 3715 kb)Fully aligned and concatenated phylogeny data matrix (TXT 79 kb)- - \\' \\' - \\'This work is licensed under a Creative Commons Attribution-NonCommercial-NoDerivative Works 3.0 License. ',\n",
       " 'Creative Commons Attribution License, which permitsunrestricted use, distribution and reproduction in any medium, provided the original author and source are credited.\"- - \"Sprent, J. I. Evolving ideas of legume evolution and diversity: a taxonomic perspective on the occurrence of nodulation. New Phytol. 174, 11u201325 (2007)._ _ _ Gage, D. J. Infection and invasion of roots by symbiotic, nitrogen-fixing rhizobia during nodulation of temperate legumes. Microbiol Mol Biol Rev 68, 280u2013300 (2004)._ _ _ Lerouge, P. et al. Symbiotic host-specificity of Rhizobium meliloti is determined by a sulphated and acylated glucosamine oligosaccharide signal. Nature 344, 781u2013 784 (1990)._ _ _ _ Lopez-Lara, I. M. et al. Structural identification of the lipo-chitin oligosaccharide nodulation signals of Rhizobium loti. Mol. Microbiol. 15, 627u2013638 (1995)._ _ _ Radutoiu, S. et al. Plant recognition of symbiotic bacteria requires two LysM receptor-like kinases. Nature 425, 585u2013592 (2003)._ _ _ _ Madsen, E. B. et al. A receptor kinase gene of the LysM type is involved in legume perception of rhizobial signals. Nature 425, 637u2013640 (2003)._ _ _ _ Arrighi, J. F. et al. The Medicago truncatula lysin motif-receptor-like kinase gene family includes NFP and new nodule-expressed genes. Plant Physiol. 142, 265u2013279 (2006)._ _ _ Miwa, H., Sun, J., Oldroyd, G. E. & Downie, J. A. Analysis of Nod-factor-induced calcium signaling in root hairs of symbiotically defective mutants of Lotus japonicus. Mol. Plant Microbe Interact. 19, 914u2013923 (2006)._ _ _ Radutoiu, S. et al. LysM domains mediate lipochitin-oligosaccharide recognition and Nfr genes extend the symbiotic host range. EMBO J. 26, 3923u20133935 (2007)._ _ _ Stracke, S. et al. A plant receptor-like kinase required for both bacterial and fungal symbiosis. Nature 417, 959u2013962 (2002)._ _ _ _ Endre, G. et al. A receptor kinase gene regulating symbiotic nodule development. Nature 417, 962u2013966 (2002)._ _ _ _ Ane, J. M. et al. Medicago truncatula DMI1 required for bacterial and fungal symbioses in legumes. Science 303, 1364u2013 1367 (2004)._ _ _ _ Imaizumi-Anraku, H. et al. Plastid proteins crucial for symbiotic fungal and bacterial entry into plant roots. Nature 433, 527u2013531 (2005)._ _ _ _ Charpentier, M. et al. Lotus japonicus CASTOR and POLLUX are ion channels essential for perinuclear calcium spiking in legume root endosymbiosis. Plant Cell 20, 3467u20133479 (2008)._ _ _ Kanamori, N. et al. A nucleoporin is required for induction of Ca2 spiking in legume nodule development and essential for rhizobial and fungal symbiosis. Proc. Natl. Acad. Sci. USA 103, 359u2013364 (2006)._ _ _ _ Saito, K. et al. NUCLEOPORIN85 is required for calcium spiking, fungal and bacterial symbioses, and seed production in Lotus japonicus. Plant Cell 19, 610u2013624 (2007)._ _ _ Tirichine, L. et al. Deregulation of a Ca/calmodulin-dependent kinase leads to spontaneous nodule development. Nature 441, 1153u20131156 (2006)._ _ _ _ Levy, J. et al. A putative Ca and calmodulin-dependent protein kinase required for bacterial and fungal symbioses. Science 303, 1361u2013 1364 (2004)._ _ _ _ Mitra, R. M. et al. A Ca/calmodulin-dependent protein kinase required for symbiotic nodule development: Gene identification by transcript-based cloning. Proc. Natl. Acad. Sci. USA 101, 4701u20134705 (2004)._ _ _ _ Yano, K. et al. CYCLOPS, a mediator of symbiotic intracellular accommodation. Proc. Natl. Acad. Sci. USA 105, 20540u201320545 (2008)._ _ _ _ Sieberer, B. J. et al. A nuclear-targeted cameleon demonstrates intranuclear Ca2+ spiking in Medicago truncatula root hairs in response to rhizobial nodulation factors. Plant Physiol. 151, 1197u20131206 (2009)._ _ _ Schauser, L., Roussis, A., Stiller, J. & Stougaard, J. A plant regulator controlling development of symbiotic root nodules. Nature 402, 191u2013195 (1999)._ _ _ _ Marsh, J. F. et al. Medicago truncatula NIN is essential for rhizobial-independent nodule organogenesis induced by autoactive calcium/calmodulin-dependent protein kinase. Plant Physiol. 144, 324u2013335 (2007)._ _ _ _ Smit, P. et al. NSP1 of the GRAS protein family is essential for rhizobial Nod factor-induced transcription. Science 308, 1789u20131791 (2005)._ _ _ _ Kalo, P. et al. Nodulation signaling in legumes requires NSP2, a member of the GRAS family of transcriptional regulators. Science 308, 1786u20131789 (2005)._ _ _ _ Heckmann, A. B. et al. Lotus japonicus nodulation requires two GRAS domain regulators, one of which is functionally conserved in a non-legume. Plant Physiol. 142, 1739u20131750 (2006)._ _ _ Murakami, Y. et al. Positional cloning identifies Lotus japonicus NSP2, a putative transcription factor of the GRAS family, required for NIN and ENOD40 gene expression in nodule initiation. DNA Res. 13, 255u2013265 (2006)._ _ _ Middleton, P. H. et al. An ERF transcription factor in Medicago truncatula that is essential for Nod factor signal transduction. Plant Cell 19, 1221u20131234 (2007)._ _ _ Asamizu, E., Shimoda, Y., Kouchi, H., Tabata, S. & Sato, S. A positive regulatory role for LjERF1 in the nodulation process is revealed by systematic analysis of nodule-associated transcription factors of Lotus japonicus. Plant Physiol. 147, 2030u20132040 (2008)._ _ _ Hogslund, N. et al. Dissection of symbiosis and organ development by integrated transcriptome analysis of lotus japonicus mutant and wild-type plants. PLoS One 4, e6556 (2009)._ _ _ Mitra, R. M., Shaw, S. L. & Long, S. R. Six nonnodulating plant mutants defective for Nod factor-induced transcriptional changes associated with the legume-rhizobia symbiosis. Proc. Natl. Acad. Sci. USA 101, 10217u2013 10222 (2004)._ _ _ _ Tirichine, L. et al. A gain-of-function mutation in a cytokinin receptor triggers spontaneous root nodule organogenesis. Science 315, 104u2013107 (2007)._ _ _ _ Murray, J. D. et al. A cytokinin perception mutant colonized by rhizobium in the absence of nodule organogenesis. Science 315, 101u2013 104 (2007)._ _ _ _ Yokota, K. et al. Rearrangement of actin cytoskeleton mediates invasion of Lotus japonicus roots by Mesorhizobium loti. Plant Cell 21, 267u2013284 (2009)._ _ _ Yano, K. et al. CERBERUS, a novel U-box protein containing WD-40 repeats, is required for formation of the infection thread and nodule development in the legume-Rhizobium symbiosis. Plant J. 60, 168u2013180 (2009)._ _ _ Tirichine, L., James, E. K., Sandal, N. & Stougaard, J. Spontaneous root-nodule formation in the model legume Lotus japonicus: a novel class of mutants nodulates in the absence of rhizobia. Mol. Plant Microbe Interact. 19, 373u2013382 (2006)._ _ _ Gleason, C., Chaudhuri, S., Yang, T., Munoz, A., Poovaiah, B. W. & Oldroyd, G. E. Nodulation independent of rhizobia induced by a calcium-activated kinase lacking autoinhibition. Nature 441, 1149u20131152 (2006)._ _ _ _ Limpens, E. et al. Formation of organelle-like N2-fixing symbiosomes in legume root nodules is controlled by DMI2. Proc. Natl. Acad. Sci. USA 102, 10375u201310380 (2005)._ _ _ _ Capoen, W., Goormachtig, S., De Rycke, R., Schroeyers, K. & Holsters, M. SrSymRK, a plant receptor essential for symbiosome formation. Proc. Natl. Acad. Sci. USA 102, 10369u201310374 (2005)._ _ _ _ Messinese, E. et al. A novel nuclear protein interacts with the symbiotic DMI3 calcium- and calmodulin-dependent protein kinase of Medicago truncatula. Mol. Plant Microbe Interact. 20, 912u2013921 (2007)._ _ _ Rodpothong, P. et al. Nodulation gene mutants of Mesorhizobium loti R7A-nodZ and nolL mutants have host-specific phenotypes on Lotus spp. Mol. Plant Microbe Interact. 22, 1546u20131554 (2009)._ _ _ James, E. K., Minchin, F. R., Iannetta, P. P. M. & Sprent, J. I. Temporal relationships between nitrogenase and intercellular glycoprotein in developing white lupin nodules. Ann. Bot. 79, 493u2013503 (1997)._ _ _ Gonzalez-Sama, A., Lucas, M. M., de Felipe, M. R. & Pueyo, J. J. An unusual infection mechanism and nodule morphogenesis in white lupin (Lupinus albus). New Phytol 163, 371u2013380 (2004)._ _ Esseling, J. J., Lhuissier, F. G. P. & Emons, A. M. C. A nonsymbiotic root hair tip growth phenotype in NORK-mutated legumes: Implications for nodulation factor-induced signaling and formation of a multifacetted root hair pocket for bacteria. Plant Cell 16, 933u2013944 (2004)._ _ _ D\\'Haeze, W. et al. Roles for azorhizobial nod factors and surface polysaccharides in intercellular invasion and nodule penetration, respectively. Mol. Plant Microbe Interact. 11, 999u20131008 (1998)._ _ _ Lohmann, G. V. et al. Evolution and regulation of the Lotus japonicus LysM receptor gene family. Mol. Plant Microbe Interact. 23, 510u2013521 (2010)._ _ _ D\\'Haeze, W., Mergaert, P., Prome, J. C. & Holsters, M. Nod factor requirements for efficient stem and root nodulation of the tropical legume Sesbania rostrata. J. Biol. Chem. 275, 15676u201315684 (2000)._ _ _ Goormachtig, S., Capoen, W., James, E. K. & Holsters, M. Switch from intracellular to intercellular invasion during water stress-tolerant legume nodulation. Proc. Natl. Acad. Sci. USA 101, 6303u20136308 (2004)._ _ _ _ Den Herder, J. V. C., De Rycke, R., Corich, V., Holsters, M. & Goormachtig, S. Nod factor perception during infection thread growth fine-tunes nodulation. Mol. Plant Microbe Interact. 20, 129u2013137 (2007)._ _ _ Capoen, W. et al. Calcium spiking patterns and the role of the calcium/calmodulin-dependent kinase CCaMK in lateral root base nodulation of Sesbania rostrata. Plant Cell 21, 1526u20131540 (2009)._ _ _ Giraud, E. et al. Legumes symbioses: absence of Nod genes in photosynthetic bradyrhizobia. Science 316, 1307u20131312 (2007)._ _ _ Karas, B. et al. Invasion of Lotus japonicus root hairless 1 by Mesorhizobium loti involves the nodulation factor-dependent induction of root hairs. Plant Physiol. 13, 1331u20131344 (2005)._ _ _ James, E. K. & Sprent, J. I. Development of N2-fixing nodules on the wetland Lotus uliginosus exposed to conditions of flooding. New. Phytol. 142, 219u2013231 (1999)._ _ Handberg, K. & Stougaard, J. Lotus-Japonicus, an autogamous, diploid legume species for classical and molecular-genetics. Plant J. 2, 487u2013496 (1992)._ _ \"- - The Danish National Research Foundation funded this study. We thank Nick Brewin for the MAC236 antibody, Krzysztof Szczyglowski for hit1 seeds and Masayoshi Kawaguchi for nup85 seeds.- - Author notes - \"LexEFla TirichinePresent address: u2020Present address: GxE9nomique, Environnementale et Evolutive Section 3 CNRS UMR8197, Institut de Biologie de l\\'Ecole Normale SupxE9rieure (IBENS), 46 rue d\\'Ulm, 75230 Paris Cedex 05, France, \" - Authors and Affiliations - \"Department of Molecular Biology, Centre for Carbohydrate Recognition and Signalling, Aarhus University, Gustav Wieds Vej 10, DK-8000, Aarhus C, DenmarkLene H. Madsen,_ LexEFla Tirichine,_Anna Jurkiewicz,_Anne B. Heckmann,_Anita S. Bek_&_Jens StougaardDepartment of Microbiology and Immunology, University of Otago, PO Box 56, Dunedin 9054, New ZealandJohn T. Sullivan_&_Clive W. RonsonEPI division, Scottish Crop Research Institute, Invergowrie, Dundee, DD2 5DA, UKEuan K. James\" - \"AuthorsLene H. MadsenYou can also search for this author in _LexEFla TirichineYou can also search for this author in _Anna JurkiewiczYou can also search for this author in _John T. SullivanYou can also search for this author in _Anne B. HeckmannYou can also search for this author in _Anita S. BekYou can also search for this author in _Clive W. RonsonYou can also search for this author in _Euan K. JamesYou can also search for this author in _Jens StougaardYou can also search for this author in _\" - Contributions - L.H.M., L.T., A.J. and A.B.H. isolated and characterized the synthetic mutants. A.J., A.S.B., E.K.J. and L.H.M. performed the microscopy. J.T.S. and C.W.R. constructed the M. loti mutants, L.T. observed the first partially infected nodules and initiated the work using synthetic mutants. J.S. coordinated and wrote the paper. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing financial interests. - \\' \\'- - Supplementary Figure S1 and Supplementary Tables S1-S4 (PDF 749 kb)- - \\' \\' - \\'This work is licensed under a Creative Commons Attribution-NonCommercial-Share Alike 3.0 License. ',\n",
       " 'Creative Commons Attribution License. RDH14 polyclonal antibody HPA056686 (MilliporeSigma; St Louis, MO); Human assay: A-431 fixed with PFA, dilution: 1:49; Human assay: U-2 OS fixed with PFA, dilution: 1:49.\\' - \"Figure 3Western blotting showing GST pull-down of TMEM206/PACC1 from SK-N-SH cell lysate using wild-type (WT) versus mutant RDH14-GST constructs. A TMEM206 (PACC1) rabbit polyclonal antibody (1:1000 dilution; catalogue #TA342366; Origene Technologies, Rockville, MD) was used. (A) Input to lane 1 was SK-N-SH cell lysate (as a positive control). Lanes 2 and 3 show extracted protein for the RDH14-GST constructs, WT and mutant, respectively (as negative controls). In lanes 4, 5, and 6, the GST pull-down eluted protein, for WT and mutant RDH14-GST, and GST alone (as negative control), respectively (i.e. bait protein against prey input). (B) Flow-through from lanes 2u20136 above was run and probed with the TMEM206 antibody. Western hybridization using GST antibodies was also performed to check sample loading (Supplementary Fig. ).\" - \"Although analysis of the effects of knocking out or knocking down RDH14 on neuronal differentiation and maturation was not feasible, as it is difficult to predict the effects of lack of retinol dehydrogenase 14 enzyme when utilizing retinoic acid as the standard differentiation agent on neuroblastoma cell lines, we were able to effectively knock-down expression of endogenous RDH14 in SK-N-SH cells. Western analysis of well-studied factors associated with cell pluripotency and dendritic marker (SOX2 and MAP2, respectively) and putative RDH14 interactors (BPRIN2, TMEM206, also MAP2), show a trend towards decreased expression in the RDH14-silenced cells for SOX2, MAP2 and TMEM206 (Fig._). RT-qPCR also showed reduced transcription for SOX2 in RDH14-knockdown cells, although non- significant (Fig._). RT-qPCR was also performed on mock versus RDH14 silenced cells for neuronal differentiation factor genes OCT4, RAR-A, NEUROD1, NESTIN, and KI67, but no change in transcription levels was observed (data not shown; raw data is available from the corresponding author upon request).\" - \"Figure 4Analysis of the RDH14 knock down effects on SOX2, MAP2, TMEM206_(PACC1) and BRINP2 (FAM5B)_in SK-N-SH cells. (A) Western blot analysis of the RDH14-siRNA transfection showing knock down of RDH14 protein, using anti-RDH14 rabbit polyclonal antibody (1:1000 dilution; cat# HPA056686; Atlas Antibodies, Bromma, Sweden). (B) qRT-PCR showing effect of RDH14-siRNA on transcription of SOX2 in SK-N-SH cells. (C) Representative western blot images showing the effects of RDH14 siRNA knock down on SOX2 and MAP2 protein levels using anti-SOX2 monoclonal antibody (1:2000 dilution; cat# ab79351, Abcam Inc, Toronto, ON), and anti-MAP2 polyclonal antibody (1:2000 dilution; cat# ab79351, Abcam Inc, Toronto, ON). (D) Effect of RDH14 siRNA knock down on TMEM206/PACC1 and FAM5B/BRINP2 protein levels using anti-TMEM206 rabbit polyclonal antibody (1:1000 dilution; cat# TA342366; Origene Tech., Rockville, MD), and anti-FAM5B rabbit polyclonal antibody (1:500 dilution; cat# orb474203; Biorbyt Tech., St Louis, MO).\"- - \\'RDH14 encodes a member of the SDR7C family short-chain dehydrogenase reductase enzymes, which, in addition to RDH14 (SDR7C4), includes RDH11 (SDR7C1), RDH12 (SDR7C2) and RDH13 (SDR7C3) (see for review). SDR7C enzymes are involved in retinoic acid catabolism and homeostasis, catalyzing the conversion of retinaldehyde to retinol, as well as the reverse, conversion of retinol to retinaldehyde. Retinaldehyde is converted by aldehyde dehydrogenases to retinoic acid (RA). All-trans RA is a critical developmental morphogen, important for the fine control of differentiation and cell-patterning. RA is critical to embryonic developmental structures, including forebrain and hindbrain, but also likely to continue to function throughout adulthood (reviewed by). The RDH14 protein is reported to be ubiquitously expressed, including brain, but notably not in the eye, whereas other SDR7C members are expressed in specific non-CNS tissues (RDH11: prostate, testis, kidney; RDH13: retina, endocrine tissue, lung, gastrointestinal tract, liver, pancreas, kidney, skin, male and female-specific tissues; ; N.B. RDH12 protein data is unavailable, but transcription is highest in skin). RNA in situ hybridization studies of RDH12 show localization to the base of photoreceptor inner segments. Biallelic mutations in RDH12 cause Leber congenital amaurosis-13 (LCA13; MIM 612712). Biallelic truncating mutations in RDH11 cause a form of retinitis pigmentosa (RP) with delayed psychomotor development, learning difficulties, and distinct craniofacial and physical dysmorphologies (MIM 616108). RDH14 exhibits retinaldehyde reductase activity, with preference for all-trans-retinaldehyde as a substrate over 9-cis-retinaldehyde, with higher catalytic efficiency compared to RDH11, and with reductase activity albeit at a lower level. The variant reported here would leave the single putative transmembrane domain intact, but would remove the co-factor binding domain at Gly50, and the entire SDR enzymatic domain including the putative active site at Tyr217.\\' - \"Although reports indicate RDH14 is expressed in microsomes, our own data using over-expression of an RDH14-GFP construct (in HEK293 cells and SK-N-SH neuroblastoma cells, as well as data from The Human Protein Atlas () using antibodies against endogenous RDH14 in A-431 (squamous carcinoma) and U-2-OS (osteosarcoma) cells, suggest that RDH14 protein is also strongly expressed in the nucleoplasm (Fig._ ). This is somewhat surprising, as, although RA has nuclear action, it is assumed that its synthesis is cytoplasmic. Also, no nuclear localization signal could be identified (using SeqNLS (), or NLStradamus (), NLS Mapper (), NucPred (). Moreover localization predictions suggest RDH14 to by overwhelmingly cytoplasmic, and with no predicted DNA or RNA binding motifs (). Interestingly, another retinol dehydrogenase, RDH8 (all-trans), is also shown by immunocytochemistry to be nucleoplasmic (The Human Protein Atlas; ).\" - \"We explored the validity of several reported proteinu2013protein interactions for RDH14 as reported through . We confirmed experimentally through GST-pull-down, using a RDH14-N-terminal GST construct that RDH14 binds with proton-activated chloride channel 1 (PACC1, previously TMEM206), as reported through , using affinity capture-mass spectrometry. GST-pull-down using an RDH14 construct with an early truncating mutation showed greatly reduced binding to PACC1 (Fig._ ). PACC1, which is transcribed at very high levels in the brain, is reported to form an acid-sensitive outwardly rectifying (ASOR) anion channel. Disruption of PACC1 protects against acid-induced neuronal apoptosis and axonal degeneration. The putative interaction with BRINP2 was not confirmed, however. The interaction between RDH14 and PACC1 is corroborated by single-cell RNAseq data from developing and adolescent mice that supports co-expression of the two genes () in many brain cell types, particularly strongly in noradrenergic and cholinergic neuronal cells in sympathetic system. We speculate that RDH14, which possesses a single transmembrane domain, may have roles in addition to, and possibly independent of, its enzymatic function, however further work is required.\" - \"Interestingly, disruption of retinoic acid signaling in the brain is linked with a variety of common intellectual disability disorders. For instance, one of the most common missense mutations in the Rett syndrome gene, MECP2, disrupts binding at the nuclear receptor corepressor (NCoR)/silencing mediator of retinoic acid and thyroid receptors (SMRT) corepressor complex. The genetic defect leading to Smith-Magenis syndrome (SMS) is deletion of the RAI1 (retinoic acid-induced 1) gene on 17p11.2. Also, Fragile X mutations block synaptic retinoic acid signaling, impairing neuronal homeostatic plasticity. Also, mutations in the retinoic acid receptor beta gene, RARB, lead to intellectual disability with progressive motor impairment. Another reported RDH14-interacting protein from the BioGRID dataset is the bone morphogenic protein/retinoic acid inducible neural-specific protein 2, BRINP2. Interestingly, copy number variants (CNV) at BRINP2s orthologue, BRINP1 (the BRINP1/ASTN2 locus) have previously been linked with neurodevelopmental disorders, with a gain CNV in a patient with autism spectrum disorder (ASD), anxiety and learning disability, and a loss CNV in a patient with developmental delay and seizures, suggesting one or both of these genes having a role in neurodevelopmental disorders. The BRINP1/ASTN2 locus has also shown genetic linkage (LOD scoreu2009 =u20094.11) with major psychiatric disorders in a single large pedigree from Spain (Pol-Fuster et al.). However, GST pull-down using our WT and mutant RDH14-GST constructs was unable to confirm binding to BRINP2 (FAM5B) (see Supplementary Fig. ).\" - \"In summary, owing to the intra-familial genetic heterogeneity in this family, additional analytic steps were required in order to make sense of the genetic architecture and likely disease-candidates. In addition to identifying a known disease-causing mutation in the ADGRG1/GPR56 polymicrogyria gene in two of the_ four available_affected members, the remaining two available_affecteds both have a biallelic mutation in the retinol dehydrogenase gene, RDH14. MRI analysis from these two individuals at ages 20_months and 14_years suggests a progressive childhood-onset form of cerebellar atrophy, in addition to ID. While no supporting evidence yet exists from additional intellectual disability families or from animal studies, the relevance of RDH14 to homeostasis of RA in the brain makes this a strong disease gene candidate.\"- - Recruitment - \"We identified a large consanguineous Pakistani family presenting with an apparently autosomal dominant form of hyperkeratosis across multiple branches, among which, in addition, there are four living and one deceased member with apparently autosomal recessive ID, including one male individual who presented with both ID and hyperkeratosis (see Fig._). Members of the family, who originate from the Bannu district in Khyber Pakhtunkhwa province in Pakistan, were recruited for a research study through Islamia College Peshawar, Pakistan, and Centre for Addiction and Mental Health (CAMH), Toronto, Canada. The study was approved by institutional ethics committees: Institutional Bioethical Committee (IBC), Islamia College Peshawar, under notification number 529/ORIC/ICP, and the Research Ethics Board, CAMH, Toronto. Informed written consent was obtained from all participants and/or their legal guardians, and the research was performed in accordance with relevant guidelines, and in accordance with the Declaration of Helsinki.\" - Genetic/genomic analysis - \"After genomic DNA extraction using standard phenol/chloroform methods, Illumina CoreExome microarrays were run for individuals III-1, III-5, III-6, III-10 (affected), and III-2, III-7 and III-8 (unaffected) using an Illumina iScan microarray scanner (Illumina Inc, San Diego, USA). Microarray data was imported into Illumina Genome Suites, through which genotypes were extracted in PLINK format. Analysis of genotypes using the Homozygosity Mapper program () was performed using default settings, firstly using all ID subjects classified as affected, and then, when no single HBD regions (>u20091_Mb in length) was identified, the process was repeated reiteratively with different ID subjects reclassified as unaffecteds.\" - Whole exome sequencing - \"Whole exome sequencing was performed for affected individuals III-1, III-5 and III-6. Whole exome paired-end sequencing was performed by Macrogen Inc. Republic of Korea Genome Sequencing Facility, using 2_u03BCg of genomic DNA (260/280u2009 >u20091.7). Exome capture was performed using the Agilent 51_Mb SureSelect All Exon V4 kit (Agilent Technologies, Santa Clara, CA), and sequenced using the HiSeq 2000 platform (Illumina).\" - Sequence analytic pipeline - \"Sequence reads were processed through an in-house custom analytic pipeline to map and call variants, as previously described. Briefly, Illumina sequence reads were aligned with BWA-mem (v 0.7.13) using the hg19 genome reference. After alignment GATK (v 3.2.2) was used for base recalibration, indel realignment, and the GATK Unified Genotyper was used for variant calling. Variants called were generated in the standard VCF version 4.1 and annotated with Annovar. Once annotation was performed, filtration of variants was performed by computationally and manually extracting variants that were in HBD regions provided by the autozygosity mapping HomozygosityMapper. The next step was to filter variants based on sequencing quality/read depth, homozygosity of allele and mutation type. Prioritization was performed by scoring truncating mutations such as stop and frameshift loss of function mutations higher than missense mutations and inframe indels. Missense mutations were scored based on SIFT and Polyphen2, scores for prioritization, as well as meta-analysis algorithms Condel and CADD. Allele frequencies were then used to filter out variants that were too common in the population (>u2009 1 in 10,000). All variants surviving filtration were checked against the gnomAD control database ().\" - Sanger sequencing confirmation and segregation - Sanger sequencing was used to confirm the ADGRG1 and RDH14 mutations, and to check the segregation across the family. Sequencing was performed under service through Eurofins Genomics (Luxembourg). The PCR/sequencing primers used are listed in . - Cloning of RDH14, and site-directed mutagenesis - \"The RDH14 coding sequence was amplified by PCR using attB1/2-flanked primers. The PCR template consisted of cDNA that was generated by reverse transcription of human brain RNA. Two sequential Gateway recombination reactions were utilized to generate the desired RDH14 expression vectors. Briefly, the attB1/2-flanked RDH14 amplicon was recombined into the donor vector pDONR221 using BP Clonase II, and then recombined into the desired Gateway expression vector using LR Clonase II. The Gateway expression vectors pDEST53 and pDEST15 (Thermo Fisher Scientific, Waltham, MA) were utilized to generate N-terminal GFP- and GST-tagged RDH14 recombinant proteins, respectively. To generate the mutant GST-RDH14 variant, c.114_115insC (resulting in p.(Gly39Argfs*97) and identical to the mutation identified in the Pakistani family), site-directed mutagenesis using non-overlapping primers was employed. Briefly, PCR using the Q5 high-fidelity DNA polymerase and primers containing the desired mutation was used to replicate a linear version of the wildtype RDH14-pDEST15 vector, followed by phosphorylation of the 5u2032 ends of the amplicon using T4 polynucleotide kinase and subsequent ligation. Clones were screened for the desired mutation using Sanger sequencing (Supplementary Fig. ). All primer sequences are listed in Supplementary Table .\" - Immunocytochemistry - \"HEK293T cells [human embryonic kidney cells (ATCC CRL-3216TM)] were seeded onto 0.1_mg/mL poly-d-lysine-coated glass coverslips in a 24-well plate. When cells reached a confluence of approximately 50u201370%, they were transiently transfected with 500_ng of the RDH14-pDEST53 vector using Lipofectamine 3000. 24_h post-transfection, cells were washed three times with ice-cold phosphate-buffered saline (PBS) and then fixed for 10_min at room temperature with 4% paraformaldehyde in 0.1_ M PBS, pH 7.4, followed by three washes with PBS for 5_min. Cells were then permeabilized for 10_min at room temperature with 0.25% Triton X-100 in PBS. Following permeabilization, cells were washed three times with PBS supplemented with 0.1% Tween-20 (PBST) for 5_min, and then blocked for 1_h at room temperature in 1% bovine serum albumen (BSA) and 22.52_mg/mL glycine in PBST. Cells were then incubated with the primary anti-GFP antibody (ab290; Abcam, Cambridge, MA) in blocking solution (1:1000 dilution) overnight at 4_xB0C with gentle agitation in a humidified polystyrene container. The following day, cells were washed three times with PBST for 5_min, followed by incubation with the fluorophore-conjugated secondary antibody (A32731; Thermo Fisher Scientific, Waltham, MA) in blocking solution (1:1000) at room temperature for 2_h with gentle agitation and covered by aluminum foil (from this step, all subsequent_steps were carried out under the cover of aluminum foil). Following incubation with the secondary antibody, cells were washed three times with PBST for 5_min, and then washed once with PBS, followed by incubation with DAPI (4u2032,6-diamidino-2-phenylindole; Roche-10236276001; MilliporeSigma; St Louis, MO) at a concentration of 1_u03BCg/mL (dissolved in methanol) at room temperature for 5_min with gentle agitation. Cells were then washed twice with PBS for 5_min and mounted onto microscope slides using Dako mounting medium (Agilent Technologies, Santa Clara, CA).\" - GST pull-down assays - \"GST-tagged RDH14 and RDH14-Mut constructs were transformed in E. coli BL21 (D3) competent cells (C2527). GST, GST-RDH14 and GST-RDH14-Mut fusion proteins were prepared according to the Pierceu2122 GST Protein Interaction Pull-Down Kit manufacturers protocol (Cat# 21516, Pierce Biotech., Rockford, IL). Briefly, GST, GST-RDH14 and GST-RDH14-Mut fusion proteins were expressed in E. coli followed by isolation and treatment of E. coli with pull-down lysis buffer for 30_min on ice. Similarly, SK-N-SH cells (Human neuroblastoma (ATCC, Manassas, VA, USA)) were grown in Dulbeccos Modified Eagle Medium (DMEM; (MilliporeSigma; St Louis, MO) containing 1% antibiotics and 10% FBS. SK-N-SH cell lysate was prepared using pull-down lysis buffer as described above and used as prey protein. GST fusion proteins were used as bait and equal amount were immobilized on an equilibrated glutathione agarose resin columns for 2_h at 4_xB0C. Columns were centrifuged, bait flow-through proteins were isolated and the columns were washed 5 times with wash solution (1:1 of TBS and pull-down buffer). Next, 800_ xB5L of prey SK-N-SH cell lysate was added to the columns and incubated at 4_ xB0C overnight. Columns were centrifuged, prey flow-through protein was isolated and then washed for a total of 5 times with wash solution. The resin-bound proteins were eluted in 10_mM glutathione elution buffer. The concentration of eluted proteins were measured and samples were prepared for western blot gel analysis.\" - \"For western blot analysis, protein was run on SDS-PAGE gels, transferred to PVDF nitrocellulose membrane and then incubated with 5% BSA blocking solution. The primary TMEM206 (PACC1), BRINP2 (FAM5B) and GST antibodies were attached and incubated overnight at 4_xB0C. The TMEM206/PACC1 protein was queried using anti-TMEM206 rabbit polyclonal antibody (1:1000 dilution; cat# TA342366; Origene Tech., Rockville, MD). The FAM5B/BRINP2 protein was queried using anti-FAM5B rabbit polyclonal antibody (1:500 dilution; cat# orb474203; Biorbyt Tech., St Louis, MO). GST was checked through hybridization with an anti-GST rabbit polyclonal antibody (1:1000 dilution; # 2622S; Cell Signaling Tech., Danvers, MA) (see Supplementary Fig. ). The next day, membranes were washed with 1u2009xD7u2009 TBST solution (Tris Buffered Saline, with Tween 20, pH 8.0) followed by incubation with the HRP secondary antibodies for 1u20132_h. Finally, membranes were sprayed with fluorescent solution and bands were visualized and obtained using the Gel Doc XRu2009+u2009System with Image Lab Software (Bio-Rad Laboratories, Hercules, CA).\" - SiRNA knockdown of RDH14 in SK-N-SH cells - \"The human neuroblastoma cell line SK-N-SH was cultured in DMEM medium supplemented with 10% FBS and 1% antibiotics at 37_xB0C in humidified air containing 5% CO2. When cells reached 80% confluency afteru2009~u200924_h, the media was replaced with fresh serum and antibiotic free DMEM. The RDH14 protein expression was knocked down with the RDH14 gene-silencing siRNA at a concentration of 75_ pmol and 25 nMol per transfection for 40_h (Cat # AM16708, Thermo Fisher Scientific, Waltham, MA). Lipofectamineu2122 3000 transfection reagent (Cat # L3000015, Invitrogen) was used to perform the transfection, followed by incubation at 37_xB0C and 5% CO2. The transfection reagents were also added to a u2018 mock group without siRNA as control. After 40_h, cells were washed with ice cold PBS and collected for protein lysate preparation in RIPA buffer for immunoblotting. For western blot analysis, protein was run on SDS-PAGE gels, transferred to PVDF nitrocellulose membrane and then incubated with 5% BSA blocking solution. The primary TMEM206 (PACC1), BRINP2 (FAM5B), GST, RDH14, SOX2, MAP2, and u03B2-actin antibodies were attached and incubated overnight at 4_xB0 C. The membranes were then washed with 1u2009xD7u2009TBST solution followed by incubation with the HRP secondary antibodies for 1u20132_h. Finally, membranes were sprayed with fluorescent solution and bands were visualized and obtained using the Bio-Rads Gel Doc XRu2009+u2009System with Image Lab Software.\" - RNA isolation and RT-qPCR - \"SK-N-SH cells intended for RT-qPCR were cultured in 6-well plates. 48_h after transient u2018mock- or siRNA-transfection, cells were washed with PBS, and RNA was isolated using the NucleoSpin RNA kit (Machery-Nagel-Fisher Scientific, DxFCrel, Germany) according to the manufacturers instructions. The initial cell lysis was performed directly in the 6-well plate, and RNA was eluted in RNase-free water. RNA integrity was verified by agarose gel electrophoresis. RNA (1_u03BCg) was reverse-transcribed into cDNA using the iScript cDNA synthesis kit (Bio-Rad). For qPCR, reactions were prepared using the PowerUp SYBR Green Master Mix (Thermo Fisher Scientific, Waltham, MA). For each reaction, cDNA samples were diluted either 10-, 100- or 1000-fold to ensure that amplification was occurring within the dynamic range for the given primer pair (primer sequences and annealing temperatures are provided). For each experimental condition (mock- or siRNA-transfected cells), four technical replicates were prepared for three biological replicates. Reactions were amplified in 384-well plates using the QuantStudio Real-Time PCR System software on the ViiA7 System instrument (Thermo Fisher Scientific, Waltham, MA) with the u0394u0394Ct experimental setting and standard amplification properties. Each reaction was subjected to melt-curve analysis to ensure the presence of only one amplicon, and a non-template control reaction was amplified in parallel for each primer pair. For each biological replicate, Ct values from two reference genes, GAPDH and 18S, were averaged (Ct-reference) and used for normalization. For normalization of experimental genes for each biological replicate, Ct values from the four technical replicates were averaged, and the Ct-reference value was subtracted (u0394Ct). In order to evaluate gene expression relative to the mock-transfection, the geometric mean of the u0394Ct values from only the three mock-transfection biological replicates was calculated, and subtracted from the u0394Ct values from each biological replicate across both mock- and siRNA-transfected conditions (u0394 u0394Ct). Lastly, 2 was calculated for each biological replicate.\"- - \\' \\' - \\'Information on the variant reported here has been submitted to the ClinVar database housed by NCBI: ClinVar accession #SCV001167015. The microarray and WES data that support the findings of this study are available from the corresponding author upon reasonable request.\\' - \\' \\'- - \"Riazuddin, S. et al. Exome sequencing of Pakistani consanguineous families identifies 30 novel candidate genes for recessive intellectual disability. Mol. Psychiatry 22, 1604u20131614 (2017)._ _ _ Anazi, S. et al. Expanding the genetic heterogeneity of intellectual disability. Hum. Genet. 136, 1419u20131429 (2017)._ _ Harripaul, R. et al. Mapping autosomal recessive intellectual disability: Combined microarray and exome sequencing identifies 26 novel candidate genes in 192 consanguineous families. Mol. Psychiatry 23, 973u2013984 (2018)._ _ _ Hu, H. et al. Genetics of intellectual disability in consanguineous families. Mol. Psychiatry 24, 1027u20131039 (2019)._ _ _ Maddirevula, S. et al. Autozygome and high throughput confirmation of disease genes candidacy. Genet. Med. 21, 736u2013742 (2019)._ _ _ Khan, M. I. et al. Whole-exome sequencing analysis reveals co-segregation of a COL20A1 missense mutation in a Pakistani family with striate palmoplantar keratoderma. Genes Genomics 40, 789u2013795 (2018)._ _ _ Piao, X. et al. G protein-coupled receptor-dependent development of human frontal cortex. Science 30, 2033u20132036 (2004)._ _ _ Ullrich, F. et al. Identification of TMEM206 proteins as pore of PAORAC/ASOR acid-sensitive chloride channels. Elife 8, e49187 (2019)._ _ _ ParxE9s, X., FarrxE9s, J., Kedishvili, N. & Duester, G. Medium- and short-chain dehydrogenase/reductase gene and protein families: Medium-chain and short-chain dehydrogenases/reductases in retinoid metabolism. Cell Mol. Life Sci. 65, 3936u20133949 (2008)._ _ Belyaeva, O. V. & Kedishvili, N. Y. Human pancreas protein 2 (PAN2) has a retinal reductase activity and is ubiquitously expressed in human tissues. FEBS Lett. 531, 489u2013493 (2002)._ _ _ Rhinn, M. & DollxE9, P. Retinoic acid signalling during development. Development 139, 843u2013858 (2012)._ _ _ Haeseleer, F. et al. Dual-substrate specificity short chain retinol dehydrogenases from the vertebrate retina. J. Biol. Chem. 277, 45537u201345546 (2002)._ _ _ Janecke, A. R. et al. Mutations in RDH12 encoding a photoreceptor cell retinol dehydrogenase cause childhood-onset severe retinal dystrophy. Nat. Genet. 36, 850u2013854 (2004)._ _ _ Xie, Y. A. et al. New syndrome with retinitis pigmentosa is caused by nonsense mutations in retinol dehydrogenase RDH11. Hum. Mol. Genet. 23, 5774u20135780 (2014)._ _ _ Huttlin, E. L. et al. Architecture of the human interactome defines protein communities and disease networks. Nature 545, 505u2013 509 (2017)._ _ _ _ Yang, J. et al. PAC, an evolutionarily conserved membrane protein, is a proton-activated chloride channel. Science 364, 395u2013399 (2019)._ _ _ _ Zeisel, A. et al. Molecular architecture of the mouse nervous system. Cell 174, 999-1014.e22 (2018)._ _ _ Kruusvee, V. et al. Structure of the MeCP2-TBLR1 complex reveals a molecular basis for Rett syndrome and related disorders. Proc. Natl. Acad. Sci. U.S.A. 114, E3243u2013 E3250 (2017)._ _ _ Zhang, Z. et al. The fragile X mutation impairs homeostatic plasticity in human neurons by blocking synaptic retinoic acid signaling. Sci. Transl. Med. 10, eaar4338 (2018)._ _ Srour, M. et al. Gain-of-function mutations in RARB cause intellectual disability with progressive motor impairment. Hum. Mutat. 37, 786u2013793 (2016)._ _ _ Lionel, A. C. et al. Disruption of the ASTN2/TRIM32 locus at 9q33.1 is a risk factor in males for autism spectrum disorders, ADHD and other neurodevelopmental phenotypes. Hum. Mol. Genet. 23, 2752u20132768 (2014)._ _ _ Pol-Fuster, J., CaxF1ellas, F., Ruiz-Guerra, L., Medina-Dols, A., Bisbal-CarrixF3, B., Ortega-Vila, B., LlinxE0 s, J., Hernandez-Rodriguez, J., LladxF3, J., Olmos, G., Strauch, K., Heine-SuxF1 er, D., Vives-BauzxE0, C. & Flaquer A. The conserved ASTN2/BRINP1 locus at 9q33.1-33.2 is associated with major psychiatric disorders in a large pedigree from Southern Spain. Sci. Rep. 11, 14529 (2021)._ _ _ _ Seelow, D., Schuelke, M., Hildebrandt, F. & NxFCrnberg, P. HomozygosityMapperu2014 An interactive approach to homozygosity mapping. Nucleic Acids Res. 37, W593u2013 W599 (2009)._ _ _ Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv preprint . (2013).Yang, H. & Wang, K. Genomic variant annotation and prioritization with ANNOVAR and wANNOVAR. Nat. Protoc. 10, 1556u20131566 (2015)._ _ _ GonzxE1lez-PxE9rez, A. & LxF3pez-Bigas, N. Improving the assessment of the outcome of nonsynonymous SNVs with a consensus deleteriousness score, Condel. Am. J. Hum. Genet. 88, 440u2013449 (2011)._ _ Rentzsch, P., Witten, D., Cooper, G. M., Shendure, J. & Kircher, M. CADD: Predicting the deleteriousness of variants throughout the human genome. Nucleic Acids Res. 47, D886u2013D894 (2019)._ _ _ UhlxE9 n, M. et al. Tissue-based map of the human proteome. Science 347, 1260419 (2015)._ _ \"- - \\'The authors thank the family for their participation in this study. This work was supported by grants from the Canadian Institutes of Health Research to JBV (#MOP-102758 and #PJT-156402).\\'- - Author notes - \\'These authors contributed equally: Stephen F. Pastore and Tahir Muhammad.\\' - Authors and Affiliations - \"Molecular Neuropsychiatry and Development (MiND) Lab, Campbell Family Mental Health Research Institute, Centre for Addiction and Mental Health, 250 College Street, Toronto, ON, M5T 1R8, CanadaStephen F. Pastore,_Tahir Muhammad,_Ricardo Harripaul,_Rebecca Lau_&_John B. VincentInstitute of Medical Science, University of Toronto, Toronto, ON, CanadaStephen F. Pastore,_Tahir Muhammad,_Ricardo Harripaul,_Rebecca Lau_&_John B. VincentDepartment of Pathology, North-West School of Medicine, Hayatabad, Peshawar, Khyber Pakhtunkhwa, PakistanMuhammad Tariq Masood KhanDepartment of Zoology, Islamia College Peshawar, Peshawar, Khyber Pakhtunkhwa, PakistanMuhammad Ismail KhanDepartment of Diagnostic Radiology, Queens University, Kingston, ON, CanadaOmar IslamDepartment of Biology and Institute of Basic Sciences, Sungshin Womens University, Seoul, Republic of KoreaChangsoo KangDepartment of Psychiatry, Queens University Kingston, Kingston, ON, CanadaMuhammad AyubCentre for Omic Sciences, Islamia College Peshawar, Peshawar, Khyber Pakhtunkhwa, PakistanMusharraf JelaniDepartment of Psychiatry, University of Toronto, Toronto, ON, CanadaJohn B. Vincent\" - \"AuthorsStephen F. PastoreYou can also search for this author in _Tahir MuhammadYou can also search for this author in _Ricardo HarripaulYou can also search for this author in _Rebecca LauYou can also search for this author in _Muhammad Tariq Masood KhanYou can also search for this author in _Muhammad Ismail KhanYou can also search for this author in _Omar IslamYou can also search for this author in _Changsoo KangYou can also search for this author in _Muhammad AyubYou can also search for this author in _Musharraf JelaniYou can also search for this author in _John B. VincentYou can also search for this author in _\" - Contributions - Study design was by J.B.V., M.J., R.H., S.P. and M.A. Recruitment and clinical evaluation was performed by M.T.M.K., M.I.K., and O.A. Experimentation was performed by S.P., T.M., R.L., C.K., with molecular and genetic data analysis by C.K., R.H., T.M. and J.B.V. The manuscript was written, and images prepared by J.B.V., S.P., and T.M. All authors reviewed the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution Noncommercial Share Alike 3.0 Unported Licence. ',\n",
       " 'Creative Commons Attribution licence.- - \"Perez-Mato, J. M. et al. Competing structural instabilities in the ferroelectric Aurivillius compound SrBi2Ta2O9. Phys. Rev. B70, 214111 (2004)._ _ McCabe, E. E. et al. Proper ferroelectricity in the Dionu2013 Jacobson material CsBi2Ti2NbO10: experiment and Theory. Chem. Mater.27, 8298u2013 8309 (2015)._ _ Bousquet, E. et al. Improper ferroelectricity in perovskite oxide artificial superlattices. Nature452, 732u2013736 (2008)._ _ Benedek, N. A. & Fennie, C. J. Hybrid improper ferroelectricity: a mechanism for controllable polarization-magnetization coupling. Phys. Rev. Lett.106, 107204 (2011)._ _ Pitcher, M. J. et al. Tilt engineering of spontaneous polarization and magnetization above 300 K in a bulk layered perovskite. Science347, 420u2013424 (2015)._ _ Mangeri, J., Pitike, K. C., Alpay, S. P. & Nakhmanson, S. Amplitudon and phason modes of electrocaloric energy interconversion. Npj Comput. Mater. 2, 16020 (2016)._ _ Kimura, T. et al. Interplane tunneling magnetoresistance in a layered manganite crystal. Science274, 1698u20131701 (1996)._ _ Mary, T. A., Evans, J. S. O., Vogt, T., Subramanian, M. A. & Sleight, A. W. Negative thermal expansion from 0.3 to 1050 Kelvin in ZrW2O8. Science272, 90u201392 (1996)._ _ Gillery, F. H. & Bush, E. A. Thermal contraction of u03B2-eucryptite (Li2O.Al2O3.2SiO2) by x-ray and dilatometer methods. J. Am. Ceram. Soc.42, 175u2013177 (1959)._ _ Williams, D. J., Partin, D. E., Lincoln, F. J., Kouvetakis, J. & OKeeffe, M. The disordered crystal structures of Zn(CN)2 and Ga(CN)3. J. Solid State Chem.134, 164u2013169 (1997)._ _ Phillips, A. E., Goodwin, A. L., Halder, G. J., Southon, P. D. & Kepert, C. J. Nanoporosity and exceptional negative thermal expansion in single-network cadmium cyanide. Angew. Chem. Int. Ed.47, 1396u20131399 (2008)._ _ Ravindran, T. R., Arora, A. K., Chandra, S., Valskumar, M. C. & Shekar, N. V. Soft modes and negative thermal expansion in Zn(CN)2 from Raman spectroscopy and first principles calculations. Phys. Rev. B76, 054302 (2007)._ _ Adak, S. et al. Thermal expansion in 3d-metal Prussian blue analogsu2014a survey study. J. Solid State Chem.184, 2854u20132861 (2011)._ _ Dove, M. T., Trachenko, K. O., Tucker, M. G. & Keen, D. A. Rigid unit modes in framework structures: Theory, experiment and applications. Rev. Mineral. Geochem.39, 1u201333 (2000)._ _ Dove, M. T., Heine, V. & Hammond, K. D. Rigid unit modes in framework silicates. Min. Mag.59, 629u2013639 (1995)._ _ Barrera, G. D., Bruno, J. A. O., Barron, T. H. K. & Allan, N. L. Topical review: negative thermal expansion. J. Phys. Condens. Matter17, 217u2013252 (2005)._ _ Senn, M. S. et al. Negative thermal expansion in hybrid improper ferroelectric Ruddlesdenu2013Popper perovskites by symmetry trapping. Phys. Rev. Lett. 114, 035701 (2015)._ _ Senn, M. S. et al. Symmetry switching of negative thermal expansion by chemical control. J. Am. Chem. Soc.138, 5479u2013 5482 (2016)._ _ Ranjbar, B. & Kennedy, B. J. Anisotropic thermal expansion in Sr2RhO4u2014a variable temperature synchrotron x-ray diffraction study. Solid State Sci.49, 43u201346 (2015)._ _ Takahashi, J. & Kamegashira, N. X-ray structural study of calcium manganese oxide by Rietveld analysis at high temperatures. MRS Bull.28, 565u2013573 (1993)._ _ Vogt, T. & Buttrey, D. Temperature dependent structural behavior of Sr2RhO4. J. Solid State Chem.123, 186u2013189 (1996)._ _ Huang, Q. et al. Neutron powder diffraction study of the crystal structures of Sr2RhO4 and Sr2IrO4 at room temperature and at 10 K. J. Solid State Chem.112, 355u2013361 (1994)._ _ Shannon, R. D. Revised effective ionic radii and systematic studies of interatomic distances in halides and chalcogenides. Acta. Crystallogr. A, 32, 751u2013767 (1976).Barron, T. H. K. & Munn, R. W. Analysis of the thermal expansion of anisotropic solids: Application to zinc. Philos. Mag.15, 85u2013103 (1967)._ _ Huang, L.-F., Lu, X.-Z. & Rondinelli, J. M. Tunable negative thermal expansion in layered perovskites from quasi-two-dimensional vibrations. Phys. Rev. Lett.117, 115901 (2016)._ _ Achary, S. N. et al. Anisotropic thermal expansion behavior in tetragonal Sr2MgWO6. Mater. Res. Bull.41, 674u2013682 (2006)._ _ Cordrey, K. J. et al. Structural and dielectric studies of the phase behaviour of the topological ferroelectric La1u2212xNdxTaO4. Dalton. Trans.44, 10673u201310680 (2015)._ _ GrxFCneisen, E. & Goens, E. Thermal expansion of zinc and cadmium. Z. Phys.29, 141u2013156 (1924)._ _ Munn, R. W. Role of the elastic constants in negative thermal expansion of axial solids. J. Phys. C5, 535u2013542 (1972)._ _ Ogborn, J. M., Collings, I. E., Moggach, S. A., Thompson, A. L. & Goodwin, A. L. Supramolecular mechanics in a metal-organic framework. Chem. Sci.3, 3011u20133017 (2012)._ _ Cai, W. & Katrusiak, A. Giant negative linear compression positively coupled to massive thermal expansion in a metalu2013organic framework. Nat. Commun.5, 4337 (2014)._ Goodwin, A. L. et al. Colossal positive and negative thermal expansion in the framework material Ag3[Co(CN)6]. Science319, 794u2013797 (2008)._ _ Fang, H., Dove, M. T. & Refson, K. Agu2014Ag dispersive interaction and physical properties of Ag3[Co(CN)6]. Phys. Rev. B90, 054302 (2014)._ _ Cox, D. E., Shirane, G., Birgeneau, R. J. & Macchesney, J. B. Neutron-diffraction study of magnetic ordering in Ca2MnO4. Phys. Rev. 188, 930u2013932 (1969)._ _ Dove, M. T. Theory of displacive phase transitions in minerals. Am. Mineral82, 213u2013 244 (1997)._ _ Clark, S. J. et al. First principles methods using CASTEP. Z. Kristall. 220, 567u2013570 (2005)._ Payne, M. C., Teter, M. P., Allan, D. C., Arias, T. & Joannopoulos, J. D. Iterative minimization techniques for ab initio total-energy calculations: molecular dynamics and conjugate gradients. Rev. Mod. Phys.64, 1045u20131097 (1992)._ _ Perdew, J. P. et al. Restoring the density-gradient expansion for exchange in solids and surfaces. Phys. Rev. Lett.100, 136406 (2008)._ _ He, L. et al. Accuracy of generalized gradient approximation functionals for density-functional perturbation theory calculations. Phys. Rev. B89, 064305 (2014)._ _ Rappe, A. M., Rabe, K. M., Kaxiras, E. & Joannopoulos, J. D. Optimized pseudopotentials. Phys. Rev. B 41, 1277u20131230 (1990)._ _ Monkhorst, H. J. & Pack, J. D. Special points for Brillouin-zone integrations. Phys. Rev. B13, 5188u2013 5192 (1976)._ _ Parrinello, M. & Rahman, A. Polymorphic transitions in single crystals: a new molecular dynamics method. J. Appl. Phys. 52, 7182u20137190 (1981)._ _ Murnaghan, F. D. The compressibility of media under extreme pressures. Proc. Natl Acad. Sci. USA30, 244u2013247 (1944)._ _ \"- - \"We acknowledge the ISIS neutron spallation source for access to HRPD (RB1520000)._ Preliminary sample characterisation was performed at I11, Diamond Light Source (EE13284). M.S.S. and N.C.B. acknowledge the Royal Commission for the Exhibition of 1851 for a fellowship. N.C.B. is supported by an Imperial College Research Fellowship and a Royal Society Research Grant. M.S.S. acknowledges the John Fell Fund, Oxford (DPD09750) for funding to support this project. C.A. is supported by a studentship in the Centre for Doctoral Training on Theory and Simulation of Materials at Imperial College London funded by the EPSRC (EP/L015579/1). Calculations were performed on the Imperial College London high-performance computing facility. This work was supported by the Thomas Young Centre under grant TYC-101. We are grateful to the UK Materials and Molecular Modelling Hub for computational resources, which is partially funded by EPSRC (EP/P020194/1).\"- - Authors and Affiliations - \"Department of Materials, Imperial College London, London, SW7 2AZ, UKChris Ablitt,_ Arash A. Mostofi_&_Nicholas C. BristoweDepartment of Chemistry, University of Oxford, South Parks Road, Oxford, OX1 3QR, UKSarah Craddock_&_Mark S. SennDepartment of Chemistry, University of Warwick, Gibbet Hill, Coventry, CV4 7AL, UKMark S. SennDepartment of Physics, Imperial College London, London, SW7 2AZ, UKArash A. MostofiSchool of Physical Sciences, University of Kent, Canterbury, CT2 7NH, UKNicholas C. Bristowe\" - \"AuthorsChris AblittYou can also search for this author in _Sarah CraddockYou can also search for this author in _Mark S. SennYou can also search for this author in _Arash A. MostofiYou can also search for this author in _Nicholas C. BristoweYou can also search for this author in _\" - Contributions - C.A. conducted all calculations. M.S.S., A.A.M. and N.C.B. advised on calculations. S.C. synthesised experimental compounds, S.C. and M.S.S. performed neutron diffraction experiments and S.C., M.S.S. and C.A. analysed the data resulting from these experiments. M.S.S., A.A.M. and N.C.B. designed and organised the study. All authors discussed the results and participated in writing the paper. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare that they have no competing financial interests. - \\' \\'- - \\' Publisher\\'\\'s note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution u2013 ShareAlike 3.0 license (CC-BY-SA).Corel Corporation, Corel Painter Education, Version 14.1.0.1105 (2015).\"- - This research was funded by the Cotton Research and Development Corporation and the National Centre for Groundwater Research and Training (funded by the Australian Research Council and the National Water Commission). We also acknowledge the National Collaborative Research Infrastructure Strategy groundwater funding used to purchase the Picarro G2132-i.- - Authors and Affiliations - \"Connected Water Initiative Research Centre, UNSW Australia, UNSW Sydney, 2052, NSW, AustraliaCharlotte P. Iverach,_Dioni I. CendxF3n,_Andy Baker_&_Bryce F. J. KellyAustralian Nuclear Science and Technology Organisation, New Illawarra Rd, Lucas Heights, 2234, NSW, AustraliaDioni I. CendxF3n_&_Stuart I. HankinRoyal Holloway, University of London, Egham Hill, Egham, Surrey, TW20 0EX, United KingdomDavid Lowry,_Rebecca E. Fisher_&_Euan G. NisbetSchool of Environmental Sciences, University of East Anglia, Norwich, Norfolk, NR4 7TJ, United KingdomJames L. France\" - \"AuthorsCharlotte P. IverachYou can also search for this author in _Dioni I. CendxF3nYou can also search for this author in _Stuart I. HankinYou can also search for this author in _David LowryYou can also search for this author in _Rebecca E. FisherYou can also search for this author in _James L. FranceYou can also search for this author in _Euan G. NisbetYou can also search for this author in _Andy BakerYou can also search for this author in _Bryce F. J. KellyYou can also search for this author in _\" - Contributions - Experimental conceptualisation and design was carried out by D.I.C., E.G.N. and B.F.J.K. Fieldwork was conducted by C.P.I., D.I.C., S.I.H., R.E.F., D.L., J.L.F. and B.F.J.K. Geochemical analyses were conducted by D.I.C. and S.I.H. Funding for the work was won by B.F.J.K., D.I.C., A.B. and E.G.N. The manuscript was written by C.P.I. and B.F.J.K with input from all authors.- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing financial interests. - \\' \\'- - \\'\\'- - \\' \\' - \"This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution-Non Commercial 4.0 International license. The authors would like to acknowledge HCPSL and Dr Donna Glassop for providing access to their field trials. Acknowledgment as well to Dr Emilie Fillol from SRA for providing sugarcane weed samples. Acknowledgment to the two CSIRO internal reviewers Dr Rebecca Haling and Dr Graham Bonnett for their valuable comments and suggestions to improve the quality of the paper.- - Authors and Affiliations - \"CSIRO Agriculture and Food, 306 Carmody Road, St Lucia, QLD, 4067, AustraliaJ. S. Pierre,_J. M. Perroux_&_A. L. RaeMolecular Diagnostics Centre, South Australian Research and Development Institute, 2b Hartley Grove, Urrbrae, SA, 5064, AustraliaD. Giblot-Ducray_&_A. C. McKayCSIRO National Research Collections Australia, Canberra, ACT, 2601, AustraliaD. M. Hartley\" - \"AuthorsJ. S. PierreYou can also search for this author in _D. Giblot-DucrayYou can also search for this author in _A. C. McKayYou can also search for this author in _D. M. HartleyYou can also search for this author in _J. M. PerrouxYou can also search for this author in _A. L. RaeYou can also search for this author in _\" - Contributions - J.S.P. and A.L.R. conceived the original project and research plans with the contribution of A.C.M. and D.G.D. J.S.P. and J.M.P. conducted most of the experiments. A.C.M. and D.G.D. set up the SARDI molecular diagnostic laboratory for high throughput delivery of the sugarcane assay and processed and analysed all samples submitted. D.M.H. provided advice on TaqMan assay design. J.S.P. analysed the data. J.S.P. wrote the article with the contribution and editing from A.L.R., D.G.D. and A.C.M. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing Interests - \\' \\' - \"The South Australian Research and Development Institute (SARDI) has Intellectual Property Rights to a body of secret and confidential technical data and skills concerning u201CDNA soil extraction protocolu201D. Drs McKay and Giblot-Ducray representing SARDI provide DNA extraction and testing services available to research groups worldwide and provide testing services to grains and horticultural producers in Australia. SARDI is willing to licence the technology internationally. Drs McKay and Giblot-Ducray, currently or have received funding in the past from Grains Research and Development Corporation (GRDC), Horticulture Innovation Australia (HIA), South Australian Grains Industry Trust (SAGIT), Meat and Livestock Australia (MLA), Fisheries Research and Development Corporation (FRDC) and Sugar Research Australia (SRA) for provision of research involving u201CDNA soil extractionu201D services from soil and sediments. Drs McKay and Giblot-Ducray receive no personal benefit from the provision of the services or exploitation of the Intellectual Property Rights and are bound by confidentiality conditions through their employment with SARDI.\" - \\' \\'- - \"Publishers note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution-Non-Commercial-ShareAlike 3.0 Unported licence. ',\n",
       " 'Creative Commons Attribution-NonCommercial- NoDerivs 3.0 Unported License. ',\n",
       " 'Creative Commons Attribution-NonCommercial- ShareALike 3.0 Unported License. ',\n",
       " 'Creative Commons Attribution-NonCommercial-No Derivate Works 3.0 Unported License. ',\n",
       " 'Creative Commons Attribution-NonCommercial-No Derivative Works 3.0 License. ',\n",
       " 'Creative Commons Attribution-NonCommercial-No Derivative Works 3.0 Unported License. ',\n",
       " 'Creative Commons Attribution-NonCommercial-No Derivative Works 4.0 Unported License. ',\n",
       " 'Creative Commons Attribution-NonCommercial-No Derivs 3.0 Unported License. ',\n",
       " 'Creative Commons Attribution-NonCommercial-No Derivs Works 3.0 Unported License. ',\n",
       " 'Creative Commons Attribution-NonCommercial-NoDerivative Works 3.0 Unported License. ',\n",
       " 'Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported License. ',\n",
       " \"Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported License. The images in this article are included in the article''s Creative Commons license, unless indicated otherwise in the image credit; if the image is not included under the Creative Commons license, users will need to obtain permission from the license holder in order to reproduce the image. \",\n",
       " 'Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported license. ',\n",
       " \"Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported license. The images in this article are included in the article''s Creative Commons license, unless indicated otherwise in the image credit; if the image is not included under the Creative Commons license, users will need to obtain permission from the license holder in order to reproduce the image. \",\n",
       " 'Creative Commons Attribution-NonCommercial-NoDerivs 4.0 Inter-national License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the mate-rial. ',\n",
       " 'Creative Commons Attribution-NonCommercial-NoDerivs 4.0 International License. The images or other third party material in this article are included in the article s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " \"Creative Commons Attribution-NonCommercial-NoDerivs 4.0 International License. The images or other third party material in this article are included in the article''s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder in order to reproduce the material. \",\n",
       " \"Creative Commons Attribution-NonCommercial-NoDerivs 4.0 International License. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. \",\n",
       " 'Creative Commons Attribution-NonCommercial-NoDerivs 4.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " \"Creative Commons Attribution-NonCommercial-NoDerivs 4.0 Unported License. The images or other third party material in this article are included in the article''s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. \",\n",
       " 'Creative Commons Attribution-NonCommercial-Share Alike 3.0 License. ',\n",
       " 'Creative Commons Attribution-NonCommercial-Share Alike 3.0 Unported License. ',\n",
       " 'Creative Commons Attribution-NonCommercial-Share Alike 3.0 Unported License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution-NonCommercial-ShareALike 3.0 Unported License. ',\n",
       " 'Creative Commons Attribution-NonCommercial-ShareALike 4.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License. ',\n",
       " \"Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License. The images in this article are included in the article''s Creative Commons license, unless indicated otherwise in the image credit; if the image is not included under the Creative Commons license, users will need to obtain permission from the license holder in order to reproduce the image. \",\n",
       " 'Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported licence. ',\n",
       " 'Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported license. ',\n",
       " \"Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported license. The images in this article are included in the article''s Creative Commons license, unless indicated otherwise in the image credit; if the image is not included under the Creative Commons license, users will need to obtain permission from the license holder in order to reproduce the image. \",\n",
       " 'Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License. The images or other third party material in this article are included in the article s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " \"Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License. The images or other third party material in this article are included in the article''s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder in order to reproduce the material. \",\n",
       " \"Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License. The images or other third party material in this article are included in the article''s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. \",\n",
       " \"Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License. The images or other third party material in this article are included in the article's Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. \",\n",
       " 'Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. ',\n",
       " 'Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International Public License (u201CCC BY-NC-SA 4.0u201D). Note: We do not recommend combining the training data set and the set of designed measurements into a single dataset. Even though both are from the same measurement system, the dynamic range is different between the two sets (as expected) potentially resulting in disagreement between the reported values for a sequence that exists in both sets._ are provided with this paper.\" - \\' \\'- - \\' \\' - \\'Code is available on github: and on Zenodo under for academic and/or non-profit internal research purposes.\\' - \\' \\'- - \"Arnold, F. H. Directed evolution: bringing new chemistry to life. Angew. Chem. Int. Ed. 57, 4143u20134148 (2018)._ _ _ Kelley, B. Developing therapeutic monoclonal antibodies at pandemic pace. Nat. Biotechnol. 38, 540u2013545 (2020)._ _ _ _ _ Yang, R. et al. Rapid assessment of oxidation via middle-down LCMS correlates with methionine side-chain solvent-accessible surface area for 121 clinical stage monoclonal antibodies. mAbs 9, 646u2013653 (2017)._ _ _ _ _ Jain, T. et al. Biophysical properties of the clinical-stage antibody landscape. Proc. Natl Acad. Sci. USA. 114, 944u2013 949 (2017)._ _ _ _ _ _ Pantazes, R. J. & Maranas, C. D. OptCDR: a general computational method for the design of antibody complementarity determining regions for targeted epitope binding. Protein Eng., Des. Selection 23, 849u2013858 (2010)._ _ _ Li, T., Pantazes, R. J. & Maranas, C. D. OptMAVEn u2013 A New Framework for the de novo Design of Antibody Variable Region Models Targeting Specific Antigen Epitopes. PLOS ONE 9, e105954 (2014)._ _ _ _ _ Fischman, S. & Ofran, Y. Computational design of antibodies. Curr. Opin. Struct. Biol. 51, 156u2013162 (2018)._ _ _ _ Rao, R. et al. Evaluating protein transfer learning with TAPE. Adv. Neural Inf. Process Syst. 32, 9689u20139701 (2019)._ _ _ Alley, E. C., Khimulya, G., Biswas, S., AlQuraishi, M. & Church, G. M. Unified rational protein engineering with sequence-based deep representation learning. Nat. Methods 16, 1315u20131322 (2019)._ _ _ _ _ Bepler, T. & Berger, B. Learning the protein language: evolution, structure, and function. Cell Syst. 12, 654u2013 669.e3 (2021)._ _ _ _ _ Bepler, T. & Berger, B. Learning protein sequence embeddings using information from structure. In International Conference on Learning Representations (2019).Zhang, Z. et al. Protein Representation Learning by Geometric Structure Pretraining. (2022).Yang, K. et al. Analyzing learned molecular representations for property prediction. J. Chem. Inf. Model. 59, 3370u20133388 (2019)._ _ _ _ _ Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Nature 596, 583u2013589 (2021)._ _ _ _ _ _ Liu, G. et al. Antibody complementarity determining region design using high-capacity machine learning. Bioinformatics 36, 2126u2013 2133 (2020)._ _ _ _ Stokes, J. M. et al. A deep learning approach to antibiotic discovery. Cell 180, 688u2013702.e13 (2020)._ _ _ _ _ Mason, D. M. et al. Optimization of therapeutic antibodies by predicting antigen specificity from antibody sequence via deep learning. Nat. Biomed. Eng. 5, 600u2013612 (2021)._ _ _ _ Saka, K. et al. Antibody design using LSTM based deep generative model from phage display library for affinity maturation. Sci. Rep. 11, 5852 (2021)._ _ _ _ _ _ Shin, J.-E. et al. Protein design and variant prediction using autoregressive generative models. Nat. Commun. 12, 2403 (2021)._ _ _ _ _ _ Shan, S. et al. Deep learning guided optimization of human antibody against SARS-CoV-2 variants with broad neutralization. Proc. Natl Acad. Sci. 119, e2122954119 (2022)._ _ _ _ _ Khan, A. et al. Toward real-world automated antibody design with combinatorial Bayesian optimization. Cell Rep. Methods 3, 100374 (2023)._ _ _ _ _ Vaswani, A. et al. Attention is All you Need. in Advances in Neural Information Processing Systems vol. 30 (Curran Associates, Inc., 2017).Devlin, J., Chang, M.-W., Lee, K. & Toutanova, K. BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding. in Proceedings of the 2019 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies, Volume 1 (Long and Short Papers) 4171u20134186 (Association for Computational Linguistics, 2019).Ricke, D. O. Two Different Antibody-Dependent Enhancement (ADE) Risks for SARS-CoV-2 Antibodies. Front Immunol. 12, 640093 (2021)._ _ _ _ _ Engelhart, E. et al. A dataset comprised of binding interactions for 104,972 antibodies against a SARS-CoV-2 peptide. Sci. Data 9, 653 (2022)._ _ _ _ _ Leslie, S. mit-ll/AlphaSeq_Antibody_Dataset: AlphaSeq Antibody Datasets 1 & 2. (2023)El-Gebali, S. et al. The Pfam protein families database in 2019. Nucleic Acids Res. 47, D427u2013D432 (2019)._ _ _ _ Kovaltsuk, A. et al. Observed Antibody Space: A Resource for Data Mining Next-Generation Sequencing of Antibody Repertoires. J. Immunol 201, 2502u20132509 (2018)._ _ _ _ Rasmussen, C. E. & Williams, C. K. I. Gaussian Processes for Machine Learning. (2005).Cock, P. J. A. et al. Biopython: freely available Python tools for computational molecular biology and bioinformatics. Bioinformatics 25, 1422u20131423 (2009)._ _ _ _ _ Li, L. et al. Antibody Representation Learning for Drug Discovery. Preprint at (2022).Kingma, D. P. & Ba, J. Adam: A Method for Stochastic Optimization. Preprint at (2017).Paszke, A. et al. PyTorch: An Imperative Style, High-Performance Deep Learning Library. in Advances in Neural Information Processing Systems vol. 32 (Curran Associates, Inc., 2019).Artificial Intelligence: A Modern Approach, 4th US ed. (2022).Katoch, S., Chauhan, S. S. & Kumar, V. A review on genetic algorithm: past, present, and future. Multimed. Tools Appl 80, 8091u20138126 (2021)._ _ _ Levine, R. A., Yu, Z., Hanley, W. G. & Nitao, J. J. Implementing random scan Gibbs samplers. Comput. Stat. 20, 177u2013196 (2005)._ _ _ _ Tran, T. D., Hofrichter, J. & Jost, J. An introduction to the mathematical structure of the Wrightu2013Fisher model of population genetics. Theory Biosci. 132, 73u201382 (2013)._ _ _ Sinai, S. et al. AdaLead: A simple and robust adaptive greedy search algorithm for sequence design. Preprint at (2020).High-throughput characterization of proteinu2013protein interactions by reprogramming yeast mating | PNAS. (2017).Li, H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics 34, 3094u20133100 (2018)._ _ _ _ _ Hinton, G. E. & Roweis, S. Stochastic Neighbor Embedding. in Advances in Neural Information Processing Systems vol. 15 (MIT Press, 2002).Kramer, O. Scikit-Learn. in Machine Learning for Evolution Strategies (ed. Kramer, O.) 45u201353 (Springer International Publishing, 2016).Wold, S., Esbensen, K. & Geladi, P. Principal component analysis. Chemometrics Intell. Lab. Syst. 2, 37u201352 (1987)._ _ _ Bjellqvist, B. et al. The focusing positions of polypeptides in immobilized pH gradients can be predicted from their amino acid sequences. ELECTROPHORESIS 14, 1023u20131031 (1993)._ _ _ _ Bjellqvist, B., Basse, B., Olsen, E. & Celis, J. E. Reference points for comparisons of two-dimensional maps of proteins from different human cell types defined in a pH scale where isoelectric points correlate with polypeptide compositions. Electrophoresis 15, 529u2013539 (1994)._ _ _ _ Tabb, D. L. An algorithm for isoelectric point estimation. (2003).Kyte, J. & Doolittle, R. F. A simple method for displaying the hydropathic character of a protein. J. Mol. Biol. 157, 105u2013132 (1982)._ _ _ _ Lin, Li, Spaeth, John, Gupta, Esther & Shing, Leslie. Machine Learning Optimization of Candidate Antibody Yields Highly Diverse Sub-nanomolar Affinity Antibody Libraries. (2023)\"- - We would like to thank the scientists at A-Alpha Bio for assistance in generation of data; Eric Schwoebel, Joshua Dettman, and Tim Lu for thoughtful discussion of the research program and approach; Jack McGowan and Irene Stapleford for graphic design support; and the many other colleagues at MIT LL who have supported this project. This research was funded by the Under Secretary of Defense for Research and Engineering under Air Force Contract No. FA8702-15-D-0001. Any opinions, findings, conclusions or recommendations expressed in this work are those of the author(s) and do not necessarily reflect the views of the Under Secretary of Defense for Research and Engineering.- - Author notes - \\'Tristan BeplerPresent address: Simons Electron Microscopy Center, New York Structural Biology Center, New York, NY, USAMatthew E. WalshPresent address: Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, USAThese authors contributed equally: Rajmonda S. Caceres, Tristan Bepler, Matthew E. Walsh.\\' - Authors and Affiliations - \"Massachusetts Institute of Technology Lincoln Laboratory, Lexington, MA, USALin Li,_Esther Gupta,_John Spaeth,_Leslie Shing,_Rafael Jaimes,_Rajmonda S. Caceres_&_Matthew E. WalshA-Alpha Bio, Inc., Seattle, WA, USAEmily Engelhart_ &_Randolph LopezResearch Laboratory of Electronics, Massachusetts Institute of Technology, Cambridge, MA, USATristan Bepler\" - \"AuthorsLin LiYou can also search for this author in _Esther GuptaYou can also search for this author in _John SpaethYou can also search for this author in _Leslie ShingYou can also search for this author in _Rafael JaimesYou can also search for this author in _Emily EngelhartYou can also search for this author in _Randolph LopezYou can also search for this author in _Rajmonda S. CaceresYou can also search for this author in _Tristan BeplerYou can also search for this author in _Matthew E. WalshYou can also search for this author in _\" - Contributions - R.C., T.B., and M.W., conceived the project; R.C. and M.W. designed the data collection process; L.S. performed in silico mutagenesis; L.L., R.C., and T.B. designed model architectures; L.L., E.G., J.S., and T.B. performed algorithm development; L.L., E.G., J.S., and L.S. trained models; L.L., E.G., and L.S. evaluated models; L.L., E.G., J.S., T.B., and M.W. performed data analysis; E.E. and R.L. generated empirical data; L.L., R.J., E.E., R.C., T.B., and M.W. wrote and edited the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - T.B. is the co-founder and CEO of NE47 Bio Inc., a company that provides machine learning services and software for protein engineering. R.L. is a co-founder and current employee of AAlpha Bio, Inc. (A-Alpha Bio) and owns stock/stock options of A-Alpha Bio. E.E. is an employee of A-Alpha Bio and owns stock/stock options of A-Alpha Bio. The remaining authors, L.L., E.G., J.S., L.S., R.J., R.C. and M.W. declare no financial and non-financial competing interests. Massachusetts Institute of Technology has filed a provisional patent application (63/373682) on the end-to-end method for generating scFv libraries; inventors include L.L., E.G., J.S., L.S., R.J., R.C., T.B. and M.W. A-Alpha Bio has a patent (US10988759B2) relating to certain research described in this article. - \\' \\' - \"xA9 2023 Massachusetts Institute of Technology. Subject to FAR52.227-11 Patent Rights - Ownership by the contractor (May 2014). Delivered to the U.S. Government with Unlimited Rights, as defined in DFARS Part 252.227-7013 or 7014 (Feb 2014). Notwithstanding any copyright notice, U.S. Government rights in this work are defined by DFARS 252.227-7013 or DFARS 252.227-7014 as detailed above. Use of this work other than as specifically authorized by the U.S. Government may violate any copyrights that exist in this work.\" - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks Rodrigo Aguiar, Ivan Anishchenko and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " \"Creative Commons Attribution-Noncommercial-No Derivative Works 3.0 Unported License - ' ' - ''- - \",\n",
       " 'Creative Commons Attribution-Share Alike 2.0 Generic license, which allow freely to share (to copy, distribute and transmit the work), and to remix (to adapt the work).\"- - \"Specimens of N. vespillo were obtained from pheromone traps used in commercial forests to catch the European spruce bark beetle Ips typographus. The traps were deployed in small forest complexes (52xB0 05u2032 23u02BA N 16xB0 24u2032 01u02BA E) in the Ziemin Forest District (Kou015Bcian Forest Inspectorate; SW Poland), and the use of such traps in forests by forestry services does not require special permits. All the material was obtained in Juneu2013August 2017; and all the specimens of N. vespillo used in the study, were provided to us by the forest services. The presence of N. vespillo in such traps is of a secondary nature: they had been lured there by the odour of wet insect carcasses, decomposing as a result of the poor drainage of rainwater from the traps.\" - \"The orange and black elytral fragments were isolated under a binocular microscope (xD7u200910) using a microscalpel. A single sample consisted of the relevant elytral fragments from six beetles. A total of 240 specimens of N. vespillo were used in the study: this yielded 39 samples with orange fragments and 39 samples with black fragments (see Supplementary Information file, Table ). Samples of orange (marked \"A\") and black (marked \"B\") fragments were placed in pairs in plastic bags and numbered.\" - Chemical analysis - Instruments - \"An inductively-coupled plasma optical emission spectrometer (Agilent 5110, Agilent, USA) was used to determine the elemental composition of the samples. The simultaneously axial and radial view of the plasma was enabled by synchronous vertical dual view (SVDV). The standard conditions were applied: Radio Frequency (RF) power 1.2_kW, nebulizer gas flow 0.7 L min, auxiliary gas flow 1.0 L min, plasma gas flow 12.0 L min, viewing height for radial plasma observation 8_mm, detector CCD (Charge Coupled Device) temperature u2009u2212u200940_xB0C, signal accusation time 5_s for 3 replicates. A Mars 5 (CEM, Matthews, USA) microwu2009 ave sample preparation system was used for sample digestion.\" - Procedure - \"Accurately weighed (range;u2009xB1u2009SE) 0.0027u20130.0091_g (xB1u2009 0.00013_g) dry samples were digested in 2_mL concentrated nitric acid in closed Teflon containers in a Mars 6 (CEM, USA) microwave sample preparation system. After digestion, the samples were diluted with water to a final volume of 6.0_ mL.\" - Analytical method validation - \"The detection limits were estimated at the level of 0.01_mg_kg dry weight (ppm. d.w.) for all the elements determined (as 3-sigma criteria). The uncertainty of the overall analytical procedure (including sample preparation) was 20%. Traceability was checked using reference materials: CRM S-1u2014loess soil; CRM NCSDC (73,349)u2014bush branches and leaves; CRM 2709u2014soil; CRM 405 and CRM 667u2014estuarine sediments. The recovery (80u2013120%) was acceptable for most of the elements determined. The recovery of the uncertified elements was defined by the standard addition method.\" - Statistical treatment - \"Initially, we assessed the overall frequency of element detectability (i.e. measurement above the detection limit) in the orange versus black elytral regions (nu2009 =u200939 in each case), using the t-test for paired comparisons and its non-parametric equivalent, the Wilcoxon signed rank test, which takes into account the small number of paired observations for many elements.\" - \"Owing to the very considerable variance in the concentrations of some elements and the large number of non-detects (excluded from our major quantitative analysis) among the measurements (see u201Cu201D), most results are presented in a descriptive fashion. For the same reasons, we employed two different approaches to data analysis when calculating average elemental concentrations and assessing potential differences between the black and orange regions of the elytra: (i) collectively across all the beetles examined, and (ii) measured simultaneously as pseudo-pairs per group of six individual beetles.\" - \"First, we examined the overall differences in concentrationsof individual elements, as well as the total concentration (u2211) of all elements and the sum of different REEs between the black and orange elytral regions across all the available data points (Table ; Supplementary Information file, Table ). We used one-way ANOVA to test this, checking the variables (elemental concentrations) for normality (Kolmogorovu2013Smirnov test) for all ANOVAs. When necessary, data log-transformations were carried out to meet the parametric assumption prior to analysis. In principle, however, this treatment had no effect on the final ANOVA results. Moreover, because log-transformation did not improve the distribution of some variables, we repeated the between-samples comparison using the non-parametric Mannu2013 Whitney test.\" - \"Table 1 Average (xB1u2009SE) concentrations (ppm d.w.) of chemical elements measured by ICP-OES in two adjacent regions of an insect cuticle with a contrasting colour pattern, namely, the black and orange regions of the elytra of the aposematic burying beetle Nicrophorus vespillo (Coleoptera: Silphidae; see Fig._).\" - Second, to reduce the large overall variance in elemental concentrations within the entire population of N. vespillo with the aim of achieving more consistent results in the context of individual variation, the data on elemental concentrations were arranged in the form of pseudo-pairs, i.e. corresponding measurements of orange (\"A\") and black (\"B\") elytral samples derived from six beetles (Table ). We used the t-test for dependent samples to test the differences for these paired measurements. Owing to the large number of non-detects for some chemical elements, fewer samples (i.e. simultaneous paired observations for orange and black elytral samples) were used in the t-test analysis than for the full data set presented in Table . - \"Table 2 Average (xB1u2009SE) concentrations (ppm d.w.) of the chemical elements measured simultaneously in pseudo-pairs in two adjacent regions of an insect cuticle with a contrasting colour pattern, namely, the black and orange regions of the elytra of the aposematic burying beetle Nicrophorus vespillo (Coleoptera: Silphidae).\" - \"The statistical analyses were performed using Statistica ver. 12.5 and Excel software. A probability of Pu2009<u20090.05 was considered statistically significant.\" - Ethics declarations - We did not employ invasive techniques for this study.- - \"ICP-OES analysis of 78 fragments of orange and black elytral regions from N. vespillo (Fig._) revealed 53 elements (ranging in atomic weight from Na to Bi) present above the detection limit (Table ). The frequency of detectability of individual elements varied strongly, however, and only ten elements (Ba, Cu, Fe, K, Mg, Mn, P, Rb, Sb and Zn) occurred in concentrations exceeding the detection limit in each sample. The number of non-detects for the remaining elements varied greatly from 1 (Pb) to as many as 77 (Nb and V) (Table ). We found no statistical differences in the frequency of detectabilityof the 53 elements in the orange versus black elytral regions (nu2009=u200939 in each case) when using either the t-test or the Wilcoxon test for paired comparisons (Pu2009u2265u20090.771).\" - \"The overall concentration (u2211 concentrations) of all 53 elements (Table ) was low, being merely 112.8 (95% CI 96.4; 129.2) ppm d.w. and 103.5 (95% CI 87.2; 119.4) ppm d.w. in the black and orange elytral regions, respectively. Correspondingly, the total concentration of all the elements in the orange regions was 9% lower than in the black ones. However, this difference was not statistically significant (one-way ANOVA, F1,76u2009=u20090.683, Pu2009=u20090.411), presumably because of the mostly extremely large variance in overall concentration (u03C3 u2009=u20092558 and 2417 in the black and orange regions, respectively).\" - \"The rare earth elements exhibited the opposite distribution. The sum of REEs included the concentrations of 13 elements (Ce, Dy, Er, Gd, Ho, La, Lu, Nd, Pr, Sc, Sm, Tb and Y) and was 17% lower in the black regions of the elytra than in the adjacent orange ones (Table ). The variance in the sum of REE concentrations measured in the black and orange regions was negligible (u03C3u2009=u2009 0.08 and 0.011, respectively), and the difference between these two samples did not approach statistical significance (ANOVA, F1,76u2009=u20090.923, Pu2009 =u20090.340).\" - \"Further analysis of the variance in concentrations of individual elements showed that 14 elements displayed variancesu2009>u20091 (Fig._). Specifically, we found an enormous variance in Ca concentrations (u03C3u2009>u20091800); at the same time, this was the most abundant element, making up an average of 30.2% (95% CI 25.0%; 35.5%; nu2009=u200976) of the sum of all elemental concentrations per sample. Concentrations of 12 other elements (K, P, Na, Mg, Ba, Sb, Fe, Cu, Al, As, Zr and Hf) exhibited variances of up to 145, although this was also dependent on the colour of the elytra sample, e.g. the variances in K, P and Na concentrations were much higher in the black region (Fig._). The variance in concentration of all the other elements listed in Table was negligible, alwaysu2009u2264u20090.69.\" - \"Figure 2Thirteen elements measured in the black and orange regions of the elytra with variance (u03C3) of concentrationsu2009>u20091; elements shown in order of decreasing variance. The small panel depicts elements with relatively lower variances in their concentrations: the respective variances for As, Zr and Hf measured in the black regions were 0.052, 0.019 and 0.006, while that for Al measured in the orange regions was 0.09. The variance in concentration of all the other elements listed in Table in both these elytral regions was alwaysu2009 u2264u20090.69.\" - \"To visualize the variation in the distribution of individual elements, we calculated the ratios of their concentrations measured in the black and orange elytral regions in all the samples (Fig._). For this purpose, we used the concentrations of 48 elements, for which sample sizes allowed this calculation (see Table ). Interestingly, we found that the concentrations of half (24) of these elements were higher in the black elytral region (Fig._), ranging from 1.011 (Ca) to 1.969 (Al). In contrast, the concentrations of the other 24 elements were higher in the orange regions of the elytra than in the black ones, the difference ranging from 0.052 (Hf) to 0.998 (Zn).\" - \"More detailed analysis for individual elements, based on a rigorous statistical treatment aimed at assessing the differences between the black and orange regions of the elytra, showed that the concentrations of four elements varied significantly between these two samplesu2014Al (ANOVA, F1,19u2009=u20096.94, Pu2009=u2009 0.016) and Cu (ANOVA, F1,76u2009=u20094.79, Pu2009=u20090.032)u2014and in the non-parametric analysis for Mn and W. Specifically, the concentrations of these four elements in the black elytral regions were respectively higher by 97%, 41%, 14% and 47% than in the orange regions (see Table for elemental concentrations, and the results of the Mannu2013Whitney test for Mn and W).\" - \"Lastly, the analysis using the t-test for dependent samples based on data on elemental concentrations set up as pseudo-pairs (i.e. corresponding measurements from six different beetles), presented in Table , shows that concentrations of Cu (t37u2009=u2009u2212u20092.67, Pu2009=u20090.011), Na (t34u2009 =u2009u2212u20092.18, Pu2009=u20090.036) and Se (t3u2009=u2009u2212u2009 5.03, Pu2009=u20090.015) were significantly higher in the black regions than in the orange regions of the elytra by 40%, 46% and 97%, respectively. Furthermore, another marginally significant difference was displayed by the Al concentration (t6u2009=u2009u2212u20092.11, Pu2009=u20090.079), being 56% higher in the black regions of the elytra (Table ).\"- - In this paper we provide novel and basic quantitative information on the concentrations of tens of chemical elements occurring within a specific animal tissue, an insect exoskeleton, and confirm the potential usefulness of ICP-OES as an effective tool for determining the ionomic profiles of microsamples of highly sclerotized insect tissues. Based on our estimated concentration ratios of elements measured in the black and orange elytral regions of N. vespillo, our results suggest that the distributions of all the elements combined, rare earth elements and nearly every individual element are unequal between these two differently coloured parts of the elytra, although the trend of these associations varies between elements. In a broader sense, our study appears to confirm the idea that dark (melanin-rich) regions can act as sinks/reservoirs for certain trace elements/metals. This was the case for inter alia Al, Se, Ce, W, Cu, Na, Si, Er, Sc, Mg, Ni and Pb, whereas a group of other elements (Lu, In, La, Te, Pd, Ru, As, Sn, Zr and Hf) tended to be concentrated in the orange elytral regions. - \"It should be emphasized, however, that rigorous statistical treatment yielded much less ambiguous results and that concentrations of only a minority of elements varied significantly between the black and orange elytral parts. Specifically, depending on the type of data analysis approach (across all beetles, and measured simultaneously as pseudo-pairs), we found that the black elytral regions were apparently characterized by higher concentrations of Al, Cu, Mn and W (demonstrated by one or both of these approaches) and Na and Se (pseudo-pairs). Basically, close inspection showed that the concentrations of these six elements obtained with both these approaches (cf. Tables and ) were broadly consistent, confirming their apparently higher levels in the black elytral region. This is also clearly evident from Fig._, where all these elements are placed on the left-hand side of that chart. In addition, we found that the between-sample differences for almost all the other elements listed in Tables and showed consistent trends. Thus, in spite of some minor inconsistencies (e.g. for Tb or Y), these two approaches are relevant to investigations of colour-assisted variation in elemental composition in our sample of beetle elytra. It is hard to state unequivocally, however, that the above-mentioned metals (or others, like REEs) are responsible for the differences in colouration between the black and orange elytral regions of N. vespillo. More importantly, since our samples are cross-sections through all the elytral layers, establishing such a relationship would require the examination of uniformly coloured layers of the beetle elytra.\" - \"Another important aspect of our investigation was to describe the magnitude of the variability in the concentrations of different elements. In particular, there was very considerable variance in the concentrations of essential elements (Ca, K, P, Na, Mg and Ba) measured in both the black and orange elytral regions (see Fig._). Such a large variability is extremely surprising, and at present we are unable to pinpoint the exact reasons for this. We may speculate that such a high variance in essential element concentrations could in part be attributed to individual differences, e.g. those resulting from the consumption of animal carcasses of different quality/chemical composition, or from sexual differences, especially that female burying beetles spend much more time at a carcass caring for their offspring than males. Furthermore, the intensity of colouration of both the orange and black elytral markings of burying beetles is variable (as found in a related species of burying beetle,N. vespilloides), which may be responsible for the different content of pigments and/or elements. Moreover, as a form of chemical (antipredator) defence, both male and female burying beetles secrete anal exudates, with females producing more of this fluid; to some degree, this could also affect the entire metal pool in the body. Consequently, the depletion of essential elements (or their intensive passage/trafficking) also seems to be more likely in females. Lastly, interferences between elements (in particular Ca) and the resulting lowering of measurement quality are also possible (e.g.). On the other hand, the variances in concentrations of other elements, including REEs, are low (u03C3u2009u2264u20090.69), which in combination with their low concentrations indicate either stringent physiological control or a poor ability to bind with the exoskeleton.\" - Importantly, because the concentrations of chemical elements in different coloured regions of the insect integument were not measured in previous studies, we can only compare our data with those earlier studies in a broader ecotoxicological context, i.e. at the level of the whole body and the chitinous integument. Consequently, inspection of the concentrationswe obtained, which are basically at normal physiological levels given the whole-body concentrations of the vast majority of elements (cf.), reveals high levels of As and Sb. Another recent study has shown that large amounts of As (up to 254.9 As ppm d.w.) bind with beetle elytra. At the same time, whole-body concentrations of Sb in different invertebrate taxa were generally much lower (up to 3.26 Sb ppm d.w.) than in our study (up to 35.47 Sb ppm d.w.; Table ). Nonetheless, there are data showing much higher Sb concentrations in organisms from contaminated mine sites (up to 66.4 Sb ppm d.w.). It remains an open question, however, whether the high concentrations of these metalloids can be related to the insecticides used in the pheromone traps from which we collected the analytical material.- - Two major conclusions with obvious research perspectives can be drawn from our study. - Firstly, in a broader perspective, beetle elytra or insect exoskeletons are readily available, e.g. from remains following a predation event, faeces, pellets, guano deposits, roadkill, museum collections or even palaeobiological material, so further studies employing such analytical material in the biomonitoring of environmental quality are both feasible highly desirable. Such studies permit an effective assessment of changes in environmental quality, with respect to both the historical context and current land use. - \"Secondly, at the organism level, various analytical techniques are used to determine the content of chemical elements in biological samples: atomic absorption spectrometry (FAAS and/or GFAAS), inductively coupled plasma with detection of optical emission spectrometry (ICP-OES), and mass spectrometry (ICP-MS). All of these techniques are equally relevant and useful. From the methodological point of view, it should borne in mind that in our study, ICP-OES measured the concentrations of a large number of elements, but at a relatively high detection limit (>u20090.01_ ppm; see u201Cu201D). Some trace elements, however, in particular those associated with pigments present in thin layer(s) of elytra or specific subcellular structures and/or cuticular macromolecules (e.g. lipids/wax, proteins; reviewed in), most likely occur in different coloured parts of beetle elytra (or various parts of an insects parti-coloured integument) in much lower concentrations. The one basic issue relating to our methodology is that the drawing of logical inferences regarding the source of differences in the elemental composition of insect exoskeletons or beetle elytra is that our sampling covered the entire thickness of the chitninous integument. This clearly implies the need for future studies employing many more throughput methods of chemical analysis for determining such low contents of chemical elements in individual insect micro-samples. We highlight the fact that progress in understanding the basic relationship between insect colouration and variation in elemental composition requires the micro-sampling of the homogeneous layers of exoskeletons. Consequently, although the ecological significance of our study seems to be weak, given the absence of an argument that there is a functional correspondence between the colour of elytra and their variability in the ionomic profile, we strongly encourage further exploration of the basic question on metal partitioning between differently coloured parts of sclerotized insect integuments in order to account for individual differences, e.g., sex, body size and colour variation.\"- - \\' \\' - \\'All the data analysed during this study are included in this published article (and its Supplementary Information file). Table : Concentrations (ppm d.w.) of chemical elements measured by ICP-OES in the black and orange regions of the elytra of the burying beetle Nicrophorus vespillo.\\' - \\' \\'- - \"Liu, Y. & Simon, J. D. Isolation and biophysical studies of natural eumelanins: Applications of imaging technologies and ultrafast spectroscopy. Pigm. Cell Res. 16, 606u2013618 (2003)._ _ _ Seago, A. E., Brady, P., Vigneron, J.-P. & Schultz, T. Gold bugs and beyond: A review of iridescence and structural colour mechanisms in beetles (Coleoptera). J. R. Soc. Inter. 6, S165u2013S184 (2009)._ _ Shamim, G., Ranjan, S. K., Pandey, D. M. & Ramani, R. Biochemistry and biosynthesis of insect pigments. Eur. J. Entomol. 111, 149u2013164 (2014)._ _ _ McNamara, M. E. The taphonomy of colour in fossil insects and feathers. Palaeontology 56, 557u2013575 (2013)._ _ McGraw, K. Melanins, metals, and mate quality. Oikos 102, 402u2013406 (2003)._ _ _ Lifshitz, N. & Clair, C. C. Coloured ornamental traits could be effective and non-invasive indicators of pollution exposure for wildlife. Conserv. Physiol. 4, 28. (2016)._ _ _ Baxter, I. Ionomics: The functional genomics of elements. Brief. Funct. Genom. 9, 149u2013156 (2010)._ _ _ Kikkawa, H., Ogita, Z. & Fujito, S. Nature of pigments derived from tyrosine and tryptophan in animals. Science 121, 43u2013 47 (1955)._ _ _ _ Hare, L. & Campbell, P. G. Temporal variations of trace metals in aquatic insects. Fresh. Biol. 27, 13u201327 (1992)._ _ _ Roth, M. Metals in invertebrate animals of a forest ecosystem. In Biogeochemistry of Trace Metals (ed. Adriano, D. C.) 299u2013328 (Lewis Publisher, Boca Raton, 1992)._ Bednarska, A. J., u015Awiu0105tek, Z. M., Paciorek, K. & Kubiu0144ska, N. Effect of cadmium bioavailability in food on its compartmentalisation in carabids. Ecotoxicology 26, 1259u20131270 (2017)._ _ _ Oru0142owski, G. et al. Breaking down insect stoichiometry into chitin-based and internal elemental traits: Patterns and correlates of continent-wide intraspecific variation in the largest European saproxylic beetle. Environ. Poll. 262, 114064. (2020)._ _ _ Abdelnour, S. A. et al. Impacts of rare earth elements on animal health and production: Highlights of cerium and lanthanum. Sci. Total Environ. 672, 1021u20131032 (2019)._ _ _ _ Adeel, M. et al. Cryptic footprints of rare earth elements on natural resources and living organisms. Environ. Int. 127, 785u2013 800 (2019)._ _ _ Polidori, C. et al. Strong phylogenetic constraint on transition metal incorporation in the mandibles of the hyper-diverse Hymenoptera (Insecta). Organ. Diver. Evol. 20, 511u2013526 (2020)._ _ Sun, J. & Bhushan, B. Structure and mechanical properties of beetle wings: A review. RSC Adv. 2(33), 12606. (2012)._ _ _ Vincent, J. F. V. & Wegst, U. Design and mechanical properties of insect cuticle. Arthrop. Struc. Dev. 33, 187u2013199 (2004)._ _ Heinrich, B. A heretofore unreported instant color change in a beetle, Nicrophorus tomentosus Weber (Coleoptera: Silphidae). Northeast. Nat. 19, 345u2013352 (2012)._ _ Biro, L. P. & Vigneron, J.-P. Photonic nanoarchitectures in butterflies and beetles: Valuable sources for bioinspiration. Laser Photonics Rev. 5, 27u201351 (2011)._ _ _ Stavenga, D. G., Wilts, B. D., Leertouwer, H. L. & Hariyama, T. Polarized iridescence of the multilayered elytra of the Japanese jewel beetle, Chrysochroa fulgidissima. Phil. Trans. R. Soc. B 366, 709u2013723 (2011)._ _ Yoshioka, S. & Kinoshita, S. Direct determination of the refractive index of natural multilayer systems. Phys. Rev. E 83, 051917. (2011)._ _ _ _ Li, L., Guo, C., Li, X., Xu, S. & Han, C. Microstructure and mechanical properties of rostrum in Cyrtotrachelus longimanus (Coleoptera: Curculionidae). Anim. Cell Syst. 21, 199u2013206 (2017)._ _ _ Sun, M. et al. Compound microstructures and wax layer of beetle elytral surfaces and their influence on wetting properties. PLoS ONE 7, e46710. (2012)._ _ _ _ _ _ Armada, N. A. & McNamara, M. E. Spatial variation of cuticular trace elements in patterned insects (GSA Annual Meeting in Denver, Colorado, USA u2013 2016) Paper No. 195u20137. Available at (2016).Lindstedt, C., Boncoraglio, G., Cotter, S., Gilbert, J. & Kilner, R. Aposematism in the burying beetle? Dual function of anal fluid in parental care and chemical defense. Behav. Ecol. 28, 1414u20131422 (2017)._ _ Gohli, J. & HxF6gstedt, G. Explaining the evolution of warning coloration: Secreted secondary defence chemicals may facilitate the evolution of visual aposematic signals. PLoS ONE 4, e5779. (2009)._ _ _ _ _ _ Bezzerides, A. L., McGraw, K. J., Parker, R. S. & Husseini, J. Elytra color as a signal of chemical defense in the Asian ladybird beetle Harmonia axyridis. Behav. Ecol. Soc. 61, 1401u20131408 (2007)._ _ Galko, J. et al. Effectiveness of pheromone traps for the European spruce bark beetle: A comparative study of four commercial products and two new models. For. J. 62, 207u2013215 (2016)._ Agilent Technologies. ICP-OES Application eHandbook (Agilent Technologies, Inc. 2017. May 26, 2017. 5991u20138147EN). Available at (2017).Siwulski, M. et al. The effect of different substrates on the growth of six cultivated mushroom species and composition of macro and trace elements in their fruiting bodies. Eur. Food Res. Tech. 245, 419u2013 431 (2019)._ _ _ StatSoft. Statistica (Data Analysis Software System), Version 12.1. (Tulsa: USA) (2014).Otronen, M. The effect of body size on the outcome of fights in burying beetles. Ann. Zool. Fenn. 25, 191u2013201 (1988)._ Butler, O., Cairns, W., Cook, J. & Davidson, C. Atomic spectrometry update Environmental analysis. J. Anal. Atom. Spectr. 25, 103u2013141 (2010)._ _ _ Oru0142owski, G. et al. Edge effect imprint on elemental traits of plant-invertebrate food web components of oilseed rape fields. Sci. Total Environ. 687, 1285u20131294 (2019)._ _ _ Duran, M., Kara, A., Akyildiz, A. & Ozdemir, A. Antimony and heavy metals accumulation in some macroinvertebrates in the Yesilirmak River (N Turkey) near the Sb-mining area. Bull. Environ. Contam. Toxicol. 78, 395u2013399 (2007)._ _ _ Dovick, M. A., Kulp, T. R., Arkle, R. S. & Pilliod, D. S. Bioaccumulation trends of arsenic and antimony in a freshwater ecosystem affected by mine drainage. Environ. Chem. 13, 149u2013159 (2015)._ _ Nocera, J. J. et al. Historical pesticide applications coincided with an altered diet of aerially foraging insectivorous chimney swifts. Proc. R. Soc. B 279, 3114u20133120 (2012)._ _ _ McRae, R., Bagchi, P., Sumalekshmy, S. & Fahrni, C. J. In situ imaging of metals in cells and tissues. Chem. Rev. 109, 4780u2013 4827 (2009)._ _ _ \"- - \"We would like to thank to two anonymous reviewers, whose constructive comments helped us to considerably improve the manuscript. We thank Daria Zarabska-Bou017C ejewicz for measuring the thicknesses of the black and orange layers in the elytra of N. vespillo; and Peter Senn for improving the English. G.O. thanks Violetta Ciach-Melewska for the informative discussion about the chemical composition of pigments.\"- - Authors and Affiliations - \"Institute for Agricultural and Forest Environment, Polish Academy of Sciences, Bukowska 19, 60-809, Poznau0144, PolandGrzegorz Oru0142owskiDepartment of Analytical Chemistry, Adam Mickiewicz University, Uniwersytetu Poznanskiego 8, 61-614, Poznau0144, PolandPrzemysu0142aw Niedzielski_&_Ju0119drzej ProchDepartment of Nature Conservation, Faculty of Biological Sciences, University of Zielona GxF3ra, Prof. Z. Szafrana 1, 65-516, ZielonaGxF3ra, PolandJerzy Karg\" - \"AuthorsGrzegorz Oru0142owskiYou can also search for this author in _Przemysu0142aw NiedzielskiYou can also search for this author in _Jerzy KargYou can also search for this author in _Ju0119drzej ProchYou can also search for this author in _\" - Contributions - G.O. designed the project together with J.K. J.K. collected data. P.N. and J.P. conducted the chemical analysis. G.O. and P.N. interpreted the results and wrote the manuscript with help, guidance and review from J.K. and J.P. All co-authors approved the final version of the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution-Share Alike 3.0 Unported (CC BY-SA 3.0).\" - \"The skyrmions are nucleated as described in a previous work by applying an in-plane magnetic field pulse on top of a static out-of-plane field at the spin reorientation transition. One of the main advantages of this device is the automatic initialization: by, ideally, nucleating a single skyrmion in the device, the system immediately relaxes to the required ground state as the skyrmion resides primarily in the central region of the confinement due to the skyrmion-edge repulsion as shown in Fig._. Thereby and in particular without the necessity of further input or adjustment of the local pinning effect, the ground state, in which the system resets itself when no potential is applied, is achieved. Thus, the interplay of thermal dynamics and edge repulsion acts as an auto-initialization and reset mechanism simultaneously.\" - \"Fig. 2: Skyrmion displacement.Subtracted Kerr-microscopy images of an equilateral triangular device with an edge length of 36u2009xB5m (tip to tip), each with a single skyrmion (dark gray) in a the ground state without and bu2013e with applied electrical potentials. In brackets below are the input patterns of the respective state. e includes exemplarily the four regions used to mimic MTJs in our analysis._Red arrows show the most probable directions in which the skyrmion will move due to thermal excitation. The chromium/gold contacts overlap the thin film in the corners and on the bottom edge (the latter is unused for the results shown).\" - Current-induced skyrmion dynamics - \"As seen in Fig._, the skyrmion without any applied voltage rests in the middle region of the triangle. For example, by applying a positive potential at one corner, a negative at an opposite corner, and ground (null potential) to the remaining one, we observe the current-induced motion of the skyrmion towards the corner with the applied positive potential. We observe current-induced motion starting at 5u2009xD7u200910u2009Au2009m in a direction opposite to the direction of the current flow, which is consistent with previous work but at a current density that is about four orders of magnitude smaller. This drastically reduced threshold current density allows for ultra-low power operation. As the current density is increased, the effects of the torques become more and more dominating until increasing the current density further eventually results in pushing the skyrmion into one of the corners, where the current density is highest (see_), and the skyrmion annihilates. Here we choose a current density in the range of 10u2009Au2009m, which is sufficient for reliable and quick operation while not resulting in any skyrmion annihilation events.\" - \"Since the current density increases monotonically towards the tip of the triangle element, we calculate the current density_value of the device at the half-width of the triangle at the corner from which the current originates. While we have a geometrically symmetric system, the resistance between the pairs of contacts/corners slightly differs due to fabrication irregularities. Hence, the same voltage potential leads to different current density values, which, in addition to possible pinning sites, influences the skyrmion occurrence. A deviation from the ideal equilateral triangle does not impede operation, since the thermally activated diffusive dynamics allow the skyrmions to explore the full state space. Such irregularities are accounted for by default in the training of the linear read-out (see detailed discussion in the_). Hence, we demonstrate that our RC method automatically compensates for sample/setup imperfections. A detailed discussion on how thermal effects can leverage device imperfections and thereby drastically increase the robustness and reliability of operation can be found in the_.\" - Evaluation and linear regression - \"The general approach is to manipulate the skyrmion dynamics via the applied electric potentials and measure the probability for the skyrmion to be present within different regions of the sample. For now, the read-out is performed optically by imaging and tracking the skyrmions. For a scaled-down device, this could, for instance, be technically done using average tunnel magnetoresistance (TMR), as the TMR depends on the presence of a skyrmion in the relevant region. To mimic read-out via MTJs, we employ four circular read-out regions with a radius of 2,2u2009xB5m within the confinement arranged center-symmetrically as indicated in Fig._. The effects of the skyrmion size on the read-out are discussed in detail in the_. The images in Fig._ show only exemplarily the displacement of a single skyrmion from the ground state in Fig._. Note, that we have studied multiple devices that all show qualitatively the same behavior. The results shown below stem from a device with a 40u2009xB5m edge length.\" - The local skyrmion occurrence probabilities within the four circular regions are processed externally via linear read-out, i.e., the output is given by the weighted sum of the probabilities plus an offset. Thereby, the same device can perform a multitude of different operations depending on the weights. - \"The skyrmion occurrence is measured using Kerr-microscopy for 13,000 frames at 16 frames per second. To obtain a series of local occurrence probabilities for the four regions, we average the skyrmion occurrence over 62.5u2009s time intervals. For each input combination, we use the first 4 of the resulting 13 sets of local occurrences probabilities to optimize (train) the weights for linear read-out using the Scikit-learn software package. The Output Q is then given by\" - \"$$Q={W}_{{{{{{rm{left}}}}}}}{P}_{{{{{{rm{left}}}}}}}+{W}_{{{{{{rm{right}}}}}}}{P}_{{{{{{ rm{right}}}}}}}+{W}_{{{{{{rm{top}}}}}}}{P}_{{{{{{rm{top}}}}}}}+{W}_{{{{{{ rm{middle}}}}}}}{P}_{{{{{{rm{middle}}}}}}}+{W}_{{{{{{rm{intercept}}}}}}},$$ (1) \" - \"where ({W}_{{{{{{rm{region}}}}}}}) is the weight of the probability ({P}_{{{{{{rm{region}}}}}}}) for the skyrmion to be in the specific circular region and ({W}_{{{{{{rm{intercept}}}}}}}) is an offset. The probability ({P}_{{{{{{rm{region}}}}}}}) itself depends on the input patterns of the device ([0 0], [0 1], [1 0], and [1 1]) (see Fig._).\" - \\'Fig. 3: Logic operations.Outputs of the linear read-out optimized for different Boolean operations. For each input combination ([0 0], [0 1], [1 0], [1 1]) the output Q of the linear read-out is shown for 13 sets of local skyrmion occurrence probabilities. The light blue and black parts of the curves indicate the sets used for training and testing, respectively. The dashed horizontal line indicates a possible threshold for perceptron read-out.\\' - \"Figure_ shows the output of the linear read-out, trained for the Boolean operations AND, NAND, OR, NOR, XOR, and XNOR. They are performed using the left and right bottom contacts as inputs, 0 and 1, respectively, while constantly keeping the top contact at 0. The training set is indicated as the light blue part of the curves, whereas the test set is black. For all Boolean operations, we observe values close to the expected output and a good separation even on the test set. The corresponding optimized weights can be found in the_ (Supplementary Table_ ). The dashed lines show possible thresholds for perceptron read-out, i.e., values above and below the threshold are assigned to 1 and 0, respectively. In particular, we have demonstrated NAND and NOR functionality, each representing a functionally complete set of logical connectives, and the non-separable XOR functionality. The latter demonstrates that this system with just one confined skyrmion is already sufficiently complex to perform nonlinearly separable tasks, which is impossible, for instance, using a conventional single-layer perceptron read-out alone.\" - The signal-to-noise ratio (({{{{{{mathrm{SNR}}}}}}})) of the read-out is defined as - \"$${{{{{bf{{{SNR}}}}}}}},:=,frac{leftlangle {{{{{boldsymbol{T}}}}}} rightrangle -leftlangle {{{{{boldsymbol{F}}}}}}rightrangle }{{{{{{{ boldsymbol{sigma }}}}}}}_{{{{{{boldsymbol{T}}}}}}}+{{{{{{boldsymbol{sigma }}}}}}}_{{{{{{boldsymbol{F}}}}}}}},$$ (2) \" - \"where T and F are the sub-sets of linear read-out outputs ({{{{{rm{Q}}}}}} ), for which the corresponding Boolean operation applied to the device input patterns give True or False, respectively. Angled brackets indicate the mean value and ({{{upsigma }}}_{{{{{{rm{T}}}}}}}) (({{{upsigma }}}_{{{{{{rm{F}}}}}}} )) is the standard deviation of the T (F) subset. Averaged over the six different Boolean operations, the data in Fig._ exhibits ({{{{{rm{SNR}}}}}}; > ;5).\" - \"The ({{{{{rm{SNR}}}}}}) decreases as the time interval used to obtain the local skyrmion occurrence probabilities is decreased. For example, if the time interval is divided by two while the ratio between the number of sets used for training and testing is kept constant, the ({{{{{rm{SNR}}}}}}) decreases to ({{{{{rm{SNR}}}}}}approx 4). The corresponding signal of the logic operations can be found in supplementary Fig._. Note here that the timescale of skyrmion diffusion is exponentially dependent on the sample temperature, which is chosen to create diffusion compatible with the sampling rate of the Kerr microscope. A read-out via magnetic tunnel junctions would allow overcoming this limitation. Additionally, the diffusion is faster for smaller devices, as the characteristic timescale of directed diffusive motion scales with the square of the corresponding length scale. Thus, a nanometer scale device would allow for very fast measurements, as discussed in the_.\" - \"By using_additionally the top contact of the device as a variable input, we can also realize three-input composite logic operations as demonstrated in supplementary Fig._. The average SNR of the three-input operations_with SNR >3 is_still slightly lower than for the two-input operations but still demonstrates reliable operation.\"- - While the presented Brownian RC already suffices to realize nonlinearly separable operations for a single confined skyrmion, we emphasize that the concept can easily be generalized by exploiting more input combinations and by increasing the number of skyrmions in the geometry. With higher numbers of skyrmions, the system exhibits more complex dynamics like (in-)commensurable ground states and thus enhances the accessible states for the RC. Moreover, the complexity can be further enhanced by linking multiple confined geometries, which could provide an ultra-low-energy alternative to neuromorphic computing based on arrays of nanoscale spintronic oscillators. Another advantage is the possibility of the combination of the all-magnetic linear read-out with in-memory computing, for example, magnetic random-access memory. Furthermore, the scalability of this concept to nanoscale dimensions reduces the displacement distances, which would give rise to latency in the nanosecond regime. - Finally, we want to acknowledge that after completion of our work, we became aware of a related work, which also uses skyrmions for non-conventional computing. - This conceptually new reservoir computing idea based on Brownian dynamics of magnetic skyrmions in confined geometries bypasses challenges for existing theoretical proposals for skyrmion RC. Moreover, leveraging the stochastic motion of the skyrmion allows operation at current densities several orders of magnitude smaller than existing spintronic reservoir computation concepts. The present demonstration is based on a single confined skyrmion, which is already sufficient to realize non-separable operations and the universal set of Boolean functions. Generalizing this concept to multiple confined skyrmions and scaling to nanoscale dimensions allow for a highly promising path to ultra-low energy neuromorphic computing.- - Sample parameters - The thin film layer stack used was sputtered by a Singulus Rotaris magnetron sputtering system, consisting of Ta(5)/Co20Fe60B20(0.95)/Ta(0.09)/MgO(2)/Ta(5) with the thickness of the layers in nanometers in parentheses. Subscripted numbers are the relative atomic concentration of the respective element in percentage. The sample is specifically tailored for low pinning and skyrmions above room temperature, which exhibit thermal diffusion. Even though this Cobalt-Iron-Boron-based thin film inherits a very flat energy landscape and thus low pinning, which also enables thermal diffusion of skyrmions, pinning can influence the position of the skyrmion in relation to the corners. - \"The structures were patterned by electron beam lithography (EBL) using a Raith Electron Beam Pioneer system and then etched by Argon ion etching using an IonSys Model 500 ion beam etching system. 15u2009xB5m long wires in 120xB0 relation to each other at the corners were used in the layout of the triangle to ensure a better electrical connection with the gold pads. The width of the wires is between 2 and 5u2009xB5m, the skyrmions used in this work do not enter the wire and stay in the triangle due to the skyrmion-edge repulsion. Different device sizes ranging from 22 to 40u2009xB5m edge length were tested, but for the presented results, a device size of 40u2009xB5m was used. For the electrical contacts, a lift-off technique was used after EBL was done for the layout of the pads. The contact pads consist of 5u2009nm of chromium and 60u2009nm of gold on top and have a base size of 250u2009xB5mxB2u2009xD7u2009250u2009 xB5mxB2. Contacting is established using an aluminum wire which is bonded from the pads to the samples homemade holder.\" - Measurement setup - \"The sample itself is placed on a QC-17-1.0-2.5MS Peltier element to achieve the necessary temperature stability at around 330u2009K at ambient air, measured by a Pt100 resistive heat sensor, to realize the skyrmion phase and the size of skyrmions appropriate for the operation. The achievable temperature range was 285u2013360u2009K with temperature stability of 0.3u2009K. The size of the skyrmions and their thermal diffusion directly depend on the temperature. Albeit necessary for the skyrmions in our stack, the increased temperature also leads to increased diffusion, which thus can lead to a necessity of higher current densities to keep a skyrmion in one corner. If the thermal energy is too high, the skyrmion can jump/move towards the middle or other pinning sites in the device, which can be outside of the measured circles or, in the extreme case, annihilate.\" - \"We observe the magnetic structures with a commercial Evico GmbH magneto-optical Kerr effect (MOKE) microscope with a CCD camera connected to a PC. The magnetic out-of-plane field is supplied by a self-built electromagnetic coil, while the in-plane magnetic field pulse is supplied by a rotatable electromagnetic coil from the microscopes manufacturer. Images and videos are taken using the polar magneto-optical Kerr effect, recorded by a Hamamatsu Digital CCD Camera (C8484-03G02) with a CCD spatial resolution of 1344u2009xD7u20091024 pixels. The videos are recorded with a frame rate of 16 frames per second at an exposure time of 62.5u2009ms. To achieve this frame rate, we use a 2u2009xD7u2009 2 binning (four physical pixels are averaged to one virtual pixel in the image), resulting in a resolution of 672u2009xD7u2009512 pixels with a field of view of 125u2009xD7u200995u2009xB5m. Differential images between skyrmion hosting states_and saturation states were used to enhance the contrast. This leads to black-and-white subtraction errors at the edges of the device in the images (owing to incomplete overlapping), which causes an operation error when the sample/structure moves under the microscope due to thermal drift or mechanical strain. We used a stable thermal equilibrated state and increased mechanical stiffness to reduce the drift. Additionally, a script for repositioning the device in the pictures was used to further reduce the thermal and mechanical drift which occurs during recording, thus ensuring more correct tracking.\" - Skyrmion imaging and induced motion - \"Skyrmions are nucleated by setting an out-of-plane (OOP) field in the u03BC T regime and applying an in-plane pulse field of 35u2009mT, by switching off the field after it saturated the sample for a second. The amount of skyrmions nucleated in the structure depends on the set OOP field, since the radius of the skyrmions can be influenced by the OOP field and the number of skyrmions the geometrical structure can harbor. By increasing the OOP field, the size of the skyrmions is reduced until skyrmions start annihilating and only one single skyrmion is left. Decreasing the OOP field again to a fixed value leads to a reproducible skyrmion size. The skyrmion moves around the center of the device due to thermal motion but is mostly restricted to staying in the center due to the edge repulsion of the device. For a scaled-down version of our device, one could achieve skyrmion nucleation due to spin-orbit torques by current flowing through magnetic tunnel junctions, which are also suggested for read-out, thus performing a double purpose.\" - The electric potentials are applied by two independent voltage sources (Keithley 2400 SourceMeter), which are connected to the ground through a custom-made breakout box. The latter prevents the device from breaking due to unintentional current flow or induction of current in the wires/cables due to the magnetic fields. The breakout box allows to safely apply the voltage potential to the sample. - \"Depending on the structure, the potentials for the current-induced skyrmion motion range from 2 up to 5.5u2009mV, which corresponds to a few u03BCA of electrical current. Considering the geometry, the current densities used at the half-width of the triangle range from 2u2009xD7u200910u2009Au2009m to 3u2009xD7u2009 10u2009Au2009m. Although the geometry is symmetrical, the resistance between pairs of the corners of the triangle is not_exactly the same and varies due to possible film inhomogeneities, the connections of the gold pads, and the varying bonding connection quality of the attached wires. This is compensated by applying different weights in the linear regression, which also takes possible pinning sites in the triangle into account. With increasing current density, the temperature increases, in particular at the tips of the triangle element. The increased temperature enhances the motion of the skyrmion and its likelihood of annihilating, if the current density, and thus the temperature, becomes too high.\" - Image analysis - \"For the analysis of the videos, the skyrmion in the device was tracked using the trackpy package for Python and its center position for every frame was compared to the overlaying mask of the four center-symmetric circles on the device. The number of frames, in which the skyrmion resides in a certain circular region are summed up and put into relation to the overall number of frames in the video, resulting in the probability of the skyrmion being in a certain area while a fixed current is applied. In the_, we employ a more elaborate scheme to calculate the occurrence probabilities taking into account skyrmion size effects; we demonstrate that the consequent changes in the occurrence probabilities have little effect on the operation of the device. Figure_ in the supplementary is the heatmap of the probability of a position for a single skyrmion in the device. One can see that the likelihood at certain positions is increased, indicating the existence of pinning sites, on which it is energetically favorable for the skyrmion to stay. Thermal activation leads to motion around said sites. This thermal activation can be increased using a higher temperature, until the skyrmion becomes either too small for our experimental setups resolution or annihilates.\" - \"As an outlook, using the three different input types with three physical inputs, 27 possible combinations would be possible, although some combinations have either no function (e.g., [- - -]) or are redundant due to gauge invariance of the electric potential (e.g., state combination (a) - - 0 and (b) 0 0 +, resulting in the skyrmion to move towards the corner with (a) 0 or (b) +). When more skyrmions are present in confinement, the systems responses to the inputs become more complex. This is expected to be most prominent for skyrmion numbers incommensurate with the geometry. Additionally, combining multiple devices would lead to even more states, thus leading to even higher capacity.\"- - \\' \\' - The data supporting the findings of this work are available from the corresponding authors upon reasonable request. - \\' \\'- - \\' \\' - The computer codes used for data analysis are available upon reasonable request from the corresponding author. - \\' \\'- - \"Fert, A., Reyren, N. & Cros, V. Magnetic skyrmions: advances in physics and potential applications. Nat. Rev. Mater. 2, 17031 (2017)._ _ _ _ Bogdanov, A. N. & Panagopoulos, C. Physical foundations and basic properties of magnetic skyrmions. Nat. Rev. Phys. 2, 492u2013498 (2020)._ _ Dohi, T., Reeve, R. M. & KlxE4ui, M. Thin film skyrmionics. Annu. Rev. Condens. Matter Phys. 13, 73u201395 (2022).Everschor-Sitte, K., Masell, J., Reeve, R. M. & KlxE4ui, M. Perspective: magnetic skyrmionsu2014 Overview of recent progress in an active research field. J. Appl. Phys. 124, 240901 (2018)._ _ _ Jiang, W. et al. Skyrmions in magnetic multilayers. Phys. Rep. 704, 1u201349 (2017)._ _ _ _ _ Finocchio, G., BxFCttner, F., Tomasello, R., Carpentieri, M. & KlxE4ui, M. Magnetic skyrmions: from fundamental to applications. J. Phys. D Appl. Phys. 49, 423001 (2016)._ _ _ Wiesendanger, R. Nanoscale magnetic skyrmions in metallic films and multilayers: a new twist for spintronics. Nat. Rev. Mater. 1, 16044 (2016)._ _ _ _ Tokura, Y. & Kanazawa, N. Magnetic skyrmion materials. Chem. Rev. 121, 2857u20132897 (2021)._ _ _ _ Jonietz, F. et al. Spin transfer torques in MnSi at ultralow current densities. Science 330, 1648u20131651 (2010)._ _ _ _ _ Jiang, W. et al. Blowing magnetic skyrmion bubbles. Science 349, 283u2013286 (2015)._ _ _ _ _ Woo, S. et al. Observation of room-temperature magnetic skyrmions and their current-driven dynamics in ultrathin metallic ferromagnets. Nat. Mater. 15, 501u2013506 (2016)._ _ _ _ _ Litzius, K. et al. Skyrmion Hall effect revealed by direct time-resolved X-ray microscopy. Nat. Phys. 13, 170u2013175 (2017)._ _ _ Dohi, T., DuttaGupta, S., Fukami, S. & Ohno, H. Formation and current-induced motion of synthetic antiferromagnetic skyrmion bubbles. Nat. Commun. 10, 5153 (2019)._ _ _ _ _ Litzius, K. et al. The role of temperature and drive current in skyrmion dynamics. Nat. Electron. 3, 30u201336 (2020)._ _ _ Prychynenko, D. et al. Magnetic skyrmion as a nonlinear resistive element: a potential building block for reservoir computing. Phys. Rev. Appl. 9, 014034 (2018)._ _ _ _ Pinna, D. et al. Skyrmion gas manipulation for probabilistic computing. Phys. Rev. Appl. 9, 064018 (2018)._ _ _ _ ZxE1 zvorka, J. et al. Thermal skyrmion diffusion used in a reshuffler device. Nat. Nanotechnol. 14, 658u2013661 (2019)._ _ _ _ Jibiki, Y. et al. Skyrmion Brownian circuit implemented in continuous ferromagnetic thin film. Appl. Phys. Lett. 117, 082402 (2020)._ _ _ _ Song, K. M. et al. Skyrmion-based artificial synapses for neuromorphic computing. Nat. Electron. 3, 148u2013155 (2020)._ _ Brems, M. A., KlxE4ui, M. & Virnau, P. Circuits and excitations to enable Brownian token-based computing with skyrmions. Appl. Phys. Lett. 119, 132405 (2021)._ _ _ _ Fert, A., Cros, V. & Sampaio, J. Skyrmions on the track. Nat. Nanotechnol. 8, 152u2013156 (2013)._ _ _ _ _ Nagaosa, N. & Tokura, Y. Topological properties and dynamics of magnetic skyrmions. Nat. Nanotechnol. 8, 899u2013 911 (2013)._ _ _ _ _ Pepper, R. A. et al. Skyrmion states in thin confined polygonal nanostructures. J. Appl. Phys. 123, 093903 (2018)._ _ _ Song, C. et al. Commensurability between element symmetry and the number of skyrmions governing skyrmion diffusion in confined geometries. Adv. Funct. Mater. 31, 2010739 (2021)._ _ _ Li, S. et al. Magnetic skyrmions for unconventional computing. Mater. Horiz. 8, 854u2013868 (2021)._ _ _ _ Grollier, J., Querlioz, D. & Stiles, M. D. Spintronic nanodevices for bioinspired computing. Proc. IEEE 104, 2024u20132039 (2016)._ _ Tanaka, G. et al. Recent advances in physical reservoir computing: a review. Neural Netw. 115, 100u2013123 (2019)._ _ _ Marinella, M. J. & Agarwal, S. Efficient reservoir computing with memristors. Nat. Electron. 2, 437u2013438 (2019)._ _ Pinna, D., Bourianoff, G. & Everschor-Sitte, K. Reservoir computing with random skyrmion textures. Phys. Rev. Appl. 14, 054020 (2020)._ _ _ _ Torrejon, J. et al. Neuromorphic computing with nanoscale spintronic oscillators. Nature 547, 428u2013431 (2017)._ _ _ _ _ Tomasello, R. et al. Electrical detection of single magnetic skyrmion at room temperature. AIP Adv. 7, 056022 (2017)._ _ _ Pedregosa, F. et al. Scikit-learn: machine learning in Python. J. Mach. Learn. Res. 12, 2825u20132830 (2011)._ _ _ Einstein, A. xDC ber die von der molekularkinetischen Theorie der WxE4rme geforderte Bewegung von in ruhenden FlxFCssigkeiten suspendierten Teilchen. Ann. Phys. 322, 549u2013 560 (1905)._ _ _ Romera, M. et al. Vowel recognition with four coupled spin-torque nano-oscillators. Nature 563, 230u2013234 (2018)._ _ _ _ _ Zahedinejad, M. et al. Memristive control of mutual spin Hall nano-oscillator synchronization for neuromorphic computing. Nat. Mater. 21, 81u201387 (2022)._ _ _ _ _ Houshang, A. et al. Spin-wave-beam driven synchronization of nanocontact spin-torque oscillators. Nat. Nanotechnol. 11, 280u2013286 (2016)._ _ _ _ _ Jung, S. et al. A crossbar array of magnetoresistive memory devices for in-memory computing. Nature 601, 211u2013 216 (2022)._ _ _ _ _ Yokouchi, T. et al. Pattern recognition with neuromorphic computing using magnetic fieldu2013 induced dynamics of skyrmions. Sci. Adv. 8, eabq5652 (2022)._ _ _ _ Yang, S. et al. Electrical generation and deletion of magnetic skyrmionu2010bubbles via vertical current injection. Adv. Mater. 33, 2104406 (2021)._ _ _ BxFCttner, F. et al. Field-free deterministic ultrafast creation of magnetic skyrmions by spinu2013orbit torques. Nat. Nanotechnol. 12, 1040u20131044 (2017)._ _ _ _ Allan, D. B., Caswell, T., Keim, N. C., van der Wel, C. M. & Verweij, R. W. Trackpy: fast, flexible particle-tracking toolkit u2014 trackpy 0.5.0 documentation. soft-matter/trackpy: Trackpy v0.5.0 (2021).Crocker, J. C. & Grier, D. G. Methods of digital video microscopy for colloidal studies. J. Colloid Interface Sci. 179, 298u2013310 (1996)._ _ _ _ Gruber, R. et al. Skyrmion pinning energetics in thin film systems. Nat. Commun. 13, 3144 (2022)._ _ _ _ _ _ \"- - \"The work in Mainz (K.R., M.A.B., G.B., J.R., F.K., and M.K.) was supported by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) projects 403502522 (SPP 2137 Skyrmionics), 49741853, and 268565370 (SFB TRR173 projects A01 and B02) as well as TopDyn and the Zeiss foundation through the Center for Emergent Algorithmic Intelligence. J.H.M. acknowledges funding the Shell-NWO/FOM-initiative u201CComputational sciences for energy researchu201D of Shell and Chemical Sciences, Earth and Life Sciences, Physical Sciences, FOM, and STW. The work is a highly interactive collaboration supported by the Horizon 2020 Framework Program of the European Commission under FET-Open grant agreement no. 863155 (s-Nebula) and ERC-2019-SyG no. 856538 (3D MAGiC) and the Horizon Europe project no. 101070290 (NIMFEIA), which M.K. and J.H.M. acknowledge. M.A.B. thanks the DFG TRR146 for partial financial support. We would also like to acknowledge helpful discussions with Peter Virnau. T.D. gratefully acknowledges financial support from the Canon Foundation in Europe.\"- - Open Access funding enabled and organized by Projekt DEAL.- - Authors and Affiliations - \"Institut fxFCr Physik, Johannes Gutenberg-UniversitxE4t Mainz, Staudingerweg 7, 55128, Mainz, GermanyKlaus Raab,_Maarten A. Brems,_Grischa Beneke,_Takaaki Dohi,_Jan RothxF6rl,_Fabian Kammerbauer_&_Mathias KlxE4uiRadboud University, Institute for Molecules and Materials, Heyendaalseweg 135, 6525 AJ, Nijmegen, The NetherlandsJohan H. MentinkGraduate School of Excellence Materials Science in Mainz, Staudingerweg 9, 55128, Mainz, GermanyMathias KlxE4ui\" - \"AuthorsKlaus RaabYou can also search for this author in _Maarten A. BremsYou can also search for this author in _Grischa BenekeYou can also search for this author in _Takaaki DohiYou can also search for this author in _Jan RothxF6rlYou can also search for this author in _Fabian KammerbauerYou can also search for this author in _Johan H. MentinkYou can also search for this author in _Mathias KlxE4uiYou can also search for this author in _\" - Contributions - M. K. and J.H.M. devised and supervised the study. K.R. designed and fabricated the sample. K.R. and G.B. carried out the measurements with the support of T.D. G.B. evaluated the video data. M.A.B. and J.R. analyzed the evaluated video data and performed training on the linear read-out. The paper was written by K.R., M.A.B., M.K., and J.H.M. All authors commented on the results and contributed to writing the paper. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution-Share Alike 3.0 Unported license ().\\'- - \"The chemistry of all first-row transition metals is very sensitive to their environment. In the presence of water, each metal ion forms hydrated ions which undergo pH and concentration-dependent chemistry that is dictated by the presence of metabolites, proteins, and other biological components (Fig._). It is important to recognize that redox active metal ions do not exist as free ions in cells. As a result, these metals undergo speciation chemistry governed by the metal ions oxidation state, the local pH, the ionic strength, and the stability of metal complexes with biological molecules. Depending on the specific conditions, several of the metal ions form multinuclear species in aqueous solution, and as such many activities and functions will not be linear but very sensitive to concentrations and association with biomolecules. Appreciation of classical speciation chemistry increases when it becomes obvious that the identification of components in the system demystifies poorly understood processes in biology.\"- - \"Typically, the first-row transition metal ions are bound to proteins in the cellular system. More than 30% of the proteins in the genome bind metal ions, and some of these bind the metal ions with such_a high affinity that the metal ion is difficult to separate from the protein. Protein complexation to a metal ion prevents the redox formation of undesired reactive oxygen species (ROS) by the metal ion. This, however, requires that the bound metal ions are utilized by transfer of the metal ion to a target protein. Processes involving metal transfer reactions can be difficult to study because the reacting metal peptide complex is very likely to have very similar spectroscopic signatures to the product metal peptide complex. Importantly, changes in affinities of the metal ions in different oxidation states are important in facilitating cell uptake and transfer reactions. Manganese, iron, cobalt, and copper are the four essential elements that exhibit rich redox chemistry under physiological conditions. These metal ions are associated with multiple enzymes, and are involved in many cellular redox processes.\"- - \"Iron is essential for human life as hemoglobin and as an intricate part of respiration enzymes, particularly those containing hemes. The chemistry of both Fe(II) and Fe(III) is important for binding and function of many other redox proteins, including non-heme proteins. Ligand coordination to iron is important to allow the redox cycling without the involvement of Fenton chemistry and generation of uncontrolled ROS at the cellular level. Similarly, copper is involved in function of many redox enzymes as well. Cu(I), Cu(II), and Cu(III) are bound to many proteins and cellular components with high affinity. Although manganese and cobalt have less prominent roles than iron and copper, they too are involved in specific processes essential for life. For example, manganese is a cofactor for the enzyme superoxide dismutase responsible for scavenging ROS. Cobalt is bound to a heme in vitamin B12 which is the only vitamin to contain a metal ion. Studies of these systems continue to be important. An increased consideration of the metal coordination chemistry will benefit the insights into these systems and uncover new details about their modes of action. Studies with zinc, which is the fifth essential element and the only non-redox active cation, are also important because zinc has either a structural or functional role in more than 300 different proteins. One role of zinc is in maintaining the folding of the DNA-binding domains of eukaryotic transcription factors including zinc finger transcription factors. Recent investigations have demonstrated numerous previously unknown activities including hormone-like activities which highlights the notion that even non-redox active ions can have crucial roles in biological systems. The large number of clinical trials involving zinc shown in Table_ demonstrates the interest in this metal. Importantly, future investigations of these metals in biological systems should be done in the context of metal coordination chemistry within and outside the range of normal homeostasis.\" - Table 1 Clinical trials involving compounds containing first-row transition metals registered with the National Institutes of Health (NIH).- - Three additional elements have some reported beneficial effects include chromium, vanadium, and nickel. Chromium, considered to be an essential metal for some time, is probably the most controversial element in the periodic table. Careful speciation studies have shown that Cr(III) is not as inert as previously believed and can convert to Cr(V) and Cr(VI) ions which are highly toxic. Like chromium, both beneficial and toxic effects have been reported for vanadium and nickel. For many years, vanadium compounds were developed as insulin enhancing agents. Current studies focus on using vanadium-based compounds as anticancer agents which include flavonoid vanadium complexes and coordination complexes for immunotherapy applications. Nickel appears to be an important part of the microflora in the human gut where it is a cofactor for the enzyme urease. On the other hand, Ni-containing alloys often used in jewelry are known to cause an allergenic response in about 30% of women. This application is countered by to the use of Ni-containing alloy implants added to mend broken bones and represents an example of beneficial and toxic effects of this metal.- - \"Titanium is a first-row transition metal ion that has no known biological function despite being readily transported in the human blood where it readily binds to proteins such as human serum albumin. Titanium metal is a common component of alloys ranging from dental implants to orthopedic prosthetics and many clinical trials have been completed and others are underway. However, even for this supposedly non-toxic element, there are some reports that the metal alloys are not completely stable; some cationic forms of titanium leach from the metal surfaces. The biological and potentially toxic response of each system should be carefully considered, particularly since such a large number of clinical trials are ongoing with this element (Table_). Scandium is non-essential to human health and has no known biological function in the biosphere. Its low abundance has precluded many studies until recently. Potential applications of scandium are currently being investigated in two clinical trials (Table_).\"- - Essential nutrients for human health include the 13 known vitamins A, C, D, E, K, and the eight B vitamins (thiamine (B1), riboflavin (B2), niacin (B3), pantothenic acid (B5), pyroxidine (B6), biotin (B7), folate (B9), and cobalamin (B12). Unlike minerals, vitamins in biological systems can be metabolized to carbon dioxide and water while metal ions must be removed by excretion. The bioprocessing of these metal ions and their recycling remain a complex matter, which require an evaluation of their speciation chemistry. Unfortunately, studies integrating metal speciation with pharmacokinetic and pharmacodynamic properties of metal ions are also quite costly. Nonetheless, metabolism provides additional avenues for formation of active biological substances. That is, if the ligand bound to a metal is changed during bioprocessing, a new complex with a different coordination and speciation chemistry is formed which may also have beneficial biological activity and significantly prolong the effects of the originally administered therapeutic.- - \"Finally, we wish to point out the utility of these elements as therapeutic and diagnostic agents. To document their use, we have tabulated the number of completed and active clinical trials that have been carried out with these ten elements (Table_). Since five of the first-row transition metals are essential elements, many of their applications are related to bringing the concentration of these elements into the normal concentration range so it is neither too low or too high, thus causing disease. Accordingly, a large number of clinical trials are associated with iron, zinc, copper, cobalt, and manganese. Many applications of titanium and nickel relate to their physical properties and concern their use as alloys in implants. Development of therapeutic and diagnostic agents continues because of the increasing need to monitor and cure diseases. However, technical advances change the requirements for the agents, as illustrated by the recent report where the compounds reactivity was considered an advantage because injections were made directly into the tumor, and the reactivity of the compound affects cancerous tissue. Multiple uses of the first-row transition metal ions in medicine are already in place in the clinic and other promising uses are being developed with the potential of improving human health.\"- - \"Sigel, H. & Sigel, A. The bio-relevant metals of the periodic table of the elements. Z. Naturforsch. 74, 461u2013471 (2019)._ _ _ Van Cleave, C. & Crans, D. C. The first row transition metals in the periodic table of medicine. Inorganics 7, 111 (2019)._ _ Chambers, A. et al. An exposure-response curve for copper excess and deficiency. J. Tox. Environ. Health, Part B 13, 546u2013578 (2010)._ _ _ Hawkins, S. What is a u201CHeavy Metalu201D? J. Chem. Ed. 74, 1374 (1997)._ _ Baes, C. F. & Mesmer, R. S. The Hydrolysis of Cations. (John Wiley & Sons, New York, 1976).Marcio, L. et al. The peculiar behavior of Picha in the formation of metallacrown complexes with Cu(II), Ni(II) and Zn(II), in aqueous solution. Dalton Trans. 44, 3237u20133250 (2015)._ _ Rae, T. D., Schmidt, P. J., Pufahl, R. A., Culotta, V. C. & OHalloran, T. V. Undetectable intracellular free copper: the requirement of a copper chaperone for superoxide dismutase. Science 284, 805u2013808 (1999)._ _ _ Kiss, Y., Enyeda, E. A. & Jakusch, T. Development of the application of speciation in chemistry. Coor. Chem. Rev. 352, 401u2013 423 (2017)._ _ _ Levina, A., Crans, D. C. & Lay, P. A. Speciation of metal drugs, supplements and toxins in media and bodily fluids controls in vitro activities Coor. Chem. Rev. 352, 473u2013498 (2017)._ _ Mounicou, S., Szpunar, J. & Lobinski, R. Metallomics: the concept and methodology. Chem. Soc. Rev. 38, 1119u20131138 (2009)._ _ _ Beuning, C., Mestre-VoegtlxE9, B., Faller, P., Hureau, C. & Crans, D. C. Measurement of inter-peptidic Cu(II) exchange rate constants by static fluorescence quenching of tryptophan. Inorg. Chem. 57, 4791u2013 4794 (2018)._ _ _ Valko, M., Morris, H. & Cronin, M. T. D. Metals, toxicity and oxidative stress. Curr. Med. Chem. 12, 1161u2013 1208 (2005)._ _ _ Casini, A., Vessieres, A. & Meier-Menches, S. M. (Eds.) Metal-based Anticancer Agents (The Royal Sociey of Chemistry, 2019).Sigel, A., Sigel, H., Freisinger, E. & Sigel, R. K. O. Metallo-drugs: development and action of anticancer agents. Met. Ions Life Sci. 18, (2018).Selman, M. et al. Multi-modal potentiation of oncolytic virotherapy by vanadium compounds. Mol. Ther. 26, 56u201369 (2018)._ _ _ Barry, N. P. E. & Sadler, P. J. Exploration of the medical periodic table: toward new targets. Chem. Commun. 49, 5106u20135131 (2013)._ _ _ Levina, A. et al. A short-lived but highly cytotoxic vanadium(V) complex as a promising drug candidate for brain cancer treatment by intratumoral injections. Angew. Chem. in press (2020).\"- - D.C.C. and K.K. thank Drs. Christopher Roberts and Deborah A. Roess and Miss Kaitlin Doucette for reading and commenting on this manuscript. We thank Cameron Van Cleave for assistance with preparation of Fig. 1.- - Authors and Affiliations - \"Department of Chemistry, 1301 Center Ave., Colorado State University, Fort Collins, CO, 80523, USADebbie C. Crans_&_Kateryna KostenkovaCell and Molecular Biology Program, 1301 Center Ave., Colorado State University, Fort Collins, CO, 80523, USADebbie C. Crans\" - \"AuthorsDebbie C. CransYou can also search for this author in _Kateryna KostenkovaYou can also search for this author in _\" - Contributions - D.C.C. initiated and supervised the project. K.K. carried out literature searches and together D.C.C. and K.K. fine-tuned ideas and finished writing the paper. K.K. also prepared the figure and the table. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution-Share Alike 3.0 Unported license, Author Hayden 120). - \"In Norway there are early finds of hemp and flax pollen from the inner Oslo Fjord area, about 350 BCu2013450 AD (considerably more hemp than flax pollen was found). Pollen and hemp seeds were found in a settlement in Hamar from around 400 AD. There are finds of hemp pollen from around 650 and 800 AD, from the inland of xC5seral in Vest-Agder and both hemp and flax seeds were found in the famous Viking burial mound, the Oseberg ship in Vestfold county from around 800 AD.\" - \"From Sweden in JxE4mtland, RxF6dxF6n parish, near Lake StorsjxF6n there are finds of hemp pollen, about 100u2013200 AD. The first flax seeds were found in HxE4lsingland, in Trogsta about 200 century AD. Another early find from HxE4lsingland of flax seeds, in a bog near a settlement in Forsa indicate that flax was cultivated and retted there about 300 AD. There are also finds of bundles of flax from KxE4rinsjxF6, in the province of Halland, from about 200u2013 300 AD. From the Viking Age, 800u20131050 AD there are more finds of hemp pollen and hemp seeds, for example from the area Lake MxE4laren, near Stockholm and in the area Lake StorsjxF6n, near RxF6dxF6n. In addition there are finds of pollen, seeds and fibres of hemp from the area of Lake Siljan in Dalarna. Both flax and hemp seeds were found in the area south of Lake Siljan, near Leksand.\" - \"The finds of hemp and flax pollen and seeds do not prove conclusively that textile production really took place. Only finds of retted fibres can do that. It is possible that the plants were grown for other purposes, i.e. the use of the seeds as food or oil. This applies both to hemp and flax. Unfortunately the preservation conditions in Scandinavia are such that plant fibre textiles normally do not survive well. There are some early remains of plant fibre textiles from Sweden from around 400u2013500 AD, from FullerxF6 in the province Uppland and from Augerum in Blekinge in the south. More textile remains are preserved from the Viking and Middle Ages, for example at the Viking site of Birka at Lake MxE4laren. There is also a find from Norway, in the Oseberg ship, from about 800 AD, consisting of a woven tapestry made of wool together with plant fibres, but the plant fibres were largely decomposed.\" - Despite the archeological finds showing that hemp and flax were both grown in the north of Scandinavia in the Viking and Early Middle Ages and despite the fact that hemp fibres can be almost as fine as flax fibres it has hitherto generally been believed that hemp was used primarily for coarse textiles i.e. ropes and sailcloth and that flax was the common textile plant for fine clothing and household textiles in Northern Scandinavia in this period. - \"This is reflected in the fact that throughout the literature, including but not restricted to the literature on Scandinavian Viking and Early Middle Ages, most fine plant fibre textile remains are referred to as flax without any mentioning of analytical tests. One of the few exceptions from this is Agnes Geijer, who in the 1930s investigated the textile finds from the Swedish Viking settlement Birka, near Lake MxE4laren. She introduced the term FH (Flax/Hemp) for all plant fibre material and called for a scientific method for proper identification u201CIt was the difficulty involved in firmly distinguishing between the two bast species that led me to employ the cryptic designation FH (Flax/Hemp) in the original classification. Such finds were made in about forty-five graves.u201D . To this day these fibres remain unidentified.\" - Part of the problem is that a systematic investigation of plant fibre textiles has been hampered by the difficulty of fibre identification. Fibres from animals (wool, fur and hair) and plant fibres in general can easily be distinguished because animal fibres have scales, but to distinguish between individual plant fibres is much more difficult. Samples have often been identified as flax purely on the basis of examinations with standard, white light, compound microscopy. This is insufficient to secure a proper identification. - \"The 10 textiles that we investigate here are all woven and made of plant fibres combined with wool. They have been carbon-14 dated to 780u20131420 AD. They are believed to be locally produced on the basis of the patterns and textile techniques. They all belonged to different churches and are among the oldest and finest preserved Scandinavian textiles found above ground. A considerable number of medieval textiles have been preserved in Swedish churches. The reason is that the Reformation in Sweden did not cause such immediate, drastic changes in the church dxE9cor as, in most other countries, for example Denmark. The old church textiles in Sweden were widely used even after the Reformation.\"- - \"All 5 pieces of the famous Swedish xD6verhogdal Viking wall-hanging were examined, see for a picture. For a picture of the Norwegain Lomen coverlet see and for a picture of the Marby hanging, . For an overview of all textiles, including fibre identification, see . The locations where the textiles were found are indicated on the map in . In the literature all these textiles are listed as being flax, without any proper references to the methods that have been used for the fibre identifications.\" - \\'Table 1 Overview of all the analysis results: The samples are listed in chronological order. Microscopy images can be found in the online \\' - \"Figure 2The xD6verhogdal Viking wall-hangings.The two top pieces Ia (Length: 164u2005cm (top edge, Width: 33, 4u201333, 5u2005cm) and Ib (Length: 195u2005 cm, Width: 33u201335u2005cm) are shown to be made of a mixture of hemp and flax. For the other pieces only flax were found (see ). Published with permission of Jamtli Museum, Sweden.\" - \"Figure 3The Lomen coverlet (Length: 130u2005cm, Width: 82u2005cm).Published with permission of the National Museum of Art, Architecture and Design, Oslo, Norway.\" - \"Figure 4The Marby hanging (Length: 81u201385u2005cm, Width: 16u201317, 5u2005 cm).Published with permission of The Swedish History Museum xA9 Gabriel Hildebrand/Statens Historiska Museum.\" - \"Our results show that 4 of the 10 investigated textile samples contained hemp fibres. They do not differ in quality from the other textiles. The Lomen coverlet was the only sample were only hemp fibres were found. Three other samples were found to contain a mixture of hemp and flax: The Marby hanging and the xD6 verhogdal wall-hangings Ia and Ib. In the other xD6verhogdal pieces only flax fibres were identified. This is an important result in its own right, since it provides an independent support for the theory that the pieces Ia and Ib are woven on the same warp. To the best of our knowledge the xD6verhogdal and Marby measurements provide the first proof that mixed plant fibres were used in the Scandinavian Viking and Early Medieval Textile production.\" - It is possible that mixed fibres may have been used in other textiles as well and that we just did not find them. In some cases the museums only provided threads from the warp or weft. It is possible that different material may have been used in the warp and the weft. We can also not know for sure if the plant fibre material used in the examined textiles is really locally produced. In principle it could be imported. A strontium isotope analysis could possible one day provide the answer to this.- - \\'Our results show that the role of hemp in the Scandinavian cultural history needs to be reconsidered. Hemp was not used solely for the production of coarse textiles such as ropes and sailcloth; it was also used in the production of fine household textiles in Scandinavian Viking and Middle Ages. The big question, from a cultural history and historical research management point of view is: Why use one material instead of another? When hemp and when flax?\\' - \\'Until the true distribution of the use of hemp and flax in Scandinavia and other places has been properly investigated through detailed fibre identification analysis, we cannot hope to come up with a conclusive answer to this, but we may speculate on the reasons:\\' - \"While hemp and flax can both grow well on most good soil suitable for crops, hemp can also, to some extend grow on soils with high pH and muddy soils not suitable for many other crops without the need for much soil preparation (i.e. plowing, burn-beating, rotation of crop or lie fallow). This type of soil is often found in mountain areas and in coniferous forest areas, in swamps and marshes and around lakes. The xD6verhogdal wall-hangings and Lomen coverlet were found in mountain areas and in march- and lake rich areas. The literature says that hemp cultivation was especially common in JxE4mtland during the 1700s and 1800s because the area has the characteristic soil needed. For optimal growth of flax the requirements are different. For example a nitrogen rich soil does not give a nice flax fibre. Flax grows well in a climate with uniform high humidity and rainfall where early summer drought does not occur. The coastal area of HxE4lsingland, a region that was known for flax cultivation from the Middle Ages has such a climate and also weeding was not so important here, due to good availability of suitable soil and favorable crop rotation.\" - \"The simple answer to the question u201Cwhen hemp and when flax?u201D could be that the preference for hemp or flax varied depending on the local growth conditions.\" - \"In later times flax became the most common textile plant for clothes and household textiles. It became so common that the word linen, became essentially synonymous with a flax textile. The origin of the word u201Clinenu201D is debated, but it seems that originally it simply meant a woven fabric of a plant fibre. During the end of the eighteenth century in the European linen trade, flax and hemp was often blended. In u201CThe workwoman\\'s guideu201D from 1840, in a chapter about different qualities of linen, it is written that u201CThe Suffolk hemp is considered the bestu201D. In another earlier book u201CThe English Housewifeu201D from 1683, the author Markham describes linen and its characteristics and writes that housewives must become proficient in the design of all types of linen fabric, whether it be of hemp or flax, he writes; \"For from these two only the largest cloth come and do so both in England and in other countries\".\" - Eventually flax and hemp lost their important position to cotton and nowadays the word linen is often used for a plain weave fabric of cotton and cloth made of flax sometimes mistakenly referred to as cotton. - \"To summarize: In this paper we have shown that hemp was used in Northern Scandinavia in the Viking and Early Middle Ages for the production of fine household textiles. Of the 10 textiles investigated, 4 were found to be made with hemp: The famous Swedish xD6verhogdal wall-hangings Ia and Ib, supporting the theory that the two pieces are woven on the same warp, the Marby hanging and the Norwegian Lomen coverlet. The three first textiles were found to be made with mixed fibres. To the best of our knowledge these measurements provide the first proof that mixed plant fibres were used in the Scandinavian Viking and Early Medieval Textile production. Our results show that the role of hemp in the Scandinavian cultural history needs to be reconsidered. More generally it demonstrates how the lack of proper fibre identification in textile research can lead to a distorted view of the relative importance of different textile plants.\"- - Hemp and flax are both bast fibres. Bast fibres (with the exception of wood bast) originate from the stem of a plant. Bast fibres are very similar in appearance and cannot be distinguished conclusively using standard white light, compound microscopy, as discussed in the previous section. DNA analysis of bast fibres is usually not possible. It is generally difficult to extract DNA even from modern bast fibres. Further, the removal of the fibres from the plant stem (retting) involves exposure to water, thus increasing the likelihood of extensive hydrolytic damage to the DNA. Fortunately there is one characteristic feature, which can be used for identification purposes. The cell walls in bast fibres contain the so-called fibrils. Fibrils are bundles of cellulose chains. In one of the cell wall layers these cellulose bundles rotate around the fibre. This is referred to as S-twist or Z twist of a fibre. Flax and nettle are S-twist and hemp is Z-twist (). - \\'Figure 5Fibrillar orientations of bast fibres.To the left: Z-twist, to the right, S-twist.\\' - For more than 60 years the so-called modified Herzog test has been available as a way of measuring the fibrillar orientation of fibres using polarized light microscopy. The modified Herzog test exploits the fact that the cellulose chains are bifringent. Unfortunately the Herzog test has not been widely used in the textile research community, except for a few cases, for example in reference. Part of the problem with the Herzog test is that it was based on empirical evidence and sometimes it does not work. Recently one of the authors of this paper (Holst) and co-worker (Haugan) published an analytical model for the Herzog test showing why it works, why it never gives a false result and why it is sometimes not possible to identify a fibrillar orientation with the test. - In the modified Herzog test the fibre sample is placed between crossed polars and rotated to extinction (sample turns black). A red plate compensator is then inserted. A color change, dependent on the fibrillar orientation, will then occur in the sample part that was at extinction before. A S-twist fibre will turn yellow and a Z-twist fibre blue when close to parallel to the analyzer. The exact color change and angle will vary depending on the thickness of the fibre, cell membrane etc.. - \"Samples of threads, about 1u20133u2005cm long were collected from each textile by conservators at the associated museums. Single fibres were extracted from each sample and mounted on microscope slides following the procedure described in reference. It is important that the mounting is done with great care. Only a few single fibre fragments, about 1u20133u2005mm is needed for identification, but the older textile material tends to be brittle and this makes the procedure more difficult. All experiments were done using an Olympus BX-51P compound microscope equipped with a full wave compensator of wavelength 530u2005nm and objectives of the type UiS2 series, Ach N. After mounting, the microscope slides were labeled solely with numbers so that it was possible for one of the authors (Holst) and two assistants, who did not prepare the fibres to perform identifications as an independent blind test. The fibre identifications were also performed by the person who prepared the fibres (Skoglund). No conflicting measurements were obtained. For each sample 2u20133 fibres were tested separately.\" - The fibrillar orientation of all textile samples could successfully be identified. The results are listed in . Note that strictly speaking we can only identify the hemp samples, since the samples with S-twist could in principle also be nettle. A complete overview with images of all experimental results can be found in the online for this paper. - \"Images of the modified Herzog test performed on the xD6verhogdal wall-hanging Ia can be found in .\" - \"Figure 6The modified Herzog Test performed on the xD6verhogdal Viking wall-hanging piece Ia.The two top pictures (a and b) show a Z-twist fibre (Hemp) and the two bottom pictures (c and d) show a S-twist fibre (Flax). The analyzer is oriented vertically.\"- - \"Hafsten, U. Jordbrukskulturens historie i Oslo och MjxF8strakten belyst ved pollenanalytiske undersxF8kelser. Viking. Norsk arkeologisk xE5rsbok. Oslo 70 (1958).Soltveg, E.-C. Makrofossilanalyse av prxF8ver fra hustomt I, II, III og V, Valum, Hamar k., Hedmark. Oppdragsrapport, Universitetet i Stavanger, Arkeologisk museum 1994, Arkeologisk museum i Stavanger. Stavanger (1994).Jessen, C. & Stylegar, F.-A. xD8degxE5rden Sosteli i xC5seral fra romertid til vikingtid. Viking. Norsk arkeologisk xE5rbok. Oslo 139 (2012).Nockert, M. & Possnert, G. Att datera textilier. Gidlunds fxF6rlag. SxF6dertxE4lje 10, 69u201389 (2002)._ Hafsten, U. Jordbrukskulturens historie i Oslo och MjxF8strakten belyst ved pollenanalytiske undersxF8kelser. Viking. Norsk arkeologisk xE5rsbok. Oslo 71 (1958).BrxFCgger, A. W. & Shetelig, H. Osebergfundet. Bind V. Kristiania (1927).Robertsson, A.-M. Vikingatida hampodling. JxE4mten. xD6 stersund 183u2013188 (1992).Nockert, M. The HxF6gom find and other Migration Period textiles and costumes in Scandinavia. University of UmexE5. Department of archaeology. RiksantikvariexE4mbetet. UmexE5 66, 67 (1991)._ Liedgren, L. JxE4rnxE5ldersgxE5rdar i HxE4lsingland. GxE5rdar, borgar och hamnar i det xE4ldsta HxE4lsingland. Red. G. Skoglund. UmexE5 (1981).Viklund, K. Jordbrukskris i Norrland i slutet av den xE4ldre jxE4rnxE5 ldern. Arkeologi i norr 2. UmexE5 95u2013105 (1989).Arbman, H. KxE4ringsjxF6 n. Studier i hallxE4ndsk jxE4rnxE5lder. Kungl. Vitterhets-, historie-och antikvitetsakademiens handlingar. Stockholm (1945).PxE5hlsson, I. VxE4stannorstjxE4rn: en pollenalytisk undersxF6kning. Rapport. The Central Board of National Antiquities and the National Historical museum. RAxC4 1981:6. Stockholm 12 (1981).Geijer, A. Birka III: Die Textilfunde aus den GrxE4bern, Untersuchungen und Studien/von Agnes Geijer. Vitterhets-, historie-och antikvitetsakademien. Stockholm (1938).Christensen, A. E. & Nockert, M. Osebergfunnet. Bd 4. Textilene. Kulturhistorisk museum. Oslo (2006).Bergfjord, C. et al. Nettle as a distinct Bronze Age textile plant. Scientific Reports 2, 664 10.1038/srep00664 (2012)._ _ _ _ _ Andersson, E. The common thread. Textile production during the Late Iron Age u2013 Viking Age. Institutionen fxF6r arkeologi. Lund 36 (1999).Gutarp, E. M. Medieval Manner of Dress, Documents, Images and Surviving Examples of Northern EuropxE9, Emphasizing Gotland in the Baltic Sea. Gotlands Fornsals FxF6rlag. Visby 46 (2001).NylxE9n HemslxF6jd, A.-M. Den svenska hemslxF6jden fram till 1800-talets slut av Anna-Maja NylxE9n. Stockholm (First published 1968) 118 (1995).Barber, E. J. W. Prehistoric Textiles: the development of cloth in the Neolithic and Bronze Ages with special reference to Aegean. Princeton University Press, Princeton, N. J. 15 (1992).Geijer Agnes, A. The Textile Finds from Birka. Cloth and Clothing in Medieval Europe. Essays in Memory of Professor E. M Carus-Wilson. London 87 (1983).Cook, J. G. Handbook of Textile Fibres. 1. Natural Fibres. Redwood Books. Wiltshire. (First published 1959) 99 (1993).Catling, D. & Grayson, J. Identification of vegetable fibres. J. W. Arrowsmith Ltd. Bristol (1982).Bergfjord, C. et al. Comment on u2018u2018 30,000-Year-Old Wild Flax Fibres. Science 325 634-b (2009)._ Franzen, A.-M. & Nockert, M. Bonaderna frxE5n Skog och xD6verhogdal. Almqvist & Wiksell Internationel. Stockholm (1992).Geijer, A. Oriental Textiles in Sweden. Copenhagen 13 (1951).Estham, I. Kyrkliga textilier: arv, utveckling och vxE5rd/Inger Estham; under medverkan av Gertrud Ingers och Gerd Reimers. Skeab. Stockholm (First published 1976) 12 (1980).Dunbar, M. & Murphy, T. M. DNA analysis of Natural Fiber Rope. Journal of Forensic Sciences 54, 108u2013 13 (2009)._ _ Lindahl, T. Instability and decay of the primary structure of DNA. Nature 362, 709u2013715 (1993)._ _ _ _ _ Hofreiter, M., Serre, D., Poinar, H. N., Kuch, M. & PxE4xE4bo, S. S. Ancient DNA. Nature Reviews Genetics 2, 353u2013359 (2001)._ _ Herzog, A. Mikrophotographischer Atlas der technisch wictigen Pflanzenfasern, second ed., Akademie-Verlag. Leipzig (1955).Goodway, M. Fibre identification in practice. Journal of the American Institute for Conservation 26, 27u201344 (1987)._ _ Haugan, E. & Holst, B. Determining the fibrillar orientation of bast fibres with polarized light microscopy: The modified Herzog test (red plate test) explained, Accepted. Journal of Microscopy 10.1111/jmi.12079 (2013).Bergfjord, C. & Holst, B. A procedure for identifying textile bast fibres using microscopy: Flax, nettle/ramie, hemp and jute. Ultramicroscopy 110, 1192u20131197 (2010)._ _ _ _ Benson, L. V., Hattori, E. M., Taylor, H. E., Poulson, S. R. & Jolie, E. A. Isotopesourcing of prehistoric willow and tule textiles recovered from western Greate Basin rock shelters and caves u2013 proof of concept. Journal of Archealogical Science 33, 1588u20131599 (2006)._ _ Piotrowski, S. & Carus, M. Ecological benefits of hemp and flax cultivation and products. Nova-Institute (2011). (Retrieved 2013-05-08).FrxF6ier, K. Lin och Hampa. Lts FxF6rlag. Stockholm 91, 33 (1960)._ Frankow, I. Ingrid 1982. Hampa som textilmaterial. Orsaker till anvxE4ndning. Materialets roll i sjxE4lvhushxE5 llet. Institutet fxF6r folklivsforskning. Stockholm 10, 10u201312 (1982)._ Guinchard, J. Sweden: historical and statistical handbook/Second part: industries. VII: 494 (1914). (Retrieved 2013-05-08).Geijer, A. A History of Textile Art. (transl. by Roger Tanner) Pasold research fund in ass. with Sotheby Parke Bernet. London (1979).Montgomery, F. Textiles in America, 1650u2013 1870: a dictionary based on original documents, prints and paintings, commercial records, American merchant\\'s papers, shopkeepers\\'advertisements and pattern books with original swatches of cloth. Montgomery;foreword by Linda Eaton. W. W. Norton & Company Ltd. New York 277u2013278 (2007).The workwomans guide, containing instructions in cutting out and completing articles of wearing apparel, by a lady, Repr. Bloomfield books & Publ. Doncaster (First published 1840) (1975).Markham, G. The english Housewife. Containing the inward and outward virtues which ought to be in a complete woman; as her skill in physic, cookery, banquetingstuff, distillation, perfumes, wool, hemp, flax, dairies, brewing, baking and all other things belonging to a household. Printed for Hannah. London (Gervase Markham 1568u20131637).Haugan, E. & Holst, B. Flax look-alikes: Pitfalls of ancient plant fibre identification. Archaeometry 10.1111/arcm.12054 (2013).\"- - This work was supported by The Agnes Geijer Foundation for Nordic Textile Research History and Bergens Research Foundation. We thank the museums in Sweden and Norway for their support and for allowing us to borrow textile threads from their collections. We thank A. S. Dinger and A. S. Palau for additional fibre tests. We also thank A. S. Dinger for preparing the images used in Figure 6 and in the online supplementary material document.- - Authors and Affiliations - \"Department of Physics and Technology, University of Bergen, Allegaten 55, NO-5007, Bergen, NorwayG. Skoglund_&_B. HolstDepartment of Art History, Textile Studies, University of Uppsala, ThunbergsvxE4gen 3 H, 751 26, Uppsala, SwedenM. Nockert\" - \"AuthorsG. SkoglundYou can also search for this author in _M. NockertYou can also search for this author in _B. HolstYou can also search for this author in _\" - Contributions - Authors G.S. and B.H. were responsible for writing the main manuscript text. G.S. was responsible for selecting the samples and preparing the microscopy slides. B.H. was responsible for the microscopy analysis, which was performed in cooperation with G.S. and two assistants (see acknowledgements). G.S. and M.N. were responsible for the interpretation of the data in a textile historical context.- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing financial interests. - \\' \\'- - \\'Online Supplementary Information for: Viking and Early Middle Ages Northern Scandinavian Textiles proven to be made with Hemp\\'- - \\' \\' - \\'This work is licensed under a Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported License. ',\n",
       " 'Creative Commons Attribution-Share Alike 3.0 Unported licensing (); Panel g: Creative Commons Attribution-Share Alike 3.0 Unported licensing ().\"- - \"Language is one of the u201Cbreakoutu201D phenomena which define humanity. A language is a codified system of representation in which objects are represented as symbols. u201CGrammarsu201D are basic rules which allow for u201Csyntaxesu201D (concatenated rules which make sense within a languages rules) at each level of representation to function. The connection between lower-level precisely-defined rules and more abstract language rules means that low-level rules interactively and stochastically construct u201Cmeaningu201D in languages. Chemically, this allows for indirect coupling of structure-based catalysis with larger reflex arcs. Viewed in this light, the phenomenon of hierarchical emergence is reminiscent of the analogical mapping between the genetic code and human language.\"- - \"Language involves the mental encoding of concept, followed by expression via speech, followed by auditory or visual reception, and finally mental decoding into received u201Cmeaning.u201D Molecular interactions may not be directly u201Caboutu201D anything, molecules react and interact according to rules. The concept of u201Csomething being about something elseu201D depends on concatenated rules and systemic context, and is a form of meta-catalysis.\"- - Undoubtedly, understanding these interlinked and emergent phenomena will benefit from close collaboration between experimental and computational chemists. For example, discovering small molecule catalysts requires algorithms which can evaluate potential intermolecular interactions and their effects on network kinetics. Literature-based prediction is currently only poorly capable of predicting the effects of such catalysis, however medicinal chemists routinely use docking techniques to rapidly screen non-covalent interactions of drug candidates with enzymes. These methods may be adaptable to rapid screening for potential catalytic interactions among reaction network products expanded using graph transformation rules. - At the level of understanding phase separation and self-assembly, the molecular properties (e.g., Kow, LogP, LogD, etc.) that enable non-covalent molecular aggregation can be estimated using chemoinformatics techniques. However, these methods are presently imprecise since the formation of micelles or vesicles depends on understanding higher order solvent-interaction effects, requiring the use of still other computational techniques. Such effects require their own parameterization and evaluation, but it should be possible to cull the local regions of network chemistry which can give rise to them to save computation time, and careful in vitro screening of large amphiphile libraries would improve the predictivity of computational methods. - \"Rapid methods for the discovery of meta-catalysis may provide insight into the observed u201Cjumpsu201D evident in evolution. Meta-catalysis is indirect, and may help explain the origins of heritable and mutable information coding, which enable the responsiveness and adaptivity of reaction networks to external stimuli. To discover meta-catalytic phenomena, in silico generative networks need to be analyzed using computationally intensive tests at the level of interactional catalysis, which is not presently simple, and then re-evaluated with every other molecule in a network, to see if new u201Cmeaningfulu201D hypergraphs are created. The models in turn need careful vetting using wet chemistry.\" - \"The RNA World concept provides an example of what is lacking in origins models. Chemists have provided ever more u201Cprebiotically plausibleu201D syntheses of RNA, and SELEX experiments have shown it is easy to isolate and amplify molecules which bind target molecules, but a recent computational exploration of nucleic acid space found more than a million possible alternative backbones to deoxyribose/ribose life as we know it uses. None of these approaches can presently address how reaction networks make compounds in thermodynamically and kinetically feasible ways, and predict how the resulting products interact to modify the reaction networks which produce them.\"- - The transition from reaction mechanisms predetermined by physical chemistry to catalyzed network-pruning reactions, to self-sorting phase generating reactions and indirect catalysis form fundamental questions on the origin of life. Both experimental and computational approaches will help understand these transitions. However, numerous problems need to be solved to be able to apply computation in meaningful, tractable ways. Borrowing and adapting techniques from other disciplines is likely the most straightforward method of making progress in this area. Refining these approaches will help focus studies in experimentally testable ways.- - \"Neveu, M., Hays, L. E., Voytek, M. A., New, M. H. & Schulte, M. D. The ladder of life detection. Astrobiology 18, 1375u20131402 (2018)._ _ Cleland, C. E. The Quest for a Universal Theory of Life: Searching for Life As We Dont Know It (Cambridge University Press, 2019).Sandford, S. A., Nuevo, M., Bera, P. P. & Lee, T. J. Prebiotic astrochemistry and the formation of molecules of astrobiological interest in interstellar clouds and protostellar disks. Chem. Rev. 120, 4616u20134659 (2020)._ _ _ Meringer, M. & Cleaves, H. J. Exploring astrobiology using in silico molecular structure generation. Philos. Trans. A Math. Phys. Eng. Sci. 375, 20160344 (2017)._ _ _ Andersen, J. L. et al. Navigating the chemical space of HCN polymerization and hydrolysis: guiding graph grammars by mass spectrometry data. Entropy 15, 4066u20134083 (2013)._ _ _ Wou0142os, A., et al. Synthetic connectivity, emergence, and self-regeneration in the network of prebiotic chemistry. Science 369, eaaw1955 (2020).Miller, S. L., Schopf, J. W. & Lazcano, A. Oparin s u201COrigin of Lifeu201D: sixty years later. J. Mol. Evol. 44, 351u2013353 (1997)._ _ _ Newman, M. E. J. Networks: An Introduction (Oxford University Press, Oxford, 2010).Heylighen, F. in Encyclopedia of Library and Information Sciences, (eds Bates, M. J. & Maack, M. N.) (2008).DxEDazu2010 MuxF1oz, S. L., Boddy, A. M., Dantas, G., Waters, C. M. & Bronstein, J. L. Contextual organismality: beyond pattern to process in the emergence of organisms. Evolution 70, 2669u20132677 (2016)._ _ GxE1nti, T. Organization of chemical reactions into dividing and metabolizing units: the chemotons. BioSystems 7, 15u201321 (1975)._ _ Kauffman, S. A. Autocatalytic sets of proteins. J. Theor. Biol. 119, 1u201324 (1986)._ _ _ Maturana, H. R. & Varela, F. J. Autopoiesis and Cognition: The Realization of the Living. Vol. 42. (Springer Science & Business Media, 1991).Penfold, N. J., Yeow, J., Boyer, C. & Armes, S. P. Emerging trends in polymerization-induced self-assembly. ACS Macro Lett. 8, 1029u20131054 (2019)._ _ _ Jia, T. Z. et al. Membraneless polyester microdroplets as primordial compartments at the origins of life. Proc. Natl Acad. Sci. USA 116, 15830u2013 15835 (2019)._ _ _ England, J. L. Statistical physics of self-replication. J. Chem. Phys. 139, 121923 (2013)._ _ Crick, F. Central dogma of molecular biology. Nature 227, 561u2013563 (1970)._ _ _ Maynard Smith, J. & Szathmary, E. The Major Transitions in Evolution (Oxford University Press, 1997).McCarthy, C. The KekulxE9 problem. Nautilus 19, 22u201331 (2017)._ Lyu, J. et al. Ultra-large library docking for discovering new chemotypes. Nature 566, 224u2013229 (2019)._ _ _ Israelachvili, J. N. Intermolecular and Surface Forces (Elsevier Science, 2011).Yadav, M., Kumar, R. & Krishnamurthy, R. Chemistry of abiotic nucleotide synthesis. Chem. Rev. 120, 4766u20134805 (2020)._ _ _ Ellington, A. D. & Szostak, J. W. In vitro selection of RNA molecules that bind specific ligands. Nature 346, 818u2013822 (1990)._ _ _ Cleaves, H. J., Butch, C., Burger, P. B., Goodwin, J. & Meringer, M. One among millions: the chemical space of nucleic acid-like molecules. J. Chem. Inf. Model. 59, 4266u20134277 (2019)._ _ _ \"- - The authors wish to thank the Earth-Life Science Institute (ELSI) and the ELSI Origins Network (EON) which supported the research which brought the authors of this work into contact. EON was supported by a grant from the John Templeton Foundation. The opinions expressed in this publication are those of the authors and do not necessarily reflect the views of the John Templeton Foundation.- - Authors and Affiliations - \"Department of Biomedical Engineering, College of Engineering and Applied Sciences, Nanjing University, Nanjing, ChinaChristopher J. ButchEarth-Life Science Institute, Tokyo Institute of Technology, Tokyo, JapanChristopher J. Butch_&_H. James Cleaves IIBlue Marble Space Institute for Science, Seattle, WA, USAChristopher J. Butch_&_H. James Cleaves IIGerman Aerospace Center (DLR), Earth Observation Center (EOC), Oberpfaffenhofen-Wessling, GermanyMarkus MeringerPhysics Department, Norwich University, Northfield, VT, USAJean-Sebastien GagnonDepartment of Earth and Planetary Sciences, Harvard University, Cambridge, MA, USAJean-Sebastien GagnonInstitute for Advanced Study, Princeton, NJ, USAH. James Cleaves IICenter for Chemical Evolution, Georgia Institute of Technology, Atlanta, GA, USAH. James Cleaves II\" - \"AuthorsChristopher J. ButchYou can also search for this author in _Markus MeringerYou can also search for this author in _Jean-Sebastien GagnonYou can also search for this author in _H. James Cleaves IIYou can also search for this author in _\" - Contributions - C.B., M.M., J.G., and H.J.C. contributed equally to the development and preparation of the manuscript and figure. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution-Share Alike 4.0 license.\\'- - Expression constructs - \"Mutations were introduced into a human MLKL DNA template (from DNA2.0, CA) using oligonucleotide-directed overlap PCR or were synthesized by ATUM (CA) and subcloned into pF TRE3G PGK puro as BamHIu2013EcoRI fragments. Genes encoding N-terminally FLAG-tagged wild type or mutant human RIPK3 were either generated using oligonucleotide-directed overlap PCR from a human RIPK3 DNA template (ATUM) or synthesized by ATUM and ligated into pF TRE3G PGK puro._Oligonucleotide sequences are presented in Supplementary_Table . Insert sequences were verified by Sanger sequencing (Micromon DNA Sequencing Facility, VIC, Australia). Vector DNA was co-transfected into HEK293T cells with pVSVg and pCMV u0394R8.2 helper plasmids to generate lentiviral particles, which were transduced into MLKLu2009 or RIPK3 HT29 cells and selected for genomic integration using puromycin (1.25u2009u03BCg/mL for MLKL cells, 2.5u2009u03BCg/mL for RIPK3 cells; StemCell Technologies) using established procedures.\" - Reagents and antibodies - \"Primary antibodies used in this study were: rat anti-MLKL (clone 3H1, produced in-house; 1:1000 dilution; available as MABC604, EMD Millipore, Billerica, MA, USA), rat anti-human MLKL pseudokinase domain (clone 7G2, produced in-house; 1:2000 dilution; soon available from_EMD Millipore as MABC1636), rabbit anti-human MLKL phospho-S358 (AB187091, Abcam; 1:2000), mouse anti-Actin (C4) HRP (sc-47778 HRP, Santa Cruz Biotechnology; 1:10000), rat anti-human RIPK3 (clone 1H2, produced in-house; 1:1000; available as MABC1640, EMD Millipore, Billerica, MA, USA), rabbit anti-human RIPK3 phospho-S227 (D6W2T, CST; 1:2000). Recombinant hTNF-Fc was produced in-house, while the Smac mimetic, Compound A and the pan-caspase inhibitor, IDN-6556/Emricasan, were provided by Tetralogic Pharmaceuticals. The RIPK3 inhibitors, GSKu2032843 (ref. ) and BMS Compound 10 (ref. ), were kindly provided by Anaxis_Pty Ltd (Australia).\" - Recombinant protein expression and purification - \"Human MLKL pseudokinase domain (residues 190u2013471): human RIPK3 kinase domain (residues 1u2013316; harboring C3S and C110A substitutions to prevent disulfide bond formation) complex was co-expressed and purified from Sf21 insect cells using an established procedure. The co-expression bacmid was prepared in DH10MultiBac Escherichia coli (ATG Biosynthetics) from a pFastBac Htb vector encoding a TEV protease-cleavable His6 fusion N-terminal to the human MLKL pseudokinase domain (residues 190u2013471), with a p10 promoter-human RIPK3 (residues 1u2013316; C3S, C110A)-HSV TK poly(A) signal (from pAceBac2; ATG Biosynthetics) cassette introduced into pFastBac Htb:human MLKL (190u2013471) by In-Fusion (Takara Bio) cloning into the non-coding region between BbsI and SnaBI sites of the pFastBac Htb vector backbone. A bacmid encoding human RIPK3 (2u2013316; C3S, C110A) with an N-terminal, TEV protease-cleavable His6 tag was prepared from pFastBac Htb via recombination in_DH10MultiBac E. coli cells. Sf21 insect cells were cultured in Insect-XPRESS (Lonza) media. Bacmids were introduced into Sf21 cells by Cellfectin II (Thermo Fisher Scientific) mediated transfection in six-well plates using the Bac-to-Bac protocol (Thermo Fisher Scientific) as detailed previously. The resulting P1 baculovirus was harvested after 4 days static incubation at 27u2009xB0C and added at 1% v/v to 100u2009mL Sf21 cells at 1.5u2009xD7 u200910 density, which were shaken at 27u2009xB0C, 130u2009r.p.m. for 4 days before the supernatant P2 virus was harvested. 50u2009mL P2 virus was added to 0.5u2009L Sf21 (2.5u20133u2009xD7u200910 cells/mL) in 2.8u2009L Fernbach flasks, and cells cultured for 48u2009h at 27u2009xB0C, 90u2009r.p.m., using an established procedure. Cells were harvested at 500u2009xD7u2009g and pellets snap frozen in liquid N2 and either thawed immediately for lysis or stored at u221280u2009xB0C until required. Cells were lysed by sonication in 0.5u2009 M NaCl, 20u2009mM Tris-HCl (pH 8), 20% v/v glycerol, 10u2009mM imidazole (pH 8), 0.5u2009mM TCEP [tris-(2-carboxyethyl)phosphine] supplemented with EDTA-free cOmplete Protease Inhibitor (Roche). Throughout purification, the proteins were maintained at 4u2009xB0C. The lysate was clarified by centrifugation (40,000u2009 xD7u2009g, 30u2009min) and then mixed with HisTag Ni-NTA resin (Roche), which has been pre-equilibrated with low imidazole buffer [0.5u2009M NaCl, 20u2009 mM Tris-HCl (pH 8.0), 5u2009mM imidazole (pH 8.0), 20% v/v glycerol] for 1u2009 h at 4u2009xB0C before the beads were pelleted at 500u2009xD7u2009g and washed extensively with low imidazole buffer and a wash buffer containing 35u2009 mM imidazole. The protein was eluted in high imidazole buffer [0.5u2009M NaCl, 20u2009mM Tris-HCl (pH 8.0), 250u2009mM imidazole (pH 8.0), 20% v/v glycerol]. The eluted His6-RIPK3 protein or His6-MLKL:RIPK3 complex were further purified by cleaving the His6 tag with TEV protease, dialysis, Ni-pulldown to eliminate uncut material and the TEV protease, and application to a HiLoad 16/160 Superdex 200 column (Cytiva) pre-equilibrated with SEC buffer [20u2009mM HEPES (pH 7.5), 200u2009mM NaCl, 5% v/v glycerol]. Purified fractions, as assessed following resolution by reducing StainFree SDS-PAGE gel electrophoresis (Bio-Rad), were pooled and concentrated to 5u2009mg/mL using_a 30u2009kDa MWCO spin concentrator (Millipore), aliquoted, and snap frozen in liquid N2 for storage at u221280u2009 xB0C.\" - Protein crystallization and structure determination - \"MLKL:RIPK3 complex was incubated overnight at 4u2009xB0C with a fivefold molar excess of BMS Compound 10, and RIPK3 was incubated with fivefold excess of GSKu2032 843 before excess compound was eliminated using 30u2009kDa MWCO spin concentrators. Proteins were subjected to sitting drop crystallization trials conducted at the C3 Facility (Parkville, VIC) with 150u2009nL protein solution mixed with 150u2009nL reservoir volume in sitting drops, with 50u2009u03BCL reservoir volume. RIPK3 kinase:MLKL pseudokinase domain uFEFFcrystals grown in 0.2u2009 M ammonium acetate, 30% w/v polyethylene glycol 4000, 0.1u2009M trisodium citrate-citric acid buffer (pH 5.5) at 8u2009xB0C, and RIPK3 kinase domain crystals grown in 0.2u2009M magnesium chloride, 25% w/v polyethylene glycol 3350, 0.1u2009 M Tris (pH 8.5) at 8u2009xB0C were soaked in cryoprotectant (reservoir solution supplemented with 20% v/v ethylene glycol) and flash-frozen in liquid nitrogen. MLKL:RIPK3 complex X-ray diffraction data were collected at the Australian Synchrotron MX2 beamline using the EIGER X 16M detector with in-house MX control software. The MLKL:RIPK3 complex diffraction data were indexed and integrated in XDS, and merged and scaled in AIMLESS. After adding Rfree reflections using phenix.reflection_file_editor, phases were solved for the MLKL:RIPK3 complex by molecular replacement in PHASER. To minimize model bias, molecular replacement with structural models in all available conformations were attempted, including the closed (PDB: 4MWI) and open (PDB: 7JXU) human MLKL conformers, and the closed (PDB: 4M66) and open (PDB: 4M69) mouse RIPK3 conformers. The solution using the human MLKL pseudokinase domain structure from the Mb32 complex (PDB: 7JXU) and the mouse RIPK3 kinase domain from the mouse RIPK3:MLKL complex structure (PDB: 4M69) were selected based on reporting the highest LLG score and the closest match between the molecular replacement model and the electron density map. Human RIPK3:GSKu2032843 diffraction data were processed using DIALS and autoPROC, and elliptically truncated using STARANISO owing to severe anisotropy, with phases solved by molecular replacement. The default I/sigI threshold of 1.2 was used to define the resolution for elliptical truncation, where the cutoff for each axis was 5.7, 3.7, and 3.1u2009xC5 in the a*, b*, and c* axes, respectively. The highest resolution reflection present within this cutoff was at 3.23u2009xC5. Rfree flags were added using phenix.refine post molecular replacement. COOT and phenix.refine were used for iterative real-space and reciprocal-space refinement by manual model building and phase refinement. Protein interfaces and the biological assembly in solution were analyzed using PDBePISA. Interface area is defined as the difference in total accessible surface areas of isolated and interfacing structures divided by two in xC5. Structure models were validated using MolProbity. Structural figures were generated using either UCSF Chimera or PyMOL (The PyMOL Molecular Graphics System, SchrxF6 dinger, LLC.). Structure alignments were calculated using UCSF Chimera. Data collection and processing statistics are summarized in Table_.\" - Small angle X-ray scattering - \"SAXS data were collected using the inline-SEC setup at the Australian Synchrotron SAXS/WAXS beamline. A co-flow setup was implemented to maximize signal-to-noise by allowing exposure of protein to high flux X-rays without direct contact with the capillary wall, while minimizing sample dilution. RIPK3:MLKL complex (unliganded or pre-incubated with Compound 10) was eluted from a Superdex 200 5/150 Increase column (Cytiva) in 200u2009mM NaCl, 20u2009mM HEPES pH 7.5, 5% glycerol at a flow rate of 0.2u2009mL/min, where the SEC eluate passes through a capillary in the path of the SAXS beam and scattering data were collected in 1u2009s exposures serially over the course of the chromatography run. Data were reduced, 2D radially integrated, and background scatter subtracted using ScatterBrain v2.82 (developed in-house by Stephen Mudie). The background scatter was calculated by averaging exposures from the SEC run prior to protein elution, and was subtracted from the scatter arising from the protein peak off the SEC column. Exposures from the apex of the protein elution peak were averaged and subjected to analyses using the ATSAS 3.0 package including Guinier analyses in PRIMUS, Pr in GNOM, and comparisons with theoretical and experimental scatter using CRYSOL. Data collection and processing statistics are summarized in Supplementary Table_ .\" - Cell culture - \"The human colorectal adenocarcinoma HT29 (a gift from Mark Hampton), and their MLKLu2009 and RIPK3u2009, counterparts were cultured in human DMEM (Thermo Fisher) media supplemented with 8% vol/vol fetal calf serum (FCS; Sigma), with puromycin (2.5u2009u03BCg/mL; StemCell Technologies) added for lines stably transduced with inducible MLKL or RIPK3 constructs. Routine PCR testing confirmed cell lines to be mycoplasma-negative.\" - IncuCyte cell death assay - \"HT29 cells were seeded into 48-well plates at 3u2009xD7u200910 cells/well and left to settle for 16u201324u2009h prior to treatment with doxycycline (20u2009ng/mL) overnight to induce expression of the relevant MLKL construct. RIPK3 expression was induced comparably but with 2.5u2009ng/mL doxycycline. Cells were then treated with TNF (100u2009ng/mL), the Smac-mimetic compound A (500u2009nM) and the pan-caspase inhibitor IDN-6556 (5u2009u03BCM) to induce necroptosis in FluoroBrite DMEM (Thermo Fisher Scientific) media supplemented with 1% FCS (Sigma), 1u2009mM Na pyruvate (Thermo Fisher Scientific), 1u2009 mM l-GlutaMAX and SYTOX Green (Thermo Fisher Scientific) using an established protocol. Cells were imaged using default bright field and green channel settings on xD710 objective in an IncuCyte S3 System (Essen Bioscience) with scans every hour for 24u2009h. The number of SYTOX Green-positive cells per mm over time was quantified using IncuCyte S3 v2020C Rev 1 software (Essen Bioscience).\" - Western blots - \"HT29 cells were seeded into 24-well plates at 7u2009xD7u200910 cells/well and induced overnight with 20u2009ng/mL doxycycline for MLKL expression, or 2.5u2009ng/mL doxycycline for RIPK3 expression. Cells were then treated with TNF (100u2009ng/mL), the Smac-mimetic compound A (500u2009nM) and the pan-caspase inhibitor IDN-6556 (5u2009u03BCM) to induce necroptosis in DMEM media supplemented with 1% FCS. MLKL expressing cells were harvested after 3u2009h and RIPK3 expressing cells were harvested 7.5u2009h post-necroptotic induction (with TSI) in 2xD7 SDS Laemmli lysis buffer, boiled at 100u2009xB0C for 5u201310u2009min, and then resolved by 4u201315% Tris-Glycine gel (Bio-Rad) or 4u201312% Bis Tris (Invitrogen) gels. Proteins were transferred to PVDF membrane, blocked with 5% w/v skim milk powder in TBST, and probed with antibodies as indicated.\" - Thermal shift assays - \"Thermal shift assays were performed using a Corbett Real Time PCR machine with proteins diluted in SEC buffer to 10u2009u03BCM and assayed with either 40u2009 u03BCM Compound 10 or an equivalent volume of DMSO control in a total reaction volume of 25u2009u03BCL using established methods. SYPRO Orange (Sigma-Aldrich) was used as a probe with fluorescence detected at 530u2009nm. The temperature was raised in 1u2009xB0C per min steps from 25u2009xB0C to 95u2009xB0 C and fluorescence readings were taken at each interval. For each well, sample fluorescence was plotted as a function of increasing temperature. The melting temperature (Tm) corresponding to the midpoint for the protein unfolding transition was calculated by fitting the sigmoidal melt curve to the Boltzmann equation using non-linear least square fit with R Studio, with R values of >0.99. Data points after the fluorescence intensity maximum were excluded from fitting. Three technical replicates were performed and plotted for each protein and ligand, and the mean Tm are shown for each.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - \"Source data are provided for western blots and cell death assays in Fig._ and Supp. Fig._ and thermal stability shift assays in Supp. Fig._. All data, including expression construct sequences, are available from the corresponding authors upon request. The atomic coordinates for the human RIPK3 kinase domain in complex with GSKu2032843, and in complex with the human MLKL pseudokinase domain and Compound 10, have been deposited in the Protein Data Bank with the accession numbers, and , respectively. The atomic coordinates for previously reported structures used in this study are available from the Protein Data Bank with accession codes: , , , , , , , ._ are provided with this paper.\" - \\' \\'- - \"Holler, N. et al. Fas triggers an alternative, caspase-8-independent cell death pathway using the kinase RIP as effector molecule. Nat. Immunol. 1, 489u2013 495 (2000)._ _ _ _ Degterev, A. et al. Chemical inhibitor of nonapoptotic cell death with therapeutic potential for ischemic brain injury. Nat. Chem. Biol. 1, 112u2013119 (2005)._ _ _ _ Cho, Y. et al. Phosphorylation-driven assembly of the RIP1-RIP3 complex regulates programmed necrosis and virus-induced inflammation. Cell 137, 1112u20131123 (2009)._ _ _ _ _ Sun, L. et al. Mixed lineage kinase domain-like protein mediates necrosis signaling downstream of RIP3 kinase. Cell 148, 213u2013227 (2012)._ _ _ _ He, S. et al. Receptor interacting protein kinase-3 determines cellular necrotic response to TNF-alpha. Cell 137, 1100u20131111 (2009)._ _ _ _ Murphy, J. M. et al. The pseudokinase MLKL mediates necroptosis via a molecular switch mechanism. Immunity 39, 443u2013453 (2013)._ _ _ _ Kitur, K. et al. Necroptosis promotes Staphylococcus aureus clearance by inhibiting excessive inflammatory signaling. Cell Rep. 16, 2219u20132230 (2016)._ _ _ _ _ Liu, Z. et al. A class of viral inducer of degradation of the necroptosis adaptor RIPK3 regulates virus-induced inflammation. Immunity 54, 247u2013258 e247 (2021)._ _ _ _ Pearson, J. S. et al. EspL is a bacterial cysteine protease effector that cleaves RHIM proteins to block necroptosis and inflammation. Nat. Microbiol. 2, 16258 (2017)._ _ _ _ Pearson, J. S. & Murphy, J. M. Down the rabbit hole: is necroptosis truly an innate response to infection? Cell Microbiol. 19, e12750 (2017)._ _ Petrie, E. J. et al. Viral MLKL homologs subvert necroptotic cell death by sequestering cellular RIPK3. Cell Rep. 28, 3309u20133319 e3305 (2019)._ _ _ _ Rickard, J. A. et al. TNFR1-dependent cell death drives inflammation in Sharpin-deficient mice. Elife 3, e03464 (2014).Rickard, J. A. et al. RIPK1 regulates RIPK3-MLKL-driven systemic inflammation and emergency hematopoiesis. Cell 157, 1175u20131188 (2014)._ _ _ _ Hildebrand, J. M. et al. A missense mutation in the MLKL brace region promotes lethal neonatal inflammation and hematopoietic dysfunction. Nat. Commun. 11, 3150 (2020)._ _ _ _ _ _ Lu, Z. et al. Necroptosis signalling promotes inflammation, airway remodelling and emphysema in COPD. Am. J. Respir. Crit. Care Med. 204, 667u2013 681 (2021).Muller, T. et al. Necroptosis and ferroptosis are alternative cell death pathways that operate in acute kidney failure. Cell Mol. Life Sci. 74, 3631u20133645 (2017)._ _ _ _ Pefanis, A., Ierino, F. L., Murphy, J. M. & Cowan, P. J. Regulated necrosis in kidney ischemia-reperfusion injury. Kidney Int. 96, 291u2013301 (2019)._ _ _ Hildebrand, J. M. et al. A family harboring an MLKL loss of function variant implicates impaired necroptosis in diabetes. Cell Death Dis. 12, 345 (2021)._ _ _ _ _ Pierdomenico, M. et al. Necroptosis is active in children with inflammatory bowel disease and contributes to heighten intestinal inflammation. Am. J. Gastroenterol. 109, 279u2013287 (2014)._ _ _ _ Tovey Crutchfield, E. C., Garnish, S. E. & Hildebrand, J. M. The role of the key effector of necroptotic cell death, MLKL, in mouse models of disease. Biomolecules 11, 803 (2021).Sun, X., Yin, J., Starovasnik, M. A., Fairbrother, W. J. & Dixit, V. M. Identification of a novel homotypic interaction motif required for the phosphorylation of receptor-interacting protein (RIP) by RIP3. J. Biol. Chem. 277, 9505u20139511 (2002)._ _ _ _ Mompean, M. et al. The structure of the necrosome RIPK1-RIPK3 core, a human hetero-amyloid signaling complex. Cell 173, 1244u2013 1253 e1210 (2018)._ _ _ _ _ Wu, X. L. et al. The amyloid structure of mouse RIPK3 (receptor interacting protein kinase 3) in cell necroptosis. Nat. Commun. 12, 1627 (2021)._ _ _ _ _ _ Cho, Y. S. et al. Phosphorylation-driven assembly of the RIP1-RIP3 complex regulates programmed necrosis and virus-induced inflammation. Cell 137, 1112u20131123 (2009)._ _ _ _ _ Meng, Y., Sandow, J. J., Czabotar, P. E. & Murphy, J. M. The regulation of necroptosis by post-translational modifications. Cell Death Differ. 28, 861u2013883 (2021)._ _ _ _ Murphy, J. M. The killer pseudokinase mixed lineage kinase domain-like protein (MLKL). Cold Spring Harb. Perspect. Biol 12, a036376 (2020).Zhao, J. et al. Mixed lineage kinase domain-like is a key receptor interacting protein 3 downstream component of TNF-induced necrosis. Proc. Natl Acad. Sci. USA 109, 5322u2013 5327 (2012)._ _ _ _ _ _ Hildebrand, J. M. et al. Activation of the pseudokinase MLKL unleashes the four-helix bundle domain to induce membrane localization and necroptotic cell death. Proc. Natl Acad. Sci. USA 111, 15072u201315077 (2014)._ _ _ _ _ _ Murai, S. et al. A FRET biosensor for necroptosis uncovers two different modes of the release of DAMPs. Nat. Commun. 9, 4457 (2018)._ _ _ _ _ Samson, A. L. et al. A toolbox for imaging RIPK1, RIPK3, and MLKL in mouse and human cells. Cell Death Differ. (2021).Samson, A. L., Garnish, S. E., Hildebrand, J. M. & Murphy, J. M. Location, location, location: a compartmentalized view of necroptotic signaling. Sci. Signal. 14, eabc6178 (2021)._ _ _ _ Samson, A. L. et al. MLKL trafficking and accumulation at the plasma membrane control the kinetics and threshold for necroptosis. Nat. Commun. 11, 3151 (2020)._ _ _ _ _ _ Quarato, G. et al. Sequential engagement of distinct MLKL phosphatidylinositol-binding sites executes necroptosis. Mol. Cell 61, 589u2013601 (2016)._ _ _ _ _ Dondelinger, Y. et al. MLKL compromises plasma membrane integrity by binding to phosphatidylinositol phosphates. Cell Rep. 7, 971u2013981 (2014)._ _ _ _ Cai, Z. et al. Plasma membrane translocation of trimerized MLKL protein is required for TNF-induced necroptosis. Nat. Cell Biol. 16, 55u201365 (2014)._ _ _ _ Petrie, E. J., Hildebrand, J. M. & Murphy, J. M. Insane in the membrane: a structural perspective of MLKL function in necroptosis. Immunol. Cell Biol. 95, 152u2013 159 (2017)._ _ _ _ Sethi, A. et al. Membrane permeabilization is mediated by distinct epitopes in mouse and human orthologs of the necroptosis effector, MLKL. Preprint at (2021).Davies, K. A. et al. The brace helices of MLKL mediate interdomain communication and oligomerisation to regulate cell death by necroptosis. Cell Death Differ. 25, 1567u20131580 (2018)._ _ _ _ _ Tanzer, M. C. et al. Evolutionary divergence of the necroptosis effector MLKL. Cell Death Differ. 23, 1185u20131197 (2016)._ _ _ _ _ Davies, K. A. et al. Distinct pseudokinase domain conformations underlie divergent activation mechanisms among vertebrate MLKL orthologues. Nat. Commun. 11, 3060 (2020)._ _ _ _ _ _ Fletcher-Etherington, A. et al. Human cytomegalovirus protein pUL36: a dual cell death pathway inhibitor. Proc. Natl Acad. Sci. USA 117, 18771u201318779 (2020)._ _ _ _ _ Omoto, S. et al. Suppression of RIP3-dependent necroptosis by human cytomegalovirus. J. Biol. Chem. 290, 11635u201311648 (2015)._ _ _ _ _ Steain, M. et al. Varicella zoster virus encodes a viral decoy RHIM to inhibit cell death. PLoS Pathog. 16, e1008473 (2020)._ _ _ _ _ Palmer, S., Chappidi, S., Pinkham, C. & Hancks, D. C. Evolutionary profile for (host and viral) MLKL indicates its activities as a battlefront for extensive counteradaptation. Mol. Biol. Evol. (2021).Petrie, E. J., Czabotar, P. E. & Murphy, J. M. The structural basis of necroptotic cell death signaling. Trends Biochem Sci. 44, 53u2013 63 (2019)._ _ _ _ Xie, T. et al. Structural insights into RIP3-Mediated necroptotic signaling. Cell Rep. 5, 70u201378 (2013)._ _ _ _ Garnish, S. E. et al. Conformational interconversion of MLKL and disengagement from RIPK3 precede cell death by necroptosis. Nat. Commun. 12, 2211 (2021)._ _ _ _ _ _ Hart, A. C. et al. Identification of RIPK3 type II inhibitors using high-throughput mechanistic studies in Hit triage. ACS Med. Chem. Lett. 11, 266u2013271 (2020)._ _ _ _ Chen, W. et al. Diverse sequence determinants control human and mouse receptor interacting protein 3 (RIP3) and mixed lineage kinase domain-like (MLKL) interaction in necroptotic signaling. J. Biol. Chem. 288, 16247u201316261 (2013)._ _ _ _ _ Murphy, J. M. et al. Insights into the evolution of divergent nucleotide-binding mechanisms among pseudokinases revealed by crystal structures of human and mouse MLKL. Biochem. J. 457, 369u2013377 (2014)._ _ _ _ Petrie, E. J. et al. Conformational switching of the pseudokinase domain promotes human MLKL tetramerization and cell death by necroptosis. Nat. Commun. 9, 2422 (2018)._ _ _ _ _ Pierotti, C. L. et al. Potent inhibition of necroptosis by simultaneously targeting multiple effectors of the pathway. ACS Chem. Biol. 15, 2702u20132713 (2020)._ _ _ _ Kornev, A. P., Haste, N. M., Taylor, S. S. & Eyck, L. F. Surface comparison of active and inactive protein kinases identifies a conserved activation mechanism. Proc. Natl Acad. Sci. USA 103, 17783u201317788 (2006)._ _ _ _ _ _ Kaiser, W. J. et al. Toll-like receptor 3-mediated necrosis via TRIF, RIP3, and MLKL. J. Biol. Chem. 288, 31268u201331279 (2013)._ _ _ _ _ Krissinel, E. & Henrick, K. Inference of macromolecular assemblies from crystalline state. J. Mol. Biol. 372, 774u2013797 (2007)._ _ _ _ Jacobsen, A. V. et al. HSP90 activity is required for MLKL oligomerisation and membrane translocation and the induction of necroptotic cell death. Cell Death Dis. 7, e2051 (2016)._ _ _ _ _ Petrie, E. J. et al. Identification of MLKL membrane translocation as a checkpoint in necroptotic cell death using Monobodies. Proc. Natl Acad. Sci. USA 117, 8468u20138475 (2020)._ _ _ _ _ Kim, K. S. et al. Discovery of pyrrolopyridine-pyridone based inhibitors of Met kinase: synthesis, X-ray crystallographic analysis, and biological activities. J. Med. Chem. 51, 5330u2013 5341 (2008)._ _ _ _ Daub, H. et al. Kinase-selective enrichment enables quantitative phosphoproteomics of the kinome across the cell cycle. Mol. Cell 31, 438u2013448 (2008)._ _ _ _ Dephoure, N. et al. A quantitative atlas of mitotic phosphorylation. Proc. Natl Acad. Sci. USA 105, 10762u201310767 (2008)._ _ _ _ _ _ Chen, W. et al. Diverse sequence determinants control human and mouse receptor interacting protein 3 (RIP3) and mixed lineage kinase domain-like (MLKL) interaction in necroptotic signaling. J. Biol. Chem. 288, 16247u201316261 (2013)._ _ _ _ _ Bossen, C. et al. Interactions of tumor necrosis factor (TNF) and TNF receptor family members in the mouse and human. J. Biol. Chem. 281, 13964u201313971 (2006)._ _ _ _ Vince, J. E. et al. IAP antagonists target cIAP1 to induce TNFalpha-dependent apoptosis. Cell 131, 682u2013693 (2007)._ _ _ _ Babon, J. J. & Murphy, J. M. In vitro JAK kinase activity and inhibition assays. Methods Mol. Biol. 967, 39u201355 (2013)._ _ _ _ Murphy, J. M. et al. A robust methodology to subclassify pseudokinases based on their nucleotide binding properties. Biochem. J. 457, 323u2013334 (2014)._ _ _ _ Aragao, D. et al. MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron. J. Synchrotron Radiat. 25, 885u2013891 (2018)._ _ _ _ _ Kabsch, W. Integration, scaling, space-group assignment and post-refinement. Acta Crystallogr. D Biol. Crystallogr. 66, 133u2013144 (2010)._ _ _ _ _ Evans, P. R. & Murshudov, G. N. How good are my data and what is the resolution? Acta Crystallogr. D Biol. Crystallogr. 69, 1204u20131214 (2013)._ _ _ _ _ Liebschner, D. et al. Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix. Acta Crystallogr. D. Struct. Biol. 75, 861u2013877 (2019)._ _ _ _ _ McCoy, A. J. et al. Phaser crystallographic software. J. Appl Crystallogr. 40, 658u2013674 (2007)._ _ _ _ _ Beilsten-Edmands, J. et al. Scaling diffraction data in the DIALS software package: algorithms and new approaches for multi-crystal scaling. Acta Crystallogr. D Struct. Biol. 76, 385u2013399 (2020)._ _ _ _ _ Vonrhein, C. et al. Data processing and analysis with the autoPROC toolbox. Acta Crystallogr. D Biol. Crystallogr. 67, 293u2013302 (2011)._ _ _ _ _ Afonine, P. V. et al. Joint X-ray and neutron refinement with phenix.refine. Acta Crystallogr. D Biol. Crystallogr. 66, 1153u20131163 (2010)._ _ _ _ _ Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of Coot. Acta Crystallogr. D Biol. Crystallogr. 66, 486u2013501 (2010)._ _ _ _ _ Chen, V. B. et al. MolProbity: all-atom structure validation for macromolecular crystallography. Acta Crystallogr. D Biol. Crystallogr. 66, 12u201321 (2010)._ _ _ _ Pettersen, E. F. et al. UCSF Chimerau2014 a visualization system for exploratory research and analysis. J. Comput. Chem. 25, 1605u20131612 (2004)._ _ _ _ Kirby, N. et al. Improved radiation dose efficiency in solution SAXS using a sheath flow sample environment. Acta Crystallogr. D Struct. Biol. 72, 1254u20131266 (2016)._ _ _ _ _ Ryan, T. M. et al. An optimized SEC-SAXS system enabling high X-ray dose for rapid SAXS assessment with correlated UV measurements for biomolecular structure analysis. J. Appl. Crystallogr. 51, 97u2013111 (2018)._ _ _ Manalastas-Cantos, K. et al. ATSAS 3.0: expanded functionality and new tools for small-angle scattering data analysis. J. Appl. Crystallogr. 54, 343u2013355 (2021)._ _ _ _ _ Lucet, I. S. & Murphy, J. M. Characterization of ligand binding to pseudokinases using a thermal shift assay. Methods Mol. Biol. 1636, 91u2013104 (2017)._ _ _ _ \"- - We thank Dr. Janet Newman and the C3 Facility for assistance with screening for protein crystallization conditions, and the Australian Synchrotron MX and SAXS/WAXS beamline staff for assistance with data collection. This research was undertaken using the MX2 crystallography beamline at the Australian Synchrotron, Victoria, Australia, and made use of the ACRF Detector. We acknowledge scholarship support for Y.M. (Melbourne Research Scholarship; AINSE PGRA scholarship) and S.E.G. (Australian Government Research Training Program Stipend Scholarship; Wendy Dowsett Scholarship). We are grateful to the National Health and Medical Research Council for fellowship (G.L., 1117089; P.E.C., 1079700; J.M.M., 1172929), grant (A.L.S. and J.J.S., 2002965) and infrastructure (IRIISS 9000653) support; Anaxis Pharma Pty Ltd for funding support; the Australian Cancer Research Foundation; and the Victorian Government Operational Infrastructure Support scheme.- - Author notes - \\'These authors contributed equally: Yanxiang Meng, Katherine A. Davies.These authors jointly supervised this work: Peter E. Czabotar, James M. Murphy.\\' - Authors and Affiliations - \"Walter and Eliza Hall Institute of Medical Research, 1G Royal Parade, Parkville, VIC, 3052, AustraliaYanxiang Meng,_Katherine A. Davies,_Cheree Fitzgibbon,_ Samuel N. Young,_Sarah E. Garnish,_Christopher R. Horne,_Cindy Luo,_Lung-Yu Liang,_Angus D. Cowan,_Andre L. Samson,_Guillaume Lessene,_Jarrod J. Sandow,_ Peter E. Czabotar_&_James M. MurphyDepartment of Medical Biology, University of Melbourne, Parkville, VIC, 3052, AustraliaYanxiang Meng,_Katherine A. Davies,_ Cheree Fitzgibbon,_Sarah E. Garnish,_Christopher R. Horne,_Lung-Yu Liang,_ Angus D. Cowan,_Andre L. Samson,_Guillaume Lessene,_Jarrod J. Sandow,_Peter E. Czabotar_&_James M. MurphySYNthesis med chem, 30 Flemington Rd, Parkville, VIC, 3052, AustraliaJean-Marc Garnier\" - \"AuthorsYanxiang MengYou can also search for this author in _Katherine A. DaviesYou can also search for this author in _Cheree FitzgibbonYou can also search for this author in _Samuel N. YoungYou can also search for this author in _Sarah E. GarnishYou can also search for this author in _Christopher R. HorneYou can also search for this author in _Cindy LuoYou can also search for this author in _Jean-Marc GarnierYou can also search for this author in _Lung-Yu LiangYou can also search for this author in _Angus D. CowanYou can also search for this author in _Andre L. SamsonYou can also search for this author in _Guillaume LesseneYou can also search for this author in _Jarrod J. SandowYou can also search for this author in _Peter E. CzabotarYou can also search for this author in _James M. MurphyYou can also search for this author in _\" - Contributions - Y.M. and K.A.D. designed, performed, and analyzed experiments, performed structural studies and wrote the paper with J.M.M.; C.F., S.N.Y., C.R.H., C.L., L.-Y.L., and A.D.C. designed and performed experiments; S.E.G., A.L.S., J.-M.G., G.L., and J.J.S. analyzed and interpreted data; P.E.C. and J.M.M. designed the project, analyzed, and interpreted data; all authors commented on the manuscript. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - Y.M., K.A.D., C.F., S.N.Y., S.E.G., C.L., J.-M.G., L.-Y.L., A.D.C., A.L.S., G.L., P.E.C., and J.M.M. contribute, or have contributed, to a project developing necroptosis inhibitors in collaboration with Anaxis Pharma Pty Ltd. The remaining authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks Hiroyasu Nakano and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution-ShareAlike 3.0 Unported license. c Diplocaulus, by Gareth Monger, available under Creative Commons Attribution 3.0 Unported license - \"Fig. 3Dispersal and vicariance rates of amniote clades through time. a All Amniota; b Synapsida; c Parareptilia; d Eureptilia. Solid grey lines represent vicariance. Dashed black lines represent dispersal. Thick lines represent the mean of the rates obtained from each stochastic mapping iteration. The thinner lines represent a standard error above and below. Silhouettes from phylopic.org. b Dimetrodon, by Dmitry Bogdanov, available under Creative Commons Attribution-ShareAlike 3.0 Unported license. c Scutosaurus, by Chris Jennings (vectorized by A. VerrixE8 re), available under Public Domain Dedication 1.0 license. d Concordia, by Steven Blackwood, available under Creative Commons Attribution-ShareAlike 3.0 Unported license\" - \"Tetrapod dispersal rates recover throughout the early Permian, reaching a peak during the early Capitanian (Fig._). The late Capitanian vicariance peak, and the trough in dispersal that follows, is visible in all amniote clades and also the temnospondyls (Figs._u2013). Only one lepospondyl is present by this time (Diplocaulus minimus from Morocco), so vicariance is obviously not possible in this clade.\" - Late Carboniferous dispersal patterns - The dispersal and vicariance patterns of amphibians and amniotes indicate differing responses to the geological and climate changes occurring during the latter half of the Paleozoic. For much of the Carboniferous, the tropical regions were covered by the coal forests, a dense belt of tropical rainforests. The general climate trend throughout the Permian was towards warming and drying, albeit punctuated by wet phases relating to the waxing and waning of the Gondwana ice sheet. Gradually, the substantial polar icecap present since the Devonian disappeared, the tropical belt across the equator narrowed, and the arid zones surrounding it expanded. - \\'At the end of the Carboniferous, during the Moscovian stage, the rainforest collapse is thought to have occurred: the coal forests covering the equatorial regions were reduced to islands of rainforest in intramontane basins of the Variscan mountains surrounded by more arid habitats. This was accompanied by a period of mass extinction among plants and a shift from a lycopsid-dominated flora to a fern-dominated flora. The transition does not appear to have occurred at a consistent rate across the equatorial latitudes, with macrofloral and palynological evidence suggesting coal development continued into the earliest Kasimovian in American localities, but after this the last remnants of the lycopsid-dominated flora disappeared. Cleal et al., after thoroughly reviewing the sedimentological and palaeobotanical changes across Moscovian and Kasimovian Euramerica, suggested that increased tectonic activity in the Variscan orogeny altered drainage patterns in Euramerica, creating conditions less suitable for the lycopsid flora and driving the disappearance of the coal swamps.\\' - The family-level diversity curves of Sahney et al. indicated an increase in faunal provinciality following the Moscovian, which they attributed to the rainforest collapse. It was suggested that the habitat fragmentation (the separation of the coal swamps into rainforest islands) was responsible. In support of this hypothesis, they observed that amniotes were less severely affected by the environmental changes at this time, a result they attributed to the amniotic egg and the impermeable skin giving them independence from water. - The dispersal patterns of amniotes and amphibians identified by this study might appear to support the conclusions of Sahney et al.. Both amphibians and amniotes exhibit a fall in dispersal rate and a peak in vicariance rate during the latest Carboniferous. The differing patterns observed in amphibians and amniotes are also supported here. The reduction of dispersal rate in amphibians is more substantial, indicating a greater sensitivity to the changing climate. Moreover, although the aridification trends continued throughout the Permian, the dispersal rates of amniotes recovered while those of amphibians remained low. It might appear that amniotes were more efficient at dispersal in the drier, more open habitat than the amphibians. - Before assuming that the results of this study emphatically support the conclusions of Sahney et al., two caveats must be noted. First, it is not entirely clear to what extent the two are directly comparable, since they are examining biogeography at different scales; Sahney et al. were comparing the degree of endemism between formations and basins, while the results presented here illustrate dispersal patterns between continental-scale areas. Thus, our results do not provide any information on the habitat fragmentation model put forwards by Sahney et al., but instead illustrate the development of geographic or climate barriers between considerably larger regions. The second caveat to note is that the model of Sahney et al. makes less sense when viewed in the context of the more recent data regarding the rainforest collapse. Rather than an extended period of habitat fragmentation into the rainforest islands, detailed examination of the plant and pollen records indicate that the lycopsid-dominated coal swamps had disappeared by the earliest Kasimovian, thereafter replaced by fern-dominated, more open environments. Therefore, the period in which we should observe the increased endemism at the formation level is extremely short, restricted to the late Moscovian. By the Kasmovian, the habitats were more uniform and should not exhibit continued isolation and reduced dispersal between them. - \\'Despite these caveats, it is possible that the rainforest collapse may still have been responsible for the continental-scale results illustrated here. As mentioned above, the collapse was not synchronous across the continents, rather representing a contraction and progressive western shift. If the coal swamps disappeared in Europe prior to North America, there would be an environmental barrier between the two, potentially restricting dispersal and driving vicariance. This explanation, however, does not resolve the second issue described in the previous paragraph: why would the reduced dispersal of amphibians continue into the early Permian with no environmental heterogeneity to prevent it?\\' - An alternative explanation that would mitigate this objection might be the increased tectonic activity at the Variscan orogeny occurring during the Moscovian. This tectonic activity caused substantial uplift and terrestrial deformation, continuing into the Permian. This mountain-building phase would have generated a substantial physical barrier between the equatorial localities, restricting not only movement between Europe and North America, but also migration between Gondwana and Laurasia. Elsewhere during the Moscovian, further uplift and mountain-building was occurring, creating further barriers. These include the Uralides between Eastern Europe and Asia, and the Appalachides between North and South America. - A serious issue requiring discussion is the conflict between the results presented here and those found by Dunne et al.. The latter employed a networking approach to assess the biogeographic connectedness of localities during the Carboniferous and Cisuralian, implementing a correction to account for the phylogenetic non-independence of the taxa. They identified an increase in biogeographic connectedness between the Carboniferous and the Cisuralian, implying greater similarity of the faunas in the early Permian. This would seem to indicate increased dispersal across the boundary, the opposite signal to that observed here. - \\'There are a number of possible explanations for the conflicting results. One possibility might be the differing temporal resolution of the studies: Here, dispersal rates are calculated at the substage level, while the finest resolution at which Dunne et al. calculated biogeographic connectedness was in three bins, each containing two or three stages. However, this seems unlikely to be the cause of the discrepancy. The Gzhelian-Sakmarian dispersal rates identified here are consistently lower than those from earlier in the Carboniferous, but this interval has increased biogeographic connectedness in the analysis of Dunne et al..\\' - Another possible explanation is that the difference represents the different selection of regional divisions. Dunne et al. defined their bioregions using cluster analysis of palaeocoordinates of localities, and so both North America and Europe were divided into numerous bioregions. Here, North America was divided into only two regions divided by the Hueco Seaway, and Western Europe was treated as a single region. The larger areas employed in this study means that dispersal between smaller subregions would not be counted. In order to test this possibility, the BioGeoBEARS analysis was repeated, using the same regions employed by Dunne et al.. Since their study defines new bioregions following the Carboniferous-Permian boundary, a procedure not permitted by BioGeoBEARS, the supertree was time sliced to the end of the Carboniferous. This does not preclude the possibility of assessing the different results, as Dunne et al. noted an increase in biogeographic connectedness during the Gzhelian, which may still be tested. - \"This last analysis provided results that were more consistent with those obtained by Dunne et al. (Fig._). Dispersal rates were low from the Tournasian until the Serpukovian but rose throughout the latter stages of the Carboniferous, reaching a peak in the Gzhelian. That this result is obtained when both North America and Europe are divided into multiple subregions indicates that different biogeographic patterns are occurring at different scales. Dispersal between the smaller-scale regions appears to become easier towards the end of the Carboniferous, although between the larger regions it becomes more difficult. This supports the inference that the principal barriers to dispersal in the late Carboniferous were the physical barriers between continental-scale regions rather than the local environmental barriers. The island-biogeography effect posited by Sahney et al., with small, local islands of forest biogeographically isolated by open areas, is firmly rejected: dispersal between these local areas increased during the Gzhelian.\" - Fig. 4Carboniferous dispersal rates of all Tetrapoda, using the regional divisions of Dunne et al.. Thick lines represent the mean of the rates obtained from each stochastic mapping iteration. The thinner lines represent a standard error above and below - Late Permian dispersal patterns - \"While amphibian dispersal appears to have been restricted throughout the Permian, amniote dispersal increased throughout most of the Cisuralian. This has been noted previously; while the late Permian equatorial faunas contain amphibians characteristic of early Permian equatorial localities, the amniotes are more similar to those found in contemporary temperate regions. Interestingly, the eureptiles show the same increase in dispersal as is found in synapsids. This contradicts previous suggestions that Captorhinidae (the most diverse eureptile family during the Permian) were better suited to arid environments. It is only in the equatorial regions that this clade is abundant during much of the Permian; they do_not become abundant in the temperate regions until the Lopingian. However, while remaining rare in temperate regions, they were no less capable of dispersal than synapsids.\" - The first substantial decline in amniote dispersal does not occur until the late Capitanian, coinciding with a peak in vicariance rate. No significant change is observed in amphibians (perhaps simply due to them already being highly provincial). - \\'The causes of this sudden and massive reduction of movement in amniotes is unclear, and our understanding is hampered by the lack of information regarding climate and environmental changes at this time. Environmental upheaval perhaps linked to flood volcanism has been suggested, as has relative climatic stability throughout the Guadalupian and Lopingian. The nature and extent of the changes, and particularly their effect on terrestrial environments are particularly unclear. Data from plants is contradictory, with some suggestion of a substantial extinction at this time, but other suggestions that no substantial change in either diversity or floral composition occurred. It seems unlikely that the lack of inferred dispersal events is due to geographically restricted sampling: the Wuchiapingian and Changshingian are among the few Paleozoic time bins where data is known from both palaeotemperate and equatorial latitudes in both Laurasia and Gondwana. In fact, these stages contain a geographically wider sample than any other, with tetrapods known from all biogeographic regions under study except North America and northern South America.\\' - \"A recent study using isotope data from amniote bones from the Karoo provides an indication of environmental changes in the temperate regions during the Permian. The late Capitanian peak in u03B4C values was interpreted as an increase in water stress in plants i.e a period of substantially reduced humidity. By contrast, examinations of the geology and flora of contemporary palaeoequatorial formations in northern Africa and Europe found that the late Capitanian and early Wuchiapingian represented a wet phase interrupting the general trend towards aridification through the Permian. It is possible that the contrasting trends in climate change at the different latitudes restricted the possibility of dispersal between the two.\" - Conclusions - Past discussions of biogeographic patterns within the Paleozoic Pangaea, and the extent of dispersal or vicariance across the continent, have been hindered by the limited use of quantitative methodology in making historical biogeographic inferences. As such, there has been considerable disagreement regarding the provinciality and cosmopolitanism of the tetrapod faunas, as well as the potential dispersal routes across Pangaea. - \\'Quantitative analysis provides evidence of the existence of barriers to dispersal across Pangaea, and the timing of decreases in dispersal rates and peaks in vicariance rate provide evidence of their development. Two episodes of reduced dispersal are observed: in the late Carboniferous in amphibians and at the end of the Guadalupian in amniotes. Both dispersal troughs are accompanied by vicariance peaks. Although the nature of the new barriers must remain speculative for the present, the first shift in dispersal rate is attributed to increased mountain-building at that time, while the second is hypothesised to be due to climate barriers.\\' - \"To conclude, it is important to note that historical biogeographic analyses not only provide information on the evolutionary history of the group under study, but also provide an additional line of evidence regarding broader patterns of earths history. Where more usual lines of evidence regarding continental arrangements or the development and break down of dispersal routes e.g. the Pangaea B scenario, the Carthaysian Bridge as a route from Gondwana to Laurasia, the relationships of the different faunas to each other provide a vital line of evidence.\"- - Supertree construction - \"The basis for the biogeographic analysis was a formal supertree of early tetrapods, designed to maximise taxonomic inclusivity. The full list of 49 source trees, finalised in April 2017, and the criteria for their inclusion is present in Supplementary Data_ and Supplementary Note_, respectively. The source trees were combined using the method u201CMatrix Representation with Parsimonyu201D (MRP). Under this method, each node in each source tree is treated as a character. Taxa within a particular node in a particular source tree receive a score of u201C1u201D for the relevant character, taxa outside it receive a score of u201C0u201D, and taxa not included in that source tree are scored as u201C ?u201D. The resulting matrix may be analysed using parsimony. The MRP matrix was generated using the programme Supertree0.85b. The matrix was analysed in the Willi Hennig Society edition of TNT, using the driven search at level 100. The most parsimonious trees were searched for 100 times, and then a branch-and-bound search was carried out using each most parsimonious tree previously found as a start. The final supertree (the strict consensus) contains 593 terminal taxa, and is available in Supplementary Data_. A summary version is presented in Fig._. Most of the_terminal taxa are genera, but they_also include some as-yet unnamed specimens and some species-level relationships where these have been tested. Clades which are not known from the fossil record until after the Permian but have ghost lineages extending into the Paleozoic were kept in the tree during time calibration but were dropped during the biogeographic analyses.\" - Fig. 5Summary version of the supertree used in the analysis - \"The supertree was time calibrated using the method of Lloyd et al.. This method was itself based on an approach by Hedman, whereby the observed age of a node relative to its successive outgroups could be used to make inferences about sampling, and thereby assess how far back in time the node should be extended. Lloyd et al. modified this method to be applied to an entire phylogeny. The tree was time calibrated in R 3.3.2. In order to date the root node, stratigraphically consistent outgroups were required. In this case, the Devonian tetrapods Ymeria, Ichthyostega and Metaxygnathus were used. A maximum age constraint was placed on the root of 409.4 million years: the age of the split between tetrapods and lungfish inferred by a recent molecular clock study. To account for the uncertainty surrounding the relationships and ages of taxa, 100 time calibrated trees were generated. For each of these, polytomies in the tree were randomly resolved and age ranges of each taxon were drawn from a uniform probability distribution covering the full possible range in which the taxon could have lived. The age ranges employed are shown in Supplementary Data_. The analyses described below are carried out on all 100 trees.\" - BioGeoBEARS - \"To infer the timing and phylogenetic position of dispersal, vicariance and local extinction events, ancestral geographic ranges were inferred using a likelihood-based model comparison in the R package BioGeoBEARS. The functions in this package implement three different biogeographic models: DEC, DIVA and BayArea, each of these allowing a different combination and geographic extent of biogeographic events. The phylogeny and the geographic ranges of the tips were analysed using all three models, with the Akaike information criterion used to identify the best-fitting model. This was found to be the DIVA model (Supplementary Table_ ), which allows dispersal and local extinction along lineages, sympatric speciation of a lineage within a single area (but not sympatric speciation covering multiple region), and the vicariant origin of a species over a variety of range sizes.\" - \"The tetrapod-bearing formations of the Carboniferous and Permian were grouped into 13 bioregions. These were separated by a combination of physical barriers such as mountain ranges and internal seaways, and latitudinal lines intended to reflect climatic boundaries (see Supplementary Note_ for detailed descriptions of the regions). The regions are: western North America; eastern North America, northern South America, southern South America, western Europe, eastern Europe, eastern Asia, northern Africa, southern Africa, Madagascar, India, Australia and Antarctica (see Supplementary Data_ for the areas to which each taxon was assigned).\" - Treefitting and the area-adjacency matrix - \\'The biogeographic analyses carried out in BioGeoBEARS incorporate a matrix indicating which areas are adjacent to each other, thus informing the algorithm about possible dispersal routes. While the majority of the area-adjacency matrix could be inferred with little debate, there are contentious issues which will influence it, for example: (1) whether the Pangaea B scenario, whereby North America is positioned more westerly than usually depicted and western Europe is adjacent to Northern South America, should be followed; (2) whether dispersal should be allowed along the Cathaysian Bridge between eastern Gondwana (India and Australia) and Eastern Laurasia (East Asia); (3) whether it is necessary to treat the North American provinces on either side of the Hueco seaway as separate biogeographic regions.\\' - In order to answer such disputes, the supertree was subjected to a treefitting analysis. This analysis uses a phylogeny and a series of event costs to calculate the optimal biogeographic reconstruction (area cladogram) with the combination of events with minimum cost. - \"The treefitting analysis was carried out in Treefitter 1.3B1 using the heuristic search. The event costs used are: vicarianceu2009=u20090.01, sympatric speciationu2009 =u20090.01, extinctionu2009=u20091, dispersalu2009=u20092. This cost scheme, whereby the cost of vicariance and sympatric speciation are minimised, is designed to maximise the likelihood of finding phylogenetically conserved distribution patterns.\" - \"Upchurch and Hunn recommended time slicing (analysing taxa in each time bin separately) when using co-phylogeny reconstruction to analyse biogeography since area relationships change through time as barriers break down and new barriers develop. While these authors carried out time slicing by dropping all tips not present in a particular time slice, we modify this method by including tips with ghost lineages (unsampled portions of the fossil record that may be inferred from the phylogeny) extending into the time slice. The time slices tested were the Mississippian, Pennsylvanian, Cisuralian, Guadalupian and Lopingian. An area cladogram was calculated for each time slice (Supplementary Fig._).\" - The treefitting analyses were able to provide solutions to the three disputes discussed. In all area cladograms, the eastern North American region is more closely related to that of western Europe rather than western North America. It is therefore judged that the Hueco seaway as a significant barrier, while dispersal between eastern North America and western Europe was easier. - In none of the area cladograms does western Europe group with South America. While in most of the time bins the northern South American region is found with other Gondwanan regions, during the Mississippian it groups with western North America. These results argue strongly against the Pangaea B hypothesis; there is no evidence of close links between the faunas of South America and Europe. - \"Finally, there is strong evidence that the Cathaysian bridge has provided a link between the faunas of Gondwana and Laurasia. This is contra the suggestion of Cisneros et al., who argued that, since no Paleozoic tetrapod fossils have been found in the regions of East Asia which then formed the Cathaysian Archipelago (South China and Korea), tetrapod dispersal between Gondwana and Laurasia took place via western Pangaea. However, in all area cladograms with the exception of the Lopingian time slice, East Asia is found to group more closely with Gondwanan faunas of Southern Africa than with the Eastern European fauna as argued by Cisneros et al.. Thus, the lack of Paleozoic tetrapods in the Cathaysian regions is likely to be an artefact of preservation, and it is likely this was a frequent dispersal route between the South African and Eastern European regions, at least until the Guadalupian. The fact that in all Permian area cladograms the Eastern European region is found more closely related to the Gondwanan regions rather than western Europe would suggest that the Cathaysian Bridge was actually the preferred dispersal route between Gondwana and Laurasia, the Variscan Orogeny perhaps forming a more substantial barrier than suggested by Cisneros et al.. The area-adjacency matrix inferred from these results and used in the BioGeoBEARS analysis is presented in Supplementary Table_.\" - Founder-event speciations - Founder-event speciation, represented in BioGeoBEARS as speciation events where one of the two descendant lineages of a node remains in the ancestral area and the other jumps to a new area, was not permitted in the analyses in this paper. While it has been shown to be important in island clades, the model has been criticised for biasing towards cladogenetic dispersal events over anagenetic. As the model incorporates no speciation-rate parameters, cladogenetic events like founder-event speciation are assumed to take place at fixed nodes, rather than being an outcome of a time-dependant diversification process. Anagenetic dispersal events, on the other hand, are treated as a character evolving in a stochastic manner similar to the MK model, and are therefore time-dependant. This difference increases the probabilities of inferring cladogenetic events such as founder-event speciation over anagentic dispersal. A further criticism was of the j parameter, not as a rate of founder-speciation, but as a free parameter indicating the weight given to the easier mode of dispersal. A final issue is that the founder events are also allowed to effectively bypass the area-adjacency matrix. While this is reasonable in island clusters, it produces unrealistic results a supercontinent where, short of circumnavigating the continent, the only possible dispersal routes for terrestrial animals are via adjacent areas. - Stochastic mapping to infer dispersal and vicariance rates - \"The BioGeoBEARS analysis identifies the probability that a node in the phylogeny was present in a particular area or combination of areas. One might infer the geographic ranges of the ancestral nodes by assuming this is represented by the most range with the highest probability (Figs._u2013). However, a better alternative is to use these probabilities to infer possible biogeographic histories via a stochastic mapping approach: drawing ancestral regions using the probabilities inferred by the BioGeoBEARS analysis, and from these deducing the phylogenetic position and timing of biogeographic events required to obtain the observed biogeographic history.\" - \"Fig. 6Example of a reconstruction of ancestral geographic ranges of sauropsids. Tree was randomly selected from the 100 time calibrated trees. a Parareptilia. b Eureptilia. Node labels represent the geographic range of that node with the highest probability, deduced by BioGeoBEARS. au2009=u2009Western Europe; bu2009 =u2009Western North America; cu2009=u2009Eastern North America; du2009=u2009 Eastern Europe; eu2009=u2009East Asia; fu2009=u2009Northern South America; gu2009=u2009Northern Africa; hu2009=u2009Southern South America; iu2009 =u2009Southern Africa; ju2009=u2009Antarctica; ku2009=u2009Madagascar; lu2009 =u2009India; mu2009=u2009Australia\" - \"Fig. 7Example of a reconstruction of ancestral geographic ranges of Synapsids. Tree was randomly selected from the 100 time calibrated trees. a Pelycosaurian-grade synapsids, Biarmosuchia and Dinocephalia. b Neotherapsida. Node labels represent the geographic range of that node with the highest probability, deduced by BioGeoBEARS. au2009=u2009Western Europe; bu2009=u2009Western North America; cu2009=u2009 Eastern North America; du2009=u2009Eastern Europe; eu2009=u2009East Asia; fu2009=u2009Northern South America; gu2009=u2009Northern Africa; hu2009 =u2009Southern South America; iu2009=u2009Southern Africa; ju2009=u2009Antarctica; ku2009=u2009Madagascar; lu2009=u2009India; mu2009=u2009Australia\" - \"Fig. 8Example of a reconstruction of ancestral geographic ranges of amphibians. Tree was randomly selected from the 100 time calibrated trees. a Temnospondyli. b Lepospondyli. Node labels represent the geographic range of that node with the highest probability, deduced by BioGeoBEARS. au2009=u2009Western Europe; bu2009=u2009Western North America; cu2009=u2009Eastern North America; du2009 =u2009Eastern Europe; eu2009=u2009East Asia; fu2009=u2009Northern South America; gu2009=u2009Northern Africa; hu2009=u2009Southern South America; iu2009 =u2009Southern Africa; ju2009=u2009Antarctica; ku2009=u2009Madagascar; lu2009 =u2009India; mu2009=u2009Australia\" - For each of the 100 time calibrated phylogenies, 100 biogeographic histories were generated using stochastic mapping; thus 10,000 total evolutionary histories were tested. The timing of dispersal and vicariance events were extracted, and the number of each event in each time bin was counted. The time bins employed were informal substages, obtained by dividing each of the international stages in half. As each bin differs in length, the event counts of each bin were divided by the length of the bin in order to obtain a rate. - \"The raw numbers of dispersal and vicariance events in each time bin may be misleading as a measure of dispersal or vicariance rate, due to the variation in the number of lineages and nodes in each time bin (Supplementary Figs._u2013). If, for example, vicariance events were distributed at random across the nodes in the tree, time bins containing more nodes would exhibit a higher vicariance count simply by chance. Therefore, for each of the 10,000 stochastic maps, a null model was also generated by simulating 100 biogeographic histories over the time calibrated phylogeny. A starting area/range of areas was selected at random from the same set of regions used in the BioGeoBEARS analysis. With this starting area assigned to the root node of the relevant phylogeny, the biogeographic history would evolve stochastically, with the same biogeographic events permitted as were allowed by the DIVA model. The probabilities of dispersal, vicariance and local extinctions occurring at any point in time along a lineage/at a node taken from the results of the BioGeoBEARS analysis. The possible ranges of each node/lineage were constrained by the same parameters employed in the BioGeoBEARS analysis: the area-adjacency matrix limited the areas to which a lineage could disperse, and the regions employed could not be further subdivided i.e. a vicariance event could not occur at a node where the range covers only a single area. The R function to simulate the biogeographic history is provided in Supplementary Data_.\" - Having simulated the 100 null histories and calculated the mean dispersal and vicariance rates of each, the null rates were subtracted from the observed rates. Thus, a negative dispersal/vicariance rate would indicate that the observed rate is less than would be expected given a randomly evolved biogeographic history over the same phylogeny. - Code availability - \"All custom code used in this study is available in Supplementary Data_.\"- - \\' \\' - All data generated and analysed during this study are included in this published article (and its supplementary information files). A reporting summary for this article is available as a Supplementary Information file. - \\' \\'- - \"Coates, M. I., Ruta, M. & Friedman, M. Ever since Owen: changing perspectives on the early evolution of tetrapods. Annu. Rev. Ecol. Evol. Syst. 39, 571u2013 592 (2008)._ _ Olson, E. C. Community evolution and the origin of mammals. Ecology 47, 291u2013302 (1966)._ _ Colbert, E. H. Continental drift and the distributions of fossil reptiles. Proc. NATO Adv. Sci. Inst. 1, 395u2013412 (1973)._ Cracraft, J. Continental drift and vertebrate distribution. Annu. Rev. Ecol. Evol. Syst. 5, 215u2013261 (1974)._ _ Rubidge, B. S. & Hopson, J. A. A new anomodont therapsid from South Africa and its bearing on the ancestry of Dicynodontia. S. Afr. J. Sci. 86, 43u201345 (1990)._ Shubin, N. H. & Sues, H. D. Biogeography of early Mesozoic continental tetrapods: patterns and implications. Paleobiology 17, 214u2013230 (1991)._ _ Rubidge, B. S. New South African fossil links with the earliest mammal-like reptile (therapsid) faunas from Russia. S. Afr. J. Sci. 89, 460u2013461 (1993)._ Rubidge, B. S. Biostratigraphy of the Beaufort Group (Karoo Supergroup). Biostratigraphic Ser. 1, 1u201346 (1995)._ Sidor, C. A. & Welman, J. A second specimen of Lemurosaurus pricei (Therapsida: Burnetiamorpha). J. Vert. Paleontol. 23, 631u2013642 (2003)._ _ Smith, R. M. H., Rubidge, B. S. & Sidor, C. A. A new burnetiid (Therapsida: Biarmosuchia) from the upper Permian of South Africa and its biogeographic implications. J. Vert. Paleontol. 26, 331u2013343 (2006)._ _ Cisneros, J. C. et al. Carnivorous dinocephalian from the Middle Permian of Brazil and tetrapod dispersal in Pangaea. Proc. Natl Acad. Sci. 109, 1584u20131588 (2012)._ _ _ _ Tucker, M. E. & Benton, M. J. Triassic environments, climates and reptile evolution. Palaeogeogr. Palaeoclimatol. Palaeoecol. 40, 361u2013379 (1982)._ _ Modesto, S. P. & Rybczynski, N. in The Age of the Dinosaurs in Russia and Mongolia (eds Benton, M. J., Shishkin, M. A., Unwin, D. M., Kurochkin, E. N.) pp. 17u201335 (Cambridge Univeristy Press, Cambridge, 2000).Sidor, C. A. et al. Permian tetrapods from the Sahara show climate-controlled endemism in Pangaea. Nature 434, 886u2013889 (2005)._ _ _ _ Ezcurra, M. D. Biogeography of Triassic tetrapods: evidence for provincialism and driven sympatric cladogenesis in the early evolution of modern tetrapod lineages. Proc. R. Soc. B 277, 2547u2013 2552 (2010)._ _ Whiteside, J. H., Grogan, D. S., Olsen, P. E. & Kent, D. V. Climatically driven biogeographic provinces of Late Triassic tropical Pangea. Proc. Natl Acad. Sci. 108, 8972u20138977 (2011)._ _ _ _ Brusatte, S. L. et al. First record of Mesozoic terrestrial vertebrates from Lithuania: phytosaurs (Diapsida: Archosauriformes) of probable Late Triassic age, with a review of phytosaur biogeography. Geol. Mag. 150, 110u2013122 (2013)._ _ _ Sidor, C. A. The vertebrate fauna of the Upper Permian of Nigeru2013VIII. Nigerpeton ricqlesi (Temnospondyli: Cochleosauridae) and tetrapod biogeographic provinces. C. R. Palevol. 12, 463u2013472 (2013)._ _ Sidor, C. A. et al. Provincialization of terrestrial faunas following the end-Permian mass extinction. Proc. Natl Acad. Sci. 110, 8129u20138133 (2013)._ _ _ _ Brocklehurst, N., Day, M. O., Rubidge, B. S. & FrxF6bisch, J. Olsons extinction and the latitudinal biodiversity gradient of tetrapods in the Permian. Proc. R. Soc. B 284, 20170231 (2017)._ _ Button, D. J., Lloyd, G. T., Ezcurra, M. D. & Butler, R. J. Mass extinctions drove increased global faunal cosmopolitanism on the supercontinent Pangaea. Nat. Comm. 8, 733 (2017)._ _ _ Angielczyk, K. D. & Kurkin, A. A. Phylogenetic analysis of Russian Permian dicynodonts (Therapsida: Anomodontia): implications for Permian biostratigraphy and Pangaean biogeography. Zool. J. Linn. Soc. 139, 157u2013212 (2003)._ _ Modesto, S. P., Scott, D. M. & Reisz, R. R. A new parareptile with temporal fenestration from the Middle Permian of South Africa. Can. J. Earth Sci. 46, 9u201320 (2009)._ _ _ Liu, J., Rubidge, B. S. & Li, J. New basal synapsid supports Laurasian origin for therapsids. Acta Palaeontol. Pol. 54, 393u2013 400 (2009)._ _ Tsuji, L. A., MxFCller, J. & Reisz, R. R. Microleter mckinzieorum gen. et sp. nov. from the Lower Permian of Oklahoma: the basalmost parareptile from Laurasia. J. Syst. Palaeontol. 8, 245u2013255 (2010)._ _ Cleal, C. J. et al. Late Moscovian terrestrial biotas and palaeoenvironments of Variscan Euramerica. Neth. J. Geosci. 88, 181u2013 278 (2009)._ Kessler, J. L. L., Soreghan, G. S. & Wacker, H. Equatorial aridity in western Pangaea: Lower Permian loessite and dolomitic palaeosols in northeastern New Mexico. J. Sediment. Res. 71, 817u2013832 (2001)._ _ _ Rees, P. M. et al. Permian phytogeographic patterns and climate data/model comparisons. J. Geol. 110, 1u201331 (2002)._ _ _ _ Tabor, N. J. & Poulsen, C. J. Palaeoclimate across the Late Pennsylvanianu2013Early Permian tropical palaeolatitudes: a review of climate indicators, their distribution and relation to palaeophysiographic climate factors. Palaeogeogr. Palaeoclim. Palaeoecol. 268, 293u2013310 (2008)._ _ Izart, A. F. et al. Palaeoclimate reconstruction from biomarker geochemistry and stable isotopes of n-alkanes from Carboniferous and Early Permian humic coals and limnic sediments in western and eastern Europe. Org. Geochem. 43, 125u2013149 (2012)._ _ _ Roscher, M. & Schneider, J. W. Permo-Carboniferous climate: Early Pennsylvanian to Late Permian climate development of central Europe in a regional and global context. Geol. Soc. Lond. Spec. Publ. 265, 95u2013136 (2006)._ _ _ DiMichele, W. A., Tabor, N. J., Chaney, D. S. & Nelson, W. J. 2006. From wetlands to wet spots: environmental tracking and the fate of Carboniferous elements in Early Permian tropical floras. Geol. Soc. Spec. Pap. 399, 223u2013 248 (2006)._ DiMichele, W. A., Montanez., I. P., Poulsen, C. J. & Tabor, N. 2009. Climate and vegetational regime shifts in the late Paleozoic ice age earth. Geobiology 7, 200u2013226 (2009)._ _ _ Cascalesu2010MixF1ana, B. & Cleal, C. J. The plant fossil record reflects just two great extinction events. Terra Nova 26, 195u2013200 (2014)._ _ _ Cascales-MixF1ana, B., Diez, J. B., Gerrienne, P. & Cleal, C. J. A palaeobotanical perspective on the great end-Permian biotic crisis. Hist. Biol. 28, 1066u20131074 (2016)._ _ Sahney, S., Benton, M. J. & Falcon-Lang, H. J. Rainforest collapse triggered Carboniferous tetrapod diversification in Euramerica. Geology 38, 1079u20131082 (2010)._ _ _ Montanez, I. P. et al. CO2-forced climate and vegetation instability during Late Paleozoic deglaciation. Science 315, 87u2013 91 (2007)._ _ _ _ Dewey, J. F. Plate tectonics and the evolution of the British Isles. J. Geol. Soc. 139, 371u2013412 (1982)._ _ _ Badham, J. P. N. Strike-slip orogensu2014an explanation for the Hercynides. J. Geol. Soc. 139, 493u2013504 (1982)._ _ _ Rowley, D. B. et al. Carboniferous paleogeographic, phytogeographic, and paleoclimatic reconstructions. Int. J. Coal Geol. 5, 7u2013 42 (1985)._ _ Dunne, E. M. et al. Diversity change during the rise of tetrapods and the impact of the u2018Carboniferous rainforest collapse . Proc. R. Soc. B 285, 20172730 (2018)._ _ Modesto, S. P., Flear, V. J., Dilney, M. M. & Reisz, R. R. A large moradisaurine tooth plate from the Lower Permian of Texas and its biostratigraphic implications. J. Vert. Paleontol. 36, e1221832 (2016)._ _ Wignall, P. B. et al. Volcanism, mass extinction, and carbon isotope fluctuations in the Middle Permian of China. Science 324, 1179u20131182 (2009)._ _ _ _ Jerram, D. A. et al. Submarine palaeoenvironments during Emeishan flood basalt volcanism, SW China: implications for plumeu2013lithosphere interaction during the Capitanian, Middle Permian (u2018end Guadalupian ) extinction event. Palaeogeogr. Palaeoclimatol. Palaeoecol. 441, 65u201373 (2015)._ _ Bond, D. P. G. et al. The Middle Permian (Capitanian) mass extinction on land and in the oceans. Earth Sci. Rev. 102, 100u2013116 (2010)._ _ Davydov, V. Late Paleozoic Climate: Constraints on warming and cooling from the benthic foraminifera record. Joint Meeting of The Geological Society of America, Soil Science Society of America, American Society of Agronomy, Crop Science Society of America, Gulf Coast Association of Geological Societies with the Gulf Coast Section of SEPM Houston, Texas, (2008).Day, M. O. et al. When and how did the terrestrial mid-Permian mass extinction occur? Evidence from the tetrapod record of the Karoo Basin, South Africa. Proc. R. Soc. B 282, 20150834 (2015)._ _ Stevens, L. G., Hilton, J., Bond, D. P., Glasspool, I. J. & Jardine, P. E. Radiation and extinction patterns in Permian floras from North China as indicators for environmental and climate change. J. Geol. Soc. 168, 607u2013619 (2011)._ _ _ Rey, K. et al. Global climate perturbations during the Permo-Triassic mass extinctions recorded by continental tetrapods from South Africa. Gondwana Res. 37, 384u2013396 (2016)._ _ _ _ Hmich, D., Schneider, J. W., Saber, H., Voigt, S. & El Wartiti, M. New continental Carboniferous and Permian faunas of Morocco: implications for biostratigraphy, palaeobiogeography and palaeoclimate. Geol. Soc. Lond. Spec. Publ. 265, 297u2013 324 (2006)._ _ _ Baum, B. R. Combining trees as a way of combining data sets for phylogenetic inference, and the desirability for combining gene trees. Taxon 41, 1u201310 (1992)._ _ Ragan, M. A. Phylogenetic inference based on matrix representation of trees. Mol. Phylogenet. Evol. 1, 53u201358 (1992)._ _ _ Salamin, N., Hodkinson, T. R. & Savolainen, V. Building supertrees: an empirical assessment using the grass family (Poaceae). Syst. Biol. 51, 134u2013150 (2002)._ _ Goloboff, P. A., Farris, J. S. & Nixon, K. C. TNT, a free program for phylogenetic analysis. Cladistics 24, 774u2013778 (2008)._ _ Lloyd, G. T., Bapst, D. W., Friedman, M. & Davis, K. E. Probabilistic divergence time estimation without branch lengths: dating the origins of dinosaurs, avian flight and crown birds. Biol. Lett. 12, 20160609 (2016)._ _ Hedman, M. M. Constraints on clade ages from fossil outgroups. Paleobiology 36, 16u201331 (2010)._ _ R Development Core Team. R: a language and environment for statistical computing (R Foundation for Statistical Computing, Vienna, 2016).Betancur, R. R., OrtxED, G. & Pyron, R. A. Fossilu2010based comparative analyses reveal ancient marine ancestry erased by extinction in rayu2010finned fishes. Ecol. Lett. 18, 441u2013450 (2015)._ _ Matzke, N. J. Model selection in historical biogeography reveals that founder-event speciation is a crucial process in island clades. Syst. Biol. 63, 951u2013970 (2014)._ _ Ronquist, F. Dispersal-vicariance analysis: a new approach to the quantification of historical biogeography. Syst. Biol. 46, 195u2013 203 (1997)._ _ Landis, M. J., Matzke, N. J., Moore, B. R. & Huelsenbeck, J. P. Bayesian analysis of biogeography when the number of areas is large. Syst. Biol. 62, 789u2013804 (2013)._ _ Arthaud, F. & Matte, P. Late Paleozoic strike-slip faulting in southern Europe and northern Africa: Result of a right-lateral shear zone between the Appalachians and the Urals. Geol. Soc. Am. Bul. 88, 1305u20131320 (1977)._ _ u015EengxF6r, A. C. & Atayman, S. The Permian extinction and the Tethys: an exercise in global geology. Geol. Soc Am. Spec Paper 448 1u201397 (2009).Lucas, S. G. Global Permian tetrapod biostratigraphy and biochronology. Geol. Soc. Lond. Spec. Pub 265, 65u201393 (2006)._ _ _ Lucas, S. G. Traces of a Permian Seacoast: Prehistoric Trackways National Monument. (New Mexico Museum of Natural History and Science, Albuquerque, New Mexico, USA, 2011).Ronquist, F. Phylogenetic approaches in coevolution and biogeography. Zool. Scr. 26, 313u2013322 (1998)._ _ SanmartxEDn, I. & Ronquist, F. New solutions to old problems: widespread taxa, redundant distributions and missing areas in eventu2013based biogeography. Anim. Biodivers. Conserve 25, 75u201393 (2002)._ Upchurch, P. & Hunn, C. A. u201CTimeu201D: the neglected dimension in cladistics biogeography. Geobios 35, 277u2013286 (2002)._ _ Ree, R. H. & Sanmartin, I. Conceptual and statistical problems with the DECu2009+u2009J model of founder-event speciation and it comparison with DEC via model selection. J. Biogeogr. 45, 741u2013749 (2018). (In Press)Ree, R. H. & Smith, S. Maximum likelihood inference of geographic range evolution by dispersal, local extinction, and cladogenesis. Syst. Biol. 57, 4u201314 (2008).\"- - \"We are grateful to Richard J. Butler and Graeme T. Lloyd for their helpful comments and discussion. The research of N.B. and J.F. was funded by Deutsche Forschungsgemeinschaft grant number FR 2457/5-1 awarded to J.F. The research of E.M.D. and D.D.C. was funded by the European Unions Horizon 2020 research and innovation programme 2014u20132018 under grant agreement 637483 (ERC Starting Grant TERRA to Richard J. Butler).\"- - Authors and Affiliations - \"Department of Earth Sciences, University of Oxford, South Parks Road, Oxford, OX1 3AN, UKNeil BrocklehurstMuseum fxFCr Naturkunde, Leibniz-Institut fxFC r Evolutions- und BiodiversitxE4tsforschung, InvalidenstraxDFe 43, 10115, Berlin, GermanyNeil Brocklehurst_&_Ju04E7rg Fru04E7bischSchool of Geography, Earth and Environmental Sciences, University of Birmingham, Birmingham, B15 2TT, UKEmma M. Dunne_&_Daniel D. CashmoreInstitut fxFCr Biologie, Humboldt-UniversitxE4 t zu Berlin, InvalidenstraxDFe 42, Berlin, 10115, GermanyJu04E7rg Fru04E7bisch\" - \"AuthorsNeil BrocklehurstYou can also search for this author in _Emma M. DunneYou can also search for this author in _Daniel D. CashmoreYou can also search for this author in _Ju04E7rg Fru04E7bischYou can also search for this author in _\" - Contributions - N.B., E.M.D. and D.D.C. collected data. N.B. designed and carried out analyses. N.B., E.M.D., D.D.C. and J.F. contributed to the writing of the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution-ShareAlike 3.0 license. The license terms can be found on the following link:_._Map of Bratislava () is licensed Creative Commons Attribution-ShareAlike 4.0 license. The license terms can be found on the following link:__(Edited by Tamas M, Inkscape, 1.0beta1,_ ).\" - Characterization of the wastewater treatment plants (WWTPs) - \"This study focused on the incidence of selected SARS-CoV-2 genes in two different Slovak (Bratislava) WWTPs. The first one is located in Bratislavau2014Centrum that treats almost all wastewater from the significant part of Bratislava. The second one is in Bratislavau2014Petrzalka that treats wastewater from Petrzalka (Table ). WWTPs use a mechanical and biological stage (stage nitrification only); the produced sludge is digested and produced biogas energy is recovered.\" - Table 1 Basic characteristics of the WWTPs. - Sampling - \"Automatic sampler devices were used to collect raw wastewater samples at the inflow point of WWTPs. Monitoring of wastewater was carried out at Bratislavau2014 Centrum and Bratislavau2014Petrzalka from September 2020 until March 2021. During the sampling campaign more than 50 samples of influent were taken (Tables , ).\" - Table 2 CT values measured by RT-qPCR of 4 genes (ORF1ab, S, E, RdRp) of influent at WWTPs. ND represents dates with performed measurement but no detected positivity of the samples. NM represents dates with not performed measurement of the samples. - \"Samples with a volume of 50_ml were taken every 15_min for a period of 24_ h to obtain a daily representative sample (in total 4.8_l). The collected specimen was frozen within 2_h at u2212u200920_xB0C and kept at this temperature until processing. This procedure follows a protocol established in several previous studies to prevent the material from biodegradation.\" - Sample processing - \"50_ml of wastewater was centrifuged at 4700_g for 30_min, and then 45_ml of clear supernatant was subjected to ultracentrifugation 125,893_g for 90_ min. After ultracentrifugation, the pellet was resuspended in 1_ml of TRI reagent. RNA was isolated using Direct-zol RNA Miniprep (Zymo Research) according to the manufacturers instructions. Each sample was eluted in 100_xB5l of RNase-free water and RNA quantity was measured using NanoDrop ND-2000 (Nanodrop Inc.). A maximum of 8_xB5g of RNA per sample was concentrated by vacuum evaporation using Eppendorf Concentrator plus (Eppendorf).\" - SARS-CoV-2 detection using quantitative RT-qPCR - \"The concentrated sample was resuspended in 11_xB5l of RNase-free water. A RevertAid First Strand cDNA synthesis kit (ThermoFisher Scientific) in a total volume of 20_u03BCl was used for reverse transcription according to the manufacturer\\'s instructions. 2_xB5l of the reverse transcription reaction product was analyzed in one RT-qPCR reaction using PerfeCTa qPCR FastMix II, low ROX (VWR) and specific primers for E, RdRP (described by WHO) and ORF1ab, S (described by Sigma Aldrich) genes (Supplementary Table ). Initial denaturation was set to 10_min at 95_ xB0C followed by 45 cycles of denaturation (95_xB0C, 30_s), annealing (59_ xB0C, 30_s) and extension (72_xB0C, 30_s) using Stratagene Mx3005P (Agilent). Copies were estimated based on standard curves using synthetic SARS-CoV-2 RNA Control 1 (Australia/VIC01/2020, Genbank ID: MT007544.1).\" - RT-qPCR product confirmation by Sanger sequencing - RT-qPCR products of each measured gene were further confirmed by Sanger sequencing. Out of all samples 8 were chosen for the confirmation. Sanger sequencing of the amplicons was carried by Microsynth Austria GmbH (Vienna, Austria). Quality control of sequences were assessed and confirmed in the BLAST standard database of nucleotide collection for coronavirus detection. Only highly similar sequences were reported. Reports from sequencing and BLAST results are accessible in Supplementary_material_2. - Calculation of viral particles in sewage and mathematical modeling - Regression models (simple linear, Double Squared Root and Square root-Y logarithmic-X) have been used to describe the studied relations between the obtained data from wastewater and reported data related to the COVID-19 situation. The time-series analysis was used to compare the dependence between the wastewater time series and various time lags of the positive RT-qPCR tests (and Death cases) time series. To determine how well the time lags match up with the wastewater time series and in particular, at what point the best match (dependence with the highest R) occurs, the cross-correlation function was applied. A generalized additive model (GAM) was used to illustrate the smoothed curves of relevant time series. The one-way ANOVA described the differences between the means of the positive RT-qPCR tests in particular days of the week based on the Studentized range statistic, Tukey\\'s \\'Honest Significant Difference\\' method. - The method\\'s detection limit was calculated as the number of monitored inhabitants divided by minimal positive cases reported at the date with positive RT-qPCR wastewater analysis for SARS-CoV-2. - All statistical and mathematical operations were performed in the R environment and software program StatGraphics.- - Feces are the main contribution of SARS-CoV-2 presence in sewage, thus selected target genes were likewise detected by several research groups in feces or from rectal swabs. Detection of several genes is vital as the RNA virus in wastewater is exposed to many environmental factors. Therefore, RNA may be strongly disintegrated resulting in poor amplification and variable detection of individual viral particles. We focused on detecting four genes ORF1ab, S, E and RdRp gene capturing different regions of the virus. Similar target genes (S, RdRP, ORF1ab) were also selected by La Rossa et al.. On the contrary, other researchers were investigating the presence of 3 different regions of nucleocapsid (N) gene in wastewater. Currently, there is no consensus of targets recommended for detecting SARS-CoV-2 in sewage. - \"In our study 29 out of 52 analyzed wastewater samples (56%) were found to be positive. Table displays CT values of detected genes in both WWTPs at given dates. First positive results were obtained on 29 September with detected S gene at both WWTPs and ORF1ab gene at WWTP Bratislava-Petrzalka. In Bratislavau2014 Centrum lowest CT detected was 31.8, whereas in Bratislavau2014Petrzalka, it was 32.7 on the same date. At least one of the targets was detected since this date at chosen WWTPs. The most often detected target was the ORF1ab gene with positivity of 33% (22 out of 66) from all positively detected targets.\" - Further, amplicons generated by RT-qPCR from 8 samples were confirmed by Sanger sequencing. This was done to prevent false negative results often obtained by RT-qPCR analysis as wastewater contains various RNA fragments and cross-reacting molecules. 7 out of 8 passed quality control check and the amplicon sequences were aligned to coronavirus strain with 5 sequences confirming SARS-CoV-2 as first hits in the BLAST database (Supplementary table , Supplementary_material_2). - \"According to calibration curves, CTs presented in Table were recalculated to the number of viral particles per milliliter. To calculate the daily loads of SARS-CoV-2 in wastewater, viral particles per ml were then multiplied by daily influents measured at the WWTPs (Supplementary Table ). Then reported numbers of positive RT-qPCR tests and reported numbers of death cases were compared to the number of viral particles in wastewater in Bratislava (Fig._).\" - Figure 3Reported numbers of positive RT-qPCR tests and reported numbers of death cases versus wastewater viral particles data model in Bratislava. The monitored time period was from 6 September 2020 to 14th March 2021. - \"By interpolating the measured viral particles in the wastewater, we obtained a model useful for comparing the time series of positive RT-qPCR tests, death cases and viral particles (Fig._). Since the graph reveals apparent time shifts, we focused on the time lags between particular time series in our analysis.\" - The following Table describes the dependencies between the time lags of reported positive RT-qPCR tests or death cases and the number of viral particles found on individual days in the wastewater of Bratislava. - Table 3 The values of R and time lags for dependence between the number of viral particles in wastewater and reported positive RT-qPCR tests and death cases. - \"It can be seen that the values of R for dependence between viral particles in wastewater and reported positive RT-qPCR tests and death cases (Table , Supplementary Fig. , ) are better for the weekly time series than for the daily time series. This led us to focus our analysis on weekly time series. This decision is supported by the results of the ANOVA test where the significant difference among the means of the number of positive RT-qPCR tests in the particular days of the week was confirmed (p-valueu2009=u20090.0054, F-valueu2009=u20093.178, DFu2009 =u20096, Supplementary Fig. ). Therefore, we decided to use the time series of cumulative data with a 7_days period. The weekly time series of measured viral particles in wastewater reported numbers of positive RT-qPCR tests and reported numbers of death cases without/with the appropriate time lags are depicted in supplementary Fig. .\" - \"By analyzing the dependence between the weekly time series of measured viral particles in the wastewater and reported numbers of positive RT-qPCR tests with a time lag of 2_weeks, the models for estimation of the number of positive RT-qPCR tests were obtained. The results of fitting a linear (Ru2009=u2009 81.72%, p-valueu2009<u20090.0001, F-valueu2009=u2009116.2, DFu2009=u2009 26, Supplementary Fig. ) and double square root (Ru2009=u200983.78%, p-valueu2009 <u20090.0001, F-valueu2009=u2009134.3, DFu2009=u200926, Supplementary Fig. ) models describe the relationship between the weekly time series of measured viral particles in the wastewater and reported numbers of positive RT-qPCR tests with the time lag of 2_weeks. The equations of the fitted models are:\" - \\'Linear:\\' - $$positive,PCR,lag2 = 76.578 + 0.0000135226 cdot viral,particles$$ - \\'Double square root:\\' - $$positive,PCR,lag2 = {(7.23532+ 0.00233866 .sqrt{viral,particles})}^{2}$$ - The models allow estimating the number of positive RT-qPCR tests with model fitting more than 80% (Figure ). - \"By analyzing the dependence between the weekly time series of measured viral particles in the wastewater and reported numbers of death cases with a time lag of 4_weeks, the models for estimation of the number of death cases were also obtained. The results of fitting a linear (Ru2009=u200948.06%, p-valueu2009 <u20090.0001, F-valueu2009=u200924.06, DFu2009=u200926, Supplementary Fig. ) and square root-Y logarithmic-X (Ru2009=u200983.21%, p-valueu2009<u2009 0.0001, F-valueu2009=u2009128.83, DFu2009=u200926, Supplementary Fig. ) models describe the relationship between the weekly time series of measured viral particles in the wastewater and reported numbers of death cases with the time lag of 4_ weeks. The values of R show that the non-linear model is significantly better than the linear. The equations of the fitted models are:\" - \\'Linear: \\' - $$deaths,lag4 = 2.71168 + 0.000000326376 cdot viral,particles$$ - \\'Square root-Y logarithmic-X:\\' - $$deaths,lag4 = {(-1.75359 + 0.26318 cdot ln (viral,particles))}^{2}$$ - The non-linear model allows estimating the number of death cases with model fitting more than 80% (Supplementary Fig. ). - \"Further, we looked at the detection limit of wastewater monitoring related to the number of positive RT-qPCR cases within a 12_days shift. Individually, the detection limit of WWTP Bratislavau2014Vrakuna is 1 RT-qPCR positive case per 25,000 people on 15/3/2021, while Bratislavau2014Petrzalka reached 1 per 4,808 on 19/1/2021. The results are also limited by reporting the positive RT-qPCR cases as separate data for each city district has been available since 10/11/2020. For Bratislava the detection limit was 1 per 8,099 on 29/9/2020 when both WWTPs were combined. It is important to stress that the results are related and strongly biased to reported RT-qPCR positive cases which vary between the countries, their testing capacities and contact tracing. In comparison, the study by Ahmed et al., could not detect SARS-CoV-2 in wastewater until 100 reported cases per 100,000 people. On the contrary, Medema et al., was capable of detecting SARS-CoV-2 in the wastewater before the first confirmed results in the monitored area. Another reason for such contrasting results is probably the different quality of the wastewater and various methodological approach influencing the detection limit. Each sewer has unique properties such as temperature, pH, presence of chemical substances and biological composition. All of these factors play a role in the degradation of viral particles. Therefore investigation of these properties is crucial and each methodological approach to detect SARS-CoV-2 from wastewater is unique for a given sewer system. However, some unification of the methodology would be vital for future comparison of the results.\" - \"Until now there was no confirmed case of infection from wastewater thus oral-fecal route is unlikely. The reason may be the inactivation of SARS-CoV-2 by gastrointestinal fluids or an unfavorable wastewater environment. The amount of the virus shed to the feces may vary in time as well as between the patients. Once the virus or its particles enter the sewage system, it is diluted by other types of water (industrial, rainfalls, water from snow melting, etc.) and is exposed to various physical and chemical factors. So far, viral RNA appears to be stable until reaching the primary settling tank of WWTP. Compared to non-enveloped viruses, SARS-CoV-2 has an affinity to wastewater solids, so a portion of the virus is probably sorbed on sewage walls and later to primary sludge. On the contrary, wastewater surveillance offers several benefits once we have a working monitoring system. In September 2020, Larsen and Wigginton published in Nature Biotechnology that theoretically, wastewater surveillance can be in 7_days lead before rapid diagnostics tests and 13u201315_days before delayed diagnostic tests. This is in agreement with our observations where results from wastewater dated to the day of sampling were in 14_days (2_weeks) lead before standard clinical RT-qPCR testing. If we add 1u20132_days of sample pretreatment and analysis we are reaching for 12u201313_days advance before reported positive RT-qPCR tests. Novelty in the presented study is that a similar correlation can be found with deaths reported in 28_days (4_weeks) after the detected increase of viral particles in the wastewater. Although we can not specify the number of infected persons, observation of trends in near real-time can help us understand community transmission. In communities with low capacity of clinical testing and delays of diagnostics wastewater surveillance can be a temporary solution. Moreover, as it is relatively cost-effective and less invasive, wastewater monitoring can be used in low-income countries. However in communities with working traditional testing, data obtained from wastewater are only additional information in controlling pandemics. They can be used to check the reliability of the introduction of novel technologies or protocols.\" - For successful monitoring of SARS-CoV-2 in the wastewater on a larger scale, public authorities must realize the importance of possible scalability of the method. National agencies should show interest and provide financial and material support for research teams and water companies in the form of grants. Monitoring of COVID-19 and its mutations will require modification of the methods when compared to the monitoring of drugs or metabolites. Optimistic presumption for successful cooperation is the fact that across Europe there is a working consortium of institutes and universities focusing on monitoring illegal drugs in the wastewaters. European Commission at the beginning of March created the HERA incubator. One of the focuses is the systematic surveillance of the wastewaters, including genetic sequencing at WWTPs with a connected population of over 150,000 individuals. - In the presented study, we detected RNA SARS-CoV-2 in wastewaters and displayed mathematical correlations between tested wastewater samples, positive RT-qPCR tests and death cases in Bratislava, Slovakia. The obtained results and subsequent mathematical modeling will be able to serve in the future as an early warning system for the occurrence of a local site of infection and at the same time will allow to predict the load on the health system up to two weeks in advance. Because each wastewater has its own characteristics (pH, temperature, (bio)chemical composition, etc.), it is necessary to approach this in other monitored localities when taking and processing samples, evaluating the results and creating the appropriate mathematical model.- - \"Larsen, D. A. & Wigginton, K. R. Tracking COVID-19 with wastewater. Nat. Biotechnol. 38, 1151u20131153 (2020)._ _ _ Xagoraraki, I. & OBrien, E. Wastewater-based epidemiology for early detection of viral outbreaks. In Women in Water Quality: Investigations by Prominent Female Engineers (ed. OBannon, D. J.) 75u201397 (Springer International Publishing, 2020). ._ _ Ort, C. et al. Spatial differences and temporal changes in illicit drug use in Europe quantified by wastewater analysis. Addict. Abingdon Engl. 109, 1338u20131352 (2014)._ _ Thomas, K. V. et al. Comparing illicit drug use in 19 European cities through sewage analysis. Sci. Total Environ. 432, 432u2013439 (2012)._ _ _ _ Wastewater-based epidemiology and drugs topic page | . .Prado, T. et al. Wastewater-based epidemiology as a useful tool to track SARS-CoV-2 and support public health policies at municipal level in Brazil. Water Res. 191, 116810 (2021)._ _ _ WHO Coronavirus (COVID-19) Dashboard | WHO Coronavirus (COVID-19) Dashboard With Vaccination Data. .Chen, J. et al. Clinical progression of patients with COVID-19 in Shanghai, China. J. Infect. 80, e1u2013e6 (2020)._ _ _ Chen, L., Lou, J., Bai, Y. & Wang, M. COVID-19 disease with positive fecal and negative pharyngeal and sputum viral tests. Am. J. Gastroenterol. 115, 790 (2020)._ _ Oran, D. P. & Topol, E. J. Prevalence of asymptomatic SARS-CoV-2 infection. Ann. Intern. Med. 173, 362u2013367 (2020)._ _ Yang, R., Gui, X. & Xiong, Y. Comparison of clinical characteristics of patients with asymptomatic vs symptomatic coronavirus disease 2019 in Wuhan, China. JAMA Netw. Open 3, e2010182 (2020)._ _ Yeo, C., Kaushal, S. & Yeo, D. Enteric involvement of coronaviruses: Is faecal-oral transmission of SARS-CoV-2 possible?. Lancet Gastroenterol. Hepatol. 5, 335u2013337 (2020)._ _ Xiao, F. et al. Evidence for gastrointestinal infection of SARS-CoV-2. Gastroenterology 158, 1831-1833.e3 (2020)._ _ _ Summary of probable SARS cases with onset of illness from 1 November 2002 to 31 July 2003. .Xu, Y. et al. Characteristics of pediatric SARS-CoV-2 infection and potential evidence for persistent fecal viral shedding. Nat. Med. 2, 1u20134. (2020)._ _ _ Zang, R. et al. TMPRSS2 and TMPRSS4 promote SARS-CoV-2 infection of human small intestinal enterocytes. Sci. Immunol. 5, 2 (2020)._ _ Cheung, K. S. et al. Gastrointestinal manifestations of SARS-CoV-2 infection and virus load in fecal samples from a hong kong cohort: Systematic review and meta-analysis. Gastroenterology 159, 81u201395 (2020)._ _ _ Wu, Y. et al. Prolonged presence of SARS-CoV-2 viral RNA in faecal samples. Lancet Gastroenterol. Hepatol. 5, 434u2013435 (2020)._ _ Foladori, P. et al. SARS-CoV-2 from faeces to wastewater treatment: What do we know?. A review. Sci. Total Environ. 743, 140444 (2020)._ _ _ _ PxE9 rez-CataluxF1a, A. et al. Detection Of Genomic Variants Of SARS-CoV-2 Circulating In Wastewater By High-Throughput Sequencing. (2021).BrandeburovxE1, P. et al. Wastewater-based epidemiology to assess the occurrence of new psychoactive substances and alcohol consumption in Slovakia. Ecotoxicol. Environ. Saf. 200, 110762 (2020)._ _ R: The R Project for Statistical Computing. .Chen, Y. et al. The presence of SARS-CoV-2 RNA in the feces of COVID-19 patients. J. Med. Virol. 92, 833u2013840 (2020)._ _ _ Tim, B. et al. An alternative approach for bioanalytical assay development for wastewater-based epidemiology of SARS-CoV-2. MedRxiv (2021)._ _ La Rosa, G. et al. First detection of SARS-CoV-2 in untreated wastewaters in Italy. Sci. Total Environ. 736, 139652 (2020)._ _ _ Randazzo, W. et al. SARS-CoV-2 RNA in wastewater anticipated COVID-19 occurrence in a low prevalence area. Water Res. 181, 115942 (2020)._ _ _ Wikramaratna, P. S., Paton, R. S., Ghafari, M. & LourenxE7o, J. Estimating the false-negative test probability of SARS-CoV-2 by RT-PCR. Euro Surveill. Bull. Eur. Sur Mal. Transm. Eur. Commun. Dis. Bull. 25, 2 (2020)._ Hata, A., Hara-Yamamura, H., Meuchi, Y., Imai, S. & Honda, R. Detection of SARS-CoV-2 in wastewater in Japan during a COVID-19 outbreak. Sci. Total Environ. 758, 143578 (2021)._ _ _ _ Medema, G., Heijnen, L., Elsinga, G., Italiaander, R. & Brouwer, A. Presence of SARS-coronavirus-2 RNA in sewage and correlation with reported COVID-19 prevalence in the early stage of the epidemic in The Netherlands. Environ. Sci. Technol. Lett. (2020)._ _ _ Mohan, S. V., Hemalatha, M., Kopperi, H., Ranjith, I. & Kumar, A. K. SARS-CoV-2 in environmental perspective: Occurrence, persistence, surveillance, inactivation and challenges. Chem. Eng. J. 405, 126893 (2021)._ _ _ Tran, H. N. et al. SARS-CoV-2 coronavirus in water and wastewater: A critical review about presence and concern. Environ. Res. 193, 110265 (2021)._ _ _ Peccia, J. et al. Measurement of SARS-CoV-2 RNA in wastewater tracks community infection dynamics. Nat. Biotechnol. 38, 1164u20131167 (2020)._ _ _ Rooney, C. M., Moura, I. B. & Wilcox, M. H. Tracking COVID-19 via sewage. Curr. Opin. Gastroenterol. 37, 4u20138 (2021)._ _ _ de Aguiar-Oliveira, M. et al. Wastewater-based epidemiology (WBE) and viral detection in polluted surface water: A valuable tool for COVID-19 surveillanceu2014A brief review. Int. J. Environ. Res. Public. Health 17, 2 (2020)._ _ Adelodun, B., Ajibade, F. O., Ibrahim, R. G., Bakare, H. O. & Choi, K.-S. Snowballing transmission of COVID-19 (SARS-CoV-2) through wastewater: Any sustainable preventive measures to curtail the scourge in low-income countries?. Sci. Total Environ. 742, 140680 (2020)._ _ _ _ NORMAN u2013 SCORE joint initiative to facilitate data comparison between u201CSARS-CoV-2 in sewageu201D studies | NORMAN. .Lescure, F.-X. et al. Clinical and virological data of the first cases of COVID-19 in Europe: a case series. Lancet Infect. Dis. 20, 697u2013706 (2020)._ _ _ Zheng, S. et al. Viral load dynamics and disease severity in patients infected with SARS-CoV-2 in Zhejiang province, China, January-March 2020: retrospective cohort study. BMJ 369, 1443 (2020)._ _ HERA Incubator. European Commission - European Commission. .\"- - \"We would like to thank Bratislava Water Company BVS a.s. This article was supported by the Slovak Research and Development Agency under project numbers APVV-17-0183, APVV-19-0250 and PP-COVID-20-0019. \"This article was written thanks to the generous support under the Operational Program Integrated Infrastructure for the project: \"Strategic research in the field of SMART monitoring, treatment and preventive protection against coronavirus (SARS-CoV-2) \", Project no. 313011ASS8, co-financed by the European Regional Development Fund.\" \"This article was financially supported by project VIR-SCANu2014Wastewater monitoring data as an early warning tool to alert COVID-19 in the population (\"EOSCsecretariat.eu has received funding from the European Union\\'s Horizon Program call H2020-INFRAEOSC- 05-2018-2019, grant Agreement number 831644.\").\"- - Author notes - \"These authors contributed equally: Nau010Fa Krivou0148xE1kovxE1, Andrea u0160 oltxFDsovxE1, Michal TamxE1u0161 and Zdenko TakxE1u010D.\" - Authors and Affiliations - \"Department of Environmental Engineering, Faculty of Chemistry and Food Technology, Slovak University of Technology in Bratislava, RadlinskxE9ho 9, 812 37, Bratislava, Slovak RepublicMichal TamxE1u0161,_Miroslav FehxE9r,_Anna KrivjanskxE1 ,_Ivana HorxE1kovxE1,_Noemi Beliu0161ovxE1,_Paula BxEDmovxE1,_Andrea Butor u0160kulcovxE1_&_TomxE1u0161 Mackuu013EakDepartment of Molecular Biology, Faculty of Natural Sciences, Comenius University, Ilkoviu010Dova 6, 842 15, Bratislava, SlovakiaAndrea u0160oltxFDsovxE1,_JxE1n Krahulec_ &_Andrej FicekInstitute for Clinical and Translational Research, Biomedical Research Center, Slovak Academy of Sciences, DxFAbravska Cesta 9, 84505, Bratislava, SlovakiaAndrea u0160oltxFDsovxE1Department of Inorganic Technology, Faculty of Chemical and Food Technology, Slovak University of Technology in Bratislava, RadlinskxE9ho 9, 812 37, Bratislava, SlovakiaMiroslav GxE1lInstitute of Information Engineering, Automation, and Mathematics, Department of Mathematics, Faculty of Chemical and Food Technology, Slovak University of Technology in Bratislava, RadlinskxE9ho 9, 812 37, Bratislava, SlovakiaNau010Fa Krivou0148xE1kovxE1 ,_Zdenko TakxE1u010D_&_MarixE1n Gall\" - \"AuthorsNau010Fa Krivou0148xE1kovxE1You can also search for this author in _Andrea u0160oltxFDsovxE1You can also search for this author in _Michal TamxE1u0161You can also search for this author in _Zdenko TakxE1u010DYou can also search for this author in _JxE1n KrahulecYou can also search for this author in _Andrej FicekYou can also search for this author in _Miroslav GxE1lYou can also search for this author in _MarixE1n GallYou can also search for this author in _Miroslav FehxE9 rYou can also search for this author in _Anna KrivjanskxE1 You can also search for this author in _Ivana HorxE1 kovxE1You can also search for this author in _Noemi Beliu0161ovxE1You can also search for this author in _Paula BxEDmovxE1You can also search for this author in _Andrea Butor u0160kulcovxE1You can also search for this author in _TomxE1u0161 Mackuu013EakYou can also search for this author in _\" - Contributions - \"T.M, M.T., M.Gal wrote the main manuscript text and created graphical abstract N.K., Z.T., M.Gall created the mathematical modeling, computational code and appropriate figures.A.F., J.K., A.So. performed RNA isolation and RT-qPCR analysisA.Sk., N.B., I.H., A.K., M.F. collected wastewater samples._All authors revised the manuscript.\" - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Attribution-ShareAlike 4.0 International License.\\' - \"The Joinpoint analyses for ASIR in males and females are shown in Table and Fig._\" - \"Table 3 Joinpoint analysis for KC age-standardized incidence rates in EU15u2009 +u2009countries and WHO regions for years 1990u20132019 in males (A), females (B). \" - \"Figure 3Trends in age-standardized incidence rates (ASIR) per 100,000 for KC in EU15u2009+u2009countries and WHO regions between 1990 and 2019. Open squares indicate males; and filled circles, females.\" - \"Global ASIR for males increased between 1990 and 1994 (EAPC 2.6 [2.1u20133.2]), but from 1994 to 1997, the change was not statistically significant (EAPC: 0.8 [u2212_1.0 to 2.5]). Between 1997 and 2009, the ASIR increased again (EAPC; 1.6 [1.4u20131.7]), and since 2009, the trajectory in ASIR has been mostly flat (EAPC 0.1 [u2212_0.1 to 0.2]). Amongst EU15u2009+u2009countries, recent trends have been mostly flat or negative. Most recently, the greatest increase has been observed in the UK between 2006 and 2019 (EAPC: 0.6 [0.5u20130.7]). The greatest decrease has been observed in Norway between 2011 and 2019 (EAPC: u2212_2.1 [u2212_2.6 to u2212_1.5]).\" - \"The rate of increase in ASIR has been variable amongst the WHO regions. Between 2011 and 2019, the Eastern Mediterranean region observed the greatest increase in ASIR (EAPC: 3.1 [2.9u20133.3]). Conversely, the Americas have seen the slowest rate of increase between 2002 and 2019 (EAPC: 0.1 [0.1u20130.2]).\" - \"Global ASIR for females increased between 1990 and 1993 (EAPC; 1.5 [0.8u2013 2.3)], the rate of growth slowed between 1993 and 2009 (EAPC; 0.7 [0.6u20130.7]). Between 2009 and 2014, ASIR decreased (EAPC; u2212_0.7 [u2212_1.1 to u2212 _0.2]), while the curve has been flat between 2014 and 2019 (EAPC; 0.0 [u2212 _0.3 to 0.4]). As with males, recent ASIR trends in females have mostly been flat or negative. The observed increase in Australia between 2017 and 2019 (EAPC: 1.1 [u2212_2.6 to 4.9]) was statistically non-significant. The greatest decrease has been observed in Austria between 1990 and 2019 (EAPC: u2212_0.9 [u2212 _1 to u2212_0.9]).\" - \"Recent ASIR trends in WHO regions for females have mostly been positive. The Eastern Mediterranean region has seen the greatest increase between 1999 and 2019 (EAPC: 2.6 [2.5u20132.6]). The Americas are the only region to see a fall recently, between 2003 and 2019 (EAPC: u2212_0.5 [u2212_0.5 to u2212_ 0.4]).\" - \"Trends in KC ASMR, 1990u20132019\" - \"Across the entire study period, there was a rise in ASMR globally for males (+u2009 19.4%) and a fall for females (u2212_2.3%). ASMR mainly increased across most WHO regions. The Eastern Mediterranean region had the greatest increase across the study period for males and females,u2009+u200987.8% andu2009+u200943.0%, respectively. The Americas had the smallest increase for males (+u20098.3%) and was the only region to see a decrease in females (u2212_16.4%). Recently, the Eastern Mediterranean region observed the greatest increase between 2011 and 2019 (EAPC: 2.3 [2.1u20132.4]). The Western Pacific is the only region to observe a decrease recently, between 2011 and 2019 (EAPC: u2212_0.4 [u2212 _0.7 to u2212_0.2]).\" - \"Among EU 15u2009+u2009countries, Denmark had the greatest increase in ASMR across the study period for males and females,u2009+u200941.7% and 37.7%, respectively. Conversely, Austria had the greatest decrease for males and females, u2212_33.8% and u2212_35.8% respectively.\" - \"Tables and and, Figs._, depict gender-specific trends in KC ASMR across EU 15u2009+u2009countries and 6 WHO regions.\" - \"The Joinpoint analyses for ASMR in males and females are shown in Table and Fig._\" - \"Table 4 Joinpoint analysis for KC age-standardized mortality rates in EU15u2009 +u2009countries and WHO regions for years 1990u20132019 in males (A), females (B) and both sexes (C).\" - \"Figure 4Trends in age-standardized mortality rates (ASMR) per 100,000 for KC in EU15u2009+u2009countries and WHO regions between 1990 and 2019. Open squares indicate males; and filled circles, females.\" - \"Global ASMR for males increased between 1990 and 1993 (EAPC: 1.9 [1u20132.8]). Between 1993 and 2009, the rate of increase slowed (EAPC: 0.9 [0.8u20130.9]). Between 2009 and 2019, there has been a non-significant decrease in ASMR (EAPC: u2212_0.2 [u2212_0.3 to 0]).\" - \"Amongst EU 15u2009+u2009countries, recent trends have mostly been negative, except in the USA, in which there has been a recent growth in ASMR between 2014 and 2019 (EAPC 0.7 [0.2u20131.1]). Recently, Norway has observed the greatest decrease between 2012 and 2019 (EAPC: u2212_2.2 [u2212_2.6 to u2212_1.8]). WHO regions have mostly seen an increase in ASMR recently.\" - \"Global, ASMR for females increased between 1990 and 1994 (EAPC: 0.9 [0.5u2013 1.2]). The trend reversed but was statistically non-significant between 1994 and 1997 (EAPC: u2212_0.4 [u2212_1.5 to 0.6]). Again, there was an increase in ASMR between 1997 and 2006 (EAPC: 0.3 [0.1u20130.4]). The trend between 2006 and 2019 showed an improvement in mortality (EAPC: u2212_0.6 [u2212_ 0.6 to u2212_0.5]). The South-East Asian region has seen the largest increase between 2012 and 2019 (EAPC: 1.5 [1.2u20131.8]). Europe was the only region with negative growth recently, between 2005 and 2019 (EAPC: u2212_0.2 [u2212 _0.3 to u2212_0.2]).\" - \"Amongst EU 15u2009+u2009countries, recent trends have been mixed. The greatest increase but statistically non-significant was observed in the USA between 2017 and 2019 (EAPC: 1 [0u20132]). The greatest decrease was observed in Denmark between 1998 and 2019 (EAPC: u2212_1.6 [u2212_1.7 to u2212_1.5]). WHO regions have mostly seen an increase in ASMR recently.\" - \"Trends in KC disability-adjusted life-years, 1990u20132019\" - \"Across the 29-year study period, there was a rise in DALYs globally for males (+u200913.1%) and a fall for females (u2212_9.4%). Similar trends were seen across all six regions as well. The Eastern Mediterranean region saw the greatest increase for males (+u200976.6%), whereas the Americas saw the smallest rise (+u20092.1%). Apart from the Americas (u2212_23.5%), all regions saw a rise in DALYs for females, led by the Eastern Mediterranean region (+u200936.3%).\" - \"In the EU15u2009+u2009countries, Denmark saw the greatest rise in DALYs for both males and females,u2009+u200938.6% andu2009+u200930.2 respectively. Austria saw the greatest decrease in DALYs across the study period for males and females, u2212_39.5% and u2212_41.2%, respectively. Figures_ and depict gender-specific trends in KC DALYs across EU 15u2009+u2009countries and 6 WHO regions.\" - \"Tables and and, Figs._ and depict gender-specific trends in KC DALY across EU 15u2009+u2009countries and 6 WHO regions.\" - \"The Joinpoint analyses for DALYs in males and females are shown in Table and Fig._\" - \"Table 5 Joinpoint analysis for KC age-standardized DALYs in EU15u2009+u2009 countries and WHO regions for years 1990u20132019 for both sexes.\" - \"Figure 5Trends in disability-adjusted life years (DALYs) per 100,000 for KC in EU15u2009+u2009countries and WHO regions between 1990 and 2019. Open squares indicate males; and filled circles, females.\" - \"Globally, DALYs for males were increasing from 1990 to 1994 (EAPC: 1.8 [1.3u2013 2.2]), followed by a slight decrease from 1994 to 1998 (EAPC: u2212_0.1 [u2212 _0.9 to 1.4]) and an increase between 1998 to 2005 (EAPC: 1.2 [0.9u20131.4]). Since 2005, DALYs have plateaued (EAPC: u2212_0.2 [u2212_0.2 to 0.1]).\" - \"Amongst EU15u2009+u2009countries, recent trends have mainly shown reductions, with the greatest decrease amongst males in Ireland from 2009 to 2019 (EAPC: u2212_1.5 [u2212_1.8 to 1.1)]. The greatest rise recently was seen in the USA from 2013 to 2019 (EAPC: 0.5 [0.1u20130.8]).\" - \"Trends have been mixed amongst WHO regions recently. The Eastern Mediterranean region has had the greatest increase recently, between 2000 and 2019 (EAPC; 1.9 [1.9u20132.0]). Meanwhile, the Western Pacific region had the greatest decrease recently, between 2011 and 2019 (EAPC; u2212_0.5 [u2212_0.8 to u2212_0.1]). Also, trends across WHO regions have mainly been non-significantly positive recently. The greatest recent rise was seen in South-East Asia between 2012 and 2019 (EAPC: 1.4 [1.1u20131.7]). The Americas were the only region with a decrease, albeit not significant (EAPC; u2212_0.1 [u2212_0.3 to u2212 _0.1]).\" - \"Similar DALYs trends were seen globally for females. Between 1990 and 1994, there was a moderate increase (EAPC: 0.4 [u2212_0.1 to 0.9)]. Since 1994, there has been a non-significant decrease till 2005. Since 2005, there has been a significant decrease (EAPC: u2212_0.7 [u2212_0.7 to u2212_0.6]). Amongst EU15u2009+u2009countries, recent trends have been mostly negative for males. The most recent significant increase was observed in Australia between 2016 and 2019 (EAPC: 0.9 [0.3u20131.6]). The greatest decrease was observed in Belgium between 1999 and 2019 (EAPC: u2212_1.3 [u2212_1.5 to u2212_1.1]).\" - \"Trends in KC MIR, 1990u20132019\" - \"Across the study period, there was a decrease in MIR globally for both males (u2212_13.7%) and females (u2212_13.9%). All EU 15u2009+u2009countries saw a decrease in MIR across the study period. Portugal saw the greatest decrease in males with a u2212_29.0% fall, while Ireland saw the greatest decrease in females with a u2212_26.6% fall. Sweden saw the smallest decrease in males and females with only a u2212_11.7% and u2212_8.5% fall, respectively. These trends were reflected across WHO regions, where all regions saw decreases in MIR across the study period. The Western Pacific region saw the greatest decrease in MIR in males with a u2212_27.3% reduction, while the Western Pacific and Eastern Mediterranean region were equivalent for females with u2212_27.9% reductions. The Americas saw the smallest decrease in MIR for both males and females over the study period, u2212_6.9% and u2212_10.2%, respectively.\"- - \"In this study, we aimed to analyze the trends of incidence, mortality, mortality to incidence ratio, and DALYs associated with KC among EU15u2009+u2009countries and 6 WHO regions, using the GBD study data and Joinpoint regression analysis. During a 29-year interval, while incidence and mortality from KC increased in most of the included countries, a drop in MIR was noted in all countries. Trends in DALYs were variable between countries. High-income countries had the highest values of ASIR, ASMR, and DALYs, concurring with previous studies. This could be partly attributed to a better performing cancer registry system and a higher prevalence of KC-related risk factors.\" - \"We found that the incidence of KC has been increasing in most EU15u2009+u2009 countries, which is congruent with previous reports. The rise in incidence is likely in part due to greater detection of early-stage KC on cross-sectional imaging and partly due to the increasing prevalence of smoking, obesity, and hypertension which are among the strongest risk factors of KC. In terms of smoking, an addition of more than 200 million daily smokers was noted between 1980 and 2012, probably contributing to the development of KC worldwide. However, recent declines in smoking rates have occurred in developed countries due to anti-smoking campaigns. This trend may be reflected by the decline in EAPC seen in the last decade in the high-income and upper-middle-income countries as per World Bank classification. Similarly, worldwide obesity prevalence had doubled during the past decades, with increasing trends seen especially in developed countries. This increase in obesity prevalence parallels our results showing that developed countries had a notable increase in incidence with the highest elevations noted in the World Bank upper-middle-income category for males. Occupational exposures to toxic compounds such as cadmium have also been found to elevate the risk of KC among males, however, to a lesser extent. Even though the contribution ratio of occupational exposure to trichloroethylene to kidney cancer incidence and mortality was relatively weak in the GBD database, protection is mandatory in the organic/chlorinated solvent industry. Recent analysis of risk factor contribution to global cancer burden in 2019 showed that for both sex combined, 33.8% of deaths could be attributed to risk factors analyzed with 19% attributed to High body-mass index, with 33% absolute increase since 2010, 18.1% attributed to smoking, with 19.1% absolute increase since 2010,u2009<u20091% attributed to occupational exposure to trichloroethylene, with 40.5% absolute increase since 2010.\" - \"Another potential cause of the rising incidence is the higher utilization of imaging and discovery of incidental small renal masses, which account for nearly half of the new cases of renal cell carcinoma. Detection of lower stage disease is associated with a better prognosis, and therefore the earlier detection of KC, as per the stage migration phenomenon, may be responsible for some of the improvements in patient outcomes. Asymptomatic masses are more frequently diagnosed given the increased clinical use of modern imaging such as abdominal CT scans for evaluating urological and non-urological symptoms. Indeed, a study in Canada showed an annual increase of CT imaging rates by 11.6% between 2000 and 2006, which decreased to 3.7% from 2013 to 2016. This is comparable to our results where Canada had an EAPC of ASIR of 4.4 for females in the years 1998u2013 2009 and 5.9 for males in the years 2001u20132004, compared to a lesser value of -1.6 in the years 2006u20132016 for females and -0.1 for the years 2008u2013 2019 in males. Similarly, CT use continuously increased for assessing abdominal pain in the emergency departments of the United States from 1997 to 2016, concurring with an increased ASIR for males in the USA. The increased frequency of radiological diagnosis would not be the sole reason behind the rise in ASIR in low-income countries since it has been demonstrated that these countries do not have equal access to imaging modalities compared to high-income countries. An improvement in healthcare access is noted in the Middle East and North Africa (MENA) region in the past few decades with a rise in life expectancy at birth from 65_years in 1990 to 71_years in 2012, which could reflect on the increased diagnosis of KC. However, significant disparities among and within countries still exist, especially with the unregulated intervention of the private sector to fill the gaps of the governments insufficient coverage, raising concern for equity. Thus, a multifactorial model would be the ultimate explanation for the increased KC incidence in different countries.\" - \"Lower MIRs worldwide may be partly due to the stage migration with the detection of lower stage disease on cross-sectional imaging. These tumors are associated with a lower likelihood of cancer mortality, and 30% were indolent on post-operative pathology. However, a recent article highlighted a paucity of imaging modalities in low to middle-income countries. Thus, over-diagnosis is not the only explanation for the universal drop in MIR. We showed that mortality is also decreasing in most of the included countries. One plausible explanation for the mortality drop would be the approval of 10 novel drugs by the US Food and Drug Administration (FDA) for the treatment of metastatic RCC between 2005 and 2016, with the first approved targeted therapy being Sorafenib, a tyrosine kinase inhibitor (TKI). The inclusion of targeted therapies in the therapeutic arsenal of KC was proven by several studies to positively impact patients survival. Our results mirror these findings as we noticed a drop in EAPCs of ASMRs from 2005 onwards, during the era of targeted therapies, in the high-income and upper-middle-income classes in the World Bank classification. On the other hand, EAPCs of ASMRs were still increasing after 2005 in middle-low to low-income countries, possibly due to a lack of access to comprehensive multidisciplinary team care, clinical trials, and advanced treatment modalities such as stereotactic radiotherapy medicine. Additionally, now a subset of patients with metastatic RCC can be cured with immune-oncology (IO-IO) combinations but these recently entered into practice in 2018. Therefore, future analysis will give information about its effects on survival and mortality. Poor access to treatment and imaging modalities plays a role in increased mortality rates in such regions since tumors are diagnosed at a later stage and are treated in a suboptimal manner. Indeed, we highlighted that middle and low-income countries had higher increases in ASMRs and smaller drops in MIR than high-income countries. Similarly, a previous article had comparable conclusions to ours, having found a positive correlation between the availability of imaging examinations/effective systemic therapies in high-income countries and a favorable prognosis; the authors also concluded that MIRs were negatively correlated with the human development index and the current health expenditure per capita. In this setting, Fay et al. emphasized the importance of clinical trial enrollment as a solution to health disparities by concluding to a similar overall survival among patients with metastatic RCC recruited in clinical trials across different geographic regions, despite different baseline characteristics. We observed that the DALYs were decreasing in most of included Western European countries, especially in females. However, the DALYs remained stable in the European region. This is probably attributed to an increase in DALYs in Eastern Europe as opposed to a decrease in Western Europe.\" - Finally, as we demonstrated an increase in incidence and mortality from KC in most included countries and regions, future focus is needed to implement strategies for the detection of early-stage KC. Extra efforts need to be put into decreasing the prevalence of smoking, obesity, and hypertension, which are among the strongest risk factors of KC, through campaigns and medical attention. Although MIRs are decreasing, thus indicating an improvement in outcomes, equal access to treatment would ameliorate even more the situation worldwide, especially by using novel targeted and IO therapies.- - \"Limitations following the use of the GBD database were noted previously by our group and the GBD Study collaborators. Regarding this study, the first limitation is the presence of alterations in data coding systems and country-specific practice during the study period, markedly a shift from the use of ICD 9 to ICD 10. However, the GBD authors map mortalities to causes of death lists, adjusting by such to the different coding systems. Secondly, variability in the reliability of death certification exists both within and across countries, with worldwide errors in death certification ranging from 39 to 61%. Also, only 39% of deaths globally were registered in 2012. Considering that Europe, the Americas, and Australasia were ranked among the best continents with civil registration and vital statistics, we demonstrated valid data of the EU15u2009+u2009countries and 6 WHO regions assessed. To balance the under-registration, the GBD uses garbage-code distribution algorithms and corrections, which relate to deaths resulting from poorly defined diagnoses or those that cannot be the single underlying cause of death. Thirdly, we could not extract the subcategorized data by individual KC histological subtypes from the GBD Study results tool, which should be considered when interpreting the results. Indeed, histopathological subtypes and stages of KC amend to different clinical significance and urgent management. Finally, since our study is an observational analysis of the trends in the burden of KC across 29_years in EU15u2009+u2009countries and 6 WHO regions, causal inferences cannot be concluded. Moreover, as seen in observational analyses, some potential confounders were not accounted for by using sex-specific, age-standardized incidence and mortality rates.\"- - \"While the incidence and mortality from KC rose in most EU15u2009+u2009countries and WHO regions from 1990 to 2019, the universal drop in MIR suggests an overall improvement in KC outcomes. This is likely multifactorial, including earlier detection of KC and improved treatments. These results are also of interest from a public health perspective as they highlight the importance of adjustable risk factor modification. A further study among lower-income countries would help identify the burden of disease across these regions and subsequently enable appropriate strategies to be developed to bridge disparities across countries and optimize patient care.\"- - \\' \\' - The datasets analysed during the current study are available in the GBD repository, . - \\' \\'- - \\'ASIR: Age-standardized incidence rate ASMR: Age-standardized mortality rate CT: Computed tomography DALYs: Disability-adjusted life years EAPC: Estimated annual percentage change EU: European Union GBD: Global burden of disease ICD: International classification of diseases IO: Immuno-oncology KC: Kidney cancer RCC: Renal cell carcinoma UK: United Kingdom USA: United States of America WHO: World Health Organization \\'- - \"Cai, Q. et al. Temporal trends of kidney cancer incidence and mortality from 1990 to 2016 and projections to 2030. Transl. Androl. Urol. 9(2), 166u2013 181 (2020)._ _ Sung, H. et al. Global cancer statistics 2020: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J. Clin. 71(3), 209u2013249 (2021)._ _ Rini, B. I., Campbell, S. C. & Escudier, B. Renal cell carcinoma. Lancet 373(9669), 1119u20131132 (2009)._ _ _ Moch, H., Cubilla, A. L., Humphrey, P. A., Reuter, V. E. & Ulbright, T. M. The 2016 WHO classification of tumours of the urinary system and male genital organs-part A: Renal, penile, and testicular tumours. Eur. Urol. 70(1), 93u2013105 (2016)._ _ Capitanio, U. et al. Epidemiology of renal cell carcinoma. Eur. Urol. 75(1), 74u201384 (2019)._ _ Gild, P., Ehdaie, B. & Kluth, L. A. Effect of obesity on bladder cancer and renal cell carcinoma incidence and survival. Curr. Opin. Urol. 27(5), 409u2013414 (2017)._ _ Lotan, Y. et al. Renal-cell carcinoma risk estimates based on participants in the prostate, lung, colorectal, and ovarian cancer screening trial and national lung screening trial. Urol. Oncol. 34(4), 167e9-216 (2016)._ _ Hidayat, K., Du, X., Zou, S. Y. & Shi, B. M. Blood pressure and kidney cancer risk: Meta-analysis of prospective studies. J. Hypertens. 35(7), 1333u20131344 (2017)._ _ _ Lowrance, W. T., Ordonez, J., Udaltsova, N., Russo, P. & Go, A. S. CKD and the risk of incident cancer. J. Am. Soc. Nephrol. 25(10), 2327u20132334 (2014)._ _ _ Kidney cancer fact sheet. International Agency for Cancer Research. WHO. . Accessed Dec. 2020. (). (Accessed on April 26 2022).Znaor, A., Lortet-Tieulent, J., Laversanne, M., Jemal, A. & Bray, F. International variations and trends in renal cell carcinoma incidence and mortality. Eur. Urol. 67(3), 519u2013 530 (2015)._ _ Ljungberg, B. et al. European association of urology guidelines on renal cell carcinoma: The 2019 Update. Eur. Urol. 75(5), 799u2013810 (2019)._ _ Chow, W. H. & Devesa, S. S. Contemporary epidemiology of renal cell cancer. Cancer J. 14(5), 288u2013301 (2008)._ _ _ Levi, F. et al. The changing pattern of kidney cancer incidence and mortality in Europe. BJU Int. 101(8), 949u2013 958 (2008)._ _ Salciccioli, J. D. et al. Respiratory disease mortality in the United Kingdom compared with EU15+ countries in 1985u2013 2015: Observational study. BMJ 363, k4680 (2018)._ _ Goodall R, Salciccioli JD, Davies AH, Marshall D, Shalhoub J. Trends in peripheral arterial disease incidence and mortality in EU15+ countries 1990u20132017. Eur J Prev Cardiol. 2020:2047487319899626.Global, regional, and national incidence, prevalence, and years lived with disability for 354 diseases and injuries for 195 countries and territories, 1990u20132017: a systematic analysis for the Global Burden of Disease Study 2017. Lancet. 2018;392(10159):1789u2013858.Global, regional, and national age-sex-specific mortality for 282 causes of death in 195 countries and territories, 1980u20132017: a systematic analysis for the Global Burden of Disease Study 2017. Lancet. 2018;392(10159):1736u201388.Al-Balah, A., Goodall, R., Salciccioli, J. D., Marshall, D. C. & Shalhoub, J. Mortality from abdominal aortic aneurysm: Trends in European Union 15+ countries from 1990 to 2017. Br. J. Surg. 107(11), 1459u20131467 (2020)._ _ _ Hughes, W. et al. Editors choiceu2014trends in lower extremity amputation incidence in European Union 15+ countries 1990u20132017. Eur. J. Vasc. Endovasc. Surg. 60(4), 602u2013612 (2020)._ _ Ferlay, J. et al. Cancer incidence and mortality worldwide: Sources, methods and major patterns in GLOBOCAN 2012. Int. J. Cancer 136(5), E359u2013E386 (2015)._ _ _ Kim, H. J., Fay, M. P., Feuer, E. J. & Midthune, D. N. Permutation tests for joinpoint regression with applications to cancer rates. Stat. Med. 19(3), 335u2013351 (2000)._ _ _ Bai, X., Yi, M., Dong, B., Zheng, X. & Wu, K. The global, regional, and national burden of kidney cancer and attributable risk factor analysis from 1990 to 2017. Exp Hematol Oncol. 9, 27 (2020)._ _ Zucchetto, A. et al. History of treated hypertension and diabetes mellitus and risk of renal cell cancer. Ann. Oncol. 18(3), 596u2013600 (2007)._ _ _ Hu, J., Mao, Y. & White, K. Renal cell carcinoma and occupational exposure to chemicals in Canada. Occup. Med. (Lond.) 52(3), 157u2013164 (2002)._ _ _ Ng, M. et al. Smoking prevalence and cigarette consumption in 187 countries, 1980u20132012. JAMA 311(2), 183u2013192 (2014)._ _ _ Finucane, M. M. et al. National, regional, and global trends in body-mass index since 1980: Systematic analysis of health examination surveys and epidemiological studies with 960 country-years and 9.1 million participants. Lancet 377(9765), 557u2013567 (2011)._ _ Malik, V. S., Willet, W. C. & Hu, F. B. Nearly a decade onu2014trends, risk factors and policy implications in global obesity. Nat. Rev. Endocrinol. 16(11), 615u2013 616 (2020)._ _ Hu, J., Mao, Y., White, K., Group TCCRER. Renal cell carcinoma and occupational exposure to chemicals in Canada. Occup. Med. 52(3), 157u2013164 (2002)._ _ _ GBDCRF Collaborators. The global burden of cancer attributable to risk factors, 2010u201319: A systematic analysis for the Global Burden of Disease Study 2019. Lancet 400(10352), 563u2013 591 (2022)._ _ Siegel, R. L., Miller, K. D. & Jemal, A. Cancer statistics, 2016. CA Cancer J. Clin. 66(1), 7u201330 (2016)._ _ Kane, C. J., Mallin, K., Ritchey, J., Cooperberg, M. R. & Carroll, P. R. Renal cell cancer stage migration: Analysis of the national cancer data base. Cancer 113(1), 78u201383 (2008)._ _ Sun, M. et al. Age-adjusted incidence, mortality, and survival rates of stage-specific renal cell carcinoma in North America: A trend analysis. Eur. Urol. 59(1), 135u2013 141 (2011)._ _ Smith-Bindman, R. et al. Trends in use of medical imaging in US health care systems and in Ontario, Canada, 2000u2013 2016. JAMA 322(9), 843u2013856 (2019)._ _ Wang, R. C. et al. Trends in use of diagnostic imaging for abdominal pain in U.S. emergency departments. AJR Am. J. Roentgenol. 216(1), 200u2013208 (2021)._ _ Hricak, H. et al. Medical imaging and nuclear medicine: A lancet oncology commission. Lancet Oncol. 22(4), e136u2013e172 (2021)._ _ Mate, K., Bryan, C., Deen, N. & McCall, J. Review of health systems of the Middle East and North Africa Region. In International Encyclopedia of Public Health (eds Mate, K. et al.) 347u201356 (Elsevier, 2017)._ _ Vaishampayan, U. et al. The effect of targeted therapy on overall survival in advanced renal cancer: A study of the national surveillance epidemiology and end results registry database. Clin. Genitourin. Cancer 12(2), 124u2013 129 (2014)._ _ Rao, A., Wiggins, C. & Lauer, R. C. Survival outcomes for advanced kidney cancer patients in the era of targeted therapies. Ann. Transl. Med. 6(9), 165 (2018)._ _ Vachhani, P. & George, S. VEGF inhibitors in renal cell carcinoma. Clin. Adv. Hematol. Oncol. 14(12), 1016u20131028 (2016)._ Mihaly, Z., Sztupinszki, Z., Surowiak, P. & Gyorffy, B. A comprehensive overview of targeted therapy in metastatic renal cell carcinoma. Curr. Cancer Drug Targets 12(7), 857u2013872 (2012)._ _ _ Shek, D. et al. Epidemiologic trends in renal cell carcinoma in the cytokine and post-cytokine eras: A registry analysis of 28,252 patients. Clin. Genitourin. Cancer 10(2), 93u201398 (2012)._ _ Kim, J. H. et al. Association of prevalence of benign pathologic findings after partial nephrectomy with preoperative imaging patterns in the United States from 2007 to 2014. JAMA Surg. 154(3), 225u2013231 (2019)._ _ Shannon, B. A., Cohen, R. J., de Bruto, H. & Davies, R. J. The value of preoperative needle core biopsy for diagnosing benign lesions among small, incidentally detected renal masses. J. Urol. 180(4), 1257u2013 1261 (2008)._ _ Lopes Gde L, Jr., de Souza JA, Barrios C. Access to cancer medications in low- and middle-income countries. Nat Rev Clin Oncol. 2013;10(6):314-22.Elaidi, R. et al. Comparative efficacy of first-line immune-based combination therapies in metastatic renal cell carcinoma: A systematic review and network meta-analysis. Cancers (Basel) 12(6), 1673 (2020)._ _ _ Sung, W. W., Ko, P. Y., Chen, W. J., Wang, S. C. & Chen, S. L. Trends in the kidney cancer mortality-to-incidence ratios according to health care expenditures of 56 countries. Sci. Rep. 11(1), 1479 (2021)._ _ _ _ Fay, A. P., McKay, R. R., Lin, X., Simantov, R. & Choueiri, T. K. Impact of geographic regions on overall survival in patients with metastatic renal cell carcinoma: Results from an international clinical trials database. J. Glob. Oncol. 4, 1u201314 (2018)._ Safiri, S. et al. The burden of kidney cancer and its attributable risk factors in 195 countries and territories, 1990u20132017. Sci. Rep. 10(1), 13862 (2020)._ _ _ Tahbaz, R., Schmid, M. & Merseburger, A. S. Prevention of kidney cancer incidence and recurrence: Lifestyle, medication and nutrition. Curr. Opin. Urol. 28(1), 62u201379 (2018)._ _ Sims, J. N. et al. Racial disparities and preventive measures to renal cell carcinoma. Int. J. Environ. Res. Public Health 15(6), 1089 (2018)._ _ GBD datasets. . (Accessed on April 26 2022).Burger, E. H., van der Merwe, L. & Volmink, J. Errors in the completion of the death notification form. S. Afr. Med. J. 97(11), 1077u20131081 (2007)._ Katsakiori, P. F., Panagiotopoulou, E. C., Sakellaropoulos, G. C., Papazafiropoulou, A. & Kardara, M. Errors in death certificates in a rural area of Greece. Rural Remote Health 7(4), 822 (2007)._ _ Lu, T. H. et al. Factors associated with errors in death certificate completion. A national study in Taiwan. J. Clin. Epidemiol. 54(3), 232u2013238 (2001)._ _ _ Mikkelsen, L. et al. A global assessment of civil registration and vital statistics systems: Monitoring data quality and progress. Lancet 386(10001), 1395u20131406 (2015)._ _ \"- - Author notes - \\'These authors contributed equally: Chinmay Jani and Nour Abdallah.These authors jointly supervised this work: Rana R. McKay and Dominic C. Marshall.\\' - Authors and Affiliations - \"Department of Internal Medicine, Mount Auburn Hospital, 330 Mount Auburn St, Cambridge, MA, 02138, USAChinmay Jani_&_Padmanabh BhattHarvard Medical School, Boston, MA, USAChinmay Jani_&_Padmanabh BhattDepartment of Urology Research, Glickman Urological and Kidney Institute, Cleveland Clinic, Cleveland, OH, USANour Abdallah_&_Justin SalciccioliDepartment of Pediatrics, Case Western Reserve University, Cleveland, OH, USAChristian MouchatiUniversity Hospitals Cleveland Medical Center, Cleveland, OH, USAChristian MouchatiSmt. NHL Municipal Medical College, Ahmedabad, Gujarat, IndiaRuchi JaniNIT Kurukshetra, Kurukshetra, IndiaRajesh SharmaImperial College HealthCare NHS Trust, London, UKGeorgina HanburyDivision of Pulmonary and Critical Care, Brigham and Womens Hospital, Boston, MA, USAJustin SalciccioliDepartment of Medicine, Division of Pulmonary and Critical Care, Medical College of Wisconsin, Milwaukee, WI, USAHarpreet SinghAcademic Section of Vascular Surgery, Department of Surgery and Cancer, Imperial College London, London, UKJoseph ShalhoubUniversity of California San Diego, San Diego, CA, USARana R. McKayDepartment of Surgery and Cancer, Imperial College London, London, UKDominic C. Marshall\" - \"AuthorsChinmay JaniYou can also search for this author in _Nour AbdallahYou can also search for this author in _Christian MouchatiYou can also search for this author in _Ruchi JaniYou can also search for this author in _Rajesh SharmaYou can also search for this author in _Padmanabh BhattYou can also search for this author in _Georgina HanburyYou can also search for this author in _Justin SalciccioliYou can also search for this author in _Harpreet SinghYou can also search for this author in _Joseph ShalhoubYou can also search for this author in _Rana R. McKayYou can also search for this author in _Dominic C. MarshallYou can also search for this author in _\" - Contributions - Each of the authors significantly contributed to this manuscript. Drs. C.J., N.A., C.M., R.R.M. and D.C.M. made significant contributions to the concept and design of this study. Drs. J.S., C.J. and D.C.M. were significantly involved in the statistical analysis. Drs. C.J., N.A., C.M., P.B., G.H., H.S., J.S., R.R.M. and D.C.M., Miss R.J. and, Mr R.S. were greatly involved in drafting manuscript. Drs. C.J., N.A., R.R.M. and D.C.M. were involved in critical revision of the manuscript and contributed vital intellectual content. All authors have reviewed the manuscript. Dr. C.J. (corresponding author, guarantor) takes responsibility for the content of the manuscript, including the data and analysis. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons BY-ND 3.0 license. LeFabShop is acknowledged for the Notre Dame model, which is available under the Creative Commons BY-SA 3.0 license. Funding: This work was supported by EPFL Innogrant, EPFL Enable, The Bridge Proof of Concept Program (Innosuisse and FNSNF 20B1-1_184178) and the Gebert RxFCf Stiftung (GRSu2013057/18).\"- - Author notes - \\'These authors contributed equally: Damien Loterie, Paul Delrot.\\' - Authors and Affiliations - \"Laboratory of Applied Photonics Devices, School of Engineering, Ecole Polytechnique FxE9dxE9rale de Lausanne, CH-1015, Lausanne, SwitzerlandDamien Loterie,_ Paul Delrot_&_Christophe Moser\" - \"AuthorsDamien LoterieYou can also search for this author in _Paul DelrotYou can also search for this author in _Christophe MoserYou can also search for this author in _\" - Contributions - D.L. and P.D. designed and conducted the research. D.L., P.D. and C.M. contributed to analysis and to writing and refining the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors are inventors on a PCT patent application related to this work (PCT/IB2018/056389). All authors declare that they have no other competing interests. - \\' \\'- - Peer review information Nature Communications thanks George Chryssolouris and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons CC BY 4.0 licence: you are free to use and modify the code on the proviso that you make any changes freely available, acknowledge the original authors in derivative works and do not release said works under a more restrictive licence.\\' - \\' \\'- - \"Choquet, D., Sainlos, M. & Sibarita, J. B. Advanced imaging and labelling methods to decipher brain cell organization and function. Nat. Rev. Neurosci. 22, 237u2013 255 (2021).Kusumi, A., Tsunoyama, T. A., Hirosawa, K. M., Kasai, R. S. & Fujiwara, T. K. Tracking single molecules at work in living cells. Nat. Chem. Biol. 10, 524u2013532 (2014)._ _ _ _ Choquet, D. Linking nanoscale dynamics of AMPA receptor organization to plasticity of excitatory synapses and learning. J. Neurosci. 38, 9318u20139329 (2018)._ _ _ _ _ Goncalves, J. et al. Nanoscale co-organization and coactivation of AMPAR, NMDAR, and mGluR at excitatory synapses. Proc. Natl Acad. Sci. USA 117, 14503u201314511 (2020)._ _ _ _ _ _ Bademosi, A. T. et al. In vivo single-molecule tracking at the Drosophila presynaptic motor nerve terminal. J. Vis. Exp. 131, e56952 (2018).Bademosi, A. T. et al. In vivo single-molecule imaging of syntaxin1A reveals polyphosphoinositide- and activity-dependent trapping in presynaptic nanoclusters. Nat. Commun. 8, 13660 (2017)._ _ _ _ _ Bademosi, A. T. et al. Trapping of syntaxin1a in presynaptic nanoclusters by a clinically relevant general anesthetic. Cell Rep. 22, 427u2013 440 (2018)._ _ _ _ Chai, Y. J. et al. Munc18-1 is a molecular chaperone for alpha-synuclein, controlling its self-replicating aggregation. J. Cell Biol. 214, 705u2013718 (2016)._ _ _ _ _ Gormal, R. S. et al. Modular transient nanoclustering of activated beta2-adrenergic receptors revealed by single-molecule tracking of conformation-specific nanobodies. Proc. Natl Acad. Sci. USA 117, 30476u2013 30487 (2020)._ _ _ _ _ _ Harper, C. B. et al. An epilepsy-associated SV2A mutation disrupts synaptotagmin-1 expression and activity-dependent trafficking. J. Neurosci. 40, 4586u20134595 (2020)._ _ _ _ _ Kasula, R. et al. The Munc18-1 domain 3a hinge-loop controls syntaxin-1A nanodomain assembly and engagement with the SNARE complex during secretory vesicle priming. J. Cell Biol. 214, 847u2013858 (2016)._ _ _ _ _ Padmanabhan, P. et al. Need for speed: super-resolving the dynamic nanoclustering of syntaxin-1 at exocytic fusion sites. Neuropharmacology 169, 107554 (2020)._ _ _ _ Haas, K. T. et al. Pre-post synaptic alignment through neuroligin-1 tunes synaptic transmission efficiency. Elife 7, e31755 (2018).Ester, M., Kriegel, H. P., Sander, J. & Xu, X. A density-based algorithm for discovering clusters in large spatial databases with noise. In Proc. 2nd International Conference on Knowledge Discovery and Data Mining (KDD96), 226u2013231 (1996).Levet, F. et al. SR-Tesseler: a method to segment and quantify localization-based super-resolution microscopy data. Nat. Methods 12, 1065u2013 1071 (2015)._ _ _ _ Andronov, L., Orlov, I., Lutz, Y., Vonesch, J. L. & Klaholz, B. P. ClusterViSu, a method for clustering of protein complexes by Voronoi tessellation in super-resolution microscopy. Sci. Rep. 6, 24084 (2016)._ _ _ _ _ _ Khater, I. M., Nabi, I. R. & Hamarneh, G. A review of super-resolution single-molecule localization microscopy cluster analysis and quantification methods. Patterns 1, 100038 (2020)._ _ _ _ _ GriffixE9, J. et al. Dynamic Bayesian cluster analysis of live-cell single molecule localization microscopy datasets. Small Methods 2, 1800008 (2018).Finkel, A. Quad trees, a data structure for retrieval on composite keys. Acta Inform. 4, 1u20139 (1974)._ _ _ _ Gutmann, A. R-trees: a dynamic index structure for spatial searching. In Proc. 1984 ACM SIGMOD International Conference on Management of Data (SIGMOD u201884), 47u201357 (1984).Figueiredo, M. An R-tree collision detection algorithm for polygonal models. In Proc. IASTED International Conference (2009).Zhai, R. G. & Bellen, H. J. The architecture of the active zone in the presynaptic nerve terminal. Physiology 19, 262u2013 270 (2004)._ _ _ Sudhof, T. C. The presynaptic active zone. Neuron 75, 11u201325 (2012)._ _ _ _ _ Tang, A. H. et al. A trans-synaptic nanocolumn aligns neurotransmitter release to receptors. Nature 536, 210u2013214 (2016)._ _ _ _ _ _ Nieves, D. J. et al. A framework for evaluating the performance of SMLM cluster analysis algorithms. Nat. Methods 20, 259u2013 267 (2021).Betzig, E. et al. Imaging intracellular fluorescent proteins at nanometer resolution. Science 313, 1642u20131645 (2006)._ _ _ _ _ Hess, S. T., Girirajan, T. P. & Mason, M. D. Ultra-high resolution imaging by fluorescence photoactivation localization microscopy. Biophys. J. 91, 4258u20134272 (2006)._ _ _ _ _ _ Manley, S. et al. High-density mapping of single-molecule trajectories with photoactivated localization microscopy. Nat. Methods 5, 155u2013 157 (2008)._ _ _ _ Sudhof, T. C. & Rothman, J. E. Membrane fusion: grappling with SNARE and SM proteins. Science 323, 474u2013 477 (2009)._ _ _ _ _ Han, L. et al. Rescue of Munc18-1 and -2 double knockdown reveals the essential functions of interaction between Munc18 and closed syntaxin in PC12 cells. Mol. Biol. Cell 20, 4962u20134975 (2009)._ _ _ _ _ Rickman, C., Meunier, F. A., Binz, T. & Davletov, B. High affinity interaction of syntaxin and SNAP-25 on the plasma membrane is abolished by botulinum toxin E. J. Biol. Chem. 279, 644u2013651 (2004)._ _ _ _ Meunier, F. A. & Gutierrez, L. M. Captivating new roles of F-actin cortex in exocytosis and bulk endocytosis in neurosecretory cells. Trends Neurosci. 39, 605u2013613 (2016)._ _ _ _ Malintan, N. T. et al. Abrogating Munc18-1-SNARE complex interaction has limited impact on exocytosis in PC12 cells. J. Biol. Chem. 284, 21637u201321646 (2009)._ _ _ _ _ Martin, S. et al. The Munc18-1 domain 3a loop is essential for neuroexocytosis but not for syntaxin-1A transport to the plasma membrane. J. Cell Sci. 126, 2353u20132360 (2013)._ _ _ _ Papadopulos, A. et al. Activity-driven relaxation of the cortical actomyosin II network synchronizes Munc18-1-dependent neurosecretory vesicle docking. Nat. Commun. 6, 6297 (2015)._ _ _ _ _ Wallis, T. P. et al. Saturated free fatty acids and association with memory formation. Nat. Commun. 12, 3443 (2021)._ _ _ _ _ _ Joensuu, M., Wallis, T. P., Saber, S. H. & Meunier, F. A. Phospholipases in neuronal function: a role in learning and memory? J. Neurochem. 153, 300u2013333 (2020)._ _ _ _ Hamilton, J. D. Time Series Analysis (Princeton University Press, 1994).Lehmann, E. L. & Romano, J. P. Testing Statistical Hypotheses (Springer, 2005).Ullrich, A. et al. Dynamical organization of syntaxin-1A at the presynaptic active zone. PLoS Comput. Biol. 11, e1004407 (2015)._ _ _ _ Jahn, R. & Fasshauer, D. Molecular machines governing exocytosis of synaptic vesicles. Nature 490, 201u2013207 (2012)._ _ _ _ _ _ Angelov, B. & Angelova, A. Nanoscale clustering of the neurotrophin receptor TrkB revealed by super-resolution STED microscopy. Nanoscale 9, 9797u20139804 (2017)._ _ _ _ Monnier, N. et al. Inferring transient particle transport dynamics in live cells. Nat. Methods 12, 838u2013840 (2015)._ _ _ _ _ Persson, F., Linden, M., Unoson, C. & Elf, J. Extracting intracellular diffusive states and transition rates from single-molecule tracking data. Nat. Methods 10, 265u2013269 (2013)._ _ _ Padmanabhan, P., Martinez-Marmol, R., Xia, D., Gotz, J. & Meunier, F. A. Frontotemporal dementia mutant Tau promotes aberrant Fyn nanoclustering in hippocampal dendritic spines. Elife 8, e45040 (2019).Joensuu, M. et al. Subdiffractional tracking of internalized molecules reveals heterogeneous motion states of synaptic vesicles. J. Cell Biol. 215, 277u2013292 (2016)._ _ _ _ _ Ripley, B. D. Modeling spatial patterns. J. R. Stat. Soc. B Stat. Methodol. 39, 172u2013212 (1977)._ Giannone, G., Hosy, E., Sibarita, J. B., Choquet, D. & Cognet, L. High-content super-resolution imaging of live cell by uPAINT. Methods Mol. Biol. 950, 95u2013110 (2013)._ _ _ _ Subach, F. V. et al. Photoactivatable mCherry for high-resolution two-color fluorescence microscopy. Nat. Methods 6, 153u2013159 (2009)._ _ _ _ _ Rickman, C. & Duncan, R. R. Munc18/syntaxin interaction kinetics control secretory vesicle dynamics. J. Biol. Chem. 285, 3965u20133972 (2010)._ _ _ _ Lang, T. & Jahn, R. Core proteins of the secretory machinery. Handb. Exp. Pharmacol. 184, 107u2013127 (2008).Cisse, I. I. et al. Real-time dynamics of RNA polymerase II clustering in live human cells. Science 341, 664u2013 667 (2013)._ _ _ _ _ Salavessa, L. et al. Cytokine receptor cluster size impacts its endocytosis and signaling. Proc. Natl Acad. Sci. USA 118, e2024893118 (2021).Joensuu, M. L. et al. Synaptotagmin 1 mediates toxicity of botulinum neurotoxin type A. EMBOJ e112095 , (2022).Martinez-Marmol, R. et al. Fyn nanoclustering requires switching to an open conformation and is enhanced by FTLD-Tau biomolecular condensates. Mol. Psychiatry 28, 946u2013 962 (2023).Small, C. H. et al. SV2A-Syt1 interaction controls surface nanoclustering and access to recycling synaptic vesicles. Prepint at bioRxiv (2022).Kubala, M. H., Kovtun, O., Alexandrov, K. & Collins, B. M. Structural and thermodynamic analysis of the GFP:GFP-nanobody complex. Protein Sci. 19, 2389u20132401 (2010)._ _ _ _ _ Joensuu, M. et al. Visualizing endocytic recycling and trafficking in live neurons by subdiffractional tracking of internalized molecules. Nat. Protoc. 12, 2590u20132622 (2017)._ _ _ _ Kechkar, A., Nair, D., Heilemann, M., Choquet, D. & Sibarita, J. B. Real-time analysis and visualization for single-molecule based super-resolution microscopy. PLoS ONE 8, e62918 (2013)._ _ _ _ _ _ \"- - \"We thank the IT department at the Queensland Brain Institute (QBI), Rumelo Amor and all past and present members of the Advanced Microscopy and Microanalysis Facility at QBI for their outstanding microscopy support. We further thank Giuseppe Balistreri for helpful discussions on the manuscript. The work was supported by an Australian Research Council (ARC) Discovery Project grant (DP190100674), an ARC Linkage Infrastructure Equipment and Facilities grant (LE130100078) and a National Health and Medical Research Council (NHMRC) Senior Research Fellowship (GNT1155794) to F.A.M. as well as an ARC Discovery Early Career Research Award (DE190100565) to M.J., an NHMRC Project Grant (GNT 1147600) to N.D._and an ARC Discovery Project (DP180102956) to D.O.\"- - Author notes - \\'Merja JoensuuPresent address: Australian Institute for Bioengineering and Nanotechnology, The University of Queensland, Brisbane, QLD, 4072, Australia\\' - Authors and Affiliations - \"Clem Jones Centre for Ageing Dementia Research, Queensland Brain Institute, The University of Queensland, Brisbane, QLD, 4072, AustraliaTristan P. Wallis,_ Anmin Jiang,_Huiyi Hou,_Kye Kudo,_Alex J. McCann,_Merja Joensuu,_Rachel S. Gormal_&_FrxE9dxE9ric A. MeunierSchool of Mathematics and Physics, The University of Queensland, Brisbane, QLD, 4072, AustraliaKyle Young,_Dietmar Oelz_&_ Hien NguyenQueensland Brain Institute, The University of Queensland, Brisbane, QLD, 4072, AustraliaNela DurisicSchool of Biomedical Sciences, The University of Queensland, Brisbane, QLD, 4072, AustraliaFrxE9dxE9ric A. Meunier\" - \"AuthorsTristan P. WallisYou can also search for this author in _Anmin JiangYou can also search for this author in _Kyle YoungYou can also search for this author in _Huiyi HouYou can also search for this author in _Kye KudoYou can also search for this author in _Alex J. McCannYou can also search for this author in _Nela DurisicYou can also search for this author in _Merja JoensuuYou can also search for this author in _Dietmar OelzYou can also search for this author in _Hien NguyenYou can also search for this author in _Rachel S. GormalYou can also search for this author in _FrxE9dxE9ric A. MeunierYou can also search for this author in _\" - Contributions - \"F.A.M. and T.P.W. conceived the project. T.P.W. implemented NASTIC in Python. H.H. and A.J.M. assisted with code debugging and GUI implementation. A.J.M. contributed additional ROI handling code._K.Y. implemented vector autoregression routines. K.K. performed quantitative comparisons of clustering algorithms and visualization of parameter matrices. D.O. and H.N. performed the tests of spatiotemporal data heterogeneity. M.J. and R.S.G. provided extensive testing, bug reporting and feature suggestions. A.J. performed super-resolution experiments under the supervision of M.J. T.P.W. performed all clustering analyses with input from R.S.G. N.D. provided valuable input into MSD analyses. T.P.W., F.A.M. and R.S.G. wrote the manuscript with input from the other authors.\" - Corresponding authors - \"Correspondence to , or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \"Nature Communications thanks Ismail Khater, Ana Garcia-Saez, and the other anonymous reviewer(s) for their contribution to the peer review of this work. This article has been peer reviewed as part of Springer Natures initiative.\" - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons CC-BY-NC-ND. (Left panel) schematic design of CTChip FR1 showing the input slots for buffer and sample, output slots for CTCs enriched fraction and WBCs-rich waste. (Right panel) Schematic cross section showing the hematogenous cells (in white and red) and CTCs (in green) at three different positions (X, Y, Z as in the left panel). (b) (Left panel) schematic design of the DLA biochip showing the input slot for the pre-diluted sample and the output slots for CTCs enriched fraction and WBCs-rich waste. (Right panel) Schematic cross section showing the hematogenous cells (in grey and red) and CTCs (in green) at two different positions (1 and 2 as in the left panel).Cross-sectional views and Dean vortices generated in curvilinear microchannel for cell separation. (a) Adapted from. - To test the performance of the DLA biochip with mDLA products, we analyzed six replicates for each cell line for each experimental condition (54 experiments in total). - Tumor cell detection of pre-labeled cancer cells after enrichment using flow cytometry - \"After enrichment of spiked mDLA samples with the CTChip FR1 or the DLA_chip, the processed samples were analyzed by flow cytometry on a MoFlo XDP flow cytometer (Beckman Coulter, Germany) to enumerate the spiked cells. Intact cells were identified based on size and granularity measured as the height of Forward and Side Scatter signals (FSC-Height and SSCxADHeight) while single cells were discriminated from aggregates using the width of the Side Scatter signals (SSCxAD Width). In addition, Hup-T4 cells were defined as Hoechst/CTO; Sk-BR-3 as Hoechst/CTG; and LNCaP as Hoechst/CTdR events (see Supplementary Fig. ). Recovery rate was defined as the ratio between the number of detected cells and the number of initially spiked cells. In order to compare the recovery rate efficiency of the two chips, we determined a global mean tumor cell recovery rate, calculated as the mean of the recoveries of all 3 cell lines cells in all 3 experimental conditions, obtained with each technology.\" - Data analysis - \"Statistical analyses of the data were performed in GraphPad Prism version 7.00 (GraphPad Software, San Diego, CA, USA). A non-parametric two-tailed test (Mannu2013 Whitney U test) was used for computing statistical significances. p value of less than 0.05 was considered significant.\"- - Establishing mDLA products - \"Since the availability of DLA products for testing new applications and protocols is very limited, we sought alternative, more easily available blood products for testing purposes. Obvious candidates were BC prepared by the centrifugation of anti-coagulated whole blood and consisting of a concentrated suspension of WBCs and platelets. BCs are available in larger quantities (50u2013100_mL) for research purposes from blood banks or commercial providers. In order to test whether BCs can be used as surrogates for DLA products, we compared their blood cell profiles. BC collected from healthy donors (nu2009=u200918) had on average higher red blood cell (RBC) fraction (pu2009<u20090.0001), lower platelet fraction (pu2009<u20090.0001), and lower WBC fraction (pu2009<u2009 0.0001) compared to patient-derived DLA products (nu2009=u200926) (Fig._A). Because nucleated cells are the most relevant and challenging blood cells for CTC enrichment (Supplementary Fig. ), we focused on the different WBC populations. We observed a significant difference in the prevalence (unpaired Mannu2013 Whitney U tests) of each WBCs population between DLA products and BCs (Fig._ B). In order to obtain better DLA surrogates, we tested processing BC with density gradient centrifugation employing the standard Ficoll-Paque PLUS (du2009=u2009 1.077_g/mL) which would preferentially enrich the mononuclear cells with lower density (1.067u20131.077_g/mL) and the Ficoll-Paque PLUS 1.084_g/mL with a density outside the density window targeted by DLA procedure (1.055u2013 1.08_g/mL) and less sharp for granulocytes with higher density (>u20091.080_ g/mL). Surprisingly, no significant difference between WBC composition after centrifugation with the two different media could be observed and both media resulted in a WBC composition similar to DLA products (see Supplementary Fig. ), indicating that both could be used alike to produce suitable mDLA products. We considered the fact that the basophil counts were still lower compared to DLA products (Fig._B) as negligible due to the extremely low prevalence of this population.\" - \"Figure 3Composition of DLA samples, unprocessed and processed buffy coats. (a) Relative distribution of RBC, platelets and WBCs with (b) indication of the fraction of the different leucocyte subpopulations determined with CELL-DYN Ruby hematology analyzer and representation of the median values with interquartile range. Whiskers represent total range. The difference between the median of the percentage of five individual populations of WBCs in the three different highly concentrated blood products as in (a) was compared. Statistical analysis (Mannu2013Whitney U test): *Pu2009<u2009.05; **Pu2009<u2009.01; ***Pu2009 <u2009.001; ****Pu2009<u2009.0001.\" - Size determination of the different cancer cell lines - \"Since the cell size influences the cell enrichment efficiency on spiral microfluidics chips, we determined the diameter of the three cell line cells tested here. We measured a mean diameter of 13.7u2009xB1u20091.4_xB5m (range 10.5 to 15.9_xB5m) for Sk-BR-3, 15.1u2009xB1u20092.4_xB5m (range 11.8 to 20.5) for Hup-T4, and 15.9u2009xB1u20093.4_xB5m (range 12.3 to 26.4), for LNCaP (see Supplementary Fig. ). Although minor differences in sizes were observed (see Supplementary Fig. ), we expected a similar performance for all three cell lines. In contrast, the diameter for the WBCs (8.5u2009xB1u20090.9_xB5 m, range 6.4 to 10_xB5m) was significantly smaller than the diameter of the cell line cells (pu2009<u20090.0001).\" - Recovery of pre-labeled tumor cells spiked in mDLA samples enriched using the ClearCell FX system and the DLA biochip - \"Next, we used the mDLA samples to challenge the spiral chip technology. As the commercially available ClearCell FX platform with the CTChip FR1 had been designed for peripheral blood samples, we initially tested recovery on this system using samples with 50u2009xD7u200910 WBCs (equivalent to u0334 15_mL_PB, experimental condition 1, nu2009=u20093 for each cell line) (Fig._A). The mean tumor cell recovery rate from mDLAs containing 50u2009xD7u200910 WBCs for all three cell lines was 30% (nu2009=u20099). Next, we tested the device with mDLA samples containing 200u2009xD7u200910 WBCs (equivivalent to u0334 60_mL_PB, experimental condition 2, nu2009=u20094 for each cell line) (Fig._B) and 400u2009xD7 u200910 WBCs (equivivalent to u0334 120_mL_PB, experimental condition 3, nu2009 =u20094 for each cell line) (Fig._C). With the higher WBC number in experimental condition 2, the global mean recovery rate dropped to 23% (nu2009=u200912) (Fig._B) and in all experiments using the experimental condition 3 the output straw of the device clogged and did not allow sample enrichment (Fig._C). The global mean tumor cell recovery rate with the CTChip FR1 in experimental conditions 1 and 2 was 25% (nu2009=u200921).\" - \"Figure 4Spiked-in cells recovery rates after enrichment with the CTChipFR1 and DLA biochip. (a) 1000, (b) 4000 and (c) 8000 cancer cells from each of three differentially pre-labelled cell lines were spiked with flow cytometry respectively into (a) 50u2009xD7u200910, (b) 200u2009xD7u200910 and (c) 400u2009xD7 u200910 WBCs from mimicking-DLA products. Enriched cells were enumerated again by flow cytometry and compared with the number of initially spiked cells to determine recovery rates. The difference between median recovery rates in percentage were compared after enrichment with the ClearCellFX System (green boxes) and DLA biochip (black boxes). Whiskers represent total range. Statistical analysis (Mannu2013Whitney U test): *Pu2009<u20090.05; **Pu2009<u20090.01.\" - \"We next tested the DLA biochip for processing similar samples. For a direct comparison with the CTChip FR1 chip, the mDLA products were prepared according to the same experimental workflow as used for the CTChip FR1 and containing 50u2009xD7 u200910, 200u2009xD7u200910 and 400u2009xD7u200910 WBCs (nu2009=u2009 6 for each cell line in each experimental condition) (Fig._). All enrichment cycles were performed without any visible clogging or other problems concerning the flow through the microfluidic chip. Despite a higher recovery rate for LNCaP in mDLA samples with 400u2009xD7u200910 WBCs, no significant recovery differences between the different cell lines were noted. The mean tumor cell recovery rate from mDLAs containing 50u2009xD7u200910 WBCs (equivivalent to u0334 15_ mL_PB) was 47% (nu2009=u200918), containing 200u2009xD7u200910 WBCs (equivalent to u0334 60_mL_PB) was 47% (nu2009=u200918), and containing 400u2009xD7 u200910 WBCs (equivalent to u0334 120_mL_PB) was 50% (nu2009=u200918), and those recoveries were not statistically different across the three experimental conditions. The global mean tumor cell recovery rate for the DLA biochip was 48% (nu2009=u200954) and significantly higher than the recovery calculated for the FR1 chip.\" - WBC contamination after DLA biochip enrichment - \"CTC-screening or other subsequent applications (e.g. molecular analyses) might be further challenging after enrichment in case of high WBC carryover. To get information on the level of WBC carryover and on the importance of the second enrichment cycle, we determined the WBC content remaining in the sample after the first (S1) and second (S2) round of DLA biochip enrichment (Fig._A). As expected (Fig._C), the number of co-enriched WBCs correlated positively with the number of input cells and decreased after performing the second round of enrichment. We also determined the distribution of the input cells through the different cellular fractions generated during DLA biochip enrichment of mDLA samples (waste of cycle 1 (W1), and samples after enrichment cycles 1 and 2 (S1 and S2) (Fig._B). We determined that 96.84%u2009xB1u20092.66% (s.d.) of the initial WBCs were present on the waste of the first enrichment cycle (W1) and an additional 3.14%u2009xB1u20092.65% (s.d.) were present on the waste of the second enrichment cycle (W2) (Fig._B). In the final enriched sample obtained after the two cycles (S2) we could find 0.017%u2009xB1u20090.013% (s.d.) (which corresponded to an average of 60.000 WBCs).\" - \"Figure 5Evaluation of WBC contamination after one and two cycles of enrichment with the DLA biochip. mDLA products containing 50u2009xD7u200910, 200u2009 xD7u200910 and 400u2009xD7u200910 WBCs were enriched with the DLA biochip and the (a) Mean number of WBCs in the enriched fraction after one (S1, analyzed with CELL-DYN Ruby hematology analyzer), and two enrichment cycles (S2, analyzed with flowcytometry). Lines represent standard deviation. (b) Mean percentages with standard deviation of WBCs in mDLA products containing 400u2009xD7u2009 10 WBCs and enriched with the DLA biochip were determined in the S2, in the waste of the first enrichment cycle (W1) and in the waste of the second enrichment cycle (W2). (c) Representative snapshot of the WBCs flow during the first and the second enrichment cycle, tracked using the Photron Fastcam SA3 (MEC, Indiana) connected to the Olympus IX71 inverted microscope (Olympus, US).\"- - \"DLA is a promising and clinically safe approach to screen liters of blood for CTCs. While its dramatic effect on CTC detection frequency and yield has been already validated, investigating the DLA products for CTCs remains challenging. DLA is a pre-enrichment approach that requires additional enrichment and detection steps, for example, like those done with the CellSearch system. But although this method has been so far the most effective to detect CTCs in DLA products, it is limited to 200u2009xD7u200910 WBCs per run (equivalent to u0334 60_ mL of PB and around 5% of a typical 40_mL DLA product). Therefore, more scalable and economic approaches are needed to unlock the full potential of DLA. In this context, microfluidic- or filtration-based label-free CTC enrichment methods appear interesting (despite used so far only with 100u2009xD7u200910 cells), because they might allow a more continuous sample processing with the possibility to analyze higher sample volumes in a cost-effective manner (e.g. by avoiding expensive conjugated antibody cocktails for positive enrichment). Particularly promising are hydrodynamic-based spiral microfluidic devices, that take advantage of Dean and inertial forces to continuously separate the cells. For our present work, we tested a novel inertial microfluidic based spiral biochip that was capable to process up to 4_mL DLA samples with high WBC content (400u2009xD7 u200910 WBCs, equivalent to u0334 120_mL_PB). Although this represents only 10% of the DLA volume, the approach is easily scalable by repeating the process on the chip with further portions of the same DLA product and/or by running several chips in parallel. Importantly, our data indicated a good recovery rate for all three different cell lines independent from the WBC input. This can be explained by intrinsic properties of the DLA biochip with its trapezoidal cross-section, whose design is based on similar existing devices, but optimized to process WBC-rich DLA products allowing maximal WBC depletion and CTC retention. Indeed, it was already demonstrated that a trapezoidal channel cross-section can increase the resolution of size-based separation of WBCs from the more abundant RBCs. The trapezoidal geometry alters the shape of velocity field and contributes to a better positioning of smaller cells near the outer wall, far enough to not disturbing the position of larger cells near the inner wall. This improved separation with minimized interaction between particles with different sizes allows devices with trapezoidal geometry to process higher number of background. The chip design is simple and easy to operate despite requiring two-cycle enrichment for DLA samples. Importantly, the DLA biochip did not clog as previously reported for filtration devices, for deterministic lateral displacement (DLD) methods, and as we could observe by processing mDLAs with the highest WBCs counts using the CTChip FR1 (Fig._C). It is conceivable that the larger dimensions of the DLA biochip channels, when compared to the standard, might help to prevent clogging. Moreover, the trapezoidal cross-section of the spiral chip, coupled with a larger number of turns, allowed to operate the chip in a sheath-less configuration unlike the CTChip FR1. We believe that all of these important characteristics of the technology contributed to the successful processing and tumor cell enrichment of samples with high WBC concentrations.\" - \"Although we are aware of the fact that tumor cell lines cannot properly reflect the heterogeneous patientxB4s CTCs, they can help to determine the technical performance of a new enrichment method in a standardized fashion. Therefore, we tested the spiral chips with cell lines frequently used in CTC assays and covering the spectrum of three major cancer entities. Since it is difficult to use precious DLA samples for establishing new methods or technical validations, we evaluated if BC after density gradient centrifugations might represent a good surrogate of patient-derived DLA based on its hematological composition. Our data (Fig._) provides evidence that the WBC composition of the mDLA is similar to the patient-derived DLA products highly commending them for standardized testing purposes. Our data revealed the superiority of the DLA biochip over the ClearCell FX System for further CTC-enrichment of DLA products, even if challenged with up to 400u2009xD7u200910 WBCs per run (equivalent to u0334 120_mL_PB). The suitability of the CTChip FR1 for our purposes was already questioned when comparing the tumor cell recoveries of mDLAs with a WBC content of 50u2009xD7u200910 WBCs. Based on own previous experiments done with 7.5_ mL blood samples, we expected a tumor cell recovery in average of 60% (data not shown) oru2009>u200985% based on data from the literature. Notably, the processing of mDLAs containing just 50u2009xD7u200910 WBCs (equivalent to u0334 15_ mL) already resulted in a mean tumor cell recovery of just 30% (Fig._A), which might be in part explained by the different rheological properties of DLA products compared to peripheral blood, but also reveal the limitations of this system to operate outside its intended use.\" - \"A further relevant aspect that determines the performance of a CTC enrichment method is the WBCs contamination in the enriched fraction. In fact, a high WBC carryover might be not compatible with CTC detection and isolation methods or may even interfere with CTC-specific gene expression profiling. For the case of DLA products with their excessive WBC counts, this issue becomes more significant. In this aspect, the DLA biochip revealed a high WBC depletion of 3.9u2009xB1 u20090.3 logs after the two cycles corresponding to a depletion of 99.98% of WBCs. Importantly, the second enrichment cycle alone was responsible for removing around 3% of WBCs (equivalent to approximately 12u2009xD7u200910 WBCs on experimental condition 3).\" - In summary, our results suggest the newly designed spiral DLA biochip as a promising tool to enrich CTCs from DLA products with a relatively low carryover of WBCs. The scalability of the method further suggests that it could potentially allow the processing of complete DLA products.- - \"Hayes, D. F. et al. Circulating tumor cells at each follow-up time point during therapy of metastatic breast cancer patients predict progression-free and overall survival. Clin. Cancer Res. 12, 4218u20134224. (2006)._ _ _ _ de Bono, J. S. et al. Circulating tumor cells predict survival benefit from treatment in metastatic castration-resistant prostate cancer editorial comment. J Urol. 181, 2535u20132536 (2009).Cohen, S. J. et al. Circulating endothelial cells (CEC) and circulating tumor cells (CTC) in patients (pts) with metastatic colorectal cancer (mCRC). J. Clin. Oncol. 24, 153su2013153s (2006)._ Toss, A., Mu, Z., Fernandez, S. & Cristofanilli, M. CTC enumeration and characterization: Moving toward personalized medicine. Ann. Transl. Med. 2, 108. (2014)._ _ _ _ _ Coumans, F. A., Ligthart, S. T., Uhr, J. W. & Terstappen, L. W. Challenges in the enumeration and phenotyping of CTC. Clin. Cancer Res. 18, 5711u20135718. (2012)._ _ _ Fischer, J. C. et al. Diagnostic leukapheresis enables reliable detection of circulating tumor cells of nonmetastatic cancer patients. Proc. Natl. Acad. Sci. U S A 110, 16580u201316585. (2013)._ _ _ _ _ Stoecklein, N. H., Fischer, J. C., Niederacher, D. & Terstappen, L. W. Challenges for CTC-based liquid biopsies: Low CTC frequency and diagnostic leukapheresis as a potential solution. Expert Rev. Mol. Diagn. 16, 147u2013164. (2016)._ _ _ _ Fehm, T. N. et al. Diagnostic leukapheresis for CTC analysis in breast cancer patients: CTC frequency, clinical experiences and recommendations for standardized reporting. Cytometry A 93, 1213u20131219. (2018)._ _ _ Andree, K. C. et al. Toward a real liquid biopsy in metastatic breast and prostate cancer: Diagnostic leukapheresis increases CTC yields in a European prospective multicenter study (CTCTrap). 1097u20130215 (electronic) 0020u20137136 (linking) (2018).Hou, H. W. et al. Isolation and retrieval of circulating tumor cells using centrifugal forces. Sci. Rep.-Uk 3 (2013).Kuntaegowdanahalli, S. S., Bhagat, A. A., Kumar, G. & Papautsky, I. Inertial microfluidics for continuous particle separation in spiral microchannels. Lab Chip 9, 2973u20132980. (2009)._ _ _ _ Nivedita, N., Ligrani, P. & Papautsky, I. Dean flow dynamics in low-aspect ratio spiral microchannels. Sci. Rep. 7, 44072. (2017)._ _ _ _ _ Khoo, B. L. et al. Clinical validation of an ultra high-throughput spiral microfluidics for the detection and enrichment of viable circulating tumor cells. PLoS ONE 9, e99409. (2014)._ _ _ _ _ _ Wu, J. et al. Magnetic-based enrichment of rare cells from high concentrated blood samples. Cancers (Basel) 12, (2020).Lee, Y., Guan, G. F. & Bhagat, A. A. ClearCell (R) FX, a label-free microfluidics technology for enrichment of viable circulating tumor cells. Cytom. Part A 93a, 1251u20131254, (2018).Wu, A., Bhagat, A. A., Leong, M. C. & Lim, C. T. ClearCell FX: A microfluidic system for label-free circulating tumor cell enrichment. J. Clin. Oncol. 32, (2014).Guan, G. et al. Spiral microchannel with rectangular and trapezoidal cross-sections for size based particle separation. Sci. Rep. 3, 1475. (2013)._ _ _ _ _ Zipursky, A., Bow, E., Seshadri, R. S. & Brown, E. J. Leukocyte density and volume in normal subjects and in patients with acute lymphoblastic leukemia. Blood 48, 361u2013371 (1976)._ _ _ Reinhardt, F. et al. Diagnostic leukapheresis enables reliable transcriptomic profiling of single circulating tumor cells to characterize inter-cellular heterogeneity in terms of endocrine resistance. Cancers (Basel) 11, (2019).Franken, A. et al. Label-free enrichment and molecular characterization of viable circulating tumor cells from diagnostic leukapheresis products. Clin. Chem. 65, 549u2013 558. (2019)._ _ _ _ Brychta, N. et al. Isolation of circulating tumor cells from pancreatic cancer by automated filtration. Oncotarget 8, 86143u201386156. (2017)._ _ _ _ Bhagat, A. A., Kuntaegowdanahalli, S. S. & Papautsky, I. Continuous particle separation in spiral microchannels using Dean flows and differential migration. Lab Chip 8, 1906u20131914. (2008)._ _ _ _ Warkiani, M. E. et al. Slanted spiral microfluidics for the ultra-fast, label-free isolation of circulating tumor cells. Lab Chip 14, 128u2013137. (2014)._ _ _ _ Wu, L., Guan, G., Hou, H. W., Bhagat, A. A. & Han, J. Separation of leukocytes from blood using spiral channel with trapezoid cross-section. Anal Chem. 84, 9324u20139331. (2012)._ _ _ _ Ji, H. M. et al. Silicon-based microfilters for whole blood cell separation. Biomed. Microdev. 10, 251u2013257. (2008)._ _ McGrath, J., Jimenez, M. & Bridle, H. Deterministic lateral displacement for particle separation: A review. Lab Chip 14, 4139u20134158. (2014)._ _ _ _ Sollier, E. et al. Size-selective collection of circulating tumor cells using Vortex technology. Lab Chip 14, 63u201377. (2014)._ _ _ _ Sieuwerts, A. M. et al. Molecular characterization of circulating tumor cells in large quantities of contaminating leukocytes by a multiplex real-time PCR. Breast Cancer Res. Treat. 118, 455u2013468. (2009)._ _ _ _ \"- - \\'This work was supported by the Brigitte und Dr. Konstanze Wegener Foundation [Project #01 to N.H.S. and W.T.K.) and the Innovative Medicines Initiative Joint Undertaking (IMI JU) in conjunction with CANCER-ID [Grant Agreement #115749 to N.H.S.].\\'- - \\'Open Access funding enabled and organized by Projekt DEAL. \\'- - Author notes - \\'These authors contributed equally: R. P. L. Neves and N. H. Stoecklein.\\' - Authors and Affiliations - \"Department of General, Visceral and Pediatric Surgery, University Hospital, Medical Faculty, Heinrich-Heine-University Duesseldorf, Moorenstr. 5, Bldg. 12.46, 40225, Duesseldorf, GermanyR. Guglielmi,_G. van Dalum,_J. Wu,_B. Behrens,_W. T. Knoefel,_R. P. L. Neves_&_N. H. StoeckleinBiolidics Limited, Singapore, SingaporeZ. LaiInstitute for Transplantation Diagnostics and Cell Therapeutics, University Hospital, Medical Faculty, Heinrich-Heine-University Duesseldorf, Duesseldorf, GermanyK. RabaInstitute for Health Innovation and Technology (iHealthtech), National University of Singapore, Singapore, SingaporeA. A. S. BhagatDepartment of Biomedical Engineering, National University of Singapore, Singapore, SingaporeA. A. S. Bhagat\" - \"AuthorsR. GuglielmiYou can also search for this author in _Z. LaiYou can also search for this author in _K. RabaYou can also search for this author in _ G. van DalumYou can also search for this author in _ J. WuYou can also search for this author in _B. BehrensYou can also search for this author in _A. A. S. BhagatYou can also search for this author in _W. T. KnoefelYou can also search for this author in _R. P. L. NevesYou can also search for this author in _N. H. StoeckleinYou can also search for this author in _\" - Contributions - R.G. conducted the experiments, analyzed the data and prepared the manuscript. Z.L. and A.A.S. Bhagat designed and fabricated the DLA biochips, and helped to write the manuscript. K.R. and J.W. provided technically assistance in the experiments. B.B. helped to analyze the data and reviewed the manuscript. G.v.D. helped to design the study and to analyze the data. W.T.K. helped to design the study and writing the manuscript. N.H.S. and R.P.L.N. designed the study, helped analyzing the data, preparing the figures and writing the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - Z. Lai is an employee of Biolidics Ltd (Singapore). Z.Lai and A. A. S. Bhagat are shareholders of Biolidics Ltd (Singapore). The remaining authors R. Guglielmi, K. Raba, G. van Dalum, J. Wu, B. Behrens, W.T. Knoefel, R. P. L. Neves and N. H. Stoecklein declare no conflict of interest. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons CC0 1.0 Universal Public Domain Dedication at ].\" - Maps - \"Maps have been added both using the Map extension and by developing an R application. In the latter case, more advanced features can be implemented. As an example, in the case of anatomical maps, placeholders radii have been related to the cardinality of the set of patients having a particular random symptom (Fig._ b), while a heatmap has been used to represent the density of random locations in geographic maps (Fig._a). The possible data analyses and representations are only limited by the underlying R framework.\" - Answers to the qualitative survey - \"Out of the 8 sub-units for which the 12 implemented databases were developed (Table_), 5 replied. The results were encouraging, having an average score of (4.3 pm 0.48) over a maximum of 5. More details have been reported in Supplementary Materials, Appendix .\" - The first question about the ease of data exploration obtained an average score of (4.4 pm 0.5) over 5. Question 2 on the speed of entry received a low vote scoring an average of (3.4 pm 1.6) over 5. Question 3 is about the user interface and received an average score of (4.2 pm 0.8). Question 4 refers to plots and statistical analyses, and received a score of (4.3 pm 0.5), however, it presents a slice of unexpressed opinions. The last question obtained the lowest number of answers and scored an average of (4.7 pm 0.6).- - This work presents a first, complete tool for designing a Semantic MediaWiki instance to be used as a semantic database enriched with statistical analyses. The choice of R Shiny as the underlying engine for computations permits extending it with a wide range of embedded interactive applications. - \"In fact, this software should allow for a more straightforward and direct design process and foster a potentially widespread adoption of such a flexible semantic platform as Semantic MediaWiki in all those data modeling scenarios in which one or more u201Cone-to-many relationsu201D are present in the design due to functional dependencies in the data themselves. In the case annotations can be linked to known formal (bio)-ontologies, special semantic properties can be used to establish equivalences translated into owl:SameAs statements, and in this way different datasets can be connected and collaboration on inter-specialty data becomes possible, possibly leading to a globally u201CMedical scientific wiki databaseu201D.\" - The database built by importing the XML file generated by tsv2smw is already using simple forms for entering data based on the PageForms extension. The only attention was to provide a uniqueness check, since the unique constraint in Semantic MediaWiki merely signals conflicts but does not prevent them. It was indeed decided to name the entities with progressive numbers, in order to implement a more general abstraction. In fact, when personal details are modeled it would perhaps be simpler to use personal data for identifying them uniquely. However, there is a MediaWiki facility named DISPLAYTITLE which allows for changing dynamically the titles of the pages therefore this should not be a problem. - \"Another interesting point that has been addressed is the explicit representation of missing data: besides the usual classification, a field could be without a value because it has not yet been considered, therefore some Boolean properties have been extended to a u201Cthree-valuedu201D logic that allows for a third possibility which stands for u201Cnot availableu201D (u201Ctertium daturu201D ).\" - Currently, the system is hosted in a closed intranet behind perimeter firewalls, and in exported data, sensitive information is omitted. Should privacy concerns arise, a symmetric deterministic client-side encryption will be enforced on certain sensitive fields that are not useful for downstream processing. An important point will be the treatment of geographical coordinates since they are sensitive but useful for geographic clustering; a possible solution will be to aggregate locations until a certain population threshold will be reached, in order to make individuals identification very hard. More pervasive data alterations should be limited since analyses on encrypted data would be more complex to manage. Two-factor authentication is available to users. - Prolog predicates could be more familiar and simpler than SPARQL syntax for some users and this language is already adopted in some systems such as AllegroGraph or Loki; its syntax is not difficult, being based on facts, rules, and goals. To be able to operate directly from the site, a special page based on Tau-Prolog, an open-source Prolog interpreter written in JavaScript, could be included in order to perform limited-depth queries. Alternatively, a SWISH embedded site could be added. In both cases, a set of predefined queries could be provided according to the underlying properties and types. - \\'The possibility of interfacing R scripts is ideal for better decoupling the development of (intelligent) analysis modules by people with more specialized skills: a biostatistician could comfortably work in his own development environment and then transfer the application she created to the developer in order to integrate it in the site.\\' - \"Also, Anatomical maps can be created using R and they can be used to trace, for example, the successive locations of tumor relapses in individual patients or, given a location, all patients with that particular symptom (Fig._b). The advantage of a u201Csymbolicu201D approach is that a first level of abstraction is already introduced, which could be further exploited through hierarchical clustering or clustering on categories. All this expressive power comes precisely from the possibility to use R, or also another data analysis framework, for dealing with data.\" - Some limitations emerged and will be addressed. First, the flexibility of the underlying TSV schema should be enhanced, since there persist some peculiarities typical to the context in which the system is currently used, namely the core patient-visit relation at the base of all the currently developed databases. This flexibility requires careful parsing of the feeble structure equipping the TSV input files. It could be also useful to give the possibility to operate on differential TSV schemata in order to make maintenance easier and allow for simpler amendments. - Another critical aspect concerns the possibility of sharing common categories in a wiki-farm configuration. To segregate relevant data to different research groups, a more robust and secure way is to set up a set of wikis; an alternative would be to configure a single site using namespaces. In this case, a triple-store could be a viable solution, and greater integration of Semantic MediaWiki with Wikidata or SPARQL endpoints , in general, would be ideal. - \"A system validation among users demonstrated the usability of the developed Semantic MediaWiki interface by bio-medical users and it suggested improvements for future interface development. There were no major negative comments by collaborators related to data exploration (Question 1). According to users answers, semantic properties and data exploration are topics of interest and could benefit biomedical researchers work. Data entry is time-consuming, which may explain the relatively low scores (Question 2). The input controls are in standard HTML, and they cannot be made easier. Question 3 is about the u201Cintuitivenessu201D of the system. The relatively low scores reflect, in our opinion, the organization of the menu commands. This aspect could be improved by instrumenting the graphical interface and timing the users mouse actions. Question 4 was about statistical graphs and received relatively few answers, possibly because the end-users are medical doctors who are not as familiar with statistics. Those who voted indicated high interest, revealing the importance of a tool for statistically plotting the results. The last question was about data export, and it received few answers. This result might be due to the relatively small amount of stored data. Also, biostatisticians use this function more commonly than medical doctors we used to test the survey. Data export can improve by making the format similar to the more familiar Excel sheets.\"- - Semantic databases built with wiki platforms are flexible and easy for end-users even in the case of complex biomedical data, thanks to the possibility to extend them and to customize the interface, however, they must be created manually or through embedded forms, and therefore it is difficult to create and manage large sets of properties with them. - In this paper, a tool for creating Semantic MediaWiki XML schemata from specifications in a simple TSV format has been proposed along with a method for embedding statistical applications in R Shiny and for exporting Prolog predicates. This allows for increasing the expressiveness of the platform by producing plots, statistical analyses, and rich maps. Additional embedded applications will be studied for other data types, for example, survival curves for time series, Gaussian mixture models for spatial clustering, and Natural Language Processing algorithms for medical reports.- - \\' \\' - A demonstration of the system containing public data adapted from a study about COVID-19 with additional random data is available online at ; access can be allowed by the corresponding author on reasonable request. - \\' \\'- - \\' \\' - \"Since the setup process involving Semantic MediaWiki and its required extensions, the R Shiny server, and an underlying relational database management system is very complex, a set of Docker images orchestrated by Compose has been created. All code has been published as open-source with GNU Public License v3 and it is available at and (Supplementary materials, Appendix_).\" - \\' \\'- - \"Telenti, A. & Jiang, X. Treating medical data as a durable asset. Nat. Genet. 52, 1005u20131010 (2020)._ _ _ Banks, M. A. Sizing up big data. Nat. Med. 26, 5u20137 (2020)._ _ _ Vayena, E. Value from health data: European opportunity to catalyse progress in digital health. Lancet 397, 652u2013653 (2021)._ _ _ Bravo, A., PixF1ero, J., Queralt-Rosinach, N., Rautschka, M. & Furlong, L. I. Extraction of relations between genes and diseases from text and large-scale data analysis: Implications for translational research. BMC Bioinform. (2015)._ _ Andrearczyk, V. et al. Overview of the hecktor challenge at miccai 2020: Automatic head and neck tumor segmentation in pet/ct. In 3D Head and Neck Tumor Segmentation in PET/CT Challenge 1u201321 (Springer, 2020)._ Menze, B. H. et al. The multimodal brain tumor image segmentation benchmark (brats). IEEE Trans. Med. Imaging 34, 1993u20132024 (2014)._ _ Bejnordi, B. E. et al. Diagnostic assessment of deep learning algorithms for detection of lymph node metastases in women with breast cancer. JAMA 318, 2199u20132210 (2017)._ _ Tomczak, K., Czerwiu0144 ska, P. & Wiznerowicz, M. The cancer genome atlas (TCGA): An immeasurable source of knowledge. Contemp. Oncol. 19, A68 (2015)._ Clark, K. et al. The cancer imaging archive (TCIA): Maintaining and operating a public information repository. J. Digit. Imaging 26, 1045u20131057 (2013)._ _ Jarque-Bou, N. J., Atzori, M. & MxFCller, H. A large calibrated database of hand movements and grasps kinematics. Sci. Data 7, 1u201310 (2020)._ _ Atzori, M. et_al. Building the ninapro database: A resource for the biorobotics community. In 2012 4th IEEE RAS & EMBS International Conference on Biomedical Robotics and Biomechatronics (BioRob), 1258u20131265 (IEEE, 2012).Cognolato, M. et al. Gaze, visual, myoelectric, and inertial data of grasps for intelligent prosthetics. Sci. Data 7, 1u201315 (2020)._ Dhrangadhariya, A., Jimenez-delToro, O., Andrearczyk, V., Atzori, M. & MxFCller, H. Exploiting biomedical literature to mine out a large multimodal dataset of rare cancer studies. In Medical Imaging 2020: Imaging Informatics for Healthcare, Research, and Applications Vol. 11318 113180A (International Society for Optics and Photonics, 2020)._ Consortium T. U. UniProt: The universal protein knowledgebase in 2021. Nucleic Acids Res. 49, D480u2013D489. (2020)._ _ _ Kanehisa, M., Sato, Y., Kawashima, M., Furumichi, M. & Tanabe, M. KEGG as a reference resource for gene and protein annotation. Nucleic Acids Res. 44, D457u2013D462. (2015)._ _ _ Robinson, P. N. & Bauer, S. Introduction to Bio-Ontologies 1st edn. (Chapman and Hall, 2011)._ _ Wang, X. et_al. TraPath: Fast regular path query evaluation on large-scale RDF graphs. In Proceedings of International Conference on Web-Age Information Management, pp. 372u2013383. (2014).Rindflesch, T. C., Kilicoglu, M., Halilan, F., Rosemblat, G. & Shin, D. Semantic MEDLINE: An advanced information management application for biomedicine. Inf. Serv. Use 31, 15u201321. (2011)._ _ _ Kilicoglu, H., Shin, D., Fiszman, M., Rosemblat, G. & Rindflesch, T. C. SemMedDB: A PubMed-scale repository of biomedical semantic predications. Bioinformatics 28, 3158u2013 3160 (2012)._ _ _ Percha, B. & Altman, R. B. A global network of biomedical relationships derived from text. Bioinformatics 34, 2614u2013 2624. (2018)._ _ _ Berners-Lee, T., Hendler, J. & Lassila, O. The semantic web. Sci. Am. 284, 34u201343 (2001)._ _ _ Antezana, E., Kuiper, M. & Mironov, V. Biological knowledge management: The emerging role of the Semantic Web technologies. Brief. Bioinform. 10, 392u2013407. (2009)._ _ Chen, H., Yu, T. & Chen, J. Y. Semantic web meets integrative biology: A survey. Brief. Bioinform. 14, 109u2013125. (2012)._ _ Cyganiak, R., Hyland-Wood, D. & Lanthaler, M. RDF 1.1 Concepts and Abstract Syntax. Tech. Rep., W3C (2014).Harris, S. & Seaborne, A. SPARQL 1.1 Query Language (2013).Grafkin, P. et_al. SPARQL Query Builders: Overview and Comparison. In Johansson, B. & VencovskxFD, F. (eds.) Joint Proceedings of the BIR 2016 Workshops, vol. 1684 of CEUR Workshop Proceedings (CEUR-WS.org, 2016).Prasarnphanich, P. & Wagner, C. The role of Wiki Technology and altruism in collaborative knowledge creation. J. Comput. Inf. Syst. 49, 33u201341. (2009)._ _ Hepp, M., Bachlechner, D. & Siorpaes, K. OntoWiki: Community-Driven Ontology Engineering and Ontology Usage Based on Wikis. In Proceedings of the 2006 International Symposium on Wikis, WikiSym 06, 143u2014144. (Association for Computing Machinery, New York, NY, USA, 2006).Baumeister, J. et_al. KnowWEu2014A Wiki for Knowledge Base Development. In KESE@ECAI (2012).Nalepa, G._J. Lokiu2014Semantic Wiki with logical knowledge representation. Trans. Comput. Collect. Intell. 3, 96u2013 114 (under review).KrxF6tzsch, M., Vrandeu010Diu0107, D., VxF6lkel, M., Haller, H. & Studer, R. Semantic Wikipedia. Web Semant. 5, 251u2013261. (2007)._ _ Davies, J., Grobelnik, M., Mladeniu0107, D., Vrandeu010D iu0107, D. & KrxF6tzsch, M. Semantic MediaWiki Vol. 13 (Springer, 2009)._ Bergamini, C. et al. LocalFocus: A big data service platform for local communities and smarter cities. IEEE Commun. Mag. 56, 116u2013123. (2018)._ _ Gherdevich, D. PolimaWiki: Un site contributif pour lxE9tude du pouvoir des listes au Moyen xC2ge. MxE9dixE9vales 73, 149u2013167 (2017)._ _ Abugessaisa, I. et al. FANTOM5 transcriptome catalog of cellular states based on Semantic MediaWiki. Database (2016)._ _ Koren, Y., Gambke, S. et_al. PageForms extension. . Accessed 19 Nov 2021.Koren, Y. & Garg, A. Page Schemas extension. . Accessed 19 Nov 2021.Chang, W. et_al. Shiny: Web Application Framework for R. . Accessed 14 Dec 2022.Topway.it. PageProperties extension. . Accessed 14 Dec 2022.Lampa, S. et al. RDFIO: Extending Semantic MediaWiki for interoperable biomedical data management. J. Biomed. Semant. 8, 1u201313. (2017)._ _ Rosenbusch, H., Wanders, F. & Pit, I. L. The semantic scale network: An online tool to detect semantic overlap of psychological scales and prevent scale redundancies. Psychol. Methods 25, 380u2013392 (2020)._ _ Christensen, A. P. & Kenett, Y. N. Semantic network analysis (SEMNA): A tutorial on preprocessing, estimating, and analyzing semantic networks. Psychol. Methods (2021)._ _ Buchanan, E._M. et_al. Word Space Creator: A visual tool for semantic space visualization. . Accessed 14 Dec 2022.Papadias, E., Kokla, M. & Tomai, E. Educing knowledge from text: Semantic information extraction of spatial concepts and places. AGILE GISci. Ser. 2, 38. (2021)._ _ _ Ait-Mlouk, A., Vu, X.-S. & Jiang, L. Winfra: A web-based platform for semantic data retrieval and data analytics. Mathematics (2020)._ _ Klinke, S. R extension. . Accessed 13 Dec 2022.Klinke, S. Shiny extension. . Accessed 13 Dec 2022.Badaloni, S. & Falda, M. Temporal-based medical diagnoses using a Fuzzy Temporal Reasoning System. J. Intell. Manuf. 21, 145u2013153. (2010)._ _ _ Falda, M., Rossi, F. & Venable, K. B. Dynamic consistency of fuzzy conditional temporal problems. J. Intell. Manuf. 21, 75u201388. (2010)._ _ Bos, J. W., Lauter, K. & Naehrig, M. Private predictive analysis on encrypted medical data. J. Biomed. Inform. 50, 234u2013243. (2014) (Special Issue on Informatics Methods in Medical Privacy)._ _ Wikitext help. . Accessed 19 Nov 2021.Allemang, D. & Hendler, J. Semantic Web for the Working Ontologist: Effective modeling in RDFS and OWL 2nd edn. (Morgan Kaufmann Publishers Inc., 2011)._ Sikos, L. Mastering structured data on the Semantic Web: From HTML5 Microdata to Linked Open Data (Apress, 2015)._ _ Deitel, P. & Deitel, H. C How to Program 8th edn. (Pearson, 2015)._ _ Robinson, P. N. & Bauer, S. Introduction to Bio-ontologies (CRC Press, 2011)._ _ Koren, Y. & et al. Semantic Drilldown extension. . Accessed 19 Nov 2021.Mercier-Linteau, A. & Garrett, A. SemanticACL extension. . Accessed 19 Nov 2021.Kinzler, D. Lockdown extension. . Accessed 22 Nov 2021.De_Dauw, J., Hong_Kong, J., Gambke, S., Koren, Y. et_al. Semantic result formats extension. . Accessed 04 Mar 2022.De_ Dauw, J. et_al. Maps extension. . Accessed 04 Mar 2022.Falda, M. PropChainsHelper extension. . Accessed 15 Jan 2022.Yao, X. & Li, G. Big spatial vector data management: A review. Big Earth Data 2, 1u201322. (2018)._ _ _ McCracken, D. D. & Reilly, E. D. Backus-Naur Form (BNF) 129u2013131 (Wiley, 2003)._ Damas, L. & Milner, R. Principal Type-Schemes for Functional Programs. In Proceedings of the 9th ACM SIGPLAN-SIGACT Symposium on Principles of Programming Languages, POPL 82, 207u2013212, (Association for Computing Machinery, New York, NY, USA, 1982).Lavezzo, E. et al. Suppression of a SARS-CoV-2 outbreak in the Italian municipality of Vo. Nature 584, 425u2013429. (2020)._ _ _ _ SMW pack. . Accessed 6 Jan 2022.Mediawiki technical metadata. . Accessed 19 Nov 2021.Rubin, D. B. Inference and missing data. Biometrika 63, 581u2013592 (1976)._ _ _ Gueron, S., Langley, A. & Lindell, Y. AES-GCM-SIV: Nonce misuse-resistant authenticated encryption. RFC 8452, Internet Research Task Force (2019). .Prolog tutorial. . Accessed 19 Nov 2021.Flach, P. Simply Logical: Intelligent Reasoning by Example (Wiley, 1998)._ _ Tau-Prolog. . Accessed 23 Dec 2021.Wielemaker, J., Lager, T. & Riguzzi, F. SWISH: SWI-Prolog for Sharing. In Proceedings of the International Workshop on User-Oriented Logic Programming (IULP 2015), 99u2013113 (2015).\"- - \"This work was supported by the u201CDepartment of excellence 2018-2022u201D initiative of the Italian Ministry of education (MIUR) awarded to the Department of Neuroscience-University of Padua.\"- - Authors and Affiliations - \"Neuroscience Department, University of Padova, Padova, ItalyMarco Falda,_Manfredo Atzori_&_Maurizio CorbettaInstitute of Information Systems, University of Applied Sciences Western Switzerland (HES-SO Valais), Sierre, SwitzerlandManfredo AtzoriPadova Neuroscience Center (PNC), Clinica Neurologica, and Venetian Institute of Molecular Medicine, VIMM, Padova, ItalyManfredo Atzori_&_Maurizio CorbettaDepartment of Neurology, Radiology, Neuroscience Washington University School of Medicine, St. Louis, MO, USAMaurizio Corbetta\" - \"AuthorsMarco FaldaYou can also search for this author in _Manfredo AtzoriYou can also search for this author in _Maurizio CorbettaYou can also search for this author in _\" - Contributions - M.F. conceived and implemented the idea to extend semantic wikis, and wrote the paper. M.A. and M.C. contributed to the ideation of the project concerning the Neuroscience Department databases, supervised and revised the work. All authors approved the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons CC0 1.0 Universal Public Domain Dedication. Universe: The Observable Universe by Pablo Carlos Budassi from Wikipedia under Attribution-ShareAlike 3.0 Unported (CC BY-SA 3.0) in Public Domain.\" - \"The significance of the crossover temperature Tx in relation to the Curie temperature TC is particularly important when discussing the nature of the magnetic ordering in 2D magnets at zero anisotropy for Tu2009>u20090 K. We investigate this behaviour through colour maps of the spin ordering after 40 million Monte Carlo steps comparing different system sizes and temperatures (Fig._). At very low temperatures Tu2009=u20092.5 K, where there is a high degree of order, the spin directions are highly correlated, as indicated by a mostly uniform colouring. Although the temperatures are near zero, the system is superparamagnetic indicating that over time the magnetisation direction fluctuates, and the effect is most apparent for the smallest sizes where the average direction has moved significantly from the initial direction Su2223u2223z. At higher temperatures, the deviation of the spin directions within the sample increases as indicated by the more varied colouring. To quantitatively assess the spin deviations we plot the statistical distribution of angle between the spin direction and the mean direction for different temperatures for each size (Supplementary Fig._). For an isotropic distribution on the unit sphere there is a (sin (theta )) weighting, which is seen at the highest temperature for all system sizes. For lower temperatures where the spin directions are more correlated, the distribution is biased towards lower angles. Qualitatively there is little difference in the spin distributions for the different samples. At Tu2009=u200920 K, there is, however, a systematic trend in the peak angle increasing from u03B8u2009=u200940xB0 for the 50u2009 xD7u200950u2009nm flake (Supplementary Fig._a) to around u03B8u2009=u2009 60xB0 at 1000u2009xD7u20091000u2009nm (Supplementary Fig._) indicating an increased level of disorder averaged over the whole sample. This effect is straightforwardly explained by the size dependence of spin-spin correlations (Supplementary Fig._ ). At small sizes the spins are strongly exchange coupled, preventing large local deviations of the spin directions. At longer length scales available for the larger systems, the variations in the magnetisation direction are also larger. Surprisingly, our calculations reveal that this effect is weak: even for very large flakes of a micrometre in size, only a small increase can be observed in the position of the peak in the angle distribution at a fixed temperature. Above the crossover temperature, the spin-spin correlation length becomes very small compared to the system size with rapid local changes in the magnetisation direction, indicative of a completely disordered paramagnetic state. Our analysis reveals that the spins in finite-sized 2D isotropic magnets are strongly aligned due to short-range order at non-zero temperatures and up to the crossover temperature.\" - \"Fig. 4: Temperature-dependent magnetic order.Visualisations of the magnetic spin configurations for the honeycomb lattice starting from an ordered state as a function of system size (vertical column) at different temperatures. The spins are projected following the colour scale shown in the sphere on the left. The bottom row shows a local view of the spins inside a 5u2009nmu2009xD7u2009 5 nm area at the location outlined by the small boxes in the 1000u2009xD7u2009 1000u2009nm snapshots.\"- - \"Mathematically a phase transition is defined as a non-analytic change in the state variable for the system, such as the particle density or the magnetisation in the case of spin systems. For any finite system the state variable is continuous by definition due to a finite number of particles, forming a continuous path of intermediate states between two distinct physical phases. The same is true for a magnetic system, forming a continuous path between an ordered and a paramagnetic state. A priori then, it is impossible to have a true phase transition for any finite magnetic samples which are routinely implemented in device platforms. Yet, nanoscale magnets that are far from infinite have been studied for decades and exhibit a clear crossover from magnetically ordered to paramagnetic phases, occurring for systems only a few nanometres in size. The crossover temperature in a finite-size system hence can be described as an inflection point in u3008 u2223mu2223u3009. The precise definition of a phase transition is significant when considering the main conclusions of Mermin and Wagner, which explicitly only apply in the case of an infinite system. As our results clearly show, sample sizes measured experimentally are not classifiable as infinite and, therefore, not subject to the Mermin-Wagner theorem. It is noteworthy that 3D compounds have weak dependence of their critical temperature on magnetic anisotropy. Similar analysis performed for a finite 3D bulk system (Supplementary Fig._a, b) show that the inclusion of anisotropy barely changes the results for Tc. This suggests that magnetism is an exchange-driven effect in both two and three dimensions.\" - \"On the practical side, heterostructures with conventional metallic magnetic materials could establish preferential directions of the magnetisation through anisotropic exchange and dipolar couplings. However, it is important to point out that the short-range order is enforced by the isotropic exchange couplings, and even a low anisotropy may suffice for stabilizing the direction of the magnetisation in the vdW layers, i.e., from underlying magnetic substrates. We can imagine micrometre-sized samples where all spins are still correlated at finite temperatures so it could represent a single bit. However, for miniaturization purposes multiple nanometre-sized bits are required on the same sample in order to be implemented in recording media. This is typically achieved by magnetic domains, but there are no domains in an isotropic model since the domain wall width is infinite. However, if vdW layers can be grown with grain boundaries, like in 2D mosaics, which are large enough that each grain area would have a uniform magnetisation, then a magnetic monolayer would have as many bits as available on the material surface. The underlying substrate hence would set the magnetisation direction for further implementations. This spin-interface engineering would be a considerable step towards on-demand magnetic properties at the atomic level given the flexibility on the orientation of the magnetic moments without a predefined direction at the layer. While the anisotropy circumvents the Mermin-Wagner theorem and causes the critical temperature Tc to be nonzero in infinitely large systems, in finite samples the short-range order persists up to much higher temperatures (Txu2009 >u2009Tc) since Tx is proportional to the isotropic exchange rather than the anisotropy. Indeed, the long tail features observed in the intrinsic magnetisation (Fig._) extending above the crossover temperature suggest that short-range order is present. In addition, the existence of short-range order in bulk magnetic systems near and above the Curie temperature has been experimentally and theoretically discovered in elemental transition metals. These studies indicate the persistence of magnetic ordering within the supposedly disordered phase above the Curie temperature, where any ordered phase is primarily controlled by exchange interactions as in the case for 2D magnets. For instance, in bcc-Fe a short-range order within 5.4 xC5 was found which is much smaller than the magnitudes obtained in our simulations for vdW materials.\" - \"In conclusion, we presented large-scale spin dynamics simulations and analytical calculations on the temperature dependence of the intrinsic magnetisation in 2D magnetic materials described by an isotropic Heisenberg model. We found that short-range magnetic order at non-zero temperature is a robust feature of isotropic 2D magnets even at experimentally accessible length and time scales. Our data show that the often asserted Mermin-Wagner limit does not apply to 2D materials on real laboratory sample sizes . Since the spins are aligned due to the exchange interactions already in the isotropic model, the direction of the magnetisation may be stabilized by geometrical factors or finite-size effects. These findings open up possibilities for a wider range of 2D magnetic materials in device applications than previously envisioned. Furthermore, the limited applicability of the analytical Merminu2013Wagner theorem opens similar possibilities in other fields such as superconductivity and liquid crystal systems, where the relevant length scale of correlations is known to be much greater than that required for experimental measurements and applications. Our results suggest that if the magnetic anisotropy can be controlled to a certain degree until it completely vanishes, new effects of strongly correlated spins or more unusual disordered states may be observed.\"- - \\'We used atomistic simulations methods implemented in the VAMPIRE software to compute the magnetic properties of 2D magnetic materials. The energy of our system is calculated using the spin Hamiltonian:\\' - \"$${{{{{{{mathcal{H}}}}}}}}=-mathop{sum}limits_{i < j}{J}_{ij}{{{{{{{{ bf{S}}}}}}}}}_{i}cdot {{{{{{{{bf{S}}}}}}}}}_{j}-Kmathop{sum}limits_{i}{ left({S}_{i}^{z}right)}^{2},$$ (4) \" - \"where Si,j are unit vectors describing the local spin directions on magnetic sites i,u2009j and Jij is the exchange constant between spins. An easy-axis magnetocrystalline anisotropy constant K can be included as well, with negligible modifications of the results as described in the text. Simulations were run for system sizes of 50u2009nm, 100u2009nm, 500u2009nm and 1000u2009nm laterally along the x and y directions with periodic boundary conditions (PBCs), and 1 atomic layer thick along the z direction. Similar PBCs were used in the analytical model. However, simulation results using open boundary conditions (OBCs) ended up in similar conclusions (Supplementary Fig._). For the honeycomb lattice, the simulations were initialised in either a perfectly ordered state aligned along the z direction or a random state corresponding to infinite temperature. For these simulations the final u3008u2223mu2223u3009(T) curves were identical to each other. However, at low temperatures it took ten times as many steps to reach the final equilibrium state from the random state, so for the remaining structures only simulations starting from the ordered states were run. The systems were integrated using a Monte Carlo integrator using a uniform sampling algorithm to remove any bias introduced from more advanced algorithms. To investigate the temperature dependence, the simulation temperature was varied from 0 to 90 K in 2.5 K steps. 40u2009xD7u200910 Monte Carlo steps were run for each temperature step. This was split into 39u2009xD7u200910 equilibration steps and then 10 time steps from which the statistics were calculated. The Monte Carlo simulations use a pseudo-random number sequence generated by the Mersenne Twister algorithm due to its high quality, avoiding correlations in the generated random numbers and with an exceptionally long period of 2u2009u2212u2009 1u2009~u200910. The parallel implementation generates different random seeds on each processor to ensure no correlation between the generated random numbers.\" - \"The time-dependent simulations in Fig. c, d were performed by solving the stochastic Landauu2013Lifshitzu2013Gilbert equation:\" - \"$$frac{partial {{{{{{{{bf{S}}}}}}}}}_{i}}{partial t}=-frac{{gamma }_{e}}{1+{lambda }^{2}}left[{{{{{{{{bf{S}}}}}}}}}_{i}times {{{{{{{{ bf{B}}}}}}}}}_{{{{{{{{rm{eff}}}}}}}}}+lambda {{{{{{{{bf{S}}}}}}}}}_{i} times left({{{{{{{{bf{S}}}}}}}}}_{i}times {{{{{{{{bf{B}}}}}}}}}_{{{{{{{{ rm{eff}}}}}}}}}right)right],$$ (5) \" - \"which models the interaction of an atomic spin moment Si with an effective magnetic field ({{{{{{{{bf{B}}}}}}}}}_{{{mbox{eff}}}}= - 1/{{{mathrm{mu}}}}_{{{ mathrm{s}}}} , partial {{{{{{{mathcal{H}}}}}}}}/partial {{{{{{{{bf{S}}}}}}}}}_{i} ). The effective field causes the atomic moments to precess around the field, where the frequency of precession is determined by the gyromagnetic ratio of an electron (u03B3eu2009=u20091.76u2009xD7u200910 rad sT) and u03BBu2009 =u20091 is the damping constant. The large value of u03BB was used to accelerate the relaxation dynamics in order to be computationally achievable (u2009~u2009 72 hours). For a different damping, one has to wait longer or shorter for this to happen. Based on the system sizes used in our computations, this can vary betweenu2009~u20095 days up to several weeks, which is not practical. However, once the system is at equilibrium, the value of the damping is not important. Moreover, a large damping would correspond to large fluctuations on the magnitude of the magnetisation and its direction. Lower damping would lead to naturally slower dynamics of the magnetisation. Nevertheless, we barely noticed any at the timescale included in our work (Fig._). It is worth mentioning that no damping parameter are used in the Monte Carlo calculations which support our conclusions. The effect of temperature is taken into account using Langevin dynamics (as in Eq. ()), where the thermal fluctuations are represented by a Gaussian white noise term. At each time step the instantaneous thermal field acting on each spin is given by\" - \"$${{{{{{{{bf{B}}}}}}}}}_{{{mbox{th}}}}^{i}=sqrt{frac{2lambda {k}_{B}T}{ gamma {mu }_{s}Delta t}}{{{{{{{boldsymbol{Gamma }}}}}}}}(t)$$ (6) \" - \"where kB is the Boltzmann constant, T is the system temperature and u0393(t) is a vector of standard (mean 0, variance 1) normal variables which are independent in components and in time. The thermal field is added to the effective field in order to simulate a heat bath. The system was integrated using a Heun numerical scheme.\"- - \\' \\' - The data that support the findings of this study are available within the paper and its Supplementary Information. - \\' \\'- - \"Dieny, B. et al. Opportunities and challenges for spintronics in the microelectronics industry. Nat. Electron. 3, 446u2013459 (2020)._ _ Sander, D. et al. The 2017 magnetism roadmap. J. Phys. D: Appl. Phys. 50, 363001 (2017)._ _ Wang, Q. H. et al. The magnetic genome of two-dimensional van der Waals materials. ACS Nano. 16, 6960u20137079 (2022)._ _ _ _ _ Hohenberg, P. C. Existence of long-range order in 1 and 2 dimensions. Phys. Rev. 158, 383u2013386 (1967)._ _ _ _ Mermin, N. D. & Wagner, H. Absence of ferromagnetism or antiferromagnetism in one- or two-dimensional isotropic Heisenberg models. Phys. Rev. Lett. 17, 1133u20131136 (1966)._ _ _ _ Wahab, D. A. et al. Quantum rescaling, domain metastability, and hybrid domain-walls in 2d CrI3 magnets. Adv. Mater. 33, 2004138 (2021)._ _ _ Kim, M. et al. Micromagnetometry of two-dimensional ferromagnets. Nat. Electron. 2, 457u2013463 (2019)._ _ _ Kapikranian, O., Berche, B. & Holovatch, Y. Quasi-long-range ordering in a finite-size 2d classical Heisenberg model. J. Phys. A: Math. Theor. 40, 3741u20133748 (2007)._ _ _ _ _ Palle, G. & Sunko, D. K. Physical limitations of the Hohenberg-Mermin-Wagner theorem. J. Phys. A: Math. Theor. 54, 315001 (2021)._ _ _ _ _ Jongh, L. J. D. & Miedema, A. R. Experiments on simple magnetic model systems. Adv. Phys. 50, 947u20131170 (2001)._ _ _ Pomerantz, M. Experiments on literally two-dimensional magnets. Surf. Sci. 142, 556u2013570 (1984)._ _ _ _ Wehr, J., Niederberger, A., Sanchez-Palencia, L. & Lewenstein, M. Disorder versus the Mermin-Wagner-Hohenberg effect: From classical spin systems to ultracold atomic gases. Phys. Rev. B. 74, 224448 (2006)._ _ _ Niederberger, A. et al. Disorder-induced order in two-component Bose-Einstein condensates. Phys. Rev. Lett. 100, 030403 (2008)._ _ _ _ _ Niederberger, A. et al. Disorder-induced order in quantum XY chains. Phys. Rev. A. 82, 013630 (2010)._ _ _ Crawford, N. On random field-induced ordering in the classical xy model. J. Stat. Phys. 142, 11u201342 (2011)._ _ _ _ _ Crawford, N. Random field induced order in low dimension. EPL (Europhys. Lett.) 102, 36003 (2013)._ _ _ _ Imry, Y. & Ma, S.-k Random-field instability of the ordered state of continuous symmetry. Phys. Rev. Lett. 35, 1399u20131401 (1975)._ _ _ _ Halperin, B. I. On the Hohenbergu2013Merminu2013Wagner theorem and its limitations. J. Stat. Phys. 175, 521u2013529 (2019)._ _ _ _ _ Stanley, H. E. & Kaplan, T. A. Possibility of a Phase Transition for the Two-Dimensional Heisenberg Model. Phys. Rev. Lett. 17, 913u2013915 (1966)._ _ _ _ Shenker, S. H. & Tobochnik, J. Monte Carlo renormalization-group analysis of the classical Heisenberg model in two dimensions. Phys. Rev. B. 22, 4462u2013 4472 (1980)._ _ _ _ BlxF6te, H. W. J., Guo, W. & Hilhorst, H. J. Phase transition in a two-dimensional Heisenberg model. Phys. Rev. Lett. 88, 047203 (2002)._ _ _ _ Tomita, Y. Finite-size scaling analysis of pseudocritical region in two-dimensional continuous-spin systems. Phys. Rev. E. 90, 032109 (2014)._ _ _ Roduner, E. Size matters: why nanomaterials are different. Chem. Soc. Rev. 35, 583u2013592 (2006)._ _ _ _ Singh, R. Unexpected magnetism in nanomaterials. J. Magn. Magn. Mater. 346, 58u201373 (2013)._ _ _ _ Kachkachi, H. & Garanin, D. A. Boundary and finite-size effects in small magnetic systems. Phys. A: Stat. Mech. its Appl. 300, 487u2013504 (2001)._ _ _ _ Sun, Q.-C. et al. Magnetic domains and domain wall pinning in atomically thin CrBr3 revealed by nanoscale imaging. Nat. Commun. 12, 1989 (2021)._ _ _ _ _ Abdul-Wahab, D. et al. Domain wall dynamics in two-dimensional van der Waals ferromagnets. Appl. Phys. Rev. 8, 041411 (2021)._ _ _ _ Miao, N. & Sun, Z. Computational design of two-dimensional magnetic materials. WIREs Computational Mol. Sci. 12, e1545 (2022)._ _ _ Gong, C. et al. Discovery of intrinsic ferromagnetism in two-dimensional van der Waals crystals. Nature 546, 265u2013269 (2017)._ _ _ _ _ Huang, B. et al. Layer-dependent ferromagnetism in a van der Waals crystal down to the monolayer limit. Nature 546, 270u2013 273 (2017)._ _ _ _ _ Alzate-Cardona, J. D., Sabogal-SuxE1rez, D., Evans, R. F. L. & Restrepo-Parra, E. Optimal phase space sampling for Monte Carlo simulations of Heisenberg spin systems. J. Phys.: Condens. Matter 31, 095802 (2019)._ _ _ MinxE1 r, J., Bornemann, S., u0160ipr, O., Polesya, S. & Ebert, H. Magnetic properties of CO clusters deposited on Pt(111). Appl. Phys. A. 82, 139u2013144 (2006)._ _ _ u0160ipr, O. et al. Magnetic moments, exchange coupling, and crossover temperatures of CO clusters on Pt(111) and Au(111). J. Phys.: Condens. Matter 19, 096203 (2007)._ _ Brown, W. F. Thermal fluctuations of a single-domain particle. Phys. Rev. 130, 1677u2013 1686 (1963)._ _ _ Bussian, D. A. et al. Tunable magnetic exchange interactions in manganese-doped inverted coreu2013shell ZnSeu2013 CdSe nanocrystals. Nat. Mater. 8, 35u201340 (2009)._ _ _ _ _ Rinehart, J. D., Fang, M., Evans, W. J. & Long, J. R. A n23u2013radical-bridged terbium complex exhibiting magnetic hysteresis at 14 K. J. Am. Chem. Soc. 133, 14236u201314239 (2011)._ _ _ _ Baumgarten, M. Tuning the magnetic exchange interactions in organic biradical networks. Phys. status solidi (b) 256, 1800642 (2019)._ _ _ Garanin, D. A. Spherical model for anisotropic ferromagnetic films. J. Phys. A: Math. Gen. 29, L257u2013L262 (1996)._ _ _ _ Garanin, D. A. Ordering in magnetic films with surface anisotropy. J. Phys. A: Math. Gen. 32, 4323u20134342 (1999)._ _ _ Nightingale, M. P. & BlxF6te, H. W. J. Dynamic exponent of the two-dimensional Ising model and Monte Carlo computation of the subdominant eigenvalue of the stochastic matrix. Phys. Rev. Lett. 76, 4548u2013 4551 (1996)._ _ _ _ _ Leggett, A. J.Lecture 9 from Lecture Notes Physics in Two Dimensions delivered at the University of Illinois at Urbana-Champaign36003 (2013).Stanley, H.Introduction to Phase Transitions and Critical Phenomena. International series of monographs on physics (Oxford University Press, 1971). .Coey, J. M. D. J. Magn. Magn. Mater. (Cambridge University Press, Cambridge, 2010).Yao, W., Wu, B. & Liu, Y. Growth and grain boundaries in 2d materials. ACS Nano 14, 9320u20139346 (2020)._ _ _ _ Irkhin, V. Y., Katanin, A. A. & Katsnelson, M. I. Self-consistent spin-wave theory of layered Heisenberg magnets. Phys. Rev. B. 60, 1082u20131099 (1999)._ _ _ _ Grechnev, A., Irkhin, V. Y., Katsnelson, M. I. & Eriksson, O. Thermodynamics of a two-dimensional Heisenberg ferromagnet with dipolar interaction. Phys. Rev. B. 71, 024427 (2005)._ _ _ Haines, E. M., Clauberg, R. & Feder, R. Short-range magnetic order near the Curie temperature iron from spin-resolved photoemission. Phys. Rev. Lett. 54, 932u2013934 (1985)._ _ _ _ _ Maetz, C. J., Gerhardt, U., Dietz, E., Ziegler, A. & Jelitto, R. J. Evidence for short-range magnetic order in Ni above Tc. Phys. Rev. Lett. 48, 1686u20131689 (1982)._ _ _ _ Antropov, V. Magnetic short-range order above the curie temperature of Fe and Ni. Phys. Rev. B. 72, 140406 (2005)._ _ _ Illing, B. et al. Mermin-Wagner fluctuations in 2d amorphous solids. Proc. Natl Acad. Sci. USA 114, 1856u20131861 (2017)._ _ _ _ _ _ Verzhbitskiy, I. A. et al. Controlling the magnetic anisotropy in Cr2Ge2Te6 by electrostatic gating. Nat. Electron. 3, 460u2013465 (2020)._ _ _ Kartsev, A., Augustin, M., Evans, R. F. L., Novoselov, K. S. & Santos, E. J. G. Biquadratic exchange interactions in two-dimensional magnets. npj Comput. Mater. 6, 150 (2020)._ _ _ Alliati, I. M., Evans, R. F. L., Novoselov, K. S. & Santos, E. J. G. Relativistic domain-wall dynamics in van der Waals antiferromagnet MnPS3. npj Comput. Mater. 8, 3 (2022)._ _ _ _ Strungaru, M., Augustin, M. & Santos, E. J. G. Ultrafast laser-driven topological spin textures on a 2d magnet. npj Comput. Mater. 8, 169 (2022)._ _ _ Augustin, M., Jenkins, S., Evans, R. F. L., Novoselov, K. S. & Santos, E. J. G. Properties and dynamics of meron topological spin textures in the two-dimensional magnet CrCl3. Nat. Commun. 12, 185 (2021)._ _ _ _ _ Dabrowski, M. et al. All-optical control of spin in a 2d van der waals magnet. Nat. Commun. 13, 5976 (2022)._ _ _ _ Evans, R. F. L. et al. Atomistic spin model simulations of magnetic nanomaterials. J. Phys.: Condens. Matter 26, 103202 (2014)._ _ _ Matsumoto, M. & Nishimura, T. Mersenne Twister: a 623-dimensionally equidistributed uniform pseudo-random number generator. ACM Trans. Model. Comput. Simul. 8, 3u2013 30 (1998)._ _ _ Brown, W. Thermal fluctuation of fine ferromagnetic particles. IEEE Trans. Magn. 15, 1196u20131208 (1979)._ _ _ \"- - \"We thank David Mermin, Mikhail Katsnelson, and Bertrand Halperin for valuable discussions. L.R. gratefully acknowledges funding by the National Research, Development and Innovation Office of Hungary via Project No. K131938 and by the Young Scholar Fund at the University of Konstanz. U.A. gratefully acknowledges funding by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation)-Project-ID 328545488-TRR 227, Project No. A08; and grants PID2021-122980OB-C55 and RYC-2020-030605-I funded by MCIN/AEI/10.13039/501100011033 and by u201CERDF A way of making Europeu201D and u201CESF Investing in your futureu201D. E.J.G.S. acknowledges computational resources through CIRRUS Tier-2 HPC Service (ec131 Cirrus Project) at EPCC funded by the University of Edinburgh and EPSRC (EP/P020267/1); ARCHER UK National Supercomputing Service () via Project d429. E.J.G.S. acknowledges the Spanish Ministry of Sciences grant program u201CEuropa-Excelenciau201D under grant number EUR2020-112238, the EPSRC Early Career Fellowship (EP/T021578/1), and the University of Edinburgh for funding support. K.S.N. is supported by the Ministry of Education, Singapore, under its Research Centre of Excellence award to the Institute for Functional Intelligent Materials (I-FIM, project No. EDUNC-33-18-279-V12) and by the Royal Society (UK, grant number RSRPR 190000). For the purpose of open access, the authors have applied a Creative Commons Attribution (CC BY) licence to any Author Accepted Manuscript version arising from this submission.\"- - Authors and Affiliations - \"Department of Physics, University of York, York, YO10 5DD, UKSarah Jenkins_ &_Richard F. L. EvansTWIST Group, Institut fxFCr Physik, Johannes Gutenberg UniversitxE4t, 55128, Mainz, GermanySarah JenkinsTWIST Group, Institut fxFC r Physik, UniversitxE4t Duisburg-Essen, Campus Duisburg, 47057, Duisburg, GermanySarah JenkinsFachbereich Physik, UniversitxE4t Konstanz, D-78457, Konstanz, GermanyLevente RxF3zsaDahlem Center for Complex Quantum Systems and Fachbereich Physik, Freie UniversitxE4t Berlin, 14195, Berlin, GermanyUnai AtxitiaInstituto de Ciencia de Materiales de Madrid, CSIC, Cantoblanco, 28049, Madrid, SpainUnai AtxitiaInstitute for Functional Intelligent Materials, National University of Singapore, Singapore, 117544, SingaporeKostya S. NovoselovInstitute for Condensed Matter Physics and Complex Systems, School of Physics and Astronomy, The University of Edinburgh, Edinburgh, EH9 3FD, United KingdomElton J. G. SantosDonostia International Physics Center (DIPC), 20018 Donostia-San SebastixE1n, Basque Country, SpainElton J. G. SantosHiggs Centre for Theoretical Physics, The University of Edinburgh, Edinburgh, EH9 3FD, United KingdomElton J. G. Santos\" - \"AuthorsSarah JenkinsYou can also search for this author in _Levente RxF3zsaYou can also search for this author in _Unai AtxitiaYou can also search for this author in _Richard F. L. EvansYou can also search for this author in _Kostya S. NovoselovYou can also search for this author in _Elton J. G. SantosYou can also search for this author in _\" - Contributions - E.J.G.S. conceived the idea and supervised the project. S.J. performed the atomistic simulations with inputs from E.J.G.S. and R.F.L.E. L.R. and U.A. developed the semi-analytical model and undertook the numerical simulations. E.J.G.S. wrote the paper with a draft initially prepared by S.J. and R.F.L.E. and also with inputs from K.S.N., U.A. and L.R. All authors contributed to this work, read the manuscript, discussed the results, and agreed on the included contents. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \"Nature Communications thanks Antonio T. Costa, Zhimei Sun and the other, anonymous, reviewer(s) for their contribution to the peer review of this work._ are available.\" - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons CCBY).\"- - Hazard-rate-based models of RT - \"In all, 24 subjects generated ~3500 RTs each. In all three modalities, the RTs were strongly modulated by the u2018go time probability distributions. Specifically, the two symmetric PDFs presented, PDExp and PDFlip, lead to nearly symmetric patterns of RT. All explanatory variables were derived from the HR and PDF of each of the presented u2018go time distributions. First, we fitted the RT data with a linear model of the commonly proposed variable u201C temporally-blurred, mirrored HRu201D. This popular model failed to fit the data adequately. In the PDExp condition (Fig._), the explanatory variable captured the behavior of the RT data mostly in the latter half of u2018go times. In the early part of the distribution, there were significant deviations between data and explanatory variable, reflected in low R values of the fitted models.\" - \"Fig. 3: Models based on the hazard rate (HR) fail to capture reaction time (RT) data.a The commonly used temporally-blurred HR (black line) in its linearly transformed (u201Cmirroru201D) version fitted to auditory (red), visual (blue), and somatosensory (violet) RT data from PDExp condition. The HR model deviates from RT in the early range of u2018go times. b As a but for PDFlip condition. Note the substantial, qualitative mismatch between the HR model and the RT data. c Fits of the temporally-blurred, reciprocal (u201C1 / u201D) HR to RT in the PDExp condition. The fit at the early u2018go times is not improved by the reciprocal model, compare with a. d As c but for PDFlip condition. The reciprocal relationship between HR model and RT improves the fit in the PDFlip condition, compare with c. See Supplementary Figs._ and for detailed summary of all other fitted HR-based models.\" - \"In the PDFlip condition (Fig._), the deviation of the explanatory variable from the observed RTs was striking in all modalities, suggesting that Hypotheses A and B do not hold as canonical rules across different probability distributions. The results indicate that the mirrored HR is not employed by the brain to model event probability across time. We next tested whether the inverse relation between RT and HR can be better captured by a simple non-linear transformation. To do this, we replaced the mirrored transformation with the reciprocal, in which the HR is inverted by division instead of being linearly mirrored. This variable, the u201Ctemporally-blurred, reciprocal HRu201D, significantly improved the fit to RT in the PDFlip condition in all modalities (Fig._). However, in the PDExp condition we observed no improvement but rather a deterioration in the fit (Fig._). Thus, the HR is an unlikely transformation to model event probability across time.\" - PDF-based models of RT - \"We next turned to models based on a more fundamental, u2018core probability parameter, the PDF itself. Similar to HR, we first examined the variable u201C temporally-blurred, mirrored PDFu201D. Although the shape of the explanatory variable captured trends in the data, the fit to RT was poor, especially in the PDExp condition (Supplementary Fig._). We then examined the non-linear, reciprocal transformation by using the u201Ctemporally-blurred, reciprocal PDFu201D as an explanatory variable (Fig._).\" - \"Fig. 4: Models based on probabilistically-blurred, reciprocal PDF capture reaction time (RT).a Model fits to auditory (red), visual (blue), and somatosensory (violet) RT based on the temporally-blurred, reciprocal PDF (black line) in the PDExp condition. b As a but for PDFlip condition. Since temporal blurring increases with u2018go time, it has a stronger smoothing impact on the model at longer u2018go times. This feature is not reflected in the data (u2018 go time span 1.2 to 1.43u2009s highlighted). c Model fits to RT based on the probabilistically-blurred, reciprocal PDF in the PDExp condition, d As c but for PDFlip condition. e Comparison between two reciprocal PDF-based models (u201Ctemporally-blurredu201D and u201Cprobabilistically- blurredu201D), in each of four equally spaced bins across the u2018go time span. e In the PDExp condition, the quotient of squared residuals (u201Ctemporally-blurredu201D /u201Cprobabilistically-blurredu201D) is relatively close to 1 in all bins, indicating residuals of similar magnitude across the two models and thus a similar goodness-of-fit. f As e but for the PDFlip condition, residuals are bigger in the temporally-blurred model compared to the probabilistically-blurred model in all bins, especially in bin #4 (highlighted). See Supplementary Figs._ and for a detailed summary of all other fitted PDF-based models.\" - \"The fit to the data was significantly improved, as shown by the similar behavior of model and data across u2018go time, confirmed by the high R values. Evidently, the reciprocal transformation of the PDF provided a much better fit than the mirror transformation as well as both transformations of the HR. In the PDFlip condition in all modalities, we observed that the explanatory variable settles to a plateau of values quite early, at a u2018go time of around 1u2009s, while the actual data continue to have a negative slope (Fig._, gray shading). Additionally, in the early range of u2018go times (e.g. 0.5u2013 0.9u2009s) the explanatory variable decreases with a steeper slope than the data. These systematic differences between model and data were introduced by the temporal blurring. Therefore, in the early range of u2018go times the Gaussian kernel has a small variance and causes only little blurring, less than the data suggest. In the late range of u2018go times, the Gaussian kernel has a large variance, leading to stronger blurring of the explanatory variable, much more than the data suggest.\" - Event probability modulates elapsed time estimation - \"These observations lead us to question the concept of monotonically increasing uncertainty that scales with elapsed time itself. Instead, we hypothesize that uncertainty in elapsed time estimation is modulated by temporal probability. This results in a blurring kernel with large variance in the early, less probable range of u2018go times in the PDFlip condition and in a blurring kernel with small variance in the late, more probable range of u2018go times. We applied this probabilistic blurring to the reciprocal PDF and fitted this variable to RT. This simple operation drastically improved the fit in the PDFlip condition (Fig._). In the PDExp condition (Fig._), the fit was almost identical to the temporal blurring case (Fig._). This is expected, as in both blurring cases the variance of the Gaussian kernel monotonically increased with the magnitude of u2018go times. To quantify differences in model fit between temporally and probabilistically blurred variables, we divided the range of u2018go times into four equal-sized bins. Within each bin, the sum of squared residuals was calculated for the u201Ctemporally-u201D and u201Cprobabilistically-blurred, reciprocal PDFu201D. In the PDExp condition (Fig._) the quotient is relatively close to 1 in all bins, indicating residuals of similar magnitude across the two models and thus a similar goodness-of-fit. In the PDFlip condition (Fig._ ) the quotient is generally larger, and always higher than 1, which shows that the probabilistically blurred model yielded smaller residuals in all bins and in all modalities. Clearly, probabilistic blurring represented the uncertainty in elapsed time estimation better than the commonly employed temporal blurring._ For comparison and completeness, three more variables, the u201Cprobabilistically-blurred, mirrored HRu201D, the u201Cprobabilistically-blurred, reciprocal HRu201D and the u201Cprobabilistically-blurred, mirrored PDFu201D were also investigated (Supplementary Figs._u2013). None of these variables provided a better fit than the variable u201Cprobabilistically-blurred, reciprocal PDFu201D. We conclude that, in all three modalities, the probabilistically-blurred, reciprocal PDFu2014but not the HRu2014is the most adequate model for mapping the brain s temporal-probabilistic input onto its output, i.e. reaction times.\" - Control analyses - \"The above results were further validated by two control experiments and various control analyses. These included split-data analyses and single-subject analyses (Supplementary Note_, Supplementary Figs._u2013), a control experiment without catch trials (Supplementary Note_, Supplementary Fig._), and a control experiment using a Gaussian u2018go time distribution (Supplementary Note_, Supplementary Fig._).\" - \"Modality-specific processes at short u2018go times\" - \"All three distributions (exponential, flipped exponential, and Gaussian, for the latter see Supplementary Fig._) show that there is an upward bend in RT curves at short u2018go times in vision and somatosensation. This effect is absent in the case of audition, in which the models fit the RT curves even at short u2018go times. Note that neither the HR-based, nor the PDF-based models contain a parameter for modality-specificity. Given the differences across sensory modalities, a dedicated model component is needed to account for modality-specificity. To quantify these effects, we used the auditory condition as a reference and subtracted the auditory RT from visual and somatosensory RT. The difference curves show that the highest u2206RT values occur at the shortest u2018go times and thereafter u2206RT monotonically decreases with u2018go time (Fig._). This pattern was observed in all conditions, regardless of distribution. This confirmed the hypothesis that the processing of events at shorter u2018 go times differs between audition and vision/somatosensation. The process leading to the observed u2206RT curves, appears to be captured well by a simple exponential function of u2018go time (Eq._(), Fig._ black fit curves). Adding this fitted exponential model to the PDF-based model from the auditory condition drastically improved the model fit in vision and somatosensation (Fig._ ). The hypothesized exponential process may be related to differences in time estimation between audition and the other modalities, as audition has been suggested to process events faster and more precisely. Alternatively, the modality-specificity in RT might result from differences in the processing of probability over time.\" - \"Fig. 5: Exponential function of u2018go time captures RT differences between modalities.a Within exponential, flipped exponential and Gaussian conditions, u2206RT curves were calculated by subtracting auditory RT curves from visual (blue) and somatosensory (magenta) ones. The u2206RT curves show a decrease over the range of u2018go times, most pronounced at shorter u2018go times, for all distributions. An exponential model (black curves) of u2018 go time (Eq._()) captured the u2206RT curves behavior. b The fitted exponential model was added to the reciprocal, probabilistically-blurred PDF model fitted to auditory RT and plotted against visual and somatosensory RT. This combined model captured the previous deviations between data and model at shorter u2018go times in all conditions in both vision and somatosensation.\" - Modality-specific and modality-independent components of RT - \"The striking similarities in processing temporal-probabilistic structures across sensory modalities suggest shared neural processes, while the differences, e.g. in processing speed, likely reflect modality-specificity. The RT distributions are the result of the superposition of such neural mechanisms. Accordingly, we characterize the RT distributions based on the hypothesis of separate, additive contributions to the mapping of an event PDF onto the corresponding RT. One contribution presumably is modality-specific, reflecting more peripheral processing stages, including latencies in sensory signal transduction and feed-forward information processing. The other contribution is hypothesized to reflect modality-independent processes associated more with the processing of the u2018go time PDF itself. QQ-plots revealed overall similarities in RT distribution across PDExp and PDFlip conditions, but also heavier right tails in the PDFlip condition in all three modalities (Fig._).\" - \"Fig. 6: Reaction time (RT) variance is sensitive to the probability distribution of events.a In all three sensory modalities, QQ-Plots reveal heavier right tails in the RT distributions in the PDFlip condition. b Mean of median RT in PDExp (u2018E) and PDFlip (u2018F) conditions. Average RT is not sensitive to the presented probability distribution. (planned contrasts, *Pu2009<u2009 0.05, **Pu2009<u20090.01, two-tailed Students t-test). c Mean of interquartile range of RT in PDExp and PDFlip condition. RT variance is higher in the PDFlip condition (planned contrasts, *Pu2009<u20090.05, **Pu2009<u20090.01, two-tailed Students t-test). d The Ex-Gaussian distribution is a convolution of an exponential and a Gaussian PDF. e The ex-Gaussian model fits RT histograms on the group level (e) and on the single-subject level (see Supplementary Fig._ ). fu2013h Ex-Gaussian fit parameters from single-subject fits. f Mean of Gaussian u03BC resembles mean of median RT (compare to b, planned contrasts, *Pu2009 <u20090.05, **Pu2009<u20090.01, two-tailed Students t-test). g Mean of exponential u03C4 resembles interquartile range of RT (compare to c, planned contrasts, *Pu2009<u20090.05, **Pu2009<u20090.01, two-tailed Student s t-test). h Mean of Gaussian u03C3 is not sensitive to the presented probability distribution (planned contrasts, *Pu2009<u20090.05, **Pu2009<u20090.01, two-tailed Students t-test). Error bars denote standard deviation. For ANOVAs, see Supplementary Tables_u2013.\" - \"We next investigated these distributional differences. Participants RTs exhibited two characteristic patterns, one modality-specific, the other modality-independent. In the modality-specific pattern, median RT was faster in the auditory condition than in both visual (u221242.4u2009xB1u200926.6u2009ms, meanu2009xB1u2009 standard deviation, SD, Pu2009=u20098.2u2009xD7u200910; Tukeys honestly significant difference test) and somatosensory conditions (u221254.8u2009xB1 u200933.7u2009ms, meanu2009xB1u2009SD, Pu2009=u20091.1u2009xD7u2009 10; Tukey) (Fig._). In RT variance, a different modality-specific pattern emerged. Interquartile range (IQRRT) was smaller in vision (u221221.3u2009xB1u2009 26.4u2009ms, meanu2009xB1u2009SD, Pu2009=u20090.00026; Tukey) and audition (u221214.6u2009xB1u200926.2u2009ms, meanu2009xB1u2009SD, Pu2009=u2009 0.0198; Tukey) compared to somatosensation (Fig._). In the modality-independent pattern, the change in u2018go time PDF had no effect on median RT (Fig._ ). However, IQRRT was significantly larger in the PDFlip condition as compared to PDExp (Fig._). The magnitude of the difference in IQRRT between probabilistic conditions did not differ between sensory modalities (F(2,22)u2009=u20090.04, Pu2009=u20090.96, one-way ANOVA (for analysis of variance and planned contrasts on median RT and IQRRT see Tables_u2013S4).\" - \"In Fig._ differences between auditory RT, on one side, and visual and somatosensory on the other, are evident at shorter u2018go times. In the visual and somatosensory cases there are characteristic upward bends that are not seen in the auditory modality. To investigate whether these differences in RT curves between modalities drive the above results on median RT and IQRRT, the respective analyses were performed again on RT from a subset of u2018go times (0.5667u2009 s to 1.4u2009s), thus eliminating the potentially confounding differences in RT curves. The analyses yielded highly similar results to the ones above (Supplementary Fig._).\" - \"To identify how the systematic differences in RT distribution (Fig._) relate to the observed modality-specific and modality-independent patterns, we modeled the RT distributions with an exponential-Gaussian PDF. This two-process model is the convolution of an exponential PDF (parameter u03C4) and a Gaussian PDF (parameters u03BC and u03C3) (Fig._). It states that the generation of RT depends on a sum of peripheral Gaussian processes and a central decision process that is hypothesized to be exponential. The model provides an excellent fit to the data in all conditions, both at the group level (Fig._) and at the single-subject level (Supplementary Fig._). Both Gaussian parameters were sensitive to changes in sensory modality but were insensitive to changes in the u2018go time PDF (Fig._), in line with the models implicit claim of peripheral Gaussian processes. In contrast, the exponential parameter u03C4 was sensitive to the u2018go time PDF (Fig._), having larger values in the PDflip condition as compared to PDExp. This pattern of u03C4 closely mirrors the behavior of IQRRT (Fig._) (See Supplementary Tables_u2013 for analysis of variance and planned contrasts on u03BC, u03C3, and u03C4). We found that Gaussian u03BC captured the behavior of median RT in all three modalities (Supplementary Fig._ ). Likewise, and also independent of modality, u03C4 captured IQRRT (Supplementary Fig._). Taken together, these findings suggest that only the process(es) reflected in the exponential parameter u03C4 use information on the u2018go time PDF. Taken together, our findings on the influence of the u2018go time PDF on the RT distribution support (i) the broader hypothesis of peripheral and central processes involved in event anticipation as well as (ii) the specific claim of an exponential process shared by all three modalities.\"- - \"We investigated how the brain infers probability as a function of time based on sensory input, by analyzing RT to temporally distributed events. In audition, vision, and somatosensation, subjects evidently extracted and used the probabilistic information encoded in the trial structure to predict event onsets, as shown by their distinct patterns of RT modulation. Previous work has proposed the hazard rate as a model of probability over time used by the brain to anticipate events and plan responses. We demonstrate that it is not the HR but the PDFu2014 a simpler and more stable variableu2014which better captures the data.\" - We found that RT was sensitive to changes in event probability density. Notably, the observed RT modulation in the PDExp condition indicates that subjects inferred temporal-probabilistic information from exponentially distributed events. This is remarkable because it is commonly assumed that an exponential distribution renders temporal prediction impossible, arguably due to its flat HR in the case of no catch trials. - \"We first modeled the RTs using various explanatory variables based on linear and non-linear transformations of the HR. None of these variables adequately captured RT patterns across the different u2018go time distributions. Therefore, the HR is not a likely parameter the brain uses to relate an event PDF to its corresponding RT. Instead, we demonstrate that a non-linear transformation of the PDF, the reciprocal, better captures the RT behavior across different PDFs in three sensory modalities.\" - \"The models based on the reciprocal PDF clearly provided the better fits to RT in all conditions as compared to the reciprocal HR. Interestingly, the reciprocal HR is just the reciprocal PDF multiplied by the survival function, as ( frac{1}{{{mathrm{HR}}}} = left( {1 - {mathrm{CDF}}} right) times frac{1}{{{mathrm{PDF}}}}). In this equation the survival function appears as a time-varying scaling factor of the reciprocal PDF. Following this revealing result that the PDF, and not the HR, is the most likely parameter modeled by the brain, and given that ({mathrm{HR}} = frac{1}{{{mathrm{survival}} ;{mathrm{function}}}} times {mathrm{PDF}}), it follows that the brain did not employ the scaling factor (1/survival function) in order to scale the PDF. Instead the scaling appeared to be better approximated by a fixed value, uniform for the entire range of u2018go times. Nonetheless, it seems possible that in contexts other than simple event anticipation, the brain might use the survival function as a scaling factor for event probability density in which case the HR may be an appropriate parameter of probability in time.\" - \"We note that in a reward-based context, the HR has been shown to adequately describe temporal expectation in a 100% rewarded condition, but not when uncertainty of reward is introduced. In the experiments we performed here, no reward was delivered. Therefore, the fact that the PDF, but not the HR, provided the best model of RT might be related to the absence of reward. Although it is clearly beyond the scope of our study to identify how reward modulates anticipation (which includes estimation of event probability and of elapsed time, see below), a simple, intuitive hypothesis can be formulated. If the brain employed a scaling factor for the PDF, which in our case is uniform across time, then under reward conditions this scaling factor could approximate (frac{1}{{{mathrm{survival}} ;{mathrm{function}}}}), and the resulting parameter encoded would be the HR, as ({mathrm{HR}} = frac{1}{{{mathrm{survival}};{mathrm{function}}}} times {mathrm{PDF}}). The survival function is defined as 1u2014CDF, where the CDF captures the cumulative probability that an event should have happened up to and including the current time instance. The computation of this accumulated probability is cognitively demanding and would likely be facilitated by motivating effects of expected reward. This simple hypothesis could describe a basic mechanism by which the brain incorporates reward into event anticipation (Supplementary Fig._). It suggests a possibility how the PDF-based model we describe here could link to the HR-based models in the reward literature.\" - \"Commonly, HR-based models of RT incorporate the concept of temporal blurring which assumes that the uncertainty in elapsed time estimation increases linearly with time: longer intervals carry higher uncertainty in their estimation than shorter intervals. This implies that the brains capacity to react fast and accurately to longer timespans is limited compared to shorter timespans, irrespective of the accuracy of the brains estimate of event probability in time. In other words, the error in time estimation ultimately constrains the brains benefit from temporal-probabilistic inference. By modeling RT, we found that probabilistically-blurred models fitted the data better than the temporally-blurred ones. In particular, the RTs at the most probable, longer u2018go times in the PDFlip condition were much better captured by the probabilistically-blurred model compared to the temporally-blurred one (Fig._ ). This finding challenges the common assumption that the brain models elapsed time with uncertainty that increases with time per se. Instead it seems that uncertainty in time estimation also scales with probability. We suggest that by modeling its environments temporal-probabilistic structure, the brain can overcome what has been considered a built-in limitation: the uncertainty in time estimation.\" - \"In addition to event probability in time and uncertainty in time estimation, a third source of uncertainty can be quantified in event anticipation: the uncertainty of event occurrence. Suppose a u2018go cue will certainly have occurred by the end of a trial. In this case there is no uncertainty of event occurrence towards the right extremum of the u2018go timespan. In contrast, u2018 go cue occurrence may remain uncertain when in a percentage of trials no event occurs (catch trials). The HR has been proposed in both contexts, without catch trials and with catch trials, as an important model of probability in time. Notably, in a setting with catch trials, certainty of event occurrence is reflected by the CDF asymptotically approaching 1. This leads to a steep increase of the HRs slope towards the end of the u2018go time period, irrespective of probability density (Supplementary Fig._)u2014the exponential PDF being a rare exception to this. Commonly, this CDF-based up-weighting of u2018go time probability is argued to reflect anticipation, which conforms with intuition as it maximizes event probability towards the right extremum of a timespan when the event will inevitably occur. However, as a result of the CDF approaching 1, the HRs slope becomes very steep and its values approach infinity. Since there is a lower bound to reaction time, this behavior towards the end of a timespan challenges the concept of the HR as a model of RT. Although the commonly employed temporal blurring remedies this problem somewhat by reducing HR values as time increases, the conceptual issue of an ever-increasing variable remains.\" - \"Another general problem of the HR concerns its instability in calculation in both contexts with and without catch trials. The HR is difficult to estimate from empirical data because technically, it requires several steps: computation of PDF, integration of PDF to arrive at CDF, transformation of CDF to arrive at the survival function, division of PDF by survival function. Even small errors in the representation of PDF or its CDF will lead to large, unpredictable errors in HR which could have considerable consequences for an organism relying on the HR as its model of temporal probability. The term 1/PDF, on the other hand, can be interpreted as u201Cone-in-manyu201Du2014a simpler and more stable computation. For example, if the probability of an event is 0.02, the brain might interpret it as u201C1/0.02u201D, i.e. u201Cone-in-fiftyu201D. This interpretation links the 1/PDF model closely to surprisal as defined in Shannon s information theory, ({mathrm{log}}_2left( {frac{1}{p}} right)) bits, where p is the probability of an event. This suggests that the brain might be computing a simplified version of surprisal. It should also be noted that in some of the previous work on the topic, the observed relation between RT and HR might also have been well captured by the PDF, as both HR and PDF can be monotonically in- or decreasing.\" - \"In addition to the influence of event probability, basic timing mechanisms may be central to the anticipatory processes investigated here. The estimation of elapsed time is highly relevant in common interval timing tasks which involve time estimation, production and reproduction and also in the simple RT task presented here. This invites brief discussion of the potential underlying mechanisms. A popular hypothesis proposes two dissociable circuits underlying interval timing, a more u201Cautomaticu201D system that involves the cerebellum and may be used more in the short sub-second range, and another, cognitively controlled system, incorporating the basal ganglia, and related cortical structures that may be involved in processing timespans in the seconds range. Our behavioral design does not permit differentiation between neural mechanisms. Still, the differences observed between sensory modalities at shorter u2018go times point towards a related differentiation between components of timing processes: some that are of a more peripheralu2014arguably u201Cautomaticu201Du2014 nature, and others that are of a more centralu2014eventually more cognitively controlledu2014nature.\" - \"It has been suggested that temporal discrimination is more precise in audition than in vision and somatosensation. Therefore, we used audition as a reference condition, and modeled the across-modality u2206RT curves with an exponential function of u2018go time. These exponential models of u2206RT were added as a component to the PDF-based model which drastically improved the fit, accounting for the modality-specificities. This modeling procedure relies on the use of audition as a reference modality, which naturally raises further questions: is audition itself free of modality-specific effects? Which components of timing are indeed modality-specific and which components are modality-general? These important aspects cannot conclusively be addressed by this experiment. Nonetheless, although we cannot further specify the processes underlying the exponential model component, we hypothesize that the exponential function of u2018go time reflects differences between modalities in fundamental timing processes in the sub-second range. A substantial body of previous work proposed that in interval discrimination for timespans above 0.2u2009s (up to 1u2009s) and in a variety of timing tasks between 0.1 and 1.5u2009s, the Weber fraction for time estimation is close to constant. Such evidence from interval discrimination suggests that the modality-specific RT modulation at short u2018go times we observed may be unrelated to elapsed time estimation itself. Instead, the impact of event probability on time estimation appears to be a likely source of modality-specific RT modulation. Given that a large literature on the processing of probability in time investigates the relationship between sensory input and behavioral output, we emphasize the importance of the contingencies of sensory input modalities for any potential inferences made.\" - To further investigate the influence of sensory input modality on processes involved in temporal-probabilistic inference, we analyzed the RT distributions. We observed modality-specific differences in average RT that are in agreement with existing findings covering a wide range of simple RT and go/no-go tasks. Here they are seen in the context of event anticipation. - \"The RT distributions could be decomposed into the sum of a modality-independent PDF which was exponential and a second, modality-dependent PDF which was Gaussian. The Gaussian parameter u03BC corresponded to the modality-specific offsets in average RT. The Gaussian parameter u03C3 displayed a different modality-specific pattern. Neither Gaussian parameters u03BC nor u03C3 were affected by changes in the event PDF of the input. In contrast, the exponential parameter u03C4 was sensitive to the event PDF. u03C4 also captured the difference in RT variance between the two probability distributions in all three sensory modalities. In the long tradition of the ex-Gaussian model of RT the exponential part has been interpreted as a central and the Gaussian part as a peripheral process but also the opposite assignment has been made. Our purely behavioral study cannot conclusively answer whether the exponential process reflects central, modality-independent computations or whether it reflects more peripheral computations in the neural substrate of each sensory modality. Nonetheless, our behavioral findings suggest that all three sensory modalities share a similar exponential processu2014 which in turn makes specific predictions for what neural activity patterns should be sought on recordings from the relevant sensory and supra-sensory regions.\" - In sum, we expect that our findings will aid efforts in the understanding of the neural mechanisms involved in predictive processes. The results demonstrate that irrespective of sensory input modality, the brain models its environment\\'s temporal-probabilistic structure using a non-linear transformation of the PDF, but not the hazard rate.- - Ethical approval - The experiments were approved by the Ethics Council of the Max-Planck Society. Written informed consent was given by all participants before the experiment. - Subjects - \"In all, 24 human subjects (13 female), aged 19u201333, participated in the auditory, visual, and somatosensory experiments. Of these, 18 subjects (13 female), aged 19u201333 participated in the Gaussian control experiment (Supplementary Fig._ ) and 12 other subjects (9 female), aged 20u201333 participated in the experiment without catch trials (Supplementary Fig._). All were right-handed and had normal or corrected-to-normal vision and reported no hearing impairment and no history of neurological disorder. Participants were naive to the purpose of the experiment. They received u20AC10 per hour for participating.\" - Task and procedure - \"In auditory, visual, and somatosensory conditions, subjects performed a simple u2018set - u2018go task in which a u2018set cue was followed by a u2018go cue. The timespan between the onset of both cues, termed the u2018go time, was a random variable that was drawn from a specific probability distribution. Subjects were asked to foveate a central black fixation dot and respond as fast as possible to the onset of the u2018go cue with a button press on a response device using the right index finger. After a button press, a small black circle appeared for 0.2u2009s around the central fixation dot indicating the end of the trial. In some trials, no u2018go cue appeared (u2018catch trials), in which case participants were instructed to not press the button. In these catch trials, a small black circle appeared 1.9u2009 s after u2018set cue onset, indicating again the end of the trial. The experiment consisted of two separate sessions taking place at the same time of the day on two consecutive days. A single session consisted of four blocks per sensory modality (vision, hearing, touch) and lasted ~2.5u2009h. Per block, 165 trials were presented (including 15 u2018catch trials), resulting in 1980 trials per session for each subject (3980 trials for two sessions), and a total of 95,040 trials for all subjects. A short training block was run before the first block of each sensory modality on both days to familiarize subjects with the task. During all experimental blocks, subjects wore headphones and positioned their heads on a forehead-and-chin rest (Head Support Tower, SR Research Ltd.) at a fixed distance of 60u2009cm relative to the computer monitor. Each subjects dominant eye, as determined by Miles test was tracked at a sampling frequency of 1,000u2009Hz (Eyelink DM-890, SR Research Ltd.). Subjects were asked to restrict eye blinking to the timespan after a button press, i.e. during the ITI. Trials in which visual fixation was not maintained within a radius of 2.5xB0 visual angle around the central fixation point for more than 300u2009ms during the u2018go time were automatically discarded for data analysis. All stimuli were generated using MatLab (The MathWorks, Natick MA, USA) and the Psychophysics Toolbox on a Fujitsu Celsius M730 computer running Windows 7 (64u2009bit). The experiment took place in a dimly lit, soundproof booth.\" - Auditory stimuli - \"Two white noise bursts (50u2009ms duration, 8u2009ms cosine ramp, onset and offset) served as u2018set and u2018go cues. They were presented at 60u2009dB SPL above hearing threshold at 1u2009kHz, as determined by pure tone audiometry according to ISO 8253. All auditory stimuli were output by a high-quality interface (RME Fireface UCX) and delivered diotically using electrodynamic headphones (Beyerdynamic DT 770 PRO) driven by a headphone amp (Lake People GT-109). The sound pressure level was calibrated to 75u2009dB(A) individually for each transducer while using a temporal weighting suited for impulsive stimuli (/u03C4u2009=u200935u2009ms). To this end we used an IEC 603184 artificial ear simulator (model G.R.A.S. 43AG) with according pinnae and a IEC 60942 class 1 sound level calibrator (Larson Davis_CAL200) and an IEC 942 class 1 pistonphone with barometric correction as calibration source (G.R.A.S. Type 42AA).\" - Visual stimuli - \"The visual u2018set cue (duration 50u2009ms) consisted of two checkerboard patterns which were presented simultaneously. One was positioned in an upper quadrant, the other in the lower quadrant on the opposite side. The u2018go cue consisted of two checkerboard patterns the same location but the with black-white pattern reversed. Each checkerboard subtended 6.6u2009xD7u20096.6 degrees of visual angle and consisted of 7u2009xD7u20097 black and white squares, each subtending 0.9u2009xD7u20090.9 degrees of visual angle. The center of each checkerboard was positioned at a horizontal distance of 12.6 degrees of visual angle and at a vertical distance of 7.1xB0 from the center of a central, black fixation dot. The site of presentation alternated between the left and right side on a trial-by-trial basis. Visual stimuli were presented on a BenQ XL2420-B monitor (resolution 1920u2009xD7u20091080, refresh rate 60u2009 Hz) which was set to a gray background.\" - Somatosensory stimuli - \"Two short electric pulses (duration 200u2009xB5s) were presented as u2018 set and u2018go cues using a constant current stimulator (Digitimer DS7A). Each subjects perceptual threshold was determined by increasing stimulus intensity (mean current) until the subject first reported a sensation and then decreasing it until it was no longer perceived. The perceptual threshold was recorded as the lowest ascending stimulus intensity at which the subject reported sensation. For the experimental task, the electric current was set to a higher intensity (mean currentu2009=u20097.9u2009xB1u20093.9u2009 mA) that the subject judged comfortable, yet easily perceptible.\" - Temporal probabilities - \"The u2018go time was a random variable drawn from one of two probability distributions (Fig._) that was fixed throughout two consecutive blocks of trials. The distributions were chosen by parametrically searching the family of Weibull distributions for cases that would fulfill the two following criteria:\" - 1.The one distribution should be the left-right flipped version of the other so that this symmetry in PDFs would help to identify the effect of elapsed time itself on the modulation of RTs by the probability distributions.2.The two distributions should have HRs with opposite slopes, in order to investigate the modulation of RTs by HR. - \"A parametric search identified a Weibull distribution with parameters ku2009 =u20091 and lu2009=u20090.33:\" - \"$$fleft( t right) = frac{k}{l}left( {frac{t}{l}} right)^{k - 1}e^{ - left( {frac{t}{l}} right)^k}$$ (1) \" - \"The identified shape value ku2009=u20091 reduces the Weibull to an exponential distribution:\" - \"$$fleft( t right) = frac{1}{l}e^{ - left( {frac{t}{l}} right)}$$ (2) \" - \"The x-axis of both distributions was discretized with the step size of the monitor refresh rate. This is based on the assumption that this level of discretization is not perceivable in the context of the u2018set - u2018go design, rendering the distributions continuous to the brain. The y-axis was also discretized as the PDF described number of trials at each discrete u2018go time point. Both distributions were delayed by 0.4u2009s giving a range of u2018go times from 0.4 to 1.4u2009s. To minimize sequential effects, the u2018go times were randomized with the constraint that no more than two consecutive trials had the same u2018go time. The intertrial interval (ITI, range 1.4 to 2.4u2009s) was randomly drawn from a uniform distribution. During each session in auditory, visual, and somatosensory conditions half of the blocks followed the exponential distribution (PDExp), the other half followed its flipped counterpart (PDFlip). The probabilistic structure changed after two blocks without notification. To control for order effects, the conditions (sensory modalities and probability distributions) were organized in a Latin square design, based on which modality and distribution were shuffled across subjects and days.\" - Exponential-Gaussian model - To quantitatively investigate the distributional properties of RT between conditions, we used the exponential-Gaussian distribution as a well-established parametric two-process model of RT. - \"$$fleft( {t;mu ,sigma ,tau } right) = frac{1}{{2tau }}e^{frac{1}{{2 tau }}left( {2mu + frac{{sigma ^2}}{tau } - 2t} right)}erfcleft( {frac{{2mu + frac{{sigma ^2}}{tau } - t}}{{sqrt 2 sigma }}} right)$$ (3) \" - where erfc is the complementary error function defined as - \"$${erfc}left( x right) = frac{2}{{sqrt pi }}mathop {smallint } nolimits_x^infty e^{ - t^2}{mathrm{d}}t$$ (4) \" - \"The parameters u03BC and u03C3 are the mean and standard deviation respectively of the Gaussian constituent of the ex-Gaussian distribution. Parameter u03C4 is the exponent of the exponential constituent distribution.\" - \"The best-fitting Ex-Gaussian parameters were obtained for each subject in each condition using a least-squares fitting algorithm. We observed no systematic difference in adj. R between conditions (Supplementary Fig._). Therefore the ex-Gaussian proved to be a well-fitting model of RT in all conditions.\" - \"Temporally blurred (u2018subjective) PDFs\" - \"The PDFs were blurred by a temporal uncertainty kernel that scales with elapsed time from a reference time point. More intuitively, the longer the elapsed interval to be estimated, the bigger the uncertainty about its length. In recent work it has been hypothesized that this uncertainty kernel has a Gaussian shape and its standard deviation increases linearly with time as (sigma = varphi cdot t), where t is the elapsed time and u03C6 is the scale factor by which the standard deviation u03C3 of the Gaussian uncertainty kernel increases. This intuitively implies that the ratio of the standard deviation of temporal uncertainty to the elapsed time is constant and for this reason the variable u03C6 has been termed a Weber fraction of the estimation of elapsed time.\" - \\'Each of the employed distributions is characterized by the following three functions:\\' - $$p( {t_{{mathrm{go}}}} ):{mathrm{Probability}};{mathrm{Density}};{mathrm{Function}};{mathrm{of}};{mathrm{Reaction}};{mathrm{Time}};{mathrm{as}};{mathrm{a}};{mathrm{function}};{mathrm{of}};^{prime} {mathrm{go}}^{prime} ;{mathrm{time}};t_{{mathrm{go}}}$$ - \"$$c( {t_{{mathrm{go}}}} ):{mathrm{Cumulative}};{mathrm{Density}};{ mathrm{Function}};{mathrm{of}};{mathrm{Reaction}};{mathrm{Time}} ;{mathrm{as}};{mathrm{a}};{mathrm{function}};{mathrm{of}};^{ prime} {mathrm{go}}^{prime}; {mathrm{time}},t_{{mathrm{go}}} , ,,,,,,,,,,,,,,,,= int_{0}^{t_{go}} pleft( u right)du$$ (5) \" - \"$$h( {t_{{mathrm{go}}}} ):{mathrm{Hazard}};{mathrm{Rate}};{mathrm{of}} ;{mathrm{Reaction}};{mathrm{Times}};{mathrm{as}};{mathrm{a}};{ mathrm{function}};{mathrm{of}};^{prime} {mathrm{go}}^{prime}; { mathrm{time}};t_{{mathrm{go}}} = frac{{p(t_{{mathrm{go}}})}}{{1 - c(t_{{ mathrm{go}}})}}$$ (6) \" - \\'Each of the PDFs were blurred with a Gaussian kernel with variance increasing with time. The equations for the corresponding subjective functions are:\\' - \"$$p_S( {t_{{mathrm{go}}}} ) = frac{1}{{varphi t_{{mathrm{go}}}sqrt {2pi } }}mathop {smallint }nolimits_{ - infty }^infty pleft( tau right) cdot e^{ - left( {tau - t_{{mathrm{go}}}} right)^2/left( {2varphi ^2t_{{mathrm{go}}}^2} right)}dtau$$ (7) \" - \"$$c_S( {t_{{mathrm{go}}}} ) = int_{0}^{t_{mathrm{go}}} p_Sleft( u right)du$$ (8) \" - \"$$h_S( {t_{{mathrm{go}}}} ) = frac{{p_S(t_{{mathrm{go}}})}}{{1 - c_S(t_{{ mathrm{go}}})}}$$ (9) \" - \"From Eq._() it is evident that for a given u2018go time tgo the PDF is convolved with a Gaussian kernel centered at tgo.\" - \"The distributions used in this work were not continuous but discrete. They were only represented at the possible time points of stimulus presentation in either auditory, visual or somatosensory conditions. The stimuli presentation instances for PDExp and PDFlip ranged between [0.4 1.4] sec at steps of 2/60u2009s (every 2 frames with frame rate 60 frames per second). The computation of the CDF from the discrete PDF was performed using trapezoidal integration. The hazard rate was computed from these discrete versions of a PDF and a CDF. The relatively small sampling interval resulted in discrete CDFs and hazard rates closely approximating the expected continuous versions of these functions from the analytical solutions (Supplementary Fig._).\" - \"The shortest u2018go time is 0.4u2009s after u2018set cue onset. At this time point the uncertainty kernel has standard deviation (varphi cdot 0.4). Similarly at the longest u2018go time of 1.4u2009s this uncertainty has standard deviation (varphi cdot 1.4). In order to implement Eq._ for the computation of the subjective PDF, the definition of the PDF was extended to the left and right of the actual stimulus presentation interval as:\" - \"$$p_{{mathrm{padded}}}( {t_{{mathrm{go}}}} ) = left{ {begin{array}{*{20}{c}} {0,(0.4 - 3 cdot varphi cdot 0.4) le t, <, 0.4} {p( {t_{{ mathrm{go}}}} ),0.4 le t le 1.4} {0,1.4, > , t ge (1.4 + 3 cdot varphi cdot 1.4)} end{array}} right.$$ (10) \" - \"The extensions were equal to 3 standard deviations of the Gaussian uncertainty function at the shortest and longest u2018go times. This 3-standard-deviation extension was selected as it encapsulates the 99.7% of the Gaussian uncertainty. Then the integral in Eq._() was computed between these new extrema [(left( {0.4 - 3 cdot varphi cdot 0.4} right)), ((1.4 + 3 cdot varphi cdot 1.4))] instead of the impractical interval of minus to plus infinity. The selection of the value of u03C6 was based on previous research. With this value of u03C6u2009=u20090.21 the temporal range of the extended PDF as defined in Eq._() becomes [0.148 2.28] s which is also the range of integration in the computation of the subjective PDF in Eq._().\" - \"The PDF of each distribution, p(tgo) was normalized so that its integral does not amount to one but to 0.9091, which is the total probability that an event will occur over the timespan covered by the distribution. The remaining 9.09% of trials (30 out of 330) are catch trials in which no u2018go cue was presented. Consequently the maximum value of the resulting CDF c(tgo) is 0.9091. The catch trials introduced some uncertainty about whether the event will occur at all. The function of the catch trials in the context of modeling temporal-probabilistic structures is to prevent the brains estimation of a PDF from being confounded by the expectation of a conditional event, i.e. the mandatory occurrence of a u2018go cue at the end of a given u2018go time range. It is easy to see from Eq._() that the introduction of catch trials and the corresponding reduction of maximum CDF to a value significantly smaller than one stabilizes the computation of the hazard rate at the right extremum of the PDF, i.e. the part where the CDF would approach one in the absence of catch trials. Here it has to be mentioned that although the maximum CDF value is 0.9091, the denominator in the computation of the HR remain 1-c(tgo) and not 0.9091u2013c(tgo). This is because this denominator represents the probability that nothing has happened up to time point tgo. This term includes also the probability that nothing has happened up to tgo because the current trial is a catch trial. So this denominator could be alternatively defined as ccatchu2009+u2009cmaxu2212c(tgo), where cmax is the maximum CDF value of the stimulus PDF, equal to 0.9091, and ccatch is the probability that no u2018go cue appears at all, which is equal to 0.0909. As these two terms sum to one, the denominator in the computation of the HR is correctly stated as in Eq._().\" - Probabilistically blurred PDFs - \"In the definition of the subjective function in Eq._(), the PDF was convolved with a Gaussian function, which represented the uncertainty in elapsed time estimation at a given time point. This uncertainty has been hypothesized to increase linearly with time, irrespective of the probability density function of event occurrence. In addition to this, here an alternative hypothesis was investigated in which the uncertainty in elapsed time estimation depends on the probability density function of event occurrence. The hypothesis states that u2018go times with high probability of event occurrence are associated with low uncertainty in time estimation based on the rationale that the brain predicts the onset of upcoming events of high probability more accurately. In contrast, u2018go times with low probability carry higher uncertainty, even if the time they span is short.\" - \"This probabilistic blurring of elapsed time estimation was implemented in a similar fashion to the temporal blurring described in Eq._(). However the standard deviation of the Gaussian kernel does not scale linearly with time as in the temporal blurring case ((sigma = varphi cdot t)), instead it scales according to the PDF of event occurrence. In order to use realistic variance values in the blurring Gaussian kernel the minimum and maximum values of the standard deviation were set accordingly to the temporal blurring case as\" - \"$$sigma _{{mathrm{min}}} = varphi cdot t_{{mathrm{min}}} = varphi cdot 0.4;{mathrm{and}};sigma _{{mathrm{max}}} = varphi cdot t_{{mathrm{max}}} = varphi cdot 1.4$$ (11) \" - \"The PDF under investigation was then scaled so that its minimum value is u03C3 min and its maximum value u03C3max.\" - \\'If pmin and pmax are the minimum and maximum values respectively of the PDF under investigation then the function used for computing the standard deviation of the Gaussian kernel based on the PDF p(t)was defined as:\\' - \"$$sleft( t right) = left[ {1 - frac{{left( {pleft( t right) - p_{{ mathrm{min}}}} right)}}{{(p_{{mathrm{max}}} - p_{{mathrm{min}}})}}} right] cdot left( {sigma _{{mathrm{max}}} - sigma _{{mathrm{min}}}} right) + sigma _{{mathrm{min}}}$$ (12) \" - \"From the first term inside the brackets it is obvious that when the probability p(t) is low the standard deviation of the Gaussian kernel approaches u03C3 max while when the probability becomes big, s(t) approaches u03C3min.\" - \\'Based on this function for determining the standard deviation of the blurring Gaussian kernel the probabilistically blurred PDF pp(t)was computed as:\\' - \"$$p_pleft( {t_{{mathrm{go}}}} right) = frac{1}{{s(t)sqrt {2pi } }} mathop {smallint }nolimits_{ - infty }^infty pleft( tau right) cdot e^{ - left( {tau - t_{{mathrm{go}}}} right)^2/left( {2s(t)^2} right)}{mathrm{d}}tau$$ (13) \" - \"This equation describes that at a given time instance t, according to the PDF of event occurrence, the Gaussian uncertainty on the estimation of elapsed time has standard deviation s(t). The minimum and maximum values of standard deviation u03C3min and u03C3max are set to (varphi cdot 0.4) and (varphi cdot 1.4), as already described earlier. In order to compare this probabilistic blurring hypothesis directly to the initial temporal blurring hypothesis, the value of u03C6 was likewise set to 0.21.\" - \"Finally, in order to implement the Gaussian blurring of Eq._() at the extrema of u2018go-times, the definition of the PDF was extended to the left and right of the actual stimulus presentation interval by three standard deviations of the corresponding smoothing Gaussian kernels, similarly to the temporally blurred case described in Eq._(), as:\" - \"$$p_{{mathrm{padded}}}( {t_{{mathrm{go}}}} ) = left{ {begin{array}{*{20}{c}} {0,(0.4 - 3 cdot s(0.4)) le t, <, 0.4} {p( {t_{{mathrm{go}}}} ),0.4 le t le 1.4} {0,1.4, > , t ge (1.4 + 3 cdot s(1.4))} end{array}} right.$$ (14) \" - Notice that the extensions depend on the standard deviation function s(t), which depends on the probability density function. - Selection of explanatory variables for modeling RT - \"The selection of explanatory variables for modeling RT was driven by an expected inverse relation of RT with the PDF and HR of the u2018go times. This is based on the rationale that for high relative values of PDF or HR, RT is expected to be small and vice versa. Previous work demonstrated a negative correlation between the u2018subjective HR and RT in the order of u22120.3. As mentioned earlier, computation of the HR in the brain would require three separate calculations: computation of PDF, its integration for deriving the CDF, and their division for computing HR. The computation of the PDF is the most basic and necessary step in the sequence outlined above and for this reason it was considered here as an alternative factor that can directly affect RT, even before the CDF and HR are computed.\" - \"Here two different functions with an inverse character were selected, a u2018 mirror and a u2018reciprocal function. The u2018mirror function just reflects a function mirrored around its mean. Here it is used to capture linear anti-correlations between RT and the explanatory variables PDF and HR.\" - \"$$x_{{mathrm{mp}}}( {t_{{mathrm{go}}}} ) = - ( {p( {t_{{mathrm{go}}}} ) - bar p} ) + bar p = - p( {t_{{mathrm{go}}}} ) + 2 cdot bar p$$ (15) \" - \"$$x_{{mathrm{mh}}}( {t_{{mathrm{go}}}} ) = - ( {h( {t_{{mathrm{go}}}} ) - bar h} ) + bar h = - h( {t_{{mathrm{go}}}} ) + 2 cdot bar h = - frac{{p( {t_{{mathrm{go}}}} )}}{{1 - c( {t_{{mathrm{go}}}} )}} + 2 cdot bar h$$ (16) \" - where - \"xmp: u201Cmirroru201Dof the PDF\" - \"xmh: u201Cmirroru201Dof the hazard rate of the PDF\" - \\'(bar p): Mean PDF, (bar h) : mean HR\\' - \"$$x_{{mathrm{op}}}( {t_{{mathrm{go}}}} ) = frac{1}{{p( {t_{{mathrm{go}}}} )}}$$ (17) \" - \"$$x_{{mathrm{oh}}}( {t_{{mathrm{go}}}} ) = frac{1}{{h( {t_{{mathrm{go}}}} )}} = frac{{1 - c( {t_{{mathrm{go}}}} )}}{{p( {t_{{mathrm{go}}}} )}} = ( {1 - c( {t_{{mathrm{go}}}} )} ) cdot x_{{mathrm{op}}}( {t_{{mathrm{go}}}} )$$ (18) \" - where - \"xop: u2018reciprocal PDF\" - \"xoh: u2018reciprocal hazard rate of the PDF.\" - \"The u2018reciprocal function simply takes the reciprocal of a function, e.g. 1/PDF. This is used to capture a non-linear anti-correlation.Similar variables are defined for the temporally and probabilistically blurred PDF cases, namely xopt, xoht, xmpt, xmht, for temporal blurring and xopp, xohp, xmpp, xmhp, where for probabilistic blurring where the third letter in the subscript indicates the type of blurring. As it can be seen from Eq._() for any given u2018go time tgo, the variable xoh(tgo) is a scaled version of the variable xop(tgo). This scaling depends on the CDF and therefore it is non-linear across u2018 go times. The same holds for variablesc xmp(tgo) and xmh(tgo), as can be seen in Eqs._() and (). As the scaling between these pairs of variables is non-linear (dependent on (( {1 - c( {t_{{mathrm{go}}}} )} ){mathrm{and}} ;frac{1}{{1 - c(t_{{mathrm{go}}})}}) respectively) it is also expected that their relation to RT will not be linearly identical. This also justified their treatment as different variables that are not independent but non-linearly related.\" - \"Modeling RT by u2018mirror and u2018reciprocal functions with a linear model\" - \"The eight u2018blurredexplanatory variables xopt, xoht, xmpt, xmht, for temporal blurring and xopp, xohp, xmpp, xmhp for probabilistic blurring, were constructed to investigate their linear relation to RT. These variables were derived with (varphi = 0.21) based on previous research.\" - \"Additionally, the four variables xop, xoh, xmp, and xmh derived directly from the original PDF, CDF and HR, without any Gaussian blurring, were also fit to RT for comparison. A linear model was built for each of these eight explanatory variables. An Ordinary Least Squares (OLS) regression was employed for the computation of the regression coefficients using the MatLab (The MathWorks, Natick MA, USA) fit function. Any assumption about the distribution of the residuals of the models was omitted, as we had no evidence that they should follow a Gaussian distribution. Adjusted R was used as a measure of goodness-of-fit for comparing the different models relation to RT.\" - Comparing temporal and probabilistic blurring - \"One of the expected differences between the two blurring methodologies was located at the early and late extrema of the u2018go time interval. This is because in the case of temporal blurring the smoothing kernel has always much higher variance at the late extremum, as compared to the early, independent of the PDF used and this should be expected to result in greater smearing of RT curves towards the late extremum due to the always greater uncertainty. This should not be the case in probabilistic blurring, where the standard deviation of uncertainty in interval estimation depends on the PDF used. So for example the uncertainty in the early extremum of the exponential PDF should be very similar to that at the late extremum of the flipped-exponential due to the PDF symmetry. This should also result in identical smearing of the RT curves at these two different extrema for these two different PDFs.\" - \"In order to investigate if the explanatory variables based on temporally or probabilistically blurred PDFs capture better the behavior of RT curves at different parts of the u2018go time range, the u2018go time range was divided in 4 equally spaced bins and the goodness-of-fit of all models for the different explanatory variables was computed in each bin. The metric employed for comparing the goodness-of-fit of these models was the sum of squared residuals in each bin.\" - For each distribution (PDExp and PDFlip), each modality (visual, auditory, somatosensory), each of the 4 bins and each of the explanatory variables that was derived based on the blurred PDF, the quotient of the RT model residuals of the temporally blurred case over that of the probabilistically blurred case was computed. This procedure indicated which blurring method provided the model that fits the RT of a specific bin better. - Modeling of RT relative to auditory condition - \"The auditory condition was used as a reference condition and auditory RTs were subtracted from visual RTs and from somatosensory RTs in both PDExp and PDFlip conditions. The resulting across-modality u2206RT curves were fitted with an exponential function of u2018go time:\" - \"$$f(t_{{mathrm{go}}}) = a;e^{b;t_{{mathrm{go}}}} + c$$ (19) \" - \"The condition-specific probability per u2018go time (PDExp or PDFlip) was used as weights in the fitting algorithm (ordinary least-squares in MATLAB s fit function, Fig._). Finally, a combined, additive model of RT was constructed based on the reciprocal, probabilistically-blurred PDF fitted to the auditory RT and the exponential model fitted to the across-modality u2206RT curves:\" - \"$$x_{{mathrm{oppc}}}( {t_{{mathrm{go}}}} ) = frac{1}{{p_p( {t_{{mathrm{go}}}} )}} + a;e^{b;t_{{mathrm{go}}}} + c$$ (20) \" - \"Adjusted R was calculated to evaluate the goodness-of-fit of the combined, additive model (Fig._).\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - The data that support the findings of this study are available from the corresponding author upon reasonable request. - \\' \\'- - \"Gallistel, C. R. & Gibbon, J. Time, rate, and conditioning. Psychol. Rev. 107, 289u2013344 (2000)._ _ _ _ Paton, J. J. & Buonomano, D. V. The neural basis of timing: distributed mechanisms for diverse functions. Neuron 98, 687u2013705 (2018)._ _ _ _ _ de Hemptinne, C., Nozaradan, S., Duvivier, Q., Lefevre, P. & Missal, M. How do primates anticipate uncertain future events? J. Neurosci. 27, 4334u20134341 (2007)._ _ _ _ _ Ghose, G. M. & Maunsell, J. H. Attentional modulation in visual cortex depends on task timing. Nature 419, 616u2013620 (2002)._ _ _ _ _ Janssen, P. & Shadlen, M. N. A representation of the hazard rate of elapsed time in macaque area LIP. Nat. Neurosci. 8, 234u2013 241 (2005)._ _ _ _ Sharma, J. et al. Spatial attention and temporal expectation under timed uncertainty predictably modulate neuronal responses in monkey V1. Cereb. Cortex 25, 2894u20132906 (2015)._ _ _ Fiorillo, C. D., Newsome, W. T. & Schultz, W. The temporal precision of reward prediction in dopamine neurons. Nat. Neurosci. 11, 966u2013973 (2008)._ _ _ _ Cravo, A. M., Rohenkohl, G., Wyart, V. & Nobre, A. C. Endogenous modulation of low frequency oscillations by temporal expectations. J. Neurophysiol. 106, 2964u20132972 (2011)._ _ _ _ Schoffelen, J. M., Oostenveld, R. & Fries, P. Neuronal coherence as a mechanism of effective corticospinal interaction. Science 308, 111u2013113 (2005)._ _ _ _ _ Cui, X., Stetson, C., Montague, P. R. & Eagleman, D. M. Ready.go: amplitude of the FMRI signal encodes expectation of cue arrival time. PLoS Biol. 7, e1000167 (2009)._ _ _ _ _ Klein-Flugge, M. C., Hunt, L. T., Bach, D. R., Dolan, R. J. & Behrens, T. E. Dissociable reward and timing signals in human midbrain and ventral striatum. Neuron 72, 654u2013664 (2011)._ _ _ _ _ Bueti, D., Bahrami, B., Walsh, V. & Rees, G. Encoding of temporal probabilities in the human brain. J. Neurosci. 30, 4343u20134352 (2010)._ _ _ _ _ Nobre, A. C. & van Ede, F. Anticipated moments: temporal structure in attention. Nat. Rev. Neurosci. 19, 34u201348 (2018)._ _ _ _ Oswal, A., Ogden, M. & Carpenter, R. H. The time course of stimulus expectation in a saccadic decision task. J. Neurophysiol. 97, 2722u20132730 (2007)._ _ _ _ Luce, R. D. Response Times: Their Role in Inferring Elementary Mental Organization (Oxford Univ. Press; Clarendon Press, 1986).Buzsaki, G. & Llinas, R. Space and time in the brain. Science 358, 482u2013485 (2017)._ _ _ _ _ _ Rice, J. & Rosenblatt, M. Estimators of the log survivor function and hazard function. Sankhya Indian J. Stat. Ser. A 38, 60u201378 (1976).Buhusi, C. V. & Meck, W. H. What makes us tick? Functional and neural mechanisms of interval timing. Nat. Rev. Neurosci. 6, 755u2013765 (2005)._ _ _ Gibbon, J. Scalar expectancy theory and Webers law in animal timing. Psychol. Rev. 84, 279u2013325 (1977)._ _ Takahashi, Y. K., Langdon, A. J., Niv, Y. & Schoenbaum, G. Temporal specificity of reward prediction errors signaled by putative dopamine neurons in rat VTA depends on ventral striatum. Neuron 91, 182u2013193 (2016)._ _ _ _ _ Krakauer, J. W., Ghazanfar, A. A., Gomez-Marin, A., MacIver, M. A. & Poeppel, D. Neuroscience needs behavior: correcting a reductionist bias. Neuron 93, 480u2013490 (2017)._ _ _ _ J.V., S. Information Theory: A Tutorial Introduction. p. 31u201333 (Sebtel Press, 2015).Welford, A. T. Reaction Times. (Academic Press, London, 1980).Repp, B. H. & Su, Y. H. Sensorimotor synchronization: a review of recent research (2006-2012). Psychon. Bull. Rev. 20, 403u2013452 (2013)._ _ _ Williams, E. A., Yuksel, E. M., Stewart, A. J. & Jones, L. A. Modality differences in timing and the filled-duration illusion: testing the pacemaker rate explanation. Atten. Percept. Psychophys. 81, 823u2013845 (2019)._ _ _ Sherrick, C. E. & Cholewiak, R. W. in Handbook of perception and human performance, Vol. I: Sensory processes and perception (eds K. E. Boff, L. Kaufman, & J. P. Thomas) p. 1u201312, Vol. 58 (Wiley, 1986).Di Russo, F., Martinez, A., Sereno, M. I., Pitzalis, S. & Hillyard, S. A. Cortical sources of the early components of the visual evoked potential. Hum. Brain Mapp. 15, 95u2013111 (2002)._ _ _ Heath, R. G. & Galbraith, G. C. Sensory evoked responses recorded simultaneously from human cortex and scalp. Nature 212, 1535u2013 1537 (1966)._ _ _ _ _ Kemp, E. H., CoppexE9 , G. E. & Robinson, E. H. Electric responses of the brain stem to unilateral auditory stimulation. Am. J. Physiol. 120, 304u2013322 (1937)._ _ Libet, B., Alberts, W. W., Wright, E. W. Jr. & Feinstein, B. Responses of human somatosensory cortex to stimuli below threshold for conscious sensation. Science 158, 1597u20131600 (1967)._ _ _ _ _ Marshall, W. H., Talbot, S. A. & Ades, H. W. Cortical response of the anaesthesized cat to gross photic and electrical afferent stimulation. J. Neurophysiol. 6, 1u201315 (1943)._ _ Palva, S., Linkenkaer-Hansen, K., Naatanen, R. & Palva, J. M. Early neural correlates of conscious somatosensory perception. J. Neurosci. 25, 5248u20135258 (2005)._ _ _ _ _ Parasuraman, R. & Beatty, J. Brain events underlying detection and recognition of weak sensory signals. Science 210, 80u201383 (1980)._ _ _ _ _ Super, H., Spekreijse, H. & Lamme, V. A. Two distinct modes of sensory processing observed in monkey primary visual cortex (V1). Nat. Neurosci. 4, 304u2013310 (2001)._ _ _ _ Hohle, R. H. Inferred components of reaction times as functions of foreperiod duration. J. Exp. Psychol. 69, 382u2013386 (1965)._ _ _ _ Starkweather, C. K., Gershman, S. J. & Uchida, N. The medial prefrontal cortex shapes dopamine reward prediction errors under state uncertainty. Neuron 98, 616u2013629.e616 (2018)._ _ _ _ _ Starkweather, C. K., Babayan, B. M., Uchida, N. & Gershman, S. J. Dopamine reward prediction errors reflect hidden-state inference across time. Nat. Neurosci. 20, 581u2013589 (2017)._ _ _ _ _ Mauk, M. D. & Buonomano, D. V. The neural basis of temporal processing. Annu Rev. Neurosci. 27, 307u2013340 (2004)._ _ _ _ Grondin, S. & Rousseau, R. Judging the relative duration of multimodal short empty time intervals. Percept. Psychophys. 49, 245u2013256 (1991)._ _ _ _ Rousseau, R., Poirier, J. & Lemyre, L. Duration discrimination of empty time intervals marked by intermodal pulses. Percept. Psychophys. 34, 541u2013548 (1983)._ _ _ _ Zarco, W., Merchant, H., Prado, L. & Mendez, J. C. Subsecond timing in primates: comparison of interval production between human subjects and rhesus monkeys. J. Neurophysiol. 102, 3191u2013 3202 (2009)._ _ _ _ Jones, L. A., Poliakoff, E. & Wells, J. Good vibrations: human interval timing in the vibrotactile modality. Q. J. Exp. Psychol. 62, 2171u20132186 (2009)._ _ Stauffer, C. C., Haldemann, J., Troche, S. J. & Rammsayer, T. H. Auditory and visual temporal sensitivity: evidence for a hierarchical structure of modality-specific and modality-independent levels of temporal information processing. Psychol. Res. 76, 20u201331 (2012)._ _ _ Gibbon, J., Malapani, C., Dale, C. L. & Gallistel, C. Toward a neurobiology of temporal cognition: advances and challenges. Curr. Opin. Neurobiol. 7, 170u2013184 (1997)._ _ _ _ Matzke, D. & Wagenmakers, E. J. Psychological interpretation of the ex-Gaussian and shifted Wald parameters: a diffusion model analysis. Psychon. Bull. Rev. 16, 798u2013817 (2009)._ _ _ McGill, W. J. in Handbook of mathematical psychology. Vol. 1 (eds R. D. Luce, R. R. Bush, & E. Galanter) 309u2013360 (Wiley, 1963).Miles, W. R. Ocular dominance demonstrated by unconscious sighting. J. Exp. Psychol. 12, 113u2013126 (1929)._ _ Brainard, D. H. The psychophysics toolbox. Spat. Vis. 10, 433u2013436 (1997)._ _ _ _ Burbeck, S. L. & Luce, R. D. Evidence from auditory simple reaction times for both change and level detectors. Percept. Psychophys. 32, 117u2013 133 (1982)._ _ _ _ Rakitin, B. C. et al. Scalar expectancy theory and peak-interval timing in humans. J. Exp. Psychol. Anim. Behav. Process. 24, 15u201333 (1998)._ _ _ _ \"- - We thank Caroline Mirkes, Claudia Lehr, Freya Materne, and Cornelius Abel for help with data acquisition and technical support.- - Author notes - \\'These authors contributed equally: Matthias Grabenhorst, Georgios Michalareas.\\' - Authors and Affiliations - \"Neuroscience Department, Max-Planck-Institute for Empirical Aesthetics, GrxFC neburgweg 14, 60322, Frankfurt, GermanyMatthias Grabenhorst,_Georgios Michalareas_ &_David PoeppelDepartment of Psychology, Center for Neural Science, 6 Washington Place, New York, NY, 10003, USALaurence T. Maloney_&_David Poeppel\" - \"AuthorsMatthias GrabenhorstYou can also search for this author in _Georgios MichalareasYou can also search for this author in _Laurence T. MaloneyYou can also search for this author in _David PoeppelYou can also search for this author in _\" - Contributions - M.G., G.M. and D.P. designed the research. M.G. performed the experiments. M.G. and G.M. analyzed the data. All authors discussed the results. M.G., G.M., L.T.M. and D.P. wrote the paper. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks Yoshiko Yabe and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Deed CC0.\\' - \"A substantial fraction of particles acting as cloud seeds (called cloud condensation nuclei, CCN) originate from nucleation. Particle growth plays a critically important role here: to act as CCN, the formed tiny particles usually need to reach tens of nanometers (nm) in diameter. Growth competes with particle loss through coagulationu2014 the faster the particles grow, the larger the fraction that survive to CCN size. An order-of-magnitude growth in diameter from e.g., 5 to 50u2009nm increases the particle volume by three orders of magnitude. The composition of the particles is therefore largely defined by the compounds that grow them by condensation, often organic vapors. Comprehension of particle growth and its reasonable description in global and regional scale models is needed to improve our understanding of aerosol-cloud interactions, which are one of the most uncertain aspects of anthropogenic climate forcing.\"- - \"Particle size distribution observations and diameter growth rates derived from them are essential for understanding the growth of freshly formed nanoparticles. Measurements from various environments show particle growth rates of the order of 1u201310u2009nm/h. Currently available data provide information of variability and seasonality of nanoparticle growth, although the majority of the observations are from the Northern hemisphere, specifically from ground-based remote or rural continental sites in Europe.\" - \"Knowledge about the properties of the vapors that grow the particles has increased but is still incomplete (Fig._). Sulfuric acid, which plays a key role in nucleation, and other inorganic compounds such as ammonia, contribute to condensational growth in many environments. However, it has become evident that organic vapors are responsible for most of the particle growth in environments with a source of organic precursor gases. The advancement of mass spectrometric techniques in the last few years has enabled the identification of source groups and compounds contributing to particle growth, for example highly oxygenated organic molecules. However, their multi-phase nature and complexity have made the detection of individual organic molecules and their contributions to nanoparticle growth challenging. Given the fundamental role volatility plays in evaporation and condensation, one simplification is grouping condensable organics into experimentally determined volatility classes: extremely low, low, semi, and intermediate volatile organic compounds abbreviated as ELVOC, LVOC, SVOC, and IVOC, respectively. There is general understanding that the smaller the particles, the lower the volatility of a compound has to be in order for it to condense. In other words, the larger the particles, the larger the contribution of SVOC to their growth.\" - \"Fig. 2: Evolution of understanding of nanoparticle growth in boreal forests.Various evidence has contributed to building the picture of the significant role of organic vapors in nanoparticle growth in the boreal forest. Initially, frequent new particle formation and growth was observed without identification of the condensing vapors (Kulmala et al.). Later, sulfuric acid was found to explain a fraction of the growth (e.g., Kuang et al.), and direct and indirect evidence led to understanding of the significant role of organics (shown by e.g., Ehn et al. and reviewed in Olenius et al.) which was supported by laboratory simulations (TrxF6stl et al.). Recently, it was shown that the detected organic vaporsu2014 accounting for their volatilitiesu2014could explain the observed nanoparticle growth at a boreal forest site with a small contribution of sulfuric acid (Mohr et al.). psatu2009=u2009saturation vapor pressure, cu2009=u2009gas phase concentration, AMSu2009=u2009Aerosol Mass Spectrometer, CIMSu2009=u2009 Chemical Ionization Mass Spectrometer, OOAu2009=u2009oxygenated organic aerosol, HOMu2009=u2009Highly oxygenated organic molecule.\" - \"In the absence of particle-phase chemistry or transport limitations, condensation of known compounds to homogeneous liquid droplets is well described by basic mass transport models. Hand-in-hand with the increasing amount of experimental data, also the availability and robustness of different predictive models for the thermodynamics of atmospherically relevant compounds and mixtures has increased. These steps forward have allowed for the development of models describing the growth and evaporation of atmospheric >5u2009nm particles with good accuracy if the concentrations and molecular properties of the vapor phase compounds are known. We have recently been able to capture nanoparticle growth in the boreal forest, and to characterize the organic vapors involved, by using a combination of chemical ionization mass spectrometry and a condensation model. Therein we confirm the crucial role of oxidation products of biogenic volatile organic compounds for particle growth in this environment. Also on global and regional scales, the inclusion of a description of nanoparticle growth by organic compounds was shown to improve predictions of aerosol size distributions, hence suggesting that this process is important for capturing e.g., aerosol impacts on clouds.\" - Nanoparticle growth spans different phases, thermodynamic and kinetic regimes, and spatial and temporal scales. Further scientific progress on this topic requires a multidisciplinary and multiangle approach. In the following we will discuss a few examples of open research questions and ongoing research efforts.- - \"To date, a consistent picture for the role of organic precursors other than monoterpenes, particularly u03B1-pinene, is still developing. Particle growth mechanisms in environments other than the boreal forests, for example in the marine boundary layer, are not well understood. Ongoing research efforts concentrate for instance on isoprene, the most abundant VOC, and its oxidation products. Isoprene suppresses NPF and lowers SOA yields in mixed precursor environments. There is indication that isoprene oxidation products may contribute to the high particle numbers observed in the upper troposphere over the Amazon. At such high altitudes the lower temperature decreases the volatility of the molecules and shifts the equilibrium towards the particle phase. With the recent elucidation of the importance of NPF in urban environments, we expect progress on understanding particle growth in polluted environments to be imminent, despite the increased complexity of the mix of anthropogenic and natural precursor gases and oxidants.\" - Compared to gas or nanoparticle composition observations, growth rate measurements are more common and considered more robust. However, the interpretation of point observations of a particle population passing a measurement site in terms of growth is challenging due to possible effects of changes in air mass or vertical boundary layer mixing. In addition, traditional methods for calculating the growth rate based on mode peak diameter may not provide pure condensational growth rates when coagulation is significant. Also, these methods are fundamentally limited by the time resolution of the instrumentation. Development of promising new methods based on particle population dynamics is ongoing. - Net particle growth is a competition between condensation and evaporation. Whereas the former is driven by the concentration of the condensing vapors, the volatilities of the condensed compounds govern the latter. Pure compound saturation vapor concentrations Csat are therefore needed for accurate predictions of particle growth. However, the complex mixtures formed in the growing particles may not behave ideally and particle composition dependent activities need to be accounted for. There are large uncertainties how particle-phase chemical reactions or the formation of a viscous phase influence particle-phase volatilities, diffusivity and nanoparticle growth. This has led to e.g., the use of mass accommodation coefficients below unity for organic molecules colliding on SOA particles. We advise against below unity values due to the different ways diffusivity limitations or other SOA properties affect particle growth dynamics. - Experimentally determining Csat of all oxidized organic compounds in the atmosphere is nearly impossible. Hence, parametrizations of Csat, or the effective vapor concentration C* including non-ideality effects, with e.g., the molecular composition as input are increasingly used to obtain estimates of organic volatility distributions. However, the lack of molecular structure information, and again, far-from-complete representation of the complexity of the range of atmospheric compounds in developing the parametrizations, results in orders-of-magnitude uncertainties in saturation vapor concentrations. As the experimental techniques for identifying molecular composition of oxidized compounds are becoming more common, it is crucial to develop more precise methods to convert the molecular compositions into volatility information.- - Ultimately, an increased understanding at the molecular process level should lead to an improved description of the climate system at the global level. Assessing the climate impact of nanoparticle formation and growth therefore depends also crucially on our ability to represent them in large-scale models. Besides improving the fundamental understanding of the properties of the condensing species, it is also worthwhile to systematically investigate the minimum number of variables needed to represent their effects on the evolution of aerosol size distribution and composition. The optimal compromise between representing the complex particle growth processes and computational demands depends on the question at hand. - \"A few words on particle growth below 5u2009nm: At these sizes, the standard theoretical approaches applied for condensation and evaporation are not applicable anymore. Together with experimental challenges related to sampling and detection of the vapor and cluster populations, these issues make the interpretation of sub-5u2009nm particle growth observations challenging. Further work in the development of molecular resolution models and their systematic upscaling for various atmospherically relevant systems are therefore needed.\" - We expect the steep instrumental progress to continue in the next decade, and higher time- and chemical resolution data will become more widely available from various environments and different systems. The ability to process this wealth of data will likely become critical for improving our understanding on SOA, and emerging techniques based on e.g., machine learning will be needed. To support such analysis and the building of comprehensive understanding of the organic molecules in various environments, establishing open databases that combine the complex data and allow for broad use of the accumulating data will be helpful. We expect an increasing number of large-scale models to implement and/or improve the volatility representation for the condensing organic molecules in the coming decade, e.g., with a VBS (volatility basis set) scheme. This will allow for better assessment of the climate impacts of the atmospheric NPF, including nanoparticle growth.- - \"Hallquist, M. et al. The formation, properties and impact of secondary organic aerosol: current and emerging issues. Atmos. Chem. Phys. 9, 5155u20135236 (2009)._ _ _ Olenius, T., Yli-Juuti, T., Elm, J, Kontkanen, J. & Riipinen I. New particle formation and growth: creating a new atmospheric phase interface. In Physical Chemistry of Gas-Liquid Interfaces (eds. Faust, J. A. & House, J. E.), 315u2013352 (Elsevier, 2018).Gordon, H. et al. Causes and importance of new particle formation in the present-day and preindustrial atmospheres. J. Geophys. Res. Atmos. 122, 8739u20138760 (2017)._ _ Scott, C. E. et al. Impact of gas-to-particle partitioning approaches on the simulatedradiative effects of biogenic secondary organic aerosol. Atmos. Chem. Phys. 15, 12989u201313001 (2015)._ _ _ IPCC. Climate Change 2013: The Physical Science Basis. Contribution of Working Group I to the Fifth Assessment Report of the Intergovernmental Panel on Climate Change (eds. Stocker, T. F. et al.), (Cambridge University Press, 2013).Nieminen, T. et al. Global analysis of continental boundary layer new particle formation based on long-term measurements. Atmos. Chem. Phys. 18, 14737u201314756 (2018)._ _ _ Ehn, M. et al. A large source of low-volatility secondary organic aerosol. Nature 506, 476u2013479 (2014)._ _ _ TrxF6stl, J. et al. The role of low-volatility organic compounds in initial particle growth in the atmosphere. Nature 533, 527u2013531 (2016)._ _ Stolzenburg, D. et al. Rapid growth of organic aerosol nanoparticles over a wide tropospheric temperature range. Proc. Natl Acad. Sci. USA 115, 9122u20139127 (2018)._ _ _ Mohr, C. et al. Molecular identification of organic vapors driving atmospheric nanoparticle growth. Nat. Commun. 10, 4442 (2019)._ _ _ Bilde, M. et al. Saturation vapor pressures and transition enthalpies of low-volatility organic molecules of atmospheric relevance: from dicarboxylic acids to complex mixtures. Chem. Rev. 115, 4115u20134156 (2015)._ _ _ Patoulias, D. et al. Simulation of the size-composition distribution of atmospheric nanoparticles over Europe. Atmos. Chem. Phys. 18, 13639u2013 13654 (2018)._ _ _ McFiggans, G. et al. Secondary organic aerosol reduced by mixture of atmospheric vapours. Nature 565, 587u2013 593 (2019)._ _ _ Andreae, M. O. et al. Aerosol characteristics and particle production in the upper troposphere over the Amazon Basin. Atmos. Chem. Phys. 18, 921u2013961 (2018)._ _ _ Yao, L. et al. Atmospheric new particle formation from sulfuric acid and amines in a Chinese megacity. Science 361, 278u2013281 (2018)._ _ _ Donahue, N. M., Epstein, S. A., Pandis, S. N. & Robinson, A. L. A two-dimensional volatility basis set: 1. organic-aerosol mixing thermodynamics. Atmos. Chem. Phys. 11, 3303u20133318 (2011)._ _ _ Kulmala, M., Toivonen, A., MxE4kelxE4, J. M. & Laaksonen, A. Analysis of the growth of nucleation mode particles observed in Boreal forest. Tellus B 50, 449u2013462 (1998)._ _ Kuang, C. et al. An improved criterion for new particle formation in diverse atmospheric environments. Atmos. Chem. Phys. 10, 8469u20138480 (2010)._ _ _ \"- - The authors would like to thank their research groups and colleagues working on all aspects related to nanoparticle growth for fruitful discussions and past and future research efforts and insights.- - Authors and Affiliations - \"Department of Applied Physics, University of Eastern Finland, Kuopio, FinlandTaina Yli-JuutiDepartment of Environmental Science (ACES), Stockholm University, Stockholm, SwedenClaudia Mohr_&_Ilona Riipinen\" - \"AuthorsTaina Yli-JuutiYou can also search for this author in _Claudia MohrYou can also search for this author in _Ilona RiipinenYou can also search for this author in _\" - Contributions - T.Y.-J., C.M., and I.R. contributed equally to writing this comment. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons Licence CCO Public Domain image: ) were put together in Adobe Photoshop (version 21.2.1) and are not to anatomical scale. Figure was made using GraphPad Prism version 8.4.3 for macOS, GraphPad Software, San Diego, California USA, .\"- - \"Since the inception of the Monro-Kellie doctrine, it has often been assumed that brain tissue itself plays a static role in the ICP compliance response, while CSF and blood act as the dynamic components, redirecting in attempt to maintain ICP homeostasis. However, clear in vitro evidence shows that neurons possess regulatory mechanisms to adjust cell volume in response to stressors such as high environmental pressure; so it seems unreasonable to assume that brain tissue has no role in compliance. With careful tissue preservation, strict a priori exclusion criterion, and precise quantitative stereological assessment methods, we demonstrate that large ischemic and hemorrhagic strokes in adult rodents causes widespread tissue and cell shrinkage in areas well outside the injury. Given the nature and timing of tissue compliance, we surmise that it reflects ICP compensation following large space occupying-lesions. Notably, we found in experiment 1 that tissue shrinkage was greatest on D1 (i.e. before peak ICP at 24u201372_h, historically), with a subsequent rebound in the following 6_days. We emphasize that this shrinkage is distinct from tissue deformation or midline shift, which have long been recognized. Under TEM, neurons simultaneously displayed signs of active volume regulation and sublethal injury, indicating that tissue compliance may have functional consequences throughout the brain. Overall, these findings challenge the current dogma regarding pressureu2013 volume relationships (Fig._b,c), and indicates that neurons may adaptively regulate their volume in response to elevated ICP, at least after severe insults.\" - \"Following ischemic and hemorrhagic stroke, neurons across brain regions decreased in both somata volume and extracellular space (Fig._d,g). These data, along with our earlier findings, lead us to a modified interpretation of the Monro-Kellie doctrine (Fig._e,f). In this revision, ICP compliance consists of the reduction of vascular blood and CSF in the cranium (well-established), along with a temporary adaptive change in tissue compliance to provide further compensation after more severe strokes. Likely, this extra compliance mechanism is only invoked following severe brain injury, as indicated by our previous serendipitous and retrospective observations in the AWB model, and perhaps only after other compliance mechanisms have been heavily engaged. There is no simple way to measure the contribution of each mechanism, but consider this: the peak of the tissue shrinkage effect occurred on D1 after ICH, with a remarkable 42% average reduction in neuronal soma volume and 34% increase in packing density, along with an average soma volume reduction of 41% in astrocytes. Similarly, following MCAO, there was a 23% average reduction in contralateral neuronal soma volume, and a 22% increase in packing density at that time. Even a small reduction in cell volume and a concurrent increase in packing density, if across the brain, could free up appreciable space to buffer heightened ICP. As noted, freeing up just 5% of the volume occupied by gray matter would represent a savings of 65_mL in the human cranium, allowing considerable room for mass effect, given that the average hematoma size isu2009 ~u200935 mL. High ICP is the common element of the three models where we have found tissue compliance: AWB model (mass effect predominantly due to extensive hematoma), collagenase model (mass effect due to hematoma and significant edema) and MCAO (mass effect solely from edema). Therefore, the most parsimonious explanation is that elevated ICP is primarily driving tissue compliance.\" - \"Changes in cell volume often occur through regulated processes (commonly referred to as regulatory volume decrease; RVD) in order to maintain a homeostatic set-point. It is well-established that neurons and astrocytes have the capacity to reversibly regulate their volume in response to high pressure and other noxious stimuli, but the mechanosensation mechanism through which these RVDs occur has yet to be fully identified, and is beyond this studys scope. Recently, RVDs have been demonstrated to have mechanistic overlap with early apoptotic shrinkage; during the first stage of apoptotic shrinkage, a neuron can decrease in volume by 20u201340% via ion and osmolyte extrusion alone. However, in the present study, we demonstrated via FJC cell counts that tissue compliance occurred in the absence of any notable cell death. Although RVDs and apoptotic shrinkage do share some mechanisms, under TEM there were no signs of apoptosis (e.g. chromatin condensation and apoptotic bodies), supporting the FJCu2009+u2009count data. The fact that cell volume and cortical thickness across most regions began to rebound by D7 post-ICH without a change in cell death (vs. SHAMs) strongly suggests that the volume reduction we observed is not from apoptotic shrinkage. At the very least, it reflects the fact that these cells are not reaching the point of no return at which programmed cell death processes proceed. While our experiments looking as late as D7 post-ICH suggest that shrunken cells will survive, longer term survival studies are still required to prove this.\" - \"Although we saw no definitive evidence of cell death in CA1 and S1 in Experiment 1, TEM suggests that tissue shrinkage has consequences. Despite the distance from the hematoma, there was evidence of sublethal damage (e.g. electron dense mitochondria, perturbed myelin, axonal atrophy) in neurons across both hemispheres. Compared to our past TEM work characterizing ischemic cell death, it was obvious that the sub-lethal damage observed in this study did not include features of irreversible ischemic injury, which suggests that the underlying mechanisms are due to tissue compliance rather than ongoing cell death. The dramatic intranuclear/cytoplasmic vacuolization and crenated membranes indicate that neurons are actively recycling membrane to reduce cell volume, but further study is required to validate thisu2014 although we did not notice other pathological changes within the neuropil of cortex and hippocampus, it is possible that additional quantitative TEM might reveal other perturbations. Concerningly, the numerous potential signs of subcellular injury suggest that the tissue compliance processes are detrimental to cellular health, potentially impacting brain function. As a key translational step, future work should ascertain the functional impact of these histological findings.\" - \"We used multiple stroke models to determine whether or not tissue shrinkage is unique to hemorrhagic stroke. We did not intend to directly compare models, but perhaps the greater shrinkage after ICH versus ischemia is due to both the mass effect of the hematoma and edema contributing to greater elevations in ICP. However, the models were not matched for lesion size or ICP response, so future work is required to delineate those relationships. The apparent lack of tissue compliance following HIE likely reflects the developmental differences between rat pups and adults; pups do not begin to undergo skull suture fusion until PD12, which is not complete untilu2009~u2009PD22, although other mechanisms may be at play (e.g., greater reduction in CSF levels). Due to this developmental timeline, ICP is rarely measured in HIE studies; the cranial sutures have not yet fused by PD8, meaning that pressure is free to equalize after injury prior to suture fusion. Accordingly, the tissue compliance effect following HIE appeared to negligible in experiment 3. These findings do not rule out the possibility that tissue shrinkage can occur in rat pups, as the timing or injury severity threshold to initiate tissue compliance could differ among brain injuries (e.g. later development of injury in HIE vs. adult MCAO). This also holds true for our findings in ischemic stroke; future study is required to elucidate the time course of tissue compliance in ischemia and other forms of brain injury, due to differences in ICP timing and magnitude that are dependent on the specific pathophysiology of the insult.\" - Two key questions are whether tissue shrinkage-mediated compliance occurs in humans, and if it is a viable therapeutic target. In order to answer those, one must consider many other issues, such as the dynamic interplay among all compliance mechanisms and ICP, as well as studying the impact of age, sex, and various co-morbidities. Shrinkage, not just of edematous tissue but distal regions too, may also be affected by osmotherapies, such as hypertonic saline or mannitol; this hypothesis is based upon the fact that regulatory volume decreases in neurons occur via the movement of ions and organic osmolytes across the plasma membrane. While osmotherapies may augment tissue compliance, we must consider that they may also enhance subcellular perturbations, or perhaps cause cell death. There is some evidence in other clinical fields that suggests that tissue compliance may occur in humans. For example, one case study documented brain-wide cell compression as a result of tumour mass effect and high ICP in several individuals, but without global loss of neurons, similar to what we found here. This at least suggests that our findings may generalize to humans. However, if tissue compliance does not occur in humans, at least acutely after stroke, it represents a fundamental species difference, and one that may help partly explain other differences between clinical stroke and animal models (e.g. higher survival rates in rodents following severe strokes). Before considering that, other factors (old age, hypertension, etc.) that may block the tissue shrinkage response have to be defined, which we are currently evaluating. - There are a number of additional considerations that remain unexplored. Astrocytes had similar volume reductions after ICH, but this was a post-hoc analysis, and further study is needed to confirm and extend those initial observations (e.g., after other forms of brain injury, in other brain regions, and with additional methods to better delineate sub-type responses). ICP was not directly measured in animals used for histology (to minimize complexity and potential confounds), however, we have repeatedly demonstrated intracranial hypertension in the same ICH and MCAO models as used presently. We emphasize the challenging nature of directly relating ICP to any given compensatory mechanism due to the dynamic presence of several others, especially given the complexity of interpreting the ICP response (measured parameters could include average, peak, spiking behaviour, etc.). Due to the anatomical constraints of the cranium, simultaneously measuring ICP and other compliance components along with determining hematoma and edema volume accurately without introducing any confounds is currently impossible. Blood gases, glucose, and pressure were not measured in the present study because our past work demonstrates that these parameters only vary modestly, and are unrelated to brain injury. More directly, in an unpublished post-hoc analysis of data from our recent study, we found that fluctuations in blood pressure and glucose do not correlate with tissue compliance; we also found that anesthetic does not appear to influence tissue compliance. Lastly, the studies characterizing mechanosensitive volume changes in healthy neurons have predominantly been done in vitro, and the effect of pathologically high ICP has not directly been evaluated in relation to cell volume regulatory mechanisms. As such, exploring these mechanisms in vivo in regard to distal tissue compliance remains a vital future direction if we are to more effectively and safely manipulate intracranial compliance. - \"Our findings challenge the long-held view that brain tissue is an immutable component of ICP pressureu2013volume relationship. As such, we propose a key modification to the Monro-Kellie doctrine (Fig._e,f) to include adaptive cell volume changes as part of a series of compliance responses to elevated ICP. We acknowledge that further work is required to determine the relative contribution of this and other mechanisms to intracranial compliance. Furthermore, the present study suggests that widespread sublethal damage occurs in neurons undergoing shrinkage, perhaps as a direct result of shrinkage mechanisms (e.g. loss of protective osmolytes), and again additional study is required to uncover the mechanism(s) responsible. If this is the case, then there is a functional cost to neurons as they reduce their volume to supplement ICP compliance, and driving it further might create additional cellular damage outside the damaged and edematous zone. Going forward, we believe that effective control of intracranial hypertension, at least for severe brain injuries, will require consideration to widespread cell volume perturbations.\"- - Subjects - \"All experiments followed the ARRIVE guidlines and the Canadian Council on Animal Care Guidelines and were approved (protocols #960, #363, and #557) by the Biological Sciences and Health Sciences Animal Care and Use Committees at the University of Alberta. In total, 82 male Sprague Dawley rats were used. We purchased 62 male Sprague Dawley rats (350u2013400_g,u2009~u20092u20133_months of age) from Charles River (Saint Constant, Quebec). Long Evans rats were purchased from Charles River (Montreal, QB), bred, and delivered vaginally. Within 48_ h of birth, rat litters were culled to 10 pups, reducing weight variability. The Yager Lab bred 20 male Long Evans rat pups, reared by their dams and used on post-natal days 7u20138. Rats were kept in a temperature and humidity-controlled environment with a 7:00u201319:00_h light cycle, during which all experiments were performed; all Sprague Dawley rats were solo-housed following surgical procedures for the duration of experiments. Water and standard rat chow were provided ad libitum, with water consumption monitored over the duration of experiment 1.\" - Experimental protocols - In order to have confidence in these findings, we followed guidelines on translational rigour. Our experiments used a priori power analyses to determine group sizes. We aimed to have at least 80% power to detect a 25% difference in neuron soma volume and density, based on our past tissue response work in multiple ICH models. We ensured that this sample size allowed for an average stereological coefficient of error of less than 0.1 for each endpoint, as convention dictates. The manipulation of experimental parameters such as timing and type of brain injury were used to examine the generalizability and robustness of the tissue shrinkage effect. All rats were assigned to groups using a random number generator (random.org). All analyses were conducted by an experimenter blinded to group identity. Unbiased stereology was used to ensure high accuracy for all cell morphology quantification, and a strict tissue quality exclusion criterion was followed to maintain optimal conditions for stereological assessment. The experimental timelines and endpoints are summarized in Table . - \"In experiment 1 (Nu2009=u200936), adult rats were given a collagenase-induced ICH, and euthanized 1 (ICH-D1; nu2009=u20098), 3 (ICH-D3; nu2009=u20098), or 7_days (ICH-D7; nu2009=u20098) later. Control rats (SHAM; nu2009=u2009 12) were given a sham ICH procedure, and for blinding purposes, further randomized to 1 (nu2009=u20094), 3 (nu2009=u20094), or 7-day survival times (nu2009 =u20094), with the a priori intention to combine these subgroups if no statistical differences were detected, as there was no plausible expectation of subgroup differences. In experiment 1, neurons were assessed in areas CA1, CA3, S1, and the contralateral striatum, and astrocytes were assessed in ipsilateral CA1 and contralateral striatum. In experiment 2 (Nu2009=u200916), adult rats either received an MCAO (MCAO-D1; nu2009=u20099) or sham (SHAM; nu2009=u20097) procedure and were euthanized 24_h later. Regions were only assessed contralaterally due to widespread edema and injury in the injured hemisphere. In experiment 3 (Nu2009=u200920), post-natal day 7 (PD7) rat pups received either an HIE (HIE-D1; nu2009=u200910) or sham (SHAM; nu2009=u200910) procedure and were euthanized 24_h later. Areas CA1 and S1 were analyzed contralaterally in experiment 3. In experiment 4 (Nu2009=u200910), adult rats received a collagenase ICH (nu2009=u20095) or a sham (nu2009=u20095) procedure, and were euthanized 24_h later. Areas CA1 and S1 were assessed bilaterally in experiment 4.\" - Exclusion criteria - Exclusion criteria were determined prior to initiating the study, and applied in a blinded manner, as accurate stereological analysis relies on optimal tissue quality. Prior to microscopic examination, if tissue had noticeable differences in staining penetration or other major histological processing damage (e.g. dark neurons, freezing damage), rats were entirely excluded from analysis. Rats were excluded from representative regional analysis (i.e. not the overall experiment) if there were local tissue distortions in the region of interest, as these tissue disturbances can alter the local cellular density and other parameters that play a role in stereological accuracy. - Intracerebral hemorrhage - \"Rats were anesthetized with isoflurane (4% induction, 2% maintenance in 60% N2O and remainder O2), and maintained at 37_xB0C via rectal temperature probe and heating pad. The ICH surgical procedure was performed as previously described. A burr hole was drilled at 0.5_mm anterior and 3.5_mm left to bregma. Bacterial collagenase (Type IV-S, Sigma, 3.0 u03BCL of 0.6 U/u03BCL in saline) was injected at a depth of 6.5_mm. Bupivacaine hydrochloride (0.5_mg/kg SC) was used as a post-operative analgesic and 5_mL of 0.9% saline was injected subcutaneously to ensure hydration. For sham procedures, rats were kept under isoflurane for equivalent time (~u200925_min). An incision was made, and connective tissue cleared, but no hole was drilled in order to maintain normal ICP by avoiding a drop in ICP due to equalization with atmospheric pressure, or have an increase in ICP due to an inflammatory response. Sham-operated rats had identical doses of bupivacaine and saline.\" - Middle cerebral artery occlusion - \"Rats were anesthetized with isoflurane and maintained at 37_xB0C. Bupivacaine was administered subcutaneously at the ventral neck region incision site. Rats in the MCAO-D1 group were subjected to 2_h of middle cerebral artery occlusion via the intraluminal suture occlusion model, as described previously. A 4u2013 0 silk suture was used to temporarily ligate the right common carotid artery and the internal carotid artery. A 4u20130 monofilament suture with a 0.41_mm diameter (Item #404156PK10, Doccol Corp, Redlands, CA, USA) was used to occlude the middle cerebral artery. After the procedure, the midline neck incision was closed with 4u20130 vicryl absorbable suture (Ethicon, Somerville, NJ, USA), and bupivacaine was given. All rats received 5_mL of 0.9% saline subcutaneously to ensure hydration. For sham procedures, a ventral midline neck incision was made, and blood vessels were isolated but not ligated or occluded; all SHAM rats were given the same duration of anesthetic as MCAO-D1 rats, and identical doses of analgesic and saline.\" - Hypoxic-ischemic encephalopathy - The classic Vannucci-Rice model of HIE was used in this experiment, as outlined previously. For detailed methods, refer to Supplemental Material (SM) 1.1. - Histology - \"For euthanasia, rats from experiment 1 and 3 were given 100_mg/kg of sodium pentobarbital IP, while rat pups in experiment 2 were given an isoflurane overdose; all animals were then transcardially perfused with 0.9% saline followed by 10% neutral buffered formalin (Fisher Scientific, Toronto, ON, Canada). Brains were carefully extracted, and immersed in fresh 10% neutral buffered formalin for at least a week to allow for optimal fixative penetration. Prior to cryosectioning, brains were immersed in a 30% sucrose-formalin solution until they sank,u2009 ~u200948u201372_h. Beginning at a random rostral start point to ensure systematic random sampling, brains were sectioned at 20_u03BCm, with every 10 pair of serial sections kept, and placed on two slide sets. Slides were stained with cresyl violet as described previously. In experiment one, an extra set of slides were taken to allow for Fluoro-Jade C staining according to the manufacturer s protocol (#AG325, MilliporeSigma, Oaksville, ON, Canada). Tissue processing was identical across groups; however, to further address the possibility that tissue processing was somehow responsible for the tissue compliance effect, we conducted pilot work assessing cell volume and density in both transcardially perfused and immersion fixed ICH and SHAM tissue, and found no significant differences in the extent of tissue compliance between the two methods (SM 1.2).\" - Hydration levels - At the time of euthanasia, abdominal muscle was taken to assess water content. Please refer to SM 1.3 for detailed methods. - Stereological analysis - Recently, it has been suggested that combining several stereological dissectors with local stereological assessment probes, such as the nucleator, may provide explanatory accuracy equivalent with serial reconstruction studies. Accordingly, the following experiments performed in this study were analyzed using a combinatorial stereological approach to ensure precision and accuracy. To assess cell volume in the present study, the nucleator probe was used in combination with the physical dissector. To assess packing density, both the physical and optical dissector probes were used. For more detailed methods, please refer to SM 1.4. - Lesion volume and cortical thickness assessment - \"Lesion volume and cortical thickness were assessed in experiments 1u20133 using Image J software (v. 1.52A, NIH). In experiments 2u20133, sections for stereology were taken so that the same tissue could be used to determine both lesion volume and cortical thickness. For the lesion volume assessment, 20_u03BCm coronal sections were assessed every 200_u03BCm from approximatelyu2009+u20095.45_ mm tou2009u2212u20096.00_mm from bregma, as done previously. In experiment 1, the section containing the maximum hematoma area was assessed instead of lesion volume, as these correlate closely, and we did not section through the entire lesion to accurately gauge lesion volume. In experiments 1u20133, the average cortical thickness of each hemisphere was calculated by measuring the distance between the dorsal aspect of the cingulum to cortical layer II in five sections rangingu2009+u20091.89 tou2009u2212u20092.0 from bregma.\" - Transmission electron microscopy - \"At 24_h post-stroke, each rat in experiment four was given 100_mg/kg of sodium pentobarbital IP, and transcardially perfused with heparinized 0.9% saline followed by Karnovskys fixative (glutaraldehyde, paraformaldehyde, sodium cacodylate buffer, balanced to a pH of 7.4). The head of each rat was removed, placed intact into fresh fixative, and stored at 4_xB0C overnight to prevent histological artefact. After 24_h immersed in situ, brains were removed and areas CA1 and S1 dissected out into 1u2009xD7u20092_mm blocks (maintaining orientation). After processing, blocks were embedded in resin, sectioned at 90_nm, and mounted on grids. First, toluidine blue was used to stain semi-thick (1_u03BCm) sections to locate the area of interest, then ultra-thin sections_(90_nm) were taken and stained (uranyl acetate, lead citrate), then imaged via electron microscopy. Overall,u2009~u20092000 neurons across areas CA1 and S1 were examined. Samples were imaged under a Philips-FEI Morgagni 268 80_kV (Phillips-FEI, Oregon, USA) transmission electron microscope at various magnifications for signs of perturbation compared to SHAMs.\" - Statistical analysis - \"Prior to analysis, the Shapirou2013Wilk test was used on all datasets to assess normality; in all cases, the result was non-significant, indicating a normally distributed population. In experiments 1u20133, cell volume, cell density, cortical thickness, and FJC counts were analyzed using a two-way ANOVA with Sidaks multiple comparisons test. The two factors compared were region (within-subject) and experimental group (between-subjects). Sham rats were collapsed across time and treated as one experimental group to increase statistical power, as there were no statistical differences due to time among sham sub-groups. Hemisphere was originally treated as a third factor, but no hemisphere differences were found, so analysis was simplified. When main effects were significant, the simple effects found via Sidaks test were reported. Abdominal muscle water content in experiment 1 were analyzed using a one-way ANOVA with Sidak s test. Cohens d was used as a measure of effect size for all endpoints to avoid overreliance on statistical significance, and was calculated using G*Power (Version 3.9.1.4). When using Cohens d as a measure of effect size, typical benchmark thresholds are du2009=u20090.2, 0.5, and 0.8 for small, medium and large effects, respectively. Average amount of tissue compliance across regions in each model was calculated by averaging the percent reduction in each region compared to SHAMs. Descriptive statistics are meanu2009xB1u2009 95% confidence intervals (C.I.), as recommended. All analyses were done in GraphPad Prism v.6, with the exception of experiment 1, which was analyzed using GraphPad Prism v.8. Stereology calculations were done using RStudio (Version 1.2.1335, RStudio Inc.).\"- - \\' \\' - All data is presented in a supplementary file (Supplementary Dataset 2), along with an additional supplemental material file outlining our stereology methods in further detail (Supplemental Material 1). - \\' \\'- - \"Wilson, M. H. Monrou2013Kellie 2.0: the dynamic vascular and venous pathophysiological components of intracranial pressure. J. Cereb. Blood Flow Metab. 36, 1338u2013 1350 (2016)._ _ _ _ Iencean, S. M. & Ciurea, A. V. Intracranial hypertension: classification and patterns of evolution. J. Med. Life 1, 101u2013107 (2008)._ _ _ McDowell, M. M., Ducruet, A. F. & Friedlander, R. M. Management of cerebral edema/intracranial pressure in ischemic stroke. In Primer on Cerebrovascular Diseases 738u2013 742 (Academic Press, 2017).Stevens, R. D., Shoykhet, M. & Cadena, R. Emergency Neurological Life Support: Intracranial Hypertension and Herniation. Neurocrit. Care 23, 76u201382 (2015)._ _ _ Mokri, B. The Monro-Kellie hypothesis: applications in CSF volume depletion. Neurology 56, 1746u20131748 (2001)._ _ _ _ Allen, R. Intracranial pressure: a review of clinical problems, measurement techniques and monitoring methods. J. Med. Eng. Technol. 10, 299u2013320 (1986)._ _ _ _ Ziai, W. C. et al. A multicenter, randomized, double-blinded, placebo-controlled phase III study of clot lysis evaluation of accelerated resolution of intraventricular hemorrhage (CLEAR III). Int. J. Stroke 9, 536u2013542 (2014)._ _ _ Lundberg, N. The Saga of the Monrou2013Kellie doctrine. In Intracranial Pressure V 68u2013 76 (Springer Berlin Heidelberg, 1983).Benarroch, E. E. Neuron-astrocyte interactions: partnership for normal function and disease in the central nervous system. Mayo Clin. Proc. 80, 1326u20131338 (2005)._ _ _ _ Toney, G. M. Regulation of neuronal cell volume: from activation to inhibition to degeneration. J. Physiol. 588, 3347u20133348 (2010)._ _ _ _ _ Morris, C. E., Wang, J. A. & Markin, V. S. The invagination of excess surface area by shrinking neurons. Biophys. J. 85, 223u2013 235 (2003)._ _ _ _ _ Mills, L. R. & Morris, C. E. Neuronal plasma membrane dynamics evoked by osmomechanical perturbations. J. Membr. Biol. 166, 223u2013235 (1998)._ _ _ _ Wan, X., Harris, J. A. & Morris, C. E. Responses of neurons to extreme osmomechanical stress. J. Membr. Biol. 145, 21u201331 (1995)._ _ _ _ Luders, E. et al. Brain size and grey matter volume in the healthy human brain. NeuroRep. 13, 2371u20132374 (2002)._ _ Broderick, J. P., Brott, T. G., Duldner, J. E., Tomsick, T. & Huster, G. Volume of intracerebral hemorrhage a powerful and easy-to-use predictor of 30-day mortality. Stroke 24, 987u2013993 (1993)._ _ _ _ Williamson, M. R. & Colbourne, F. Evidence for decreased brain parenchymal volume after large intracerebral hemorrhages: a potential mechanism limiting intracranial pressure rises. Transl. Stroke Res. 8, 386u2013 396 (2017)._ _ _ _ Kerr, N. L. HARKing: hypothesizing after the results are known. Personal. Soc. Psychol. Rev. 2, 196u2013217 (1998)._ _ _ Dirnagl, U. et al. A concerted appeal for international cooperation in preclinical stroke research. Stroke 44, 1754u20131760 (2013)._ _ _ _ Dirnagl, U. Resolving the Tension Between Exploration and Confirmation in Preclinical Biomedical Research 71u201379 (Springer, Cham, 2019)._ Hoff, J. T. & Xi, G. Brain edema from intracerebral hemorrhage. In Brain Edema XII 11u201315 (Springer Vienna, 2003).Xi, G., Keep, R. F. & Hoff, J. T. Mechanisms of brain injury after intracerebral haemorrhage. Lancet Neurol. 5, 53u201363 (2006)._ _ _ John, R. F. & Colbourne, F. Delayed localized hypothermia reduces intracranial pressure following collagenase-induced intracerebral hemorrhage in rat. Brain Res. 1633, 27u201336 (2016)._ _ _ _ Hiploylee, C. & Colbourne, F. Intracranial pressure measured in freely moving rats for days after intracerebral hemorrhage. Exp. Neurol. 255, 49u201355 (2014)._ _ _ Ishikawa, M., Yoshitomi, T., Zorumski, C. F. & Izumi, Y. Effects of acutely elevated hydrostatic pressure in a rat ex vivo retinal preparation. Investig. Opthalmology Vis. Sci. 51, 6414 (2010)._ _ Silasi, G., MacLellan, C. L. & Colbourne, F. Use of telemetry blood pressure transmitters to measure intracranial pressure (ICP) in freely moving rats. Curr. Neurovasc. Res. 6, 62u201369 (2009)._ _ _ Williamson, M. R., John, R. F. & Colbourne, F. Measurement of Intracranial Pressure in Freely Moving Rats. In Traumatic and Ischemic Injury 17u201325 (2018).Murtha, L. A. et al. Intracranial pressure elevation after ischemic stroke in rats: cerebral edema is not the only cause, and short-duration mild hypothermia is a highly effective preventive therapy. J. Cereb. Blood Flow Metab. 35, 592u2013600 (2015)._ _ _ Mujsce, D. J., Towfighi, J., Stern, D. & Vannucci, R. C. Mannitol therapy in perinatal hypoxic-ischemic brain damage in rats. Stroke 21, 1210u20131214 (1990)._ _ _ _ Dohrmann, G. J., Wagner, F. C. & Bucy, P. C. Transitory traumatic paraplegia: electron microscopy of early alterations in myelinated nerve fibers. J. Neurosurg. 36, 407u2013415 (1972)._ _ _ _ Etten, K. The Effect of Hydrostatic Pressure on Neuronal Cell Morphology in Vitro (Clemson University, Clemson, 2017)._ Robaglia, A. A., Cau, P., Bottini, J. & Seite, R. Effects of isolation and high helium pressure on the nucleolus of sympathetic neurons in the rat superior cervical ganglion. J. Auton. Nerv. Syst. 27, 207u2013219 (1989)._ _ _ _ Falcone, G. J. et al. Predictors of hematoma volume in deep and lobar supratentorial intracerebral hemorrhage. JAMA Neurol. 70, 988u2013994 (2013)._ _ _ Hoffmann, E. K., Lambert, I. H. & Pedersen, S. F. Physiology of cell volume regulation in vertebrates. Physiol. Rev. 89, 193u2013277 (2009)._ _ _ _ Mongin, A. A. & Orlov, S. N. Mechanisms of cell volume regulation and possible nature of the cell volume sensor. Pathophysiology 8, 77u201388 (2001)._ _ _ _ NxFAxF1ez, R., Sancho-MartxEDnez, S. M., Novoa, J. M. L. & LxF3pez-HernxE1ndez, F. J. Apoptotic volume decrease as a geometric determinant for cell dismantling into apoptotic bodies. Cell Death Differ. 17, 1665u20131671 (2010)._ _ _ Heimlich, G., Bortner, C. D. & Cidlowski, J. A. Apoptosis and cell volume regulation: the importance of ions and ion channels. In Cell Volume and Signalling (eds. Peter Lauf & Norma Adragna) 193u2013194 (Advances in Experimental Medicine and Biology, 2004).Pasantes-Morales, H. Channels and volume changes in the life and death of the cell. Mol. Pharmacol. 90, 358u2013370 (2016)._ _ _ _ Askalan, R. et al. Mechanisms of neurodegeneration after severe hypoxic-ischemic injury in the neonatal rat brain. Brain Res. 1629, 94u2013103 (2015)._ _ _ _ Saito, A. et al. Oxidative stress and neuronal death/survival signaling in cerebral ischemia. Mol. Neurobiol. 31, 105u2013116 (2005)._ _ _ _ Colbourne, F., Sutherland, G. R. & Auer, R. N. Electron microscopic evidence against apoptosis as the mechanism of neuronal death in global ischemia. J. Neurosci. 19, 4200u20134210 (1999)._ _ _ _ _ Grova, M. et al. Animal models of cranial suture biology. J. Craniofac. Surg. 23, 1954u20131958 (2012).Ordaz, B. et al. Osmolytes and mechanisms involved in regulatory volume decrease under conditions of sudden or gradual osmolarity decrease. Neurochem. Res. 29, 65u201372 (2004)._ _ _ _ Cohen, B. A. et al. Brain compression without global neuronal loss in meningiomas: whole-brain proton MR spectroscopy report of 2 cases. AJNR. Am. J. Neuroradiol. 26, 2178u20132182 (2005)._ _ _ MacLellan, C. L. et al. Intracerebral hemorrhage models in rat: comparing collagenase to blood infusion. J. Cereb. Blood Flow Metab. 28, 516u2013 525 (2008)._ _ _ _ DeBow, S. & Colbourne, F. Brain temperature measurement and regulation in awake and freely moving rodents. Methods 30, 167u2013171 (2003)._ _ _ _ Wilkinson, C. M. et al. The collagenase model of intracerebral hemorrhage in awake, freely moving animals: the effects of isoflurane. Brain Res. 1728, 146593 (2020)._ _ _ _ Kilkenny, C., Browne, W. J., Cuthill, I. C., Emerson, M. & Altman, D. G. Improving bioscience research reporting: the ARRIVE guidelines for reporting animal research. PLoS Biol. 8, e1000412 (2010)._ _ _ _ _ Fisher, M. et al. Update of the stroke therapy academic industry roundtable preclinical reccomendations. Stroke 40, 2224u20132250 (2009)._ _ MxF8ller, A., Strange, P. & Gundersen, H. J. G. Efficient estimation of cell volume and number using the nucleator and the disector. J. Microsc. 159, 61u2013 71 (1990)._ _ _ Brown, D. L. Practical stereology applications for the pathologist. Vet. Pathol. 54, 358u2013368 (2017)._ _ _ Gundersen, H. J. G., Jensen, E. B. V., Kieu, K. & Nielsen, J. The efficiency of systematic sampling in stereologyu2014reconsidered. J. Microsc. 193, 199u2013211 (1999)._ _ _ _ Golub, V. M. et al. Neurostereology protocol for unbiased quantification of neuronal injury and neurodegeneration. Front. Aging Neurosci. 7, 196 (2015)._ _ _ _ _ Deniz, xD6. G. et al. A concise review of optical, physical, and isotropic fractionator techniques in neuroscience studies, including recent developments. J. Neurosci. Methods 310, 45u201353 (2018)._ _ _ _ Fix, A. S. & Garman, R. H. Practical aspects of neuropathology: a technical guide for working with the nervous system. Toxicol. Pathol. 28, 122u2013131 (2000)._ _ _ _ Longa, E. et al. Reversible middle cerebral artery occlusion without craniectomy in rats. Stroke 20, 84u201391 (1989)._ _ _ _ Nguyen, A., Armstrong, E. A. & Yager, J. Y. Unilateral common carotid artery ligation as a model of perinatal asphyxia: the original rice-vannucci model. Neuromethods 104, 1u201313 (2015)._ _ MacLellan, C. L. et al. Gauging recovery after hemorrhagic stroke in rats: implications for cytoprotection studies. J. Cereb. Blood Flow Metab. 26, 1031u20131042 (2006)._ _ _ MacLellan, C. L., Davies, L. M., Fingas, M. S. & Colbourne, F. The influence of hypothermia on outcome after intracerebral hemorrhage in rats. Stroke 37, 1266u20131270 (2006)._ _ _ Napper, R. M. A. Total number is important: using the disector method in design-based stereology to understand the structure of the rodent brain. Front. Neuroanat. 12, 16 (2018)._ _ _ _ _ MacLellan, C., Shuaib, A. & Colbourne, F. Failure of delayed and prolonged hypothermia to favorably affect hemorrhagic stroke in rats. Brain Res. 958, 192u2013200 (2002)._ _ _ _ Garman, R. H. Artifacts in routinely immersion fixed nervous tissue. Toxicol. Pathol. 18, 149u2013153 (1990)._ _ _ _ Piper, S. K. et al. Exact replication: foundation of science or game of chance?. PLOS Biol. 17, e3000188 (2019)._ _ _ _ _ Lakens, D. Calculating and reporting effect sizes to facilitate cumulative science: a practical primer for t-tests and ANOVAs. Front. Psychol. 4, 1u201312 (2013)._ _ \"- - We thank Ed Armstrong, Arlene Oatway, and Yonglie Ma for technical assistance. We specially thank Arlene Oatway for remote TEM imaging during the 2020 pandemic closures.- - \"This work was funded by the_Heart and Stroke Foundation of Canada (Grant_#G-16u2013 00014065 awarded to FC; Grant #G-19u20130026316 awarded to IRW), and the Canadian Institute for Health Research (Grant #166087 awarded to FC). FC is supported by a Canada Research Chair award. ACJK and CMW are each supported by a CIHR CGS-M, and ACJK was also supported by AIHS and the University of Alberta Undergraduate Research Initiative.\"- - Authors and Affiliations - \"Department of Psychology, Faculty of Science, University of Alberta, Edmonton, AB, CanadaAnna C. J. Kalisvaart,_Cassandra M. Wilkinson_&_Frederick ColbourneNeuroscience and Mental Health Institute, University of Alberta, Edmonton, CanadaSherry Gu,_ Tiffany F. C. Kung,_Ian R. Winship,_Frank K. H. van Landeghem_&_Frederick ColbourneDepartment of Pediatrics, Faculty of Medicine and Dentistry, University of Alberta, Edmonton, CanadaJerome YagerDepartment of Psychiatry, Faculty of Medicine and Dentistry, University of Alberta, Edmonton, CanadaIan R. WinshipDepartment of Laboratory Medicine and Pathology, University of Alberta Hospital, Edmonton, CanadaFrank K. H. van Landeghem\" - \"AuthorsAnna C. J. KalisvaartYou can also search for this author in _Cassandra M. WilkinsonYou can also search for this author in _Sherry GuYou can also search for this author in _Tiffany F. C. KungYou can also search for this author in _Jerome YagerYou can also search for this author in _Ian R. WinshipYou can also search for this author in _Frank K. H. van LandeghemYou can also search for this author in _Frederick ColbourneYou can also search for this author in _\" - Contributions - A.C.J.K., C.M.W., I.R.W., J.Y. and F.C. were involved in concept and experimental design. A.C.J.K., C.M.W., S.G., and T.K. collected the data. A.C.J.K., F.C., F.K.H.L., and C.M.W. were involved in data analysis. A.C.J.K., C.M.W., S.G., and T.K. drafted the manuscript, which all authors edited and approved. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons License Attribution-NonCommercial-ShareAlike-CC BY-NC-SA. The source code cannot be used for commercial purposes. Address all correspondence to Edward J. Delp, ace@ecn.purdue.edu. - \\' \\'- - \"Lucas, A. M. et al. Open-source deep-learning software for bioimage segmentation. Mol. Biol. Cell 32, 823u2013829 (2021)._ _ _ _ _ Piccinini, F. et al. Software tools for 3d nuclei segmentation and quantitative analysis in multicellular aggregates. Comput. Struct. Biotechnol. J. 18, 1287u20131300 (2020)._ _ _ _ _ Stringer, C., Wang, T., Michaelos, M. & Pachitariu, M. Cellpose: A generalist algorithm for cellular segmentation. Nat. Methods 18, 100u2013106 (2021)._ _ _ _ Greenwald, N. F. et al. Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning. Nat. Biotechnol. 40, 555u2013565 (2022)._ _ _ _ Kromp, F. et al. Evaluation of deep learning architectures for complex immunofluorescence nuclear image segmentation. IEEE Trans. Med. Imaging 40, 1934u20131949 (2021)._ _ _ Dunn, K. W. et al. Deepsynth: Three-dimensional nuclear segmentation of biological images using neural networks trained with synthetic data. Sci. Rep. 9, 18295u2013 18309 (2019)._ _ _ _ _ _ Fu, C. et_al. Three dimensional fluorescence microscopy image synthesis and segmentation. In Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition Workshops 2302u20132310 (2018). noteSalt Lake City, UT.Yushkevich, P. A. et al. User-guided 3D active contour segmentation of anatomical structures: Significantly improved efficiency and reliability. Neuroimage 31, 1116u20131128 (2006)._ _ _ Berger, D. R., Seung, H. S. & Lichtman, J. W. Vast (volume annotation and segmentation tool): Efficient manual and semi-automatic labeling of large 3d image stacks. Front. Neural Circ. 12, 88 (2018)._ _ Hollandi, R., DixF3sdi, xC1., Hollandi, G., Moshkov, N. & HorvxE1th, P. Annotatorj: An imagej plugin to ease hand annotation of cellular compartments. Mol. Biol. Cell 31, 2179u20132186 (2020)._ _ _ _ _ Borland, D. et al. Segmentor: A tool for manual refinement of 3d microscopy annotations. BMC Bioinform. 22, 1u2013 12 (2021)._ _ Wang, J. & Perez, L. The effectiveness of data augmentation in image classification using deep learning. (arXiv preprint) (2017).Mikou0142ajczyk, A. & Grochowski, M. Data augmentation for improving deep learning in image classification problem. Int. Interdiscip. PhD Works. 20, 117u2013122 (2018)._ Yang, L. et al. Nuset: A deep learning tool for reliably separating and analyzing crowded cells. PLoS Comput. Biol. 16, e1008193 (2020)._ _ _ _ _ Weigert, M., Schmidt, U., Haase, R., Sugawara, K. & Myers, G. Star-convex polyhedra for 3d object detection and segmentation in microscopy. In Proceedings of the IEEE/CVF Winter Conference on Applications of Computer Vision 3666u20133673 (2020).Sadanandan, S. K., Ranefall, P., Le Guyader, S. & WxE4hlby, C. Automated training of deep convolutional neural networks for cell segmentation. Sci. Rep. 7, 1u20137 (2017)._ _ _ Caicedo, J. C. et al. Evaluation of deep learning strategies for nucleus segmentation in fluorescence images. Cytometry A 95, 952u2013965 (2019)._ _ _ _ Baniukiewicz, P., Lutton, E. J., Collier, S. & Bretschneider, T. Generative adversarial networks for augmenting training data of microscopic cell images. Front. Comput. Sci. 10, 25 (2019)._ Goodfellow, I. J. et al. Generative adversarial networks. IEEE Signal Process. Mag. 1406, 2661 (2014)._ Liu, L. et al. Deep learning for generic object detection: A survey. Int. J. Comput. Vision 128, 261u2013318 (2020)._ _ _ Carneiro, G., Zheng, Y., Xing, F. & Yang, L. Review of deep learning methods in mammography, cardiovascular, and microscopy image analysis. Deep Learn. Convolut. Neural Netw. Med. Image Comput. 20, 11u201332 (2017)._ _ Ronneberger, O., Fischer, P. & Brox, T. U-net: Convolutional networks for biomedical image segmentation. Med. Image Comput. Comput. Assist. Intervention 9351, 231u2013 241 (2015)._ xC7ixE7ek, xD6., Abdulkadir, A., Lienkamp, S., Brox, T. & Ronneberger, O. 3D u-net: Learning dense volumetric segmentation from sparse annotation. Med. Image Comput. Comput. Assist. Intervention 9901, 424u2013432 (2016)._ Milletari, F., Navab, N. & Ahmadi, S. V-net: Fully convolutional neural networks for volumetric medical image segmentation. In International Conference on 3D Vision 565u2013571 (2016).Balakrishnan, G., Zhao, A., Sabuncu, M. R., Guttag, J. & Dalca, A. V. Voxelmorph: A learning framework for deformable medical image registration. IEEE Trans. Med. Imaging 38, 1788u2013 1800 (2019)._ _ Graham, S. et al. Hover-net: Simultaneous segmentation and classification of nuclei in multi-tissue histology images. Med. Image Anal. 58, 101563 (2019)._ _ _ Fu, C. et_al. Nuclei segmentation of fluorescence microscopy images using convolutional neural networks. In Proceedings of the IEEE International Symposium on Biomedical Imaging 704u2013708 (2017).Ho, D._J., Fu, C., Salama, P., Dunn, K._W. & Delp, E._J. Nuclei segmentation of fluorescence microscopy images using three dimensional convolutional neural networks. In IEEE Conference on Computer Vision and Pattern Recognition Workshops 834u2013842 (2017).Schmidt, U., Weigert, M., Broaddus, C. & Myers, G. Cell detection with star-convex polygons. In Medical Image Computing and Computer Assisted Intervention 265u2013273 (2018).Ho, D. J. et al. Sphere estimation network: Three-dimensional nuclei detection of fluorescence microscopy images. J. Med. Imaging 7, 1u201316 (2020)._ _ Lux, F. & Matula, P. Dic image segmentation of dense cell populations by combining deep learning and watershed. In IEEE International Symposium on Biomedical Imaging 236u2013239 (2019).Arbelle, A., Cohen, S. & Raviv, T. R. Dual-task convlstm-unet for instance segmentation of weakly annotated microscopy videos. IEEE Trans. Med. Imaging 41, 1948u20131960. (2022)._ _ Bao, R., Al-Shakarji, N._M., Bunyak, F. & Palaniappan, K. Dmnet: Dual-stream marker guided deep network for dense cell segmentation and lineage tracking. In IEEE International Conference on Computer Vision Workshops 3354u20133363 (2021).Scherr, T., LxF6ffler, K., BxF6hland, M. & Mikut, R. Cell segmentation and tracking using cnn-based distance predictions and a graph-based matching strategy. PLoS One 15, e0243219 (2020)._ _ _ _ _ Mandal, S. & Uhlmann, V. Splinedist: Automated cell segmentation with spline curves. In Proceedings of the International Symposium on Biomedical Imaging 1082u20131086 (2021).Ruiz-Santaquiteria, J., Bueno, G., Deniz, O., Vallez, N. & Cristobal, G. Semantic versus instance segmentation in microscopic algae detection. Eng. Appl. Artif. Intell. 87, 103271 (2020)._ _ Hama, H. et al. Scale: A chemical approach for fluorescence imaging and reconstruction of transparent mouse brain. Nat. Neurosci. 14, 1481u20131488 (2011)._ _ _ _ Clendenon, S. G., Young, P. A., Ferkowicz, M., Phillips, C. & Dunn, K. W. Deep tissue fluorescent imaging in scattering specimens using confocal microscopy. Microsc. Microanal. 17, 614u2013 617 (2011)._ _ _ _ _ _ Chen, A. et_al. 3d ground truth annotations of nuclei in 3d microscopy volumes. bioRxiv (2022).Simard, P._Y., Steinkraus, D. & Platt, J._C. Best practices for convolutional neural networks applied to visual document analysis. In Proceedings of the International Conference on Document Analysis and Recognition 958u2013963 (2003).Chen, A. et_al. Three dimensional synthetic non-ellipsoidal nuclei volume generation using Bezier curves. In Proceedings of the IEEE International Symposium on Biomedical Imaging (2021).Wu, L. et_al. Rcnn-slicenet: A slice and cluster approach for nuclei centroid detection in three-dimensional fluorescence microscopy images. InProceedings of the IEEE Conference on Computer Vision and Pattern Recognition Workshops 3750u20133760 (2021).Wu, L., Chen, A., Salama, P., Dunn, K._W. & Delp, E._J. An ensemble learning and slice fusion strategy for three-dimensional nuclei instance segmentation. In Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition Workshops (2022).Zhu, J., Park, T., Isola, P. & Efros, A._A. Unpaired image-to-image translation using cycle-consistent adversarial networks. In Proceedings of the IEEE International Conference on Computer Vision 2242u20132251 (2017).Isensee, F., Jaeger, P. F., Kohl, S. A. A., Petersen, J. & Maier-Hein, K. H. nnu-net: A self-configuring method for deep learning-based biomedical image segmentation. Nat. Methods 18, 203u2013211. (2021)._ _ _ _ Yang, X., Li, H. & Zhou, X. Nuclei segmentation using marker-controlled watershed, tracking using mean-shift, and Kalman filter in time-lapse microscopy. IEEE Trans. Circ. Syst. I Regul. Pap. 53, 2405u2013 2414 (2006)._ _ Rizk, A. et al. Segmentation and quantification of subcellular structures in fluorescence microscopy images using squassh. Nat. Protoc. 9, 586u2013596 (2014)._ _ _ _ McQuin, C. et al. Cellprofiler 3.0: Next-generation image processing for biology. PLoS Biol. 16, e2005970-1u201317 (2018)._ _ Hirsch, P. & Kainmueller, D. An auxiliary task for learning nuclei segmentation in 3d microscopy images. In booktitleProceedings of the Third Conference on Medical Imaging with Deep Learning, vol. 121 of Proceedings of Machine Learning Research (Arbel, T. et_al. eds) 304u2013321 (2020).Englbrecht, F., Ruider, I. E. & Bausch, A. R. Automatic image annotation for fluorescent cell nuclei segmentation. PLoS One 16, 1u201313. (2021)._ _ _ Lee, M. Y. et al. Cellseg: A robust, pre-trained nucleus segmentation and pixel quantification software for highly multiplexed fluorescence images. BMC Bioinform. 23, 46. (2022)._ _ Cutler, K. J. et al. Omnipose: A high-precision morphology-independent solution for bacterial cell segmentation. Nat. Methods 19, 1438u20131448. (2022)._ _ _ _ _ Mougeot, G. et al. Deep learningu2014promises for 3D nuclear imaging: a guide for biologists. J. Cell Sci. (2022)._ _ _ _ Winfree, S. et al. Quantitative three-dimensional tissue cytometry to study kidney tissue and resident immune cells. J. Am. Soc. Nephrol. 28, 2108u2013 2118 (2017)._ _ _ _ _ Kumar, N. et al. A dataset and a technique for generalized nuclear segmentation for computational pathology. IEEE Trans. Med. Imaging 36, 1550u20131560 (2017)._ _ _ Hosang, J., Benenson, R., DollxE1r, P. & Schiele, B. What makes for effective detection proposals?. IEEE Trans. Pattern Anal. Mach. Intell. 38, 814u2013830 (2016)._ _ _ Shen, X. & Stamos, I. 3d object detection and instance segmentation from 3d range and 2d color images. Sensors (2021)._ _ _ _ Basu, A., Senapati, P., Deb, M., Rai, R. & Dhal, K. G. A survey on recent trends in deep learning for nucleus segmentation from histopathology images. Evol. Syst. (2023)._ _ SchxFCrch, C. M. et al. Coordinated cellular neighborhoods orchestrate antitumoral immunity at the colorectal cancer invasive front. Cell 182, 1341-1359.e19. (2020)._ _ _ _ _ Caicedo, J. C. et al. Nucleus segmentation across imaging experiments: The 2018 data science bowl. Nat. Methods 16, 1247u20131253. (2019)._ _ _ _ _ Everingham, M. et al. The pascal visual object classes challenge: A retrospective. Int. J. Comput. Vision 111, 98u2013136 (2015)._ _ Padilla, R., Netto, S._L. & da_Silva, E. A._B. A survey on performance metrics for object-detection algorithms. In Proceedings of the International Conference on Systems, Signals and Image Processing 237u2013242 (2020).Davis, J. & Goadrich, M. The relationship between precision-recall and roc curves. In Proceedings of the international conference on Machine learning 233u2013240 (2006).Winfree, S. User-accessible machine learning approaches for cell segmentation and analysis in tissue. Front. Physiol. (2022)._ _ _ _ He, K., Zhang, X., Ren, S. & Sun, J. Deep residual learning for image recognition. In IEEE Conference on Computer Vision and Pattern Recognition 770u2013778 (2016).Oktay, O. et_al. Attention u-net: Learning where to look for the pancreas. (arXiv preprint) (2018).Han, S. et_al. Nuclei counting in microscopy images with three dimensional generative adversarial networks. In Proceedings of the SPIE Conference on Medical Imaging10949, 753 u2013 763 (2019).Zhang, D. et_al. Nuclei instance segmentation with dual contour-enhanced adversarial network. In Proceedings of the IEEE International Symposium on Biomedical Imaging 409u2013412 (2018).Lin, T., Goyal, P., Girshick, R., He, K. & DollxE1r, P. Focal loss for dense object detection. In Proceedings of the IEEE International Conference on Computer Vision 2999u20133007 (2017).Salehi, M., Sadegh, S., Erdogmus, D. & Gholipour, A. Tversky loss function for image segmentation using 3d fully convolutional deep networks. IN Proceedings of International Workshop on Machine Learning in Medical Imaging 379u2013387 (2017).Soille, P. & Vincent, L._M. Determining watersheds in digital pictures via flooding simulations. Visual Communications and Image Processing90: Fifth in a Series1360, 240u2013250 (1990).Di_Stefano, L. & Bulgarelli, A. A simple and efficient connected components labeling algorithm. In Proceedings of the International Conference on Image Analysis and Processing 322u2013327 (1999).Svoboda, D., Kozubek, M. & Stejskal, S. Generation of digital phantoms of cell nuclei and simulation of image formation in 3d image cytometry. Cytometry Part A J. Int. Soc. Adv. Cytometry 75, 494u2013509 (2009)._ _ McKinley, S. & Levine, M. Cubic spline interpolation. Coll. Redwoods 45, 1049u20131060 (1998)._ Santella, A., Du, Z., Nowotschin, S., Hadjantonakis, A. K. & Bao, Z. A hybrid blob-slice model for accurate and efficient detection of fluorescence labeled nuclei in 3d. BMC Bioinform. 11, 1u201313 (2010)._ _ Isola, P., Zhu, J., Zhou, T. & Efros, A._A. Image-to-image translation with conditional adversarial networks. In Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition 5967u20135976 (2017).Kingma, D._P. & Ba, J. Adam: A method for stochastic optimization. (arXiv preprint) (2017).Wang, J. et al. 3D GAN image synthesis and dataset quality assessment for bacterial biofilm. Bioinformatics 38, 4598u20134604. (2022)._ _ _ _ _ Lin, T., Goyal, P., Girshick, R., He, K. & DollxE1r, P. Focal loss for dense object detection. In IEEE International Conference on Computer Vision 2999u20133007 (2017).Rahman, M._A. & Wang, Y. Optimizing intersection-over-union in deep neural networks for image segmentation. In International Symposium on Visual Computing 234u2013244 (2016).Rezatofighi, H. et_al. Generalized intersection over union: A metric and a loss for bounding box regression. In Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition 658u2013666 (2019).Powers, D._M. Evaluation: from precision, recall and f-measure to roc, informedness, markedness and correlation. (arXiv preprint) (2020).Lin, Z. et_al. Nucmm dataset: 3d neuronal nuclei instance segmentation at sub-cubic millimeter scale. In International Conference on Medical Image Computing and Computer-Assisted Intervention 164u2013174 (2021).\"- - \"This research was partially supported by a George M. OBrien Award from the National Institutes of Health under Grant NIH/NIDDK P30 DK079312 and the endowment of the Charles William Harrison Distinguished Professorship at Purdue University. The original image volumes used in this paper were provided by Malgorzata Kamocka, Sherry Clendenon, and Michael Ferkowicz at Indiana University. We gratefully acknowledge their cooperation.\"- - Authors and Affiliations - \"Video and Image Processing Laboratory, School of Electrical and Computer Engineering, Purdue University, West_Lafayette, IN, 47907, USALiming Wu,_Alain Chen_&_ Edward J. DelpDepartment of Electrical and Computer Engineering, Indiana University-Purdue University Indianapolis, Indianapolis, IN, 46202, USAPaul SalamaDepartment of Pathology and Microbiology, University of Nebraska Medical Center, Omaha, NE, 68198, USASeth WinfreeSchool of Medicine, Indiana University, Indianapolis, IN, 46202, USAKenneth W. Dunn\" - \"AuthorsLiming WuYou can also search for this author in _Alain ChenYou can also search for this author in _Paul SalamaYou can also search for this author in _Seth WinfreeYou can also search for this author in _Kenneth W. DunnYou can also search for this author in _Edward J. DelpYou can also search for this author in _\" - Contributions - L.W., E.D., and P.S. conceived and designed NISNet3D. L.W. and A.C. designed the convolutional network and developed the segmentation approach, generated the synthetic data, conducted the segmentations, generated ground truth data, and conducted the comparisons with alternative methods. L.W., P.S., A.C., and E.D. wrote the manuscript. L.W., A.C., P.S., S.W., K.D., and E.D. revised the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons License, 2002. Lecture 10: Batteries, EMF, Energy Conservation, Power, Kirchhoffs Rules, Circuits, Kelvin Water Dropper (Center for Future Civic Media, Massachusetts Institute of Technology, MIT, 2008).Marin, A. G. et al. The microfluidic Kelvin water dropper. Lab Chip 13, 4503u20134506 (2013)._ _ _ _ Marx, V. Pouring over liquid handling. Nat. Methods 11, 33 (2013)._ _ _ Choi D. et al. Spontaneous electrical charging of droplets by conventional pipetting. Sci. Rep. 3, (2013).Choi, D. et al. A smart pipet tip: triboelectricity and thermoelectricity assisted in situ evaluation of electrolyte concentration. Nano Energy 38, 419u2013427 (2017)._ _ _ Wu, C. et al. Nanogenerator: a foundation of the energy for the new era. Adv. Energy Mater. 9, 1802906 (2019)._ _ _ McCarty, L. S. & Whitesides, G. M. Electrostatic charging due to separation of ions at interfaces: Contact electrification of ionic electrets. Angew. Chem. Int. Ed. 47, 2188u2013 2207 (2008)._ _ _ Xu, W. et al. A droplet-based electricity generator with high instantaneous power density. Nature 578, 392u2013 396 (2020)._ _ _ _ _ Faubel, M. & Steiner, B. Strong bipolar electrokinetic charging of thin liquid jets emerging from 10 u03BCm PtIr nozzles. Ber. der Bunsenges. f.xFCr. physikalische Chem. 96, 1167u20131172 (1992)._ _ _ Schwierz, N. et al. Hydrogen and electric power generation from liquid microjets: design principles for optimizing conversion efficiency. J. Phys. Chem. C. 120, 14513u201314521 (2016)._ _ _ Sarver, T., Al-Qaraghuli, A. & Kazmerski, L. L. A comprehensive review of the impact of dust on the use of solar energy: history, investigations, results, literature, and mitigation approaches. Renew. Sustain. Energy Rev. 22, 698u2013733 (2013)._ _ Pratt, T. H. Electrostatic ignitions of fires and explosions (John Wiley & Sons, 2010).Wilkinson, M. Large deviation analysis of rapid onset of rain showers. Phys. Rev. Lett. 116, 018501 (2016)._ _ _ _ _ _ Shavlov, A. V. & Zdzhumandzhi, V. A. Metastable states and coalescence of charged water drops inside clouds and fog. J. Aerosol Sci. 91, 54u201361 (2016)._ _ _ _ Vemulapalli, G. K. & Kukolich, S. G. Why does a stream of water deflect in an electric field? J. Chem. Educ. 73, 887u2013888 (1996)._ _ _ Ziaei-Moayyed, M., Goodman, E. & Williams, P. Electrical deflection of polar liquid streams: a misunderstood demonstration. J. Chem. Educ. 77, 1520u20131524 (2000)._ _ _ Burgo, T. A. L., Galembeck, F. & Pollack, G. H. Where is water in the triboelectric series? J. Electrost. 80, 30u201333 (2016)._ _ Matyushov, D. V. Electrophoretic mobility without charge driven by polarisation of the nanoparticleu2013water interface. Mol. Phys. 112, 2029u2013 2039 (2014)._ _ _ _ Mishra, H. et al. Bronsted basicity of the air-water interface. Proc. Natl Acad. Sci. USA 109, 18679u2013 18683 (2012)._ _ _ _ _ _ Saykally, R. J. Air/water interface: two sides of the acid-base story. Nat. Chem. 5, 82u2013 84 (2013)._ _ _ _ Hauner, I. M., Deblais, A., Beattie, J. K., Kellay, H. & Bonn, D. The dynamic surface tension of water. J. Phys. Chem. Lett. 8, 1599u20131603 (2017)._ _ _ _ _ Mishra, H. et al. Anions dramatically enhance proton transfer through aqueous interfaces. Proc. Natl Acad. Sci. USA 109, 10228u201310232 (2012)._ _ _ _ _ _ Mishra, H. et al. Quantum chemical insights into the dissociation of nitric acid on the surface of aqueous electrolytes. Int. J. Quantum Chem. 113, 413u2013417 (2013)._ _ _ Agmon, N. et al. Protons and hydroxide ions in aqueous systems. Chem. Rev. 116, 7642u20137672 (2016)._ _ _ _ _ Galembeck, F. et al. Friction, tribochemistry and triboelectricity: recent progress and perspectives. RSC Adv. 4, 64280u2013 64298 (2014)._ _ _ Lach, S., Yoon, S. M. & Grzybowski, B. A. Tactic, reactive, and functional droplets outside of equilibrium. Chem. Soc. Rev. 45, 4766u20134796 (2016)._ _ _ _ Gallo, A. et al. The chemical reactions in electrosprays of water do not always correspond to those at the pristine airu2013water interface. Chem. Sci. 10, 2566u20132577 (2019)._ _ _ _ Colussi A. J. & Enami S. Comment on u201CThe chemical reactions in electrosprays of water do not always correspond to those at the pristine airu2013water interfaceu201D by A. Gallo Jr, A. S. F. Farinha, M. Dinis, A.-H. Emwas, A. Santana, R. J. Nielsen, W. A. Goddard III and H. Mishra, Chem. Sci. (2019).Gallo A. et al. Reply to the u2018Comment on u201CThe chemical reactions in electrosprays of water do not always correspond to those at the pristine airu2013water interfaceu201D by A. J. Colussi and S. Enami, Chem. Sci., 2019, 10, Chem. Sci., (2019).Carpenter, A. P., Tran, E., Altman, R. M. & Richmond, G. L. Formation and surface-stabilizing contributions to bare nanoemulsions created with negligible surface charge. Proc. Natl Acad. Sci. USA 116, 9214u20139219 (2019)._ _ _ _ _ Pullanchery, S., Kulik, S., Okur, H. I., de Aguiar, H. B. & Roke, S. On the stability and necessary electrophoretic mobility of bare oil nanodroplets in water. J. Chem. Phys. 152, 241104 (2020)._ _ _ _ _ Enami, S., Mishra, H., Hoffmann, M. R. & Colussi, A. J. Protonation and oligomerization of gaseous isoprene on mildly acidic surfaces: implications for atmospheric chemistry. J. Phys. Chem. A 116, 6027u20136032 (2012)._ _ _ _ Lacks, D. J. & Mohan Sankaran, R. Contact electrification of insulating materials. J. Phys. D Appl. Phys. 44, 453001 (2011)._ _ _ _ Karraker, K. A. & Radke, C. J. Disjoining pressures, zeta potentials and surface tensions of aqueous non-ionic surfactant/electrolyte solutions: theory and comparison to experiment. Adv. Colloid Interface Sci. 96, 231u2013264 (2002)._ _ _ _ Zangi, R. & Engberts, J. B. F. N. Physisorption of hydroxide ions from aqueous solution to a hydrophobic surface. J. Am. Chem. Soc. 127, 2272u20132276 (2005)._ _ _ _ Beattie, J. K. The intrinsic charge on hydrophobic microfluidic substrates. Lab Chip 6, 1409u20131411 (2006)._ _ _ _ Beattie, J. K. Comment on autoionization at the surface of neat water: is the top layer pH neutral, basic, or acidic? by R. Vacha, V. Buch, A. Milet, J. P. Devlin and P. Jungwirth, Phys. Chem. Chem. Phys., 2007, 9, 4736. Phys. Chem. Chem. Phys. 10, 330u2013331 (2008)._ _ _ _ Beattie J. K., Djerdjev A. M. & Warr G. G. The surface of neat water is basic. 61, 31-39 (2009).Creux, P., Lachaise, J., Graciaa, A., Beattie, J. K. & Djerdjev, A. M. Strong specific hydroxide ion binding at the pristine oil/water and air/water interfaces. J. Phys. Chem. B 113, 14146u201314150 (2009)._ _ _ _ Gray-Weale, A. & Beattie, J. K. An explanation for the charge on waters surface. Phys. Chem. Chem. Phys. 11, 10994u2013 11005 (2009)._ _ _ _ Mundy, C. J., Kuo, I. F. W., Tuckerman, M. E., Lee, H. S. & Tobias, D. J. Hydroxide anion at the air-water interface. Chem. Phys. Lett. 481, 2u20138 (2009)._ _ _ _ Tian, C. S. & Shen, Y. R. Structure and charging of hydrophobic material/water interfaces studied by phase-sensitive sum-frequency vibrational spectroscopy. Proc. Natl Acad. Sci. USA 106, 15148u201315153 (2009)._ _ _ _ _ _ Marinova, K. G. et al. Charging of oilu2212water interfaces due to spontaneous adsorption of hydroxyl ions. Langmuir 12, 2045u20132051 (1996)._ _ _ Healy, T. & Fuerstenau, D. The isoelectric point/point-of zero-charge of interfaces formed by aqueous solutions and nonpolar solids, liquids, and gases. J. Colloid Interface Sci. 309, 183u2013188 (2007)._ _ _ _ _ Petersen, M. K., Iyengar, S. S., Day, T. J. F. & Voth, G. A. The hydrated proton at the water liquid/vapor interface. J. Phys. Chem. B 108, 14804u201314806 (2004)._ _ _ Iuchi, S., Chen, H. N., Paesani, F. & Voth, G. A. Hydrated excess proton at water-hydrophobic interfaces. J. Phys. Chem. B 113, 4017u20134030 (2009)._ _ _ _ Kathmann, S. M., Kuo, I. F. W., Mundy, C. J. & Schenter, G. K. Understanding the surface potential of water. J. Phys. Chem. B 115, 4369u2013 4377 (2011)._ _ _ _ Petersen, P. B. & Saykally, R. J. Evidence for an enhanced hydronium concentration at the liquid water surface. J. Phys. Chem. B 109, 7976u20137980 (2005)._ _ _ _ Zhang, C. et al. Water at hydrophobic interfaces delays proton surface-to-bulk transfer and provides a pathway for lateral proton diffusion. Proc. Natl Acad. Sci. USA 109, 9744u20139749 (2012)._ _ _ _ _ _ Yamaguchi, S., Kundu, A., Sen, P. & Tahara, T. Communication: quantitative estimate of the water surface pH using heterodyne-detected electronic sum frequency generation. J. Chem. Phys. 137, 151101 (2012)._ _ _ _ _ Rizzuto, A. M., Cheng, E. S., Lam, R. K. & Saykally, R. J. Surprising effects of hydrochloric acid on the water evaporation coefficient observed by Raman thermometry. J. Phys. Chem. C. 121, 4420u2013 4425 (2017)._ _ _ Smolentsev, N. & Roke, S. Self-assembly at water nanodroplet interfaces quantified with nonlinear light scattering. Langmuir 36, 9317u20139322 (2020)._ _ _ _ VxE1cha, R. et al. The orientation and charge of water at the hydrophobic oil dropletu2013water interface. J. Am. Chem. Soc. 133, 10204u201310210 (2011)._ _ _ _ Vacha, R. et al. Charge transfer between water molecules as the possible origin of the observed charging at the surface of pure water. J. Phys. Chem. Lett. 3, 107u2013111 (2012)._ _ _ Poli, E., Jong, K. H. & Hassanali, A. Charge transfer as a ubiquitous mechanism in determining the negative charge at hydrophobic interfaces. Nat. Commun. 11, 901 (2020)._ _ _ _ _ _ Yan, X. B. et al. Central role of bicarbonate anions in charging water/hydrophobic interfaces. J. Phys. Chem. Lett. 9, 96u2013103 (2018)._ _ _ _ _ Harper, W. R. Liquids giving no electrification by bubbling. Br. J. Appl. Phys. 4, S19u2013S22 (1953)._ _ _ Roger, K. & Cabane, B. Why are hydrophobic/water interfaces negatively charged? Angew. Chem. Int. Ed. 51, 5625u20135628 (2012)._ _ _ Jena, K. C., Scheu, R. & Roke, S. Surface impurities are not responsible for the charge on the oil/water interface: a comment. Angew. Chem. Int. Ed. 51, 12938u201312940 (2012)._ _ _ Beattie, J. K. & Gray-Weale, A. Oil/water interface charged by hydroxide ions and deprotonated fatty acids: a comment. Angew. Chem. Int. Ed. 51, 12941u201312942 (2012)._ _ _ Roger, K. & Cabane, B. Uncontaminated hydrophobic/water interfaces are uncharged: a reply. Angew. Chem. Int. Ed. 51, 12943u201312945 (2012)._ _ _ Uematsu, Y., Bonthuis, D. J. & Netz, R. R. Charged surface-active impurities at nanomolar concentration induce Jones-Ray effect. J. Phys. Chem. Lett. 9, 189u2013193 (2018)._ _ _ _ Cole, J. J., Barry, C. R., Knuesel, R. J., Wang, X. & Jacobs, H. O. Nanocontact electrification: patterned surface charges affecting adhesion, transfer, and printing. Langmuir 27, 7321u20137329 (2011)._ _ _ _ Diaz, A. F. & Felix-Navarro, R. M. A semi-quantitative tribo-electric series for polymeric materials: the influence of chemical structure and properties. J. Electrost. 62, 277u2013290 (2004)._ _ _ Horn, R. G., Smith, D. T. & Grabbe, A. Contact electrification induced by monolayer modification of a surface and relation to acidu2013base interactions. Nature 366, 442u2013443 (1993)._ _ _ _ Lowell, J. & Truscott, W. S. Triboelectrification of Identical Insulators .2. Theory and further experiments. J. Phys. D. Appl. Phys. 19, 1281u2013 1298 (1986)._ _ _ _ Liu, C. & Bard, A. J. Electrostatic electrochemistry at insulators. Nat. Mater. 7, 505 (2008)._ _ _ _ _ Liu, C. Y. & Bard, A. J. Chemical redox reactions induced by cryptoelectrons on a PMMA surface. J. Am. Chem. Soc. 131, 6397u20136401 (2009)._ _ _ _ Baytekin, B., Baytekin, H. T. & Grzybowski, B. A. What really drives chemical reactions on contact charged surfaces? J. Am. Chem. Soc. 134, 7223u20137226 (2012)._ _ _ _ Piperno, S., Cohen, H., Bendikov, T., Lahav, M. & Lubomirsky, I. The absence of redox reactions for palladium(II) and copper(II) on electrostatically charged teflon: relevance to the concept of u201Ccryptoelectronsu201D. Angew. Chem. Int. Ed. 50, 5654u20135657 (2011)._ _ _ Israelachvili, J. N. Intermolecular and Surface Forces (Third Edition). (Academic Press, Elsevier, 2011).Keithley Instruments I. Low Level Measurement Handbook. Precision DC Current, Voltage, and Resistance Measurements (2004).Duffin, A. M. & Saykally, R. J. Electrokinetic hydrogen generation from liquid water microjets. J. Phys. Chem. C. 111, 12031u201312037 (2007)._ _ _ Enami, S., Mishra, H., Hoffmann, M. R. & Colussi, A. J. Hofmeister effects in micromolar electrolyte solutions. J. Chem. Phys. 136, 154707 (2012)._ _ _ _ _ Beattie, J. K. et al. pH and the surface tension of water. J. Colloid Interface Sci. 422, 54u201357 (2014)._ _ _ _ _ Mazzini, V. & Craig, V. S. J. What is the fundamental ion-specific series for anions and cations? Ion specificity in standard partial molar volumes of electrolytes and electrostriction in water and non-aqueous solvents. Chem. Sci. 8, 7052u20137065 (2017)._ _ _ _ _ Burgo, T. A. L. et al. Triboelectricity: macroscopic charge patterns formed by self-arraying ions on polymer surfaces. Langmuir 28, 7407u20137416 (2012)._ _ _ _ Strom, G., Fredriksson, M. & Stenius, P. Contact angles, work of adhesion, and interfacial-tensions at a dissolving hydrocarbon surface. J. Colloid Interface Sci. 119, 352u2013361 (1987)._ _ _ Lide D. R. CRC Handbook of Chemistry and Physics, 84th Edition. (Taylor & Francis, 2003).Li, S. et al. Contributions of different functional groups to contact electrification of polymers. Adv. Mater. 32, 2001307 (2020).Vigil, G., Xu, Z. H., Steinberg, S. & Israelachvili, J. Interactions of silica surfaces. J. Colloid Interface Sci. 165, 367u2013385 (1994)._ _ _ _ Yutkin, M. P., Mishra, H., Patzek, T. W., Lee, J. & Radke, C. J. Bulk and surface aqueous speciation of calcite: implications for low-salinity waterflooding of carbonate reservoirs. SPEJ. 23, 84u2013101 (2018)._ _ _ Atkins P., de Paula J. Atkins Physical Chemistry (Eight Edition). (Oxford University Press, 2006).Lam, R. K. et al. Reversed interfacial fractionation of carbonate and bicarbonate evidenced by X-ray photoemission spectroscopy. J. Chem. Phys. 146, 094703 (2017)._ _ _ _ Jubb, A. M., Hua, W. & Allen, H. C. Organization of water and atmospherically relevant ions and solutes: vibrational sum frequency spectroscopy at the vapor/liquid and liquid/solid interfaces. Acc. Chem. Res. 45, 110u2013119 (2012)._ _ _ _ Mishra, H. et al. Time-dependent wetting behavior of PDMS surfaces with bioinspired, hierarchical structures. ACS Appl. Mater. Interfaces 8, 8168u20138174 (2016)._ _ _ _ Domingues, E. M., Arunachalam, S., Nauruzbayeva, J. & Mishra, H. Biomimetic coating-free surfaces for long-term entrapment of air under wetting liquids. Nat. Commun. 9, 3606 (2018)._ _ _ _ _ _ Wong, W. S. Y. et al. Microdroplet contaminants: when and why superamphiphobic surfaces are not self-cleaning. ACS Nano 14, 3836u2013 3846 (2020)._ _ _ _ _ Lowell, J. & Akande, A. R. Contact electrification-why is it variable? J. Phys. D: Appl. Phys. 21, 125u2013137 (1988)._ _ _ _ Meagher, L. & Craig, V. S. J. Effect of dissolved gas and salt on the hydrophobic force between polypropylene surfaces. Langmuir 10, 2736u20132742 (1994)._ _ _ Arunachalam, S., Ahmad, Z., Das, R. & Mishra, H. Counterintuitive wetting transitions in doubly reentrant cavities as a function of surface make-up, hydrostatic pressure, and cavity aspect ratio. Adv. Mater. Interfaces (2020).Subramanian, N. et al. Evaluating the potential of superhydrophobic nanoporous alumina membranes for direct contact membrane distillation. J. Colloid Interface Sci. 533, 723u2013 732 (2019)._ _ _ _ _ Pillai, S. et al. A molecular to macro level assessment of direct contact membrane distillation for separating organics from water. J. Membr. Sci. 608, 118140 (2020)._ _ _ Das, R., Ahmad, Z., Nauruzbayeva, J. & Mishra, H. Biomimetic coating-free superomniphobicity. Sci. Rep. 10, 7934 (2020)._ _ _ _ _ _ Acharya, G. et al. The hydrogel template method for fabrication of homogeneous nano/microparticles. J. Controlled Release 141, 314u2013319 (2010)._ _ _ Millipore, M. Milli-QxAE Advantage A10xAE Water Purification Systems. User-adapted ultrapure water. (EMD Millipore Corporation, Billerica, MA, U.S.A., 2013).Shrestha, B. R. et al. Nuclear quantum effects in hydrophobic nanoconfinement J. Phys. Chem. Lett. 10, 5530u20135535 (2019)._ _ _ _ Domingues, E. M., Arunachalam, S. & Mishra, H. Doubly reentrant cavities prevent catastrophic wetting transitions on intrinsically wetting surfaces. ACS Appl. Mater. Interfaces 9, 21532u201321538 (2017)._ _ _ _ \"- - \"H.M. and J.C.S. acknowledge funding from KAUST. J.N. and H.M. thank Mrs. Emilie Dauzon (KAUST), who initiated this research through investigating electrification at solid-solid interfaces in HMs Group as a visiting student in the year 2015. The co-authors thank Mr. Farizal Hakiki for performing permittivity measurements used in Fig._, and Dr. Michael Cusack (KAUST) for scientific editing. The authors thank Mr. Ivan Gromicho, Scientific Illustrator at KAUST, for preparing Fig._ .\"- - Author notes - \\'These authors contributed equally: Jamilya Nauruzbayeva, Zhonghao Sun, Adair Gallo Jr.\\' - Authors and Affiliations - \"King Abdullah University of Science and Technology (KAUST), Water Desalination and Reuse Center (WDRC), Division of Biological and Environmental Sciences and Engineering, Thuwal, 23955 - 6900, Saudi ArabiaJamilya Nauruzbayeva,_Adair Gallo Jr.,_Mahmoud Ibrahim_&_Himanshu MishraKing Abdullah University of Science and Technology, Ali I. Al-Naimi Petroleum Engineering Research Center (ANPERC), Division of Physical Science and Engineering, Thuwal, 23955 - 6900, Saudi ArabiaZhonghao Sun_&_J. Carlos Santamarina\" - \"AuthorsJamilya NauruzbayevaYou can also search for this author in _Zhonghao SunYou can also search for this author in _Adair Gallo Jr.You can also search for this author in _Mahmoud IbrahimYou can also search for this author in _J. Carlos SantamarinaYou can also search for this author in _Himanshu MishraYou can also search for this author in _\" - Contributions - \"H.M. conceived the project, co-supervised the research with JCS, and coordinated the collaboration. H.M. and JCS conceived experiments detailed in Fig. 2A; J.N. and Z.S. planned the experiments reported in Fig. 2B; H.M. conceived the experiment reported in Fig. 2C; Z.S. and J.N. conceived the experiment detailed in Fig. 4; H.M. and A.G.J. conceived the experiment detailed in Fig. 5B. Z.S. and J.N. performed the experiments described in Figs. 2u20133 and analyzed the data; J.N. performed the experiments reported in Figs. 4u20135 and analyzed the data. M.I. and J.N. characterized purity of hexadecane using porous alumina membranes and interfacial tension measurements. H.M. derived the scaling relationships to explain Fig. 3; H.M., J.N., Z.S., J.C.S. and A.G.J. analyzed the data; H.M. and A.G.J. pieced together the results to realize the complete picture. H.M. wrote the paper and the co-authors edited it.\" - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks Vincent Craig, Manfred Heuberger and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons License. b Instantaneous color-changing of the EL-integrated display in response to the background light variation. Blue, green, and orange light-emitting EL units were printed on the display. A 200-V AC power voltage at a frequency of 1000u2009Hz was set. Scale bar: 1u2009cm. c Schematic illustrating the chameleon-inspired background-matching strategy of the ELbot. The chameleon sketches are adapted from Freepick under the CC0 Creative Commons License. d Instantaneous color-changing of the ELbot in response to the background light variation. Scale bar: 2u2009cm.\" - Discussion - In summary, we present a streamlined approach for fabricating mechanically compliant and flexible EL devices through developing high-performance 3D printable EL inks. Enabled by the good interfacial robustness of the 3DP EL devices, the 3DP EL devices exhibited stable electroluminescent performance even under mechanical deformation. Through integrating the 3D printed EL devices with a soft robot, a chameleon-inspired artificial camouflage was designed and fabricated to demonstrate the potential of our 3DP EL devices in automated background-matching applications. - Differing from previous artificial camouflages made of shape-memory polymer or stimuli-responsive polymer, of which the response is slow and the reversibility is poor upon external stimuli, our ELbot is able to attain perceivably instant response and reversible color transformation with comparable timescale to physiological response through actively controlling the AC voltage supply in harmony with the external background color. This instant matching capability is ideal for camouflage applications. It should be pointed out that the above demonstration shows the ability of our ELbot in response to a specific monochromatic background. Although not demonstrated in this study, to enable response to unlimited background color, the EL unit can be printed in a pixel-wise arrangement, where each pixel is made of three subpixels (red, green, blue ELE inks) to yield a complete coverage of the RGB color gamut. In addition, spatially color-matching response could also be readily realized through programming each pixel according to the local environment information. - Looking ahead, the multi-material 3D printing strategy proposed here offer advantages of rapid prototyping and customizability in the EL device fabrication. With the proposed 3D printing strategy, intricate EL devices can be custom-made for specific applications, opening up new avenues for the next generation of completely soft light-emitting devices, smart displays, and camouflaging systems.- - Ink composition and preparation - \"To fabricate a fully 3D printed electroluminescent device, three different UV-curable composite inks were formulated, specifically for the ion conducting elastomer (ICE) layer, electroluminescent elastomer (ELE) layer and insulting dielectric elastomer (IDE) layer. The ICE ink was composed of poly(acrylic acid) (PAA) supplemented with ionic monomers (i.e., 3-dimethyl(methacryloyloxyethyl) ammonium propane sulfonate, DMAPS) and an ionic liquid (i.e., 1-ethyl-3-methylimidazolium ethyl sulfate, EMES). There were three ICE ink recipes used in our work, namely ICE1, ICE2, and ICE3. For ICE1, the weight percentages of PAA, DMAPS, and EMES in the ink were set at 12.4 wt.%, 61.7 wt.%, and 25.9 wt.%. For ICE2, the corresponding weight percentages were 6.6 wt.%, 65.8 wt.% and 27.6 wt.%. For ICE3, the corresponding weight percentages were 4.5 wt.%, 67.2 wt.%, and 28.3 wt.%. These inks were created for comparing their mechanical properties as shown in Supplementary Fig._, and ICE1 was typically used in the study. Next, the crosslinker PEGDA400 (0.8 wt.%), photo-initiator I2959 (0.5 wt.%) and silica (5, 10, or 15 wt.%) were added into the PAA/DMAPS/EMES mixture solution in methanol under 3-min vigorous mixing in a planetary mixer (AR-100, Thinky) at 2000u2009rpm. The solvent was removed through evaporation, followed by defoaming for 2u2009min at 2200u2009rpm using a planetary mixer. The as-obtained ICE ink was stored in dark prior to use.\" - \"The IDE ink was prepared by dissolving 0.2u2009g PVDF-HFP in 8u2009mL TEAc under magnetic stirring for 12u2009h. The crosslinker PEGDA400 (1 wt.%), photo-initiator TPO (1 wt.%) and silica nanoparticles (5 wt.%) were added into the viscous paste, and mixed vigorously within the planetary mixer at 2500u2009rpm for 1u2009 min, giving the insulating dielectric ink. The ELE ink was prepared by adding ZnS phosphors (40 wt.%) into the insulating dielectric ink, under 3-min vigorous mixing in the planetary mixer at 2000u2009rpm, following by defoaming at 2200u2009 rpm for 1u2009min. Blue, green and orange light-emitting ELE inks were prepared with the use of ZnS particles doped with different transition metals. All these freshly-prepared inks for DIW were loaded to a UV-blocking 5-mL syringe barrel (EFD Nordson), and stored in dark prior to further 3D printing.\" - 3D printing procedure - \"3D printing of the EL devices was conducted on a 3D Bio-Architect multi-nozzle workstation (Regenovo). To achieve stable 3D printing with continuous and uniform filaments, we first performed a printability study to optimize the printing parameters (i.e., printing speed and printing pressure) for the ICE, IDE, and ELE inks. The optimized parameters are summarized below. The ICE ink was extruded through a nozzle (0.26u2009mm in diameter) at a printing speed of 12u2009 mm s under a pressure of 0.3u2009MPa at room temperature; The IDE ink was extruded through a nozzle (0.21u2009mm in diameter) at a printing speed of 10u2009 mm s under a pressure of 0.5u2009MPa; The ELE ink was extruded through a nozzle (0.21u2009mm in diameter) at a printing speed of 5u2009mm s under a pressure of 0.5u2009MPa.\" - \"To fabricate flexible EL devices, the 3D printed structures were first designed using Solidworks (Dassault Systemes), and were then converted into G-code. The ICE, IDE, and ELE inks within three individual UV-blocking syringe barrels (5u2009 mL) were loaded to the workstation. An ICE layer was first printed as the substrate, followed by printing ELE patterns using one or more ELE inks with the surrounding edges filled with the IDE ink (Supplementary Fig._. The device was then finished up with printing another ICE layer on top. During 3D printing, UV radiation (365u2009nm wavelength, 400u2009mW cm and power intensity of 50%) was applied in situ to simultaneously cure the 3D printed structures. After printing, the as-printed devices were further cured for 60u2009s under UV radiation.\" - Fabrication of quadrupedal soft robots - \"The pneumatically-actuated quadrupedal robots were fabricated following previously described protocols deploying the soft lithography strategy. They were prepared by casting PDMS precursor into a customized mold designed with SolidWorks (Dassault Systemes). The mold consisted of an upper mold and a lower mold, and were printed with a FDM 3D printer (Ultimaker S5). Specifically, the top layer with air channels was prepared by casting a flexible silicone rubber (Ecoflex 00-30, Smooth-on, Inc) into the upper mold (Supplementary Fig._, while the bottom layer was prepared by casting a rigid PDMS precursor (Sylgard 184, Dow Corning) into the lower mold. After degassing in a vacuum drier (Sciencetool, DV-9252), the Ecoflex precursor was partially cured under room temperature for 1u2009h, and the PDMS precursor was partially cured at 75u2009C for 20u2009min. To achieve a robust interface between the top and bottom layers, the partially cured top layer was attached to the partially-cured bottom layer, and the assembly was further cured at 75u2009C for another 1u2009h. Five soft and flexible silicone rubber tubes (1u2009mm in outer diameter) were then inserted into the designed air channels in the assembly and were further sealed with Sylgard 184 PDMS precursor under 75u2009C for 1u2009h. To achieve independent control of the four feet and the central body of the quadrupedal soft robots, air was pumped to each tube separately via syringe manipulation. Movements, such as wriggling and crawling, were driven under alternating inflation and deflation of the quadrupedal soft robots (Supplementary Fig._.\" - Fabrication and assembly of the ELbots - \"To enhance the interfacial covalent bonding between the silicone elastomer of the quadrupedal soft robot and the EL units printed in the following step, the surface of the quadrupedal soft robot was first functionalized with 3-(trimethoxysilyl)propyl methacrylate (TMSPMA) silane. Specifically, the as-prepared robot was thoroughly cleaned with de-ionized water and ethanol in sequence, followed by complete drying under nitrogen flow. Surface of the robot was activated with oxygen plasma (30 W, 200 mtorr; Harrick Plasma PDC-002, 1u2009min), prior to 2u2009h incubation in the TMSPMA silane solution (4.15u2009g TMSPMA in 100u2009mL ethanol). After surface treatment, the soft robot was fixed on the DIW platform. The ICE, ELE, and IDE inks were then printed on the soft robot via multi-material 3D printing using the aforementioned method, as shown in Supplementary Fig._. Once the 3D printing process was completed, the EL soft robot was exposed to UV radiation for another 600u2009s for complete curing. The top and bottom ICE layers were then connected to copper wires (0.8u2009mm in diameter) using a conductive Ag paste. Two quadrupedal soft robots were fabricated. The first quadrupedal soft robot was embodied with four EL units that can emit blue light, and the second robot was embodied with various EL units that can emit either blue, green or orange light. A commercially available light sensor (AS7341, ams AG, 20u2009 xD7u200920u2009mm) was used as the light sensing system, which was placed to the soft robots at the position as indicated in Fig._.\" - Control logic of the ELbots - \"The control logic of the EL integrated soft robots is shown in Fig._. To implement spatially background matching, the control of the first ELbot (with four blue light-emitting EL units) relied on the detection of the light intensity of the background using the light sensor. A microcontroller (Arduino UNO) was used to control the on/off of the relay of each EI unit according to the light intensity, which in turn switching on/off the AC supply (150u2013600 V) of the corresponding EL unit. On the other hand, the control of the second ELbot (with various light-emitting EL units) relied on the detection of the light wavelength, where the corresponding relay of the EL unit was independently switched on/off in accordance with the measured wavelength of the background environment.\" - Fabrication and control logic of the real-time adaptive multicolor EL display - \"Blue, green, and orange-light emitting EL units were printed on a 50-u03BCm PET sheet in an arranged configuration, as illustrated in Supplementary Fig._ , using the aforementioned multi-material printing method. Similar to the control logic used in the ELbots, a light sensor (AS7341, ams AG, 20u2009xD7u2009 20u2009mm) was placed near the display for measuring the wavelength of the background light, and an Arduino microcontroller was used to control the on/off of the EL units that can emit specific colors according to the background light wavelength.\"- - \\' \\' - \"The data that support the findings of this study are provided in the source data file._ are provided with this paper.\" - \\' \\'- - \"Sun, H. et al. Smart responsive phosphorescent materials for data recording and security protection. Nat. Commun. 5, 3601 (2014)._ _ _ Lee, G., Kong, M., Park, D., Park, J. & Jeong, U. Electro-photoluminescence color change for deformable visual encryption. Adv. Mater. 32, 1907477 (2017)._ _ Lee, G., Kong, M., Park, D., Park, J. & Jeong, U. Electro-photoluminescence color change for deformable visual encryption. Adv. Mater. 32, 1907477 (2020)._ _ _ Kim, H. et al. Biomimetic chameleon soft robot with artificial crypsis and disruptive coloration skin. Nat. Commun. 12, 4658 (2021)._ _ _ _ Son, D. et al. An integrated self-healable electronic skin system fabricated via dynamic reconstruction of a nanostructured conducting network. Nat. Nanotech. 13, 1057u20131065 (2018)._ _ _ _ Zhou, Y. et al. Stretchable high-permittivity nanocomposites for epidermal alternating-current electroluminescent displays. ACS Mater. Lett. 1, 511u2013518 (2019)._ _ _ Wang, J., Yan, C., Chee, K. J. & Lee, P. S. Highly stretchable and self-deformable alternating current electroluminescent devices. Adv. Mater. 27, 2876u20132882 (2015)._ _ _ Lee, B. et al. Ultraflexible and transparent electroluminescent skin for real-time and super-resolution imaging of pressure distribution. Nat. Commun. 11, 663 (2020)._ _ _ _ Morin, S. A. et al. Camouflage and display for soft machines. Science 337, 828u2013832 (2012)._ _ _ _ Tan, Y. J. et al. A transparent, self-healing and high-u03BA dielectric for low-field-emission stretchable optoelectronics. Nat. Mater. 19, 182u2013188 (2020)._ _ _ _ Larson, C. et al. Highly stretchable electroluminescent skin for optical signaling and tactile sensing. Science 351, 1071u20131074 (2016)._ _ _ _ Shi, X. et al. Large-area display textiles integrated with functional systems. Nature 591, 240u2013245 (2021)._ _ _ _ Zhang, Z. et al. A colour-tunable, weavable fibre-shaped polymer light-emitting electrochemical cell. Nat. Photonics 9, 233u2013238 (2015)._ _ _ _ Yin, H., Zhu, Y., Youssef, K., Yu, Z. & Pei, Q. Structures and materials in stretchable electroluminescent devices. Adv. Mater. 2106184 (2021).Yang, C. H., Chen, B., Zhou, J., Chen, Y. M. & Suo, Z. Electroluminescence of giant stretchability. Adv. Mater. 28, 4480u20134484 (2016)._ _ _ Duan, L. et al. Solution processable small molecules for organic light-emitting diodes. J. Mater. Chem. 20, 6392u20136407 (2010)._ _ _ Trung, T. Q. & Lee, N.-E. Recent progress on stretchable electronic devices with intrinsically stretchable components. Adv. Mater. 29, 1603167 (2017)._ _ Chen, G. et al. Bioinspired 3D printing of functional materials by harnessing enzyme-induced biomineralization. Adv. Funct. Mater. 32, 2113262 (2022)._ _ Zhang, Y. S. et al. 3d extrusion bioprinting. Nat. Rev. Methods Prim. 1, 1u201320 (2021)._ _ _ _ Truby, R. L. & Lewis, J. A. Printing soft matter in three dimensions. Nature 540, 371u2013378 (2016)._ _ _ _ Studart, A. R. Additive manufacturing of biologically-inspired materials. Chem. Soc. Rev. 45, 359u2013376 (2016)._ _ _ Ligon, S. C., Liska, R., Stampfl, J., Gurr, M. & MxFClhaupt, R. Polymers for 3d printing and customized additive manufacturing. Chem. Rev. 117, 10212u201310290 (2017)._ _ _ Lewis, J. A. & Ahn, B. Y. Three-dimensional printed electronics. Nature 518, 42u201343 (2015)._ _ _ _ Chen, C. et al. Additive manufacturing of piezoelectric materials. Adv. Funct. Mater. 30, 2005141 (2020)._ _ _ Skylar-Scott, M. A., Mueller, J., Visser, C. W. & Lewis, J. A. Voxelated soft matter via multimaterial multinozzle 3d printing. Nature 575, 330u2013335 (2019)._ _ _ _ Lewis, J. A. Direct ink writing of 3d functional materials. Adv. Funct. Mater. 16, 2193u20132204 (2006)._ _ _ Wehner, M. et al. An integrated design and fabrication strategy for entirely soft, autonomous robots. Nature 536, 451u2013455 (2016)._ _ _ _ Lei, Z. & Wu, P. A highly transparent and ultra-stretchable conductor with stable conductivity during large deformation. Nat. Commun. 10, 3429 (2019)._ _ _ Chen, J. et al. Mimicking natures butterflies: Electrochromic devices with dual-sided differential colorations. Adv. Mater. 33, 2007314 (2021)._ _ _ Teyssier, J., Saenko, S. V., Van Der Marel, D. & Milinkovitch, M. C. Photonic crystals cause active colour change in chameleons. Nat. Commun. 6, 1u20137 (2015)._ _ Reiter, S. & Laurent, G. Visual perception and cuttlefish camouflage. Curr. Opin. Neurobiol. 60, 47u201354 (2020)._ _ _ Shepherd, R. F. et al. Multigait soft robot. Proc. Natl Acad. Sci. USA. 108, 20400u201320403 (2011)._ _ _ _ Wang, Y., Cui, H., Zhao, Q. & Du, X. Chameleon-inspired structural-color actuators. Matter 1, 626u2013638 (2019)._ _ Du, X. et al. Reconfiguration, camouflage, and color-shifting for bioinspired adaptive hydrogel-based millirobots. Adv. Funct. Mater. 30, 1909202 (2020)._ _ _ \"- - J.L. acknowledges the financial support from Natural Science Foundation of Guangdong Province (2022A1515010152 and 2020A1515110288), Basic Research Program of Shenzhen (JCYJ20210324105211032 and RCBS20210609103713046), and MechERE Centers at MIT and SUSTech (Y01346002). This work was also supported in part by the Science, Technology and Innovation Commission of Shenzhen Municipality (ZDSYS20200811143601004) and the Additive Manufacturing Innovation Center at SUSTech. The authors also would like to acknowledge the technical support from SUSTech Core Research Facilities, Prof. Hong Wang and Prof. Kai Wang for their generous access to dielectric and luminance measurement facilities, respectively.- - Authors and Affiliations - \"Department of Mechanical and Energy Engineering, Southern University of Science and Technology, Shenzhen, 518055, ChinaPei Zhang,_Iek Man Lei,_Guangda Chen,_ Jingsen Lin,_Xingmei Chen,_Jiajun Zhang,_Chengcheng Cai,_Xiangyu Liang_&_ Ji LiuShenzhen Key Laboratory of Biomimetic Robotics and Intelligent Systems, Department of Mechanical and Energy Engineering, Southern University of Science and Technology, Shenzhen, 518055, ChinaJi LiuGuangdong Provincial Key Laboratory of Human-Augmentation and Rehabilitation Robotics in Universities, Southern University of Science and Technology, Shenzhen, 518055, ChinaJi Liu\" - \"AuthorsPei ZhangYou can also search for this author in _Iek Man LeiYou can also search for this author in _Guangda ChenYou can also search for this author in _Jingsen LinYou can also search for this author in _Xingmei ChenYou can also search for this author in _Jiajun ZhangYou can also search for this author in _Chengcheng CaiYou can also search for this author in _Xiangyu LiangYou can also search for this author in _Ji LiuYou can also search for this author in _\" - Contributions - J.L. conceived the ideas. J.L. and P.Z. designed the experiments. P.Z., I.M.L., G.C., J.S.L., X.C., J.Z., C.C., X.L., and J.L. performed experiments and analysed the experimental data. J.L. and P.Z. wrote the manuscript with input from all authors. J.L. supervised the study. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - \"Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work._ are available.\" - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons License.\" - \"Numerous reviews highlight recent efforts to mimic the DOPA-metal cross-link strategy with mussel-inspired catechol-functionalized polymers; yet, none of these materials reproduces the complex hierarchical structure of the cuticle or its properties. Indeed, little is understood about how the nanostructure of the cuticle and granules is achieved. It is known that the proteins that comprise the cuticle are stored within secretory vesicles in the mussel footu2014 the organ responsible for synthesizing and stockpiling the proteins that make the byssus (Fig._). Self-assembly of the cuticle occurs within minutes via release of vesicle contents into a groove running along the foot, during which they coalesce and spread over the already formed collagenous core of the byssus fiber (Fig._). However, many questions remain concerning how the intricate cuticle substructure emerges via self-assembly and how metal ions are infiltrated to cure the formed cuticle.\" - Here, we report an in-depth compositional and ultrastructural investigation of the structure and self-assembly of the mussel byssus cuticle. Specifically, we investigate the 3D nanostructure and elemental composition of the secretory vesicles and the cuticle itself, utilizing a combination of focused ion beam scanning electron microscopy (FIB-SEM), transmission electron microscopy (TEM) and scanning transmission electron microscopy with energy dispersive X-ray spectroscopy (STEM-EDS). By examining the different stages of assembly, we gain important new insights into the formation process and function of this complex biological material, with bearing on the design of technically and biomedically relevant composite materials.- - Nanostructural investigation of cuticle secretory vesicles - \"Previous investigations have identified that the precursor proteins forming the cuticle are stored in secretory vesicles in the mussel foot within a region known as the enzyme or cuticle gland (Fig._). Transmission electron microscopy (TEM) imaging of post-stained thin sections of the cuticle gland of Mytilus edulis revealed that each micron-sized secretory vesicle possesses at least two distinctive regionsu2014a darker outer phase (more heavily stained with Os) and a lighter, less-stained inner phase (Fig._). The distinctive biphasic brain-like texture of the inner phase consisting of lighter stained connected layers was previously observed in cuticle secretory vesicles of another species, Mytilus galloprovincialis, and is highly reminiscent of native thread cuticle granule structure (Fig._)u2014leading us to name this the proto-granule and the outer phase, the proto-matrix. A third, very lightly stained crescent-shaped phase is also observed at the outer periphery of most cuticle secretory vesicles. FIB-SEM of chemically fixed foot tissue samples enabled 3D rendering of secretory vesicles in a small region of the cuticle gland close to the secretion point, revealing a tightly packed arrangement of nearly spherical vesicles (Fig._ ). Consistent with TEM, there are at least three discernible phases in these vesicles that stain differently, corresponding to the proto-granule, proto-matrix and crescent phase (Fig._). Volumetric analysis of the 3D data of 28 individual cuticle secretory vesicles indicates similar distribution of volume fractions of the three phases in all vesicles suggesting a highly regulated formation process (Supplementary Fig._).\" - \"Fig. 2: TEM and FIB-SEM imaging of cuticle secretory vesicles in mussel foot tissue.a TEM image of osmium stained cuticle secretory vesicle from region similar to Fig._. Contrast is achieved by different osmium staining with the proto-matrix (pm) staining the heaviest, the crescent phase (cp) staining the lightest and the proto-granules (pg) staining in between. Scale baru2009=u2009200u2009 nm. b FIB-SEM image of osmium stained region of cuticle gland similar to Fig._ . Contrast is inverted compared to TEM, but pg, pm and cp can clearly be differentiated. Scale baru2009=u20091000u2009nm. c Using electron density contrast, secretory vesicles and their inner structure were reconstructed in 3D from an image stack consisting of 392 images. d Magnified and cropped 3D image of a cuticle secretory vesicle showing all three phases. e STEM-EDS compositional analysis of native cuticle granule and matrix regions, showing distribution of nitrogen and sulfur in the region in the STEM-HAADF image in the left panel. Scale baru2009=u2009 500u2009nm. f Relative sulfur wt% (not calibrated) collected from a transect across four secretory vesicles in dotted box in e reveals that pg has approximately one half the amount of sulfur as pm.\" - Compositional investigation of cuticle secretory vesicles - \"Compositional analysis of cuticle secretory vesicles was performed with STEM-EDS, revealing an elevated and uniform nitrogen content in cuticle secretory vesicles compared with the surrounding intracellular material (Fig._). In contrast, there is an approximately two to three-fold higher sulfur content in the proto-matrix relative to the proto-granules (Fig._). Notably, recent transcriptomic studies have identified a family of putative cysteine-rich proteins (mfp-16u2013mfp-19) within the cuticle gland of a related species Mytilus californianus. This is consistent with previous cytochemical studies predicting the presence of cysteine-rich proteins within the cuticle secretory vesicles of Mytilus galloprovinicialis. The fact that the proto-granule contains a lower sulfur signal substantiates the previous supposition that the granules mainly contain mefp-1, which completely lacks cysteine. Furthermore, cysteine is a strong target of OsO4 staining, which helps explain the contrast observed in TEM and FIB-SEM. Notably, within the sensitivity limit, we did not detect peaks indicative of transition metal ions (e.g., Fe and V) in the EDS spectra from unstained cuticle secretory vesicles (Supplementary Fig._), even though these metals were previously detected in the mature cuticle. This result is consistent with recent findings suggesting that metal ions are added into the cuticle in a post-secretion curing step.\" - Structural investigation of cuticle formation - \"To investigate the dynamic transition from the storage phase of cuticle secretory vesicles into a hard and flexible mesostructured coating, TEM was performed on vesicles secreted into the foot groove. Vesicle secretion was induced by injection of 0.56 M_KCl into the pedal nerve of the foot, as previously used to study thread formation. TEM images of the distal region of induced threads chemically fixed several minutes into the formation process reveal that cuticle secretory vesicle contents fuse and coalesce already within the secretory ducts leading from the glands to the foot groove (Fig._). This reveals that the proto-matrix retains a liquid-like behavior as it coalesces with the contents of other vesicles, showing no apparent boundaries. However, the proto-granules remain as separate entities with the biphasic brainy structure intact. Within the groove, the coalesced secretory vesicles spread over the collagenous core forming a thin layer, as previously shown with histology and confocal Raman spectroscopy (Figs._, ). This granular morphology is very similar to the native cuticle structure (Fig._ ). Taken together with the TEM-EDS measurements (Fig._), these observations suggest that the cuticle secretory vesicles contain two immiscible protein phases represented by the proto-matrix and proto-granule that are pre-organized within the vesicles to facilitate rapid assembly of a hierarchically structured coating.\" - \"Fig. 3: Cuticle formation in the mussel foot groove via coalescence of cuticle secretory vesicle contents.a Schematic overview of transverse section of mussel foot tissue showing anatomy of byssal glands. Cuticle secretory vesicles (in red) are lined up at the edge of the cuticle gland, ready to be secreted into the groove, aided by cilia. During cuticle assembly, the secretory vesicle contents cluster and coalesce into the groove spreading over the formed core surface. b TEM image from a stained section of an induced foot in the region indicated in a, showing coalesced cuticle secretory vesicles about to be released into the groove. Notably, the proto-matrix (pm) of several vesicles merges to form a continuous matrix, while the proto-granules (pg) retain their brain-like structure. Scale baru2009=u2009500u2009nm. c TEM image from stained section of an induced foot in the region indicated in a, showing a formed induced thread with intact cuticle. Scale baru2009=u2009500u2009nm. d Magnified region from c showing details of the induced cuticle including granule and matrix. Scale baru2009 =u2009200u2009nm.\" - Nanostructural investigation of native cuticle - \"The nanoscale structural and compositional details of the native cuticle were next investigated with FIB-SEM and STEM-EDS on chemically-fixed distal thread samples, both of which revealed that M. edulis thread cuticles consist of a single layer of granules, rather than the several layers observed in other Mytilus species. FIB-SEM analysis enabled the visualization of the matrix and granules, as well as the convoluted brain-like internal substructure of the granules (Fig._ ). 3D reconstruction of the image stacks revealed the connectivity of the matrix phase and heavily stained part of the granule in three dimensions and further indicates that the lightly stained (darker) region within the granules is comprised of a connected network of flattened layers resembling a bicontinuous phase of a micro phase-separated block co-polymer (Fig._). Image analysis reveals the flattened layers of multiple granules within a given thread possess a highly-defined thickness of ~20u2009nm, although this varied between different threads (Fig._ and Supplementary Fig._; Supplementary Table_). Additionally, the flattened layers within the granules are oriented along a common direction approximately 45xB0 to the fiber axis (Fig._)u2014the orientation of which is consistent between different granules within the same thread. The 3-dimensional (spherically averaged) autocorrelation functions of the different measured granules are similar to each other within a given thread, indicating a high homogeneity in their structural features (Supplementary Fig._).\" - \"Fig. 4: FIB-SEM 3D-reconstruction of intragranular nanostructure in native thread cuticle.a FIB-SEM image of osmium stained region of native distal thread showing substructure of the core and cuticle, including granule and matrix in the cuticle and previously described voids in the core. Scale baru2009=u2009500u2009 nm. b Higher magnification FIB-SEM image of a single granule. Contrast arises from heavily staining (hs) and lightly staining (ls) regions within a single granule. n.b. contrast is inverted compared to TEM, with elements of higher atomic mass appearing brighter. Scale baru2009=u2009200u2009nm. c 3D reconstruction of FIB-SEM image stack using the contrast of the lightly staining (ls) phase of the granules and the core voids (which are parallel to the fiber axis). d Granule cropped using clipping planes parallel to the one indicated in c revealing internal structure of the ls phase, consisting of bicontinuous flattened layers with a thickness of ~20u2009nm. The transparent region between layers constitutes the heavily staining (hs) phase from b. Scale baru2009=u200980u2009nm. e STEM-EDS compositional analysis of native cuticle granule and matrix regions, showing distribution of nitrogen, sulfur, iron, and vanadium in the region in the STEM-HAADF image on the left. Scale baru2009=u2009100u2009nm.\" - Compositional investigation of native cuticle - \"At first glance, STEM-EDS compositional studies of the mature cuticle are largely consistent with those measured on the cuticle secretory vesicles. Similar to the proto-granules and proto-matrix in the vesicles, the granules and matrix have uniform nitrogen levels, while the matrix phase has a significantly higher sulfur signal than the granules (Fig._, S4). However, while transition metal ions were not detected in the cuticle secretory vesicles (Supplementary Fig._ ), both Fe and V were detected by EDS in the native cuticle (Fig._). Unexpectedly, however, metal distribution was micro-partitioned within the cuticle, with V explicitly associated with the granules and Fe associated only with the matrix with a sharp interface at the boundary. This is consistent with previous Raman spectroscopic measurements across different Mytilid species showing that DOPA-V coordination is predominantly detected in native cuticles, rather than DOPA-Fe coordinationu2014although it should be noted that Fe can be artificially introduced and coordinated following removal of V with EDTA. Because the metals are not present in the secretory vesicles, this finding also implies that the metal ions spontaneously segregate between the matrix and granules when introduced during distal thread formation.\"- - \"The findings of this study suggest that the granular mesostructure of the cuticle is achieved through a membrane-bound liquidu2013liquid phase separation (LLPS) consisting of immiscible fluid protein phases. Co-existing condensed liquid protein phases are observed in nucleolar subcompartments and believed to be important for tuning the vectorial transport and processing of rRNA. In contrast, within the cuticle secretory vesicles, the co-existing LLPS leads to a micro-scale to nano-scale distribution of proteins with different functional groups (i.e., DOPA and Cysteine) that contribute to cross-linking the liquid phase to solid when triggered during assembly. This effectively tunes the viscoelastic properties of the final material at the mesoscale. Based on our current understanding of the role of metal coordination complexes in the byssus cuticle, we posit that microscale partitioning of specific metal coordination cross-links between the granules and matrix has important implications for the dynamic properties of this material, and it is important to understand why and how this occurs. Current models postulate different mechanical properties in the matrix and granulesu2014 either hard granular inclusions in a soft matrix or soft, water-absorbent granules in a stiffer matrix (at least under low hydration conditions). However, both models rest on results from quasistatic mechanical indentation performed at a single loading rate, which do not access the dynamic nature of the metal coordination bonds. On the other hand, rheological investigations of mussel-inspired catechol-enriched metallopolymers have demonstrated clearly that DOPA-V coordination bonds possess a more than 10-fold longer bond lifetime (and thus, slower relaxation time) than DOPA-Fe bonds. This is functionally relevant considering that mussels face an enormous range of mechanical loading rates from rapid crashing waves with velocities up to 30u2009m/s to the extremely slow loading from predators such as sea stars. Thus, it seems plausible that the physical demands of life in the intertidal have led to evolution of an adaptive coating that responds to a wide range of loading conditions (e.g., strain rates, extensions, and forces), achieved through hierarchical organization of dynamic bonds.\" - \"Fabricating this functional composite structure reproducibly requires a remarkable degree of process control during assembly, which is apparently achieved by pre-organizing the various cuticle components into co-existing phase-separated liquid phases within the secretory vesicles prior to assembly (Fig._). Until recently, only DOPA-rich mefp-1 was confirmed to be present in the cuticle (Fig._). However, as mentioned, transcriptomics has identified four new putative cuticle proteins (mfp-16u2013mfp-19) in a closely related species, all of which are enriched in cysteine and likely present primarily in the proto-matrix/matrix, based on our STEM-EDS findings and previous cytochemical evidence. The rapid coalescence of the proto-matrix into a solid material during formation is likely related to the well-known ability of cysteine to participate in various covalent and non-covalent interactions. However, as previously proposed, cysteines in byssus proteins likely also perform a crucial role in redox cycling of the DOPA catechol moiety. Both putative roles will be discussed later in more detail. Evidence that mefp-1 is localized in the granule comes from the previous Raman-based localization of DOPA-V interactions in the granules, the different susceptibilities of the granule to chymotrypsin vs. pepsin digestion in cytochemical studies (Supplementary Fig._) and the lower sulfur content in the granules observed here with STEM-EDS (Figs._, ).\" - \"Given these compositional findings, what is then the source of complex nano-structure observed in the secretory vesicles and thus, the final cuticle structure? We propose that the bicontinuous structure and lower sulfur signal of granules arises from the immiscibility of mefp-1 and the Cys-rich protein, which is driven by the amphiphilic block co-polymer-like structure of mefp-1 (Fig. S6). Mefp-1 consists of numerous repeats of the decapeptide consensus motif [AKPSYPPTYK]n in which the tyrosine (Y) residues are largely converted to DOPA (total content of 10u201315u2009mol%) and known to interact with metal ions (Fe, V) (Figs._ i, and Supplementary Fig._). The repetitive region of mefp-1 possesses a hydrophilic character due to the high content of conserved Lysine (K) residues. Recombinant expression of truncated mfp-1 consisting of 12 or 22 repeats of the hydrophilic decapeptide have been demonstrated to undergo spontaneous LLPS under high salt conditions mediated via pi-cation interactions, offering support to the coacervate-like nature in the cuticle secretory vesicles. However, at the N-terminus of native mefp-1 and mgfp-1 from M. galloprovincialis is an 60u201380 amino acid non-repetitive domain that is markedly less hydrophilic, giving the overall protein an amphiphilic profile (i.e., block co-polymer-like) (Fig._ and Supplementary Fig._). We propose here that the non-repetitive domain provides the impetus for formation of a bicontinuous phase (also observed in M. galloprovincialis), forming the well-defined ~20u2009nm thick layers characteristic of the granules, while the positively charged DOPA-rich phase interacts with the cysteine-rich proteins to produce the heavily stained phase of variable thickness between the flattened layers (Fig._). The fact that osmium has a very high affinity to cysteine under the alkaline staining conditions supports this model. The matrix on the other hand, seems to consist mostly of the cysteine-rich proteins.\" - \"Fig. 5: Schematic model of cuticle assembly via co-existing condensed liquid phase vesicles.To achieve the high degree of compositional, structural and mechanical hierarchy observed in the native byssus cuticle, mussels store the protein precursors as a LLPS of two co-existing phases. The main protein component mfp-1 and the Cys-rich proteins (mfp-16u201319) are immiscible due to the amphiphilic nature of mfp-1, leading to phase separation into a bicontinuous structure that characterizes the proto-granule. During secretion and cuticle assembly, the proto-matrix of nearby secretory vesicles fuses forming the continuous matrix of the cuticle, possibly cross-linked via cysteine residues. The intricate nanostructure of the proto-granule is maintained in the newly formed cuticle and contains a much higher local concentration of DOPA than the surrounding matrix. Thus, when metals are added to the thread in a secondary curing, the metal ions that have the highest stability complex with DOPA (i.e., vanadium) diffuse into the granule where they become concentrated relative to iron. This results in different viscoelastic behavior of the granule and matrix, which is likely an adaptive function to life on the highly dynamic rocky seashore. Cysteine may play a secondary, but equally important role as a reducing agent that counteracts the spontaneous oxidation of DOPA to DOPA-quinone under basic conditions, which enables strong metal coordination cross-link formation.\" - \"Based on this new model of how the observed meso-structures and nano-structures form in the cuticle secretory vesicles, we consider next how the condensed fluid phase within the vesicles suddenly transitions into a hard, yet stretchy composite material. Our TEM investigation of induced thread formation indicates that the cysteine-rich proto-matrix fuses during assembly, creating the continuous matrix of the cuticle in which the pre-assembled granules are embedded. As mentioned, cysteine is a highly promiscuous cross-linker with the ability to form covalent bonds via disulfide linkages or through oxidative cross-linking with DOPA residues. In fact, 5-cysteinyl-DOPA was previously purified from byssus material. However, considering that nearly 85% of cross-links in the cuticle depend on metal ions, cysteine-based covalent bonds are not the dominant cross-linking mechanism in the matrix. Based on the co-localization of cysteine and Fe in the matrix, we find the possibility that sulfuru2013Fe interactions may contribute as load-bearing cross-links highly compelling. While typically found in enzymes involved with redox pathways in cells, recent AFM single molecule force spectroscopic reveal that Feu2013S clusters can function as reversible sacrificial cross-links in proteins with mechanical breaking force comparable to other metal coordination complexes.\" - \"A secondary, but equally important role of the cysteine residues within the granules may be as a reducing agent to stall spontaneous DOPA oxidation in the basic environment of the ocean (Fig._). Another cysteine-rich protein called mfp-6 was previously identified in the adhesive secretion of the byssus and was shown in vitro to prevent DOPA from spontaneously oxidizing to DOPA-quinone (a less effective adherent) through redox cycling. Similarly, DOPA-quinone is inefficient at forming strong metal coordination complexes. Considering that at least 85% of cross-linking in the cuticle is based on metal coordination bonds, the proposed reducing role of cysteine may be crucial to achieving the dynamic properties, and is likely enhanced by the proposed co-localization with the DOPA-rich repetitive domain of mefp-1 in the granules (Fig._).\" - Shortly after the cuticle forms during assembly via fusion of the proto-matrix, transition metal ions (Fe and V) are apparently added, providing secondary mechanical reinforcement via metal coordination bonding. Currently, it is unclear if metal addition is an active or passive process. Cuticles stripped of metal with the metal chelator EDTA are able to uptake metal passively from millimolar metal chloride solutions. However, pulse-chase studies have clearly shown that radiolabeled Fe is actively taken up by mussels and is stored temporarily in the soft tissue from where it is transferred into the byssus fibers over a period of several days. Regardless of the source of the metal ions, Raman spectroscopic investigation of the cuticle revealed that DOPA is able to bind both V and Fe within the cuticle when exposed to only the individual ions during in vitro experiments. However, all Raman spectra reported for native byssal threads across several species are dominated by the DOPA-V signal, rather than DOPA-Fe. It is tempting to propose that this partitioning may reflect the inherent differences in dissociation rates of the DOPA-Fe and DOPA-V complexes previously revealed by rheological studies of mussel-inspired catechol-functionalized hydrogels, showing that DOPA-V has more than an order of magnitude longer bond lifetime than DOPA-Fe complexes. The slower bond dissociation of DOPA-V implies that it will be less likely to exchange with Fe once coordinated, leading to its concentration in the mfp-1-enriched granules over time through dissociation and diffusion. However, the longer bond lifetime, not only affects the distribution of metals, but as already pointed out, will also determine local viscoelastic relaxation behavior defined by the compartmentalization of mfp-1 within the granules. Taking for granted that varying the viscoelastic response on the mesoscale is important to the function of the cuticle (e.g., being hard and extensible, retaining water in dry conditions and/or responding to a range of loading rates), this assembly mechanism provides a degree of control over dynamic mechanical response that is not possible in current engineered polymers. Indeed, mimicking this fabrication process and thus, this remarkable degree of control over soft matter response could inspire the design of new responsive smart polymers for a range of applications from flexible electronics and actuated structures to drug delivery and dynamic tissue scaffolds.- - Materials - \"Blue mussels (M. edulis) purchased from the Alfred-Wegener-Institut were maintained at ~14u2009xB0C in an aquarium with artificial salt water. Native threads were harvested from mussels upon arrival and were stored at 4u2009xB0C in water prior to use. Investigations were performed either on native byssal threads (5u20138u2009cm in length) or on the foot organ of adult mussels removed with a scalpel (nu2009=u20092). To investigate cuticle assembly, protein secretion was induced by injecting a small volume of 0.56u2009M KCl solution in the base of the mussel feet. Induced mussel feet were dissected after at ~5u2009min following KCl injection to investigate vesicle secretion and after ~20u2009 min to investigate the induced thread cuticle structure. We have complied with all relevant ethical regulations for testing and research of Mytilus edulis.\" - Chemical fixation and embedding - \"Dissected feet (induced or not) were carefully rinsed with cold water, blotted with a paper towel to remove mucus and pre-fixed for 30u2009min at 4u2009xB0 C in 3% glutaraldehyde, 1.5% paraformaldehyde, 650u2009mM sucrose in 0.1u2009 M cacodylate buffer pH 7.2. The foot tissue was then cut into thin cross-sections comprising the groove and part of the gland tissue and then fixed for 2u2009 h at 4u2009xB0C in the same buffer as above. Fixed samples were rinsed 5xD7 with 0.1u2009M cacodylate buffer, pH 7.2 at 4u2009xB0C and post-fixed with 1% OsO4 for 1u2009h at 4u2009xB0C. Tissue samples prepared for elemental analysis were not treated with OsO4. Samples were rinsed again in 0.1u2009 M cacodylate buffer pH 7.2 (3u2009xD7u20095u2009min at 4u2009xB0C), followed by series dehydration in ethanol (50%, 70%, 90%, 3u2009xD7u2009100%) for 10u2009min each step at RT. Dehydrated samples were embedded either in low viscosity SpurrxB4s resin (Electron Microscopy Sciences, # 14300) for TEM/STEM-EDS or in Hard Plus resin 812 (Electron Microscopy Sciences, # 14115) for FIB-SEM and polymerized at 70u2009xB0C for at least 48u2009h. Ultrathin sections of 100u2009nm for TEM investigations were prepared using a PowerTome Model XL ultramicrotome (Boeckeler Instruments, Inc.) and mounted on carbon coated Cu grids (200 mesh) for imaging and on Lacey carbon coated Cu grids (200 mesh) for EDS measurements. In order to reveal the internal structure of the protogranule, some grids were post-stained with 2% uranyl acetate for 10u2009min.\" - \"The distal region of native threads were washed 3u2009xD7u20095u2009min in cold double distilled H2O and cut into small pieces of ~3u2009mm in length. Fixation was carried out for 1u2009h at 4u2009xB0C in 2.5% glutaraldehyde and 1.5% paraformaldehyde in 0.1u2009M cacodylate buffer pH 7.4. Samples were rinsed 3u2009xD7u200910u2009min in 0.1u2009M cacodylate buffer at 4u2009 xB0C before post-fixation with 1% OsO4 for 1u2009h. Samples for EDS measurements were not stained with OsO4. A second rinsing step in 0.1u2009M cacodylate buffer pH 7.4 (3u2009xD7u20095u2009min at 4u2009xB0C) was followed by dehydration in ethanol (50%, 70%, 90%, 3u2009xD7u2009100%) for 10u2009min each step at RT. Threads were embedded in low viscosity SpurrxB4s resin (Electron Microscopy Sciences, # 14300) at 65u2009xB0C over 2 days. The resulting resin blocks were trimmed to the region of interest and sectioned to 100u2009nm using an ultramicrotome (PowerTome Model XL). Ultrathin sections were mounted on lacey carbon coated copper grids (200 mesh) for imaging and EDS measurements.\" - Transmission electron microscopy - \"TEM was performed with a Zeiss EM 912 Omega with an acceleration voltage of 120u2009 kV and a Jeol JEM ARM200F equipped with a cold field-emission electron source and a silicon drift detector (SSD), operated at 200u2009kV acceleration voltage and 15u2009xB5A emission current. TEM mode was used exclusively for imaging (Bright Field imaging) at magnifications of 10,000xD7 and 16,000xD7, whereas STEM mode was used for energy dispersive spectroscopy (EDS) and high angle annular dark field (HAADF) imaging. In STEM mode a fine electron probe scans the surface of the sample pixel-by-pixel enabling identification of the area of the sample that generates certain characteristic X-rays with nanometric resolution. Overview elemental maps of foot sections were acquired at a magnification of 50,000xD7 with a pixel size of 17u2009nmu2009xD7u200917u2009nm and an exposure of 1u2009s per pixel. For thread sections, elemental maps (59u2009xD7u2009 89 pixel) were acquired at a magnification of 200,000xD7 with a pixel size of 9.3u2009xD7u20099.3u2009nm and an exposure of 1u2009s per pixel using a SDD.\" - FIB-SEM - \"Resin blocks containing samples were polished in order to expose the tissue or thread at the block surface. Samples were sputter-coated with three Carbon layers (~5u2009nm each) and one platinum layer (~5u201310u2009nm) and transferred to the Zeiss Crossbeam 540 (Carl Zeiss Microscopy GmbH, Germany). At the region of interest, a trench for SEM imaging was milled into the sample surface using the 65u2009nA FIB current at 30u2009kV acceleration voltage. The resulting cross-section was finely polished using the 1.5u2009nA FIB probe at 30u2009 kV. Measurement of foot tissue and threads required different parameters. Thin slices of samples were removed in a serial manner by FIB milling (300u2009 pA, 30u2009kV, slice thickness 17.5u2009nm for foot tissue; 700u2009pA, 30u2009 kV, slide thickness 10.5u2009nm for threads). After each milling step, the specimen was imaged by SEM using the secondary and backscattered electron detector (acceleration voltageu2009=u20092u2009kV for foot tissue and 2.5u2009kV for threads). For foot tissue and threads, the image resolution was 2048u2009xD7u20091536 pixels and 1024u2009xD7u2009785 pixels, respectively with a lateral image pixel size of 12.4u2009nm and 4.8u2009nm, respectively. Images were recorded using line averaging (Nu2009=u20094 for foot tissue and Nu2009=u200931 for threads) and a dwell time of 200u2009ns.\" - FIB-SEM data processing - \"The resulting secondary and back-scattered electron images were processed using the SPYDER3 (Scientific Python Development Environment) (Python 3.6) software. Custom-written python scripts were developed and provided by Luca Bertinetti. For data analysis of cuticle gland tissue, images were automatically aligned using enhanced correlation coefficient alignment. Total variation denoising was performed by applying the ChambollexB4s projection algorithm (100,000 iterations, weight 0.07, 0.001 eps) in 3D mode. Segmentation of cuticle vesicles was performed using the ZIB version of Amira 3D (Thermo Fisher Scientific, USA). Vesicle shapes of 28 individual granules were segmented manually from 392 processed back-scattered electron images using the brush tool. The inner phase, which corresponds to the proto-granules as well as the proto-matrix phase were automatically segmented using the Magic Wand tool. 3D visualization of both phases was realized by volume rendering of segmented structures.\" - For data analysis of the thread cuticle structure, secondary electron images were automatically aligned using the Fourier shift theorem for detecting the translational shift in the frequency domain and vertical stripes arising from the waterfall effect by FIB milling were removed by Fourier filtering. Total variation denoising was performed by applying the Chambolle algorithm in 3D mode. The images were inverted afterwards with Fiji and Sauvola\\'s local thresholding computation was applied with a block size of 11, a k value of 0.005 and r value of 1.7. As the thresholded images contained regions resulting from statistical noise in the image, only thresholded regions containing minimum 40 pixels were selected. Images were median 3D filtered with Fiji and x, y, z radii were set to 1.5. Segmentation of five adjoining cuticle granules was performed using the ZIB version of Amira 3D (Thermo Fisher Scientific, USA). Granule shapes were segmented manually from 352 processed and inverted secondary electron images using the brush tool. In a second step, the lightly staining (ls) granule phase was segmented automatically from local threshold computed and median 3D filtered image stacks using the Magic Wand tool. - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - The datasets generated during and/or analysed during the current study are available from the corresponding author on reasonable request. - \\' \\'- - 31 March 2020An amendment to this paper has been published and can be accessed via a link at the top of the paper.- - \"Sai, H. et al. Hierarchical porous polymer scaffolds from block copolymers. Science 341, 530u2013534 (2013)._ _ _ _ Lutz, J. F., Lehn, J. M., Meijer, E. W. & Matyjaszewski, K. From precision polymers to complex materials and systems. Nat. Rev. Mater. 1, 16024 (2016)._ _ _ _ Lutolf, M. P. & Hubbell, J. A. Synthetic biomaterials as instructive extracellular microenvironments for morphogenesis in tissue engineering. Nat. Biotechnol. 23, 47u201355 (2005)._ _ _ Webber, M. J., Appel, E. A., Meijer, E. W. & Langer, R. Supramolecular biomaterials. Nat. Mater. 15, 13u201326 (2016)._ _ _ _ Lehn, J. M. Perspectives in chemistryu2014steps towards complex matter. Angew. Chem. Int. Ed. 52, 2836u20132850 (2013)._ _ _ Lehn, J. M. Toward self-organization and complex matter. Science 295, 2400u2013 2403 (2002)._ _ _ _ Huo, Q. S., Margolese, D. I. & Stucky, G. D. Surfactant control of phases in the synthesis of mesoporous silica-based materials. Chem. Mater. 8, 1147u20131160 (1996)._ _ _ Zhao, D. Y. et al. Triblock copolymer syntheses of mesoporous silica with periodic 50 to 300 angstrom pores. Science 279, 548u2013552 (1998)._ _ _ _ Meseck, G. R., Terpstra, A. S. & MacLachlan, M. J. Liquid crystal templating of nanomaterials with natures toolbox. Curr. Opin. Colloid Interface Sci. 29, 9u201320 (2017)._ _ _ Fratzl, P. & Weinkamer, R. Natures hierarchical materials. Prog. Mater. Sci. 52, 1263u20131334 (2007)._ _ _ Degtyar, E., Harrington, M. J., Politi, Y. & Fratzl, P. The mechanical role of metal ions in biogenic protein-based materials. Angew. Chem. Int. Ed. 53, 12026u201312044 (2014)._ _ _ Heim, M., Keerl, D. & Scheibel, T. Spider silk: from soluble protein to extraordinary fiber. Angew. Chem. Int. Ed. 48, 3584u20133596 (2009)._ _ _ Priemel, T., Degtyar, E., Dean, M. N. & Harrington, M. J. Rapid self-assembly of complex biomolecular architectures during mussel byssus biofabrication. Nat. Commun. 8, 14539 (2017)._ _ _ Baer, A. et al. Mechanoresponsive lipid-protein nanoglobules facilitate reversible fibre formation in velvet worm slime. Nat. Commun. 8, 974 (2017)._ _ _ Winegard, T. et al. Coiling and maturation of a high-performance fibre in hagfish slime gland thread cells. Nat. Commun. 5, 3534 (2014)._ _ _ Denny, M. W. & Gaylord, B. Marine ecomechanics. Annu. Rev. Mar. Sci. 2, 89u2013114 (2010)._ _ _ Harrington, M. J., Masic, A., Holten-Andersen, N., Waite, J. H. & Fratzl, P. Iron-clad fibers: a metal-based biological strategy for hard flexible coatings. Science 328, 216u2013 220 (2010)._ _ _ _ Holten-Andersen, N., Fantner, G. E., Hohlbauch, S., Waite, J. H. & Zok, F. W. Protective coatings on extensible biofibres. Nat. Mater. 6, 669u2013672 (2007)._ _ _ _ Holten-Andersen, N. et al. Metals and the integrity of a biological coating: the cuticle of mussel byssus. Langmuir 25, 3323u20133326 (2009)._ _ _ Schmitt, C. N. Z. et al. Mechanical homeostasis of a DOPA-enriched biological coating from mussels in response to metal variation. J. R. Soc. Interface 12, 20150466 (2015)._ _ Monnier, C. A., DeMartini, D. G. & Waite, J. H. Intertidal exposure favors the soft-studded armor of adaptive mussel coatings. Nat. Commun. 9, 3424 (2018)._ _ _ Lee, B. P., Messersmith, P. B., Israelachvili, J. N. & Waite, J. H. Mussel-inspired adhesives and coatings. Annu. Rev. Mater. Res. 41, 99u2013132 (2011)._ _ _ _ Li, L., Smitthipong, W. & Zeng, H. Mussel-inspired hydrogels for biomedical and environmental applications. Polym. Chem. 6, 353u2013358 (2015)._ _ _ Krogsgaard, M., Nue, V. & Birkedal, H. Mussel-inspired materials: self-healing through coordination chemistry. Chem. Eur. J. 22, 844u2013857 (2016)._ _ _ Harrington, M. J., Jehle, F. & Priemel, T. Mussel byssus structure-function and fabrication as inspiration for biotechnological production of advanced materials. Biotechnol. J. 13, e1800133 (2018)._ _ Vitellaro-Zuccarello, L. Ultrastructural and cytochemical study on the enzyme gland of the foot of a mollusk. Tissue Cell 13, 701u2013713 (1981)._ _ Waite, J. H. The formation of mussel byssus: anatomy of a natural manufacturing process. Results Probl. Cell. Differ. 19, 27u201354 (1992)._ _ _ DeMartini, D. G., Errico, J. M., Sjoestroem, S., Fenster, A. & Waite, J. H. A cohort of new adhesive proteins identified from transcriptomic analysis of mussel foot glands. J. R. Soc. Interface 14, 20170151 (2017)._ _ Bahr, G. F. Osmium tetroxide and ruthenium tetroxide and their reactions with biologically important substances. Exp. Cell Res. 7, 457u2013 479 (1954)._ _ _ Hayat, M. A. Principles and Techniques of Electron Microscopy. Biological Applications. (Cambridge University Press, Cambridge, 2000)Tamarin, A., Lewis, P. & Askey, J. Structure and formation of byssus attachment plaque in Mytilus. J. Morphol. 149, 199u2013221 (1976)._ _ _ Bates, F. S. & Fredrickson, G. H. Block copolymer thermodynamicsu2014theory and experiment. Annu. Rev. Phys. Chem. 41, 525u2013 557 (1990)._ _ _ _ Feric, M. et al. Coexisting liquid phases underlie nucleolar subcompartments. Cell 165, 1686u20131697 (2016)._ _ _ Holten-Andersen, N. et al. Metal-coordination: using one of natures tricks to control soft material mechanics. J. Mater. Chem. B 2, 2467u20132472 (2014)._ _ _ Creighton, T. E. Proteins: Structures and Molecular Properties. 2nd edn (W. H. Freeman and Company, New York, 1993).Yu, J. et al. Mussel protein adhesion depends on interprotein thiol-mediated redox modulation. Nat. Chem. Biol. 7, 588u2013 590 (2011)._ _ _ Kim, S. et al. Salt triggers the simple coacervation of an underwater adhesive when cations meet aromatic pi electrons in seawater. ACS Nano 11, 6764u20136772 (2017)._ _ _ Zhao, H. & Waite, J. H. Linking adhesive and structural proteins in the attachment plaque of Mytilus californianus. J. Biol. Chem. 281, 26150u2013 26158 (2006)._ _ _ Li, J. Y. & Li, H. B. Mechanical unfolding pathway of the high-potential iron-sulfur protein revealed by single-molecule atomic force microscopy: toward a general unfolding mechanism for iron-sulfur proteins. J. Phys. Chem. B 122, 9340u20139349 (2018)._ _ _ Lee, H., Scherer, N. F. & Messersmith, P. B. Single-molecule mechanics of mussel adhesion. Proc. Natl Acad. Sci. USA 103, 12999u201313003 (2006)._ _ _ _ George, S. G., Pirie, B. J. S. & Coombs, T. L. The kinetics of accumulation and excretion of ferric hydroxide in Mytilus edulis and its distribution in the tissues. J. Exp. Mar. Biol. Ecol. 23, 71u2013 84 (1976)._ _ _ \"- - The authors acknowledge funding from the Max Planck Society and the German Research Foundation (DFG individual grant HA 6369 5). M.J.H. acknowledges support from the Natural Sciences and Engineering Research Council of Canada (NSERC Discovery Grant RGPIN-2018-05243).- - Authors and Affiliations - \"Department of Biomaterials, Max Planck Institute of Colloids and Interfaces, 14424, Potsdam, GermanyFranziska Jehle,_Elena MacxEDas-SxE1nchez,_Sanja Sviben,_Peter Fratzl,_Luca Bertinetti_&_Matthew J. HarringtonDepartment of Chemistry, McGill University, 801 Sherbrooke Street West, MontrxE9al, QC, H3A 0B8, CanadaMatthew J. Harrington\" - \"AuthorsFranziska JehleYou can also search for this author in _Elena MacxEDas-SxE1nchezYou can also search for this author in _Sanja SvibenYou can also search for this author in _Peter FratzlYou can also search for this author in _Luca BertinettiYou can also search for this author in _Matthew J. HarringtonYou can also search for this author in _\" - Contributions - M.J.H., L.B. and P.F. devised and oversaw the study. L.B., F.J., S.S. and E.M.-S. performed experiments and processed data. M.J.H. and F.J. wrote the paper. All authors contributed to editing the paper. - Corresponding authors - \"Correspondence to or .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks Emily Carrington and the other anonymous reviewer(s) for their contribution to the peer review of this work. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons license (CypA/CA complex); ref. 7, under a Creative Commons license (HIV-1 capsid); ref. 153, Springer Nature Ltd (NMDA); courtesy of Lorenzo Casalino, UC San Diego (influenza A, top); ref. 144, Elsevier (25-bp DNA); ref. 149, Wiley (Fip35); ref. 152, Cell Press (influenza A, bottom); ref. 151, PLOS (Nuclear pore complex); ref. 39, Wiley (GATA4 gene).u201D\"- - The error has been corrected in the online version of the article.- - Authors and Affiliations - \"Department of Chemistry, New York University, New York, NY, USATamar Schlick_ &_Stephanie Portillo-LedesmaCourant Institute of Mathematical Sciences, New York University, New York, NY, USATamar SchlickNew York Universityu2013East China Normal University Center for Computational Chemistry at New York University Shanghai, Shanghai, ChinaTamar Schlick\" - \"AuthorsTamar SchlickYou can also search for this author in _Stephanie Portillo-LedesmaYou can also search for this author in _\" - Corresponding author - \"Correspondence to .\"- - \\'\\'- - ',\n",
       " 'Creative Commons license (HIV-1 capsid); ref. , Springer Nature Ltd (NMDA); courtesy of Lorenzo Casalino, UC San Diego (influenza A, top); ref. , Elsevier (25-bp DNA); ref. , Wiley (Fip35); ref. , Cell Press (influenza A, bottom); ref. , PLOS (Nuclear pore complex); ref. , Wiley (GATA4 gene).\" - Scientists studying chemical and biological systems, from small molecules to huge viruses, now routinely combine computer simulations and a variety of experimental information to determine or predict structures, energies, kinetics, mechanisms and functions of these fascinating and important systems. Pioneers and leaders of the field who pushed the envelopes of applications and technologies through large simulation programs and state-of-the-art methodologies have unveiled the molecules of life in action, similar to what the light microscopes and X-ray techniques did in the seventeenth and nineteenth centuries. Biomolecular modeling and simulation applications have allowed us to pose and answer new questions and pursue difficult challenges, in both basic and applied research. Problems range from unraveling the folding pathways of proteins and identification of new therapeutic targets for common human diseases to the design of novel materials and pharmaceuticals. With the recent emergence of the coronavirus pandemic, all these tools are being utilized in numerous community efforts for simulating COVID-19 related systems. Similar to the exponential growth so familiar to us now in connection with the spread of COVID-19 infections, exponential progress is only realized when we take stock of long timeframes. - \"A key element in this success is the relentless pursuit and exploitation of the state-of-the-art technology by the biomolecular simulation community. In fact, the excellent utilization of supercomputers and technology by modelers led to comparable performance for landmark simulations with the worlds fastest computers (Fig. ). The simulation time of biomolecular complexes scales up by about three orders of magnitude every decade, and this progress is faster than Moores law, which projected a doubling every two years. While some aspects of this doubling have been debated, it has also been argued that such doubling of computation/performance has held over 100u2009years in many fields.\" - \"Fig. 2: Performance of landmark simulations compared with the worlds fastest supercomputers and Moores law trend.Plot of the computational system ranked first (blue) and the highest ranked academic computer (orange) as reported in Rmax according to the LINPACK benchmark as assembled in the Top500 supercomputer lists (). Rmax is the unit used to define computer performance in TFLOPS (trillion floating point operations per second). Landmark simulations (green diamonds) are dated assuming calculations were performed about a year before publication, except for the publications in 1998, which we assumed were performed in 1996. These include, from 1996 to the present, 25-bp DNA using National Center for Supercomputing Applications (NCSA) Silicon Graphics Inc. (SGI) machines; villin protein using the Cray T3E900; bc1 membrane complex using the Cray T3E900; 12-bp DNA using MareNostrum/Barcelona; Fip35 protein using NCSA Abe clusters; nuclear core complex using Blue Waters; influenza A virus using the Jade Supercomputer; CypA/CA complex using Blue Waters; HIV-1 capsid using Titan Cray XK7; GATA4 gene using Trinity Phase 2; and influenza A virus H1N1 using Blue Waters. As Blue Waters has opted out of the Top500, we use estimates of sustained system performance/sustained petascale performance (SSP/SPP) from 2012 and 2020. For system size and simulation time of each landmark simulation, see Fig. .\" - \"Today, concurrent advances in many technological fields have led to exponential growth in allied fields. Take, for example, the dramatic drop in the cost of gene sequencing technology, from US$2.7 billion for the Human Genome Project to US$1,000 today to sequence an individuals genome. Not only can this information be used for personalized medicine, but we can also sequence genomes such as of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in hours and apply such sequence variant information to map the disease spread across the world in nearly real time. Fields such as artificial intelligence, nanotechnology, energy and robotics are all benefiting from exponential growth as well. This in turn means that urgent problems can be solved to improve our lives, health and environment, from wind farms to vaccines. In particular, biomolecular modeling and simulation is thriving in this ever-evolving landscape, evidenced by many successes, and in experiments driven by modeling.\" - \"A detailed account of this triumphant field trajectory is described separately in our recent field perspective. A review on the success of computations in this field was also recently published by Dill et al.. In our field perspective, we covered metrics of the fields rise in popularity and productivity, examples of success and failure, collaborations between experimentalists and modelers, and the impact of community initiatives and exercises. Here, we focus on two aspects of technology advances that are relevant to numerous areas of computational science at large: knowledge-based methods versus physics-based methods, and the role of hardware versus software in driving the field.\"- - \"Physics-type models based on molecular mechanics principles have been successfully applied to molecular systems since the 1960s, providing insights into structures and mechanisms involved in biomolecular rearrangements, flexibility, pathways and function. In these methods, energy functions that treat molecules as physical systems, similar to balls connected by springs, are used to express biomolecules in terms of fundamental vibrations, rotations and non-bonded interactions. Target data taken from experiments on relevant molecular entities are used to parametrize these functions, which are then applied to larger systems composed of the same basic chemical subgroups. Thus, experimental data are used in constructing these general functions but in a fundamental way, such as the nature of Cu2013O bonds or the rotational flexibility around alpha carbons.\" - Knowledge-based methods, in contrast, lack a fundamental energy framework. Instead, various structural, energetic or functional data are used to train a computer program into discovering these trends in related systems from known chemical and biophysical information on specific molecular systems. Thus, such approaches use available data to make extrapolative predictions regarding related biological and chemical systems. - While physics-based models have been in continuous usage, knowledge-based methods have gained momentum since the 2000s with the increasing amount of both available data and computational power for handling voluminous data. Although physics-based approaches remain essential for understanding mechanisms, knowledge-based methods are inevitably succeeding in specific applications and overtaking many fields of science and engineering. As we argue below, both are important to develop, and their combination can be particularly fruitful. - Physics-based methods - \"Physics-based methods offer us a conceptual understanding of biological processes. Indeed, the development of improved all-atom force fields for biomolecular simulations in both functional form and parameters has been crucial to the increasing accuracy of modeling many biological processes of large systems. Force fields for proteins, nucleic acids, membranes and small organic molecules have been applied to study problems such as protein folding, enzymatic mechanisms, ligand binding/unbinding, membrane insertion mechanisms and many others. Current fourth-generation force fields have introduced polarization effects, important for processes or systems with induced electronic polarization, such as intrinsically disordered proteins, metal/protein interactions and membrane permeation mechanisms. Despite 50u2009 years of developments, force fields are far from perfect, and further refinement, expansion, standardization and validation can be expected in the future. Transferability to a wide range of biomolecular systems and u2018convergence among different force fields will continue to be issues. In parallel to all-atom force fields, numerous coarse-grained potentials have been developed for many systems, but these are far less unified compared with all-atom force fields. Much development can be expected in the near future in this area as the complexity of biomolecular problems of interest increases.\" - \"In the area of protein structure prediction, for example, the physics-based coarse-grained united-residue (UNRES) force field developed by the lab of the late Harold Scheraga demonstrated exceptional results in predicting the orientations of domains in the tenth Critical Assessment of Protein Structure Prediction (CASP10). Such predictions were free of biases from structural databases and relied on energetically favorable residue/residue interactions (u2018first principles).\" - Physics-based methods are essential for studying protein dynamics and folding pathways. For example, UNRES and many all-atom force fields have been successful in the study of folding pathways of several proteins, including a small protein inside its chaperonin. Besides folding mechanisms, kinetic and thermodynamic parameters can be determined. - Many other areas of applications demonstrate how molecular mechanics and dynamics simulations provide insights on structures and mechanisms. These include structures of viruses (including SARS-CoV-2), pathways in DNA repair or folding of chromatin fibers. In drug discovery, molecular docking has shown to be successful for high-throughput screening. For instance, restrained-temperature multiple-copy molecular dynamics (MD) replica-exchange combined with molecular docking suggested molecules that bind to the spike protein of the SARS-CoV-2 virus. - The most common concerns in such molecular mechanics approaches involve insufficient conformational sampling and limited simulation length compared with biological timeframes. Other drawbacks are approximations due to force-field imperfections and other model simplifications, absence of adequate statistical information and the lack of general applicability to all molecular systems. Increases in computer power, advances in enhanced simulation techniques and fourth-generation force fields with incorporated polarizabilities are helping overcome these limitations. For example, the Frontera petascale computing system allowed multiple microsecond all-atom MD simulations of the SARS-CoV-2 spike glycoprotein embedded in the viral membrane. Enhanced sampling simulations combining parallel tempering with well-tempered ensemble metadynamics revealed how phosphorylation of intrinsically disordered proteins regulates their binding to their interacting partners. The polarizable force-field AMOEBA has allowed the determination of the phosphate binding mode to the phosphate binding protein, which has remained controversial for a long time. - Knowledge-based methods - Knowledge-based methods are less conceptually demanding than physics-based models and can in principle overcome the approximations of physics-based methods. - In the field of protein folding and structure prediction, knowledge-based methods, such as homology, threading and minithreading modeling have shown to be more effective than physics-based methods in some cases. Other successful algorithms use information on evolutionary coupled residues, namely, residues involved in compensatory mutations. Such information can be detected from multiple sequence alignments and used to predict protein structures de novo with high accuracy, as observed in CASP11. - In particular, the artificial intelligence approach by Google AlphaFold, a co-evolution-based method, upstaged the CASP13 exercise held in 2018, outperforming other methods for protein structure prediction (Fig. ). More recently, analyses of CASP14 (2020) results with the updated AlphaFold2 revealed unprecedented levels of accuracy across all targets. - \"Fig. 3: Applications of biomolecular modeling made possible by technology advances.a, AlphaFold workflow. Deep neural networks are trained with known structures deposited in the Protein Data Bank to predict protein structures de novo. The distribution of distances and angles are obtained and then the scores are optimized with gradient descent to improve the designs. b, RNA mesoscale modeling. Target residues for drugs or gene editing in SARS-2-CoV frameshifting element (FSE) identified by graph theory combined with all-atom microsecond MD simulations made possible by a new four-petaflop u2018Greene supercomputer at New York University. c, Chromatin mesoscale modeling. HOXC mesoscale model at nucleosome resolution constructed with experimental information on nucleosome positioning, histone tail acetylation and linker histone binding. Shown is the unfolded gene (left) and the folded structure of the gene (right). d, Cloud computing. Targeted protein sites related to SARS-CoV-2 with bound ligands from the billion compound database screened with VirtualFlow platform that makes use of cloud computing and cloud storage. Figure reproduced with permission from: a, ref. , Springer Nature America, Inc. (structure image); b, ref. , Cell Press; c, ref. , PNAS; d, , Harvard Univ. See original works for full names and abbreviations.\" - The increasing amount of high-resolution structural data for protein/ligand binding, as deposited in the Protein Data Bank, has accelerated the use of knowledge-based methods in drug discovery, a key application of biomolecular modeling. For example, the crystal structure of the main protease of the SARS-CoV-2 virus was solved unliganded and in complex with a peptide mimetic inhibitor, providing the basis for the development of improved inhibitors using knowledge-based methods. Also related to COVID-19, artificial intelligence tools have been used to identify potential drugs against SARS-CoV-2. - \"Of course, the accuracy of knowledge-based methods depends on the quality and size of the database available, similarity between the underlying database and the systems studied, and the analysis methods applied. Even in large databases, some systems are underrepresented. These include, for example, RNAs with higher-order junctions, where few experimental data exist, and intrinsically disordered proteins, which are difficult to solve by conventional X-ray or NMR techniques. Such problems may be alleviated in principle as more data become available. Nonetheless, unbalanced databases can produce erroneous results. For example, models trained with databases of ligandu2013protein complexes where ligands that bind weakly are underrepresented can overestimate binding affinities.\" - For some applications, such as deriving force fields by machine learning protocols, access to a large and diverse high-quality training dataset obtained by quantum mechanics calculations is essential to obtain reliable results for general applications. However, there are no known criteria of sufficiency. How many molecular descriptors are required to satisfactorily explain ligand binding or chemical reactivity? How many large non-coding RNAs are diverse enough to represent the universe of RNA folds for these systems? - Combined knowledge and physics-based methods - Fortunately, combinations of knowledge and physics-based approaches can merge the strengths of each technique, integrating specific molecular information with learned patterns. For instance, maximum entropy and Bayesian approaches integrate simulations with experimental data. They generate structural ensembles for the systems using MD or Monte Carlo simulations and incorporate them by imposing restraints to reproduce experimental data. - Protein-folding approaches can be improved by the use of hybrid energy functions that combine physics-based with knowledge-based components. For example, physics-based functions can be modified with structural restraints from NMR experiments, torsion angle correction terms for the backbone or side chains of residues or hydrogen-bonding potentials based on high-resolution protein crystal structures. - In protein structure refinement, combinations of physics-based and knowledge-based approaches have shown to be particularly successful. For example, in the CASP10 exercise, MD simulations from the Shaw and Zhang groups showed that experimental constraints were crucial for refining predicted structures. Pure physics-based methods were unsuccessful at correcting non-native conformations toward native states. Recently, it was reported that refinements with MD simulations of models obtained with AlphaFold substantially improve the predicted structures. - \"In computer-aided drug design, quantitative structure/property relationship (QSPR) models combine experimental and quantum mechanical descriptors to improve the prediction of Gibbs free energies of solvation. MD simulations combined with machine learning algorithms can help create improved quantitative structureu2013 activity relationship (QSAR) models.\" - In the long run, inferring mechanisms is critical for understanding and addressing complex problems in biophysics. Force fields will not likely disappear any time soon, despite the growing success of knowledge-based methods. As shown in public citizen projects, such as Foldit for protein folding, combinations of both physics-based and knowledge-based methods will probably work best. Importantly, human intuition and insight is needed to fully merge both approaches and properly interpret the computational findings.- - Rigorous and efficient algorithms are essential for the success of any biomolecular modeling or simulation. New algorithms are required to address problems as they emerge, as well as to utilize new technologies and hardware developments. Classic examples of algorithms that enhanced the reliability and efficiency of biomolecular simulations include the particle-mesh Ewald method for treatment of electrostatics and symplectic and resonance-free methods for long-time integration in MD simulations. Hardware advances, in addition, are essential for expanding system size and simulation timeframes. The continuing increase in computer power, in combination with parallel computing, has been crucial in the development of the field of biomolecular simulations. Both hardware and software will be essential to the continued success of the field. - Algorithms and software advances - Outstanding progress has been reported in developing software to enhance sampling, reduce computational cost and integrate information from machine learning and artificial intelligence methods to solve biological problems. Algorithms that utilize novel hardware such as graphics processing units (GPUs) and coupled processors have also been impactful. Enhanced sampling methods and particle-based methods such as Ewald summations have revolutionized how molecular simulations are performed and how conformational transitions can be captured, for example, to connect experimental endpoints. MD algorithms such as multiple timestep approaches, in contrast, have achieved far less impact than hardware innovations, due to a relatively small net computational gain. However, their framework may be useful in combination with other improvements such as enhanced sampling algorithms or optimized particle-mesh Ewald algorithms. The complexity and size of the biological systems of interest increases every year and thus continued algorithm development is crucial to obtain reliable methods that balance accuracy and performance. - \"Density functional theory (DFT), used in quantum mechanical (QM) applications since the 1990s, has become one of the most popular QM methods to study biomolecules. DFT has a computational cost similar to semi-empirical methods but higher accuracy. New DFT functionals are continuously being developed to improve the description of dispersion and for special applications. The high efficiency of DFT implies that larger and more complex systems can be studied, expanding the applications and predictive power of electronic structure theory, and promoting collaborations between modelers and experimentalists. This high efficiency has also been exploited by MD methods; DFT-based MD simulation methods, such as Caru2013Parrinello MD and ab initio MD, are widely applied to study electronic processes in biological systems, such as chemical reactions.\" - \"Because of the computational cost of QM methods and the large size of most biological systems, the development of combined quantum mechanics/molecular mechanics (QM/MM) methods was fundamental to advance electronic structure calculations of biological systems. In particular, computational enzymology has driven the development of these methods since the pioneering work of Warshel and Levitt on the reaction mechanism of the lysozyme. By partitioning the system into an electronic active region and the rest, which is treated at a molecular mechanical level, computational effort is centered in the part of the system where it is needed, and the overall cost is substantially reduced. Nowadays, several QM/MM methods differing in the scheme used to compute QM/MM energies, the treatment of the boundary region and the QM to MM interactions are applied to study many enzymatic mechanisms, metalu2013protein interactions, photochemical processes and redox processes, among others. Adaptive QM/MM methods that reassign the QM and MM regions on the fly have also been developed. These methods are particularly important to study ions in solution or in biomolecules, and chemical reactions in explicit solvent.\" - Recent QM/MM methods employ machine learning (ML) potentials in place of MM calculations. Such QM/ML schemes can avoid problems associated with force fields as well as boundary issues between the QM and MM regions. Other recent developments use neural networks coupled with QM/MM algorithms; the neural networks are used to predict potential energy surfaces at an ab initio/MM level from semi-empirical/MM calculations. - Many of the biological processes of interest occur on timescales that are not easily accessible by conventional MD simulations. Thus, a variety of enhanced sampling algorithms have been developed. These methods improve the sampling efficiency by reducing energy barriers and allowing the systems to escape local minima in the potential energy surface. Speedups compared with conventional MD can be around one order of magnitude or more. Methods based on collective variables such as umbrella sampling, metadynamics and steered MD have advanced the field with applications to ligand binding/unbinding, conformational changes of proteins and nucleic acids, free energy profiles along enzymatic reactions and ligand unbinding, and protein folding. Methods that do not require definition of specific collective variables or reaction coordinates, such as replica exchange MD and accelerated MD have shown to be particularly successful when defining a collective variable is difficult, for example, when exploring transition pathways and intermediate states. Markov state models (MSMs) can help describe pathways between different relevant metastable states identified by experiments or MD. For example, when studying the folding of a dimeric protein, an MSM of the metastable states on the free-energy surface has identified the states that describe the folding process, as well as the specific inter-residue interactions that can lead to kinetic traps. Physics-based protein folding has benefited from the application of MSMs that combine many short independent trajectories. Related thermodynamic integration and free-energy perturbation methods, which calculate free-energy differences between initial and final states, have also helped determine protein/ligand binding constants, membrane/water partition coefficients, pKa values and folding free energies to connect simulations to experimental measurements. - \"Enhanced sampling techniques are now being combined with machine learning to improve the selection of collective variables and to develop new methods. Clearly, artificial intelligence and ML algorithms are changing the way we do molecular modeling. Coupled with the growth of data, GPU-accelerated scientific computing and physics-based techniques, these algorithms are revolutionizing the field. Since the pioneering work of Behler and Parrinello on the use of neural networks to represent DFT potential energy surfaces and thus to describe chemical processes, ML has been applied to design all-atom and coarse-grained force fields, analyze MD simulations, develop enhanced sampling techniques and construct MSMs, among others. As discussed above, Googles AlphaFold performance in CASP13 and CASP14 showed how impactful these kinds of algorithms can be for predicting protein structure. Artificial intelligence platforms for drug discovery have also led to clinical trials for COVID-19 treatments in record times.\" - Multiscale models - A special case of algorithms that has potential to revolutionize the field involves multiscale models. Crucial for bridging the gap between experimental and computational timeframes, such models increase spatial and temporal resolution by use of coarse graining, interpolation and other ways to connect all the information on different levels. - \"The 2013 Nobel Prize in Chemistry that recognized Karplus, Levitt and Warshel for their work on developing multiscale models has underscored the importance of these models. In the 1970s, bridging molecular mechanics with quantum mechanics defined indeed a new way of simulating molecular systems. The first hybrid model of this type by Warshel and Karplus was initially intended to study chemical properties and reactions of planar molecules but was later extended to study enzymatic reactions. Todays models are numerous and varied. While useful in practice, they are generally tailored to specific systems and lack a rigorous theoretical framework.\" - For example, numerous coarse-grained protein models have been developed and applied to protein dynamics, folding and flexibility, protein structure prediction, protein interactions and membrane proteins, as recently reviewed. - Coarse-grained models have also been developed to study nucleic acids. Possibly due to small volumes of structural data, high charge density and wide structural diversity, they have progressed somewhat slower than for proteins, especially in the case of RNA. - DNA coarse-grained models allow us to study, in reasonable time, large DNA systems that could not be approached by all-atom models. The reduction in the degrees of freedom achieved by coarse-grained models has allowed the study of thousands of base pair systems in scales of microseconds to milliseconds. Crucial studies include self-assemblies of large DNA molecules, the denaturalization process, the hybridization process important for many biological functions, the topology of DNA mini-circles and the sequence-dependence of single-stranded DNA structures. - The flexibility of RNAs and the huge spectrum of possible conformations make their modeling challenging, and numerous coarse-grained models that differ in the number of beads per nucleotide and interactions included in the model and their treatment have been developed. A different coarse-grained approach using two-dimensional and three-dimensional graphs to represent RNA structure has also proven useful to analyze and design novel RNAs, including the SARS-CoV-2 frameshifting element (Fig. ). - Coarse-grained models have also been applied to biomembranes, systems of thousands of lipids that undergo large-scale transitions in the microsecond-to-millisecond regime. Membrane protein dynamics, virion capsid assembly, lipid recognition by proteins and many remodeling processes have been successfully captured in such coarse-grained applications. - \"Finally, to study DNA complexed with proteins, such as in the context of chromatin fibers, multiscale approaches are essential, as recently reviewed. These approaches derive the chromatin model from the atomistic DNA, nucleosomes and linker histones. Successful models by the groups of the late Langowski, Wedemann, NordenskixF6 ld, Olson, Spakowitz, de Pablo and ours have been applied to understand the mechanisms that regulate chromatin compaction and function. For example, our recent three-dimensional folding of the HOXC gene cluster (~55u2009kbp) at nucleosome resolution by mesoscale modeling revealed how epigenetic factors act together to regulate chromatin folding (Fig. ). The next challenge for these types of model is to merge the kilobase to megabase levels of understanding chromatin while retaining a basic dependency on the physical parameters that dictate fiber conformations.\" - Multiscale models are as much art as science, as they require subjective decisions on what parts to approximate and what parts to resolve. Yet much information guides these models, and important biological problems serve as motivators. Overall, innovative advances in both algorithms and hardware, especially in multiscale modeling, will be pivotal for the progress of the biological sciences in the coming years. - Hardware advances - The computational biology and chemistry communities have utilized hardware exceptionally well. This is evident from the expectation curve in Fig. and from the computer technology plot in Fig. . We see that hardware innovations have propelled the field of biomolecular simulations forward by around six orders of magnitude over three decades as reflected by simulation length and size of biomolecular systems. - In the first decade of the twenty-first century, hardware innovations such as the supercomputers Anton and Blue Waters propelled the field by expanding the limits of both system size and simulation time that is possible. Today, nanosecond simulations of a 160-million-atom influenza virus or the 1-billion-atom GATA4 gene have become possible. - At the same time, the introduction of GPUs for biomolecular simulations by NVIDIA broke new grounds. GPUs are specialized electronic circuits designed to rapidly manipulate and alter memory to accelerate computations. Such GPUs contain hundreds of arithmetic units and possess a high degree of parallelism, allowing performance levels tens or hundreds of times higher than a single central processing unit (CPU) core with tailored software. - \"The acceleration of MD simulations by GPU computing and supercomputers substantially reduced the gap between experimental and theoretical scales. For example, as mentioned above, the worlds second-fastest supercomputeru2014Summit from the Oak Ridge National Laboratory, with more than 27,000 NVIDIA GPUs and 9,000 IBM Power9 CPUsu2014was used to explore SARS-CoV-2 virus inhibitors among more than 8,000 compounds. Such simulations were conducted in just a few days, with 77 compound candidates found. GPU-based algorithms for free-energy calculations can achieve a speedup of 200 compared with CPU-based methods. QM/MM GPU-based methods have also accelerated calculations focused on enzymatic mechanisms. For example, GPU-based DFT in the framework of hybrid QM/MM calculations such as ONIOM or additive QM/MM realize speedup factors of 20 to 30 compared with CPU-based calculations. The Folding@home distributed computing project, dedicated to understanding the role of protein folding in several diseases, is conducting most calculations on GPUs by using simulation packages adapted to this architecture. Recently, over a million citizen scientists helped solve COVID-19 challenges; they combined ~280,000 GPUs, reaching the exascale and generating more than 0.1u2009s of simulation. These simulations helped understand how the SARS-CoV-2 virus spike surface protein attaches to the receptors in human cells. MD software adapted to GPU-accelerated architectures is also being used to perform enormous cell-scale simulations, important to mimic realistic cellular environments and to study viral and bacterial infections.\" - \"Cloud-based computing is surging as a viable alternative to supercomputers, providing researchers with remote high-performance computing platforms for large-scale simulations, analysis and visualization. Acquisition and maintenance of such hardware is not affordable for individual research groups, but feasible for institutions and companies. For example, Googles Exacycle has been used to conduct millisecond simulations of the G-protein-coupled receptor u03B2 2AR that revealed its activation pathway, important for the design of drugs to treat heart diseases. Recently, in an unprecedented study, the Google Cloud Platform and Google Cloud Storage were combined to screen around 1 billion compounds against 15 SARS-CoV-2 proteins and 2 human proteins involved in the infection (Fig. ). A high-performance version of the popular visualization program VMD has been implemented on the Amazon cloud, as well as the MD toolkit QwikMD and the molecular dynamics flexible fitting (MDFF) method for structure refinement from cryo-electron microscopy densities. These efforts allow scientists worldwide to access powerful computational equipment and software packages in a cost-effective way.\" - Overall, tailored computers for molecular simulations, such as Anton, can accelerate the calculation of computationally expensive interactions with specialized software, while general-purpose supercomputers or cloud computing that parallelize MD calculations across multiple processors with thousands of GPUs or CPUs can accelerate performance (for example, trillions of calculations per second) for large systems. - \"Although hardware advances have overwhelmed software advances, both are clearly needed for optimal performance. Hardware bottlenecks will inevitably emerge as computer storage limits are reached. Yet, whether or not Moores law will continue to be realized, software advances will always be important. Certainly, engineers and mathematicians will not be out of jobs.\" - Figure summarizes key software, hardware and algorithm developments that helped breakthrough studies. - \"Fig. 4: Key developments in algorithms, software and hardware that advanced the field.1970s: simulations of <1,000-atom systems and few picoseconds in vacuo were possible due to the development of digital computers and algorithms to treat long-range Coulomb interactions. The image depicts the structure of the small protein BPTI that was simulated for 8.8u2009ps without hydrogen atoms and with four water molecules. 1980s: simulations that considered solvent effects became possible, and algorithms such as SHAKE, to constrain covalent bonds involving hydrogen atoms, allowed the study of systems with explicit hydrogens. The image depicts the 125u2009ps simulation of the 12u2009bp DNA in complex with the lac repressor protein in aqueous solution using the simple three-point charge water model. 1990s: QM/MM methods can perform geometry optimizations, MD and Monte Carlo simulations. The image depicts the acetyl-CoA enolization mechanism by the citrate synthase enzyme studied with AM1/CHARMM. 2000s: GPU-based MD simulations, specialized supercomputers such as Anton, shared resources such as Folding@home, enhanced sampling algorithms and Markov state models (MSMs) all contributed to advance protein folding. The image depicts the long 100u2009 u03BCs simulation of the Fip35 folding conducted in Anton (red) compared with the X-ray structure (blue). 2010s: all-atom and coarse-grained MD simulations of viruses performed on supercomputers such as Blue Waters became common. The image depicts the MD simulation of the all-atom HIV capsid using the MDFF method that uses cryo-electron microscopy data to guide simulations. 2020s: physical whole-cell models are being developed to fully understand how biomolecules behave inside cells and to study interactions between them, for example, within viruses and cells. The image depicts the model of the interaction between the spike protein in the SARS-CoV-2 surface and the ACE2 receptor on a human cell surface being developed in the Amaro Lab. From left to right, images adapted with permission from: second image, ref. , Wiley; third image, ref. , Wiley; fourth image, ref. , AAAS; fifth image, ref. , Springer Nature Ltd; sixth image, , Amaro Lab.\"- - Technology has driven many advances that affect our everyday life, from cellphones and personal medical devices, to solar energy and coping in times of physical isolation during the current COVID-19 pandemic. Biomolecular modelers have consistently leveraged technology to solve important practical problems efficiently and will undoubtedly continue to do so. Machine learning and other data science approaches are now offering new tools for discovery in numerous fields. These tools for predicting structures, dynamics and functions of biomolecules can be combined with physics-based approaches not only to find solutions but also to understand associated mechanisms. Algorithms such as MSMs, neural networks, multiscale modeling, enhanced conformational sampling and comparative modeling can be leveraged as never before, especially in combination with these data-science approaches. - We expect that force-field based methods will remain essential for the understanding of mechanisms of biomolecular systems, but knowledge-based methods will certainly gain momentum. Although the recent breathtaking results from AlphaFold2 might tempt us to believe that the physics-based era is over, the range of complex problems beyond protein folding is unlikely to be easily solved by knowledge-based methods alone. Novel computing platforms will also play an important role in the future of biomolecular simulations. As quantum computing, neuromorphic computing and other architectures enter the arena, we can be sure that they will be exploited avidly by the biomolecular community. Despite the extraordinary technical impact of computers on our field and the incredible potential of artificial intelligence techniques to address many scientific problems, human intuition and intelligence will continue to be instrumental for developing ideas and pursuing new research avenues. After all, such human talent is responsible for artificial intelligence design and implementation in the first place and will probably continue to do so. - Finally, gone are the days when modelers worked in isolation. Whether on Zoom or sharing a bench, teams of multidisciplinary scientists (and likely automated machines too in the future) are collaborating to address essential problems of life, from energy to vaccines. Despite some bumps, exponential growth appears a reality in the near future, and the field of biomolecular modeling and simulation will undoubtedly continue to incorporate, innovate and digitize the inner workings of biological systems to solve the secrets of life and to develop solutions for treating human disease, improving global health and enhancing our environment.- - \\'16 November 2021A Correction to this paper has been published: \\'- - \"Schlick, T., Collepardo-Guevara, R., Halvorsen, L. A., Jung, S. & Xiao, X. Biomolecular modeling and simulation: a field coming of age. Q. Rev. Biophys. 44, 191u2013 228 (2011)._ _ Schaefer, H. F. Methylene: a paradigm for computational quantum chemistry. Science 231, 1100u20131107 (1986)._ _ Maddox, J. Statistical mechanics by numbers. Nature 334, 561 (1988)._ _ Munos, B. Lessons from 60 years of pharmaceutical innovation. Nat. Rev. Drug Discov. 8, 959u2013968 (2009)._ _ Hayden, E. C. Human genome at then: life is complicated. Nature 464, 664u2013667 (2010)._ _ Lindorff-Larsen, K., Piana, S., Dror, R. O. & Shaw, D. E. How fast-folding proteins fold. Science 334, 517u2013 520 (2011)._ _ Perilla, J. R. & Schulten, K. Physical properties of the HIV-1 capsid from all-atom molecular dynamics simulations. Nat. Commun. 8, 15959 (2017)._ _ Acharya, A. et al. Supercomputer-based ensemble docking drug discovery pipeline with application to COVID-19. J. Chem. Inf. Model. 60, 5832u20135852 (2020)._ _ Schlick, T. The 2013 Nobel Prize in Chemistry celebrates computations in chemistry and biology. SIAM News 46, 1u20134 (2013)._ Vendruscolo, M. & Dobson, C. M. Protein dynamics: Moores Law in molecular biology. Curr. Biol. 21, R68u2013R70 (2011)._ _ Moore, G. E. Cramming more components onto integrated circuits. Electronics 38, 114u2013 117 (1965).Ismail, S., Malone, M. S. & Van Geest, Y. Exponential Organizations. Why New Organizations Are Ten Times Better, Faster, and Cheaper Than Yours (and What to Do About It) (Diversion Publishing, 2014).Wetterstrand, K. A. DNA Sequencing Costs: Data from the NHGRI Large-Scale Genome Sequencing Program (NIH, 2016); Forster, P., Forster, L., Renfrew, C. & Forster, M. Phylogenetic network analysis of SARS-CoV-2 genomes. Proc. Natl Acad. Sci. USA 117, 9241u20139243 (2020)._ _ Schlick, T. et al. Biomolecular modeling and simulation: a prospering multidisciplinary field. Annu. Rev. Biophys. 50, 267u2013301 (2021)._ _ Brini, E., Simmerling, C. & Dill, K. Protein storytelling through physics. Science 370, eaaz3041 (2020)._ _ Cornell, W. D. et al. A second generation force field for the simulation of proteins, nucleic acids, and organic molecules. J. Am. Chem. Soc. 117, 5179u20135197 (1995)._ _ MacKerell, A. D., Wiorkiewicz-Kuczera, J. & Karplus, M. An all-atom empirical energy function for the simulation of nucleic acids. J. Am. Chem. Soc. 117, 11946u201311975 (1995)._ _ Oostenbrink, C., Villa, A., Mark, A. E. & Van Gunsteren, W. F. A biomolecular force field based on the free enthalpy of hydration and solvation: the GROMOS force-field parameter sets 53A5 and 53A6. J. Comput. Chem. 25, 1656u20131676 (2004)._ _ Dror, R. O., Dirks, R. M., Grossman, J. P., Xu, H. & Shaw, D. E. Biomolecular simulation: a computational microscope for molecular biology. Annu. Rev. Biophys. 41, 429u2013452 (2012)._ _ Huggins, D. J. et al. Biomolecular simulations: from dynamics and mechanisms to computational assays of biological activity. Wiley. Interdiscip. Rev. Comput. Mol. Sci. 9, e1393 (2019)._ _ Patel, S. & Brooks, C. L. CHARMM fluctuating charge force field for proteins: I parameterization and application to bulk organic liquid simulations. J. Comput. Chem. 25, 1u2013 16 (2004)._ _ Lopes, P. E. M. et al. Polarizable force field for peptides and proteins based on the classical drude oscillator. J. Chem. Theory Comput. 9, 5430u20135449 (2013)._ _ Zhang, C. et al. AMOEBA polarizable atomic multipole force field for nucleic acids. J. Chem. Theory Comput. 14, 2084u20132108 (2018)._ _ Inakollu, V. S., Geerke, D. P., Rowley, C. N. & Yu, H. Polarisable force fields: what do they add in biomolecular simulations? Curr. Opin. Struct. Biol. 61, 182u2013190 (2020)._ _ Jing, Z. et al. Polarizable force fields for biomolecular simulations: recent advances and applications. Annu. Rev. Biophys. 48, 371u2013394 (2019)._ _ Dauber-Osguthorpe, P. & Hagler, A. T. Biomolecular force fields: where have we been, where are we now, where do we need to go and how do we get there? J. Comput. Aided Mol. Des. 33, 133u2013203 (2019)._ _ van der Spoel, D. Systematic design of biomolecular force fields. Curr. Opin. Struct. Biol. 67, 18u2013 24 (2021)._ _ Noid, W. G. Perspective: coarse-grained models for biomolecular systems. J. Chem. Phys. 139, 90901 (2013)._ _ Kamerlin, S. C. L., Vicatos, S., Dryga, A. & Warshel, A. Coarse-grained (multiscale) simulations in studies of biophysical and chemical systems. Annu. Rev. Phys. Chem. 62, 41u201364 (2011)._ _ He, Y. et al. Lessons from application of the UNRES force field to predictions of structures of CASP10 targets. Proc. Natl Acad. Sci. USA 110, 14936u201314941 (2013)._ _ Maisuradze, G. G., Senet, P., Czaplewski, C., Liwo, A. & Scheraga, H. A. Investigation of protein folding by coarse-grained molecular dynamics with the UNRES force field. J. Phys. Chem. A 114, 4471u20134485 (2010)._ _ Piana, S., Lindorff-Larsen, K. & Shaw, D. E. Protein folding kinetics and thermodynamics from atomistic simulation. Proc. Natl Acad. Sci. USA 109, 17845u201317850 (2012)._ _ Miao, Y., Feixas, F., Eun, C. & McCammon, J. A. Accelerated molecular dynamics simulations of protein folding. J. Comput. Chem. 36, 1536u20131549 (2015)._ _ Piana, S. & Shaw, D. E. Atomic-level description of protein folding inside the GroEL cavity. J. Phys. Chem. B 122, 11440u201311449 (2018)._ _ Durrant, J. D. et al. Mesoscale all-atom influenza virus simulations suggest new substrate binding mechanism. ACS Cent. Sci. 6, 189u2013 196 (2020)._ _ Yu, A. et al. A multiscale coarse-grained model of the SARS-CoV-2 virion. Biophys. J. (2020).Radhakrishnan, R. et al. Regulation of DNA repair fidelity by molecular checkpoints: u2018gates in DNA polymerase u03B2s substrate selection. Biochemistry 45, 15142u2013 15156 (2006)._ _ Jung, J. et al. Scaling molecular dynamics beyond 100,000 processor cores for large-scale biophysical simulations. J. Comput. Chem. 40, 1919u20131930 (2019)._ _ Bascom, G. & Schlick, T. in Translational Epigenetics Vol. 2 (eds Lavelle, C. & Victor, J.-M.) 123u2013 147 (Academic Press, 2018).Wedemann, G. & Langowski, J. Computer simulation of the 30-nanometer chromatin fiber. Biophys. J. 82, 2847u20132859 (2002)._ _ Smith, M. D. & Smith, J. C. Repurposing therapeutics for COVID-19: supercomputer-based docking to the SARS-CoV-2 viral spike protein and viral spike proteinu2013human ACE2 interface. Preprint at (2020).Van Gunsteren, W. F. et al. Biomolecular modeling: goals, problems, perspectives. Angew. Chem. Int. Ed. 45, 4064u20134092 (2006)._ _ Casalino, L. et al. Beyond shielding: the roles of glycans in the SARS-CoV-2 spike protein. ACS Cent. Sci. 6, 1722u20131734 (2020)._ _ Liu, N., Guo, Y., Ning, S. & Duan, M. Phosphorylation regulates the binding of intrinsically disordered proteins via a flexible conformation selection mechanism. Commun. Chem. 3, 123 (2020)._ _ Qi, R. et al. Elucidating the phosphate binding mode of phosphate-binding protein: the critical effect of buffer solution. J. Phys. Chem. B 122, 6371u20136376 (2018)._ _ Warme, P. K., Momany, F. A., Rumball, S. V., Tuttle, R. W. & Scheraga, H. A. Computation of structures of homologous proteins. Alpha-lactalbumin from lysozyme. Biochemistry 13, 768u2013782 (1974)._ _ Jones, D. & Thornton, J. Protein fold recognition. J. Comput. Aided Mol. Des. 7, 439u2013 456 (1993)._ _ Rohl, C. A., Strauss, C. E. M., Misura, K. M. S. & Baker, D. Protein structure prediction using Rosetta. Methods Enzymol. 383, 66u201393 (2004)._ _ Abriata, L. A., TamxF2, G. E., Monastyrskyy, B., Kryshtafovych, A. & Dal Peraro, M. Assessment of hard target modeling in CASP12 reveals an emerging role of alignment-based contact prediction methods. Proteins 86, 97u2013112 (2018)._ _ Marks, D. S. et al. Protein 3D structure computed from evolutionary sequence variation. PLoS ONE 6, e28766 (2011)._ _ Ovchinnikov, S. et al. Improved de novo structure prediction in CASP11 by incorporating coevolution information into Rosetta. Proteins 84, 67u201375 (2016)._ _ Kryshtafovych, A., Schwede, T., Topf, M., Fidelis, K. & Moult, J. Critical assessment of methods of protein structure prediction (CASP)u2014round XIII. Proteins 87, 1011u20131020 (2019)._ _ Callaway, E. u2018It will change everything: DeepMinds AI makes gigantic leap in solving protein structures. Nature 588, 203u2013204 (2020)._ _ Zhang, L. et al. Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved u03B1-ketoamide inhibitors. Science 368, 409u2013412 (2020)._ _ Mohammad, T. et al. Identification of high-affinity inhibitors of SARS-CoV-2 main protease: towards the development of effective COVID-19 therapy. Virus Res. 288, 198102 (2020)._ _ Zhou, Y. et al. Artificial intelligence in COVID-19 drug repurposing. Lancet Digit. Health. 2, e667u2013e676 (2020)._ _ Laing, C. et al. Predicting helical topologies in RNA junctions as tree graphs. PLoS ONE 8, e71947 (2013)._ _ Durrant, J. D. & McCammon, J. A. NNScore: a neural-network-based scoring function for the characterization of proteinu2013ligand complexes. J. Chem. Inf. Model. 50, 1865u20131871 (2010)._ _ Wang, C. & Zhang, Y. Improving scoring-docking-screening powers of proteinu2013ligand scoring functions using random forest. J. Comput. Chem. 38, 169u2013177 (2017)._ _ Botu, V., Batra, R., Chapman, J. & Ramprasad, R. Machine learning force fields: construction, validation, and outlook. J. Phys. Chem. C 121, 511u2013522 (2017)._ _ Pitera, J. W. & Chodera, J. D. On the use of experimental observations to bias simulated ensembles. J. Chem. Theory Comput. 8, 3445u20133451 (2012)._ _ Hummer, G. & KxF6finger, J. Bayesian ensemble refinement by replica simulations and reweighting. J. Chem. Phys. 143, 243150 (2015)._ _ Park, H., Lee, G. R., Heo, L. & Seok, C. Protein loop modeling using a new hybrid energy function and its application to modeling in inaccurate structural environments. PLoS ONE 9, e113811 (2014)._ _ Kortemme, T., Morozov, A. V. & Baker, D. An orientation-dependent hydrogen bonding potential improves prediction of specificity and structure for proteins and proteinu2013protein complexes. J. Mol. Biol. 326, 1239u2013 1259 (2003)._ _ Raval, A., Piana, S., Eastwood, M. P., Dror, R. O. & Shaw, D. E. Refinement of protein structure homology models via long, all-atom molecular dynamics simulations. Proteins 80, 2071u20132079 (2012)._ _ Zhang, J., Liang, Y. & Zhang, Y. Atomic-level protein structure refinement using fragment-guided molecular dynamics conformation sampling. Structure 19, 1784u20131795 (2011)._ _ Heo, L. & Feig, M. High-accuracy protein structures by combining machine-learning with physics-based refinement. Proteins 88, 637u2013642 (2020)._ _ Borhani, T. N., GarcxEDa-MuxF1oz, S., Vanesa Luciani, C., Galindo, A. & Adjiman, C. S. Hybrid QSPR models for the prediction of the free energy of solvation of organic solute/solvent pairs. Phys. Chem. Chem. Phys. 21, 13706u201313720 (2019)._ _ Ash, J. & Fourches, D. Characterizing the chemical space of ERK2 kinase inhibitors using descriptors computed from molecular dynamics trajectories. J. Chem. Inf. Model. 57, 1286u20131299 (2017)._ _ Koepnick, B. et al. De novo protein design by citizen scientists. Nature 570, 390u2013394 (2019)._ _ Darden, T., York, D. & Pedersen, L. Particle mesh Ewald: an Nu22C5log(N) method for Ewald sums in large systems. J. Chem. Phys. 98, 10089u201310092 (1993)._ _ Hockney, R. W. & Eastwood, J. W. Computer Simulation Using Particles (Taylor & Francis, 1988).Verlet, L. Computer u2018experiments on classical fluids. I. Thermodynamical properties of Lennardu2013Jones molecules. Phys. Rev. 159, 98u2013103 (1967)._ _ Barth, E. & Schlick, T. Overcoming stability limitations in biomolecular dynamics. I. Combining force splitting via extrapolation with Langevin dynamics in LN. J. Chem. Phys. 109, 1617u20131632 (1998)._ _ Radhakrishnan, R. & Schlick, T. Orchestration of cooperative events in DNA synthesis and repair mechanism unraveled by transition path sampling of DNA polymerase u03B2s closing. Proc. Natl Acad. Sci. USA 101, 5970u20135975 (2004)._ _ Chen, P. Y. & Tuckerman, M. E. Molecular dynamics based enhanced sampling of collective variables with very large time steps. J. Chem. Phys. 148, 24106 (2018)._ _ Batcho, P. F., Case, D. A. & Schlick, T. Optimized particle-mesh Ewald/multiple-time step integration for molecular dynamics simulations. J. Chem. Phys. 115, 4003u20134018 (2001)._ _ Hohenberg, P. & Kohn, W. Inhomogeneous electron gas. Phys. Rev. 136, B864u2013B871 (1964)._ _ _ Kohn, W. & Sham, L. J. Self-consistent equations including exchange and correlation effects. Phys. Rev. 140, A1133u2013A1138 (1965)._ _ _ Su, N. Q. & Xu, X. Development of new density functional approximations. Annu. Rev. Phys. Chem. 68, 155u2013182 (2017)._ _ Ban, F., Rankin, K. N., Gauld, J. W. & Boyd, R. J. Recent applications of density functional theory calculations to biomolecules. Theor. Chem. Acc. 108, 1u201311 (2002)._ _ Car, R. & Parrinello, M. Unified approach for molecular dynamics and density-functional theory. Phys. Rev. Lett. 55, 2471u20132474 (1985)._ _ Marx, D. & Hutter, J. Ab Initio Molecular Dynamics: Basic Theory and Advanced Methods (Cambridge Univ. Press, 2009); Iftimie, R., Minary, P. & Tuckerman, M. E. Ab initio molecular dynamics: concepts, recent developments, and future trends. Proc. Natl Acad. Sci. USA 102, 6654u20136659 (2005)._ _ Senn, H. M. & Thiel, W. QM/MM methods for biomolecular systems. Angew. Chem. Int. Ed. 48, 1198u20131229 (2009)._ _ Warshel, A. & Levitt, M. Theoretical studies of enzymic reactions: dielectric, electrostatic and steric stabilization of the carbonium ion in the reaction of lysozyme. J. Mol. Biol. 103, 227u2013249 (1976)._ _ Carloni, P., Rothlisberger, U. & Parrinello, M. The role and perspective of ab initio molecular dynamics in the study of biological systems. Acc. Chem. Res. 35, 455u2013464 (2002)._ _ Wallrapp, F. H. & Guallar, V. Mixed quantum mechanics and molecular mechanics methods: looking inside proteins. Wiley Interdiscip. Rev. Comput. Mol. Sci. 1, 315u2013 322 (2011)._ _ Zheng, M. & Waller, M. P. Adaptive quantum mechanics/molecular mechanics methods. Wiley Interdiscip. Rev. Comput. Mol. Sci. 6, 369u2013385 (2016)._ _ Zhang, Y. J., Khorshidi, A., Kastlunger, G. & Peterson, A. A. The potential for machine learning in hybrid QM/MM calculations. J. Chem. Phys. 148, 241740 (2018)._ _ Shen, L., Wu, J. & Yang, W. Multiscale quantum mechanics/molecular mechanics simulations with neural networks. J. Chem. Theory Comput. 12, 4934u20134946 (2016)._ _ Yang, Y. I., Shao, Q., Zhang, J., Yang, L. & Gao, Y. Q. Enhanced sampling in molecular dynamics. J. Chem. Phys. 151, 70902 (2019)._ _ Liao, Q. in Progress in Molecular Biology and Translational Science Vol. 170 (eds. Strodel, B. & Barz, B.) 177u2013213 (Academic Press, 2020).Pan, A. C., Weinreich, T. M., Piana, S. & Shaw, D. E. Demonstrating an order-of-magnitude sampling enhancement in molecular dynamics simulations of complex protein systems. J. Chem. Theory Comput. 12, 1360u2013 1367 (2016)._ _ Torrie, G. M. & Valleau, J. P. Nonphysical sampling distributions in Monte Carlo free-energy estimation: umbrella sampling. J. Comput. Phys. 23, 187u2013199 (1977)._ _ Laio, A. & Parrinello, M. Escaping free-energy minima. Proc. Natl Acad. Sci. USA 99, 12562u201312566 (2002)._ _ Lu, H. & Schulten, K. Steered molecular dynamics simulations of force-induced protein domain unfolding. Proteins 35, 453u2013463 (1999)._ _ Sugita, Y. & Okamoto, Y. Replica-exchange molecular dynamics method for protein folding. Chem. Phys. Lett. 314, 141u2013151 (1999)._ _ Hamelberg, D., Mongan, J. & McCammon, J. A. Accelerated molecular dynamics: a promising and efficient simulation method for biomolecules. J. Chem. Phys. 120, 11919u201311929 (2004)._ _ Piana, S., Lindorff-Larsen, K. & Shaw, D. E. Atomistic description of the folding of a dimeric protein. J. Phys. Chem. B 117, 12935u2013 12942 (2013)._ _ Husic, B. E. & Pande, V. S. Markov state models: from an art to a science. J. Am. Chem. Soc. 140, 2386u20132396 (2018)._ _ Schwantes, C. R., McGibbon, R. T. & Pande, V. S. Perspective: Markov models for long-timescale biomolecular dynamics. J. Chem. Phys. 141, 90901 (2014)._ _ Straatsma, T. P. & Berendsen, H. J. C. Free energy of ionic hydration: analysis of a thermodynamic integration technique to evaluate free energy differences by molecular dynamics simulations. J. Chem. Phys. 89, 5876u20135886 (1988)._ _ Chipot, C. & Pohorille, A. Free Energy Calculations (Springer-Verlag, 2007).Deng, Y. & Roux, B. Computations of standard binding free energies with molecular dynamics simulations. J. Phys. Chem. B 113, 2234u20132246 (2009)._ _ Chipot, C. in New Algorithms for Macromolecular Simulation (eds. Leimkuhler, B. et al.) 185u2013 211 (Springer, 2006); SchxF6berl, M., Zabaras, N. & Koutsourelakis, P. S. Predictive collective variable discovery with deep Bayesian models. J. Chem. Phys. 150, 24109 (2019)._ _ Bonati, L., Zhang, Y. Y. & Parrinello, M. Neural networks-based variationally enhanced sampling. Proc. Natl Acad. Sci. USA 116, 17641u201317647 (2019)._ _ _ _ Zhang, J., Yang, Y. I. & NoxE9, F. Targeted adversarial learning optimized sampling. J. Phys. Chem. Lett. 10, 5791u20135797 (2019)._ _ Behler, J. & Parrinello, M. Generalized neural-network representation of high-dimensional potential-energy surfaces. Phys. Rev. Lett. 98, 146401 (2007)._ _ NoxE9, F., Tkatchenko, A., MxFCller, K. R. & Clementi, C. Machine learning for molecular simulation. Annu. Rev. Phys. Chem. 71, 361u2013 390 (2020)._ _ Warshel, A. & Karplus, M. Calculation of ground and excited state potential surfaces of conjugated Molecules. I. Formulation and parametrization. J. Am. Chem. Soc. 94, 5612u20135625 (1972)._ _ Kmiecik, S. et al. Coarse-grained protein models and their applications. Chem. Rev. 116, 7898u20137936 (2016)._ _ Dans, P. D., Walther, J., GxF3mez, H. & Orozco, M. Multiscale simulation of DNA. Curr. Opin. Struct. Biol. 37, 29u201345 (2016)._ _ Potoyan, D. A., Savelyev, A. & Papoian, G. A. Recent successes in coarse-grained modeling of DNA. Wiley Interdiscip. Rev. Comput. Mol. Sci. 3, 69u201383 (2013)._ _ u0160poner, J. et al. RNA structural dynamics as captured by molecular simulations: a comprehensive overview. Chem. Rev. 118, 4177u2013 4338 (2018)._ _ Dawson, W. K., Maciejczyk, M., Jankowska, E. J. & Bujnicki, J. M. Coarse-grained modeling of RNA 3D structure. Methods 103, 138u2013156 (2016)._ _ Schlick, T., Zhu, Q., Jain, S. & Yan, S. Structure-altering mutations of the SARS-CoV-2 frameshifting RNA element. Biophys. J. 120, 1040u20131053 (2021)._ _ Marrink, S. J. & Tieleman, D. P. Perspective on the Martini model. Chem. Soc. Rev. 42, 6801u20136822 (2013)._ _ Cascella, M. & Vanni, S. in Chemical Modelling Vol. 12 (eds. Springborg, M. & Joswig, J.-O.) 1u2013 52 (Royal Society of Chemistry, 2016).Soares, T. A., Vanni, S., Milano, G. & Cascella, M. Toward chemically resolved computer simulations of dynamics and remodeling of biological membranes. J. Phys. Chem. Lett. 8, 3586u20133594 (2017)._ _ Portillo-Ledesma, S. & Schlick, T. Bridging chromatin structure and function over a range of experimental spatial and temporal scales by molecular modeling. Wiley Interdiscip. Rev. Comput. Mol. Sci. 10, wcms.1434 (2020)._ _ Bendandi, A., Dante, S., Zia, S. R., Diaspro, A. & Rocchia, W. Chromatin compaction multiscale modeling: a complex synergy between theory, simulation, and experiment. Front. Mol. Biosci. 7, 15 (2020)._ _ Stehr, R. et al. Exploring the conformational space of chromatin fibers and their stability by numerical dynamic phase diagrams. Biophys. J. 98, 1028u20131037 (2010)._ _ Fan, Y., Korolev, N., Lyubartsev, A. P. & NordenskixF6ld, L. An advanced coarse-grained nucleosome core particle model for computer simulations of nucleosomeu2013nucleosome interactions under varying ionic conditions. PLoS ONE 8, e54228 (2013)._ _ Kulaeva, O. I. et al. Internucleosomal interactions mediated by histone tails allow distant communication in chromatin. J. Biol. Chem. 287, 20248u2013 20257 (2012)._ _ MacPherson, Q., Beltran, B. & Spakowitz, A. J. Bottom-up modeling of chromatin segregation due to epigenetic modifications. Proc. Natl Acad. Sci. USA 115, 12739u201312744 (2018)._ _ Lequieu, J., CxF3rdoba, A., Moller, J. & de Pablo, J. J. 1CPN: a coarse-grained multi-scale model of chromatin. J. Chem. Phys. 150, 215102 (2019)._ _ Bascom, G., Myers, C. & Schlick, T. Mesoscale modeling reveals formation of an epigenetically driven hoxc gene hubs. Proc. Natl Acad. Sci. USA 116, 4955u20134962 (2018)._ _ Stone, J. E., Hardy, D. J., Ufimtsev, I. S. & Schulten, K. GPU-accelerated molecular modeling coming of age. J. Mol. Graph. Model. 29, 116u2013125 (2010)._ _ Harger, M. et al. Tinker-OpenMM: absolute and relative alchemical free energies using AMOEBA on GPUs. J. Comput. Chem. 38, 2047u20132055 (2017)._ _ JxE1sz, xC1., RxE1k, xC1., LadjxE1nszki, I., Tornai, G. J. & Cserey, G. Towards chemically accurate QM/MM simulations on GPUs. J. Mol. Graph. Model. 96, 107536 (2020)._ _ Nitsche, M. A., Ferreria, M., Mocskos, E. E. & Lebrero, M. C. G. GPU accelerated implementation of density functional theory for hybrid QM/MM simulations. J. Chem. Theory Comput. 10, 959u2013967 (2014)._ _ Active CPUs and GPUs by OS. Folding@home (accessed 1 March 2021).Zimmerman, M. I. et al. Citizen scientists create an exascale computer to combat COVID-19. Preprint at bioRxiv (2020).Acun, B. et al. Scalable molecular dynamics with NAMD on the summit system. IBM J. Res. Dev. 62, 1u20139 (2018)._ _ Kohlhoff, K. J. et al. Cloud-based simulations on Google Exacycle reveal ligand modulation of GPCR activation pathways. Nat. Chem. 6, 15u201321 (2013)._ _ Gorgulla, C. et al. A multi-pronged approach targeting SARS-CoV-2 proteins using ultra-large virtual screening. iScience 24, 102021 (2021)._ _ Stone, J. E., Messmer, P., Sisneros, R. & Schulten, K. High performance molecular visualization: in-situ and parallel rendering with EGL. In Proc. 2016 IEEE 30th International Parallel Distributed Processing Symposium 1014u2013 1023 (IEEE, 2016).Ribeiro, J. V. et al. QwikMDu2014integrative molecular dynamics toolkit for novices and experts. Sci. Rep. 6, 26536 (2016)._ _ Singharoy, A. et al. Molecular dynamics-based refinement and validation for sub-5u2009xC5 cryo-electron microscopy maps. Elife 5, e16105 (2016)._ _ Shaw, D. E. et al. Anton, a special-purpose machine for molecular dynamics simulation. Commun. ACM 51, 91u201397 (2008)._ _ Young, M. A. & Beveridge, D. L. Molecular dynamics simulations of an oligonucleotide duplex with adenine tracts phased by a full helix turn. J. Mol. Biol. 281, 675u2013687 (1998)._ _ Duan, Y. & Kollman, P. A. Pathways to a protein folding intermediate observed in a 1-microsecond simulation in aqueous solution. Science 282, 740u2013744 (1998)._ _ Izrailev, S., Crofts, A. R., Berry, E. A. & Schulten, K. Steered molecular dynamics simulation of the Rieske subunit motion in the cytochrome bc1 complex. Biophys. J. 77, 1753u20131768 (1999)._ _ PxE9rez, A., Luque, F. J. & Orozco, M. Dynamics of B-DNA on the microsecond time scale. J. Am. Chem. Soc. 129, 14739u201314745 (2007)._ _ Freddolino, P. L., Liu, F., Gruebele, M. & Schulten, K. Ten-microsecond molecular dynamics simulation of a fast-folding WW domain. Biophys. J. 94, L75u2013 L77 (2008)._ _ Zanetti-Polzi, L. et al. Parallel folding pathways of Fip35 WW domain explained by infrared spectra and their computer simulation. FEBS Lett. 591, 3265u20133275 (2017)._ _ Shaw, D. E. et al. Atomic-level characterization of the structural dynamics of proteins. Science 330, 341u2013346 (2010)._ _ Gamini, R., Han, W., Stone, J. E. & Schulten, K. Assembly of Nsp1 nucleoporins provides insight into nuclear pore complex gating. PLoS Comput. Biol. 10, e1003488 (2014)._ _ Reddy, T. et al. Nothing to sneeze at: a dynamic and integrative computational model of an influenza a virion. Structure 23, 584u2013 597 (2015)._ _ Song, X. et al. Mechanism of NMDA receptor channel block by MK-801 and memantine. Nature 556, 515u2013519 (2018)._ _ Liu, C. et al. Cyclophilin A stabilizes the HIV-1 capsid through a novel non-canonical binding site. Nat. Commun. 7, 10714 (2016)._ _ Senior, A. W. et al. Improved protein structure prediction using potentials from deep learning. Nature 577, 706u2013710 (2020)._ _ Miao, Y. et al. Accelerated structure-based design of chemically diverse allosteric modulators of a muscarinic G protein-coupled receptor. Proc. Natl Acad. Sci. USA 113, E5675u2013E5684 (2016)._ _ Rollins, N. J. et al. Inferring protein 3D structure from deep mutation scans. Nat. Genet. 51, 1170u20131176 (2019)._ _ McCammon, J. A., Gelin, B. R. & Karplus, M. Dynamics of folded proteins. Nature 267, 585u2013 590 (1977)._ _ de Vlieg, J., Berendsen, H. J. C. & van Gunsteren, W. F. An NMRu2010based molecular dynamics simulation of the interaction of the lac repressor headpiece and its operator in aqueous solution. Proteins 6, 104u2013127 (1989)._ _ Field, M. J., Bash, P. A. & Karplus, M. A combined quantum mechanical and molecular mechanical potential for molecular dynamics simulations. J. Comput. Chem. 11, 700u2013733 (1990)._ _ Lane, T. J., Shukla, D., Beauchamp, K. A. & Pande, V. S. To milliseconds and beyond: challenges in the simulation of protein folding. Curr. Opin. Struct. Biol. 23, 58u201365 (2013)._ _ Ode, H., Nakashima, M., Kitamura, S., Sugiura, W. & Sato, H. Molecular dynamics simulation in virus research. Front. Microbiol. 3, 258 (2012)._ _ Mulholland, A. J. & Richards, W. G. Acetyl-CoA enolization in citrate synthase: a quantum mechanical/molecular mechanical (QM/MM) study. Proteins 27, 9u201325 (1997)._ _ Zhao, G. et al. Mature HIV-1 capsid structure by cryo-electron microscopy and all-atom molecular dynamics. Nature 497, 643u2013646 (2013)._ _ \"- - Support from the National Institutes of Health, National Institutes of General Medical Sciences Awards R01GM055264 and R35-GM122562, NSF RAPID Award (2030377) from the Division of Mathematical Sciences, and Philip-Morris USA Inc. and Philips-Morris International to T.S. are gratefully acknowledged. Computing support from NYU HPC facility and team was instrumental for our own research throughout the decades.- - Authors and Affiliations - \"Department of Chemistry, New York University, New York, NY, USATamar Schlick_ &_Stephanie Portillo-LedesmaCourant Institute of Mathematical Sciences, New York University, New York, NY, USATamar SchlickNew York Universityu2013East China Normal University Center for Computational Chemistry at New York University Shanghai, Shanghai, ChinaTamar Schlick\" - \"AuthorsTamar SchlickYou can also search for this author in _Stephanie Portillo-LedesmaYou can also search for this author in _\" - Contributions - T.S. conceived the manuscript. T.S. and S.P.-L. contributed to the writing and revision of the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Computational Science thanks the anonymous reviewers for their contribution to the peer review of this work. Fernando Chirigati was the primary editor on this Perspective and managed its editorial process and peer review in collaboration with the rest of the editorial team. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - ',\n",
       " 'Creative Commons license CC BY 3.0. - \"The optimized assay was first tested on control DNA samples containing SMN1 only (NA17117), SMN2 only (NA23255), or both SMN1 and SMN2 (NA03815). Representative images of the dPCR results from the analysis software (Fig._A) shows strong resolution of the partitions. All SMN probes (SMN1 exon 7, SMN2 exon 7 and total SMN intron 1) were normalized against the reference gene (RPPH1) that contains 2 copies per genome. The scatter plots for SMN1 exon 7 (Fig._B), SMN2 exon 7 (Fig._C), and total SMN intron 1 (Fig._D) shows robust separation between the target and reference probes. The multiplex SMA genotyping assay run on the Absolute Q shows high specificity and sensitivity in classifying the control samples with SMN1:SMN2 copy numbers of 0:3 (NA23255), 3:0 (NA17117), and 1:1 (NA03815). For NA03815, the Absolute Q dPCR data showed copy numbers for SMN1 exon 7, SMN2 exon 7, and total SMN intron 1 as 1.1, 1.1 and 2.2, respectively.\" - \"Figure 4Representative partition image (A) and scatter plots (Bu2013D) of the SMA 4-color multiplexing dPCR assay using NA03815 genomic DNA as the sample. The scatter plots are shown for SMN1 exon 7 (B; FAM), SMN2 exon 7 (C; VIC) and total SMN intron 1 (D; TYE665), all relative to the reference gene RPPH1 (TAMRA). The analysis software subtracted the pre-PCR image intensities from the post-PCR image intensities so as to remove fluorescent signals which did not exhibit amplification behavior, thereby eliminating false positive signals. If the pre-PCR partitions were brighter than the post-PCR partitions, then the analysis software would record negative fluorescent units. These negative units, however, did not impact the copy number quantification results.\" - \"The Absolute Q dPCR assays require less template genomic DNA (2.5_ng) than the QuantStudio 3D dPCR assays which uses 30u201360_ng of EcoRI-digested genomic DNA. Genomic DNA up to 25_ng was also tested on the Absolute Q. There are no apparent differences in SMN1 and SMN2 copy number quantification completed with 2.5_ng template or with 25_ng genomic DNA (Supplementary Table ).\" - Quantification of control gDNA SMN1 and SMN2 copy numbers using absolute Q dPCR - \"Copy numbers of SMN1, SMN2 and total SMN in 10 genomic DNA test samples from Coriell Cell Repositories were quantified using the SMA multiplex 4-color assay. All 10 Coriell DNA samples were run in triplicate on the Absolute Q dPCR System for assay verification and repeatability. The intra-assay variabilityu2014 measured by %CVu2014between the SMN1 (Fig._A), SMN2 (Fig._B) and total SMN (Fig._C) is low, demonstrating strong repeatability. The intraclass correlation coefficients (ICCs) were 0.993 (95% confidence interval (CI) 0.979u20130.998) for the SMN1 assay, 0.995 (95% CI 0.985u20130.999) for the SMN2 assay and 0.987 (95% CI 0.963u20130.996) for the total SMN assay. The SMN1 and SMN2 copy numbers measured with this multiplex assay are concordant with those provided by Coriell Cell Repositories, including one sample with 5 copies of SMN2. Additionally, the 6 copies of total SMN for NA03814 correctly matched the corresponding sums of SMN1 and SMN2 copy numbers. The total SMN copy numbers in all of the samples except for NA11254 were equal to their corresponding sums of SMN1 and SMN2 copy numbers.\" - \"Figure 5Repeatability of the Absolute Q dPCR assays for SMA genotyping. The control DNA samples (nu2009=u200910) from Coriell Cell Repositories were assayed for SMN1 (A), SMN2 (B) and total SMN (C) copy numbers. Each sample was assayed in triplicate. The intra-assay precision, measured by %CV, for each assay was listed below each sample on the plots. The intra-assay precision measurements were not calculated (N/C) for those samples with a copy number less than 1.00.\" - SMN1 and SMN2 copy number validation of blinded clinical samples - \"Absolute Q and QuantStudio 3D array dPCR SMA assays were compared on a set of blinded samples derived from patient lines (nu2009=u200915). This set of samples contained gDNAs from 13 SMA patients and 2 non-SMA controls. Using the SMN1 copy number assay, the Absolute Q dPCR correctly identified all SMA samples with homozygous deletions of SMN1 (12 out of 15) within the cohort. The Absolute Q dPCR identified one of the patient-derived sample (MND10) as containing 1 copy of SMN1 and one copy of SMN2. This sample was from an individual harboring a SMN1(p.A2G) missense mutation. The Absolute Q dPCR SMN1 (Fig._A), SMN2 (Fig._ B) and total SMN (Fig._C) copy number measurements of the blinded clinical samples were concordant with those results obtained using QuantStudio 3D array dPCR, including a sample (MND12) with 5 copies of total SMN. Bland Altman analysis of the results from SMN1 (Fig._D), SMN2 (Fig._E) and total SMN (Fig._F and Table ) probes demonstrated strong agreement between Absolute Q and QuantStudio 3D dPCR assays. For each assay, 93% of the test samples fell within the limits of agreement. Additionally, the total SMN copy numbers from 14 blinded clinical samples agree with the sum of the copy numbers from SMN1 and SMN2 assays. In a single sample (MND03), the total SMN copy number at exon 7 (4 copies) was greater than the sum of the SMN1 exon 7 (0 copies) and SMN2 exon 7 (2 copies) copy numbers. Based on this result, this sample was hypothesized to have 2 copies of either SMN1 or SMN2 containing a partial deletion of exon 7. The partial deletion of exon 7 in this sample was confirmed orthogonally by junctional PCR (data not shown) and by GS.\" - \"Figure 6Agreement between the QuantStudio 3D and the Absolute Q dPCR assays. (Au2013C) Scatterplots for SMN1 (A), SMN2 (B) and total SMN (C) copy numbers measured with the QuantStudio 3D and the Absolute Q dPCR assays. The line of equality for each set of assays is shown as a dashed line in each plot. (Du2013 F) Bland Altman difference plots for SMN1 (D), SMN2 (E) and total SMN (F) assays. For each assay, the bias is shown as a solid blue line that is labeled as the mean and the limits of agreement defined and labeled as the meanu2009xB1u2009 1.96SD (standard deviation).\" - Table 1 SMN1 exon 7, SMN2 exon 7 and total SMN intron 1 copy numbers for patient-derived cell lines measured by QuantStudio 3D array dPCR and combination absolute Q dPCR.- - Accurate quantification of SMN1 and SMN2 copy numbers is essential for the diagnosis of SMA as well as for the development of therapeutic strategies to treat the disease. This study describes the development of a multiplex dPCR assay that provides accurate and rapid SMN1 and SMN2 copy number quantification. The Absolute Q dPCR System provides reliable measurements of SMN1 and SMN2 copy numbers with a workflow that is similar to qPCR, reducing hands-on time and time-to-results when compared to other currently available dPCR platforms. - \"Some dPCR copy number variation assays rely on digestion of the template genomic DNA with a restriction endonuclease. This is because sequence repeats of two or more copies can be in close proximity to each other, increasing the probability that they will be partitioned in the same microchamber and result in a miscalculation of the copy number. Restriction endonuclease digestion of the template DNA helps to reduce the probability of proximal gene copies partitioning in the same microchamber. In this study, SMN1 and SMN2 copy numbers were accurately measured without restriction endonuclease digestion of the sample genomic DNA. Each target-specific assay was designed with distinct fluorophores that can reliably differentiate between SMN1 and SMN2; even though these genes share highly homologous DNA sequences. Additionally, the high partition numbers of the microfluidic device (20,000 partitions) and the low DNA input amount (~u20092 to 3_ng per reaction) facilitate single molecule partitioning. Finally, SMN1 and SMN2 genes, each approximately 28_kb in length, are both located in the same region of chromosome 5 but they are separated from each other by about 875_kb of genomic sequence. Mechanical sheering of the genomic DNA during DNA sample isolation may further fragment this region thereby reducing the likelihood of both genes being in the same partition. Eliminating the restriction digestion step in the copy number assay, therefore, simplifies the workflow of the assay. Future studies would assess the effects of specimen source and DNA isolation procedure on the need for either a shearing or digestion step prior to dPCR analysis.\" - In addition to eliminating the need for a pre-PCR digestion step, the dPCR workflow is simplified by integration of multiple systems into a single instrument. The first-generation of dPCR systems have separate instruments for microfluidic partitioning of the PCR reactions and template DNA, PCR amplification and raw data acquisition. This required significant hands-on-time and training for each unique workflow, which, in turn, increased the probability of operator error or process variability. The Absolute Q dPCR System combines sample partitioning, thermocycling, and raw data acquisition into a single benchtop instrument. As a result, the Absolute Q dPCR System has an average run-time that is 73% faster than first-generation dPCR systems. The shortened run-time along with minimal hands-on operation could facilitate platform adoption into diagnostic and prognostic settings. - \"In addition to determining SMN1 and SMN2 copy numbers, dPCR can also identify partial deletions in SMN1/SMN2 and gene conversion events that generate hybrid SMN genes. By comparing total SMN copy number to the sum of SMN1 exon 7 and SMN2 exon 7 copy numbers, partial deletions of SMN1 or SMN2 can be identified. One of the patient-derived cell lines tested in this study (MND003) contained a partial deletion of exon 7, which was confirmed orthogonally (data not shown). Additionally, one of the control samples (NA11254) possibly contains a partial deletion as the total SMN copy number was found to be 3 at intron 1 while the sum of SMN1 and SMN2 copy numbers at exon 7 was 2. The discrepant results with this sample are not a consequence of a lack of specificity in the intron 1 primers or probe since the in silico specificity scores for these sequences were high, i.e. all had a BLAT score of 22, which exceeds the minimal threshold of 20. These results need to be confirmed orthogonally by junctional PCR or GS in order to verify that the partial deletion is indeed the reason for the discrepancy. Partial SMN deletions, particularly resulting from the loss of exons 7 and/or 8 (SMN1/2u039478), have been previously reported in the literature using several approaches including MLPA and GS. Gene conversion events can create hybrid SMN genes where part of the gene is SMN1 and another part is SMN2. The most commonly identified hybrid gene is SMN1 with c.840T (i.e. SMN2), however there are other SMN1/SMN2 hybrid gene variations. Other SMN1/SMN2 hybrid genes have different exon 7 inclusion efficiencies compared to SMN1 or SMN2. It is important to correctly identify hybrid SMN genes as they can potentially lead to unexpected responses to current therapeutic options. Future work involving the development of a panel of assays targeting the single nucleotide difference between SMN1 and SMN2 at exon 8, as well as the intronic single nucleotide differences, would be useful for the identification of partial deletions in SMN genes as well as hybrid SMN genes.\" - \"A limitation of the copy number assay described in this study is its inability to detect silent SMA carriers (2u2009+u20090) where both copies of SMN1 are present in cis on the same chromosome. A single nucleotide polymorphism within SMN1 (g.27134T>G; rs143838139) is associated with the cis configuration of SMN1. In addition, this dPCR assay cannot detect intragenic point mutations in SMN1, which account for 3u20135% of SMA cases. Future studies are needed to develop dPCR assays that are able to detect these single nucleotide changes. These assays could potentially be integrated into an expanded multiplexing assay panel alongside the current SMN1/SMN2 differentiating dPCR assays described here.\" - \"Newborn screening of SMA is becoming more available and routine in the United States as well as many other countries. Dried blood spots (DBS) are a common source material for SMA newborn screening. A 3-mm diameter DBS typically yieldsu2009 ~u2009140_ng gDNA (in a volume of 50 u03BCL) using column-based DNA extraction. This study demonstrates that SMN1 and SMN2 copy numbers can be accurately measured with as little as 2u20133_ng gDNA, so the Absolute Q and MAP plates could reliably be used for orthogonal validation of SMA carrier status or SMN2 copy number. As the technology is further developed, quadraplex dPCR could potentially become a primary SMA screening tool.\" - The acquisition cost for the Absolute Q dPCR System is comparable to high-end qPCR instrumentation, and somewhat lower than other currently available dPCR platforms. In this study, the consumable (SMA reagents and MAP plates) costs were $2.36 per test. This is less than typical per-sample costs for available droplet dPCR systems ($3.80 per sample) and other technologies such as mass spectrometry ($3.00 per sample). These per-sample costs are exclusive of the cost associated with labor, which is reduced with the lower hands-on time of the Absolute Q dPCR System. - The Absolute Q dPCR quadruplex assay approach could be extended to identification of copy number variations associated with other diseases, in addition to SMA. dPCR has been used to identify rare gene variants as well as differences in copy number of multiple genes associated with pediatric-onset disorders (reviewed in). We have demonstrated that this system can identify a rare single nucleotide variant (EGFR(T790M)) associated with non-small cell lung carcinoma as well as fusion genes (BCR-ABL1 and CCDC88C-FLT3) that are associated with different types of cancer. - \"Accurate quantification of SMN1 and SMN2 copy numbers is essential for the proper diagnosis of SMA and the identification of carriers. Similarly, accurate measurement of SMN2 copy number is essential to determine therapeutic regimens for treating children with SMA. Digital PCRu2014including the Absolute Q dPCR System presented hereu2014can accurately, reliably and rapidly quantify SMN1 and SMN2 copy numbers in genomic DNA samples. Future work will assess the feasibility of this dPCR platform for diagnostic as well as prognostic purposes using well-established assay validation approaches and different types of DNA samples.\"- - DNA samples - \"Control samples of human genomic DNA (nu2009=u200910) were obtained from Coriell Cell Repositories (Camden, NJ; Table ). Most of these samples (nu2009=u2009 8) had well characterized SMN1 and SMN2 copy numbers provided by Coriell Cell Repositories. Among the 10 samples, 5 were derived from SMA patients with 0 copies of SMN1. The copy numbers for NA11254 and NA17117 were measured using high-resolution melting analysis and array digital PCR (personal communications). NA03814 and NA03815 samples were from known SMA carriers, i.e. 1 copy of SMN1.\" - Table 2 Control DNA samples from the coriell cell repositories previously measured SMN1 and SMN2 copy numbers for the control DNA samples were obtained from Coriell cell repositories. - \"Patient-derived samples (nu2009=u200915) were procured from the Motor Neuron Diseases Research Laboratory (Nemours Alfred I. duPont Hospital for Children). Genomic DNA was isolated from patient-derived fibroblast and lymphoblastoid cell lines and the SMN1 and SMN2 copy numbers were measured using the QuantStudio 3D Digital PCR System (Life Technologies, Waltham, MA) as described previously. Approval for cell line generation was provided by the Nemours Institutional Review Board and this study is registered on ClinicalTrials.gov (NCT01754441 and NCT02532244). This study was approved by the Nemours Institutional Review Board (#764456). Written informed consent or assent was obtained for each patient-derived cell line. Each cell line was de-identified. For all samples tested, the tester was blinded to the previously measured SMN1 and SMN2 copy numbers. All procedures performed in this study involving human participants were in accordance with the ethical standards of the institutional and/or research committees and with the 1964 Helsinki declaration and its later amendments of comparable ethical standards.\" - Absolute Q digital PCR system and microfluidic array partitioning (MAP) plates - \"The Combinati Absolute Q Digital PCR System consists of Microfluidic Array Partitioning (MAP) consumable plates and a fully integrated instrument that automates partitioning of reagents in the plate, PCR thermocycling and 5-color fluorescence image acquisition. The MAP plate has a standard microplate footprint capable of running up to 16 samples per dPCR run. Each unit is loaded with 10 xB5L of sample mixture and more than 90% of the loaded sample is partitioned and analyzed inu2009~u2009 20,000 pico-scale partitions. The partition volume in MAP plates is defined by the physical dimensions of the microarray chambers and not by a stochastic process such as fluid emulsions, which is important because consistent partition volume is a critical component of the dPCR statistical correction model. The physical array ensures that all samples across all plates yieldu2009~u200920,000 analyzed partitions and minimal sample is lost to dead volume or compromised partitions.\" - \"The Absolute Q digital PCR System has a walk-away workflow identical to traditional qPCR. The MAP plate is loaded via pipette with 10 u03BCL of PCR mix and an overlay of 10 u03BCL Isolation Buffer in each well. The wells are then capped with specialized gaskets. The plate is then placed into the Absolute Q tray and retracted into the system. The Absolute Q uses positive pressure from an on-board compressor to partition the sample within the MAP plate without the need for microfluidic valves, sealing films or other moving parts. The MAP slides are constructed of a cyclo olefin polymer (COP; 80_u03BCm) film that seals the microfluidic features that are molded into a separate piece of thicker material. Four identical slides are bonded to a rigid, microtiter format plate frame that includes the loading wells to complete the plate assembly. The thin film becomes gas permeable when positive pressure is applied to a well containing reagents. As the reagent enters the microfluidic features, air is passed out of the partition through the film. This allows reagent to completely fill dead-ended partitions and prevents any bubbles from forming inside of the microfluidic features. The Isolation Buffer overlay follows the reagent and physically separates the reagent reaction volumes to complete the partitioning. Positive pressure applied to the consumable during the PCR thermocycling prevents any evaporation and ensures that bubbles will not form and disrupt the isolated micro-reactions.\" - \"Before and after PCR thermocycling, entire arrays are imaged with up to five optical channels configured for the most commonly used dyes, including a ubiquitous quality control dye (ROX) used to verify proper partition filling and finding. The images taken before PCR are subtracted from the after-PCR images to remove background noise. Combinati Analysis software automatically applies optical crosstalk compensation and classifies the partitions using a convoluted neural network algorithm to eliminate false positives/ negatives and ensure robust quantification results. The full dPCR process occurs within the single benchtop instrument without operators interaction after setting up the protocol parameters (Fig._). The streamlined workflow reduces the potential for contamination, minimizes human handling errors, and reduces the time to result. No fluids ever contact instrument components, so minimal system maintenance is required.\" - Digital PCR primer and probe design - \"The primer and probe oligos were designed using PrimerQuest Tool (Integrated DNA Technologies, Coralville, IA). All primers were checked for target sequence specificity using NCBI Primer-BLAST and UCSC In-Silico PCR (, last accessed May 27, 2020) using default settings. For Primer-BLAST, each primer was required to have at least 2 total mismatches to unintended targets, including at least 2 mismatches with the last 5 base pairs at the 3u2032 end. For In-Silico PCR, the minimum match size was 15 and a minimum BLAT score of 20, using the GRCh38/hg38 human genome assembly. Primers and probes were also evaluated for primer dimers and cross primer interactions using Multiple Primer Analyzer (Thermo Fisher Scientific, Waltham, MA) with the optimal sensitivity value (3) for dimer detection. All of the oligos were synthesized by Integrated DNA Technologies and Thermo Fisher Scientific. The assay includes a total of 3 pairs of primers and 4 probes with the SMN1 and SMN2 targets sharing a common pair of primers. The FAM-labeled SMN1 and VIC-labeled SMN2 probes were designed utilizing the signature c.840 single nucleotide difference between exon 7 of SMN1 (c.840C) and SMN2 (c.840T). The oligos for total SMN copy number were designed upstream in intron 1 targeting both SMN1 and SMN2 genes, and the probe was labeled with TYE665. The TAMRA-labeled probe for RNase P (RPPH1; OMIM #608513) targeted the single exon region of RPPH1 gene (Fig._). All probes have either QSY or IBRQ dark quenchers on their 3u2032 ends (Table ).\" - Table 3 List of primers and probes used for the multiplex dPCR assay. - Digital PCR - \"The control DNA samples were diluted to 10_ng/xB5L working stock. The blinded DNA samples were measured using Qubit 4 fluorometer (Invitrogen) and diluted to 10_ng/xB5L. The PCR reaction mix contained 1X Combinati MasterMix, 2.5u2013 25_ng human gDNA, 1800_nmol/L SMN1/SMN2 primers, 900_nmol/L RPPH1 primers, 900_nmol/L total SMN primers, and 250_nmol/L of each probe. Each MAP plate well was loaded by hand via pipette with 10 xB5L of PCR reaction mix then overlaid with 10 xB5L Isolation Buffer. Gasket caps were placed and the plate was put into the Absolute Q and run with the following conditions: 3_min activation at 95_xB0C, 40 cycles of 5_s at 95_xB0C and 30_s at 62_xB0C. The fully automated Absolute Q Control Software (v1.0.20, ) controls the sample digitization into the MAP partitions, thermal cycling, and imaging.\" - Data analysis - \\'The copy numbers were determined using the Absolute Q Analysis Software (v10.5.4). The software automatically calculates the optimal positive/ negative threshold and in this case the positive signals were five times higher than the negative signals. For each unit, the software displays the total viable partition count and the positive partition count. The software has a setting to identify the sample types as a copy number variation assay and applies the appropriate calculations to automatically display the copy numbers for the samples. The copy numbers of SMN1, SMN2 and total SMN are normalized by the reference control RPPH1 using the following equations:\\' - $$SMN1 mathrm{CN}=2 frac{lnleft(frac{N}{N-{N}_{1}}right)}{lnleft(frac{N}{N-{N}_{4}}right)}$$ - $$SMN2 mathrm{CN}=2 frac{lnleft(frac{N}{N-{N}_{2}}right)}{lnleft(frac{N}{N-{N}_{4}}right)}$$ - $$mathrm{total} SMN mathrm{CN}=2 frac{lnleft(frac{N}{N-{N}_{3}}right)}{lnleft(frac{N}{N-{N}_{4}}right)}$$ - \"where Nu2009=u2009total number of viable partitions, N1u2009=u2009number of SMN1 positive partitions, N2u2009=u2009number of SMN2 positive partitions, N3u2009=u2009number of total SMN positive partitions and N4u2009=u2009number of RPPH1 positive partitions.\" - Statistical analysis - The coefficient of variability (%CV), defined as the standard deviation divided by the mean value for each set of replicates, was used to assess repeatability, or intra-assay precision. Reliability was also measured by the intraclass correlation coefficient (ICC) using SPSS v.25 (IBM, Armonk, NY). A Bland Altman agreement analysis using SigmaPlot v.12.0 (Systat Software, Inc., San Jose, CA) was used to measure the agreement between dPCR methods.- - \\' \\' - The datasets generated or analyzed during the current study are available from the corresponding author on reasonable request. - \\' \\'- - \"Crawford, T. O. & Pardo, C. A. The neurobiology of childhood spinal muscular atrophy. Neurobiol. Dis. 3, 97u2013110 (1996)._ _ _ _ Tisdale, S. & Pellizzoni, L. Disease mechanisms and therapeutic approaches in spinal muscular atrophy. J. Neurosci. 35, 8691u20138700 (2015)._ _ _ _ _ Pearn, J. Incidence, prevalence and gene frequency studies of chronic childhood spinal muscular atrophy. J. Med. Genet. 15, 409u2013413 (1978)._ _ _ _ _ CuscxF3, I. et al. Prenatal diagnosis for risk of spinal muscular atrophy. Br. J. Obstet. Gynaecol. 109, 1244u20131249 (2002)._ _ Lefebvre, S. et al. Identification and characterization of a spinal muscular atrophy-determining gene. Cell 80, 155u2013165 (1995)._ _ _ _ Lorson, C. L., Hahnen, E., Androphy, E. J. & Wirth, B. A single nucleotide in the SMN gene regulates splicing and is responsible for spinal muscular atrophy. Proc. Natl. Acad. Sci. USA 96, 6307u20136311 (1999)._ _ _ _ _ _ Monani, U. R. et al. A single nucleotide difference that alters splicing patterns distinguishes the SMA gene SMN1 from the copy gene SMN2. Hum. Mol. Genet. 8, 1177u20131183 (1999)._ _ _ _ Kolb, S. J. & Kissel, J. T. Spinal muscular atrophy. Neurol. Clin. 33, 831u2013846 (2015)._ _ _ _ Butchbach, M. E. R. Copy number variations in the survival motor neuron genes: implications for spinal muscular atrophy and other neurodegenerative diseases. Front. Mol. Biosci. 3, 7 (2016)._ _ _ _ _ _ Chiriboga, C. A. et al. Results from a phase I study of nusinersen (ISIS-SMNrx) in children with spinal muscular atrophy. Neurology 86, 890u2013897 (2016)._ _ _ _ _ Finkel, R. S. et al. Treatment of infantile-onset spinal muscular atrophy with nusinersen: a phase 2, open-label, dose-escalation study. Lancet 388, 3017u2013 3026 (2016)._ _ _ _ HachxE9, M. et al. Intrathecal injections in children with spinal muscular atrophy: nusinersen clinical trial experience. J. Child Neurol. 31, 899u2013906 (2016)._ _ _ _ Mendell, J. R. et al. Single-dose gene-replacement therapy for spinal muscular atrophy. N. Engl. J. Med. 377, 1713u20131722 (2017)._ _ _ _ Sturm, S. et al. A phase 1 healthy male volunteer single escalating dose study of the pharmacokinetics and pharmacodynamics of risdiplam (RG7916, RO7034067), a SMN2 splicing modifier. Br. J. Clin. Pharmacol. 85, 181u2013193 (2019)._ _ _ _ Glascock, J. et al. Treatment algorithm for infants diagnosed with spinal muscular atrophy through newborn screening. J. Neuromuscul. Dis. 5, 145u2013158 (2018)._ _ _ _ Glascock, J. et al. Revised recommendations for the treatment of infants diagnosed with spinal muscular atrophy via newborn screening who have 4 copies of SMN2. J. Neuromuscul. Dis. 7, 97u2013100 (2020)._ _ _ _ FeldkxF6tter, M., Schwarzer, V., Wirth, R., Wienker, T. F. & Wirth, B. Quantitative analyses of SMN1 and SMN2 based on real-time LightCycler PCR: fast and highly reliable carrier testing and prediction of severity of spinal muscular atrophy. Am. J. Hum. Genet. 70, 358u2013368 (2002)._ _ _ Arkblad, E. L. et al. Multiplex ligation-dependent probe amplification improves diagnostics in spinal muscular atrophy. Neuromuscul. Disord. 16, 830u2013838 (2006)._ _ _ Huang, C. H. et al. Copy number analysis of survival motor neuron genes by multiplex ligation-dependent probe amplification. Genet. Med. 9, 241u2013248 (2007)._ _ _ GxF3mez-Curet, I. et al. Robust quantification of the SMN gene copy number by real-time Taqman PCR. Neurogenetics 8, 271u2013278 (2007)._ _ _ _ AlxEDas, L. et al. Accuracy of marker analysis, quantitative real-time polymerase chain reaction and multiple ligation-dependent probe amplification to determine SMN2 copy number in patients with spinal muscular atrophy. Genet. Test. Mol. Biomarkers 15, 587u2013594 (2011)._ _ _ _ Prior, T. W., Nagan, N., Sugarman, E. A., Batish, S. D. & Braastad, C. Technical standards and guidelines for spinal muscular atrophy testing. Genet. Med. 13, 686u2013694 (2011)._ _ _ Chen, X. et al. Spinal muscular atrophy diagnosis and carrier screening from genome sequencing data. Genet. Med. 22, 945u2013953 (2020)._ _ _ _ _ Butchbach, M. E. R. Applicability of digital PCR to the investigation of pediatric-onset genetic disorders. Biomol. Detect. Quant. 10, 9u201314 (2016)._ _ Sykes, P. J. et al. Quantitation of targets for PCR by use of limiting dilution. Biotechniques 13, 444u2013449 (1992)._ _ _ Vogelstein, B. & Kinzler, K. W. Digital PCR. Proc. Natl. Acad. Sci. USA 96, 9236u20139241 (1999)._ _ _ _ _ _ Stabley, D. L. et al. SMN1 And SMN2 copy numbers in cell lines derived from patients with spinal muscular atrophy as measured by array digital PCR. Mol. Genet. Genom. Med. 3, 248u2013 257 (2015)._ _ _ Vidal-Folch, N. et al. Multiplex droplet digital PCR method applicable to newborn screening, carrier status and assessment of spinal muscular atrophy. Clin. Chem. 64, 1753u20131761 (2018)._ _ _ _ Dueck, M. E. et al. Precision cancer monitoring using a novel, fully integrated, microfluidic array partitioning digital PCR platform. Sci. Rep. 9, 19606 (2019)._ _ _ _ _ _ Baer, M., Nilsen, T. W., Costigan, C. & Altman, S. Structure and transcription of a human gene for H1 RNA, the RNA component of human RNase P. Nucleic Acids Res. 18, 97 (1990)._ _ _ _ _ Stabley, D. L. et al. Establishing a reference dataset for the authentication of spinal muscular atrophy cell lines using STR profiling and digital PCR. Neuromuscul. Disord. 27, 439u2013446 (2017)._ _ _ _ Kent, W. J. BLAT: the BLAST-like alignment tool. Genome Res. 12, 656u2013664 (2002)._ _ _ _ _ Vijzelaar, R. et al. The frequency of SMN gene variants lacking exon 7 and 8 is highly population dependent. PLoS ONE 14, E0220211 (2019)._ _ _ _ _ Burghes, A. H. M. When is a deletion not a deletion? When it is converted. Am. J. Hum. Genet. 61, 9u2013 15 (1997)._ _ _ _ _ Kashima, T., Rao, N. & Manley, J. L. An intronic element contributes to splicing repression in spinal muscular atrophy. Proc. Natl. Acad. Sci. USA 104, 3426u20133431 (2007)._ _ _ _ _ _ Wu, X. et al. A-44G transition in SMN2 intron 6 protects patients with spinal muscular atrophy. Hum. Mol. Genet. 26, 2768u20132780 (2017)._ _ _ _ _ Luo, M. et al. An ashkenazi jewish SMN1 haplotype specific to duplication alleles improves pan-ethnic carrier screening for spinal muscular atrophy. Genet. Med. 16, 149u2013156 (2014)._ _ _ _ Chien, Y. H. et al. Presymptomatic diagnosis of spinal muscular atrophy through newborn screening. J. Pediatr. 190, 124u2013129 (2017)._ _ _ Kraszewski, J. N. et al. Pilot study of population-based newborn screening for spinal muscular atrophy in New York state. Genet. Med. 20, 608u2013613 (2018)._ _ _ Durie, D. et al. Quantification of DNA in neonatal dried blood spots by adenine tandem mass spectrometry. Anal. Chem. 90, 801u2013806 (2018)._ _ _ _ Lin, Y. et al. Newborn screening for spinal muscular atrophy in China using DNA mass spectrometry. Front. Genet. 10, 1255 (2019)._ _ _ _ _ Vidal-Folch, N. et al. A droplet digital PCR method for severe combined immunodeficiency newborn screening. J. Mol. Diagn. 19, 755u2013 765 (2017)._ _ _ _ Milosevic, D. et al. Applying standard clinical chemistry assay validation to droplet digital PCR quantitative liquid biopsy testing. Clin. Chem. 64, 1732u20131742 (2018)._ _ _ _ Ye, J. et al. Primer-BLAST: a tool to design target-specific primers for polymerase chain reaction. BMC Bioinform. 13, 134 (2012)._ _ _ Bland, J. M. & Altman, D. G. Statistical methods for assessing agreement between two methods of clinical measurement. Lancet 327, 307u2013310 (1986)._ _ \"- - We would like to thank those SMA patients and their families who provided samples for this study. This work was supported by Combinati, the Nemours Foundation (M.E.R.B.) and the National Institutes of Health (R44OD023028 to P.H. and P20GM103464 to M.E.R.B.).- - Authors and Affiliations - \"Combinati Inc., 2450 Embarcadero Way, Palo Alto, CA, 94303, USALingxia Jiang,_ Robert Lin,_Steve Gallagher,_Andrew Zayac_&_Paul HungCenter for Applied Clinical Genomics, Nemours Biomedical Research, Nemours Alfred I. duPont Hospital for Children, Wilmington, DE, USAMatthew E. R. ButchbachCenter for Pediatric Research, Nemours Biomedical Research, Nemours Alfred I. duPont Hospital for Children, Wilmington, DE, USAMatthew E. R. ButchbachDepartment of Pediatrics, Sidney Kimmel College of Medicine, Thomas Jefferson University, Philadelphia, PA, USAMatthew E. R. ButchbachDepartment of Biological Sciences, University of Delaware, Newark, DE, USAMatthew E. R. Butchbach\" - \"AuthorsLingxia JiangYou can also search for this author in _Robert LinYou can also search for this author in _Steve GallagherYou can also search for this author in _Andrew ZayacYou can also search for this author in _Matthew E. R. ButchbachYou can also search for this author in _Paul HungYou can also search for this author in _\" - Contributions - L.J. and M.E.R.B. conceived and designed the experiments. L.J., R.L., S.G., A.Z., M.E.R.B. and P.H. performed the experiments and analyzed the data. L.J., R.L., A.Z., M.E.R.B. and P.H. wrote the paper. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons license CC-BY 3.0; Ben Davis, ROu2013In the Smashicons Garden 2 - Solid Collection (soil improvers); Wahyuntitle, ID (ornamental); Hamishu2013In the Environments & Nature Collection (hedges and shelters); By Icongeek26u2013In the Fruits and vegetables Collection (human food); Adrien Coquet, FR (food additives); H V P (vertebrate food); Tomi Triyana, ID (leaf) / Kiran Shastry, IN (silkworm) (invertebrate food); Vectors Marketu2013In the Beach and Camping Glyph Icons Collection (fuelwood); Arthur Shlain, RUu2013In the Charcoal Collection (charcoal); By Icongeek26u2013In the Power Collection (biofuels); By Firza Alamsyah, IDu2013 In the Autumn Collection (timber); By Symbolon, ITu2013In the Pixa Collection (stems/cane); iconixaru2013In the Sewing - Solid Collection (fibre); kareemovic3000 (seeds and fruits); Marco Galtarossa, IT (tannins and dyestuffs); Bakunetsu Kaitou2013In the Construction Collection (resins and gums); Nikita Kozin, RUu2013 In the Car Service Filled Collection (lipids); GeoNeo1, GBu2013In the Christmas Collection (waxes); DinosoftLabu2013In the Shopping and E-commerce Glyphs icons vol 1 Collection (scents); Xinh Studiou2013In the Simplie plants Collection (rubber tree) / Gan Khoon Layu2013In the Farmer Farming Agriculture Plantation Industry Collection (rubber tapper) (rubber); Mavadee, THu2013In the Hospital Collection (medicines); Diego Naive, BR (poison) / Peter van Driel, NLu2013 In the An Icon I need tomorrow Collection (rat) (vertebrate poison); Diego Naive, BR (poison) / By SBTS, INu2013In the Smartfarm BlackFill Collection (bug) (invertebrate poison); ProSymbols, USu2013In the Cells, Organs, Medical Cannabis Glyph Icons Collection (smoking materials and drugs); vanilau2013In the a Collection (yin yang) / ani rofiqah, IDu2013In the tree Collection (tree) (symbolism, magic and inspiration).\"- - Authors and Affiliations - \"Universidad de AlcalxE1, Department of Life Sciences, Global Change Ecology and Evolution Group, AlcalxE1 de Henares, Madrid, SpainRafael Molina-Venegas\" - \"AuthorsRafael Molina-VenegasYou can also search for this author in _\" - Contributions - RM-V is the only author of the research. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The author declares no competing interests. - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons license Non-Commercial 3.0. (B) Macroscopic appearance of a first-generation TEHV after in-vitro culture. (C) In the first-generation TEHV design the leaflets are exposed to compressive strain (dark blue areas in the coaptation and hinge regions) in radial direction. Image was re-adapted from Sanders et al. under the Creative Commons license 4.0. (D) Schematic of the cross-section of a second-generation TEHV, characterized by a more profound hinge region (dotted square). Image was created using the software Tinkercad under Creative Commons license Non-Commercial 3.0. (E) Macroscopic appearance of a second-generation TEHV after in-vitro culture performed with an insert to impose valve geometry. (F) In second-generation TEHV design radial tissue compression was limited to the hinge region, thereby not affecting the leaflets coaptation area. Image was re-adapted from Sanders et al. under the Creative Commons license 4.0. - Remarkably, after TPVR in sheep, second-generation TEHVs demonstrated clinical-grade and preserved long-term performance with the absence of relevant regurgitation and only minimal leaflet shortening, as predicted by the computational models. While both generations of TEM-based valves showed profound and continuous remodeling potential over time, comprising host cellular infiltration, neo-matrix deposition, and endothelialization, first-generation TEHVs presented with maladaptive remodeling phenomena (i.e. leaflet shortening and thickening) starting several weeks post-implantation. To the contrary, second-generation, computational modeling-inspired TEHVs, exhibited tissue remodeling towards native-like configurations. - \"So far, previous studies have identified two important parameters that are relevant in the remodeling process of TEHVs. First, the geometry, which influences the stress and strain distribution in the leaflets thereby determining long-term valve functionality. Second, the presence and distribution of u03B1-smooth muscle actin positive (u03B1-SMA) cells within the valves, which if over-expressed in the leaflets, induce leaflet retraction, leading to dysfunction and ultimate valve failure. However, despite the identification of these predictors, the impact and role of inflammatory cells in the remodeling processes, that determines failure or success of the implanted TEM-based TEHVs is not yet fully understood. Hence, the in-depth understanding of the underlying inflammatory processes, especially during the early stages of the remodeling process, are mandatory to determine and guide the outcome thereby enabling safe clinical translation of TEM-based TEHVs.\" - \"Inflammatory host cells such as monocyte-derived naxEFve macrophages can polarize in response to the local mechanical and biochemical environmental stimuli and/or implanted biomaterial, and can differentiate into a variety of cells that have specific biological functions. The extreme ends of this polarization range are referred to as the pro-inflammatory macrophage 1 (M1) and the anti-inflammatory macrophage 2 (M2) type. While M1 macrophages are typically present during an inflammatory phase, M2 cells are primarily active during reparative and constructive remodeling.\" - \"Macrophages play a pivotal role in the initiation and perpetuation of inflammation and remodeling processes, thereby determining either the chronicity or the resolution of the inflammatory processu2014and this has been demonstrated also for tissue-engineered implants such as vascular grafts. In these studies, M2 macrophages were shown to help the recruitment of tissue remodeling cells (e.g.: smooth muscle cells, endothelial cells (ECs), and other immune cells) by expressing anti-inflammatory cytokines, finally leading to adaptive and functional remodeling. To the contrary, the constant presence of the M1 phenotype, which gives rise to a chronic inflammation and scar tissue formation, will finally result into a negative or maladaptive remodeling.\" - \"In this study, we assessed the role and impact of inflammatory host cells on the (mal)-adaptive remodeling processes observed in the hinge area of TEM-based TEHVs. To achieve this, we performed an independent, in-depth qualitative and quantitative (immuno-)histological analysis in tissue samples derived from the hinge area of our previous first- and second-generation TEHV preclinical studies (Fig._). We analyzed key cellular players of the inflammatory and remodeling cascades such as M1 and M2 macrophages, u03B1-SMA cells, and ECs. We focused our analysis on the hinge region of the TEHVs as this area is exposed to a maximum degree of mechanical strain, which is known to be a strong regulator of valvular cell behavior and differentiation,that in turn strongly influences the (mal)adaptive remodeling phenomena in TEHVs.\"- - First-generation TEHV analyses - Qualitative evaluation of inflammatory host cell infiltration - \"H&E and MG staining showed abundant cell infiltration over time throughout the valve (Fig._ and Supplementary Fig._). In the explanted TEHVs from the acute animals, no cellular infiltration within the valve tissue was visible (Fig._ A,E,I); the only cells detected were those trapped in the fibrin thrombus (Fig._ I, star symbol), i.e. erythrocytes (Fig._I, triangle) and leukocytes (Fig._ I, arrow). At 8_weeks, cells were predominantly present in the wall and hinge regions of the valve (Fig._B,F). In addition, our analysis revealed a progressive colonization of cells in the hinge region (at 16 and 24_weeks), which further develop towards the leaflets (Fig._C,G,D,H). The analysis of the area surrounding the scaffold remnants in the hinge region showed the presence of fibrous tissue of heterogeneous thickness, with eosinophilic cells containing a multilobed nucleus (Fig._J, white arrow), multifocal ordered macrophages containing blue granules in the cytoplasm (Fig._J,L, arrow head), multinucleated giant cells (Fig._L, cross symbol), and isolated lymphocytes (small spherical nucleus with abundant dark staining) forming lymph follicles (Fig._Ju2013L), all indicators of an ongoing chronic inflammatory response of the host towards the implanted TEHV and signs of incomplete polymer degradation at 24_weeks. Over time, u03B1 -SMA expression decreased in the wall region and gradually increased towards the hinge region and the leaflet (Fig._Mu2013P). Abundant infiltration of CD3 T lymphocytes was observed at the 8_weeks follow-up (Fig._R). After 16 (Fig._ S) and 24_weeks (Fig._T), lymphocytes distribution changed from random distribution to multifocal, with clusters detected around polymer leftovers or in follicles. Presence of endogenous ECs was demonstrated via SEM and immunohistochemistry for CD31 and VWF (Supplementary Fig._) and was in line with the previously reported results for our first-generation TEHVs. Interestingly, at 8_weeks inhomogeneous EC coverage was observed via SEM, CD31 and VWF staining (Supplementary Fig._Au2013E). In addition, we observed a large number of platelets (VWF) entrapped in the pannus (Supplementary Fig._B). At later time-points (16 and 24_weeks), the valves demonstrated a confluent, homogenous and persistent spindle-shaped EC layer covering the entire surface (Supplementary Fig._Fu2013O) as also confirmed by the immunohistochemistry for CD31 and VWF (Supplementary Fig._ Fu2013G and Ku2013L). Importantly, the CD31 ECs in the 16_weeks explants expressed the pro-thrombotic factor vWF (Supplementary Fig._G). Cutting artefacts may have prevented the complete depiction of ECs and platelets presence at 24_ weeks for the CD31 and vWF stainings (Supplementary Fig._Ku2013L), while ECs confluency could be confirmed by SEM imaging (Supplementary Fig._Mu2013O).\" - \"Figure 2Hinge region of first-generation TEHVs at different time-points. Representative images of the H&E staining performed on the cross section of the valve. (Au2013 D) 5u2009xD7u2009magnification pictures of the hinge region of the valve (scale bars 500_u03BCm). (Eu2013H) 20u2009xD7u2009magnification pictures of the hinge region marked by the dotted rectangles in panels (Au2013D) (scale bars 100_u03BCm). (Iu2013L) 40u2009xD7u2009magnification images of the hinge region marked by dotted rectangles in panels (Eu2013H) (scale bars 50_u03BC m). Presence of contractile cells is detected with immunohistochemical images at 8 (N), 16 (O), and 24 (P) weeks. u03B1-SMAcells are already detected in the hinge area at the earliest time point considered and, over time, these cells are also present in the leaflet. Acute control valve (M) does not present u03B1 -SMAcells. Scale bars: 500_u03BCm, zoom: 5u2009xD7u2009. Presence of T-lymphocytes is detected with immunohistochemical images at 8 (R), 16 (S), and 24 (T) weeks. Acute control valve (Q) does not present T-lymphocytes. T-lymphocytes cluster around polymer remnants or in follicles (yellow arrow). Scale bars: 100_u03BC m, zoom: 10u2009xD7u2009. In panel I arrow points at leukocytes, star symbol indicates fibrin, and triangle marks erythrocytes. In panel (J) arrow heads point at macrophages and the white arrow indicates eosinophils. In panel (L), the cross symbol shows multinucleated giant cells and the arrowhead macrophages.\" - Quantitative evaluation of macrophages in TEHV hinge area - \"Contrarily to what is observed for the control wall of TEHVs, the amount of cell infiltration in the hinge region, measured by cell nuclei quantification with DAPI-staining, did not gradually decreased over time (Fig._Au2013D). The number of stained M1 slightly decreased over time, whereas M2 macrophages dramatically increased beyond 8_weeks approaching the total number of M1 at 24_weeks. However, an overall prevalence of the M1 over the M2 phenotype is observed (Fig._E), suggesting an ongoing chronic inflammatory response in the hinge area of the TEHVs.\" - \"Figure 3Analysis of cell phenotype for the hinge area of first-generation TEHVs. Immunofluorescence images at the hinge area stained for M1 (red) and M2 (green) macrophages at 8 (A), 16 (B), and 24 (C) weeks follow-up. (D) Total cell number quantification in the wall and hinge region of first-generation TEHVs is showing an abundant cell presence indicating a high ongoing inflammatory response. (E) Quantification of M1 and M2 macrophages in the hinge region at the different time-points considered shows a prevalence of M1 over M2 subpopulation, with the number of M1 macrophages gradually decreasing over time while M2 macrophages is increasing. For immunofluorescence pictures: scale bars: 100_u03BCm, zoom 1u2009xD7u2009. For every measurement, cell number is normalized per u03BC m.\" - Morphological evaluation - \"Morphological changes of the TEHV in terms of leaflet length and hinge thickness were evaluated for all available time points (Table ). Compared to the leaflet length in the acute animals (controls) (Fig._A, black column), leaflets showed an initial shortening of 19% at the 8-weeks time point (Fig._A, grey bar) which then further progressed over time to a total shortening of approximately 70% at 24-weeks time point (Fig._A, white bar), which led to TEHVs functional failure. The hinge thickness increased over time with a total thickening of 66.5% at 24_weeks in-vivo when compared to the hinge thickness in the acute animals (controls) (Fig._B).\" - Table 1 Overview of the available samples for first- and second-generation TEHVs. (*) Unpublished data derived from an independent pilot study, in which TEHVs were cultured using the same culture conditions. - \"Figure 4Tissue geometry of first-generation TEHVs. (A) Quantification of leaflet lengths from histological sections of first-generation TEHVs shows gradual leaflet shortening over time. (B) Quantification of hinge thickness from histological sections of first-generation TEHVs demonstrates increased values over time. Acute control TEHVs represent the initial leaflet length and hinge thickness of acutely-implanted valves. Measurements are expressed as average valueu2009 xB1u2009standard deviation.\" - Second-generation TEHV analysis - Qualitative evaluation of inflammatory host cell infiltration - \"H&E and MG staining showed few and sparse infiltrated cells in the hinge region at any time point considered (Fig._ and Supplementary Fig._). In the acute explant, few cells were detected in the fibrin clot (Fig._A,D,G): erythrocytes (Fig._G, triangle) and leukocytes (Fig._G, arrow). After 8_weeks in-vivo, cells were predominantly present in the wall regions of the valve, close to the native tissues (Fig._B,E). At 52_weeks follow-up, instead, cells were homogenously distributed in the different valve regions, with some cellular aggregates in the proximity of the few polymeric scaffold remnants (Fig._C,F). Cells expressing u03B1-SMA were only detected in the wall area of the valve, suggesting the presence of a functional vascular wall. Independently of the time point considered, few to none u03B1-SMA cells were detected in the hinge and leaflet region (Fig._Ju2013L). In second-generation TEHVs, CD3 T lymphocytes were overall less abundant compared to first-generation valves (Fig._Mu2013 O). Most of the CD3 T lymphocytes were detected 8_weeks after implantation, accumulated in the pannus at the hinge area (Fig._N). After 52_weeks, only sporadic CD3 cells were detected (Fig._O). Finally and in line with our previously reported results, second-generation TEHVs were covered by a confluent EC layer already after 8_weeks, as shown by SEM imaging and immunohistochemistry (Supplementary Fig._). Most importantly, the confluent endothelium was retained up to 52_ weeks.\" - \"Figure 5Hinge region of second-generation TEHVs at different time-points. Representative images of the H&E staining performed on the cross section of the valve. (Au2013 C) 5u2009xD7u2009magnification pictures of the hinge region of the valve (scale bars 500_u03BCm). (Du2013F) 20u2009xD7u2009magnification pictures of the hinge region marked by the dotted rectangles in panels (Au2013C) (scale bars 100_u03BCm). (Gu2013I) 40u2009xD7u2009magnification of the hinge region marked by dotted rectangles in panels (Du2013F) (scale bars 50_u03BCm). In panel (F), SP points at a hole left by a suture point. In panel (G), arrow points at leukocytes and triangle indicates erythrocytes. Immunohistochemical images of the hinge area of second-generation TEHVs stained for u03B1-SMA at 8_weeks (K), and 52_weeks (L) follow-up. For this novel valve geometry, few to none u03B1-SMA cells are detected in the hinge area or in the leaflet at the time-point considered. Acute control valve (G) does not present u03B1-SMA cells. Scale bars: 500_u03BCm, zoom: 5u2009xD7u2009. Presence of T-lymphocytes is detected with immunohistochemical images at 8 (N) and 52 (O) weeks. Scale bars: 100_ u03BCm, zoom: 10u2009xD7u2009.\" - Quantitative evaluation of macrophages in TEHV hinge area - \"Quantification of the cell nuclei stained with DAPI (Fig._Au2013B) revealed a low number of cells in the hinge region at both 8_weeks and 52_weeks, values comparable to those of the wall region of the valve (Fig._C). Overall, a minor presence of macrophages, with a prevalence of M1 over M2 subpopulation (Fig._ D) was observed, suggesting a generally negligible inflammation of the hinge area.\" - \"Figure 6Analysis of cell phenotype for the hinge area of second-generation TEHVs. Immunofluorescence images at the hinge area stained for M1 (green) and M2 (red) macrophages at 8_weeks (A) and 52_weeks (B) follow-up. (C) Total cell number quantification in the wall and hinge region of second-generation TEHVs. (D) Quantification of M1 and M2 macrophages in the hinge region at the different time-points considered shows a mild prevalence of M1 over M2 positive cells. For immunofluorescence pictures: scale bars: 100_u03BCm, zoom 1u2009xD7u2009 . For every measurement, cell number is normalized per u03BCm.\" - Morphological evaluation - \"Comparable to what was reported for first-generation THEVs, second-generation TEHVs displayed an initial 23% shortening of the leaflet length at 8_weeks. However, and in contrast to the first-generation, the shortening did not progress over time suggesting that an equilibrium of leaflet length was achieved in the 52_weeks tissue samples (Fig._A, Supplementary Table ), which is an important parameter to ensure long-term valve performance. In addition, the hinge region (Fig._B) showed a 52% increase of thickness at 8_weeks, which however normalized to the initial control values of the acute explants after 52_weeks (Fig._ B, Supplementary Table ).\" - \"Figure 7Tissue geometry of second-generation TEHVs. (A) Quantification of leaflet lengths from histological sections of second-generation TEHVs. (B) Quantification of hinge thickness from histological sections of second-generation TEHVs. Acute TEHVs represent the initial leaflet length and hinge thickness of acutely-implanted valves. Measurements are expressed as average valueu2009xB1u2009standard deviation.\"- - Inflammation is an inescapable process, which naturally follows implantation of any device. Tissue-engineered implants designed for in-situ approaches aim at inducing a controlled inflammatory response to ultimately enable orchestrated healing and remodeling cascades. Indeed, the impact of host cells on the ongoing inflammatory response and remodeling process changes over time and strongly depends on the type of implanted biomaterial, its geometry, and also on the overall hemodynamic conditions to which it is subjected. - \"In this study, we investigated the inflammatory host cell response in two different generations of TEM-based TEHV explants, which have been previously implanted as pulmonary valve replacements in a translational sheep model. To better understand that, we performed an independent, in-depth qualitative and quantitative (immuno-)histological tissue analysis focusing on the key cellular players of the inflammatory and remodeling processes comprising M1 and M2 macrophages, u03B1-SMA cells, lymphocytes, and ECs. In agreement with previously published studies, we report the active role of inflammatory and immune cells in both the regenerative and the maladaptive remodeling of TEM-based TEHVs. Our results indicate that for first-generation TEHVs, the abundance of macrophages in the hinge area is associated with a considerable number of lymphocytes and u03B1-SMA cells and resulting tissue fibrosis. To the contrary, the physiological-like geometry of second-generation TEHVs resulted in a milder inflammation with a reduced presence of macrophages, lymphocytes, and u03B1-SMA cells. These results highlight how a milder inflammatory response (represented by a limited number of macrophages, lymphocytes, and u03B1-SMA cells in the tissue) resulting from a more physiological-like geometry might elicit an adaptive remodeling of the implanted TEM-based TEHV. On the other hand, a strong inflammatory response with abundance of macrophages, as observed for the simplified first-generation TEHV geometry, can cause a maladaptive tissue remodeling, with leaflet thickening and shortening. Our main findings have been summarized in Fig._, where we highlight the differences in valve performance, in leaflet remodeling and, importantly, in the presence of M1 and M2 macrophages, u03B1-SMA cells, and ECs in first- or second-generation TEHVs depending on the adopted geometry.\" - \"Figure 8Summary of host cell infiltration behavior in response to the geometries of first- and second-generation TEM-based TEHVs. (A) We hypothesize that first-generation TEHVs presented a non-physiological geometry, which hindered the physiological formation of blood vortexes (arrows), thereby favoring the deposition of erythrocytes and leukocytes in the hinge region shortly after implantation. At 8_weeks follow-up, first-generation TEHVs showed sustained short-term performance with high expression of M1 macrophages and u03B1-SMA cells markers. These inflammatory cells were mostly localized in the implanted TEM material, while few to no cells were detected in the newly formed pannus at the hinge area. Endothelialization was incomplete at this stage. After 24_weeks follow-up, first-generation TEHVs exhibited moderate to severe insufficiency, with a chronic inflammation and extensive maladaptive remodeling of the TEM material and pannus. This maladaptive remodeling was associated with evident hinge region thickening, abundant M1 and M2 macrophages, and u03B1 -SMA cells expression. Image was created using the software Tinkercad under Creative Commons license Non-Commercial 3.0 and Servier Medical Art under Creative Commons license 3.0. (B) Second-generation TEHVs presented a computational modeling-inspired geometry, which allowed a more physiological performance of the TEHVs after implantation, thereby possibly favoring vortex formation and blood wash-out from the hinge area. After 8_weeks, second-generation TEHVs displayed sustained performance, with few M1 macrophages and negligible presence of M2 and u03B1 -SMA cells in the hinge region. At this timepoint, endothelialization of the valve surfaces, and in particular, of the hinge area was complete. At 52_weeks, second-generation TEHVs still exhibited excellent performance, with a similar cellular composition to the 8_weeks timepoint, comprising M1 macrophages and negligible presence of M2 and u03B1-SMA cells. This inflammatory response highlighted the resolution and adaptive positive remodeling by the host towards second-generation TEM-based TEHVs. Image was created using the software Tinkercad under Creative Commons license Non-Commercial 3.0 and Servier Medical Art under Creative Commons license 3.0.\" - Maladaptive remodeling of first-generation TEHVs - \"u03B1-SMA cells have been repeatedly reported as a key cell type involved in leaflet shortening and retraction of TEHVs, however their exact role has never been fully elucidated so far. In this study, we have identified two potential regulators that may play an important role in the infiltration and/or differentiation of u03B1-SMA cells in the hinge area of our first-generation TEHVs: macrophages and ECs.\" - \"In first-generation valves, the chronic inflammatory response was confirmed by a general abundance of M1 over M2 macrophages, and an overall greater number of macrophages and lymphocytes when compared to second-generation TEHVs. The ongoing chronic inflammation may be explained by the presence of polymeric scaffold remnants until the 24_weeks follow-up, that are characterized by surrounding fibrous tissue, eosinophils, macrophages, multinucleated giant cells and lymphocytes (Fig._), thus impacting on the degree of inflammation of the host towards the implant.\" - \"Macrophages are known to actively impact on the degree and severity of damage and fibrosis by recruiting u03B1-SMA cells to the inflammation site, and by the transition from macrophages to myofibroblasts. Macrophages are responsible for the release of pro-inflammatory cytokines and metalloproteinases that enhance the inflammatory response and influence the in-vivo wound healing by contributing to myofibroblasts proliferation and recruitment. Briefly, in response to a tissue injury, macrophages first express the pro-inflammatory M1 phenotype, releasing TNF-u03B1, IL-1u03B2, IL-12, and IL-23. Subsequently, M2 macrophages take over by secreting IL-10 and TGF-u03B2 to reduce the inflammation. During this process, macrophages express a complex phenotype, where cell polarization changes from M1 to M2 over time, thereby actively regulating fibrotic tissue formation and wound healing. In the presented study, we observed a steady presence of M1 macrophages, associated to an increase in M2 macrophages over time in first-generation TEHVs. Interestingly, M2 macrophages have been repeatedly reported to also influence fibrosis, by undergoing macrophage-to-myofibroblast transition, by promoting the fibrogenic activities of fibroblasts, and by inducing u03B1-SMA expression in fibroblasts. In addition, in-vitro experiments co-culturing macrophages and u03B1-SMA cells demonstrated reduced apoptotic rates for both cell types, suggesting a beneficial interaction which promotes mutual survival.\" - \"The strong inflammatory response and the presence of u03B1-SMA cells observed in first-generation TEHVs can be associated to the high strain profile exerted on the matrix as well as to the incomplete endothelialization at the earliest time points. Physiologically, changes in mechanical milieu can influence cell differentiation mechanisms, matrix remodeling, and the endothelialization process, and are often correlated to valvular pathologies. Native valvular ECs are indeed responsible for the anti-thrombogenic properties, as well as for the retention of the quiescent state of valvular interstitial cells. We may therefore hypothesize that the lack of ECs has caused the activation of the infiltrated interstitial cells towards contractile u03B1-SMA cells during the early stages of TEHV remodeling. In line with our (immuno-)histological findings, we also observed leaflet shortening, hinge area thickening, and wall-leaflet fusion phenomena of first-generation TEHVs. These adverse remodeling events were assumed to be mainly triggered by the presence of contractile u03B1SMA cells and may be a consequence of the simplified TEHV geometry, comprising a non-physiological hinge design. This may have contributed to the continuous thrombus deposition and pannus formation at the hinge region, which may have finally hindered the development of a confluent endothelium during the first 8_weeks after implantation. Notably, at every time point considered in our present analysis, the hinge region of first-generation TEHVs presented with an abundant amount of inflammation-associated cells (i.e.: leukocytes, eosinophils, macrophages, multinucleated giant cells, and u03B1 -SMA cells) which are involved in the formation of the pannus and the associated fibrotic tissue (Fig._A).\" - \"As previously reported, first-generation TEHVs showed good early recellularization, self-repair capacity, and good valve performance within the first 8_weeks in-vivo. However, mild to moderate insufficiency was first observed at 16_weeks, and valvular performance gradually worsened over time, with moderate-to severe insufficiency at 24_weeks, which was mainly caused by wall/leaflet fusion phenomena (Fig._ A).\" - \"Taken together, the results of the present study extend our understanding of the maladaptive remodeling observed in our first-generation TEHVs. As highlighted by Fig._A, our findings support the hypothesis that the extended presence of M1, combined with the increasing number of M2 macrophages over time and the lack of a confluent endothelium in first-generation TEHVs significantly impacted valve remodeling by inducing myofibroblast differentiation, causing pannus formation and subsequent increase in the hinge thickness during the first remodeling phases.\" - Second-generation TEHVs, towards a native-like remodeling - \"To date, independently of the tissue engineering technology used, several studiesu2014 among which also our first-generation TEHVu2014reported a gradual loss of TEHV performance within few months after implantation, due to maladaptive tissue remodeling (thickening, leaflet shortening). However, here we show that the tissue analysis of our second-generation TEHVs did not reveal signs of pannus formation nor thickening of the hinge area. Importantly, second-generation TEHVs presented with much less leaflet shortening when compared to their first-generation counterparts. Notably, it appears that both TEHVs generations undergo an early shortening within the first 8_weeks, which then continues in the first-generation valves leading to functional failure (negative remodeling-driven continuous shortening), while the leaflet length remains completely stable and unchanged over time in the second-generation TEHVs, thereby maintaining an excellent performance profile (functional remodeling).\" - \"In fact, the improved and more physiological geometry of second-generation TEHVs is not only beneficial from a mechanical point of view, but also from an inflammatory host cell response perspective (Fig._B). Indeed, the long-term performance demonstrated by our second-generation TEHVs was characterized by an overall low inflammatory cell infiltration, negligible amount of macrophages, and, importantly, the absence of u03B1-SMA cells in the hinge region at all the time-points considered (acute, 8_weeks, and 52_weeks). This, in relation to the complete endothelialization of the hinge area already at 8_weeks after implantation, resulted in an overall milder tissue inflammation (Fig._B).\" - \"Finally, this controlled inflammatory response is reflected by the observation of positive tissue remodeling outcomes. In this regard, we previously demonstrated that, similarly to a native valve, our computational modeling-inspired valve geometry can better sustain strain on the valve leaflets, thereby ensuring a more physiological long-term performance. Indeed, one of the major causes of leaflet shortening has been identified in leaflet compression. While first-generation TEHVs showed an overall compression of the leaflet, from the coaptation to the hinge region, second-generation TEHV geometry could efficiently limit leaflet compression to just the coaptation area. Second, we did neither detect thickening nor pannus formation in the hinge area. We may therefore hypothesize that second-generation TEHV geometry seem to better mimic native leaflet anatomy in the hinge area, thereby promoting the wash-out of fibrin deposits and preventing blood stagnation (Fig._B). The resulting favorable hemodynamics may therefore have enabled long-term performance of such second-generation TEHVs.\" - 8 weeks in-vivo follow-up, a turning point in the TEHV remodeling process? - \"As highlighted by our findings both generations of TEHVs presented with promising cellular infiltration and self-repair (remodeling) potential up to 8_weeks in-vivo. However, it is important to recognize that the remodeling process of first- and second-generation TEHV progresses into two distinct directions beyond 8_weeks (Fig._A,B). So far, the relevance of TEHV geometry was mainly considered on a mechanical and/or functional level. In the present study, we demonstrate that the TEHV geometry also strongly influences the outcome on a host cell response level (Fig._A,B).\" - \"At 8_weeks post implantation, first- and second-generation TEHVs showed comparable remodeling of the leaflet, with similar values of leaflet lengths (23% vs. 19% shortening for first- and second-generation, respectively) and hinge thicknesses. To the contrary, the differences observed in the number, phenotype, and distribution of infiltrating cells was substantially different underscoring the importance and impact of TEHV geometry. On one hand, first-generation TEHVs showed abundant cell infiltration, with approximately 20 times more cells per u03BCm than second-generation TEHVs at the same time point. Next, abundance of inflammatory cells, such as eosinophilic cells, M1 macrophages, u03B1-SMA cells, together with lack of a confluent endothelium in the hinge area were detected.\" - \"On the other hand, computational modeling-inspired second-generation TEHVs demonstrated an overall lower cellular infiltration with the negligible presence of M1/M2 and u03B1-SMA cells already 8_weeks after implantation. Remarkably, endothelialization of the hinge region was complete already at this time point. Hence, despite a comparable leaflet morphology, our qualitative and quantitative cell analysis indicates a substantial difference between the valves already at 8_weeks that resulted in two distinct remodeling outcomes (maladaptive first-generation TEHVs versus adaptive and functional in second-generation TEHVs) beyond 8_weeks. This finding highlights the importance of early host cell response in controlling long-term valvular remodeling, and provides important insight on time and type of cells that could be included in in-silico models to further enhance the outcome on tissue remodeling prediction.\"- - \"This study is an in-depth evaluation of TEHV explants obtained from previously performed in-vivo transcatheter implantations of two independent studies. Consequently, the available time points and number of samples were limited and unpaired between the experiments (Table ). Hence, the possibility to perform statistical analyses to compare first- and second-generation THEVs was limited. Despite this, we showed that both TEHVs generations underwent an initial leaflet shortening at 8_weeks follow-up. However, first-generation TEHVs experienced a progressive leaflet shortening, leading to a 70% reduction in length at 24_weeks post-implantation. To the contrary, leaflet shortening for second-generation TEHV did not progress over time thereby enabling the TEHV to remain functional for up to 1_year. This result is in line with what was previously reported for second-generation TEHVs, even though a direct comparison between these studies is not possible because of 1) the different amount of samples used to evaluate the leaflet length (11 vs 3 valve explants) and 2) the different analysis techniques adopted to measure the leaflet length (caliper measurement vs. histology section measurement). In this regard, it has been showed that a collagenous tissue processed for histology (i.e.: using formaldehyde fixation and paraffin wax embedding) can be subjected to a 33% shrinkage. Our study might therefore also highlight the importance of choosing the most appropriate measurement strategy to precisely report explanted TEHV remodeling measurements. In addition, immunohistochemistry proved to be a suboptimal technique to evaluate the presence of ECs due to cutting artefacts that may have caused cell detachment and lack of visible staining. In this regard, future studies should also better assess the integrity and functionality of the ECs layer, as well as evaluate the presence of activated platelets and microvesicles in the plasma. In this regard, analyses of anticoagulation and anti-thrombotic properties of ECs should be carefully considered when planning future in-vivo studies.\" - \"Next, we here focused our analysis on the hinge area of the TEHVs that, due to its important load-bearing activity, is subjected to high deformations. Indeed, mechanical strain is a well-known regulator of cell behavior and differentiation, and can therefore impact the (mal)adaptive remodeling phenomena in TEHVs. However, future in-vivo studies should consider additional time points as well as different TEHVs regions (e.g. the leaflets), to assess the first stages of inflammation (3u20134_weeks follow up) and the resolution of the inflammatory status (12u2013 24_months).\" - Finally, future studies should investigate how TEHV geometry will impact on the mechanical and hemodynamic valvular environment, also taking into consideration blood flow patterns through the valve, to better understand the mechanobiology of valvular remodeling. In this context, in-depth understanding of the molecular pathways underlying the complex interactions between inflammatory host cells, matrix remodeling, and scaffold degradation processes over time, may be crucial to better predict or even to anticipate (mal)adaptive remodeling phenomena.- - \"This study highlights the strong interplay between TEHV geometry and its related inflammatory host cell response, which ultimately drives the tissue remodeling outcomes in TEHVs. First-generation TEHVs, with a simple valve-geometry, experienced a chronic inflammatory response characterized by abundant presence of M1 and M2 macrophages and u03B1-SMA cells, which finally led to leaflet retraction and valve insufficiency. In contrast, second-generation TEHVs displayed an overall lower inflammatory response with only few M1 and M2 macrophages and u03B1-SMA cells being present. These results highlight how an analytical, computational-inspired TEHV geometry does not only impact the mechanical strain distribution on the leaflets which is critical for valve performance, but also how it orchestrates host cell response towards functional remodeling, thereby enabling long-term performance of a TEHV. Our findings further provide important insight into the (inflammatory) host cell response and the related remodeling processes of TEM-based TEHVs which represents a key requisite for the safe and successful clinical translation of such technologies.\"- - \"In this study, we performed a pooled, in-depth analysis of TEM-based TEHV tissue explants (nu2009=u200920) from our previously reported in-vivo studies, as well as from additional unpublished data. The ethics committee and local authorities (VeterinxE4ramt, Gesundheitsdirektion, Kanton ZxFCrich [197/2010], Switzerland and the Regional Office for Health and Social Affairs Berlin, LAGeSo, Berlin; approval.\" - \"no. G0111/11) approved the previously reported in-vivo studies, which were performed in compliance with the Guide for the Care and Use of Laboratory Animals, published by the National Institutes of Health (NIH publication No. 85u201323), as well as the guidelines of the European and German societies of Laboratory Animal Science (FELASA, GV-SOLAS) and the ARRIVE guidelines (Animal Research; Reporting of in-vivo Experiments). Detailed information about the manufacturing, in-vitro characterization, and in-vivo testing of first- and second-generation TEHV have been previously reported. Compared to first-generation TEHVs, the use of an insert to impose a computational modelling-inspired valve geometry was the only difference in the manufacturing protocol of second-generation TEHVs.\" - \"According to the follow-up schemes of our in-vivo studies, a total of fifteen TEHV tissue samples (nu2009=u200915) were obtained from explants at the following time points: acute (considered as control), 8_weeks, 16_weeks, 24_weeks, and 52_weeks (1_year). The overview of available samples is reported in Table . In addition, the tissue samples for the acute (nu2009=u20092) and 8_weeks time point (nu2009=u20093) of the second-generation valve were obtained from an independent pilot series of animals (nu2009=u20095) which has not been published before (unpublished data), and were produced using the same culture conditions. Qualitative and quantitative analyses have been performed as described below to evaluate the inflammatory host cell response to the implanted TEHVs particularly in the hinge region of the leaflets, as it acknowledged to be the most subjected area to maladaptive remodeling phenomena.\" - Qualitative tissue analysis - Samples from explanted TEHVs (Table ) were analyzed qualitatively to characterize the presence of inflammatory host cells using immunofluorescence, histology, immunohistochemistry, and scanning electron microscopy (SEM) techniques. - (Immuno)histochemistry and immunofluorescence stainings - \"In brief, TEHVs samples were fixed in formalin, embedded into paraffin, and cut consecutively into 5_xB5m slices with a sliding microtome at room temperature. The specimens were deparaffinized by treatment with xylene followed by a series of decreasing concentrations of ethanol (100u201350%). Cellular infiltration and tissue appearance were evaluated by using Haematoxilin and Eosin (H&E) and Masson Goldner (MG) staining, and CD31, von Willebrand Factor (VWF), and u03B1 -SMA staining. The slides were visualized by Zeiss Mirax Midi microscope (Carl Zeiss Microimaging) and assessed with Pannoramic Viewer (3DHistech).\" - \"To characterize the phenotype of the infiltrated cells within the explanted TEHVs, immunofluorescence was performed following antigen retrieval using citrate (10_ mM citric acid at pH 6.0) which was then brought to boiling temperature (96_xB0 C) for 20_min. Primary antibody for CCR7 (M1 marker) and CD163 (M2 marker) were incubated for over-night at 4_xB0C. After washing with PBS, secondary antibodies were applied for 2_h (Supplementary Table ). Cell nuclei were counterstained with DAPI. The stained sections were visualized using a fluorescence slide scanner (ZEISS Axio Scan.Z1, Carl Zeiss AG). Image processing was performed using the Axio Scan.Z1 ZEN lite imaging software (Blue version, 2012; Carl Zeiss AG).\" - Scanning electron microscopy (SEM) - SEM was used to assess and confirm the degree of endothelialization for the two TEHVs geometries. For further details see the Supplementary Material section. - Quantitative tissue analysis - Inflammatory host cell infiltration - \"Cellular infiltration was quantified automatically and manually with ImageJ software in three pictures across the hinge area and in two consecutive sections for a total thickness of 10_xB5m (20u2009xD7u2009magnification; area comprising 420,000 xB5m per picture). Pictures of stained sections were converted into grey-value images (16-bit). Quantification was performed by counting the cells positive for CCR7, CD163, or a combination of both (double stained cells) and compared to the total number of cells, detected via DAPI. The total number of cells infiltrating the wall region was used as control value for the complete cellular infiltration in the hinge area.\" - Morphological evaluation - \"Hinge thickness and leaflet length were measured manually for both first- and second-generation TEHVs using the Panoramic Viewer software (3DHistech, Ltd.). For both TEHVs geometries, hinge thickness was measured in two different sections of 5_u03BCm each, and in three locations per section, for a total of 6 measurements. Leaflet length was measured in two different sections of 5_u03BCm each, by using the highest number of segments possible to increase the accuracy of the measurements. Analyzed samples comprised the following follow-up time-points: acute, 8_weeks, 16_weeks, 24_weeks, and 52_weeks (Table ).\" - Statistics - \"Measurements are represented as mean valueu2009xB1u2009standard deviation unless stated otherwise. Leaflet length and hinge thickness measurements were averaged for each sample (Supplementary Table ).\"- - \\' \\' - Data are available upon request and completion of a Materials Transfer Agreement (MTA). - \\' \\'- - \"Leopold, J. A. Cellular mechanisms of aortic valve calcification. Circ. Cardiovasc. Interv. 5, 605u2013614 (2012)._ _ _ _ Langer, R. & Vacanti, J. P. Tissue engineering. Science (80-) 260, 920u2013926 (1993)._ _ _ _ Fioretta, E. S. et al. Next-generation tissue-engineered heart valves with repair, remodelling and regeneration capacity. Nat Rev Cardiol. (2020).Fioretta, E. S., Dijkman, P. E., Emmert, M. Y. & Hoerstrup, S. P. The future of heart valve replacement: recent developments and translational challenges for heart valve tissue engineering. J Tissue Eng Regen Med (2017)._ _ _ Wissing, T. B., Bonito, V., Bouten, C. V. C. & Smits, A. I. P. M. Biomaterial-driven in situ cardiovascular tissue engineeringu2014 a multi-disciplinary perspective. NPJ Regen. Med. 2, 18 (2017)._ _ _ _ Driessen-Mol, A. et al. Transcatheter implantation of homologous u2018off-the-shelf tissue-engineered heart valves with self-repair capacity: long-term functionality and rapid in vivo remodeling in sheep. J Am Coll Cardiol 63, 1320u20131329 (2014)._ _ _ Emmert, M. Y. et al. Computational modeling guides tissue-engineered heart valve design for long-term in vivo performance in a translational sheep model. Sci Transl Med 10 , 1u201313 (2018).Sanders, B. et al. Improved geometry of decellularized tissue engineered heart valves to prevent leaflet retraction. Ann. Biomed. Eng. 44, 1061u20131071 (2016)._ _ _ Loerakker, S., Ristori, T. & Baaijens, F. P. A computational analysis of cell-mediated compaction and collagen remodeling in tissue-engineered heart valves. J. Mech. Behav. Biomed. Mater. 58, 173u2013187 (2016)._ _ _ _ Hans, C. P. et al. Transcriptomics analysis reveals new insights into the roles of Notch1 signaling on macrophage polarization. Sci. Rep. 9, 7999 (2019)._ _ _ _ _ _ Londono, R. & Badylak, S. F. Biologic scaffolds for regenerative medicine: mechanisms of in vivo remodeling. Ann. Biomed. Eng. 43, 577u2013592 (2015)._ _ _ Syedain, Z. et al. Tissue engineering of acellular vascular grafts capable of somatic growth in young lambs. Nat. Commun. 7, 1u2013 9 (2016)._ _ _ Roh, J. D. et al. Tissue-engineered vascular grafts transform into mature blood vessels via an inflammation-mediated process of vascular remodeling. Proc. Natl. Acad. Sci. U. S. A. 107, 4669u2013 4674 (2010)._ _ _ _ _ _ Matsumura, G., Miyagawa-Tomita, S., Shinoka, T., Ikada, Y. & Kurosawa, H. First evidence that bone marrow cells contribute to the construction of tissue-engineered vascular autografts in vivo. Circulation 108, 1729u20131734 (2003)._ _ _ Hibino, N. et al. The innate immune system contributes to tissue-engineered vascular graft performance. Faseb J. 29, 2431u20132438 (2015)._ _ _ _ _ Fukunishi, T. et al. Tissue-engineered small diameter arterial vascular grafts from cell-free nanofiber PCL/Chitosan scaffolds in a sheep model. PLoS ONE 11, e0158555 (2016)._ _ _ _ _ Lawson, J. H. et al. Bioengineered human acellular vessels for dialysis access in patients with end-stage renal disease: two phase 2 single-arm trials. Lancet (Lond. Engl.) 387, 2026u20132034 (2016)._ _ Kirkton, R. D. et al. Bioengineered human acellular vessels recellularize and evolve into living blood vessels after human implantation. Sci. Transl. Med. 11, 1u201311 (2019).Franz, S., Rammelt, S., Scharnweber, D. & Simon, J. C. Immune responses to implantsu2014a review of the implications for the design of immunomodulatory biomaterials. Biomaterials 32, 6692u2013 6709 (2011)._ _ _ _ Mosser, D. M. & Edwards, J. P. Exploring the full spectrum of macrophage activation. Nat. Rev. Immunol. 8, 958u2013969 (2008)._ _ _ _ _ Boehler, R. M., Graham, J. G. & Shea, L. D. Tissue engineering tools for modulation of the immune response. Biotechniques 51, 239u2013240, 242, 244 passim (2011)._ _ _ _ _ Tidball, J. G. & Villalta, S. A. Regulatory interactions between muscle and the immune system during muscle regeneration. Am. J. Physiol. Regul. Integr. Comp. Physiol. 298, R1173u2013 R1187 (2010)._ _ _ _ _ Naito, Y. et al. Vascular tissue engineering: towards the next generation vascular grafts. Adv. Drug. Deliv. Rev. 63, 312u2013323 (2011)._ _ _ _ Katayama, S., Umetani, N., Sugiura, S. & Hisada, T. The sinus of Valsalva relieves abnormal stress on aortic valve leaflets by facilitating smooth closure. J. Thorac. Cardiovasc. Surg. 136, 1528u20131535, 1535.e1 (2008)._ _ _ Balachandran, K., Sucosky, P. & Yoganathan, A. P. Hemodynamics and mechanobiology of aortic valve inflammation and calcification. Int. J. Inflam. 2011, 263870 (2011)._ _ _ _ Mariani, E., Lisignoli, G., BorzxEC, R. M. & Pulsatelli, L. Biomaterials: foreign bodies or tuners for the immune response?. Int. J. Mol. Sci. 20, 636 (2019)._ _ _ _ Morris, A. H., Stamer, D. K. & Kyriakides, T. R. The host response to naturally-derived extracellular matrix biomaterials. Semin. Immuno.l 29, 72u201391 (2017)._ _ _ Chester, A. H. et al. The living aortic valve: from molecules to function. Glob. Cardiol. Sci. Pr. 2014, 52u201377 (2014)._ _ Ayoub, S. et al. Heart valve biomechanics and underlying mechanobiology. Compr. Physiol. 6, 1743u20131780 (2016)._ _ _ _ Klopfleisch, R. & Jung, F. The pathology of the foreign body reaction against biomaterials. J. Biomed. Mater. Res. A 105, 927u2013940 (2017)._ _ _ _ Loerakker, S., Argento, G., Oomens, C. W. & Baaijens, F. P. Effects of valve geometry and tissue anisotropy on the radial stretch and coaptation area of tissue-engineered heart valves. J. Biomech. 46, 1792u20131800 (2013)._ _ _ _ Ristori, T., Obbink-Huizer, C., Oomens, C. W., Baaijens, F. P. & Loerakker, S. Efficient computational simulation of actin stress fiber remodeling. Comput. Methods Biomech. Biomed. Eng. 19, 1347u20131358 (2016)._ _ _ Kumar, V. & Sharma, A. Neutrophils: Cinderella of innate immune system. Int. Immunopharmacol. 10, 1325u20131334 (2010)._ _ _ _ Wang, Y.-Y. et al. Macrophage-to-myofibroblast transition contributes to interstitial fibrosis in chronic renal allograft injury. J. Am. Soc. Nephrol. 28, 2053u2013 2067 (2017)._ _ _ _ _ Braga, T. T., Agudelo, J. S. H. & Camara, N. O. S. Macrophages during the fibrotic process: M2 as friend and foe. Front. Immunol. 6, 602 (2015)._ _ _ _ _ Shapouri-Moghaddam, A. et al. Macrophage plasticity, polarization, and function in health and disease. J. Cell. Physiol. 233, 6425u2013 6440 (2018)._ _ _ _ Daley, J. M., Brancato, S. K., Thomay, A. A., Reichner, J. S. & Albina, J. E. The phenotype of murine wound macrophages. J. Leukoc. Biol. 87, 59u201367 (2010)._ _ _ _ Hesketh, M., Sahin, K. B., West, Z. E. & Murray, R. Z. Macrophage phenotypes regulate scar formation and chronic wound healing. Int. J. Mol. Sci. 18 (2017).Zhu, Z., Ding, J., Ma, Z., Iwashina, T. & Tredget, E. E. Alternatively activated macrophages derived from THP-1 cells promote the fibrogenic activities of human dermal fibroblasts. Wound Repair Regen. 25, 377u2013 388 (2017)._ _ _ Glim, J. E., Niessen, F. B., Everts, V., van Egmond, M. & Beelen, R. H. J. Platelet derived growth factor-CC secreted by M2 macrophages induces alpha-smooth muscle actin expression by dermal and gingival fibroblasts. Immunobiology 218, 924u2013929 (2013)._ _ _ _ Cai, Q., Lanting, L. & Natarajan, R. Interaction of monocytes with vascular smooth muscle cells regulates monocyte survival and differentiation through distinct pathways. Arter. Thromb Vasc Biol 24, 2263u2013 2270 (2004)._ _ _ Gould, S. T., Matherly, E. E., Smith, J. N., Heistad, D. D. & Anseth, K. S. The role of valvular endothelial cell paracrine signaling and matrix elasticity on valvular interstitial cell activation. Biomaterials 35, 3596u20133606 (2014)._ _ _ _ _ Butcher, J. T. & Nerem, R. M. Valvular endothelial cells regulate the phenotype of interstitial cells in co-culture: effects of steady shear stress. Tissue Eng 12, 905u2013915 (2006)._ _ _ _ Jana, S., Tranquillo, R. T. & Lerman, A. Cells for tissue engineering of cardiac valves. J. Tissue Eng. Regen. Med. 10, 804u2013824 (2016)._ _ _ _ Cushing, M. C., Liao, J. T. & Anseth, K. S. Activation of valvular interstitial cells is mediated by transforming growth factor-beta1 interactions with matrix molecules. Matrix Biol. 24, 428u2013 437 (2005)._ _ _ _ Benton, J. A., DeForest, C. A., Vivekanandan, V. & Anseth, K. S. Photocrosslinking of gelatin macromers to synthesize porous hydrogels that promote valvular interstitial cell function. Tissue Eng. Part A 15, 3221u20133230 (2009)._ _ _ _ _ Reimer, J. et al. Implantation of a tissue-engineered tubular heart valve in growing lambs. Ann. Biomed. Eng. (2016)._ _ _ _ Syedain, Z. et al. 6-Month aortic valve implantation of an off-the-shelf tissue-engineered valve in sheep. Biomaterials 73, 175u2013 184 (2015)._ _ _ _ _ Schmidt, D. et al. Minimally-invasive implantation of living tissue engineered heart valves: a comprehensive approach from autologous vascular cells to stem cells. J. Am. Coll. Cardiol. 56, 510u2013520 (2010)._ _ _ Weber, B. et al. Off-the-shelf human decellularized tissue-engineered heart valves in a non-human primate model. Biomaterials 34, 7269u20137280 (2013)._ _ _ _ Flanagan, T. C. et al. In vivo remodeling and structural characterization of fibrin-based tissue-engineered heart valves in the adult sheep model. Tissue Eng. Part A 15, 2965u20132976 (2009)._ _ _ _ Gottlieb, D. et al. In vivo monitoring of function of autologous engineered pulmonary valve. J. Thorac. Cardiovasc. Surg. 139, 723u2013731 (2010)._ _ _ Schmitt, B. et al. Percutaneous pulmonary valve replacement using completely tissue-engineered off-the-shelf heart valves: six-month in vivo functionality and matrix remodelling in sheep. EuroIntervention 12, 62u201370 (2016)._ _ _ Chatterjee, S. Artefacts in histopathology. J. Oral Maxillofac. Pathol. 18, S111u2013S116 (2014)._ _ _ _ Jensen, E. C. Quantitative analysis of histological staining and fluorescence using ImageJ. Anat. Rec. 296, 378u2013381 (2013)._ _ \"- - \"The authors gratefully acknowledge the funding from the European Union s Seventh Framework Program (FP7/2007-2013) under grant agreement nxB0 242008. S.E.M. was supported by the University of Zurich Forschnugskredit (FK-14-031). E.S.F. was partly funded through the Swiss National Science Foundation (PZ00P3_180138). M.Y.E. and S.L. have received funding from the European Research Council (ERC) under the European Unions Horizon 2020 research and innovation programme, grant agreement no. 852814 (TAVI4Life) and grant agreement no. 802967 (MechanoSignaling). We are thankful to Flurin Arner for supporting and contributing to this work.\"- - Author notes - \\'These authors contributed equally: Sarah E. Motta and Emanuela S. Fioretta.\\' - Authors and Affiliations - \"Institute for Regenerative Medicine (IREM), University of Zurich, Wagistrasse 12, 8952, Schlieren, SwitzerlandSarah E. Motta,_Emanuela S. Fioretta,_Petra E. Dijkman,_Laura Frese,_Simon P. Hoerstrup_&_Maximilian Y. EmmertWyss Translational Center Zurich, University and ETH Zurich, Zurich, SwitzerlandSarah E. Motta,_ Valentina Lintas,_Simon P. Hoerstrup_&_Maximilian Y. EmmertInstitute of Veterinary Pathology, University of Zurich, Zurich, SwitzerlandMonika HilbeDepartment of Cardiothoracic and Vascular Surgery, German Heart Center Berlin, Berlin, GermanyNikola Cesarovic,_Volkmar Falk_&_Maximilian Y. EmmertDepartment of Health Sciences and Technology, Swiss Federal Institute of Technology, Zurich, SwitzerlandNikola Cesarovic_&_Volkmar FalkDepartment of Biomedical Engineering, Eindhoven University of Technology, Eindhoven, The NetherlandsSandra Loerakker_&_Frank P. T. BaaijensInstitute for Complex Molecular Systems, Eindhoven University of Technology, Eindhoven, The NetherlandsSandra LoerakkerDepartment of Cardiovascular Surgery, CharitxE9 UniversitxE4tsmedizin Berlin, Berlin, GermanyVolkmar Falk_&_Maximilian Y. Emmert\" - \"AuthorsSarah E. MottaYou can also search for this author in _Emanuela S. FiorettaYou can also search for this author in _Valentina LintasYou can also search for this author in _Petra E. DijkmanYou can also search for this author in _Monika HilbeYou can also search for this author in _Laura FreseYou can also search for this author in _Nikola CesarovicYou can also search for this author in _Sandra LoerakkerYou can also search for this author in _Frank P. T. BaaijensYou can also search for this author in _Volkmar FalkYou can also search for this author in _Simon P. HoerstrupYou can also search for this author in _Maximilian Y. EmmertYou can also search for this author in _\" - Contributions - S.P.H. and F.P.T.B. obtained the funding. S.E.M, E.S.F., and P.E.D. designed the study concept. S.E.M. and M.H. performed the experiments. S.E.M. analyzed the data. S.E.M., E.S.F., V.L. P.E.D., M.H., L.F., N.C., and S.L. performed post-mortem evaluation. S.E.M, E.S.F., V.L., P.E.D., M.H., L.F., N.C., S.L., F.P.T.B., V.F., S.P.H., and M.Y.E. drafted and/or revised the manuscript for critical content. S.E.M. wrote the manuscript. Administrative, technical, or supervisory tasks were handled by S.E.M., M.Y.E, and S.P.H. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - \"V. Falk has disclosed relevant financial activities outside the submitted work with the following commercial entities: Medtronic GmbH, Boston Scientific and Edwards Lifesciences in relation to educational grants (including travel support) fees for lectures and speeches, fees for professional consultation and research and study funds. F.P.T. Baaijens and S.P. Hoerstrup are shareholders at Xeltis BV and LifeMatrix AG. M.Y. Emmert is a shareholder at LifeMatrix AG. S. Loerakker, F.P.T. Baaijens, and S.P. Hoerstrup are inventors on patent u201CControlling TEHV geometry by using predefined inserts during cultureu201D (WO2015044190 A1) held by Eindhoven University of Technology. This patent covers how the shape of the heart valves can be controlled and maintained during the in vitro culture phase. The authors declare no competing interests.\" - \\' \\'- - Publisher\\'s note - Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.- - \\'\\'- - \\' \\' - Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article\\'s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article\\'s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons license.) b The insect world includes a very large variety of tessellated (patterned) cuticles with bumps, pits, pixels, bands, spots or patches and a diversity of structural colors (Staatliches Museum fxFCr Naturkunde, Karlsruhe, Germany. Image by H. Zell. Creative Commons license). c Two-band iridescent cuticle of Chrysina gloriosa. d Green and silver bands at their interface as observed by optical microscopy (reflection mode, unpolarized light). e TEM images of transverse structures at the interface between green and silver bands and close to the top surface. White dots underline the stripes in the fingerprint texture. The transition between curved and straight stripes is observed to be very gentle. The last part of the cuticle consists of a wax layer, which may serve to restrict water loss and prevent desiccation. f Reflection spectra of green and silver bands.\" - \"For circularly polarized light propagating along the helix axis with the same handedness as the helix, a CLC exhibits a Bragg reflection band over the wavelength range of (nou2013ne)p, where no and ne are the ordinary and extraordinary refractive indices and p is the helical pitch. The central wavelength u03BB0 is: u03BB 0u2009=u2009np, where nu2009=u2009(nou2009+u2009ne)/2 is the average refractive index.\" - \\'In biomimicry, many of the same species come up: geckos, spiders, and butterflies. The latter are dominant in the field of photonic structures. Air layers and pillar-supported chitinous layers alternate in the twist-free, multilayer structure of butterfly wings, which leads to optical interference. There are many replication methods, which are based on three concepts that use wings as templates: coating, modification and filling. The morphology of the replicas can be hard to control. The choice of the method depends on the requirements of the situation.\\' - \\'Biomimetic materials aiming at reproducing the cuticles of beetles as closely as possible are especially rare. They have addressed the untextured, gold scarab beetle Plusiotis resplendens. The samples consist in the superposition of three layers: a nematic LC layer, which acts as a half-wave retardation plate, is inserted between two left-handed cholesteric layers. The fabrication methods used a UV-crosslinked elastomer or chiral cellulose nanocrystals after water evaporation. In the latter case, a non-uniformity of the reflection colors at the surface of the sample resulting from the evaporation process is reported. These three-layer systems present interfaces. When optical performance is desired, interfaces likely add parasitic reflections and light scattering to Bragg reflections. Besides, the cuticle of P. resplendens is interface-free: the cholesteric-nematic interfaces do not indeed exhibit any discontinuity.\\' - \"In contrast to nontextured reflective cuticles, tessellations as found in the cuticles of many insects (Fig._), and a few fruits are challenging to duplicate via a biomimetic design. First, the pitch length and the helical axis orientation must be position-dependent across the material, and formation of periodic patterns at its surface is simultaneously required. The primary challenge consists of finding the design rules to manufacture a single-piece material that presents variable physical properties without resorting to an accumulation of layers and a juxtaposition of patterns. Second, tessellated cuticles are associated with the concept of multiscale structures. Biomimetic materials must thus include anisotropic architectures at the nanoscale, micrometre scale and macroscale. Third, the double versatility of colors and patterns must ideally be achieved through a simple and single stage to make the self-organization process closer to morphogenesis. Finally, an optimal use of resources has to be targeted, as in nature: reducing the number of materials to be implemented since living organisms use only a few chemical elements (C, H, O, N, P and S, typically) to self-organize into a multitude of complex structures; avoiding metallic nanoparticles and metallic layers to realize an environmentally responsible approach; and considering the recyclability of materials from the perspective of mass-market production.\" - In this paper, we approach the problem of the high-fidelity capture of the structural complexity observed in nature by focusing on patterned, iridescent carapaces of insects with a complex twisted organization of chitin fibres. At a small scale (the micrometre or submicrometre scale), the helical pitch and orientation of the helical structure depend on the position in three dimensions, and the formation of macroscopic periodic patterns is required. We used LC oligomers to mimic biological LCs. We attempted to reproduce the textural, structural and color properties of these biological LCs at several length scales. We made optimal use of resources and stages during the fabrication procedure, in the spirit of eco-design. By means of a single sequence based on self-organization, precise control of a single-piece sample structure composed of two different colored patterns with the same unique pitch gradient was enabled. The functionalities of both materials are relevant to optical communication and camouflage. In addition to this set of conceptual advances, we present a concrete use of the material for advanced optical tags in cryptography.- - Lessons from nature - \"With the previous objectives in mind, we focus on the emblematic case of the scarab beetle Chrysina gloriosa (Fig._), which is found from southwestern North America to Central America in pine, pine-oak, and juniper forests. The cuticle of C. gloriosa exhibits green and silver bands with selective and broad light reflections, respectively. The green bands exhibit a cellular polygonal texture in optical microscopy images, whereas the silver bands exhibit a non-patterned planar texture (Fig._). Previous studies on the cuticle of C. gloriosa have reported the following: the cellular structure of the concentric rings in the green bands, as established through TEM investigations or fluorescence confocal microscopy; the variation in the optical properties with the incidence angle, as established through Mueller matrix spectroscopic ellipsometry; the existence of an array of multiwavelength micromirrors in the green bands; and the generation of self-healing Bessel beams from polygonal cells.\" - In the following, we set out the lessons learned from nature about the structure and optical response of both bands. - \"Cross-sectional transmission electron microscopy (TEM) views of the green and silver bands show the typical cholesteric fingerprint (stripe) texture (Fig._ ). The axis of the helix lies in the plane of the image. The fingerprint texture of the (unlayered) cholesteric structure appears as a network of alternating bright and dark stripes. This periodic network thus arises from periodic modulation, along the helix axis, of the orientation of the molecular director (the orientation of the elongated chitin fibres depicted in Fig._) and the refractive index. The helical axis is perpendicular to the stripes. The distance between two identical stripes is related to the half pitch of the twisted organization of the chitin fibres. For the green bands, the cuticle displays curved stripes in the upper part of the exocuticle, which means that the orientation of the helical axis is spatially changing, and parallel stripes in most of the endocuticle region. For the silver bands, the stripes are parallel in the whole chitinous part of the cuticle, the orientation of the helical axis is fixed. For both bands, a pitch gradient occurs from the top of the exocuticle to the bottom of the endocuticle (Supplementary Fig._).\" - Before proceeding to the analysis of the optical spectra, a preliminary comment must be made here regarding the fact that between 0 and 50% of the unpolarized incident light is reflected or transmitted, which is true for all of the reflection and transmission spectra presented in this study. Unlike common mirrors, which reverse the polarization of the light they reflect, the cholesteric planar texture reflects circularly polarized light with the same handedness as that of the twist of the helix, while light with the other polarization is transmitted. This behavior is referred to as the polarization-selectivity rule, which is valid only at normal incidence. Consequently, up to 50% of the ambient unpolarized light at the selected wavelength can be reflected. The other component (50% of the ambient unpolarized light at the selected wavelength) is transmitted. Only first-order Bragg reflection is possible at normal incidence. All orders of reflection may occur at oblique incidence or when the helical structure is tilted, and the reflected or transmitted light is elliptically polarized. - \"The reflectance spectrum ascribed to the silver band in Fig._ exhibits a broadband reflection from 450 to 800u2009nm with a variable reflectance. This broad band can be identified from the pitch gradient shown in the TEM micrograph of a cross section (Supplementary Fig._) and the related measurements, which give a pitch p varying from ~280 to 900u2009nm (Supplementary Fig._). By taking nu2009 =u20091.63 for the value of the average refractive index of the chitinous part of the cuticle of C. gloriosa (in ref. (p. 214), the ordinary and extraordinary indices are set to 1.59 and 1.68 for Plusiotis gloriosa (a synonym now invalid for C. gloriosa), which leads to a mean refraction index equal to ~1.63.) and using the Bragg law at normal incidence, u03BB0u2009=u2009np, as given in the introduction, we obtain a wavelength band of ~455u20131470u2009nm (the measurements in Fig._ stop at 850u2009nm because of the technical limitations of the spectrophotometer). The silver band thus plays the role of a flat metallic reflector operating over the visible spectrum and into the infrared spectrum. The oscillations, or ripples, modulating the reflection spectrum in Fig._ can be attributed to longitudinal interferences associated with the existence of a singular region named the emergence line between the exo- and endo-cuticles (i.e., a trough in the periodicity variation), as observed by SEM and discussed in ref. . The presence of a ripple structure superimposed on a broadband plateau is also present in the spectrum of the gold cuticle of the scarab beetle Chrysina aurigans.\" - \"The reflectance spectrum ascribed to the green band exhibits a broadband reflection from 450 to 800u2009nm. The pitch gradients in the silver and green bands occur in ranges of comparable amplitudes, as shown in Supplementary Fig._. However, for the green band, the reflectance is very low (between 8 and 10%) from 450 to 800u2009nm, with a characteristic green peak at 520u2009nm, because the green bands act as green diffusers. Diffuse reflection occurs when light is reflected at many different angles rather than at just one angle as in the case of specular reflection. The green bands are green diffusers because the arrangement on a curved surface of polygonal cells, with curved and concave stripes (Fig._ and Supplementary Fig._), with a spatially varying helical axis, significantly broadens the angular distribution of reflection. Both kinds of curvature (the surface of the cuticle, and the orientation of the helical axis inside each cell) remove the angular dependence of the reflection wavelength normally associated with Bragg reflection when the cholesteric material is a flat film with a pattern-free planar texture. The result is a nearly angle-independent greenish appearance of the beetle at long viewing distances, which may enable the avoidance of detection by predators in green foliage. The disruptive green-silver patterning is a good match for the habitat of C. gloriosa since the foliage of the juniper tree is green with white resin flecks. This situation might provide the insect with efficient camouflage.\" - It should be noted that the reflective and absorptive properties of the wax layer at short wavelengths may influence the shape of the reflection spectrum at the beginning of the visible region, as discussed in the case of Chrysina aurigans. - \"To summarize, Fig._ shows a 3D view that displays the color and structure properties to be simultaneously addressed in the goal of biomimetic design. The biomimetic material must be a monolayer; exhibit a disruptive character of green and silver colors; display the continuity of the twisted structures in all directions (no disruption in the pitch distribution and in the orientation of the helical axis from region to region) with, in the upper part of the film, an alternate mix of variable and fixed orientations of the helical axis; include a pitch gradient related to reflection colors ranging from green to near IR; and exhibit pitch variations in similar ranges.\" - \\'Fig. 2: Fundamental principles to mimic the tessellated cuticle.a Tridimensional two-band structure to be considered. b Physical design in five stages.\\' - Biomimetic design - Recent successes in utilizing soft materials for the design of advanced photonic structures promise compelling opportunities, but they rarely capture the range of structural complexity observed in natural optical materials. Using synthetic LC materials for biomimetic purposes is an underutilized solution in the toolbox of researchers and engineers, partly due to challenges in fabrication, integration, and structural control at the nanoscale and microscale of soft components. However, LCs are first-rate candidates because they offer anisotropic self-organization, multiscale architectures, and a diversity of materials, such as fluids, polymers or gels, that can be inserted between two solid or flexible substrates or handled as free-standing films. - \"The design procedure we propose uses vitrifiable cholesteric oligomers with a siloxane backbone. The rapid cooling of samples below the glass-transition temperature, from the viscous cholesteric phase to room temperature, prevents crystallization and preserves the cholesteric structure and colors. Polysiloxanes form a group of materials with unusual properties, which may include high thermal stability, low surface energy and a low glass-transition temperature. The present biomimetic design takes advantage of these properties. LC polysiloxane oligomers from Wacker Chemie GmbH were used (Supplementary Fig._). The molecule is a siloxane cyclic chain with two types of side chains attached via spacers: an achiral mesogen and a (chiral) cholesterol-bearing mesogen. When used to coat a glass or plastic substrate, this compound exhibits typical iridescent colors that range from blue to red with the tuning of the molar percentage of chiral mesogens in the oligomer molecule, which can vary from 31 to 50%. LC polysiloxane oligomers have been the subject of various studies, including basic investigations of their reflective properties, the fabrication of optical notch filters and variable bandwidth reflectors, low-frequency mechanical relaxation measurements of molecular motions near the glass-transition temperature, the color gamut of planar films and flakes based on colorimetry, the electro-optical behavior of cholesteric flakes, mechano-optical effects, lasing from dye-doped materials, the self-organization of metallic nanoparticles into various cholesteric textures and the control of the stretching and compression of ultrashort laser pulses.\" - \"Polygonal textures, consisting of arrays of micrometre-scale polygonal cells (Supplementary Fig._), can be observed in flat CLC films when the helical axes are strongly tilted with respect to the substrates. This situation can be observed in an open film via hybrid anchoring of molecules, which preferentially align tangentially to the solid substrate on which the CLC is coated (planar anchoring conditions) and perpendicularly to the air interface (perpendicular anchoring conditions). Supplementary Fig._ shows a three-dimensional representation of the polygonal texture obtained by combining various microscopy methods and provides more information on the formation of the polygonal texture in the materials under study. Polygonal fingerprint textures were first imaged via optical microscopy in large-pitch low-molar-mass CLCs in the early 1970s. In the mid-1990s, polygonal textures in the film plane and in cross sections were observed via TEM and AFM. In the 2010s, investigations of polygonal textures focused on glass-forming oligomers such as those used in the present study, which does not focus on the polygonal texture itself.\" - \"The biomimetic design is based on the control of the thermally driven diffusion between one green and one infrared light-reflecting layer accompanied by local changes in the molecular anchoring conditions in specific regions of the film. It involves five stages (Fig._). Stage 1: A cell made of cholesteric films coated on glass substrates facing each other, reflecting the light in the green and IR spectrum, respectively, is assembled at RT. Stage 2: The sandwich cell is heated at 120u2009xB0C for 30u2009s to adhere the films together. Stage 3: After quenching of the sample at RT, the upper substrate is removed. Stage 4: Spaced bands of an isotropic polymer, polyvinyl alcohol (PVA), in solution in water are deposited on the free surface of the bilayer film. Stage 5: After drying of the aqueous solution, the bilayer film is heated in the cholesteric phase at 120u2009xB0C for 1u2009h. Three major events simultaneously occur due to the thermally driven diffusion between the two films in the direction perpendicular to their surfaces: a physical transverse pitch gradient is built as the result of the chemical gradient in the chiral species; the polygonal texture nucleates in the regions in contact with air between the polymer bands (the anchoring of molecules becomes perpendicular in the upper part of these regions), whereas the planar texture is preserved in the regions below the polymer bands (the anchoring of molecules is kept as planar in these regions); and the bilayer system becomes a single layer. Finally, the film is quenched at RT. The structural and optical properties are maintained in the solid glassy film.\" - \"The fifth stage of the design is the crucial, one-step, stage. By offering a single sequence based on self-organization, precise control of the sample structure composed of two different colored bands with the same unique pitch gradient is enabled. An association of the alternate green and silver bands was obtained by drawing spaced bands on the free surface of the film before the thermal diffusion began using a pencil dipped in PVA solution. Alternatively, band-shaped adhesive tape could be fixed on the surface of the film, which is then spin-coated with the solution (see Methods). The tape was removed before the solution was completely dried. Many patterns other than bands can thus be drawn, which offers a wide, choice on demand for designing reflective patterns. PVA is commonly used in the field of thermotropic LCs to induce a molecular orientation parallel to the support. Here, it was used to protect specific regions targeted to become silver bands from air. By analogy, it protects the film from texture evolution (planar to polygonal texture transition), while the wax layer visible in Fig._ and Supplementary Fig._ protects the chitinous part of the cuticle from the external environment, with the difference being that the polymer layer only covers the silver bands, while the wax layer covers the green and silver bands. Finally, the PVA bands can be dissolved in water if required. Patterning techniques, such as metallic layer sputtering deposition, moulding, alignment methods or lithography, would require many stages to achieve the present patterning objectives and would lead to discontinuous layers and structures. In addition, the range of producible patterns, the properties of surface coatings related to these techniques and their durability are supposedly restricted.\" - \\'Biomimetic samples: texture, structure and optical behavior\\' - \"Figure_ shows the biomimetic samples when coated on a glass substrate or as a free-standing film. The silver bands appear as black because the film (coated on a glass substrate) is placed on a black substrate and transmits light. It is semitransparent: indeed, a cholesteric film reflects at most 50% of unpolarized ambient light (as explained previously). The perception of colors in the macroscopic images is highly dependent on the ambient lighting conditions and on the viewing angle. This is the reason why, in Fig._, some silver bands of the beetle partially appear as black: the cuticle is curved, and the light reflected by the specular mirrors illuminated by the flash on part of the curved surface is not collected by the camera. On the other hand, all of the green bands are seen as green because the light reflected by the green bands is diffusely reflected. This is a consequence of the existence of an array of polygonal cells in which the orientation of the helical axis is spatially variable. Magnified views of a set of green and silver bands for which the whitish aspect of the silver band is more visible are given in Supplementary Fig._. Additional comments regarding the differences in the perceived colors, from the biological to the biomimetic material, will be given below in the discussion of the optical spectra.\" - \\'Fig. 3: Features of the biomimetic samples.a Biomimetic samples on a glass substrate and as a free-standing film. b Optical micrograph of the interfacial region between silver (top) and green (bottom) bands (reflection mode, unpolarized light). c TEM image of transverse structures at the interface between green and silver bands and close to the top surface. White dots underline the stripes of the fingerprint texture. The transition between curved and straight stripes is observed to be very gentle. d Reflectance spectrum of green and silver bands. e Transmittance spectrum of green and silver bands with the same wavelength scale as in d. Inset: the same spectrum displayed with the full operating wavelength band. f Multicriterion comparative table of features of the cuticle and biomimetic sample. The mentions in green (resp. red) report a similarity (resp. discrepancy). The mention in orange corresponds to an in-between situation.\\' - Biomimetic samples consist of a single piece that does not include any foreign material. By analogy, solidification by thermal quenching of the viscous material, at the end of the ultimate fifth stage, plays the role (transition from a soft to an hard material) of cuticle sclerotization after the cuticle has expanded to its final size and shape. - \"The optical micrograph of the cuticle (Fig._) shows a well-defined boundary between bands, which is more pronounced than in the biological material (Fig._ ). At the sub-micrometre scale, TEM investigations of cross sections (Fig._) show that the transition between bands is very gentle in the case of the biomimetic material. This set of observations validates the continuity of the transverse structures and the role of the orientation of the helical axisu2014from tilted to perpendicularu2014in the macroscopic color differences. The texture periodicityu2014 related to the helical half-pitchu2014as a function of the film depth is determined from cross-sectional TEM images (Supplementary Fig._). The pitch ranges in the biologicalu2014ca. 280u2013900u2009nm (Supplementary Fig._)u2014and biomimeticu2014ca. 300u20131320u2009nm (Supplementary Fig._)u2014materials are comparable in their orders of magnitude.\" - \"The reflection spectra ascribed to the biomimetic silver and green bands (Fig._ ) have comparable shapes. Both bands behave as broadband reflectors from ~500 to 850u2009nm. This broad wavelength range is comparable to that of the cuticular silver band in Fig._. Some differences in reflectance between the biomimetic silver and green bands are apparent: the spectrum ascribed to the silver band exhibits enhanced reflectance from 500 to 700u2009nm because here, the green band continues to exhibit diffuse reflection, whereas the silver band behaves as a specular mirror. However, the reflectance of the biomimetic green band is much more important than that of the cuticular green band. A green peak at 570u2009nm is observed for the biomimetic green band, whereas the corresponding peak is located at 520u2009nm for the cuticular band. This distinction contributes to the difference in the perceived green reflection color between the biological material and the biomimetic one. Other reasons, of a conceptual nature, may also be invoked to explain why the reflection colors and optical textures of the biomimetic samples are much less vibrant than those of C. gloriosa. First, the chemical compositions of the biomimetic and biological materials are very different. In some beetles, the presence of uric acid in the chitin matrix plays a major role in imparting color by enhancing the birefringence and reflectivity. Second, the thin and transparent wax layer may play a role in the perception of colors. It may scatter light and contribute to the shiny appearance of some beetles. In the case of the cuticle of Chrysina boucardi, the overall chromatic effect has been described as deeper in color because the cuticle is covered with wax, a situation that was noted to be evocative of the metal flake paint used on automobiles.\" - \"While the opacity of the cuticle prevents the measurement of light transmission, the transmission spectra of the biomimetic sample show a broad bandgap from 500 to 1000u2009nm (inset in Fig._). The fact that the plateau begins at 500u2009 nm is consistent with the sudden increase in reflectance beginning at ~500u2009 nm in Fig._. The transmittance spectra for both bands are very similar, providing further confirmation that the difference in reflection colors is due to the dual presence of variable and fixed orientations of the helical axis and not to significant pitch variations in the bulk of the material. This broad band is objectified by the pitch gradient shown in the TEM micrograph of a cross section and the related measurements, which give a pitch varying from 300 to 1320u2009nm (Supplementary Fig._). By taking nu2009=u20091.5 for the value of the average refractive index of the polysiloxane film and by using the Bragg law u03BB0u2009=u2009np, at normal incidence, we obtain a wavelength band of ~450u20131980u2009nm. The wavelengths at the left edge of the Bragg band are comparable: 500u2009nm, as displayed in Fig._, vs. 450u2009nm, as calculated. A wavelength discrepancy appears at the right edge: 1000u2009nm vs. 1980 nm. We assume that the number of helix turns ascribed to the twist from 666u2009 nm (1000u2009nm:1.5) to 1320u2009nm (1980u2009nm:1.5) is too low to allow a contribution to the Bragg band from this part of the pitch gradient. The reflectance indeed depends on the number of pitches (i.e., the number of helix turns), which means that the twist needs to develop within a minimal film thickness to make possible the appearance of a Bragg peak that is detectable under standard spectrophotometry conditions. Our hypothesis is supported by the sudden jump in the half-pitch in the 333u2013660u2009nm range for a small sample thickness of ~6u2009u03BC m (vs. the total sample thickness of 40u2009u03BCm), as shown in Supplementary Fig._. This thickness cannot permit a sufficiently large number of helix turns and the detection of related Bragg reflections.\" - Multicriterion comparison - \"A criterion-by-criterion comparative table of the cuticle and biomimetic sample is given in Fig._. Eight of twelve criteria are rated as high. The four criteria learned from the physical study of the biological material and reported in Fig._ as essential to claim the biomimicry status of the sample include the monolayer status, 3D structural continuity, 2D color discontinuity, and transverse pitch gradient. The biomimetic sample is chemically a monocomponent, while the cuticle is a chitin protein composite. The compaction can be taken as high in the cuticle and as medium in the biomimetic sample because the combination of common physical properties occurs within thicknesses equal to ca. 15 and 40u2009u03BCm, respectively. Two characteristics are clearly different: reflection colors are vibrant in the cuticle and much less vibrant in the biomimetic sample, and polygonal cells are concave in the cuticle and convex in the biomimetic sample.\" - Biomimetic applications in optical communication - The twisted CLC patterns issued from the present design pave the way to many possibilities of practical uses. - \"They may inspire future applications in optics, in the field of non-specular properties such as deflection and lensing in geometric phase planar optics. The phase of light that is Bragg-reflected off the helical structure can be controlled over 0u20132u03C0 depending on the spatial phase of the helical structure. The manufacture of planar elements with arbitrary reflected wave fronts via orientation control is thus desired. This requires the helix phase to vary depending on position, which objective is reached with on-demand chiral patterns provided by the present design.\" - Insects and humans have common concerns in terms of energy regulation, optical communication and camouflage. Concerning the last two functions, the fabrication of biomimetic samples inspires us to design tags in the field of cryptography and encryption. Graphically and spectrally encoded tags have become appealing for carrying information about tagged products because of their high coding capacity and wide applicability. Incorporating anti-counterfeiting tags with physical unclonable functions (PUFs) into objects is a promising solution for their authentication. Ongoing research hints at future LC-based technologies that may benefit from the peculiar behavior of CLCs subject to curvature, as reported for the case of cholesteric shells and droplets with the objective of generating unclonable patterns. - \"We provide two physical methods in relation to the size of the pattern to code and the technical solution to be implemented: the objective consists of encoding and hiding a signature (taggant) with different colors at different scales, from the millimetre to micrometre scale (Fig._), with an optical readout visible to the naked eye (Fig._) or by optical microscopy (Fig._).\" - \\'Fig. 4: Practical uses in visual cryptography.Multiscale optical tags with taggants from the a millimetre to b micrometre scale.\\' - \"For millimetre-size taggants (Fig._), the free surface of the green film is signed by the user with a pencil dipped into a polymer (PVA) solution. After water evaporation, the signature is invisible to the naked eye (Supplementary Fig._) and difficult to discern with the help of a microscope since the polymer film is thin, isotropic and transparent. It is revealed after thermal treatment (120u2009xB0C for 1u2009h) as the consequence of the nucleation and growth of the polygonal texture in the polymer-free parts of the film. The difference between the planar (u201CJoeu201D signature) and polygonal textures leads to color differences. Micrometre-size taggants (Fig._) can be formed by fixing a copper grid to the film and depositing a polymer solution on the gridu2013 film system. The grid is then removed, and the film is annealed. Grid patterns of taggants with geometrical features at different scales, from several tens of micrometres to a few micrometres, are revealed after thermal treatment and can be achieved by using different spin-coating conditions to deposit the polymer solution (see Methods), or grids with different geometrical features. Glass-forming CLCs are easily malleable, and their structure can be locked in solid paint after quenching. Solidified hybrid textures can be formed on a support that can be safely applied on an object by printing it on a plastic film or embedding it in a resin. Under illumination, the tags reveal patterns that, according to the taggant size, are visible to the naked eye, without or with the use of polarizers, or detectable by optical microscopy. The hybrid planar-polygonal texture provides the possibility of forming an image (the PUF tag) when it is embedded in a polygon-textured matrix, producing a specific optical response (the PUF pattern), which has to be recorded in a list of PUF keys (a database). An assembly of cells carries an additional level of PUF since the relative positions, sizes and optical patterns of cells are unique, not clonable and unpredictable. C. gloriosa may display information and hide in the background. Therefore, the comparable role of artificial tags and biological patterns brings an additional criterion to the biomimetic nature of the present biomimetic samples in terms of physical functions.\" - By using CLC oligomers we have engineered a single-layer material mimicking the striped cuticle of the scarab beetle C. gloriosa. Textural, structural and color properties were reproduced at several length scales. The functionalities of both materials were relevant to the topics of optical communication and camouflage. A multicriterion comparison revealed a very high level of biomimicry. The double versatility of colors and patterns was achieved during a simple and single stage based on a thermally driven self-organization process. The fabrication of biomimetic samples has inspired us to design proof-of-concept prototypes of optical tags in the field of cryptography and encryption. The design strategy involves an optimal use of resources in the spirit of eco-design, and a high versatility of chiral patterns unreached by the current manufacturing techniques.- - Biological samples - A male specimen was collected at Madera Canyon (Santa Cruz County, Arizona, USA) in July 2014. The cuticle samples including green and silver bands correspond to the elytra and were removed with a razor blade. - Synthetic materials - \"LC vitrifiable side-chain oligomers with a siloxane backbone were obtained from Wacker-Chemie GmbH, Munich, Germany (Supplementary Fig._).\" - Alignment and sacrificial layers used in the manufacture of cholesteric films - \"A PVA layer formed on the glass substrates supporting the cholesteric films was used as a promoter of the Grandjean planar texture or as a sacrificial layer to obtain free-standing films after the sample was placed in water. PVA as a powder (P8136, Sigma-Aldrich) was dissolved at 4 wt.% in distilled water. The aqueous solution was deposited on the substrate using a spin coater (Ossila Ltd.). The thickness of the dried film was ~2u2009u03BCm. A single sequence of spin-coating consisted of the deposition of 1.30u2009ml of solution during a program consisting of three sequences with the following speed rates and durations: 5000u2009rpm@20u2009su20144000u2009rpm@40u2009su20143000u2009rpm@60u2009 s. This operation was repeated three times.\" - Preparation of individual films for the biomimetic samples - \"Two distinct green-reflecting (SG) and infrared-reflecting (IR) samples were prepared on a heating stage at 120u2009xB0C. SG was used as provided by the supplier. IR was a blend made with 40 wt.% of a red-reflecting compound (SR) and 60 wt.% of a nematic compound (SN). A 50-u03BCm-thick film of SG was coated with a razor blade on a 150-u03BCm-thick cover plate. The sample was then quenched at room temperature (RT). The mean reflection wavelength of the film was 520u2009 nm (Supplementary Fig._). A 20-u03BCm-thick IR film was sandwiched between a 1-mm-thick glass plate and a 150-u03BCm-thick cover plate separated by 20-u03BC m-thick spacers. The sample was then quenched at RT. The mean reflection wavelength of the film was 1600 nm (Supplementary Fig._). The cover plate was sharply removed from the film with the aid of pressurized air to avoid the formation of scratches at the surface of the film. That the free surface of the film remained smooth after the operation was checked by optical microscopy. When the realization of free-standing samples was the goal, the glass plate that supported the IR film was previously covered with an 80-u03BCm-thick PVA film made from a 30 wt.% solution. The sample was then placed in a vial of distilled water. It detached from the plate in <1u2009h at RT. The PVA bands on the top surface also dissolved in water. The sample thus consisted of a monolithic piece of glassy polymer that did not include any foreign material.\" - Formation of bands in the biomimetic samples - \"To draw bands, an alternate solution to the use of a pencil dipped in PVA solution was to place 75-u03BCm-thick dicing tape (6033, Loadpoint), cut with dimensions close to those of the green bands of C. gloriosa, on the free surface of the bilayer film (Supplementary Fig._), at the end of stage nxB03 in Fig._. The surface of the sample with tape (typical size: 1.5u2009cm) was covered with 1.30u2009ml of PVA solution and spin-coated according to a program consisting of three sequences with the following speed rates and durations: 5000u2009 rpm@20u2009su20144000u2009rpm@40u2009su20143000u2009rpm@60u2009s. This operation was repeated three times. The tape was gently removed from the film with the help of tweezers before the PVA film was completely dry.\" - Preparation of functional films for optical cryptography at the micrometre scale - \"A copper grid (1GC 200 Pelco, Ted Pella) with square patterns (hole widthu2009 =u200990u2009u03BCm, bar width = 37 u03BCm) was fixed with adhesive tape on a 20-u03BCm-thick SG open film. A quantity of 1.30u2009ml of PVA solution was spin-coated on the free surface of the film at 900u2009rpm for 6u2009 s for a variable number of sequences, therefore obtaining a variable quantity of solution. The promotion of one type of pattern size at the detriment of other size depended on this process. The thicknesses of the thread patterns (related to the grid barsu2014the solution was trapped by capillarity below them) with mean sizes of 35, 6 and 2u2009u03BCm (Fig._) corresponded to 6, 4 and 3 sequences. The grid was then removed, and the sample was annealed at 120u2009xB0C for 1u2009h.\" - Inclusions of samples for ultramicrotomy - \"A small piece of material was embedded in EMBedu2013812 resin (Electron Microscopy Sciences), which was then cured at 62u2009xB0C for 2 days (biological sample) or at 37u2009xB0C for 11 days (biomimetic sample).\" - Preparation of TEM samples by ultramicrotomy - A diamond knife at ambient temperature was used to cut 80-nm-thick ultrathin slices with a Leica UCT ultramicrotome. The material was cut perpendicular to the film surface (cross sections). Slices were placed on single-slot Formvar-coated copper grids (GS2x1-C3, Gilder Grids Ltd.). - TEM conditions - \"A Jeol JEMu20131400 microscope operating at 80u2009kV equipped with a Gatan Orius SC1000B CCD camera was used.\" - Periodicity profiles from TEM images - Quantitative image analysis was performed with DigitalMicrograph software from Gatan. The intensities of the gray levels on a given surface of the images were analysed using the Profile option. - Optical micrographs - Textures were photographed with an Olympus BX51 stereomicroscope equipped with an Olympus DP73 digital camera. - Optical measurements - \"The reflectance and transmittance spectra of materials were collected at RT and normal incidence with unpolarized incident light. An HR2000CG-UVu2013NIR spectrophotometer from Ocean Optics (reflectance spectra) or a Cary 5000 UVu2013Visu2013NIR spectrometer from Varian (transmittance spectra) was used. Baselines were recorded with air (resp. a mirror) in the beam path for transmittance (resp. reflectance) measurements.\" - Reporting summary - \"Further information on research design is available in the_ linked to this article.\"- - \\' \\' - The data supporting the findings of this study are available within the paper and its Supplementary Information file and from the corresponding author upon reasonable request. - \\' \\'- - \"Ball, P. The Self-Made Tapestryu2014Pattern Formation in Nature (Oxford University Press, Oxford, 1999).Lee, D. Natures Paletteu2014The Science of Plant Colour (The University of Chicago Press, Chicago, 2007).Kolle, M. & Lee, S. Progress and opportunities in soft photonics and biologically inspired optics. Adv. Mater. 30, 1702669 (2018)._ Yang, Y. et al. Recent progress in biomimetic additive manufacturing technology: from materials to functional structures. Adv. Mater. 30, 1706539 (2018)._ Wegst, U. G. K., Bai, H., Saiz, E., Tomsia, A. P. & Ritchie, R. O. Bioinspired structural materials. Nat. Mater. 14, 23u201336 (2015)._ _ _ _ Filippov, A. E., Kovalev, A. & Gorb, S. N. Numerical simulation of the pattern formation of the springtail cuticle nanostructures. J. R. Soc. Interface 15, 20180217 (2018)._ _ _ Lenau, T. & Barfoed. M. Colours and metallic sheen in beetle shellsu2014a biomimetic search for material structuring principles causing light interference. Adv. Eng. Mat. 10, 299u2013314 (2008)._ de Gennes, P. G. & Prost, J. (eds) in_The Physics of Liquid Crystals 263u2013329 (Oxford University Press, Oxford, 1993).Snell-Rood, E. Bring biologists into biomimetics. Nature 529, 277u2013278 (2016)._ _ _ _ Kaiser, N. & Pulker, H. K. Optical Interference Coatings (Springer, New York, 2003).Zhu, Y., Zhang, W. & Zhang, D. Fabrication of sensor materials inspired by butterfly wings. Adv. Mater. Technol. 2, 160020 (2017)._ Matranga, A. et al. Biomimetic reflectors fabricated using self-organising, self-aligning liquid crystal polymers. Adv. Mater. 25, 520u2013523 (2013)._ _ _ Fernandes, S. N. et al. Mind the microgap in iridescent cellulose nanocrystal films. Adv. Mater. 29, 1603560 (2017)._ Caveney, S. Cuticle reflectivity and optical activity in scarab beetles: the role of uric acid. Proc. R. Soc. Lond. B 178, 205u2013225 (1971)._ _ _ _ Seago, A. E., Brady, P., Vigneron, J.-P. & Schultz, T. D. Gold bugs and beyond: a review of iridescence and structural color mechanisms in beetles (Coleoptera). J. R. Soc. Interface 6, S165u2013 S184 (2009)._ _ Stavenga, D. K., Wilts, B. D., Leertouwer, H. L. & Hariyama, T. Polarized iridescence of the multilayered elytra of the Japanese jewel beetle, Chrysochroa fulgidissima. Philos. Trans. B 366, 709u2013 723 (2011)._ Yu, K., Fan, T., Lou, S. & Zhang, D. Biomimetic optical materials: integration of natures design for manipulation of light. Prog. Mater. Sci. 58, 825u2013873 (2013)._ Vignolini, S. et al. Pointillist structural color in Pollia fruit. Proc. Natl Acad. Sci. USA 109, 15712u201315715 (2012)._ _ _ _ Vignolini, S. et al. Structural color from helicoidal cell-wall architecture in fruits of Margarita nobilis. J. R. Soc. Interface 13, 20160645 (2016)._ _ _ Pace, A. Cholesteric liquid crystal-like structure of the cuticle of Plusiotis gloriosa. Science 176, 678u2013680 (1972)._ _ _ Sharma, V., Crne, M., Park, J. O. & Srinivasarao, M. Structural origin of circularly polarized iridescence in jeweled beetles. Science 325, 449u2013451 (2009)._ _ _ _ Fernandez del Rio, L., Arwin, H. & JxE4rrendahl, K. Polarizing properties and structural characteristics of the cuticle of the scarab beetle Chrysina gloriosa. Thin Solid Films 571, 410u2013415 (2014)._ _ _ Fernandez del Rio, L., Arwin, H. & JxE4rrendahl, K. K. Polarization of light reflected from Chrysina gloriosa under various illuminations. Mat. Today.: Proc. 1S, 172u2013176 (2014)._ Agez, G., Bayon, C. & Mitov, M. Multiwavelength micromirrors in the cuticle of scarab beetle Chrysina gloriosa. Acta Biomater. 48, 357u2013367 (2017)._ _ _ Bouchal, P. et al. Non-diffracting light in nature: anomalously reflected self-healing Bessel beams from jewel scarabs. APL Photon 4, 126102 (2019)._ _ Kelker, H. & Hatz, R. (eds) in Handbook of Liquid Crystals 293u2013337 (Verlag Chemie, Weinheim, 1980).HernxE1ndez-JimxE9nez, M. et al. Qualitative correlation between structural chirality through the cuticle of Chrysina aurigans scarabs and left-handed circular polarization of the reflected light. Opt. Mater. Express 4, 2632u20132645 (2014)._ _ Libby, E. et al. Light reflection by the cuticle of C. aurigans scarabs: a biological broadband reflector of left handed circularly polarized light. J. Opt. 16, 082001 (2014)._ _ Thomas, B., Seago, A. & Robacker, D. C. Reflections on golden scarabs. Am. Entomol. 53, 224u2013 230 (2007)._ Azofeifa, D. E. et al. A quantitative assessment approach of feasible optical mechanisms contributing to structural color of golden-like Chrysina aurigans scarab beetles. J. Quant. Spectrosc. Rad. Transf. 160, 63u201374 (2015)._ _ _ Kolle, M. et al. Mimicking the colorful wing scale structure of the Papilio blumei butterfly. Nat. Nanotech. 5, 511u2013515 (2010)._ _ _ Mark, E. & Lin, G. The Polysiloxanes (Oxford University Press, Oxford, 2015).Kreuzer F.-H. et al. in Organosilicon Chemistry III (eds Auner, N. & Weiss, J.) 566u2013 586 (Wiley-VCH, Weinheim, 1997).Bunning, T. J. & Kreuzer, F.-H. Cyclosiloxane-based liquid crystalline materials. Trends Polym. Sci. 3, 318u2013323 (1995)._ _ Eberle, H.-J., Miller, A. & Kreuzer, F.-H. Inverse angle dependence of the reflection colors of cholesteric polymeric liquid crystals mixed with pigments. Liq. Cryst. 5, 907u2013916 (1989)._ Tsai, M. L., Chen, S. H. & Jacobs, S. D. Optical notch filter using thermotropic liquid crystalline polymers. Appl. Phys. Lett. 54, 2395 (1989)._ _ _ Mitov, M., Binet, C., Boudet, A. & Bourgerette, C. Glassy cholesteric broadband reflectors with a pitch gradient: material design, optical properties and microstructure. Mol. Cryst. Liq. Cryst. 358, 209u2013223 (2001)._ _ Etienne, S., David, L., Mitov, M., Sixou, P. & Ngai, K. L. Mechanical spectroscopy of side-chain liquid crystalline polymers in the glass transition range. Macromolecules 28, 5758u20135764 (1995)._ _ _ Korenic, E. M., Jacobs, S. D., Faris, S. M. & Li, L. Color gamut of cholesteric liquid-crystal films and flakes by standard colorimetry. Colour. Res. Appl. 23, 210u2013220 (1998)._ Trajkovska, A., Varshneya, R., Kosc, T. Z., Marshall, K. L. & Jacobs, S. D. Enhanced electro-optic behavior for shaped polymer cholesteric liquid-crystal flakes made using soft lithography. Adv. Funct. Mater. 15, 217u2013222 (2005)._ Shibaev, P. V. et al. Color changing and lasing stretchable cholesteric films. Opt. Express 16, 2965u20132970 (2008)._ _ _ _ Shibaev, P. V. in Liquid Crystal Microlasers (eds Blinov, L. M. & Bartolino, R.) 127u2013140 (Transworld Research Network, Kerala, 2010).Mitov, M., Portet, C., Bourgerette, C., Snoeck, E. & Verelst, M. Long range structuring of nanoparticles by mimetism with a cholesteric liquid crystal. Nat. Mater. 1, 229u2013231 (2002)._ _ _ _ Bitar, R., Agez, G. & Mitov, M. Cholesteric liquid crystal self-organization of gold nanoparticles. Soft Matter 7, 8198u2013 8206 (2011)._ _ _ Neradovskiy, M., Scarangella, A., Jullien, A. & Mitov, M. Dispersion of 20 fs pulses through band edges of cholesteric liquid crystals. Opt. Express 27, 021794 (2019)._ _ Bouligand, Y. Recherches sur les textures des xE9tats mxE9somorphesu2014 Les champ polygonaux dans les cholestxE9riques. J. Phys. 33, 715u2013736 (1972)._ _ Saupe, A. Disclinations and properties of the directorfield in nematic and cholesteric liquid crystals. Mol. Cryst. Liq. Cryst. 21, 211u2013 238 (1973)._ _ Bunning, T. J. et al. Cholesteric liquid crystals: image contrast in the TEM. Liq. Cryst. 16, 769u2013781 (1994)._ _ Meister, R., Dumoulin, H., HallxE9, M.-A. & Pieranski, P. The anchoring of a cholesteric liquid crystal at the free surface. J. Phys. II 6, 827u2013844 (1996)._ _ Meister, R., HallxE9 , M.-A., Dumoulin, H. & Pieranski, P. Structure of the cholesteric focal conic domains at the free surface. Phys. Rev. E 54, 3771u20133782 (1996)._ _ _ Bobrovsky, A., Mochalov, K., Chistyakov, A., Oleinikov, V. & Shibaev, V. Features of double-spiral u201Cvalley-hillsu201D surface topography formation in photochromic cholesteric oligomer-based films and their changes under polarized light action. Macromol. Chem. Phys. 213, 2639u2013 2646 (2012)._ _ Bobrovsky, A., Sinitsyna, O. V., Abramchuk, A., Yaminsky, I. & Shibaev, V. Atomic force microscopy study of surface topography of films of cholesteric oligomer- and polymer-based mixtures with photovariable helix pitch. Phys. Rev. E 87, 012503 (2013)._ _ Bayon, C., Agez, G. & Mitov., M. Wavelength-tunable light shaping with cholesteric liquid crystal microlenses. Lab Chip 14, 2063u20132071 (2014)._ _ _ Sinitsyna, O. V., Bobrovsky, A. Y., Meshkov, G. B., Yaminsky, I. V. & Shibaev, V. P. Surface relief changes in cholesteric cyclosiloxane oligomer films at different temperatures. J. Phys. Chem. B 119, 12708u201312713 (2015)._ _ _ Bayon, C., Agez, G. & Mitov Size-effect of oligomeric cholesteric liquid-crystal microlenses on the optical specifications. Opt. Lett. 40, 4763u20134766 (2015)._ _ _ _ Sinitsyna, O. V., Bobrovsky, A. Y., Meshkov, G. B., Yaminsky, I. V. & Shibaev, V. P. Direct observation of changes in focal conic domains of cholesteric films induced by ultraviolet irradiation. J. Phys. Chem. B 121, 5407u20135412 (2017)._ _ _ Jullien, A., Scarangella, A., Bortolozzo, U., Residori, S. & Mitov, M. Nanoscale hyperspectral imaging of of cholesteric liquid crystals with oblique helicity. Soft Matter 15, 3256u20133263 (2019)._ _ _ _ Cui, Y., Zola, R. S., Yang, Y.-C. & Yang, D.-K. Alignment layers with variable anchoring strengths from polyvinyl alcohol. J. Appl. Phys. 111, 063520 (2012)._ _ Kumar, C. et al. Replicating the complexity of natural surfaces: technique validation and applications for biomimetics, ecology and evolution. Philos. Trans. Roy. Soc. A-Math. Phys. Eng. Sci. 377, 20180265 (2019)._ _ _ Chigrinov, V. G., Kozenkov, V. M. & Kwok, H.-S. Photoalignment of Liquid Crystalline Materials: Physics and Applications_(John Wiley & Sons, Ltd., Chichester, 2008)._ Nie, Z. & Kumacheva, E. Patterning surfaces with functional polymers. Nat. Mater. 7, 277u2013290 (2008)._ _ _ _ van Oosten, C. L., Bastiaansen, C. W. M. & Broer, D. J. Printed artificial cilia from liquid-crystal network actuators modularly driven by light. Nat. Mater. 8, 677u2013682 (2009)._ _ _ Andersen, S. O. in Insect Molecular Biology and Biochemistry_Ch. 6 (ed. Gilbert, L. I.) (Academic Press, London, 2012)._Zografopoulos, D. C., Kriezis, E. E., Mitov, M. & Binet, C. Theoretical and experimental optical studies of cholesteric liquid crystal films with thermally induced pitch gradients. Phys. Rev. E 73, 061701 (2006)._ _ St. John, W. D., Fritz, W. J., Lu, Z. J. & Yang, D.-K. Bragg reflection from cholesteric liquid-crystals. Phys. Rev. E 51, 1191u20131198 (1995)._ _ Kobashi, J., Yoshida, H. & Ozaki, M. Planar optics with patterned chiral liquid crystals. Nat. Photon 10, 389u2013392 (2016)._ _ _ Chen, P. et al. Chirality invertible superstructure mediated active planar optics. Nat. Commun. 10, 2518 (2019)._ _ _ _ Ma, L.-L. et al. Light-activated liquid crystalline hierarchical architecture toward photonics. Adv. Optical Mater. 7, 1900393 (2019)._ Mitov, M. Cholesteric liquid crystals in living matter. Soft Matter 13, 4176u20134209 (2017)._ _ _ _ Pappu, R., Recht, B., Taylor, J. & Gershenfeld, N. Physical one-way functions. Science 297, 2026u20132030 (2002)._ _ _ _ Arppe, R. & SxF8rensen, T. J. Physical unclonable functions generated through chemical methods for anti-counterfeiting. Nat. Rev. Chem. 1, 0031 (2017)._ _ Geng, Y. et al. High-fidelity spherical cholesteric liquid crystal Bragg reflectors generating unclonable patterns for secure authentication. Sci. Rep. 6, 26840 (2016)._ _ _ _ _ Lenzini, G. et al. Security in the shell: an optical physical unclonable function made of shells of cholesteric liquid crystals. In Proc. 2017 IEEE Workshop on Information Forensics and Security (WIFS)_1u20136 (eds_Furon, T. & Troncoso, C.) (IEEE, 2017).Schwartz, M. et al. Cholesteric liquid crystal shells as enabling material for information-rich design and architecture. Adv. Mater. 30, 1707382 (2018)._ Zhang, Y., Paris, O., Terrill, N. J. & Gupta, H. S. Uncovering three-dimensional gradients in fibrillar orientation in an impact-resistant biological armour. Sci. Rep. 6, 26249 (2016)._ _ _ _ _ Mitov, M., Soldan, V. & Balor, S. Observation of an anisotropic texture inside the wax layer of insect. Arthropod Struct. Dev. 47, 622u2013626 (2018)._ _ Blomquist, G. J. & BagnxE8res, A.-G. Insect Hydrocarbonsu2014 Biology, Biochemistry and Chemical Ecology_(Cambridge University Press, Cambridge, 2010)._ \"- - \"M.M. thanks the Agence Nationale de la Recherche (France) for financial support (Grant: ANR-17-CE30u20130025) and Dr. E. Hanelt from Wacker-Chemie GmbH (Munich, Germany) for providing him with oligomers.\"- - Authors and Affiliations - \"Centre dElaboration de MatxE9riaux et dEtudes Structurales, CEMES, CNRS, Toulouse, FranceAdriana Scarangella_&_Michel MitovCentre de Biologie IntxE9grative, CBI, Microscopie Electronique IntxE9grative, METi, CNRS, University of Toulouse, Toulouse, FranceVanessa Soldan\" - \"AuthorsAdriana ScarangellaYou can also search for this author in _Vanessa SoldanYou can also search for this author in _Michel MitovYou can also search for this author in _\" - Contributions - M.M. led the conceptual development of the research, oversaw data collection, performed the analysis of results, organized the contents of the manuscript, prepared the figures and wrote the manuscript. A.S. contributed to the conceptual development of samples, carried out the manufacture of biomimetic samples along with their optical characterization, and participated in the analysis of results. V.S. carried out the conceptual development of operations related to the TEM investigations, cut the inclusions by ultramicrotomy and carried out TEM observations. M.M. prepared the inclusions for TEM observation and carried out the conception and manufacture of samples for cryptography. All authors revised the manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - Peer review information Nature Communications thanks Huai Yang, Jan Lagerwall and Seungwoo Lee for their contribution to the peer review of this work. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons license: . Finally, we would like to thank Dr. Bethany Moore for helpful conversations and insights.\\'- - Authors and Affiliations - \"Department of Biomedical Engineering, University of Michigan, Ann Arbor, MI, USAAaron H. Morris,_Kevin R. Hughes,_Robert S. Oakes,_Michelle M. Cai_&_ Lonnie D. SheaDepartment of Microbiologyu2013Immunology and Interdepartmental Immunobiology Center, Northwestern University Feinberg School of Medicine, Chicago, IL, USAStephen D. MillerDepartment of Neurology, University of Michigan Medical School, Ann Arbor, MI, USADavid N. IraniDepartment of Chemical Engineering, University of Michigan, Ann Arbor, MI, USALonnie D. Shea\" - \"AuthorsAaron H. MorrisYou can also search for this author in _Kevin R. HughesYou can also search for this author in _Robert S. OakesYou can also search for this author in _Michelle M. CaiYou can also search for this author in _Stephen D. MillerYou can also search for this author in _David N. IraniYou can also search for this author in _Lonnie D. SheaYou can also search for this author in _\" - Contributions - A.H.M., K.R.H., R.S.O., and L.D.S. conceived aspects of project. A.H.M., K.R.H., R.S.O., and M.M.C. performed the experiments and analyzed the data. A.H.M., K.R.H., R.S.O., and L.D.S. designed the experiments. D.N.I., S.D.M., and L.D.S. helped analyze and interpret the data and contributed to the supervision of the project. A.H.M., K.R.H., and R.S.O. visualized the data and generated the display items. A.H.M., K.R.H., and L.D.S. drafted the original paper. A.H.M., K.R.H., R.S.O., M.M.C., D.N.I., S.D.M., and L.D.S. edited and provided input into all aspects of the paper. A.H.M., S.D.M., and L.D.S. acquired funding to support this project. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - \"This paper uses nanoparticles for which L.D.S. and S.D.M. have an interest in Cour Pharmaceutical, who have licensed the technology. L.D.S., A.H.M., R.S.O., and K.R.H. are co-inventors on pending patent applications related to engineered immunological niches. The authors declare no further_competing interests.\" - \\' \\'- - Peer review information Nature Communications thanks Robert Weissert and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Peer reviewer reports are available. - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons license: .\\' - \"Likewise, beam steering must also incorporate cases in which a direct line-of-sight path is not available. As discussed above, an IRS could be used to realize a reflected path thereby increasing coverage and avoiding blockage. However, building the device is not enough; in order to function properly, the networks control plane must discover and configure this path, including properties of the IRS itself: for example, if the IRS provides a non-specular reflection, it must know the targeted incoming and outgoing angles. Since a network may alternate serving users in time, the IRS would need to reconfigure not only due to user mobility, but also according to which users are transmitting and receiving. Beam steering devices must also consider the new wavefronts described in Section 3. For example, in typical demonstrations of beams such as OAM, the transmitter, and receiver are manually aligned and are typically placed broadside. To employ such wavefronts in a mobile network will require adaptation not only for location, but also for the relative orientations of the transmitter and receiver when they are not ideally oriented.\" - \"As noted above, the aforementioned techniques based on the idea of an IRS have previously been considered for use at lower frequencies, but their implementation takes on new urgency at these higher frequencies. In addition, there are some approaches which have been more widely employed at lower frequencies, and which can also offer valuable capabilities in the THz range. One good illustration is the assessment of angle-of-arrival for a mobile user via cooperative estimation of spatial correlations among multiple antennas, for example in a massive MIMO architecture. This possibility has recently been considered by Peng et al. in the context of a THz network. Such legacy control-plane methods can play an important role, but will in general need some rethinking in view of the highly directional nature of transmissions in these networks, as well as the possibility (discussed above)_that the user could be in the near field of the array.\" - \"Todays wireless networks provide multi-user capabilities, in which an access point or base station transmits to (or receives from) multiple users simultaneously in order to increase aggregate data rate and decrease latency. Realizing this capability with THz frequencies will require two new advances. First, waveforms and modulation formats must be designed to support simultaneous transmission, incorporating that users will not be co-located and will be using directional transmissions. In some downlink cases, spatial separation of receivers combined with narrow beams may provide a simple starting point. Yet, for the uplink, and even for the downlink when users are close together, interference and co-stream management must be carefully controlled. Second, even if a network has the physical capability to realize a multi-user transmission, control plane mechanisms are needed to coordinate the transmission. Namely, the control plane must identify and localize the users, determine the appropriate spectrum and modulations to use, trigger the transmission at the correct time, and so on. In some cases, these functions are sufficiently similar to those of existing networks that comparable methods can be used; in other cases, entirely new protocols will need to be developed. For example, while traditionally medium access control protocols are driven by the transmitter, an alternative procedure based on receiver-initiated link synchronization, in which a receiver periodically polls potential transmitters as its antenna sweeps / scans the space, can increase the reliability and throughput and reduce latency.\"- - \"The many challenges discussed in this article have inspired a great deal of research over the last few years (only a small fraction of which could be included here). One challenge, not discussed above, involves the potential interference of wireless signals at these frequencies with existing (for the most part, passive) users involved in earth sensing or radio astronomy. Numerous research communities employ highly sensitive receivers to harvest information about the status of our atmosphere and the molecular composition of astronomical objects. It is critical that any active communication services that exploit frequencies above 100u2009GHz must be designed to avoid interference with these important existing communities. Of course, because of the higher atmospheric attenuation and the high gain of transmitting antennas, issues of sharing and interference may be quite different at these high frequencies. More research is required, for example, to establish the limits for interference, or to demonstrate antenna configurations whose side lobes are designed to avoid interference with overhead satellites.\" - \"Unsurprisingly, the daunting nature of the technical challenges has also inspired some skepticism from some researchers in the field. A few have noted that R&D expenditures in THz systems from many of the major telecommunications companies remain only a small fraction of their total R&D budget. Of course, this is not surprising, since the market for these systems also remains tiny. At this juncture, one should not expect massive private sector investment in a technology that is probably at least a decade away. Another oft-stated concern relates to the need for such systems. Twenty years ago, a common refrain was that nobody would ever require frequency bands above 10u2009GHz for consumer applications; ten years ago, it was 60u2009GHz; today, with the emergence of the first commercial backhaul devices operating in D band, the threshold has now moved to 140u2009 GHz. In fact, we choose to regard this moving target as an optimistic indicator of the rapid progress in the field. This progress is embodied in exciting recent publications, including breakthrough new link demonstrations and rapid advances in solid-state device technology.\" - Of course, there are valid reasons for concern; the challenges discussed in this article are indeed formidable. Nevertheless, we feel that the research results of the last few years have established that THz technologies are a promising foundation for future needs in wireless networks, which seem likely to exploit these frequencies for at least some of their key functions. While many open questions remain, there is at this point a clear and compelling motivation to pursue the goal of THz wireless.- - \"IEEE 802.11 Working Group. Enhancements for very high throughput for operation in license-exempt bands above 45 GHz. IEEE P802.11ay/D3.0 (2019).KxFCrner, T., Mittleman, D. M. & Nagatsuma, T. THz Communications: Paving the Way towards Wireless Tbps (Springer, 2021).Shafie, A. et al. Terahertz communications for 6G and beyond wireless networks: challenges, key advancements, and opportunities. IEEE Network (IEEE, 2022).Hirata, A. et al. 120-GHz-band wireless link technologies for outdoor 10-Gbit/s data transmission. IEEE Trans. Microw. Theory Tech. 60, 881u2013895 (2012)._ _ _ Petrov, V., KxFCrner, T. & Hosako, I. First standardization efforts for sub-terahertz band communications towards 6G. IEEE Commun. Mag. 58, 28u201333 (2020)._ _ Balanis, C. A. Antenna Theory: Analysis and Design 4th ed. (John Wiley & Sons, 2016).Kleine-Ostmann, T. & Nagatsuma, T. A review on terahertz communications research. J. Infrared Millim. Terahertz Waves 32, 143u2013171 (2011)._ _ Fu, Z. & Chen, R. T. Five-bit substrate guided wave true-time delay module working up to 2.4 Thz with a packing density of 2.5 lines/cm2 for phased array antenna applications. Opt. Eng. 6, 1838u20131844 (1998)._ _ _ International Telecommunication Union, Radiocommunication Sector (ITU-R), Recommendation P.676-11, Attenuation by atmospheric gases. (2016).OHara, J. F. & Grischkowsky, D. R. Comment on the veracity of the ITU-R recommendation for atmospheric attenuation at terahertz frequencies. IEEE Trans. Terahertz Sci. Technol. 8, 372u2013375 (2018)._ _ _ Yang, Y., Mandehgar, M. & Grischkowsky, D. THz-TDS characterization of the digital communication channels of the atmosphere and the enabled applications. J. Infrared Millim. Terahertz Waves 36, 97u2013129 (2015)._ _ Fang, Z., Hornbuckle, M. & Mittleman, D. M. Secure communication channels using atmosphere-limited line-of-sight terahertz links. IEEE Trans. Terahertz Sci. Technol. 12, 363u2013369 (2022)._ _ _ Kallfass, I. et al. 64 Gbit/s Transmission over 850 m Fixed Wireless Link at 240 GHz Carrier Frequency. J. Infrared Millim. Terahertz Waves 36, 221u2013 233 (2015)._ _ Wu, Q. et al. A 21 km 5 Gbps real time wireless communication system at 0.14 THz. In 42nd International Conference on Infrared, Millimeter, and Terahertz Waves (IRMMW-THz) (IEEE, 2017).Sen, P., Siles, J. V., Thawdar, N. & Jornet, J. M. Multi-kilometer multi-gigabit-per-second (sub) terahertz communications. Nat. Electron. (2022)._ _ Federici, J. F., Ma, J. & Moeller, L. Review of weather impact on outdoor terahertz wireless communication links. Nano Commun. Netw. 10, 13u201326 (2016)._ _ Sen, P. et al. Terahertz communications can work in rain and snow: Impact of adverse weather conditions on channels at 140 GHz. In 6th ACM Workshop on Millimeter-Wave and Terahertz Networks and Sensing Systems 13u201318 (ACM, 2022).Su, K., Moeller, L., Barat, R. B. & Federici, J. F. Experimental comparison of performance degradation from terahertz and infrared wireless links in fog. J. Opt. Soc. Am. A 29, 179u2013184 (2012)._ _ _ Han, C. et al. Terahertz wireless channels: a holistic survey on measurement, modeling, and analysis. IEEE Commun. Surv. Tutor. 24, 1670u2013 1707 (2022)._ _ De Beelde, B. et al. Material characterization and radio channel modeling at D-band frequencies. IEEE Access 9, 153528u2013 153539 (2021)._ _ Priebe, S., Kannicht, M., Jacob, M. & KxFCrner, T. Ultra broadband indoor channel measurements and calibrated ray tracing propagation modeling at THz frequencies. J. Commun. Netw. 15, 547u2013 558 (2013)._ _ Abbasi, N. A. et al. THz band channel measurements and statistical modeling for urban D2D environments. IEEE Trans. Wirel. Commun. (2022)._ _ Sheikh, F., El-Hadidy, M. & Kaiser, T. Terahertz band: Indoor ray-tracing channel model considering atmospheric attenuation. In IEEE International Symposium on Antennas and Propagation & USNC/URSI National Radio Science Meeting 1782-1783 (IEEE, 2015).Ma, J., Shrestha, R., Moeller, L. & Mittleman, D. M. Channel performance of indoor and outdoor terahertz wireless links. APL Photon 3, 051601 (2018)._ _ _ Jansen, C. et al. Diffuse scattering from rough surfaces in THz communication channels. IEEE Trans. Terahertz Sci. Tech. 1, 462u2013472 (2011)._ _ _ Ma, J., Shrestha, R., Zhang, W., Moeller, L. & Mittleman, D. M. Terahertz wireless links using diffuse scattering from rough surfaces. IEEE Trans. Terahertz Sci. Tech. 9, 463u2013470 (2019)._ _ _ _ Messenger, R., Strecker, K., Ekin, S. & OHara, J. F. Dispersion from diffuse reflectors and its effect on terahertz wireless communication performance. IEEE Trans. Terahertz Sci. Tech. 11, 695u2013703 (2021)._ _ _ _ Ma, J. et al. Security and eavesdropping in terahertz wireless links. Nature 563, 89u201393 (2018)._ _ _ _ _ Mei, Y., Ma, Y., Ma, J., Moeller, L. & Federici, J. F. Eavesdropping risk evaluation on terahertz wireless channels in atmospheric turbulence. IEEE Access 9, 101916u2013101923 (2021)._ _ Herold, C., Doeker, T., Eckhardt, J. M. & KxFCrner, T. Investigation of eavesdropping opportunities in a meeting room scenario for THz communications. In 16th European Conference on Antennas and Propagation (EuCAP) (IEEE, 2022).Ju, S. & Rappaport, T. S. Sub-terahertz spatial statistical MIMO channel model for urban microcells at 142 GHz. In IEEE Global Communications Conference (GLOBECOM) (IEEE, 2021).Toskala, A., Holma, H. & Nakamura, T. 5G Technology: 3GPP New Radio (John Wiley & Sons, 2020).Wu, Q., Zhang, S., Zheng, B., You, C. & Zhang, R. Intelligent reflecting surface-aided wireless communications: a tutorial. IEEE Trans. Commun. 69, 3313u20133351 (2021)._ _ _ Rouhi, K., Hosseininejad, S. E., Abadal, S., Khalily, M. & Tafazolli, R. Multi-channel near-field terahertz communications using reprogrammable graphene-based digital metasurface. J. Lightwave Technol. 39, 6893u20136907 (2021)._ _ _ _ Zhang, H. et al. Beam focusing for near-field multiuser MIMO communications. IEEE Trans. Wirel. Commun. 21, 7476u20137490 (2022)._ _ Jo, O., Kim, J.-J., Yoon, J., Choi, D. & Hong, W. Exploitation of dual-polarization diversity for 5G millimeter-wave MIMO beamforming systems. IEEE Trans. Antennas Propag. 65, 6646u20136655 (2017)._ _ _ Durnin, J., Miceli, J. J. & Eberly, J. H. Comparison of Bessel and Gaussian beams. Opt. Lett. 13, 79u201380 (1988)._ _ _ _ _ Efremidis, N. K., Chen, Z., Segev, M. & Christodoulides, D. N. Airy beams and accelerating waves: an overview of recent advances. Optica 6, 686u2013701 (2019)._ _ _ _ Zhou, H. et al. Utilizing multiplexing of structured THz beams carrying orbital-angular-momentum for high-capacity communications. Opt. Express 30, 25418u201325432 (2022)._ _ _ _ _ Edfors, O. & Johansson, A. J. Is orbital angular momentum (OAM) based radio communication an unexploited area? IEEE Trans. Antennas Propag. 60, 1126u20131131 (2012)._ _ _ _ _ Li, J.-S. & Chen, J.-Z. Multi-beam and multi-mode orbital angular momentum by utilizing a single metasurface. Opt. Express 29, 27332u201327339 (2021)._ _ _ _ Khan, M. I. W. et al. A 0.31-THz orbital-angular-momentum (OAM) wave transceiver in CMOS with bits-to-OAM mode mapping. IEEE J. Solid State Circuits 57, 1344u20131357 (2021)._ _ _ Venkatesh, S., Lu, X., Saeidi, H. & Sengupta, K. A high-speed programmable and scalable terahertz holographic metasurface based on tiled CMOS chips. Nat. Electron 3, 785u2013793 (2020)._ _ _ Monroe, N. M. et al. Electronic THz pencil beam forming and 2D steering for high angular-resolution operation: a 98xD798 unit, 265u2009GHz CMOS reflectarray with in-unit digital beam shaping and squint correction. In IEEE International Solid-State Circuits Conference (ISSCC) (2022).Shrestha, R., Guerboukha, H., Fang, Z., Knightly, E. & Mittleman, D. M. Jamming a terahertz wireless link. Nat. Commun. 13, 3045 (2022)._ _ _ _ _ _ Woo, J. et al. Physical-layer security for THz communications via orbital angular momentum waves. In IEEE Workshop on Signal Processing Systems (SiPS) (IEEE, 2022).Hadani, R. et al. Orthogonal time frequency space (OTFS) modulation for millimeter-wave communications systems. In International Microwave Symposium (IMS) 681u2013683 (IEEE, 2017).Grischkowsky, D., Keiding, S., Exter, M. V. & Fattinger, C. Far-infrared time-domain spectroscopy with terahertz beams of dielectrics and semiconductors. J. Opt. Soc. Am. B 7, 2006u20132015 (1990)._ _ _ _ Chen, P., Wang, Y. & Babakhani, A. A 4u2009ps amplitude reconfigurable impulse radiator with THz-TDS characterization method in 0.13 u03BCm SiGe BiCMOS. In IEEE MTT-S International Microwave Symposium (IMS) (IEEE, 2016).Jornet, J. M. & Akyildiz, I. F. Femtosecond-long pulse-based modulation for terahertz band communication in nanonetworks. IEEE Trans. Commun. 62, 1742u20131754 (2014)._ _ Bodet, D., Sen, P., Hossain, Z., Thawdar, N. & Jornet, J. M. Hierarchical bandwidth modulations for ultra-broadband communications in the terahertz band. IEEE Trans. Wireless Commun. (2022).Bosso, C. et al. Ultrabroadband spread spectrum techniques for secure dynamic spectrum sharing above 100u2009GHz between active and passive users. In International Symposium on Dynamic Spectrum Access Networks (DySPAN) 45u201352 (IEEE, 2021).Abu-Surra, S. et al. End-to-end 6u2009G terahertz wireless platform with adaptive transmit and receive beamforming. in IEEE International Conference on Communications Workshops (ICC Workshops) 897u2013903 (IEEE, 2022).Watts, C. M. et al. Terahertz compressive imaging with metamaterial spatial light modulators. Nat. Photon 8, 605u2013609 (2014)._ _ _ _ Sun, C., Chang, Q., Zhao, R., Wang, Y. & Wang, J. Terahertz imaging based on sparse MIMO array. In International Conference on Microwave and Millimeter Wave Technology (ICMMT) (IEEE, 2020).Singh, A., Andrello, M., Einarsson, E., Thawdar, N. & Jornet, J. M. Design and operation of a smart grapheneu2013metal hybrid reflectarray at THz frequencies. In 14th European Conference on Antennas and Propagation (EuCAP) (IEEE, 2020).Bandurin, D. A. et al. Resonant terahertz detection using graphene plasmons. Nat. Commun. 9, 5392 (2018)._ _ _ _ _ _ Tu, N. H. et al. Active spatial control of terahertz plasmons in graphene. Commun. Mater. 1, 7 (2020)._ _ Jornet, J. M. & Akyildiz, I. F. Graphene-based plasmonic nano-antenna for terahertz band communication in nanonetworks. IEEE J. Sel. Areas Commun. 31, 685u2013 694 (2013)._ _ Yeang, C.-P., Wornell, G. W. & Zheng, L. Oversampling transmit and receive antenna arrays. In International Conference on Acoustics, Speech and Signal Processing 2522u20132525 (IEEE, 2010).Jackson, D. R., Caloz, C. & Itoh, T. Leaky wave antennas. Proc. IEEE 100, 2194u2013 2206 (2012)._ _ Ma, J., Karl, N. J., Bretin, S., Ducournau, G. & Mittleman, D. M. Frequency-division multiplexer and demultiplexer for terahertz wireless links. Nat. Commun. 8, 729 (2017)._ _ _ _ _ Matsumoto, H., Watanabe, I., Kasamatsu, A. & Monnai, Y. Integrated terahertz radar based on leaky-wave coherence tomography. Nat. Electron 3, 122u2013 129 (2020)._ _ Ghasempour, Y., Shrestha, R., Charous, A., Knightly, E. & Mittleman, D. M. Single-shot link discovery for terahertz wireless networks. Nat. Commun. 11, 2017 (2020)._ _ _ _ _ _ Li, R., Yan, H. & Cabric, D. Rainbow-link: beam-alignment-free and grant-free mmW multiple access using true-time-delay array. IEEE J. Sel. Areas Commun. 40, 1692u20131705 (2022)._ _ Nguyen, H. V., Abielmona, S., Rennings, A. & Caloz, C. Pencil-beam full-space scanning 2D CRLH leaky-wave antenna array. In International Symposium on Signals, Systems and Electronics 139u2013142 (IEEE, 2007).Kamoda, H., Iwasaki, T., Tsumochi, J., Kuki, T. & Hashimoto, O. 60-GHz electronically reconfigurable large reflectarray using single-bit phase shifters. IEEE Trans. Antennas Propag. 59, 2524u2013 2531 (2011)._ _ _ Zhao, Y. et al. High-speed efficient terahertz modulation based on tunable collective-individual state conversion within an active 3u2009nm two-dimensional electron gas metasurface. Nano Lett. 19, 7588u20137597 (2019)._ _ _ _ _ Shaikhanov, Z., Hassan, F., Guerboukha, H., Mittleman, D. & Knightly, E. Metasurface-in-the-middle attack: from theory to experiment. In 15th ACM Conference on Security and Privacy in Wireless and Mobile Networks (WiSec u201822) 257u2013267 (ACM, 2022).Kanhere, O. & Rappaport, T. S. Outdoor sub-THz position location and tracking using field measurements at 142u2009GHz. In IEEE International Conference on Communications (ICC) (IEEE, 2021).Zhao, M. et al. Through-wall human mesh recovery using radio signals. In IEEE/CVF International Conference on Computer Vision (ICCV) 10112u2013 10121 (IEEE, 2019).Lin, F. et al. Cardiac Scan: A non-contact and continuous heart-based user authentication system. In 23rd Annual International Conference on Mobile Computing and Networking (MobiCom u201817) 315u2013328 (ACM, 2017).Kanhere, O. & Rappaport, T. S. Position location for futuristic cellular communications: 5G and beyond. IEEE Commun. Mag. 59, 70u201375 (2021)._ _ Saeidi, H., Venkatesh, S., Lu, X. & Sengupta, K. THz prism: one shot simultaneous localization of multiple wireless nodes with leaky-wave THz antennas and transceivers in CMOS. IEEE J. Solid-State Circuits 56, 3840u20133854 (2021)._ _ _ Domae, B. W., Boljanovic, V., Li, R. & Cabric, D. Machine learning prediction for phase-less millimeter-wave beam tracking. In IEEE 23rd International Workshop on Signal Processing Advances in Wireless Communication (SPAWC) (IEEE, 2022). Peng, B., Guan, K. & KxFCrner, T. Cooperative dynamic angle of arrival estimation considering spaceu2013time correlations for terahertz communications. IEEE Trans. Wirel. Commun. 17, 6029u20136041 (2018)._ _ Xia, Q., Hossain, Z., Medley, M. & Jornet, J. M. Link-layer synchronization and medium access control protocol for terahertz-band communication networks. IEEE Trans. Mob. Comput. 20, 2u201318 (2021)._ _ Xing, Y. & Rappaport, T. S. Terahertz wireless communications: co-sharing for terrestrial and satellite systems above 100u2009GHz. IEEE Commun. Lett. 25, 3156u20133160 (2021)._ _ Singh, A. et al. A D-band radio-on-glass module for spectrally-efficient and low-cost wireless backhaul. In IEEE Radio Frequency Integrated Circuits Symposium (RFIC) 99u2013 102 (IEEE, 2020).KxFCrner, T. & Kawanishi, T. Demonstrating 300u2009GHz wireless backhaul links u2013 The ThoR approach. In 47th International Conference on Infrared, Millimeter and Terahertz Waves (IRMMW-THz) (IEEE, 2022).Maes, D. et al. UTC photodiodes on silicon nitride enabling 100 Gbit/s Terahertz links at 300u2009GHz. In European Conference on Optical Communication (ECOC) (IEEE, 2022).Rodwell, M. et al. Transistors for 100-300u2009GHz wireless. In IEEE 51st European Solid-State Device Research Conference (ESSDERC) (IEEE, 2021).Wang, C. & Rebeiz, G. A 2-channel 136-156u2009GHz dual down-conversion I/Q receiver with 30u2009dB gain and 9.5u2009dB NF using CMOS 22u2009nm FDSOI. In IEEE Radio Frequency Integrated Circuits Symposium (RFIC) (IEEE, 2021).Chen, Z., Choi, W. & O, K. K. 300-GHz double-balanced up-converter using asymmetric MOS varactors in 65-nm CMOS. IEEE J. Solid-State Circuits 57, 2336u20132347 (2022)._ _ _ Mehta, Y., Thomas, S. & Babakhani, A. A 140u2013 220-GHz low-noise amplifier with 6-dB minimum noise figure and 80-GHz bandwidth in 130-nm SiGe BiCMOS. IEEE Microwave Wireless Comp. Lett. 33, 200u2013203 (2022).Pang, X. et al. Bridging the terahertz gap: photonics-assisted free-space communications from the submillimeter-wave to the mid-infrared. J. Lightwave Technol. 40, 3149u20133162 (2022)._ _ _ \"- - The authors gratefully acknowledge the assistance of Kaushik Sengupta, Hichem Guerboukha, and Duschia Bodet in assembling figures for this manuscript. D.M.M. acknowledges funding support from the US National Science Foundation (grant numbers 1923733, 1954780, 2148132, and 2211616) and the Air Force Research Laboratory (award number FA8750-19-1-0500). E.W.K. acknowledges funding support from Cisco, Intel, the US National Science Foundation (grant numbers 1955075, 1923782, 1824529, and 2148132), and the Army Research Laboratory (grant W911NF-19-2-0269). J.M.J. acknowledges funding support from the US National Science Foundation (grant numbers 1955004, 2011411, and 2117814) and the Air Force Research Laboratory (award number FA8750-19-1-0200).- - Authors and Affiliations - Department of Electrical and Computer Engineering, Institute for the Wireless Internet of Things, Northeastern University, 360 Huntington Avenue, Boston, MA, 02115, USAJosep M. JornetDepartment of Electrical & Computer Engineering, Rice University, 6100 Main St., Houston, TX, 77005, USAEdward W. KnightlySchool of Engineering, Brown University, 184 Hope St., Providence, RI, 02912, USADaniel M. Mittleman - \"AuthorsJosep M. JornetYou can also search for this author in _Edward W. KnightlyYou can also search for this author in _Daniel M. MittlemanYou can also search for this author in _\" - Contributions - D.M.M., J. M.J., and E.W.K. contributed equally to the writing of this manuscript. - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks Robert Elschner, Shuai Nie and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. ',\n",
       " 'Creative Commons licenses. Naturally, we recognize that not all researchers will have access to the resources required for open-access publication, which is why authors can choose to publish their work without charge behind a subscription pay wall. We are sometimes asked whether the choice of publication license influences the decisions of the editors, and the answer is resolutely \\'no\\'u2014editors are blind to the author\\'s choice until the work is ready for acceptance.\"- - \"And what of quality? Only time will tell, but no doubt our authors, and those whose papers we have regretfully rejected, will attest to the rigour of peer review at Nature Communications. Our selection criteria, upheld by our Editorial Advisory Panel, together with decisions made after peer review result in publication of approximately 20% of submissionsu2014a number that suggests our aspirations of being a high-quality science journal with somewhat specialist appeal were well-grounded.\"- - \"Nature Communications cannot rest on its laurels, however. Submissions continue to growu2014more than 270 new manuscripts were submitted in March aloneu2014 and the number of published papers will increase accordingly. Our web technologies must therefore keep pace with this rapidly expanding archive. Indeed, since Nature Communications launched we have made our content available through mobile device applications to enable browsing on the go; launched a personalization tool on our homepage for users to customize their experience; and introduced an online commenting facility to encourage scientific dialogue.\"- - So what does the future hold for Nature Communications? We are interested in understanding what you, the scientific community, would like to see (or indeed hear), and we encourage you to use the commenting tool at the foot of the HTML version of this article to pass on your thoughts.- - \\'\\'- - ',\n",
       " 'Creative Commons. .\"- - This work was supported by funding from NSF (AGS1554998) and the National Geographic Society (NGS-39815) to J.L.O. and the Karst Waters Institute to C.B.d.W. J.H. acknowledges financial support by the Max Planck Graduate Center Mainz. We thank Mike Davies and Bruce Rogers of the Western Cave Conservancy for guidance in the field and Susan Petrie of the Peninsula Open Space Trust for cave access.- - Authors and Affiliations - \"Department of Chemistry, Johannes Gutenberg-University Mainz, Mainz, GermanyJulia Homann_&_Thorsten HoffmannDepartment of Earth and Environmental Sciences, Vanderbilt University, Nashville, TN, USAJessica L. Oster_&_Cameron B. de WetDepartment of Geography and Environmental Sciences, Northumbria University, Newcastle upon Tyne, UKSebastian F. M. Breitenbach\" - \"AuthorsJulia HomannYou can also search for this author in _Jessica L. OsterYou can also search for this author in _Cameron B. de WetYou can also search for this author in _Sebastian F. M. BreitenbachYou can also search for this author in _Thorsten HoffmannYou can also search for this author in _\" - Contributions - \"J.H., J.L.O., C.B.d.W., S.F.M.B., and T.H. designed the study. J.H. conducted the sample preparation and data analysis for levoglucosan and LOPs. C.B.d.W. and J.L.O. subsampled the stalagmite for analysis, constructed the age model, and performed the changepoint analysis on u03B4C. All co-authors participated in discussions, interpretation of the data, and writing of the manuscript.\" - Corresponding author - \"Correspondence to .\"- - \\' \\' - Competing interests - \\' \\' - The authors declare no competing interests. - \\' \\'- - \\' \\' - Peer review information - \\' \\' - Nature Communications thanks the anonymous reviewers for their contribution to the peer review of this work. - \\' \\'- - \"Publishers note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\"- - \\'\\'- - \\' \\' - \"Open Access This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the articles Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the articles Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. '}"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lis_texts=[i[\"Lisence\"] for i in raw_data_list]\n",
    "set(lis_texts)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "mi",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.3"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
